(window.webpackJsonp=window.webpackJsonp||[]).push([[147],{501:function(t,a,e){"use strict";e.r(a);var s=e(1),r=Object(s.a)({},(function(){var t=this,a=t.$createElement,e=t._self._c||a;return e("ContentSlotsDistributor",{attrs:{"slot-key":t.$parent.slotKey}},[e("p",[t._v("GitHub: "),e("a",{attrs:{href:"https://github.com/smorabit/hdWGCNA/",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://github.com/smorabit/hdWGCNA/"),e("OutboundLink")],1),e("br"),t._v("\nDoc："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"custom-block note"},[e("p",{staticClass:"custom-block-title"},[t._v("相关阅读")]),t._v(" "),e("ol",[e("li",[e("RouterLink",{attrs:{to:"/pages/ffdcc0/"}},[t._v("文献阅读 --- WGCNA：an R package for weighted correlation network analysis")])],1),t._v(" "),e("li",[e("RouterLink",{attrs:{to:"/pages/9ffdf7/"}},[t._v("WGCNA：an R package for weighted correlation network analysis")])],1),t._v(" "),e("li",[e("RouterLink",{attrs:{to:"/pages/25c02b/"}},[t._v("文献阅读 --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease")])],1),t._v(" "),e("li",[e("RouterLink",{attrs:{to:"/pages/a24249/"}},[t._v("文献阅读 --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data")])],1),t._v(" "),e("li",[e("RouterLink",{attrs:{to:"/pages/649303/"}},[t._v("hdWGCNA：high dimensional WGCNA")])],1)])]),t._v(" "),e("p",[e("strong",[t._v("作者介绍")])]),t._v(" "),e("table",[e("thead",[e("tr",[e("th",{staticStyle:{"text-align":"center"}},[t._v("Samuel Morabito")]),t._v(" "),e("th",{staticStyle:{"text-align":"center"}},[t._v("Vivek Swarup")])])]),t._v(" "),e("tbody",[e("tr",[e("td",{staticStyle:{"text-align":"center"}},[e("img",{attrs:{src:"https://smorabit.github.io/assets/img/sam_2022.jpg",width:"200"}})]),t._v(" "),e("td",{staticStyle:{"text-align":"center"}},[e("img",{attrs:{src:"https://ccbs.uci.edu/wp-content/uploads/sites/3/2019/09/Swarup_jpeg-130x160.jpeg",width:"200"}})])]),t._v(" "),e("tr",[e("td",{staticStyle:{"text-align":"center"}},[t._v("University of California Irvine(UCI)")]),t._v(" "),e("td",{staticStyle:{"text-align":"center"}},[t._v("University of California Irvine(UCI)")])]),t._v(" "),e("tr",[e("td",{staticStyle:{"text-align":"center"}},[e("a",{attrs:{href:"mailto:smorabit@uci.edu"}},[t._v("smorabit@uci.edu")])]),t._v(" "),e("td",{staticStyle:{"text-align":"center"}},[e("a",{attrs:{href:"mailto:vswarup@uci.edu"}},[t._v("vswarup@uci.edu")])])])])]),t._v(" "),e("h2",{attrs:{id:"hdwgcna-介绍"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#hdwgcna-介绍"}},[t._v("#")]),t._v(" hdWGCNA 介绍")]),t._v(" "),e("p",[t._v("hdWGCNA 是一个 R 包，用于在 scRNA-seq 或空间转录组学等高维转录组学数据中执行"),e("code",[t._v("加权基因共表达网络分析(WGCNA)")]),t._v("。hdWGCNA 是高度模块化的，可以构建跨多尺度细胞和空间层次结构的共表达网络。hdWGNCA 识别出相互关联的基因的稳健模块，并通过各种生物学知识源为这些模块提供解释。hdWGCNA 需要格式化为 "),e("a",{attrs:{href:"https://satijalab.org/seurat/index.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("Seurat"),e("OutboundLink")],1),t._v(" 对象的数据，这是单细胞数据最普遍的格式之一。")]),t._v(" "),e("p",[t._v("WGCNA 发表的相关文献:")]),t._v(" "),e("ul",[e("li",[e("a",{attrs:{href:"https://doi.org/10.1101/2022.09.22.509094",target:"_blank",rel:"noopener noreferrer"}},[t._v("Morabito et al. bioRxiv 2022"),e("OutboundLink")],1)]),t._v(" "),e("li",[e("a",{attrs:{href:"https://doi.org/10.1038/s41588-021-00894-z",target:"_blank",rel:"noopener noreferrer"}},[t._v("Morabito & Miyoshi et al. Nature Genetics 2021"),e("OutboundLink")],1)])]),t._v(" "),e("h2",{attrs:{id:"installation"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#installation"}},[t._v("#")]),t._v(" Installation")]),t._v(" "),e("p",[t._v("建议为 hdWGCNA 创建一个 R conda 环境")]),t._v(" "),e("div",{staticClass:"language-shell line-numbers-mode"},[e("pre",{pre:!0,attrs:{class:"language-shell"}},[e("code",[e("span",{pre:!0,attrs:{class:"token comment"}},[t._v("# create new conda environment for R")]),t._v("\nconda create "),e("span",{pre:!0,attrs:{class:"token parameter variable"}},[t._v("-n")]),t._v(" hdWGCNA "),e("span",{pre:!0,attrs:{class:"token parameter variable"}},[t._v("-c")]),t._v(" conda-forge r-base r-essentials\n\n"),e("span",{pre:!0,attrs:{class:"token comment"}},[t._v("# activate conda environment")]),t._v("\nconda activate hdWGCNA\n")])]),t._v(" "),e("div",{staticClass:"line-numbers-wrapper"},[e("span",{staticClass:"line-number"},[t._v("1")]),e("br"),e("span",{staticClass:"line-number"},[t._v("2")]),e("br"),e("span",{staticClass:"line-number"},[t._v("3")]),e("br"),e("span",{staticClass:"line-number"},[t._v("4")]),e("br"),e("span",{staticClass:"line-number"},[t._v("5")]),e("br")])]),e("p",[t._v("接下来，打开 R 并安装所需的依赖项：")]),t._v(" "),e("div",{staticClass:"language-R line-numbers-mode"},[e("pre",{pre:!0,attrs:{class:"language-r"}},[e("code",[e("span",{pre:!0,attrs:{class:"token comment"}},[t._v("# install BiocManager")]),t._v("\ninstall.packages"),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v("(")]),e("span",{pre:!0,attrs:{class:"token string"}},[t._v('"BiocManager"')]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(")")]),t._v("\n\n"),e("span",{pre:!0,attrs:{class:"token comment"}},[t._v("# install Bioconductor core packages")]),t._v("\nBiocManager"),e("span",{pre:!0,attrs:{class:"token operator"}},[t._v("::")]),t._v("install"),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v("(")]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(")")]),t._v("\n\n"),e("span",{pre:!0,attrs:{class:"token comment"}},[t._v("# install additional packages:")]),t._v("\ninstall.packages"),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v("(")]),t._v("c"),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v("(")]),e("span",{pre:!0,attrs:{class:"token string"}},[t._v('"Seurat"')]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(",")]),t._v(" "),e("span",{pre:!0,attrs:{class:"token string"}},[t._v('"WGCNA"')]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(",")]),t._v(" "),e("span",{pre:!0,attrs:{class:"token string"}},[t._v('"igraph"')]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(",")]),t._v(" "),e("span",{pre:!0,attrs:{class:"token string"}},[t._v('"devtools"')]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(")")]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(")")]),t._v("\n")])]),t._v(" "),e("div",{staticClass:"line-numbers-wrapper"},[e("span",{staticClass:"line-number"},[t._v("1")]),e("br"),e("span",{staticClass:"line-number"},[t._v("2")]),e("br"),e("span",{staticClass:"line-number"},[t._v("3")]),e("br"),e("span",{staticClass:"line-number"},[t._v("4")]),e("br"),e("span",{staticClass:"line-number"},[t._v("5")]),e("br"),e("span",{staticClass:"line-number"},[t._v("6")]),e("br"),e("span",{staticClass:"line-number"},[t._v("7")]),e("br"),e("span",{staticClass:"line-number"},[t._v("8")]),e("br")])]),e("p",[t._v("然后，使用 devtools 安装 hdWGCNA 软件包")]),t._v(" "),e("div",{staticClass:"language-R line-numbers-mode"},[e("pre",{pre:!0,attrs:{class:"language-r"}},[e("code",[t._v("devtools"),e("span",{pre:!0,attrs:{class:"token operator"}},[t._v("::")]),t._v("install_github"),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v("(")]),e("span",{pre:!0,attrs:{class:"token string"}},[t._v("'smorabit/hdWGCNA'")]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(",")]),t._v(" ref"),e("span",{pre:!0,attrs:{class:"token operator"}},[t._v("=")]),e("span",{pre:!0,attrs:{class:"token string"}},[t._v("'dev'")]),e("span",{pre:!0,attrs:{class:"token punctuation"}},[t._v(")")]),t._v("\n")])]),t._v(" "),e("div",{staticClass:"line-numbers-wrapper"},[e("span",{staticClass:"line-number"},[t._v("1")]),e("br")])]),e("h2",{attrs:{id:"hdwgcna-vignettes"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#hdwgcna-vignettes"}},[t._v("#")]),t._v(" hdWGCNA Vignettes")]),t._v(" "),e("h3",{attrs:{id:"_1-co-expression-network-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_1-co-expression-network-analysis"}},[t._v("#")]),t._v(" 1 Co-expression network analysis")]),t._v(" "),e("p",[t._v("这些教程涵盖了在单细胞转录组学数据中执行共表达网络分析的要点，以及可视化关键结果。")]),t._v(" "),e("h4",{attrs:{id:"_1-1-hdwgcna-in-single-cell-data"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_1-1-hdwgcna-in-single-cell-data"}},[t._v("#")]),t._v(" 1.1 hdWGCNA in single-cell data")]),t._v(" "),e("p",[t._v("本教程涵盖了使用 hdWGCNA 在单细胞转录组学数据中构建共表达网络的基本功能。"),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/basic_tutorial.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/basic_tutorial.html"),e("OutboundLink")],1),t._v(" "),e("br"),t._v("\n中文教程："),e("a",{attrs:{href:"http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E8%BD%AC%E5%BD%95%E7%BB%84/007.hdWGCNA/hdWGCNA%20in%20single-cell%20data.nb.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/单细胞空间转录组/007.hdWGCNA/hdWGCNA in single-cell data.nb.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/basic_tutorial/Zhou_featureplot_hMEs_selected_wide.png",alt:"1.1"}})])]),e("h4",{attrs:{id:"_1-2-hdwgcna-in-spatial-transcriptomics-data"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_1-2-hdwgcna-in-spatial-transcriptomics-data"}},[t._v("#")]),t._v(" 1.2 hdWGCNA in spatial transcriptomics data")]),t._v(" "),e("p",[t._v("本教程涵盖了使用 hdWGCNA 在空间转录组学数据中构建共表达网络的基本功能。"),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/ST_basics.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/ST_basics.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/ST_basics/spatial_clusters.png",alt:"1.2"}})])]),e("h4",{attrs:{id:"_1-3-isoform-co-expression-network-analysis-with-pacbio-mas-seq"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_1-3-isoform-co-expression-network-analysis-with-pacbio-mas-seq"}},[t._v("#")]),t._v(" 1.3 Isoform co-expression network analysis with PacBio MAS-Seq")]),t._v(" "),e("p",[t._v("本教程介绍了使用 hdWGCNA 和 PacBio-MAS-Seq 数据进行同源异构体共表达网络分析的基本知识。"),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/isoform_pbmcs.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/isoform_pbmcs.html"),e("OutboundLink")],1)]),t._v(" "),e("h4",{attrs:{id:"_1-4-network-visualization"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_1-4-network-visualization"}},[t._v("#")]),t._v(" 1.4 Network visualization")]),t._v(" "),e("p",[t._v("本教程重点介绍了几种可视化 hdWGCNA 共表达网络的方法。"),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/network_visualizations.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/network_visualizations.html"),e("OutboundLink")],1),t._v(" "),e("br"),t._v("\n中文教程："),e("a",{attrs:{href:"http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E8%BD%AC%E5%BD%95%E7%BB%84/007.hdWGCNA/Network%20Visualization.nb.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/单细胞空间转录组/007.hdWGCNA/Network Visualization.nb.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/network_vis/hubgene_umap_igraph.png",alt:"1.4"}})])]),e("h3",{attrs:{id:"_2-biological-context-for-co-expression-modules"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_2-biological-context-for-co-expression-modules"}},[t._v("#")]),t._v(" 2 Biological context for co-expression modules")]),t._v(" "),e("p",[t._v("这些教程将为我们的共表达模块提供进一步的生物学背景，有可能揭示这些模块所涉及的实验条件和生物学过程。")]),t._v(" "),e("h4",{attrs:{id:"_2-1-differential-module-eigengene-dme-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_2-1-differential-module-eigengene-dme-analysis"}},[t._v("#")]),t._v(" 2.1 Differential module eigengene (DME) analysis")]),t._v(" "),e("p",[t._v("本教程介绍了如何比较实验组之间的 module eigengenes。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/differential_MEs.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/differential_MEs.html"),e("OutboundLink")],1),t._v(" "),e("br"),t._v("\n中文教程："),e("a",{attrs:{href:"http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E8%BD%AC%E5%BD%95%E7%BB%84/007.hdWGCNA/Differential%20module%20eigengene%20(DME)%20analysis.nb.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/单细胞空间转录组/007.hdWGCNA/Differential module eigengene (DME) analysis.nb.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/DMEs/test_DME_volcano.png",alt:"2.1"}})])]),e("h4",{attrs:{id:"_2-2-module-trait-correlation"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_2-2-module-trait-correlation"}},[t._v("#")]),t._v(" 2.2 Module trait correlation")]),t._v(" "),e("p",[t._v("这篇教程介绍了如何使用 module eigengenes 或 module expression scores 来关联连续变量和分类变量，揭示哪些模块与不同的实验条件或协变量相关。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/module_trait_correlation.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/module_trait_correlation.html"),e("OutboundLink")],1),t._v(" "),e("br"),t._v("\n中文教程："),e("a",{attrs:{href:"http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E8%BD%AC%E5%BD%95%E7%BB%84/007.hdWGCNA/Module%20Trait%20Correlation.nb.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/单细胞空间转录组/007.hdWGCNA/Module Trait Correlation.nb.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/mt_correlation/ME_Trait_correlation_fdr.png",alt:"2.2"}})])]),e("h4",{attrs:{id:"_2-3-enrichment-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_2-3-enrichment-analysis"}},[t._v("#")]),t._v(" 2.3 Enrichment analysis")]),t._v(" "),e("p",[t._v("本教程展示了如何使用 Enrichr 将每个共表达模块的基因成员与策划的基因列表进行比较，从而指向共表达模块的生物学功能。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/enrichment_analysis.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/enrichment_analysis.html"),e("OutboundLink")],1),t._v(" "),e("br"),t._v("\n中文教程："),e("a",{attrs:{href:"http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E8%BD%AC%E5%BD%95%E7%BB%84/007.hdWGCNA/Enrichment%20analysis.nb.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/单细胞空间转录组/007.hdWGCNA/Enrichment analysis.nb.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/enrichment/GO_dotplot.png",alt:"2.3"}})])]),e("h3",{attrs:{id:"_3-exploring-modules-in-external-datasets"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_3-exploring-modules-in-external-datasets"}},[t._v("#")]),t._v(" 3 Exploring modules in external datasets")]),t._v(" "),e("h4",{attrs:{id:"_3-1-projecting-modules-to-new-datasets"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_3-1-projecting-modules-to-new-datasets"}},[t._v("#")]),t._v(" 3.1 Projecting modules to new datasets")]),t._v(" "),e("p",[t._v("本教程介绍如何将共表达模块从 reference 投射到 query 数据集。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/projecting_modules.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/projecting_modules.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/projection/compare_umaps.png",alt:"3.1"}})])]),e("h4",{attrs:{id:"_3-2-module-preservation-and-reproducibility"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_3-2-module-preservation-and-reproducibility"}},[t._v("#")]),t._v(" 3.2 Module preservation and reproducibility")]),t._v(" "),e("p",[t._v("本教程介绍了使用外部数据集评估共表达网络保守性和可重复性的统计方法。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/module_preservation.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/module_preservation.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/projection/module_preservation_summary.png",alt:"3.2"}})])]),e("h4",{attrs:{id:"_3-3-cross-species-and-cross-modality-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_3-3-cross-species-and-cross-modality-analysis"}},[t._v("#")]),t._v(" 3.3 Cross-species and cross-modality analysis")]),t._v(" "),e("p",[t._v("本教程介绍了如何在 reference 和 query 之间数据模态或物种不匹配的特殊情况下，将共表达模块从 reference 映射到 query 数据集。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/projecting_modules_cross.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/projecting_modules_cross.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/projection/atac_umap_covplot.png",alt:"3.3"}})])]),e("h3",{attrs:{id:"_4-advanced-topics"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_4-advanced-topics"}},[t._v("#")]),t._v(" 4 Advanced topics")]),t._v(" "),e("h4",{attrs:{id:"_4-1-consensus-network-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_4-1-consensus-network-analysis"}},[t._v("#")]),t._v(" 4.1 Consensus network analysis")]),t._v(" "),e("p",[t._v("在本教程中，我们使用 hdWGCNA 执行一致性共表达网络分析。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/consensus_wgcna.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/consensus_wgcna.html"),e("OutboundLink")],1)]),t._v(" "),e("h4",{attrs:{id:"_4-2-co-expression-module-dynamics-with-pseudotime"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_4-2-co-expression-module-dynamics-with-pseudotime"}},[t._v("#")]),t._v(" 4.2 Co-expression module dynamics with pseudotime")]),t._v(" "),e("p",[t._v("本教程介绍了在具有拟时序信息的数据集中的共表达网络分析。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/pseudotime.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/pseudotime.html"),e("OutboundLink")],1)]),t._v(" "),e("h4",{attrs:{id:"_4-3-motif-analysis"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_4-3-motif-analysis"}},[t._v("#")]),t._v(" 4.3 Motif Analysis")]),t._v(" "),e("p",[t._v("官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/motif_analysis.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/motif_analysis.html"),e("OutboundLink")],1)]),t._v(" "),e("h4",{attrs:{id:"_4-4-alternative-metacell-algorithms"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_4-4-alternative-metacell-algorithms"}},[t._v("#")]),t._v(" 4.4 Alternative metacell algorithms")]),t._v(" "),e("p",[t._v("在本教程中，我们演示了一个高级分析，展示了如何使用替代 metacell 聚合算法进行共表达网络分析。")]),t._v(" "),e("h3",{attrs:{id:"_5-other"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_5-other"}},[t._v("#")]),t._v(" 5 Other")]),t._v(" "),e("h4",{attrs:{id:"_5-1-module-customization"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_5-1-module-customization"}},[t._v("#")]),t._v(" 5.1 Module customization")]),t._v(" "),e("p",[t._v("本教程介绍如何更改 hdWGCNA 模块的默认名称和颜色。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/customization.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/customization.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/customization/featureplot.png",alt:"5.1"}})])]),e("h4",{attrs:{id:"_5-2-using-sctransform-normalized-data"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#_5-2-using-sctransform-normalized-data"}},[t._v("#")]),t._v(" 5.2 Using SCTransform normalized data")]),t._v(" "),e("p",[t._v("本教程介绍如何在 hdWGCNA 中使用 SCTransform normalized 数据。 "),e("br"),t._v("\n官网教程："),e("a",{attrs:{href:"https://smorabit.github.io/hdWGCNA/articles/sctransform.html",target:"_blank",rel:"noopener noreferrer"}},[t._v("https://smorabit.github.io/hdWGCNA/articles/sctransform.html"),e("OutboundLink")],1)]),t._v(" "),e("div",{staticClass:"center-container"},[e("p",[e("img",{attrs:{src:"https://smorabit.github.io/hdWGCNA/articles/figures/sctransform/dendro_SCT_compare.png",alt:"5.2"}})])])])}),[],!1,null,null,null);a.default=r.exports}}]);