(window.webpackJsonp=window.webpackJsonp||[]).push([[27],{359:function(a,t,e){"use strict";e.r(t);var r={data:()=>({cardData1:[{id:"1",cardSrc:"/pages/d207e7/",cardImgSrc:"https://satijalab.org/seurat/articles/assets/seurat_banner.jpg",cardName:"Seurat",cardContent:"R toolkit for single cell genomics"},{cardSrc:"/pages/a35823/",cardImgSrc:"https://avatars.githubusercontent.com/u/40582930?s=48&v=4",cardName:"DoubletFinder",cardContent:"R package for detecting doublets in single-cell RNA sequencing data"},{cardSrc:"/pages/7d0fcf/",cardImgSrc:"https://raw.githubusercontent.com/scverse/anndata/main/docs/_static/img/anndata_schema.svg",cardName:"anndata",cardContent:"A Python package for handling annotated data matrices"},{cardSrc:"/pages/9d48f3/",cardImgSrc:"https://www.bioconductor.org/images/logo_bioconductor.gif",cardName:"MetaNeighbor",cardContent:"A method to rapidly assess cell type identity using both functional and random gene sets"},{cardSrc:"/pages/8dbd6f/",cardImgSrc:"https://camo.githubusercontent.com/35082624fe578d4201a9c9f4b7f1aeb7b9ddae658af644cfe062b01dfd81175f/68747470733a2f2f75706c6f61642e77696b696d656469612e6f72672f77696b6970656469612f636f6d6d6f6e732f7468756d622f632f63622f5468655f6361745f2d5f616e5f696e74726f64756374696f6e5f746f5f7468655f73747564795f6f665f6261636b626f6e65645f616e696d616c732532435f657370656369616c6c795f6d616d6d616c735f253238313838312532395f25323832303537373835303736322532392e6a70672f36343070782d5468655f6361745f2d5f616e5f696e74726f64756374696f6e5f746f5f7468655f73747564795f6f665f6261636b626f6e65645f616e696d616c732532435f657370656369616c6c795f6d616d6d616c735f253238313838312532395f25323832303537373835303736322532392e6a7067",cardName:"scrattch.hicat",cardContent:"Hierarchical, Iterative Clustering for Analysis of Transcriptomics"},{cardSrc:"/pages/9ffdf7/",cardImgSrc:"https://avatars.githubusercontent.com/u/6899542?s=48&v=4",cardName:"WGCNA",cardContent:"Weighted Correlation Network Analysis"},{cardSrc:"/pages/649303/",cardImgSrc:"https://smorabit.github.io/hdWGCNA/reference/figures/logo.png",cardName:"hdWGCNA",cardContent:"High dimensional weighted gene co-expression network analysis"},{cardSrc:"/pages/b62f01/",cardImgSrc:"https://avatars.githubusercontent.com/u/8060918?s=48&v=4",cardName:"monocle3",cardContent:"An analysis toolkit for single-cell RNA-seq"},{cardSrc:"/pages/f26000/",cardImgSrc:"http://www.cellchat.org/images/CellChatDB_Logo_front.png",cardName:"CellChat",cardContent:"Inference and analysis of cell-cell communication"},{cardSrc:"/pages/7a726f/",cardImgSrc:"https://avatars.githubusercontent.com/u/83494306?s=48&v=4",cardName:"NeuronChat",cardContent:"Inferring neuron-neuron communications from single-cell transcriptomics"},{cardSrc:"/pages/9e5aae/",cardImgSrc:"https://samuel-marsh.github.io/scCustomize/reference/figures/scCustomize_Logo.svg",cardName:"scCustomize",cardContent:"A collection of functions created and/or curated to aid in the visualization and analysis of single-cell data using R"},{cardSrc:"/pages/c7e56a/",cardImgSrc:"https://raw.githubusercontent.com/broadinstitute/Tangram/master/figures/tangram_large.png",cardName:"Tangram",cardContent:"Spatial alignment of single cell transcriptomic data."},{cardSrc:"/pages/61a60f/",cardImgSrc:"https://scanpy.readthedocs.io/en/stable/_static/Scanpy_Logo_BrightFG.svg",cardName:"scanpy",cardContent:"Single-cell analysis in Python. Scales to >1M cells."},{cardSrc:"/pages/37f62d/",cardImgSrc:"https://squidpy.readthedocs.io/en/stable/_static/squidpy_horizontal.png",cardName:"Squidpy",cardContent:"Spatial Single Cell Analysis in Python"}],cardData2:[{id:"2",cardSrc:"/pages/006dc9/",cardImgSrc:"https://raw.githubusercontent.com/dmlc/dmlc.github.io/master/img/logo-m/xgboost.png",cardName:"XGBoost",cardContent:"An optimized distributed gradient boosting library designed to be highly efficient, flexible and portable"},{cardSrc:"/pages/ee4b70/",cardImgSrc:"https://glmnet.stanford.edu/reference/figures/logo.png",cardName:"glmnet",cardContent:"Lasso and Elastic-Net Regularized Generalized Linear Models"}],cardData4:[{id:"4",cardSrc:"/pages/8633a5/",cardImgSrc:"https://avatars.githubusercontent.com/u/31731892?s=48&v=4",cardName:"nnUNet",cardContent:"A self-configuring method for deep learning-based biomedical image segmentation"}]})},c=e(1),n=Object(c.a)(r,(function(){var a=this,t=a.$createElement,e=a._self._c||t;return e("ContentSlotsDistributor",{attrs:{"slot-key":a.$parent.slotKey}},[e("h2",{attrs:{id:"单细胞空间组"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#单细胞空间组"}},[a._v("#")]),a._v(" 单细胞空间组")]),a._v(" "),e("ClientOnly",[e("Card",{attrs:{cardData:a.cardData1,cardListSize:3,carTitlColor:"#000",carHoverColor:"#000"}})],1),a._v(" "),e("h2",{attrs:{id:"机器学习"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#机器学习"}},[a._v("#")]),a._v(" 机器学习")]),a._v(" "),e("ClientOnly",[e("Card",{attrs:{cardData:a.cardData2,cardListSize:3,carTitlColor:"#000",carHoverColor:"#000"}})],1),a._v(" "),e("h2",{attrs:{id:"三维基因组"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#三维基因组"}},[a._v("#")]),a._v(" 三维基因组")]),a._v(" "),e("h2",{attrs:{id:"图像处理"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#图像处理"}},[a._v("#")]),a._v(" 图像处理")]),a._v(" "),e("ClientOnly",[e("Card",{attrs:{cardData:a.cardData4,cardListSize:3,carTitlColor:"#000",carHoverColor:"#000"}})],1),a._v(" "),e("h2",{attrs:{id:"ngs相关"}},[e("a",{staticClass:"header-anchor",attrs:{href:"#ngs相关"}},[a._v("#")]),a._v(" NGS相关")])],1)}),[],!1,null,null,null);t.default=n.exports}}]);