(window.webpackJsonp=window.webpackJsonp||[]).push([[0],[]]);!function(n){function e(e){for(var i,o,s=e[0],l=e[1],c=e[2],d=0,p=[];d<s.length;d++)o=s[d],Object.prototype.hasOwnProperty.call(a,o)&&a[o]&&p.push(a[o][0]),a[o]=0;for(i in l)Object.prototype.hasOwnProperty.call(l,i)&&(n[i]=l[i]);for(g&&g(e);p.length;)p.shift()();return r.push.apply(r,c||[]),t()}function t(){for(var n,e=0;e<r.length;e++){for(var t=r[e],i=!0,s=1;s<t.length;s++){var l=t[s];0!==a[l]&&(i=!1)}i&&(r.splice(e--,1),n=o(o.s=t[0]))}return n}var i={},a={1:0},r=[];function o(e){if(i[e])return i[e].exports;var t=i[e]={i:e,l:!1,exports:{}};return n[e].call(t.exports,t,t.exports,o),t.l=!0,t.exports}o.e=function(n){var e=[],t=a[n];if(0!==t)if(t)e.push(t[2]);else{var i=new Promise((function(e,i){t=a[n]=[e,i]}));e.push(t[2]=i);var r,s=document.createElement("script");s.charset="utf-8",s.timeout=120,o.nc&&s.setAttribute("nonce",o.nc),s.src=function(n){return o.p+"assets/js/"+({}[n]||n)+"."+{2:"b95133a4",3:"3abf1aa3",4:"17f28c7e",5:"ff06ff0e",6:"78d855d4",7:"ca82cfad",8:"113240f8",9:"ea9f0c60",10:"3a38bc3c",11:"6804d4e2",12:"5dfbdbbd",13:"0fa098c9",14:"330b018e",15:"b8bb4a86",16:"781d3ca8",17:"14bdc8cc",18:"14ae8e93",19:"bb80f057",20:"04e82b51",21:"2bb87dd4",22:"9f6217ab",23:"5e274244",24:"b9574f9e",25:"137ded62",26:"875ef211",27:"21478680",28:"a17d8c74",29:"82ee0284",30:"f0a41074",31:"67a239e0",32:"2a07296f",33:"3c91d254",34:"fedaa0c2",35:"b2bf202a",36:"ba0c39cd",37:"6c5d6dea",38:"6898e0d0",39:"51032bd3",40:"54330115",41:"0e5fc4df",42:"b8579e68",43:"f0d87263",44:"dadf352d",45:"48126a39",46:"daffcf96",47:"910b1864",48:"ef8b1d7c",49:"627389f0",50:"a8db01db",51:"7ee63ade",52:"704a676f",53:"885a2d7c",54:"15136c57",55:"86145f9f",56:"619fcbc5",57:"d5ab5e8b",58:"ed36b9cb",59:"a6d66110",60:"80777eb6",61:"e0ad0e0b",62:"170fe53a",63:"987c67c8",64:"d110743c",65:"b891a616",66:"f0afa649",67:"a2999e6f",68:"4304a9a3",69:"ca4ccc73",70:"82988a8d",71:"c647d9ae",72:"1a7bd048",73:"d0d8f205",74:"2487173f",75:"96c44e6a",76:"97334731",77:"0f523633",78:"a2b1b565",79:"bc117446",80:"e9de3364",81:"de3e1a8d",82:"d6b092b3",83:"480ad2a7",84:"9ccf3129",85:"d3b4c265",86:"a9d37445",87:"87970e32",88:"ae6c8b1a",89:"004ff48b",90:"574ad6dc",91:"919f7599",92:"0403f50b",93:"f1c10d6e",94:"68180157",95:"43aaa156",96:"5c07dd45",97:"34e6505b",98:"ee88afe4",99:"d636b596",100:"07ad7c69",101:"cf0e9c52",102:"b677846a",103:"d978cd23",104:"aaca6a9e",105:"b70bc017",106:"6357bc2a",107:"52ba7c90",108:"9bdac9ca",109:"0424b457",110:"5e8624f2",111:"9f280b10",112:"e3e184c2",113:"9dc61991",114:"6a888a2a",115:"0c868c7e",116:"5f982486",117:"05d64b6c",118:"de1820da",119:"9e48d583",120:"e79c6680",121:"9d3d53fd",122:"7704c3e0",123:"1d7bd3ac",124:"66f0d159",125:"ffbca55f",126:"faa2bafe",127:"4cd6096d",128:"c063ce35",129:"6ff200b7",130:"bd3a34a0",131:"b1d386f6",132:"f2e1f89a",133:"0dd4a6e4",134:"03528a2c",135:"5f50f01f",136:"6038686c",137:"c3668c24",138:"52048225",139:"8a83e4a5",140:"0c9bf078",141:"bd662f0c",142:"03c93ff2",143:"d81dbfa4",144:"e2baf63f",145:"bbc19ffd",146:"c8bad20d",147:"76385b63",148:"0f773ba8",149:"06f2933d",150:"80df8c58",151:"3d18fd70",152:"2e71a772",153:"0b4b3e96",154:"10e02163",155:"7c69bdf4",156:"660f1ea4",157:"b0c3c63c",158:"2d79cfc7",159:"839ca4dd",160:"4c0aa430",161:"26031b01",162:"7382ebc3",163:"1b782d65",164:"29f66df7",165:"aefb0b28",166:"5a1e69d1",167:"9ec8f2b4",168:"8e72cb0f",169:"00457935",170:"02889291",171:"18d34ca2",172:"9eead420",173:"526e718c",174:"7b94dd0c",175:"b34180db",176:"df484597",177:"bd35bd12",178:"c0a11a10",179:"e31946e2",180:"764682cc",181:"977f1390",182:"2692f937",183:"e8af2276",184:"b4e1ac36",185:"6e4ecc97",186:"989245c1",187:"80231977",188:"d083d8df",189:"26567c30",190:"3873532c",191:"05e3fbd2",192:"4b8ea243",193:"28a8daba",194:"d573e176",195:"8f7be313",196:"e671f76e",197:"fd0b0e77",198:"9d6d57a8",199:"b01e4408",200:"41fbcbdb",201:"112b825c",202:"78dabbb8",203:"d712629e",204:"3244b025",205:"20d2a208"}[n]+".js"}(n);var l=new Error;r=function(e){s.onerror=s.onload=null,clearTimeout(c);var t=a[n];if(0!==t){if(t){var i=e&&("load"===e.type?"missing":e.type),r=e&&e.target&&e.target.src;l.message="Loading chunk "+n+" failed.\n("+i+": "+r+")",l.name="ChunkLoadError",l.type=i,l.request=r,t[1](l)}a[n]=void 0}};var c=setTimeout((function(){r({type:"timeout",target:s})}),12e4);s.onerror=s.onload=r,document.head.appendChild(s)}return Promise.all(e)},o.m=n,o.c=i,o.d=function(n,e,t){o.o(n,e)||Object.defineProperty(n,e,{enumerable:!0,get:t})},o.r=function(n){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(n,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(n,"__esModule",{value:!0})},o.t=function(n,e){if(1&e&&(n=o(n)),8&e)return n;if(4&e&&"object"==typeof n&&n&&n.__esModule)return n;var t=Object.create(null);if(o.r(t),Object.defineProperty(t,"default",{enumerable:!0,value:n}),2&e&&"string"!=typeof n)for(var i in n)o.d(t,i,function(e){return n[e]}.bind(null,i));return t},o.n=function(n){var e=n&&n.__esModule?function(){return n.default}:function(){return n};return o.d(e,"a",e),e},o.o=function(n,e){return Object.prototype.hasOwnProperty.call(n,e)},o.p="/",o.oe=function(n){throw console.error(n),n};var s=window.webpackJsonp=window.webpackJsonp||[],l=s.push.bind(s);s.push=e,s=s.slice();for(var c=0;c<s.length;c++)e(s[c]);var g=l;r.push([109,0]),t()}([function(n,e,t){"use strict";t.r(e);
/*!
 * Vue.js v2.6.14
 * (c) 2014-2021 Evan You
 * Released under the MIT License.
 */
var i=Object.freeze({});function a(n){return null==n}function r(n){return null!=n}function o(n){return!0===n}function s(n){return"string"==typeof n||"number"==typeof n||"symbol"==typeof n||"boolean"==typeof n}function l(n){return null!==n&&"object"==typeof n}var c=Object.prototype.toString;function g(n){return"[object Object]"===c.call(n)}function d(n){return"[object RegExp]"===c.call(n)}function p(n){var e=parseFloat(String(n));return e>=0&&Math.floor(e)===e&&isFinite(n)}function u(n){return r(n)&&"function"==typeof n.then&&"function"==typeof n.catch}function m(n){return null==n?"":Array.isArray(n)||g(n)&&n.toString===c?JSON.stringify(n,null,2):String(n)}function h(n){var e=parseFloat(n);return isNaN(e)?n:e}function f(n,e){for(var t=Object.create(null),i=n.split(","),a=0;a<i.length;a++)t[i[a]]=!0;return e?function(n){return t[n.toLowerCase()]}:function(n){return t[n]}}f("slot,component",!0);var b=f("key,ref,slot,slot-scope,is");function v(n,e){if(n.length){var t=n.indexOf(e);if(t>-1)return n.splice(t,1)}}var y=Object.prototype.hasOwnProperty;function x(n,e){return y.call(n,e)}function C(n){var e=Object.create(null);return function(t){return e[t]||(e[t]=n(t))}}var A=/-(\w)/g,w=C((function(n){return n.replace(A,(function(n,e){return e?e.toUpperCase():""}))})),k=C((function(n){return n.charAt(0).toUpperCase()+n.slice(1)})),T=/\B([A-Z])/g,E=C((function(n){return n.replace(T,"-$1").toLowerCase()}));var F=Function.prototype.bind?function(n,e){return n.bind(e)}:function(n,e){function t(t){var i=arguments.length;return i?i>1?n.apply(e,arguments):n.call(e,t):n.call(e)}return t._length=n.length,t};function S(n,e){e=e||0;for(var t=n.length-e,i=new Array(t);t--;)i[t]=n[t+e];return i}function _(n,e){for(var t in e)n[t]=e[t];return n}function P(n){for(var e={},t=0;t<n.length;t++)n[t]&&_(e,n[t]);return e}function q(n,e,t){}var I=function(n,e,t){return!1},z=function(n){return n};function j(n,e){if(n===e)return!0;var t=l(n),i=l(e);if(!t||!i)return!t&&!i&&String(n)===String(e);try{var a=Array.isArray(n),r=Array.isArray(e);if(a&&r)return n.length===e.length&&n.every((function(n,t){return j(n,e[t])}));if(n instanceof Date&&e instanceof Date)return n.getTime()===e.getTime();if(a||r)return!1;var o=Object.keys(n),s=Object.keys(e);return o.length===s.length&&o.every((function(t){return j(n[t],e[t])}))}catch(n){return!1}}function N(n,e){for(var t=0;t<n.length;t++)if(j(n[t],e))return t;return-1}function B(n){var e=!1;return function(){e||(e=!0,n.apply(this,arguments))}}var D=["component","directive","filter"],R=["beforeCreate","created","beforeMount","mounted","beforeUpdate","updated","beforeDestroy","destroyed","activated","deactivated","errorCaptured","serverPrefetch"],M={optionMergeStrategies:Object.create(null),silent:!1,productionTip:!1,devtools:!1,performance:!1,errorHandler:null,warnHandler:null,ignoredElements:[],keyCodes:Object.create(null),isReservedTag:I,isReservedAttr:I,isUnknownElement:I,getTagNamespace:q,parsePlatformTagName:z,mustUseProp:I,async:!0,_lifecycleHooks:R},L=/a-zA-Z\u00B7\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u037D\u037F-\u1FFF\u200C-\u200D\u203F-\u2040\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD/;function O(n,e,t,i){Object.defineProperty(n,e,{value:t,enumerable:!!i,writable:!0,configurable:!0})}var G=new RegExp("[^"+L.source+".$_\\d]");var H,U="__proto__"in{},V="undefined"!=typeof window,Z="undefined"!=typeof WXEnvironment&&!!WXEnvironment.platform,W=Z&&WXEnvironment.platform.toLowerCase(),X=V&&window.navigator.userAgent.toLowerCase(),$=X&&/msie|trident/.test(X),J=X&&X.indexOf("msie 9.0")>0,K=X&&X.indexOf("edge/")>0,Y=(X&&X.indexOf("android"),X&&/iphone|ipad|ipod|ios/.test(X)||"ios"===W),Q=(X&&/chrome\/\d+/.test(X),X&&/phantomjs/.test(X),X&&X.match(/firefox\/(\d+)/)),nn={}.watch,en=!1;if(V)try{var tn={};Object.defineProperty(tn,"passive",{get:function(){en=!0}}),window.addEventListener("test-passive",null,tn)}catch(n){}var an=function(){return void 0===H&&(H=!V&&!Z&&"undefined"!=typeof global&&(global.process&&"server"===global.process.env.VUE_ENV)),H},rn=V&&window.__VUE_DEVTOOLS_GLOBAL_HOOK__;function on(n){return"function"==typeof n&&/native code/.test(n.toString())}var sn,ln="undefined"!=typeof Symbol&&on(Symbol)&&"undefined"!=typeof Reflect&&on(Reflect.ownKeys);sn="undefined"!=typeof Set&&on(Set)?Set:function(){function n(){this.set=Object.create(null)}return n.prototype.has=function(n){return!0===this.set[n]},n.prototype.add=function(n){this.set[n]=!0},n.prototype.clear=function(){this.set=Object.create(null)},n}();var cn=q,gn=0,dn=function(){this.id=gn++,this.subs=[]};dn.prototype.addSub=function(n){this.subs.push(n)},dn.prototype.removeSub=function(n){v(this.subs,n)},dn.prototype.depend=function(){dn.target&&dn.target.addDep(this)},dn.prototype.notify=function(){var n=this.subs.slice();for(var e=0,t=n.length;e<t;e++)n[e].update()},dn.target=null;var pn=[];function un(n){pn.push(n),dn.target=n}function mn(){pn.pop(),dn.target=pn[pn.length-1]}var hn=function(n,e,t,i,a,r,o,s){this.tag=n,this.data=e,this.children=t,this.text=i,this.elm=a,this.ns=void 0,this.context=r,this.fnContext=void 0,this.fnOptions=void 0,this.fnScopeId=void 0,this.key=e&&e.key,this.componentOptions=o,this.componentInstance=void 0,this.parent=void 0,this.raw=!1,this.isStatic=!1,this.isRootInsert=!0,this.isComment=!1,this.isCloned=!1,this.isOnce=!1,this.asyncFactory=s,this.asyncMeta=void 0,this.isAsyncPlaceholder=!1},fn={child:{configurable:!0}};fn.child.get=function(){return this.componentInstance},Object.defineProperties(hn.prototype,fn);var bn=function(n){void 0===n&&(n="");var e=new hn;return e.text=n,e.isComment=!0,e};function vn(n){return new hn(void 0,void 0,void 0,String(n))}function yn(n){var e=new hn(n.tag,n.data,n.children&&n.children.slice(),n.text,n.elm,n.context,n.componentOptions,n.asyncFactory);return e.ns=n.ns,e.isStatic=n.isStatic,e.key=n.key,e.isComment=n.isComment,e.fnContext=n.fnContext,e.fnOptions=n.fnOptions,e.fnScopeId=n.fnScopeId,e.asyncMeta=n.asyncMeta,e.isCloned=!0,e}var xn=Array.prototype,Cn=Object.create(xn);["push","pop","shift","unshift","splice","sort","reverse"].forEach((function(n){var e=xn[n];O(Cn,n,(function(){for(var t=[],i=arguments.length;i--;)t[i]=arguments[i];var a,r=e.apply(this,t),o=this.__ob__;switch(n){case"push":case"unshift":a=t;break;case"splice":a=t.slice(2)}return a&&o.observeArray(a),o.dep.notify(),r}))}));var An=Object.getOwnPropertyNames(Cn),wn=!0;function kn(n){wn=n}var Tn=function(n){this.value=n,this.dep=new dn,this.vmCount=0,O(n,"__ob__",this),Array.isArray(n)?(U?function(n,e){n.__proto__=e}(n,Cn):function(n,e,t){for(var i=0,a=t.length;i<a;i++){var r=t[i];O(n,r,e[r])}}(n,Cn,An),this.observeArray(n)):this.walk(n)};function En(n,e){var t;if(l(n)&&!(n instanceof hn))return x(n,"__ob__")&&n.__ob__ instanceof Tn?t=n.__ob__:wn&&!an()&&(Array.isArray(n)||g(n))&&Object.isExtensible(n)&&!n._isVue&&(t=new Tn(n)),e&&t&&t.vmCount++,t}function Fn(n,e,t,i,a){var r=new dn,o=Object.getOwnPropertyDescriptor(n,e);if(!o||!1!==o.configurable){var s=o&&o.get,l=o&&o.set;s&&!l||2!==arguments.length||(t=n[e]);var c=!a&&En(t);Object.defineProperty(n,e,{enumerable:!0,configurable:!0,get:function(){var e=s?s.call(n):t;return dn.target&&(r.depend(),c&&(c.dep.depend(),Array.isArray(e)&&Pn(e))),e},set:function(e){var i=s?s.call(n):t;e===i||e!=e&&i!=i||s&&!l||(l?l.call(n,e):t=e,c=!a&&En(e),r.notify())}})}}function Sn(n,e,t){if(Array.isArray(n)&&p(e))return n.length=Math.max(n.length,e),n.splice(e,1,t),t;if(e in n&&!(e in Object.prototype))return n[e]=t,t;var i=n.__ob__;return n._isVue||i&&i.vmCount?t:i?(Fn(i.value,e,t),i.dep.notify(),t):(n[e]=t,t)}function _n(n,e){if(Array.isArray(n)&&p(e))n.splice(e,1);else{var t=n.__ob__;n._isVue||t&&t.vmCount||x(n,e)&&(delete n[e],t&&t.dep.notify())}}function Pn(n){for(var e=void 0,t=0,i=n.length;t<i;t++)(e=n[t])&&e.__ob__&&e.__ob__.dep.depend(),Array.isArray(e)&&Pn(e)}Tn.prototype.walk=function(n){for(var e=Object.keys(n),t=0;t<e.length;t++)Fn(n,e[t])},Tn.prototype.observeArray=function(n){for(var e=0,t=n.length;e<t;e++)En(n[e])};var qn=M.optionMergeStrategies;function In(n,e){if(!e)return n;for(var t,i,a,r=ln?Reflect.ownKeys(e):Object.keys(e),o=0;o<r.length;o++)"__ob__"!==(t=r[o])&&(i=n[t],a=e[t],x(n,t)?i!==a&&g(i)&&g(a)&&In(i,a):Sn(n,t,a));return n}function zn(n,e,t){return t?function(){var i="function"==typeof e?e.call(t,t):e,a="function"==typeof n?n.call(t,t):n;return i?In(i,a):a}:e?n?function(){return In("function"==typeof e?e.call(this,this):e,"function"==typeof n?n.call(this,this):n)}:e:n}function jn(n,e){var t=e?n?n.concat(e):Array.isArray(e)?e:[e]:n;return t?function(n){for(var e=[],t=0;t<n.length;t++)-1===e.indexOf(n[t])&&e.push(n[t]);return e}(t):t}function Nn(n,e,t,i){var a=Object.create(n||null);return e?_(a,e):a}qn.data=function(n,e,t){return t?zn(n,e,t):e&&"function"!=typeof e?n:zn(n,e)},R.forEach((function(n){qn[n]=jn})),D.forEach((function(n){qn[n+"s"]=Nn})),qn.watch=function(n,e,t,i){if(n===nn&&(n=void 0),e===nn&&(e=void 0),!e)return Object.create(n||null);if(!n)return e;var a={};for(var r in _(a,n),e){var o=a[r],s=e[r];o&&!Array.isArray(o)&&(o=[o]),a[r]=o?o.concat(s):Array.isArray(s)?s:[s]}return a},qn.props=qn.methods=qn.inject=qn.computed=function(n,e,t,i){if(!n)return e;var a=Object.create(null);return _(a,n),e&&_(a,e),a},qn.provide=zn;var Bn=function(n,e){return void 0===e?n:e};function Dn(n,e,t){if("function"==typeof e&&(e=e.options),function(n,e){var t=n.props;if(t){var i,a,r={};if(Array.isArray(t))for(i=t.length;i--;)"string"==typeof(a=t[i])&&(r[w(a)]={type:null});else if(g(t))for(var o in t)a=t[o],r[w(o)]=g(a)?a:{type:a};else 0;n.props=r}}(e),function(n,e){var t=n.inject;if(t){var i=n.inject={};if(Array.isArray(t))for(var a=0;a<t.length;a++)i[t[a]]={from:t[a]};else if(g(t))for(var r in t){var o=t[r];i[r]=g(o)?_({from:r},o):{from:o}}else 0}}(e),function(n){var e=n.directives;if(e)for(var t in e){var i=e[t];"function"==typeof i&&(e[t]={bind:i,update:i})}}(e),!e._base&&(e.extends&&(n=Dn(n,e.extends,t)),e.mixins))for(var i=0,a=e.mixins.length;i<a;i++)n=Dn(n,e.mixins[i],t);var r,o={};for(r in n)s(r);for(r in e)x(n,r)||s(r);function s(i){var a=qn[i]||Bn;o[i]=a(n[i],e[i],t,i)}return o}function Rn(n,e,t,i){if("string"==typeof t){var a=n[e];if(x(a,t))return a[t];var r=w(t);if(x(a,r))return a[r];var o=k(r);return x(a,o)?a[o]:a[t]||a[r]||a[o]}}function Mn(n,e,t,i){var a=e[n],r=!x(t,n),o=t[n],s=Hn(Boolean,a.type);if(s>-1)if(r&&!x(a,"default"))o=!1;else if(""===o||o===E(n)){var l=Hn(String,a.type);(l<0||s<l)&&(o=!0)}if(void 0===o){o=function(n,e,t){if(!x(e,"default"))return;var i=e.default;0;if(n&&n.$options.propsData&&void 0===n.$options.propsData[t]&&void 0!==n._props[t])return n._props[t];return"function"==typeof i&&"Function"!==On(e.type)?i.call(n):i}(i,a,n);var c=wn;kn(!0),En(o),kn(c)}return o}var Ln=/^\s*function (\w+)/;function On(n){var e=n&&n.toString().match(Ln);return e?e[1]:""}function Gn(n,e){return On(n)===On(e)}function Hn(n,e){if(!Array.isArray(e))return Gn(e,n)?0:-1;for(var t=0,i=e.length;t<i;t++)if(Gn(e[t],n))return t;return-1}function Un(n,e,t){un();try{if(e)for(var i=e;i=i.$parent;){var a=i.$options.errorCaptured;if(a)for(var r=0;r<a.length;r++)try{if(!1===a[r].call(i,n,e,t))return}catch(n){Zn(n,i,"errorCaptured hook")}}Zn(n,e,t)}finally{mn()}}function Vn(n,e,t,i,a){var r;try{(r=t?n.apply(e,t):n.call(e))&&!r._isVue&&u(r)&&!r._handled&&(r.catch((function(n){return Un(n,i,a+" (Promise/async)")})),r._handled=!0)}catch(n){Un(n,i,a)}return r}function Zn(n,e,t){if(M.errorHandler)try{return M.errorHandler.call(null,n,e,t)}catch(e){e!==n&&Wn(e,null,"config.errorHandler")}Wn(n,e,t)}function Wn(n,e,t){if(!V&&!Z||"undefined"==typeof console)throw n;console.error(n)}var Xn,$n=!1,Jn=[],Kn=!1;function Yn(){Kn=!1;var n=Jn.slice(0);Jn.length=0;for(var e=0;e<n.length;e++)n[e]()}if("undefined"!=typeof Promise&&on(Promise)){var Qn=Promise.resolve();Xn=function(){Qn.then(Yn),Y&&setTimeout(q)},$n=!0}else if($||"undefined"==typeof MutationObserver||!on(MutationObserver)&&"[object MutationObserverConstructor]"!==MutationObserver.toString())Xn="undefined"!=typeof setImmediate&&on(setImmediate)?function(){setImmediate(Yn)}:function(){setTimeout(Yn,0)};else{var ne=1,ee=new MutationObserver(Yn),te=document.createTextNode(String(ne));ee.observe(te,{characterData:!0}),Xn=function(){ne=(ne+1)%2,te.data=String(ne)},$n=!0}function ie(n,e){var t;if(Jn.push((function(){if(n)try{n.call(e)}catch(n){Un(n,e,"nextTick")}else t&&t(e)})),Kn||(Kn=!0,Xn()),!n&&"undefined"!=typeof Promise)return new Promise((function(n){t=n}))}var ae=new sn;function re(n){!function n(e,t){var i,a,r=Array.isArray(e);if(!r&&!l(e)||Object.isFrozen(e)||e instanceof hn)return;if(e.__ob__){var o=e.__ob__.dep.id;if(t.has(o))return;t.add(o)}if(r)for(i=e.length;i--;)n(e[i],t);else for(a=Object.keys(e),i=a.length;i--;)n(e[a[i]],t)}(n,ae),ae.clear()}var oe=C((function(n){var e="&"===n.charAt(0),t="~"===(n=e?n.slice(1):n).charAt(0),i="!"===(n=t?n.slice(1):n).charAt(0);return{name:n=i?n.slice(1):n,once:t,capture:i,passive:e}}));function se(n,e){function t(){var n=arguments,i=t.fns;if(!Array.isArray(i))return Vn(i,null,arguments,e,"v-on handler");for(var a=i.slice(),r=0;r<a.length;r++)Vn(a[r],null,n,e,"v-on handler")}return t.fns=n,t}function le(n,e,t,i,r,s){var l,c,g,d;for(l in n)c=n[l],g=e[l],d=oe(l),a(c)||(a(g)?(a(c.fns)&&(c=n[l]=se(c,s)),o(d.once)&&(c=n[l]=r(d.name,c,d.capture)),t(d.name,c,d.capture,d.passive,d.params)):c!==g&&(g.fns=c,n[l]=g));for(l in e)a(n[l])&&i((d=oe(l)).name,e[l],d.capture)}function ce(n,e,t){var i;n instanceof hn&&(n=n.data.hook||(n.data.hook={}));var s=n[e];function l(){t.apply(this,arguments),v(i.fns,l)}a(s)?i=se([l]):r(s.fns)&&o(s.merged)?(i=s).fns.push(l):i=se([s,l]),i.merged=!0,n[e]=i}function ge(n,e,t,i,a){if(r(e)){if(x(e,t))return n[t]=e[t],a||delete e[t],!0;if(x(e,i))return n[t]=e[i],a||delete e[i],!0}return!1}function de(n){return s(n)?[vn(n)]:Array.isArray(n)?function n(e,t){var i,l,c,g,d=[];for(i=0;i<e.length;i++)a(l=e[i])||"boolean"==typeof l||(c=d.length-1,g=d[c],Array.isArray(l)?l.length>0&&(pe((l=n(l,(t||"")+"_"+i))[0])&&pe(g)&&(d[c]=vn(g.text+l[0].text),l.shift()),d.push.apply(d,l)):s(l)?pe(g)?d[c]=vn(g.text+l):""!==l&&d.push(vn(l)):pe(l)&&pe(g)?d[c]=vn(g.text+l.text):(o(e._isVList)&&r(l.tag)&&a(l.key)&&r(t)&&(l.key="__vlist"+t+"_"+i+"__"),d.push(l)));return d}(n):void 0}function pe(n){return r(n)&&r(n.text)&&!1===n.isComment}function ue(n,e){if(n){for(var t=Object.create(null),i=ln?Reflect.ownKeys(n):Object.keys(n),a=0;a<i.length;a++){var r=i[a];if("__ob__"!==r){for(var o=n[r].from,s=e;s;){if(s._provided&&x(s._provided,o)){t[r]=s._provided[o];break}s=s.$parent}if(!s)if("default"in n[r]){var l=n[r].default;t[r]="function"==typeof l?l.call(e):l}else 0}}return t}}function me(n,e){if(!n||!n.length)return{};for(var t={},i=0,a=n.length;i<a;i++){var r=n[i],o=r.data;if(o&&o.attrs&&o.attrs.slot&&delete o.attrs.slot,r.context!==e&&r.fnContext!==e||!o||null==o.slot)(t.default||(t.default=[])).push(r);else{var s=o.slot,l=t[s]||(t[s]=[]);"template"===r.tag?l.push.apply(l,r.children||[]):l.push(r)}}for(var c in t)t[c].every(he)&&delete t[c];return t}function he(n){return n.isComment&&!n.asyncFactory||" "===n.text}function fe(n){return n.isComment&&n.asyncFactory}function be(n,e,t){var a,r=Object.keys(e).length>0,o=n?!!n.$stable:!r,s=n&&n.$key;if(n){if(n._normalized)return n._normalized;if(o&&t&&t!==i&&s===t.$key&&!r&&!t.$hasNormal)return t;for(var l in a={},n)n[l]&&"$"!==l[0]&&(a[l]=ve(e,l,n[l]))}else a={};for(var c in e)c in a||(a[c]=ye(e,c));return n&&Object.isExtensible(n)&&(n._normalized=a),O(a,"$stable",o),O(a,"$key",s),O(a,"$hasNormal",r),a}function ve(n,e,t){var i=function(){var n=arguments.length?t.apply(null,arguments):t({}),e=(n=n&&"object"==typeof n&&!Array.isArray(n)?[n]:de(n))&&n[0];return n&&(!e||1===n.length&&e.isComment&&!fe(e))?void 0:n};return t.proxy&&Object.defineProperty(n,e,{get:i,enumerable:!0,configurable:!0}),i}function ye(n,e){return function(){return n[e]}}function xe(n,e){var t,i,a,o,s;if(Array.isArray(n)||"string"==typeof n)for(t=new Array(n.length),i=0,a=n.length;i<a;i++)t[i]=e(n[i],i);else if("number"==typeof n)for(t=new Array(n),i=0;i<n;i++)t[i]=e(i+1,i);else if(l(n))if(ln&&n[Symbol.iterator]){t=[];for(var c=n[Symbol.iterator](),g=c.next();!g.done;)t.push(e(g.value,t.length)),g=c.next()}else for(o=Object.keys(n),t=new Array(o.length),i=0,a=o.length;i<a;i++)s=o[i],t[i]=e(n[s],s,i);return r(t)||(t=[]),t._isVList=!0,t}function Ce(n,e,t,i){var a,r=this.$scopedSlots[n];r?(t=t||{},i&&(t=_(_({},i),t)),a=r(t)||("function"==typeof e?e():e)):a=this.$slots[n]||("function"==typeof e?e():e);var o=t&&t.slot;return o?this.$createElement("template",{slot:o},a):a}function Ae(n){return Rn(this.$options,"filters",n)||z}function we(n,e){return Array.isArray(n)?-1===n.indexOf(e):n!==e}function ke(n,e,t,i,a){var r=M.keyCodes[e]||t;return a&&i&&!M.keyCodes[e]?we(a,i):r?we(r,n):i?E(i)!==e:void 0===n}function Te(n,e,t,i,a){if(t)if(l(t)){var r;Array.isArray(t)&&(t=P(t));var o=function(o){if("class"===o||"style"===o||b(o))r=n;else{var s=n.attrs&&n.attrs.type;r=i||M.mustUseProp(e,s,o)?n.domProps||(n.domProps={}):n.attrs||(n.attrs={})}var l=w(o),c=E(o);l in r||c in r||(r[o]=t[o],a&&((n.on||(n.on={}))["update:"+o]=function(n){t[o]=n}))};for(var s in t)o(s)}else;return n}function Ee(n,e){var t=this._staticTrees||(this._staticTrees=[]),i=t[n];return i&&!e||Se(i=t[n]=this.$options.staticRenderFns[n].call(this._renderProxy,null,this),"__static__"+n,!1),i}function Fe(n,e,t){return Se(n,"__once__"+e+(t?"_"+t:""),!0),n}function Se(n,e,t){if(Array.isArray(n))for(var i=0;i<n.length;i++)n[i]&&"string"!=typeof n[i]&&_e(n[i],e+"_"+i,t);else _e(n,e,t)}function _e(n,e,t){n.isStatic=!0,n.key=e,n.isOnce=t}function Pe(n,e){if(e)if(g(e)){var t=n.on=n.on?_({},n.on):{};for(var i in e){var a=t[i],r=e[i];t[i]=a?[].concat(a,r):r}}else;return n}function qe(n,e,t,i){e=e||{$stable:!t};for(var a=0;a<n.length;a++){var r=n[a];Array.isArray(r)?qe(r,e,t):r&&(r.proxy&&(r.fn.proxy=!0),e[r.key]=r.fn)}return i&&(e.$key=i),e}function Ie(n,e){for(var t=0;t<e.length;t+=2){var i=e[t];"string"==typeof i&&i&&(n[e[t]]=e[t+1])}return n}function ze(n,e){return"string"==typeof n?e+n:n}function je(n){n._o=Fe,n._n=h,n._s=m,n._l=xe,n._t=Ce,n._q=j,n._i=N,n._m=Ee,n._f=Ae,n._k=ke,n._b=Te,n._v=vn,n._e=bn,n._u=qe,n._g=Pe,n._d=Ie,n._p=ze}function Ne(n,e,t,a,r){var s,l=this,c=r.options;x(a,"_uid")?(s=Object.create(a))._original=a:(s=a,a=a._original);var g=o(c._compiled),d=!g;this.data=n,this.props=e,this.children=t,this.parent=a,this.listeners=n.on||i,this.injections=ue(c.inject,a),this.slots=function(){return l.$slots||be(n.scopedSlots,l.$slots=me(t,a)),l.$slots},Object.defineProperty(this,"scopedSlots",{enumerable:!0,get:function(){return be(n.scopedSlots,this.slots())}}),g&&(this.$options=c,this.$slots=this.slots(),this.$scopedSlots=be(n.scopedSlots,this.$slots)),c._scopeId?this._c=function(n,e,t,i){var r=Ge(s,n,e,t,i,d);return r&&!Array.isArray(r)&&(r.fnScopeId=c._scopeId,r.fnContext=a),r}:this._c=function(n,e,t,i){return Ge(s,n,e,t,i,d)}}function Be(n,e,t,i,a){var r=yn(n);return r.fnContext=t,r.fnOptions=i,e.slot&&((r.data||(r.data={})).slot=e.slot),r}function De(n,e){for(var t in e)n[w(t)]=e[t]}je(Ne.prototype);var Re={init:function(n,e){if(n.componentInstance&&!n.componentInstance._isDestroyed&&n.data.keepAlive){var t=n;Re.prepatch(t,t)}else{(n.componentInstance=function(n,e){var t={_isComponent:!0,_parentVnode:n,parent:e},i=n.data.inlineTemplate;r(i)&&(t.render=i.render,t.staticRenderFns=i.staticRenderFns);return new n.componentOptions.Ctor(t)}(n,Ke)).$mount(e?n.elm:void 0,e)}},prepatch:function(n,e){var t=e.componentOptions;!function(n,e,t,a,r){0;var o=a.data.scopedSlots,s=n.$scopedSlots,l=!!(o&&!o.$stable||s!==i&&!s.$stable||o&&n.$scopedSlots.$key!==o.$key||!o&&n.$scopedSlots.$key),c=!!(r||n.$options._renderChildren||l);n.$options._parentVnode=a,n.$vnode=a,n._vnode&&(n._vnode.parent=a);if(n.$options._renderChildren=r,n.$attrs=a.data.attrs||i,n.$listeners=t||i,e&&n.$options.props){kn(!1);for(var g=n._props,d=n.$options._propKeys||[],p=0;p<d.length;p++){var u=d[p],m=n.$options.props;g[u]=Mn(u,m,e,n)}kn(!0),n.$options.propsData=e}t=t||i;var h=n.$options._parentListeners;n.$options._parentListeners=t,Je(n,t,h),c&&(n.$slots=me(r,a.context),n.$forceUpdate());0}(e.componentInstance=n.componentInstance,t.propsData,t.listeners,e,t.children)},insert:function(n){var e,t=n.context,i=n.componentInstance;i._isMounted||(i._isMounted=!0,et(i,"mounted")),n.data.keepAlive&&(t._isMounted?((e=i)._inactive=!1,it.push(e)):nt(i,!0))},destroy:function(n){var e=n.componentInstance;e._isDestroyed||(n.data.keepAlive?function n(e,t){if(t&&(e._directInactive=!0,Qe(e)))return;if(!e._inactive){e._inactive=!0;for(var i=0;i<e.$children.length;i++)n(e.$children[i]);et(e,"deactivated")}}(e,!0):e.$destroy())}},Me=Object.keys(Re);function Le(n,e,t,s,c){if(!a(n)){var g=t.$options._base;if(l(n)&&(n=g.extend(n)),"function"==typeof n){var d;if(a(n.cid)&&void 0===(n=function(n,e){if(o(n.error)&&r(n.errorComp))return n.errorComp;if(r(n.resolved))return n.resolved;var t=Ue;t&&r(n.owners)&&-1===n.owners.indexOf(t)&&n.owners.push(t);if(o(n.loading)&&r(n.loadingComp))return n.loadingComp;if(t&&!r(n.owners)){var i=n.owners=[t],s=!0,c=null,g=null;t.$on("hook:destroyed",(function(){return v(i,t)}));var d=function(n){for(var e=0,t=i.length;e<t;e++)i[e].$forceUpdate();n&&(i.length=0,null!==c&&(clearTimeout(c),c=null),null!==g&&(clearTimeout(g),g=null))},p=B((function(t){n.resolved=Ve(t,e),s?i.length=0:d(!0)})),m=B((function(e){r(n.errorComp)&&(n.error=!0,d(!0))})),h=n(p,m);return l(h)&&(u(h)?a(n.resolved)&&h.then(p,m):u(h.component)&&(h.component.then(p,m),r(h.error)&&(n.errorComp=Ve(h.error,e)),r(h.loading)&&(n.loadingComp=Ve(h.loading,e),0===h.delay?n.loading=!0:c=setTimeout((function(){c=null,a(n.resolved)&&a(n.error)&&(n.loading=!0,d(!1))}),h.delay||200)),r(h.timeout)&&(g=setTimeout((function(){g=null,a(n.resolved)&&m(null)}),h.timeout)))),s=!1,n.loading?n.loadingComp:n.resolved}}(d=n,g)))return function(n,e,t,i,a){var r=bn();return r.asyncFactory=n,r.asyncMeta={data:e,context:t,children:i,tag:a},r}(d,e,t,s,c);e=e||{},wt(n),r(e.model)&&function(n,e){var t=n.model&&n.model.prop||"value",i=n.model&&n.model.event||"input";(e.attrs||(e.attrs={}))[t]=e.model.value;var a=e.on||(e.on={}),o=a[i],s=e.model.callback;r(o)?(Array.isArray(o)?-1===o.indexOf(s):o!==s)&&(a[i]=[s].concat(o)):a[i]=s}(n.options,e);var p=function(n,e,t){var i=e.options.props;if(!a(i)){var o={},s=n.attrs,l=n.props;if(r(s)||r(l))for(var c in i){var g=E(c);ge(o,l,c,g,!0)||ge(o,s,c,g,!1)}return o}}(e,n);if(o(n.options.functional))return function(n,e,t,a,o){var s=n.options,l={},c=s.props;if(r(c))for(var g in c)l[g]=Mn(g,c,e||i);else r(t.attrs)&&De(l,t.attrs),r(t.props)&&De(l,t.props);var d=new Ne(t,l,o,a,n),p=s.render.call(null,d._c,d);if(p instanceof hn)return Be(p,t,d.parent,s,d);if(Array.isArray(p)){for(var u=de(p)||[],m=new Array(u.length),h=0;h<u.length;h++)m[h]=Be(u[h],t,d.parent,s,d);return m}}(n,p,e,t,s);var m=e.on;if(e.on=e.nativeOn,o(n.options.abstract)){var h=e.slot;e={},h&&(e.slot=h)}!function(n){for(var e=n.hook||(n.hook={}),t=0;t<Me.length;t++){var i=Me[t],a=e[i],r=Re[i];a===r||a&&a._merged||(e[i]=a?Oe(r,a):r)}}(e);var f=n.options.name||c;return new hn("vue-component-"+n.cid+(f?"-"+f:""),e,void 0,void 0,void 0,t,{Ctor:n,propsData:p,listeners:m,tag:c,children:s},d)}}}function Oe(n,e){var t=function(t,i){n(t,i),e(t,i)};return t._merged=!0,t}function Ge(n,e,t,i,c,g){return(Array.isArray(t)||s(t))&&(c=i,i=t,t=void 0),o(g)&&(c=2),function(n,e,t,i,s){if(r(t)&&r(t.__ob__))return bn();r(t)&&r(t.is)&&(e=t.is);if(!e)return bn();0;Array.isArray(i)&&"function"==typeof i[0]&&((t=t||{}).scopedSlots={default:i[0]},i.length=0);2===s?i=de(i):1===s&&(i=function(n){for(var e=0;e<n.length;e++)if(Array.isArray(n[e]))return Array.prototype.concat.apply([],n);return n}(i));var c,g;if("string"==typeof e){var d;g=n.$vnode&&n.$vnode.ns||M.getTagNamespace(e),c=M.isReservedTag(e)?new hn(M.parsePlatformTagName(e),t,i,void 0,void 0,n):t&&t.pre||!r(d=Rn(n.$options,"components",e))?new hn(e,t,i,void 0,void 0,n):Le(d,t,n,i,e)}else c=Le(e,t,n,i);return Array.isArray(c)?c:r(c)?(r(g)&&function n(e,t,i){e.ns=t,"foreignObject"===e.tag&&(t=void 0,i=!0);if(r(e.children))for(var s=0,l=e.children.length;s<l;s++){var c=e.children[s];r(c.tag)&&(a(c.ns)||o(i)&&"svg"!==c.tag)&&n(c,t,i)}}(c,g),r(t)&&function(n){l(n.style)&&re(n.style);l(n.class)&&re(n.class)}(t),c):bn()}(n,e,t,i,c)}var He,Ue=null;function Ve(n,e){return(n.__esModule||ln&&"Module"===n[Symbol.toStringTag])&&(n=n.default),l(n)?e.extend(n):n}function Ze(n){if(Array.isArray(n))for(var e=0;e<n.length;e++){var t=n[e];if(r(t)&&(r(t.componentOptions)||fe(t)))return t}}function We(n,e){He.$on(n,e)}function Xe(n,e){He.$off(n,e)}function $e(n,e){var t=He;return function i(){var a=e.apply(null,arguments);null!==a&&t.$off(n,i)}}function Je(n,e,t){He=n,le(e,t||{},We,Xe,$e,n),He=void 0}var Ke=null;function Ye(n){var e=Ke;return Ke=n,function(){Ke=e}}function Qe(n){for(;n&&(n=n.$parent);)if(n._inactive)return!0;return!1}function nt(n,e){if(e){if(n._directInactive=!1,Qe(n))return}else if(n._directInactive)return;if(n._inactive||null===n._inactive){n._inactive=!1;for(var t=0;t<n.$children.length;t++)nt(n.$children[t]);et(n,"activated")}}function et(n,e){un();var t=n.$options[e],i=e+" hook";if(t)for(var a=0,r=t.length;a<r;a++)Vn(t[a],n,null,n,i);n._hasHookEvent&&n.$emit("hook:"+e),mn()}var tt=[],it=[],at={},rt=!1,ot=!1,st=0;var lt=0,ct=Date.now;if(V&&!$){var gt=window.performance;gt&&"function"==typeof gt.now&&ct()>document.createEvent("Event").timeStamp&&(ct=function(){return gt.now()})}function dt(){var n,e;for(lt=ct(),ot=!0,tt.sort((function(n,e){return n.id-e.id})),st=0;st<tt.length;st++)(n=tt[st]).before&&n.before(),e=n.id,at[e]=null,n.run();var t=it.slice(),i=tt.slice();st=tt.length=it.length=0,at={},rt=ot=!1,function(n){for(var e=0;e<n.length;e++)n[e]._inactive=!0,nt(n[e],!0)}(t),function(n){var e=n.length;for(;e--;){var t=n[e],i=t.vm;i._watcher===t&&i._isMounted&&!i._isDestroyed&&et(i,"updated")}}(i),rn&&M.devtools&&rn.emit("flush")}var pt=0,ut=function(n,e,t,i,a){this.vm=n,a&&(n._watcher=this),n._watchers.push(this),i?(this.deep=!!i.deep,this.user=!!i.user,this.lazy=!!i.lazy,this.sync=!!i.sync,this.before=i.before):this.deep=this.user=this.lazy=this.sync=!1,this.cb=t,this.id=++pt,this.active=!0,this.dirty=this.lazy,this.deps=[],this.newDeps=[],this.depIds=new sn,this.newDepIds=new sn,this.expression="","function"==typeof e?this.getter=e:(this.getter=function(n){if(!G.test(n)){var e=n.split(".");return function(n){for(var t=0;t<e.length;t++){if(!n)return;n=n[e[t]]}return n}}}(e),this.getter||(this.getter=q)),this.value=this.lazy?void 0:this.get()};ut.prototype.get=function(){var n;un(this);var e=this.vm;try{n=this.getter.call(e,e)}catch(n){if(!this.user)throw n;Un(n,e,'getter for watcher "'+this.expression+'"')}finally{this.deep&&re(n),mn(),this.cleanupDeps()}return n},ut.prototype.addDep=function(n){var e=n.id;this.newDepIds.has(e)||(this.newDepIds.add(e),this.newDeps.push(n),this.depIds.has(e)||n.addSub(this))},ut.prototype.cleanupDeps=function(){for(var n=this.deps.length;n--;){var e=this.deps[n];this.newDepIds.has(e.id)||e.removeSub(this)}var t=this.depIds;this.depIds=this.newDepIds,this.newDepIds=t,this.newDepIds.clear(),t=this.deps,this.deps=this.newDeps,this.newDeps=t,this.newDeps.length=0},ut.prototype.update=function(){this.lazy?this.dirty=!0:this.sync?this.run():function(n){var e=n.id;if(null==at[e]){if(at[e]=!0,ot){for(var t=tt.length-1;t>st&&tt[t].id>n.id;)t--;tt.splice(t+1,0,n)}else tt.push(n);rt||(rt=!0,ie(dt))}}(this)},ut.prototype.run=function(){if(this.active){var n=this.get();if(n!==this.value||l(n)||this.deep){var e=this.value;if(this.value=n,this.user){var t='callback for watcher "'+this.expression+'"';Vn(this.cb,this.vm,[n,e],this.vm,t)}else this.cb.call(this.vm,n,e)}}},ut.prototype.evaluate=function(){this.value=this.get(),this.dirty=!1},ut.prototype.depend=function(){for(var n=this.deps.length;n--;)this.deps[n].depend()},ut.prototype.teardown=function(){if(this.active){this.vm._isBeingDestroyed||v(this.vm._watchers,this);for(var n=this.deps.length;n--;)this.deps[n].removeSub(this);this.active=!1}};var mt={enumerable:!0,configurable:!0,get:q,set:q};function ht(n,e,t){mt.get=function(){return this[e][t]},mt.set=function(n){this[e][t]=n},Object.defineProperty(n,t,mt)}function ft(n){n._watchers=[];var e=n.$options;e.props&&function(n,e){var t=n.$options.propsData||{},i=n._props={},a=n.$options._propKeys=[];n.$parent&&kn(!1);var r=function(r){a.push(r);var o=Mn(r,e,t,n);Fn(i,r,o),r in n||ht(n,"_props",r)};for(var o in e)r(o);kn(!0)}(n,e.props),e.methods&&function(n,e){n.$options.props;for(var t in e)n[t]="function"!=typeof e[t]?q:F(e[t],n)}(n,e.methods),e.data?function(n){var e=n.$options.data;g(e=n._data="function"==typeof e?function(n,e){un();try{return n.call(e,e)}catch(n){return Un(n,e,"data()"),{}}finally{mn()}}(e,n):e||{})||(e={});var t=Object.keys(e),i=n.$options.props,a=(n.$options.methods,t.length);for(;a--;){var r=t[a];0,i&&x(i,r)||(o=void 0,36!==(o=(r+"").charCodeAt(0))&&95!==o&&ht(n,"_data",r))}var o;En(e,!0)}(n):En(n._data={},!0),e.computed&&function(n,e){var t=n._computedWatchers=Object.create(null),i=an();for(var a in e){var r=e[a],o="function"==typeof r?r:r.get;0,i||(t[a]=new ut(n,o||q,q,bt)),a in n||vt(n,a,r)}}(n,e.computed),e.watch&&e.watch!==nn&&function(n,e){for(var t in e){var i=e[t];if(Array.isArray(i))for(var a=0;a<i.length;a++)Ct(n,t,i[a]);else Ct(n,t,i)}}(n,e.watch)}var bt={lazy:!0};function vt(n,e,t){var i=!an();"function"==typeof t?(mt.get=i?yt(e):xt(t),mt.set=q):(mt.get=t.get?i&&!1!==t.cache?yt(e):xt(t.get):q,mt.set=t.set||q),Object.defineProperty(n,e,mt)}function yt(n){return function(){var e=this._computedWatchers&&this._computedWatchers[n];if(e)return e.dirty&&e.evaluate(),dn.target&&e.depend(),e.value}}function xt(n){return function(){return n.call(this,this)}}function Ct(n,e,t,i){return g(t)&&(i=t,t=t.handler),"string"==typeof t&&(t=n[t]),n.$watch(e,t,i)}var At=0;function wt(n){var e=n.options;if(n.super){var t=wt(n.super);if(t!==n.superOptions){n.superOptions=t;var i=function(n){var e,t=n.options,i=n.sealedOptions;for(var a in t)t[a]!==i[a]&&(e||(e={}),e[a]=t[a]);return e}(n);i&&_(n.extendOptions,i),(e=n.options=Dn(t,n.extendOptions)).name&&(e.components[e.name]=n)}}return e}function kt(n){this._init(n)}function Tt(n){n.cid=0;var e=1;n.extend=function(n){n=n||{};var t=this,i=t.cid,a=n._Ctor||(n._Ctor={});if(a[i])return a[i];var r=n.name||t.options.name;var o=function(n){this._init(n)};return(o.prototype=Object.create(t.prototype)).constructor=o,o.cid=e++,o.options=Dn(t.options,n),o.super=t,o.options.props&&function(n){var e=n.options.props;for(var t in e)ht(n.prototype,"_props",t)}(o),o.options.computed&&function(n){var e=n.options.computed;for(var t in e)vt(n.prototype,t,e[t])}(o),o.extend=t.extend,o.mixin=t.mixin,o.use=t.use,D.forEach((function(n){o[n]=t[n]})),r&&(o.options.components[r]=o),o.superOptions=t.options,o.extendOptions=n,o.sealedOptions=_({},o.options),a[i]=o,o}}function Et(n){return n&&(n.Ctor.options.name||n.tag)}function Ft(n,e){return Array.isArray(n)?n.indexOf(e)>-1:"string"==typeof n?n.split(",").indexOf(e)>-1:!!d(n)&&n.test(e)}function St(n,e){var t=n.cache,i=n.keys,a=n._vnode;for(var r in t){var o=t[r];if(o){var s=o.name;s&&!e(s)&&_t(t,r,i,a)}}}function _t(n,e,t,i){var a=n[e];!a||i&&a.tag===i.tag||a.componentInstance.$destroy(),n[e]=null,v(t,e)}!function(n){n.prototype._init=function(n){var e=this;e._uid=At++,e._isVue=!0,n&&n._isComponent?function(n,e){var t=n.$options=Object.create(n.constructor.options),i=e._parentVnode;t.parent=e.parent,t._parentVnode=i;var a=i.componentOptions;t.propsData=a.propsData,t._parentListeners=a.listeners,t._renderChildren=a.children,t._componentTag=a.tag,e.render&&(t.render=e.render,t.staticRenderFns=e.staticRenderFns)}(e,n):e.$options=Dn(wt(e.constructor),n||{},e),e._renderProxy=e,e._self=e,function(n){var e=n.$options,t=e.parent;if(t&&!e.abstract){for(;t.$options.abstract&&t.$parent;)t=t.$parent;t.$children.push(n)}n.$parent=t,n.$root=t?t.$root:n,n.$children=[],n.$refs={},n._watcher=null,n._inactive=null,n._directInactive=!1,n._isMounted=!1,n._isDestroyed=!1,n._isBeingDestroyed=!1}(e),function(n){n._events=Object.create(null),n._hasHookEvent=!1;var e=n.$options._parentListeners;e&&Je(n,e)}(e),function(n){n._vnode=null,n._staticTrees=null;var e=n.$options,t=n.$vnode=e._parentVnode,a=t&&t.context;n.$slots=me(e._renderChildren,a),n.$scopedSlots=i,n._c=function(e,t,i,a){return Ge(n,e,t,i,a,!1)},n.$createElement=function(e,t,i,a){return Ge(n,e,t,i,a,!0)};var r=t&&t.data;Fn(n,"$attrs",r&&r.attrs||i,null,!0),Fn(n,"$listeners",e._parentListeners||i,null,!0)}(e),et(e,"beforeCreate"),function(n){var e=ue(n.$options.inject,n);e&&(kn(!1),Object.keys(e).forEach((function(t){Fn(n,t,e[t])})),kn(!0))}(e),ft(e),function(n){var e=n.$options.provide;e&&(n._provided="function"==typeof e?e.call(n):e)}(e),et(e,"created"),e.$options.el&&e.$mount(e.$options.el)}}(kt),function(n){var e={get:function(){return this._data}},t={get:function(){return this._props}};Object.defineProperty(n.prototype,"$data",e),Object.defineProperty(n.prototype,"$props",t),n.prototype.$set=Sn,n.prototype.$delete=_n,n.prototype.$watch=function(n,e,t){if(g(e))return Ct(this,n,e,t);(t=t||{}).user=!0;var i=new ut(this,n,e,t);if(t.immediate){var a='callback for immediate watcher "'+i.expression+'"';un(),Vn(e,this,[i.value],this,a),mn()}return function(){i.teardown()}}}(kt),function(n){var e=/^hook:/;n.prototype.$on=function(n,t){var i=this;if(Array.isArray(n))for(var a=0,r=n.length;a<r;a++)i.$on(n[a],t);else(i._events[n]||(i._events[n]=[])).push(t),e.test(n)&&(i._hasHookEvent=!0);return i},n.prototype.$once=function(n,e){var t=this;function i(){t.$off(n,i),e.apply(t,arguments)}return i.fn=e,t.$on(n,i),t},n.prototype.$off=function(n,e){var t=this;if(!arguments.length)return t._events=Object.create(null),t;if(Array.isArray(n)){for(var i=0,a=n.length;i<a;i++)t.$off(n[i],e);return t}var r,o=t._events[n];if(!o)return t;if(!e)return t._events[n]=null,t;for(var s=o.length;s--;)if((r=o[s])===e||r.fn===e){o.splice(s,1);break}return t},n.prototype.$emit=function(n){var e=this,t=e._events[n];if(t){t=t.length>1?S(t):t;for(var i=S(arguments,1),a='event handler for "'+n+'"',r=0,o=t.length;r<o;r++)Vn(t[r],e,i,e,a)}return e}}(kt),function(n){n.prototype._update=function(n,e){var t=this,i=t.$el,a=t._vnode,r=Ye(t);t._vnode=n,t.$el=a?t.__patch__(a,n):t.__patch__(t.$el,n,e,!1),r(),i&&(i.__vue__=null),t.$el&&(t.$el.__vue__=t),t.$vnode&&t.$parent&&t.$vnode===t.$parent._vnode&&(t.$parent.$el=t.$el)},n.prototype.$forceUpdate=function(){this._watcher&&this._watcher.update()},n.prototype.$destroy=function(){var n=this;if(!n._isBeingDestroyed){et(n,"beforeDestroy"),n._isBeingDestroyed=!0;var e=n.$parent;!e||e._isBeingDestroyed||n.$options.abstract||v(e.$children,n),n._watcher&&n._watcher.teardown();for(var t=n._watchers.length;t--;)n._watchers[t].teardown();n._data.__ob__&&n._data.__ob__.vmCount--,n._isDestroyed=!0,n.__patch__(n._vnode,null),et(n,"destroyed"),n.$off(),n.$el&&(n.$el.__vue__=null),n.$vnode&&(n.$vnode.parent=null)}}}(kt),function(n){je(n.prototype),n.prototype.$nextTick=function(n){return ie(n,this)},n.prototype._render=function(){var n,e=this,t=e.$options,i=t.render,a=t._parentVnode;a&&(e.$scopedSlots=be(a.data.scopedSlots,e.$slots,e.$scopedSlots)),e.$vnode=a;try{Ue=e,n=i.call(e._renderProxy,e.$createElement)}catch(t){Un(t,e,"render"),n=e._vnode}finally{Ue=null}return Array.isArray(n)&&1===n.length&&(n=n[0]),n instanceof hn||(n=bn()),n.parent=a,n}}(kt);var Pt=[String,RegExp,Array],qt={KeepAlive:{name:"keep-alive",abstract:!0,props:{include:Pt,exclude:Pt,max:[String,Number]},methods:{cacheVNode:function(){var n=this.cache,e=this.keys,t=this.vnodeToCache,i=this.keyToCache;if(t){var a=t.tag,r=t.componentInstance,o=t.componentOptions;n[i]={name:Et(o),tag:a,componentInstance:r},e.push(i),this.max&&e.length>parseInt(this.max)&&_t(n,e[0],e,this._vnode),this.vnodeToCache=null}}},created:function(){this.cache=Object.create(null),this.keys=[]},destroyed:function(){for(var n in this.cache)_t(this.cache,n,this.keys)},mounted:function(){var n=this;this.cacheVNode(),this.$watch("include",(function(e){St(n,(function(n){return Ft(e,n)}))})),this.$watch("exclude",(function(e){St(n,(function(n){return!Ft(e,n)}))}))},updated:function(){this.cacheVNode()},render:function(){var n=this.$slots.default,e=Ze(n),t=e&&e.componentOptions;if(t){var i=Et(t),a=this.include,r=this.exclude;if(a&&(!i||!Ft(a,i))||r&&i&&Ft(r,i))return e;var o=this.cache,s=this.keys,l=null==e.key?t.Ctor.cid+(t.tag?"::"+t.tag:""):e.key;o[l]?(e.componentInstance=o[l].componentInstance,v(s,l),s.push(l)):(this.vnodeToCache=e,this.keyToCache=l),e.data.keepAlive=!0}return e||n&&n[0]}}};!function(n){var e={get:function(){return M}};Object.defineProperty(n,"config",e),n.util={warn:cn,extend:_,mergeOptions:Dn,defineReactive:Fn},n.set=Sn,n.delete=_n,n.nextTick=ie,n.observable=function(n){return En(n),n},n.options=Object.create(null),D.forEach((function(e){n.options[e+"s"]=Object.create(null)})),n.options._base=n,_(n.options.components,qt),function(n){n.use=function(n){var e=this._installedPlugins||(this._installedPlugins=[]);if(e.indexOf(n)>-1)return this;var t=S(arguments,1);return t.unshift(this),"function"==typeof n.install?n.install.apply(n,t):"function"==typeof n&&n.apply(null,t),e.push(n),this}}(n),function(n){n.mixin=function(n){return this.options=Dn(this.options,n),this}}(n),Tt(n),function(n){D.forEach((function(e){n[e]=function(n,t){return t?("component"===e&&g(t)&&(t.name=t.name||n,t=this.options._base.extend(t)),"directive"===e&&"function"==typeof t&&(t={bind:t,update:t}),this.options[e+"s"][n]=t,t):this.options[e+"s"][n]}}))}(n)}(kt),Object.defineProperty(kt.prototype,"$isServer",{get:an}),Object.defineProperty(kt.prototype,"$ssrContext",{get:function(){return this.$vnode&&this.$vnode.ssrContext}}),Object.defineProperty(kt,"FunctionalRenderContext",{value:Ne}),kt.version="2.6.14";var It=f("style,class"),zt=f("input,textarea,option,select,progress"),jt=f("contenteditable,draggable,spellcheck"),Nt=f("events,caret,typing,plaintext-only"),Bt=f("allowfullscreen,async,autofocus,autoplay,checked,compact,controls,declare,default,defaultchecked,defaultmuted,defaultselected,defer,disabled,enabled,formnovalidate,hidden,indeterminate,inert,ismap,itemscope,loop,multiple,muted,nohref,noresize,noshade,novalidate,nowrap,open,pauseonexit,readonly,required,reversed,scoped,seamless,selected,sortable,truespeed,typemustmatch,visible"),Dt="http://www.w3.org/1999/xlink",Rt=function(n){return":"===n.charAt(5)&&"xlink"===n.slice(0,5)},Mt=function(n){return Rt(n)?n.slice(6,n.length):""},Lt=function(n){return null==n||!1===n};function Ot(n){for(var e=n.data,t=n,i=n;r(i.componentInstance);)(i=i.componentInstance._vnode)&&i.data&&(e=Gt(i.data,e));for(;r(t=t.parent);)t&&t.data&&(e=Gt(e,t.data));return function(n,e){if(r(n)||r(e))return Ht(n,Ut(e));return""}(e.staticClass,e.class)}function Gt(n,e){return{staticClass:Ht(n.staticClass,e.staticClass),class:r(n.class)?[n.class,e.class]:e.class}}function Ht(n,e){return n?e?n+" "+e:n:e||""}function Ut(n){return Array.isArray(n)?function(n){for(var e,t="",i=0,a=n.length;i<a;i++)r(e=Ut(n[i]))&&""!==e&&(t&&(t+=" "),t+=e);return t}(n):l(n)?function(n){var e="";for(var t in n)n[t]&&(e&&(e+=" "),e+=t);return e}(n):"string"==typeof n?n:""}var Vt={svg:"http://www.w3.org/2000/svg",math:"http://www.w3.org/1998/Math/MathML"},Zt=f("html,body,base,head,link,meta,style,title,address,article,aside,footer,header,h1,h2,h3,h4,h5,h6,hgroup,nav,section,div,dd,dl,dt,figcaption,figure,picture,hr,img,li,main,ol,p,pre,ul,a,b,abbr,bdi,bdo,br,cite,code,data,dfn,em,i,kbd,mark,q,rp,rt,rtc,ruby,s,samp,small,span,strong,sub,sup,time,u,var,wbr,area,audio,map,track,video,embed,object,param,source,canvas,script,noscript,del,ins,caption,col,colgroup,table,thead,tbody,td,th,tr,button,datalist,fieldset,form,input,label,legend,meter,optgroup,option,output,progress,select,textarea,details,dialog,menu,menuitem,summary,content,element,shadow,template,blockquote,iframe,tfoot"),Wt=f("svg,animate,circle,clippath,cursor,defs,desc,ellipse,filter,font-face,foreignobject,g,glyph,image,line,marker,mask,missing-glyph,path,pattern,polygon,polyline,rect,switch,symbol,text,textpath,tspan,use,view",!0),Xt=function(n){return Zt(n)||Wt(n)};var $t=Object.create(null);var Jt=f("text,number,password,search,email,tel,url");var Kt=Object.freeze({createElement:function(n,e){var t=document.createElement(n);return"select"!==n||e.data&&e.data.attrs&&void 0!==e.data.attrs.multiple&&t.setAttribute("multiple","multiple"),t},createElementNS:function(n,e){return document.createElementNS(Vt[n],e)},createTextNode:function(n){return document.createTextNode(n)},createComment:function(n){return document.createComment(n)},insertBefore:function(n,e,t){n.insertBefore(e,t)},removeChild:function(n,e){n.removeChild(e)},appendChild:function(n,e){n.appendChild(e)},parentNode:function(n){return n.parentNode},nextSibling:function(n){return n.nextSibling},tagName:function(n){return n.tagName},setTextContent:function(n,e){n.textContent=e},setStyleScope:function(n,e){n.setAttribute(e,"")}}),Yt={create:function(n,e){Qt(e)},update:function(n,e){n.data.ref!==e.data.ref&&(Qt(n,!0),Qt(e))},destroy:function(n){Qt(n,!0)}};function Qt(n,e){var t=n.data.ref;if(r(t)){var i=n.context,a=n.componentInstance||n.elm,o=i.$refs;e?Array.isArray(o[t])?v(o[t],a):o[t]===a&&(o[t]=void 0):n.data.refInFor?Array.isArray(o[t])?o[t].indexOf(a)<0&&o[t].push(a):o[t]=[a]:o[t]=a}}var ni=new hn("",{},[]),ei=["create","activate","update","remove","destroy"];function ti(n,e){return n.key===e.key&&n.asyncFactory===e.asyncFactory&&(n.tag===e.tag&&n.isComment===e.isComment&&r(n.data)===r(e.data)&&function(n,e){if("input"!==n.tag)return!0;var t,i=r(t=n.data)&&r(t=t.attrs)&&t.type,a=r(t=e.data)&&r(t=t.attrs)&&t.type;return i===a||Jt(i)&&Jt(a)}(n,e)||o(n.isAsyncPlaceholder)&&a(e.asyncFactory.error))}function ii(n,e,t){var i,a,o={};for(i=e;i<=t;++i)r(a=n[i].key)&&(o[a]=i);return o}var ai={create:ri,update:ri,destroy:function(n){ri(n,ni)}};function ri(n,e){(n.data.directives||e.data.directives)&&function(n,e){var t,i,a,r=n===ni,o=e===ni,s=si(n.data.directives,n.context),l=si(e.data.directives,e.context),c=[],g=[];for(t in l)i=s[t],a=l[t],i?(a.oldValue=i.value,a.oldArg=i.arg,ci(a,"update",e,n),a.def&&a.def.componentUpdated&&g.push(a)):(ci(a,"bind",e,n),a.def&&a.def.inserted&&c.push(a));if(c.length){var d=function(){for(var t=0;t<c.length;t++)ci(c[t],"inserted",e,n)};r?ce(e,"insert",d):d()}g.length&&ce(e,"postpatch",(function(){for(var t=0;t<g.length;t++)ci(g[t],"componentUpdated",e,n)}));if(!r)for(t in s)l[t]||ci(s[t],"unbind",n,n,o)}(n,e)}var oi=Object.create(null);function si(n,e){var t,i,a=Object.create(null);if(!n)return a;for(t=0;t<n.length;t++)(i=n[t]).modifiers||(i.modifiers=oi),a[li(i)]=i,i.def=Rn(e.$options,"directives",i.name);return a}function li(n){return n.rawName||n.name+"."+Object.keys(n.modifiers||{}).join(".")}function ci(n,e,t,i,a){var r=n.def&&n.def[e];if(r)try{r(t.elm,n,t,i,a)}catch(i){Un(i,t.context,"directive "+n.name+" "+e+" hook")}}var gi=[Yt,ai];function di(n,e){var t=e.componentOptions;if(!(r(t)&&!1===t.Ctor.options.inheritAttrs||a(n.data.attrs)&&a(e.data.attrs))){var i,o,s=e.elm,l=n.data.attrs||{},c=e.data.attrs||{};for(i in r(c.__ob__)&&(c=e.data.attrs=_({},c)),c)o=c[i],l[i]!==o&&pi(s,i,o,e.data.pre);for(i in($||K)&&c.value!==l.value&&pi(s,"value",c.value),l)a(c[i])&&(Rt(i)?s.removeAttributeNS(Dt,Mt(i)):jt(i)||s.removeAttribute(i))}}function pi(n,e,t,i){i||n.tagName.indexOf("-")>-1?ui(n,e,t):Bt(e)?Lt(t)?n.removeAttribute(e):(t="allowfullscreen"===e&&"EMBED"===n.tagName?"true":e,n.setAttribute(e,t)):jt(e)?n.setAttribute(e,function(n,e){return Lt(e)||"false"===e?"false":"contenteditable"===n&&Nt(e)?e:"true"}(e,t)):Rt(e)?Lt(t)?n.removeAttributeNS(Dt,Mt(e)):n.setAttributeNS(Dt,e,t):ui(n,e,t)}function ui(n,e,t){if(Lt(t))n.removeAttribute(e);else{if($&&!J&&"TEXTAREA"===n.tagName&&"placeholder"===e&&""!==t&&!n.__ieph){var i=function(e){e.stopImmediatePropagation(),n.removeEventListener("input",i)};n.addEventListener("input",i),n.__ieph=!0}n.setAttribute(e,t)}}var mi={create:di,update:di};function hi(n,e){var t=e.elm,i=e.data,o=n.data;if(!(a(i.staticClass)&&a(i.class)&&(a(o)||a(o.staticClass)&&a(o.class)))){var s=Ot(e),l=t._transitionClasses;r(l)&&(s=Ht(s,Ut(l))),s!==t._prevClass&&(t.setAttribute("class",s),t._prevClass=s)}}var fi,bi={create:hi,update:hi};function vi(n,e,t){var i=fi;return function a(){var r=e.apply(null,arguments);null!==r&&Ci(n,a,t,i)}}var yi=$n&&!(Q&&Number(Q[1])<=53);function xi(n,e,t,i){if(yi){var a=lt,r=e;e=r._wrapper=function(n){if(n.target===n.currentTarget||n.timeStamp>=a||n.timeStamp<=0||n.target.ownerDocument!==document)return r.apply(this,arguments)}}fi.addEventListener(n,e,en?{capture:t,passive:i}:t)}function Ci(n,e,t,i){(i||fi).removeEventListener(n,e._wrapper||e,t)}function Ai(n,e){if(!a(n.data.on)||!a(e.data.on)){var t=e.data.on||{},i=n.data.on||{};fi=e.elm,function(n){if(r(n.__r)){var e=$?"change":"input";n[e]=[].concat(n.__r,n[e]||[]),delete n.__r}r(n.__c)&&(n.change=[].concat(n.__c,n.change||[]),delete n.__c)}(t),le(t,i,xi,Ci,vi,e.context),fi=void 0}}var wi,ki={create:Ai,update:Ai};function Ti(n,e){if(!a(n.data.domProps)||!a(e.data.domProps)){var t,i,o=e.elm,s=n.data.domProps||{},l=e.data.domProps||{};for(t in r(l.__ob__)&&(l=e.data.domProps=_({},l)),s)t in l||(o[t]="");for(t in l){if(i=l[t],"textContent"===t||"innerHTML"===t){if(e.children&&(e.children.length=0),i===s[t])continue;1===o.childNodes.length&&o.removeChild(o.childNodes[0])}if("value"===t&&"PROGRESS"!==o.tagName){o._value=i;var c=a(i)?"":String(i);Ei(o,c)&&(o.value=c)}else if("innerHTML"===t&&Wt(o.tagName)&&a(o.innerHTML)){(wi=wi||document.createElement("div")).innerHTML="<svg>"+i+"</svg>";for(var g=wi.firstChild;o.firstChild;)o.removeChild(o.firstChild);for(;g.firstChild;)o.appendChild(g.firstChild)}else if(i!==s[t])try{o[t]=i}catch(n){}}}}function Ei(n,e){return!n.composing&&("OPTION"===n.tagName||function(n,e){var t=!0;try{t=document.activeElement!==n}catch(n){}return t&&n.value!==e}(n,e)||function(n,e){var t=n.value,i=n._vModifiers;if(r(i)){if(i.number)return h(t)!==h(e);if(i.trim)return t.trim()!==e.trim()}return t!==e}(n,e))}var Fi={create:Ti,update:Ti},Si=C((function(n){var e={},t=/:(.+)/;return n.split(/;(?![^(]*\))/g).forEach((function(n){if(n){var i=n.split(t);i.length>1&&(e[i[0].trim()]=i[1].trim())}})),e}));function _i(n){var e=Pi(n.style);return n.staticStyle?_(n.staticStyle,e):e}function Pi(n){return Array.isArray(n)?P(n):"string"==typeof n?Si(n):n}var qi,Ii=/^--/,zi=/\s*!important$/,ji=function(n,e,t){if(Ii.test(e))n.style.setProperty(e,t);else if(zi.test(t))n.style.setProperty(E(e),t.replace(zi,""),"important");else{var i=Bi(e);if(Array.isArray(t))for(var a=0,r=t.length;a<r;a++)n.style[i]=t[a];else n.style[i]=t}},Ni=["Webkit","Moz","ms"],Bi=C((function(n){if(qi=qi||document.createElement("div").style,"filter"!==(n=w(n))&&n in qi)return n;for(var e=n.charAt(0).toUpperCase()+n.slice(1),t=0;t<Ni.length;t++){var i=Ni[t]+e;if(i in qi)return i}}));function Di(n,e){var t=e.data,i=n.data;if(!(a(t.staticStyle)&&a(t.style)&&a(i.staticStyle)&&a(i.style))){var o,s,l=e.elm,c=i.staticStyle,g=i.normalizedStyle||i.style||{},d=c||g,p=Pi(e.data.style)||{};e.data.normalizedStyle=r(p.__ob__)?_({},p):p;var u=function(n,e){var t,i={};if(e)for(var a=n;a.componentInstance;)(a=a.componentInstance._vnode)&&a.data&&(t=_i(a.data))&&_(i,t);(t=_i(n.data))&&_(i,t);for(var r=n;r=r.parent;)r.data&&(t=_i(r.data))&&_(i,t);return i}(e,!0);for(s in d)a(u[s])&&ji(l,s,"");for(s in u)(o=u[s])!==d[s]&&ji(l,s,null==o?"":o)}}var Ri={create:Di,update:Di},Mi=/\s+/;function Li(n,e){if(e&&(e=e.trim()))if(n.classList)e.indexOf(" ")>-1?e.split(Mi).forEach((function(e){return n.classList.add(e)})):n.classList.add(e);else{var t=" "+(n.getAttribute("class")||"")+" ";t.indexOf(" "+e+" ")<0&&n.setAttribute("class",(t+e).trim())}}function Oi(n,e){if(e&&(e=e.trim()))if(n.classList)e.indexOf(" ")>-1?e.split(Mi).forEach((function(e){return n.classList.remove(e)})):n.classList.remove(e),n.classList.length||n.removeAttribute("class");else{for(var t=" "+(n.getAttribute("class")||"")+" ",i=" "+e+" ";t.indexOf(i)>=0;)t=t.replace(i," ");(t=t.trim())?n.setAttribute("class",t):n.removeAttribute("class")}}function Gi(n){if(n){if("object"==typeof n){var e={};return!1!==n.css&&_(e,Hi(n.name||"v")),_(e,n),e}return"string"==typeof n?Hi(n):void 0}}var Hi=C((function(n){return{enterClass:n+"-enter",enterToClass:n+"-enter-to",enterActiveClass:n+"-enter-active",leaveClass:n+"-leave",leaveToClass:n+"-leave-to",leaveActiveClass:n+"-leave-active"}})),Ui=V&&!J,Vi="transition",Zi="transitionend",Wi="animation",Xi="animationend";Ui&&(void 0===window.ontransitionend&&void 0!==window.onwebkittransitionend&&(Vi="WebkitTransition",Zi="webkitTransitionEnd"),void 0===window.onanimationend&&void 0!==window.onwebkitanimationend&&(Wi="WebkitAnimation",Xi="webkitAnimationEnd"));var $i=V?window.requestAnimationFrame?window.requestAnimationFrame.bind(window):setTimeout:function(n){return n()};function Ji(n){$i((function(){$i(n)}))}function Ki(n,e){var t=n._transitionClasses||(n._transitionClasses=[]);t.indexOf(e)<0&&(t.push(e),Li(n,e))}function Yi(n,e){n._transitionClasses&&v(n._transitionClasses,e),Oi(n,e)}function Qi(n,e,t){var i=ea(n,e),a=i.type,r=i.timeout,o=i.propCount;if(!a)return t();var s="transition"===a?Zi:Xi,l=0,c=function(){n.removeEventListener(s,g),t()},g=function(e){e.target===n&&++l>=o&&c()};setTimeout((function(){l<o&&c()}),r+1),n.addEventListener(s,g)}var na=/\b(transform|all)(,|$)/;function ea(n,e){var t,i=window.getComputedStyle(n),a=(i[Vi+"Delay"]||"").split(", "),r=(i[Vi+"Duration"]||"").split(", "),o=ta(a,r),s=(i[Wi+"Delay"]||"").split(", "),l=(i[Wi+"Duration"]||"").split(", "),c=ta(s,l),g=0,d=0;return"transition"===e?o>0&&(t="transition",g=o,d=r.length):"animation"===e?c>0&&(t="animation",g=c,d=l.length):d=(t=(g=Math.max(o,c))>0?o>c?"transition":"animation":null)?"transition"===t?r.length:l.length:0,{type:t,timeout:g,propCount:d,hasTransform:"transition"===t&&na.test(i[Vi+"Property"])}}function ta(n,e){for(;n.length<e.length;)n=n.concat(n);return Math.max.apply(null,e.map((function(e,t){return ia(e)+ia(n[t])})))}function ia(n){return 1e3*Number(n.slice(0,-1).replace(",","."))}function aa(n,e){var t=n.elm;r(t._leaveCb)&&(t._leaveCb.cancelled=!0,t._leaveCb());var i=Gi(n.data.transition);if(!a(i)&&!r(t._enterCb)&&1===t.nodeType){for(var o=i.css,s=i.type,c=i.enterClass,g=i.enterToClass,d=i.enterActiveClass,p=i.appearClass,u=i.appearToClass,m=i.appearActiveClass,f=i.beforeEnter,b=i.enter,v=i.afterEnter,y=i.enterCancelled,x=i.beforeAppear,C=i.appear,A=i.afterAppear,w=i.appearCancelled,k=i.duration,T=Ke,E=Ke.$vnode;E&&E.parent;)T=E.context,E=E.parent;var F=!T._isMounted||!n.isRootInsert;if(!F||C||""===C){var S=F&&p?p:c,_=F&&m?m:d,P=F&&u?u:g,q=F&&x||f,I=F&&"function"==typeof C?C:b,z=F&&A||v,j=F&&w||y,N=h(l(k)?k.enter:k);0;var D=!1!==o&&!J,R=sa(I),M=t._enterCb=B((function(){D&&(Yi(t,P),Yi(t,_)),M.cancelled?(D&&Yi(t,S),j&&j(t)):z&&z(t),t._enterCb=null}));n.data.show||ce(n,"insert",(function(){var e=t.parentNode,i=e&&e._pending&&e._pending[n.key];i&&i.tag===n.tag&&i.elm._leaveCb&&i.elm._leaveCb(),I&&I(t,M)})),q&&q(t),D&&(Ki(t,S),Ki(t,_),Ji((function(){Yi(t,S),M.cancelled||(Ki(t,P),R||(oa(N)?setTimeout(M,N):Qi(t,s,M)))}))),n.data.show&&(e&&e(),I&&I(t,M)),D||R||M()}}}function ra(n,e){var t=n.elm;r(t._enterCb)&&(t._enterCb.cancelled=!0,t._enterCb());var i=Gi(n.data.transition);if(a(i)||1!==t.nodeType)return e();if(!r(t._leaveCb)){var o=i.css,s=i.type,c=i.leaveClass,g=i.leaveToClass,d=i.leaveActiveClass,p=i.beforeLeave,u=i.leave,m=i.afterLeave,f=i.leaveCancelled,b=i.delayLeave,v=i.duration,y=!1!==o&&!J,x=sa(u),C=h(l(v)?v.leave:v);0;var A=t._leaveCb=B((function(){t.parentNode&&t.parentNode._pending&&(t.parentNode._pending[n.key]=null),y&&(Yi(t,g),Yi(t,d)),A.cancelled?(y&&Yi(t,c),f&&f(t)):(e(),m&&m(t)),t._leaveCb=null}));b?b(w):w()}function w(){A.cancelled||(!n.data.show&&t.parentNode&&((t.parentNode._pending||(t.parentNode._pending={}))[n.key]=n),p&&p(t),y&&(Ki(t,c),Ki(t,d),Ji((function(){Yi(t,c),A.cancelled||(Ki(t,g),x||(oa(C)?setTimeout(A,C):Qi(t,s,A)))}))),u&&u(t,A),y||x||A())}}function oa(n){return"number"==typeof n&&!isNaN(n)}function sa(n){if(a(n))return!1;var e=n.fns;return r(e)?sa(Array.isArray(e)?e[0]:e):(n._length||n.length)>1}function la(n,e){!0!==e.data.show&&aa(e)}var ca=function(n){var e,t,i={},l=n.modules,c=n.nodeOps;for(e=0;e<ei.length;++e)for(i[ei[e]]=[],t=0;t<l.length;++t)r(l[t][ei[e]])&&i[ei[e]].push(l[t][ei[e]]);function g(n){var e=c.parentNode(n);r(e)&&c.removeChild(e,n)}function d(n,e,t,a,s,l,g){if(r(n.elm)&&r(l)&&(n=l[g]=yn(n)),n.isRootInsert=!s,!function(n,e,t,a){var s=n.data;if(r(s)){var l=r(n.componentInstance)&&s.keepAlive;if(r(s=s.hook)&&r(s=s.init)&&s(n,!1),r(n.componentInstance))return p(n,e),u(t,n.elm,a),o(l)&&function(n,e,t,a){var o,s=n;for(;s.componentInstance;)if(s=s.componentInstance._vnode,r(o=s.data)&&r(o=o.transition)){for(o=0;o<i.activate.length;++o)i.activate[o](ni,s);e.push(s);break}u(t,n.elm,a)}(n,e,t,a),!0}}(n,e,t,a)){var d=n.data,h=n.children,f=n.tag;r(f)?(n.elm=n.ns?c.createElementNS(n.ns,f):c.createElement(f,n),v(n),m(n,h,e),r(d)&&b(n,e),u(t,n.elm,a)):o(n.isComment)?(n.elm=c.createComment(n.text),u(t,n.elm,a)):(n.elm=c.createTextNode(n.text),u(t,n.elm,a))}}function p(n,e){r(n.data.pendingInsert)&&(e.push.apply(e,n.data.pendingInsert),n.data.pendingInsert=null),n.elm=n.componentInstance.$el,h(n)?(b(n,e),v(n)):(Qt(n),e.push(n))}function u(n,e,t){r(n)&&(r(t)?c.parentNode(t)===n&&c.insertBefore(n,e,t):c.appendChild(n,e))}function m(n,e,t){if(Array.isArray(e)){0;for(var i=0;i<e.length;++i)d(e[i],t,n.elm,null,!0,e,i)}else s(n.text)&&c.appendChild(n.elm,c.createTextNode(String(n.text)))}function h(n){for(;n.componentInstance;)n=n.componentInstance._vnode;return r(n.tag)}function b(n,t){for(var a=0;a<i.create.length;++a)i.create[a](ni,n);r(e=n.data.hook)&&(r(e.create)&&e.create(ni,n),r(e.insert)&&t.push(n))}function v(n){var e;if(r(e=n.fnScopeId))c.setStyleScope(n.elm,e);else for(var t=n;t;)r(e=t.context)&&r(e=e.$options._scopeId)&&c.setStyleScope(n.elm,e),t=t.parent;r(e=Ke)&&e!==n.context&&e!==n.fnContext&&r(e=e.$options._scopeId)&&c.setStyleScope(n.elm,e)}function y(n,e,t,i,a,r){for(;i<=a;++i)d(t[i],r,n,e,!1,t,i)}function x(n){var e,t,a=n.data;if(r(a))for(r(e=a.hook)&&r(e=e.destroy)&&e(n),e=0;e<i.destroy.length;++e)i.destroy[e](n);if(r(e=n.children))for(t=0;t<n.children.length;++t)x(n.children[t])}function C(n,e,t){for(;e<=t;++e){var i=n[e];r(i)&&(r(i.tag)?(A(i),x(i)):g(i.elm))}}function A(n,e){if(r(e)||r(n.data)){var t,a=i.remove.length+1;for(r(e)?e.listeners+=a:e=function(n,e){function t(){0==--t.listeners&&g(n)}return t.listeners=e,t}(n.elm,a),r(t=n.componentInstance)&&r(t=t._vnode)&&r(t.data)&&A(t,e),t=0;t<i.remove.length;++t)i.remove[t](n,e);r(t=n.data.hook)&&r(t=t.remove)?t(n,e):e()}else g(n.elm)}function w(n,e,t,i){for(var a=t;a<i;a++){var o=e[a];if(r(o)&&ti(n,o))return a}}function k(n,e,t,s,l,g){if(n!==e){r(e.elm)&&r(s)&&(e=s[l]=yn(e));var p=e.elm=n.elm;if(o(n.isAsyncPlaceholder))r(e.asyncFactory.resolved)?F(n.elm,e,t):e.isAsyncPlaceholder=!0;else if(o(e.isStatic)&&o(n.isStatic)&&e.key===n.key&&(o(e.isCloned)||o(e.isOnce)))e.componentInstance=n.componentInstance;else{var u,m=e.data;r(m)&&r(u=m.hook)&&r(u=u.prepatch)&&u(n,e);var f=n.children,b=e.children;if(r(m)&&h(e)){for(u=0;u<i.update.length;++u)i.update[u](n,e);r(u=m.hook)&&r(u=u.update)&&u(n,e)}a(e.text)?r(f)&&r(b)?f!==b&&function(n,e,t,i,o){var s,l,g,p=0,u=0,m=e.length-1,h=e[0],f=e[m],b=t.length-1,v=t[0],x=t[b],A=!o;for(0;p<=m&&u<=b;)a(h)?h=e[++p]:a(f)?f=e[--m]:ti(h,v)?(k(h,v,i,t,u),h=e[++p],v=t[++u]):ti(f,x)?(k(f,x,i,t,b),f=e[--m],x=t[--b]):ti(h,x)?(k(h,x,i,t,b),A&&c.insertBefore(n,h.elm,c.nextSibling(f.elm)),h=e[++p],x=t[--b]):ti(f,v)?(k(f,v,i,t,u),A&&c.insertBefore(n,f.elm,h.elm),f=e[--m],v=t[++u]):(a(s)&&(s=ii(e,p,m)),a(l=r(v.key)?s[v.key]:w(v,e,p,m))?d(v,i,n,h.elm,!1,t,u):ti(g=e[l],v)?(k(g,v,i,t,u),e[l]=void 0,A&&c.insertBefore(n,g.elm,h.elm)):d(v,i,n,h.elm,!1,t,u),v=t[++u]);p>m?y(n,a(t[b+1])?null:t[b+1].elm,t,u,b,i):u>b&&C(e,p,m)}(p,f,b,t,g):r(b)?(r(n.text)&&c.setTextContent(p,""),y(p,null,b,0,b.length-1,t)):r(f)?C(f,0,f.length-1):r(n.text)&&c.setTextContent(p,""):n.text!==e.text&&c.setTextContent(p,e.text),r(m)&&r(u=m.hook)&&r(u=u.postpatch)&&u(n,e)}}}function T(n,e,t){if(o(t)&&r(n.parent))n.parent.data.pendingInsert=e;else for(var i=0;i<e.length;++i)e[i].data.hook.insert(e[i])}var E=f("attrs,class,staticClass,staticStyle,key");function F(n,e,t,i){var a,s=e.tag,l=e.data,c=e.children;if(i=i||l&&l.pre,e.elm=n,o(e.isComment)&&r(e.asyncFactory))return e.isAsyncPlaceholder=!0,!0;if(r(l)&&(r(a=l.hook)&&r(a=a.init)&&a(e,!0),r(a=e.componentInstance)))return p(e,t),!0;if(r(s)){if(r(c))if(n.hasChildNodes())if(r(a=l)&&r(a=a.domProps)&&r(a=a.innerHTML)){if(a!==n.innerHTML)return!1}else{for(var g=!0,d=n.firstChild,u=0;u<c.length;u++){if(!d||!F(d,c[u],t,i)){g=!1;break}d=d.nextSibling}if(!g||d)return!1}else m(e,c,t);if(r(l)){var h=!1;for(var f in l)if(!E(f)){h=!0,b(e,t);break}!h&&l.class&&re(l.class)}}else n.data!==e.text&&(n.data=e.text);return!0}return function(n,e,t,s){if(!a(e)){var l,g=!1,p=[];if(a(n))g=!0,d(e,p);else{var u=r(n.nodeType);if(!u&&ti(n,e))k(n,e,p,null,null,s);else{if(u){if(1===n.nodeType&&n.hasAttribute("data-server-rendered")&&(n.removeAttribute("data-server-rendered"),t=!0),o(t)&&F(n,e,p))return T(e,p,!0),n;l=n,n=new hn(c.tagName(l).toLowerCase(),{},[],void 0,l)}var m=n.elm,f=c.parentNode(m);if(d(e,p,m._leaveCb?null:f,c.nextSibling(m)),r(e.parent))for(var b=e.parent,v=h(e);b;){for(var y=0;y<i.destroy.length;++y)i.destroy[y](b);if(b.elm=e.elm,v){for(var A=0;A<i.create.length;++A)i.create[A](ni,b);var w=b.data.hook.insert;if(w.merged)for(var E=1;E<w.fns.length;E++)w.fns[E]()}else Qt(b);b=b.parent}r(f)?C([n],0,0):r(n.tag)&&x(n)}}return T(e,p,g),e.elm}r(n)&&x(n)}}({nodeOps:Kt,modules:[mi,bi,ki,Fi,Ri,V?{create:la,activate:la,remove:function(n,e){!0!==n.data.show?ra(n,e):e()}}:{}].concat(gi)});J&&document.addEventListener("selectionchange",(function(){var n=document.activeElement;n&&n.vmodel&&ba(n,"input")}));var ga={inserted:function(n,e,t,i){"select"===t.tag?(i.elm&&!i.elm._vOptions?ce(t,"postpatch",(function(){ga.componentUpdated(n,e,t)})):da(n,e,t.context),n._vOptions=[].map.call(n.options,ma)):("textarea"===t.tag||Jt(n.type))&&(n._vModifiers=e.modifiers,e.modifiers.lazy||(n.addEventListener("compositionstart",ha),n.addEventListener("compositionend",fa),n.addEventListener("change",fa),J&&(n.vmodel=!0)))},componentUpdated:function(n,e,t){if("select"===t.tag){da(n,e,t.context);var i=n._vOptions,a=n._vOptions=[].map.call(n.options,ma);if(a.some((function(n,e){return!j(n,i[e])})))(n.multiple?e.value.some((function(n){return ua(n,a)})):e.value!==e.oldValue&&ua(e.value,a))&&ba(n,"change")}}};function da(n,e,t){pa(n,e,t),($||K)&&setTimeout((function(){pa(n,e,t)}),0)}function pa(n,e,t){var i=e.value,a=n.multiple;if(!a||Array.isArray(i)){for(var r,o,s=0,l=n.options.length;s<l;s++)if(o=n.options[s],a)r=N(i,ma(o))>-1,o.selected!==r&&(o.selected=r);else if(j(ma(o),i))return void(n.selectedIndex!==s&&(n.selectedIndex=s));a||(n.selectedIndex=-1)}}function ua(n,e){return e.every((function(e){return!j(e,n)}))}function ma(n){return"_value"in n?n._value:n.value}function ha(n){n.target.composing=!0}function fa(n){n.target.composing&&(n.target.composing=!1,ba(n.target,"input"))}function ba(n,e){var t=document.createEvent("HTMLEvents");t.initEvent(e,!0,!0),n.dispatchEvent(t)}function va(n){return!n.componentInstance||n.data&&n.data.transition?n:va(n.componentInstance._vnode)}var ya={model:ga,show:{bind:function(n,e,t){var i=e.value,a=(t=va(t)).data&&t.data.transition,r=n.__vOriginalDisplay="none"===n.style.display?"":n.style.display;i&&a?(t.data.show=!0,aa(t,(function(){n.style.display=r}))):n.style.display=i?r:"none"},update:function(n,e,t){var i=e.value;!i!=!e.oldValue&&((t=va(t)).data&&t.data.transition?(t.data.show=!0,i?aa(t,(function(){n.style.display=n.__vOriginalDisplay})):ra(t,(function(){n.style.display="none"}))):n.style.display=i?n.__vOriginalDisplay:"none")},unbind:function(n,e,t,i,a){a||(n.style.display=n.__vOriginalDisplay)}}},xa={name:String,appear:Boolean,css:Boolean,mode:String,type:String,enterClass:String,leaveClass:String,enterToClass:String,leaveToClass:String,enterActiveClass:String,leaveActiveClass:String,appearClass:String,appearActiveClass:String,appearToClass:String,duration:[Number,String,Object]};function Ca(n){var e=n&&n.componentOptions;return e&&e.Ctor.options.abstract?Ca(Ze(e.children)):n}function Aa(n){var e={},t=n.$options;for(var i in t.propsData)e[i]=n[i];var a=t._parentListeners;for(var r in a)e[w(r)]=a[r];return e}function wa(n,e){if(/\d-keep-alive$/.test(e.tag))return n("keep-alive",{props:e.componentOptions.propsData})}var ka=function(n){return n.tag||fe(n)},Ta=function(n){return"show"===n.name},Ea={name:"transition",props:xa,abstract:!0,render:function(n){var e=this,t=this.$slots.default;if(t&&(t=t.filter(ka)).length){0;var i=this.mode;0;var a=t[0];if(function(n){for(;n=n.parent;)if(n.data.transition)return!0}(this.$vnode))return a;var r=Ca(a);if(!r)return a;if(this._leaving)return wa(n,a);var o="__transition-"+this._uid+"-";r.key=null==r.key?r.isComment?o+"comment":o+r.tag:s(r.key)?0===String(r.key).indexOf(o)?r.key:o+r.key:r.key;var l=(r.data||(r.data={})).transition=Aa(this),c=this._vnode,g=Ca(c);if(r.data.directives&&r.data.directives.some(Ta)&&(r.data.show=!0),g&&g.data&&!function(n,e){return e.key===n.key&&e.tag===n.tag}(r,g)&&!fe(g)&&(!g.componentInstance||!g.componentInstance._vnode.isComment)){var d=g.data.transition=_({},l);if("out-in"===i)return this._leaving=!0,ce(d,"afterLeave",(function(){e._leaving=!1,e.$forceUpdate()})),wa(n,a);if("in-out"===i){if(fe(r))return c;var p,u=function(){p()};ce(l,"afterEnter",u),ce(l,"enterCancelled",u),ce(d,"delayLeave",(function(n){p=n}))}}return a}}},Fa=_({tag:String,moveClass:String},xa);function Sa(n){n.elm._moveCb&&n.elm._moveCb(),n.elm._enterCb&&n.elm._enterCb()}function _a(n){n.data.newPos=n.elm.getBoundingClientRect()}function Pa(n){var e=n.data.pos,t=n.data.newPos,i=e.left-t.left,a=e.top-t.top;if(i||a){n.data.moved=!0;var r=n.elm.style;r.transform=r.WebkitTransform="translate("+i+"px,"+a+"px)",r.transitionDuration="0s"}}delete Fa.mode;var qa={Transition:Ea,TransitionGroup:{props:Fa,beforeMount:function(){var n=this,e=this._update;this._update=function(t,i){var a=Ye(n);n.__patch__(n._vnode,n.kept,!1,!0),n._vnode=n.kept,a(),e.call(n,t,i)}},render:function(n){for(var e=this.tag||this.$vnode.data.tag||"span",t=Object.create(null),i=this.prevChildren=this.children,a=this.$slots.default||[],r=this.children=[],o=Aa(this),s=0;s<a.length;s++){var l=a[s];if(l.tag)if(null!=l.key&&0!==String(l.key).indexOf("__vlist"))r.push(l),t[l.key]=l,(l.data||(l.data={})).transition=o;else;}if(i){for(var c=[],g=[],d=0;d<i.length;d++){var p=i[d];p.data.transition=o,p.data.pos=p.elm.getBoundingClientRect(),t[p.key]?c.push(p):g.push(p)}this.kept=n(e,null,c),this.removed=g}return n(e,null,r)},updated:function(){var n=this.prevChildren,e=this.moveClass||(this.name||"v")+"-move";n.length&&this.hasMove(n[0].elm,e)&&(n.forEach(Sa),n.forEach(_a),n.forEach(Pa),this._reflow=document.body.offsetHeight,n.forEach((function(n){if(n.data.moved){var t=n.elm,i=t.style;Ki(t,e),i.transform=i.WebkitTransform=i.transitionDuration="",t.addEventListener(Zi,t._moveCb=function n(i){i&&i.target!==t||i&&!/transform$/.test(i.propertyName)||(t.removeEventListener(Zi,n),t._moveCb=null,Yi(t,e))})}})))},methods:{hasMove:function(n,e){if(!Ui)return!1;if(this._hasMove)return this._hasMove;var t=n.cloneNode();n._transitionClasses&&n._transitionClasses.forEach((function(n){Oi(t,n)})),Li(t,e),t.style.display="none",this.$el.appendChild(t);var i=ea(t);return this.$el.removeChild(t),this._hasMove=i.hasTransform}}}};kt.config.mustUseProp=function(n,e,t){return"value"===t&&zt(n)&&"button"!==e||"selected"===t&&"option"===n||"checked"===t&&"input"===n||"muted"===t&&"video"===n},kt.config.isReservedTag=Xt,kt.config.isReservedAttr=It,kt.config.getTagNamespace=function(n){return Wt(n)?"svg":"math"===n?"math":void 0},kt.config.isUnknownElement=function(n){if(!V)return!0;if(Xt(n))return!1;if(n=n.toLowerCase(),null!=$t[n])return $t[n];var e=document.createElement(n);return n.indexOf("-")>-1?$t[n]=e.constructor===window.HTMLUnknownElement||e.constructor===window.HTMLElement:$t[n]=/HTMLUnknownElement/.test(e.toString())},_(kt.options.directives,ya),_(kt.options.components,qa),kt.prototype.__patch__=V?ca:q,kt.prototype.$mount=function(n,e){return function(n,e,t){var i;return n.$el=e,n.$options.render||(n.$options.render=bn),et(n,"beforeMount"),i=function(){n._update(n._render(),t)},new ut(n,i,q,{before:function(){n._isMounted&&!n._isDestroyed&&et(n,"beforeUpdate")}},!0),t=!1,null==n.$vnode&&(n._isMounted=!0,et(n,"mounted")),n}(this,n=n&&V?function(n){if("string"==typeof n){var e=document.querySelector(n);return e||document.createElement("div")}return n}(n):void 0,e)},V&&setTimeout((function(){M.devtools&&rn&&rn.emit("init",kt)}),0),e.default=kt},function(n,e,t){"use strict";function i(n,e,t,i,a,r,o,s){var l,c="function"==typeof n?n.options:n;if(e&&(c.render=e,c.staticRenderFns=t,c._compiled=!0),i&&(c.functional=!0),r&&(c._scopeId="data-v-"+r),o?(l=function(n){(n=n||this.$vnode&&this.$vnode.ssrContext||this.parent&&this.parent.$vnode&&this.parent.$vnode.ssrContext)||"undefined"==typeof __VUE_SSR_CONTEXT__||(n=__VUE_SSR_CONTEXT__),a&&a.call(this,n),n&&n._registeredComponents&&n._registeredComponents.add(o)},c._ssrRegister=l):a&&(l=s?function(){a.call(this,(c.functional?this.parent:this).$root.$options.shadowRoot)}:a),l)if(c.functional){c._injectStyles=l;var g=c.render;c.render=function(n,e){return l.call(e),g(n,e)}}else{var d=c.beforeCreate;c.beforeCreate=d?[].concat(d,l):[l]}return{exports:n,options:c}}t.d(e,"a",(function(){return i}))},function(n,e,t){"use strict";var i=t(57),a=i.all;n.exports=i.IS_HTMLDDA?function(n){return"function"==typeof n||n===a}:function(n){return"function"==typeof n}},function(n,e,t){"use strict";var i=function(n){return n&&n.Math===Math&&n};n.exports=i("object"==typeof globalThis&&globalThis)||i("object"==typeof window&&window)||i("object"==typeof self&&self)||i("object"==typeof global&&global)||function(){return this}()||this||Function("return this")()},function(n,e,t){"use strict";var i=t(28),a=Function.prototype,r=a.call,o=i&&a.bind.bind(r,r);n.exports=i?o:function(n){return function(){return r.apply(n,arguments)}}},function(n,e,t){"use strict";n.exports=function(n){try{return!!n()}catch(n){return!0}}},function(n,e,t){"use strict";var i=t(5);n.exports=!i((function(){return 7!==Object.defineProperty({},1,{get:function(){return 7}})[1]}))},function(n,e){var t=Array.isArray;n.exports=t},function(n,e,t){var i=t(71),a="object"==typeof self&&self&&self.Object===Object&&self,r=i||a||Function("return this")();n.exports=r},function(n,e,t){"use strict";var i=t(4),a=t(33),r=i({}.hasOwnProperty);n.exports=Object.hasOwn||function(n,e){return r(a(n),e)}},function(n,e,t){"use strict";var i=t(2),a=t(57),r=a.all;n.exports=a.IS_HTMLDDA?function(n){return"object"==typeof n?null!==n:i(n)||n===r}:function(n){return"object"==typeof n?null!==n:i(n)}},function(n,e,t){var i=t(166),a=t(169);n.exports=function(n,e){var t=a(n,e);return i(t)?t:void 0}},function(n,e,t){"use strict";var i=t(27),a=t(33),r=t(34),o=t(132),s=t(134);i({target:"Array",proto:!0,arity:1,forced:t(5)((function(){return 4294967297!==[].push.call({length:4294967296},1)}))||!function(){try{Object.defineProperty([],"length",{writable:!1}).push()}catch(n){return n instanceof TypeError}}()},{push:function(n){var e=a(this),t=r(e),i=arguments.length;s(t+i);for(var l=0;l<i;l++)e[t]=arguments[l],t++;return o(e,t),t}})},function(n,e,t){"use strict";t.d(e,"e",(function(){return i})),t.d(e,"b",(function(){return r})),t.d(e,"j",(function(){return o})),t.d(e,"g",(function(){return l})),t.d(e,"h",(function(){return c})),t.d(e,"i",(function(){return g})),t.d(e,"c",(function(){return d})),t.d(e,"f",(function(){return p})),t.d(e,"l",(function(){return u})),t.d(e,"m",(function(){return m})),t.d(e,"d",(function(){return f})),t.d(e,"k",(function(){return b})),t.d(e,"n",(function(){return v})),t.d(e,"a",(function(){return x}));t(12);const i=/#.*$/,a=/\.(md|html)$/,r=/\/$/,o=/^[a-z]+:/i;function s(n){return decodeURI(n).replace(i,"").replace(a,"")}function l(n){return o.test(n)}function c(n){return/^mailto:/.test(n)}function g(n){return/^tel:/.test(n)}function d(n){if(l(n))return n;if(!n)return"404";const e=n.match(i),t=e?e[0]:"",a=s(n);return r.test(a)?n:a+".html"+t}function p(n,e){const t=n.hash,a=function(n){const e=n&&n.match(i);if(e)return e[0]}(e);if(a&&t!==a)return!1;return s(n.path)===s(e)}function u(n,e,t){if(l(e))return{type:"external",path:e};t&&(e=function(n,e,t){const i=n.charAt(0);if("/"===i)return n;if("?"===i||"#"===i)return e+n;const a=e.split("/");t&&a[a.length-1]||a.pop();const r=n.replace(/^\//,"").split("/");for(let n=0;n<r.length;n++){const e=r[n];".."===e?a.pop():"."!==e&&a.push(e)}""!==a[0]&&a.unshift("");return a.join("/")}(e,t));const i=s(e);for(let e=0;e<n.length;e++)if(s(n[e].regularPath)===i)return Object.assign({},n[e],{type:"page",path:d(n[e].path)});return console.error(`[vuepress] No matching page found for sidebar item "${e}"`),{}}function m(n,e,t,i){const{pages:a,themeConfig:r}=t,o=i&&r.locales&&r.locales[i]||r;if("auto"===(n.frontmatter.sidebar||o.sidebar||r.sidebar))return h(n);const s=o.sidebar||r.sidebar;if(s){const{base:t,config:i}=function(n,e){if(Array.isArray(e))return{base:"/",config:e};for(const i in e)if(0===(t=n,/(\.html|\/)$/.test(t)?t:t+"/").indexOf(encodeURI(i)))return{base:i,config:e[i]};var t;return{}}(e,s);return"auto"===i?h(n):i?i.map(n=>function n(e,t,i,a=1){if("string"==typeof e)return u(t,e,i);if(Array.isArray(e))return Object.assign(u(t,e[0],i),{title:e[1]});{a>3&&console.error("[vuepress] detected a too deep nested sidebar group.");const r=e.children||[];return 0===r.length&&e.path?Object.assign(u(t,e.path,i),{title:e.title}):{type:"group",path:e.path,title:e.title,sidebarDepth:e.sidebarDepth,initialOpenGroupIndex:e.initialOpenGroupIndex,children:r.map(e=>n(e,t,i,a+1)),collapsable:!1!==e.collapsable}}}(n,a,t)):[]}return[]}function h(n){const e=f(n.headers||[]);return[{type:"group",collapsable:!1,title:n.title,path:null,children:e.map(e=>({type:"auto",title:e.title,basePath:n.path,path:n.path+"#"+e.slug,children:e.children||[]}))}]}function f(n){let e;return(n=n.map(n=>Object.assign({},n))).forEach(n=>{2===n.level?e=n:e&&(e.children||(e.children=[])).push(n)}),n.filter(n=>2===n.level)}function b(n){return Object.assign(n,{type:n.items&&n.items.length?"links":"link"})}function v(n){return Object.prototype.toString.call(n).match(/\[object (.*?)\]/)[1].toLowerCase()}function y(n){let e=n.frontmatter.date||n.lastUpdated||new Date,t=new Date(e);return"Invalid Date"==t&&e&&(t=new Date(e.replace(/-/g,"/"))),t.getTime()}function x(n,e){return y(e)-y(n)}},function(n,e){n.exports=function(n){return null!=n&&"object"==typeof n}},function(n,e,t){var i=t(17),a=t(151),r=t(152),o=i?i.toStringTag:void 0;n.exports=function(n){return null==n?void 0===n?"[object Undefined]":"[object Null]":o&&o in Object(n)?a(n):r(n)}},function(n,e,t){"use strict";var i=t(6),a=t(18),r=t(36);n.exports=i?function(n,e,t){return a.f(n,e,r(1,t))}:function(n,e,t){return n[e]=t,n}},function(n,e,t){var i=t(8).Symbol;n.exports=i},function(n,e,t){"use strict";var i=t(6),a=t(66),r=t(104),o=t(26),s=t(56),l=TypeError,c=Object.defineProperty,g=Object.getOwnPropertyDescriptor;e.f=i?r?function(n,e,t){if(o(n),e=s(e),o(t),"function"==typeof n&&"prototype"===e&&"value"in t&&"writable"in t&&!t.writable){var i=g(n,e);i&&i.writable&&(n[e]=t.value,t={configurable:"configurable"in t?t.configurable:i.configurable,enumerable:"enumerable"in t?t.enumerable:i.enumerable,writable:!1})}return c(n,e,t)}:c:function(n,e,t){if(o(n),e=s(e),o(t),a)try{return c(n,e,t)}catch(n){}if("get"in t||"set"in t)throw l("Accessors not supported");return"value"in t&&(n[e]=t.value),n}},function(n,e,t){"use strict";var i=t(4),a=i({}.toString),r=i("".slice);n.exports=function(n){return r(a(n),8,-1)}},function(n,e,t){var i=t(156),a=t(157),r=t(158),o=t(159),s=t(160);function l(n){var e=-1,t=null==n?0:n.length;for(this.clear();++e<t;){var i=n[e];this.set(i[0],i[1])}}l.prototype.clear=i,l.prototype.delete=a,l.prototype.get=r,l.prototype.has=o,l.prototype.set=s,n.exports=l},function(n,e,t){var i=t(73);n.exports=function(n,e){for(var t=n.length;t--;)if(i(n[t][0],e))return t;return-1}},function(n,e,t){var i=t(11)(Object,"create");n.exports=i},function(n,e,t){var i=t(178);n.exports=function(n,e){var t=n.__data__;return i(e)?t["string"==typeof e?"string":"hash"]:t.map}},function(n,e,t){var i=t(46);n.exports=function(n){if("string"==typeof n||i(n))return n;var e=n+"";return"0"==e&&1/n==-1/0?"-0":e}},function(n,e,t){var i,a;
/* NProgress, (c) 2013, 2014 Rico Sta. Cruz - http://ricostacruz.com/nprogress
 * @license MIT */void 0===(a="function"==typeof(i=function(){var n,e,t={version:"0.2.0"},i=t.settings={minimum:.08,easing:"ease",positionUsing:"",speed:200,trickle:!0,trickleRate:.02,trickleSpeed:800,showSpinner:!0,barSelector:'[role="bar"]',spinnerSelector:'[role="spinner"]',parent:"body",template:'<div class="bar" role="bar"><div class="peg"></div></div><div class="spinner" role="spinner"><div class="spinner-icon"></div></div>'};function a(n,e,t){return n<e?e:n>t?t:n}function r(n){return 100*(-1+n)}t.configure=function(n){var e,t;for(e in n)void 0!==(t=n[e])&&n.hasOwnProperty(e)&&(i[e]=t);return this},t.status=null,t.set=function(n){var e=t.isStarted();n=a(n,i.minimum,1),t.status=1===n?null:n;var l=t.render(!e),c=l.querySelector(i.barSelector),g=i.speed,d=i.easing;return l.offsetWidth,o((function(e){""===i.positionUsing&&(i.positionUsing=t.getPositioningCSS()),s(c,function(n,e,t){var a;return(a="translate3d"===i.positionUsing?{transform:"translate3d("+r(n)+"%,0,0)"}:"translate"===i.positionUsing?{transform:"translate("+r(n)+"%,0)"}:{"margin-left":r(n)+"%"}).transition="all "+e+"ms "+t,a}(n,g,d)),1===n?(s(l,{transition:"none",opacity:1}),l.offsetWidth,setTimeout((function(){s(l,{transition:"all "+g+"ms linear",opacity:0}),setTimeout((function(){t.remove(),e()}),g)}),g)):setTimeout(e,g)})),this},t.isStarted=function(){return"number"==typeof t.status},t.start=function(){t.status||t.set(0);var n=function(){setTimeout((function(){t.status&&(t.trickle(),n())}),i.trickleSpeed)};return i.trickle&&n(),this},t.done=function(n){return n||t.status?t.inc(.3+.5*Math.random()).set(1):this},t.inc=function(n){var e=t.status;return e?("number"!=typeof n&&(n=(1-e)*a(Math.random()*e,.1,.95)),e=a(e+n,0,.994),t.set(e)):t.start()},t.trickle=function(){return t.inc(Math.random()*i.trickleRate)},n=0,e=0,t.promise=function(i){return i&&"resolved"!==i.state()?(0===e&&t.start(),n++,e++,i.always((function(){0==--e?(n=0,t.done()):t.set((n-e)/n)})),this):this},t.render=function(n){if(t.isRendered())return document.getElementById("nprogress");c(document.documentElement,"nprogress-busy");var e=document.createElement("div");e.id="nprogress",e.innerHTML=i.template;var a,o=e.querySelector(i.barSelector),l=n?"-100":r(t.status||0),g=document.querySelector(i.parent);return s(o,{transition:"all 0 linear",transform:"translate3d("+l+"%,0,0)"}),i.showSpinner||(a=e.querySelector(i.spinnerSelector))&&p(a),g!=document.body&&c(g,"nprogress-custom-parent"),g.appendChild(e),e},t.remove=function(){g(document.documentElement,"nprogress-busy"),g(document.querySelector(i.parent),"nprogress-custom-parent");var n=document.getElementById("nprogress");n&&p(n)},t.isRendered=function(){return!!document.getElementById("nprogress")},t.getPositioningCSS=function(){var n=document.body.style,e="WebkitTransform"in n?"Webkit":"MozTransform"in n?"Moz":"msTransform"in n?"ms":"OTransform"in n?"O":"";return e+"Perspective"in n?"translate3d":e+"Transform"in n?"translate":"margin"};var o=function(){var n=[];function e(){var t=n.shift();t&&t(e)}return function(t){n.push(t),1==n.length&&e()}}(),s=function(){var n=["Webkit","O","Moz","ms"],e={};function t(t){return t=t.replace(/^-ms-/,"ms-").replace(/-([\da-z])/gi,(function(n,e){return e.toUpperCase()})),e[t]||(e[t]=function(e){var t=document.body.style;if(e in t)return e;for(var i,a=n.length,r=e.charAt(0).toUpperCase()+e.slice(1);a--;)if((i=n[a]+r)in t)return i;return e}(t))}function i(n,e,i){e=t(e),n.style[e]=i}return function(n,e){var t,a,r=arguments;if(2==r.length)for(t in e)void 0!==(a=e[t])&&e.hasOwnProperty(t)&&i(n,t,a);else i(n,r[1],r[2])}}();function l(n,e){return("string"==typeof n?n:d(n)).indexOf(" "+e+" ")>=0}function c(n,e){var t=d(n),i=t+e;l(t,e)||(n.className=i.substring(1))}function g(n,e){var t,i=d(n);l(n,e)&&(t=i.replace(" "+e+" "," "),n.className=t.substring(1,t.length-1))}function d(n){return(" "+(n.className||"")+" ").replace(/\s+/gi," ")}function p(n){n&&n.parentNode&&n.parentNode.removeChild(n)}return t})?i.call(e,t,e,n):i)||(n.exports=a)},function(n,e,t){"use strict";var i=t(10),a=String,r=TypeError;n.exports=function(n){if(i(n))return n;throw r(a(n)+" is not an object")}},function(n,e,t){"use strict";var i=t(3),a=t(53).f,r=t(16),o=t(100),s=t(38),l=t(67),c=t(128);n.exports=function(n,e){var t,g,d,p,u,m=n.target,h=n.global,f=n.stat;if(t=h?i:f?i[m]||s(m,{}):(i[m]||{}).prototype)for(g in e){if(p=e[g],d=n.dontCallGetSet?(u=a(t,g))&&u.value:t[g],!c(h?g:m+(f?".":"#")+g,n.forced)&&void 0!==d){if(typeof p==typeof d)continue;l(p,d)}(n.sham||d&&d.sham)&&r(p,"sham",!0),o(t,g,p,n)}}},function(n,e,t){"use strict";var i=t(5);n.exports=!i((function(){var n=function(){}.bind();return"function"!=typeof n||n.hasOwnProperty("prototype")}))},function(n,e,t){"use strict";var i=t(49),a=t(54);n.exports=function(n){return i(a(n))}},function(n,e,t){"use strict";var i=t(3),a=t(2),r=function(n){return a(n)?n:void 0};n.exports=function(n,e){return arguments.length<2?r(i[n]):i[n]&&i[n][e]}},function(n,e,t){"use strict";var i=t(2),a=t(115),r=TypeError;n.exports=function(n){if(i(n))return n;throw r(a(n)+" is not a function")}},function(n,e,t){"use strict";var i=t(3),a=t(63),r=t(9),o=t(65),s=t(61),l=t(60),c=i.Symbol,g=a("wks"),d=l?c.for||c:c&&c.withoutSetter||o;n.exports=function(n){return r(g,n)||(g[n]=s&&r(c,n)?c[n]:d("Symbol."+n)),g[n]}},function(n,e,t){"use strict";var i=t(54),a=Object;n.exports=function(n){return a(i(n))}},function(n,e,t){"use strict";var i=t(126);n.exports=function(n){return i(n.length)}},function(n,e,t){"use strict";var i=t(28),a=Function.prototype.call;n.exports=i?a.bind(a):function(){return a.apply(a,arguments)}},function(n,e,t){"use strict";n.exports=function(n,e){return{enumerable:!(1&n),configurable:!(2&n),writable:!(4&n),value:e}}},function(n,e,t){"use strict";var i=t(3),a=t(38),r=i["__core-js_shared__"]||a("__core-js_shared__",{});n.exports=r},function(n,e,t){"use strict";var i=t(3),a=Object.defineProperty;n.exports=function(n,e){try{a(i,n,{value:e,configurable:!0,writable:!0})}catch(t){i[n]=e}return e}},function(n,e,t){var i=t(150),a=t(14),r=Object.prototype,o=r.hasOwnProperty,s=r.propertyIsEnumerable,l=i(function(){return arguments}())?i:function(n){return a(n)&&o.call(n,"callee")&&!s.call(n,"callee")};n.exports=l},function(n,e,t){var i=t(11)(t(8),"Map");n.exports=i},function(n,e){n.exports=function(n){var e=typeof n;return null!=n&&("object"==e||"function"==e)}},function(n,e,t){var i=t(170),a=t(177),r=t(179),o=t(180),s=t(181);function l(n){var e=-1,t=null==n?0:n.length;for(this.clear();++e<t;){var i=n[e];this.set(i[0],i[1])}}l.prototype.clear=i,l.prototype.delete=a,l.prototype.get=r,l.prototype.has=o,l.prototype.set=s,n.exports=l},function(n,e){n.exports=function(n){var e=-1,t=Array(n.size);return n.forEach((function(n){t[++e]=n})),t}},function(n,e){n.exports=function(n){return"number"==typeof n&&n>-1&&n%1==0&&n<=9007199254740991}},function(n,e,t){var i=t(7),a=t(46),r=/\.|\[(?:[^[\]]*|(["'])(?:(?!\1)[^\\]|\\.)*?\1)\]/,o=/^\w*$/;n.exports=function(n,e){if(i(n))return!1;var t=typeof n;return!("number"!=t&&"symbol"!=t&&"boolean"!=t&&null!=n&&!a(n))||(o.test(n)||!r.test(n)||null!=e&&n in Object(e))}},function(n,e,t){var i=t(15),a=t(14);n.exports=function(n){return"symbol"==typeof n||a(n)&&"[object Symbol]"==i(n)}},function(n,e){n.exports=function(n){return n}},function(n,e,t){"use strict";var i=t(27),a=t(3),r=t(135),o=t(136),s=a.WebAssembly,l=7!==Error("e",{cause:7}).cause,c=function(n,e){var t={};t[n]=o(n,e,l),i({global:!0,constructor:!0,arity:1,forced:l},t)},g=function(n,e){if(s&&s[n]){var t={};t[n]=o("WebAssembly."+n,e,l),i({target:"WebAssembly",stat:!0,constructor:!0,arity:1,forced:l},t)}};c("Error",(function(n){return function(e){return r(n,this,arguments)}})),c("EvalError",(function(n){return function(e){return r(n,this,arguments)}})),c("RangeError",(function(n){return function(e){return r(n,this,arguments)}})),c("ReferenceError",(function(n){return function(e){return r(n,this,arguments)}})),c("SyntaxError",(function(n){return function(e){return r(n,this,arguments)}})),c("TypeError",(function(n){return function(e){return r(n,this,arguments)}})),c("URIError",(function(n){return function(e){return r(n,this,arguments)}})),g("CompileError",(function(n){return function(e){return r(n,this,arguments)}})),g("LinkError",(function(n){return function(e){return r(n,this,arguments)}})),g("RuntimeError",(function(n){return function(e){return r(n,this,arguments)}}))},function(n,e,t){"use strict";var i=t(4),a=t(5),r=t(19),o=Object,s=i("".split);n.exports=a((function(){return!o("z").propertyIsEnumerable(0)}))?function(n){return"String"===r(n)?s(n,""):o(n)}:o},function(n,e,t){"use strict";n.exports={}},function(n,e){n.exports=function(n){return n.webpackPolyfill||(n.deprecate=function(){},n.paths=[],n.children||(n.children=[]),Object.defineProperty(n,"loaded",{enumerable:!0,get:function(){return n.l}}),Object.defineProperty(n,"id",{enumerable:!0,get:function(){return n.i}}),n.webpackPolyfill=1),n}},function(n,e){var t=/^\s+|\s+$/g,i=/^[-+]0x[0-9a-f]+$/i,a=/^0b[01]+$/i,r=/^0o[0-7]+$/i,o=parseInt,s="object"==typeof global&&global&&global.Object===Object&&global,l="object"==typeof self&&self&&self.Object===Object&&self,c=s||l||Function("return this")(),g=Object.prototype.toString,d=Math.max,p=Math.min,u=function(){return c.Date.now()};function m(n){var e=typeof n;return!!n&&("object"==e||"function"==e)}function h(n){if("number"==typeof n)return n;if(function(n){return"symbol"==typeof n||function(n){return!!n&&"object"==typeof n}(n)&&"[object Symbol]"==g.call(n)}(n))return NaN;if(m(n)){var e="function"==typeof n.valueOf?n.valueOf():n;n=m(e)?e+"":e}if("string"!=typeof n)return 0===n?n:+n;n=n.replace(t,"");var s=a.test(n);return s||r.test(n)?o(n.slice(2),s?2:8):i.test(n)?NaN:+n}n.exports=function(n,e,t){var i,a,r,o,s,l,c=0,g=!1,f=!1,b=!0;if("function"!=typeof n)throw new TypeError("Expected a function");function v(e){var t=i,r=a;return i=a=void 0,c=e,o=n.apply(r,t)}function y(n){return c=n,s=setTimeout(C,e),g?v(n):o}function x(n){var t=n-l;return void 0===l||t>=e||t<0||f&&n-c>=r}function C(){var n=u();if(x(n))return A(n);s=setTimeout(C,function(n){var t=e-(n-l);return f?p(t,r-(n-c)):t}(n))}function A(n){return s=void 0,b&&i?v(n):(i=a=void 0,o)}function w(){var n=u(),t=x(n);if(i=arguments,a=this,l=n,t){if(void 0===s)return y(l);if(f)return s=setTimeout(C,e),v(l)}return void 0===s&&(s=setTimeout(C,e)),o}return e=h(e)||0,m(t)&&(g=!!t.leading,r=(f="maxWait"in t)?d(h(t.maxWait)||0,e):r,b="trailing"in t?!!t.trailing:b),w.cancel=function(){void 0!==s&&clearTimeout(s),c=0,i=l=a=s=void 0},w.flush=function(){return void 0===s?o:A(u())},w}},function(n,e,t){"use strict";var i=t(6),a=t(35),r=t(111),o=t(36),s=t(29),l=t(56),c=t(9),g=t(66),d=Object.getOwnPropertyDescriptor;e.f=i?d:function(n,e){if(n=s(n),e=l(e),g)try{return d(n,e)}catch(n){}if(c(n,e))return o(!a(r.f,n,e),n[e])}},function(n,e,t){"use strict";var i=t(55),a=TypeError;n.exports=function(n){if(i(n))throw a("Can't call method on "+n);return n}},function(n,e,t){"use strict";n.exports=function(n){return null==n}},function(n,e,t){"use strict";var i=t(112),a=t(58);n.exports=function(n){var e=i(n,"string");return a(e)?e:e+""}},function(n,e,t){"use strict";var i="object"==typeof document&&document.all,a=void 0===i&&void 0!==i;n.exports={all:i,IS_HTMLDDA:a}},function(n,e,t){"use strict";var i=t(30),a=t(2),r=t(59),o=t(60),s=Object;n.exports=o?function(n){return"symbol"==typeof n}:function(n){var e=i("Symbol");return a(e)&&r(e.prototype,s(n))}},function(n,e,t){"use strict";var i=t(4);n.exports=i({}.isPrototypeOf)},function(n,e,t){"use strict";var i=t(61);n.exports=i&&!Symbol.sham&&"symbol"==typeof Symbol.iterator},function(n,e,t){"use strict";var i=t(62),a=t(5),r=t(3).String;n.exports=!!Object.getOwnPropertySymbols&&!a((function(){var n=Symbol("symbol detection");return!r(n)||!(Object(n)instanceof Symbol)||!Symbol.sham&&i&&i<41}))},function(n,e,t){"use strict";var i,a,r=t(3),o=t(113),s=r.process,l=r.Deno,c=s&&s.versions||l&&l.version,g=c&&c.v8;g&&(a=(i=g.split("."))[0]>0&&i[0]<4?1:+(i[0]+i[1])),!a&&o&&(!(i=o.match(/Edge\/(\d+)/))||i[1]>=74)&&(i=o.match(/Chrome\/(\d+)/))&&(a=+i[1]),n.exports=a},function(n,e,t){"use strict";var i=t(64),a=t(37);(n.exports=function(n,e){return a[n]||(a[n]=void 0!==e?e:{})})("versions",[]).push({version:"3.32.1",mode:i?"pure":"global",copyright:" 2014-2023 Denis Pushkarev (zloirock.ru)",license:"https://github.com/zloirock/core-js/blob/v3.32.1/LICENSE",source:"https://github.com/zloirock/core-js"})},function(n,e,t){"use strict";n.exports=!1},function(n,e,t){"use strict";var i=t(4),a=0,r=Math.random(),o=i(1..toString);n.exports=function(n){return"Symbol("+(void 0===n?"":n)+")_"+o(++a+r,36)}},function(n,e,t){"use strict";var i=t(6),a=t(5),r=t(103);n.exports=!i&&!a((function(){return 7!==Object.defineProperty(r("div"),"a",{get:function(){return 7}}).a}))},function(n,e,t){"use strict";var i=t(9),a=t(121),r=t(53),o=t(18);n.exports=function(n,e,t){for(var s=a(e),l=o.f,c=r.f,g=0;g<s.length;g++){var d=s[g];i(n,d)||t&&i(t,d)||l(n,d,c(e,d))}}},function(n,e,t){"use strict";var i=t(125);n.exports=function(n){var e=+n;return e!=e||0===e?0:i(e)}},function(n,e,t){"use strict";var i=t(137),a=t(26),r=t(138);n.exports=Object.setPrototypeOf||("__proto__"in{}?function(){var n,e=!1,t={};try{(n=i(Object.prototype,"__proto__","set"))(t,[]),e=t instanceof Array}catch(n){}return function(t,i){return a(t),r(i),e?n(t,i):t.__proto__=i,t}}():void 0)},function(n,e){n.exports=function(n,e){for(var t=-1,i=e.length,a=n.length;++t<i;)n[a+t]=e[t];return n}},function(n,e){var t="object"==typeof global&&global&&global.Object===Object&&global;n.exports=t},function(n,e,t){var i=t(20),a=t(161),r=t(162),o=t(163),s=t(164),l=t(165);function c(n){var e=this.__data__=new i(n);this.size=e.size}c.prototype.clear=a,c.prototype.delete=r,c.prototype.get=o,c.prototype.has=s,c.prototype.set=l,n.exports=c},function(n,e){n.exports=function(n,e){return n===e||n!=n&&e!=e}},function(n,e,t){var i=t(15),a=t(41);n.exports=function(n){if(!a(n))return!1;var e=i(n);return"[object Function]"==e||"[object GeneratorFunction]"==e||"[object AsyncFunction]"==e||"[object Proxy]"==e}},function(n,e){var t=Function.prototype.toString;n.exports=function(n){if(null!=n){try{return t.call(n)}catch(n){}try{return n+""}catch(n){}}return""}},function(n,e,t){var i=t(182),a=t(14);n.exports=function n(e,t,r,o,s){return e===t||(null==e||null==t||!a(e)&&!a(t)?e!=e&&t!=t:i(e,t,r,o,n,s))}},function(n,e,t){var i=t(78),a=t(185),r=t(79);n.exports=function(n,e,t,o,s,l){var c=1&t,g=n.length,d=e.length;if(g!=d&&!(c&&d>g))return!1;var p=l.get(n),u=l.get(e);if(p&&u)return p==e&&u==n;var m=-1,h=!0,f=2&t?new i:void 0;for(l.set(n,e),l.set(e,n);++m<g;){var b=n[m],v=e[m];if(o)var y=c?o(v,b,m,e,n,l):o(b,v,m,n,e,l);if(void 0!==y){if(y)continue;h=!1;break}if(f){if(!a(e,(function(n,e){if(!r(f,e)&&(b===n||s(b,n,t,o,l)))return f.push(e)}))){h=!1;break}}else if(b!==v&&!s(b,v,t,o,l)){h=!1;break}}return l.delete(n),l.delete(e),h}},function(n,e,t){var i=t(42),a=t(183),r=t(184);function o(n){var e=-1,t=null==n?0:n.length;for(this.__data__=new i;++e<t;)this.add(n[e])}o.prototype.add=o.prototype.push=a,o.prototype.has=r,n.exports=o},function(n,e){n.exports=function(n,e){return n.has(e)}},function(n,e,t){var i=t(195),a=t(201),r=t(84);n.exports=function(n){return r(n)?i(n):a(n)}},function(n,e,t){(function(n){var i=t(8),a=t(197),r=e&&!e.nodeType&&e,o=r&&"object"==typeof n&&n&&!n.nodeType&&n,s=o&&o.exports===r?i.Buffer:void 0,l=(s?s.isBuffer:void 0)||a;n.exports=l}).call(this,t(51)(n))},function(n,e){var t=/^(?:0|[1-9]\d*)$/;n.exports=function(n,e){var i=typeof n;return!!(e=null==e?9007199254740991:e)&&("number"==i||"symbol"!=i&&t.test(n))&&n>-1&&n%1==0&&n<e}},function(n,e,t){var i=t(198),a=t(199),r=t(200),o=r&&r.isTypedArray,s=o?a(o):i;n.exports=s},function(n,e,t){var i=t(74),a=t(44);n.exports=function(n){return null!=n&&a(n.length)&&!i(n)}},function(n,e,t){var i=t(11)(t(8),"Set");n.exports=i},function(n,e,t){var i=t(41);n.exports=function(n){return n==n&&!i(n)}},function(n,e){n.exports=function(n,e){return function(t){return null!=t&&(t[n]===e&&(void 0!==e||n in Object(t)))}}},function(n,e,t){var i=t(89),a=t(24);n.exports=function(n,e){for(var t=0,r=(e=i(e,n)).length;null!=n&&t<r;)n=n[a(e[t++])];return t&&t==r?n:void 0}},function(n,e,t){var i=t(7),a=t(45),r=t(212),o=t(215);n.exports=function(n,e){return i(n)?n:a(n,e)?[n]:r(o(n))}},function(n,e,t){},function(n,e,t){},function(n,e,t){},function(n,e,t){},function(n,e,t){},function(n,e,t){var i=t(148),a=t(153),r=t(224),o=t(232),s=t(241),l=t(108),c=r((function(n){var e=l(n);return s(e)&&(e=void 0),o(i(n,1,s,!0),a(e,2))}));n.exports=c},function(n,e,t){"use strict";
/*!
 * escape-html
 * Copyright(c) 2012-2013 TJ Holowaychuk
 * Copyright(c) 2015 Andreas Lubbe
 * Copyright(c) 2015 Tiancheng "Timothy" Gu
 * MIT Licensed
 */var i=/["'&<>]/;n.exports=function(n){var e,t=""+n,a=i.exec(t);if(!a)return t;var r="",o=0,s=0;for(o=a.index;o<t.length;o++){switch(t.charCodeAt(o)){case 34:e="&quot;";break;case 38:e="&amp;";break;case 39:e="&#39;";break;case 60:e="&lt;";break;case 62:e="&gt;";break;default:continue}s!==o&&(r+=t.substring(s,o)),s=o+1,r+=e}return s!==o?r+t.substring(s,o):r}},function(n,e,t){var i;"undefined"!=typeof self&&self,i=function(){return function(n){var e={};function t(i){if(e[i])return e[i].exports;var a=e[i]={i:i,l:!1,exports:{}};return n[i].call(a.exports,a,a.exports,t),a.l=!0,a.exports}return t.m=n,t.c=e,t.d=function(n,e,i){t.o(n,e)||Object.defineProperty(n,e,{configurable:!1,enumerable:!0,get:i})},t.n=function(n){var e=n&&n.__esModule?function(){return n.default}:function(){return n};return t.d(e,"a",e),e},t.o=function(n,e){return Object.prototype.hasOwnProperty.call(n,e)},t.p="",t(t.s=7)}([function(n,e,t){n.exports=!t(6)((function(){return 7!=Object.defineProperty({},"a",{get:function(){return 7}}).a}))},function(n,e){n.exports=function(n){return"object"==typeof n?null!==n:"function"==typeof n}},function(n,e){n.exports=function(n,e,t,i,a){var r,o=n=n||{},s=typeof n.default;"object"!==s&&"function"!==s||(r=n,o=n.default);var l,c="function"==typeof o?o.options:o;if(e&&(c.render=e.render,c.staticRenderFns=e.staticRenderFns),i&&(c._scopeId=i),a?(l=function(n){(n=n||this.$vnode&&this.$vnode.ssrContext||this.parent&&this.parent.$vnode&&this.parent.$vnode.ssrContext)||"undefined"==typeof __VUE_SSR_CONTEXT__||(n=__VUE_SSR_CONTEXT__),t&&t.call(this,n),n&&n._registeredComponents&&n._registeredComponents.add(a)},c._ssrRegister=l):t&&(l=t),l){var g=c.functional,d=g?c.render:c.beforeCreate;g?c.render=function(n,e){return l.call(e),d(n,e)}:c.beforeCreate=d?[].concat(d,l):[l]}return{esModule:r,exports:o,options:c}}},function(n,e){var t=n.exports="undefined"!=typeof window&&window.Math==Math?window:"undefined"!=typeof self&&self.Math==Math?self:Function("return this")();"number"==typeof __g&&(__g=t)},function(n,e){var t=n.exports={version:"2.5.7"};"number"==typeof __e&&(__e=t)},function(n,e,t){var i=t(23),a=t(24),r=t(26),o=Object.defineProperty;e.f=t(0)?Object.defineProperty:function(n,e,t){if(i(n),e=r(e,!0),i(t),a)try{return o(n,e,t)}catch(n){}if("get"in t||"set"in t)throw TypeError("Accessors not supported!");return"value"in t&&(n[e]=t.value),n}},function(n,e){n.exports=function(n){try{return!!n()}catch(n){return!0}}},function(n,e,t){var i,a,r;a=[e,t(8),t(11)],void 0===(r="function"==typeof(i=function(n,e,t){"use strict";Object.defineProperty(n,"__esModule",{value:!0}),n.Tabs=n.Tab=void 0;var i=r(e),a=r(t);function r(n){return n&&n.__esModule?n:{default:n}}n.default={install:function(n){n.component("tab",i.default),n.component("tabs",a.default)}},n.Tab=i.default,n.Tabs=a.default})?i.apply(e,a):i)||(n.exports=r)},function(n,e,t){var i=t(2)(t(9),t(10),null,null,null);n.exports=i.exports},function(n,e,t){var i,a,r;a=[e],void 0===(r="function"==typeof(i=function(n){"use strict";Object.defineProperty(n,"__esModule",{value:!0}),n.default={props:{id:{default:null},name:{required:!0},prefix:{default:""},suffix:{default:""},isDisabled:{default:!1}},data:function(){return{isActive:!1,isVisible:!0}},computed:{header:function(){return this.prefix+this.name+this.suffix},computedId:function(){return this.id?this.id:this.name.toLowerCase().replace(/ /g,"-")},hash:function(){return this.isDisabled?"#":"#"+this.computedId}}}})?i.apply(e,a):i)||(n.exports=r)},function(n,e){n.exports={render:function(){var n=this.$createElement;return(this._self._c||n)("section",{directives:[{name:"show",rawName:"v-show",value:this.isActive,expression:"isActive"}],staticClass:"tabs-component-panel",attrs:{"aria-hidden":!this.isActive,id:this.computedId,role:"tabpanel"}},[this._t("default")],2)},staticRenderFns:[]}},function(n,e,t){var i=t(2)(t(12),t(29),null,null,null);n.exports=i.exports},function(n,e,t){var i,a,r;a=[e,t(13)],void 0===(r="function"==typeof(i=function(n,e){"use strict";Object.defineProperty(n,"__esModule",{value:!0});var t,i=(t=e)&&t.__esModule?t:{default:t};n.default={props:{cacheLifetime:{default:5},options:{type:Object,required:!1,default:function(){return{useUrlFragment:!0,defaultTabHash:null}}}},data:function(){return{tabs:[],activeTabHash:"",activeTabIndex:0,lastActiveTabHash:""}},computed:{storageKey:function(){return"vue-tabs-component.cache."+window.location.host+window.location.pathname}},created:function(){this.tabs=this.$children},mounted:function(){var n=this;if(window.addEventListener("hashchange",(function(){return n.selectTab(window.location.hash)})),this.findTab(window.location.hash))this.selectTab(window.location.hash);else{var e=i.default.get(this.storageKey);this.findTab(e)?this.selectTab(e):null!==this.options.defaultTabHash&&this.findTab("#"+this.options.defaultTabHash)?this.selectTab("#"+this.options.defaultTabHash):this.tabs.length&&this.selectTab(this.tabs[0].hash)}},methods:{findTab:function(n){return this.tabs.find((function(e){return e.hash===n}))},selectTab:function(n,e){e&&!this.options.useUrlFragment&&e.preventDefault();var t=this.findTab(n);t&&(t.isDisabled?e.preventDefault():this.lastActiveTabHash!==t.hash?(this.tabs.forEach((function(n){n.isActive=n.hash===t.hash})),this.$emit("changed",{tab:t}),this.activeTabHash=t.hash,this.activeTabIndex=this.getTabIndex(n),this.lastActiveTabHash=this.activeTabHash=t.hash,i.default.set(this.storageKey,t.hash,this.cacheLifetime)):this.$emit("clicked",{tab:t}))},setTabVisible:function(n,e){var t=this.findTab(n);t&&(t.isVisible=e,t.isActive&&(t.isActive=e,this.tabs.every((function(n,e,t){return!n.isVisible||(n.isActive=!0,!1)}))))},getTabIndex:function(n){var e=this.findTab(n);return this.tabs.indexOf(e)},getTabHash:function(n){var e=this,t=this.tabs.find((function(t){return e.tabs.indexOf(t)===n}));if(t)return t.hash},getActiveTab:function(){return this.findTab(this.activeTabHash)},getActiveTabIndex:function(){return this.getTabIndex(this.activeTabHash)}}}})?i.apply(e,a):i)||(n.exports=r)},function(n,e,t){var i,a,r;a=[e,t(14),t(15)],void 0===(r="function"==typeof(i=function(n,e,t){"use strict";Object.defineProperty(n,"__esModule",{value:!0});var i=r(e),a=r(t);function r(n){return n&&n.__esModule?n:{default:n}}var o=function(){function n(){(0,i.default)(this,n)}return(0,a.default)(n,[{key:"get",value:function(n){var e=JSON.parse(localStorage.getItem(n));return e?new Date(e.expires)<new Date?(localStorage.removeItem(n),null):e.value:null}},{key:"set",value:function(n,e,t){var i=(new Date).getTime(),a=new Date(i+6e4*t);localStorage.setItem(n,JSON.stringify({value:e,expires:a}))}}]),n}();n.default=new o})?i.apply(e,a):i)||(n.exports=r)},function(n,e,t){"use strict";e.__esModule=!0,e.default=function(n,e){if(!(n instanceof e))throw new TypeError("Cannot call a class as a function")}},function(n,e,t){"use strict";e.__esModule=!0;var i,a=t(16),r=(i=a)&&i.__esModule?i:{default:i};e.default=function(){function n(n,e){for(var t=0;t<e.length;t++){var i=e[t];i.enumerable=i.enumerable||!1,i.configurable=!0,"value"in i&&(i.writable=!0),(0,r.default)(n,i.key,i)}}return function(e,t,i){return t&&n(e.prototype,t),i&&n(e,i),e}}()},function(n,e,t){n.exports={default:t(17),__esModule:!0}},function(n,e,t){t(18);var i=t(4).Object;n.exports=function(n,e,t){return i.defineProperty(n,e,t)}},function(n,e,t){var i=t(19);i(i.S+i.F*!t(0),"Object",{defineProperty:t(5).f})},function(n,e,t){var i=t(3),a=t(4),r=t(20),o=t(22),s=t(28),l=function(n,e,t){var c,g,d,p=n&l.F,u=n&l.G,m=n&l.S,h=n&l.P,f=n&l.B,b=n&l.W,v=u?a:a[e]||(a[e]={}),y=v.prototype,x=u?i:m?i[e]:(i[e]||{}).prototype;for(c in u&&(t=e),t)(g=!p&&x&&void 0!==x[c])&&s(v,c)||(d=g?x[c]:t[c],v[c]=u&&"function"!=typeof x[c]?t[c]:f&&g?r(d,i):b&&x[c]==d?function(n){var e=function(e,t,i){if(this instanceof n){switch(arguments.length){case 0:return new n;case 1:return new n(e);case 2:return new n(e,t)}return new n(e,t,i)}return n.apply(this,arguments)};return e.prototype=n.prototype,e}(d):h&&"function"==typeof d?r(Function.call,d):d,h&&((v.virtual||(v.virtual={}))[c]=d,n&l.R&&y&&!y[c]&&o(y,c,d)))};l.F=1,l.G=2,l.S=4,l.P=8,l.B=16,l.W=32,l.U=64,l.R=128,n.exports=l},function(n,e,t){var i=t(21);n.exports=function(n,e,t){if(i(n),void 0===e)return n;switch(t){case 1:return function(t){return n.call(e,t)};case 2:return function(t,i){return n.call(e,t,i)};case 3:return function(t,i,a){return n.call(e,t,i,a)}}return function(){return n.apply(e,arguments)}}},function(n,e){n.exports=function(n){if("function"!=typeof n)throw TypeError(n+" is not a function!");return n}},function(n,e,t){var i=t(5),a=t(27);n.exports=t(0)?function(n,e,t){return i.f(n,e,a(1,t))}:function(n,e,t){return n[e]=t,n}},function(n,e,t){var i=t(1);n.exports=function(n){if(!i(n))throw TypeError(n+" is not an object!");return n}},function(n,e,t){n.exports=!t(0)&&!t(6)((function(){return 7!=Object.defineProperty(t(25)("div"),"a",{get:function(){return 7}}).a}))},function(n,e,t){var i=t(1),a=t(3).document,r=i(a)&&i(a.createElement);n.exports=function(n){return r?a.createElement(n):{}}},function(n,e,t){var i=t(1);n.exports=function(n,e){if(!i(n))return n;var t,a;if(e&&"function"==typeof(t=n.toString)&&!i(a=t.call(n)))return a;if("function"==typeof(t=n.valueOf)&&!i(a=t.call(n)))return a;if(!e&&"function"==typeof(t=n.toString)&&!i(a=t.call(n)))return a;throw TypeError("Can't convert object to primitive value")}},function(n,e){n.exports=function(n,e){return{enumerable:!(1&n),configurable:!(2&n),writable:!(4&n),value:e}}},function(n,e){var t={}.hasOwnProperty;n.exports=function(n,e){return t.call(n,e)}},function(n,e){n.exports={render:function(){var n=this,e=n.$createElement,t=n._self._c||e;return t("div",{staticClass:"tabs-component"},[t("ul",{staticClass:"tabs-component-tabs",attrs:{role:"tablist"}},n._l(n.tabs,(function(e,i){return t("li",{directives:[{name:"show",rawName:"v-show",value:e.isVisible,expression:"tab.isVisible"}],key:i,staticClass:"tabs-component-tab",class:{"is-active":e.isActive,"is-disabled":e.isDisabled},attrs:{role:"presentation"}},[t("a",{staticClass:"tabs-component-tab-a",attrs:{"aria-controls":e.hash,"aria-selected":e.isActive,href:e.hash,role:"tab"},domProps:{innerHTML:n._s(e.header)},on:{click:function(t){n.selectTab(e.hash,t)}}})])}))),n._v(" "),t("div",{staticClass:"tabs-component-panels"},[n._t("default")],2)])},staticRenderFns:[]}}])},n.exports=i()},function(n,e,t){"use strict";t.r(e);var i={name:"CodeBlock",props:{title:{type:String,required:!0},active:{type:Boolean,default:!1}}},a=(t(244),t(1)),r=Object(a.a)(i,(function(){var n=this.$createElement;return(this._self._c||n)("div",{staticClass:"theme-code-block",class:{"theme-code-block__active":this.active}},[this._t("default")],2)}),[],!1,null,"4f1e9d0c",null);e.default=r.exports},function(n,e,t){"use strict";t.r(e);var i={name:"CodeGroup",data:()=>({codeTabs:[],activeCodeTabIndex:-1}),watch:{activeCodeTabIndex(n){this.codeTabs.forEach(n=>{n.elm.classList.remove("theme-code-block__active")}),this.codeTabs[n].elm.classList.add("theme-code-block__active")}},mounted(){this.codeTabs=(this.$slots.default||[]).filter(n=>Boolean(n.componentOptions)).map((n,e)=>(""===n.componentOptions.propsData.active&&(this.activeCodeTabIndex=e),{title:n.componentOptions.propsData.title,elm:n.elm})),-1===this.activeCodeTabIndex&&this.codeTabs.length>0&&(this.activeCodeTabIndex=0)},methods:{changeCodeTab(n){this.activeCodeTabIndex=n}}},a=(t(245),t(1)),r=Object(a.a)(i,(function(){var n=this,e=n.$createElement,t=n._self._c||e;return t("div",{staticClass:"theme-code-group"},[t("div",{staticClass:"theme-code-group__nav"},[t("ul",{staticClass:"theme-code-group__ul"},n._l(n.codeTabs,(function(e,i){return t("li",{key:e.title,staticClass:"theme-code-group__li"},[t("button",{staticClass:"theme-code-group__nav-tab",class:{"theme-code-group__nav-tab-active":i===n.activeCodeTabIndex},on:{click:function(e){return n.changeCodeTab(i)}}},[n._v("\n            "+n._s(e.title)+"\n          ")])])})),0)]),n._v(" "),n._t("default"),n._v(" "),n.codeTabs.length<1?t("pre",{staticClass:"pre-blank"},[n._v("// Make sure to add code blocks to your code group")]):n._e()],2)}),[],!1,null,"2f5f1757",null);e.default=r.exports},function(n,e,t){"use strict";var i=t(2),a=t(18),r=t(105),o=t(38);n.exports=function(n,e,t,s){s||(s={});var l=s.enumerable,c=void 0!==s.name?s.name:e;if(i(t)&&r(t,c,s),s.global)l?n[e]=t:o(e,t);else{try{s.unsafe?n[e]&&(l=!0):delete n[e]}catch(n){}l?n[e]=t:a.f(n,e,{value:t,enumerable:!1,configurable:!s.nonConfigurable,writable:!s.nonWritable})}return n}},function(n,e,t){"use strict";n.exports=["constructor","hasOwnProperty","isPrototypeOf","propertyIsEnumerable","toLocaleString","toString","valueOf"]},function(n,e,t){"use strict";var i=t(142),a=String;n.exports=function(n){if("Symbol"===i(n))throw TypeError("Cannot convert a Symbol value to a string");return a(n)}},function(n,e,t){"use strict";var i=t(3),a=t(10),r=i.document,o=a(r)&&a(r.createElement);n.exports=function(n){return o?r.createElement(n):{}}},function(n,e,t){"use strict";var i=t(6),a=t(5);n.exports=i&&a((function(){return 42!==Object.defineProperty((function(){}),"prototype",{value:42,writable:!1}).prototype}))},function(n,e,t){"use strict";var i=t(4),a=t(5),r=t(2),o=t(9),s=t(6),l=t(117).CONFIGURABLE,c=t(118),g=t(119),d=g.enforce,p=g.get,u=String,m=Object.defineProperty,h=i("".slice),f=i("".replace),b=i([].join),v=s&&!a((function(){return 8!==m((function(){}),"length",{value:8}).length})),y=String(String).split("String"),x=n.exports=function(n,e,t){"Symbol("===h(u(e),0,7)&&(e="["+f(u(e),/^Symbol\(([^)]*)\)/,"$1")+"]"),t&&t.getter&&(e="get "+e),t&&t.setter&&(e="set "+e),(!o(n,"name")||l&&n.name!==e)&&(s?m(n,"name",{value:e,configurable:!0}):n.name=e),v&&t&&o(t,"arity")&&n.length!==t.arity&&m(n,"length",{value:t.arity});try{t&&o(t,"constructor")&&t.constructor?s&&m(n,"prototype",{writable:!1}):n.prototype&&(n.prototype=void 0)}catch(n){}var i=d(n);return o(i,"source")||(i.source=b(y,"string"==typeof e?e:"")),n};Function.prototype.toString=x((function(){return r(this)&&p(this).source||c(this)}),"toString")},function(n,e,t){"use strict";var i=t(63),a=t(65),r=i("keys");n.exports=function(n){return r[n]||(r[n]=a(n))}},function(n,e,t){"use strict";var i=t(4),a=t(9),r=t(29),o=t(123).indexOf,s=t(50),l=i([].push);n.exports=function(n,e){var t,i=r(n),c=0,g=[];for(t in i)!a(s,t)&&a(i,t)&&l(g,t);for(;e.length>c;)a(i,t=e[c++])&&(~o(g,t)||l(g,t));return g}},function(n,e){n.exports=function(n){var e=null==n?0:n.length;return e?n[e-1]:void 0}},function(n,e,t){n.exports=t(252)},function(n,e,t){"use strict";var i=t(27),a=t(129).left,r=t(130),o=t(62);i({target:"Array",proto:!0,forced:!t(131)&&o>79&&o<83||!r("reduce")},{reduce:function(n){var e=arguments.length;return a(this,n,e,e>1?arguments[1]:void 0)}})},function(n,e,t){"use strict";var i={}.propertyIsEnumerable,a=Object.getOwnPropertyDescriptor,r=a&&!i.call({1:2},1);e.f=r?function(n){var e=a(this,n);return!!e&&e.enumerable}:i},function(n,e,t){"use strict";var i=t(35),a=t(10),r=t(58),o=t(114),s=t(116),l=t(32),c=TypeError,g=l("toPrimitive");n.exports=function(n,e){if(!a(n)||r(n))return n;var t,l=o(n,g);if(l){if(void 0===e&&(e="default"),t=i(l,n,e),!a(t)||r(t))return t;throw c("Can't convert object to primitive value")}return void 0===e&&(e="number"),s(n,e)}},function(n,e,t){"use strict";n.exports="undefined"!=typeof navigator&&String(navigator.userAgent)||""},function(n,e,t){"use strict";var i=t(31),a=t(55);n.exports=function(n,e){var t=n[e];return a(t)?void 0:i(t)}},function(n,e,t){"use strict";var i=String;n.exports=function(n){try{return i(n)}catch(n){return"Object"}}},function(n,e,t){"use strict";var i=t(35),a=t(2),r=t(10),o=TypeError;n.exports=function(n,e){var t,s;if("string"===e&&a(t=n.toString)&&!r(s=i(t,n)))return s;if(a(t=n.valueOf)&&!r(s=i(t,n)))return s;if("string"!==e&&a(t=n.toString)&&!r(s=i(t,n)))return s;throw o("Can't convert object to primitive value")}},function(n,e,t){"use strict";var i=t(6),a=t(9),r=Function.prototype,o=i&&Object.getOwnPropertyDescriptor,s=a(r,"name"),l=s&&"something"===function(){}.name,c=s&&(!i||i&&o(r,"name").configurable);n.exports={EXISTS:s,PROPER:l,CONFIGURABLE:c}},function(n,e,t){"use strict";var i=t(4),a=t(2),r=t(37),o=i(Function.toString);a(r.inspectSource)||(r.inspectSource=function(n){return o(n)}),n.exports=r.inspectSource},function(n,e,t){"use strict";var i,a,r,o=t(120),s=t(3),l=t(10),c=t(16),g=t(9),d=t(37),p=t(106),u=t(50),m=s.TypeError,h=s.WeakMap;if(o||d.state){var f=d.state||(d.state=new h);f.get=f.get,f.has=f.has,f.set=f.set,i=function(n,e){if(f.has(n))throw m("Object already initialized");return e.facade=n,f.set(n,e),e},a=function(n){return f.get(n)||{}},r=function(n){return f.has(n)}}else{var b=p("state");u[b]=!0,i=function(n,e){if(g(n,b))throw m("Object already initialized");return e.facade=n,c(n,b,e),e},a=function(n){return g(n,b)?n[b]:{}},r=function(n){return g(n,b)}}n.exports={set:i,get:a,has:r,enforce:function(n){return r(n)?a(n):i(n,{})},getterFor:function(n){return function(e){var t;if(!l(e)||(t=a(e)).type!==n)throw m("Incompatible receiver, "+n+" required");return t}}}},function(n,e,t){"use strict";var i=t(3),a=t(2),r=i.WeakMap;n.exports=a(r)&&/native code/.test(String(r))},function(n,e,t){"use strict";var i=t(30),a=t(4),r=t(122),o=t(127),s=t(26),l=a([].concat);n.exports=i("Reflect","ownKeys")||function(n){var e=r.f(s(n)),t=o.f;return t?l(e,t(n)):e}},function(n,e,t){"use strict";var i=t(107),a=t(101).concat("length","prototype");e.f=Object.getOwnPropertyNames||function(n){return i(n,a)}},function(n,e,t){"use strict";var i=t(29),a=t(124),r=t(34),o=function(n){return function(e,t,o){var s,l=i(e),c=r(l),g=a(o,c);if(n&&t!=t){for(;c>g;)if((s=l[g++])!=s)return!0}else for(;c>g;g++)if((n||g in l)&&l[g]===t)return n||g||0;return!n&&-1}};n.exports={includes:o(!0),indexOf:o(!1)}},function(n,e,t){"use strict";var i=t(68),a=Math.max,r=Math.min;n.exports=function(n,e){var t=i(n);return t<0?a(t+e,0):r(t,e)}},function(n,e,t){"use strict";var i=Math.ceil,a=Math.floor;n.exports=Math.trunc||function(n){var e=+n;return(e>0?a:i)(e)}},function(n,e,t){"use strict";var i=t(68),a=Math.min;n.exports=function(n){return n>0?a(i(n),9007199254740991):0}},function(n,e,t){"use strict";e.f=Object.getOwnPropertySymbols},function(n,e,t){"use strict";var i=t(5),a=t(2),r=/#|\.prototype\./,o=function(n,e){var t=l[s(n)];return t===g||t!==c&&(a(e)?i(e):!!e)},s=o.normalize=function(n){return String(n).replace(r,".").toLowerCase()},l=o.data={},c=o.NATIVE="N",g=o.POLYFILL="P";n.exports=o},function(n,e,t){"use strict";var i=t(31),a=t(33),r=t(49),o=t(34),s=TypeError,l=function(n){return function(e,t,l,c){i(t);var g=a(e),d=r(g),p=o(g),u=n?p-1:0,m=n?-1:1;if(l<2)for(;;){if(u in d){c=d[u],u+=m;break}if(u+=m,n?u<0:p<=u)throw s("Reduce of empty array with no initial value")}for(;n?u>=0:p>u;u+=m)u in d&&(c=t(c,d[u],u,g));return c}};n.exports={left:l(!1),right:l(!0)}},function(n,e,t){"use strict";var i=t(5);n.exports=function(n,e){var t=[][n];return!!t&&i((function(){t.call(null,e||function(){return 1},1)}))}},function(n,e,t){"use strict";var i=t(3),a=t(19);n.exports="process"===a(i.process)},function(n,e,t){"use strict";var i=t(6),a=t(133),r=TypeError,o=Object.getOwnPropertyDescriptor,s=i&&!function(){if(void 0!==this)return!0;try{Object.defineProperty([],"length",{writable:!1}).length=1}catch(n){return n instanceof TypeError}}();n.exports=s?function(n,e){if(a(n)&&!o(n,"length").writable)throw r("Cannot set read only .length");return n.length=e}:function(n,e){return n.length=e}},function(n,e,t){"use strict";var i=t(19);n.exports=Array.isArray||function(n){return"Array"===i(n)}},function(n,e,t){"use strict";var i=TypeError;n.exports=function(n){if(n>9007199254740991)throw i("Maximum allowed index exceeded");return n}},function(n,e,t){"use strict";var i=t(28),a=Function.prototype,r=a.apply,o=a.call;n.exports="object"==typeof Reflect&&Reflect.apply||(i?o.bind(r):function(){return o.apply(r,arguments)})},function(n,e,t){"use strict";var i=t(30),a=t(9),r=t(16),o=t(59),s=t(69),l=t(67),c=t(139),g=t(140),d=t(141),p=t(144),u=t(145),m=t(6),h=t(64);n.exports=function(n,e,t,f){var b=f?2:1,v=n.split("."),y=v[v.length-1],x=i.apply(null,v);if(x){var C=x.prototype;if(!h&&a(C,"cause")&&delete C.cause,!t)return x;var A=i("Error"),w=e((function(n,e){var t=d(f?e:n,void 0),i=f?new x(n):new x;return void 0!==t&&r(i,"message",t),u(i,w,i.stack,2),this&&o(C,this)&&g(i,this,w),arguments.length>b&&p(i,arguments[b]),i}));if(w.prototype=C,"Error"!==y?s?s(w,A):l(w,A,{name:!0}):m&&"stackTraceLimit"in x&&(c(w,x,"stackTraceLimit"),c(w,x,"prepareStackTrace")),l(w,x),!h)try{C.name!==y&&r(C,"name",y),C.constructor=w}catch(n){}return w}}},function(n,e,t){"use strict";var i=t(4),a=t(31);n.exports=function(n,e,t){try{return i(a(Object.getOwnPropertyDescriptor(n,e)[t]))}catch(n){}}},function(n,e,t){"use strict";var i=t(2),a=String,r=TypeError;n.exports=function(n){if("object"==typeof n||i(n))return n;throw r("Can't set "+a(n)+" as a prototype")}},function(n,e,t){"use strict";var i=t(18).f;n.exports=function(n,e,t){t in n||i(n,t,{configurable:!0,get:function(){return e[t]},set:function(n){e[t]=n}})}},function(n,e,t){"use strict";var i=t(2),a=t(10),r=t(69);n.exports=function(n,e,t){var o,s;return r&&i(o=e.constructor)&&o!==t&&a(s=o.prototype)&&s!==t.prototype&&r(n,s),n}},function(n,e,t){"use strict";var i=t(102);n.exports=function(n,e){return void 0===n?arguments.length<2?"":e:i(n)}},function(n,e,t){"use strict";var i=t(143),a=t(2),r=t(19),o=t(32)("toStringTag"),s=Object,l="Arguments"===r(function(){return arguments}());n.exports=i?r:function(n){var e,t,i;return void 0===n?"Undefined":null===n?"Null":"string"==typeof(t=function(n,e){try{return n[e]}catch(n){}}(e=s(n),o))?t:l?r(e):"Object"===(i=r(e))&&a(e.callee)?"Arguments":i}},function(n,e,t){"use strict";var i={};i[t(32)("toStringTag")]="z",n.exports="[object z]"===String(i)},function(n,e,t){"use strict";var i=t(10),a=t(16);n.exports=function(n,e){i(e)&&"cause"in e&&a(n,"cause",e.cause)}},function(n,e,t){"use strict";var i=t(16),a=t(146),r=t(147),o=Error.captureStackTrace;n.exports=function(n,e,t,s){r&&(o?o(n,e):i(n,"stack",a(t,s)))}},function(n,e,t){"use strict";var i=t(4),a=Error,r=i("".replace),o=String(a("zxcasd").stack),s=/\n\s*at [^:]*:[^\n]*/,l=s.test(o);n.exports=function(n,e){if(l&&"string"==typeof n&&!a.prepareStackTrace)for(;e--;)n=r(n,s,"");return n}},function(n,e,t){"use strict";var i=t(5),a=t(36);n.exports=!i((function(){var n=Error("a");return!("stack"in n)||(Object.defineProperty(n,"stack",a(1,7)),7!==n.stack)}))},function(n,e,t){var i=t(70),a=t(149);n.exports=function n(e,t,r,o,s){var l=-1,c=e.length;for(r||(r=a),s||(s=[]);++l<c;){var g=e[l];t>0&&r(g)?t>1?n(g,t-1,r,o,s):i(s,g):o||(s[s.length]=g)}return s}},function(n,e,t){var i=t(17),a=t(39),r=t(7),o=i?i.isConcatSpreadable:void 0;n.exports=function(n){return r(n)||a(n)||!!(o&&n&&n[o])}},function(n,e,t){var i=t(15),a=t(14);n.exports=function(n){return a(n)&&"[object Arguments]"==i(n)}},function(n,e,t){var i=t(17),a=Object.prototype,r=a.hasOwnProperty,o=a.toString,s=i?i.toStringTag:void 0;n.exports=function(n){var e=r.call(n,s),t=n[s];try{n[s]=void 0;var i=!0}catch(n){}var a=o.call(n);return i&&(e?n[s]=t:delete n[s]),a}},function(n,e){var t=Object.prototype.toString;n.exports=function(n){return t.call(n)}},function(n,e,t){var i=t(154),a=t(210),r=t(47),o=t(7),s=t(221);n.exports=function(n){return"function"==typeof n?n:null==n?r:"object"==typeof n?o(n)?a(n[0],n[1]):i(n):s(n)}},function(n,e,t){var i=t(155),a=t(209),r=t(87);n.exports=function(n){var e=a(n);return 1==e.length&&e[0][2]?r(e[0][0],e[0][1]):function(t){return t===n||i(t,n,e)}}},function(n,e,t){var i=t(72),a=t(76);n.exports=function(n,e,t,r){var o=t.length,s=o,l=!r;if(null==n)return!s;for(n=Object(n);o--;){var c=t[o];if(l&&c[2]?c[1]!==n[c[0]]:!(c[0]in n))return!1}for(;++o<s;){var g=(c=t[o])[0],d=n[g],p=c[1];if(l&&c[2]){if(void 0===d&&!(g in n))return!1}else{var u=new i;if(r)var m=r(d,p,g,n,e,u);if(!(void 0===m?a(p,d,3,r,u):m))return!1}}return!0}},function(n,e){n.exports=function(){this.__data__=[],this.size=0}},function(n,e,t){var i=t(21),a=Array.prototype.splice;n.exports=function(n){var e=this.__data__,t=i(e,n);return!(t<0)&&(t==e.length-1?e.pop():a.call(e,t,1),--this.size,!0)}},function(n,e,t){var i=t(21);n.exports=function(n){var e=this.__data__,t=i(e,n);return t<0?void 0:e[t][1]}},function(n,e,t){var i=t(21);n.exports=function(n){return i(this.__data__,n)>-1}},function(n,e,t){var i=t(21);n.exports=function(n,e){var t=this.__data__,a=i(t,n);return a<0?(++this.size,t.push([n,e])):t[a][1]=e,this}},function(n,e,t){var i=t(20);n.exports=function(){this.__data__=new i,this.size=0}},function(n,e){n.exports=function(n){var e=this.__data__,t=e.delete(n);return this.size=e.size,t}},function(n,e){n.exports=function(n){return this.__data__.get(n)}},function(n,e){n.exports=function(n){return this.__data__.has(n)}},function(n,e,t){var i=t(20),a=t(40),r=t(42);n.exports=function(n,e){var t=this.__data__;if(t instanceof i){var o=t.__data__;if(!a||o.length<199)return o.push([n,e]),this.size=++t.size,this;t=this.__data__=new r(o)}return t.set(n,e),this.size=t.size,this}},function(n,e,t){var i=t(74),a=t(167),r=t(41),o=t(75),s=/^\[object .+?Constructor\]$/,l=Function.prototype,c=Object.prototype,g=l.toString,d=c.hasOwnProperty,p=RegExp("^"+g.call(d).replace(/[\\^$.*+?()[\]{}|]/g,"\\$&").replace(/hasOwnProperty|(function).*?(?=\\\()| for .+?(?=\\\])/g,"$1.*?")+"$");n.exports=function(n){return!(!r(n)||a(n))&&(i(n)?p:s).test(o(n))}},function(n,e,t){var i,a=t(168),r=(i=/[^.]+$/.exec(a&&a.keys&&a.keys.IE_PROTO||""))?"Symbol(src)_1."+i:"";n.exports=function(n){return!!r&&r in n}},function(n,e,t){var i=t(8)["__core-js_shared__"];n.exports=i},function(n,e){n.exports=function(n,e){return null==n?void 0:n[e]}},function(n,e,t){var i=t(171),a=t(20),r=t(40);n.exports=function(){this.size=0,this.__data__={hash:new i,map:new(r||a),string:new i}}},function(n,e,t){var i=t(172),a=t(173),r=t(174),o=t(175),s=t(176);function l(n){var e=-1,t=null==n?0:n.length;for(this.clear();++e<t;){var i=n[e];this.set(i[0],i[1])}}l.prototype.clear=i,l.prototype.delete=a,l.prototype.get=r,l.prototype.has=o,l.prototype.set=s,n.exports=l},function(n,e,t){var i=t(22);n.exports=function(){this.__data__=i?i(null):{},this.size=0}},function(n,e){n.exports=function(n){var e=this.has(n)&&delete this.__data__[n];return this.size-=e?1:0,e}},function(n,e,t){var i=t(22),a=Object.prototype.hasOwnProperty;n.exports=function(n){var e=this.__data__;if(i){var t=e[n];return"__lodash_hash_undefined__"===t?void 0:t}return a.call(e,n)?e[n]:void 0}},function(n,e,t){var i=t(22),a=Object.prototype.hasOwnProperty;n.exports=function(n){var e=this.__data__;return i?void 0!==e[n]:a.call(e,n)}},function(n,e,t){var i=t(22);n.exports=function(n,e){var t=this.__data__;return this.size+=this.has(n)?0:1,t[n]=i&&void 0===e?"__lodash_hash_undefined__":e,this}},function(n,e,t){var i=t(23);n.exports=function(n){var e=i(this,n).delete(n);return this.size-=e?1:0,e}},function(n,e){n.exports=function(n){var e=typeof n;return"string"==e||"number"==e||"symbol"==e||"boolean"==e?"__proto__"!==n:null===n}},function(n,e,t){var i=t(23);n.exports=function(n){return i(this,n).get(n)}},function(n,e,t){var i=t(23);n.exports=function(n){return i(this,n).has(n)}},function(n,e,t){var i=t(23);n.exports=function(n,e){var t=i(this,n),a=t.size;return t.set(n,e),this.size+=t.size==a?0:1,this}},function(n,e,t){var i=t(72),a=t(77),r=t(186),o=t(189),s=t(205),l=t(7),c=t(81),g=t(83),d="[object Object]",p=Object.prototype.hasOwnProperty;n.exports=function(n,e,t,u,m,h){var f=l(n),b=l(e),v=f?"[object Array]":s(n),y=b?"[object Array]":s(e),x=(v="[object Arguments]"==v?d:v)==d,C=(y="[object Arguments]"==y?d:y)==d,A=v==y;if(A&&c(n)){if(!c(e))return!1;f=!0,x=!1}if(A&&!x)return h||(h=new i),f||g(n)?a(n,e,t,u,m,h):r(n,e,v,t,u,m,h);if(!(1&t)){var w=x&&p.call(n,"__wrapped__"),k=C&&p.call(e,"__wrapped__");if(w||k){var T=w?n.value():n,E=k?e.value():e;return h||(h=new i),m(T,E,t,u,h)}}return!!A&&(h||(h=new i),o(n,e,t,u,m,h))}},function(n,e){n.exports=function(n){return this.__data__.set(n,"__lodash_hash_undefined__"),this}},function(n,e){n.exports=function(n){return this.__data__.has(n)}},function(n,e){n.exports=function(n,e){for(var t=-1,i=null==n?0:n.length;++t<i;)if(e(n[t],t,n))return!0;return!1}},function(n,e,t){var i=t(17),a=t(187),r=t(73),o=t(77),s=t(188),l=t(43),c=i?i.prototype:void 0,g=c?c.valueOf:void 0;n.exports=function(n,e,t,i,c,d,p){switch(t){case"[object DataView]":if(n.byteLength!=e.byteLength||n.byteOffset!=e.byteOffset)return!1;n=n.buffer,e=e.buffer;case"[object ArrayBuffer]":return!(n.byteLength!=e.byteLength||!d(new a(n),new a(e)));case"[object Boolean]":case"[object Date]":case"[object Number]":return r(+n,+e);case"[object Error]":return n.name==e.name&&n.message==e.message;case"[object RegExp]":case"[object String]":return n==e+"";case"[object Map]":var u=s;case"[object Set]":var m=1&i;if(u||(u=l),n.size!=e.size&&!m)return!1;var h=p.get(n);if(h)return h==e;i|=2,p.set(n,e);var f=o(u(n),u(e),i,c,d,p);return p.delete(n),f;case"[object Symbol]":if(g)return g.call(n)==g.call(e)}return!1}},function(n,e,t){var i=t(8).Uint8Array;n.exports=i},function(n,e){n.exports=function(n){var e=-1,t=Array(n.size);return n.forEach((function(n,i){t[++e]=[i,n]})),t}},function(n,e,t){var i=t(190),a=Object.prototype.hasOwnProperty;n.exports=function(n,e,t,r,o,s){var l=1&t,c=i(n),g=c.length;if(g!=i(e).length&&!l)return!1;for(var d=g;d--;){var p=c[d];if(!(l?p in e:a.call(e,p)))return!1}var u=s.get(n),m=s.get(e);if(u&&m)return u==e&&m==n;var h=!0;s.set(n,e),s.set(e,n);for(var f=l;++d<g;){var b=n[p=c[d]],v=e[p];if(r)var y=l?r(v,b,p,e,n,s):r(b,v,p,n,e,s);if(!(void 0===y?b===v||o(b,v,t,r,s):y)){h=!1;break}f||(f="constructor"==p)}if(h&&!f){var x=n.constructor,C=e.constructor;x==C||!("constructor"in n)||!("constructor"in e)||"function"==typeof x&&x instanceof x&&"function"==typeof C&&C instanceof C||(h=!1)}return s.delete(n),s.delete(e),h}},function(n,e,t){var i=t(191),a=t(192),r=t(80);n.exports=function(n){return i(n,r,a)}},function(n,e,t){var i=t(70),a=t(7);n.exports=function(n,e,t){var r=e(n);return a(n)?r:i(r,t(n))}},function(n,e,t){var i=t(193),a=t(194),r=Object.prototype.propertyIsEnumerable,o=Object.getOwnPropertySymbols,s=o?function(n){return null==n?[]:(n=Object(n),i(o(n),(function(e){return r.call(n,e)})))}:a;n.exports=s},function(n,e){n.exports=function(n,e){for(var t=-1,i=null==n?0:n.length,a=0,r=[];++t<i;){var o=n[t];e(o,t,n)&&(r[a++]=o)}return r}},function(n,e){n.exports=function(){return[]}},function(n,e,t){var i=t(196),a=t(39),r=t(7),o=t(81),s=t(82),l=t(83),c=Object.prototype.hasOwnProperty;n.exports=function(n,e){var t=r(n),g=!t&&a(n),d=!t&&!g&&o(n),p=!t&&!g&&!d&&l(n),u=t||g||d||p,m=u?i(n.length,String):[],h=m.length;for(var f in n)!e&&!c.call(n,f)||u&&("length"==f||d&&("offset"==f||"parent"==f)||p&&("buffer"==f||"byteLength"==f||"byteOffset"==f)||s(f,h))||m.push(f);return m}},function(n,e){n.exports=function(n,e){for(var t=-1,i=Array(n);++t<n;)i[t]=e(t);return i}},function(n,e){n.exports=function(){return!1}},function(n,e,t){var i=t(15),a=t(44),r=t(14),o={};o["[object Float32Array]"]=o["[object Float64Array]"]=o["[object Int8Array]"]=o["[object Int16Array]"]=o["[object Int32Array]"]=o["[object Uint8Array]"]=o["[object Uint8ClampedArray]"]=o["[object Uint16Array]"]=o["[object Uint32Array]"]=!0,o["[object Arguments]"]=o["[object Array]"]=o["[object ArrayBuffer]"]=o["[object Boolean]"]=o["[object DataView]"]=o["[object Date]"]=o["[object Error]"]=o["[object Function]"]=o["[object Map]"]=o["[object Number]"]=o["[object Object]"]=o["[object RegExp]"]=o["[object Set]"]=o["[object String]"]=o["[object WeakMap]"]=!1,n.exports=function(n){return r(n)&&a(n.length)&&!!o[i(n)]}},function(n,e){n.exports=function(n){return function(e){return n(e)}}},function(n,e,t){(function(n){var i=t(71),a=e&&!e.nodeType&&e,r=a&&"object"==typeof n&&n&&!n.nodeType&&n,o=r&&r.exports===a&&i.process,s=function(){try{var n=r&&r.require&&r.require("util").types;return n||o&&o.binding&&o.binding("util")}catch(n){}}();n.exports=s}).call(this,t(51)(n))},function(n,e,t){var i=t(202),a=t(203),r=Object.prototype.hasOwnProperty;n.exports=function(n){if(!i(n))return a(n);var e=[];for(var t in Object(n))r.call(n,t)&&"constructor"!=t&&e.push(t);return e}},function(n,e){var t=Object.prototype;n.exports=function(n){var e=n&&n.constructor;return n===("function"==typeof e&&e.prototype||t)}},function(n,e,t){var i=t(204)(Object.keys,Object);n.exports=i},function(n,e){n.exports=function(n,e){return function(t){return n(e(t))}}},function(n,e,t){var i=t(206),a=t(40),r=t(207),o=t(85),s=t(208),l=t(15),c=t(75),g=c(i),d=c(a),p=c(r),u=c(o),m=c(s),h=l;(i&&"[object DataView]"!=h(new i(new ArrayBuffer(1)))||a&&"[object Map]"!=h(new a)||r&&"[object Promise]"!=h(r.resolve())||o&&"[object Set]"!=h(new o)||s&&"[object WeakMap]"!=h(new s))&&(h=function(n){var e=l(n),t="[object Object]"==e?n.constructor:void 0,i=t?c(t):"";if(i)switch(i){case g:return"[object DataView]";case d:return"[object Map]";case p:return"[object Promise]";case u:return"[object Set]";case m:return"[object WeakMap]"}return e}),n.exports=h},function(n,e,t){var i=t(11)(t(8),"DataView");n.exports=i},function(n,e,t){var i=t(11)(t(8),"Promise");n.exports=i},function(n,e,t){var i=t(11)(t(8),"WeakMap");n.exports=i},function(n,e,t){var i=t(86),a=t(80);n.exports=function(n){for(var e=a(n),t=e.length;t--;){var r=e[t],o=n[r];e[t]=[r,o,i(o)]}return e}},function(n,e,t){var i=t(76),a=t(211),r=t(218),o=t(45),s=t(86),l=t(87),c=t(24);n.exports=function(n,e){return o(n)&&s(e)?l(c(n),e):function(t){var o=a(t,n);return void 0===o&&o===e?r(t,n):i(e,o,3)}}},function(n,e,t){var i=t(88);n.exports=function(n,e,t){var a=null==n?void 0:i(n,e);return void 0===a?t:a}},function(n,e,t){var i=t(213),a=/[^.[\]]+|\[(?:(-?\d+(?:\.\d+)?)|(["'])((?:(?!\2)[^\\]|\\.)*?)\2)\]|(?=(?:\.|\[\])(?:\.|\[\]|$))/g,r=/\\(\\)?/g,o=i((function(n){var e=[];return 46===n.charCodeAt(0)&&e.push(""),n.replace(a,(function(n,t,i,a){e.push(i?a.replace(r,"$1"):t||n)})),e}));n.exports=o},function(n,e,t){var i=t(214);n.exports=function(n){var e=i(n,(function(n){return 500===t.size&&t.clear(),n})),t=e.cache;return e}},function(n,e,t){var i=t(42);function a(n,e){if("function"!=typeof n||null!=e&&"function"!=typeof e)throw new TypeError("Expected a function");var t=function(){var i=arguments,a=e?e.apply(this,i):i[0],r=t.cache;if(r.has(a))return r.get(a);var o=n.apply(this,i);return t.cache=r.set(a,o)||r,o};return t.cache=new(a.Cache||i),t}a.Cache=i,n.exports=a},function(n,e,t){var i=t(216);n.exports=function(n){return null==n?"":i(n)}},function(n,e,t){var i=t(17),a=t(217),r=t(7),o=t(46),s=i?i.prototype:void 0,l=s?s.toString:void 0;n.exports=function n(e){if("string"==typeof e)return e;if(r(e))return a(e,n)+"";if(o(e))return l?l.call(e):"";var t=e+"";return"0"==t&&1/e==-1/0?"-0":t}},function(n,e){n.exports=function(n,e){for(var t=-1,i=null==n?0:n.length,a=Array(i);++t<i;)a[t]=e(n[t],t,n);return a}},function(n,e,t){var i=t(219),a=t(220);n.exports=function(n,e){return null!=n&&a(n,e,i)}},function(n,e){n.exports=function(n,e){return null!=n&&e in Object(n)}},function(n,e,t){var i=t(89),a=t(39),r=t(7),o=t(82),s=t(44),l=t(24);n.exports=function(n,e,t){for(var c=-1,g=(e=i(e,n)).length,d=!1;++c<g;){var p=l(e[c]);if(!(d=null!=n&&t(n,p)))break;n=n[p]}return d||++c!=g?d:!!(g=null==n?0:n.length)&&s(g)&&o(p,g)&&(r(n)||a(n))}},function(n,e,t){var i=t(222),a=t(223),r=t(45),o=t(24);n.exports=function(n){return r(n)?i(o(n)):a(n)}},function(n,e){n.exports=function(n){return function(e){return null==e?void 0:e[n]}}},function(n,e,t){var i=t(88);n.exports=function(n){return function(e){return i(e,n)}}},function(n,e,t){var i=t(47),a=t(225),r=t(227);n.exports=function(n,e){return r(a(n,e,i),n+"")}},function(n,e,t){var i=t(226),a=Math.max;n.exports=function(n,e,t){return e=a(void 0===e?n.length-1:e,0),function(){for(var r=arguments,o=-1,s=a(r.length-e,0),l=Array(s);++o<s;)l[o]=r[e+o];o=-1;for(var c=Array(e+1);++o<e;)c[o]=r[o];return c[e]=t(l),i(n,this,c)}}},function(n,e){n.exports=function(n,e,t){switch(t.length){case 0:return n.call(e);case 1:return n.call(e,t[0]);case 2:return n.call(e,t[0],t[1]);case 3:return n.call(e,t[0],t[1],t[2])}return n.apply(e,t)}},function(n,e,t){var i=t(228),a=t(231)(i);n.exports=a},function(n,e,t){var i=t(229),a=t(230),r=t(47),o=a?function(n,e){return a(n,"toString",{configurable:!0,enumerable:!1,value:i(e),writable:!0})}:r;n.exports=o},function(n,e){n.exports=function(n){return function(){return n}}},function(n,e,t){var i=t(11),a=function(){try{var n=i(Object,"defineProperty");return n({},"",{}),n}catch(n){}}();n.exports=a},function(n,e){var t=Date.now;n.exports=function(n){var e=0,i=0;return function(){var a=t(),r=16-(a-i);if(i=a,r>0){if(++e>=800)return arguments[0]}else e=0;return n.apply(void 0,arguments)}}},function(n,e,t){var i=t(78),a=t(233),r=t(238),o=t(79),s=t(239),l=t(43);n.exports=function(n,e,t){var c=-1,g=a,d=n.length,p=!0,u=[],m=u;if(t)p=!1,g=r;else if(d>=200){var h=e?null:s(n);if(h)return l(h);p=!1,g=o,m=new i}else m=e?[]:u;n:for(;++c<d;){var f=n[c],b=e?e(f):f;if(f=t||0!==f?f:0,p&&b==b){for(var v=m.length;v--;)if(m[v]===b)continue n;e&&m.push(b),u.push(f)}else g(m,b,t)||(m!==u&&m.push(b),u.push(f))}return u}},function(n,e,t){var i=t(234);n.exports=function(n,e){return!!(null==n?0:n.length)&&i(n,e,0)>-1}},function(n,e,t){var i=t(235),a=t(236),r=t(237);n.exports=function(n,e,t){return e==e?r(n,e,t):i(n,a,t)}},function(n,e){n.exports=function(n,e,t,i){for(var a=n.length,r=t+(i?1:-1);i?r--:++r<a;)if(e(n[r],r,n))return r;return-1}},function(n,e){n.exports=function(n){return n!=n}},function(n,e){n.exports=function(n,e,t){for(var i=t-1,a=n.length;++i<a;)if(n[i]===e)return i;return-1}},function(n,e){n.exports=function(n,e,t){for(var i=-1,a=null==n?0:n.length;++i<a;)if(t(e,n[i]))return!0;return!1}},function(n,e,t){var i=t(85),a=t(240),r=t(43),o=i&&1/r(new i([,-0]))[1]==1/0?function(n){return new i(n)}:a;n.exports=o},function(n,e){n.exports=function(){}},function(n,e,t){var i=t(84),a=t(14);n.exports=function(n){return a(n)&&i(n)}},function(n,e,t){},function(n,e,t){},function(n,e,t){"use strict";t(90)},function(n,e,t){"use strict";t(91)},function(n,e,t){},function(n,e,t){},function(n,e,t){"use strict";t(92)},function(n,e,t){n.exports=function(n){var e={};function t(i){if(e[i])return e[i].exports;var a=e[i]={i:i,l:!1,exports:{}};return n[i].call(a.exports,a,a.exports,t),a.l=!0,a.exports}return t.m=n,t.c=e,t.d=function(n,e,i){t.o(n,e)||Object.defineProperty(n,e,{enumerable:!0,get:i})},t.r=function(n){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(n,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(n,"__esModule",{value:!0})},t.t=function(n,e){if(1&e&&(n=t(n)),8&e)return n;if(4&e&&"object"==typeof n&&n&&n.__esModule)return n;var i=Object.create(null);if(t.r(i),Object.defineProperty(i,"default",{enumerable:!0,value:n}),2&e&&"string"!=typeof n)for(var a in n)t.d(i,a,function(e){return n[e]}.bind(null,a));return i},t.n=function(n){var e=n&&n.__esModule?function(){return n.default}:function(){return n};return t.d(e,"a",e),e},t.o=function(n,e){return Object.prototype.hasOwnProperty.call(n,e)},t.p="",t(t.s="fb15")}({"01f9":function(n,e,t){"use strict";var i=t("2d00"),a=t("5ca1"),r=t("2aba"),o=t("32e9"),s=t("84f2"),l=t("41a0"),c=t("7f20"),g=t("38fd"),d=t("2b4c")("iterator"),p=!([].keys&&"next"in[].keys()),u=function(){return this};n.exports=function(n,e,t,m,h,f,b){l(t,e,m);var v,y,x,C=function(n){if(!p&&n in T)return T[n];switch(n){case"keys":case"values":return function(){return new t(this,n)}}return function(){return new t(this,n)}},A=e+" Iterator",w="values"==h,k=!1,T=n.prototype,E=T[d]||T["@@iterator"]||h&&T[h],F=E||C(h),S=h?w?C("entries"):F:void 0,_="Array"==e&&T.entries||E;if(_&&(x=g(_.call(new n)))!==Object.prototype&&x.next&&(c(x,A,!0),i||"function"==typeof x[d]||o(x,d,u)),w&&E&&"values"!==E.name&&(k=!0,F=function(){return E.call(this)}),i&&!b||!p&&!k&&T[d]||o(T,d,F),s[e]=F,s[A]=u,h)if(v={values:w?F:C("values"),keys:f?F:C("keys"),entries:S},b)for(y in v)y in T||r(T,y,v[y]);else a(a.P+a.F*(p||k),e,v);return v}},"0a49":function(n,e,t){var i=t("9b43"),a=t("626a"),r=t("4bf8"),o=t("9def"),s=t("cd1c");n.exports=function(n,e){var t=1==n,l=2==n,c=3==n,g=4==n,d=6==n,p=5==n||d,u=e||s;return function(e,s,m){for(var h,f,b=r(e),v=a(b),y=i(s,m,3),x=o(v.length),C=0,A=t?u(e,x):l?u(e,0):void 0;x>C;C++)if((p||C in v)&&(f=y(h=v[C],C,b),n))if(t)A[C]=f;else if(f)switch(n){case 3:return!0;case 5:return h;case 6:return C;case 2:A.push(h)}else if(g)return!1;return d?-1:c||g?g:A}}},"0bfb":function(n,e,t){"use strict";var i=t("cb7c");n.exports=function(){var n=i(this),e="";return n.global&&(e+="g"),n.ignoreCase&&(e+="i"),n.multiline&&(e+="m"),n.unicode&&(e+="u"),n.sticky&&(e+="y"),e}},"0d58":function(n,e,t){var i=t("ce10"),a=t("e11e");n.exports=Object.keys||function(n){return i(n,a)}},1169:function(n,e,t){var i=t("2d95");n.exports=Array.isArray||function(n){return"Array"==i(n)}},"11e9":function(n,e,t){var i=t("52a7"),a=t("4630"),r=t("6821"),o=t("6a99"),s=t("69a8"),l=t("c69a"),c=Object.getOwnPropertyDescriptor;e.f=t("9e1e")?c:function(n,e){if(n=r(n),e=o(e,!0),l)try{return c(n,e)}catch(n){}if(s(n,e))return a(!i.f.call(n,e),n[e])}},1495:function(n,e,t){var i=t("86cc"),a=t("cb7c"),r=t("0d58");n.exports=t("9e1e")?Object.defineProperties:function(n,e){a(n);for(var t,o=r(e),s=o.length,l=0;s>l;)i.f(n,t=o[l++],e[t]);return n}},1991:function(n,e,t){var i,a,r,o=t("9b43"),s=t("31f4"),l=t("fab2"),c=t("230e"),g=t("7726"),d=g.process,p=g.setImmediate,u=g.clearImmediate,m=g.MessageChannel,h=g.Dispatch,f=0,b={},v=function(){var n=+this;if(b.hasOwnProperty(n)){var e=b[n];delete b[n],e()}},y=function(n){v.call(n.data)};p&&u||(p=function(n){for(var e=[],t=1;arguments.length>t;)e.push(arguments[t++]);return b[++f]=function(){s("function"==typeof n?n:Function(n),e)},i(f),f},u=function(n){delete b[n]},"process"==t("2d95")(d)?i=function(n){d.nextTick(o(v,n,1))}:h&&h.now?i=function(n){h.now(o(v,n,1))}:m?(r=(a=new m).port2,a.port1.onmessage=y,i=o(r.postMessage,r,1)):g.addEventListener&&"function"==typeof postMessage&&!g.importScripts?(i=function(n){g.postMessage(n+"","*")},g.addEventListener("message",y,!1)):i="onreadystatechange"in c("script")?function(n){l.appendChild(c("script")).onreadystatechange=function(){l.removeChild(this),v.call(n)}}:function(n){setTimeout(o(v,n,1),0)}),n.exports={set:p,clear:u}},"1fa8":function(n,e,t){var i=t("cb7c");n.exports=function(n,e,t,a){try{return a?e(i(t)[0],t[1]):e(t)}catch(e){var r=n.return;throw void 0!==r&&i(r.call(n)),e}}},"20d6":function(n,e,t){"use strict";var i=t("5ca1"),a=t("0a49")(6),r="findIndex",o=!0;r in[]&&Array(1)[r]((function(){o=!1})),i(i.P+i.F*o,"Array",{findIndex:function(n){return a(this,n,arguments.length>1?arguments[1]:void 0)}}),t("9c6c")(r)},"214f":function(n,e,t){"use strict";var i=t("32e9"),a=t("2aba"),r=t("79e5"),o=t("be13"),s=t("2b4c");n.exports=function(n,e,t){var l=s(n),c=t(o,l,""[n]),g=c[0],d=c[1];r((function(){var e={};return e[l]=function(){return 7},7!=""[n](e)}))&&(a(String.prototype,n,g),i(RegExp.prototype,l,2==e?function(n,e){return d.call(n,this,e)}:function(n){return d.call(n,this)}))}},"230e":function(n,e,t){var i=t("d3f4"),a=t("7726").document,r=i(a)&&i(a.createElement);n.exports=function(n){return r?a.createElement(n):{}}},2350:function(n,e){n.exports=function(n){var e=[];return e.toString=function(){return this.map((function(e){var t=function(n,e){var t=n[1]||"",i=n[3];if(!i)return t;if(e&&"function"==typeof btoa){var a=(o=i,"/*# sourceMappingURL=data:application/json;charset=utf-8;base64,"+btoa(unescape(encodeURIComponent(JSON.stringify(o))))+" */"),r=i.sources.map((function(n){return"/*# sourceURL="+i.sourceRoot+n+" */"}));return[t].concat(r).concat([a]).join("\n")}var o;return[t].join("\n")}(e,n);return e[2]?"@media "+e[2]+"{"+t+"}":t})).join("")},e.i=function(n,t){"string"==typeof n&&(n=[[null,n,""]]);for(var i={},a=0;a<this.length;a++){var r=this[a][0];"number"==typeof r&&(i[r]=!0)}for(a=0;a<n.length;a++){var o=n[a];"number"==typeof o[0]&&i[o[0]]||(t&&!o[2]?o[2]=t:t&&(o[2]="("+o[2]+") and ("+t+")"),e.push(o))}},e}},"23c6":function(n,e,t){var i=t("2d95"),a=t("2b4c")("toStringTag"),r="Arguments"==i(function(){return arguments}());n.exports=function(n){var e,t,o;return void 0===n?"Undefined":null===n?"Null":"string"==typeof(t=function(n,e){try{return n[e]}catch(n){}}(e=Object(n),a))?t:r?i(e):"Object"==(o=i(e))&&"function"==typeof e.callee?"Arguments":o}},2621:function(n,e){e.f=Object.getOwnPropertySymbols},"27ee":function(n,e,t){var i=t("23c6"),a=t("2b4c")("iterator"),r=t("84f2");n.exports=t("8378").getIteratorMethod=function(n){if(null!=n)return n[a]||n["@@iterator"]||r[i(n)]}},"28a5":function(n,e,t){t("214f")("split",2,(function(n,e,i){"use strict";var a=t("aae3"),r=i,o=[].push,s="length";if("c"=="abbc".split(/(b)*/)[1]||4!="test".split(/(?:)/,-1)[s]||2!="ab".split(/(?:ab)*/)[s]||4!=".".split(/(.?)(.?)/)[s]||".".split(/()()/)[s]>1||"".split(/.?/)[s]){var l=void 0===/()??/.exec("")[1];i=function(n,e){var t=String(this);if(void 0===n&&0===e)return[];if(!a(n))return r.call(t,n,e);var i,c,g,d,p,u=[],m=(n.ignoreCase?"i":"")+(n.multiline?"m":"")+(n.unicode?"u":"")+(n.sticky?"y":""),h=0,f=void 0===e?4294967295:e>>>0,b=new RegExp(n.source,m+"g");for(l||(i=new RegExp("^"+b.source+"$(?!\\s)",m));(c=b.exec(t))&&!((g=c.index+c[0][s])>h&&(u.push(t.slice(h,c.index)),!l&&c[s]>1&&c[0].replace(i,(function(){for(p=1;p<arguments[s]-2;p++)void 0===arguments[p]&&(c[p]=void 0)})),c[s]>1&&c.index<t[s]&&o.apply(u,c.slice(1)),d=c[0][s],h=g,u[s]>=f));)b.lastIndex===c.index&&b.lastIndex++;return h===t[s]?!d&&b.test("")||u.push(""):u.push(t.slice(h)),u[s]>f?u.slice(0,f):u}}else"0".split(void 0,0)[s]&&(i=function(n,e){return void 0===n&&0===e?[]:r.call(this,n,e)});return[function(t,a){var r=n(this),o=null==t?void 0:t[e];return void 0!==o?o.call(t,r,a):i.call(String(r),t,a)},i]}))},"2aba":function(n,e,t){var i=t("7726"),a=t("32e9"),r=t("69a8"),o=t("ca5a")("src"),s=Function.toString,l=(""+s).split("toString");t("8378").inspectSource=function(n){return s.call(n)},(n.exports=function(n,e,t,s){var c="function"==typeof t;c&&(r(t,"name")||a(t,"name",e)),n[e]!==t&&(c&&(r(t,o)||a(t,o,n[e]?""+n[e]:l.join(String(e)))),n===i?n[e]=t:s?n[e]?n[e]=t:a(n,e,t):(delete n[e],a(n,e,t)))})(Function.prototype,"toString",(function(){return"function"==typeof this&&this[o]||s.call(this)}))},"2aeb":function(n,e,t){var i=t("cb7c"),a=t("1495"),r=t("e11e"),o=t("613b")("IE_PROTO"),s=function(){},l=function(){var n,e=t("230e")("iframe"),i=r.length;for(e.style.display="none",t("fab2").appendChild(e),e.src="javascript:",(n=e.contentWindow.document).open(),n.write("<script>document.F=Object<\/script>"),n.close(),l=n.F;i--;)delete l.prototype[r[i]];return l()};n.exports=Object.create||function(n,e){var t;return null!==n?(s.prototype=i(n),t=new s,s.prototype=null,t[o]=n):t=l(),void 0===e?t:a(t,e)}},"2b4c":function(n,e,t){var i=t("5537")("wks"),a=t("ca5a"),r=t("7726").Symbol,o="function"==typeof r;(n.exports=function(n){return i[n]||(i[n]=o&&r[n]||(o?r:a)("Symbol."+n))}).store=i},"2d00":function(n,e){n.exports=!1},"2d95":function(n,e){var t={}.toString;n.exports=function(n){return t.call(n).slice(8,-1)}},"2e08":function(n,e,t){var i=t("9def"),a=t("9744"),r=t("be13");n.exports=function(n,e,t,o){var s=String(r(n)),l=s.length,c=void 0===t?" ":String(t),g=i(e);if(g<=l||""==c)return s;var d=g-l,p=a.call(c,Math.ceil(d/c.length));return p.length>d&&(p=p.slice(0,d)),o?p+s:s+p}},"2f21":function(n,e,t){"use strict";var i=t("79e5");n.exports=function(n,e){return!!n&&i((function(){e?n.call(null,(function(){}),1):n.call(null)}))}},"2fdb":function(n,e,t){"use strict";var i=t("5ca1"),a=t("d2c8");i(i.P+i.F*t("5147")("includes"),"String",{includes:function(n){return!!~a(this,n,"includes").indexOf(n,arguments.length>1?arguments[1]:void 0)}})},"31f4":function(n,e){n.exports=function(n,e,t){var i=void 0===t;switch(e.length){case 0:return i?n():n.call(t);case 1:return i?n(e[0]):n.call(t,e[0]);case 2:return i?n(e[0],e[1]):n.call(t,e[0],e[1]);case 3:return i?n(e[0],e[1],e[2]):n.call(t,e[0],e[1],e[2]);case 4:return i?n(e[0],e[1],e[2],e[3]):n.call(t,e[0],e[1],e[2],e[3])}return n.apply(t,e)}},"32e9":function(n,e,t){var i=t("86cc"),a=t("4630");n.exports=t("9e1e")?function(n,e,t){return i.f(n,e,a(1,t))}:function(n,e,t){return n[e]=t,n}},"33a4":function(n,e,t){var i=t("84f2"),a=t("2b4c")("iterator"),r=Array.prototype;n.exports=function(n){return void 0!==n&&(i.Array===n||r[a]===n)}},"37e7":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M25.468 6.947a1.004 1.004 0 0 0-1.03.057L18 11.384V7.831a1.001 1.001 0 0 0-1.562-.827l-12 8.164a1 1 0 0 0 0 1.654l12 8.168A.999.999 0 0 0 18 24.164v-3.556l6.438 4.382A.999.999 0 0 0 26 24.164V7.831c0-.371-.205-.71-.532-.884z"}})])}}},3846:function(n,e,t){t("9e1e")&&"g"!=/./g.flags&&t("86cc").f(RegExp.prototype,"flags",{configurable:!0,get:t("0bfb")})},"386b":function(n,e,t){var i=t("5ca1"),a=t("79e5"),r=t("be13"),o=/"/g,s=function(n,e,t,i){var a=String(r(n)),s="<"+e;return""!==t&&(s+=" "+t+'="'+String(i).replace(o,"&quot;")+'"'),s+">"+a+"</"+e+">"};n.exports=function(n,e){var t={};t[n]=e(s),i(i.P+i.F*a((function(){var e=""[n]('"');return e!==e.toLowerCase()||e.split('"').length>3})),"String",t)}},"38fd":function(n,e,t){var i=t("69a8"),a=t("4bf8"),r=t("613b")("IE_PROTO"),o=Object.prototype;n.exports=Object.getPrototypeOf||function(n){return n=a(n),i(n,r)?n[r]:"function"==typeof n.constructor&&n instanceof n.constructor?n.constructor.prototype:n instanceof Object?o:null}},"41a0":function(n,e,t){"use strict";var i=t("2aeb"),a=t("4630"),r=t("7f20"),o={};t("32e9")(o,t("2b4c")("iterator"),(function(){return this})),n.exports=function(n,e,t){n.prototype=i(o,{next:a(1,t)}),r(n,e+" Iterator")}},"456d":function(n,e,t){var i=t("4bf8"),a=t("0d58");t("5eda")("keys",(function(){return function(n){return a(i(n))}}))},4588:function(n,e){var t=Math.ceil,i=Math.floor;n.exports=function(n){return isNaN(n=+n)?0:(n>0?i:t)(n)}},4630:function(n,e){n.exports=function(n,e){return{enumerable:!(1&n),configurable:!(2&n),writable:!(4&n),value:e}}},4713:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M26.667 5.333H5.334h-.001a2.667 2.667 0 0 0-2.666 2.666V24.001a2.667 2.667 0 0 0 2.666 2.666h21.335a2.667 2.667 0 0 0 2.666-2.666V8v-.001a2.667 2.667 0 0 0-2.666-2.666h-.001zM5.333 16h5.333v2.667H5.333V16zm13.334 8H5.334v-2.667h13.333V24zm8 0h-5.333v-2.667h5.333V24zm0-5.333H13.334V16h13.333v2.667z"}})])}}},"475a":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M.622 18.334h19.54v7.55l11.052-9.412-11.052-9.413v7.549H.622v3.725z"}})])}}},"48d3":function(n,e,t){"use strict";Object.defineProperty(e,"__esModule",{value:!0}),function(n){for(var t in n)e.hasOwnProperty(t)||(e[t]=n[t])}(t("b349"));var i=t("66a1");e.modifiers=i.modifiers},4917:function(n,e,t){t("214f")("match",1,(function(n,e,t){return[function(t){"use strict";var i=n(this),a=null==t?void 0:t[e];return void 0!==a?a.call(t,i):new RegExp(t)[e](String(i))},t]}))},"499e":function(n,e,t){"use strict";function i(n,e){for(var t=[],i={},a=0;a<e.length;a++){var r=e[a],o=r[0],s={id:n+":"+a,css:r[1],media:r[2],sourceMap:r[3]};i[o]?i[o].parts.push(s):t.push(i[o]={id:o,parts:[s]})}return t}t.r(e),t.d(e,"default",(function(){return u}));var a="undefined"!=typeof document;if("undefined"!=typeof DEBUG&&DEBUG&&!a)throw new Error("vue-style-loader cannot be used in a non-browser environment. Use { target: 'node' } in your Webpack config to indicate a server-rendering environment.");var r={},o=a&&(document.head||document.getElementsByTagName("head")[0]),s=null,l=0,c=!1,g=function(){},d=null,p="undefined"!=typeof navigator&&/msie [6-9]\b/.test(navigator.userAgent.toLowerCase());function u(n,e,t,a){c=t,d=a||{};var o=i(n,e);return m(o),function(e){for(var t=[],a=0;a<o.length;a++){var s=o[a];(l=r[s.id]).refs--,t.push(l)}e?m(o=i(n,e)):o=[];for(a=0;a<t.length;a++){var l;if(0===(l=t[a]).refs){for(var c=0;c<l.parts.length;c++)l.parts[c]();delete r[l.id]}}}}function m(n){for(var e=0;e<n.length;e++){var t=n[e],i=r[t.id];if(i){i.refs++;for(var a=0;a<i.parts.length;a++)i.parts[a](t.parts[a]);for(;a<t.parts.length;a++)i.parts.push(f(t.parts[a]));i.parts.length>t.parts.length&&(i.parts.length=t.parts.length)}else{var o=[];for(a=0;a<t.parts.length;a++)o.push(f(t.parts[a]));r[t.id]={id:t.id,refs:1,parts:o}}}}function h(){var n=document.createElement("style");return n.type="text/css",o.appendChild(n),n}function f(n){var e,t,i=document.querySelector('style[data-vue-ssr-id~="'+n.id+'"]');if(i){if(c)return g;i.parentNode.removeChild(i)}if(p){var a=l++;i=s||(s=h()),e=y.bind(null,i,a,!1),t=y.bind(null,i,a,!0)}else i=h(),e=x.bind(null,i),t=function(){i.parentNode.removeChild(i)};return e(n),function(i){if(i){if(i.css===n.css&&i.media===n.media&&i.sourceMap===n.sourceMap)return;e(n=i)}else t()}}var b,v=(b=[],function(n,e){return b[n]=e,b.filter(Boolean).join("\n")});function y(n,e,t,i){var a=t?"":i.css;if(n.styleSheet)n.styleSheet.cssText=v(e,a);else{var r=document.createTextNode(a),o=n.childNodes;o[e]&&n.removeChild(o[e]),o.length?n.insertBefore(r,o[e]):n.appendChild(r)}}function x(n,e){var t=e.css,i=e.media,a=e.sourceMap;if(i&&n.setAttribute("media",i),d.ssrId&&n.setAttribute("data-vue-ssr-id",e.id),a&&(t+="\n/*# sourceURL="+a.sources[0]+" */",t+="\n/*# sourceMappingURL=data:application/json;base64,"+btoa(unescape(encodeURIComponent(JSON.stringify(a))))+" */"),n.styleSheet)n.styleSheet.cssText=t;else{for(;n.firstChild;)n.removeChild(n.firstChild);n.appendChild(document.createTextNode(t))}}},"4a59":function(n,e,t){var i=t("9b43"),a=t("1fa8"),r=t("33a4"),o=t("cb7c"),s=t("9def"),l=t("27ee"),c={},g={};(e=n.exports=function(n,e,t,d,p){var u,m,h,f,b=p?function(){return n}:l(n),v=i(t,d,e?2:1),y=0;if("function"!=typeof b)throw TypeError(n+" is not iterable!");if(r(b)){for(u=s(n.length);u>y;y++)if((f=e?v(o(m=n[y])[0],m[1]):v(n[y]))===c||f===g)return f}else for(h=b.call(n);!(m=h.next()).done;)if((f=a(h,v,m.value,e))===c||f===g)return f}).BREAK=c,e.RETURN=g},"4b41":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 28 32"}},[t("path",{attrs:{d:"M13.728 6.272v19.456q0 .448-.352.8t-.8.32-.8-.32l-5.952-5.952H1.152q-.48 0-.8-.352t-.352-.8v-6.848q0-.48.352-.8t.8-.352h4.672l5.952-5.952q.32-.32.8-.32t.8.32.352.8zM20.576 16q0 1.344-.768 2.528t-2.016 1.664q-.16.096-.448.096-.448 0-.8-.32t-.32-.832q0-.384.192-.64t.544-.448.608-.384.512-.64.192-1.024-.192-1.024-.512-.64-.608-.384-.544-.448-.192-.64q0-.48.32-.832t.8-.32q.288 0 .448.096 1.248.48 2.016 1.664T20.576 16zm4.576 0q0 2.72-1.536 5.056t-4 3.36q-.256.096-.448.096-.48 0-.832-.352t-.32-.8q0-.704.672-1.056 1.024-.512 1.376-.8 1.312-.96 2.048-2.4T22.848 16t-.736-3.104-2.048-2.4q-.352-.288-1.376-.8-.672-.352-.672-1.056 0-.448.32-.8t.8-.352q.224 0 .48.096 2.496 1.056 4 3.36T25.152 16zm4.576 0q0 4.096-2.272 7.552t-6.048 5.056q-.224.096-.448.096-.48 0-.832-.352t-.32-.8q0-.64.704-1.056l.384-.192q.256-.128.416-.192.8-.448 1.44-.896 2.208-1.632 3.456-4.064T27.424 16t-1.216-5.152-3.456-4.064q-.64-.448-1.44-.896-.128-.096-.416-.192t-.384-.192q-.704-.416-.704-1.056 0-.448.32-.8t.832-.352q.224 0 .448.096 3.776 1.632 6.048 5.056T29.728 16z"}})])}}},"4bf8":function(n,e,t){var i=t("be13");n.exports=function(n){return Object(i(n))}},"4d26":function(n,e,t){var i;
/*!
  Copyright (c) 2017 Jed Watson.
  Licensed under the MIT License (MIT), see
  http://jedwatson.github.io/classnames
*/!function(){"use strict";var t={}.hasOwnProperty;function a(){for(var n=[],e=0;e<arguments.length;e++){var i=arguments[e];if(i){var r=typeof i;if("string"===r||"number"===r)n.push(i);else if(Array.isArray(i)&&i.length){var o=a.apply(null,i);o&&n.push(o)}else if("object"===r)for(var s in i)t.call(i,s)&&i[s]&&n.push(s)}}return n.join(" ")}n.exports?(a.default=a,n.exports=a):void 0===(i=function(){return a}.apply(e,[]))||(n.exports=i)}()},5147:function(n,e,t){var i=t("2b4c")("match");n.exports=function(n){var e=/./;try{"/./"[n](e)}catch(t){try{return e[i]=!1,!"/./"[n](e)}catch(n){}}return!0}},"52a7":function(n,e){e.f={}.propertyIsEnumerable},"52f0":function(n,e,t){var i={"./loading.svg":"885d","./loop-all.svg":"f866","./loop-none.svg":"c3ab","./loop-one.svg":"5527","./lrc.svg":"4713","./menu.svg":"906b","./order-list.svg":"475a","./order-random.svg":"7a1a","./pause.svg":"daf8","./play.svg":"84d8","./right.svg":"bf5c","./skip.svg":"37e7","./volume-down.svg":"bdba","./volume-off.svg":"adec","./volume-up.svg":"4b41"};function a(n){var e=r(n);return t(e)}function r(n){var e=i[n];if(!(e+1)){var t=new Error("Cannot find module '"+n+"'");throw t.code="MODULE_NOT_FOUND",t}return e}a.keys=function(){return Object.keys(i)},a.resolve=r,n.exports=a,a.id="52f0"},"551c":function(n,e,t){"use strict";var i,a,r,o,s=t("2d00"),l=t("7726"),c=t("9b43"),g=t("23c6"),d=t("5ca1"),p=t("d3f4"),u=t("d8e8"),m=t("f605"),h=t("4a59"),f=t("ebd6"),b=t("1991").set,v=t("8079")(),y=t("a5b8"),x=t("9c80"),C=t("a25f"),A=t("bcaa"),w=l.TypeError,k=l.process,T=k&&k.versions,E=T&&T.v8||"",F=l.Promise,S="process"==g(k),_=function(){},P=a=y.f,q=!!function(){try{var n=F.resolve(1),e=(n.constructor={})[t("2b4c")("species")]=function(n){n(_,_)};return(S||"function"==typeof PromiseRejectionEvent)&&n.then(_)instanceof e&&0!==E.indexOf("6.6")&&-1===C.indexOf("Chrome/66")}catch(n){}}(),I=function(n){var e;return!(!p(n)||"function"!=typeof(e=n.then))&&e},z=function(n,e){if(!n._n){n._n=!0;var t=n._c;v((function(){for(var i=n._v,a=1==n._s,r=0,o=function(e){var t,r,o,s=a?e.ok:e.fail,l=e.resolve,c=e.reject,g=e.domain;try{s?(a||(2==n._h&&B(n),n._h=1),!0===s?t=i:(g&&g.enter(),t=s(i),g&&(g.exit(),o=!0)),t===e.promise?c(w("Promise-chain cycle")):(r=I(t))?r.call(t,l,c):l(t)):c(i)}catch(n){g&&!o&&g.exit(),c(n)}};t.length>r;)o(t[r++]);n._c=[],n._n=!1,e&&!n._h&&j(n)}))}},j=function(n){b.call(l,(function(){var e,t,i,a=n._v,r=N(n);if(r&&(e=x((function(){S?k.emit("unhandledRejection",a,n):(t=l.onunhandledrejection)?t({promise:n,reason:a}):(i=l.console)&&i.error&&i.error("Unhandled promise rejection",a)})),n._h=S||N(n)?2:1),n._a=void 0,r&&e.e)throw e.v}))},N=function(n){return 1!==n._h&&0===(n._a||n._c).length},B=function(n){b.call(l,(function(){var e;S?k.emit("rejectionHandled",n):(e=l.onrejectionhandled)&&e({promise:n,reason:n._v})}))},D=function(n){var e=this;e._d||(e._d=!0,(e=e._w||e)._v=n,e._s=2,e._a||(e._a=e._c.slice()),z(e,!0))},R=function(n){var e,t=this;if(!t._d){t._d=!0,t=t._w||t;try{if(t===n)throw w("Promise can't be resolved itself");(e=I(n))?v((function(){var i={_w:t,_d:!1};try{e.call(n,c(R,i,1),c(D,i,1))}catch(n){D.call(i,n)}})):(t._v=n,t._s=1,z(t,!1))}catch(n){D.call({_w:t,_d:!1},n)}}};q||(F=function(n){m(this,F,"Promise","_h"),u(n),i.call(this);try{n(c(R,this,1),c(D,this,1))}catch(n){D.call(this,n)}},(i=function(n){this._c=[],this._a=void 0,this._s=0,this._d=!1,this._v=void 0,this._h=0,this._n=!1}).prototype=t("dcbc")(F.prototype,{then:function(n,e){var t=P(f(this,F));return t.ok="function"!=typeof n||n,t.fail="function"==typeof e&&e,t.domain=S?k.domain:void 0,this._c.push(t),this._a&&this._a.push(t),this._s&&z(this,!1),t.promise},catch:function(n){return this.then(void 0,n)}}),r=function(){var n=new i;this.promise=n,this.resolve=c(R,n,1),this.reject=c(D,n,1)},y.f=P=function(n){return n===F||n===o?new r(n):a(n)}),d(d.G+d.W+d.F*!q,{Promise:F}),t("7f20")(F,"Promise"),t("7a56")("Promise"),o=t("8378").Promise,d(d.S+d.F*!q,"Promise",{reject:function(n){var e=P(this);return(0,e.reject)(n),e.promise}}),d(d.S+d.F*(s||!q),"Promise",{resolve:function(n){return A(s&&this===o?F:this,n)}}),d(d.S+d.F*!(q&&t("5cc5")((function(n){F.all(n).catch(_)}))),"Promise",{all:function(n){var e=this,t=P(e),i=t.resolve,a=t.reject,r=x((function(){var t=[],r=0,o=1;h(n,!1,(function(n){var s=r++,l=!1;t.push(void 0),o++,e.resolve(n).then((function(n){l||(l=!0,t[s]=n,--o||i(t))}),a)})),--o||i(t)}));return r.e&&a(r.v),t.promise},race:function(n){var e=this,t=P(e),i=t.reject,a=x((function(){h(n,!1,(function(n){e.resolve(n).then(t.resolve,i)}))}));return a.e&&i(a.v),t.promise}})},5527:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 33 32"}},[t("path",{attrs:{d:"M9.333 9.333h13.333v4L27.999 8l-5.333-5.333v4h-16v8h2.667V9.334zm13.334 13.334H9.334v-4L4.001 24l5.333 5.333v-4h16v-8h-2.667v5.333zM17.333 20v-8H16l-2.667 1.333v1.333h2v5.333h2z"}})])}}},5537:function(n,e,t){var i=t("8378"),a=t("7726"),r=a["__core-js_shared__"]||(a["__core-js_shared__"]={});(n.exports=function(n,e){return r[n]||(r[n]=void 0!==e?e:{})})("versions",[]).push({version:i.version,mode:t("2d00")?"pure":"global",copyright:" 2018 Denis Pushkarev (zloirock.ru)"})},"55dd":function(n,e,t){"use strict";var i=t("5ca1"),a=t("d8e8"),r=t("4bf8"),o=t("79e5"),s=[].sort,l=[1,2,3];i(i.P+i.F*(o((function(){l.sort(void 0)}))||!o((function(){l.sort(null)}))||!t("2f21")(s)),"Array",{sort:function(n){return void 0===n?s.call(r(this)):s.call(r(this),a(n))}})},"5ca1":function(n,e,t){var i=t("7726"),a=t("8378"),r=t("32e9"),o=t("2aba"),s=t("9b43"),l=function(n,e,t){var c,g,d,p,u=n&l.F,m=n&l.G,h=n&l.S,f=n&l.P,b=n&l.B,v=m?i:h?i[e]||(i[e]={}):(i[e]||{}).prototype,y=m?a:a[e]||(a[e]={}),x=y.prototype||(y.prototype={});for(c in m&&(t=e),t)d=((g=!u&&v&&void 0!==v[c])?v:t)[c],p=b&&g?s(d,i):f&&"function"==typeof d?s(Function.call,d):d,v&&o(v,c,d,n&l.U),y[c]!=d&&r(y,c,p),f&&x[c]!=d&&(x[c]=d)};i.core=a,l.F=1,l.G=2,l.S=4,l.P=8,l.B=16,l.W=32,l.U=64,l.R=128,n.exports=l},"5cc5":function(n,e,t){var i=t("2b4c")("iterator"),a=!1;try{var r=[7][i]();r.return=function(){a=!0},Array.from(r,(function(){throw 2}))}catch(n){}n.exports=function(n,e){if(!e&&!a)return!1;var t=!1;try{var r=[7],o=r[i]();o.next=function(){return{done:t=!0}},r[i]=function(){return o},n(r)}catch(n){}return t}},"5dbc":function(n,e,t){var i=t("d3f4"),a=t("8b97").set;n.exports=function(n,e,t){var r,o=e.constructor;return o!==t&&"function"==typeof o&&(r=o.prototype)!==t.prototype&&i(r)&&a&&a(n,r),n}},"5eda":function(n,e,t){var i=t("5ca1"),a=t("8378"),r=t("79e5");n.exports=function(n,e){var t=(a.Object||{})[n]||Object[n],o={};o[n]=e(t),i(i.S+i.F*r((function(){t(1)})),"Object",o)}},"610a":function(n,e,t){var i=t("ab57");"string"==typeof i&&(i=[[n.i,i,""]]),i.locals&&(n.exports=i.locals);(0,t("499e").default)("4573c8b0",i,!0,{sourceMap:!1,shadowMode:!1})},"613b":function(n,e,t){var i=t("5537")("keys"),a=t("ca5a");n.exports=function(n){return i[n]||(i[n]=a(n))}},"626a":function(n,e,t){var i=t("2d95");n.exports=Object("z").propertyIsEnumerable(0)?Object:function(n){return"String"==i(n)?n.split(""):Object(n)}},"65d9":function(n,e,t){"use strict";
/**
  * vue-class-component v6.3.2
  * (c) 2015-present Evan You
  * @license MIT
  */Object.defineProperty(e,"__esModule",{value:!0});var i,a=(i=t("8bbf"))&&"object"==typeof i&&"default"in i?i.default:i,r="undefined"!=typeof Reflect&&Reflect.defineMetadata;function o(n,e,t){(t?Reflect.getOwnMetadataKeys(e,t):Reflect.getOwnMetadataKeys(e)).forEach((function(i){var a=t?Reflect.getOwnMetadata(i,e,t):Reflect.getOwnMetadata(i,e);t?Reflect.defineMetadata(i,a,n,t):Reflect.defineMetadata(i,a,n)}))}var s={__proto__:[]}instanceof Array;var l=["data","beforeCreate","created","beforeMount","mounted","beforeDestroy","destroyed","beforeUpdate","updated","activated","deactivated","render","errorCaptured"];function c(n,e){void 0===e&&(e={}),e.name=e.name||n._componentTag||n.name;var t=n.prototype;Object.getOwnPropertyNames(t).forEach((function(n){if("constructor"!==n)if(l.indexOf(n)>-1)e[n]=t[n];else{var i=Object.getOwnPropertyDescriptor(t,n);void 0!==i.value?"function"==typeof i.value?(e.methods||(e.methods={}))[n]=i.value:(e.mixins||(e.mixins=[])).push({data:function(){var e;return(e={})[n]=i.value,e}}):(i.get||i.set)&&((e.computed||(e.computed={}))[n]={get:i.get,set:i.set})}})),(e.mixins||(e.mixins=[])).push({data:function(){return function(n,e){var t=e.prototype._init;e.prototype._init=function(){var e=this,t=Object.getOwnPropertyNames(n);if(n.$options.props)for(var i in n.$options.props)n.hasOwnProperty(i)||t.push(i);t.forEach((function(t){"_"!==t.charAt(0)&&Object.defineProperty(e,t,{get:function(){return n[t]},set:function(e){n[t]=e},configurable:!0})}))};var i=new e;e.prototype._init=t;var a={};return Object.keys(i).forEach((function(n){void 0!==i[n]&&(a[n]=i[n])})),a}(this,n)}});var i=n.__decorators__;i&&(i.forEach((function(n){return n(e)})),delete n.__decorators__);var c,g,d=Object.getPrototypeOf(n.prototype),p=d instanceof a?d.constructor:a,u=p.extend(e);return function(n,e,t){Object.getOwnPropertyNames(e).forEach((function(i){if("prototype"!==i){var a=Object.getOwnPropertyDescriptor(n,i);if(!a||a.configurable){var r,o,l=Object.getOwnPropertyDescriptor(e,i);if(!s){if("cid"===i)return;var c=Object.getOwnPropertyDescriptor(t,i);if(r=l.value,o=typeof r,null!=r&&("object"===o||"function"===o)&&c&&c.value===l.value)return}0,Object.defineProperty(n,i,l)}}}))}(u,n,p),r&&(o(c=u,g=n),Object.getOwnPropertyNames(g.prototype).forEach((function(n){o(c.prototype,g.prototype,n)})),Object.getOwnPropertyNames(g).forEach((function(n){o(c,g,n)}))),u}function g(n){return"function"==typeof n?c(n):function(e){return c(e,n)}}g.registerHooks=function(n){l.push.apply(l,n)},e.default=g,e.createDecorator=function(n){return function(e,t,i){var a="function"==typeof e?e:e.constructor;a.__decorators__||(a.__decorators__=[]),"number"!=typeof i&&(i=void 0),a.__decorators__.push((function(e){return n(e,t,i)}))}},e.mixins=function(){for(var n=[],e=0;e<arguments.length;e++)n[e]=arguments[e];return a.extend({mixins:n})}},"66a1":function(n,e,t){"use strict";var i=this&&this.__assign||Object.assign||function(n){for(var e,t=1,i=arguments.length;t<i;t++)for(var a in e=arguments[t])Object.prototype.hasOwnProperty.call(e,a)&&(n[a]=e[a]);return n};function a(n,e,t){for(var i=0,a=e;i<a.length;i++){if(!(0,a[i])(n))return}t&&t(n)}Object.defineProperty(e,"__esModule",{value:!0});var r={esc:27,tab:9,enter:13,space:32,up:38,down:40,del:[8,46],left:37,right:39};function o(n){for(var e=[],t=0,i=n;t<i.length;t++){var a=i[t];if("number"==typeof a)e.push(a);else{var o=r[a];"number"==typeof o?e.push(o):e.push.apply(e,o)}}switch(e.length){case 0:return function(n){return!1};case 1:var s=e[0];return function(n){return n.keyCode===s};default:return function(n){return e.indexOf(n.keyCode)>=0}}}function s(n,e,t,i,a){Object.defineProperty(n,t,{get:function(){var r=l(e.concat([i]),a);return Object.defineProperty(n,t,{value:r,enumerable:!0}),r},enumerable:!0,configurable:!0})}function l(n,e){function t(e){return e instanceof Function?function(t){return a(t,n,e)}:void a(e,n)}if(e.keyboard||e.mouse){var l=i({},e,{keyboard:!1,mouse:!1});e.keyboard&&(function(n,e,t){var i=function(i){var a=i;if("left"===a||"right"===a)return"continue";var o=r[a];if("number"==typeof o)s(n,e,a,(function(n){return n.keyCode===o}),t);else{var l=o[0],c=o[1];s(n,e,a,(function(n){return n.keyCode===l||n.keyCode===c}),t)}};for(var a in r)i(a)}(t,n,l),function(n,e,t){Object.defineProperty(n,"keys",{get:function(){var n=this,i=function(){for(var i=[],a=0;a<arguments.length;a++)i[a]=arguments[a];var r="keys:"+i.toString(),l=n[r];if(void 0!==l)return l;var c=o(i);return s(n,e,r,c,t),n[r]};return Object.defineProperty(this,"keys",{value:i,enumerable:!0}),i},enumerable:!0,configurable:!0})}(t,n,l)),e.mouse&&s(t,n,"middle",(function(n){return 1===n.button}),l),s(t,n,"left",(function(n){return 37===n.keyCode||0===n.button}),l),s(t,n,"right",(function(n){return 39===n.keyCode||2===n.button}),l)}if(e.exact){l=i({},e,{exact:!1,modkey:!1});!function(n,e,t){Object.defineProperty(n,"exact",{get:function(){var n=this,i=function(){for(var i=[],a=0;a<arguments.length;a++)i[a]=arguments[a];var r="exact:"+i.toString(),o=n[r];if(void 0!==o)return o;var l={ctrl:!1,shift:!1,alt:!1,meta:!1};i.forEach((function(n){return l[n]=!0}));var c=function(n){return!!n.ctrlKey===l.ctrl&&!!n.shiftKey===l.shift&&!!n.altKey===l.alt&&!!n.metaKey===l.meta};return s(n,e,r,c,t),n[r]};return Object.defineProperty(this,"exact",{value:i,enumerable:!0}),i},enumerable:!0,configurable:!0})}(t,n,l)}if(e.modkey){l=i({},e,{exact:!1});s(t,n,"ctrl",(function(n){return n.ctrlKey}),l),s(t,n,"shift",(function(n){return n.shiftKey}),l),s(t,n,"alt",(function(n){return n.altKey}),l),s(t,n,"meta",(function(n){return n.metaKey}),l),s(t,n,"noctrl",(function(n){return!n.ctrlKey}),l),s(t,n,"noshift",(function(n){return!n.shiftKey}),l),s(t,n,"noalt",(function(n){return!n.altKey}),l),s(t,n,"nometa",(function(n){return!n.metaKey}),l)}return s(t,n,"stop",(function(n){return n.stopPropagation()||!0}),e),s(t,n,"prevent",(function(n){return n.preventDefault()||!0}),e),s(t,n,"self",(function(n){return n.target===n.currentTarget}),e),t}e.modifiers=l([],{keyboard:!0,mouse:!0,modkey:!0,exact:!0})},6762:function(n,e,t){"use strict";var i=t("5ca1"),a=t("c366")(!0);i(i.P,"Array",{includes:function(n){return a(this,n,arguments.length>1?arguments[1]:void 0)}}),t("9c6c")("includes")},6821:function(n,e,t){var i=t("626a"),a=t("be13");n.exports=function(n){return i(a(n))}},"69a8":function(n,e){var t={}.hasOwnProperty;n.exports=function(n,e){return t.call(n,e)}},"6a99":function(n,e,t){var i=t("d3f4");n.exports=function(n,e){if(!i(n))return n;var t,a;if(e&&"function"==typeof(t=n.toString)&&!i(a=t.call(n)))return a;if("function"==typeof(t=n.valueOf)&&!i(a=t.call(n)))return a;if(!e&&"function"==typeof(t=n.toString)&&!i(a=t.call(n)))return a;throw TypeError("Can't convert object to primitive value")}},"6b54":function(n,e,t){"use strict";t("3846");var i=t("cb7c"),a=t("0bfb"),r=t("9e1e"),o=/./.toString,s=function(n){t("2aba")(RegExp.prototype,"toString",n,!0)};t("79e5")((function(){return"/a/b"!=o.call({source:"a",flags:"b"})}))?s((function(){var n=i(this);return"/".concat(n.source,"/","flags"in n?n.flags:!r&&n instanceof RegExp?a.call(n):void 0)})):"toString"!=o.name&&s((function(){return o.call(this)}))},7333:function(n,e,t){"use strict";var i=t("0d58"),a=t("2621"),r=t("52a7"),o=t("4bf8"),s=t("626a"),l=Object.assign;n.exports=!l||t("79e5")((function(){var n={},e={},t=Symbol(),i="abcdefghijklmnopqrst";return n[t]=7,i.split("").forEach((function(n){e[n]=n})),7!=l({},n)[t]||Object.keys(l({},e)).join("")!=i}))?function(n,e){for(var t=o(n),l=arguments.length,c=1,g=a.f,d=r.f;l>c;)for(var p,u=s(arguments[c++]),m=g?i(u).concat(g(u)):i(u),h=m.length,f=0;h>f;)d.call(u,p=m[f++])&&(t[p]=u[p]);return t}:l},7514:function(n,e,t){"use strict";var i=t("5ca1"),a=t("0a49")(5),r=!0;"find"in[]&&Array(1).find((function(){r=!1})),i(i.P+i.F*r,"Array",{find:function(n){return a(this,n,arguments.length>1?arguments[1]:void 0)}}),t("9c6c")("find")},7726:function(n,e){var t=n.exports="undefined"!=typeof window&&window.Math==Math?window:"undefined"!=typeof self&&self.Math==Math?self:Function("return this")();"number"==typeof __g&&(__g=t)},"77f1":function(n,e,t){var i=t("4588"),a=Math.max,r=Math.min;n.exports=function(n,e){return(n=i(n))<0?a(n+e,0):r(n,e)}},"79e5":function(n,e){n.exports=function(n){try{return!!n()}catch(n){return!0}}},"7a1a":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M22.667 4l7 6-7 6 7 6-7 6v-4h-3.653l-3.76-3.76 2.827-2.827L20.668 20h2v-8h-2l-12 12h-6v-4h4.347l12-12h3.653V4zm-20 4h6l3.76 3.76L9.6 14.587 7.013 12H2.666V8z"}})])}}},"7a56":function(n,e,t){"use strict";var i=t("7726"),a=t("86cc"),r=t("9e1e"),o=t("2b4c")("species");n.exports=function(n){var e=i[n];r&&e&&!e[o]&&a.f(e,o,{configurable:!0,get:function(){return this}})}},"7f20":function(n,e,t){var i=t("86cc").f,a=t("69a8"),r=t("2b4c")("toStringTag");n.exports=function(n,e,t){n&&!a(n=t?n:n.prototype,r)&&i(n,r,{configurable:!0,value:e})}},"7f7f":function(n,e,t){var i=t("86cc").f,a=Function.prototype,r=/^\s*function ([^ (]*)/;"name"in a||t("9e1e")&&i(a,"name",{configurable:!0,get:function(){try{return(""+this).match(r)[1]}catch(n){return""}}})},8079:function(n,e,t){var i=t("7726"),a=t("1991").set,r=i.MutationObserver||i.WebKitMutationObserver,o=i.process,s=i.Promise,l="process"==t("2d95")(o);n.exports=function(){var n,e,t,c=function(){var i,a;for(l&&(i=o.domain)&&i.exit();n;){a=n.fn,n=n.next;try{a()}catch(i){throw n?t():e=void 0,i}}e=void 0,i&&i.enter()};if(l)t=function(){o.nextTick(c)};else if(!r||i.navigator&&i.navigator.standalone)if(s&&s.resolve){var g=s.resolve(void 0);t=function(){g.then(c)}}else t=function(){a.call(i,c)};else{var d=!0,p=document.createTextNode("");new r(c).observe(p,{characterData:!0}),t=function(){p.data=d=!d}}return function(i){var a={fn:i,next:void 0};e&&(e.next=a),n||(n=a,t()),e=a}}},8378:function(n,e){var t=n.exports={version:"2.5.7"};"number"==typeof __e&&(__e=t)},"84d8":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 16 31"}},[t("path",{attrs:{d:"M15.552 15.168q.448.32.448.832 0 .448-.448.768L1.856 25.28q-.768.512-1.312.192T0 24.192V7.744q0-.96.544-1.28t1.312.192z"}})])}}},"84f2":function(n,e){n.exports={}},"86cc":function(n,e,t){var i=t("cb7c"),a=t("c69a"),r=t("6a99"),o=Object.defineProperty;e.f=t("9e1e")?Object.defineProperty:function(n,e,t){if(i(n),e=r(e,!0),i(t),a)try{return o(n,e,t)}catch(n){}if("get"in t||"set"in t)throw TypeError("Accessors not supported!");return"value"in t&&(n[e]=t.value),n}},"885d":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M4 16C4 9.4 9.4 4 16 4s12 5.4 12 12c0 1.2-.8 2-2 2s-2-.8-2-2c0-4.4-3.6-8-8-8s-8 3.6-8 8 3.6 8 8 8c1.2 0 2 .8 2 2s-.8 2-2 2C9.4 28 4 22.6 4 16z"}})])}}},"8b97":function(n,e,t){var i=t("d3f4"),a=t("cb7c"),r=function(n,e){if(a(n),!i(e)&&null!==e)throw TypeError(e+": can't set as prototype!")};n.exports={set:Object.setPrototypeOf||("__proto__"in{}?function(n,e,i){try{(i=t("9b43")(Function.call,t("11e9").f(Object.prototype,"__proto__").set,2))(n,[]),e=!(n instanceof Array)}catch(n){e=!0}return function(n,t){return r(n,t),e?n.__proto__=t:i(n,t),n}}({},!1):void 0),check:r}},"8bbf":function(n,e){n.exports=t(0)},"906b":function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 22 32"}},[t("path",{attrs:{d:"M20.8 14.4q.704 0 1.152.48T22.4 16t-.48 1.12-1.12.48H1.6q-.64 0-1.12-.48T0 16t.448-1.12T1.6 14.4h19.2zM1.6 11.2q-.64 0-1.12-.48T0 9.6t.448-1.12T1.6 8h19.2q.704 0 1.152.48T22.4 9.6t-.48 1.12-1.12.48H1.6zm19.2 9.6q.704 0 1.152.48t.448 1.12-.48 1.12-1.12.48H1.6q-.64 0-1.12-.48T0 22.4t.448-1.12T1.6 20.8h19.2z"}})])}}},9093:function(n,e,t){var i=t("ce10"),a=t("e11e").concat("length","prototype");e.f=Object.getOwnPropertyNames||function(n){return i(n,a)}},"96cf":function(n,e){!function(e){"use strict";var t=Object.prototype,i=t.hasOwnProperty,a="function"==typeof Symbol?Symbol:{},r=a.iterator||"@@iterator",o=a.asyncIterator||"@@asyncIterator",s=a.toStringTag||"@@toStringTag",l="object"==typeof n,c=e.regeneratorRuntime;if(c)l&&(n.exports=c);else{(c=e.regeneratorRuntime=l?n.exports:{}).wrap=h;var g={},d={};d[r]=function(){return this};var p=Object.getPrototypeOf,u=p&&p(p(E([])));u&&u!==t&&i.call(u,r)&&(d=u);var m=y.prototype=b.prototype=Object.create(d);v.prototype=m.constructor=y,y.constructor=v,y[s]=v.displayName="GeneratorFunction",c.isGeneratorFunction=function(n){var e="function"==typeof n&&n.constructor;return!!e&&(e===v||"GeneratorFunction"===(e.displayName||e.name))},c.mark=function(n){return Object.setPrototypeOf?Object.setPrototypeOf(n,y):(n.__proto__=y,s in n||(n[s]="GeneratorFunction")),n.prototype=Object.create(m),n},c.awrap=function(n){return{__await:n}},x(C.prototype),C.prototype[o]=function(){return this},c.AsyncIterator=C,c.async=function(n,e,t,i){var a=new C(h(n,e,t,i));return c.isGeneratorFunction(e)?a:a.next().then((function(n){return n.done?n.value:a.next()}))},x(m),m[s]="Generator",m[r]=function(){return this},m.toString=function(){return"[object Generator]"},c.keys=function(n){var e=[];for(var t in n)e.push(t);return e.reverse(),function t(){for(;e.length;){var i=e.pop();if(i in n)return t.value=i,t.done=!1,t}return t.done=!0,t}},c.values=E,T.prototype={constructor:T,reset:function(n){if(this.prev=0,this.next=0,this.sent=this._sent=void 0,this.done=!1,this.delegate=null,this.method="next",this.arg=void 0,this.tryEntries.forEach(k),!n)for(var e in this)"t"===e.charAt(0)&&i.call(this,e)&&!isNaN(+e.slice(1))&&(this[e]=void 0)},stop:function(){this.done=!0;var n=this.tryEntries[0].completion;if("throw"===n.type)throw n.arg;return this.rval},dispatchException:function(n){if(this.done)throw n;var e=this;function t(t,i){return o.type="throw",o.arg=n,e.next=t,i&&(e.method="next",e.arg=void 0),!!i}for(var a=this.tryEntries.length-1;a>=0;--a){var r=this.tryEntries[a],o=r.completion;if("root"===r.tryLoc)return t("end");if(r.tryLoc<=this.prev){var s=i.call(r,"catchLoc"),l=i.call(r,"finallyLoc");if(s&&l){if(this.prev<r.catchLoc)return t(r.catchLoc,!0);if(this.prev<r.finallyLoc)return t(r.finallyLoc)}else if(s){if(this.prev<r.catchLoc)return t(r.catchLoc,!0)}else{if(!l)throw new Error("try statement without catch or finally");if(this.prev<r.finallyLoc)return t(r.finallyLoc)}}}},abrupt:function(n,e){for(var t=this.tryEntries.length-1;t>=0;--t){var a=this.tryEntries[t];if(a.tryLoc<=this.prev&&i.call(a,"finallyLoc")&&this.prev<a.finallyLoc){var r=a;break}}r&&("break"===n||"continue"===n)&&r.tryLoc<=e&&e<=r.finallyLoc&&(r=null);var o=r?r.completion:{};return o.type=n,o.arg=e,r?(this.method="next",this.next=r.finallyLoc,g):this.complete(o)},complete:function(n,e){if("throw"===n.type)throw n.arg;return"break"===n.type||"continue"===n.type?this.next=n.arg:"return"===n.type?(this.rval=this.arg=n.arg,this.method="return",this.next="end"):"normal"===n.type&&e&&(this.next=e),g},finish:function(n){for(var e=this.tryEntries.length-1;e>=0;--e){var t=this.tryEntries[e];if(t.finallyLoc===n)return this.complete(t.completion,t.afterLoc),k(t),g}},catch:function(n){for(var e=this.tryEntries.length-1;e>=0;--e){var t=this.tryEntries[e];if(t.tryLoc===n){var i=t.completion;if("throw"===i.type){var a=i.arg;k(t)}return a}}throw new Error("illegal catch attempt")},delegateYield:function(n,e,t){return this.delegate={iterator:E(n),resultName:e,nextLoc:t},"next"===this.method&&(this.arg=void 0),g}}}function h(n,e,t,i){var a=e&&e.prototype instanceof b?e:b,r=Object.create(a.prototype),o=new T(i||[]);return r._invoke=function(n,e,t){var i="suspendedStart";return function(a,r){if("executing"===i)throw new Error("Generator is already running");if("completed"===i){if("throw"===a)throw r;return F()}for(t.method=a,t.arg=r;;){var o=t.delegate;if(o){var s=A(o,t);if(s){if(s===g)continue;return s}}if("next"===t.method)t.sent=t._sent=t.arg;else if("throw"===t.method){if("suspendedStart"===i)throw i="completed",t.arg;t.dispatchException(t.arg)}else"return"===t.method&&t.abrupt("return",t.arg);i="executing";var l=f(n,e,t);if("normal"===l.type){if(i=t.done?"completed":"suspendedYield",l.arg===g)continue;return{value:l.arg,done:t.done}}"throw"===l.type&&(i="completed",t.method="throw",t.arg=l.arg)}}}(n,t,o),r}function f(n,e,t){try{return{type:"normal",arg:n.call(e,t)}}catch(n){return{type:"throw",arg:n}}}function b(){}function v(){}function y(){}function x(n){["next","throw","return"].forEach((function(e){n[e]=function(n){return this._invoke(e,n)}}))}function C(n){var e;this._invoke=function(t,a){function r(){return new Promise((function(e,r){!function e(t,a,r,o){var s=f(n[t],n,a);if("throw"!==s.type){var l=s.arg,c=l.value;return c&&"object"==typeof c&&i.call(c,"__await")?Promise.resolve(c.__await).then((function(n){e("next",n,r,o)}),(function(n){e("throw",n,r,o)})):Promise.resolve(c).then((function(n){l.value=n,r(l)}),o)}o(s.arg)}(t,a,e,r)}))}return e=e?e.then(r,r):r()}}function A(n,e){var t=n.iterator[e.method];if(void 0===t){if(e.delegate=null,"throw"===e.method){if(n.iterator.return&&(e.method="return",e.arg=void 0,A(n,e),"throw"===e.method))return g;e.method="throw",e.arg=new TypeError("The iterator does not provide a 'throw' method")}return g}var i=f(t,n.iterator,e.arg);if("throw"===i.type)return e.method="throw",e.arg=i.arg,e.delegate=null,g;var a=i.arg;return a?a.done?(e[n.resultName]=a.value,e.next=n.nextLoc,"return"!==e.method&&(e.method="next",e.arg=void 0),e.delegate=null,g):a:(e.method="throw",e.arg=new TypeError("iterator result is not an object"),e.delegate=null,g)}function w(n){var e={tryLoc:n[0]};1 in n&&(e.catchLoc=n[1]),2 in n&&(e.finallyLoc=n[2],e.afterLoc=n[3]),this.tryEntries.push(e)}function k(n){var e=n.completion||{};e.type="normal",delete e.arg,n.completion=e}function T(n){this.tryEntries=[{tryLoc:"root"}],n.forEach(w,this),this.reset(!0)}function E(n){if(n){var e=n[r];if(e)return e.call(n);if("function"==typeof n.next)return n;if(!isNaN(n.length)){var t=-1,a=function e(){for(;++t<n.length;)if(i.call(n,t))return e.value=n[t],e.done=!1,e;return e.value=void 0,e.done=!0,e};return a.next=a}}return{next:F}}function F(){return{value:void 0,done:!0}}}(function(){return this}()||Function("return this")())},9744:function(n,e,t){"use strict";var i=t("4588"),a=t("be13");n.exports=function(n){var e=String(a(this)),t="",r=i(n);if(r<0||r==1/0)throw RangeError("Count can't be negative");for(;r>0;(r>>>=1)&&(e+=e))1&r&&(t+=e);return t}},"9b43":function(n,e,t){var i=t("d8e8");n.exports=function(n,e,t){if(i(n),void 0===e)return n;switch(t){case 1:return function(t){return n.call(e,t)};case 2:return function(t,i){return n.call(e,t,i)};case 3:return function(t,i,a){return n.call(e,t,i,a)}}return function(){return n.apply(e,arguments)}}},"9c6c":function(n,e,t){var i=t("2b4c")("unscopables"),a=Array.prototype;null==a[i]&&t("32e9")(a,i,{}),n.exports=function(n){a[i][n]=!0}},"9c80":function(n,e){n.exports=function(n){try{return{e:!1,v:n()}}catch(n){return{e:!0,v:n}}}},"9def":function(n,e,t){var i=t("4588"),a=Math.min;n.exports=function(n){return n>0?a(i(n),9007199254740991):0}},"9e1e":function(n,e,t){n.exports=!t("79e5")((function(){return 7!=Object.defineProperty({},"a",{get:function(){return 7}}).a}))},a25f:function(n,e,t){var i=t("7726").navigator;n.exports=i&&i.userAgent||""},a481:function(n,e,t){t("214f")("replace",2,(function(n,e,t){return[function(i,a){"use strict";var r=n(this),o=null==i?void 0:i[e];return void 0!==o?o.call(i,r,a):t.call(String(r),i,a)},t]}))},a5b8:function(n,e,t){"use strict";var i=t("d8e8");function a(n){var e,t;this.promise=new n((function(n,i){if(void 0!==e||void 0!==t)throw TypeError("Bad Promise constructor");e=n,t=i})),this.resolve=i(e),this.reject=i(t)}n.exports.f=function(n){return new a(n)}},aa77:function(n,e,t){var i=t("5ca1"),a=t("be13"),r=t("79e5"),o=t("fdef"),s="["+o+"]",l=RegExp("^"+s+s+"*"),c=RegExp(s+s+"*$"),g=function(n,e,t){var a={},s=r((function(){return!!o[n]()||""!=""[n]()})),l=a[n]=s?e(d):o[n];t&&(a[t]=l),i(i.P+i.F*s,"String",a)},d=g.trim=function(n,e){return n=String(a(n)),1&e&&(n=n.replace(l,"")),2&e&&(n=n.replace(c,"")),n};n.exports=g},aae3:function(n,e,t){var i=t("d3f4"),a=t("2d95"),r=t("2b4c")("match");n.exports=function(n){var e;return i(n)&&(void 0!==(e=n[r])?!!e:"RegExp"==a(n))}},ab57:function(n,e,t){(n.exports=t("2350")(!1)).push([n.i,'.aplayer{background:#fff;font-family:Arial,Helvetica,sans-serif;margin:5px;-webkit-box-shadow:0 2px 2px 0 rgba(0,0,0,.07),0 1px 5px 0 rgba(0,0,0,.1);box-shadow:0 2px 2px 0 rgba(0,0,0,.07),0 1px 5px 0 rgba(0,0,0,.1);border-radius:2px;overflow:hidden;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;line-height:normal;position:relative}.aplayer *{-webkit-box-sizing:content-box;box-sizing:content-box}.aplayer svg{width:100%;height:100%}.aplayer svg circle,.aplayer svg path{fill:#fff}.aplayer.aplayer-withlist .aplayer-info{border-bottom:1px solid #e9e9e9}.aplayer.aplayer-withlist .aplayer-list{display:block;width:100%}.aplayer.aplayer-withlist .aplayer-icon-order,.aplayer.aplayer-withlist .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon.aplayer-icon-menu{display:inline}.aplayer.aplayer-withlrc .aplayer-pic{height:90px;width:90px}.aplayer.aplayer-withlrc .aplayer-info{margin-left:90px;height:90px;padding:10px 7px 0 7px}.aplayer.aplayer-withlrc .aplayer-lrc{display:block}.aplayer.aplayer-narrow{width:66px}.aplayer.aplayer-narrow .aplayer-info,.aplayer.aplayer-narrow .aplayer-list{display:none}.aplayer.aplayer-narrow .aplayer-body,.aplayer.aplayer-narrow .aplayer-pic{height:66px;width:66px}.aplayer.aplayer-fixed{position:fixed;bottom:0;left:0;right:0;margin:0;z-index:99;overflow:visible;max-width:400px;-webkit-box-shadow:none;box-shadow:none}.aplayer.aplayer-fixed .aplayer-list{margin-bottom:65px;border:1px solid #eee;border-bottom:none;-webkit-box-sizing:border-box;box-sizing:border-box}.aplayer.aplayer-fixed .aplayer-body{position:fixed;bottom:0;left:0;right:0;margin:0;z-index:99;background:#fff;padding-right:18px;-webkit-transition:width .3s ease;transition:width .3s ease;max-width:400px;width:calc(100% - 18px)}.aplayer.aplayer-fixed .aplayer-lrc{display:block;position:fixed;bottom:10px;left:0;right:0;margin:0;z-index:98;pointer-events:none;text-shadow:-1px -1px 0 #fff}.aplayer.aplayer-fixed .aplayer-lrc:after,.aplayer.aplayer-fixed .aplayer-lrc:before{display:none}.aplayer.aplayer-fixed .aplayer-info{-webkit-transform:scaleX(1);transform:scaleX(1);-webkit-transform-origin:0 0;transform-origin:0 0;-webkit-transition:all .3s ease;transition:all .3s ease;border-bottom:none;border-top:1px solid #e9e9e9}.aplayer.aplayer-fixed .aplayer-info .aplayer-music{width:calc(100% - 105px)}.aplayer.aplayer-fixed .aplayer-miniswitcher{display:block}.aplayer.aplayer-fixed.aplayer-narrow .aplayer-info{display:block;-webkit-transform:scaleX(0);transform:scaleX(0)}.aplayer.aplayer-fixed.aplayer-narrow .aplayer-body{width:66px!important}.aplayer.aplayer-fixed.aplayer-narrow .aplayer-miniswitcher .aplayer-icon{-webkit-transform:rotateY(0);transform:rotateY(0)}.aplayer.aplayer-fixed .aplayer-icon-back,.aplayer.aplayer-fixed .aplayer-icon-forward,.aplayer.aplayer-fixed .aplayer-icon-lrc,.aplayer.aplayer-fixed .aplayer-icon-play{display:inline-block}.aplayer.aplayer-fixed .aplayer-icon-back,.aplayer.aplayer-fixed .aplayer-icon-forward,.aplayer.aplayer-fixed .aplayer-icon-menu,.aplayer.aplayer-fixed .aplayer-icon-play{position:absolute;bottom:27px;width:20px;height:20px}.aplayer.aplayer-fixed .aplayer-icon-back{right:75px}.aplayer.aplayer-fixed .aplayer-icon-play{right:50px}.aplayer.aplayer-fixed .aplayer-icon-forward{right:25px}.aplayer.aplayer-fixed .aplayer-icon-menu{right:0}.aplayer.aplayer-arrow .aplayer-icon-loop,.aplayer.aplayer-arrow .aplayer-icon-order,.aplayer.aplayer-mobile .aplayer-icon-volume-down,.aplayer.aplayer-mobile .aplayer-icon-volume-up{display:none}.aplayer.aplayer-loading .aplayer-info .aplayer-controller .aplayer-loading-icon{display:block}.aplayer.aplayer-loading .aplayer-info .aplayer-controller .aplayer-bar-wrap .aplayer-bar .aplayer-played .aplayer-thumb{-webkit-transform:scale(1);transform:scale(1)}.aplayer .aplayer-body{position:relative}.aplayer .aplayer-icon{width:15px;height:15px;border:none;background-color:transparent;outline:none;cursor:pointer;opacity:.8;vertical-align:middle;padding:0;font-size:12px;margin:0;display:inline-block}.aplayer .aplayer-icon path{-webkit-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.aplayer .aplayer-icon-back,.aplayer .aplayer-icon-forward,.aplayer .aplayer-icon-lrc,.aplayer .aplayer-icon-order,.aplayer .aplayer-icon-play{display:none}.aplayer .aplayer-icon-lrc-inactivity svg{opacity:.4}.aplayer .aplayer-icon-forward{-webkit-transform:rotate(180deg);transform:rotate(180deg)}.aplayer .aplayer-lrc-content{display:none}.aplayer .aplayer-pic{position:relative;float:left;height:66px;width:66px;background-size:cover;background-position:50%;-webkit-transition:all .3s ease;transition:all .3s ease;cursor:pointer}.aplayer .aplayer-pic:hover .aplayer-button{opacity:1}.aplayer .aplayer-pic .aplayer-button{position:absolute;border-radius:50%;opacity:.8;text-shadow:0 1px 1px rgba(0,0,0,.2);-webkit-box-shadow:0 1px 1px rgba(0,0,0,.2);box-shadow:0 1px 1px rgba(0,0,0,.2);background:rgba(0,0,0,.2);-webkit-transition:all .1s ease;transition:all .1s ease}.aplayer .aplayer-pic .aplayer-button path{fill:#fff}.aplayer .aplayer-pic .aplayer-hide{display:none}.aplayer .aplayer-pic .aplayer-play{width:26px;height:26px;border:2px solid #fff;bottom:50%;right:50%;margin:0 -15px -15px 0}.aplayer .aplayer-pic .aplayer-play svg{position:absolute;top:3px;left:4px;height:20px;width:20px}.aplayer .aplayer-pic .aplayer-pause{width:16px;height:16px;border:2px solid #fff;bottom:4px;right:4px}.aplayer .aplayer-pic .aplayer-pause svg{position:absolute;top:2px;left:2px;height:12px;width:12px}.aplayer .aplayer-info{margin-left:66px;padding:14px 7px 0 10px;height:66px;-webkit-box-sizing:border-box;box-sizing:border-box}.aplayer .aplayer-info .aplayer-music{overflow:hidden;white-space:nowrap;text-overflow:ellipsis;margin:0 0 13px 5px;-webkit-user-select:text;-moz-user-select:text;-ms-user-select:text;user-select:text;cursor:default;padding-bottom:2px;height:20px}.aplayer .aplayer-info .aplayer-music .aplayer-title{font-size:14px}.aplayer .aplayer-info .aplayer-music .aplayer-author{font-size:12px;color:#666}.aplayer .aplayer-info .aplayer-controller{position:relative;display:-webkit-box;display:-ms-flexbox;display:flex}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap{margin:0 0 0 5px;padding:4px 0;cursor:pointer!important;-webkit-box-flex:1;-ms-flex:1;flex:1}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap:hover .aplayer-bar .aplayer-played .aplayer-thumb{-webkit-transform:scale(1);transform:scale(1)}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap .aplayer-bar{position:relative;height:2px;width:100%;background:#cdcdcd}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap .aplayer-bar .aplayer-loaded{position:absolute;left:0;top:0;bottom:0;background:#aaa;height:2px;-webkit-transition:all .5s ease;transition:all .5s ease}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap .aplayer-bar .aplayer-played{position:absolute;left:0;top:0;bottom:0;height:2px;-webkit-transition:background-color .3s ease;transition:background-color .3s ease}.aplayer .aplayer-info .aplayer-controller .aplayer-bar-wrap .aplayer-bar .aplayer-played .aplayer-thumb{position:absolute;top:0;right:5px;margin-top:-4px;margin-right:-10px;height:10px;width:10px;border-radius:50%;cursor:pointer;-webkit-transition:all .3s ease-in-out;transition:all .3s ease-in-out;-webkit-transform:scale(0);transform:scale(0)}.aplayer .aplayer-info .aplayer-controller .aplayer-time{position:relative;right:0;bottom:4px;height:17px;color:#999;font-size:11px;padding-left:7px}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-time-inner{vertical-align:middle}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon{cursor:pointer;-webkit-transition:all .2s ease;transition:all .2s ease}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon path{fill:#666}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon.aplayer-icon-loop{margin-right:2px}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon:hover path{fill:#000}.aplayer .aplayer-info .aplayer-controller .aplayer-time .aplayer-icon.aplayer-icon-menu,.aplayer .aplayer-info .aplayer-controller .aplayer-time.aplayer-time-narrow .aplayer-icon-menu,.aplayer .aplayer-info .aplayer-controller .aplayer-time.aplayer-time-narrow .aplayer-icon-mode{display:none}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap{position:relative;display:inline-block;margin-left:3px;cursor:pointer!important}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap:hover .aplayer-volume-bar-wrap{height:40px}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap .aplayer-volume-bar-wrap{position:absolute;bottom:15px;right:-3px;width:25px;height:0;z-index:99;overflow:hidden;-webkit-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap .aplayer-volume-bar-wrap.aplayer-volume-bar-wrap-active{height:40px}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap .aplayer-volume-bar-wrap .aplayer-volume-bar{position:absolute;bottom:0;right:10px;width:5px;height:35px;background:#aaa;border-radius:2.5px;overflow:hidden}.aplayer .aplayer-info .aplayer-controller .aplayer-volume-wrap .aplayer-volume-bar-wrap .aplayer-volume-bar .aplayer-volume{position:absolute;bottom:0;right:0;width:5px;-webkit-transition:all .1s ease;transition:all .1s ease}.aplayer .aplayer-info .aplayer-controller .aplayer-loading-icon{display:none}.aplayer .aplayer-info .aplayer-controller .aplayer-loading-icon svg{position:absolute;-webkit-animation:rotate 1s linear infinite;animation:rotate 1s linear infinite}.aplayer .aplayer-lrc{display:none;position:relative;height:30px;text-align:center;overflow:hidden;margin:-10px 0 7px}.aplayer .aplayer-lrc:before{top:0;height:10%;background:-webkit-gradient(linear,left top,left bottom,from(#fff),to(hsla(0,0%,100%,0)));background:linear-gradient(180deg,#fff 0,hsla(0,0%,100%,0));filter:progid:DXImageTransform.Microsoft.gradient(startColorstr="#ffffff",endColorstr="#00ffffff",GradientType=0)}.aplayer .aplayer-lrc:after,.aplayer .aplayer-lrc:before{position:absolute;z-index:1;display:block;overflow:hidden;width:100%;content:" "}.aplayer .aplayer-lrc:after{bottom:0;height:33%;background:-webkit-gradient(linear,left top,left bottom,from(hsla(0,0%,100%,0)),to(hsla(0,0%,100%,.8)));background:linear-gradient(180deg,hsla(0,0%,100%,0) 0,hsla(0,0%,100%,.8));filter:progid:DXImageTransform.Microsoft.gradient(startColorstr="#00ffffff",endColorstr="#ccffffff",GradientType=0)}.aplayer .aplayer-lrc p{font-size:12px;color:#666;line-height:16px!important;height:16px!important;padding:0!important;margin:0!important;-webkit-transition-property:font-size,color,opacity;transition-property:font-size,color,opacity;-webkit-transition-timing-function:ease-out;transition-timing-function:ease-out;-webkit-transition-duration:.5s;transition-duration:.5s;opacity:.4;overflow:hidden}.aplayer .aplayer-lrc p.aplayer-lrc-current{opacity:1;overflow:visible;height:auto!important;min-height:16px}.aplayer .aplayer-lrc.aplayer-lrc-hide{display:none}.aplayer .aplayer-lrc .aplayer-lrc-contents{width:100%;-webkit-transition:all .5s ease-out;transition:all .5s ease-out;-webkit-user-select:text;-moz-user-select:text;-ms-user-select:text;user-select:text;cursor:default}.aplayer .aplayer-list{overflow:auto;-webkit-transition:all .5s ease;transition:all .5s ease;will-change:height;display:none;overflow:hidden;list-style-type:none;margin:0;padding:0;overflow-y:auto}.aplayer .aplayer-list::-webkit-scrollbar{width:5px}.aplayer .aplayer-list::-webkit-scrollbar-thumb{border-radius:3px;background-color:#eee}.aplayer .aplayer-list::-webkit-scrollbar-thumb:hover{background-color:#ccc}.aplayer .aplayer-list li{position:relative;height:32px;line-height:32px;padding:0 15px;font-size:12px;border-top:1px solid #e9e9e9;cursor:pointer;-webkit-transition:all .2s ease;transition:all .2s ease;overflow:hidden;margin:0}.aplayer .aplayer-list li:first-child{border-top:none}.aplayer .aplayer-list li:hover{background:#efefef}.aplayer .aplayer-list li.aplayer-list-light{background:#e9e9e9}.aplayer .aplayer-list li.aplayer-list-light .aplayer-list-cur{display:inline-block}.aplayer .aplayer-list li .aplayer-list-cur{display:none;width:3px;height:22px;position:absolute;left:0;top:5px;-webkit-transition:background-color .3s ease;transition:background-color .3s ease;cursor:pointer}.aplayer .aplayer-list li .aplayer-list-index{color:#666;margin-right:12px;cursor:pointer}.aplayer .aplayer-list li .aplayer-list-author{color:#666;float:right;cursor:pointer}.aplayer .aplayer-notice{opacity:0;position:absolute;z-index:1;top:50%;left:50%;-webkit-transform:translate(-50%,-50%);transform:translate(-50%,-50%);font-size:12px;border-radius:4px;padding:5px 10px;-webkit-transition:all .3s ease-in-out;transition:all .3s ease-in-out;overflow:hidden;color:#fff;pointer-events:none;background-color:#f4f4f5;color:#909399}.aplayer .aplayer-miniswitcher{display:none;position:absolute;top:0;right:0;bottom:0;height:100%;background:#e6e6e6;width:18px;border-radius:0 2px 2px 0}.aplayer .aplayer-miniswitcher .aplayer-icon{height:100%;width:100%;-webkit-transform:rotateY(180deg);transform:rotateY(180deg);-webkit-transition:all .3s ease;transition:all .3s ease}.aplayer .aplayer-miniswitcher .aplayer-icon path{fill:#666}.aplayer .aplayer-miniswitcher .aplayer-icon:hover path{fill:#000}@-webkit-keyframes aplayer-roll{0%{left:0}to{left:-100%}}@keyframes aplayer-roll{0%{left:0}to{left:-100%}}@-webkit-keyframes rotate{0%{-webkit-transform:rotate(0);transform:rotate(0)}to{-webkit-transform:rotate(1turn);transform:rotate(1turn)}}@keyframes rotate{0%{-webkit-transform:rotate(0);transform:rotate(0)}to{-webkit-transform:rotate(1turn);transform:rotate(1turn)}}',""])},ac6a:function(n,e,t){for(var i=t("cadf"),a=t("0d58"),r=t("2aba"),o=t("7726"),s=t("32e9"),l=t("84f2"),c=t("2b4c"),g=c("iterator"),d=c("toStringTag"),p=l.Array,u={CSSRuleList:!0,CSSStyleDeclaration:!1,CSSValueList:!1,ClientRectList:!1,DOMRectList:!1,DOMStringList:!1,DOMTokenList:!0,DataTransferItemList:!1,FileList:!1,HTMLAllCollection:!1,HTMLCollection:!1,HTMLFormElement:!1,HTMLSelectElement:!1,MediaList:!0,MimeTypeArray:!1,NamedNodeMap:!1,NodeList:!0,PaintRequestList:!1,Plugin:!1,PluginArray:!1,SVGLengthList:!1,SVGNumberList:!1,SVGPathSegList:!1,SVGPointList:!1,SVGStringList:!1,SVGTransformList:!1,SourceBufferList:!1,StyleSheetList:!0,TextTrackCueList:!1,TextTrackList:!1,TouchList:!1},m=a(u),h=0;h<m.length;h++){var f,b=m[h],v=u[b],y=o[b],x=y&&y.prototype;if(x&&(x[g]||s(x,g,p),x[d]||s(x,d,b),l[b]=p,v))for(f in i)x[f]||r(x,f,i[f],!0)}},adec:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 28 32"}},[t("path",{attrs:{d:"M13.728 6.272v19.456q0 .448-.352.8t-.8.32-.8-.32l-5.952-5.952H1.152q-.48 0-.8-.352t-.352-.8v-6.848q0-.48.352-.8t.8-.352h4.672l5.952-5.952q.32-.32.8-.32t.8.32.352.8z"}})])}}},b349:function(n,e,t){"use strict";var i,a=this&&this.__extends||(i=Object.setPrototypeOf||{__proto__:[]}instanceof Array&&function(n,e){n.__proto__=e}||function(n,e){for(var t in e)e.hasOwnProperty(t)&&(n[t]=e[t])},function(n,e){function t(){this.constructor=n}i(n,e),n.prototype=null===e?Object.create(e):(t.prototype=e.prototype,new t)}),r=this&&this.__assign||Object.assign||function(n){for(var e,t=1,i=arguments.length;t<i;t++)for(var a in e=arguments[t])Object.prototype.hasOwnProperty.call(e,a)&&(n[a]=e[a]);return n},o=this&&this.__importDefault||function(n){return n&&n.__esModule?n:{default:n}};Object.defineProperty(e,"__esModule",{value:!0});var s=o(t("8bbf")),l=function(n){function e(){var e=null!==n&&n.apply(this,arguments)||this;return e._tsxattrs=void 0,e.$scopedSlots=void 0,e}return a(e,n),e}(s.default);e.Component=l,e.createComponent=function(n){return s.default.extend(n)};var c={convert:function(n){return n},extendFrom:function(n){return n}};function g(n,e){return{create:function(t){var i=t.mixins?t.mixins.concat(e):e;return n.extend(r({},t,{mixins:i}))},mixin:function(t){return g(n,e.concat([t]))}}}e.ofType=function(){return c},e.withNativeOn=function(n){return n},e.withHtmlAttrs=function(n){return n},e.withUnknownProps=function(n){return n},e.componentFactory={create:function(n){return s.default.extend(n)},extendFrom:function(n){return g(n,[])},mixin:function(n){return g(s.default,[n])}},e.componentFactoryOf=function(){return e.componentFactory},e.component=e.componentFactory.create,e.extendFrom=e.componentFactory.extendFrom},bcaa:function(n,e,t){var i=t("cb7c"),a=t("d3f4"),r=t("a5b8");n.exports=function(n,e){if(i(n),a(e)&&e.constructor===n)return e;var t=r.f(n);return(0,t.resolve)(e),t.promise}},bdba:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 28 32"}},[t("path",{attrs:{d:"M13.728 6.272v19.456q0 .448-.352.8t-.8.32-.8-.32l-5.952-5.952H1.152q-.48 0-.8-.352t-.352-.8v-6.848q0-.48.352-.8t.8-.352h4.672l5.952-5.952q.32-.32.8-.32t.8.32.352.8zM20.576 16q0 1.344-.768 2.528t-2.016 1.664q-.16.096-.448.096-.448 0-.8-.32t-.32-.832q0-.384.192-.64t.544-.448.608-.384.512-.64.192-1.024-.192-1.024-.512-.64-.608-.384-.544-.448-.192-.64q0-.48.32-.832t.8-.32q.288 0 .448.096 1.248.48 2.016 1.664T20.576 16z"}})])}}},be13:function(n,e){n.exports=function(n){if(null==n)throw TypeError("Can't call method on  "+n);return n}},bf5c:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 32 32"}},[t("path",{attrs:{d:"M22 16L11.895 5.4 10 7.387 18.211 16 10 24.612l1.895 1.988 8.211-8.613z"}})])}}},c366:function(n,e,t){var i=t("6821"),a=t("9def"),r=t("77f1");n.exports=function(n){return function(e,t,o){var s,l=i(e),c=a(l.length),g=r(o,c);if(n&&t!=t){for(;c>g;)if((s=l[g++])!=s)return!0}else for(;c>g;g++)if((n||g in l)&&l[g]===t)return n||g||0;return!n&&-1}}},c3ab:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 29 32"}},[t("path",{attrs:{d:"M2.667 7.027l1.707-1.693 22.293 22.293-1.693 1.707-4-4H9.334v4l-5.333-5.333 5.333-5.333v4h8.973l-8.973-8.973v.973H6.667v-3.64l-4-4zm20 10.306h2.667v5.573l-2.667-2.667v-2.907zm0-10.666v-4L28 8l-5.333 5.333v-4H11.76L9.093 6.666h13.573z"}})])}}},c5f6:function(n,e,t){"use strict";var i=t("7726"),a=t("69a8"),r=t("2d95"),o=t("5dbc"),s=t("6a99"),l=t("79e5"),c=t("9093").f,g=t("11e9").f,d=t("86cc").f,p=t("aa77").trim,u=i.Number,m=u,h=u.prototype,f="Number"==r(t("2aeb")(h)),b="trim"in String.prototype,v=function(n){var e=s(n,!1);if("string"==typeof e&&e.length>2){var t,i,a,r=(e=b?e.trim():p(e,3)).charCodeAt(0);if(43===r||45===r){if(88===(t=e.charCodeAt(2))||120===t)return NaN}else if(48===r){switch(e.charCodeAt(1)){case 66:case 98:i=2,a=49;break;case 79:case 111:i=8,a=55;break;default:return+e}for(var o,l=e.slice(2),c=0,g=l.length;c<g;c++)if((o=l.charCodeAt(c))<48||o>a)return NaN;return parseInt(l,i)}}return+e};if(!u(" 0o1")||!u("0b1")||u("+0x1")){u=function(n){var e=arguments.length<1?0:n,t=this;return t instanceof u&&(f?l((function(){h.valueOf.call(t)})):"Number"!=r(t))?o(new m(v(e)),t,u):v(e)};for(var y,x=t("9e1e")?c(m):"MAX_VALUE,MIN_VALUE,NaN,NEGATIVE_INFINITY,POSITIVE_INFINITY,EPSILON,isFinite,isInteger,isNaN,isSafeInteger,MAX_SAFE_INTEGER,MIN_SAFE_INTEGER,parseFloat,parseInt,isInteger".split(","),C=0;x.length>C;C++)a(m,y=x[C])&&!a(u,y)&&d(u,y,g(m,y));u.prototype=h,h.constructor=u,t("2aba")(i,"Number",u)}},c69a:function(n,e,t){n.exports=!t("9e1e")&&!t("79e5")((function(){return 7!=Object.defineProperty(t("230e")("div"),"a",{get:function(){return 7}}).a}))},ca5a:function(n,e){var t=0,i=Math.random();n.exports=function(n){return"Symbol(".concat(void 0===n?"":n,")_",(++t+i).toString(36))}},cadf:function(n,e,t){"use strict";var i=t("9c6c"),a=t("d53b"),r=t("84f2"),o=t("6821");n.exports=t("01f9")(Array,"Array",(function(n,e){this._t=o(n),this._i=0,this._k=e}),(function(){var n=this._t,e=this._k,t=this._i++;return!n||t>=n.length?(this._t=void 0,a(1)):a(0,"keys"==e?t:"values"==e?n[t]:[t,n[t]])}),"values"),r.Arguments=r.Array,i("keys"),i("values"),i("entries")},cb7c:function(n,e,t){var i=t("d3f4");n.exports=function(n){if(!i(n))throw TypeError(n+" is not an object!");return n}},cd1c:function(n,e,t){var i=t("e853");n.exports=function(n,e){return new(i(n))(e)}},ce10:function(n,e,t){var i=t("69a8"),a=t("6821"),r=t("c366")(!1),o=t("613b")("IE_PROTO");n.exports=function(n,e){var t,s=a(n),l=0,c=[];for(t in s)t!=o&&i(s,t)&&c.push(t);for(;e.length>l;)i(s,t=e[l++])&&(~r(c,t)||c.push(t));return c}},d263:function(n,e,t){"use strict";t("386b")("fixed",(function(n){return function(){return n(this,"tt","","")}}))},d2c8:function(n,e,t){var i=t("aae3"),a=t("be13");n.exports=function(n,e,t){if(i(e))throw TypeError("String#"+t+" doesn't accept regex!");return String(a(n))}},d3f4:function(n,e){n.exports=function(n){return"object"==typeof n?null!==n:"function"==typeof n}},d53b:function(n,e){n.exports=function(n,e){return{value:e,done:!!n}}},d8e8:function(n,e){n.exports=function(n){if("function"!=typeof n)throw TypeError(n+" is not a function!");return n}},daf8:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 17 32"}},[t("path",{attrs:{d:"M14.08 4.8q2.88 0 2.88 2.048v18.24q0 2.112-2.88 2.112t-2.88-2.112V6.848q0-2.048 2.88-2.048zm-11.2 0q2.88 0 2.88 2.048v18.24q0 2.112-2.88 2.112T0 25.088V6.848Q0 4.8 2.88 4.8z"}})])}}},dcbc:function(n,e,t){var i=t("2aba");n.exports=function(n,e,t){for(var a in e)i(n,a,e[a],t);return n}},e11e:function(n,e){n.exports="constructor,hasOwnProperty,isPrototypeOf,propertyIsEnumerable,toLocaleString,toString,valueOf".split(",")},e853:function(n,e,t){var i=t("d3f4"),a=t("1169"),r=t("2b4c")("species");n.exports=function(n){var e;return a(n)&&("function"!=typeof(e=n.constructor)||e!==Array&&!a(e.prototype)||(e=void 0),i(e)&&null===(e=e[r])&&(e=void 0)),void 0===e?Array:e}},ebd6:function(n,e,t){var i=t("cb7c"),a=t("d8e8"),r=t("2b4c")("species");n.exports=function(n,e){var t,o=i(n).constructor;return void 0===o||null==(t=i(o)[r])?e:a(t)}},f559:function(n,e,t){"use strict";var i=t("5ca1"),a=t("9def"),r=t("d2c8"),o="".startsWith;i(i.P+i.F*t("5147")("startsWith"),"String",{startsWith:function(n){var e=r(this,n,"startsWith"),t=a(Math.min(arguments.length>1?arguments[1]:void 0,e.length)),i=String(n);return o?o.call(e,i,t):e.slice(t,t+i.length)===i}})},f576:function(n,e,t){"use strict";var i=t("5ca1"),a=t("2e08"),r=t("a25f");i(i.P+i.F*/Version\/10\.\d+(\.\d+)? Safari\//.test(r),"String",{padStart:function(n){return a(this,n,arguments.length>1?arguments[1]:void 0,!0)}})},f605:function(n,e){n.exports=function(n,e,t,i){if(!(n instanceof e)||void 0!==i&&i in n)throw TypeError(t+": incorrect invocation!");return n}},f751:function(n,e,t){var i=t("5ca1");i(i.S+i.F,"Object",{assign:t("7333")})},f866:function(n,e,t){"use strict";t.r(e),e.default={functional:!0,render:function(n,e){var t=e._c;return t("svg",{class:[e.data.class,e.data.staticClass],style:[e.data.style,e.data.staticStyle],attrs:{xmlns:"http://www.w3.org/2000/svg",viewBox:"0 0 29 32"}},[t("path",{attrs:{d:"M9.333 9.333h13.333v4L27.999 8l-5.333-5.333v4h-16v8h2.667V9.334zm13.334 13.334H9.334v-4L4.001 24l5.333 5.333v-4h16v-8h-2.667v5.333z"}})])}}},fab2:function(n,e,t){var i=t("7726").document;n.exports=i&&i.documentElement},fb15:function(n,e,t){"use strict";var i;(t.r(e),"undefined"!=typeof window)&&((i=window.document.currentScript)&&(i=i.src.match(/(.+\/)[^/]+\.js(\?.*)?$/))&&(t.p=i[1]));t("f751");function a(n,e,t){return e in n?Object.defineProperty(n,e,{value:t,enumerable:!0,configurable:!0,writable:!0}):n[e]=t,n}function r(n){for(var e=1;e<arguments.length;e++){var t=null!=arguments[e]?arguments[e]:{},i=Object.keys(t);"function"==typeof Object.getOwnPropertySymbols&&(i=i.concat(Object.getOwnPropertySymbols(t).filter((function(n){return Object.getOwnPropertyDescriptor(t,n).enumerable})))),i.forEach((function(e){a(n,e,t[e])}))}return n}t("c5f6");function o(n,e){if(null==n)return{};var t,i,a=function(n,e){if(null==n)return{};var t,i,a={},r=Object.keys(n);for(i=0;i<r.length;i++)t=r[i],e.indexOf(t)>=0||(a[t]=n[t]);return a}(n,e);if(Object.getOwnPropertySymbols){var r=Object.getOwnPropertySymbols(n);for(i=0;i<r.length;i++)t=r[i],e.indexOf(t)>=0||Object.prototype.propertyIsEnumerable.call(n,t)&&(a[t]=n[t])}return a}function s(n){return function(n){if(Array.isArray(n)){for(var e=0,t=new Array(n.length);e<n.length;e++)t[e]=n[e];return t}}(n)||function(n){if(Symbol.iterator in Object(n)||"[object Arguments]"===Object.prototype.toString.call(n))return Array.from(n)}(n)||function(){throw new TypeError("Invalid attempt to spread non-iterable instance")}()}t("ac6a"),t("551c"),t("6762"),t("2fdb"),t("7f7f"),t("7514"),t("6b54");function l(n,e){return function(n){if(Array.isArray(n))return n}(n)||function(n,e){var t=[],i=!0,a=!1,r=void 0;try{for(var o,s=n[Symbol.iterator]();!(i=(o=s.next()).done)&&(t.push(o.value),!e||t.length!==e);i=!0);}catch(n){a=!0,r=n}finally{try{i||null==s.return||s.return()}finally{if(a)throw r}}return t}(n,e)||function(){throw new TypeError("Invalid attempt to destructure non-iterable instance")}()}t("20d6"),t("96cf");function c(n,e,t,i,a,r,o){try{var s=n[r](o),l=s.value}catch(n){return void t(n)}s.done?e(l):Promise.resolve(l).then(i,a)}function g(n){return function(){var e=this,t=arguments;return new Promise((function(i,a){var r=n.apply(e,t);function o(n){c(r,i,a,o,s,"next",n)}function s(n){c(r,i,a,o,s,"throw",n)}o(void 0)}))}}t("d263");function d(n,e){if(!(n instanceof e))throw new TypeError("Cannot call a class as a function")}function p(n,e){for(var t=0;t<e.length;t++){var i=e[t];i.enumerable=i.enumerable||!1,i.configurable=!0,"value"in i&&(i.writable=!0),Object.defineProperty(n,i.key,i)}}function u(n,e,t){return e&&p(n.prototype,e),t&&p(n,t),n}function m(n){return(m="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(n){return typeof n}:function(n){return n&&"function"==typeof Symbol&&n.constructor===Symbol&&n!==Symbol.prototype?"symbol":typeof n})(n)}function h(n){return(h="function"==typeof Symbol&&"symbol"===m(Symbol.iterator)?function(n){return m(n)}:function(n){return n&&"function"==typeof Symbol&&n.constructor===Symbol&&n!==Symbol.prototype?"symbol":m(n)})(n)}function f(n,e){return!e||"object"!==h(e)&&"function"!=typeof e?function(n){if(void 0===n)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return n}(n):e}function b(n){return(b=Object.setPrototypeOf?Object.getPrototypeOf:function(n){return n.__proto__||Object.getPrototypeOf(n)})(n)}function v(n,e){return(v=Object.setPrototypeOf||function(n,e){return n.__proto__=e,n})(n,e)}function y(n,e){if("function"!=typeof e&&null!==e)throw new TypeError("Super expression must either be null or a function");n.prototype=Object.create(e&&e.prototype,{constructor:{value:n,writable:!0,configurable:!0}}),e&&v(n,e)}var x=t("48d3"),C=t("65d9"),A=t.n(C),w=t("8bbf"),k=t.n(w);function T(n){return Object(C.createDecorator)((function(e,t){void 0===e.inject&&(e.inject={}),Array.isArray(e.inject)||(e.inject[t]=n||t)}))}function E(n){return Object(C.createDecorator)((function(e,t){var i=e.provide;if("function"!=typeof i||!i.managed){var a=e.provide;(i=e.provide=function(){var n=Object.create(("function"==typeof a?a.call(this):a)||null);for(var e in i.managed)n[i.managed[e]]=this[e];return n}).managed={}}i.managed[t]=n||t}))}function F(n){return void 0===n&&(n={}),Object(C.createDecorator)((function(e,t){(e.props||(e.props={}))[t]=n}))}function S(n,e){void 0===e&&(e={});var t=e.deep,i=void 0!==t&&t,a=e.immediate,r=void 0!==a&&a;return Object(C.createDecorator)((function(e,t){"object"!=typeof e.watch&&(e.watch=Object.create(null)),e.watch[n]={handler:t,deep:i,immediate:r}}))}var _=t("4d26"),P=t.n(_);t("cadf"),t("456d");function q(n){var e=arguments.length>1&&void 0!==arguments[1]?arguments[1]:3e3;return new Promise((function(t,i){var a=(new Date).getTime(),r=setInterval((function(){n()?(t(),clearInterval(r)):e>0&&(new Date).getTime()-a>e&&(i(),clearInterval(r))}),100)}))}var I,z=["abort","canplay","canplaythrough","durationchange","emptied","ended","error","loadeddata","loadedmetadata","loadstart","pause","play","playing","progress","ratechange","readystatechange","seeked","seeking","stalled","suspend","timeupdate","volumechange","waiting"],j=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},N=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)};!function(n){n[n.HAVE_NOTHING=0]="HAVE_NOTHING",n[n.HAVE_METADATA=1]="HAVE_METADATA",n[n.HAVE_CURRENT_DATA=2]="HAVE_CURRENT_DATA",n[n.HAVE_FUTURE_DATA=3]="HAVE_FUTURE_DATA",n[n.HAVE_ENOUGH_DATA=4]="HAVE_ENOUGH_DATA"}(I||(I={}));var B=function(n){function e(){var n;return d(this,e),(n=f(this,b(e).call(this))).audio=new Audio,n.audioTracks=n.audio.audioTracks,n.autoplay=n.audio.autoplay,n.buffered=n.audio.buffered,n.controls=n.audio.controls,n.crossOrigin=n.audio.crossOrigin,n.currentSrc=n.audio.currentSrc,n.currentTime=n.audio.currentTime,n.defaultMuted=n.audio.defaultMuted,n.defaultPlaybackRate=n.audio.defaultPlaybackRate,n.duration=n.audio.duration,n.ended=n.audio.ended,n.error=n.audio.error,n.loop=n.audio.loop,n.mediaKeys=n.audio.mediaKeys,n.muted=n.audio.muted,n.networkState=n.audio.networkState,n.paused=n.audio.paused,n.playbackRate=n.audio.playbackRate,n.played=n.audio.played,n.preload=n.audio.preload,n.readyState=n.audio.readyState,n.seekable=n.audio.seekable,n.seeking=n.audio.seeking,n.src=n.audio.src,n.textTracks=n.audio.textTracks,n.volume=n.audio.volume,z.forEach((function(e){n.audio.addEventListener(e,(function(e){n.sync()}))})),n}return y(e,n),u(e,[{key:"sync",value:function(){var n=this;Object.keys(this.$data).forEach((function(e){"audio"!==e&&(n[e]=n.audio[e])}))}},{key:"loaded",value:function(){var n=this;return q((function(){return n.readyState>=I.HAVE_FUTURE_DATA}),0)}},{key:"srcLoaded",value:function(){var n=this;return q((function(){return n.src}),0)}},{key:"render",value:function(){return null}}]),e}(k.a),D=B=j([A.a,N("design:paramtypes",[])],B),R=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},M=function(n){function e(){var n;return d(this,e),(n=f(this,b(e).apply(this,arguments))).key="aplayer-setting",n.store=n.get(n.key),n}return y(e,n),u(e,[{key:"get",value:function(n){return JSON.parse(localStorage.getItem(n)||"[]")}},{key:"set",value:function(n){this.store=n,localStorage.setItem(this.key,JSON.stringify(n))}},{key:"render",value:function(){return null}}]),e}(k.a),L=M=R([A.a],M),O=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},G=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"isMobile",get:function(){var n=this.$ssrContext?this.$ssrContext.userAgent:window.navigator.userAgent;return/mobile/i.test(n)}}]),e}(k.a),H=G=O([A.a],G),U=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},V=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},Z=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"handleClick",value:function(n){this.$emit("click",n)}},{key:"render",value:function(){var n=arguments[0];return n("div",{class:"aplayer-pic",style:this.style,on:{click:this.handleClick}},[this.$slots.default])}},{key:"style",get:function(){var n=this.aplayer,e=n.options,t=n.currentTheme,i=n.currentMusic.cover||e.defaultCover;return{backgroundImage:i&&'url("'.concat(i,'")'),backgroundColor:t}}}]),e}(x.Component);U([T(),V("design:type",Object)],Z.prototype,"aplayer",void 0);var W=Z=U([A.a],Z),X=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},$=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},J=function(n){return t("52f0")("./".concat(n,".svg")).default},K=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"render",value:function(){var n=arguments[0],e=J(this.type);return n(e)}}]),e}(x.Component);X([F({type:String,required:!0}),$("design:type",String)],K.prototype,"type",void 0);var Y=K=X([A.a],K);t("55dd"),t("28a5"),t("a481"),t("4917");var Q=function(){function n(){d(this,n),this.xhr=new XMLHttpRequest}return u(n,[{key:"download",value:function(n){var e=this,t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:"";return new Promise((function(i,a){e.xhr.open("get",n),e.xhr.responseType=t,e.xhr.onload=function(){var n=e.xhr.status;(n>=200&&n<300||304===n)&&i(e.xhr.response)},e.xhr.onabort=a,e.xhr.onerror=a,e.xhr.ontimeout=a,e.xhr.send()}))}}]),n}(),nn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},en=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},tn=function(n){function e(){var n;return d(this,e),(n=f(this,b(e).apply(this,arguments))).lrc="",n.xhr=new Q,n.isLoading=!1,n}var t;return y(e,n),u(e,[{key:"getLyricFromCurrentMusic",value:function(){var n=this;return new Promise((function(e,t){var i=n.aplayer,a=i.lrcType,r=i.currentMusic;switch(a){case 0:e("");break;case 1:e(r.lrc);break;case 3:e(r.lrc?n.xhr.download(r.lrc):"");break;default:t(new Error("Illegal lrcType: ".concat(a)))}}))}},{key:"parseLRC",value:function(n){var e=/\[(\d+):(\d+)[.|:](\d+)\](.+)/,t=/\[(\d+):(\d+)[.|:](\d+)\]/g,i=/\[(\d+):(\d+)]()(.+)/,a=/\[(\d+):(\d+)]/g,r=this.lrc.match(/\[offset:\s*(-?\d+)\]/),o=r?Number(r[1]):0,s=[];return n.replace(/\\n/g,"\n").split("\n").forEach((function(n){return function n(r){var l=r.match(e)||r.match(i);if(l&&5===l.length){var c=60*(Number(l[1])||0)*1e3+1e3*(Number(l[2])||0)+(Number(l[3])||0)+o,g=l[4].replace(t,"").replace(a,"");g&&(s.push({time:c,text:g}),n(l[4]))}}(n)})),s.length>0&&s.sort((function(n,e){return n.time-e.time})),s}},{key:"handleChange",value:(t=g(regeneratorRuntime.mark((function n(){return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:return n.prev=0,this.isLoading=!0,this.lrc="",n.next=5,this.getLyricFromCurrentMusic();case 5:this.lrc=n.sent;case 6:return n.prev=6,this.isLoading=!1,n.finish(6);case 9:case"end":return n.stop()}}),n,this,[[0,,6,9]])}))),function(){return t.apply(this,arguments)})},{key:"render",value:function(){var n=arguments[0],e=this.visible,t=this.style,i=this.parsed,a=this.current,r=this.noLyric;return n("div",{class:P()({"aplayer-lrc":!0,"aplayer-lrc-hide":!e})},[n("div",{class:"aplayer-lrc-contents",style:t},[i.length>0?i.map((function(e,t){return n("p",{key:e.time,class:P()({"aplayer-lrc-current":a.time===e.time})},[e.text])})):n("p",{class:"aplayer-lrc-current"},[r])])])}},{key:"noLyric",get:function(){return this.aplayer.currentMusic.id?this.isLoading?"(*) ..":this.lrc?"(*) ":"(,, . ,,) ":"(   ) "}},{key:"parsed",get:function(){return this.parseLRC(this.lrc)}},{key:"current",get:function(){var n=this.aplayer,e=n.media,t=n.currentPlayed,i=this.parsed.filter((function(n){return n.time<t*e.duration*1e3}));return i&&i.length>0?i[i.length-1]:this.parsed[0]}},{key:"transitionDuration",get:function(){return this.parsed.length>1?500:0}},{key:"translateY",get:function(){var n=this.current,e=this.parsed;if(e.length<=0)return 0;var t=e.indexOf(n);return-1*(t===e.length-1?16*(t-1):16*t)}},{key:"style",get:function(){return{transitionDuration:"".concat(this.transitionDuration,"ms"),transform:"translate3d(0, ".concat(this.translateY,"px, 0)")}}}]),e}(x.Component);nn([F({type:Boolean,required:!1,default:!0}),en("design:type",Boolean)],tn.prototype,"visible",void 0),nn([T(),en("design:type",Object)],tn.prototype,"aplayer",void 0),nn([S("aplayer.lrcType",{immediate:!0}),S("aplayer.currentMusic.lrc",{immediate:!0}),en("design:type",Function),en("design:paramtypes",[]),en("design:returntype",Promise)],tn.prototype,"handleChange",null);var an=tn=nn([A.a],tn),rn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},on=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},sn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"render",value:function(){var n=arguments[0],e=this.music,t=this.aplayer.fixed;return n("div",{class:"aplayer-info"},[n("div",{class:"aplayer-music"},[n("span",{class:"aplayer-title"},[e.name]),n("span",{class:"aplayer-author"},[e.artist])]),t?null:n(an),this.$slots.default])}},{key:"music",get:function(){var n=this.aplayer.currentMusic;return{name:n.name,artist:n.artist?" - ".concat(n.artist):""}}}]),e}(x.Component);rn([T(),on("design:type",Object)],sn.prototype,"aplayer",void 0);var ln=sn=rn([A.a],sn),cn=(t("f559"),t("f576"),function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o}),gn=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},dn=function(n){function e(){var n;return d(this,e),(n=f(this,b(e).apply(this,arguments))).isDragMove=!1,n}return y(e,n),u(e,[{key:"thumbMove",value:function(n){this.isDragMove=!0,this.$emit("panMove",n)}},{key:"thumbUp",value:function(n){document.removeEventListener(this.dragMove,this.thumbMove),document.removeEventListener(this.dragEnd,this.thumbUp),this.isDragMove=!1,this.$emit("panEnd",n)}},{key:"mounted",value:function(){var n=this;this.$el.addEventListener(this.dragStart,(function(e){n.$emit("panStart",e),document.addEventListener(n.dragMove,n.thumbMove),document.addEventListener(n.dragEnd,n.thumbUp)}))}},{key:"render",value:function(){var n=arguments[0];return n("div",{class:this.classNames,style:{touchAction:"none",userSelect:"none",webkitUserDrag:"none",webkitTapHighlightColor:"rgba(0, 0, 0, 0)"}},[this.$slots.default])}},{key:"classNames",get:function(){return a({},this.panMoveClass,this.isDragMove)}},{key:"dragStart",get:function(){return this.isMobile?"touchstart":"mousedown"}},{key:"dragMove",get:function(){return this.isMobile?"touchmove":"mousemove"}},{key:"dragEnd",get:function(){return this.isMobile?"touchend":"mouseup"}}]),e}(x.Component);cn([F({type:String,required:!1}),gn("design:type",String)],dn.prototype,"panMoveClass",void 0);var pn=dn=cn([A()({mixins:[H]})],dn),un=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},mn=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},hn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"handleClick",value:function(){this.$emit("click")}},{key:"render",value:function(){var n=arguments[0];return n("button",{attrs:{type:"button"},class:"aplayer-icon aplayer-icon-".concat(this.type),on:{click:this.handleClick}},[n(Y,{attrs:{type:this.icon}})])}}]),e}(x.Component);un([F({type:String,required:!0}),mn("design:type",String)],hn.prototype,"type",void 0),un([F({type:String,required:!0}),mn("design:type",String)],hn.prototype,"icon",void 0);var fn=hn=un([A.a],hn),bn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},vn=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},yn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"handleChange",value:function(n){var e=this.$refs.progressBar,t=e.getBoundingClientRect().left,i=((n.type.startsWith("touch")?n.changedTouches[0].clientX:n.clientX)-t)/e.offsetWidth;i=Math.min(i,1),i=Math.max(i,0),this.handleChangeProgress(n,i)}},{key:"render",value:function(){var n=arguments[0],e=this.aplayer,t=e.currentTheme,i=e.currentLoaded,a=e.currentPlayed;return n(pn,{class:"aplayer-bar-wrap",on:{panMove:this.handleChange,panEnd:this.handleChange}},[n("div",{ref:"progressBar",class:"aplayer-bar"},[n("div",{class:"aplayer-loaded",style:{width:"".concat(100*i,"%")}}),n("div",{class:"aplayer-played",style:{width:"".concat(100*a,"%"),backgroundColor:t}},[n("span",{class:"aplayer-thumb",style:{backgroundColor:t}},[n("span",{class:"aplayer-loading-icon"},[n(Y,{attrs:{type:"loading"}})])])])])])}}]),e}(x.Component);bn([T(),vn("design:type",Object)],yn.prototype,"aplayer",void 0),bn([T(),vn("design:type",Function)],yn.prototype,"handleChangeProgress",void 0);var xn=yn=bn([A.a],yn),Cn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},An=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},wn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"timeSecondsFormat",value:function(){var n=arguments.length>0&&void 0!==arguments[0]?arguments[0]:0,e=Math.floor(n/60)||0,t=Math.floor(n%60)||0;return"".concat(e.toString().padStart(2,"0"),":").concat(t.toString().padStart(2,"0"))}},{key:"handleToggleVolume",value:function(){var n=this.aplayer,e=n.currentVolume,t=n.currentSettings;this.handleChangeVolume(e>0?0:t.volume)}},{key:"handleClickVolumeBar",value:function(n){this.handlePanMove(n)}},{key:"handlePanMove",value:function(n){var e=this.$refs.volumeBar,t=e.getBoundingClientRect().bottom;if(!(t<=0)){var i=n.type.startsWith("touch")?n.changedTouches[0].clientY:n.clientY,a=Math.round(t-i)/e.offsetHeight;a=Math.min(a,1),a=Math.max(a,0),this.handleChangeVolume(a)}}},{key:"render",value:function(){var n=arguments[0],e=this.ptime,t=this.dtime,i=this.volumeIcon,a=this.aplayer,r=a.lrcType,o=a.currentTheme,s=a.currentVolume,l=a.currentOrder,c=a.currentLoop;return n("div",{class:"aplayer-controller"},[n(xn),n("div",{class:"aplayer-time"},[n("span",{class:"aplayer-time-inner"},[n("span",{class:"aplayer-ptime"},[e])," /"," ",n("span",{class:"aplayer-dtime"},[t])," "]),n("span",{class:"aplayer-icon aplayer-icon-back",on:{click:this.handleSkipBack}},[n(Y,{attrs:{type:"skip"}})]),n("span",{class:"aplayer-icon aplayer-icon-play",on:{click:this.handleTogglePlay}},[n(Y,{attrs:{type:this.playIcon}})]),n("span",{class:"aplayer-icon aplayer-icon-forward",on:{click:this.handleSkipForward}},[n(Y,{attrs:{type:"skip"}})]),n("div",{class:"aplayer-volume-wrap"},[n(fn,{attrs:{type:"volume-".concat(i),icon:"volume-".concat(i)},on:{click:this.handleToggleVolume}}),n(pn,{class:"aplayer-volume-bar-wrap",attrs:{panMoveClass:"aplayer-volume-bar-wrap-active"},on:{panMove:this.handlePanMove}},[n("div",{ref:"volumeBar",class:"aplayer-volume-bar",on:{click:this.handleClickVolumeBar}},[n("div",{class:"aplayer-volume",style:{height:"".concat(100*s,"%"),backgroundColor:o}})])])])," ",n(fn,{attrs:{type:"order",icon:"order-".concat(l)},on:{click:this.handleToggleOrderMode}})," ",n(fn,{attrs:{type:"loop",icon:"loop-".concat(c)},on:{click:this.handleToggleLoopMode}})," ",n(fn,{attrs:{type:"menu",icon:"menu"},on:{click:this.handleTogglePlaylist}}),0!==r?n(fn,{attrs:{type:"lrc",icon:"lrc"},on:{click:this.handleToggleLyric}}):null])])}},{key:"playIcon",get:function(){return this.aplayer.media.paused?"play":"pause"}},{key:"volumeIcon",get:function(){var n=this.aplayer.currentVolume;return n<=0?"off":n>=.95?"up":"down"}},{key:"ptime",get:function(){var n=this.aplayer,e=n.media,t=n.currentPlayed;return this.timeSecondsFormat(t*e.duration)}},{key:"dtime",get:function(){return this.timeSecondsFormat(this.aplayer.media.duration)}}]),e}(x.Component);Cn([T(),An("design:type",Object)],wn.prototype,"aplayer",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleSkipBack",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleSkipForward",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleTogglePlay",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleToggleOrderMode",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleToggleLoopMode",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleTogglePlaylist",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleToggleLyric",void 0),Cn([T(),An("design:type",Function)],wn.prototype,"handleChangeVolume",void 0);var kn=wn=Cn([A.a],wn),Tn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},En=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},Fn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}return y(e,n),u(e,[{key:"handleTogglePlay",value:function(){this.$emit("togglePlay")}},{key:"handleSkipBack",value:function(){this.$emit("skipBack")}},{key:"handleSkipForward",value:function(){this.$emit("skipForward")}},{key:"handleToggleOrderMode",value:function(){this.$emit("toggleOrderMode")}},{key:"handleToggleLoopMode",value:function(){this.$emit("toggleLoopMode")}},{key:"handleTogglePlaylist",value:function(){this.$emit("togglePlaylist")}},{key:"handleToggleLyric",value:function(){this.$emit("toggleLyric")}},{key:"handleChangeVolume",value:function(n){this.$emit("changeVolume",n)}},{key:"handleChangeProgress",value:function(n,e){this.$emit("changeProgress",n,e)}},{key:"handleMiniSwitcher",value:function(){this.$emit("miniSwitcher")}},{key:"render",value:function(){var n=arguments[0],e=this.playIcon,t=this.notice;return n("div",{class:"aplayer-body"},[n(W,{on:{click:this.handleTogglePlay}},[n("div",{class:"aplayer-button aplayer-".concat(e)},[n(Y,{attrs:{type:e}})])]),n(ln,[n(kn,{on:{skipBack:this.handleSkipBack,skipForward:this.handleSkipForward,togglePlay:this.handleTogglePlay,toggleOrderMode:this.handleToggleOrderMode,toggleLoopMode:this.handleToggleLoopMode,togglePlaylist:this.handleTogglePlaylist,toggleLyric:this.handleToggleLyric,changeVolume:this.handleChangeVolume,changeProgress:this.handleChangeProgress}})]),n("div",{class:"aplayer-notice",style:{opacity:t.opacity}},[t.text]),n("div",{class:"aplayer-miniswitcher",on:{click:this.handleMiniSwitcher}},[n(fn,{attrs:{type:"miniswitcher",icon:"right"}})])])}},{key:"playIcon",get:function(){return this.aplayer.media.paused?"play":"pause"}}]),e}(x.Component);Tn([F({type:Object,required:!0}),En("design:type",Object)],Fn.prototype,"notice",void 0),Tn([T(),En("design:type",Object)],Fn.prototype,"aplayer",void 0),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleTogglePlay",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleSkipBack",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleSkipForward",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleToggleOrderMode",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleToggleLoopMode",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleTogglePlaylist",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[]),En("design:returntype",void 0)],Fn.prototype,"handleToggleLyric",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[Number]),En("design:returntype",void 0)],Fn.prototype,"handleChangeVolume",null),Tn([E(),En("design:type",Function),En("design:paramtypes",[Object,Number]),En("design:returntype",void 0)],Fn.prototype,"handleChangeProgress",null);var Sn,_n,Pn=Fn=Tn([A.a],Fn),qn=function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o},In=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},zn=function(n){function e(){return d(this,e),f(this,b(e).apply(this,arguments))}var t;return y(e,n),u(e,[{key:"handleChangeScrollTop",value:(t=g(regeneratorRuntime.mark((function n(){return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:return n.next=2,this.$nextTick();case 2:this.visible&&(this.$refs.list.scrollTop=this.scrollTop);case 3:case"end":return n.stop()}}),n,this)}))),function(){return t.apply(this,arguments)})},{key:"render",value:function(){var n=this,e=arguments[0],t=this.listHeight,i=this.dataSource,a=this.currentMusic,r=this.aplayer.currentTheme;return e("ol",{ref:"list",class:"aplayer-list",style:{height:"".concat(t,"px")}},[i.map((function(t,i){return e("li",{key:t.id,class:P()({"aplayer-list-light":t.id===a.id}),on:{click:function(){return n.$emit("change",t,i)}}},[e("span",{class:"aplayer-list-cur",style:{backgroundColor:r}}),e("span",{class:"aplayer-list-index"},[i+1])," ",e("span",{class:"aplayer-list-title"},[t.name]),e("span",{class:"aplayer-list-author"},[t.artist])])}))])}},{key:"listHeight",get:function(){var n=this.visible,e=this.dataSource;return n?Math.min(33*e.length,Number(this.aplayer.listMaxHeight)):0}}]),e}(x.Component);qn([F({type:Boolean,required:!1,default:!0}),In("design:type",Boolean)],zn.prototype,"visible",void 0),qn([F({type:Object,required:!0}),In("design:type","function"==typeof(Sn="undefined"!=typeof APlayer&&APlayer.Audio)?Sn:Object)],zn.prototype,"currentMusic",void 0),qn([F({type:Array,required:!0}),In("design:type","function"==typeof(_n="undefined"!=typeof Array&&Array)?_n:Object)],zn.prototype,"dataSource",void 0),qn([F({type:Number,required:!0}),In("design:type",Number)],zn.prototype,"scrollTop",void 0),qn([T(),In("design:type",Object)],zn.prototype,"aplayer",void 0),qn([S("scrollTop",{immediate:!0}),S("dataSource",{immediate:!0,deep:!0}),S("visible"),In("design:type",Function),In("design:paramtypes",[]),In("design:returntype",Promise)],zn.prototype,"handleChangeScrollTop",null);var jn,Nn,Bn,Dn,Rn,Mn,Ln=zn=qn([A.a],zn),On=(t("610a"),function(n,e,t,i){var a,r=arguments.length,o=r<3?e:null===i?i=Object.getOwnPropertyDescriptor(e,t):i;if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.decorate)o=Reflect.decorate(n,e,t,i);else for(var s=n.length-1;s>=0;s--)(a=n[s])&&(o=(r<3?a(o):r>3?a(e,t,o):a(e,t))||o);return r>3&&o&&Object.defineProperty(e,t,o),o}),Gn=function(n,e){if("object"===("undefined"==typeof Reflect?"undefined":h(Reflect))&&"function"==typeof Reflect.metadata)return Reflect.metadata(n,e)},Hn=[],Un=new L,Vn=null;"undefined"!=typeof BroadcastChannel&&(Vn=new BroadcastChannel("aplayer"));var Zn=function(n){function e(){var n;return d(this,e),(n=f(this,b(e).apply(this,arguments))).isDraggingProgressBar=!1,n.isAwaitChangeProgressBar=!1,n.isMini=null!==n.mini?n.mini:n.fixed,n.isArrow=!1,n.canPlay=!n.isMobile&&n.autoplay,n.listVisible=!n.listFolded,n.lyricVisible=!0,n.img=new Image,n.xhr=new Q,n.media=new D,n.player=n.media.audio,n.store=Un,n.currentMusic={id:NaN,name:"",artist:"(   )",url:""},n.currentPlayed=0,n.currentVolume=n.volume,n.currentLoop=n.loop,n.currentOrder=n.order,n.currentTheme=n.currentMusic.theme||n.theme,n.notice={text:"",time:2e3,opacity:0},n}var t,i,a,c,p;return y(e,n),u(e,[{key:"handleChangePlayList",value:(p=g(regeneratorRuntime.mark((function n(e,t){var i,a,r,o,s,c,g,d;return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:if(t&&(i=e.length,a=t.length,i!==a&&(i<=0?this.$emit("listClear"):i>a?this.$emit("listAdd"):(this.currentOrderIndex<0&&(r=this.currentMusic,o=r.id,s=r.url,c=t.findIndex((function(n){return n.id===o||n.url===s})),Object.assign(this.currentMusic,t[c-1])),this.canPlay=!this.player.paused,this.$emit("listRemove")))),!(this.orderList.length>0)){n.next=6;break}return this.currentMusic.id?(this.canPlay=!this.player.paused,d=this.orderList[this.currentOrderIndex]||this.orderList[0],Object.assign(this.currentMusic,d)):(g=l(this.currentList,1),this.currentMusic=g[0]),n.next=5,this.$nextTick();case 5:this.canPlay=!0;case 6:case"end":return n.stop()}}),n,this)}))),function(n,e){return p.apply(this,arguments)})},{key:"handleChangeCurrentMusic",value:(c=g(regeneratorRuntime.mark((function n(e,t){var i,a,r=this;return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:if(e.theme?this.currentTheme=e.theme:(i=e.cover||this.options.defaultCover)&&setTimeout(g(regeneratorRuntime.mark((function n(){return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:return n.prev=0,n.next=3,r.getThemeColorFromCover(i);case 3:r.currentTheme=n.sent,n.next=9;break;case 6:n.prev=6,n.t0=n.catch(0),r.currentTheme=e.theme||r.theme;case 9:case"end":return n.stop()}}),n,this,[[0,6]])})))),!e.url){n.next=15;break}if((void 0!==t&&t.url)===e.url&&this.player.src===e.url){n.next=14;break}return this.currentPlayed=0,t&&t.id&&(this.handleChangeSettings(),this.$emit("listSwitch",e)),n.next=7,this.getAudioUrl(e);case 7:(a=n.sent)&&(this.player.src=a),this.player.playbackRate=e.speed||1,this.player.preload=this.preload,this.player.volume=this.currentVolume,this.player.currentTime=0,this.player.onerror=function(n){r.showNotice(n.toString())};case 14:this.canPlay&&this.play();case 15:case"end":return n.stop()}}),n,this)}))),function(n,e){return c.apply(this,arguments)})},{key:"handleChangeVolume",value:function(n){this.currentVolume=n}},{key:"handleChangeCurrentVolume",value:function(){this.player.volume=this.currentVolume,this.$emit("update:volume",this.currentVolume)}},{key:"handleChangeCurrentTime",value:function(){this.isDraggingProgressBar||this.isAwaitChangeProgressBar||(this.currentPlayed=this.media.currentTime/this.media.duration||0)}},{key:"handleChangeSettings",value:function(){var n={currentTime:this.media.currentTime,duration:this.media.duration,paused:this.media.paused,mini:this.isMini,lrc:this.lyricVisible,list:this.listVisible,volume:this.currentVolume,loop:this.currentLoop,order:this.currentOrder,music:this.currentMusic};n.volume<=0&&(n.volume=this.currentSettings.volume),this.saveSettings(n)}},{key:"handleChangeEnded",value:function(){if(this.media.ended)switch(this.currentPlayed=0,this.currentLoop){default:case"all":this.handleSkipForward();break;case"one":this.play();break;case"none":if(this.currentIndex===this.currentList.length-1){var n=l(this.currentList,1);this.currentMusic=n[0],this.pause(),this.canPlay=!1}else this.handleSkipForward()}}},{key:"handleChangeMini",value:function(){this.isMini=this.mini}},{key:"handleChangeCurrentMini",value:(a=g(regeneratorRuntime.mark((function n(e,t){var i;return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:return n.next=2,this.$nextTick();case 2:i=this.$refs.container,this.isArrow=i&&i.offsetWidth<=300,void 0!==t&&(this.$emit("update:mini",this.isMini),this.handleChangeSettings());case 5:case"end":return n.stop()}}),n,this)}))),function(n,e){return a.apply(this,arguments)})},{key:"handleChangeLoop",value:function(){this.currentLoop=this.loop}},{key:"handleChangeCurrentLoop",value:function(){this.$emit("update:loop",this.currentLoop),this.handleChangeSettings()}},{key:"handleChangeOrder",value:function(){this.currentOrder=this.order}},{key:"handleChangeCurrentOrder",value:function(){this.$emit("update:order",this.currentOrder),this.handleChangeSettings()}},{key:"handleChangeListVisible",value:function(){this.$emit(this.listVisible?"listShow":"listHide"),this.$emit("update:listFolded",this.listVisible),this.handleChangeSettings()}},{key:"handleChangeLyricVisible",value:function(){this.$emit(this.lyricVisible?"lrcShow":"lrcHide"),this.handleChangeSettings()}},{key:"play",value:(i=g(regeneratorRuntime.mark((function n(){return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:return n.prev=0,this.mutex&&this.pauseOtherInstances(),n.next=4,this.player.play();case 4:n.next=10;break;case 6:n.prev=6,n.t0=n.catch(0),this.showNotice(n.t0.message),this.player.pause();case 10:case"end":return n.stop()}}),n,this,[[0,6]])}))),function(){return i.apply(this,arguments)})},{key:"pause",value:function(){this.player.pause()}},{key:"toggle",value:function(){this.media.paused?this.play():this.pause()}},{key:"seeking",value:(t=g(regeneratorRuntime.mark((function n(e){var t,i,a=arguments;return regeneratorRuntime.wrap((function(n){for(;;)switch(n.prev=n.next){case 0:if(t=!(a.length>1&&void 0!==a[1])||a[1],n.prev=1,this.isAwaitChangeProgressBar=!0,"none"!==this.preload){n.next=11;break}if(this.player.src){n.next=7;break}return n.next=7,this.media.srcLoaded();case 7:return i=this.player.paused,n.next=10,this.play();case 10:t&&i&&this.pause();case 11:return t&&this.pause(),n.next=14,this.media.loaded();case 14:this.player.currentTime=e*this.media.duration,t||(this.play(),Vn&&this.mutex&&Vn.postMessage("mutex")),n.next=21;break;case 18:n.prev=18,n.t0=n.catch(1),this.showNotice(n.t0.message);case 21:return n.prev=21,this.isAwaitChangeProgressBar=!1,n.finish(21);case 24:case"end":return n.stop()}}),n,this,[[1,18,21,24]])}))),function(n){return t.apply(this,arguments)})},{key:"seek",value:function(n){this.seeking(n/this.media.duration,this.media.paused)}},{key:"switch",value:function(n){switch(h(n)){case"number":this.currentMusic=this.orderList[Math.min(Math.max(0,n),this.orderList.length-1)];break;default:var e=this.orderList.find((function(e){return"string"==typeof e.name&&e.name.includes(n)}));e&&(this.currentMusic=e)}}},{key:"skipBack",value:function(){var n=this.getPlayIndexByMode("skipBack");this.currentMusic=r({},this.currentList[n])}},{key:"skipForward",value:function(){var n=this.getPlayIndexByMode("skipForward");this.currentMusic=r({},this.currentList[n])}},{key:"showLrc",value:function(){this.lyricVisible=!0}},{key:"hideLrc",value:function(){this.lyricVisible=!1}},{key:"toggleLrc",value:function(){this.lyricVisible=!this.lyricVisible}},{key:"showList",value:function(){this.listVisible=!0}},{key:"hideList",value:function(){this.listVisible=!1}},{key:"toggleList",value:function(){this.listVisible=!this.listVisible}},{key:"showNotice",value:function(n){var e=this,t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:2e3,i=arguments.length>2&&void 0!==arguments[2]?arguments[2]:.8;return new Promise((function(a){e.isMini?(console.warn("aplayer notice:",n),a()):(e.notice={text:n,time:t,opacity:i},e.$emit("noticeShow"),t>0&&setTimeout((function(){e.notice.opacity=0,e.$emit("noticeHide"),a()}),t))}))}},{key:"getThemeColorFromCover",value:function(n){var e=this;return new Promise(function(){var t=g(regeneratorRuntime.mark((function t(i,a){var r,o;return regeneratorRuntime.wrap((function(t){for(;;)switch(t.prev=t.next){case 0:if(t.prev=0,"undefined"==typeof ColorThief){t.next=12;break}return t.next=4,e.xhr.download(n,"blob");case 4:r=t.sent,(o=new FileReader).onload=function(){e.img.src=o.result,e.img.onload=function(){var n=l((new ColorThief).getColor(e.img),3),t=n[0],a=n[1],r=n[2],o="rgb(".concat(t,", ").concat(a,", ").concat(r,")");i(o||e.currentMusic.theme||e.theme)},e.img.onabort=a,e.img.onerror=a},o.onabort=a,o.onerror=a,o.readAsDataURL(r),t.next=13;break;case 12:i(e.currentMusic.theme||e.theme);case 13:t.next=18;break;case 15:t.prev=15,t.t0=t.catch(0),i(e.currentMusic.theme||e.theme);case 18:case"end":return t.stop()}}),t,this,[[0,15]])})));return function(n,e){return t.apply(this,arguments)}}())}},{key:"getAudioUrl",value:function(n){var e=this;return new Promise((function(t,i){var a=n.type;if(a&&e.customAudioType&&e.customAudioType[a])"function"==typeof e.customAudioType[a]?e.customAudioType[a](e.player,n,e):console.error("Illegal customType: ".concat(a)),t();else if(a&&"auto"!==a||(a=/m3u8(#|\?|$)/i.test(n.url)?"hls":"normal"),"hls"===a)try{if(Hls.isSupported()){var r=new Hls;r.loadSource(n.url),r.attachMedia(e.player),t()}else e.player.canPlayType("application/x-mpegURL")||e.player.canPlayType("application/vnd.apple.mpegURL")?t(n.url):i(new Error("HLS is not supported."))}catch(n){i(new Error("HLS is not supported."))}else t(n.url)}))}},{key:"getPlayIndexByMode",value:function(n){var e=this.currentList.length,t=this.currentIndex;return("skipBack"===n?e+(t-1):t+1)%e}},{key:"pauseOtherInstances",value:function(){var n=this;Hn.filter((function(e){return e!==n})).forEach((function(n){return n.pause()}))}},{key:"saveSettings",value:function(n){var e=Hn.indexOf(this);null===n&&delete Hn[e],this.store.set(void 0!==this.settings[e]?this.settings.map((function(t,i){return i===e?n:t})):s(this.settings).concat([n]))}},{key:"handleSkipBack",value:function(){this.skipBack()}},{key:"handleSkipForward",value:function(){this.skipForward()}},{key:"handleTogglePlay",value:function(){this.toggle()}},{key:"handleToggleOrderMode",value:function(){this.currentOrder="list"===this.currentOrder?"random":"list"}},{key:"handleToggleLoopMode",value:function(){this.currentLoop="all"===this.currentLoop?"one":"one"===this.currentLoop?"none":"all"}},{key:"handleTogglePlaylist",value:function(){this.toggleList()}},{key:"handleToggleLyric",value:function(){this.toggleLrc()}},{key:"handleChangeProgress",value:function(n,e){this.currentPlayed=e,this.isDraggingProgressBar=n.type.includes("move"),["touchend","mouseup"].includes(n.type)&&this.seeking(e,this.media.paused)}},{key:"handleMiniSwitcher",value:function(){this.isMini=!this.isMini}},{key:"handleChangePlaylist",value:function(n,e){n.id===this.currentMusic.id?this.handleTogglePlay():this.currentMusic=this.orderList[e]}},{key:"beforeMount",value:function(){var n=this;this.store.key=this.storageName;var e=Hn.findIndex((function(n){return!n}));if(e>-1?Hn[e]=this:Hn.push(this),this.currentSettings){var t=this.currentSettings,i=t.mini,a=t.lrc,r=t.list,o=t.volume,s=t.loop,l=t.order,c=t.music,g=t.currentTime,d=t.duration,p=t.paused;this.isMini=i,this.lyricVisible=a,this.listVisible=r,this.currentVolume=o,this.currentLoop=s,this.currentOrder=l,c&&(this.currentMusic=c,!this.isMobile&&d&&this.seeking(g/d,p))}Vn&&this.mutex&&Vn.addEventListener("message",(function(e){"mutex"===e.data&&n.pause()})),z.forEach((function(e){n.player.addEventListener(e,(function(t){return n.$emit(e,t)}))}))}},{key:"beforeDestroy",value:function(){this.pause(),this.saveSettings(null),this.$emit("destroy"),this.$el.remove()}},{key:"render",value:function(){var n=arguments[0],e=this.dataSource,t=this.fixed,i=this.lrcType,a=this.isMini,r=this.isMobile,o=this.isArrow,s=this.isLoading,l=this.notice,c=this.listVisible,g=this.listScrollTop,d=this.currentMusic,p=this.lyricVisible;return n("div",{ref:"container",class:P()({aplayer:!0,"aplayer-withlist":e.length>1,"aplayer-withlrc":!t&&0!==i&&p,"aplayer-narrow":a,"aplayer-fixed":t,"aplayer-mobile":r,"aplayer-arrow":o,"aplayer-loading":s})},[n(Pn,{attrs:{notice:l},on:{skipBack:this.handleSkipBack,skipForward:this.handleSkipForward,togglePlay:this.handleTogglePlay,toggleOrderMode:this.handleToggleOrderMode,toggleLoopMode:this.handleToggleLoopMode,togglePlaylist:this.handleTogglePlaylist,toggleLyric:this.handleToggleLyric,changeVolume:this.handleChangeVolume,changeProgress:this.handleChangeProgress,miniSwitcher:this.handleMiniSwitcher}}),n(Ln,{attrs:{visible:c,scrollTop:g,currentMusic:d,dataSource:e},on:{change:this.handleChangePlaylist}}),t&&0!==i?n(an,{attrs:{visible:p}}):null])}},{key:"aplayer",get:function(){return this}},{key:"settings",get:function(){return this.store.store}},{key:"currentSettings",get:function(){return this.settings[Hn.indexOf(this)]}},{key:"currentList",get:function(){return"list"===this.currentOrder?this.orderList:this.randomList}},{key:"dataSource",get:function(){return(Array.isArray(this.audio)?this.audio:[this.audio]).filter((function(n){return n})).map((function(n,e){return r({id:e+1},n)}))}},{key:"orderList",get:function(){var n=function(n,e){return"string"==typeof n?n:n.data&&n.data.attrs&&n.data.attrs["data-".concat(e)]};return this.dataSource.map((function(e){var t=e.name,i=e.artist;return r({},o(e,["name","artist"]),{name:n(t,"name"),artist:n(i,"artist")})}))}},{key:"randomList",get:function(){return function(n){for(var e=n.length-1;e>=0;e--){var t=Math.floor(Math.random()*(e+1)),i=n[t];n[t]=n[e],n[e]=i}return n}(s(this.orderList))}},{key:"isLoading",get:function(){var n=this.preload,e=this.currentPlayed,t=this.currentLoaded,i=this.media,a=i.src,r=i.paused,o=i.duration,s=!!a&&(e>t||!o);return"none"===n?!r&&s:s}},{key:"listScrollTop",get:function(){return 33*this.currentOrderIndex}},{key:"currentIndex",get:function(){return"list"===this.currentOrder?this.currentOrderIndex:this.currentRandomIndex}},{key:"currentOrderIndex",get:function(){var n=this.currentMusic,e=n.id,t=n.url;return this.orderList.findIndex((function(n){return n.id===e||n.url===t}))}},{key:"currentRandomIndex",get:function(){var n=this.currentMusic,e=n.id,t=n.url;return this.randomList.findIndex((function(n){return n.id===e||n.url===t}))}},{key:"currentLoaded",get:function(){if(this.media.readyState<I.HAVE_FUTURE_DATA)return 0;var n=this.media.buffered.length;return n>0?this.media.buffered.end(n-1)/this.media.duration:1}}]),e}(x.Component);Zn.version="2.0.0-beta.5",On([F({type:Boolean,required:!1,default:!1}),Gn("design:type",Boolean)],Zn.prototype,"fixed",void 0),On([F({type:Boolean,required:!1,default:null}),Gn("design:type",Boolean)],Zn.prototype,"mini",void 0),On([F({type:Boolean,required:!1,default:!1}),Gn("design:type",Boolean)],Zn.prototype,"autoplay",void 0),On([F({type:String,required:!1,default:"#b7daff"}),Gn("design:type",String)],Zn.prototype,"theme",void 0),On([F({type:String,required:!1,default:"all"}),Gn("design:type","function"==typeof(jn=void 0!==Zn&&Zn.LoopMode)?jn:Object)],Zn.prototype,"loop",void 0),On([F({type:String,required:!1,default:"list"}),Gn("design:type","function"==typeof(Nn=void 0!==Zn&&Zn.OrderMode)?Nn:Object)],Zn.prototype,"order",void 0),On([F({type:String,required:!1,default:"auto"}),Gn("design:type","function"==typeof(Bn=void 0!==Zn&&Zn.Preload)?Bn:Object)],Zn.prototype,"preload",void 0),On([F({type:Number,required:!1,default:.7}),Gn("design:type",Number)],Zn.prototype,"volume",void 0),On([F({type:[Object,Array],required:!0}),Gn("design:type",Object)],Zn.prototype,"audio",void 0),On([F({type:Object,required:!1}),Gn("design:type",Object)],Zn.prototype,"customAudioType",void 0),On([F({type:Boolean,required:!1,default:!0}),Gn("design:type",Boolean)],Zn.prototype,"mutex",void 0),On([F({type:Number,required:!1,default:0}),Gn("design:type","function"==typeof(Dn=void 0!==Zn&&Zn.LrcType)?Dn:Object)],Zn.prototype,"lrcType",void 0),On([F({type:Boolean,required:!1,default:!1}),Gn("design:type",Boolean)],Zn.prototype,"listFolded",void 0),On([F({type:Number,required:!1,default:250}),Gn("design:type",Number)],Zn.prototype,"listMaxHeight",void 0),On([F({type:String,required:!1,default:"aplayer-setting"}),Gn("design:type",String)],Zn.prototype,"storageName",void 0),On([E(),Gn("design:type",Object),Gn("design:paramtypes",[])],Zn.prototype,"aplayer",null),On([S("orderList",{immediate:!0,deep:!0}),Gn("design:type",Function),Gn("design:paramtypes",[Array,Array]),Gn("design:returntype",Promise)],Zn.prototype,"handleChangePlayList",null),On([S("currentMusic",{immediate:!0,deep:!0}),Gn("design:type",Function),Gn("design:paramtypes",["function"==typeof(Rn=void 0!==Zn&&Zn.Audio)?Rn:Object,"function"==typeof(Mn=void 0!==Zn&&Zn.Audio)?Mn:Object]),Gn("design:returntype",Promise)],Zn.prototype,"handleChangeCurrentMusic",null),On([S("volume"),Gn("design:type",Function),Gn("design:paramtypes",[Number]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeVolume",null),On([S("currentVolume"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeCurrentVolume",null),On([S("media.currentTime"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeCurrentTime",null),On([S("media.$data",{deep:!0}),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeSettings",null),On([S("media.ended"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeEnded",null),On([S("mini"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeMini",null),On([S("isMini",{immediate:!0}),Gn("design:type",Function),Gn("design:paramtypes",[Boolean,Boolean]),Gn("design:returntype",Promise)],Zn.prototype,"handleChangeCurrentMini",null),On([S("loop"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeLoop",null),On([S("currentLoop"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeCurrentLoop",null),On([S("order"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeOrder",null),On([S("currentOrder"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeCurrentOrder",null),On([S("listVisible"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeListVisible",null),On([S("lyricVisible"),Gn("design:type",Function),Gn("design:paramtypes",[]),Gn("design:returntype",void 0)],Zn.prototype,"handleChangeLyricVisible",null);var Wn=Zn=On([A()({mixins:[H]})],Zn);t.d(e,"APlayer",(function(){return Wn}));e.default=function(n,e){var t=r({},{productionTip:!0,defaultCover:"https://avatars2.githubusercontent.com/u/20062482?s=270"},e);Object.assign(Wn.prototype,{options:t}),n.component("aplayer",Wn),n.component("APlayer",Wn),t.productionTip&&console.log("%c vue-aplayer %c v".concat("2.0.0-beta.5"," ").concat("dd10c50"," %c"),"background: #35495e; padding: 1px; border-radius: 3px 0 0 3px; color: #fff","background: #41b883; padding: 1px; border-radius: 0 3px 3px 0; color: #fff","background: transparent")}},fdef:function(n,e){n.exports="\t\n\v\f\r \u2028\u2029\ufeff"}})},function(n,e,t){"use strict";t(93)},function(n,e,t){"use strict";t(94)},function(n,e,t){"use strict";t.r(e);var i=t(0);
/*!
  * vue-router v3.6.5
  * (c) 2022 Evan You
  * @license MIT
  */function a(n,e){for(var t in e)n[t]=e[t];return n}var r=/[!'()*]/g,o=function(n){return"%"+n.charCodeAt(0).toString(16)},s=/%2C/g,l=function(n){return encodeURIComponent(n).replace(r,o).replace(s,",")};function c(n){try{return decodeURIComponent(n)}catch(n){0}return n}var g=function(n){return null==n||"object"==typeof n?n:String(n)};function d(n){var e={};return(n=n.trim().replace(/^(\?|#|&)/,""))?(n.split("&").forEach((function(n){var t=n.replace(/\+/g," ").split("="),i=c(t.shift()),a=t.length>0?c(t.join("=")):null;void 0===e[i]?e[i]=a:Array.isArray(e[i])?e[i].push(a):e[i]=[e[i],a]})),e):e}function p(n){var e=n?Object.keys(n).map((function(e){var t=n[e];if(void 0===t)return"";if(null===t)return l(e);if(Array.isArray(t)){var i=[];return t.forEach((function(n){void 0!==n&&(null===n?i.push(l(e)):i.push(l(e)+"="+l(n)))})),i.join("&")}return l(e)+"="+l(t)})).filter((function(n){return n.length>0})).join("&"):null;return e?"?"+e:""}var u=/\/?$/;function m(n,e,t,i){var a=i&&i.options.stringifyQuery,r=e.query||{};try{r=h(r)}catch(n){}var o={name:e.name||n&&n.name,meta:n&&n.meta||{},path:e.path||"/",hash:e.hash||"",query:r,params:e.params||{},fullPath:v(e,a),matched:n?b(n):[]};return t&&(o.redirectedFrom=v(t,a)),Object.freeze(o)}function h(n){if(Array.isArray(n))return n.map(h);if(n&&"object"==typeof n){var e={};for(var t in n)e[t]=h(n[t]);return e}return n}var f=m(null,{path:"/"});function b(n){for(var e=[];n;)e.unshift(n),n=n.parent;return e}function v(n,e){var t=n.path,i=n.query;void 0===i&&(i={});var a=n.hash;return void 0===a&&(a=""),(t||"/")+(e||p)(i)+a}function y(n,e,t){return e===f?n===e:!!e&&(n.path&&e.path?n.path.replace(u,"")===e.path.replace(u,"")&&(t||n.hash===e.hash&&x(n.query,e.query)):!(!n.name||!e.name)&&(n.name===e.name&&(t||n.hash===e.hash&&x(n.query,e.query)&&x(n.params,e.params))))}function x(n,e){if(void 0===n&&(n={}),void 0===e&&(e={}),!n||!e)return n===e;var t=Object.keys(n).sort(),i=Object.keys(e).sort();return t.length===i.length&&t.every((function(t,a){var r=n[t];if(i[a]!==t)return!1;var o=e[t];return null==r||null==o?r===o:"object"==typeof r&&"object"==typeof o?x(r,o):String(r)===String(o)}))}function C(n){for(var e=0;e<n.matched.length;e++){var t=n.matched[e];for(var i in t.instances){var a=t.instances[i],r=t.enteredCbs[i];if(a&&r){delete t.enteredCbs[i];for(var o=0;o<r.length;o++)a._isBeingDestroyed||r[o](a)}}}}var A={name:"RouterView",functional:!0,props:{name:{type:String,default:"default"}},render:function(n,e){var t=e.props,i=e.children,r=e.parent,o=e.data;o.routerView=!0;for(var s=r.$createElement,l=t.name,c=r.$route,g=r._routerViewCache||(r._routerViewCache={}),d=0,p=!1;r&&r._routerRoot!==r;){var u=r.$vnode?r.$vnode.data:{};u.routerView&&d++,u.keepAlive&&r._directInactive&&r._inactive&&(p=!0),r=r.$parent}if(o.routerViewDepth=d,p){var m=g[l],h=m&&m.component;return h?(m.configProps&&w(h,o,m.route,m.configProps),s(h,o,i)):s()}var f=c.matched[d],b=f&&f.components[l];if(!f||!b)return g[l]=null,s();g[l]={component:b},o.registerRouteInstance=function(n,e){var t=f.instances[l];(e&&t!==n||!e&&t===n)&&(f.instances[l]=e)},(o.hook||(o.hook={})).prepatch=function(n,e){f.instances[l]=e.componentInstance},o.hook.init=function(n){n.data.keepAlive&&n.componentInstance&&n.componentInstance!==f.instances[l]&&(f.instances[l]=n.componentInstance),C(c)};var v=f.props&&f.props[l];return v&&(a(g[l],{route:c,configProps:v}),w(b,o,c,v)),s(b,o,i)}};function w(n,e,t,i){var r=e.props=function(n,e){switch(typeof e){case"undefined":return;case"object":return e;case"function":return e(n);case"boolean":return e?n.params:void 0;default:0}}(t,i);if(r){r=e.props=a({},r);var o=e.attrs=e.attrs||{};for(var s in r)n.props&&s in n.props||(o[s]=r[s],delete r[s])}}function k(n,e,t){var i=n.charAt(0);if("/"===i)return n;if("?"===i||"#"===i)return e+n;var a=e.split("/");t&&a[a.length-1]||a.pop();for(var r=n.replace(/^\//,"").split("/"),o=0;o<r.length;o++){var s=r[o];".."===s?a.pop():"."!==s&&a.push(s)}return""!==a[0]&&a.unshift(""),a.join("/")}function T(n){return n.replace(/\/(?:\s*\/)+/g,"/")}var E=Array.isArray||function(n){return"[object Array]"==Object.prototype.toString.call(n)},F=O,S=z,_=function(n,e){return N(z(n,e),e)},P=N,q=L,I=new RegExp(["(\\\\.)","([\\/.])?(?:(?:\\:(\\w+)(?:\\(((?:\\\\.|[^\\\\()])+)\\))?|\\(((?:\\\\.|[^\\\\()])+)\\))([+*?])?|(\\*))"].join("|"),"g");function z(n,e){for(var t,i=[],a=0,r=0,o="",s=e&&e.delimiter||"/";null!=(t=I.exec(n));){var l=t[0],c=t[1],g=t.index;if(o+=n.slice(r,g),r=g+l.length,c)o+=c[1];else{var d=n[r],p=t[2],u=t[3],m=t[4],h=t[5],f=t[6],b=t[7];o&&(i.push(o),o="");var v=null!=p&&null!=d&&d!==p,y="+"===f||"*"===f,x="?"===f||"*"===f,C=t[2]||s,A=m||h;i.push({name:u||a++,prefix:p||"",delimiter:C,optional:x,repeat:y,partial:v,asterisk:!!b,pattern:A?D(A):b?".*":"[^"+B(C)+"]+?"})}}return r<n.length&&(o+=n.substr(r)),o&&i.push(o),i}function j(n){return encodeURI(n).replace(/[\/?#]/g,(function(n){return"%"+n.charCodeAt(0).toString(16).toUpperCase()}))}function N(n,e){for(var t=new Array(n.length),i=0;i<n.length;i++)"object"==typeof n[i]&&(t[i]=new RegExp("^(?:"+n[i].pattern+")$",M(e)));return function(e,i){for(var a="",r=e||{},o=(i||{}).pretty?j:encodeURIComponent,s=0;s<n.length;s++){var l=n[s];if("string"!=typeof l){var c,g=r[l.name];if(null==g){if(l.optional){l.partial&&(a+=l.prefix);continue}throw new TypeError('Expected "'+l.name+'" to be defined')}if(E(g)){if(!l.repeat)throw new TypeError('Expected "'+l.name+'" to not repeat, but received `'+JSON.stringify(g)+"`");if(0===g.length){if(l.optional)continue;throw new TypeError('Expected "'+l.name+'" to not be empty')}for(var d=0;d<g.length;d++){if(c=o(g[d]),!t[s].test(c))throw new TypeError('Expected all "'+l.name+'" to match "'+l.pattern+'", but received `'+JSON.stringify(c)+"`");a+=(0===d?l.prefix:l.delimiter)+c}}else{if(c=l.asterisk?encodeURI(g).replace(/[?#]/g,(function(n){return"%"+n.charCodeAt(0).toString(16).toUpperCase()})):o(g),!t[s].test(c))throw new TypeError('Expected "'+l.name+'" to match "'+l.pattern+'", but received "'+c+'"');a+=l.prefix+c}}else a+=l}return a}}function B(n){return n.replace(/([.+*?=^!:${}()[\]|\/\\])/g,"\\$1")}function D(n){return n.replace(/([=!:$\/()])/g,"\\$1")}function R(n,e){return n.keys=e,n}function M(n){return n&&n.sensitive?"":"i"}function L(n,e,t){E(e)||(t=e||t,e=[]);for(var i=(t=t||{}).strict,a=!1!==t.end,r="",o=0;o<n.length;o++){var s=n[o];if("string"==typeof s)r+=B(s);else{var l=B(s.prefix),c="(?:"+s.pattern+")";e.push(s),s.repeat&&(c+="(?:"+l+c+")*"),r+=c=s.optional?s.partial?l+"("+c+")?":"(?:"+l+"("+c+"))?":l+"("+c+")"}}var g=B(t.delimiter||"/"),d=r.slice(-g.length)===g;return i||(r=(d?r.slice(0,-g.length):r)+"(?:"+g+"(?=$))?"),r+=a?"$":i&&d?"":"(?="+g+"|$)",R(new RegExp("^"+r,M(t)),e)}function O(n,e,t){return E(e)||(t=e||t,e=[]),t=t||{},n instanceof RegExp?function(n,e){var t=n.source.match(/\((?!\?)/g);if(t)for(var i=0;i<t.length;i++)e.push({name:i,prefix:null,delimiter:null,optional:!1,repeat:!1,partial:!1,asterisk:!1,pattern:null});return R(n,e)}(n,e):E(n)?function(n,e,t){for(var i=[],a=0;a<n.length;a++)i.push(O(n[a],e,t).source);return R(new RegExp("(?:"+i.join("|")+")",M(t)),e)}(n,e,t):function(n,e,t){return L(z(n,t),e,t)}(n,e,t)}F.parse=S,F.compile=_,F.tokensToFunction=P,F.tokensToRegExp=q;var G=Object.create(null);function H(n,e,t){e=e||{};try{var i=G[n]||(G[n]=F.compile(n));return"string"==typeof e.pathMatch&&(e[0]=e.pathMatch),i(e,{pretty:!0})}catch(n){return""}finally{delete e[0]}}function U(n,e,t,i){var r="string"==typeof n?{path:n}:n;if(r._normalized)return r;if(r.name){var o=(r=a({},n)).params;return o&&"object"==typeof o&&(r.params=a({},o)),r}if(!r.path&&r.params&&e){(r=a({},r))._normalized=!0;var s=a(a({},e.params),r.params);if(e.name)r.name=e.name,r.params=s;else if(e.matched.length){var l=e.matched[e.matched.length-1].path;r.path=H(l,s,e.path)}else 0;return r}var c=function(n){var e="",t="",i=n.indexOf("#");i>=0&&(e=n.slice(i),n=n.slice(0,i));var a=n.indexOf("?");return a>=0&&(t=n.slice(a+1),n=n.slice(0,a)),{path:n,query:t,hash:e}}(r.path||""),p=e&&e.path||"/",u=c.path?k(c.path,p,t||r.append):p,m=function(n,e,t){void 0===e&&(e={});var i,a=t||d;try{i=a(n||"")}catch(n){i={}}for(var r in e){var o=e[r];i[r]=Array.isArray(o)?o.map(g):g(o)}return i}(c.query,r.query,i&&i.options.parseQuery),h=r.hash||c.hash;return h&&"#"!==h.charAt(0)&&(h="#"+h),{_normalized:!0,path:u,query:m,hash:h}}var V,Z=function(){},W={name:"RouterLink",props:{to:{type:[String,Object],required:!0},tag:{type:String,default:"a"},custom:Boolean,exact:Boolean,exactPath:Boolean,append:Boolean,replace:Boolean,activeClass:String,exactActiveClass:String,ariaCurrentValue:{type:String,default:"page"},event:{type:[String,Array],default:"click"}},render:function(n){var e=this,t=this.$router,i=this.$route,r=t.resolve(this.to,i,this.append),o=r.location,s=r.route,l=r.href,c={},g=t.options.linkActiveClass,d=t.options.linkExactActiveClass,p=null==g?"router-link-active":g,h=null==d?"router-link-exact-active":d,f=null==this.activeClass?p:this.activeClass,b=null==this.exactActiveClass?h:this.exactActiveClass,v=s.redirectedFrom?m(null,U(s.redirectedFrom),null,t):s;c[b]=y(i,v,this.exactPath),c[f]=this.exact||this.exactPath?c[b]:function(n,e){return 0===n.path.replace(u,"/").indexOf(e.path.replace(u,"/"))&&(!e.hash||n.hash===e.hash)&&function(n,e){for(var t in e)if(!(t in n))return!1;return!0}(n.query,e.query)}(i,v);var x=c[b]?this.ariaCurrentValue:null,C=function(n){X(n)&&(e.replace?t.replace(o,Z):t.push(o,Z))},A={click:X};Array.isArray(this.event)?this.event.forEach((function(n){A[n]=C})):A[this.event]=C;var w={class:c},k=!this.$scopedSlots.$hasNormal&&this.$scopedSlots.default&&this.$scopedSlots.default({href:l,route:s,navigate:C,isActive:c[f],isExactActive:c[b]});if(k){if(1===k.length)return k[0];if(k.length>1||!k.length)return 0===k.length?n():n("span",{},k)}if("a"===this.tag)w.on=A,w.attrs={href:l,"aria-current":x};else{var T=function n(e){var t;if(e)for(var i=0;i<e.length;i++){if("a"===(t=e[i]).tag)return t;if(t.children&&(t=n(t.children)))return t}}(this.$slots.default);if(T){T.isStatic=!1;var E=T.data=a({},T.data);for(var F in E.on=E.on||{},E.on){var S=E.on[F];F in A&&(E.on[F]=Array.isArray(S)?S:[S])}for(var _ in A)_ in E.on?E.on[_].push(A[_]):E.on[_]=C;var P=T.data.attrs=a({},T.data.attrs);P.href=l,P["aria-current"]=x}else w.on=A}return n(this.tag,w,this.$slots.default)}};function X(n){if(!(n.metaKey||n.altKey||n.ctrlKey||n.shiftKey||n.defaultPrevented||void 0!==n.button&&0!==n.button)){if(n.currentTarget&&n.currentTarget.getAttribute){var e=n.currentTarget.getAttribute("target");if(/\b_blank\b/i.test(e))return}return n.preventDefault&&n.preventDefault(),!0}}var $="undefined"!=typeof window;function J(n,e,t,i,a){var r=e||[],o=t||Object.create(null),s=i||Object.create(null);n.forEach((function(n){!function n(e,t,i,a,r,o){var s=a.path,l=a.name;0;var c=a.pathToRegexpOptions||{},g=function(n,e,t){t||(n=n.replace(/\/$/,""));if("/"===n[0])return n;if(null==e)return n;return T(e.path+"/"+n)}(s,r,c.strict);"boolean"==typeof a.caseSensitive&&(c.sensitive=a.caseSensitive);var d={path:g,regex:K(g,c),components:a.components||{default:a.component},alias:a.alias?"string"==typeof a.alias?[a.alias]:a.alias:[],instances:{},enteredCbs:{},name:l,parent:r,matchAs:o,redirect:a.redirect,beforeEnter:a.beforeEnter,meta:a.meta||{},props:null==a.props?{}:a.components?a.props:{default:a.props}};a.children&&a.children.forEach((function(a){var r=o?T(o+"/"+a.path):void 0;n(e,t,i,a,d,r)}));t[d.path]||(e.push(d.path),t[d.path]=d);if(void 0!==a.alias)for(var p=Array.isArray(a.alias)?a.alias:[a.alias],u=0;u<p.length;++u){0;var m={path:p[u],children:a.children};n(e,t,i,m,r,d.path||"/")}l&&(i[l]||(i[l]=d))}(r,o,s,n,a)}));for(var l=0,c=r.length;l<c;l++)"*"===r[l]&&(r.push(r.splice(l,1)[0]),c--,l--);return{pathList:r,pathMap:o,nameMap:s}}function K(n,e){return F(n,[],e)}function Y(n,e){var t=J(n),i=t.pathList,a=t.pathMap,r=t.nameMap;function o(n,t,o){var s=U(n,t,!1,e),c=s.name;if(c){var g=r[c];if(!g)return l(null,s);var d=g.regex.keys.filter((function(n){return!n.optional})).map((function(n){return n.name}));if("object"!=typeof s.params&&(s.params={}),t&&"object"==typeof t.params)for(var p in t.params)!(p in s.params)&&d.indexOf(p)>-1&&(s.params[p]=t.params[p]);return s.path=H(g.path,s.params),l(g,s,o)}if(s.path){s.params={};for(var u=0;u<i.length;u++){var m=i[u],h=a[m];if(Q(h.regex,s.path,s.params))return l(h,s,o)}}return l(null,s)}function s(n,t){var i=n.redirect,a="function"==typeof i?i(m(n,t,null,e)):i;if("string"==typeof a&&(a={path:a}),!a||"object"!=typeof a)return l(null,t);var s=a,c=s.name,g=s.path,d=t.query,p=t.hash,u=t.params;if(d=s.hasOwnProperty("query")?s.query:d,p=s.hasOwnProperty("hash")?s.hash:p,u=s.hasOwnProperty("params")?s.params:u,c){r[c];return o({_normalized:!0,name:c,query:d,hash:p,params:u},void 0,t)}if(g){var h=function(n,e){return k(n,e.parent?e.parent.path:"/",!0)}(g,n);return o({_normalized:!0,path:H(h,u),query:d,hash:p},void 0,t)}return l(null,t)}function l(n,t,i){return n&&n.redirect?s(n,i||t):n&&n.matchAs?function(n,e,t){var i=o({_normalized:!0,path:H(t,e.params)});if(i){var a=i.matched,r=a[a.length-1];return e.params=i.params,l(r,e)}return l(null,e)}(0,t,n.matchAs):m(n,t,i,e)}return{match:o,addRoute:function(n,e){var t="object"!=typeof n?r[n]:void 0;J([e||n],i,a,r,t),t&&t.alias.length&&J(t.alias.map((function(n){return{path:n,children:[e]}})),i,a,r,t)},getRoutes:function(){return i.map((function(n){return a[n]}))},addRoutes:function(n){J(n,i,a,r)}}}function Q(n,e,t){var i=e.match(n);if(!i)return!1;if(!t)return!0;for(var a=1,r=i.length;a<r;++a){var o=n.keys[a-1];o&&(t[o.name||"pathMatch"]="string"==typeof i[a]?c(i[a]):i[a])}return!0}var nn=$&&window.performance&&window.performance.now?window.performance:Date;function en(){return nn.now().toFixed(3)}var tn=en();function an(){return tn}function rn(n){return tn=n}var on=Object.create(null);function sn(){"scrollRestoration"in window.history&&(window.history.scrollRestoration="manual");var n=window.location.protocol+"//"+window.location.host,e=window.location.href.replace(n,""),t=a({},window.history.state);return t.key=an(),window.history.replaceState(t,"",e),window.addEventListener("popstate",gn),function(){window.removeEventListener("popstate",gn)}}function ln(n,e,t,i){if(n.app){var a=n.options.scrollBehavior;a&&n.app.$nextTick((function(){var r=function(){var n=an();if(n)return on[n]}(),o=a.call(n,e,t,i?r:null);o&&("function"==typeof o.then?o.then((function(n){hn(n,r)})).catch((function(n){0})):hn(o,r))}))}}function cn(){var n=an();n&&(on[n]={x:window.pageXOffset,y:window.pageYOffset})}function gn(n){cn(),n.state&&n.state.key&&rn(n.state.key)}function dn(n){return un(n.x)||un(n.y)}function pn(n){return{x:un(n.x)?n.x:window.pageXOffset,y:un(n.y)?n.y:window.pageYOffset}}function un(n){return"number"==typeof n}var mn=/^#\d/;function hn(n,e){var t,i="object"==typeof n;if(i&&"string"==typeof n.selector){var a=mn.test(n.selector)?document.getElementById(n.selector.slice(1)):document.querySelector(n.selector);if(a){var r=n.offset&&"object"==typeof n.offset?n.offset:{};e=function(n,e){var t=document.documentElement.getBoundingClientRect(),i=n.getBoundingClientRect();return{x:i.left-t.left-e.x,y:i.top-t.top-e.y}}(a,r={x:un((t=r).x)?t.x:0,y:un(t.y)?t.y:0})}else dn(n)&&(e=pn(n))}else i&&dn(n)&&(e=pn(n));e&&("scrollBehavior"in document.documentElement.style?window.scrollTo({left:e.x,top:e.y,behavior:n.behavior}):window.scrollTo(e.x,e.y))}var fn,bn=$&&((-1===(fn=window.navigator.userAgent).indexOf("Android 2.")&&-1===fn.indexOf("Android 4.0")||-1===fn.indexOf("Mobile Safari")||-1!==fn.indexOf("Chrome")||-1!==fn.indexOf("Windows Phone"))&&window.history&&"function"==typeof window.history.pushState);function vn(n,e){cn();var t=window.history;try{if(e){var i=a({},t.state);i.key=an(),t.replaceState(i,"",n)}else t.pushState({key:rn(en())},"",n)}catch(t){window.location[e?"replace":"assign"](n)}}function yn(n){vn(n,!0)}var xn={redirected:2,aborted:4,cancelled:8,duplicated:16};function Cn(n,e){return wn(n,e,xn.redirected,'Redirected when going from "'+n.fullPath+'" to "'+function(n){if("string"==typeof n)return n;if("path"in n)return n.path;var e={};return kn.forEach((function(t){t in n&&(e[t]=n[t])})),JSON.stringify(e,null,2)}(e)+'" via a navigation guard.')}function An(n,e){return wn(n,e,xn.cancelled,'Navigation cancelled from "'+n.fullPath+'" to "'+e.fullPath+'" with a new navigation.')}function wn(n,e,t,i){var a=new Error(i);return a._isRouter=!0,a.from=n,a.to=e,a.type=t,a}var kn=["params","query","hash"];function Tn(n){return Object.prototype.toString.call(n).indexOf("Error")>-1}function En(n,e){return Tn(n)&&n._isRouter&&(null==e||n.type===e)}function Fn(n,e,t){var i=function(a){a>=n.length?t():n[a]?e(n[a],(function(){i(a+1)})):i(a+1)};i(0)}function Sn(n){return function(e,t,i){var a=!1,r=0,o=null;_n(n,(function(n,e,t,s){if("function"==typeof n&&void 0===n.cid){a=!0,r++;var l,c=In((function(e){var a;((a=e).__esModule||qn&&"Module"===a[Symbol.toStringTag])&&(e=e.default),n.resolved="function"==typeof e?e:V.extend(e),t.components[s]=e,--r<=0&&i()})),g=In((function(n){var e="Failed to resolve async component "+s+": "+n;o||(o=Tn(n)?n:new Error(e),i(o))}));try{l=n(c,g)}catch(n){g(n)}if(l)if("function"==typeof l.then)l.then(c,g);else{var d=l.component;d&&"function"==typeof d.then&&d.then(c,g)}}})),a||i()}}function _n(n,e){return Pn(n.map((function(n){return Object.keys(n.components).map((function(t){return e(n.components[t],n.instances[t],n,t)}))})))}function Pn(n){return Array.prototype.concat.apply([],n)}var qn="function"==typeof Symbol&&"symbol"==typeof Symbol.toStringTag;function In(n){var e=!1;return function(){for(var t=[],i=arguments.length;i--;)t[i]=arguments[i];if(!e)return e=!0,n.apply(this,t)}}var zn=function(n,e){this.router=n,this.base=function(n){if(!n)if($){var e=document.querySelector("base");n=(n=e&&e.getAttribute("href")||"/").replace(/^https?:\/\/[^\/]+/,"")}else n="/";"/"!==n.charAt(0)&&(n="/"+n);return n.replace(/\/$/,"")}(e),this.current=f,this.pending=null,this.ready=!1,this.readyCbs=[],this.readyErrorCbs=[],this.errorCbs=[],this.listeners=[]};function jn(n,e,t,i){var a=_n(n,(function(n,i,a,r){var o=function(n,e){"function"!=typeof n&&(n=V.extend(n));return n.options[e]}(n,e);if(o)return Array.isArray(o)?o.map((function(n){return t(n,i,a,r)})):t(o,i,a,r)}));return Pn(i?a.reverse():a)}function Nn(n,e){if(e)return function(){return n.apply(e,arguments)}}zn.prototype.listen=function(n){this.cb=n},zn.prototype.onReady=function(n,e){this.ready?n():(this.readyCbs.push(n),e&&this.readyErrorCbs.push(e))},zn.prototype.onError=function(n){this.errorCbs.push(n)},zn.prototype.transitionTo=function(n,e,t){var i,a=this;try{i=this.router.match(n,this.current)}catch(n){throw this.errorCbs.forEach((function(e){e(n)})),n}var r=this.current;this.confirmTransition(i,(function(){a.updateRoute(i),e&&e(i),a.ensureURL(),a.router.afterHooks.forEach((function(n){n&&n(i,r)})),a.ready||(a.ready=!0,a.readyCbs.forEach((function(n){n(i)})))}),(function(n){t&&t(n),n&&!a.ready&&(En(n,xn.redirected)&&r===f||(a.ready=!0,a.readyErrorCbs.forEach((function(e){e(n)}))))}))},zn.prototype.confirmTransition=function(n,e,t){var i=this,a=this.current;this.pending=n;var r,o,s=function(n){!En(n)&&Tn(n)&&(i.errorCbs.length?i.errorCbs.forEach((function(e){e(n)})):console.error(n)),t&&t(n)},l=n.matched.length-1,c=a.matched.length-1;if(y(n,a)&&l===c&&n.matched[l]===a.matched[c])return this.ensureURL(),n.hash&&ln(this.router,a,n,!1),s(((o=wn(r=a,n,xn.duplicated,'Avoided redundant navigation to current location: "'+r.fullPath+'".')).name="NavigationDuplicated",o));var g=function(n,e){var t,i=Math.max(n.length,e.length);for(t=0;t<i&&n[t]===e[t];t++);return{updated:e.slice(0,t),activated:e.slice(t),deactivated:n.slice(t)}}(this.current.matched,n.matched),d=g.updated,p=g.deactivated,u=g.activated,m=[].concat(function(n){return jn(n,"beforeRouteLeave",Nn,!0)}(p),this.router.beforeHooks,function(n){return jn(n,"beforeRouteUpdate",Nn)}(d),u.map((function(n){return n.beforeEnter})),Sn(u)),h=function(e,t){if(i.pending!==n)return s(An(a,n));try{e(n,a,(function(e){!1===e?(i.ensureURL(!0),s(function(n,e){return wn(n,e,xn.aborted,'Navigation aborted from "'+n.fullPath+'" to "'+e.fullPath+'" via a navigation guard.')}(a,n))):Tn(e)?(i.ensureURL(!0),s(e)):"string"==typeof e||"object"==typeof e&&("string"==typeof e.path||"string"==typeof e.name)?(s(Cn(a,n)),"object"==typeof e&&e.replace?i.replace(e):i.push(e)):t(e)}))}catch(n){s(n)}};Fn(m,h,(function(){Fn(function(n){return jn(n,"beforeRouteEnter",(function(n,e,t,i){return function(n,e,t){return function(i,a,r){return n(i,a,(function(n){"function"==typeof n&&(e.enteredCbs[t]||(e.enteredCbs[t]=[]),e.enteredCbs[t].push(n)),r(n)}))}}(n,t,i)}))}(u).concat(i.router.resolveHooks),h,(function(){if(i.pending!==n)return s(An(a,n));i.pending=null,e(n),i.router.app&&i.router.app.$nextTick((function(){C(n)}))}))}))},zn.prototype.updateRoute=function(n){this.current=n,this.cb&&this.cb(n)},zn.prototype.setupListeners=function(){},zn.prototype.teardown=function(){this.listeners.forEach((function(n){n()})),this.listeners=[],this.current=f,this.pending=null};var Bn=function(n){function e(e,t){n.call(this,e,t),this._startLocation=Dn(this.base)}return n&&(e.__proto__=n),e.prototype=Object.create(n&&n.prototype),e.prototype.constructor=e,e.prototype.setupListeners=function(){var n=this;if(!(this.listeners.length>0)){var e=this.router,t=e.options.scrollBehavior,i=bn&&t;i&&this.listeners.push(sn());var a=function(){var t=n.current,a=Dn(n.base);n.current===f&&a===n._startLocation||n.transitionTo(a,(function(n){i&&ln(e,n,t,!0)}))};window.addEventListener("popstate",a),this.listeners.push((function(){window.removeEventListener("popstate",a)}))}},e.prototype.go=function(n){window.history.go(n)},e.prototype.push=function(n,e,t){var i=this,a=this.current;this.transitionTo(n,(function(n){vn(T(i.base+n.fullPath)),ln(i.router,n,a,!1),e&&e(n)}),t)},e.prototype.replace=function(n,e,t){var i=this,a=this.current;this.transitionTo(n,(function(n){yn(T(i.base+n.fullPath)),ln(i.router,n,a,!1),e&&e(n)}),t)},e.prototype.ensureURL=function(n){if(Dn(this.base)!==this.current.fullPath){var e=T(this.base+this.current.fullPath);n?vn(e):yn(e)}},e.prototype.getCurrentLocation=function(){return Dn(this.base)},e}(zn);function Dn(n){var e=window.location.pathname,t=e.toLowerCase(),i=n.toLowerCase();return!n||t!==i&&0!==t.indexOf(T(i+"/"))||(e=e.slice(n.length)),(e||"/")+window.location.search+window.location.hash}var Rn=function(n){function e(e,t,i){n.call(this,e,t),i&&function(n){var e=Dn(n);if(!/^\/#/.test(e))return window.location.replace(T(n+"/#"+e)),!0}(this.base)||Mn()}return n&&(e.__proto__=n),e.prototype=Object.create(n&&n.prototype),e.prototype.constructor=e,e.prototype.setupListeners=function(){var n=this;if(!(this.listeners.length>0)){var e=this.router.options.scrollBehavior,t=bn&&e;t&&this.listeners.push(sn());var i=function(){var e=n.current;Mn()&&n.transitionTo(Ln(),(function(i){t&&ln(n.router,i,e,!0),bn||Hn(i.fullPath)}))},a=bn?"popstate":"hashchange";window.addEventListener(a,i),this.listeners.push((function(){window.removeEventListener(a,i)}))}},e.prototype.push=function(n,e,t){var i=this,a=this.current;this.transitionTo(n,(function(n){Gn(n.fullPath),ln(i.router,n,a,!1),e&&e(n)}),t)},e.prototype.replace=function(n,e,t){var i=this,a=this.current;this.transitionTo(n,(function(n){Hn(n.fullPath),ln(i.router,n,a,!1),e&&e(n)}),t)},e.prototype.go=function(n){window.history.go(n)},e.prototype.ensureURL=function(n){var e=this.current.fullPath;Ln()!==e&&(n?Gn(e):Hn(e))},e.prototype.getCurrentLocation=function(){return Ln()},e}(zn);function Mn(){var n=Ln();return"/"===n.charAt(0)||(Hn("/"+n),!1)}function Ln(){var n=window.location.href,e=n.indexOf("#");return e<0?"":n=n.slice(e+1)}function On(n){var e=window.location.href,t=e.indexOf("#");return(t>=0?e.slice(0,t):e)+"#"+n}function Gn(n){bn?vn(On(n)):window.location.hash=n}function Hn(n){bn?yn(On(n)):window.location.replace(On(n))}var Un=function(n){function e(e,t){n.call(this,e,t),this.stack=[],this.index=-1}return n&&(e.__proto__=n),e.prototype=Object.create(n&&n.prototype),e.prototype.constructor=e,e.prototype.push=function(n,e,t){var i=this;this.transitionTo(n,(function(n){i.stack=i.stack.slice(0,i.index+1).concat(n),i.index++,e&&e(n)}),t)},e.prototype.replace=function(n,e,t){var i=this;this.transitionTo(n,(function(n){i.stack=i.stack.slice(0,i.index).concat(n),e&&e(n)}),t)},e.prototype.go=function(n){var e=this,t=this.index+n;if(!(t<0||t>=this.stack.length)){var i=this.stack[t];this.confirmTransition(i,(function(){var n=e.current;e.index=t,e.updateRoute(i),e.router.afterHooks.forEach((function(e){e&&e(i,n)}))}),(function(n){En(n,xn.duplicated)&&(e.index=t)}))}},e.prototype.getCurrentLocation=function(){var n=this.stack[this.stack.length-1];return n?n.fullPath:"/"},e.prototype.ensureURL=function(){},e}(zn),Vn=function(n){void 0===n&&(n={}),this.app=null,this.apps=[],this.options=n,this.beforeHooks=[],this.resolveHooks=[],this.afterHooks=[],this.matcher=Y(n.routes||[],this);var e=n.mode||"hash";switch(this.fallback="history"===e&&!bn&&!1!==n.fallback,this.fallback&&(e="hash"),$||(e="abstract"),this.mode=e,e){case"history":this.history=new Bn(this,n.base);break;case"hash":this.history=new Rn(this,n.base,this.fallback);break;case"abstract":this.history=new Un(this,n.base);break;default:0}},Zn={currentRoute:{configurable:!0}};Vn.prototype.match=function(n,e,t){return this.matcher.match(n,e,t)},Zn.currentRoute.get=function(){return this.history&&this.history.current},Vn.prototype.init=function(n){var e=this;if(this.apps.push(n),n.$once("hook:destroyed",(function(){var t=e.apps.indexOf(n);t>-1&&e.apps.splice(t,1),e.app===n&&(e.app=e.apps[0]||null),e.app||e.history.teardown()})),!this.app){this.app=n;var t=this.history;if(t instanceof Bn||t instanceof Rn){var i=function(n){t.setupListeners(),function(n){var i=t.current,a=e.options.scrollBehavior;bn&&a&&"fullPath"in n&&ln(e,n,i,!1)}(n)};t.transitionTo(t.getCurrentLocation(),i,i)}t.listen((function(n){e.apps.forEach((function(e){e._route=n}))}))}},Vn.prototype.beforeEach=function(n){return Xn(this.beforeHooks,n)},Vn.prototype.beforeResolve=function(n){return Xn(this.resolveHooks,n)},Vn.prototype.afterEach=function(n){return Xn(this.afterHooks,n)},Vn.prototype.onReady=function(n,e){this.history.onReady(n,e)},Vn.prototype.onError=function(n){this.history.onError(n)},Vn.prototype.push=function(n,e,t){var i=this;if(!e&&!t&&"undefined"!=typeof Promise)return new Promise((function(e,t){i.history.push(n,e,t)}));this.history.push(n,e,t)},Vn.prototype.replace=function(n,e,t){var i=this;if(!e&&!t&&"undefined"!=typeof Promise)return new Promise((function(e,t){i.history.replace(n,e,t)}));this.history.replace(n,e,t)},Vn.prototype.go=function(n){this.history.go(n)},Vn.prototype.back=function(){this.go(-1)},Vn.prototype.forward=function(){this.go(1)},Vn.prototype.getMatchedComponents=function(n){var e=n?n.matched?n:this.resolve(n).route:this.currentRoute;return e?[].concat.apply([],e.matched.map((function(n){return Object.keys(n.components).map((function(e){return n.components[e]}))}))):[]},Vn.prototype.resolve=function(n,e,t){var i=U(n,e=e||this.history.current,t,this),a=this.match(i,e),r=a.redirectedFrom||a.fullPath;return{location:i,route:a,href:function(n,e,t){var i="hash"===t?"#"+e:e;return n?T(n+"/"+i):i}(this.history.base,r,this.mode),normalizedTo:i,resolved:a}},Vn.prototype.getRoutes=function(){return this.matcher.getRoutes()},Vn.prototype.addRoute=function(n,e){this.matcher.addRoute(n,e),this.history.current!==f&&this.history.transitionTo(this.history.getCurrentLocation())},Vn.prototype.addRoutes=function(n){this.matcher.addRoutes(n),this.history.current!==f&&this.history.transitionTo(this.history.getCurrentLocation())},Object.defineProperties(Vn.prototype,Zn);var Wn=Vn;function Xn(n,e){return n.push(e),function(){var t=n.indexOf(e);t>-1&&n.splice(t,1)}}Vn.install=function n(e){if(!n.installed||V!==e){n.installed=!0,V=e;var t=function(n){return void 0!==n},i=function(n,e){var i=n.$options._parentVnode;t(i)&&t(i=i.data)&&t(i=i.registerRouteInstance)&&i(n,e)};e.mixin({beforeCreate:function(){t(this.$options.router)?(this._routerRoot=this,this._router=this.$options.router,this._router.init(this),e.util.defineReactive(this,"_route",this._router.history.current)):this._routerRoot=this.$parent&&this.$parent._routerRoot||this,i(this,this)},destroyed:function(){i(this)}}),Object.defineProperty(e.prototype,"$router",{get:function(){return this._routerRoot._router}}),Object.defineProperty(e.prototype,"$route",{get:function(){return this._routerRoot._route}}),e.component("RouterView",A),e.component("RouterLink",W);var a=e.config.optionMergeStrategies;a.beforeRouteEnter=a.beforeRouteLeave=a.beforeRouteUpdate=a.created}},Vn.version="3.6.5",Vn.isNavigationFailure=En,Vn.NavigationFailureType=xn,Vn.START_LOCATION=f,$&&window.Vue&&window.Vue.use(Vn);t(110);t(12),t(48);var $n={NotFound:()=>Promise.all([t.e(0),t.e(13)]).then(t.bind(null,358)),Layout:()=>Promise.all([t.e(0),t.e(2)]).then(t.bind(null,356))},Jn={"v-718147da":()=>t.e(18).then(t.bind(null,374)),"v-8bb86ff6":()=>t.e(21).then(t.bind(null,375)),"v-fb61cf0a":()=>t.e(19).then(t.bind(null,376)),"v-4f1678f4":()=>t.e(23).then(t.bind(null,377)),"v-6b3ea08c":()=>t.e(24).then(t.bind(null,378)),"v-033b6f48":()=>t.e(25).then(t.bind(null,379)),"v-634587af":()=>t.e(26).then(t.bind(null,380)),"v-a6be02e2":()=>t.e(29).then(t.bind(null,381)),"v-46d8e2d8":()=>t.e(20).then(t.bind(null,382)),"v-3cb81039":()=>t.e(27).then(t.bind(null,359)),"v-c6085372":()=>t.e(31).then(t.bind(null,383)),"v-18d4ff6e":()=>t.e(28).then(t.bind(null,384)),"v-6c8f1043":()=>t.e(32).then(t.bind(null,385)),"v-584566ea":()=>t.e(30).then(t.bind(null,386)),"v-5c47aead":()=>t.e(22).then(t.bind(null,387)),"v-5dc118b8":()=>t.e(39).then(t.bind(null,388)),"v-1172eaec":()=>t.e(34).then(t.bind(null,389)),"v-1a2241f6":()=>t.e(33).then(t.bind(null,390)),"v-5ec3cdd2":()=>t.e(36).then(t.bind(null,391)),"v-350d1abc":()=>t.e(40).then(t.bind(null,392)),"v-1a35b060":()=>t.e(41).then(t.bind(null,393)),"v-4e30e452":()=>t.e(35).then(t.bind(null,394)),"v-93263dc4":()=>t.e(37).then(t.bind(null,395)),"v-0fb1f416":()=>t.e(43).then(t.bind(null,396)),"v-f77517aa":()=>t.e(42).then(t.bind(null,397)),"v-5ec20e5c":()=>t.e(38).then(t.bind(null,398)),"v-4312c42e":()=>t.e(44).then(t.bind(null,399)),"v-e52b92e2":()=>t.e(46).then(t.bind(null,400)),"v-31e3c08b":()=>t.e(45).then(t.bind(null,401)),"v-294d13ab":()=>t.e(50).then(t.bind(null,402)),"v-3d52f327":()=>t.e(49).then(t.bind(null,403)),"v-5bcc7c9e":()=>t.e(51).then(t.bind(null,404)),"v-490122ed":()=>t.e(47).then(t.bind(null,360)),"v-0e096556":()=>t.e(52).then(t.bind(null,405)),"v-79611a61":()=>t.e(48).then(t.bind(null,406)),"v-14a9ddeb":()=>t.e(53).then(t.bind(null,407)),"v-98790e58":()=>t.e(54).then(t.bind(null,408)),"v-39f4cdd0":()=>t.e(56).then(t.bind(null,409)),"v-a89b5f26":()=>t.e(55).then(t.bind(null,361)),"v-e1b6b19c":()=>t.e(58).then(t.bind(null,410)),"v-6c95ab70":()=>t.e(60).then(t.bind(null,411)),"v-c7d2b644":()=>t.e(57).then(t.bind(null,412)),"v-7258202a":()=>t.e(59).then(t.bind(null,413)),"v-21cb322c":()=>t.e(63).then(t.bind(null,414)),"v-76ee6b70":()=>t.e(62).then(t.bind(null,415)),"v-15bfff01":()=>t.e(64).then(t.bind(null,416)),"v-7180fb28":()=>t.e(61).then(t.bind(null,417)),"v-61bb19fa":()=>t.e(65).then(t.bind(null,418)),"v-3e8128e8":()=>t.e(67).then(t.bind(null,419)),"v-5ac83554":()=>t.e(69).then(t.bind(null,420)),"v-21fa6f80":()=>t.e(66).then(t.bind(null,421)),"v-393821ce":()=>t.e(70).then(t.bind(null,422)),"v-4fbe66c8":()=>t.e(71).then(t.bind(null,423)),"v-98642aa8":()=>t.e(72).then(t.bind(null,424)),"v-64a80ee8":()=>t.e(68).then(t.bind(null,425)),"v-485075cc":()=>t.e(75).then(t.bind(null,426)),"v-79262273":()=>t.e(73).then(t.bind(null,427)),"v-40e67c36":()=>t.e(74).then(t.bind(null,428)),"v-5c36d2c2":()=>t.e(77).then(t.bind(null,429)),"v-44435d26":()=>t.e(76).then(t.bind(null,430)),"v-afb7dc3c":()=>t.e(81).then(t.bind(null,431)),"v-e7256176":()=>t.e(79).then(t.bind(null,432)),"v-e6a0d06a":()=>t.e(78).then(t.bind(null,433)),"v-d1ce449a":()=>t.e(80).then(t.bind(null,434)),"v-128dac97":()=>t.e(82).then(t.bind(null,435)),"v-722b1d75":()=>t.e(83).then(t.bind(null,436)),"v-9ab7ce14":()=>t.e(86).then(t.bind(null,437)),"v-f79bb6f6":()=>t.e(84).then(t.bind(null,438)),"v-444950d4":()=>t.e(85).then(t.bind(null,439)),"v-09d0538e":()=>t.e(87).then(t.bind(null,440)),"v-82511ca6":()=>t.e(88).then(t.bind(null,441)),"v-4ca30be4":()=>t.e(91).then(t.bind(null,442)),"v-704e8ccd":()=>t.e(89).then(t.bind(null,443)),"v-2b6b4421":()=>t.e(90).then(t.bind(null,444)),"v-9245513c":()=>t.e(95).then(t.bind(null,445)),"v-1f4da734":()=>t.e(94).then(t.bind(null,446)),"v-3eebebda":()=>t.e(92).then(t.bind(null,447)),"v-13e0acb8":()=>t.e(93).then(t.bind(null,448)),"v-cb41435c":()=>t.e(96).then(t.bind(null,449)),"v-53c700f9":()=>t.e(98).then(t.bind(null,450)),"v-f88540b8":()=>t.e(99).then(t.bind(null,451)),"v-7b5d7342":()=>t.e(97).then(t.bind(null,452)),"v-52142b16":()=>t.e(101).then(t.bind(null,453)),"v-95c1c078":()=>t.e(100).then(t.bind(null,454)),"v-6de6612b":()=>t.e(102).then(t.bind(null,455)),"v-51b78e88":()=>t.e(104).then(t.bind(null,456)),"v-6f2dd4c9":()=>t.e(106).then(t.bind(null,457)),"v-2cc66357":()=>t.e(103).then(t.bind(null,458)),"v-c424057a":()=>t.e(107).then(t.bind(null,459)),"v-6086e0c3":()=>t.e(105).then(t.bind(null,460)),"v-3c7ee81a":()=>t.e(108).then(t.bind(null,461)),"v-72b734c5":()=>t.e(110).then(t.bind(null,462)),"v-6ed06999":()=>t.e(109).then(t.bind(null,463)),"v-2607359c":()=>t.e(111).then(t.bind(null,464)),"v-49747540":()=>t.e(112).then(t.bind(null,465)),"v-35f11cad":()=>t.e(113).then(t.bind(null,466)),"v-26ae0bd4":()=>t.e(115).then(t.bind(null,467)),"v-9dcdff76":()=>t.e(117).then(t.bind(null,468)),"v-195442fc":()=>t.e(114).then(t.bind(null,469)),"v-5d420fe2":()=>t.e(119).then(t.bind(null,470)),"v-778d5c48":()=>t.e(116).then(t.bind(null,471)),"v-084ec0e5":()=>t.e(118).then(t.bind(null,472)),"v-620e68fd":()=>t.e(122).then(t.bind(null,473)),"v-48e1665e":()=>t.e(121).then(t.bind(null,474)),"v-65c7b89c":()=>t.e(120).then(t.bind(null,475)),"v-10ed743c":()=>t.e(123).then(t.bind(null,476)),"v-7bb51432":()=>t.e(126).then(t.bind(null,477)),"v-ca793f2a":()=>t.e(124).then(t.bind(null,478)),"v-9366df52":()=>t.e(125).then(t.bind(null,479)),"v-78e6cd76":()=>t.e(128).then(t.bind(null,480)),"v-c46c1bf8":()=>t.e(129).then(t.bind(null,481)),"v-15038c83":()=>t.e(131).then(t.bind(null,482)),"v-c3ba952a":()=>t.e(130).then(t.bind(null,483)),"v-1056f6de":()=>t.e(127).then(t.bind(null,484)),"v-cebb617c":()=>t.e(132).then(t.bind(null,485)),"v-01f55ac7":()=>t.e(134).then(t.bind(null,486)),"v-01625f39":()=>t.e(135).then(t.bind(null,487)),"v-2766f654":()=>t.e(133).then(t.bind(null,488)),"v-3121a636":()=>t.e(136).then(t.bind(null,489)),"v-29874988":()=>t.e(137).then(t.bind(null,490)),"v-24a06b6a":()=>t.e(138).then(t.bind(null,491)),"v-0ef330be":()=>t.e(140).then(t.bind(null,492)),"v-22df1dd0":()=>t.e(139).then(t.bind(null,493)),"v-41ca1859":()=>t.e(141).then(t.bind(null,494)),"v-616419cb":()=>t.e(142).then(t.bind(null,495)),"v-1d158259":()=>t.e(145).then(t.bind(null,496)),"v-e06f21f6":()=>t.e(143).then(t.bind(null,497)),"v-0da46b4e":()=>t.e(144).then(t.bind(null,498)),"v-8449846e":()=>t.e(146).then(t.bind(null,499)),"v-0387c6df":()=>t.e(148).then(t.bind(null,500)),"v-97fad502":()=>t.e(147).then(t.bind(null,501)),"v-bf406dce":()=>t.e(149).then(t.bind(null,502)),"v-434e1f39":()=>t.e(150).then(t.bind(null,503)),"v-d477d30a":()=>t.e(151).then(t.bind(null,504)),"v-2833738e":()=>t.e(153).then(t.bind(null,505)),"v-8d171136":()=>t.e(154).then(t.bind(null,506)),"v-35793acf":()=>t.e(155).then(t.bind(null,507)),"v-66ffee6f":()=>t.e(152).then(t.bind(null,508)),"v-5de954f1":()=>t.e(157).then(t.bind(null,509)),"v-b2b8642e":()=>t.e(156).then(t.bind(null,510)),"v-7c78e838":()=>t.e(158).then(t.bind(null,511)),"v-c279f104":()=>t.e(160).then(t.bind(null,512)),"v-4e38c05c":()=>t.e(159).then(t.bind(null,513)),"v-2a4dc049":()=>t.e(161).then(t.bind(null,514)),"v-2754ef54":()=>t.e(163).then(t.bind(null,515)),"v-c8068dca":()=>t.e(162).then(t.bind(null,516)),"v-1d78ed0e":()=>t.e(164).then(t.bind(null,517)),"v-4e102365":()=>t.e(166).then(t.bind(null,518)),"v-f12adf96":()=>t.e(165).then(t.bind(null,519)),"v-43390cd4":()=>t.e(168).then(t.bind(null,520)),"v-8a435fea":()=>t.e(167).then(t.bind(null,521)),"v-6770800e":()=>t.e(172).then(t.bind(null,522)),"v-bb151b66":()=>t.e(170).then(t.bind(null,523)),"v-636d4942":()=>t.e(169).then(t.bind(null,524)),"v-7384231e":()=>t.e(171).then(t.bind(null,525)),"v-0f0ab9bb":()=>t.e(174).then(t.bind(null,526)),"v-6d8cebcc":()=>t.e(173).then(t.bind(null,527)),"v-1e343bfb":()=>t.e(175).then(t.bind(null,528)),"v-5bf683a0":()=>t.e(177).then(t.bind(null,529)),"v-220be6fa":()=>t.e(176).then(t.bind(null,530)),"v-0c0e50cc":()=>t.e(178).then(t.bind(null,531)),"v-3ee28eee":()=>t.e(181).then(t.bind(null,532)),"v-5e86d73a":()=>t.e(179).then(t.bind(null,533)),"v-1b276464":()=>t.e(180).then(t.bind(null,534)),"v-ba6ecaea":()=>t.e(184).then(t.bind(null,535)),"v-aaf5ea96":()=>t.e(182).then(t.bind(null,536)),"v-371f414b":()=>t.e(186).then(t.bind(null,537)),"v-5c93088f":()=>t.e(187).then(t.bind(null,538)),"v-0ec22c8d":()=>t.e(183).then(t.bind(null,539)),"v-e53f033e":()=>t.e(185).then(t.bind(null,540)),"v-114caebc":()=>t.e(189).then(t.bind(null,541)),"v-5ffa3dac":()=>t.e(188).then(t.bind(null,542)),"v-41e3f643":()=>t.e(193).then(t.bind(null,543)),"v-16897589":()=>t.e(192).then(t.bind(null,544)),"v-623ea45e":()=>t.e(190).then(t.bind(null,545)),"v-6406ed3f":()=>t.e(191).then(t.bind(null,546)),"v-4a93e361":()=>t.e(196).then(t.bind(null,547)),"v-7c712705":()=>t.e(194).then(t.bind(null,548)),"v-0701bf55":()=>t.e(199).then(t.bind(null,549)),"v-7931d701":()=>t.e(195).then(t.bind(null,550)),"v-0b9c81ee":()=>t.e(197).then(t.bind(null,551)),"v-8d2263ae":()=>t.e(201).then(t.bind(null,552)),"v-14a60cb5":()=>t.e(200).then(t.bind(null,553)),"v-e022c6c4":()=>t.e(198).then(t.bind(null,554)),"v-6ea3592e":()=>t.e(204).then(t.bind(null,555)),"v-0d4bbaae":()=>t.e(202).then(t.bind(null,556)),"v-22f1f98e":()=>t.e(203).then(t.bind(null,557)),"v-42ca3e9a":()=>t.e(205).then(t.bind(null,558))};function Kn(n){const e=Object.create(null);return function(t){return e[t]||(e[t]=n(t))}}const Yn=/-(\w)/g,Qn=Kn(n=>n.replace(Yn,(n,e)=>e?e.toUpperCase():"")),ne=/\B([A-Z])/g,ee=Kn(n=>n.replace(ne,"-$1").toLowerCase()),te=Kn(n=>n.charAt(0).toUpperCase()+n.slice(1));function ie(n,e){if(!e)return;if(n(e))return n(e);return e.includes("-")?n(te(Qn(e))):n(te(e))||n(ee(e))}const ae=Object.assign({},$n,Jn),re=n=>ae[n],oe=n=>Jn[n],se=n=>$n[n],le=n=>i.default.component(n);function ce(n){return ie(oe,n)}function ge(n){return ie(se,n)}function de(n){return ie(re,n)}function pe(n){return ie(le,n)}function ue(...n){return Promise.all(n.filter(n=>n).map(async n=>{if(!pe(n)&&de(n)){const e=await de(n)();i.default.component(n,e.default)}}))}function me(n,e){"undefined"!=typeof window&&window.__VUEPRESS__&&(window.__VUEPRESS__[n]=e)}var he=t(95),fe=t.n(he),be=t(96),ve=t.n(be),ye={created(){if(this.siteMeta=this.$site.headTags.filter(([n])=>"meta"===n).map(([n,e])=>e),this.$ssrContext){const e=this.getMergedMetaTags();this.$ssrContext.title=this.$title,this.$ssrContext.lang=this.$lang,this.$ssrContext.pageMeta=(n=e)?n.map(n=>{let e="<meta";return Object.keys(n).forEach(t=>{e+=` ${t}="${ve()(n[t])}"`}),e+">"}).join("\n    "):"",this.$ssrContext.canonicalLink=Ce(this.$canonicalUrl)}var n},mounted(){this.currentMetaTags=[...document.querySelectorAll("meta")],this.updateMeta(),this.updateCanonicalLink()},methods:{updateMeta(){document.title=this.$title,document.documentElement.lang=this.$lang;const n=this.getMergedMetaTags();this.currentMetaTags=Ae(n,this.currentMetaTags)},getMergedMetaTags(){const n=this.$page.frontmatter.meta||[];return fe()([{name:"description",content:this.$description}],n,this.siteMeta,we)},updateCanonicalLink(){xe(),this.$canonicalUrl&&document.head.insertAdjacentHTML("beforeend",Ce(this.$canonicalUrl))}},watch:{$page(){this.updateMeta(),this.updateCanonicalLink()}},beforeDestroy(){Ae(null,this.currentMetaTags),xe()}};function xe(){const n=document.querySelector("link[rel='canonical']");n&&n.remove()}function Ce(n=""){return n?`<link href="${n}" rel="canonical" />`:""}function Ae(n,e){if(e&&[...e].filter(n=>n.parentNode===document.head).forEach(n=>document.head.removeChild(n)),n)return n.map(n=>{const e=document.createElement("meta");return Object.keys(n).forEach(t=>{e.setAttribute(t,n[t])}),document.head.appendChild(e),e})}function we(n){for(const e of["name","property","itemprop"])if(n.hasOwnProperty(e))return n[e]+e;return JSON.stringify(n)}var ke=t(52),Te={mounted(){window.addEventListener("scroll",this.onScroll)},methods:{onScroll:t.n(ke)()((function(){this.setActiveHash()}),300),setActiveHash(){const n=[].slice.call(document.querySelectorAll(".sidebar-link")),e=[].slice.call(document.querySelectorAll(".header-anchor")).filter(e=>n.some(n=>n.hash===e.hash)),t=Math.max(window.pageYOffset,document.documentElement.scrollTop,document.body.scrollTop),i=Math.max(document.documentElement.scrollHeight,document.body.scrollHeight),a=window.innerHeight+t;for(let n=0;n<e.length;n++){const r=e[n],o=e[n+1],s=0===n&&0===t||t>=r.parentElement.offsetTop+10&&(!o||t<o.parentElement.offsetTop-10),l=decodeURIComponent(this.$route.hash);if(s&&l!==decodeURIComponent(r.hash)){const t=r;if(a===i)for(let t=n+1;t<e.length;t++)if(l===decodeURIComponent(e[t].hash))return;return this.$vuepress.$set("disableScrollBehavior",!0),void this.$router.replace(decodeURIComponent(t.hash),()=>{this.$nextTick(()=>{this.$vuepress.$set("disableScrollBehavior",!1)})})}}}},beforeDestroy(){window.removeEventListener("scroll",this.onScroll)}},Ee=t(25),Fe=t.n(Ee),Se={mounted(){Fe.a.configure({showSpinner:!1}),this.$router.beforeEach((n,e,t)=>{n.path===e.path||i.default.component(n.name)||Fe.a.start(),t()}),this.$router.afterEach(()=>{Fe.a.done(),this.isSidebarOpen=!1})}};t(242),t(243);class _e{constructor(){this.containerEl=document.getElementById("message-container"),this.containerEl||(this.containerEl=document.createElement("div"),this.containerEl.id="message-container",document.body.appendChild(this.containerEl))}show({text:n="",duration:e=3e3}){let t=document.createElement("div");t.className="message move-in",t.innerHTML=`\n      <i style="fill: #06a35a;font-size: 14px;display:inline-flex;align-items: center;">\n        <svg style="fill: #06a35a;font-size: 14px;" t="1572421810237" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="2323" width="16" height="16"><path d="M822.811993 824.617989c-83.075838 81.99224-188.546032 124.613757-316.049383 127.86455-122.085362-3.250794-223.943563-45.87231-305.935802-127.86455s-124.613757-184.21164-127.86455-305.935802c3.250794-127.503351 45.87231-232.973545 127.86455-316.049383 81.99224-83.075838 184.21164-126.058554 305.935802-129.309347 127.503351 3.250794 232.973545 46.23351 316.049383 129.309347 83.075838 83.075838 126.058554 188.546032 129.309347 316.049383C949.231746 640.406349 905.887831 742.62575 822.811993 824.617989zM432.716755 684.111464c3.973192 3.973192 8.307584 5.779189 13.364374 6.140388 5.05679 0.361199 9.752381-1.444797 13.364374-5.417989l292.571429-287.514638c3.973192-3.973192 5.779189-8.307584 5.779189-13.364374 0-5.05679-1.805996-9.752381-5.779189-13.364374l1.805996 1.805996c-3.973192-3.973192-8.668783-5.779189-14.086772-6.140388-5.417989-0.361199-10.47478 1.444797-14.809171 5.417989l-264.397884 220.33157c-3.973192 3.250794-8.668783 4.695591-14.447972 4.695591-5.779189 0-10.835979-1.444797-15.53157-3.973192l-94.273016-72.962257c-4.334392-3.250794-9.391182-4.334392-14.447972-3.973192s-9.391182 3.250794-12.641975 7.585185l-2.889594 3.973192c-3.250794 4.334392-4.334392 9.391182-3.973192 14.809171 0.722399 5.417989 2.528395 10.11358 5.779189 14.086772L432.716755 684.111464z" p-id="2324"></path></svg>\n      </i>\n      <div class="text">${n}</div>\n    `,this.containerEl.appendChild(t),e>0&&setTimeout(()=>{this.close(t)},e)}close(n){n.className=n.className.replace("move-in",""),n.className+="move-out",n.addEventListener("animationend",()=>{n.remove()})}}var Pe={mounted(){!!/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)||this.updateCopy()},updated(){!!/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)||this.updateCopy()},methods:{updateCopy(){setTimeout(()=>{(['div[class*="language-"] pre','div[class*="aside-code"] aside']instanceof Array||Array.isArray(['div[class*="language-"] pre','div[class*="aside-code"] aside']))&&['div[class*="language-"] pre','div[class*="aside-code"] aside'].forEach(n=>{document.querySelectorAll(n).forEach(this.generateCopyButton)})},1e3)},generateCopyButton(n){if(n.classList.contains("codecopy-enabled"))return;const e=document.createElement("i");e.className="code-copy",e.innerHTML='<svg  style="color:#aaa;font-size:14px" t="1572422231464" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="3201" width="14" height="14"><path d="M866.461538 39.384615H354.461538c-43.323077 0-78.769231 35.446154-78.76923 78.769231v39.384616h472.615384c43.323077 0 78.769231 35.446154 78.769231 78.76923v551.384616h39.384615c43.323077 0 78.769231-35.446154 78.769231-78.769231V118.153846c0-43.323077-35.446154-78.769231-78.769231-78.769231z m-118.153846 275.692308c0-43.323077-35.446154-78.769231-78.76923-78.769231H157.538462c-43.323077 0-78.769231 35.446154-78.769231 78.769231v590.769231c0 43.323077 35.446154 78.769231 78.769231 78.769231h512c43.323077 0 78.769231-35.446154 78.76923-78.769231V315.076923z m-354.461538 137.846154c0 11.815385-7.876923 19.692308-19.692308 19.692308h-157.538461c-11.815385 0-19.692308-7.876923-19.692308-19.692308v-39.384615c0-11.815385 7.876923-19.692308 19.692308-19.692308h157.538461c11.815385 0 19.692308 7.876923 19.692308 19.692308v39.384615z m157.538461 315.076923c0 11.815385-7.876923 19.692308-19.692307 19.692308H216.615385c-11.815385 0-19.692308-7.876923-19.692308-19.692308v-39.384615c0-11.815385 7.876923-19.692308 19.692308-19.692308h315.076923c11.815385 0 19.692308 7.876923 19.692307 19.692308v39.384615z m78.769231-157.538462c0 11.815385-7.876923 19.692308-19.692308 19.692308H216.615385c-11.815385 0-19.692308-7.876923-19.692308-19.692308v-39.384615c0-11.815385 7.876923-19.692308 19.692308-19.692308h393.846153c11.815385 0 19.692308 7.876923 19.692308 19.692308v39.384615z" p-id="3202"></path></svg>',e.title="Copy to clipboard",e.addEventListener("click",()=>{this.copyToClipboard(n.innerText)}),n.appendChild(e),n.classList.add("codecopy-enabled")},copyToClipboard(n){const e=document.createElement("textarea");e.value=n,e.setAttribute("readonly",""),e.style.position="absolute",e.style.left="-9999px",document.body.appendChild(e);const t=document.getSelection().rangeCount>0&&document.getSelection().getRangeAt(0);e.select(),document.execCommand("copy");(new _e).show({text:"",duration:1e3}),document.body.removeChild(e),t&&(document.getSelection().removeAllRanges(),document.getSelection().addRange(t))}}},qe="auto",Ie="zoom-in",ze="zoom-out",je="grab",Ne="move";function Be(n,e,t){var i=!(arguments.length>3&&void 0!==arguments[3])||arguments[3],a={passive:!1};i?n.addEventListener(e,t,a):n.removeEventListener(e,t,a)}function De(n,e){if(n){var t=new Image;t.onload=function(){e&&e(t)},t.src=n}}function Re(n){return n.dataset.original?n.dataset.original:"A"===n.parentNode.tagName?n.parentNode.getAttribute("href"):null}function Me(n,e,t){!function(n){var e=Le,t=Oe;if(n.transition){var i=n.transition;delete n.transition,n[e]=i}if(n.transform){var a=n.transform;delete n.transform,n[t]=a}}(e);var i=n.style,a={};for(var r in e)t&&(a[r]=i[r]||""),i[r]=e[r];return a}var Le="transition",Oe="transform",Ge="transform",He="transitionend";var Ue=function(){},Ve={enableGrab:!0,preloadImage:!1,closeOnWindowResize:!0,transitionDuration:.4,transitionTimingFunction:"cubic-bezier(0.4, 0, 0, 1)",bgColor:"rgb(255, 255, 255)",bgOpacity:1,scaleBase:1,scaleExtra:.5,scrollThreshold:40,zIndex:998,customSize:null,onOpen:Ue,onClose:Ue,onGrab:Ue,onMove:Ue,onRelease:Ue,onBeforeOpen:Ue,onBeforeClose:Ue,onBeforeGrab:Ue,onBeforeRelease:Ue,onImageLoading:Ue,onImageLoaded:Ue},Ze={init:function(n){var e,t;e=this,t=n,Object.getOwnPropertyNames(Object.getPrototypeOf(e)).forEach((function(n){e[n]=e[n].bind(t)}))},click:function(n){if(n.preventDefault(),Xe(n))return window.open(this.target.srcOriginal||n.currentTarget.src,"_blank");this.shown?this.released?this.close():this.release():this.open(n.currentTarget)},scroll:function(){var n=document.documentElement||document.body.parentNode||document.body,e=window.pageXOffset||n.scrollLeft,t=window.pageYOffset||n.scrollTop;null===this.lastScrollPosition&&(this.lastScrollPosition={x:e,y:t});var i=this.lastScrollPosition.x-e,a=this.lastScrollPosition.y-t,r=this.options.scrollThreshold;(Math.abs(a)>=r||Math.abs(i)>=r)&&(this.lastScrollPosition=null,this.close())},keydown:function(n){(function(n){return"Escape"===(n.key||n.code)||27===n.keyCode})(n)&&(this.released?this.close():this.release(this.close))},mousedown:function(n){if(We(n)&&!Xe(n)){n.preventDefault();var e=n.clientX,t=n.clientY;this.pressTimer=setTimeout(function(){this.grab(e,t)}.bind(this),200)}},mousemove:function(n){this.released||this.move(n.clientX,n.clientY)},mouseup:function(n){We(n)&&!Xe(n)&&(clearTimeout(this.pressTimer),this.released?this.close():this.release())},touchstart:function(n){n.preventDefault();var e=n.touches[0],t=e.clientX,i=e.clientY;this.pressTimer=setTimeout(function(){this.grab(t,i)}.bind(this),200)},touchmove:function(n){if(!this.released){var e=n.touches[0],t=e.clientX,i=e.clientY;this.move(t,i)}},touchend:function(n){(function(n){n.targetTouches.length})(n)||(clearTimeout(this.pressTimer),this.released?this.close():this.release())},clickOverlay:function(){this.close()},resizeWindow:function(){this.close()}};function We(n){return 0===n.button}function Xe(n){return n.metaKey||n.ctrlKey}var $e={init:function(n){this.el=document.createElement("div"),this.instance=n,this.parent=document.body,Me(this.el,{position:"fixed",top:0,left:0,right:0,bottom:0,opacity:0}),this.updateStyle(n.options),Be(this.el,"click",n.handler.clickOverlay.bind(n))},updateStyle:function(n){Me(this.el,{zIndex:n.zIndex,backgroundColor:n.bgColor,transition:"opacity\n        "+n.transitionDuration+"s\n        "+n.transitionTimingFunction})},insert:function(){this.parent.appendChild(this.el)},remove:function(){this.parent.removeChild(this.el)},fadeIn:function(){this.el.offsetWidth,this.el.style.opacity=this.instance.options.bgOpacity},fadeOut:function(){this.el.style.opacity=0}},Je="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(n){return typeof n}:function(n){return n&&"function"==typeof Symbol&&n.constructor===Symbol&&n!==Symbol.prototype?"symbol":typeof n},Ke=function(){function n(n,e){for(var t=0;t<e.length;t++){var i=e[t];i.enumerable=i.enumerable||!1,i.configurable=!0,"value"in i&&(i.writable=!0),Object.defineProperty(n,i.key,i)}}return function(e,t,i){return t&&n(e.prototype,t),i&&n(e,i),e}}(),Ye=Object.assign||function(n){for(var e=1;e<arguments.length;e++){var t=arguments[e];for(var i in t)Object.prototype.hasOwnProperty.call(t,i)&&(n[i]=t[i])}return n},Qe={init:function(n,e){this.el=n,this.instance=e,this.srcThumbnail=this.el.getAttribute("src"),this.srcset=this.el.getAttribute("srcset"),this.srcOriginal=Re(this.el),this.rect=this.el.getBoundingClientRect(),this.translate=null,this.scale=null,this.styleOpen=null,this.styleClose=null},zoomIn:function(){var n=this.instance.options,e=n.zIndex,t=n.enableGrab,i=n.transitionDuration,a=n.transitionTimingFunction;this.translate=this.calculateTranslate(),this.scale=this.calculateScale(),this.styleOpen={position:"relative",zIndex:e+1,cursor:t?je:ze,transition:Ge+"\n        "+i+"s\n        "+a,transform:"translate3d("+this.translate.x+"px, "+this.translate.y+"px, 0px)\n        scale("+this.scale.x+","+this.scale.y+")",height:this.rect.height+"px",width:this.rect.width+"px"},this.el.offsetWidth,this.styleClose=Me(this.el,this.styleOpen,!0)},zoomOut:function(){this.el.offsetWidth,Me(this.el,{transform:"none"})},grab:function(n,e,t){var i=nt(),a=i.x-n,r=i.y-e;Me(this.el,{cursor:Ne,transform:"translate3d(\n        "+(this.translate.x+a)+"px, "+(this.translate.y+r)+"px, 0px)\n        scale("+(this.scale.x+t)+","+(this.scale.y+t)+")"})},move:function(n,e,t){var i=nt(),a=i.x-n,r=i.y-e;Me(this.el,{transition:Ge,transform:"translate3d(\n        "+(this.translate.x+a)+"px, "+(this.translate.y+r)+"px, 0px)\n        scale("+(this.scale.x+t)+","+(this.scale.y+t)+")"})},restoreCloseStyle:function(){Me(this.el,this.styleClose)},restoreOpenStyle:function(){Me(this.el,this.styleOpen)},upgradeSource:function(){if(this.srcOriginal){var n=this.el.parentNode;this.srcset&&this.el.removeAttribute("srcset");var e=this.el.cloneNode(!1);e.setAttribute("src",this.srcOriginal),e.style.position="fixed",e.style.visibility="hidden",n.appendChild(e),setTimeout(function(){this.el.setAttribute("src",this.srcOriginal),n.removeChild(e)}.bind(this),50)}},downgradeSource:function(){this.srcOriginal&&(this.srcset&&this.el.setAttribute("srcset",this.srcset),this.el.setAttribute("src",this.srcThumbnail))},calculateTranslate:function(){var n=nt(),e=this.rect.left+this.rect.width/2,t=this.rect.top+this.rect.height/2;return{x:n.x-e,y:n.y-t}},calculateScale:function(){var n=this.el.dataset,e=n.zoomingHeight,t=n.zoomingWidth,i=this.instance.options,a=i.customSize,r=i.scaleBase;if(!a&&e&&t)return{x:t/this.rect.width,y:e/this.rect.height};if(a&&"object"===(void 0===a?"undefined":Je(a)))return{x:a.width/this.rect.width,y:a.height/this.rect.height};var o=this.rect.width/2,s=this.rect.height/2,l=nt(),c={x:l.x-o,y:l.y-s},g=c.x/o,d=c.y/s,p=r+Math.min(g,d);if(a&&"string"==typeof a){var u=t||this.el.naturalWidth,m=e||this.el.naturalHeight,h=parseFloat(a)*u/(100*this.rect.width),f=parseFloat(a)*m/(100*this.rect.height);if(p>h||p>f)return{x:h,y:f}}return{x:p,y:p}}};function nt(){var n=document.documentElement;return{x:Math.min(n.clientWidth,window.innerWidth)/2,y:Math.min(n.clientHeight,window.innerHeight)/2}}function et(n,e,t){["mousedown","mousemove","mouseup","touchstart","touchmove","touchend"].forEach((function(i){Be(n,i,e[i],t)}))}var tt=function(){function n(e){!function(n,e){if(!(n instanceof e))throw new TypeError("Cannot call a class as a function")}(this,n),this.target=Object.create(Qe),this.overlay=Object.create($e),this.handler=Object.create(Ze),this.body=document.body,this.shown=!1,this.lock=!1,this.released=!0,this.lastScrollPosition=null,this.pressTimer=null,this.options=Ye({},Ve,e),this.overlay.init(this),this.handler.init(this)}return Ke(n,[{key:"listen",value:function(n){if("string"==typeof n)for(var e=document.querySelectorAll(n),t=e.length;t--;)this.listen(e[t]);else"IMG"===n.tagName&&(n.style.cursor=Ie,Be(n,"click",this.handler.click),this.options.preloadImage&&De(Re(n)));return this}},{key:"config",value:function(n){return n?(Ye(this.options,n),this.overlay.updateStyle(this.options),this):this.options}},{key:"open",value:function(n){var e=this,t=arguments.length>1&&void 0!==arguments[1]?arguments[1]:this.options.onOpen;if(!this.shown&&!this.lock){var i="string"==typeof n?document.querySelector(n):n;if("IMG"===i.tagName){if(this.options.onBeforeOpen(i),this.target.init(i,this),!this.options.preloadImage){var a=this.target.srcOriginal;null!=a&&(this.options.onImageLoading(i),De(a,this.options.onImageLoaded))}this.shown=!0,this.lock=!0,this.target.zoomIn(),this.overlay.insert(),this.overlay.fadeIn(),Be(document,"scroll",this.handler.scroll),Be(document,"keydown",this.handler.keydown),this.options.closeOnWindowResize&&Be(window,"resize",this.handler.resizeWindow);var r=function n(){Be(i,He,n,!1),e.lock=!1,e.target.upgradeSource(),e.options.enableGrab&&et(document,e.handler,!0),t(i)};return Be(i,He,r),this}}}},{key:"close",value:function(){var n=this,e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:this.options.onClose;if(this.shown&&!this.lock){var t=this.target.el;this.options.onBeforeClose(t),this.lock=!0,this.body.style.cursor=qe,this.overlay.fadeOut(),this.target.zoomOut(),Be(document,"scroll",this.handler.scroll,!1),Be(document,"keydown",this.handler.keydown,!1),this.options.closeOnWindowResize&&Be(window,"resize",this.handler.resizeWindow,!1);var i=function i(){Be(t,He,i,!1),n.shown=!1,n.lock=!1,n.target.downgradeSource(),n.options.enableGrab&&et(document,n.handler,!1),n.target.restoreCloseStyle(),n.overlay.remove(),e(t)};return Be(t,He,i),this}}},{key:"grab",value:function(n,e){var t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:this.options.scaleExtra,i=arguments.length>3&&void 0!==arguments[3]?arguments[3]:this.options.onGrab;if(this.shown&&!this.lock){var a=this.target.el;this.options.onBeforeGrab(a),this.released=!1,this.target.grab(n,e,t);var r=function n(){Be(a,He,n,!1),i(a)};return Be(a,He,r),this}}},{key:"move",value:function(n,e){var t=arguments.length>2&&void 0!==arguments[2]?arguments[2]:this.options.scaleExtra,i=arguments.length>3&&void 0!==arguments[3]?arguments[3]:this.options.onMove;if(this.shown&&!this.lock){this.released=!1,this.body.style.cursor=Ne,this.target.move(n,e,t);var a=this.target.el,r=function n(){Be(a,He,n,!1),i(a)};return Be(a,He,r),this}}},{key:"release",value:function(){var n=this,e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:this.options.onRelease;if(this.shown&&!this.lock){var t=this.target.el;this.options.onBeforeRelease(t),this.lock=!0,this.body.style.cursor=qe,this.target.restoreOpenStyle();var i=function i(){Be(t,He,i,!1),n.lock=!1,n.released=!0,e(t)};return Be(t,He,i),this}}}]),n}();const it=JSON.parse('{"bgColor":"rgba(0,0,0,0.6)"}'),at=Number("500");class rt{constructor(){this.instance=new tt(it)}update(n=".theme-vdoing-content img:not(.no-zoom)"){"undefined"!=typeof window&&this.instance.listen(n)}updateDelay(n=".theme-vdoing-content img:not(.no-zoom)",e=at){setTimeout(()=>this.update(n),e)}}var ot=[ye,Te,Se,Pe,{watch:{"$page.path"(){void 0!==this.$vuepress.zooming&&this.$vuepress.zooming.updateDelay()}},mounted(){this.$vuepress.zooming=new rt,this.$vuepress.zooming.updateDelay()}}],st={name:"GlobalLayout",computed:{layout(){const n=this.getLayout();return me("layout",n),i.default.component(n)}},methods:{getLayout(){if(this.$page.path){const n=this.$page.frontmatter.layout;return n&&(this.$vuepress.getLayoutAsyncComponent(n)||this.$vuepress.getVueComponent(n))?n:"Layout"}return"NotFound"}}},lt=t(1),ct=Object(lt.a)(st,(function(){var n=this.$createElement;return(this._self._c||n)(this.layout,{tag:"component"})}),[],!1,null,null,null).exports;!function(n,e,t){switch(e){case"components":n[e]||(n[e]={}),Object.assign(n[e],t);break;case"mixins":n[e]||(n[e]=[]),n[e].push(...t);break;default:throw new Error("Unknown option name.")}}(ct,"mixins",ot);const gt=[{name:"v-718147da",path:"/pages/585994/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-718147da").then(t)}},{path:"/pages/585994/index.html",redirect:"/pages/585994/"},{path:"/00./01. - /00..html",redirect:"/pages/585994/"},{name:"v-8bb86ff6",path:"/pages/4c56e7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-8bb86ff6").then(t)}},{path:"/pages/4c56e7/index.html",redirect:"/pages/4c56e7/"},{path:"/00./02. - /02..html",redirect:"/pages/4c56e7/"},{name:"v-fb61cf0a",path:"/pages/577e62/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-fb61cf0a").then(t)}},{path:"/pages/577e62/index.html",redirect:"/pages/577e62/"},{path:"/00./02. - /00..html",redirect:"/pages/577e62/"},{name:"v-4f1678f4",path:"/pages/e2e6a4/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4f1678f4").then(t)}},{path:"/pages/e2e6a4/index.html",redirect:"/pages/e2e6a4/"},{path:"/00./02. - /12..html",redirect:"/pages/e2e6a4/"},{name:"v-6b3ea08c",path:"/pages/e90196/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6b3ea08c").then(t)}},{path:"/pages/e90196/index.html",redirect:"/pages/e90196/"},{path:"/00./02. - /13..html",redirect:"/pages/e90196/"},{name:"v-033b6f48",path:"/pages/69dc52/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-033b6f48").then(t)}},{path:"/pages/69dc52/index.html",redirect:"/pages/69dc52/"},{path:"/00./02. - /14..html",redirect:"/pages/69dc52/"},{name:"v-634587af",path:"/pages/98350d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-634587af").then(t)}},{path:"/pages/98350d/index.html",redirect:"/pages/98350d/"},{path:"/00./03. - /00..html",redirect:"/pages/98350d/"},{name:"v-a6be02e2",path:"/pages/3c3418/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-a6be02e2").then(t)}},{path:"/pages/3c3418/index.html",redirect:"/pages/3c3418/"},{path:"/00./03. - /12. - .html",redirect:"/pages/3c3418/"},{name:"v-46d8e2d8",path:"/pages/09e806/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-46d8e2d8").then(t)}},{path:"/pages/09e806/index.html",redirect:"/pages/09e806/"},{path:"/00./02. - /01..html",redirect:"/pages/09e806/"},{name:"v-3cb81039",path:"/pages/11a817/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3cb81039").then(t)}},{path:"/pages/11a817/index.html",redirect:"/pages/11a817/"},{path:"/00./03. - /01..html",redirect:"/pages/11a817/"},{name:"v-c6085372",path:"/pages/8b01b8/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c6085372").then(t)}},{path:"/pages/8b01b8/index.html",redirect:"/pages/8b01b8/"},{path:"/00./03. - /14. - .html",redirect:"/pages/8b01b8/"},{name:"v-18d4ff6e",path:"/pages/967a30/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-18d4ff6e").then(t)}},{path:"/pages/967a30/index.html",redirect:"/pages/967a30/"},{path:"/00./03. - /11. - .html",redirect:"/pages/967a30/"},{name:"v-6c8f1043",path:"/pages/c3f74c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6c8f1043").then(t)}},{path:"/pages/c3f74c/index.html",redirect:"/pages/c3f74c/"},{path:"/00./03. - /15. - NGS.html",redirect:"/pages/c3f74c/"},{name:"v-584566ea",path:"/pages/726c78/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-584566ea").then(t)}},{path:"/pages/726c78/index.html",redirect:"/pages/726c78/"},{path:"/00./03. - /13. - .html",redirect:"/pages/726c78/"},{name:"v-5c47aead",path:"/pages/17c432/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5c47aead").then(t)}},{path:"/pages/17c432/index.html",redirect:"/pages/17c432/"},{path:"/00./02. - /11..html",redirect:"/pages/17c432/"},{name:"v-5dc118b8",path:"/pages/d4684b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5dc118b8").then(t)}},{path:"/pages/d4684b/index.html",redirect:"/pages/d4684b/"},{path:"/00./04.Web - /16. - Ajax.html",redirect:"/pages/d4684b/"},{name:"v-1172eaec",path:"/pages/3ca8f4/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1172eaec").then(t)}},{path:"/pages/3ca8f4/index.html",redirect:"/pages/3ca8f4/"},{path:"/00./04.Web - /11. - HTML.html",redirect:"/pages/3ca8f4/"},{name:"v-1a2241f6",path:"/pages/4efbc6/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1a2241f6").then(t)}},{path:"/pages/4efbc6/index.html",redirect:"/pages/4efbc6/"},{path:"/00./04.Web - /00.Web.html",redirect:"/pages/4efbc6/"},{name:"v-5ec3cdd2",path:"/pages/9ec206/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5ec3cdd2").then(t)}},{path:"/pages/9ec206/index.html",redirect:"/pages/9ec206/"},{path:"/00./04.Web - /13. - JS&TS.html",redirect:"/pages/9ec206/"},{name:"v-350d1abc",path:"/pages/98c9ed/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-350d1abc").then(t)}},{path:"/pages/98c9ed/index.html",redirect:"/pages/98c9ed/"},{path:"/00./04.Web - /21. - MySQL.html",redirect:"/pages/98c9ed/"},{name:"v-1a35b060",path:"/pages/e1436c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1a35b060").then(t)}},{path:"/pages/e1436c/index.html",redirect:"/pages/e1436c/"},{path:"/00./04.Web - /22. - Django.html",redirect:"/pages/e1436c/"},{name:"v-4e30e452",path:"/pages/f16f8d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4e30e452").then(t)}},{path:"/pages/f16f8d/index.html",redirect:"/pages/f16f8d/"},{path:"/00./04.Web - /12. - CSS.html",redirect:"/pages/f16f8d/"},{name:"v-93263dc4",path:"/pages/1abc5f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-93263dc4").then(t)}},{path:"/pages/1abc5f/index.html",redirect:"/pages/1abc5f/"},{path:"/00./04.Web - /14. - Markdown.html",redirect:"/pages/1abc5f/"},{name:"v-0fb1f416",path:"/pages/01057a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0fb1f416").then(t)}},{path:"/pages/01057a/index.html",redirect:"/pages/01057a/"},{path:"/00./05. - /01..html",redirect:"/pages/01057a/"},{name:"v-f77517aa",path:"/pages/3c0f51/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-f77517aa").then(t)}},{path:"/pages/3c0f51/index.html",redirect:"/pages/3c0f51/"},{path:"/00./05. - /00..html",redirect:"/pages/3c0f51/"},{name:"v-5ec20e5c",path:"/pages/d2038c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5ec20e5c").then(t)}},{path:"/pages/d2038c/index.html",redirect:"/pages/d2038c/"},{path:"/00./04.Web - /15. - Vue.html",redirect:"/pages/d2038c/"},{name:"v-4312c42e",path:"/pages/99a1ec/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4312c42e").then(t)}},{path:"/pages/99a1ec/index.html",redirect:"/pages/99a1ec/"},{path:"/00./05. - /11. - .html",redirect:"/pages/99a1ec/"},{name:"v-e52b92e2",path:"/pages/baed44/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e52b92e2").then(t)}},{path:"/pages/baed44/index.html",redirect:"/pages/baed44/"},{path:"/00./06. - /00..html",redirect:"/pages/baed44/"},{name:"v-31e3c08b",path:"/pages/7daf99/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-31e3c08b").then(t)}},{path:"/pages/7daf99/index.html",redirect:"/pages/7daf99/"},{path:"/00./05. - /12. - .html",redirect:"/pages/7daf99/"},{name:"v-294d13ab",path:"/pages/8cb41f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-294d13ab").then(t)}},{path:"/pages/8cb41f/index.html",redirect:"/pages/8cb41f/"},{path:"/00./100. - /00..html",redirect:"/pages/8cb41f/"},{name:"v-3d52f327",path:"/pages/be5d67/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3d52f327").then(t)}},{path:"/pages/be5d67/index.html",redirect:"/pages/be5d67/"},{path:"/00./07. - /11. - .html",redirect:"/pages/be5d67/"},{name:"v-5bcc7c9e",path:"/pages/13ae7d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5bcc7c9e").then(t)}},{path:"/pages/13ae7d/index.html",redirect:"/pages/13ae7d/"},{path:"/00./100. - /01. - .html",redirect:"/pages/13ae7d/"},{name:"v-490122ed",path:"/pages/9ab953/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-490122ed").then(t)}},{path:"/pages/9ab953/index.html",redirect:"/pages/9ab953/"},{path:"/00./06. - /10.R .html",redirect:"/pages/9ab953/"},{name:"v-0e096556",path:"/pages/2b095a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0e096556").then(t)}},{path:"/pages/2b095a/index.html",redirect:"/pages/2b095a/"},{path:"/00./100. - /02. - .html",redirect:"/pages/2b095a/"},{name:"v-79611a61",path:"/pages/634e82/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-79611a61").then(t)}},{path:"/pages/634e82/index.html",redirect:"/pages/634e82/"},{path:"/00./07. - /00..html",redirect:"/pages/634e82/"},{name:"v-14a9ddeb",path:"/pages/d28eb7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-14a9ddeb").then(t)}},{path:"/pages/d28eb7/index.html",redirect:"/pages/d28eb7/"},{path:"/00./99. - /00..html",redirect:"/pages/d28eb7/"},{name:"v-98790e58",path:"/pages/1a58a6/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-98790e58").then(t)}},{path:"/pages/1a58a6/index.html",redirect:"/pages/1a58a6/"},{path:"/00./99. - /03..html",redirect:"/pages/1a58a6/"},{name:"v-39f4cdd0",path:"/pages/1260b7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-39f4cdd0").then(t)}},{path:"/pages/1260b7/index.html",redirect:"/pages/1260b7/"},{path:"/01./001.OneNote.html",redirect:"/pages/1260b7/"},{name:"v-a89b5f26",path:"/navigation/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-a89b5f26").then(t)}},{path:"/navigation/index.html",redirect:"/navigation/"},{path:"/01..html",redirect:"/navigation/"},{name:"v-e1b6b19c",path:"/pages/420e29/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e1b6b19c").then(t)}},{path:"/pages/420e29/index.html",redirect:"/pages/420e29/"},{path:"/01./003.NCBI.html",redirect:"/pages/420e29/"},{name:"v-6c95ab70",path:"/pages/7100c2/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6c95ab70").then(t)}},{path:"/pages/7100c2/index.html",redirect:"/pages/7100c2/"},{path:"/02./11./01./20180609.(2018.06.09  Curr Opin Neurobiol) Transcriptional control of long-range cortical projections.html",redirect:"/pages/7100c2/"},{name:"v-c7d2b644",path:"/pages/413897/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c7d2b644").then(t)}},{path:"/pages/413897/index.html",redirect:"/pages/413897/"},{path:"/01./002.Snipaste.html",redirect:"/pages/413897/"},{name:"v-7258202a",path:"/pages/1aae84/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7258202a").then(t)}},{path:"/pages/1aae84/index.html",redirect:"/pages/1aae84/"},{path:"/02./11./01./20121220.(2012.12.20 mPFC  Neuron) The Role of Medial Prefrontal Cortex in Memory and Decision Making.html",redirect:"/pages/1aae84/"},{name:"v-21cb322c",path:"/pages/dc1352/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-21cb322c").then(t)}},{path:"/pages/dc1352/index.html",redirect:"/pages/dc1352/"},{path:"/02./11./02./20160718.(2016.07.18 MAPseq Neuron) High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA.html",redirect:"/pages/dc1352/"},{name:"v-76ee6b70",path:"/pages/7ffbf6/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-76ee6b70").then(t)}},{path:"/pages/7ffbf6/index.html",redirect:"/pages/7ffbf6/"},{path:"/02./11./02./20140402.(2014.04.02 Allen Nature) A mesoscale connectome of the mouse brain.html",redirect:"/pages/7ffbf6/"},{name:"v-15bfff01",path:"/pages/8cb60c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-15bfff01").then(t)}},{path:"/pages/8cb60c/index.html",redirect:"/pages/8cb60c/"},{path:"/02./11./02./20180328.(2018.03.28  Nature 2018) The logic of single-cell projections from  visual cortex.html",redirect:"/pages/8cb60c/"},{name:"v-7180fb28",path:"/pages/9e1c62/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7180fb28").then(t)}},{path:"/pages/9e1c62/index.html",redirect:"/pages/9e1c62/"},{path:"/02./11./01./20181116.(2018.11.16  Trends Cogn Sci) Bridging the Gap between Connectome and Transcriptome.html",redirect:"/pages/9e1c62/"},{name:"v-61bb19fa",path:"/pages/986ab0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-61bb19fa").then(t)}},{path:"/pages/986ab0/index.html",redirect:"/pages/986ab0/"},{path:"/02./11./02./20181030.(2018.10.31 PT-thalamus-medulla Nature) Distinct descending motor cortex pathways and their roles in movement.html",redirect:"/pages/986ab0/"},{name:"v-3e8128e8",path:"/pages/037be6/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3e8128e8").then(t)}},{path:"/pages/037be6/index.html",redirect:"/pages/037be6/"},{path:"/02./11./02./20190208.(2019.02.08  Mol Neurodegeneration) Differences in neurotropism and neurotoxicity among retrograde viral tracers.html",redirect:"/pages/037be6/"},{name:"v-5ac83554",path:"/pages/afda58/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5ac83554").then(t)}},{path:"/pages/afda58/index.html",redirect:"/pages/afda58/"},{path:"/02./11./02./20200709.(2020.07.09 BRICseq Cell) BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals.html",redirect:"/pages/afda58/"},{name:"v-21fa6f80",path:"/pages/af94c8/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-21fa6f80").then(t)}},{path:"/pages/af94c8/index.html",redirect:"/pages/af94c8/"},{path:"/02./11./02./20181031.(2018.10.31  Nature) Shared and distinct transcriptomic cell types across neocortical areas.html",redirect:"/pages/af94c8/"},{name:"v-393821ce",path:"/pages/4918af/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-393821ce").then(t)}},{path:"/pages/4918af/index.html",redirect:"/pages/4918af/"},{path:"/02./11./02./20200722.(2020.07.22  Nature) Distinct subnetworks of the thalamic reticular nucleus.html",redirect:"/pages/4918af/"},{name:"v-4fbe66c8",path:"/pages/fa4bef/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4fbe66c8").then(t)}},{path:"/pages/fa4bef/index.html",redirect:"/pages/fa4bef/"},{path:"/02./11./02./20200914.(2020.09.14 MAPseq Nat Neurosci) Circuit and molecular architecture of a ventral hippocampal network.html",redirect:"/pages/fa4bef/"},{name:"v-98642aa8",path:"/pages/521e00/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-98642aa8").then(t)}},{path:"/pages/521e00/index.html",redirect:"/pages/521e00/"},{path:"/02./11./02./20201214.(2020.12.14 SCM PNAS) Uncovering the genetic blueprint of the C. elegans nervous system.html",redirect:"/pages/521e00/"},{name:"v-64a80ee8",path:"/pages/da828f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-64a80ee8").then(t)}},{path:"/pages/da828f/index.html",redirect:"/pages/da828f/"},{path:"/02./11./02./20191017.(2019.10.17 BAR-seq Cell) High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing.html",redirect:"/pages/da828f/"},{name:"v-485075cc",path:"/pages/ccc053/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-485075cc").then(t)}},{path:"/pages/ccc053/index.html",redirect:"/pages/ccc053/"},{path:"/02./11./02./20211005.(2021.10.06 epi-retro-seq Nature) Epigenomic diversity of cortical projection neurons in the mouse brain.html",redirect:"/pages/ccc053/"},{name:"v-79262273",path:"/pages/5c79f3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-79262273").then(t)}},{path:"/pages/5c79f3/index.html",redirect:"/pages/5c79f3/"},{path:"/02./11./02./20201217.(2020.12.17  Cell) Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks.html",redirect:"/pages/5c79f3/"},{name:"v-40e67c36",path:"/pages/084a0b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-40e67c36").then(t)}},{path:"/pages/084a0b/index.html",redirect:"/pages/084a0b/"},{path:"/02./11./02./20210510.(2021.05.10 BARseq2 Neuroscience) Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections.html",redirect:"/pages/084a0b/"},{name:"v-5c36d2c2",path:"/pages/40cb7b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5c36d2c2").then(t)}},{path:"/pages/40cb7b/index.html",redirect:"/pages/40cb7b/"},{path:"/02./11./02./20211221.(2021.12.21 VECTORseq Cell Reports) Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neuron involved in sensorimotor integration.html",redirect:"/pages/40cb7b/"},{name:"v-44435d26",path:"/pages/d5ec9f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-44435d26").then(t)}},{path:"/pages/d5ec9f/index.html",redirect:"/pages/d5ec9f/"},{path:"/02./11./02./20211006.(2021.10.06 MOp-ul Nature) Cellular anatomy of the mouse primary motor cortex.html",redirect:"/pages/d5ec9f/"},{name:"v-afb7dc3c",path:"/pages/aaea7b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-afb7dc3c").then(t)}},{path:"/pages/aaea7b/index.html",redirect:"/pages/aaea7b/"},{path:"/02./11./02./20231018.(2023.10.18 MAPseq macaque Neuron) Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex.html",redirect:"/pages/aaea7b/"},{name:"v-e7256176",path:"/pages/94c1e9/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e7256176").then(t)}},{path:"/pages/94c1e9/index.html",redirect:"/pages/94c1e9/"},{path:"/02./11./02./20221027.(2022.10.27 MAPseq Cell) High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex.html",redirect:"/pages/94c1e9/"},{name:"v-e6a0d06a",path:"/pages/0be7e3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e6a0d06a").then(t)}},{path:"/pages/0be7e3/index.html",redirect:"/pages/0be7e3/"},{path:"/02./11./02./20220331.(2022.03.31 fMOST Nat Neurosci) Single-neuron projectome of mouse prefrontal cortex.html",redirect:"/pages/0be7e3/"},{name:"v-d1ce449a",path:"/pages/e528dd/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-d1ce449a").then(t)}},{path:"/pages/e528dd/index.html",redirect:"/pages/e528dd/"},{path:"/02./11./02./20230522.(2023.5.22 fMOST Nat Neurosci) Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex.html",redirect:"/pages/e528dd/"},{name:"v-128dac97",path:"/pages/b405a1/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-128dac97").then(t)}},{path:"/pages/b405a1/index.html",redirect:"/pages/b405a1/"},{path:"/02./11./02./20231213.(2023.12.13 epi-retro-seq Nature) Temporal controls over inter-areal cortical projection neuron fate diversity.html",redirect:"/pages/b405a1/"},{name:"v-722b1d75",path:"/pages/f20dad/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-722b1d75").then(t)}},{path:"/pages/f20dad/index.html",redirect:"/pages/f20dad/"},{path:"/02./11./02./20240202.(2024.02.02 fmost Science) Whole-brain spatial organization of hippocampal single-neuron projectomes.html",redirect:"/pages/f20dad/"},{name:"v-9ab7ce14",path:"/pages/b79c35/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-9ab7ce14").then(t)}},{path:"/pages/b79c35/index.html",redirect:"/pages/b79c35/"},{path:"/02./11./03./20220120.(2022.01.20  Cell) Vision-dependent specification of cell types and function in the developing cortex.html",redirect:"/pages/b79c35/"},{name:"v-f79bb6f6",path:"/pages/723a6a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-f79bb6f6").then(t)}},{path:"/pages/723a6a/index.html",redirect:"/pages/723a6a/"},{path:"/02./11./02./20240223.(2024.02.23 MERGE-seq eLife) High-throughput mapping of single-neuron projection and molecular features by  retrograde barcoded labelling.html",redirect:"/pages/723a6a/"},{name:"v-444950d4",path:"/pages/9c9ddc/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-444950d4").then(t)}},{path:"/pages/9c9ddc/index.html",redirect:"/pages/9c9ddc/"},{path:"/02./11./02./20240610.(2024.06.10  eLife) Deciphering the genetic code of neuronal type connectivity through bilinear modeling.html",redirect:"/pages/9c9ddc/"},{name:"v-09d0538e",path:"/pages/0b119c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-09d0538e").then(t)}},{path:"/pages/0b119c/index.html",redirect:"/pages/0b119c/"},{path:"/02./11./03./20221110.(2022.11.10 PFC Cell) Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution.html",redirect:"/pages/0b119c/"},{name:"v-82511ca6",path:"/pages/16bdaa/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-82511ca6").then(t)}},{path:"/pages/16bdaa/index.html",redirect:"/pages/16bdaa/"},{path:"/02./11./03./20221228.(2022.12.28 MERFISH Cell) Molecular and spatial signatures of mouse brain aging at single-cell resolution.html",redirect:"/pages/16bdaa/"},{name:"v-4ca30be4",path:"/pages/8bf2bf/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4ca30be4").then(t)}},{path:"/pages/8bf2bf/index.html",redirect:"/pages/8bf2bf/"},{path:"/02./11./04./20171011.(2017.10.11 Act-seq Neuron) Detecting Activated Cell Populations Using Single-Cell RNA-Seq.html",redirect:"/pages/8bf2bf/"},{name:"v-704e8ccd",path:"/pages/9b041f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-704e8ccd").then(t)}},{path:"/pages/9b041f/index.html",redirect:"/pages/9b041f/"},{path:"/02./11./04./20160628.(2016.06.28 Div-Seq Science) Div-Seq Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons.html",redirect:"/pages/9b041f/"},{name:"v-2b6b4421",path:"/pages/a61efc/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2b6b4421").then(t)}},{path:"/pages/a61efc/index.html",redirect:"/pages/a61efc/"},{path:"/02./11./04./20160816.(2016.08.16  PNAS) Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons.html",redirect:"/pages/a61efc/"},{name:"v-9245513c",path:"/pages/37a825/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-9245513c").then(t)}},{path:"/pages/37a825/index.html",redirect:"/pages/37a825/"},{path:"/02./11./04./20200626.(2020.06.26  Sci Adv) Molecular atlas of the adult mouse brain.html",redirect:"/pages/37a825/"},{name:"v-1f4da734",path:"/pages/0b79fa/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1f4da734").then(t)}},{path:"/pages/0b79fa/index.html",redirect:"/pages/0b79fa/"},{path:"/02./11./04./20200115.(2020.01.15  nature) Decoding the development of the human  hippocampus.html",redirect:"/pages/0b79fa/"},{name:"v-3eebebda",path:"/pages/c60442/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3eebebda").then(t)}},{path:"/pages/c60442/index.html",redirect:"/pages/c60442/"},{path:"/02./11./04./20190913.(2019.09.13 mPFC Nat Commun) Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction.html",redirect:"/pages/c60442/"},{name:"v-13e0acb8",path:"/pages/bb2e69/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-13e0acb8").then(t)}},{path:"/pages/bb2e69/index.html",redirect:"/pages/bb2e69/"},{path:"/02./11./04./20191111.(2019.11.11  Elife) Single-cell transcriptomic evidence for dense intracortical neuropeptide networks.html",redirect:"/pages/bb2e69/"},{name:"v-cb41435c",path:"/pages/b5aaa4/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-cb41435c").then(t)}},{path:"/pages/b5aaa4/index.html",redirect:"/pages/b5aaa4/"},{path:"/02./11./04./20201207.(2020.12.07  Nat Neurosci) Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing.html",redirect:"/pages/b5aaa4/"},{name:"v-53c700f9",path:"/pages/25c02b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-53c700f9").then(t)}},{path:"/pages/25c02b/index.html",redirect:"/pages/25c02b/"},{path:"/02./11./04./20210708.(2021.07.08 hdWGCNA Nat Genet) Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease.html",redirect:"/pages/25c02b/"},{name:"v-f88540b8",path:"/pages/f481a1/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-f88540b8").then(t)}},{path:"/pages/f481a1/index.html",redirect:"/pages/f481a1/"},{path:"/02./11./04./20220530.(2022.05.30  Nat Neurosci) Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations.html",redirect:"/pages/f481a1/"},{name:"v-7b5d7342",path:"/pages/8a50f3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7b5d7342").then(t)}},{path:"/pages/8a50f3/index.html",redirect:"/pages/8a50f3/"},{path:"/02./11./04./20210623.(2021.06.23  nature) Molecular logic of cellular diversification in the mouse cerebral cortex.html",redirect:"/pages/8a50f3/"},{name:"v-52142b16",path:"/pages/8c91b0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-52142b16").then(t)}},{path:"/pages/8c91b0/index.html",redirect:"/pages/8c91b0/"},{path:"/02./11./04./20230712.(2023.07.12 stereo-seq Cell) Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex.html",redirect:"/pages/8c91b0/"},{name:"v-95c1c078",path:"/pages/444fce/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-95c1c078").then(t)}},{path:"/pages/444fce/index.html",redirect:"/pages/444fce/"},{path:"/02./11./04./20220621.(2022.06.21 hPFC PNAS) The neuropeptide landscape of human prefrontal cortex.html",redirect:"/pages/444fce/"},{name:"v-6de6612b",path:"/pages/42c71a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6de6612b").then(t)}},{path:"/pages/42c71a/index.html",redirect:"/pages/42c71a/"},{path:"/02./11./04./20231016.(2023.10.16 mPFC MERFISH Nat Neurosci) Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain.html",redirect:"/pages/42c71a/"},{name:"v-51b78e88",path:"/pages/a1ce18/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-51b78e88").then(t)}},{path:"/pages/a1ce18/index.html",redirect:"/pages/a1ce18/"},{path:"/02./11./04./20231115.(2023.11.15  Neuron) The neuropeptidergic connectome of C. elegans.html",redirect:"/pages/a1ce18/"},{name:"v-6f2dd4c9",path:"/pages/666cd3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6f2dd4c9").then(t)}},{path:"/pages/666cd3/index.html",redirect:"/pages/666cd3/"},{path:"/02./11./04./20240424.(2024.04.24 BARseq Nature) Whole-cortex in situ sequencing reveals input-dependent area identity.html",redirect:"/pages/666cd3/"},{name:"v-2cc66357",path:"/pages/12b484/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2cc66357").then(t)}},{path:"/pages/12b484/index.html",redirect:"/pages/12b484/"},{path:"/02./11./04./20231101.(2023.11.01  Sci. Adv.) A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types.html",redirect:"/pages/12b484/"},{name:"v-c424057a",path:"/pages/3eedee/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c424057a").then(t)}},{path:"/pages/3eedee/index.html",redirect:"/pages/3eedee/"},{path:"/02./12./20220220.(2022.02.20 CRD J Pineal Res)Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma.html",redirect:"/pages/3eedee/"},{name:"v-6086e0c3",path:"/pages/895303/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6086e0c3").then(t)}},{path:"/pages/895303/index.html",redirect:"/pages/895303/"},{path:"/02./11./04./20231213.(2023.12.13 Allen bioRxiv) A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain.html",redirect:"/pages/895303/"},{name:"v-3c7ee81a",path:"/pages/c40bc0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3c7ee81a").then(t)}},{path:"/pages/c40bc0/index.html",redirect:"/pages/c40bc0/"},{path:"/02./12./20230127.(2023.01.27 -FISH rainbow Nat Commun) Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules.html",redirect:"/pages/c40bc0/"},{name:"v-72b734c5",path:"/pages/b55e03/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-72b734c5").then(t)}},{path:"/pages/b55e03/index.html",redirect:"/pages/b55e03/"},{path:"/02./12./20240704.(2024.07.04  Nat Methods) Systematic comparison of sequencing-based spatial transcriptomic methods.html",redirect:"/pages/b55e03/"},{name:"v-6ed06999",path:"/pages/e6566f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6ed06999").then(t)}},{path:"/pages/e6566f/index.html",redirect:"/pages/e6566f/"},{path:"/02./12./20240514.(2024.05.14 MiP-seq Nat. Biomed. Eng) Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing.html",redirect:"/pages/e6566f/"},{name:"v-2607359c",path:"/pages/76fbba/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2607359c").then(t)}},{path:"/pages/76fbba/index.html",redirect:"/pages/76fbba/"},{path:"/02./13./20180426.(2018.04.26 DLO Hi-C Nat Genet) Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture.html",redirect:"/pages/76fbba/"},{name:"v-49747540",path:"/pages/ffdcc0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-49747540").then(t)}},{path:"/pages/ffdcc0/index.html",redirect:"/pages/ffdcc0/"},{path:"/02./14./20081228.(2008.12.28 WGCNA BMC Bioinformatics) WGCNA an R package for weighted correlation network analysis.html",redirect:"/pages/ffdcc0/"},{name:"v-35f11cad",path:"/pages/0d1fce/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-35f11cad").then(t)}},{path:"/pages/0d1fce/index.html",redirect:"/pages/0d1fce/"},{path:"/02./14./20150413.(2015.04.13 Seurat V1 Nat Biotechnol) Spatial reconstruction of single-cell gene expression data.html",redirect:"/pages/0d1fce/"},{name:"v-26ae0bd4",path:"/pages/ebb632/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-26ae0bd4").then(t)}},{path:"/pages/ebb632/index.html",redirect:"/pages/ebb632/"},{path:"/02./14./20180206.(2018.02.06 scanpy Genome Biol) SCANPY large-scale single-cell gene expression data analysis.html",redirect:"/pages/ebb632/"},{name:"v-9dcdff76",path:"/pages/06c5a0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-9dcdff76").then(t)}},{path:"/pages/06c5a0/index.html",redirect:"/pages/06c5a0/"},{path:"/02./14./20180628.(2018.06.28 MAGIC Cell) Recovering Gene Interactions from Single-Cell Data Using Data Diffusion.html",redirect:"/pages/06c5a0/"},{name:"v-195442fc",path:"/pages/e020de/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-195442fc").then(t)}},{path:"/pages/e020de/index.html",redirect:"/pages/e020de/"},{path:"/02./14./20171204.(2017.12.04 WholeBrain Nat Neurosci) An interactive framework for whole-brain maps at cellular resolution.html",redirect:"/pages/e020de/"},{name:"v-5d420fe2",path:"/pages/d26f9e/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5d420fe2").then(t)}},{path:"/pages/d26f9e/index.html",redirect:"/pages/d26f9e/"},{path:"/02./14./20190606.(2019.06.06 Seurat V3 Cell) Comprehensive Integration of Single-Cell Data.html",redirect:"/pages/d26f9e/"},{name:"v-778d5c48",path:"/pages/f37280/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-778d5c48").then(t)}},{path:"/pages/f37280/index.html",redirect:"/pages/f37280/"},{path:"/02./14./20180402.(2018.04.02 Seurat V2 Nat Biotechnol) Integrating single-cell transcriptomic data across different conditions, technologies, and species.html",redirect:"/pages/f37280/"},{name:"v-084ec0e5",path:"/pages/71b694/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-084ec0e5").then(t)}},{path:"/pages/71b694/index.html",redirect:"/pages/71b694/"},{path:"/02./14./20190403.(2019.04.03 DoubletFinder Cell Systems) DoubletFinder Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors.html",redirect:"/pages/71b694/"},{name:"v-620e68fd",path:"/pages/c34d2c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-620e68fd").then(t)}},{path:"/pages/c34d2c/index.html",redirect:"/pages/c34d2c/"},{path:"/02./14./20201210.(2020.12.10, GCNG, Genome Biol) GCNG graph convolutional networks for inferring gene interaction from spatial transcriptomics data.html",redirect:"/pages/c34d2c/"},{name:"v-48e1665e",path:"/pages/d38252/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-48e1665e").then(t)}},{path:"/pages/d38252/index.html",redirect:"/pages/d38252/"},{path:"/02./14./20201207.(2020.12.07 nnU-Net Nat Methods) nnU-Net a self-configuring method for deep learning-based biomedical image segmentation.html",redirect:"/pages/d38252/"},{name:"v-65c7b89c",path:"/pages/c7adb0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-65c7b89c").then(t)}},{path:"/pages/c7adb0/index.html",redirect:"/pages/c7adb0/"},{path:"/02./14./20200914.(2020.09.14 scRNAseq imputation Genome Biol) A systematic evaluation of single-cell RNA-sequencing imputation methods.html",redirect:"/pages/c7adb0/"},{name:"v-10ed743c",path:"/pages/1f3ea0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-10ed743c").then(t)}},{path:"/pages/1f3ea0/index.html",redirect:"/pages/1f3ea0/"},{path:"/02./14./20201214.(2020.12.14 Cellpose Nat Methods) Cellpose a generalist algorithm for cellular segmentation.html",redirect:"/pages/1f3ea0/"},{name:"v-7bb51432",path:"/pages/2737d2/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7bb51432").then(t)}},{path:"/pages/2737d2/index.html",redirect:"/pages/2737d2/"},{path:"/02./14./20210217.(2021.02.17 CellChat Nat Commun) Inference and analysis of cell-cell communication using CellChat.html",redirect:"/pages/2737d2/"},{name:"v-ca793f2a",path:"/pages/1f53c5/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-ca793f2a").then(t)}},{path:"/pages/1f53c5/index.html",redirect:"/pages/1f53c5/"},{path:"/02./14./20201217.(2020.12.17 FAN-C Genome Biol) FAN-C a feature-rich framework for the analysis and visualisation of chromosome conformation capture data.html",redirect:"/pages/1f53c5/"},{name:"v-9366df52",path:"/pages/1c1b2b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-9366df52").then(t)}},{path:"/pages/1c1b2b/index.html",redirect:"/pages/1c1b2b/"},{path:"/02./14./20210206.(2021.02.06  COVID-19 CT Medical Physics) Toward data-efficient learning A benchmark for COVID-19 CT lung and infection segmentation.html",redirect:"/pages/1c1b2b/"},{name:"v-78e6cd76",path:"/pages/6a4bcc/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-78e6cd76").then(t)}},{path:"/pages/6a4bcc/index.html",redirect:"/pages/6a4bcc/"},{path:"/02./14./20210701.(2021.07.01  Liu et al. Front. Genet) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease.html",redirect:"/pages/6a4bcc/"},{name:"v-c46c1bf8",path:"/pages/c3bbd9/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c46c1bf8").then(t)}},{path:"/pages/c3bbd9/index.html",redirect:"/pages/c3bbd9/"},{path:"/02./14./20211028.(2021.10.28 Tangram Nat Methods) Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram.html",redirect:"/pages/c3bbd9/"},{name:"v-15038c83",path:"/pages/76476f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-15038c83").then(t)}},{path:"/pages/76476f/index.html",redirect:"/pages/76476f/"},{path:"/02./14./20220131.(2022.01.31 Squidpy Nat Methods) Squidpy a scalable framework for spatial omics analysis.html",redirect:"/pages/76476f/"},{name:"v-c3ba952a",path:"/pages/9912cc/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c3ba952a").then(t)}},{path:"/pages/9912cc/index.html",redirect:"/pages/9912cc/"},{path:"/02./14./20220113.(2022.01.13 Cell2location Nat Biotechnol) Cell2location maps fine-grained cell types in spatial transcriptomics.html",redirect:"/pages/9912cc/"},{name:"v-1056f6de",path:"/pages/b4dfe0/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1056f6de").then(t)}},{path:"/pages/b4dfe0/index.html",redirect:"/pages/b4dfe0/"},{path:"/02./14./20210624.(2021.06.24 Seurat V4 Cell) Integrated analysis of multimodal single-cell data.html",redirect:"/pages/b4dfe0/"},{name:"v-cebb617c",path:"/pages/112d0b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-cebb617c").then(t)}},{path:"/pages/112d0b/index.html",redirect:"/pages/112d0b/"},{path:"/02./14./20220511.(2022.05.11 Viv Nat Methods) Viv multiscale visualization of high-resolution multiplexed bioimaging data on the web.html",redirect:"/pages/112d0b/"},{name:"v-01f55ac7",path:"/pages/794f06/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-01f55ac7").then(t)}},{path:"/pages/794f06/index.html",redirect:"/pages/794f06/"},{path:"/02./14./20230215.(2023.02.15 NeuronChat Nat Commun) Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat.html",redirect:"/pages/794f06/"},{name:"v-01625f39",path:"/pages/8c5839/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-01625f39").then(t)}},{path:"/pages/8c5839/index.html",redirect:"/pages/8c5839/"},{path:"/02./14./20230325.(2023.03.25 Seurat V5 Nat Biotechnol) Dictionary learning for integrative, multimodal, and scalable single-cell analysis.html",redirect:"/pages/8c5839/"},{name:"v-2766f654",path:"/pages/4cba92/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2766f654").then(t)}},{path:"/pages/4cba92/index.html",redirect:"/pages/4cba92/"},{path:"/02./14./20220930.(2022.09.30, SpaceX, Bioinformatics) SpaceX gene co-expression network estimation for spatial transcriptomics.html",redirect:"/pages/4cba92/"},{name:"v-3121a636",path:"/pages/124edd/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3121a636").then(t)}},{path:"/pages/124edd/index.html",redirect:"/pages/124edd/"},{path:"/02./14./20230410.(2023.04.10 scverse Nat Biotechnol) The scverse project provides a computational ecosystem for single-cell omics data analysis.html",redirect:"/pages/124edd/"},{name:"v-29874988",path:"/pages/a24249/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-29874988").then(t)}},{path:"/pages/a24249/index.html",redirect:"/pages/a24249/"},{path:"/02./14./20230612.(2023.06.12 hdWGCNA Cell Reports Methods) hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data.html",redirect:"/pages/a24249/"},{name:"v-24a06b6a",path:"/pages/da0fb5/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-24a06b6a").then(t)}},{path:"/pages/da0fb5/index.html",redirect:"/pages/da0fb5/"},{path:"/02./14./20231122.(2023.11.22, SPACEL, Nat Commun) SPACEL characterizing spatial transcriptome architectures by deep-learning.html",redirect:"/pages/da0fb5/"},{name:"v-0ef330be",path:"/pages/02ef79/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0ef330be").then(t)}},{path:"/pages/02ef79/index.html",redirect:"/pages/02ef79/"},{path:"/02./14./20240703.(2024.07.03, ezSingleCell, Nat Commun) ezSingleCell an integrated one-stop single-cell and spatial omics analysis platform for bench scientists.html",redirect:"/pages/02ef79/"},{name:"v-22df1dd0",path:"/pages/32a729/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-22df1dd0").then(t)}},{path:"/pages/32a729/index.html",redirect:"/pages/32a729/"},{path:"/02./14./20240614.(2024.06.14, tidyomics, Nat Methods) The tidyomics ecosystem enhancing omic data analyses.html",redirect:"/pages/32a729/"},{name:"v-41ca1859",path:"/pages/d207e7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-41ca1859").then(t)}},{path:"/pages/d207e7/index.html",redirect:"/pages/d207e7/"},{path:"/03./01. - /001.Seurat.html",redirect:"/pages/d207e7/"},{name:"v-616419cb",path:"/pages/a35823/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-616419cb").then(t)}},{path:"/pages/a35823/index.html",redirect:"/pages/a35823/"},{path:"/03./01. - /002.DoubletFinder.html",redirect:"/pages/a35823/"},{name:"v-1d158259",path:"/pages/8dbd6f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1d158259").then(t)}},{path:"/pages/8dbd6f/index.html",redirect:"/pages/8dbd6f/"},{path:"/03./01. - /005.scrattch.html",redirect:"/pages/8dbd6f/"},{name:"v-e06f21f6",path:"/pages/7d0fcf/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e06f21f6").then(t)}},{path:"/pages/7d0fcf/index.html",redirect:"/pages/7d0fcf/"},{path:"/03./01. - /003.anndata.html",redirect:"/pages/7d0fcf/"},{name:"v-0da46b4e",path:"/pages/9d48f3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0da46b4e").then(t)}},{path:"/pages/9d48f3/index.html",redirect:"/pages/9d48f3/"},{path:"/03./01. - /004.MetaNeighbor.html",redirect:"/pages/9d48f3/"},{name:"v-8449846e",path:"/pages/9ffdf7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-8449846e").then(t)}},{path:"/pages/9ffdf7/index.html",redirect:"/pages/9ffdf7/"},{path:"/03./01. - /006.WGCNA.html",redirect:"/pages/9ffdf7/"},{name:"v-0387c6df",path:"/pages/b62f01/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0387c6df").then(t)}},{path:"/pages/b62f01/index.html",redirect:"/pages/b62f01/"},{path:"/03./01. - /008.Monocle.html",redirect:"/pages/b62f01/"},{name:"v-97fad502",path:"/pages/649303/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-97fad502").then(t)}},{path:"/pages/649303/index.html",redirect:"/pages/649303/"},{path:"/03./01. - /007.hdWGCNA.html",redirect:"/pages/649303/"},{name:"v-bf406dce",path:"/pages/f26000/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-bf406dce").then(t)}},{path:"/pages/f26000/index.html",redirect:"/pages/f26000/"},{path:"/03./01. - /009.CellChat.html",redirect:"/pages/f26000/"},{name:"v-434e1f39",path:"/pages/7a726f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-434e1f39").then(t)}},{path:"/pages/7a726f/index.html",redirect:"/pages/7a726f/"},{path:"/03./01. - /010.NeuronChat.html",redirect:"/pages/7a726f/"},{name:"v-d477d30a",path:"/pages/9e5aae/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-d477d30a").then(t)}},{path:"/pages/9e5aae/index.html",redirect:"/pages/9e5aae/"},{path:"/03./01. - /011.scCustomize.html",redirect:"/pages/9e5aae/"},{name:"v-2833738e",path:"/pages/61a60f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2833738e").then(t)}},{path:"/pages/61a60f/index.html",redirect:"/pages/61a60f/"},{path:"/03./01. - /013.scanpy.html",redirect:"/pages/61a60f/"},{name:"v-8d171136",path:"/pages/37f62d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-8d171136").then(t)}},{path:"/pages/37f62d/index.html",redirect:"/pages/37f62d/"},{path:"/03./01. - /014.squidpy.html",redirect:"/pages/37f62d/"},{name:"v-35793acf",path:"/pages/006dc9/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-35793acf").then(t)}},{path:"/pages/006dc9/index.html",redirect:"/pages/006dc9/"},{path:"/03./02. - /001.XGBoost.html",redirect:"/pages/006dc9/"},{name:"v-66ffee6f",path:"/pages/c7e56a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-66ffee6f").then(t)}},{path:"/pages/c7e56a/index.html",redirect:"/pages/c7e56a/"},{path:"/03./01. - /012.Tangram.html",redirect:"/pages/c7e56a/"},{name:"v-5de954f1",path:"/pages/8633a5/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5de954f1").then(t)}},{path:"/pages/8633a5/index.html",redirect:"/pages/8633a5/"},{path:"/03./04. - /001.nnUNet.html",redirect:"/pages/8633a5/"},{name:"v-b2b8642e",path:"/pages/ee4b70/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-b2b8642e").then(t)}},{path:"/pages/ee4b70/index.html",redirect:"/pages/ee4b70/"},{path:"/03./02. - /002.glmnet.html",redirect:"/pages/ee4b70/"},{name:"v-7c78e838",path:"/pages/791228/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7c78e838").then(t)}},{path:"/pages/791228/index.html",redirect:"/pages/791228/"},{path:"/04.Web/11. - HTML/01.HTML.html",redirect:"/pages/791228/"},{name:"v-c279f104",path:"/pages/bb1275/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c279f104").then(t)}},{path:"/pages/bb1275/index.html",redirect:"/pages/bb1275/"},{path:"/04.Web/12. - CSS/01.CSS.html",redirect:"/pages/bb1275/"},{name:"v-4e38c05c",path:"/pages/18d66f/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4e38c05c").then(t)}},{path:"/pages/18d66f/index.html",redirect:"/pages/18d66f/"},{path:"/04.Web/11. - HTML/02.HTML.html",redirect:"/pages/18d66f/"},{name:"v-2a4dc049",path:"/pages/592a6d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2a4dc049").then(t)}},{path:"/pages/592a6d/index.html",redirect:"/pages/592a6d/"},{path:"/04.Web/13. - JS&TS/01.JavaScript.html",redirect:"/pages/592a6d/"},{name:"v-2754ef54",path:"/pages/da13bb/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-2754ef54").then(t)}},{path:"/pages/da13bb/index.html",redirect:"/pages/da13bb/"},{path:"/04.Web/14. - Markdown/01.Markdown.html",redirect:"/pages/da13bb/"},{name:"v-c8068dca",path:"/pages/6ce63c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-c8068dca").then(t)}},{path:"/pages/6ce63c/index.html",redirect:"/pages/6ce63c/"},{path:"/04.Web/13. - JS&TS/02.TypeScript.html",redirect:"/pages/6ce63c/"},{name:"v-1d78ed0e",path:"/pages/f03da3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1d78ed0e").then(t)}},{path:"/pages/f03da3/index.html",redirect:"/pages/f03da3/"},{path:"/04.Web/15. - Vue/01.Vue3.html",redirect:"/pages/f03da3/"},{name:"v-4e102365",path:"/pages/49a104/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4e102365").then(t)}},{path:"/pages/49a104/index.html",redirect:"/pages/49a104/"},{path:"/04.Web/21. - MySQL/01.MySQL.html",redirect:"/pages/49a104/"},{name:"v-f12adf96",path:"/pages/ce8da3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-f12adf96").then(t)}},{path:"/pages/ce8da3/index.html",redirect:"/pages/ce8da3/"},{path:"/04.Web/16. - Ajax/01.AJAX.html",redirect:"/pages/ce8da3/"},{name:"v-43390cd4",path:"/pages/d7e3cb/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-43390cd4").then(t)}},{path:"/pages/d7e3cb/index.html",redirect:"/pages/d7e3cb/"},{path:"/05./01./1..html",redirect:"/pages/d7e3cb/"},{name:"v-8a435fea",path:"/pages/b75e1a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-8a435fea").then(t)}},{path:"/pages/b75e1a/index.html",redirect:"/pages/b75e1a/"},{path:"/04.Web/22. - Django/01.Django.html",redirect:"/pages/b75e1a/"},{name:"v-6770800e",path:"/pages/4a4e4d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6770800e").then(t)}},{path:"/pages/4a4e4d/index.html",redirect:"/pages/4a4e4d/"},{path:"/05./11. - /1..html",redirect:"/pages/4a4e4d/"},{name:"v-bb151b66",path:"/pages/4c1aba/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-bb151b66").then(t)}},{path:"/pages/4c1aba/index.html",redirect:"/pages/4c1aba/"},{path:"/05./01./3..html",redirect:"/pages/4c1aba/"},{name:"v-636d4942",path:"/pages/02fc82/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-636d4942").then(t)}},{path:"/pages/02fc82/index.html",redirect:"/pages/02fc82/"},{path:"/05./01./2..html",redirect:"/pages/02fc82/"},{name:"v-7384231e",path:"/pages/cbc016/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7384231e").then(t)}},{path:"/pages/cbc016/index.html",redirect:"/pages/cbc016/"},{path:"/05./01./4..html",redirect:"/pages/cbc016/"},{name:"v-0f0ab9bb",path:"/pages/3e695c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0f0ab9bb").then(t)}},{path:"/pages/3e695c/index.html",redirect:"/pages/3e695c/"},{path:"/05./11. - /3..html",redirect:"/pages/3e695c/"},{name:"v-6d8cebcc",path:"/pages/f7fe10/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6d8cebcc").then(t)}},{path:"/pages/f7fe10/index.html",redirect:"/pages/f7fe10/"},{path:"/05./11. - /2..html",redirect:"/pages/f7fe10/"},{name:"v-1e343bfb",path:"/pages/e2487d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1e343bfb").then(t)}},{path:"/pages/e2487d/index.html",redirect:"/pages/e2487d/"},{path:"/05./12. - /01..html",redirect:"/pages/e2487d/"},{name:"v-5bf683a0",path:"/pages/0952ad/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5bf683a0").then(t)}},{path:"/pages/0952ad/index.html",redirect:"/pages/0952ad/"},{path:"/05./12. - /03..html",redirect:"/pages/0952ad/"},{name:"v-220be6fa",path:"/pages/64abf4/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-220be6fa").then(t)}},{path:"/pages/64abf4/index.html",redirect:"/pages/64abf4/"},{path:"/05./12. - /02..html",redirect:"/pages/64abf4/"},{name:"v-0c0e50cc",path:"/pages/95b5f9/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0c0e50cc").then(t)}},{path:"/pages/95b5f9/index.html",redirect:"/pages/95b5f9/"},{path:"/05./12. - /04..html",redirect:"/pages/95b5f9/"},{name:"v-3ee28eee",path:"/pages/98868d/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-3ee28eee").then(t)}},{path:"/pages/98868d/index.html",redirect:"/pages/98868d/"},{path:"/06./12.R - tidyverse/02.ggplot2.html",redirect:"/pages/98868d/"},{name:"v-5e86d73a",path:"/pages/81f7ef/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5e86d73a").then(t)}},{path:"/pages/81f7ef/index.html",redirect:"/pages/81f7ef/"},{path:"/05./12. - /06..html",redirect:"/pages/81f7ef/"},{name:"v-1b276464",path:"/pages/c63497/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-1b276464").then(t)}},{path:"/pages/c63497/index.html",redirect:"/pages/c63497/"},{path:"/06./12.R - tidyverse/01.tidyverse.html",redirect:"/pages/c63497/"},{name:"v-ba6ecaea",path:"/pages/142715/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-ba6ecaea").then(t)}},{path:"/pages/142715/index.html",redirect:"/pages/142715/"},{path:"/06./15.R - /006.linkET.html",redirect:"/pages/142715/"},{name:"v-aaf5ea96",path:"/pages/29214b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-aaf5ea96").then(t)}},{path:"/pages/29214b/index.html",redirect:"/pages/29214b/"},{path:"/06./15.R - /004.ggalluvial.html",redirect:"/pages/29214b/"},{name:"v-371f414b",path:"/pages/22fafa/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-371f414b").then(t)}},{path:"/pages/22fafa/index.html",redirect:"/pages/22fafa/"},{path:"/06./15.R - /008.Biorplot.html",redirect:"/pages/22fafa/"},{name:"v-5c93088f",path:"/pages/02875c/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5c93088f").then(t)}},{path:"/pages/02875c/index.html",redirect:"/pages/02875c/"},{path:"/06./16.R - /01.quarto.html",redirect:"/pages/02875c/"},{name:"v-0ec22c8d",path:"/pages/6580bf/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0ec22c8d").then(t)}},{path:"/pages/6580bf/index.html",redirect:"/pages/6580bf/"},{path:"/06./15.R - /005.DiagrammeR.html",redirect:"/pages/6580bf/"},{name:"v-e53f033e",path:"/pages/a41a6b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e53f033e").then(t)}},{path:"/pages/a41a6b/index.html",redirect:"/pages/a41a6b/"},{path:"/06./15.R - /007.circlize.html",redirect:"/pages/a41a6b/"},{name:"v-114caebc",path:"/about/me/introduce/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-114caebc").then(t)}},{path:"/about/me/introduce/index.html",redirect:"/about/me/introduce/"},{path:"/100./01. - /01. - .html",redirect:"/about/me/introduce/"},{name:"v-5ffa3dac",path:"/pages/14b7e3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-5ffa3dac").then(t)}},{path:"/pages/14b7e3/index.html",redirect:"/pages/14b7e3/"},{path:"/07./11. - /01.1 .html",redirect:"/pages/14b7e3/"},{name:"v-41e3f643",path:"/pages/093f81/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-41e3f643").then(t)}},{path:"/pages/093f81/index.html",redirect:"/pages/093f81/"},{path:"/100./02. - /04.-.html",redirect:"/pages/093f81/"},{name:"v-16897589",path:"/pages/94f5fc/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-16897589").then(t)}},{path:"/pages/94f5fc/index.html",redirect:"/pages/94f5fc/"},{path:"/100./02. - /03.-.html",redirect:"/pages/94f5fc/"},{name:"v-623ea45e",path:"/pages/d3830b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-623ea45e").then(t)}},{path:"/pages/d3830b/index.html",redirect:"/pages/d3830b/"},{path:"/100./02. - /01.-.html",redirect:"/pages/d3830b/"},{name:"v-6406ed3f",path:"/pages/e6cfba/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6406ed3f").then(t)}},{path:"/pages/e6cfba/index.html",redirect:"/pages/e6cfba/"},{path:"/100./02. - /02.-.html",redirect:"/pages/e6cfba/"},{name:"v-4a93e361",path:"/pages/25bbfb/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-4a93e361").then(t)}},{path:"/pages/25bbfb/index.html",redirect:"/pages/25bbfb/"},{path:"/100./02. - /07.-.html",redirect:"/pages/25bbfb/"},{name:"v-7c712705",path:"/pages/50efc7/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7c712705").then(t)}},{path:"/pages/50efc7/index.html",redirect:"/pages/50efc7/"},{path:"/100./02. - /05.-Markdown.html",redirect:"/pages/50efc7/"},{name:"v-0701bf55",path:"/pages/30653b/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0701bf55").then(t)}},{path:"/pages/30653b/index.html",redirect:"/pages/30653b/"},{path:"/99./02..html",redirect:"/pages/30653b/"},{name:"v-7931d701",path:"/pages/c4fda3/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-7931d701").then(t)}},{path:"/pages/c4fda3/index.html",redirect:"/pages/c4fda3/"},{path:"/100./02. - /06.-.html",redirect:"/pages/c4fda3/"},{name:"v-0b9c81ee",path:"/message-area/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0b9c81ee").then(t)}},{path:"/message-area/index.html",redirect:"/message-area/"},{path:"/101. - .html",redirect:"/message-area/"},{name:"v-8d2263ae",path:"/pages/aef061/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-8d2263ae").then(t)}},{path:"/pages/aef061/index.html",redirect:"/pages/aef061/"},{path:"/99./03./002..html",redirect:"/pages/aef061/"},{name:"v-14a60cb5",path:"/pages/46a15a/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-14a60cb5").then(t)}},{path:"/pages/46a15a/index.html",redirect:"/pages/46a15a/"},{path:"/99./03./001..html",redirect:"/pages/46a15a/"},{name:"v-e022c6c4",path:"/pages/eccd67/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-e022c6c4").then(t)}},{path:"/pages/eccd67/index.html",redirect:"/pages/eccd67/"},{path:"/99./01..html",redirect:"/pages/eccd67/"},{name:"v-6ea3592e",path:"/tags/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-6ea3592e").then(t)}},{path:"/tags/index.html",redirect:"/tags/"},{path:"/@pages/tagsPage.html",redirect:"/tags/"},{name:"v-0d4bbaae",path:"/archives/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-0d4bbaae").then(t)}},{path:"/archives/index.html",redirect:"/archives/"},{path:"/@pages/archivesPage.html",redirect:"/archives/"},{name:"v-22f1f98e",path:"/categories/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-22f1f98e").then(t)}},{path:"/categories/index.html",redirect:"/categories/"},{path:"/@pages/categoriesPage.html",redirect:"/categories/"},{name:"v-42ca3e9a",path:"/",component:ct,beforeEnter:(n,e,t)=>{ue("Layout","v-42ca3e9a").then(t)}},{path:"/index.html",redirect:"/"},{path:"*",component:ct}],dt={title:"",description:"",base:"/",headTags:[["link",{rel:"shortcut icon",href:"/img/favicon.ico"}],["link",{rel:"stylesheet",href:"//at.alicdn.com/t/font_3114978_qe0b39no76.css"}],["meta",{name:"referrer",content:"no-referrer-when-downgrade"}],["noscript",{},'<meta http-equiv="refresh" content="0; url=https://www.youngkbt.cn/noscript/"><style>.theme-vdoing-content { display:none }'],["meta",{name:"keywords",content:"TigerZ, VuePress"}],["meta",{name:"theme-color",content:"#11a8cd"}],["script",{src:"https://cdn.staticfile.org/twikoo/1.6.7/twikoo.all.min.js"}],["script",{},'var _hmt = _hmt || [];\n          (function() {\n            var hm = document.createElement("script");\n            hm.src = "https://hm.baidu.com/hm.js?267c5680c2ffb468ca29c45ffe6801da"; \n            var s = document.getElementsByTagName("script")[0];\n            s.parentNode.insertBefore(hm, s);\n          })();\n          '],["link",{rel:"stylesheet",href:"https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css"}],["link",{rel:"stylesheet",href:"https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/2.10.0/github-markdown.min.css"}],["meta",{name:"baidu-site-verification",content:"codeva-nuy0c96SLu"}]],pages:[{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"01."}},title:"",date:"2023-09-03T16:57:24.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/585994/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/01.%E7%A7%91%E7%A0%94%E6%8A%80%E8%83%BD%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E7%A7%91%E7%A0%94%E6%8A%80%E8%83%BD.html",relativePath:"00./01. - /00..md",key:"v-718147da",path:"/pages/585994/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"",frontmatter:{title:"",date:"2023-06-15T19:19:37.000Z",permalink:"/pages/4c56e7/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E6%B1%87%E6%80%BB.html",relativePath:"00./02. - /02..md",key:"v-8bb86ff6",path:"/pages/4c56e7/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:3,title:"",slug:"-",normalizedTitle:"",charIndex:12},{level:3,title:"",slug:"-",normalizedTitle:"",charIndex:1234},{level:3,title:"",slug:"-",normalizedTitle:"",charIndex:12119},{level:3,title:"",slug:"-",normalizedTitle:"",charIndex:13581},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:15473},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:24e3},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:24632}],headersStr:"       ",content:"# \n\n\n# \n\n(2012.12.20, mPFC , Neuron) The Role of Medial Prefrontal Cortex in Memory and Decision Making\n\n(2018.06.09, , Curr Opin Neurobiol) Transcriptional control of long-range cortical projections\n\n\n(2018.11.16, , Trends Cogn Sci) Bridging the Gap between Connectome and Transcriptome\n\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://www.cell.com/cms/attachment/a5ce8700-14ac-4ec4-813c-18c15c0e9e36/gr3_lrg.jpg\n    link: /pages/1aae84/\n    name: ''\n    desc: <strong>(2012.12.20, mPFC , Neuron)</strong> The Role of Medial Prefrontal Cortex in Memory and Decision Making\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S095943881830076X-gr1_lrg.jpg\n    link: /pages/7100c2/\n    name: ''\n    desc: <strong>(2018.06.09, , Curr Opin Neurobiol)</strong> Transcriptional control of long-range cortical projections <br>\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S1364661318302535-gr1b1_lrg.jpg\n    link: /pages/9e1c62/\n    name: ''\n    desc: <strong>(2018.11.16, , Trends Cogn Sci)</strong> Bridging the Gap between Connectome and Transcriptome <br>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n# \n\n(2014.04.02, Allen, Nature) A mesoscale connectome of the mouse brain\n\n\n\n\n(2016.07.18, MAPseq, Neuron) High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA\n\n(2018.03.28,  MAPseq, Nature) The logic of single-cell projections from visual cortex\n\n\n\n(2018.10.31, PT-thalamus-medulla, Nature) Distinct descending motor cortex pathways and their roles in movement\n\n(2018.10.31, ALM_VISp, Nature) Shared and distinct transcriptomic cell types across neocortical areas\n\n\n\n(2019.02.08, , Mol Neurodegeneration) Differences in neurotropism and neurotoxicity among retrograde viral tracers\n\n\n\n(2019.10.17, BAR-seq, Cell) High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing\n\n(2020.07.09, BRICseq, Cell) BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals\n\n(2020.07.22, , Nature) Distinct subnetworks of the thalamic reticular nucleus\n\n\n\n(2020.09.14, MAPseq, Nat Neurosci) Circuit and molecular architecture of a ventral hippocampal network\n\n\n\n(2020.12.14, SCM, PNAS) Uncovering the genetic blueprint of the C. elegans nervous system\n\n\n\n(2020.12.17, PFC, Cell) Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks\n\n(2021.05.10, BARseq2, Nat Neurosci) Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections\n\n(2021.10.06, epi-retro-seq, Nature) Epigenomic diversity of cortical projection neurons in the mouse brain\n\n(2021.10.06, MOp-ul, Nature) Cellular anatomy of the mouse primary motor cortex\n\n\n\n(2021.12.21, VECTORseq, Cell Rep) Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neuron involved in sensorimotor integration\n\n(2022.03.31, fMOST, Nat Neurosci) Single-neuron projectome of mouse prefrontal cortex\n\n\n\n(2022.10.27, MAPseq, Cell) High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex\n\n(2023.5.22, fMOST, Nat Neurosci) Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n(2023.10.18, MAPseq macaque, Neuron) Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex\n\n(2023.12.13, epi-retro-seq, Nature) Brain-wide correspondence of neuronal epigenomics and distant projections\n\n(2024.02.02, fmost, Science) Whole-brain spatial organization of hippocampal single-neuron projectomes\n\n(2024.02.23, MERGE-seq, eLife) High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labelling\n\n(2024.06.10, , eLife) Deciphering the genetic code of neuronal type connectivity through bilinear modeling\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature13186/MediaObjects/41586_2014_Article_BFnature13186_Fig1_HTML.jpg?\n    link: /pages/7ffbf6/\n    name: ''\n    desc: <strong>(2014.04.02, Allen, Nature)</strong> A mesoscale connectome of the mouse brain <br><br><br>\n\n  - img: https://www.cell.com/cms/attachment/f00a0a0f-e552-497f-917a-582f5fc4b8d6/gr1.jpg\n    link: /pages/dc1352/\n    name: ''\n    desc: <strong>(2016.07.18, MAPseq, Neuron)</strong> High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnature26159/MediaObjects/41586_2018_Article_BFnature26159_Fig1_HTML.jpg?\n    link: /pages/8cb60c/\n    name: ''\n    desc: <strong>(2018.03.28,  MAPseq, Nature)</strong> The logic of single-cell projections from  visual cortex <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-018-0642-9/MediaObjects/41586_2018_642_Fig1_HTML.png?\n    link: /pages/986ab0/\n    name: ''\n    desc: <strong>(2018.10.31, PT-thalamus-medulla, Nature)</strong> Distinct descending motor cortex pathways and their roles in movement\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-018-0654-5/MediaObjects/41586_2018_654_Fig1_HTML.png?as=webp\n    link: /pages/af94c8/\n    name: ''\n    desc: <strong>(2018.10.31, ALM_VISp, Nature)</strong> Shared and distinct transcriptomic cell types across neocortical areas <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13024-019-0308-6/MediaObjects/13024_2019_308_Fig5_HTML.png?\n    link: /pages/037be6/\n    name: ''\n    desc: <strong>(2019.02.08, , Mol Neurodegeneration)</strong> Differences in neurotropism and neurotoxicity among retrograde viral tracers <br><br>\n\n  - img: https://www.cell.com/cms/attachment/1b78fa3c-f355-43de-a457-7cabdd45b7da/gr1_lrg.jpg\n    link: /pages/da828f/\n    name: ''\n    desc: <strong>(2019.10.17, BAR-seq, Cell)</strong> High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S0092867420306243-gr1_lrg.jpg\n    link: /pages/afda58/\n    name: ''\n    desc: <strong>(2020.07.09, BRICseq, Cell)</strong> BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-020-2504-5/MediaObjects/41586_2020_2504_Fig1_HTML.png?as=webp\n    link: /pages/4918af/\n    name: ''\n    desc: <strong>(2020.07.22, , Nature)</strong> Distinct subnetworks of the thalamic reticular nucleus <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-020-0705-8/MediaObjects/41593_2020_705_Fig1_HTML.png?as=webp\n    link: /pages/fa4bef/\n    name: ''\n    desc: <strong>(2020.09.14, MAPseq, Nat Neurosci)</strong> Circuit and molecular architecture of a ventral hippocampal network <br><br>\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.2009093117/asset/7a047265-e84e-4df8-98e6-ea2726bd944b/assets/images/large/pnas.2009093117fig01.jpg\n    link: /pages/521e00/\n    name: ''\n    desc: <strong>(2020.12.14, SCM, PNAS)</strong> Uncovering the genetic blueprint of the C. elegans nervous system <br><br>\n\n  - img: https://www.cell.com/cms/attachment/e3db9ab7-51be-4cb2-bf8f-39ddbd6eed69/gr1_lrg.jpg\n    link: /pages/5c79f3/\n    name: ''\n    desc: <strong>(2020.12.17, PFC, Cell)</strong> Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-021-00842-4/MediaObjects/41593_2021_842_Fig1_HTML.png?as=webp\n    link: /pages/084a0b/\n    name: ''\n    desc: <strong>(2021.05.10, BARseq2, Nat Neurosci)</strong> Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-021-03223-w/MediaObjects/41586_2021_3223_Fig1_HTML.png?as=webp\n    link: /pages/ccc053/\n    name: ''\n    desc: <strong>(2021.10.06, epi-retro-seq, Nature)</strong> Epigenomic diversity of cortical projection neurons in the mouse brain\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-021-03970-w/MediaObjects/41586_2021_3970_Fig6_HTML.png?as=webp\n    link: /pages/d5ec9f/\n    name: ''\n    desc: <strong>(2021.10.06, MOp-ul, Nature)</strong> Cellular anatomy of the mouse primary motor cortex <br><br>\n\n  - img: https://www.cell.com/cms/attachment/e23ba31d-602d-4b6e-981b-a1bb45f84c69/gr1_lrg.jpg\n    link: /pages/40cb7b/\n    name: ''\n    desc: <strong>(2021.12.21, VECTORseq, Cell Rep)</strong> Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neuron involved in sensorimotor integration\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-022-01041-5/MediaObjects/41593_2022_1041_Fig1_HTML.png?as=webp\n    link: /pages/0be7e3/\n    name: ''\n    desc: <strong>(2022.03.31, fMOST, Nat Neurosci)</strong> Single-neuron projectome of mouse prefrontal cortex <br><br>\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S0092867422012570-gr1_lrg.jpg\n    link: /pages/94c1e9/\n    name: ''\n    desc: <strong>(2022.10.27, MAPseq, Cell)</strong> High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-023-01339-y/MediaObjects/41593_2023_1339_Fig1_HTML.png?as=webp\n    link: /pages/e528dd/\n    name: ''\n    desc: <strong>(2023.5.22, fMOST, Nat Neurosci)</strong> Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S0896627323007110-gr1_lrg.jpg\n    link: /pages/aaea7b/\n    name: ''\n    desc: <strong>(2023.10.18, MAPseq macaque, Neuron)</strong> Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-023-06823-w/MediaObjects/41586_2023_6823_Fig1_HTML.png?as=webp\n    link: /pages/b405a1/\n    name: ''\n    desc: <strong>(2023.12.13, epi-retro-seq, Nature)</strong> Brain-wide correspondence of neuronal epigenomics and distant projections\n\n  - img: https://www.science.org/cms/10.1126/science.adj9198/asset/baad16a7-af44-444f-826f-6d212ca1d933/assets/images/large/science.adj9198-fa.jpg\n    link: /pages/f20dad/\n    name: ''\n    desc: <strong>(2024.02.02, fmost, Science)</strong> Whole-brain spatial organization of hippocampal single-neuron projectomes\n\n  - img: https://iiif.elifesciences.org/lax:85419%2Felife-85419-fig1-v2.tif/full/1500,/0/default.jpg\n    link: /pages/723a6a/\n    name: ''\n    desc: <strong>(2024.02.23, MERGE-seq, eLife)</strong> High-throughput mapping of single-neuron projection and molecular features by  retrograde barcoded labelling\n\n  - img: https://iiif.elifesciences.org/lax:91532%2Felife-91532-fig1-v1.tif/full/,1500/0/default.jpg\n    link: /pages/9c9ddc/\n    name: ''\n    desc: <strong>(2024.06.10, , eLife)</strong> Deciphering the genetic code of neuronal type connectivity through bilinear modeling\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n\n\n\n# \n\n(2022.01.20, , Cell) Vision-dependent specification of cell types and function in the developing cortex\n\n\n\n(2022.11.10, PFC, Cell) Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution\n\n(2022.12.28, MERFISH, Cell) Molecular and spatial signatures of mouse brain aging at single-cell resolution\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://www.cell.com/cms/10.1016/j.cell.2021.12.022/asset/d3fddf2d-2a9b-4753-a6ab-0384c1b0eee5/main.assets/gr1_lrg.jpg\n    link: /pages/b79c35/\n    name: ''\n    desc: <strong>(2022.01.20, , Cell)</strong> Vision-dependent specification of cell types and function in the developing cortex <br><br>\n\n  - img: https://www.cell.com/cms/10.1016/j.cell.2022.09.039/asset/d4ce23e7-241e-42f5-9b38-3c2ed9ad42f1/main.assets/gr1_lrg.jpg\n    link: /pages/0b119c/\n    name: ''\n    desc: <strong>(2022.11.10, PFC, Cell)</strong> Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution\n\n  - img: https://www.cell.com/cms/10.1016/j.cell.2022.12.010/asset/cc83b730-e3b0-4d13-9906-c5675fc39143/main.assets/gr1_lrg.jpg\n    link: /pages/16bdaa/\n    name: ''\n    desc: <strong>(2022.12.28, MERFISH, Cell)</strong> Molecular and spatial signatures of mouse brain aging at single-cell resolution \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n# \n\n(2016.06.28, Div-Seq, Science) Div-Seq:Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons\n\n(2016.08.16, , PNAS) Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons\n\n(2017.10.11, Act-seq, Neuron) Detecting Activated Cell Populations Using Single-Cell RNA-Seq\n\n\n\n(2019.09.13, mPFC, Nat Commun) Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction\n\n(2019.11.11, , Elife) Single-cell transcriptomic evidence for dense intracortical neuropeptide networks\n\n(2020.01.15, , Nature) Decoding the development of the human hippocampus\n\n\n\n(2020.06.26, , Sci. Adv.) Molecular atlas of the adult mouse brain\n\n\n\n(2020.12.07, , Nat Neurosci) Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing\n\n(2021.06.23, , Nature) Molecular logic of cellular diversification in the mouse cerebral cortex\n\n\n\n(2021.07.08, hdWGCNA, Nat Genet) Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n\n(2022.05.30, , Nat Neurosci) Single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations\n\n(2022.06.21, hPFC, PNAS) The neuropeptide landscape of human prefrontal cortex\n\n\n\n(2023.07.12, stereo-seq, Cell) Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex\n\n(2023.10.16,  PFC MERFISH, Nat Neurosci) Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain\n\n(2023.11.01, , Sci. Adv.) A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types\n\n(2023.11.15, , Neuron) The neuropeptidergic connectome of C. elegans\n\n\n\n(2023.12.13, Allen, Nature) A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain\n\n(2024.04.24, BARseq, Nature) Whole-cortex in situ sequencing reveals input-dependent area identity\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://www.science.org/cms/10.1126/science.aad7038/asset/98421af6-a2f9-4063-9262-c6d1a0b82a3e/assets/graphic/353_925_f1.jpeg\n    link: /pages/9b041f/\n    name: ''\n    desc: <strong>(2016.06.28, Div-Seq, Science)</strong> Div-Seq:Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.1610155113/asset/a7875690-c8f8-405a-a22b-f6498cf81933/assets/graphic/pnas.1610155113fig01.jpeg\n    link: /pages/a61efc/\n    name: ''\n    desc: <strong>(2016.08.16, , PNAS)</strong> Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons\n\n  - img: https://www.cell.com/cms/attachment/a8ffa9eb-0275-4f7e-94b0-ebec14a16894/gr1_lrg.jpg\n    link: /pages/8bf2bf/\n    name: ''\n    desc: <strong>(2017.10.11, Act-seq, Neuron)</strong> Detecting Activated Cell Populations Using Single-Cell RNA-Seq <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-019-12054-3/MediaObjects/41467_2019_12054_Fig1_HTML.png?as=webp\n    link: /pages/c60442/\n    name: ''\n    desc: <strong>(2019.09.13, mPFC, Nat Commun)</strong> Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction\n\n  - img: https://iiif.elifesciences.org/lax/47889%2Felife-47889-fig5-v2.tif/full/,1500/0/default.jpg\n    link: /pages/bb2e69/\n    name: ''\n    desc: <strong>(2019.11.11, , Elife)</strong> Single-cell transcriptomic evidence for dense intracortical neuropeptide networks\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-019-1917-5/MediaObjects/41586_2019_1917_Fig1_HTML.png?as=webp\n    link: /pages/0b79fa/\n    name: ''\n    desc: <strong>(2020.01.15, , Nature)</strong> Decoding the development of the human hippocampus <br><br>\n\n  - img: https://www.science.org/cms/10.1126/sciadv.abb3446/asset/0321fe52-76d5-401f-9b72-86e473a55a56/assets/graphic/abb3446-f1.jpeg\n    link: /pages/37a825/\n    name: ''\n    desc: <strong>(2020.06.26, , Sci. Adv.)</strong> Molecular atlas of the adult mouse brain <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-020-00736-x/MediaObjects/41593_2020_736_Fig3_HTML.png?as=webp\n    link: /pages/b5aaa4/\n    name: ''\n    desc: <strong>(2020.12.07, , Nat Neurosci)</strong> Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing\n\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-021-03670-5/MediaObjects/41586_2021_3670_Fig1_HTML.png?as=webp\n    link: /pages/8a50f3/\n    name: ''\n    desc: <strong>(2021.06.23, , Nature)</strong> Molecular logic of cellular diversification in the mouse cerebral cortex <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-021-00894-z/MediaObjects/41588_2021_894_Fig1_HTML.png?as=webp\n    link: /pages/25c02b/\n    name: ''\n    desc: <strong>(2021.07.08, hdWGCNA, Nat Genet)</strong> Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-022-01073-x/MediaObjects/41593_2022_1073_Fig1_HTML.png?as=webp\n    link: /pages/f481a1/\n    name: ''\n    desc: <strong>(2022.05.30, , Nat Neurosci)</strong> Single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.2123146119/asset/13b94f27-0870-474b-b255-e1bc569c06cc/assets/images/large/pnas.2123146119fig01.jpg\n    link: /pages/444fce/\n    name: ''\n    desc: <strong>(2022.06.21, hPFC, PNAS)</strong> The neuropeptide landscape of human prefrontal cortex <br><br>\n\n  - img: https://www.cell.com/cms/attachment/2ba65490-ca8d-4413-8652-ef034dbf5db6/gr1_lrg.jpg\n    link: /pages/8c91b0/\n    name: ''\n    desc: <strong>(2023.07.12, stereo-seq, Cell)</strong> Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-023-01455-9/MediaObjects/41593_2023_1455_Fig1_HTML.png?as=webp\n    link: /pages/42c71a/\n    name: ''\n    desc: <strong>(2023.10.16,  PFC MERFISH, Nat Neurosci)</strong> Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain\n\n  - img: https://www.science.org/cms/10.1126/sciadv.adh7693/asset/f2f443e5-7123-41e4-8dc8-586440200643/assets/images/large/sciadv.adh7693-f1.jpg\n    link: /pages/12b484/\n    name: ''\n    desc: <strong>(2023.11.01, , Sci. Adv.)</strong> A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types\n\n  - img: https://www.cell.com/cms/attachment/94939603-4f28-451e-a53f-4e9814ce0c3b/fx1_lrg.jpg\n    link: /pages/a1ce18/\n    name: ''\n    desc: <strong>(2023.11.15, , Neuron)</strong> The neuropeptidergic connectome of C. elegans <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41586-023-06812-z/MediaObjects/41586_2023_6812_Fig1_HTML.png?as=webp\n    link: /pages/895303/\n    name: ''\n    desc: <strong>(2023.12.13, Allen, Nature)</strong> A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41586-024-07221-6/MediaObjects/41586_2024_7221_Fig1_HTML.png?\n    link: /pages/666cd3/\n    name: ''\n    desc: <strong>(2024.04.24, BARseq, Nature)</strong> Whole-cortex in situ sequencing reveals input-dependent area identity\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n\n\n\n# \n\n(2022.02.20, CRD, J Pineal Res) Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma\n\n(2023.01.27, -FISH rainbow, Nat Commun) Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules\n\n(2024.05.14, MiP-seq, Nat. Biomed. Eng) Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing\n\n(2024.07.04, benchmark, Nat Methods) Systematic comparison of sequencing-based spatial transcriptomic methods\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://onlinelibrary.wiley.com/cms/asset/0c3d6310-66d6-45a2-bea0-3fae76351e68/jpi12803-fig-0001-m.jpg\n    link: /pages/3eedee/\n    name: ''\n    desc: <strong>(2022.02.20, CRD, J Pineal Res)</strong> Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-36137-4/MediaObjects/41467_2023_36137_Fig2_HTML.png?as=webp\n    link: /pages/c40bc0/\n    name: ''\n    desc: <strong>(2023.01.27, -FISH rainbow, Nat Commun)</strong> Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41551-024-01205-7/MediaObjects/41551_2024_1205_Fig2_HTML.png?as=webp\n    link: /pages/e6566f/\n    name: ''\n    desc: <strong>(2024.05.14, MiP-seq, Nat. Biomed. Eng)</strong> Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-024-02325-3/MediaObjects/41592_2024_2325_Fig1_HTML.png?as=webp\n    link: /pages/b55e03/\n    name: ''\n    desc: <strong>(2024.07.04, benchmark, Nat Methods)</strong> Systematic comparison of sequencing-based spatial transcriptomic methods\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n\n\n\n# \n\n(2018.04.26, DLO Hi-C, Nat Genet) Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41588-018-0111-2/MediaObjects/41588_2018_111_Fig1_HTML.jpg?as=webp\n    link: /pages/76fbba/\n    name: ''\n    desc: <strong>(2018.04.26, DLO Hi-C, Nat Genet)</strong> Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\n(2008.12.28, WGCNA, BMC Bioinformatics) WGCNA an R package for weighted correlation network analysis\n\n(2015.04.13, Seurat V1, Nat Biotechnol) Spatial reconstruction of single-cell gene expression data\n\n\n\n(2017.12.04, WholeBrain, Nat Neurosci) An interactive framework for whole-brain maps at cellular resolution\n\n\n\n(2018.02.06, scanpy, Genome Biol) SCANPYlarge-scale single-cell gene expression data analysis\n\n\n\n(2018.04.02, Seurat V2, Nat Biotechnol) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n\n(2019.04.03, DoubletFinder, Cell Systems) DoubletFinder Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors\n\n(2019.06.06, Seurat V3, Cell) Comprehensive Integration of Single-Cell Data\n\n\n\n(2020.09.14, scRNAseq imputation, Genome Biol) A systematic evaluation of single-cell RNA-sequencing imputation methods\n\n(2020.12.07, nnU-Net, Nat Methods) nnU-Net:a self-configuring method for deep learning-based biomedical image segmentation\n\n(2020.12.10, GCNG, Genome Biol) GCNGgraph convolutional networks for inferring gene interaction from spatial transcriptomics data\n\n(2020.12.14, Cellpose, Nat Methods) Cellpose:a generalist algorithm for cellular segmentation\n\n\n\n(2021.02.06, COVID-19 CT, Medical Physics) Toward data-efficient learning:A benchmark for COVID-19 CT lung and infection segmentation\n\n(2021.02.17, CellChat, Nat Commun) Inference and analysis of cell-cell communication using CellChat\n\n(2021.06.24, Seurat V4, Cell) Integrated analysis of multimodal single-cell data\n\n\n\n(2021.07.01, LASSO, Front. Genet) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease\n\n(2021.10.28, Tangram, Nat Methods) Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram\n\n(2022.01.13, Cell2location, Nat Biotechnol) Cell2location maps fine-grained cell types in spatial transcriptomics\n\n(2022.01.31, Squidpy, Nat Methods) Squidpy:a scalable framework for spatial omics analysis\n\n\n\n(2022.05.11, Viv, Nat Methods) Squidpy:a scalable framework for spatial omics analysis\n\n\n\n(2022.09.30, SpaceX, Bioinformatics) SpaceXgene co-expression network estimation for spatial transcriptomics\n\n(2023.02.15, NeuronChat, Nat Commun) Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat\n\n(2023.03.25, Seurat V5, Nat Biotechnol) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n(2023.04.10, scverse, Nat Biotechnol) The scverse project provides a computational ecosystem for single-cell omics data analysis\n\n(2023.06.12, hdWGCNA, Cell Reports Methods) hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n\n(2023.11.22, SPACEL, Nat Commun) SPACELcharacterizing spatial transcriptome architectures by deep-learning\n\n(2024.06.14, tidyomics, Nat Methods) The tidyomics ecosystemenhancing omic data analyses\n\n(2024.07.03, ezSingleCell, Nat Commun) ezSingleCellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists\n\nconfig:\n    target: _blank\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 5\n\ndata:\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-9-559/MediaObjects/12859_2008_Article_2544_Fig1_HTML.jpg?as=webp\n    link: /pages/ffdcc0/\n    name: ''\n    desc: <strong>(2008.12.28, WGCNA, BMC Bioinformatics)</strong> WGCNA an R package for weighted correlation network analysis\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnbt.3192/MediaObjects/41587_2015_Article_BFnbt3192_Fig1_HTML.jpg?as=webp\n    link: /pages/0d1fce/\n    name: ''\n    desc: <strong>(2015.04.13, Seurat V1, Nat Biotechnol)</strong> Spatial reconstruction of single-cell gene expression data <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-017-0027-7/MediaObjects/41593_2017_27_Fig1_HTML.jpg?as=webp\n    link: /pages/e020de/\n    name: ''\n    desc: <strong>(2017.12.04, WholeBrain, Nat Neurosci)</strong> An interactive framework for whole-brain maps at cellular resolution <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-017-1382-0/MediaObjects/13059_2017_1382_Fig1_HTML.gif?as=webp\n    link: /pages/ebb632/\n    name: ''\n    desc: <strong>(2018.02.06, scanpy, Genome Biol)</strong> SCANPYlarge-scale single-cell gene expression data analysis <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnbt.4096/MediaObjects/41587_2018_Article_BFnbt4096_Fig1_HTML.jpg?as=webp\n    link: /pages/f37280/\n    name: ''\n    desc: <strong>(2018.04.02, Seurat V2, Nat Biotechnol)</strong> Integrating single-cell transcriptomic data across different conditions, technologies, and species\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S2405471219300730-fx1_lrg.jpg\n    link: /pages/71b694/\n    name: ''\n    desc: <strong>(2019.04.03, DoubletFinder, Cell Systems)</strong> DoubletFinder Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S0092867419305598-fx1_lrg.jpg\n    link: /pages/d26f9e/\n    name: ''\n    desc: <strong>(2019.06.06, Seurat V3, Cell)</strong> Comprehensive Integration of Single-Cell Data <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fs13059-020-02132-x/MediaObjects/13059_2020_2132_Fig1_HTML.png?as=webp\n    link: /pages/c7adb0/\n    name: ''\n    desc: <strong>(2020.09.14, scRNAseq imputation, Genome Biol)</strong> A systematic evaluation of single-cell RNA-sequencing imputation methods\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-020-01008-z/MediaObjects/41592_2020_1008_Fig1_HTML.png?as=webp\n    link: /pages/d38252/\n    name: ''\n    desc: <strong>(2020.12.07, nnU-Net, Nat Methods)</strong> nnU-Net:a self-configuring method for deep learning-based biomedical image segmentation\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13059-020-02214-w/MediaObjects/13059_2020_2214_Fig1_HTML.png?as=webp\n    link: /pages/c34d2c/\n    name: ''\n    desc: <strong>(2020.12.10, GCNG, Genome Biol)</strong> GCNGgraph convolutional networks for inferring gene interaction from spatial transcriptomics data\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-020-01018-x/MediaObjects/41592_2020_1018_Fig1_HTML.png?as=webp\n    link: /pages/1f3ea0/\n    name: ''\n    desc: <strong>(2020.12.14, Cellpose, Nat Methods)</strong> Cellpose:a generalist algorithm for cellular segmentation <br><br>\n\n  - img: https://raw.githubusercontent.com/JunMa11/COVID-19-CT-Seg-Benchmark/master/utils/ImageExamples.png\n    link: /pages/1c1b2b/\n    name: ''\n    desc: <strong>(2021.02.06, COVID-19 CT, Medical Physics)</strong> Toward data-efficient learning:A benchmark for COVID-19 CT lung and infection segmentation\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-021-21246-9/MediaObjects/41467_2021_21246_Fig1_HTML.png?as=webp\n    link: /pages/2737d2/\n    name: ''\n    desc: <strong>(2021.02.17, CellChat, Nat Commun)</strong> Inference and analysis of cell-cell communication using CellChat\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-S0092867421005833-fx1_lrg.jpg\n    link: /pages/b4dfe0/\n    name: ''\n    desc: <strong>(2021.06.24, Seurat V4, Cell)</strong> Integrated analysis of multimodal single-cell data <br><br>\n\n  - img: https://www.frontiersin.org/files/Articles/658323/fgene-12-658323-HTML/image_m/fgene-12-658323-g001.jpg\n    link: /pages/6a4bcc/\n    name: ''\n    desc: <strong>(2021.07.01, LASSO, Front. Genet)</strong> Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-021-01264-7/MediaObjects/41592_2021_1264_Fig1_HTML.png?as=webp\n    link: /pages/c3bbd9/\n    name: ''\n    desc: <strong>(2021.10.28, Tangram, Nat Methods)</strong> Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41587-021-01139-4/MediaObjects/41587_2021_1139_Fig1_HTML.png?as=webp\n    link: /pages/9912cc/\n    name: ''\n    desc: <strong>(2022.01.13, Cell2location, Nat Biotechnol)</strong> Cell2location maps fine-grained cell types in spatial transcriptomics\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41592-021-01358-2/MediaObjects/41592_2021_1358_Fig1_HTML.png?as=webp\n    link: /pages/76476f/\n    name: ''\n    desc: <strong>(2022.01.31, Squidpy, Nat Methods)</strong> Squidpy:a scalable framework for spatial omics analysis <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-022-01482-7/MediaObjects/41592_2022_1482_Fig1_HTML.png?as=webp\n    link: /pages/112d0b/\n    name: ''\n    desc: <strong>(2022.05.11, Viv, Nat Methods)</strong> Squidpy:a scalable framework for spatial omics analysis <br><br>\n\n  - img: https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/38/22/10.1093_bioinformatics_btac645/3/btac645f1.jpeg?Expires=1725068870&Signature=gkOqi32igJEe3TntPWroK01XeDunvd9RCeCITtiwpCbW5ZWdu3~inp1AWlImoug9RMbLyqZtwPCfbUiLfr8QF2yFfky9QCC6Po0EUkLrpge5mm55yPLOW8fG2HkAK-BW8qHV9ppISJSC0eoqwH6aYFcgivjnUn4h3~RJku3luJPs3OaXqhtocIsey0pyhKccgcKoRaJ2qkvRomju55m7qGszpR6rz99Ke7rA0Kcq8Y5Vhba-7YD0EB8RX7Jun2UsNdsd2Uo7IeVZYXHutvq1vmuYX-3fq1040Nmgl4SwjAThoCiMte-ZeSUTWtv-qAGkNtRjALxlNP5Emk0Hix5hDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA\n    link: /pages/4cba92/\n    name: ''\n    desc: <strong>(2022.09.30, SpaceX, Bioinformatics)</strong> SpaceXgene co-expression network estimation for spatial transcriptomics \n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41467-023-36800-w/MediaObjects/41467_2023_36800_Fig1_HTML.png?as=webp\n    link: /pages/794f06/\n    name: ''\n    desc: <strong>(2023.02.15, NeuronChat, Nat Commun)</strong> Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41587-023-01767-y/MediaObjects/41587_2023_1767_Fig1_HTML.png?as=webp\n    link: /pages/8c5839/\n    name: ''\n    desc: <strong>(2023.03.25, Seurat V5, Nat Biotechnol)</strong> Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41587-023-01733-8/MediaObjects/41587_2023_1733_Fig1_HTML.png?as=webp\n    link: /pages/124edd/\n    name: ''\n    desc: <strong>(2023.04.10, scverse, Nat Biotechnol)</strong> The scverse project provides a computational ecosystem for single-cell omics data analysis\n\n  - img: https://www.cell.com/cms/attachment/2f4fe3cb-81a1-4461-ae56-05fd516d7822/fx1_lrg.jpg\n    link: /pages/a24249/\n    name: ''\n    desc: <strong>(2023.06.12, hdWGCNA, Cell Reports Methods)</strong> hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-023-43220-3/MediaObjects/41467_2023_43220_Fig1_HTML.png?as=webp\n    link: /pages/da0fb5/\n    name: ''\n    desc: <strong>(2023.11.22, SPACEL, Nat Commun)</strong> SPACELcharacterizing spatial transcriptome architectures by deep-learning\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41592-024-02299-2/MediaObjects/41592_2024_2299_Fig1_HTML.png?as=webp\n    link: /pages/32a729/\n    name: ''\n    desc: <strong>(2024.06.14, tidyomics, Nat Methods)</strong> The tidyomics ecosystemenhancing omic data analyses\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-48188-2/MediaObjects/41467_2024_48188_Fig1_HTML.png?as=webp\n    link: /pages/02ef79/\n    name: ''\n    desc: <strong>(2024.07.03, ezSingleCell, Nat Commun)</strong> ezSingleCellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n",normalizedContent:"# \n\n\n# \n\n(2012.12.20, mpfc , neuron) the role of medial prefrontal cortex in memory and decision making\n\n(2018.06.09, , curr opin neurobiol) transcriptional control of long-range cortical projections\n\n\n(2018.11.16, , trends cogn sci) bridging the gap between connectome and transcriptome\n\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://www.cell.com/cms/attachment/a5ce8700-14ac-4ec4-813c-18c15c0e9e36/gr3_lrg.jpg\n    link: /pages/1aae84/\n    name: ''\n    desc: <strong>(2012.12.20, mpfc , neuron)</strong> the role of medial prefrontal cortex in memory and decision making\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s095943881830076x-gr1_lrg.jpg\n    link: /pages/7100c2/\n    name: ''\n    desc: <strong>(2018.06.09, , curr opin neurobiol)</strong> transcriptional control of long-range cortical projections <br>\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s1364661318302535-gr1b1_lrg.jpg\n    link: /pages/9e1c62/\n    name: ''\n    desc: <strong>(2018.11.16, , trends cogn sci)</strong> bridging the gap between connectome and transcriptome <br>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n# \n\n(2014.04.02, allen, nature) a mesoscale connectome of the mouse brain\n\n\n\n\n(2016.07.18, mapseq, neuron) high-throughput mapping of single-neuron projections by sequencing of barcoded rna\n\n(2018.03.28,  mapseq, nature) the logic of single-cell projections from visual cortex\n\n\n\n(2018.10.31, pt-thalamus-medulla, nature) distinct descending motor cortex pathways and their roles in movement\n\n(2018.10.31, alm_visp, nature) shared and distinct transcriptomic cell types across neocortical areas\n\n\n\n(2019.02.08, , mol neurodegeneration) differences in neurotropism and neurotoxicity among retrograde viral tracers\n\n\n\n(2019.10.17, bar-seq, cell) high-throughput mapping of long-range neuronal projection using in situ sequencing\n\n(2020.07.09, bricseq, cell) bricseq bridges brain-wide interregional connectivity to neural activity and gene expression in single animals\n\n(2020.07.22, , nature) distinct subnetworks of the thalamic reticular nucleus\n\n\n\n(2020.09.14, mapseq, nat neurosci) circuit and molecular architecture of a ventral hippocampal network\n\n\n\n(2020.12.14, scm, pnas) uncovering the genetic blueprint of the c. elegans nervous system\n\n\n\n(2020.12.17, pfc, cell) differential encoding in prefrontal cortex projection neuron classes across cognitive tasks\n\n(2021.05.10, barseq2, nat neurosci) integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections\n\n(2021.10.06, epi-retro-seq, nature) epigenomic diversity of cortical projection neurons in the mouse brain\n\n(2021.10.06, mop-ul, nature) cellular anatomy of the mouse primary motor cortex\n\n\n\n(2021.12.21, vectorseq, cell rep) virally encoded connectivity transgenic overlay rna sequencing (vectorseq) defines projection neuron involved in sensorimotor integration\n\n(2022.03.31, fmost, nat neurosci) single-neuron projectome of mouse prefrontal cortex\n\n\n\n(2022.10.27, mapseq, cell) high-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex\n\n(2023.5.22, fmost, nat neurosci) single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n(2023.10.18, mapseq macaque, neuron) single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex\n\n(2023.12.13, epi-retro-seq, nature) brain-wide correspondence of neuronal epigenomics and distant projections\n\n(2024.02.02, fmost, science) whole-brain spatial organization of hippocampal single-neuron projectomes\n\n(2024.02.23, merge-seq, elife) high-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labelling\n\n(2024.06.10, , elife) deciphering the genetic code of neuronal type connectivity through bilinear modeling\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fnature13186/mediaobjects/41586_2014_article_bfnature13186_fig1_html.jpg?\n    link: /pages/7ffbf6/\n    name: ''\n    desc: <strong>(2014.04.02, allen, nature)</strong> a mesoscale connectome of the mouse brain <br><br><br>\n\n  - img: https://www.cell.com/cms/attachment/f00a0a0f-e552-497f-917a-582f5fc4b8d6/gr1.jpg\n    link: /pages/dc1352/\n    name: ''\n    desc: <strong>(2016.07.18, mapseq, neuron)</strong> high-throughput mapping of single-neuron projections by sequencing of barcoded rna\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fnature26159/mediaobjects/41586_2018_article_bfnature26159_fig1_html.jpg?\n    link: /pages/8cb60c/\n    name: ''\n    desc: <strong>(2018.03.28,  mapseq, nature)</strong> the logic of single-cell projections from  visual cortex <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41586-018-0642-9/mediaobjects/41586_2018_642_fig1_html.png?\n    link: /pages/986ab0/\n    name: ''\n    desc: <strong>(2018.10.31, pt-thalamus-medulla, nature)</strong> distinct descending motor cortex pathways and their roles in movement\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41586-018-0654-5/mediaobjects/41586_2018_654_fig1_html.png?as=webp\n    link: /pages/af94c8/\n    name: ''\n    desc: <strong>(2018.10.31, alm_visp, nature)</strong> shared and distinct transcriptomic cell types across neocortical areas <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1186%2fs13024-019-0308-6/mediaobjects/13024_2019_308_fig5_html.png?\n    link: /pages/037be6/\n    name: ''\n    desc: <strong>(2019.02.08, , mol neurodegeneration)</strong> differences in neurotropism and neurotoxicity among retrograde viral tracers <br><br>\n\n  - img: https://www.cell.com/cms/attachment/1b78fa3c-f355-43de-a457-7cabdd45b7da/gr1_lrg.jpg\n    link: /pages/da828f/\n    name: ''\n    desc: <strong>(2019.10.17, bar-seq, cell)</strong> high-throughput mapping of long-range neuronal projection using in situ sequencing\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s0092867420306243-gr1_lrg.jpg\n    link: /pages/afda58/\n    name: ''\n    desc: <strong>(2020.07.09, bricseq, cell)</strong> bricseq bridges brain-wide interregional connectivity to neural activity and gene expression in single animals\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-020-2504-5/mediaobjects/41586_2020_2504_fig1_html.png?as=webp\n    link: /pages/4918af/\n    name: ''\n    desc: <strong>(2020.07.22, , nature)</strong> distinct subnetworks of the thalamic reticular nucleus <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-020-0705-8/mediaobjects/41593_2020_705_fig1_html.png?as=webp\n    link: /pages/fa4bef/\n    name: ''\n    desc: <strong>(2020.09.14, mapseq, nat neurosci)</strong> circuit and molecular architecture of a ventral hippocampal network <br><br>\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.2009093117/asset/7a047265-e84e-4df8-98e6-ea2726bd944b/assets/images/large/pnas.2009093117fig01.jpg\n    link: /pages/521e00/\n    name: ''\n    desc: <strong>(2020.12.14, scm, pnas)</strong> uncovering the genetic blueprint of the c. elegans nervous system <br><br>\n\n  - img: https://www.cell.com/cms/attachment/e3db9ab7-51be-4cb2-bf8f-39ddbd6eed69/gr1_lrg.jpg\n    link: /pages/5c79f3/\n    name: ''\n    desc: <strong>(2020.12.17, pfc, cell)</strong> differential encoding in prefrontal cortex projection neuron classes across cognitive tasks\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-021-00842-4/mediaobjects/41593_2021_842_fig1_html.png?as=webp\n    link: /pages/084a0b/\n    name: ''\n    desc: <strong>(2021.05.10, barseq2, nat neurosci)</strong> integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-021-03223-w/mediaobjects/41586_2021_3223_fig1_html.png?as=webp\n    link: /pages/ccc053/\n    name: ''\n    desc: <strong>(2021.10.06, epi-retro-seq, nature)</strong> epigenomic diversity of cortical projection neurons in the mouse brain\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-021-03970-w/mediaobjects/41586_2021_3970_fig6_html.png?as=webp\n    link: /pages/d5ec9f/\n    name: ''\n    desc: <strong>(2021.10.06, mop-ul, nature)</strong> cellular anatomy of the mouse primary motor cortex <br><br>\n\n  - img: https://www.cell.com/cms/attachment/e23ba31d-602d-4b6e-981b-a1bb45f84c69/gr1_lrg.jpg\n    link: /pages/40cb7b/\n    name: ''\n    desc: <strong>(2021.12.21, vectorseq, cell rep)</strong> virally encoded connectivity transgenic overlay rna sequencing (vectorseq) defines projection neuron involved in sensorimotor integration\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-022-01041-5/mediaobjects/41593_2022_1041_fig1_html.png?as=webp\n    link: /pages/0be7e3/\n    name: ''\n    desc: <strong>(2022.03.31, fmost, nat neurosci)</strong> single-neuron projectome of mouse prefrontal cortex <br><br>\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s0092867422012570-gr1_lrg.jpg\n    link: /pages/94c1e9/\n    name: ''\n    desc: <strong>(2022.10.27, mapseq, cell)</strong> high-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-023-01339-y/mediaobjects/41593_2023_1339_fig1_html.png?as=webp\n    link: /pages/e528dd/\n    name: ''\n    desc: <strong>(2023.5.22, fmost, nat neurosci)</strong> single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s0896627323007110-gr1_lrg.jpg\n    link: /pages/aaea7b/\n    name: ''\n    desc: <strong>(2023.10.18, mapseq macaque, neuron)</strong> single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-023-06823-w/mediaobjects/41586_2023_6823_fig1_html.png?as=webp\n    link: /pages/b405a1/\n    name: ''\n    desc: <strong>(2023.12.13, epi-retro-seq, nature)</strong> brain-wide correspondence of neuronal epigenomics and distant projections\n\n  - img: https://www.science.org/cms/10.1126/science.adj9198/asset/baad16a7-af44-444f-826f-6d212ca1d933/assets/images/large/science.adj9198-fa.jpg\n    link: /pages/f20dad/\n    name: ''\n    desc: <strong>(2024.02.02, fmost, science)</strong> whole-brain spatial organization of hippocampal single-neuron projectomes\n\n  - img: https://iiif.elifesciences.org/lax:85419%2felife-85419-fig1-v2.tif/full/1500,/0/default.jpg\n    link: /pages/723a6a/\n    name: ''\n    desc: <strong>(2024.02.23, merge-seq, elife)</strong> high-throughput mapping of single-neuron projection and molecular features by  retrograde barcoded labelling\n\n  - img: https://iiif.elifesciences.org/lax:91532%2felife-91532-fig1-v1.tif/full/,1500/0/default.jpg\n    link: /pages/9c9ddc/\n    name: ''\n    desc: <strong>(2024.06.10, , elife)</strong> deciphering the genetic code of neuronal type connectivity through bilinear modeling\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n\n\n\n# \n\n(2022.01.20, , cell) vision-dependent specification of cell types and function in the developing cortex\n\n\n\n(2022.11.10, pfc, cell) human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution\n\n(2022.12.28, merfish, cell) molecular and spatial signatures of mouse brain aging at single-cell resolution\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://www.cell.com/cms/10.1016/j.cell.2021.12.022/asset/d3fddf2d-2a9b-4753-a6ab-0384c1b0eee5/main.assets/gr1_lrg.jpg\n    link: /pages/b79c35/\n    name: ''\n    desc: <strong>(2022.01.20, , cell)</strong> vision-dependent specification of cell types and function in the developing cortex <br><br>\n\n  - img: https://www.cell.com/cms/10.1016/j.cell.2022.09.039/asset/d4ce23e7-241e-42f5-9b38-3c2ed9ad42f1/main.assets/gr1_lrg.jpg\n    link: /pages/0b119c/\n    name: ''\n    desc: <strong>(2022.11.10, pfc, cell)</strong> human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution\n\n  - img: https://www.cell.com/cms/10.1016/j.cell.2022.12.010/asset/cc83b730-e3b0-4d13-9906-c5675fc39143/main.assets/gr1_lrg.jpg\n    link: /pages/16bdaa/\n    name: ''\n    desc: <strong>(2022.12.28, merfish, cell)</strong> molecular and spatial signatures of mouse brain aging at single-cell resolution \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n# \n\n(2016.06.28, div-seq, science) div-seq:single-nucleus rna-seq reveals dynamics of rare adult newborn neurons\n\n(2016.08.16, , pnas) single-cell rnaseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons\n\n(2017.10.11, act-seq, neuron) detecting activated cell populations using single-cell rna-seq\n\n\n\n(2019.09.13, mpfc, nat commun) cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction\n\n(2019.11.11, , elife) single-cell transcriptomic evidence for dense intracortical neuropeptide networks\n\n(2020.01.15, , nature) decoding the development of the human hippocampus\n\n\n\n(2020.06.26, , sci. adv.) molecular atlas of the adult mouse brain\n\n\n\n(2020.12.07, , nat neurosci) diversification of molecularly defined myenteric neuron classes revealed by single-cell rna sequencing\n\n(2021.06.23, , nature) molecular logic of cellular diversification in the mouse cerebral cortex\n\n\n\n(2021.07.08, hdwgcna, nat genet) single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n\n(2022.05.30, , nat neurosci) single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations\n\n(2022.06.21, hpfc, pnas) the neuropeptide landscape of human prefrontal cortex\n\n\n\n(2023.07.12, stereo-seq, cell) single-cell spatial transcriptome reveals cell-type organization in the macaque cortex\n\n(2023.10.16,  pfc merfish, nat neurosci) spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain\n\n(2023.11.01, , sci. adv.) a telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types\n\n(2023.11.15, , neuron) the neuropeptidergic connectome of c. elegans\n\n\n\n(2023.12.13, allen, nature) a high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain\n\n(2024.04.24, barseq, nature) whole-cortex in situ sequencing reveals input-dependent area identity\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://www.science.org/cms/10.1126/science.aad7038/asset/98421af6-a2f9-4063-9262-c6d1a0b82a3e/assets/graphic/353_925_f1.jpeg\n    link: /pages/9b041f/\n    name: ''\n    desc: <strong>(2016.06.28, div-seq, science)</strong> div-seq:single-nucleus rna-seq reveals dynamics of rare adult newborn neurons\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.1610155113/asset/a7875690-c8f8-405a-a22b-f6498cf81933/assets/graphic/pnas.1610155113fig01.jpeg\n    link: /pages/a61efc/\n    name: ''\n    desc: <strong>(2016.08.16, , pnas)</strong> single-cell rnaseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons\n\n  - img: https://www.cell.com/cms/attachment/a8ffa9eb-0275-4f7e-94b0-ebec14a16894/gr1_lrg.jpg\n    link: /pages/8bf2bf/\n    name: ''\n    desc: <strong>(2017.10.11, act-seq, neuron)</strong> detecting activated cell populations using single-cell rna-seq <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41467-019-12054-3/mediaobjects/41467_2019_12054_fig1_html.png?as=webp\n    link: /pages/c60442/\n    name: ''\n    desc: <strong>(2019.09.13, mpfc, nat commun)</strong> cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction\n\n  - img: https://iiif.elifesciences.org/lax/47889%2felife-47889-fig5-v2.tif/full/,1500/0/default.jpg\n    link: /pages/bb2e69/\n    name: ''\n    desc: <strong>(2019.11.11, , elife)</strong> single-cell transcriptomic evidence for dense intracortical neuropeptide networks\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-019-1917-5/mediaobjects/41586_2019_1917_fig1_html.png?as=webp\n    link: /pages/0b79fa/\n    name: ''\n    desc: <strong>(2020.01.15, , nature)</strong> decoding the development of the human hippocampus <br><br>\n\n  - img: https://www.science.org/cms/10.1126/sciadv.abb3446/asset/0321fe52-76d5-401f-9b72-86e473a55a56/assets/graphic/abb3446-f1.jpeg\n    link: /pages/37a825/\n    name: ''\n    desc: <strong>(2020.06.26, , sci. adv.)</strong> molecular atlas of the adult mouse brain <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-020-00736-x/mediaobjects/41593_2020_736_fig3_html.png?as=webp\n    link: /pages/b5aaa4/\n    name: ''\n    desc: <strong>(2020.12.07, , nat neurosci)</strong> diversification of molecularly defined myenteric neuron classes revealed by single-cell rna sequencing\n\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-021-03670-5/mediaobjects/41586_2021_3670_fig1_html.png?as=webp\n    link: /pages/8a50f3/\n    name: ''\n    desc: <strong>(2021.06.23, , nature)</strong> molecular logic of cellular diversification in the mouse cerebral cortex <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41588-021-00894-z/mediaobjects/41588_2021_894_fig1_html.png?as=webp\n    link: /pages/25c02b/\n    name: ''\n    desc: <strong>(2021.07.08, hdwgcna, nat genet)</strong> single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-022-01073-x/mediaobjects/41593_2022_1073_fig1_html.png?as=webp\n    link: /pages/f481a1/\n    name: ''\n    desc: <strong>(2022.05.30, , nat neurosci)</strong> single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations\n\n  - img: https://www.pnas.org/cms/10.1073/pnas.2123146119/asset/13b94f27-0870-474b-b255-e1bc569c06cc/assets/images/large/pnas.2123146119fig01.jpg\n    link: /pages/444fce/\n    name: ''\n    desc: <strong>(2022.06.21, hpfc, pnas)</strong> the neuropeptide landscape of human prefrontal cortex <br><br>\n\n  - img: https://www.cell.com/cms/attachment/2ba65490-ca8d-4413-8652-ef034dbf5db6/gr1_lrg.jpg\n    link: /pages/8c91b0/\n    name: ''\n    desc: <strong>(2023.07.12, stereo-seq, cell)</strong> single-cell spatial transcriptome reveals cell-type organization in the macaque cortex\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-023-01455-9/mediaobjects/41593_2023_1455_fig1_html.png?as=webp\n    link: /pages/42c71a/\n    name: ''\n    desc: <strong>(2023.10.16,  pfc merfish, nat neurosci)</strong> spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain\n\n  - img: https://www.science.org/cms/10.1126/sciadv.adh7693/asset/f2f443e5-7123-41e4-8dc8-586440200643/assets/images/large/sciadv.adh7693-f1.jpg\n    link: /pages/12b484/\n    name: ''\n    desc: <strong>(2023.11.01, , sci. adv.)</strong> a telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types\n\n  - img: https://www.cell.com/cms/attachment/94939603-4f28-451e-a53f-4e9814ce0c3b/fx1_lrg.jpg\n    link: /pages/a1ce18/\n    name: ''\n    desc: <strong>(2023.11.15, , neuron)</strong> the neuropeptidergic connectome of c. elegans <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41586-023-06812-z/mediaobjects/41586_2023_6812_fig1_html.png?as=webp\n    link: /pages/895303/\n    name: ''\n    desc: <strong>(2023.12.13, allen, nature)</strong> a high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41586-024-07221-6/mediaobjects/41586_2024_7221_fig1_html.png?\n    link: /pages/666cd3/\n    name: ''\n    desc: <strong>(2024.04.24, barseq, nature)</strong> whole-cortex in situ sequencing reveals input-dependent area identity\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n\n\n\n# \n\n(2022.02.20, crd, j pineal res) singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma\n\n(2023.01.27, -fish rainbow, nat commun) highly efficient and robust -fish rainbow for multiplexed in situ detection of diverse biomolecules\n\n(2024.05.14, mip-seq, nat. biomed. eng) spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing\n\n(2024.07.04, benchmark, nat methods) systematic comparison of sequencing-based spatial transcriptomic methods\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://onlinelibrary.wiley.com/cms/asset/0c3d6310-66d6-45a2-bea0-3fae76351e68/jpi12803-fig-0001-m.jpg\n    link: /pages/3eedee/\n    name: ''\n    desc: <strong>(2022.02.20, crd, j pineal res)</strong> singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41467-023-36137-4/mediaobjects/41467_2023_36137_fig2_html.png?as=webp\n    link: /pages/c40bc0/\n    name: ''\n    desc: <strong>(2023.01.27, -fish rainbow, nat commun)</strong> highly efficient and robust -fish rainbow for multiplexed in situ detection of diverse biomolecules\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41551-024-01205-7/mediaobjects/41551_2024_1205_fig2_html.png?as=webp\n    link: /pages/e6566f/\n    name: ''\n    desc: <strong>(2024.05.14, mip-seq, nat. biomed. eng)</strong> spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41592-024-02325-3/mediaobjects/41592_2024_2325_fig1_html.png?as=webp\n    link: /pages/b55e03/\n    name: ''\n    desc: <strong>(2024.07.04, benchmark, nat methods)</strong> systematic comparison of sequencing-based spatial transcriptomic methods\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n\n\n\n# \n\n(2018.04.26, dlo hi-c, nat genet) digestion-ligation-only hi-c is an efficient and cost-effective method for chromosome conformation capture\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41588-018-0111-2/mediaobjects/41588_2018_111_fig1_html.jpg?as=webp\n    link: /pages/76fbba/\n    name: ''\n    desc: <strong>(2018.04.26, dlo hi-c, nat genet)</strong> digestion-ligation-only hi-c is an efficient and cost-effective method for chromosome conformation capture\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\n(2008.12.28, wgcna, bmc bioinformatics) wgcna an r package for weighted correlation network analysis\n\n(2015.04.13, seurat v1, nat biotechnol) spatial reconstruction of single-cell gene expression data\n\n\n\n(2017.12.04, wholebrain, nat neurosci) an interactive framework for whole-brain maps at cellular resolution\n\n\n\n(2018.02.06, scanpy, genome biol) scanpylarge-scale single-cell gene expression data analysis\n\n\n\n(2018.04.02, seurat v2, nat biotechnol) integrating single-cell transcriptomic data across different conditions, technologies, and species\n\n(2019.04.03, doubletfinder, cell systems) doubletfinder doublet detection in single-cell rna sequencing data using artificial nearest neighbors\n\n(2019.06.06, seurat v3, cell) comprehensive integration of single-cell data\n\n\n\n(2020.09.14, scrnaseq imputation, genome biol) a systematic evaluation of single-cell rna-sequencing imputation methods\n\n(2020.12.07, nnu-net, nat methods) nnu-net:a self-configuring method for deep learning-based biomedical image segmentation\n\n(2020.12.10, gcng, genome biol) gcnggraph convolutional networks for inferring gene interaction from spatial transcriptomics data\n\n(2020.12.14, cellpose, nat methods) cellpose:a generalist algorithm for cellular segmentation\n\n\n\n(2021.02.06, covid-19 ct, medical physics) toward data-efficient learning:a benchmark for covid-19 ct lung and infection segmentation\n\n(2021.02.17, cellchat, nat commun) inference and analysis of cell-cell communication using cellchat\n\n(2021.06.24, seurat v4, cell) integrated analysis of multimodal single-cell data\n\n\n\n(2021.07.01, lasso, front. genet) discovery and validation of key biomarkers based on immune infiltrates in alzheimer's disease\n\n(2021.10.28, tangram, nat methods) deep learning and alignment of spatially resolved single-cell transcriptomes with tangram\n\n(2022.01.13, cell2location, nat biotechnol) cell2location maps fine-grained cell types in spatial transcriptomics\n\n(2022.01.31, squidpy, nat methods) squidpy:a scalable framework for spatial omics analysis\n\n\n\n(2022.05.11, viv, nat methods) squidpy:a scalable framework for spatial omics analysis\n\n\n\n(2022.09.30, spacex, bioinformatics) spacexgene co-expression network estimation for spatial transcriptomics\n\n(2023.02.15, neuronchat, nat commun) inferring neuron-neuron communications from single-cell transcriptomics through neuronchat\n\n(2023.03.25, seurat v5, nat biotechnol) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n(2023.04.10, scverse, nat biotechnol) the scverse project provides a computational ecosystem for single-cell omics data analysis\n\n(2023.06.12, hdwgcna, cell reports methods) hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n\n(2023.11.22, spacel, nat commun) spacelcharacterizing spatial transcriptome architectures by deep-learning\n\n(2024.06.14, tidyomics, nat methods) the tidyomics ecosystemenhancing omic data analyses\n\n(2024.07.03, ezsinglecell, nat commun) ezsinglecellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists\n\nconfig:\n    target: _blank\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 5\n\ndata:\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1186%2f1471-2105-9-559/mediaobjects/12859_2008_article_2544_fig1_html.jpg?as=webp\n    link: /pages/ffdcc0/\n    name: ''\n    desc: <strong>(2008.12.28, wgcna, bmc bioinformatics)</strong> wgcna an r package for weighted correlation network analysis\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fnbt.3192/mediaobjects/41587_2015_article_bfnbt3192_fig1_html.jpg?as=webp\n    link: /pages/0d1fce/\n    name: ''\n    desc: <strong>(2015.04.13, seurat v1, nat biotechnol)</strong> spatial reconstruction of single-cell gene expression data <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41593-017-0027-7/mediaobjects/41593_2017_27_fig1_html.jpg?as=webp\n    link: /pages/e020de/\n    name: ''\n    desc: <strong>(2017.12.04, wholebrain, nat neurosci)</strong> an interactive framework for whole-brain maps at cellular resolution <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1186%2fs13059-017-1382-0/mediaobjects/13059_2017_1382_fig1_html.gif?as=webp\n    link: /pages/ebb632/\n    name: ''\n    desc: <strong>(2018.02.06, scanpy, genome biol)</strong> scanpylarge-scale single-cell gene expression data analysis <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fnbt.4096/mediaobjects/41587_2018_article_bfnbt4096_fig1_html.jpg?as=webp\n    link: /pages/f37280/\n    name: ''\n    desc: <strong>(2018.04.02, seurat v2, nat biotechnol)</strong> integrating single-cell transcriptomic data across different conditions, technologies, and species\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s2405471219300730-fx1_lrg.jpg\n    link: /pages/71b694/\n    name: ''\n    desc: <strong>(2019.04.03, doubletfinder, cell systems)</strong> doubletfinder doublet detection in single-cell rna sequencing data using artificial nearest neighbors\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s0092867419305598-fx1_lrg.jpg\n    link: /pages/d26f9e/\n    name: ''\n    desc: <strong>(2019.06.06, seurat v3, cell)</strong> comprehensive integration of single-cell data <br><br>\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1186%2fs13059-020-02132-x/mediaobjects/13059_2020_2132_fig1_html.png?as=webp\n    link: /pages/c7adb0/\n    name: ''\n    desc: <strong>(2020.09.14, scrnaseq imputation, genome biol)</strong> a systematic evaluation of single-cell rna-sequencing imputation methods\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41592-020-01008-z/mediaobjects/41592_2020_1008_fig1_html.png?as=webp\n    link: /pages/d38252/\n    name: ''\n    desc: <strong>(2020.12.07, nnu-net, nat methods)</strong> nnu-net:a self-configuring method for deep learning-based biomedical image segmentation\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1186%2fs13059-020-02214-w/mediaobjects/13059_2020_2214_fig1_html.png?as=webp\n    link: /pages/c34d2c/\n    name: ''\n    desc: <strong>(2020.12.10, gcng, genome biol)</strong> gcnggraph convolutional networks for inferring gene interaction from spatial transcriptomics data\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41592-020-01018-x/mediaobjects/41592_2020_1018_fig1_html.png?as=webp\n    link: /pages/1f3ea0/\n    name: ''\n    desc: <strong>(2020.12.14, cellpose, nat methods)</strong> cellpose:a generalist algorithm for cellular segmentation <br><br>\n\n  - img: https://raw.githubusercontent.com/junma11/covid-19-ct-seg-benchmark/master/utils/imageexamples.png\n    link: /pages/1c1b2b/\n    name: ''\n    desc: <strong>(2021.02.06, covid-19 ct, medical physics)</strong> toward data-efficient learning:a benchmark for covid-19 ct lung and infection segmentation\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41467-021-21246-9/mediaobjects/41467_2021_21246_fig1_html.png?as=webp\n    link: /pages/2737d2/\n    name: ''\n    desc: <strong>(2021.02.17, cellchat, nat commun)</strong> inference and analysis of cell-cell communication using cellchat\n\n  - img: https://ars.els-cdn.com/content/image/1-s2.0-s0092867421005833-fx1_lrg.jpg\n    link: /pages/b4dfe0/\n    name: ''\n    desc: <strong>(2021.06.24, seurat v4, cell)</strong> integrated analysis of multimodal single-cell data <br><br>\n\n  - img: https://www.frontiersin.org/files/articles/658323/fgene-12-658323-html/image_m/fgene-12-658323-g001.jpg\n    link: /pages/6a4bcc/\n    name: ''\n    desc: <strong>(2021.07.01, lasso, front. genet)</strong> discovery and validation of key biomarkers based on immune infiltrates in alzheimer's disease\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41592-021-01264-7/mediaobjects/41592_2021_1264_fig1_html.png?as=webp\n    link: /pages/c3bbd9/\n    name: ''\n    desc: <strong>(2021.10.28, tangram, nat methods)</strong> deep learning and alignment of spatially resolved single-cell transcriptomes with tangram\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41587-021-01139-4/mediaobjects/41587_2021_1139_fig1_html.png?as=webp\n    link: /pages/9912cc/\n    name: ''\n    desc: <strong>(2022.01.13, cell2location, nat biotechnol)</strong> cell2location maps fine-grained cell types in spatial transcriptomics\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41592-021-01358-2/mediaobjects/41592_2021_1358_fig1_html.png?as=webp\n    link: /pages/76476f/\n    name: ''\n    desc: <strong>(2022.01.31, squidpy, nat methods)</strong> squidpy:a scalable framework for spatial omics analysis <br><br>\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41592-022-01482-7/mediaobjects/41592_2022_1482_fig1_html.png?as=webp\n    link: /pages/112d0b/\n    name: ''\n    desc: <strong>(2022.05.11, viv, nat methods)</strong> squidpy:a scalable framework for spatial omics analysis <br><br>\n\n  - img: https://oup.silverchair-cdn.com/oup/backfile/content_public/journal/bioinformatics/38/22/10.1093_bioinformatics_btac645/3/btac645f1.jpeg?expires=1725068870&signature=gkoqi32igjee3tntpwrok01xedunvd9rcecittiwpcbw5zwdu3~inp1awlimoug9rmblyqztwpcfbuilfr8qf2yffky9qcc6po0euklrpge5mm55yplow8fg2hkak-bw8qhv9ppisjsc0eoqwh6ayfcgivjnun4h3~rjku3lujps3oaxqhtocisey0pyhkccgckoraj2qkvromju55m7qgszpr6rz99ke7ra0kcq8y5vhba-7yd0eb8rx7jun2usndsd2uo7ievzyxhutvq1vmuyx-3fq1040nmgl4swjathocimte-zesutwtv-qagkntrjalxlnp5emk0hix5hdg__&key-pair-id=apkaie5g5crdk6rd3pga\n    link: /pages/4cba92/\n    name: ''\n    desc: <strong>(2022.09.30, spacex, bioinformatics)</strong> spacexgene co-expression network estimation for spatial transcriptomics \n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41467-023-36800-w/mediaobjects/41467_2023_36800_fig1_html.png?as=webp\n    link: /pages/794f06/\n    name: ''\n    desc: <strong>(2023.02.15, neuronchat, nat commun)</strong> inferring neuron-neuron communications from single-cell transcriptomics through neuronchat\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41587-023-01767-y/mediaobjects/41587_2023_1767_fig1_html.png?as=webp\n    link: /pages/8c5839/\n    name: ''\n    desc: <strong>(2023.03.25, seurat v5, nat biotechnol)</strong> dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n  - img: https://media.springernature.com/full/springer-static/image/art%3a10.1038%2fs41587-023-01733-8/mediaobjects/41587_2023_1733_fig1_html.png?as=webp\n    link: /pages/124edd/\n    name: ''\n    desc: <strong>(2023.04.10, scverse, nat biotechnol)</strong> the scverse project provides a computational ecosystem for single-cell omics data analysis\n\n  - img: https://www.cell.com/cms/attachment/2f4fe3cb-81a1-4461-ae56-05fd516d7822/fx1_lrg.jpg\n    link: /pages/a24249/\n    name: ''\n    desc: <strong>(2023.06.12, hdwgcna, cell reports methods)</strong> hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41467-023-43220-3/mediaobjects/41467_2023_43220_fig1_html.png?as=webp\n    link: /pages/da0fb5/\n    name: ''\n    desc: <strong>(2023.11.22, spacel, nat commun)</strong> spacelcharacterizing spatial transcriptome architectures by deep-learning\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41592-024-02299-2/mediaobjects/41592_2024_2299_fig1_html.png?as=webp\n    link: /pages/32a729/\n    name: ''\n    desc: <strong>(2024.06.14, tidyomics, nat methods)</strong> the tidyomics ecosystemenhancing omic data analyses\n\n  - img: https://media.springernature.com/lw685/springer-static/image/art%3a10.1038%2fs41467-024-48188-2/mediaobjects/41467_2024_48188_fig1_html.png?as=webp\n    link: /pages/02ef79/\n    name: ''\n    desc: <strong>(2024.07.03, ezsinglecell, nat commun)</strong> ezsinglecellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"02."}},title:"",date:"2022-11-19T10:51:23.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/577e62/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB.html",relativePath:"00./02. - /00..md",key:"v-fb61cf0a",path:"/pages/577e62/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:" --- ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"02./12."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/e2e6a4/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/12.%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84.html",relativePath:"00./02. - /12..md",key:"v-4f1678f4",path:"/pages/e2e6a4/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" --- ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"02./13."}},title:" --- ",date:"2022-12-08T20:35:03.000Z",permalink:"/pages/e90196/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/13.%E4%B8%89%E7%BB%B4%E5%9F%BA%E5%9B%A0%E7%BB%84.html",relativePath:"00./02. - /13..md",key:"v-6b3ea08c",path:"/pages/e90196/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" --- ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"02./14."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/69dc52/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6.html",relativePath:"00./02. - /14..md",key:"v-033b6f48",path:"/pages/69dc52/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03."}},title:"",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/98350d/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0.html",relativePath:"00./03. - /00..md",key:"v-634587af",path:"/pages/98350d/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03./02. - "}},title:" - ",date:"2023-05-10T20:47:21.000Z",permalink:"/pages/3c3418/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/12.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0.html",relativePath:"00./03. - /12. - .md",key:"v-a6be02e2",path:"/pages/3c3418/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/28, 19:35:41",lastUpdatedTimestamp:1687952141e3},{title:"",frontmatter:{title:"",date:"2023-02-02T11:18:04.000Z",permalink:"/pages/09e806/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/01.%E4%BD%9C%E8%80%85%E6%B1%87%E6%80%BB.html",relativePath:"00./02. - /01..md",key:"v-46d8e2d8",path:"/pages/09e806/",headers:[{level:2,title:"Gang Cao ()",slug:"gang-cao-",normalizedTitle:"gang cao ()",charIndex:0},{level:2,title:"Anthony Zador",slug:"anthony-zador",normalizedTitle:"anthony zador",charIndex:31},{level:2,title:"Hongkui Zeng ()",slug:"hongkui-zeng-",normalizedTitle:"hongkui zeng ()",charIndex:83},{level:2,title:"Liqun Luo ()",slug:"liqun-luo-",normalizedTitle:"liqun luo ()",charIndex:144},{level:2,title:"Aviv Regev",slug:"aviv-regev",normalizedTitle:"aviv regev",charIndex:190},{level:2,title:"Qing Nie ()",slug:"qing-nie-",normalizedTitle:"qing nie ()",charIndex:244},{level:2,title:"Rahul Satija",slug:"rahul-satija",normalizedTitle:"rahul satija",charIndex:299},{level:2,title:"Edward Callaway",slug:"edward-callaway",normalizedTitle:"edward callaway",charIndex:343},{level:2,title:"Jun Yan ()",slug:"jun-yan-",normalizedTitle:"jun yan ()",charIndex:409},{level:2,title:"Feng Zhang ()",slug:"feng-zhang-",normalizedTitle:"feng zhang ()",charIndex:443},{level:2,title:"Yi Zhang ()",slug:"yi-zhang-",normalizedTitle:"yi zhang ()",charIndex:502},{level:2,title:"Fabian J. Theis",slug:"fabian-j-theis",normalizedTitle:"fabian j. theis",charIndex:547},{level:2,title:"Kun Qu ()",slug:"kun-qu-",normalizedTitle:"kun qu ()",charIndex:602},{level:2,title:"Xiaowei Zhuang ()",slug:"xiaowei-zhuang-",normalizedTitle:"xiaowei zhuang ()",charIndex:633},{level:2,title:"Hongwei Dong ()",slug:"hongwei-dong-",normalizedTitle:"hongwei dong ()",charIndex:681}],headersStr:"Gang Cao () Anthony Zador Hongkui Zeng () Liqun Luo () Aviv Regev Qing Nie () Rahul Satija Edward Callaway Jun Yan () Feng Zhang () Yi Zhang () Fabian J. Theis Kun Qu () Xiaowei Zhuang () Hongwei Dong ()",content:'Gang Cao ()\n\n\n\n\n\n\n\n\n\n\n\nAnthony Zador\n\n\n\n\n\nCold Spring Harbor Laboratory\n\n\n\nHongkui Zeng ()\n\n\n\n\n\nAllen Institute for Brain Science\n\n\n\nLiqun Luo ()\n\n\n\n\n\nStanford University\n\n\n\n\n\nAviv Regev\n\n\n\n\n\nBroad Institute of MIT and Harvard\n\n\n\nQing Nie ()\n\n\n\n\n\nUniversity of California, Irvine\n\n\n\nRahul Satija\n\n\n\n\n\nNew York Genome Center\n\n\n\nEdward Callaway\n\n\n\n\n\nThe Salk Institute for Biological Studies\n\n\n\nJun Yan ()\n\n\n\n\n\n\n\n\n\nFeng Zhang ()\n\n\n\n\n\nBroad Institute of MIT and Harvard\n\n\n\nYi Zhang ()\n\n\n\n\n\nHARVARD MEDICAL SCHOOL\n\n\n\nFabian J. Theis\n\n\n\n\n\nTechnische Universitt Mnchen\n\n\n\nKun Qu ()\n\n\n\n\n\n\n\n\n\n\n\nXiaowei Zhuang ()\n\n\n\n\n\nHarvard University\n\n\n\nHongwei Dong ()\n\n\n\n\n\nUCLA\n\n\n\nconfig:\n    target: _self\n    imgHeight: 150px\n    objectFit: cover\n    lineClamp: 2\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Gang_Cao.8vmtcwxh07.webp\n    link: /pages/09e806/#gang-cao-\n    name: <div align="center">Gang Cao ()</div>\n    desc: <div align="center"> <br><br></div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Anthony-Zador.1j141mxgi3c0.webp\n    link: /pages/09e806/#anthony-zador\n    name: <div align="center">Anthony Zador</div>\n    desc: <div align="center">Cold Spring Harbor Laboratory</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Hongkui_Zeng.68jimoeruys0.webp\n    link: /pages/09e806/#hongkui-zeng-\n    name: <div align="center">Hongkui Zeng ()</div>\n    desc: <div align="center">Allen Institute for Brain Science</div>\n    \n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Liqun_Luo.5q77udpe2tg0.webp\n    link: /pages/09e806/#liqun-luo-\n    name: <div align="center">Liqun Luo ()</div>\n    desc: <div align="center">Stanford University <br><br></div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Aviv_Regev.5cn8ag0jlb80.webp\n    link: /pages/09e806/#aviv-regev\n    name: <div align="center">Aviv Regev</div>\n    desc: <div align="center">Broad Institute of MIT and Harvard</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Qing_Nie.1k6nws60lso0.webp\n    link: /pages/09e806/#qing-nie-\n    name: <div align="center">Qing Nie ()</div>\n    desc: <div align="center">University of California, Irvine</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/download.47syrpendga0.webp\n    link: /pages/09e806/#rahul-satija\n    name: <div align="center">Rahul Satija</div>\n    desc: <div align="center">New York Genome Center</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Edward_Callaway.9nzounx19p.webp\n    link: /pages/09e806/#edward-callaway\n    name: <div align="center">Edward Callaway</div>\n    desc: <div align="center">The Salk Institute for Biological Studies</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Jun_Yan.2obfcsy3v6.webp\n    link: /pages/09e806/#jun-yan-\n    name: <div align="center">Jun Yan ()</div>\n    desc: <div align="center"></div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Feng_Zhang.9nzoupswln.webp\n    link: /pages/09e806/#feng-zhang-\n    name: <div align="center">Feng Zhang ()</div>\n    desc: <div align="center">Broad Institute of MIT and Harvard</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Yi_Zhang.8hgdm4klqz.webp\n    link: /pages/09e806/#yi-zhang-\n    name: <div align="center">Yi Zhang ()</div>\n    desc: <div align="center">HARVARD MEDICAL SCHOOL</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/fabian_theis.6ik6vxejdg.webp\n    link: /pages/09e806/#fabian-j-theis\n    name: <div align="center">Fabian J. Theis</div>\n    desc: <div align="center">Technische Universitt Mnchen</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Kun_Qu.175ab9w6bx.webp\n    link: /pages/09e806/#kun-qu-\n    name: <div align="center">Kun Qu ()</div>\n    desc: <div align="center"> <br><br></div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Xiaowei_Zhuang.64dsj5rsfh.webp\n    link: /pages/09e806/#xiaowei-zhuang-\n    name: <div align="center">Xiaowei Zhuang ()</div>\n    desc: <div align="center">Harvard University</div>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/02-/01-/Hongwei_Dong.45hlsu7wcx.webp\n    link: /pages/09e806/#hongwei-dong-\n    name: <div align="center">Hongwei Dong ()</div>\n    desc: <div align="center">UCLA</div>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n\n\n\n\n\n----------------------------------------\n\n\n\n\n\n# Gang Cao ()\n\n\nNCMLS:123-/Nature\nGenetics, Nature Biomedical Engineering, Neuron, Nature\nCommunications, Science Advances, Nucleic Acids Res, Genome\nBiology12,Nature\nGenetics, Nature Reviews Neuroscience , Neuron,\nJACC\n(2018.04.26 DLO Hi-C). Lin, D., Hong, P., Zhang, S. et al.\nDigestion-ligation-only Hi-C is an efficient and\ncost-effective method for chromosome conformation capture.\nNat Genet 50, 754763 (2018).\n\n(2019.02.08 ). Sun, L., Tang, Y., Yan, K. et al.\nDifferences in neurotropism and neurotoxicity among\nretrograde viral tracers. Mol Neurodegeneration 14, 8\n(2019).\n\n(2023.01.27 -FISH rainbow). Tao, Y., Zhou, X., Sun, L. et\nal. Highly efficient and robust -FISH rainbow for\nmultiplexed in situ detection of diverse biomolecules. Nat\nCommun 14, 443 (2023).\n\n(2024.05.14 MiP-seq). Wu, X., Xu, W., Deng, L. et al.\nSpatial multi-omics at subcellular resolution via\nhigh-throughput in situ pairwise sequencing. Nat. Biomed.\nEng (2024).\n\n\n# Anthony Zador\n\n\nAnthony M. Zador Cold Spring Harbor\nLaboratory Alle Davis Harris\nCOSYNENAISYS\n2004 Zador\nMAPseq\nGoogle Scholar(2024-07-26): Citations 16497 | h-index 57 |\ni10-index 98\n\n(2016.07.18 MAPseq). Kebschull, J. M., Garcia da Silva, P.,\nReid, A. P., Peikon, I. D., Albeanu, D. F., & Zador, A. M.\n(2016). High-Throughput Mapping of Single-Neuron Projections\nby Sequencing of Barcoded RNA. Neuron, 91(5), 975-987.\n\n(2018.03.28 MAPseq). Han, Y., Kebschull, J., Campbell,\nR. et al. The logic of single-cell projections from visual\ncortex. Nature 556, 5156 (2018).\n\n(2019.10.17 BARseq). Chen X, Sun YC, Zhan H, Kebschull JM,\nFischer S, Matho K, Huang ZJ, Gillis J, Zador AM.\nHigh-Throughput Mapping of Long-Range Neuronal Projection\nUsing In Situ Sequencing. Cell. 2019 Oct\n17;179(3):772-786.e19.\n\n(2020.07.09 BRICseq). Huang L, Kebschull JM, Frth D, Musall\nS, Kaufman MT, Churchland AK, Zador AM. BRICseq Bridges\nBrain-wide Interregional Connectivity to Neural Activity and\nGene Expression in Single Animals. Cell. 2020 Jul\n9;182(1):177-188.e27.\n\n(2021.05.10 BARseq2). Sun, YC., Chen, X., Fischer, S. et al.\nIntegrating barcoded neuroanatomy with spatial\ntranscriptional profiling enables identification of gene\ncorrelates of projections. Nat Neurosci 24, 873885 (2021).\n\n(2022.10.27 MAPseq). Chen Y, Chen X, Baserdem B. et al.\nHigh-throughput sequencing of single neuron projections\nreveals spatial organization in the olfactory cortex. Cell.\n2022 Oct 27;185(22):4117-4134.e28.\n\n(2024.04.24 BARseq). Chen, X., Fischer, S., Rue, M.C.P.\net al. Whole-cortex in situ sequencing reveals\ninput-dependent area identity. Nature (2024).\n\n\n# Hongkui Zeng ()\n\n\nHongkui Zeng ()Allen Institute for Brain\nScience 2023  5  2\n 2006\nCCF\nGoogle Scholar(2024-07-26): Citations 42664 | h-index 89 |\ni10-index 193\n\n(2014.04.02 Allen). Oh, S., Harris, J., Ng, L. et al. A\nmesoscale connectome of the mouse brain. Nature 508, 207214\n(2014).\n\n(2018.10.31 ALM&VISp). Tasic, B., Yao, Z., Graybuck,\nL.T. et al. Shared and distinct transcriptomic cell types\nacross neocortical areas. Nature 563, 7278 (2018).\n\n(2023.12.13 Allen). Yao, Z., van Velthoven, C.T.J.,\nKunst, M. et al. A high-resolution transcriptomic and\nspatial atlas of cell types in the whole mouse brain. Nature\n624, 317332 (2023).\n\n\n# Liqun Luo ()\n\n\nLiqun Luo ()1966  1\n1981\n1986  1\n1987\n1992 1996\n1997 2005\n9  Jacob K. Javits 2012\n\nGoogle Scholar(2024-07-26): Citations 52678 | h-index 118 |\ni10-index 220\n\n(2020.12.17 mPFC). Lui JH, Nguyen ND, Grutzner SM, et\nal. Differential encoding in prefrontal cortex projection\nneuron classes across cognitive tasks. Cell.\n2021;184(2):489-506.e26.\n\n\n# Aviv Regev\n\n\nAviv Regev\nGenentech/Roche\nBroad Regev\n Sarah Teichmann \nHuman Cell Atlas \n(2015.04.13 Seurat V1). Satija, R., Farrell, J., Gennert, D.\net al. Spatial reconstruction of single-cell gene expression\ndata. Nat Biotechnol 33, 495502 (2015).\n\n(2016.06.28 Div-Seq). Habib N, Li Y, Heidenreich M. et al.\nDiv-Seq: Single-nucleus RNA-Seq reveals dynamics of rare\nadult newborn neurons. Science. 2016 Aug 26;353(6302):925-8.\n\n(2021.06.23 ). Di Bella, D.J., Habibi, E., Stickels,\nR.R. et al. Molecular logic of cellular diversification in\nthe mouse cerebral cortex. Nature 595, 554559 (2021).\n\n(2021.10.28 Tangram). Biancalani, T., Scalia, G., Buffoni,\nL. et al. Deep learning and alignment of spatially resolved\nsingle-cell transcriptomes with Tangram. Nat Methods 18,\n13521362 (2021).\n\n\n# Qing Nie ()\n\n\nQing Nie () University of\nCalifornia,\nIrvine\nNSF-Simons \nGoogle Scholar(2024-07-26): Citations 14981 | h-index 56 |\ni10-index 176\n\n(2021.02.17 CellChat). Jin, S., Guerrero-Juarez, C.F.,\nZhang, L. et al. Inference and analysis of cell-cell\ncommunication using CellChat. Nat Commun 12, 1088 (2021).\n\n(2023.02.15 NeuronChat). Zhao, W., Johnston, K.G., Ren, H.\net al. Inferring neuron-neuron communications from\nsingle-cell transcriptomics through NeuronChat. Nat Commun\n14, 1128 (2023).\n\n\n# Rahul Satija\n\n\nRahul Satija  (New York Genome Center, NYGC)\n (NYU)  NYGC Satija\n (Seurat)Satija\nSatija\n NIH  2020  NIH  Satija\n\nGoogle Scholar(2024-07-26): Citations 75184 | h-index 61 |\ni10-index 79\n\n(2015.04.13 Seurat V1). Satija, R., Farrell, J., Gennert, D.\net al. Spatial reconstruction of single-cell gene expression\ndata. Nat Biotechnol 33, 495502 (2015).\n\n(2018.04.02 Seurat V2). Butler, A., Hoffman, P., Smibert, P.\net al. Integrating single-cell transcriptomic data across\ndifferent conditions, technologies, and species. Nat\nBiotechnol 36, 411420 (2018).\n\n(2019.06.06 Seurat V3). Stuart T, Butler A, Hoffman P, et\nal. Comprehensive Integration of Single-Cell Data. Cell.\n2019;177(7):1888-1902.e21.\n\n(2021.06.24 Seurat V4). Hao Y, Hao S, Andersen-Nissen E, et\nal. Integrated analysis of multimodal single-cell data.\nCell. 2021;184(13):3573-3587.e29.\n\n(2023.03.25 Seurat V5). Hao, Y., Stuart, T., Kowalski, M.H.\net al. Dictionary learning for integrative, multimodal and\nscalable single-cell analysis. Nat Biotechnol (2023).\n\n\n# Edward Callaway\n\n\nEdward\nCallaway\n\n\n\nGoogle Scholar(2024-07-26): Citations 29401 | h-index 89 |\ni10-index 149\n\n(2021.10.06). Zhang, Z., Zhou, J., Tan, P. et al. Epigenomic\ndiversity of cortical projection neurons in the mouse brain.\nNature 598, 167173 (2021).\n\n(2023.12.13). Zhou, J., Zhang, Z., Wu, M. et al. Brain-wide\ncorrespondence of neuronal epigenomics and distant\nprojections. Nature 624, 355365 (2023).\n\n\n# Jun Yan ()\n\n\n1994-19981998-20032003-2006200692009320159\n(2022.03.31). Gao, L., Liu, S., Gou, L. et al. Single-neuron\nprojectome of mouse prefrontal cortex. Nat Neurosci 25,\n515529 (2022).\n\n(2023.5.22). Gao, L., Liu, S., Wang, Y. et al. Single-neuron\nanalysis of dendrites and axons reveals the network\norganization in mouse prefrontal cortex. Nat Neurosci\n(2023).\n\n\n# Feng Zhang ()\n\n\n19821022CRISPR-Cas\nGoogle Scholar(2024-07-26): Citations 170259 | h-index 111 |\ni10-index 123\n\n(2016.06.28 Div-Seq). Habib N, Li Y, Heidenreich M. et al.\nDiv-Seq: Single-nucleus RNA-Seq reveals dynamics of rare\nadult newborn neurons. Science. 2016 Aug 26;353(6302):925-8.\n\n\n# Yi Zhang ()\n\n\n19841995Fred\nRosenH3K27me3X-DNA18075000H114ScienceWatch2002~2006\nGoogle Scholar(2024-07-26): Citations 89478 | h-index 122 |\ni10-index 187\n\n(2019.09.13). Bhattacherjee, A., Djekidel, M.N., Chen, R. et\nal. Cell type-specific transcriptional programs in mouse\nprefrontal cortex during adolescence and addiction. Nat\nCommun 10, 4169 (2019).\n\n(2023.10.16). Bhattacherjee, A., Zhang, C., Watson, B.R. et\nal. Spatial transcriptomics reveals the distinct\norganization of mouse prefrontal cortex and neuronal\nsubtypes regulating chronic pain. Nat Neurosci (2023).\n\n\n# Fabian J. Theis\n\n\nFabian \n2018  2019 \n(HelmholtzAI) Fabian Theis\n\nNature BiotechlonogyNature MethodsNature\ncommunicationsCell systemsMolecular Systems\nBiologybioinformatics \nGoogle Scholar(2024-07-26): Citations 65585 | h-index 117 |\ni10-index 445\n\n(2018.02.06 SCANPY). Wolf, F., Angerer, P. & Theis, F.\nSCANPY: large-scale single-cell gene expression data\nanalysis. Genome Biol 19, 15 (2018).\n\n(2022.01.31 Squidpy). Palla, G., Spitzer, H., Klein, M. et\nal. Squidpy: a scalable framework for spatial omics\nanalysis. Nat Methods 19, 171178 (2022).\n\n(2023.04.10 scverse). Virshup, I., Bredikhin, D., Heumos, L.\net al. The scverse project provides a computational\necosystem for single-cell omics data analysis. Nat\nBiotechnol 41, 604606 (2023).\n\n\n# Kun Qu ()\n\n\n\nNatureNature\nMethodsCellCancer\nCell60Google80001000H-index\n36\nGoogle Scholar(2024-07-26): Citations 13271 | h-index 48 |\ni10-index 72\n\n(2023.11.22 SPACEL). Xu, H., Wang, S., Fang, M. et al.\nSPACEL: deep learning-based characterization of spatial\ntranscriptome architectures. Nat Commun 14, 7603 (2023).\n\n\n# Xiaowei Zhuang ()\n\n\n19721\n1987199119199620012005200620122015\n\nGoogle Scholar(2024-08-30): Citations 64703 | h-index 111 |\ni10-index 198\n\n(2022.12.28 MERFISH). William E. Allen, Timothy R.\nBlosser, Zuri A. Sullivan, et al. Molecular and spatial\nsignatures of mouse brain aging at single-cell resolution.\nCell, Volume 186, Issue 1, 2023, Pages 194-208.e18, ISSN\n0092-8674.\n\n\n# Hongwei Dong ()\n\n\nUCLA C57Bl/6\n3D\n\n\n\nGoogle Scholar(2024-08-30): Citations 19834 | h-index 42 |\ni10-index 57\n\n(2021.10.06 MOp-ul). Muoz-Castaeda, R., Zingg, B., Matho,\nK.S. et al. Cellular anatomy of the mouse primary motor\ncortex. Nature 598, 159166 (2021).',normalizedContent:'gang cao ()\n\n\n\n\n\n\n\n\n\n\n\nanthony zador\n\n\n\n\n\ncold spring harbor laboratory\n\n\n\nhongkui zeng ()\n\n\n\n\n\nallen institute for brain science\n\n\n\nliqun luo ()\n\n\n\n\n\nstanford university\n\n\n\n\n\naviv regev\n\n\n\n\n\nbroad institute of mit and harvard\n\n\n\nqing nie ()\n\n\n\n\n\nuniversity of california, irvine\n\n\n\nrahul satija\n\n\n\n\n\nnew york genome center\n\n\n\nedward callaway\n\n\n\n\n\nthe salk institute for biological studies\n\n\n\njun yan ()\n\n\n\n\n\n\n\n\n\nfeng zhang ()\n\n\n\n\n\nbroad institute of mit and harvard\n\n\n\nyi zhang ()\n\n\n\n\n\nharvard medical school\n\n\n\nfabian j. theis\n\n\n\n\n\ntechnische universitat munchen\n\n\n\nkun qu ()\n\n\n\n\n\n\n\n\n\n\n\nxiaowei zhuang ()\n\n\n\n\n\nharvard university\n\n\n\nhongwei dong ()\n\n\n\n\n\nucla\n\n\n\nconfig:\n    target: _self\n    imgheight: 150px\n    objectfit: cover\n    lineclamp: 2\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/gang_cao.8vmtcwxh07.webp\n    link: /pages/09e806/#gang-cao-\n    name: <div align="center">gang cao ()</div>\n    desc: <div align="center"> <br><br></div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/anthony-zador.1j141mxgi3c0.webp\n    link: /pages/09e806/#anthony-zador\n    name: <div align="center">anthony zador</div>\n    desc: <div align="center">cold spring harbor laboratory</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/hongkui_zeng.68jimoeruys0.webp\n    link: /pages/09e806/#hongkui-zeng-\n    name: <div align="center">hongkui zeng ()</div>\n    desc: <div align="center">allen institute for brain science</div>\n    \n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/liqun_luo.5q77udpe2tg0.webp\n    link: /pages/09e806/#liqun-luo-\n    name: <div align="center">liqun luo ()</div>\n    desc: <div align="center">stanford university <br><br></div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/aviv_regev.5cn8ag0jlb80.webp\n    link: /pages/09e806/#aviv-regev\n    name: <div align="center">aviv regev</div>\n    desc: <div align="center">broad institute of mit and harvard</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/qing_nie.1k6nws60lso0.webp\n    link: /pages/09e806/#qing-nie-\n    name: <div align="center">qing nie ()</div>\n    desc: <div align="center">university of california, irvine</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/download.47syrpendga0.webp\n    link: /pages/09e806/#rahul-satija\n    name: <div align="center">rahul satija</div>\n    desc: <div align="center">new york genome center</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/edward_callaway.9nzounx19p.webp\n    link: /pages/09e806/#edward-callaway\n    name: <div align="center">edward callaway</div>\n    desc: <div align="center">the salk institute for biological studies</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/jun_yan.2obfcsy3v6.webp\n    link: /pages/09e806/#jun-yan-\n    name: <div align="center">jun yan ()</div>\n    desc: <div align="center"></div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/feng_zhang.9nzoupswln.webp\n    link: /pages/09e806/#feng-zhang-\n    name: <div align="center">feng zhang ()</div>\n    desc: <div align="center">broad institute of mit and harvard</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/yi_zhang.8hgdm4klqz.webp\n    link: /pages/09e806/#yi-zhang-\n    name: <div align="center">yi zhang ()</div>\n    desc: <div align="center">harvard medical school</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/fabian_theis.6ik6vxejdg.webp\n    link: /pages/09e806/#fabian-j-theis\n    name: <div align="center">fabian j. theis</div>\n    desc: <div align="center">technische universitat munchen</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/kun_qu.175ab9w6bx.webp\n    link: /pages/09e806/#kun-qu-\n    name: <div align="center">kun qu ()</div>\n    desc: <div align="center"> <br><br></div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/xiaowei_zhuang.64dsj5rsfh.webp\n    link: /pages/09e806/#xiaowei-zhuang-\n    name: <div align="center">xiaowei zhuang ()</div>\n    desc: <div align="center">harvard university</div>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/02-/01-/hongwei_dong.45hlsu7wcx.webp\n    link: /pages/09e806/#hongwei-dong-\n    name: <div align="center">hongwei dong ()</div>\n    desc: <div align="center">ucla</div>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n\n\n\n\n\n----------------------------------------\n\n\n\n\n\n# gang cao ()\n\n\nncmls:123-/nature\ngenetics, nature biomedical engineering, neuron, nature\ncommunications, science advances, nucleic acids res, genome\nbiology12,nature\ngenetics, nature reviews neuroscience , neuron,\njacc\n(2018.04.26 dlo hi-c). lin, d., hong, p., zhang, s. et al.\ndigestion-ligation-only hi-c is an efficient and\ncost-effective method for chromosome conformation capture.\nnat genet 50, 754763 (2018).\n\n(2019.02.08 ). sun, l., tang, y., yan, k. et al.\ndifferences in neurotropism and neurotoxicity among\nretrograde viral tracers. mol neurodegeneration 14, 8\n(2019).\n\n(2023.01.27 -fish rainbow). tao, y., zhou, x., sun, l. et\nal. highly efficient and robust -fish rainbow for\nmultiplexed in situ detection of diverse biomolecules. nat\ncommun 14, 443 (2023).\n\n(2024.05.14 mip-seq). wu, x., xu, w., deng, l. et al.\nspatial multi-omics at subcellular resolution via\nhigh-throughput in situ pairwise sequencing. nat. biomed.\neng (2024).\n\n\n# anthony zador\n\n\nanthony m. zador cold spring harbor\nlaboratory alle davis harris\ncosynenaisys\n2004 zador\nmapseq\ngoogle scholar(2024-07-26): citations 16497 | h-index 57 |\ni10-index 98\n\n(2016.07.18 mapseq). kebschull, j. m., garcia da silva, p.,\nreid, a. p., peikon, i. d., albeanu, d. f., & zador, a. m.\n(2016). high-throughput mapping of single-neuron projections\nby sequencing of barcoded rna. neuron, 91(5), 975-987.\n\n(2018.03.28 mapseq). han, y., kebschull, j., campbell,\nr. et al. the logic of single-cell projections from visual\ncortex. nature 556, 5156 (2018).\n\n(2019.10.17 barseq). chen x, sun yc, zhan h, kebschull jm,\nfischer s, matho k, huang zj, gillis j, zador am.\nhigh-throughput mapping of long-range neuronal projection\nusing in situ sequencing. cell. 2019 oct\n17;179(3):772-786.e19.\n\n(2020.07.09 bricseq). huang l, kebschull jm, furth d, musall\ns, kaufman mt, churchland ak, zador am. bricseq bridges\nbrain-wide interregional connectivity to neural activity and\ngene expression in single animals. cell. 2020 jul\n9;182(1):177-188.e27.\n\n(2021.05.10 barseq2). sun, yc., chen, x., fischer, s. et al.\nintegrating barcoded neuroanatomy with spatial\ntranscriptional profiling enables identification of gene\ncorrelates of projections. nat neurosci 24, 873885 (2021).\n\n(2022.10.27 mapseq). chen y, chen x, baserdem b. et al.\nhigh-throughput sequencing of single neuron projections\nreveals spatial organization in the olfactory cortex. cell.\n2022 oct 27;185(22):4117-4134.e28.\n\n(2024.04.24 barseq). chen, x., fischer, s., rue, m.c.p.\net al. whole-cortex in situ sequencing reveals\ninput-dependent area identity. nature (2024).\n\n\n# hongkui zeng ()\n\n\nhongkui zeng ()allen institute for brain\nscience 2023  5  2\n 2006\nccf\ngoogle scholar(2024-07-26): citations 42664 | h-index 89 |\ni10-index 193\n\n(2014.04.02 allen). oh, s., harris, j., ng, l. et al. a\nmesoscale connectome of the mouse brain. nature 508, 207214\n(2014).\n\n(2018.10.31 alm&visp). tasic, b., yao, z., graybuck,\nl.t. et al. shared and distinct transcriptomic cell types\nacross neocortical areas. nature 563, 7278 (2018).\n\n(2023.12.13 allen). yao, z., van velthoven, c.t.j.,\nkunst, m. et al. a high-resolution transcriptomic and\nspatial atlas of cell types in the whole mouse brain. nature\n624, 317332 (2023).\n\n\n# liqun luo ()\n\n\nliqun luo ()1966  1\n1981\n1986  1\n1987\n1992 1996\n1997 2005\n9  jacob k. javits 2012\n\ngoogle scholar(2024-07-26): citations 52678 | h-index 118 |\ni10-index 220\n\n(2020.12.17 mpfc). lui jh, nguyen nd, grutzner sm, et\nal. differential encoding in prefrontal cortex projection\nneuron classes across cognitive tasks. cell.\n2021;184(2):489-506.e26.\n\n\n# aviv regev\n\n\naviv regev\ngenentech/roche\nbroad regev\n sarah teichmann \nhuman cell atlas \n(2015.04.13 seurat v1). satija, r., farrell, j., gennert, d.\net al. spatial reconstruction of single-cell gene expression\ndata. nat biotechnol 33, 495502 (2015).\n\n(2016.06.28 div-seq). habib n, li y, heidenreich m. et al.\ndiv-seq: single-nucleus rna-seq reveals dynamics of rare\nadult newborn neurons. science. 2016 aug 26;353(6302):925-8.\n\n(2021.06.23 ). di bella, d.j., habibi, e., stickels,\nr.r. et al. molecular logic of cellular diversification in\nthe mouse cerebral cortex. nature 595, 554559 (2021).\n\n(2021.10.28 tangram). biancalani, t., scalia, g., buffoni,\nl. et al. deep learning and alignment of spatially resolved\nsingle-cell transcriptomes with tangram. nat methods 18,\n13521362 (2021).\n\n\n# qing nie ()\n\n\nqing nie () university of\ncalifornia,\nirvine\nnsf-simons \ngoogle scholar(2024-07-26): citations 14981 | h-index 56 |\ni10-index 176\n\n(2021.02.17 cellchat). jin, s., guerrero-juarez, c.f.,\nzhang, l. et al. inference and analysis of cell-cell\ncommunication using cellchat. nat commun 12, 1088 (2021).\n\n(2023.02.15 neuronchat). zhao, w., johnston, k.g., ren, h.\net al. inferring neuron-neuron communications from\nsingle-cell transcriptomics through neuronchat. nat commun\n14, 1128 (2023).\n\n\n# rahul satija\n\n\nrahul satija  (new york genome center, nygc)\n (nyu)  nygc satija\n (seurat)satija\nsatija\n nih  2020  nih  satija\n\ngoogle scholar(2024-07-26): citations 75184 | h-index 61 |\ni10-index 79\n\n(2015.04.13 seurat v1). satija, r., farrell, j., gennert, d.\net al. spatial reconstruction of single-cell gene expression\ndata. nat biotechnol 33, 495502 (2015).\n\n(2018.04.02 seurat v2). butler, a., hoffman, p., smibert, p.\net al. integrating single-cell transcriptomic data across\ndifferent conditions, technologies, and species. nat\nbiotechnol 36, 411420 (2018).\n\n(2019.06.06 seurat v3). stuart t, butler a, hoffman p, et\nal. comprehensive integration of single-cell data. cell.\n2019;177(7):1888-1902.e21.\n\n(2021.06.24 seurat v4). hao y, hao s, andersen-nissen e, et\nal. integrated analysis of multimodal single-cell data.\ncell. 2021;184(13):3573-3587.e29.\n\n(2023.03.25 seurat v5). hao, y., stuart, t., kowalski, m.h.\net al. dictionary learning for integrative, multimodal and\nscalable single-cell analysis. nat biotechnol (2023).\n\n\n# edward callaway\n\n\nedward\ncallaway\n\n\n\ngoogle scholar(2024-07-26): citations 29401 | h-index 89 |\ni10-index 149\n\n(2021.10.06). zhang, z., zhou, j., tan, p. et al. epigenomic\ndiversity of cortical projection neurons in the mouse brain.\nnature 598, 167173 (2021).\n\n(2023.12.13). zhou, j., zhang, z., wu, m. et al. brain-wide\ncorrespondence of neuronal epigenomics and distant\nprojections. nature 624, 355365 (2023).\n\n\n# jun yan ()\n\n\n1994-19981998-20032003-2006200692009320159\n(2022.03.31). gao, l., liu, s., gou, l. et al. single-neuron\nprojectome of mouse prefrontal cortex. nat neurosci 25,\n515529 (2022).\n\n(2023.5.22). gao, l., liu, s., wang, y. et al. single-neuron\nanalysis of dendrites and axons reveals the network\norganization in mouse prefrontal cortex. nat neurosci\n(2023).\n\n\n# feng zhang ()\n\n\n19821022crispr-cas\ngoogle scholar(2024-07-26): citations 170259 | h-index 111 |\ni10-index 123\n\n(2016.06.28 div-seq). habib n, li y, heidenreich m. et al.\ndiv-seq: single-nucleus rna-seq reveals dynamics of rare\nadult newborn neurons. science. 2016 aug 26;353(6302):925-8.\n\n\n# yi zhang ()\n\n\n19841995fred\nrosenh3k27me3x-dna18075000h114sciencewatch2002~2006\ngoogle scholar(2024-07-26): citations 89478 | h-index 122 |\ni10-index 187\n\n(2019.09.13). bhattacherjee, a., djekidel, m.n., chen, r. et\nal. cell type-specific transcriptional programs in mouse\nprefrontal cortex during adolescence and addiction. nat\ncommun 10, 4169 (2019).\n\n(2023.10.16). bhattacherjee, a., zhang, c., watson, b.r. et\nal. spatial transcriptomics reveals the distinct\norganization of mouse prefrontal cortex and neuronal\nsubtypes regulating chronic pain. nat neurosci (2023).\n\n\n# fabian j. theis\n\n\nfabian \n2018  2019 \n(helmholtzai) fabian theis\n\nnature biotechlonogynature methodsnature\ncommunicationscell systemsmolecular systems\nbiologybioinformatics \ngoogle scholar(2024-07-26): citations 65585 | h-index 117 |\ni10-index 445\n\n(2018.02.06 scanpy). wolf, f., angerer, p. & theis, f.\nscanpy: large-scale single-cell gene expression data\nanalysis. genome biol 19, 15 (2018).\n\n(2022.01.31 squidpy). palla, g., spitzer, h., klein, m. et\nal. squidpy: a scalable framework for spatial omics\nanalysis. nat methods 19, 171178 (2022).\n\n(2023.04.10 scverse). virshup, i., bredikhin, d., heumos, l.\net al. the scverse project provides a computational\necosystem for single-cell omics data analysis. nat\nbiotechnol 41, 604606 (2023).\n\n\n# kun qu ()\n\n\n\nnaturenature\nmethodscellcancer\ncell60google80001000h-index\n36\ngoogle scholar(2024-07-26): citations 13271 | h-index 48 |\ni10-index 72\n\n(2023.11.22 spacel). xu, h., wang, s., fang, m. et al.\nspacel: deep learning-based characterization of spatial\ntranscriptome architectures. nat commun 14, 7603 (2023).\n\n\n# xiaowei zhuang ()\n\n\n19721\n1987199119199620012005200620122015\n\ngoogle scholar(2024-08-30): citations 64703 | h-index 111 |\ni10-index 198\n\n(2022.12.28 merfish). william e. allen, timothy r.\nblosser, zuri a. sullivan, et al. molecular and spatial\nsignatures of mouse brain aging at single-cell resolution.\ncell, volume 186, issue 1, 2023, pages 194-208.e18, issn\n0092-8674.\n\n\n# hongwei dong ()\n\n\nucla c57bl/6\n3d\n\n\n\ngoogle scholar(2024-08-30): citations 19834 | h-index 42 |\ni10-index 57\n\n(2021.10.06 mop-ul). munoz-castaneda, r., zingg, b., matho,\nk.s. et al. cellular anatomy of the mouse primary motor\ncortex. nature 598, 159166 (2021).',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{title:"",date:"2023-06-26T19:49:01.000Z",permalink:"/pages/11a817/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%E6%B1%87%E6%80%BB.html",relativePath:"00./03. - /01..md",key:"v-3cb81039",path:"/pages/11a817/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:13},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:22},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:32},{level:2,title:"NGS",slug:"ngs",normalizedTitle:"ngs",charIndex:41}],headersStr:"    NGS",content:"# \n\n\n# \n\n\n# \n\n\n# \n\n\n# NGS",normalizedContent:"# \n\n\n# \n\n\n# \n\n\n# \n\n\n# ngs",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03./04. - "}},title:" - ",date:"2023-05-15T16:46:19.000Z",permalink:"/pages/8b01b8/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/14.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%9B%BE%E5%83%8F%E5%A4%84%E7%90%86.html",relativePath:"00./03. - /14. - .md",key:"v-c6085372",path:"/pages/8b01b8/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/28, 19:35:41",lastUpdatedTimestamp:1687952141e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03./01. - "}},title:" - ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/967a30/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/11.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E5%92%8C%E7%A9%BA%E9%97%B4%E7%BB%84.html",relativePath:"00./03. - /11. - .md",key:"v-18d4ff6e",path:"/pages/967a30/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/28, 19:35:41",lastUpdatedTimestamp:1687952141e3},{title:" - NGS",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03./05. - NGS"}},title:" - NGS",date:"2023-05-15T16:46:33.000Z",permalink:"/pages/c3f74c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/15.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20NGS%E7%9B%B8%E5%85%B3.html",relativePath:"00./03. - /15. - NGS.md",key:"v-6c8f1043",path:"/pages/c3f74c/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/28, 19:35:41",lastUpdatedTimestamp:1687952141e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"03./03. - "}},title:" - ",date:"2023-05-15T16:45:44.000Z",permalink:"/pages/726c78/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/13.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E4%B8%89%E7%BB%B4%E5%9F%BA%E5%9B%A0%E7%BB%84.html",relativePath:"00./03. - /13. - .md",key:"v-584566ea",path:"/pages/726c78/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/28, 19:35:41",lastUpdatedTimestamp:1687952141e3},{title:" --- ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"02./11."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/17c432/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6.html",relativePath:"00./02. - /11..md",key:"v-5c47aead",path:"/pages/17c432/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" - Ajax",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/16. - Ajax"}},title:" - Ajax",date:"2024-07-26T21:48:51.000Z",permalink:"/pages/d4684b/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/16.%E5%89%8D%E7%AB%AF%20-%20Ajax.html",relativePath:"00./04.Web - /16. - Ajax.md",key:"v-5dc118b8",path:"/pages/d4684b/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - HTML",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/11. - HTML"}},title:" - HTML",date:"2023-04-30T11:05:51.000Z",permalink:"/pages/3ca8f4/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/11.%E5%89%8D%E7%AB%AF%20-%20HTML.html",relativePath:"00./04.Web - /11. - HTML.md",key:"v-1172eaec",path:"/pages/3ca8f4/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"Web",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web"}},title:"Web",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-24T20:29:46.000Z",permalink:"/pages/4efbc6/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.Web%E5%BC%80%E5%8F%91.html",relativePath:"00./04.Web - /00.Web.md",key:"v-1a2241f6",path:"/pages/4efbc6/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - JavaScript/TypeScript",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/13. - JS&TS"}},title:" - JavaScript/TypeScript",date:"2023-04-30T11:06:17.000Z",permalink:"/pages/9ec206/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/13.%E5%89%8D%E7%AB%AF%20-%20JS&TS.html",relativePath:"00./04.Web - /13. - JS&TS.md",key:"v-5ec3cdd2",path:"/pages/9ec206/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - MySQL",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/21. - MySQL"}},title:" - MySQL",date:"2024-07-26T21:50:19.000Z",permalink:"/pages/98c9ed/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/21.%E5%90%8E%E7%AB%AF%20-%20MySQL.html",relativePath:"00./04.Web - /21. - MySQL.md",key:"v-350d1abc",path:"/pages/98c9ed/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - Django",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/22. - Django"}},title:" - Django",date:"2024-07-26T21:50:22.000Z",permalink:"/pages/e1436c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/22.%E5%90%8E%E7%AB%AF%20-%20Django.html",relativePath:"00./04.Web - /22. - Django.md",key:"v-1a35b060",path:"/pages/e1436c/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - CSS",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/12. - CSS"}},title:" - CSS",date:"2023-04-30T11:05:59.000Z",permalink:"/pages/f16f8d/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/12.%E5%89%8D%E7%AB%AF%20-%20CSS.html",relativePath:"00./04.Web - /12. - CSS.md",key:"v-4e30e452",path:"/pages/f16f8d/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - Markdown",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/14. - Markdown"}},title:" - Markdown",date:"2023-04-16T21:17:16.000Z",permalink:"/pages/1abc5f/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/14.%E5%89%8D%E7%AB%AF%20-%20Markdown.html",relativePath:"00./04.Web - /14. - Markdown.md",key:"v-93263dc4",path:"/pages/1abc5f/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"05./01."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:38:52.000Z",permalink:"/pages/01057a/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/01.%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80.html",relativePath:"00./05. - /01..md",key:"v-0fb1f416",path:"/pages/01057a/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"05."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:37:25.000Z",permalink:"/pages/3c0f51/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90.html",relativePath:"00./05. - /00..md",key:"v-f77517aa",path:"/pages/3c0f51/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:" - Vue",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"04.Web/15. - Vue"}},title:" - Vue",date:"2023-04-30T11:16:00.000Z",permalink:"/pages/d2038c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/04.Web%E5%BC%80%E5%8F%91%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/15.%E5%89%8D%E7%AB%AF%20-%20Vue.html",relativePath:"00./04.Web - /15. - Vue.md",key:"v-5ec20e5c",path:"/pages/d2038c/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"05./11. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-12-04T20:46:08.000Z",permalink:"/pages/99a1ec/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/11.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E8%A5%BF%E7%93%9C%E4%B9%A6.html",relativePath:"00./05. - /11. - .md",key:"v-4312c42e",path:"/pages/99a1ec/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"06."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-28T09:38:06.000Z",permalink:"/pages/baed44/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0.html",relativePath:"00./06. - /00..md",key:"v-e52b92e2",path:"/pages/baed44/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/05/05, 10:42:51",lastUpdatedTimestamp:1683254571e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"05./12. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T09:00:26.000Z",permalink:"/pages/7daf99/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE.html",relativePath:"00./05. - /12. - .md",key:"v-31e3c08b",path:"/pages/7daf99/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"100."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:35:22.000Z",permalink:"/pages/8cb41f/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/100.%E5%85%B3%E4%BA%8E%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E5%85%B3%E4%BA%8E.html",relativePath:"00./100. - /00..md",key:"v-294d13ab",path:"/pages/8cb41f/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"07./11. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-28T09:28:26.000Z",permalink:"/pages/be5d67/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/07.%E7%94%9F%E7%89%A9%E7%9F%A5%E8%AF%86%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6%20-%20%E4%B8%81%E6%96%90.html",relativePath:"00./07. - /11. - .md",key:"v-3d52f327",path:"/pages/be5d67/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/05/05, 10:42:51",lastUpdatedTimestamp:1683254571e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"100./01. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:36:17.000Z",permalink:"/pages/13ae7d/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/100.%E5%85%B3%E4%BA%8E%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/01.%E5%85%B3%E4%BA%8E%20-%20%E8%87%AA%E6%88%91.html",relativePath:"00./100. - /01. - .md",key:"v-5bcc7c9e",path:"/pages/13ae7d/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"R ",frontmatter:{title:"R ",date:"2023-06-26T19:11:03.000Z",permalink:"/pages/9ab953/",categories:["R"],tags:[null],readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/10.R%20%E6%B1%87%E6%80%BB.html",relativePath:"00./06. - /10.R .md",key:"v-490122ed",path:"/pages/9ab953/",headers:[{level:2,title:"R books",slug:"r-books",normalizedTitle:"r books",charIndex:2},{level:2,title:"R tidyverse",slug:"r-tidyverse",normalizedTitle:"r tidyverse",charIndex:535},{level:2,title:"R ",slug:"r-",normalizedTitle:"r ",charIndex:551},{level:2,title:"R ",slug:"r-",normalizedTitle:"r ",charIndex:560}],headersStr:"R books R tidyverse R  R ",content:'# R books\n\nR for Data Science\n\n\n\nR Packages\n\n\n\nconfig:\n    target: _blank\n    imgHeight: 250px\n    objectFit: cover\n    lineClamp: 1\n\ndata:\n  - img: https://zhenghu159.github.io/r4ds-zh-cn/cover.jpg\n    link: https://zhenghu159.github.io/r4ds-zh-cn/\n    name: <div align="center"> R for Data Science </div>\n\n  - img: https://zhenghu159.github.io/r-pkgs-zh-cn/images/cover-2e-small.png\n    link: https://zhenghu159.github.io/r-pkgs-zh-cn/\n    name: <div align="center"> R Packages </div>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n\n\n\n# R tidyverse\n\n\n# R \n\n\n# R ',normalizedContent:'# r books\n\nr for data science\n\n\n\nr packages\n\n\n\nconfig:\n    target: _blank\n    imgheight: 250px\n    objectfit: cover\n    lineclamp: 1\n\ndata:\n  - img: https://zhenghu159.github.io/r4ds-zh-cn/cover.jpg\n    link: https://zhenghu159.github.io/r4ds-zh-cn/\n    name: <div align="center"> r for data science </div>\n\n  - img: https://zhenghu159.github.io/r-pkgs-zh-cn/images/cover-2e-small.png\n    link: https://zhenghu159.github.io/r-pkgs-zh-cn/\n    name: <div align="center"> r packages </div>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n\n\n\n# r tidyverse\n\n\n# r \n\n\n# r ',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - ",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"100./02. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:36:26.000Z",permalink:"/pages/2b095a/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/100.%E5%85%B3%E4%BA%8E%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99.html",relativePath:"00./100. - /02. - .md",key:"v-0e096556",path:"/pages/2b095a/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"07."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:37:25.000Z",permalink:"/pages/634e82/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/07.%E7%94%9F%E7%89%A9%E7%9F%A5%E8%AF%86%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E7%94%9F%E7%89%A9%E7%9F%A5%E8%AF%86.html",relativePath:"00./07. - /00..md",key:"v-79611a61",path:"/pages/634e82/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/05/05, 10:42:51",lastUpdatedTimestamp:1683254571e3},{title:"",frontmatter:{pageComponent:{name:"Catalogue",data:{path:"99."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-23T21:15:10.000Z",permalink:"/pages/d28eb7/",readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/99.%E7%94%9F%E6%B4%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/00.%E7%94%9F%E6%B4%BB.html",relativePath:"00./99. - /00..md",key:"v-14a9ddeb",path:"/pages/d28eb7/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{title:"",date:"2023-08-26T15:12:36.000Z",permalink:"/pages/1a58a6/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/00.%E7%9B%AE%E5%BD%95%E9%A1%B5/99.%E7%94%9F%E6%B4%BB%20-%20%E7%9B%AE%E5%BD%95%E9%A1%B5/03.%E9%9F%B3%E4%B9%90.html",relativePath:"00./99. - /03..md",key:"v-98790e58",path:"/pages/1a58a6/",headersStr:null,content:' --- 2023.05.28 \n\n\n\n --- 2017.10.12 AZA\n\n\n\nconfig:\n    target: _blank\n    imgHeight: 180px\n    objectFit: cover\n    lineClamp: 10\n\ndata:\n  - img: https://tse1.mm.bing.net/th?&id=OVP.KpXWvOqSeCk8z_-J5ZMuaQIIFF&w=330&h=186&c=7&pid=6.1&rs=1\n    link: /pages/46a15a/\n    name: <div align="center"><strong> --- 2023.05.28 </strong></div>\n\n  - img: https://tse1.mm.bing.net/th?&id=OVP.tPXiz4o-Ae88Ohuf2gYRQQIIFF&w=197&h=110&c=7&pid=1.7&rs=1\n    link: /pages/aef061/\n    name: <div align="center"><strong> --- 2017.10.12 AZA</strong></div>\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n',normalizedContent:' --- 2023.05.28 \n\n\n\n --- 2017.10.12 aza\n\n\n\nconfig:\n    target: _blank\n    imgheight: 180px\n    objectfit: cover\n    lineclamp: 10\n\ndata:\n  - img: https://tse1.mm.bing.net/th?&id=ovp.kpxwvoqseck8z_-j5zmuaqiiff&w=330&h=186&c=7&pid=6.1&rs=1\n    link: /pages/46a15a/\n    name: <div align="center"><strong> --- 2023.05.28 </strong></div>\n\n  - img: https://tse1.mm.bing.net/th?&id=ovp.tpxiz4o-ae88ohuf2gyrqqiiff&w=197&h=110&c=7&pid=1.7&rs=1\n    link: /pages/aef061/\n    name: <div align="center"><strong> --- 2017.10.12 aza</strong></div>\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"OneNote",frontmatter:{title:"OneNote",date:"2023-09-03T17:03:37.000Z",permalink:"/pages/1260b7/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/01.%E7%A7%91%E7%A0%94%E6%8A%80%E8%83%BD/001.OneNote.html",relativePath:"01./001.OneNote.md",key:"v-39f4cdd0",path:"/pages/1260b7/",headers:[{level:2,title:"OneNote ",slug:"onenote-",normalizedTitle:"onenote ",charIndex:59},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:78},{level:2,title:"OneNote ",slug:"onenote-",normalizedTitle:"onenote ",charIndex:282},{level:2,title:"OneNote ",slug:"onenote-",normalizedTitle:"onenote ",charIndex:462},{level:2,title:"OneNote ",slug:"onenote-",normalizedTitle:"onenote ",charIndex:642},{level:3,title:"1 ",slug:"_1--",normalizedTitle:"1 ",charIndex:657},{level:3,title:"2 ",slug:"_2--",normalizedTitle:"2 ",charIndex:874},{level:3,title:"3 ",slug:"_3--",normalizedTitle:"3 ",charIndex:962},{level:3,title:"4 ",slug:"_4-",normalizedTitle:"4 ",charIndex:1031},{level:3,title:"5 ",slug:"_5-",normalizedTitle:"5 ",charIndex:1137},{level:3,title:"6 ",slug:"_6-",normalizedTitle:"6 ",charIndex:1397},{level:3,title:"7  OneNote",slug:"_7--onenote",normalizedTitle:"7  onenote",charIndex:1570},{level:3,title:"8 GPT + OneNote  AI ",slug:"_8-gpt-onenote--ai-",normalizedTitle:"8 gpt + onenote  ai ",charIndex:1744}],excerpt:'<div class="custom-block note"><p class="custom-block-title"></p>\n<p><a href="https://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&amp;mid=2247484689&amp;idx=1&amp;sn=7ea7148f7cd33dbf74cf14990c6c7fd1&amp;chksm=c0a7b5ccf7d03cda22c898da96d2b386584529487ec89dcef8449a7510048b5811be91f02fb3&amp;token=1924756988&amp;lang=zh_CN#rd" target="_blank" rel="noopener noreferrer">ChatGPT  OneNote <OutboundLink/></a></p>\n</div>\n',headersStr:"OneNote   OneNote  OneNote  OneNote  1  2  3  4  5  6  7  OneNote 8 GPT + OneNote  AI ",content:"\n\nChatGPT  OneNote \n\n\n# OneNote \n\n\n\n\n# \n\n OneNote  OneNote  OneNote  OneNote \n\n\n# OneNote \n\nOneNote  Microsoft  Office 365 OneNote webOneNote \n\n\n# OneNote \n\n windows OneNote  App Store  windows10  OneNote Microsoft Store  onenote for windows10 \n\n\n# OneNote \n\n\n# 1 \n\n\n\nOneNote  R  tidyverse  R  ggplot2  ggplot2  G  ggplot() \n\n\n# 2 \n\n\n\nOneNote  Office \n\n\n# 3 \n\n\n\nOneNote \n\n\n# 4 \n\n\n\nOneNote  VSCode \n\n\n# 5 \n\n\n\nOneNote  Ctrl+FOneNote \n\n\n\n OneNote \n\n\n# 6 \n\n\n\nOneNote [][]\n\n\n# 7  OneNote\n\n\n\nOneNote ipad  OneNote ipad  pencil  OneNote \n\n\n# 8 GPT + OneNote  AI \n\n ChatGPT  Microsoft 365  AI  OneNote  AI ...",normalizedContent:"\n\nchatgpt  onenote \n\n\n# onenote \n\n\n\n\n# \n\n onenote  onenote  onenote  onenote \n\n\n# onenote \n\nonenote  microsoft  office 365 onenote webonenote \n\n\n# onenote \n\n windows onenote  app store  windows10  onenote microsoft store  onenote for windows10 \n\n\n# onenote \n\n\n# 1 \n\n\n\nonenote  r  tidyverse  r  ggplot2  ggplot2  g  ggplot() \n\n\n# 2 \n\n\n\nonenote  office \n\n\n# 3 \n\n\n\nonenote \n\n\n# 4 \n\n\n\nonenote  vscode \n\n\n# 5 \n\n\n\nonenote  ctrl+fonenote \n\n\n\n onenote \n\n\n# 6 \n\n\n\nonenote [][]\n\n\n# 7  onenote\n\n\n\nonenote ipad  onenote ipad  pencil  onenote \n\n\n# 8 gpt + onenote  ai \n\n chatgpt  microsoft 365  ai  onenote  ai ...",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{title:"",date:"2022-01-04T20:02:46.000Z",permalink:"/navigation/",article:!1,categories:[null],tags:[null],readingShow:"top"},regularPath:"/01.%E5%AF%BC%E8%88%AA%E7%AB%99.html",relativePath:"01..md",key:"v-a89b5f26",path:"/navigation/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:305},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1319},{level:3,title:"-",slug:"-",normalizedTitle:"-",charIndex:1328},{level:3,title:"NGS ",slug:"ngs-",normalizedTitle:"ngs ",charIndex:1342},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1355},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1365},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1376},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1385},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1394},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1403},{level:2,title:"Python ",slug:"python-",normalizedTitle:"python ",charIndex:1413},{level:2,title:"R ",slug:"r-",normalizedTitle:"r ",charIndex:1428},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1438},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1447},{level:3,title:"Vue",slug:"vue",normalizedTitle:"vue",charIndex:1456}],headersStr:"   - NGS        Python  R    Vue",content:"# \n\nXjgou's blog\n\n\n\n- name: Xjgou's blog\n  desc: \n  avatar: https://gouxiaojuan.github.io/authors/admin/avatar_hu57dd110a617e3774f481ac89b955aa93_44615_270x270_fill_q75_lanczos_center.jpg\n  link: https://gouxiaojuan.github.io/\n  bgColor: '#FFCEDE'\n  textColor: '#621529'\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n# \n\nEvan's blog\n\n\n\nYoung Kbt blog\n\n\n\nYihui's Blog\n\nR  statistics  life  random\n\n- name: Evan's blog\n  desc: \n  avatar: https://jsd.cdn.zzko.cn/gh/xugaoyi/image_store/blog/20200103123203.jpg\n  link: https://xugaoyi.com/\n  bgColor: '#B9D59C'\n  textColor: '#3B551F'\n\n- name: Young Kbt blog\n  desc: \n  avatar: https://cdn.jsdelivr.net/gh/Kele-Bingtang/static/user/avatar2.png\n  link: https://notes.youngkbt.cn/\n  bgColor: '#FFB6C1'\n  textColor: '#621529'\n\n- name: Yihui's Blog\n  desc: R  statistics  life  random\n  avatar: https://avatars.githubusercontent.com/u/163582?v=4\n  link: https://yihui.org/\n  bgColor: '#fce3c6'\n  textColor: '#07323c'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n\n\n   ()\n\n- name: TigerZ # \n  desc:  # \n  avatar: https://zhenghu159.github.io/avatar.jpg # \n  link: https://zhenghu159.github.io/  # \n  bgColor: '#FFB6C1'  # \n  textColor: '#621529'  # \n\n\n1\n2\n3\n4\n5\n6\n\n\n~\n\n\n# \n\n\n# -\n\n\n# NGS \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# Python \n\n\n# R \n\n\n# \n\n\n# \n\n\n# Vue",normalizedContent:"# \n\nxjgou's blog\n\n\n\n- name: xjgou's blog\n  desc: \n  avatar: https://gouxiaojuan.github.io/authors/admin/avatar_hu57dd110a617e3774f481ac89b955aa93_44615_270x270_fill_q75_lanczos_center.jpg\n  link: https://gouxiaojuan.github.io/\n  bgcolor: '#ffcede'\n  textcolor: '#621529'\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n# \n\nevan's blog\n\n\n\nyoung kbt blog\n\n\n\nyihui's blog\n\nr  statistics  life  random\n\n- name: evan's blog\n  desc: \n  avatar: https://jsd.cdn.zzko.cn/gh/xugaoyi/image_store/blog/20200103123203.jpg\n  link: https://xugaoyi.com/\n  bgcolor: '#b9d59c'\n  textcolor: '#3b551f'\n\n- name: young kbt blog\n  desc: \n  avatar: https://cdn.jsdelivr.net/gh/kele-bingtang/static/user/avatar2.png\n  link: https://notes.youngkbt.cn/\n  bgcolor: '#ffb6c1'\n  textcolor: '#621529'\n\n- name: yihui's blog\n  desc: r  statistics  life  random\n  avatar: https://avatars.githubusercontent.com/u/163582?v=4\n  link: https://yihui.org/\n  bgcolor: '#fce3c6'\n  textcolor: '#07323c'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n\n\n   ()\n\n- name: tigerz # \n  desc:  # \n  avatar: https://zhenghu159.github.io/avatar.jpg # \n  link: https://zhenghu159.github.io/  # \n  bgcolor: '#ffb6c1'  # \n  textcolor: '#621529'  # \n\n\n1\n2\n3\n4\n5\n6\n\n\n~\n\n\n# \n\n\n# -\n\n\n# ngs \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# \n\n\n# python \n\n\n# r \n\n\n# \n\n\n# \n\n\n# vue",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"NCBI",frontmatter:{title:"NCBI",date:"2024-07-26T21:08:20.000Z",permalink:"/pages/420e29/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/01.%E7%A7%91%E7%A0%94%E6%8A%80%E8%83%BD/003.NCBI.html",relativePath:"01./003.NCBI.md",key:"v-e1b6b19c",path:"/pages/420e29/",excerpt:'<div class="custom-block note"><p class="custom-block-title"></p>\n<p></p>\n<ol>\n<li><a href="https://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&amp;mid=2247487850&amp;idx=1&amp;sn=65690b84d806ecafc5e7e4454e3a5a95&amp;chksm=c0a7a1b7f7d028a1f25a3c8fc382482c43eb8e9162bec6dc0b9f827ef014e78302b16023ba98&amp;scene=178&amp;cur_album_id=3089670478165966852#rd" target="_blank" rel="noopener noreferrer"> GEO --- <OutboundLink/></a></li>\n<li><a href="https://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&amp;mid=2247487888&amp;idx=1&amp;sn=150d8b97eeedf551d8913eb7bec5b885&amp;chksm=c0a7a14df7d0285b71539109e02649856e2ed7dcb287420817d5f9fe4575ee56285ed0216553&amp;scene=178&amp;cur_album_id=3089670478165966852#rd" target="_blank" rel="noopener noreferrer"> SRA --- <OutboundLink/></a></li>\n<li><a href="https://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&amp;mid=2247487978&amp;idx=1&amp;sn=2a3875f1bf25e33513afd9fa0f47237d&amp;chksm=c0a7a137f7d02821a64db37dd499c0c4c95ef5f7d88cd497bdbe2992e6a0120a4e7c2f292851&amp;scene=178&amp;cur_album_id=3089670478165966852#rd" target="_blank" rel="noopener noreferrer"> GEO --- <OutboundLink/></a></li>\n</ol>\n</div>\n',headersStr:null,content:"\n\n\n\n 1.  GEO --- \n 2.  SRA --- \n 3.  GEO --- ",normalizedContent:"\n\n\n\n 1.  geo --- \n 2.  sra --- \n 3.  geo --- ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Transcriptional control of long-range cortical projections",frontmatter:{title:" --- Transcriptional control of long-range cortical projections",date:"2023-08-10T20:33:21.000Z",permalink:"/pages/7100c2/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/01.%E7%BB%BC%E8%BF%B0/20180609.(2018.06.09%20%E7%9A%AE%E5%B1%82%E6%8A%95%E5%B0%84%E8%BD%AC%E5%BD%95%E8%B0%83%E6%8E%A7%20Curr%20Opin%20Neurobiol)%20Transcriptional%20control%20of%20long-range%20cortical%20projections.html",relativePath:"02./11./01./20180609.(2018.06.09  Curr Opin Neurobiol) Transcriptional control of long-range cortical projections.md",key:"v-6c95ab70",path:"/pages/7100c2/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:431},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:705},{level:3,title:"Table 1. Transcriptional regulators differentially expressed in long-range projection neurons, based on protein and/or mRNA detection. L, neocortical layer",slug:"table-1-transcriptional-regulators-differentially-expressed-in-long-range-projection-neurons-based-on-protein-and-or-mrna-detection-l-neocortical-layer",normalizedTitle:"table 1. transcriptional regulators differentially expressed in long-range projection neurons, based on protein and/or mrna detection. l, neocortical layer",charIndex:717},{level:3,title:"Figure 1. Network of transcriptional regulators (bold) and axon guidance cue interactions",slug:"figure-1-network-of-transcriptional-regulators-bold-and-axon-guidance-cue-interactions",normalizedTitle:"figure 1. network of transcriptional regulators (bold) and axon guidance cue interactions",charIndex:1333},{level:3,title:"Table 2. Summary of axon guidance genes",slug:"table-2-summary-of-axon-guidance-genes",normalizedTitle:"table 2. summary of axon guidance genes",charIndex:1726}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.conb.2018.05.005" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.conb.2018.05.005<OutboundLink/></a><br>\nCite: Paolino A, Fenlon LR, Surez R, Richards LJ. Transcriptional control of long-range cortical projections. Curr Opin Neurobiol. 2018;53:57-65.</p>\n',headersStr:"Abstract Results Table 1. Transcriptional regulators differentially expressed in long-range projection neurons, based on protein and/or mRNA detection. L, neocortical layer Figure 1. Network of transcriptional regulators (bold) and axon guidance cue interactions Table 2. Summary of axon guidance genes",content:": \nDOI: https://doi.org/10.1016/j.conb.2018.05.005\nCite: Paolino A, Fenlon LR, Surez R, Richards LJ. Transcriptional control of long-range cortical projections. Curr Opin Neurobiol. 2018;53:57-65.\n\n\n\n\n\n\n\nLINDA J RICHARDS             RODRIGO SUREZ\n                             \nQueensland Brain Institute   Queensland Brain Institute\nrichards@uq.edu.au           r.suarez@uq.edu.au\n\n\n# Abstract\n\nguidance cuesguidance decisions DNA \n\n\n# Results\n\n\n# Table 1. Transcriptional regulators differentially expressed in long-range projection neurons, based on protein and/or mRNA detection. L, neocortical layer\n\n\n\nTable 1. / mRNA L\n\nretrograde tracers\n\n 1. -corticocortical projection neurons L2/3  L5\n 2. subcerebral projection neurons L5\n 3. -corticothalamic projection neurons L6\n\nL5 corticocortical and subcerebral neurons \n\n\n# Figure 1. Network of transcriptional regulators (bold) and axon guidance cue interactions\n\n\n\nFigure 1. \n\ncueCPN: callosal projection neurons, CThPN: corticothalamic projection neurons, SCPN: subcerebral projection neurons\n\n\n# Table 2. Summary of axon guidance genes\n\n\n\nTable 2. \n\ncorticocortical (SATB2highCTIP2lowTLE4low)subcerebral (SATB2lowCTIP2highTLE4low)corticothalamic (SATB2lowCTIP2medTLE4high) projection neurons (adapted from Molyneaux et al. 2015) ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.conb.2018.05.005\ncite: paolino a, fenlon lr, suarez r, richards lj. transcriptional control of long-range cortical projections. curr opin neurobiol. 2018;53:57-65.\n\n\n\n\n\n\n\nlinda j richards             rodrigo suarez\n                             \nqueensland brain institute   queensland brain institute\nrichards@uq.edu.au           r.suarez@uq.edu.au\n\n\n# abstract\n\nguidance cuesguidance decisions dna \n\n\n# results\n\n\n# table 1. transcriptional regulators differentially expressed in long-range projection neurons, based on protein and/or mrna detection. l, neocortical layer\n\n\n\ntable 1. / mrna l\n\nretrograde tracers\n\n 1. -corticocortical projection neurons l2/3  l5\n 2. subcerebral projection neurons l5\n 3. -corticothalamic projection neurons l6\n\nl5 corticocortical and subcerebral neurons \n\n\n# figure 1. network of transcriptional regulators (bold) and axon guidance cue interactions\n\n\n\nfigure 1. \n\ncuecpn: callosal projection neurons, cthpn: corticothalamic projection neurons, scpn: subcerebral projection neurons\n\n\n# table 2. summary of axon guidance genes\n\n\n\ntable 2. \n\ncorticocortical (satb2highctip2lowtle4low)subcerebral (satb2lowctip2hightle4low)corticothalamic (satb2lowctip2medtle4high) projection neurons (adapted from molyneaux et al. 2015) ",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"Snipaste",frontmatter:{title:"Snipaste",date:"2023-09-04T21:01:39.000Z",permalink:"/pages/413897/",categories:[""],tags:[null],readingShow:"top"},regularPath:"/01.%E7%A7%91%E7%A0%94%E6%8A%80%E8%83%BD/002.Snipaste.html",relativePath:"01./002.Snipaste.md",key:"v-c7d2b644",path:"/pages/413897/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:2,title:"Snipaste ",slug:"snipaste-",normalizedTitle:"snipaste ",charIndex:139},{level:2,title:"Snipaste ",slug:"snipaste-",normalizedTitle:"snipaste ",charIndex:358}],excerpt:'<h2 id=""><a class="header-anchor" href="#">#</a> </h2>\n<p> QQ  <code>Ctrl+Alt+A</code>  QQ  RGB Snipaste</p>\n',headersStr:" Snipaste  Snipaste ",content:"# \n\n QQ  Ctrl+Alt+A  QQ  RGB Snipaste\n\n\n# Snipaste \n\nSnipaste Snipaste \n\n\n# Snipaste \n\n https://zh.snipaste.com/",normalizedContent:"# \n\n qq  ctrl+alt+a  qq  rgb snipaste\n\n\n# snipaste \n\nsnipaste snipaste \n\n\n# snipaste \n\n https://zh.snipaste.com/",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- The Role of Medial Prefrontal Cortex in Memory and Decision Making",frontmatter:{title:" --- The Role of Medial Prefrontal Cortex in Memory and Decision Making",date:"2023-03-09T21:25:42.000Z",permalink:"/pages/1aae84/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/01.%E7%BB%BC%E8%BF%B0/20121220.(2012.12.20%20mPFC%20%E7%BB%BC%E8%BF%B0%20Neuron)%20The%20Role%20of%20Medial%20Prefrontal%20Cortex%20in%20Memory%20and%20Decision%20Making.html",relativePath:"02./11./01./20121220.(2012.12.20 mPFC  Neuron) The Role of Medial Prefrontal Cortex in Memory and Decision Making.md",key:"v-7258202a",path:"/pages/1aae84/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:551},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:881},{level:3,title:"Figure 1. mPFC Encodes Expectation of Positive Outcomes",slug:"figure-1-mpfc-encodes-expectation-of-positive-outcomes",normalizedTitle:"figure 1. mpfc encodes expectation of positive outcomes",charIndex:894},{level:3,title:"Figure 2. mPFC Encodes Expectation of Negative Outcomes",slug:"figure-2-mpfc-encodes-expectation-of-negative-outcomes",normalizedTitle:"figure 2. mpfc encodes expectation of negative outcomes",charIndex:1459},{level:3,title:"Figure 3. Major Anatomical Connections of Ventral mPFC",slug:"figure-3-major-anatomical-connections-of-ventral-mpfc",normalizedTitle:"figure 3. major anatomical connections of ventral mpfc",charIndex:2083},{level:3,title:"Figure 4. Schematic View of Hypothesized Inputs to and Outputs from Different Regions of Prefrontal Cortex",slug:"figure-4-schematic-view-of-hypothesized-inputs-to-and-outputs-from-different-regions-of-prefrontal-cortex",normalizedTitle:"figure 4. schematic view of hypothesized inputs to and outputs from different regions of prefrontal cortex",charIndex:2258},{level:3,title:"Figure 5. Proposed Timeline of mPFC's Role in Memory",slug:"figure-5-proposed-timeline-of-mpfc-s-role-in-memory",normalizedTitle:"figure 5. proposed timeline of mpfc's role in memory",charIndex:2548}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.neuron.2012.12.002" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.neuron.2012.12.002<OutboundLink/></a><br>\nCite: Euston DR, Gruber AJ, McNaughton BL. The role of medial prefrontal cortex in memory and decision making. Neuron. 2012 Dec 20;76(6):1057-70.</p>\n',headersStr:"Abstract: Results: Figure 1. mPFC Encodes Expectation of Positive Outcomes Figure 2. mPFC Encodes Expectation of Negative Outcomes Figure 3. Major Anatomical Connections of Ventral mPFC Figure 4. Schematic View of Hypothesized Inputs to and Outputs from Different Regions of Prefrontal Cortex Figure 5. Proposed Timeline of mPFC's Role in Memory",content:": \nDOI: https://doi.org/10.1016/j.neuron.2012.12.002\nCite: Euston DR, Gruber AJ, McNaughton BL. The role of medial prefrontal cortex in memory and decision making. Neuron. 2012 Dec 20;76(6):1057-70.\n\n\n\n\n\n\n\nDAVID EUSTON               AARON GRUBER               BRUCE MCNAUGHTON\n                                                      \nUniversity of Lethbridge   University of Lethbridge   University of Lethbridge\neuston@uleth.ca            aaron.gruber@uleth.ca      bruce.mcnaughton@uleth.ca\n\n\n# Abstract:\n\nmPFC decision makingmPFC mPFC mPFC contextmPFC  mPFC mPFC \n\n\n# Results:\n\n\n# Figure 1. mPFC Encodes Expectation of Positive Outcomes\n\n\n\nFigure 1. mPFC \n\n(A) subjective valuemPFC ventral striatumposterior cingulate cortex t \n(B) Fos-positive prelimbicinfralimbiclateral orbital cortex\n(C)  mPFC  0  2 \n\n\n# Figure 2. mPFC Encodes Expectation of Negative Outcomes\n\n\n\nFigure 2. mPFC \n\n(A)  192  mPFC \n(B) \n(C) mPFC  Box A CS 500 US Box B CS  US  z score CS 1,800mPFC  Box B  CS \n\n\n# Figure 3. Major Anatomical Connections of Ventral mPFC\n\n\n\nFigure 3.  mPFC \n\n mPFC \n\n\n# Figure 4. Schematic View of Hypothesized Inputs to and Outputs from Different Regions of Prefrontal Cortex\n\n\n\nFigure 4. \n\nmPFC \n\n\n# Figure 5. Proposed Timeline of mPFC's Role in Memory\n\n\n\nFigure 5. mPFC \n\n mPFC  30  mPFC  mPFC ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.neuron.2012.12.002\ncite: euston dr, gruber aj, mcnaughton bl. the role of medial prefrontal cortex in memory and decision making. neuron. 2012 dec 20;76(6):1057-70.\n\n\n\n\n\n\n\ndavid euston               aaron gruber               bruce mcnaughton\n                                                      \nuniversity of lethbridge   university of lethbridge   university of lethbridge\neuston@uleth.ca            aaron.gruber@uleth.ca      bruce.mcnaughton@uleth.ca\n\n\n# abstract:\n\nmpfc decision makingmpfc mpfc mpfc contextmpfc  mpfc mpfc \n\n\n# results:\n\n\n# figure 1. mpfc encodes expectation of positive outcomes\n\n\n\nfigure 1. mpfc \n\n(a) subjective valuempfc ventral striatumposterior cingulate cortex t \n(b) fos-positive prelimbicinfralimbiclateral orbital cortex\n(c)  mpfc  0  2 \n\n\n# figure 2. mpfc encodes expectation of negative outcomes\n\n\n\nfigure 2. mpfc \n\n(a)  192  mpfc \n(b) \n(c) mpfc  box a cs 500 us box b cs  us  z score cs 1,800mpfc  box b  cs \n\n\n# figure 3. major anatomical connections of ventral mpfc\n\n\n\nfigure 3.  mpfc \n\n mpfc \n\n\n# figure 4. schematic view of hypothesized inputs to and outputs from different regions of prefrontal cortex\n\n\n\nfigure 4. \n\nmpfc \n\n\n# figure 5. proposed timeline of mpfc's role in memory\n\n\n\nfigure 5. mpfc \n\n mpfc  30  mpfc  mpfc ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA",frontmatter:{title:" --- High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA",date:"2022-12-09T20:11:32.000Z",permalink:"/pages/dc1352/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20160718.(2016.07.18%20MAPseq%20Neuron)%20High-Throughput%20Mapping%20of%20Single-Neuron%20Projections%20by%20Sequencing%20of%20Barcoded%20RNA.html",relativePath:"02./11./02./20160718.(2016.07.18 MAPseq Neuron) High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA.md",key:"v-21cb322c",path:"/pages/dc1352/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:414},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:795},{level:3,title:"Fig1. Barcoding Allows High-Throughput Single-Neuron Tracing",slug:"fig1-barcoding-allows-high-throughput-single-neuron-tracing",normalizedTitle:"fig1. barcoding allows high-throughput single-neuron tracing",charIndex:807},{level:3,title:"Fig2. Barcoded Sindbis Virus Can Be Used for Projection Mapping",slug:"fig2-barcoded-sindbis-virus-can-be-used-for-projection-mapping",normalizedTitle:"fig2. barcoded sindbis virus can be used for projection mapping",charIndex:1090},{level:3,title:"Fig3. Random Labeling of Neurons with a Barcoded Virus Library Can Achieve Unique Labeling of Many Neurons",slug:"fig3-random-labeling-of-neurons-with-a-barcoded-virus-library-can-achieve-unique-labeling-of-many-neurons",normalizedTitle:"fig3. random labeling of neurons with a barcoded virus library can achieve unique labeling of many neurons",charIndex:1537},{level:3,title:"Fig4. MAPseq Reveals Large Diversity of Projections from LC",slug:"fig4-mapseq-reveals-large-diversity-of-projections-from-lc",normalizedTitle:"fig4. mapseq reveals large diversity of projections from lc",charIndex:2066},{level:3,title:"Fig5. MAPseq Provides a Robust Readout of Single-Neuron Projection Patterns",slug:"fig5-mapseq-provides-a-robust-readout-of-single-neuron-projection-patterns",normalizedTitle:"fig5. mapseq provides a robust readout of single-neuron projection patterns",charIndex:2271},{level:3,title:"Fig6. LC Neurons Tile Cortex with Their Maximum Projections, but Innervate Large Areas of Cortex at a Low Level",slug:"fig6-lc-neurons-tile-cortex-with-their-maximum-projections-but-innervate-large-areas-of-cortex-at-a-low-level",normalizedTitle:"fig6. lc neurons tile cortex with their maximum projections, but innervate large areas of cortex at a low level",charIndex:2534},{level:3,title:"Fig7. MAPseq Can Be Multiplexed to Several Injection Sites",slug:"fig7-mapseq-can-be-multiplexed-to-several-injection-sites",normalizedTitle:"fig7. mapseq can be multiplexed to several injection sites",charIndex:2818}],excerpt:'<p>:  RNA <br>\nDOI: <a href="https://doi.org/10.1016/j.neuron.2016.07.036" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.neuron.2016.07.036<OutboundLink/></a><br>\nCite: Kebschull, J. M., Garcia da Silva, P., Reid, A. P., Peikon, I. D., Albeanu, D. F., &amp; Zador, A. M. (2016). High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA. <em>Neuron</em>, 91(5), 975-987.</p>\n',headersStr:"Abstract Results Fig1. Barcoding Allows High-Throughput Single-Neuron Tracing Fig2. Barcoded Sindbis Virus Can Be Used for Projection Mapping Fig3. Random Labeling of Neurons with a Barcoded Virus Library Can Achieve Unique Labeling of Many Neurons Fig4. MAPseq Reveals Large Diversity of Projections from LC Fig5. MAPseq Provides a Robust Readout of Single-Neuron Projection Patterns Fig6. LC Neurons Tile Cortex with Their Maximum Projections, but Innervate Large Areas of Cortex at a Low Level Fig7. MAPseq Can Be Multiplexed to Several Injection Sites",content:':  RNA \nDOI: https://doi.org/10.1016/j.neuron.2016.07.036\nCite: Kebschull, J. M., Garcia da Silva, P., Reid, A. P., Peikon, I. D., Albeanu, D. F., & Zador, A. M. (2016). High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA. Neuron, 91(5), 975-987.\n\n\n\n RNA \n\n\n\nANTHONY ZADOR\n\nCold Spring Harbor Laboratory\nzador@cshl.edu\n\n\n# Abstract\n\nMAPseqMultiplexed Analysis of Projections by SequencingRNA"barcodes"mRNA mRNA  MAPseq ( locus coeruleusLC) LC MAPseq \n\n\n# Results\n\n\n# Fig1. Barcoding Allows High-Throughput Single-Neuron Tracing\n\n\n\nFig1. \n\nA. Allen  bulk mapping \nB. MAPseq  Barcode \nC. \nD. MAPseq MOI  Barcode Barcode mRNA \n\n\n# Fig2. Barcoded Sindbis Virus Can Be Used for Projection Mapping\n\n\n\nFig2. Barcoded Sindbis \n\nA.  Sindbis  barcodes  GFPbarcodes  MAPP-n\nB. () ()  Barcode RNA()  GFP() \nC. LC  (frontal cortices)  (visual cortex) \nD.  LC  MAPseq \nE.  barcode mRNA Bulk projection strength  barcode \nF.  barcode \n\n\n# Fig3. Random Labeling of Neurons with a Barcoded Virus Library Can Achieve Unique Labeling of Many Neurons\n\n\n\nFig3. \n\nA.  barcode MAPseq \nB.  barcode  45  barcode 21%  barcode\nC.  barcode MAPseq \nD.  Sindbis virus libraries  ~101810^{18}10 18  ~10610^610 6 \nE.  barcode  10610^610 6  barcode \nF. \n\n\n# Fig4. MAPseq Reveals Large Diversity of Projections from LC\n\n\n\nFig4. MAPseq  LC \n\nA.  RNA \nB-D.  MAPseq  22  995 barcodes D 6  barcodes \n\n\n# Fig5. MAPseq Provides a Robust Readout of Single-Neuron Projection Patterns\n\n\n\nFig5. MAPseq \n\nA. MAPseq\nB. \n\n\n# Fig6. LC Neurons Tile Cortex with Their Maximum Projections, but Innervate Large Areas of Cortex at a Low Level\n\n\n\nFig6. LC \n\nA.  995 \nB. \nC.  LC \n\n\n# Fig7. MAPseq Can Be Multiplexed to Several Injection Sites\n\n\n\nFig7. MAPseq \n\nA.  barcoded Sindbis \nB. \nC. LC',normalizedContent:':  rna \ndoi: https://doi.org/10.1016/j.neuron.2016.07.036\ncite: kebschull, j. m., garcia da silva, p., reid, a. p., peikon, i. d., albeanu, d. f., & zador, a. m. (2016). high-throughput mapping of single-neuron projections by sequencing of barcoded rna. neuron, 91(5), 975-987.\n\n\n\n rna \n\n\n\nanthony zador\n\ncold spring harbor laboratory\nzador@cshl.edu\n\n\n# abstract\n\nmapseqmultiplexed analysis of projections by sequencingrna"barcodes"mrna mrna  mapseq ( locus coeruleuslc) lc mapseq \n\n\n# results\n\n\n# fig1. barcoding allows high-throughput single-neuron tracing\n\n\n\nfig1. \n\na. allen  bulk mapping \nb. mapseq  barcode \nc. \nd. mapseq moi  barcode barcode mrna \n\n\n# fig2. barcoded sindbis virus can be used for projection mapping\n\n\n\nfig2. barcoded sindbis \n\na.  sindbis  barcodes  gfpbarcodes  mapp-n\nb. () ()  barcode rna()  gfp() \nc. lc  (frontal cortices)  (visual cortex) \nd.  lc  mapseq \ne.  barcode mrna bulk projection strength  barcode \nf.  barcode \n\n\n# fig3. random labeling of neurons with a barcoded virus library can achieve unique labeling of many neurons\n\n\n\nfig3. \n\na.  barcode mapseq \nb.  barcode  45  barcode 21%  barcode\nc.  barcode mapseq \nd.  sindbis virus libraries  ~101810^{18}10 18  ~10610^610 6 \ne.  barcode  10610^610 6  barcode \nf. \n\n\n# fig4. mapseq reveals large diversity of projections from lc\n\n\n\nfig4. mapseq  lc \n\na.  rna \nb-d.  mapseq  22  995 barcodes d 6  barcodes \n\n\n# fig5. mapseq provides a robust readout of single-neuron projection patterns\n\n\n\nfig5. mapseq \n\na. mapseq\nb. \n\n\n# fig6. lc neurons tile cortex with their maximum projections, but innervate large areas of cortex at a low level\n\n\n\nfig6. lc \n\na.  995 \nb. \nc.  lc \n\n\n# fig7. mapseq can be multiplexed to several injection sites\n\n\n\nfig7. mapseq \n\na.  barcoded sindbis \nb. \nc. lc',charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- A mesoscale connectome of the mouse brain",frontmatter:{title:" --- A mesoscale connectome of the mouse brain",date:"2023-06-15T21:05:15.000Z",permalink:"/pages/7ffbf6/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20140402.(2014.04.02%20Allen%E8%BF%9E%E6%8E%A5%E5%9B%BE%E8%B0%B1%20Nature)%20A%20mesoscale%20connectome%20of%20the%20mouse%20brain.html",relativePath:"02./11./02./20140402.(2014.04.02 Allen Nature) A mesoscale connectome of the mouse brain.md",key:"v-76ee6b70",path:"/pages/7ffbf6/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:291},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:721},{level:3,title:"Figure 1. Creation of the Connectivity Atlas",slug:"figure-1-creation-of-the-connectivity-atlas",normalizedTitle:"figure 1. creation of the connectivity atlas",charIndex:734},{level:3,title:"Figure 2. Whole brain projection patterns from seven representative cortical regions",slug:"figure-2-whole-brain-projection-patterns-from-seven-representative-cortical-regions",normalizedTitle:"figure 2. whole brain projection patterns from seven representative cortical regions",charIndex:1049},{level:3,title:"Figure 3. Adult mouse brain connectivity matrix",slug:"figure-3-adult-mouse-brain-connectivity-matrix",normalizedTitle:"figure 3. adult mouse brain connectivity matrix",charIndex:1251},{level:3,title:"Figure 4. A computational model of inter-regional connection strengths",slug:"figure-4-a-computational-model-of-inter-regional-connection-strengths",normalizedTitle:"figure 4. a computational model of inter-regional connection strengths",charIndex:1384},{level:3,title:"Figure 5. Topography of cortico-striatal and cortico-thalamic projections",slug:"figure-5-topography-of-cortico-striatal-and-cortico-thalamic-projections",normalizedTitle:"figure 5. topography of cortico-striatal and cortico-thalamic projections",charIndex:1713},{level:3,title:"Figure 6. A wiring diagram of connections between major cortical regions and thalamic nuclei",slug:"figure-6-a-wiring-diagram-of-connections-between-major-cortical-regions-and-thalamic-nuclei",normalizedTitle:"figure 6. a wiring diagram of connections between major cortical regions and thalamic nuclei",charIndex:2162}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/nature13186" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/nature13186<OutboundLink/></a><br>\nCite: Oh, S., Harris, J., Ng, L. et al. A mesoscale connectome of the mouse brain. Nature 508, 207214 (2014).</p>\n',headersStr:"Abstract: Results: Figure 1. Creation of the Connectivity Atlas Figure 2. Whole brain projection patterns from seven representative cortical regions Figure 3. Adult mouse brain connectivity matrix Figure 4. A computational model of inter-regional connection strengths Figure 5. Topography of cortico-striatal and cortico-thalamic projections Figure 6. A wiring diagram of connections between major cortical regions and thalamic nuclei",content:": \nDOI: https://doi.org/10.1038/nature13186\nCite: Oh, S., Harris, J., Ng, L. et al. A mesoscale connectome of the mouse brain. Nature 508, 207214 (2014).\n\n\n\n\n\n\n\nHONGKUI ZENG()\n\nAllen Institute for Brain Science\nhongkuiz@alleninstitute.org\n\n\n# Abstract:\n\nAllen EGFP EGFP 3D- Allen Mouse Brain Connectivity Atlas \n\n\n# Results:\n\n\n# Figure 1. Creation of the Connectivity Atlas\n\n\n\nFigure 1. \n\na. AAV-EGFP  TissueCyte 1000  (STP)  EGFP  1,231  100um  140 \nb.  Allen Reference Atlas \nc. \n\n\n# Figure 2. Whole brain projection patterns from seven representative cortical regions\n\n\n\nFigure 2.  7 \n\n\n 3D \n 6  EGFP \n\n\n# Figure 3. Adult mouse brain connectivity matrix\n\n\n\nFigure 3. \n\n 469  295 \n\n\n# Figure 4. A computational model of inter-regional connection strengths\n\n\n\nFigure 4. \n\na. \nb. -\nc.  Erdos-RenyiWatts-Strogatz  Barabasi-Albert \nd. \n\n\n# Figure 5. Topography of cortico-striatal and cortico-thalamic projections\n\n\n\nFigure 5. --\n\na.  80  3D  2D \nb.  a  21  8 \nc. b \nd.  3D \ne.  3D \nf.  3D \n\n\n# Figure 6. A wiring diagram of connections between major cortical regions and thalamic nuclei\n\n\n\nFigure 6. \n\n Fig. 5a  Extended Data Fig. 10 Fig. 3LGv  PF  (RT) --/-- GU/VISC  VPMpc/VPLpc  VPM/VPL",normalizedContent:": \ndoi: https://doi.org/10.1038/nature13186\ncite: oh, s., harris, j., ng, l. et al. a mesoscale connectome of the mouse brain. nature 508, 207214 (2014).\n\n\n\n\n\n\n\nhongkui zeng()\n\nallen institute for brain science\nhongkuiz@alleninstitute.org\n\n\n# abstract:\n\nallen egfp egfp 3d- allen mouse brain connectivity atlas \n\n\n# results:\n\n\n# figure 1. creation of the connectivity atlas\n\n\n\nfigure 1. \n\na. aav-egfp  tissuecyte 1000  (stp)  egfp  1,231  100um  140 \nb.  allen reference atlas \nc. \n\n\n# figure 2. whole brain projection patterns from seven representative cortical regions\n\n\n\nfigure 2.  7 \n\n\n 3d \n 6  egfp \n\n\n# figure 3. adult mouse brain connectivity matrix\n\n\n\nfigure 3. \n\n 469  295 \n\n\n# figure 4. a computational model of inter-regional connection strengths\n\n\n\nfigure 4. \n\na. \nb. -\nc.  erdos-renyiwatts-strogatz  barabasi-albert \nd. \n\n\n# figure 5. topography of cortico-striatal and cortico-thalamic projections\n\n\n\nfigure 5. --\n\na.  80  3d  2d \nb.  a  21  8 \nc. b \nd.  3d \ne.  3d \nf.  3d \n\n\n# figure 6. a wiring diagram of connections between major cortical regions and thalamic nuclei\n\n\n\nfigure 6. \n\n fig. 5a  extended data fig. 10 fig. 3lgv  pf  (rt) --/-- gu/visc  vpmpc/vplpc  vpm/vpl",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- The logic of single-cell projections from visual cortex",frontmatter:{title:" --- The logic of single-cell projections from visual cortex",date:"2022-12-09T20:48:28.000Z",permalink:"/pages/8cb60c/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20180328.(2018.03.28%20%E9%9F%A9%E8%8A%B8%E8%80%98%20Nature%202018)%20The%20logic%20of%20single-cell%20projections%20from%20%20visual%20cortex.html",relativePath:"02./11./02./20180328.(2018.03.28  Nature 2018) The logic of single-cell projections from  visual cortex.md",key:"v-15bfff01",path:"/pages/8cb60c/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:510},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:861},{level:3,title:"Fig1. Brain-wide single-cell tracing reveals the diversity of axonal projection patterns of layer-2/3 V1 neurons, with most cells projecting to more than one target area",slug:"fig1-brain-wide-single-cell-tracing-reveals-the-diversity-of-axonal-projection-patterns-of-layer-2-3-v1-neurons-with-most-cells-projecting-to-more-than-one-target-area",normalizedTitle:"fig1. brain-wide single-cell tracing reveals the diversity of axonal projection patterns of layer-2/3 v1 neurons, with most cells projecting to more than one target area",charIndex:873},{level:3,title:"Fig2. MAPseq projection mapping reveals a diversity of projection motifs",slug:"fig2-mapseq-projection-mapping-reveals-a-diversity-of-projection-motifs",normalizedTitle:"fig2. mapseq projection mapping reveals a diversity of projection motifs",charIndex:1473},{level:3,title:"Fig3. Over- and underrepresented projection motifs of neurons in V1",slug:"fig3-over-and-underrepresented-projection-motifs-of-neurons-in-v1",normalizedTitle:"fig3. over- and underrepresented projection motifs of neurons in v1",charIndex:1995}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/nature26159" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/nature26159<OutboundLink/></a><br>\nCite: Han, Y., Kebschull, J., Campbell, R. et al. The logic of single-cell projections from visual cortex. <em>Nature</em> 556, 5156 (2018).</p>\n',headersStr:"Abstract Results Fig1. Brain-wide single-cell tracing reveals the diversity of axonal projection patterns of layer-2/3 V1 neurons, with most cells projecting to more than one target area Fig2. MAPseq projection mapping reveals a diversity of projection motifs Fig3. Over- and underrepresented projection motifs of neurons in V1",content:': \nDOI: https://doi.org/10.1038/nature26159\nCite: Han, Y., Kebschull, J., Campbell, R. et al. The logic of single-cell projections from visual cortex. Nature 556, 5156 (2018).\n\n\n\n\n\n\n\nYUNYUN HAN()    ANTHONY ZADOR                   THOMAS MRSIC-FLOGEL\n                                                   \n             Cold Spring Harbor Laboratory   University College London\nyhan@hust.edu.cn   zador@cshl.edu                  t.mrsic-flogel@ucl.ac.uk\n\n\n# Abstract\n\n MAPseqprimary visual cortex 591  18  "one neuronone target area" \n\n\n# Results\n\n\n# Fig1. Brain-wide single-cell tracing reveals the diversity of axonal projection patterns of layer-2/3 V1 neurons, with most cells projecting to more than one target area\n\n\n\nFig1.  L2/3 V1 \n\na. \nb. GFP \nc. \nd. \ne.  Allen Reference Atlas \nf. \ng. 11\nh.  V1  V1 \ni.  GFP  V1 n=31\nj. V1 \nk. \nl.  V1  18 \nm.  V1  18 \n\n\n# Fig2. MAPseq projection mapping reveals a diversity of projection motifs\n\n\n\nFig2. MAPseq  motifs\n\na. MAPseq6LILMALPMAMRL\nb.  MAPseq  553  6 olfactory bulbOBn=4\nc. MAPseq6V1\nd. MAPseq neurons MAPseq neurons \ne.  MAPseq  k-means  8  cluster \nf.  A  B \n\n\n# Fig3. Over- and underrepresented projection motifs of neurons in V1\n\n\n\nFig3. V1 \n\na.  null hypothesis null hypothesis\nb. MAPseq motifs \nc. \nd-h. 5 ',normalizedContent:': \ndoi: https://doi.org/10.1038/nature26159\ncite: han, y., kebschull, j., campbell, r. et al. the logic of single-cell projections from visual cortex. nature 556, 5156 (2018).\n\n\n\n\n\n\n\nyunyun han()    anthony zador                   thomas mrsic-flogel\n                                                   \n             cold spring harbor laboratory   university college london\nyhan@hust.edu.cn   zador@cshl.edu                  t.mrsic-flogel@ucl.ac.uk\n\n\n# abstract\n\n mapseqprimary visual cortex 591  18  "one neuronone target area" \n\n\n# results\n\n\n# fig1. brain-wide single-cell tracing reveals the diversity of axonal projection patterns of layer-2/3 v1 neurons, with most cells projecting to more than one target area\n\n\n\nfig1.  l2/3 v1 \n\na. \nb. gfp \nc. \nd. \ne.  allen reference atlas \nf. \ng. 11\nh.  v1  v1 \ni.  gfp  v1 n=31\nj. v1 \nk. \nl.  v1  18 \nm.  v1  18 \n\n\n# fig2. mapseq projection mapping reveals a diversity of projection motifs\n\n\n\nfig2. mapseq  motifs\n\na. mapseq6lilmalpmamrl\nb.  mapseq  553  6 olfactory bulbobn=4\nc. mapseq6v1\nd. mapseq neurons mapseq neurons \ne.  mapseq  k-means  8  cluster \nf.  a  b \n\n\n# fig3. over- and underrepresented projection motifs of neurons in v1\n\n\n\nfig3. v1 \n\na.  null hypothesis null hypothesis\nb. mapseq motifs \nc. \nd-h. 5 ',charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Bridging the Gap between Connectome and Transcriptome",frontmatter:{title:" --- Bridging the Gap between Connectome and Transcriptome",date:"2024-03-06T15:49:31.000Z",permalink:"/pages/9e1c62/",categories:[""],tags:["-"],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/01.%E7%BB%BC%E8%BF%B0/20181116.(2018.11.16%20%E8%BF%9E%E6%8E%A5%E7%BB%84%E7%BB%BC%E8%BF%B0%20Trends%20Cogn%20Sci)%20Bridging%20the%20Gap%20between%20Connectome%20and%20Transcriptome.html",relativePath:"02./11./01./20181116.(2018.11.16  Trends Cogn Sci) Bridging the Gap between Connectome and Transcriptome.md",key:"v-7180fb28",path:"/pages/9e1c62/",headers:[{level:2,title:"Highlights",slug:"highlights",normalizedTitle:"highlights",charIndex:305},{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:523},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:773},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:785},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1894},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:3507},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:5903},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:7648},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:9609},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:1537},{level:3,title:"Box 1 Modelling the Brain as a Complex Network",slug:"box-1-modelling-the-brain-as-a-complex-network",normalizedTitle:"box 1 modelling the brain as a complex network",charIndex:10483},{level:3,title:"Figure 1. Workflows for Relating Transcriptomic Atlases to Connectome Data",slug:"figure-1-workflows-for-relating-transcriptomic-atlases-to-connectome-data",normalizedTitle:"figure 1. workflows for relating transcriptomic atlases to connectome data",charIndex:11612},{level:3,title:"Box 2 Clinical Applications of Brain-wide Gene Expression Atlases",slug:"box-2-clinical-applications-of-brain-wide-gene-expression-atlases",normalizedTitle:"box 2 clinical applications of brain-wide gene expression atlases",charIndex:12207},{level:3,title:"Figure 2. The Brain Transcriptome Follows Broad Spatial Trends That Track Brain Network Architecture",slug:"figure-2-the-brain-transcriptome-follows-broad-spatial-trends-that-track-brain-network-architecture",normalizedTitle:"figure 2. the brain transcriptome follows broad spatial trends that track brain network architecture",charIndex:13394},{level:3,title:"Figure 3. Neuronal Density and Synaptic Connectivity Vary along a Rostrocaudal Axis",slug:"figure-3-neuronal-density-and-synaptic-connectivity-vary-along-a-rostrocaudal-axis",normalizedTitle:"figure 3. neuronal density and synaptic connectivity vary along a rostrocaudal axis",charIndex:13886},{level:3,title:"Figure 4. Hierarchical Gradients in Gene Expression and Connectional Architecture",slug:"figure-4-hierarchical-gradients-in-gene-expression-and-connectional-architecture",normalizedTitle:"figure 4. hierarchical gradients in gene expression and connectional architecture",charIndex:14322},{level:3,title:"Figure 5. Transcriptional Coupling Is Elevated within Network Modules and between Network Hubs",slug:"figure-5-transcriptional-coupling-is-elevated-within-network-modules-and-between-network-hubs",normalizedTitle:"figure 5. transcriptional coupling is elevated within network modules and between network hubs",charIndex:14982}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.tics.2018.10.005" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.tics.2018.10.005<OutboundLink/></a><br>\nCite: Fornito A, Arnatkeviit A, Fulcher BD. Bridging the Gap between Connectome and Transcriptome. Trends Cogn Sci. 2019;23(1):34-50.</p>\n',headersStr:"Highlights Abstract Results       Figure Box 1 Modelling the Brain as a Complex Network Figure 1. Workflows for Relating Transcriptomic Atlases to Connectome Data Box 2 Clinical Applications of Brain-wide Gene Expression Atlases Figure 2. The Brain Transcriptome Follows Broad Spatial Trends That Track Brain Network Architecture Figure 3. Neuronal Density and Synaptic Connectivity Vary along a Rostrocaudal Axis Figure 4. Hierarchical Gradients in Gene Expression and Connectional Architecture Figure 5. Transcriptional Coupling Is Elevated within Network Modules and between Network Hubs",content:": \nDOI: https://doi.org/10.1016/j.tics.2018.10.005\nCite: Fornito A, Arnatkeviit A, Fulcher BD. Bridging the Gap between Connectome and Transcriptome. Trends Cogn Sci. 2019;23(1):34-50.\n\n\n\n\n\n\n\nALEX FORNITO\n\nMonash University\nalex.fornito@monash.edu\n\n\n# Highlights\n\n * \n\n * \n\n * \n\n * \n\n\n# Abstract\n\n\n\n\n# Results\n\n\n# \n\nnetworkgraphnodeedgeBox 1    Box 1\n\nwiring costsnetwork economy\n\n DNA candidate variantgenome-wide association\n\nFigure 1/\n\nBox 2Box 2\n\n\n# \n\n DNA  DNA  RNA  DNA  cRNA  mRNA mRNA \n\n mRNA mRNA \n\n Allen Mouse Brain Atlas (AMBA)  Allen Human Brain Atlas (AHBA)AMBA  19,000  200 m  AHBA  3702  20,000 \n\n AMBAAHBA (i) (ii) CGE (iii) Figure 1Figure 1DCGE Figure 1EFigure 1CCGE --Figure 1CE CGE  CGE \n\n GO  KEGGGO \n\n AHBA \n\n\n\n\n# \n\n CGE Figure 2A-D\n\nCGE  (Figure 2E-G \n\nrostrocaudal axisFigure 3AB MRI Figure 3AB\n\nFigure 3C(Figure 4)Figure 4D\n\n 4  4 Figure 4B 3-5  GO  2  3  19  HSE  CGE  CGE HSE \n\n T1-to-T2-weighted MRI Figure 4C T1/T2  AHBA Figure 4E 4 Figure 4ANMDA  T1/T2  T1/T2 \n\n\n\n\n# \n\nSperry  70 chemoaffinity hypothesis netrinsslitsephrins  semaphorins\n\n CGE  CGE  CGE \n\n <300 AUC  0.84AUC  0.60 GO \n\n CGE  62%  GO  568  CGE  55% \n\n AHBA  CGE \n\n GO  RNA  50  MRI (fMRI)  26  10% \n\n\n\n\n# \n\nBox 1Box 1\n\n CGE  GO  CGE Figure 5AB\n\nparticipation coefficient CGE  Figure 2A CGE  rich linkshubs  nonhubs feeder links nonhubs peripheral linksFigure 5C,D CGE  CGE Figure 2A CGE  ATP  GO  CGE \n\n CGE Figure 5E\n\n CGE  948  17,000  fMRI \n\n CGE  hubs hubs  CGE  CGE  HSE  hubs  hubs CGE  CGE \n\n\n# \n\nBox 2\n\nFigure 1 DNA  20,000  100 Box 2\n\n>80% AHBA \n\n\n# Figure\n\n\n# Box 1 Modelling the Brain as a Complex Network\n\nBox 1. \n\nedgesnodesFigure IAgraph\n\n\n\neconomical wiringhierarchical modularitysmall-world organisationnetwork hubsrich club\n\nFigure IB provincial hubsconnector hubs( Figure IA )\n\nFigure I\n\nFigure I. Graph-based Representations of the Connectome \n\n\n# Figure 1. Workflows for Relating Transcriptomic Atlases to Connectome Data\n\n\n\nFigure 1. \n\n(A)  Allen Human Brain Atlas MRI \n(B) \n(C) \n(D)  C  D \n(E)  (CGE) CGE  CGE  CGE \n\n\n# Box 2 Clinical Applications of Brain-wide Gene Expression Atlases\n\nBox 1. \n\n\n\nPLSFigure IA  ()  Figure IB  PLS  GO AHBA \n\nFigure IC  48  Figure 1D tau  MAPT  SNCA - tau  tau \n\n GO \n\nFigure I. Linking Gene Expression to Brain Changes in Clinical Disorders \n\n\n# Figure 2. The Brain Transcriptome Follows Broad Spatial Trends That Track Brain Network Architecture\n\n\n\nFigure 2. \n\n(A-D) (A)  (CGE)  (B)  MRI  (C) CGE  (D) \n(E, F)  2D umap  (E) 4211  (F) E  F \n(G) Allen Human Brain Atlas \n\n\n# Figure 3. Neuronal Density and Synaptic Connectivity Vary along a Rostrocaudal Axis\n\n\n\nFigure 3. \n\n(A) \n(B) \n(C) \n\n\n# Figure 4. Hierarchical Gradients in Gene Expression and Connectional Architecture\n\n\n\nFigure 4. \n\n(A) \n(B)  (3060 mm)  (%NSG) \n(C) T1-to-T2-weighted MRI  %NSG \n(D)  1 2 \n(E)  T1/T2 MRI \n\n\n# Figure 5. Transcriptional Coupling Is Elevated within Network Modules and between Network Hubs\n\n\n\nFigure 5. \n\n(A)  MCN  CGE \n(B)  CGE  CGE \n(C)  hub  CGE  hubs  nonhubsrich hubs feederhubs  nonhubs peripheral nonhubs  CGE\n(D)  hubs  richfeederperipheral  CGE hubs  hubs rich  CGE  CGE\n(E)  D CGE  rich ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.tics.2018.10.005\ncite: fornito a, arnatkeviciute a, fulcher bd. bridging the gap between connectome and transcriptome. trends cogn sci. 2019;23(1):34-50.\n\n\n\n\n\n\n\nalex fornito\n\nmonash university\nalex.fornito@monash.edu\n\n\n# highlights\n\n * \n\n * \n\n * \n\n * \n\n\n# abstract\n\n\n\n\n# results\n\n\n# \n\nnetworkgraphnodeedgebox 1    box 1\n\nwiring costsnetwork economy\n\n dna candidate variantgenome-wide association\n\nfigure 1/\n\nbox 2box 2\n\n\n# \n\n dna  dna  rna  dna  crna  mrna mrna \n\n mrna mrna \n\n allen mouse brain atlas (amba)  allen human brain atlas (ahba)amba  19,000  200 m  ahba  3702  20,000 \n\n ambaahba (i) (ii) cge (iii) figure 1figure 1dcge figure 1efigure 1ccge --figure 1ce cge  cge \n\n go  kegggo \n\n ahba \n\n\n\n\n# \n\n cge figure 2a-d\n\ncge  (figure 2e-g \n\nrostrocaudal axisfigure 3ab mri figure 3ab\n\nfigure 3c(figure 4)figure 4d\n\n 4  4 figure 4b 3-5  go  2  3  19  hse  cge  cge hse \n\n t1-to-t2-weighted mri figure 4c t1/t2  ahba figure 4e 4 figure 4anmda  t1/t2  t1/t2 \n\n\n\n\n# \n\nsperry  70 chemoaffinity hypothesis netrinsslitsephrins  semaphorins\n\n cge  cge  cge \n\n <300 auc  0.84auc  0.60 go \n\n cge  62%  go  568  cge  55% \n\n ahba  cge \n\n go  rna  50  mri (fmri)  26  10% \n\n\n\n\n# \n\nbox 1box 1\n\n cge  go  cge figure 5ab\n\nparticipation coefficient cge  figure 2a cge  rich linkshubs  nonhubs feeder links nonhubs peripheral linksfigure 5c,d cge  cge figure 2a cge  atp  go  cge \n\n cge figure 5e\n\n cge  948  17,000  fmri \n\n cge  hubs hubs  cge  cge  hse  hubs  hubs cge  cge \n\n\n# \n\nbox 2\n\nfigure 1 dna  20,000  100 box 2\n\n>80% ahba \n\n\n# figure\n\n\n# box 1 modelling the brain as a complex network\n\nbox 1. \n\nedgesnodesfigure iagraph\n\n\n\neconomical wiringhierarchical modularitysmall-world organisationnetwork hubsrich club\n\nfigure ib provincial hubsconnector hubs( figure ia )\n\nfigure i\n\nfigure i. graph-based representations of the connectome \n\n\n# figure 1. workflows for relating transcriptomic atlases to connectome data\n\n\n\nfigure 1. \n\n(a)  allen human brain atlas mri \n(b) \n(c) \n(d)  c  d \n(e)  (cge) cge  cge  cge \n\n\n# box 2 clinical applications of brain-wide gene expression atlases\n\nbox 1. \n\n\n\nplsfigure ia  ()  figure ib  pls  go ahba \n\nfigure ic  48  figure 1d tau  mapt  snca - tau  tau \n\n go \n\nfigure i. linking gene expression to brain changes in clinical disorders \n\n\n# figure 2. the brain transcriptome follows broad spatial trends that track brain network architecture\n\n\n\nfigure 2. \n\n(a-d) (a)  (cge)  (b)  mri  (c) cge  (d) \n(e, f)  2d umap  (e) 4211  (f) e  f \n(g) allen human brain atlas \n\n\n# figure 3. neuronal density and synaptic connectivity vary along a rostrocaudal axis\n\n\n\nfigure 3. \n\n(a) \n(b) \n(c) \n\n\n# figure 4. hierarchical gradients in gene expression and connectional architecture\n\n\n\nfigure 4. \n\n(a) \n(b)  (3060 mm)  (%nsg) \n(c) t1-to-t2-weighted mri  %nsg \n(d)  1 2 \n(e)  t1/t2 mri \n\n\n# figure 5. transcriptional coupling is elevated within network modules and between network hubs\n\n\n\nfigure 5. \n\n(a)  mcn  cge \n(b)  cge  cge \n(c)  hub  cge  hubs  nonhubsrich hubs feederhubs  nonhubs peripheral nonhubs  cge\n(d)  hubs  richfeederperipheral  cge hubs  hubs rich  cge  cge\n(e)  d cge  rich ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Distinct descending motor cortex pathways and their roles in movement",frontmatter:{title:" --- Distinct descending motor cortex pathways and their roles in movement",date:"2023-03-28T19:39:13.000Z",permalink:"/pages/986ab0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20181030.(2018.10.31%20PT-thalamus-medulla%20Nature)%20Distinct%20descending%20motor%20cortex%20pathways%20and%20their%20roles%20in%20movement.html",relativePath:"02./11./02./20181030.(2018.10.31 PT-thalamus-medulla Nature) Distinct descending motor cortex pathways and their roles in movement.md",key:"v-61bb19fa",path:"/pages/986ab0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:348},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:676},{level:3,title:"Fig1. Taxonomy of motor cortex glutamatergic neurons based on scRNA-seq",slug:"fig1-taxonomy-of-motor-cortex-glutamatergic-neurons-based-on-scrna-seq",normalizedTitle:"fig1. taxonomy of motor cortex glutamatergic neurons based on scrna-seq",charIndex:688},{level:3,title:"Fig2. Two types of PT neuron in the motor cortex",slug:"fig2-two-types-of-pt-neuron-in-the-motor-cortex",normalizedTitle:"fig2. two types of pt neuron in the motor cortex",charIndex:1112},{level:3,title:"Fig3. PT neuron type markers",slug:"fig3-pt-neuron-type-markers",normalizedTitle:"fig3. pt neuron type markers",charIndex:1600},{level:3,title:"Fig4. Cell type-specific extracellular neurophysiology",slug:"fig4-cell-type-specific-extracellular-neurophysiology",normalizedTitle:"fig4. cell type-specific extracellular neurophysiology",charIndex:1783},{level:3,title:"Fig5. Persistent preparatory activity in PTupper neurons",slug:"fig5-persistent-preparatory-activity-in-ptupper-neurons",normalizedTitle:"fig5. persistent preparatory activity in ptupper neurons",charIndex:2334},{level:3,title:"Fig6. Movement commands in PTlower neurons",slug:"fig6-movement-commands-in-ptlower-neurons",normalizedTitle:"fig6. movement commands in ptlower neurons",charIndex:2729},{level:2,title:"Extended Data Figure",slug:"extended-data-figure",normalizedTitle:"extended data figure",charIndex:3220},{level:3,title:"Extended Data Fig1. Differentially expressed genes in single-cell and bulk RNA-seq",slug:"extended-data-fig1-differentially-expressed-genes-in-single-cell-and-bulk-rna-seq",normalizedTitle:"extended data fig1. differentially expressed genes in single-cell and bulk rna-seq",charIndex:3245},{level:3,title:"Extended Data Fig2. Distribution of PT axon collaterals",slug:"extended-data-fig2-distribution-of-pt-axon-collaterals",normalizedTitle:"extended data fig2. distribution of pt axon collaterals",charIndex:3538},{level:3,title:"Extended Data Fig3. Spatial distribution of thalamus- and medulla- projecting PT neurons",slug:"extended-data-fig3-spatial-distribution-of-thalamus-and-medulla-projecting-pt-neurons",normalizedTitle:"extended data fig3. spatial distribution of thalamus- and medulla- projecting pt neurons",charIndex:3631},{level:3,title:"Extended Data Fig4. Correspondence between scRNA-seq transcriptomic clusters, projection targets, and bulk RNA-seq data",slug:"extended-data-fig4-correspondence-between-scrna-seq-transcriptomic-clusters-projection-targets-and-bulk-rna-seq-data",normalizedTitle:"extended data fig4. correspondence between scrna-seq transcriptomic clusters, projection targets, and bulk rna-seq data",charIndex:3767},{level:3,title:"Extended Data Fig5. Electrophysiology and trial-averaged spike rates for identified PT neurons",slug:"extended-data-fig5-electrophysiology-and-trial-averaged-spike-rates-for-identified-pt-neurons",normalizedTitle:"extended data fig5. electrophysiology and trial-averaged spike rates for identified pt neurons",charIndex:3958},{level:3,title:"Extended Data Fig6. Variance in trial-averaged activity explained by PT cell class",slug:"extended-data-fig6-variance-in-trial-averaged-activity-explained-by-pt-cell-class",normalizedTitle:"extended data fig6. variance in trial-averaged activity explained by pt cell class",charIndex:4105},{level:3,title:"Extended Data Fig7. Late delay epoch coding direction and similarity with other trial epochs",slug:"extended-data-fig7-late-delay-epoch-coding-direction-and-similarity-with-other-trial-epochs",normalizedTitle:"extended data fig7. late delay epoch coding direction and similarity with other trial epochs",charIndex:4236},{level:3,title:"Extended Data Fig8. Comparison between identified PT populations and simultaneously recorded untagged neurons",slug:"extended-data-fig8-comparison-between-identified-pt-populations-and-simultaneously-recorded-untagged-neurons",normalizedTitle:"extended data fig8. comparison between identified pt populations and simultaneously recorded untagged neurons",charIndex:4378},{level:3,title:"Extended Data Fig9. Decoding of trial type in PT neuron types",slug:"extended-data-fig9-decoding-of-trial-type-in-pt-neuron-types",normalizedTitle:"extended data fig9. decoding of trial type in pt neuron types",charIndex:4543},{level:3,title:"Extended Data Fig10. Movement termination signals in PTlower neurons",slug:"extended-data-fig10-movement-termination-signals-in-ptlower-neurons",normalizedTitle:"extended data fig10. movement termination signals in ptlower neurons",charIndex:4649}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-018-0642-9" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-018-0642-9<OutboundLink/></a><br>\nCite: Economo, M.N., Viswanathan, S., Tasic, B. et al. Distinct descending motor cortex pathways and their roles in movement. <em>Nature</em> 563, 7984 (2018).</p>\n',headersStr:"Abstract Results Fig1. Taxonomy of motor cortex glutamatergic neurons based on scRNA-seq Fig2. Two types of PT neuron in the motor cortex Fig3. PT neuron type markers Fig4. Cell type-specific extracellular neurophysiology Fig5. Persistent preparatory activity in PTupper neurons Fig6. Movement commands in PTlower neurons Extended Data Figure Extended Data Fig1. Differentially expressed genes in single-cell and bulk RNA-seq Extended Data Fig2. Distribution of PT axon collaterals Extended Data Fig3. Spatial distribution of thalamus- and medulla- projecting PT neurons Extended Data Fig4. Correspondence between scRNA-seq transcriptomic clusters, projection targets, and bulk RNA-seq data Extended Data Fig5. Electrophysiology and trial-averaged spike rates for identified PT neurons Extended Data Fig6. Variance in trial-averaged activity explained by PT cell class Extended Data Fig7. Late delay epoch coding direction and similarity with other trial epochs Extended Data Fig8. Comparison between identified PT populations and simultaneously recorded untagged neurons Extended Data Fig9. Decoding of trial type in PT neuron types Extended Data Fig10. Movement termination signals in PTlower neurons",content:": \nDOI: https://doi.org/10.1038/s41586-018-0642-9\nCite: Economo, M.N., Viswanathan, S., Tasic, B. et al. Distinct descending motor cortex pathways and their roles in movement. Nature 563, 7984 (2018).\n\n\n\n\n\n\n\nKAREL SVOBODA\n\nHoward Hughes Medical Institute\nsvobodak@janelia.hhmi.org\n\n\n# Abstract\n\n layer 5 \n\n\n# Results\n\n\n# Fig1. Taxonomy of motor cortex glutamatergic neurons based on scRNA-seq\n\n\n\nFig1.  scRNA-seq \n\na.  (for details, see Tasic et al.) Slco2a1Npsr1Hpgd  PT  Cajal-Retzius CT, corticothalamic neurons; IT, intratelencephalic neurons; NP, near-projecting neurons. Ptrf is also known as Cavin1\nb. ALM  t-SNE (n=4,477 glutamatergic cells) a \n\n\n# Fig2. Two types of PT neuron in the motor cortex\n\n\n\nFig2.  PT \n\na.  PT  (thalamus) (top; green hues) (medulla) (bottom; magenta hues)\nb.  thalamus-projecting  medulla-projecting  PT  (top) (bottom)\nc-d.  thalamus (green)  medulla (magenta)  L5b 3SC, superior colliculus \ne. PT  t-SNE (as in Fig. 1b; n=368 PT neurons)\nf.  Slco2a1  Npsr1Hpgd \n\n\n# Fig3. PT neuron type markers\n\n\n\nFig3. PT \n\na.  scRNA-seq (n=9,573 cells) \nb. smFISH \nc. PTupper  PTlower Npnt  Slco2a1  RNA \n\n\n# Fig4. Cell type-specific extracellular neurophysiology\n\n\n\nFig4. \n\na. 1.01.3\nb. SDR\nc. \nd. +t[t, +t]+t+t\ne. PTPTPTn=61PTn=69\nf. PT\n\n\n# Fig5. Persistent preparatory activity in PTupper neurons\n\n\n\nFig5.  PTupper \n\na. CDearlyPTPTAU\nb. PTPTCDearly\nc. PTn=1,000ab5-95%P<0.05\n\n\n# Fig6. Movement commands in PTlower neurons\n\n\n\nFig6. PTlower \n\na. CDgoPTPT\nb. PTPTCDgo\nc. bRT99%=3856CDgoPT24PT48\nd. PTn=1,000abc5-95%P<0.05\n\n\n# Extended Data Figure\n\n\n# Extended Data Fig1. Differentially expressed genes in single-cell and bulk RNA-seq\n\n\n\nExtended Data Fig1.  bulk RNA-seq \n\na,c.  scRNA-seq (a)  bulk RNA-seq (c)  ( scRNA-seq) \nb,d.  scRNA-seq (b)  bulk RNA-seq (d)  ( bulk RNA-seq) \n\n\n# Extended Data Fig2. Distribution of PT axon collaterals\n\n\n\nExtended Data Fig2. PT\n\n\n# Extended Data Fig3. Spatial distribution of thalamus- and medulla- projecting PT neurons\n\n\n\nExtended Data Fig3.  PT \n\n\n# Extended Data Fig4. Correspondence between scRNA-seq transcriptomic clusters, projection targets, and bulk RNA-seq data\n\n\n\nExtended Data Fig4. scRNA-seq  bulk RNA-seq \n\n\n# Extended Data Fig5. Electrophysiology and trial-averaged spike rates for identified PT neurons\n\n\n\nExtended Data Fig5.  PT \n\n\n# Extended Data Fig6. Variance in trial-averaged activity explained by PT cell class\n\n\n\nExtended Data Fig6.  PT \n\n\n# Extended Data Fig7. Late delay epoch coding direction and similarity with other trial epochs\n\n\n\nExtended Data Fig7. \n\n\n# Extended Data Fig8. Comparison between identified PT populations and simultaneously recorded untagged neurons\n\n\n\nExtended Data Fig8.  PT \n\n\n# Extended Data Fig9. Decoding of trial type in PT neuron types\n\n\n\nExtended Data Fig9. PT \n\n\n# Extended Data Fig10. Movement termination signals in PTlower neurons\n\n\n\nExtended Data Fig10. PTlower ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-018-0642-9\ncite: economo, m.n., viswanathan, s., tasic, b. et al. distinct descending motor cortex pathways and their roles in movement. nature 563, 7984 (2018).\n\n\n\n\n\n\n\nkarel svoboda\n\nhoward hughes medical institute\nsvobodak@janelia.hhmi.org\n\n\n# abstract\n\n layer 5 \n\n\n# results\n\n\n# fig1. taxonomy of motor cortex glutamatergic neurons based on scrna-seq\n\n\n\nfig1.  scrna-seq \n\na.  (for details, see tasic et al.) slco2a1npsr1hpgd  pt  cajal-retzius ct, corticothalamic neurons; it, intratelencephalic neurons; np, near-projecting neurons. ptrf is also known as cavin1\nb. alm  t-sne (n=4,477 glutamatergic cells) a \n\n\n# fig2. two types of pt neuron in the motor cortex\n\n\n\nfig2.  pt \n\na.  pt  (thalamus) (top; green hues) (medulla) (bottom; magenta hues)\nb.  thalamus-projecting  medulla-projecting  pt  (top) (bottom)\nc-d.  thalamus (green)  medulla (magenta)  l5b 3sc, superior colliculus \ne. pt  t-sne (as in fig. 1b; n=368 pt neurons)\nf.  slco2a1  npsr1hpgd \n\n\n# fig3. pt neuron type markers\n\n\n\nfig3. pt \n\na.  scrna-seq (n=9,573 cells) \nb. smfish \nc. ptupper  ptlower npnt  slco2a1  rna \n\n\n# fig4. cell type-specific extracellular neurophysiology\n\n\n\nfig4. \n\na. 1.01.3\nb. sdr\nc. \nd. +t[t, +t]+t+t\ne. ptptptn=61ptn=69\nf. pt\n\n\n# fig5. persistent preparatory activity in ptupper neurons\n\n\n\nfig5.  ptupper \n\na. cdearlyptptau\nb. ptptcdearly\nc. ptn=1,000ab5-95%p<0.05\n\n\n# fig6. movement commands in ptlower neurons\n\n\n\nfig6. ptlower \n\na. cdgoptpt\nb. ptptcdgo\nc. brt99%=3856cdgopt24pt48\nd. ptn=1,000abc5-95%p<0.05\n\n\n# extended data figure\n\n\n# extended data fig1. differentially expressed genes in single-cell and bulk rna-seq\n\n\n\nextended data fig1.  bulk rna-seq \n\na,c.  scrna-seq (a)  bulk rna-seq (c)  ( scrna-seq) \nb,d.  scrna-seq (b)  bulk rna-seq (d)  ( bulk rna-seq) \n\n\n# extended data fig2. distribution of pt axon collaterals\n\n\n\nextended data fig2. pt\n\n\n# extended data fig3. spatial distribution of thalamus- and medulla- projecting pt neurons\n\n\n\nextended data fig3.  pt \n\n\n# extended data fig4. correspondence between scrna-seq transcriptomic clusters, projection targets, and bulk rna-seq data\n\n\n\nextended data fig4. scrna-seq  bulk rna-seq \n\n\n# extended data fig5. electrophysiology and trial-averaged spike rates for identified pt neurons\n\n\n\nextended data fig5.  pt \n\n\n# extended data fig6. variance in trial-averaged activity explained by pt cell class\n\n\n\nextended data fig6.  pt \n\n\n# extended data fig7. late delay epoch coding direction and similarity with other trial epochs\n\n\n\nextended data fig7. \n\n\n# extended data fig8. comparison between identified pt populations and simultaneously recorded untagged neurons\n\n\n\nextended data fig8.  pt \n\n\n# extended data fig9. decoding of trial type in pt neuron types\n\n\n\nextended data fig9. pt \n\n\n# extended data fig10. movement termination signals in ptlower neurons\n\n\n\nextended data fig10. ptlower ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Differences in neurotropism and neurotoxicity among retrograde viral tracers",frontmatter:{title:" --- Differences in neurotropism and neurotoxicity among retrograde viral tracers",date:"2024-07-25T17:21:04.000Z",permalink:"/pages/037be6/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20190208.(2019.02.08%20%E5%AD%99%E4%B9%90%E5%BC%BA%20Mol%20Neurodegeneration)%20Differences%20in%20neurotropism%20and%20neurotoxicity%20among%20retrograde%20viral%20tracers.html",relativePath:"02./11./02./20190208.(2019.02.08  Mol Neurodegeneration) Differences in neurotropism and neurotoxicity among retrograde viral tracers.md",key:"v-3e8128e8",path:"/pages/037be6/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:412}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1186/s13024-019-0308-6" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13024-019-0308-6<OutboundLink/></a><br>\nCite: Sun, L., Tang, Y., Yan, K. et al. Differences in neurotropism and neurotoxicity among retrograde viral tracers. Mol Neurodegeneration 14, 8 (2019).</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1186/s13024-019-0308-6\nCite: Sun, L., Tang, Y., Yan, K. et al. Differences in neurotropism and neurotoxicity among retrograde viral tracers. Mol Neurodegeneration 14, 8 (2019).\n\n\n\n\n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# Abstract\n\nBackground: \nMethods: multi-trans-synapticmono-synapticPRVRVrAAV2-retro scRNAseq  RVRV-G rAAV2-retro \nResults:  PRV  RV  RV-G  rAAV2-retrorAAV2-retro  RV-G  rAAV2-retro  5  6  RV  TVA  EnvA-pseudotyped RV  RV-G  rAAV2-retro  RV-G  rAAV2-retroRV  rAAV \nConclusions:  rAAV2-retro RV-G  rAAV2-retroRV  rAAV \nKeywords: Neurotropic Virus, Retrograde tracing, pseudorabies virus (PRV), rabies virus (RV), rAAV2-retro, Neurotropism, Neurotoxicity, Multi-trans-synaptic Tracing, Mono-synaptic Tracing, RNA Sequencing",normalizedContent:": \ndoi: https://doi.org/10.1186/s13024-019-0308-6\ncite: sun, l., tang, y., yan, k. et al. differences in neurotropism and neurotoxicity among retrograde viral tracers. mol neurodegeneration 14, 8 (2019).\n\n\n\n\n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# abstract\n\nbackground: \nmethods: multi-trans-synapticmono-synapticprvrvraav2-retro scrnaseq  rvrv-g raav2-retro \nresults:  prv  rv  rv-g  raav2-retroraav2-retro  rv-g  raav2-retro  5  6  rv  tva  enva-pseudotyped rv  rv-g  raav2-retro  rv-g  raav2-retrorv  raav \nconclusions:  raav2-retro rv-g  raav2-retrorv  raav \nkeywords: neurotropic virus, retrograde tracing, pseudorabies virus (prv), rabies virus (rv), raav2-retro, neurotropism, neurotoxicity, multi-trans-synaptic tracing, mono-synaptic tracing, rna sequencing",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals",frontmatter:{title:" --- BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals",date:"2023-02-08T16:07:50.000Z",permalink:"/pages/afda58/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20200709.(2020.07.09%20BRICseq%20Cell)%20BRICseq%20Bridges%20Brain-wide%20Interregional%20Connectivity%20to%20Neural%20Activity%20and%20Gene%20Expression%20in%20Single%20Animals.html",relativePath:"02./11./02./20200709.(2020.07.09 BRICseq Cell) BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals.md",key:"v-5ac83554",path:"/pages/afda58/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:446},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:882},{level:3,title:"Figure 1. Mapping Brain-wide Corticocortical Projections with BRICseq",slug:"figure-1-mapping-brain-wide-corticocortical-projections-with-bricseq",normalizedTitle:"figure 1. mapping brain-wide corticocortical projections with bricseq",charIndex:894},{level:3,title:"Figure 2. Brain-wide Corticocortical Projectome Mapped by BRICseq and Its Validation",slug:"figure-2-brain-wide-corticocortical-projectome-mapped-by-bricseq-and-its-validation",normalizedTitle:"figure 2. brain-wide corticocortical projectome mapped by bricseq and its validation",charIndex:1565},{level:3,title:"Figure 3. Validation of BRICseq",slug:"figure-3-validation-of-bricseq",normalizedTitle:"figure 3. validation of bricseq",charIndex:2138},{level:3,title:"Figure 4. BRICseq Predicts Functional Connectivity",slug:"figure-4-bricseq-predicts-functional-connectivity",normalizedTitle:"figure 4. bricseq predicts functional connectivity",charIndex:2453},{level:3,title:"Figure 5. Gene Expression Patterns Predict Connectivity Determined by BRICseq",slug:"figure-5-gene-expression-patterns-predict-connectivity-determined-by-bricseq",normalizedTitle:"figure 5. gene expression patterns predict connectivity determined by bricseq",charIndex:2711},{level:3,title:"Figure 6. Comparison of BTBR and C57BL/6J Cortical Connectivity",slug:"figure-6-comparison-of-btbr-and-c57bl-6j-cortical-connectivity",normalizedTitle:"figure 6. comparison of btbr and c57bl/6j cortical connectivity",charIndex:3188},{level:3,title:"Figure 7. Topological Properties of the Ipsilateral Cortical Network",slug:"figure-7-topological-properties-of-the-ipsilateral-cortical-network",normalizedTitle:"figure 7. topological properties of the ipsilateral cortical network",charIndex:3550},{level:2,title:"Supplemental Figures",slug:"supplemental-figures",normalizedTitle:"supplemental figures",charIndex:3856},{level:3,title:"Figure S1. Additional BRICseq Protocol",slug:"figure-s1-additional-bricseq-protocol",normalizedTitle:"figure s1. additional bricseq protocol",charIndex:3881},{level:3,title:"Figure S2. Technical Notes on BRICseq",slug:"figure-s2-technical-notes-on-bricseq",normalizedTitle:"figure s2. technical notes on bricseq",charIndex:3953},{level:3,title:"Figure S3. Reproducibility and Accuracy of BRICseq",slug:"figure-s3-reproducibility-and-accuracy-of-bricseq",normalizedTitle:"figure s3. reproducibility and accuracy of bricseq",charIndex:4021},{level:3,title:"Figure S4. Function-Connection Relationship",slug:"figure-s4-function-connection-relationship",normalizedTitle:"figure s4. function-connection relationship",charIndex:4107},{level:3,title:"Figure S5. Gene Expression-Connection Relationship",slug:"figure-s5-gene-expression-connection-relationship",normalizedTitle:"figure s5. gene expression-connection relationship",charIndex:4178},{level:3,title:"Figure S6. Additional Comparison of BTBR and C57BL/6J Cortical Connectivity",slug:"figure-s6-additional-comparison-of-btbr-and-c57bl-6j-cortical-connectivity",normalizedTitle:"figure s6. additional comparison of btbr and c57bl/6j cortical connectivity",charIndex:4257},{level:3,title:"Figure S7. Additional Analysis of Topological Properties of the Ipsilateral Cortical Network",slug:"figure-s7-additional-analysis-of-topological-properties-of-the-ipsilateral-cortical-network",normalizedTitle:"figure s7. additional analysis of topological properties of the ipsilateral cortical network",charIndex:4377}],excerpt:'<p>: BRICseq <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2020.05.029" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2020.05.029<OutboundLink/></a><br>\nCite: Huang L, Kebschull JM, Frth D, Musall S, Kaufman MT, Churchland AK, Zador AM. BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals. Cell. 2020 Jul 9;182(1):177-188.e27.</p>\n',headersStr:"Abstract Results Figure 1. Mapping Brain-wide Corticocortical Projections with BRICseq Figure 2. Brain-wide Corticocortical Projectome Mapped by BRICseq and Its Validation Figure 3. Validation of BRICseq Figure 4. BRICseq Predicts Functional Connectivity Figure 5. Gene Expression Patterns Predict Connectivity Determined by BRICseq Figure 6. Comparison of BTBR and C57BL/6J Cortical Connectivity Figure 7. Topological Properties of the Ipsilateral Cortical Network Supplemental Figures Figure S1. Additional BRICseq Protocol Figure S2. Technical Notes on BRICseq Figure S3. Reproducibility and Accuracy of BRICseq Figure S4. Function-Connection Relationship Figure S5. Gene Expression-Connection Relationship Figure S6. Additional Comparison of BTBR and C57BL/6J Cortical Connectivity Figure S7. Additional Analysis of Topological Properties of the Ipsilateral Cortical Network",content:": BRICseq \nDOI: https://doi.org/10.1016/j.cell.2020.05.029\nCite: Huang L, Kebschull JM, Frth D, Musall S, Kaufman MT, Churchland AK, Zador AM. BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals. Cell. 2020 Jul 9;182(1):177-188.e27.\n\n\n\nBRICseq \n\n\n\nANTHONY ZADOR\n\nCold Spring Harbor Laboratory\nzador@cshl.edu\n\n\n# Abstract\n\nbrain-wide individual animal connectome sequencingBRICseq DNA  BRICseq neocortex BRICseq  BTBR BRICseq \n\n\n# Results\n\n\n# Figure 1. Mapping Brain-wide Corticocortical Projections with BRICseq\n\n\n\nFigure 1.  BRICseq \n\nA.  source area \nB.  MAPseq barcoded Sindbis virus  single source target areas  RNA barcodes MAPseq  single source area \nC.  BRICseq barcoded Sindbis virus  multiple source areasBRICseq  multiple source areas \nD. soma-max RNA barcodes  soma-max  BC  BC \nE.  soma-max  source cubelets  target cubelets  BC \nF.  soma-max 0.5%\nG. BRICseq pipeline\n\n\n# Figure 2. Brain-wide Corticocortical Projectome Mapped by BRICseq and Its Validation\n\n\n\nFigure 2. Brain-wide Corticocortical Projectome Mapped by BRICseq and Its Validation\n\nA.  BL6-1 -\nB.  source cubelet target cubeletcubelet  FR, frontal area; MO, motor area; SS,somatosensory area; VIS, visual area; AUD, auditory area; STR, striatum; TH, thalamus; AMY, amygdala; TEC, tectum; P/M/SC, pons/medulla/spinal cord; OB,olfactory bulb\nBRICseq  Figure S2\n\n\n# Figure 3. Validation of BRICseq\n\n\n\nFigure 3. BRICseq \n\nA. BL6-1  BL6-2  brain area-to-brain area connection maps \nB.  C57BL/6J \nCD.  BRICseq  Allen Connectome Atlas (C) Allen Atlas  BL6-1 BRICseq  PTLp (D) Allen Connectome  BL6-1 BRICseq \n\n\n# Figure 4. BRICseq Predicts Functional Connectivity\n\n\n\nFigure 4. BRICseq \n\nA. BRICseq connectivity  Ca2+ \nB. \nC. \nD. \nE. \nF. pearson \nG. \n\n\n# Figure 5. Gene Expression Patterns Predict Connectivity Determined by BRICseq\n\n\n\nFigure 5.  BRICseq \n\nA.  PCA  BL6-1  PCs  coefficients n  PCs  BL6-1 BL6-2\nBC.  BL6-1  BL6-1 BL6-2 (B) BL6-1 BL6-2 (C)  10 \nD.  10  10  PC \n\n\n# Figure 6. Comparison of BTBR and C57BL/6J Cortical Connectivity\n\n\n\nFigure 6. BTBR  C57BL/6J \n\nA. C57BL/6J  BTBR \nB.  BTBR  Cubelet-to-cubelet connection matrix\nC. C57BL/6J  BTBR \nD. C57BL/6JBL6-1 BTBRBTBR-1\nE. C57BL/6JBL6-1BTBRBTBR-1\n\n\n# Figure 7. Topological Properties of the Ipsilateral Cortical Network\n\n\n\nFigure 7. \n\nAB. C57BL/6JBL6-12-node (A)  3-node (B) \nC.  BL6-1  cubelet-to-cubelet connection matrix \nD. C57BL/6JBL6-1(C)(D)(D)\n\n\n# Supplemental Figures\n\n\n# Figure S1. Additional BRICseq Protocol\n\n\n\nFigure S1.  BRICseq \n\n\n# Figure S2. Technical Notes on BRICseq\n\n\n\nFigure S2. BRICseq\n\n\n# Figure S3. Reproducibility and Accuracy of BRICseq\n\n\n\nFigure S3. BRICseq \n\n\n# Figure S4. Function-Connection Relationship\n\n\n\nFigure S4. -\n\n\n# Figure S5. Gene Expression-Connection Relationship\n\n\n\nFigure S5. -\n\n\n# Figure S6. Additional Comparison of BTBR and C57BL/6J Cortical Connectivity\n\n\n\nFigure S6. BTBR  C57BL/6J \n\n\n# Figure S7. Additional Analysis of Topological Properties of the Ipsilateral Cortical Network\n\n\n\nFigure S7. ",normalizedContent:": bricseq \ndoi: https://doi.org/10.1016/j.cell.2020.05.029\ncite: huang l, kebschull jm, furth d, musall s, kaufman mt, churchland ak, zador am. bricseq bridges brain-wide interregional connectivity to neural activity and gene expression in single animals. cell. 2020 jul 9;182(1):177-188.e27.\n\n\n\nbricseq \n\n\n\nanthony zador\n\ncold spring harbor laboratory\nzador@cshl.edu\n\n\n# abstract\n\nbrain-wide individual animal connectome sequencingbricseq dna  bricseq neocortex bricseq  btbr bricseq \n\n\n# results\n\n\n# figure 1. mapping brain-wide corticocortical projections with bricseq\n\n\n\nfigure 1.  bricseq \n\na.  source area \nb.  mapseq barcoded sindbis virus  single source target areas  rna barcodes mapseq  single source area \nc.  bricseq barcoded sindbis virus  multiple source areasbricseq  multiple source areas \nd. soma-max rna barcodes  soma-max  bc  bc \ne.  soma-max  source cubelets  target cubelets  bc \nf.  soma-max 0.5%\ng. bricseq pipeline\n\n\n# figure 2. brain-wide corticocortical projectome mapped by bricseq and its validation\n\n\n\nfigure 2. brain-wide corticocortical projectome mapped by bricseq and its validation\n\na.  bl6-1 -\nb.  source cubelet target cubeletcubelet  fr, frontal area; mo, motor area; ss,somatosensory area; vis, visual area; aud, auditory area; str, striatum; th, thalamus; amy, amygdala; tec, tectum; p/m/sc, pons/medulla/spinal cord; ob,olfactory bulb\nbricseq  figure s2\n\n\n# figure 3. validation of bricseq\n\n\n\nfigure 3. bricseq \n\na. bl6-1  bl6-2  brain area-to-brain area connection maps \nb.  c57bl/6j \ncd.  bricseq  allen connectome atlas (c) allen atlas  bl6-1 bricseq  ptlp (d) allen connectome  bl6-1 bricseq \n\n\n# figure 4. bricseq predicts functional connectivity\n\n\n\nfigure 4. bricseq \n\na. bricseq connectivity  ca2+ \nb. \nc. \nd. \ne. \nf. pearson \ng. \n\n\n# figure 5. gene expression patterns predict connectivity determined by bricseq\n\n\n\nfigure 5.  bricseq \n\na.  pca  bl6-1  pcs  coefficients n  pcs  bl6-1 bl6-2\nbc.  bl6-1  bl6-1 bl6-2 (b) bl6-1 bl6-2 (c)  10 \nd.  10  10  pc \n\n\n# figure 6. comparison of btbr and c57bl/6j cortical connectivity\n\n\n\nfigure 6. btbr  c57bl/6j \n\na. c57bl/6j  btbr \nb.  btbr  cubelet-to-cubelet connection matrix\nc. c57bl/6j  btbr \nd. c57bl/6jbl6-1 btbrbtbr-1\ne. c57bl/6jbl6-1btbrbtbr-1\n\n\n# figure 7. topological properties of the ipsilateral cortical network\n\n\n\nfigure 7. \n\nab. c57bl/6jbl6-12-node (a)  3-node (b) \nc.  bl6-1  cubelet-to-cubelet connection matrix \nd. c57bl/6jbl6-1(c)(d)(d)\n\n\n# supplemental figures\n\n\n# figure s1. additional bricseq protocol\n\n\n\nfigure s1.  bricseq \n\n\n# figure s2. technical notes on bricseq\n\n\n\nfigure s2. bricseq\n\n\n# figure s3. reproducibility and accuracy of bricseq\n\n\n\nfigure s3. bricseq \n\n\n# figure s4. function-connection relationship\n\n\n\nfigure s4. -\n\n\n# figure s5. gene expression-connection relationship\n\n\n\nfigure s5. -\n\n\n# figure s6. additional comparison of btbr and c57bl/6j cortical connectivity\n\n\n\nfigure s6. btbr  c57bl/6j \n\n\n# figure s7. additional analysis of topological properties of the ipsilateral cortical network\n\n\n\nfigure s7. ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Shared and distinct transcriptomic cell types across neocortical areas",frontmatter:{title:" --- Shared and distinct transcriptomic cell types across neocortical areas",date:"2022-12-10T11:03:34.000Z",permalink:"/pages/af94c8/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20181031.(2018.10.31%20%E6%9B%BE%E7%BA%A2%E8%91%B5%20Nature)%20Shared%20and%20distinct%20transcriptomic%20cell%20types%20across%20neocortical%20areas.html",relativePath:"02./11./02./20181031.(2018.10.31  Nature) Shared and distinct transcriptomic cell types across neocortical areas.md",key:"v-21fa6f80",path:"/pages/af94c8/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:498},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:892},{level:3,title:"Fig1. Cell type taxonomy in ALM and VISp cortical areas",slug:"fig1-cell-type-taxonomy-in-alm-and-visp-cortical-areas",normalizedTitle:"fig1. cell type taxonomy in alm and visp cortical areas",charIndex:904},{level:3,title:"Fig2. Comparison of gene expression differences among types across cortical areas",slug:"fig2-comparison-of-gene-expression-differences-among-types-across-cortical-areas",normalizedTitle:"fig2. comparison of gene expression differences among types across cortical areas",charIndex:1188},{level:3,title:"Fig3. Glutamatergic cell types by scRNA-seq and projections",slug:"fig3-glutamatergic-cell-types-by-scrna-seq-and-projections",normalizedTitle:"fig3. glutamatergic cell types by scrna-seq and projections",charIndex:2066},{level:3,title:"Fig4. Glutamatergic cell types and markers",slug:"fig4-glutamatergic-cell-types-and-markers",normalizedTitle:"fig4. glutamatergic cell types and markers",charIndex:2439},{level:3,title:"Fig5. GABAergic cell types by scRNA-seq",slug:"fig5-gabaergic-cell-types-by-scrna-seq",normalizedTitle:"fig5. gabaergic cell types by scrna-seq",charIndex:2939}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-018-0654-5" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-018-0654-5<OutboundLink/></a><br>\nCite: Tasic, B., Yao, Z., Graybuck, L.T. et al. Shared and distinct transcriptomic cell types across neocortical areas. <em>Nature</em> 563, 7278 (2018).</p>\n',headersStr:"Abstract Results Fig1. Cell type taxonomy in ALM and VISp cortical areas Fig2. Comparison of gene expression differences among types across cortical areas Fig3. Glutamatergic cell types by scRNA-seq and projections Fig4. Glutamatergic cell types and markers Fig5. GABAergic cell types by scRNA-seq",content:": \nDOI: https://doi.org/10.1038/s41586-018-0654-5\nCite: Tasic, B., Yao, Z., Graybuck, L.T. et al. Shared and distinct transcriptomic cell types across neocortical areas. Nature 563, 7278 (2018).\n\n\n\n\n\n\n\nBOSILJKA TASIC                      HONGKUI ZENG()\n                                    \nAllen Institute for Brain Science   Allen Institute for Brain Science\nbosiljkat@alleninstitute.org        hongkuiz@alleninstitute.org\n\n\n# Abstract\n\n (neocortex)  23,822 :  (primary visual cortex)  (anterior lateral motor cortex) RNA  133  GABA (-)  (glutamatergic)  RNA \n\n\n# Results\n\n\n# Fig1. Cell type taxonomy in ALM and VISp cortical areas\n\n\n\nFig1. ALM  VISp \n\na.  ALM  VISp  Retro-seq\nb. \nc.  4020  23,822  61  GABAergic56  glutamatergic  16  classes  subclasses \n\n\n# Fig2. Comparison of gene expression differences among types across cortical areas\n\n\n\nFig2. \n\na. t-SNE  4020  n=23,822  regionclass  cluster  glutamatergic  ALM- or VISp-specific GABAergic (salt-and-pepper clusters, left t-SNE)\nb.  8778  clusters (DE)xy ALM  VISp  GABAergic cluster glutamatergic ALM  VISp clusters panels subclass  subclasses non-neuronal  classes (GABA  glutamate)\nc.  ALM- and VISp-specific   ALM  VISp  (Extended Data Fig. 10c)x cell types  Fig1  ALM  VISp  ALM-  VISp-enriched genes \nd. ALM- or VISp-specific genes  glutamatergic  GABAergic \n\n\n# Fig3. Glutamatergic cell types by scRNA-seq and projections\n\n\n\nFig3. Glutamatergic scRNA-seq \n\na. Retro-seq(ALMVISp)1b\nb. target cortex (CTX), striatum (STR), thalamus (TH), tectum (TEC), pons (P) or medulla (MY)\nc. ALM  glutamatergic \nd. VISp  glutamatergic \n\n\n# Fig4. Glutamatergic cell types and markers\n\n\n\nFig4. Glutamatergic  markers\n\na. ALM  VISp  cluster  core cell (n=10,729)edge weights (n=1,136)L6-CT-Nxph2-SlaL6b-Col8a1-RprmL6b-Hsd17b2L6b-P2ry12  Cajal-Retzius \nb.  Fig.1c  glutamatergic glutamatergic clusters  11871  8477 \nc.  cluster  0  log10 (n=11,827 cells)\n\n\n# Fig5. GABAergic cell types by scRNA-seq\n\n\n\nFig5. GABAergic  markers\n\na. Sst and Pvalba Lamp5, Serpinf1, Sncg and Vipb (n=9,021 core cells; n=1,457 intermediate cells) subclasses  Meis2 \ncd.  Fig.1c  GABAergic  Fig.4b n=4,675 out of 5,365 cells in c, and 3,908 out of 5,113 cells in d\nef.  cluster  Fig.4c n=5,365 cells in e and 5,113 cells in f",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-018-0654-5\ncite: tasic, b., yao, z., graybuck, l.t. et al. shared and distinct transcriptomic cell types across neocortical areas. nature 563, 7278 (2018).\n\n\n\n\n\n\n\nbosiljka tasic                      hongkui zeng()\n                                    \nallen institute for brain science   allen institute for brain science\nbosiljkat@alleninstitute.org        hongkuiz@alleninstitute.org\n\n\n# abstract\n\n (neocortex)  23,822 :  (primary visual cortex)  (anterior lateral motor cortex) rna  133  gaba (-)  (glutamatergic)  rna \n\n\n# results\n\n\n# fig1. cell type taxonomy in alm and visp cortical areas\n\n\n\nfig1. alm  visp \n\na.  alm  visp  retro-seq\nb. \nc.  4020  23,822  61  gabaergic56  glutamatergic  16  classes  subclasses \n\n\n# fig2. comparison of gene expression differences among types across cortical areas\n\n\n\nfig2. \n\na. t-sne  4020  n=23,822  regionclass  cluster  glutamatergic  alm- or visp-specific gabaergic (salt-and-pepper clusters, left t-sne)\nb.  8778  clusters (de)xy alm  visp  gabaergic cluster glutamatergic alm  visp clusters panels subclass  subclasses non-neuronal  classes (gaba  glutamate)\nc.  alm- and visp-specific   alm  visp  (extended data fig. 10c)x cell types  fig1  alm  visp  alm-  visp-enriched genes \nd. alm- or visp-specific genes  glutamatergic  gabaergic \n\n\n# fig3. glutamatergic cell types by scrna-seq and projections\n\n\n\nfig3. glutamatergic scrna-seq \n\na. retro-seq(almvisp)1b\nb. target cortex (ctx), striatum (str), thalamus (th), tectum (tec), pons (p) or medulla (my)\nc. alm  glutamatergic \nd. visp  glutamatergic \n\n\n# fig4. glutamatergic cell types and markers\n\n\n\nfig4. glutamatergic  markers\n\na. alm  visp  cluster  core cell (n=10,729)edge weights (n=1,136)l6-ct-nxph2-slal6b-col8a1-rprml6b-hsd17b2l6b-p2ry12  cajal-retzius \nb.  fig.1c  glutamatergic glutamatergic clusters  11871  8477 \nc.  cluster  0  log10 (n=11,827 cells)\n\n\n# fig5. gabaergic cell types by scrna-seq\n\n\n\nfig5. gabaergic  markers\n\na. sst and pvalba lamp5, serpinf1, sncg and vipb (n=9,021 core cells; n=1,457 intermediate cells) subclasses  meis2 \ncd.  fig.1c  gabaergic  fig.4b n=4,675 out of 5,365 cells in c, and 3,908 out of 5,113 cells in d\nef.  cluster  fig.4c n=5,365 cells in e and 5,113 cells in f",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Distinct subnetworks of the thalamic reticular nucleus",frontmatter:{title:" --- Distinct subnetworks of the thalamic reticular nucleus",date:"2023-03-02T20:09:08.000Z",permalink:"/pages/4918af/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20200722.(2020.07.22%20%E4%B8%98%E8%84%91%E7%BD%91%E7%8A%B6%E6%A0%B8%20Nature)%20Distinct%20subnetworks%20of%20the%20thalamic%20reticular%20nucleus.html",relativePath:"02./11./02./20200722.(2020.07.22  Nature) Distinct subnetworks of the thalamic reticular nucleus.md",key:"v-393821ce",path:"/pages/4918af/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:614},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:1015},{level:3,title:"Fig1. snRNA-seq reveals transcriptomic gradient of TRN neurons",slug:"fig1-snrna-seq-reveals-transcriptomic-gradient-of-trn-neurons",normalizedTitle:"fig1. snrna-seq reveals transcriptomic gradient of trn neurons",charIndex:1028},{level:3,title:"Fig2. Ecel1+ and Spp1+ neurons show distinct spatial distribution across the TRN",slug:"fig2-ecel1-and-spp1-neurons-show-distinct-spatial-distribution-across-the-trn",normalizedTitle:"fig2. ecel1+ and spp1+ neurons show distinct spatial distribution across the trn",charIndex:1493},{level:3,title:"Fig3. Topographical map of TRN-thalamus projections",slug:"fig3-topographical-map-of-trn-thalamus-projections",normalizedTitle:"fig3. topographical map of trn-thalamus projections",charIndex:1741},{level:3,title:"Fig4. Spp1+ and Ecel1+ TRN subpopulations show distinct electrophysiological signatures",slug:"fig4-spp1-and-ecel1-trn-subpopulations-show-distinct-electrophysiological-signatures",normalizedTitle:"fig4. spp1+ and ecel1+ trn subpopulations show distinct electrophysiological signatures",charIndex:1974},{level:3,title:"Fig5. Selective perturbation of the firing properties of Spp1+ or Ecel1+ neurons in vivo reveals their differential participation in thalamocortical rhythms",slug:"fig5-selective-perturbation-of-the-firing-properties-of-spp1-or-ecel1-neurons-in-vivo-reveals-their-differential-participation-in-thalamocortical-rhythms",normalizedTitle:"fig5. selective perturbation of the firing properties of spp1+ or ecel1+ neurons in vivo reveals their differential participation in thalamocortical rhythms",charIndex:2641}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-020-2504-5" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-020-2504-5<OutboundLink/></a><br>\nCite: Li, Y., Lopez-Huerta, V.G., Adiconis, X. et al. Distinct subnetworks of the thalamic reticular nucleus. Nature 583, 819824 (2020).</p>\n',headersStr:"Abstract: Results: Fig1. snRNA-seq reveals transcriptomic gradient of TRN neurons Fig2. Ecel1+ and Spp1+ neurons show distinct spatial distribution across the TRN Fig3. Topographical map of TRN-thalamus projections Fig4. Spp1+ and Ecel1+ TRN subpopulations show distinct electrophysiological signatures Fig5. Selective perturbation of the firing properties of Spp1+ or Ecel1+ neurons in vivo reveals their differential participation in thalamocortical rhythms",content:": \nDOI: https://doi.org/10.1038/s41586-020-2504-5\nCite: Li, Y., Lopez-Huerta, V.G., Adiconis, X. et al. Distinct subnetworks of the thalamic reticular nucleus. Nature 583, 819824 (2020).\n\n\n\n\n\n\n\nGUOPING FENG()                    ZHANYAN FU                           JOSHUA LEVIN\n                                                                          \nBroad Institute of MIT and Harvard   Broad Institute of MIT and Harvard   Broad Institute of MIT and Harvard\nfengg@mit.edu                        zfu@broadinstitute.org               jlevin@broadinstitute.org\n\n\n# Abstract:\n\nTRNTRNTRNTRNTRNTRN-TRN\n\n\n# Results:\n\n\n# Fig1. snRNA-seq reveals transcriptomic gradient of TRN neurons\n\n\n\nFig1. snRNA-seq  TRN \n\na. TRN  TRN TRN tdTomato\nb. t-SNE  clusters\nc. Gad2+  Pvalb+  biSNE embedding\nd. Spp1  Ecel1  biSNE embedding\ne.  Spp1  Ecel1 \nf.  TRN  Spp1  Ecel1 \ng. TRN Spp1+Ecel1+  DN  DP \n\n\n# Fig2. Ecel1+ and Spp1+ neurons show distinct spatial distribution across the TRN\n\n\n\nFig2. Ecel1+  Spp1+  TRN \n\na.  RNA FISH TRN  Pvalb  FISH \nb. a 5\nc. Spp1+Ecel1+DN  DP (A)(P)\n\n\n# Fig3. Topographical map of TRN-thalamus projections\n\n\n\nFig3. TRN-\n\na.  FO  HO RetroBeads TRN  FO  HO \nb.  Spp1+Ecel1+Spp1Ecel1Spp1+Ecel1+ \nc. \n\n\n# Fig4. Spp1+ and Ecel1+ TRN subpopulations show distinct electrophysiological signatures\n\n\n\nFig4. Spp1+  Ecel1+ TRN \n\na. TRNPatch-seq\nb. RBs\nc. P=0.0001tn=27Spp1+n=22Ecel1+n=10DPn=13DNAHPn=26Spp1+n=16Ecel1+n=9DPn=12DN5\nd. n=20Spp1+n=16Ecel1+n=10DPn=13DN5\ne. rbFreqAHP3D5n=26Spp1+n=16Ecel1+n=9DPn=12DN\nf. n=68\n\n\n# Fig5. Selective perturbation of the firing properties of Spp1+ or Ecel1+ neurons in vivo reveals their differential participation in thalamocortical rhythms\n\n\n\nFig5.  Spp1+  Ecel1+ \n\na. FOHOTRNPvalb+6-mCherryTRNPvalb+100 m\nb. FOHO6-AAVTRNFOHOSpp1+Ecel1+\nc. 6TRNFO6.11.661.70.8P=0.0001HO1.10.2660.30.1P=0.0081ta-cFOn=14n=1864HOn=12n=1164\nd-f. FOTRN61-4 HzdefHOTRN6fdFOP=0.0045HOP=0.75teFOP=0.038HOP=0.96WilcoxonfFOP=2 x 10^-23HOP=4.5 x 10^-13Kolmogorov-Smirnovd-fFOn=8n=86HOn=7n=8682575NS",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-020-2504-5\ncite: li, y., lopez-huerta, v.g., adiconis, x. et al. distinct subnetworks of the thalamic reticular nucleus. nature 583, 819824 (2020).\n\n\n\n\n\n\n\nguoping feng()                    zhanyan fu                           joshua levin\n                                                                          \nbroad institute of mit and harvard   broad institute of mit and harvard   broad institute of mit and harvard\nfengg@mit.edu                        zfu@broadinstitute.org               jlevin@broadinstitute.org\n\n\n# abstract:\n\ntrntrntrntrntrntrn-trn\n\n\n# results:\n\n\n# fig1. snrna-seq reveals transcriptomic gradient of trn neurons\n\n\n\nfig1. snrna-seq  trn \n\na. trn  trn trn tdtomato\nb. t-sne  clusters\nc. gad2+  pvalb+  bisne embedding\nd. spp1  ecel1  bisne embedding\ne.  spp1  ecel1 \nf.  trn  spp1  ecel1 \ng. trn spp1+ecel1+  dn  dp \n\n\n# fig2. ecel1+ and spp1+ neurons show distinct spatial distribution across the trn\n\n\n\nfig2. ecel1+  spp1+  trn \n\na.  rna fish trn  pvalb  fish \nb. a 5\nc. spp1+ecel1+dn  dp (a)(p)\n\n\n# fig3. topographical map of trn-thalamus projections\n\n\n\nfig3. trn-\n\na.  fo  ho retrobeads trn  fo  ho \nb.  spp1+ecel1+spp1ecel1spp1+ecel1+ \nc. \n\n\n# fig4. spp1+ and ecel1+ trn subpopulations show distinct electrophysiological signatures\n\n\n\nfig4. spp1+  ecel1+ trn \n\na. trnpatch-seq\nb. rbs\nc. p=0.0001tn=27spp1+n=22ecel1+n=10dpn=13dnahpn=26spp1+n=16ecel1+n=9dpn=12dn5\nd. n=20spp1+n=16ecel1+n=10dpn=13dn5\ne. rbfreqahp3d5n=26spp1+n=16ecel1+n=9dpn=12dn\nf. n=68\n\n\n# fig5. selective perturbation of the firing properties of spp1+ or ecel1+ neurons in vivo reveals their differential participation in thalamocortical rhythms\n\n\n\nfig5.  spp1+  ecel1+ \n\na. fohotrnpvalb+6-mcherrytrnpvalb+100 m\nb. foho6-aavtrnfohospp1+ecel1+\nc. 6trnfo6.11.661.70.8p=0.0001ho1.10.2660.30.1p=0.0081ta-cfon=14n=1864hon=12n=1164\nd-f. fotrn61-4 hzdefhotrn6fdfop=0.0045hop=0.75tefop=0.038hop=0.96wilcoxonffop=2 x 10^-23hop=4.5 x 10^-13kolmogorov-smirnovd-ffon=8n=86hon=7n=8682575ns",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Circuit and molecular architecture of a ventral hippocampal network",frontmatter:{title:" --- Circuit and molecular architecture of a ventral hippocampal network",categories:[""],tags:[""],date:"2022-11-18T21:33:52.000Z",permalink:"/pages/fa4bef/",readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20200914.(2020.09.14%20MAPseq%E6%B5%B7%E9%A9%AC%20Nat%20Neurosci)%20Circuit%20and%20molecular%20architecture%20of%20a%20ventral%20hippocampal%20network.html",relativePath:"02./11./02./20200914.(2020.09.14 MAPseq Nat Neurosci) Circuit and molecular architecture of a ventral hippocampal network.md",key:"v-4fbe66c8",path:"/pages/fa4bef/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:347},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:694},{level:3,title:"Fig1. MAPseq vCA1 projections ",slug:"fig1-mapseq-vca1-projections-",normalizedTitle:"fig1. mapseq vca1 projections ",charIndex:707},{level:3,title:"Fig2. vCA1 ",slug:"fig2-vca1-",normalizedTitle:"fig2. vca1 ",charIndex:1423},{level:3,title:"Fig3. Viral inputoutput tracing of vCA1",slug:"fig3-viral-input-output-tracing-of-vca1",normalizedTitle:"fig3. viral inputoutput tracing of vca1",charIndex:2076},{level:3,title:"Fig4. RV-labeled inputs to vCA1 output neurons",slug:"fig4-rv-labeled-inputs-to-vca1-output-neurons",normalizedTitle:"fig4. rv-labeled inputs to vca1 output neurons",charIndex:2741},{level:3,title:"Fig5. Transcriptional profiling of vCA1 neurons defined by connectivity",slug:"fig5-transcriptional-profiling-of-vca1-neurons-defined-by-connectivity",normalizedTitle:"fig5. transcriptional profiling of vca1 neurons defined by connectivity",charIndex:3157}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-020-0705-8" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-020-0705-8<OutboundLink/></a><br>\nCite: Gergues MM, Han KJ, Choi HS, et al. Circuit and molecular architecture of a ventral hippocampal network. <em>Nat Neurosci</em> 23, 14441452 (2020).</p>\n',headersStr:"Abstract Results Fig1. MAPseq vCA1 projections  Fig2. vCA1  Fig3. Viral inputoutput tracing of vCA1 Fig4. RV-labeled inputs to vCA1 output neurons Fig5. Transcriptional profiling of vCA1 neurons defined by connectivity",content:": \nDOI: https://doi.org/10.1038/s41593-020-0705-8\nCite: Gergues MM, Han KJ, Choi HS, et al. Circuit and molecular architecture of a ventral hippocampal network. Nat Neurosci 23, 14441452 (2020).\n\n\n\n\n\n\n\nMAZEN KHEIRBEK\n\nPsychiatry UCSF Weill Institute for Neurosciences\nmazen.kheirbek@ucsf.edu\n\n\n# Abstract\n\n(ventral hippocampus, vHPC)CA1(vCA1) vCA1  vCA1 (MAPseq) vCA1 - vCA1  vCA1 \n\n\n# Results\n\n\n# Fig1. MAPseq vCA1 projections \n\n\n\nFig1. MAPseq vCA1 projections \n\na. MAPseq 2x107  barcode Sindbis Virus vCA1 44 vCA1  7 (mPFCLSBNSTNAcCeABALH ) Bulk RNA-seq  Barcode  Barcodes  brain target  Barcode  vCA1  2494  barcode  24947 \nb. vCA1  MAPseq (GFP)\nc.  2494  vCA1  7  target  vCA1  LS \nd.  vCA1  target 77% 18.2%  targets3.73% 1.12% \ne.  A  B  LS  LS \n\n\n# Fig2. vCA1 \n\n\n\nFig2. vCA1 \n\na.  vCA1  targets\nb. vCA1  UpSet  3  7  target  LS  CeA  NAc  LS  BNST  LS   null model null model  vCA1  7  target \nc.  vCA1 \nde. x  7  target y  target  vCA1 \n\n\n# Fig3. Viral inputoutput tracing of vCA1\n\n\n\nFig3. Viral inputoutput tracing of vCA1\n\na.  vCA1  vCA1  CeA  6  vCA1  mPFC  mPFC  AAV2retro-CAG-Cre  vCA1  AAV1-FLEX-TVA-EGFP-G  mPFC  vCA1  Cre Cre  FLEX TVA-EGFP-G21  CA1  RV-EnvA-G-mCherry G(G)TVA  EnvA RV-EnvA-G-mCherry  TVA-EGFP-G  vCA1  vCA1  vCA1  Cre TVA-EGFP-G   vCA1  3\nb. 6  vCA1  RV-EnvA-G-mCherry  AAV1-FLEX-TVA-EGFP-G \nc. 6  vHPC  AP \n\n\n# Fig4. RV-labeled inputs to vCA1 output neurons\n\n\n\nFig4. RV-labeled inputs to vCA1 output neurons\n\na. RV-mCherry .\nb.  5  5  6  6  vCA1  6  vCA1 \nc. b\nd.  vCA1 \n\n\n# Fig5. Transcriptional profiling of vCA1 neurons defined by connectivity\n\n\n\nFig5. Transcriptional profiling of vCA1 neurons defined by connectivity\n\na.  vCA1  EGFP  bulk RNAseq \nb.  vCA1  mPFC  BALHNAc \nc. vCA1  mPFC \nde. GO vCA1  mPFC ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-020-0705-8\ncite: gergues mm, han kj, choi hs, et al. circuit and molecular architecture of a ventral hippocampal network. nat neurosci 23, 14441452 (2020).\n\n\n\n\n\n\n\nmazen kheirbek\n\npsychiatry ucsf weill institute for neurosciences\nmazen.kheirbek@ucsf.edu\n\n\n# abstract\n\n(ventral hippocampus, vhpc)ca1(vca1) vca1  vca1 (mapseq) vca1 - vca1  vca1 \n\n\n# results\n\n\n# fig1. mapseq vca1 projections \n\n\n\nfig1. mapseq vca1 projections \n\na. mapseq 2x107  barcode sindbis virus vca1 44 vca1  7 (mpfclsbnstnacceabalh ) bulk rna-seq  barcode  barcodes  brain target  barcode  vca1  2494  barcode  24947 \nb. vca1  mapseq (gfp)\nc.  2494  vca1  7  target  vca1  ls \nd.  vca1  target 77% 18.2%  targets3.73% 1.12% \ne.  a  b  ls  ls \n\n\n# fig2. vca1 \n\n\n\nfig2. vca1 \n\na.  vca1  targets\nb. vca1  upset  3  7  target  ls  cea  nac  ls  bnst  ls   null model null model  vca1  7  target \nc.  vca1 \nde. x  7  target y  target  vca1 \n\n\n# fig3. viral inputoutput tracing of vca1\n\n\n\nfig3. viral inputoutput tracing of vca1\n\na.  vca1  vca1  cea  6  vca1  mpfc  mpfc  aav2retro-cag-cre  vca1  aav1-flex-tva-egfp-g  mpfc  vca1  cre cre  flex tva-egfp-g21  ca1  rv-enva-g-mcherry g(g)tva  enva rv-enva-g-mcherry  tva-egfp-g  vca1  vca1  vca1  cre tva-egfp-g   vca1  3\nb. 6  vca1  rv-enva-g-mcherry  aav1-flex-tva-egfp-g \nc. 6  vhpc  ap \n\n\n# fig4. rv-labeled inputs to vca1 output neurons\n\n\n\nfig4. rv-labeled inputs to vca1 output neurons\n\na. rv-mcherry .\nb.  5  5  6  6  vca1  6  vca1 \nc. b\nd.  vca1 \n\n\n# fig5. transcriptional profiling of vca1 neurons defined by connectivity\n\n\n\nfig5. transcriptional profiling of vca1 neurons defined by connectivity\n\na.  vca1  egfp  bulk rnaseq \nb.  vca1  mpfc  balhnac \nc. vca1  mpfc \nde. go vca1  mpfc ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Uncovering the genetic blueprint of the C. elegans nervous system",frontmatter:{title:" --- Uncovering the genetic blueprint of the C. elegans nervous system",date:"2024-08-30T20:14:01.000Z",permalink:"/pages/521e00/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20201214.(2020.12.14%20SCM%20PNAS)%20Uncovering%20the%20genetic%20blueprint%20of%20the%20C.%20elegans%20nervous%20system.html",relativePath:"02./11./02./20201214.(2020.12.14 SCM PNAS) Uncovering the genetic blueprint of the C. elegans nervous system.md",key:"v-98642aa8",path:"/pages/521e00/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:353}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1073/pnas.2009093117" target="_blank" rel="noopener noreferrer">https://doi.org/10.1073/pnas.2009093117<OutboundLink/></a><br>\nCite: Kovcs, I.A.; Barabsi, D.L.; Barabsi, A.-L. Uncovering the genetic blueprint of the C. elegans nervous system. Proc. Natl. Acad. Sci. USA 2020, 117, 3357033577.</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1073/pnas.2009093117\nCite: Kovcs, I.A.; Barabsi, D.L.; Barabsi, A.-L. Uncovering the genetic blueprint of the C. elegans nervous system. Proc. Natl. Acad. Sci. USA 2020, 117, 3357033577.\n\n\n\n\n\n\n\nALBERT-LSZL BARABSI\n\nNortheastern University\na.barabasi@northeastern.edu\n\n\n# Abstract\n\n 19  5 ",normalizedContent:": \ndoi: https://doi.org/10.1073/pnas.2009093117\ncite: kovacs, i.a.; barabasi, d.l.; barabasi, a.-l. uncovering the genetic blueprint of the c. elegans nervous system. proc. natl. acad. sci. usa 2020, 117, 3357033577.\n\n\n\n\n\n\n\nalbert-laszlo barabasi\n\nnortheastern university\na.barabasi@northeastern.edu\n\n\n# abstract\n\n 19  5 ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing",frontmatter:{title:" --- High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing",date:"2022-12-11T15:49:16.000Z",permalink:"/pages/da828f/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20191017.(2019.10.17%20BAR-seq%20Cell)%20High-Throughput%20Mapping%20of%20Long-Range%20Neuronal%20Projection%20Using%20In%20Situ%20Sequencing.html",relativePath:"02./11./02./20191017.(2019.10.17 BAR-seq Cell) High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing.md",key:"v-64a80ee8",path:"/pages/da828f/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:396},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:877},{level:3,title:"Fig1. Multiplexed Projection Mapping Using In Situ Sequencing",slug:"fig1-multiplexed-projection-mapping-using-in-situ-sequencing",normalizedTitle:"fig1. multiplexed projection mapping using in situ sequencing",charIndex:889},{level:3,title:"Fig2. Validation of BARseq",slug:"fig2-validation-of-barseq",normalizedTitle:"fig2. validation of barseq",charIndex:1132},{level:3,title:"Fig3. Correlating Gene Expression and Projections Using BARseq",slug:"fig3-correlating-gene-expression-and-projections-using-barseq",normalizedTitle:"fig3. correlating gene expression and projections using barseq",charIndex:1692},{level:3,title:"Fig4. Mapping Projections of the Auditory Cortex Using BARseq",slug:"fig4-mapping-projections-of-the-auditory-cortex-using-barseq",normalizedTitle:"fig4. mapping projections of the auditory cortex using barseq",charIndex:2179},{level:3,title:"Fig5. The Laminar Organization of Projection Neurons in the Auditory Cortex",slug:"fig5-the-laminar-organization-of-projection-neurons-in-the-auditory-cortex",normalizedTitle:"fig5. the laminar organization of projection neurons in the auditory cortex",charIndex:2553},{level:3,title:"Fig6. Subtypes of IT Neurons Defined by Gene Expression in the Auditory Cortex",slug:"fig6-subtypes-of-it-neurons-defined-by-gene-expression-in-the-auditory-cortex",normalizedTitle:"fig6. subtypes of it neurons defined by gene expression in the auditory cortex",charIndex:2774},{level:3,title:"Fig7. Projections across IT Subtypes Defined by Gene Expression",slug:"fig7-projections-across-it-subtypes-defined-by-gene-expression",normalizedTitle:"fig7. projections across it subtypes defined by gene expression",charIndex:3007}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2019.09.023" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2019.09.023<OutboundLink/></a><br>\nCite: Chen X, Sun YC, Zhan H, Kebschull JM, Fischer S, Matho K, Huang ZJ, Gillis J, Zador AM. High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing. Cell. 2019 Oct 17;179(3):772-786.e19.</p>\n',headersStr:"Abstract Results Fig1. Multiplexed Projection Mapping Using In Situ Sequencing Fig2. Validation of BARseq Fig3. Correlating Gene Expression and Projections Using BARseq Fig4. Mapping Projections of the Auditory Cortex Using BARseq Fig5. The Laminar Organization of Projection Neurons in the Auditory Cortex Fig6. Subtypes of IT Neurons Defined by Gene Expression in the Auditory Cortex Fig7. Projections across IT Subtypes Defined by Gene Expression",content:": \nDOI: https://doi.org/10.1016/j.cell.2019.09.023\nCite: Chen X, Sun YC, Zhan H, Kebschull JM, Fischer S, Matho K, Huang ZJ, Gillis J, Zador AM. High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing. Cell. 2019 Oct 17;179(3):772-786.e19.\n\n\n\n\n\n\n\nANTHONY ZADOR\n\nCold Spring Harbor Laboratory\nzador@cshl.edu\n\n\n# Abstract\n\n (barcoded anatomy resolved by sequencingBARseq) RNA  Cre  BARseq  11 3,579 intratelencephalicITpyramidal tract-likePT-likecorticothalamicCT IT BARseq \n\n\n# Results\n\n\n# Fig1. Multiplexed Projection Mapping Using In Situ Sequencing\n\n\n\nFig1. \n\nA. MAPseq ()  BARseq () \nB. NGS  in situ sequencing \nC. in situ sequencing  \nD. GFP  Barcode Barcode\nE. \n\n\n# Fig2. Validation of BARseq\n\n\n\nFig2. BARseq \n\nA. GFP  Rolonies  No Barcode \nB. \nC. \nD. \nE. in situ sequences  random sequences  barcode reads \nF. in situ sequences  random sequences \nG. CTB (cholera toxin subunit B, B) contralateral auditory cortex  CTB ipsilateral auditory cortex  barcode\nH. contralateral auditory cortex  barcodes ()  CTB () \nI. contralateral auditory cortex  CTB  BARseq Venn\n\n\n# Fig3. Correlating Gene Expression and Projections Using BARseq\n\n\n\nFig3.  BARseq \n\nA.  Fezf2+ tdTomato () GFP  barcoded ()()()\nB.  1,291  projection neurons ()72 Fezf2+ neurons ()\nC. 72 Fezf2+ neurons  cortical depth \nD.  Slc17a7/Gad2 FISH()  ()\nE. D Slc17a7/Gad2  barcode sequences\nF.  Slc17a7/Gad2 (Proj+)(Proj-)\nG. barcoded  non-barcoded neurons \n\n\n# Fig4. Mapping Projections of the Auditory Cortex Using BARseq\n\n\n\nFig4.  BARseq \n\nA. BARseq  barcodes  hamming distance \nB.  6,391 projection neurons (1309 BARseq 5,082 MAPseq) \nC. BARseq  bulk GFP (barcode)\nD.  projection area  barcode counts \nE. \nF. \nG. \n\n\n# Fig5. The Laminar Organization of Projection Neurons in the Auditory Cortex\n\n\n\nFig5. \n\nA. \nB. \nC. \nD.  t-SNE\nE. \n\n\n# Fig6. Subtypes of IT Neurons Defined by Gene Expression in the Auditory Cortex\n\n\n\nFig6.  IT \n\nA. 735  scRNAseq\nB.  t-SNE\nC.  Tasic et al.  MetaNeighbor \nD. \nE. 4IT\n\n\n# Fig7. Projections across IT Subtypes Defined by Gene Expression\n\n\n\nFig7.  IT \n\nA. ITi-Ctx  4  IT \nB. 4  IT \nC. 4  IT  barcode \nD.  IT  4  IT \nE. Cdh13+/- IT  +/-  AudC \nF.  4  IT ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2019.09.023\ncite: chen x, sun yc, zhan h, kebschull jm, fischer s, matho k, huang zj, gillis j, zador am. high-throughput mapping of long-range neuronal projection using in situ sequencing. cell. 2019 oct 17;179(3):772-786.e19.\n\n\n\n\n\n\n\nanthony zador\n\ncold spring harbor laboratory\nzador@cshl.edu\n\n\n# abstract\n\n (barcoded anatomy resolved by sequencingbarseq) rna  cre  barseq  11 3,579 intratelencephalicitpyramidal tract-likept-likecorticothalamicct it barseq \n\n\n# results\n\n\n# fig1. multiplexed projection mapping using in situ sequencing\n\n\n\nfig1. \n\na. mapseq ()  barseq () \nb. ngs  in situ sequencing \nc. in situ sequencing  \nd. gfp  barcode barcode\ne. \n\n\n# fig2. validation of barseq\n\n\n\nfig2. barseq \n\na. gfp  rolonies  no barcode \nb. \nc. \nd. \ne. in situ sequences  random sequences  barcode reads \nf. in situ sequences  random sequences \ng. ctb (cholera toxin subunit b, b) contralateral auditory cortex  ctb ipsilateral auditory cortex  barcode\nh. contralateral auditory cortex  barcodes ()  ctb () \ni. contralateral auditory cortex  ctb  barseq venn\n\n\n# fig3. correlating gene expression and projections using barseq\n\n\n\nfig3.  barseq \n\na.  fezf2+ tdtomato () gfp  barcoded ()()()\nb.  1,291  projection neurons ()72 fezf2+ neurons ()\nc. 72 fezf2+ neurons  cortical depth \nd.  slc17a7/gad2 fish()  ()\ne. d slc17a7/gad2  barcode sequences\nf.  slc17a7/gad2 (proj+)(proj-)\ng. barcoded  non-barcoded neurons \n\n\n# fig4. mapping projections of the auditory cortex using barseq\n\n\n\nfig4.  barseq \n\na. barseq  barcodes  hamming distance \nb.  6,391 projection neurons (1309 barseq 5,082 mapseq) \nc. barseq  bulk gfp (barcode)\nd.  projection area  barcode counts \ne. \nf. \ng. \n\n\n# fig5. the laminar organization of projection neurons in the auditory cortex\n\n\n\nfig5. \n\na. \nb. \nc. \nd.  t-sne\ne. \n\n\n# fig6. subtypes of it neurons defined by gene expression in the auditory cortex\n\n\n\nfig6.  it \n\na. 735  scrnaseq\nb.  t-sne\nc.  tasic et al.  metaneighbor \nd. \ne. 4it\n\n\n# fig7. projections across it subtypes defined by gene expression\n\n\n\nfig7.  it \n\na. iti-ctx  4  it \nb. 4  it \nc. 4  it  barcode \nd.  it  4  it \ne. cdh13+/- it  +/-  audc \nf.  4  it ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Epigenomic diversity of cortical projection neurons in the mouse brain",frontmatter:{title:" --- Epigenomic diversity of cortical projection neurons in the mouse brain",date:"2023-12-26T15:32:02.000Z",permalink:"/pages/ccc053/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20211005.(2021.10.06%20epi-retro-seq%20Nature)%20Epigenomic%20diversity%20of%20cortical%20projection%20neurons%20in%20the%20mouse%20brain.html",relativePath:"02./11./02./20211005.(2021.10.06 epi-retro-seq Nature) Epigenomic diversity of cortical projection neurons in the mouse brain.md",key:"v-485075cc",path:"/pages/ccc053/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:513},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:1160},{level:3,title:"Fig. 1 The epigenomic landscape of cortical projection neurons",slug:"fig-1-the-epigenomic-landscape-of-cortical-projection-neurons",normalizedTitle:"fig. 1 the epigenomic landscape of cortical projection neurons",charIndex:1171},{level:3,title:"Fig. 2 Epigenetic differences between IT neurons projecting to different targets",slug:"fig-2-epigenetic-differences-between-it-neurons-projecting-to-different-targets",normalizedTitle:"fig. 2 epigenetic differences between it neurons projecting to different targets",charIndex:3798},{level:3,title:"Fig. 3 Epigenetic diversity of L5 ET neurons",slug:"fig-3-epigenetic-diversity-of-l5-et-neurons",normalizedTitle:"fig. 3 epigenetic diversity of l5 et neurons",charIndex:5606},{level:3,title:"Fig. 4 Epigenetic differences between L5 ET neurons projecting to different targets",slug:"fig-4-epigenetic-differences-between-l5-et-neurons-projecting-to-different-targets",normalizedTitle:"fig. 4 epigenetic differences between l5 et neurons projecting to different targets",charIndex:6738},{level:3,title:"Fig. 5 A L5 ET neuron type that projects to both ET and cortical targets",slug:"fig-5-a-l5-et-neuron-type-that-projects-to-both-et-and-cortical-targets",normalizedTitle:"fig. 5 a l5 et neuron type that projects to both et and cortical targets",charIndex:8008},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:9337}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-021-03223-w" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-021-03223-w<OutboundLink/></a><br>\nCite: Zhang, Z., Zhou, J., Tan, P. et al. Epigenomic diversity of cortical projection neurons in the mouse brain. <em>Nature</em> 598, 167173 (2021).</p>\n',headersStr:"Abstract Figure Fig. 1 The epigenomic landscape of cortical projection neurons Fig. 2 Epigenetic differences between IT neurons projecting to different targets Fig. 3 Epigenetic diversity of L5 ET neurons Fig. 4 Epigenetic differences between L5 ET neurons projecting to different targets Fig. 5 A L5 ET neuron type that projects to both ET and cortical targets Discussion",content:": \nDOI: https://doi.org/10.1038/s41586-021-03223-w\nCite: Zhang, Z., Zhou, J., Tan, P. et al. Epigenomic diversity of cortical projection neurons in the mouse brain. Nature 598, 167173 (2021).\n\n\n\n\n\n\n\nEDWARD CALLAWAY                             JOSEPH ECKER\n                                            \nThe Salk Institute for Biological Studies   The Salk Institute for Biological Studies\ncallaway@salk.edu                           ecker@salk.edu\n\n\n# Abstract\n\nDNA 63  cortico-cortical  cortico-subcortical  11,827  5 L5 ET L5 ET  cortico-cortical  L5 ET  L5 ET  cortico-cortical \n\n\n# Figure\n\n\n# Fig. 1 The epigenomic landscape of cortical projection neurons\n\n\n\nFig. 1 \n\na.  epi-retro-seqsnmC-seqsource regionstargets (Fig. 1a) epi-retro-seq  rAAV2-retro-Cre  INTACT target region Cre GFPsource regionsFANS GFP+ NeuN+ GFP  snmC-seq snmC-seq non-CGCH; in which 'H' denotes A, T or CmCH CG mCG\n\n epi-retro-seq  8  ('source')  10 'target' 26  cortico-cortical (CC) projections  37  cortico-subcortical projections (the primary motor cortex (MOp), primary somatosensory cortex (SSp), anterior cingulate area (ACA), and primary visual cortex (VISp)) (the striatum, thalamus, superior colliculus, ventral tegmental area (VTA) and substantia nigra, pons and medulla)source regions MOp, SSp, ACA, agranular insular cortex (AI), retrosplenial area (RSP), auditory cortex (AUD), posterior parietal cortex (PTLp) and visual cortex (VIS)\n\nb.  11,827  100 kb  mCH  gene body mCH  clusters  subclassesFig.1b 1,431  10,396 \n\ncde.  subclass Fig.1c DNA epi-retro-seq  subclasses  snmC-seq target regions subclass source regionsFig.1d DNA  DNA  subclass 1Fig.1esourcetarget0.5 mCH Fig.1e\n\nf. target regions t-SNE  IT subclassesL2/3L4L5 ITL6 IT and claustrum-cortico-cortical, CC-cortico-striatal L5 ET Fig.1fsource regions L6 Fig.1f\n\ng. source regionstarget regions mCH  AUROC Fig.1g mCH  DNA \n\n\n# Fig. 2 Epigenetic differences between IT neurons projecting to different targets\n\n\n\nFig. 2 IT \n\na.  epi-retro-seq  42.6%  IT Fig.1bcortical areatargets DNA  8  4  26 -cortico-cortical, CC 8 sources 4 targets 32  AUROC Fig.2a MOp versus ACA  AUROC  0.922 SSp versus VISp  ACA versus VISp  AUROC  0.915  0.914 SSp versus ACA  MOp versus VISp  AUROC  0.837  0.831Fig.2aFig.2a\n\nb-i. -cortico-cortical, CCsources AUD IT AUD-SSp AUD  SSp AUD-VISp AUROC  0.974Fig.2b,e AUD-ACA AUROC  0.766Fig.2c,elayersFig.2d AUD IT targets MOp  ACA  AUROC  SSp  ACA  AUROC Fig.2f,gFig.2h,i AUD  VIS-MOp  VIS-ACA  RSP  PTLp Fig.2h AUD  VIS  MOp  ACA \n\nj-k. Fig.2j,k ACA  VISp  SSp  ACA Fig.2j,kSSp  ACA  MOp  PTLp AUD  VISFig.2k\n\n\n# Fig. 3 Epigenetic diversity of L5 ET neurons\n\n\n\nFig. 3 L5 ET \n\na-c. L5 ET 4,17635.3% unbiased snmC-seq2  epi-retro-seq  6.3  L5 ET L5 ET  15  clustersFig.3aclusters  UMAP Fig.3b clusters  L5 ET Fig.3cMOp  SSp  AUDVISACA  PTLp  VISp  AUD PTLp  ACA\n\nd-f.  L5 ET clusters  2,675  CH-DMGsFig.3d 341,748  CG-DMRsFig.3e L5 ET clusters GO  CH-DMGs CG-DMRs  227 bp84.9%  CG-DMRs  5kbgene bodies  mCH  mCG  cluster Fig.3f Rora  clusters 1813  hypo-CH-methylated cluster8 RORA  clusters  CG-DMRs  RORA  cluster 8 \n\n\n# Fig. 4 Epigenetic differences between L5 ET neurons projecting to different targets\n\n\n\nFig. 4 L5 ET \n\n AI  RSP cluster  L5 ET clusters targetssource regions L5 ET  clusters L5 ET clusters\n\na-e.  clusters medullaSSp L5 ET  7  clustersFig.4a SSp-medulla  cluster 0 Fig.4b,c 7  MOp L5 ET  clusters MOp-medulla  cluster  medulla  MOp  SSp  L5 ET  AUROC  0.929  0.864 medulla  L5 ET  L5 ET  MOp(SSp)-medulla L5 ET  ET  1,380293 CH-DMGsFig.4d,e180  CH-DMGs  MOp-medulla  SSp-medulla  MOp  SSp  medulla \n\nf-i.  medulla  L5 ET  ET  mCH profiles AI  L5 ET  5  clustersFig.4f AI-pons  AI-SC  clusters Fig.4g,hgene body mCH  145  CH-DMGs AI-pons  AI-SC  AI-pons AI-thalamus Fig.4iAI-pons  AI-thalamus  AI-thalamus \n\n\n# Fig. 5 A L5 ET neuron type that projects to both ET and cortical targets\n\n\n\nFig. 5  L5 ET  ET \n\na-d.  ACA  RSP  L5 ET clusters  30  VISpFig.5a,b L5 ET cluster  ET  L5  ET  ACA  RSP  clusterFig.5a-d RSP  ACA  cluster  Ubn2 Ubn2  cluster \n\ne-f.  ET  RSP-VISp ET  VISp  AAV-retro-Cre RSPFig.5e ACA  AAV-FLEX-GFP Cre GFP RSP-VISp  ACA-VISp  VISp  RSP  ET  GFP Fig.5f ACA  ET RSP  ACA  L5  ET  L5 ET  L5 ET  \"L5 ET + CC\" \"L5 ET + IT\"  L5 ET \n\ng-h.  ACARSP  L5 ET + CC  ponscholera toxin subunit B (CTB) Alexa Fluor 647 VISpCTB Alexa Fluor 488Fig.5g pons  ACA  RSP  ACA  RSP  VISp  pons  RSP 26.6%Fig.5h ACA 7.0%Fig.5h 13.1%  14.6%\n\n\n# Discussion\n\n IT  ET  L5 ET  IT  L5 ET  ALM  MOp  SSp AI L5 ET  PTLp\n\n- RSP-VISp  ACA-VISp  L5 ET  L5 ET  IT - L5 ET RSP-VISp  ET  L5 ET + CC L5 ET  CC  26  CC  VISp  L5 ET + CC  L5 ET + CC  ET  L5 ET + CC \n\n",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-021-03223-w\ncite: zhang, z., zhou, j., tan, p. et al. epigenomic diversity of cortical projection neurons in the mouse brain. nature 598, 167173 (2021).\n\n\n\n\n\n\n\nedward callaway                             joseph ecker\n                                            \nthe salk institute for biological studies   the salk institute for biological studies\ncallaway@salk.edu                           ecker@salk.edu\n\n\n# abstract\n\ndna 63  cortico-cortical  cortico-subcortical  11,827  5 l5 et l5 et  cortico-cortical  l5 et  l5 et  cortico-cortical \n\n\n# figure\n\n\n# fig. 1 the epigenomic landscape of cortical projection neurons\n\n\n\nfig. 1 \n\na.  epi-retro-seqsnmc-seqsource regionstargets (fig. 1a) epi-retro-seq  raav2-retro-cre  intact target region cre gfpsource regionsfans gfp+ neun+ gfp  snmc-seq snmc-seq non-cgch; in which 'h' denotes a, t or cmch cg mcg\n\n epi-retro-seq  8  ('source')  10 'target' 26  cortico-cortical (cc) projections  37  cortico-subcortical projections (the primary motor cortex (mop), primary somatosensory cortex (ssp), anterior cingulate area (aca), and primary visual cortex (visp)) (the striatum, thalamus, superior colliculus, ventral tegmental area (vta) and substantia nigra, pons and medulla)source regions mop, ssp, aca, agranular insular cortex (ai), retrosplenial area (rsp), auditory cortex (aud), posterior parietal cortex (ptlp) and visual cortex (vis)\n\nb.  11,827  100 kb  mch  gene body mch  clusters  subclassesfig.1b 1,431  10,396 \n\ncde.  subclass fig.1c dna epi-retro-seq  subclasses  snmc-seq target regions subclass source regionsfig.1d dna  dna  subclass 1fig.1esourcetarget0.5 mch fig.1e\n\nf. target regions t-sne  it subclassesl2/3l4l5 itl6 it and claustrum-cortico-cortical, cc-cortico-striatal l5 et fig.1fsource regions l6 fig.1f\n\ng. source regionstarget regions mch  auroc fig.1g mch  dna \n\n\n# fig. 2 epigenetic differences between it neurons projecting to different targets\n\n\n\nfig. 2 it \n\na.  epi-retro-seq  42.6%  it fig.1bcortical areatargets dna  8  4  26 -cortico-cortical, cc 8 sources 4 targets 32  auroc fig.2a mop versus aca  auroc  0.922 ssp versus visp  aca versus visp  auroc  0.915  0.914 ssp versus aca  mop versus visp  auroc  0.837  0.831fig.2afig.2a\n\nb-i. -cortico-cortical, ccsources aud it aud-ssp aud  ssp aud-visp auroc  0.974fig.2b,e aud-aca auroc  0.766fig.2c,elayersfig.2d aud it targets mop  aca  auroc  ssp  aca  auroc fig.2f,gfig.2h,i aud  vis-mop  vis-aca  rsp  ptlp fig.2h aud  vis  mop  aca \n\nj-k. fig.2j,k aca  visp  ssp  aca fig.2j,kssp  aca  mop  ptlp aud  visfig.2k\n\n\n# fig. 3 epigenetic diversity of l5 et neurons\n\n\n\nfig. 3 l5 et \n\na-c. l5 et 4,17635.3% unbiased snmc-seq2  epi-retro-seq  6.3  l5 et l5 et  15  clustersfig.3aclusters  umap fig.3b clusters  l5 et fig.3cmop  ssp  audvisaca  ptlp  visp  aud ptlp  aca\n\nd-f.  l5 et clusters  2,675  ch-dmgsfig.3d 341,748  cg-dmrsfig.3e l5 et clusters go  ch-dmgs cg-dmrs  227 bp84.9%  cg-dmrs  5kbgene bodies  mch  mcg  cluster fig.3f rora  clusters 1813  hypo-ch-methylated cluster8 rora  clusters  cg-dmrs  rora  cluster 8 \n\n\n# fig. 4 epigenetic differences between l5 et neurons projecting to different targets\n\n\n\nfig. 4 l5 et \n\n ai  rsp cluster  l5 et clusters targetssource regions l5 et  clusters l5 et clusters\n\na-e.  clusters medullassp l5 et  7  clustersfig.4a ssp-medulla  cluster 0 fig.4b,c 7  mop l5 et  clusters mop-medulla  cluster  medulla  mop  ssp  l5 et  auroc  0.929  0.864 medulla  l5 et  l5 et  mop(ssp)-medulla l5 et  et  1,380293 ch-dmgsfig.4d,e180  ch-dmgs  mop-medulla  ssp-medulla  mop  ssp  medulla \n\nf-i.  medulla  l5 et  et  mch profiles ai  l5 et  5  clustersfig.4f ai-pons  ai-sc  clusters fig.4g,hgene body mch  145  ch-dmgs ai-pons  ai-sc  ai-pons ai-thalamus fig.4iai-pons  ai-thalamus  ai-thalamus \n\n\n# fig. 5 a l5 et neuron type that projects to both et and cortical targets\n\n\n\nfig. 5  l5 et  et \n\na-d.  aca  rsp  l5 et clusters  30  vispfig.5a,b l5 et cluster  et  l5  et  aca  rsp  clusterfig.5a-d rsp  aca  cluster  ubn2 ubn2  cluster \n\ne-f.  et  rsp-visp et  visp  aav-retro-cre rspfig.5e aca  aav-flex-gfp cre gfp rsp-visp  aca-visp  visp  rsp  et  gfp fig.5f aca  et rsp  aca  l5  et  l5 et  l5 et  \"l5 et + cc\" \"l5 et + it\"  l5 et \n\ng-h.  acarsp  l5 et + cc  ponscholera toxin subunit b (ctb) alexa fluor 647 vispctb alexa fluor 488fig.5g pons  aca  rsp  aca  rsp  visp  pons  rsp 26.6%fig.5h aca 7.0%fig.5h 13.1%  14.6%\n\n\n# discussion\n\n it  et  l5 et  it  l5 et  alm  mop  ssp ai l5 et  ptlp\n\n- rsp-visp  aca-visp  l5 et  l5 et  it - l5 et rsp-visp  et  l5 et + cc l5 et  cc  26  cc  visp  l5 et + cc  l5 et + cc  et  l5 et + cc \n\n",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks",frontmatter:{title:" --- Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks",date:"2023-02-09T16:34:14.000Z",permalink:"/pages/5c79f3/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20201217.(2020.12.17%20%E9%AA%86%E5%88%A9%E7%BE%A4%20Cell)%20Differential%20encoding%20in%20prefrontal%20cortex%20projection%20neuron%20classes%20across%20cognitive%20tasks.html",relativePath:"02./11./02./20201217.(2020.12.17  Cell) Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks.md",key:"v-79262273",path:"/pages/5c79f3/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:529},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:952},{level:3,title:"Figure 1. Transcriptomic map of Rbp4Cre-labeled PFC projection neurons",slug:"figure-1-transcriptomic-map-of-rbp4cre-labeled-pfc-projection-neurons",normalizedTitle:"figure 1. transcriptomic map of rbp4cre-labeled pfc projection neurons",charIndex:964},{level:3,title:"Figure 2. Anatomical locations of PFC transcriptomic types",slug:"figure-2-anatomical-locations-of-pfc-transcriptomic-types",normalizedTitle:"figure 2. anatomical locations of pfc transcriptomic types",charIndex:1715},{level:3,title:"Figure 3. Relationship between projection patterns and transcriptomic types in vmPFC",slug:"figure-3-relationship-between-projection-patterns-and-transcriptomic-types-in-vmpfc",normalizedTitle:"figure 3. relationship between projection patterns and transcriptomic types in vmpfc",charIndex:2020},{level:3,title:"Figure 4. vmPFC is engaged by a two-alternative forced choice task",slug:"figure-4-vmpfc-is-engaged-by-a-two-alternative-forced-choice-task",normalizedTitle:"figure 4. vmpfc is engaged by a two-alternative forced choice task",charIndex:2571},{level:3,title:"Figure 5. Differential enrichment of activity across epochs between cell classes",slug:"figure-5-differential-enrichment-of-activity-across-epochs-between-cell-classes",normalizedTitle:"figure 5. differential enrichment of activity across epochs between cell classes",charIndex:3210},{level:3,title:"Figure 6. Choice direction-specific information differs quantitatively between cell classes",slug:"figure-6-choice-direction-specific-information-differs-quantitatively-between-cell-classes",normalizedTitle:"figure 6. choice direction-specific information differs quantitatively between cell classes",charIndex:3508},{level:3,title:"Figure 7. Two additional cognitive tasks reveal how cell classes differentially encode task signals",slug:"figure-7-two-additional-cognitive-tasks-reveal-how-cell-classes-differentially-encode-task-signals",normalizedTitle:"figure 7. two additional cognitive tasks reveal how cell classes differentially encode task signals",charIndex:3865},{level:2,title:"Supplemental Figures",slug:"supplemental-figures",normalizedTitle:"supplemental figures",charIndex:4489},{level:3,title:"Figure S1. Sequencing quality control, validation of PFC neuron clustering, and other scRNAseq analysis",slug:"figure-s1-sequencing-quality-control-validation-of-pfc-neuron-clustering-and-other-scrnaseq-analysis",normalizedTitle:"figure s1. sequencing quality control, validation of pfc neuron clustering, and other scrnaseq analysis",charIndex:4514},{level:3,title:"Figure S2. Additional HCR-FISH validations and example retrograde labeling",slug:"figure-s2-additional-hcr-fish-validations-and-example-retrograde-labeling",normalizedTitle:"figure s2. additional hcr-fish validations and example retrograde labeling",charIndex:4852},{level:3,title:"Figure S3. Rbp4Cre>tdTomato+ and projection-defined neurons intermingle in tSNE space, and additional histological validation of marker expression in projection-defined neurons",slug:"figure-s3-rbp4cre-tdtomato-and-projection-defined-neurons-intermingle-in-tsne-space-and-additional-histological-validation-of-marker-expression-in-projection-defined-neurons",normalizedTitle:"figure s3. rbp4cre&gt;tdtomato+ and projection-defined neurons intermingle in tsne space, and additional histological validation of marker expression in projection-defined neurons",charIndex:null},{level:3,title:"Figure S4. Optogenetic cannulae placement, GRIN lens targeting, and Ca2+ imaging fields of view for the 2-odor task",slug:"figure-s4-optogenetic-cannulae-placement-grin-lens-targeting-and-ca2-imaging-fields-of-view-for-the-2-odor-task",normalizedTitle:"figure s4. optogenetic cannulae placement, grin lens targeting, and ca2+ imaging fields of view for the 2-odor task",charIndex:6092},{level:3,title:"Figure S5. Task-relevant Ca2+ activity dynamics vary by cell class in vmPFC, and encoding of reward omission",slug:"figure-s5-task-relevant-ca2-activity-dynamics-vary-by-cell-class-in-vmpfc-and-encoding-of-reward-omission",normalizedTitle:"figure s5. task-relevant ca2+ activity dynamics vary by cell class in vmpfc, and encoding of reward omission",charIndex:6261},{level:3,title:"Figure S6. Analysis of side-selectivity and dimensionality in the 2-odor task, and imaging fields of view in the 4-odor cued and uncued tasks",slug:"figure-s6-analysis-of-side-selectivity-and-dimensionality-in-the-2-odor-task-and-imaging-fields-of-view-in-the-4-odor-cued-and-uncued-tasks",normalizedTitle:"figure s6. analysis of side-selectivity and dimensionality in the 2-odor task, and imaging fields of view in the 4-odor cued and uncued tasks",charIndex:6430},{level:3,title:"Figure S7. Neural data is similar between the 2- and 4-odor tasks but differs in the uncued task, and analysis of mixed selectivity",slug:"figure-s7-neural-data-is-similar-between-the-2-and-4-odor-tasks-but-differs-in-the-uncued-task-and-analysis-of-mixed-selectivity",normalizedTitle:"figure s7. neural data is similar between the 2- and 4-odor tasks but differs in the uncued task, and analysis of mixed selectivity",charIndex:6625}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2020.11.046" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2020.11.046<OutboundLink/></a><br>\nCite: Lui JH, Nguyen ND, Grutzner SM, et al. Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks. Cell. 2021;184(2):489-506.e26.</p>\n',headersStr:"Abstract Results Figure 1. Transcriptomic map of Rbp4Cre-labeled PFC projection neurons Figure 2. Anatomical locations of PFC transcriptomic types Figure 3. Relationship between projection patterns and transcriptomic types in vmPFC Figure 4. vmPFC is engaged by a two-alternative forced choice task Figure 5. Differential enrichment of activity across epochs between cell classes Figure 6. Choice direction-specific information differs quantitatively between cell classes Figure 7. Two additional cognitive tasks reveal how cell classes differentially encode task signals Supplemental Figures Figure S1. Sequencing quality control, validation of PFC neuron clustering, and other scRNAseq analysis Figure S2. Additional HCR-FISH validations and example retrograde labeling Figure S3. Rbp4Cre>tdTomato+ and projection-defined neurons intermingle in tSNE space, and additional histological validation of marker expression in projection-defined neurons Figure S4. Optogenetic cannulae placement, GRIN lens targeting, and Ca2+ imaging fields of view for the 2-odor task Figure S5. Task-relevant Ca2+ activity dynamics vary by cell class in vmPFC, and encoding of reward omission Figure S6. Analysis of side-selectivity and dimensionality in the 2-odor task, and imaging fields of view in the 4-odor cued and uncued tasks Figure S7. Neural data is similar between the 2- and 4-odor tasks but differs in the uncued task, and analysis of mixed selectivity",content:": \nDOI: https://doi.org/10.1016/j.cell.2020.11.046\nCite: Lui JH, Nguyen ND, Grutzner SM, et al. Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks. Cell. 2021;184(2):489-506.e26.\n\n\n\n\n\n\n\nJAN LUI               LIQUN LUO()        STEPHEN QUAKE\n                                            \nStanford University   Stanford University   Stanford University\njanlui@stanford.edu   lluo@stanford.edu     steve@quake-lab.org\n\n\n# Abstract\n\nPFCPFC -> PAGtwo-alternative choice tasksPFC -> PAGCa2 +PAGPFC\n\n\n# Results\n\n\n# Figure 1. Transcriptomic map of Rbp4Cre-labeled PFC projection neurons\n\n\n\nFigure 1. Rbp4Cre  PFC \n\nA.  Rbp4Cre  Ai14( Cre+  tdTomato)  34-40 dmPFCvmPFCOFC(FACS) SMART-Seq2  tdTomato+  plate-based scRNAseq\nB.  3139 median ~7000 genes/cell, ~1-2 million reads/cell tSNE  Seurat  clusters\nC. Feature plots (top)  violin plots (bottom) excitatory pyramidal neurons (Vglut1+), upper (Cux1) versus deeper (Fezf2) layers, and subcortically projecting neurons (Ctip2)\nD.  C \nE. Clustree flowchart  0.3  cluster \nF.  Seurat nearest neighbor mapping cluster  Tasic et al.  ALM  VISp \n\n\n# Figure 2. Anatomical locations of PFC transcriptomic types\n\n\n\nFigure 2. PFC \n\nA. vmPFC  HCR-FISHAllen Atlas L2/3120um; L5a230um; L5b410umL6600um\nB.  vmPFC \nC. vmPFC  Npr3  Tshz2  HCR-FISH\nD. vmPFC  OtofCxcr7  Figf  Triple HCR-FISH\n\n\n# Figure 3. Relationship between projection patterns and transcriptomic types in vmPFC\n\n\n\nFigure 3. vmPFC \n\nA.  vmPFC  scRNAseq  vmPFC  tdTomato+ \nB.  vmPFCn = 440 cPFC129 DS93 NAc290 Amyg94 PAG109 Hypo cells Figure 1  7 Syt6+ L6  Nearest neighbor mapping target \nC.  PAG HCR-FISH  PAG (tdTomato) Npr3 Figf\nD. (cPFC-PAG-)(Cd44FigfCxcr7Npr3)\nE.  vmPFC  PAG-tdTomato cPFC CTB-488\n\n\n# Figure 4. vmPFC is engaged by a two-alternative forced choice task\n\n\n\nFigure 4.  vmPFC \n\nA.  2AFC (1)1(2) 4 mL \nB.  (Gad2Cre;Ai32)  ChR2(H134R)  PL \nCD. vmPFC  3 s (C)  1 s (D) 25% \nE.  4 \nF. Rbp4Cre: n = 8, PAG-projecting: n = 8, and cPFC-projecting: n = 7 mice; paired t test\nG. 32\nH. 6G Ca2+ CNMF-E33/\nI. \n\n\n# Figure 5. Differential enrichment of activity across epochs between cell classes\n\n\n\nFigure 5. \n\n(A)  Figure 4E  4  PAG \n(B) \n(C) \n(D) \n\n\n# Figure 6. Choice direction-specific information differs quantitatively between cell classes\n\n\n\nFigure 6. \n\n(A) \n(B)  PC \n(C) (B)200\n(D) \n(E) \n(F) \n\n\n# Figure 7. Two additional cognitive tasks reveal how cell classes differentially encode task signals\n\n\n\nFigure 7. \n\n(A)  4 mL \n(B) \n(C) L-2)L-1 + L-2R-1+R-2+R-/\n(D) \n(E) l-r-/\n(F)  PC \n(G) (F)/\n\n\n# Supplemental Figures\n\n\n# Figure S1. Sequencing quality control, validation of PFC neuron clustering, and other scRNAseq analysis\n\n\n\nFigure S1. PFC  scRNAseq \n\nA.  PFC  genes  reads \nB.  Allen \nC. Rbp4Cre>tdTomato+ PFC 20\nD. Tasic et al., 2018\n\n\n# Figure S2. Additional HCR-FISH validations and example retrograde labeling\n\n\n\nFigure S2.  HCR-FISH \n\nA.  dmPFC  Figure 2BAllen Atlas\nB. vmPFC  OtofCd44Cxcr7  Triple HCR-FISH\nC. vmPFC  Cd44Cxcr7Tshz2  Triple HCR-FISH\nD.  mPFC  OFC  Pld5 \nE.  CAV-Cre  Ai14  PFC  PAG mPFC \nF. CAV-Cre  PFC  PAG Cg, cingulate; PL, prelimbic; IL, infralimbic; MO, medial orbital; VO, ventral orbital; LO, lateral orbital; AI(D/V): agranular insular (dorsal/ventral); DP(dorsal peduncular) cortices\n\n\n# Figure S3. Rbp4Cre>tdTomato+ and projection-defined neurons intermingle in tSNE space, and additional histological validation of marker expression in projection-defined neurons\n\n\n\nFigure S3. Rbp4Cre>tdTomato+  tSNE \n\nA. tSNE  Figure 1n = 3139 Rbp4Cre>tdTomato+  6 n = 1155 vmPFC \nB.  cluster identities  Nearest neighbor mapping Figure 3B \nC.  cPFC tdTomato Cre  HCR-FISH  Cd44Cxcr7Figf\nD.  cPFC tdTomato Cre  HCR-FISH cPFC  vmPFC  Npr3-expressing \n\n\n# Figure S4. Optogenetic cannulae placement, GRIN lens targeting, and Ca2+ imaging fields of view for the 2-odor task\n\n\n\nFigure S4. GRIN 2-Ca2+\n\n\n# Figure S5. Task-relevant Ca2+ activity dynamics vary by cell class in vmPFC, and encoding of reward omission\n\n\n\nFigure S5.  vmPFC  Ca2+ \n\n\n# Figure S6. Analysis of side-selectivity and dimensionality in the 2-odor task, and imaging fields of view in the 4-odor cued and uncued tasks\n\n\n\nFigure S6. 24\n\n\n# Figure S7. Neural data is similar between the 2- and 4-odor tasks but differs in the uncued task, and analysis of mixed selectivity\n\n\n\nFigure S7. 24",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2020.11.046\ncite: lui jh, nguyen nd, grutzner sm, et al. differential encoding in prefrontal cortex projection neuron classes across cognitive tasks. cell. 2021;184(2):489-506.e26.\n\n\n\n\n\n\n\njan lui               liqun luo()        stephen quake\n                                            \nstanford university   stanford university   stanford university\njanlui@stanford.edu   lluo@stanford.edu     steve@quake-lab.org\n\n\n# abstract\n\npfcpfc -> pagtwo-alternative choice taskspfc -> pagca2 +pagpfc\n\n\n# results\n\n\n# figure 1. transcriptomic map of rbp4cre-labeled pfc projection neurons\n\n\n\nfigure 1. rbp4cre  pfc \n\na.  rbp4cre  ai14( cre+  tdtomato)  34-40 dmpfcvmpfcofc(facs) smart-seq2  tdtomato+  plate-based scrnaseq\nb.  3139 median ~7000 genes/cell, ~1-2 million reads/cell tsne  seurat  clusters\nc. feature plots (top)  violin plots (bottom) excitatory pyramidal neurons (vglut1+), upper (cux1) versus deeper (fezf2) layers, and subcortically projecting neurons (ctip2)\nd.  c \ne. clustree flowchart  0.3  cluster \nf.  seurat nearest neighbor mapping cluster  tasic et al.  alm  visp \n\n\n# figure 2. anatomical locations of pfc transcriptomic types\n\n\n\nfigure 2. pfc \n\na. vmpfc  hcr-fishallen atlas l2/3120um; l5a230um; l5b410uml6600um\nb.  vmpfc \nc. vmpfc  npr3  tshz2  hcr-fish\nd. vmpfc  otofcxcr7  figf  triple hcr-fish\n\n\n# figure 3. relationship between projection patterns and transcriptomic types in vmpfc\n\n\n\nfigure 3. vmpfc \n\na.  vmpfc  scrnaseq  vmpfc  tdtomato+ \nb.  vmpfcn = 440 cpfc129 ds93 nac290 amyg94 pag109 hypo cells figure 1  7 syt6+ l6  nearest neighbor mapping target \nc.  pag hcr-fish  pag (tdtomato) npr3 figf\nd. (cpfc-pag-)(cd44figfcxcr7npr3)\ne.  vmpfc  pag-tdtomato cpfc ctb-488\n\n\n# figure 4. vmpfc is engaged by a two-alternative forced choice task\n\n\n\nfigure 4.  vmpfc \n\na.  2afc (1)1(2) 4 ml \nb.  (gad2cre;ai32)  chr2(h134r)  pl \ncd. vmpfc  3 s (c)  1 s (d) 25% \ne.  4 \nf. rbp4cre: n = 8, pag-projecting: n = 8, and cpfc-projecting: n = 7 mice; paired t test\ng. 32\nh. 6g ca2+ cnmf-e33/\ni. \n\n\n# figure 5. differential enrichment of activity across epochs between cell classes\n\n\n\nfigure 5. \n\n(a)  figure 4e  4  pag \n(b) \n(c) \n(d) \n\n\n# figure 6. choice direction-specific information differs quantitatively between cell classes\n\n\n\nfigure 6. \n\n(a) \n(b)  pc \n(c) (b)200\n(d) \n(e) \n(f) \n\n\n# figure 7. two additional cognitive tasks reveal how cell classes differentially encode task signals\n\n\n\nfigure 7. \n\n(a)  4 ml \n(b) \n(c) l-2)l-1 + l-2r-1+r-2+r-/\n(d) \n(e) l-r-/\n(f)  pc \n(g) (f)/\n\n\n# supplemental figures\n\n\n# figure s1. sequencing quality control, validation of pfc neuron clustering, and other scrnaseq analysis\n\n\n\nfigure s1. pfc  scrnaseq \n\na.  pfc  genes  reads \nb.  allen \nc. rbp4cre>tdtomato+ pfc 20\nd. tasic et al., 2018\n\n\n# figure s2. additional hcr-fish validations and example retrograde labeling\n\n\n\nfigure s2.  hcr-fish \n\na.  dmpfc  figure 2ballen atlas\nb. vmpfc  otofcd44cxcr7  triple hcr-fish\nc. vmpfc  cd44cxcr7tshz2  triple hcr-fish\nd.  mpfc  ofc  pld5 \ne.  cav-cre  ai14  pfc  pag mpfc \nf. cav-cre  pfc  pag cg, cingulate; pl, prelimbic; il, infralimbic; mo, medial orbital; vo, ventral orbital; lo, lateral orbital; ai(d/v): agranular insular (dorsal/ventral); dp(dorsal peduncular) cortices\n\n\n# figure s3. rbp4cre>tdtomato+ and projection-defined neurons intermingle in tsne space, and additional histological validation of marker expression in projection-defined neurons\n\n\n\nfigure s3. rbp4cre>tdtomato+  tsne \n\na. tsne  figure 1n = 3139 rbp4cre>tdtomato+  6 n = 1155 vmpfc \nb.  cluster identities  nearest neighbor mapping figure 3b \nc.  cpfc tdtomato cre  hcr-fish  cd44cxcr7figf\nd.  cpfc tdtomato cre  hcr-fish cpfc  vmpfc  npr3-expressing \n\n\n# figure s4. optogenetic cannulae placement, grin lens targeting, and ca2+ imaging fields of view for the 2-odor task\n\n\n\nfigure s4. grin 2-ca2+\n\n\n# figure s5. task-relevant ca2+ activity dynamics vary by cell class in vmpfc, and encoding of reward omission\n\n\n\nfigure s5.  vmpfc  ca2+ \n\n\n# figure s6. analysis of side-selectivity and dimensionality in the 2-odor task, and imaging fields of view in the 4-odor cued and uncued tasks\n\n\n\nfigure s6. 24\n\n\n# figure s7. neural data is similar between the 2- and 4-odor tasks but differs in the uncued task, and analysis of mixed selectivity\n\n\n\nfigure s7. 24",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections",frontmatter:{title:" --- Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections",date:"2022-12-11T19:41:58.000Z",permalink:"/pages/084a0b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20210510.(2021.05.10%20BARseq2%20Neuroscience)%20Integrating%20barcoded%20neuroanatomy%20with%20spatial%20transcriptional%20profiling%20enables%20identification%20of%20gene%20correlates%20of%20projections.html",relativePath:"02./11./02./20210510.(2021.05.10 BARseq2 Neuroscience) Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections.md",key:"v-40e67c36",path:"/pages/084a0b/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:419},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:772},{level:3,title:"Fig1. In situ sequencing of endogenous mRNAs using BARseq2",slug:"fig1-in-situ-sequencing-of-endogenous-mrnas-using-barseq2",normalizedTitle:"fig1. in situ sequencing of endogenous mrnas using barseq2",charIndex:784},{level:3,title:"Fig2. Multiplexed detection of mRNAs using BARseq2",slug:"fig2-multiplexed-detection-of-mrnas-using-barseq2",normalizedTitle:"fig2. multiplexed detection of mrnas using barseq2",charIndex:1465},{level:3,title:"Fig3. Cadherin expression across transcriptomic neuronal types in motor cortex",slug:"fig3-cadherin-expression-across-transcriptomic-neuronal-types-in-motor-cortex",normalizedTitle:"fig3. cadherin expression across transcriptomic neuronal types in motor cortex",charIndex:2376},{level:3,title:"Fig4. Correlating gene expression to projections using BARseq2",slug:"fig4-correlating-gene-expression-to-projections-using-barseq2",normalizedTitle:"fig4. correlating gene expression to projections using barseq2",charIndex:2804},{level:3,title:"Fig5. Differential cadherin expression across major classes and cortical areas",slug:"fig5-differential-cadherin-expression-across-major-classes-and-cortical-areas",normalizedTitle:"fig5. differential cadherin expression across major classes and cortical areas",charIndex:3340},{level:3,title:"Fig6. Cadherins correlate with diverse projections of IT neurons",slug:"fig6-cadherins-correlate-with-diverse-projections-of-it-neurons",normalizedTitle:"fig6. cadherins correlate with diverse projections of it neurons",charIndex:3915},{level:3,title:"Fig7. Gene coexpression modules correlate with diverse projections of IT neurons",slug:"fig7-gene-coexpression-modules-correlate-with-diverse-projections-of-it-neurons",normalizedTitle:"fig7. gene coexpression modules correlate with diverse projections of it neurons",charIndex:4574}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-021-00842-4" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-021-00842-4<OutboundLink/></a><br>\nCite: Sun, YC., Chen, X., Fischer, S. et al. Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections. Nat Neurosci 24, 873885 (2021).</p>\n',headersStr:"Abstract Results Fig1. In situ sequencing of endogenous mRNAs using BARseq2 Fig2. Multiplexed detection of mRNAs using BARseq2 Fig3. Cadherin expression across transcriptomic neuronal types in motor cortex Fig4. Correlating gene expression to projections using BARseq2 Fig5. Differential cadherin expression across major classes and cortical areas Fig6. Cadherins correlate with diverse projections of IT neurons Fig7. Gene coexpression modules correlate with diverse projections of IT neurons",content:": \nDOI: https://doi.org/10.1038/s41593-021-00842-4\nCite: Sun, YC., Chen, X., Fischer, S. et al. Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections. Nat Neurosci 24, 873885 (2021).\n\n\n\n\n\n\n\nANTHONY ZADOR\n\nCold Spring Harbor Laboratory\nzador@cshl.edu\n\n\n# Abstract\n\nBARseq229933motor cortex()auditory cortex()31641349BARseq2\n\n\n# Results\n\n\n# Fig1. In situ sequencing of endogenous mRNAs using BARseq2\n\n\n\nFig1. BARseq2mRNAs\n\na. ABABA\nbc. BARseq2(b)BARseq2RNARNA(c)\nd. Eng(BARseq2)BARseq2 e. BARseq22(n=2) f. RNAscopeeBARseq210 m\n\n\n# Fig2. Multiplexed detection of mRNAs using BARseq2\n\n\n\nFig2.  BARseq2  mRNAs\n\na. rolonies100m\nb. a100m\nc. a10m\nd. \ne. BARseq2FISHerror barss.dn=2 slices for BARseq2 and n=3 slices for FISH\nf. BARseq2Allen100-mbin95%n=2 slices for BARseq2 and n=1 slice for ABA ISH\ng. BARseq2scRNA-seqBARseq2\nh. BARseq2scRNA-seqBARseq2scRNA-seq\ni. BARseq2scRNA-seqscRNA-seq\nj. Slc17a7Gad1Slc17a7Gad1\nk. scRNA-seqSlc17a7Gad1MethodsBARseq2\n\n\n# Fig3. Cadherin expression across transcriptomic neuronal types in motor cortex\n\n\n\nFig3. \n\na. roloniesmRNA100m\nb. acall\nc. BARseq2NPnear-projecting\nd. BARseq2 two-tailed rank-sum test\n\n\n# Fig4. Correlating gene expression to projections using BARseq2\n\n\n\nFig4.  BARseq2 \n\na. roloniesrolonygene rolony20m\nb. aaSupplementary Table 2 BC, barcode\nc. A1M1log\nde. (d)(e)(exc)(inh)\n\n\n# Fig5. Differential cadherin expression across major classes and cortical areas\n\n\n\nFig5. \n\na. raw countsycountsxbinsbar lengths)(FDR<0.05\nbc. (b)(c)yxFDR<0.05\nd. yxFDR<0.05panelstwo-tailed rank-sum testsP\n\n\n# Fig6. Cadherins correlate with diverse projections of IT neurons\n\n\n\nFig6.  IT \n\na. A1M1ITOFCMRstrSSCstrAmygVisIpViSCAudC\nb. IT\nc. b\nd. A1M1Pcdh19CSTR-IORBMOMOpSSpSSsTEa\ne. yx(FDR<0.1two-tailed rank-sum tests)\n\n\n# Fig7. Gene coexpression modules correlate with diverse projections of IT neurons\n\n\n\nFig7.  IT \n\na. ITscRNA-seq\nb. AUROC area under the receiver-operator characteristic curve\nc. binned\nd. single-cell SmartSeqIT(AUROCb)",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-021-00842-4\ncite: sun, yc., chen, x., fischer, s. et al. integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections. nat neurosci 24, 873885 (2021).\n\n\n\n\n\n\n\nanthony zador\n\ncold spring harbor laboratory\nzador@cshl.edu\n\n\n# abstract\n\nbarseq229933motor cortex()auditory cortex()31641349barseq2\n\n\n# results\n\n\n# fig1. in situ sequencing of endogenous mrnas using barseq2\n\n\n\nfig1. barseq2mrnas\n\na. ababa\nbc. barseq2(b)barseq2rnarna(c)\nd. eng(barseq2)barseq2 e. barseq22(n=2) f. rnascopeebarseq210 m\n\n\n# fig2. multiplexed detection of mrnas using barseq2\n\n\n\nfig2.  barseq2  mrnas\n\na. rolonies100m\nb. a100m\nc. a10m\nd. \ne. barseq2fisherror barss.dn=2 slices for barseq2 and n=3 slices for fish\nf. barseq2allen100-mbin95%n=2 slices for barseq2 and n=1 slice for aba ish\ng. barseq2scrna-seqbarseq2\nh. barseq2scrna-seqbarseq2scrna-seq\ni. barseq2scrna-seqscrna-seq\nj. slc17a7gad1slc17a7gad1\nk. scrna-seqslc17a7gad1methodsbarseq2\n\n\n# fig3. cadherin expression across transcriptomic neuronal types in motor cortex\n\n\n\nfig3. \n\na. roloniesmrna100m\nb. acall\nc. barseq2npnear-projecting\nd. barseq2 two-tailed rank-sum test\n\n\n# fig4. correlating gene expression to projections using barseq2\n\n\n\nfig4.  barseq2 \n\na. roloniesrolonygene rolony20m\nb. aasupplementary table 2 bc, barcode\nc. a1m1log\nde. (d)(e)(exc)(inh)\n\n\n# fig5. differential cadherin expression across major classes and cortical areas\n\n\n\nfig5. \n\na. raw countsycountsxbinsbar lengths)(fdr<0.05\nbc. (b)(c)yxfdr<0.05\nd. yxfdr<0.05panelstwo-tailed rank-sum testsp\n\n\n# fig6. cadherins correlate with diverse projections of it neurons\n\n\n\nfig6.  it \n\na. a1m1itofcmrstrsscstramygvisipviscaudc\nb. it\nc. b\nd. a1m1pcdh19cstr-iorbmomopsspssstea\ne. yx(fdr<0.1two-tailed rank-sum tests)\n\n\n# fig7. gene coexpression modules correlate with diverse projections of it neurons\n\n\n\nfig7.  it \n\na. itscrna-seq\nb. auroc area under the receiver-operator characteristic curve\nc. binned\nd. single-cell smartseqit(aurocb)",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Virally Encoded Connectivity Transgenic Overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration",frontmatter:{title:" --- Virally Encoded Connectivity Transgenic Overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration",date:"2022-12-11T20:13:45.000Z",permalink:"/pages/40cb7b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20211221.(2021.12.21%20VECTORseq%20Cell%20Reports)%20Virally%20encoded%20connectivity%20transgenic%20overlay%20RNA%20sequencing%20(VECTORseq)%20defines%20projection%20neuron%20involved%20in%20sensorimotor%20integration.html",relativePath:"02./11./02./20211221.(2021.12.21 VECTORseq Cell Reports) Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neuron involved in sensorimotor integration.md",key:"v-5c36d2c2",path:"/pages/40cb7b/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:439},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:874},{level:3,title:"Fig1. Schematic of VECTORseq",slug:"fig1-schematic-of-vectorseq",normalizedTitle:"fig1. schematic of vectorseq",charIndex:886},{level:3,title:"Fig2. VECTORseq of V1 projection neurons",slug:"fig2-vectorseq-of-v1-projection-neurons",normalizedTitle:"fig2. vectorseq of v1 projection neurons",charIndex:1184},{level:3,title:"Fig3. VECTORseq of SC",slug:"fig3-vectorseq-of-sc",normalizedTitle:"fig3. vectorseq of sc",charIndex:1555},{level:3,title:"Fig4. Analysis of candidate marker gene expression in different SC projection populations",slug:"fig4-analysis-of-candidate-marker-gene-expression-in-different-sc-projection-populations",normalizedTitle:"fig4. analysis of candidate marker gene expression in different sc projection populations",charIndex:1969},{level:3,title:"Fig5. VECTORseq of ventral midbrain inhibitory neurons",slug:"fig5-vectorseq-of-ventral-midbrain-inhibitory-neurons",normalizedTitle:"fig5. vectorseq of ventral midbrain inhibitory neurons",charIndex:2393},{level:3,title:"Fig6. Analysis of candidate marker gene expression in ZI  VM population",slug:"fig6-analysis-of-candidate-marker-gene-expression-in-zi--vm-population",normalizedTitle:"fig6. analysis of candidate marker gene expression in zi  vm population",charIndex:2685}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.celrep.2021.110131" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.celrep.2021.110131<OutboundLink/></a><br>\nCite: Cheung V, Chung P, Bjorni M, Shvareva VA, Lopez YC, Feinberg EH. Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration. <em>Cell Rep</em>. 2021 Dec 21;37(12):110131.</p>\n',headersStr:"Abstract Results Fig1. Schematic of VECTORseq Fig2. VECTORseq of V1 projection neurons Fig3. VECTORseq of SC Fig4. Analysis of candidate marker gene expression in different SC projection populations Fig5. VECTORseq of ventral midbrain inhibitory neurons Fig6. Analysis of candidate marker gene expression in ZI  VM population",content:": \nDOI: https://doi.org/10.1016/j.celrep.2021.110131\nCite: Cheung V, Chung P, Bjorni M, Shvareva VA, Lopez YC, Feinberg EH. Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration. Cell Rep. 2021 Dec 21;37(12):110131.\n\n\n\n\n\n\n\nEVAN FEINBERG\n\nUniversity of California San Francisco\nevan.feinberg@ucsf.edu\n\n\n# Abstract\n\nRNAmultiplexedRNA(Virally Encoded Connectivity Transgenic Overlay RNA sequencingVECTORseq)VECTORseqe.g.mRNAVECTORseq \n\n\n# Results\n\n\n# Fig1. Schematic of VECTORseq\n\n\n\nFig1. VECTORseq \n\n RNA  RNA RNA \n\n\n# Fig2. VECTORseq of V1 projection neurons\n\n\n\nFig2. V1  VECTORseq\n\nA.  left V1SCStrV1SCSCAAV1-hSyn-TurboRFP V1  AAV1-CAG-FLEX-tdTomato  AAV1-EF1-fDIO-EYFP 3V1\nB. V1UMAP\nC. \n\n\n# Fig3. VECTORseq of SC\n\n\n\nFig3. (SC) VECTORseq\n\nA.  left LPleft CnFright PPRF3left SCNeuN/Rbfox3\nB. SCUMAP\nC. (Flt1, endothelia; Mog, oligodendrocytes; C1qa, microglia; Aqp4, astrocytes, Slc17a6, excitatory neurons; Gad2, inhibitory neurons)\nD. SC excitatory(Slc17a6+)UMAP\nE. SC\n\n\n# Fig4. Analysis of candidate marker gene expression in different SC projection populations\n\n\n\nFig4.  SC \n\nA. AAVrg-Cre right PPRF3 left SCRNAscopeCreCre+Pitx2Cre+Pitx2+\nBC. (A) Pmfbp1  SC  PPRF (B) lp (C)\nD. (A)-(C) Gda  lp \nEF. (A)-(D) Rxfp2  SC  lp (E) PPRF (F)\n\n\n# Fig5. VECTORseq of ventral midbrain inhibitory neurons\n\n\n\nFig5.  VECTORseq\n\nA. MLRSC\nB. UMAP\nC. Gad1+/Gad2+UMAP\nD. UMAP\nE. \n\n\n# Fig6. Analysis of candidate marker gene expression in ZI  VM population\n\n\n\nFig6. ZI  VM \n\nAAVrg-Cre  ZI  RNAscope  Cre  Cre+  Pax6  Cre+  Pax6+ ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.celrep.2021.110131\ncite: cheung v, chung p, bjorni m, shvareva va, lopez yc, feinberg eh. virally encoded connectivity transgenic overlay rna sequencing (vectorseq) defines projection neurons involved in sensorimotor integration. cell rep. 2021 dec 21;37(12):110131.\n\n\n\n\n\n\n\nevan feinberg\n\nuniversity of california san francisco\nevan.feinberg@ucsf.edu\n\n\n# abstract\n\nrnamultiplexedrna(virally encoded connectivity transgenic overlay rna sequencingvectorseq)vectorseqe.g.mrnavectorseq \n\n\n# results\n\n\n# fig1. schematic of vectorseq\n\n\n\nfig1. vectorseq \n\n rna  rna rna \n\n\n# fig2. vectorseq of v1 projection neurons\n\n\n\nfig2. v1  vectorseq\n\na.  left v1scstrv1scscaav1-hsyn-turborfp v1  aav1-cag-flex-tdtomato  aav1-ef1-fdio-eyfp 3v1\nb. v1umap\nc. \n\n\n# fig3. vectorseq of sc\n\n\n\nfig3. (sc) vectorseq\n\na.  left lpleft cnfright pprf3left scneun/rbfox3\nb. scumap\nc. (flt1, endothelia; mog, oligodendrocytes; c1qa, microglia; aqp4, astrocytes, slc17a6, excitatory neurons; gad2, inhibitory neurons)\nd. sc excitatory(slc17a6+)umap\ne. sc\n\n\n# fig4. analysis of candidate marker gene expression in different sc projection populations\n\n\n\nfig4.  sc \n\na. aavrg-cre right pprf3 left scrnascopecrecre+pitx2cre+pitx2+\nbc. (a) pmfbp1  sc  pprf (b) lp (c)\nd. (a)-(c) gda  lp \nef. (a)-(d) rxfp2  sc  lp (e) pprf (f)\n\n\n# fig5. vectorseq of ventral midbrain inhibitory neurons\n\n\n\nfig5.  vectorseq\n\na. mlrsc\nb. umap\nc. gad1+/gad2+umap\nd. umap\ne. \n\n\n# fig6. analysis of candidate marker gene expression in zi  vm population\n\n\n\nfig6. zi  vm \n\naavrg-cre  zi  rnascope  cre  cre+  pax6  cre+  pax6+ ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Cellular anatomy of the mouse primary motor cortex",frontmatter:{title:" --- Cellular anatomy of the mouse primary motor cortex",date:"2023-06-20T20:42:22.000Z",permalink:"/pages/d5ec9f/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20211006.(2021.10.06%20MOp-ul%20Nature)%20Cellular%20anatomy%20of%20the%20mouse%20primary%20motor%20cortex.html",relativePath:"02./11./02./20211006.(2021.10.06 MOp-ul Nature) Cellular anatomy of the mouse primary motor cortex.md",key:"v-44435d26",path:"/pages/d5ec9f/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:749},{level:2,title:"Introduction",slug:"introduction",normalizedTitle:"introduction",charIndex:1154},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:2093},{level:3,title:"MOp-ul borders and cell types",slug:"mop-ul-borders-and-cell-types",normalizedTitle:"mop-ul borders and cell types",charIndex:2277},{level:3,title:"Laminar organization of neuron types",slug:"laminar-organization-of-neuron-types",normalizedTitle:"laminar organization of neuron types",charIndex:4376},{level:3,title:"Outputs of MOp-ul",slug:"outputs-of-mop-ul",normalizedTitle:"outputs of mop-ul",charIndex:5852},{level:3,title:"Inputs to MOp-ul",slug:"inputs-to-mop-ul",normalizedTitle:"inputs to mop-ul",charIndex:7345},{level:3,title:"BARseq projection mapping",slug:"barseq-projection-mapping",normalizedTitle:"barseq projection mapping",charIndex:8788},{level:3,title:"Single-neuron projection patterns",slug:"single-neuron-projection-patterns",normalizedTitle:"single-neuron projection patterns",charIndex:10285},{level:3,title:"Diverse PN axon projection motifs",slug:"diverse-pn-axon-projection-motifs",normalizedTitle:"diverse pn axon projection motifs",charIndex:12623},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:13463},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:14386},{level:3,title:"Fig. 1 Delineation of the MOp-ul region and its cell-type organization",slug:"fig-1-delineation-of-the-mop-ul-region-and-its-cell-type-organization",normalizedTitle:"fig. 1 delineation of the mop-ul region and its cell-type organization",charIndex:14397},{level:3,title:"Fig. 2 Brain-wide MOp-ul projection patterns by layer and class",slug:"fig-2-brain-wide-mop-ul-projection-patterns-by-layer-and-class",normalizedTitle:"fig. 2 brain-wide mop-ul projection patterns by layer and class",charIndex:14943},{level:3,title:"Fig. 3 Brain-wide inputs to MOp-ul by layer and class",slug:"fig-3-brain-wide-inputs-to-mop-ul-by-layer-and-class",normalizedTitle:"fig. 3 brain-wide inputs to mop-ul by layer and class",charIndex:15290},{level:3,title:"Fig. 4 Projection mapping with single-cell resolution using BARseq",slug:"fig-4-projection-mapping-with-single-cell-resolution-using-barseq",normalizedTitle:"fig. 4 projection mapping with single-cell resolution using barseq",charIndex:15504},{level:3,title:"Fig. 5 Full morphological reconstructions reveal diverse single-cell projection motifs",slug:"fig-5-full-morphological-reconstructions-reveal-diverse-single-cell-projection-motifs",normalizedTitle:"fig. 5 full morphological reconstructions reveal diverse single-cell projection motifs",charIndex:16023},{level:3,title:"Fig. 6 Cell type wiring diagram of the MOp-ul",slug:"fig-6-cell-type-wiring-diagram-of-the-mop-ul",normalizedTitle:"fig. 6 cell type wiring diagram of the mop-ul",charIndex:16495}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-021-03970-w" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-021-03970-w<OutboundLink/></a><br>\nCite: Muoz-Castaeda, R., Zingg, B., Matho, K.S. et al. Cellular anatomy of the mouse primary motor cortex. Nature 598, 159166 (2021).</p>\n',headersStr:"Abstract Introduction Results MOp-ul borders and cell types Laminar organization of neuron types Outputs of MOp-ul Inputs to MOp-ul BARseq projection mapping Single-neuron projection patterns Diverse PN axon projection motifs Discussion Figure Fig. 1 Delineation of the MOp-ul region and its cell-type organization Fig. 2 Brain-wide MOp-ul projection patterns by layer and class Fig. 3 Brain-wide inputs to MOp-ul by layer and class Fig. 4 Projection mapping with single-cell resolution using BARseq Fig. 5 Full morphological reconstructions reveal diverse single-cell projection motifs Fig. 6 Cell type wiring diagram of the MOp-ul",content:': \nDOI: https://doi.org/10.1038/s41586-021-03970-w\nCite: Muoz-Castaeda, R., Zingg, B., Matho, K.S. et al. Cellular anatomy of the mouse primary motor cortex. Nature 598, 159166 (2021).\n\n\n\n\n\n\n\nHONG WEI DONG ()                            PAVEL OSTEN                     Z. JOSH HUANG                   GIORGIO A. ASCOLI\n                                                                                                               \nUniversity of California, Los Angeles (UCLA)   Cold Spring Harbor Laboratory   Cold Spring Harbor Laboratory   George Mason University\nHongWeiD@mednet.ucla.edu                       osten@cshl.edu                  josh.huang@duke.edu             ascoli@gmu.edu\n\n\n# Abstract\n\nBrain Initiative Cell Census NetworkBICCNMOp-ul 3D  MOp-ul MOp-ul  24 inputoutput wiring diagram\n\n\n# Introduction\n\ngranularitymacroscale mesoscale microscale nanoscale MRI  macroscale connectomes\n\n scRNAeq CCF\n\nBICCN  MOp-ul  BARseq  CCF  MOp-ul \n\n\n# Results\n\n Web  (Extended Data Fig. 1, Methods) Supplementary Table 1  Supplementary Table 2 \n\n\n# MOp-ul borders and cell types\n\nMOp-ul \n\nprimary motor cortex, MOp-- 2D  3D  MOp-ul  3D  (Fig. 1a, Supplementary Video 1; datasets can be viewed at https://viz.neurodata.io/?json_url=https://json.neurodata.io/v1?NGStateID=LwZ24nSZk1JTHw)\n\nMOp-ul primary somatosensory, SSp Nissl- NeuroTrace-  MOp  5 L5 SSp  L5a  L5b  (Fig. 1b, Extended Data Figs. 2a, b; see also the Allen Reference Atlas (ARA) and http://brainmaps.org) MOp agranular cortex L4MOpMOs SSp  (Fig. 1b, Extended Data Fig. 2b; see also algorithmic analysis of MOpSSp border, revealing individual variations between animals in Extended Data Fig. 2c, d, Supplementary Information)\n\nVGluT1Slc17a7 MOp-SSp  L4  L5 Fig. 1a, b, Supplementary Video 2 VGluT3  MOp-ulMOs (Fig. 1a, b)AAV SSp SSp-ul MOp-ulORBvlRSPd MOsExtended Data Fig. 3a cervical (to delineate upper limb)  lumbar (to delineate lower limb)  AAVretro  (Fig. 1b, Extended Data Figs. 3b, c, 4, Supplementary Video 1)MOp L5 projecting to the intermediate and ventral horn SSp L4 projecting to the dorsal hornFig. 1b, Extended Data Figs. 4, 5i MOp-ul  AAV-RFPPhaseolus vulgaris leucoagglutinin (PHAL)  AAV-GFP  MOsMOp-ul  SSp Fig. 1b, Extended Data Fig. 5\n\n Neuroglancer  CCFv3 average template  MOp-ul  3D  3D Fig. 1c, Extended Data Fig. 2e, Supplementary Video 2 ARA  Franklin-Paxinos  Allen CCFv3 Extended Data Fig. 2f, Supplementary Information\n\n MOp-ul VGluT1+GABA -GAD2+ GABA  Cre Extended Data Fig. 3d\n\n\n# Laminar organization of neuron types\n\n\n\n 6  8  layers PNs1intratelencephalicIT L2-L6 2pyramidal tractPTextratelencephalicET L5 3-corticothalamicCT L6  MOp-ul  PNs fluorogold and cholera toxin B subunit (CTB) 15   Mop --caudoputamenCP targetsFig. 1b, Extended Data Figs. 3b, c, 7 MOp-ul  PNs Fig. 1d, Extended Data Fig. 3b, c, 7 16  IT 7  ET  3  CT laminar organization26  PN  11 1, 2, 3, 4, 5a, 5b-superficial, 5b-middle, 5b-deep, 6a-superficial, 6a-deep and 6bFig. 1d connectivity  PN Fig. 1d Nissl Extended Data Fig. 6\n\n IT 1 L2  L5 temporal association areaTEa2 L4  MOs  SSp 3 L6b Extended Data Fig. 7 PN  Cre  AAV Extended Data Fig. 8aExtended Data Fig. 8b, cL5a  L5b  IT L5a  IT  CP L5b  IT  CP L5b  IT  CP  L5a  IT  CP \n\nin situ hybridization MOp Extended Data Fig. 9\n\n\n# Outputs of MOp-ul\n\nMOp-ul \n\n CCF  MOp-ul  PHAL  MOp-ul MOp-ul  110  60 Extended Data Figs. 5, 10, Supplementary Information L2/3L4L5 ITL5 ET  L6 CT  Cre Fig. 2a, bExtended Data Fig. 11\n\n CCFv3  314 Fig. 2c, Source Data Fig. 2MOp-ul isocortexstriatumthalamus 44.9%29.0%  8.1%midbrainmedullaponsExtended Data Fig. 13d Cre Fig. 2c, Extended Data Figs. 12, 13a, 14Sepw1-L2/3Cux2-L2/3Nr5a1-L4Scnn1a-L4/5Plxnd1-L2/3 + L5  Tlx3-L5 isocortex CP IT Sim1-L5  Fezf2-L5/6 subcortical ET  Ntsr1-L6  Tle4-L6 thalamic nuclei CT  Cre  Rbp4-L5  IT  ETFig. 2a, c, Extended Data Fig. 12\n\nFig. 2cCluster 1  L5 ET  PHALAAV-GFP  Rbp4-L5 IT/ETCluster 2  L6 CT  Ntsr1-L6  Tle4-L6Cluster 3  4  IT PN Cluster 3  Cux2-L2/3Tlx3-L5Plxnd1-L2/3 + L5Cluster 4  Sepw1-L2/3Nr5a1-L4Scnn1a-L4 cluster Tle4-L6  Ntsr1-L6Extended Data Fig. 13d, left MOp-ul Fig. 2d, Extended Data Fig. 14\n\n\n# Inputs to MOp-ul\n\nMOp-ul \n\nFig. 3a, b MOp1 CTBExtended Data Figs. 7, 10, 15(2)  Cre  Cre PvalbSst  Vip(3)  (TRIO)  AAVretro-Cre Extended Data Fig. 16aCTB  MOp-ul MOpSSpSSsMOsVALPFPO VMExtended Data Figs. 10, 15 Cre Extended Data Figs. 13b, 15, 16aGPeGPiCEAcCSDR MOp-ul CTB Extended Data Fig. 15\n\nMOp-ul  CCFv3 Fig. 3c, Source Data Fig. 3isocortexthalamus 90.1%7.7%Extended Data Fig. 13f, pie chartFig. 3ccluster  CTB  Cre  cluster  TRIO  Cre  (Foxp2-L6)average n=91 versus 30 input regions, P<0.0001, two-tailed t-test\n\n Cre-  target-  MOp-ul Extended Data Fig. 16b MOp MOp Extended Data Fig. 10 MOp-ul Fig. 3d\n\n (L2/3/4)  (L5) MOp Extended Data Fig. 17aeL5 \n\n\n# BARseq projection mapping\n\nBARseq \n\nCre BARseq  RNA  BARseq10,299  MOp  39 Fig. 4aExtended Data Fig. 18af, Supplementary InformationSupplementary Information, Extended Data Fig. 18gk\n\n CTL5 ET  IT with (IT Str+) or without (IT Str) projections to the striatumFig. 4b, Extended Data Fig. 19ac, Supplementary InformationMethods 18 Fig. 4c, Extended Data Fig. 19dk, Supplementary Information BARseq PNs  11 Fig. 4d11  BARseq PN Fig. 4d\n\n IT  MOsMOs-contra L3  L5b-d  IT 72-93 MOsMOs-ipsi MOs-contra  IT  32-50  MOs-ipsiFig. 4e, Extended Data Fig. 19lExtended Data Fig. 19lMOs-ipsi  L2  SSsSSs-ipsi L2-4  MOs-ipsi  SSs-ipsi  MOp-contra Fig. 4fMOp-contra SS-contraFig. 4fL2/3 IT L5aL5b  L6a IT Fig. 4gFig. 1d\n\n\n# Single-neuron projection patterns\n\n\n\nfMOST CCFv3  140  Janelia MouseLight  121  fMOST  42  12  303  MOp  113  MOp-ul Fig. 5a, Extended Data Fig. 20a\n\nFig. 5b, Source Data Fig. 5Extended Data Fig. 20b, c\n\n Fig. 2  13 C1C13; Fig. 5b, c Cre /CTET  ITC1  IT  ET C2  CT Cre  L6 CT C2.1VPM Source Data Fig. 5 \n\nC3-C5  MOp L5 ET Fig. 5b, c, eC3  C4 C3C4 C3 3.2PSVPBC3.2 C4  ET MRNSCAPNPAGPGTRNPRNrVALVMPOPF L5b Fig. 1dC4 C4.2 RT\n\nIT  Cre  C6-C13 IT 1C7C10  C11 L2/3 C7  C13  L42C8  IT one-way ANOVA and Tukeys post hoc test, P<0.00013two-way repeated measures ANOVA, P<0.0001 interaction effect of cluster  target area C7C8C12 C13Tukeys post hoc test, P<0.05C9 AId C6C9  C12 Tukeys post hoc test, P<0.05C11 ACAv C6C9C11 C13Tukeys post hoc test, P<0.05C12 SSp-ul  SSs\n\nC11  C13  IT  C8-C10  C12  CA1 CPTukeys post hoc test, P<0.02 BARseq  IT Str  IT Str+ C8  L5  IT CEAGPeC7C11  C13  L2/3  L4  BARseq  IT ""\n\n PHAL  MOp  15 C1C5  C6 0.5%C232%C440%C812%C97.7%C112.9%C124.9%C3C7C9C131%40%ET32%CT28%IT\n\n\n# Diverse PN axon projection motifs\n\n PN \n\nFig. 5c, Extended Data Figs. 20d ETC3C5: R=0.52, 0.51 and 0.56, respectively IT C6C12: range 0.540.61 and C13: R=0.66CT C2average Spearman R=0.66 ET  IT \n\n CT Fig. 5d VM VAL  POMDlPCNCL PF ET Fig 5e1-2-3-4-IT Str+ Fig. 5f IT Str+  L2/3  L48 out of 9 cells; Fig. 5f, leftL5  IT Str+ n=3; Fig. 5f, rightIT Str- Fig. 5g IT Str- \n\n\n# Discussion\n\n 3D  24 - 300  1 \n\nFig. 6a; Extended Data Fig. 21-Fig. 6b\n\n L2  L5  TEa - ECT - PERI  L6 CT VM  CT  L6a  L6bFig. 1d L6  L6 CT  PN \n\nMOp \n\n\n# Figure\n\n\n# Fig. 1 Delineation of the MOp-ul region and its cell-type organization\n\n\n\nFig. 1  MOp-ul \n\na.  CCF average template  Neuroglancer  MOp-ul \nb.  Nissl  MOp-ul left AAVretro  Cre middle MOp-ul  SSp-ul  MOs right\nc. MOp-ul  CCF  3D \nd. Leftlaminar positionsITPT  CTprojection targetsRight (top) Cre  MOp-ul Right (bottom) 25m  MOp-ul \n\n\n# Fig. 2 Brain-wide MOp-ul projection patterns by layer and class\n\n\n\nFig. 2  Mop-ul \n\na.  MOp  Cre \nb.  CCFv3 \nc.  CCFv3  MOp  314  314 \nd.  MOp \n\n\n# Fig. 3 Brain-wide inputs to MOp-ul by layer and class\n\n\n\nFig. 3  Mop-ul \n\na. \nb. \nc. MOp-ul  CCFv3 \nd.  MOp \n\n\n# Fig. 4 Projection mapping with single-cell resolution using BARseq\n\n\n\nFig. 4  BARseq \n\na.  BARseq10,299  MOp  39 \nb.  CTL5 ET  IT with (IT Str+) or without (IT Str) projections to the striatum\nc.  18 \nd.  sublayer  subgroup subgroup\ne.  L3  L5b-d  MOs  IT  MOs \nf.  sublayer  X  Y\ng.  IT  IT \n\n\n# Fig. 5 Full morphological reconstructions reveal diverse single-cell projection motifs\n\n\n\nFig. 5 \n\na.  CCFv3  (n = 303)Fig. 1  MOp-ul \nb.  Fig. 2  13 C1C13\nc.  13  CCFv3 3D \nd.  L6 CT VAL/PO  VM dominant \ne.  L5 ET ------ -- \nfg. IT  (+) (g)  () (h)  (STR) \n\n\n# Fig. 6 Cell type wiring diagram of the MOp-ul\n\n\n\nFig. 6 MOp-ul \n\na.  MOp-ul left, black MOp-ul ETCT  IT Cre  S  MOp-ul layersrightleft\nb. MOp-ul  MOp-ul IT L2-L6b  6  IT  CT  MOp-ul  ET 1-cortico-spinal2-cortico-medullarMDRN3-cortico-tectal4-cortico-pontine ET  GPeZISTNRN  IO',normalizedContent:': \ndoi: https://doi.org/10.1038/s41586-021-03970-w\ncite: munoz-castaneda, r., zingg, b., matho, k.s. et al. cellular anatomy of the mouse primary motor cortex. nature 598, 159166 (2021).\n\n\n\n\n\n\n\nhong wei dong ()                            pavel osten                     z. josh huang                   giorgio a. ascoli\n                                                                                                               \nuniversity of california, los angeles (ucla)   cold spring harbor laboratory   cold spring harbor laboratory   george mason university\nhongweid@mednet.ucla.edu                       osten@cshl.edu                  josh.huang@duke.edu             ascoli@gmu.edu\n\n\n# abstract\n\nbrain initiative cell census networkbiccnmop-ul 3d  mop-ul mop-ul  24 inputoutput wiring diagram\n\n\n# introduction\n\ngranularitymacroscale mesoscale microscale nanoscale mri  macroscale connectomes\n\n scrnaeq ccf\n\nbiccn  mop-ul  barseq  ccf  mop-ul \n\n\n# results\n\n web  (extended data fig. 1, methods) supplementary table 1  supplementary table 2 \n\n\n# mop-ul borders and cell types\n\nmop-ul \n\nprimary motor cortex, mop-- 2d  3d  mop-ul  3d  (fig. 1a, supplementary video 1; datasets can be viewed at https://viz.neurodata.io/?json_url=https://json.neurodata.io/v1?ngstateid=lwz24nszk1jthw)\n\nmop-ul primary somatosensory, ssp nissl- neurotrace-  mop  5 l5 ssp  l5a  l5b  (fig. 1b, extended data figs. 2a, b; see also the allen reference atlas (ara) and http://brainmaps.org) mop agranular cortex l4mopmos ssp  (fig. 1b, extended data fig. 2b; see also algorithmic analysis of mopssp border, revealing individual variations between animals in extended data fig. 2c, d, supplementary information)\n\nvglut1slc17a7 mop-ssp  l4  l5 fig. 1a, b, supplementary video 2 vglut3  mop-ulmos (fig. 1a, b)aav ssp ssp-ul mop-ulorbvlrspd mosextended data fig. 3a cervical (to delineate upper limb)  lumbar (to delineate lower limb)  aavretro  (fig. 1b, extended data figs. 3b, c, 4, supplementary video 1)mop l5 projecting to the intermediate and ventral horn ssp l4 projecting to the dorsal hornfig. 1b, extended data figs. 4, 5i mop-ul  aav-rfpphaseolus vulgaris leucoagglutinin (phal)  aav-gfp  mosmop-ul  ssp fig. 1b, extended data fig. 5\n\n neuroglancer  ccfv3 average template  mop-ul  3d  3d fig. 1c, extended data fig. 2e, supplementary video 2 ara  franklin-paxinos  allen ccfv3 extended data fig. 2f, supplementary information\n\n mop-ul vglut1+gaba -gad2+ gaba  cre extended data fig. 3d\n\n\n# laminar organization of neuron types\n\n\n\n 6  8  layers pns1intratelencephalicit l2-l6 2pyramidal tractptextratelencephalicet l5 3-corticothalamicct l6  mop-ul  pns fluorogold and cholera toxin b subunit (ctb) 15   mop --caudoputamencp targetsfig. 1b, extended data figs. 3b, c, 7 mop-ul  pns fig. 1d, extended data fig. 3b, c, 7 16  it 7  et  3  ct laminar organization26  pn  11 1, 2, 3, 4, 5a, 5b-superficial, 5b-middle, 5b-deep, 6a-superficial, 6a-deep and 6bfig. 1d connectivity  pn fig. 1d nissl extended data fig. 6\n\n it 1 l2  l5 temporal association areatea2 l4  mos  ssp 3 l6b extended data fig. 7 pn  cre  aav extended data fig. 8aextended data fig. 8b, cl5a  l5b  it l5a  it  cp l5b  it  cp l5b  it  cp  l5a  it  cp \n\nin situ hybridization mop extended data fig. 9\n\n\n# outputs of mop-ul\n\nmop-ul \n\n ccf  mop-ul  phal  mop-ul mop-ul  110  60 extended data figs. 5, 10, supplementary information l2/3l4l5 itl5 et  l6 ct  cre fig. 2a, bextended data fig. 11\n\n ccfv3  314 fig. 2c, source data fig. 2mop-ul isocortexstriatumthalamus 44.9%29.0%  8.1%midbrainmedullaponsextended data fig. 13d cre fig. 2c, extended data figs. 12, 13a, 14sepw1-l2/3cux2-l2/3nr5a1-l4scnn1a-l4/5plxnd1-l2/3 + l5  tlx3-l5 isocortex cp it sim1-l5  fezf2-l5/6 subcortical et  ntsr1-l6  tle4-l6 thalamic nuclei ct  cre  rbp4-l5  it  etfig. 2a, c, extended data fig. 12\n\nfig. 2ccluster 1  l5 et  phalaav-gfp  rbp4-l5 it/etcluster 2  l6 ct  ntsr1-l6  tle4-l6cluster 3  4  it pn cluster 3  cux2-l2/3tlx3-l5plxnd1-l2/3 + l5cluster 4  sepw1-l2/3nr5a1-l4scnn1a-l4 cluster tle4-l6  ntsr1-l6extended data fig. 13d, left mop-ul fig. 2d, extended data fig. 14\n\n\n# inputs to mop-ul\n\nmop-ul \n\nfig. 3a, b mop1 ctbextended data figs. 7, 10, 15(2)  cre  cre pvalbsst  vip(3)  (trio)  aavretro-cre extended data fig. 16actb  mop-ul mopsspsssmosvalpfpo vmextended data figs. 10, 15 cre extended data figs. 13b, 15, 16agpegpiceaccsdr mop-ul ctb extended data fig. 15\n\nmop-ul  ccfv3 fig. 3c, source data fig. 3isocortexthalamus 90.1%7.7%extended data fig. 13f, pie chartfig. 3ccluster  ctb  cre  cluster  trio  cre  (foxp2-l6)average n=91 versus 30 input regions, p<0.0001, two-tailed t-test\n\n cre-  target-  mop-ul extended data fig. 16b mop mop extended data fig. 10 mop-ul fig. 3d\n\n (l2/3/4)  (l5) mop extended data fig. 17ael5 \n\n\n# barseq projection mapping\n\nbarseq \n\ncre barseq  rna  barseq10,299  mop  39 fig. 4aextended data fig. 18af, supplementary informationsupplementary information, extended data fig. 18gk\n\n ctl5 et  it with (it str+) or without (it str) projections to the striatumfig. 4b, extended data fig. 19ac, supplementary informationmethods 18 fig. 4c, extended data fig. 19dk, supplementary information barseq pns  11 fig. 4d11  barseq pn fig. 4d\n\n it  mosmos-contra l3  l5b-d  it 72-93 mosmos-ipsi mos-contra  it  32-50  mos-ipsifig. 4e, extended data fig. 19lextended data fig. 19lmos-ipsi  l2  ssssss-ipsi l2-4  mos-ipsi  sss-ipsi  mop-contra fig. 4fmop-contra ss-contrafig. 4fl2/3 it l5al5b  l6a it fig. 4gfig. 1d\n\n\n# single-neuron projection patterns\n\n\n\nfmost ccfv3  140  janelia mouselight  121  fmost  42  12  303  mop  113  mop-ul fig. 5a, extended data fig. 20a\n\nfig. 5b, source data fig. 5extended data fig. 20b, c\n\n fig. 2  13 c1c13; fig. 5b, c cre /ctet  itc1  it  et c2  ct cre  l6 ct c2.1vpm source data fig. 5 \n\nc3-c5  mop l5 et fig. 5b, c, ec3  c4 c3c4 c3 3.2psvpbc3.2 c4  et mrnscapnpagpgtrnprnrvalvmpopf l5b fig. 1dc4 c4.2 rt\n\nit  cre  c6-c13 it 1c7c10  c11 l2/3 c7  c13  l42c8  it one-way anova and tukeys post hoc test, p<0.00013two-way repeated measures anova, p<0.0001 interaction effect of cluster  target area c7c8c12 c13tukeys post hoc test, p<0.05c9 aid c6c9  c12 tukeys post hoc test, p<0.05c11 acav c6c9c11 c13tukeys post hoc test, p<0.05c12 ssp-ul  sss\n\nc11  c13  it  c8-c10  c12  ca1 cptukeys post hoc test, p<0.02 barseq  it str  it str+ c8  l5  it ceagpec7c11  c13  l2/3  l4  barseq  it ""\n\n phal  mop  15 c1c5  c6 0.5%c232%c440%c812%c97.7%c112.9%c124.9%c3c7c9c131%40%et32%ct28%it\n\n\n# diverse pn axon projection motifs\n\n pn \n\nfig. 5c, extended data figs. 20d etc3c5: r=0.52, 0.51 and 0.56, respectively it c6c12: range 0.540.61 and c13: r=0.66ct c2average spearman r=0.66 et  it \n\n ct fig. 5d vm val  pomdlpcncl pf et fig 5e1-2-3-4-it str+ fig. 5f it str+  l2/3  l48 out of 9 cells; fig. 5f, leftl5  it str+ n=3; fig. 5f, rightit str- fig. 5g it str- \n\n\n# discussion\n\n 3d  24 - 300  1 \n\nfig. 6a; extended data fig. 21-fig. 6b\n\n l2  l5  tea - ect - peri  l6 ct vm  ct  l6a  l6bfig. 1d l6  l6 ct  pn \n\nmop \n\n\n# figure\n\n\n# fig. 1 delineation of the mop-ul region and its cell-type organization\n\n\n\nfig. 1  mop-ul \n\na.  ccf average template  neuroglancer  mop-ul \nb.  nissl  mop-ul left aavretro  cre middle mop-ul  ssp-ul  mos right\nc. mop-ul  ccf  3d \nd. leftlaminar positionsitpt  ctprojection targetsright (top) cre  mop-ul right (bottom) 25m  mop-ul \n\n\n# fig. 2 brain-wide mop-ul projection patterns by layer and class\n\n\n\nfig. 2  mop-ul \n\na.  mop  cre \nb.  ccfv3 \nc.  ccfv3  mop  314  314 \nd.  mop \n\n\n# fig. 3 brain-wide inputs to mop-ul by layer and class\n\n\n\nfig. 3  mop-ul \n\na. \nb. \nc. mop-ul  ccfv3 \nd.  mop \n\n\n# fig. 4 projection mapping with single-cell resolution using barseq\n\n\n\nfig. 4  barseq \n\na.  barseq10,299  mop  39 \nb.  ctl5 et  it with (it str+) or without (it str) projections to the striatum\nc.  18 \nd.  sublayer  subgroup subgroup\ne.  l3  l5b-d  mos  it  mos \nf.  sublayer  x  y\ng.  it  it \n\n\n# fig. 5 full morphological reconstructions reveal diverse single-cell projection motifs\n\n\n\nfig. 5 \n\na.  ccfv3  (n = 303)fig. 1  mop-ul \nb.  fig. 2  13 c1c13\nc.  13  ccfv3 3d \nd.  l6 ct val/po  vm dominant \ne.  l5 et ------ -- \nfg. it  (+) (g)  () (h)  (str) \n\n\n# fig. 6 cell type wiring diagram of the mop-ul\n\n\n\nfig. 6 mop-ul \n\na.  mop-ul left, black mop-ul etct  it cre  s  mop-ul layersrightleft\nb. mop-ul  mop-ul it l2-l6b  6  it  ct  mop-ul  et 1-cortico-spinal2-cortico-medullarmdrn3-cortico-tectal4-cortico-pontine et  gpezistnrn  io',charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex",frontmatter:{title:" --- Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex",date:"2024-04-13T10:43:33.000Z",permalink:"/pages/aaea7b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20231018.(2023.10.18%20MAPseq%20macaque%20Neuron)%20Single%20basolateral%20amygdala%20neurons%20in%20macaques%20exhibit%20distinct%20connectional%20motifs%20with%20frontal%20cortex.html",relativePath:"02./11./02./20231018.(2023.10.18 MAPseq macaque Neuron) Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex.md",key:"v-afb7dc3c",path:"/pages/aaea7b/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:416},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:783},{level:3,title:"Figure 1. MAPseq of macaque BLA",slug:"figure-1-mapseq-of-macaque-bla",normalizedTitle:"figure 1. mapseq of macaque bla",charIndex:794},{level:3,title:"Figure 2. Single-neuron analysis of branching projections from BLA",slug:"figure-2-single-neuron-analysis-of-branching-projections-from-bla",normalizedTitle:"figure 2. single-neuron analysis of branching projections from bla",charIndex:1568},{level:3,title:"Figure 3. BLA specific projections to medial frontal cortex",slug:"figure-3-bla-specific-projections-to-medial-frontal-cortex",normalizedTitle:"figure 3. bla specific projections to medial frontal cortex",charIndex:2077},{level:3,title:"Figure 4. BLA specific projections to ventral frontal cortex",slug:"figure-4-bla-specific-projections-to-ventral-frontal-cortex",normalizedTitle:"figure 4. bla specific projections to ventral frontal cortex",charIndex:2521},{level:3,title:"Figure 5. Tract tracing confirms branching motifs identified by MAPseq",slug:"figure-5-tract-tracing-confirms-branching-motifs-identified-by-mapseq",normalizedTitle:"figure 5. tract tracing confirms branching motifs identified by mapseq",charIndex:2881}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.neuron.2023.09.024" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.neuron.2023.09.024<OutboundLink/></a><br>\nCite: Zeisler ZR, London L, Janssen WG, et al. Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex. Neuron. 2023;111(20):3307-3320.e5.</p>\n',headersStr:"Abstract Figure Figure 1. MAPseq of macaque BLA Figure 2. Single-neuron analysis of branching projections from BLA Figure 3. BLA specific projections to medial frontal cortex Figure 4. BLA specific projections to ventral frontal cortex Figure 5. Tract tracing confirms branching motifs identified by MAPseq",content:": \nDOI: https://doi.org/10.1016/j.neuron.2023.09.024\nCite: Zeisler ZR, London L, Janssen WG, et al. Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex. Neuron. 2023;111(20):3307-3320.e5.\n\n\n\n\n\n\n\nPETER H RUDEBECK\n\nIcahn School of Medicine at Mount Sinai\npeter.rudebeck@mssm.edu\n\n\n# Abstract\n\nBasolateral amygdala, BLAfrontal cortex- BLA  3000  BLA  BLA  BLA  motifs BLA  BLA \n\n\n# Figure\n\n\n# Figure 1. MAPseq of macaque BLA\n\n\n\nFigure 1.  BLA  MAPseq\n\n(A) 19.5 mm18 mm 1  2 3  4 \n(B) EntoHCAImOFClOFCvlPFCscACCpgACCdACCPMdNAccMD\n(C) barcode  barcode  barcode\n(D)  target  control  barcode  control  barcode \n(E)  barcode \n(F)  10  barcode target  control  barcode \n(G) barcode  A  B \n\n\n# Figure 2. Single-neuron analysis of branching projections from BLA\n\n\n\nFigure 2. BLA \n\n(A) 1,300  K-means  (k = 12)\n(B)  BLA \n(C) \n(D) \n(E)  2 3  4  (FDR)  NAcc  AI \n(F) 23  4 \n\n\n# Figure 3. BLA specific projections to medial frontal cortex\n\n\n\nFigure 3. BLA \n\n(A)  scACCpgACCdACC BLA D  V  R  C \n(B) pgACC  dACC z test for proportions, z = 3.17, p = 0.0015dACC  scACC z = 2.27p = 0.023\n(C)  scACCpgACC  dACC \n\n\n# Figure 4. BLA specific projections to ventral frontal cortex\n\n\n\nFigure 4. BLA \n\n(A)  AImOFClOFC vlPFC BLA A  P  M  L \n(B) AI  vlPFC \n(C)  mOFClOFCvlPFC  AI \n\n\n# Figure 5. Tract tracing confirms branching motifs identified by MAPseq\n\n\n\nFigure 5.  MAPseq \n\n(A) \n(B)  AI (EGFP)  NAcc (mCherry)  lOFC (EGFP)  vlPFC (mCherry) intra-aural  mm  2 mm \n(C)  intra-aural mm 1 mm 250 m\n(D) AI/NAcc  MAPseq \n(E)  (D)  lOFC/vlPFC ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.neuron.2023.09.024\ncite: zeisler zr, london l, janssen wg, et al. single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex. neuron. 2023;111(20):3307-3320.e5.\n\n\n\n\n\n\n\npeter h rudebeck\n\nicahn school of medicine at mount sinai\npeter.rudebeck@mssm.edu\n\n\n# abstract\n\nbasolateral amygdala, blafrontal cortex- bla  3000  bla  bla  bla  motifs bla  bla \n\n\n# figure\n\n\n# figure 1. mapseq of macaque bla\n\n\n\nfigure 1.  bla  mapseq\n\n(a) 19.5 mm18 mm 1  2 3  4 \n(b) entohcaimofclofcvlpfcscaccpgaccdaccpmdnaccmd\n(c) barcode  barcode  barcode\n(d)  target  control  barcode  control  barcode \n(e)  barcode \n(f)  10  barcode target  control  barcode \n(g) barcode  a  b \n\n\n# figure 2. single-neuron analysis of branching projections from bla\n\n\n\nfigure 2. bla \n\n(a) 1,300  k-means  (k = 12)\n(b)  bla \n(c) \n(d) \n(e)  2 3  4  (fdr)  nacc  ai \n(f) 23  4 \n\n\n# figure 3. bla specific projections to medial frontal cortex\n\n\n\nfigure 3. bla \n\n(a)  scaccpgaccdacc bla d  v  r  c \n(b) pgacc  dacc z test for proportions, z = 3.17, p = 0.0015dacc  scacc z = 2.27p = 0.023\n(c)  scaccpgacc  dacc \n\n\n# figure 4. bla specific projections to ventral frontal cortex\n\n\n\nfigure 4. bla \n\n(a)  aimofclofc vlpfc bla a  p  m  l \n(b) ai  vlpfc \n(c)  mofclofcvlpfc  ai \n\n\n# figure 5. tract tracing confirms branching motifs identified by mapseq\n\n\n\nfigure 5.  mapseq \n\n(a) \n(b)  ai (egfp)  nacc (mcherry)  lofc (egfp)  vlpfc (mcherry) intra-aural  mm  2 mm \n(c)  intra-aural mm 1 mm 250 m\n(d) ai/nacc  mapseq \n(e)  (d)  lofc/vlpfc ",charsets:{cjk:!0},lastUpdated:"2024/05/28, 12:17:48",lastUpdatedTimestamp:1716869868e3},{title:" --- High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex",frontmatter:{title:" --- High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex",date:"2023-03-14T16:32:39.000Z",permalink:"/pages/94c1e9/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20221027.(2022.10.27%20%E5%97%85%E8%A7%89%E7%9A%AE%E5%B1%82MAPseq%20Cell)%20High-throughput%20sequencing%20of%20single%20neuron%20projections%20reveals%20spatial%20organization%20in%20the%20olfactory%20cortex.html",relativePath:"02./11./02./20221027.(2022.10.27 MAPseq Cell) High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex.md",key:"v-e7256176",path:"/pages/94c1e9/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:642},{level:2,title:"INTRODUCTION",slug:"introduction",normalizedTitle:"introduction",charIndex:1055},{level:2,title:"RESULTS",slug:"results",normalizedTitle:"results",charIndex:2344},{level:2,title:"DISCUSSION",slug:"discussion",normalizedTitle:"discussion",charIndex:12833},{level:2,title:"Limitations of the study",slug:"limitations-of-the-study",normalizedTitle:"limitations of the study",charIndex:16987},{level:2,title:"Conclusions",slug:"conclusions",normalizedTitle:"conclusions",charIndex:17228},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:2736},{level:3,title:"Figure 1. MAPseq and BARseq projection mapping of individual olfactory bulb neurons",slug:"figure-1-mapseq-and-barseq-projection-mapping-of-individual-olfactory-bulb-neurons",normalizedTitle:"figure 1. mapseq and barseq projection mapping of individual olfactory bulb neurons",charIndex:17420},{level:3,title:"Figure 2. MAPseq reveals structured correlations in the distribution of olfactory bulb projections to their targets",slug:"figure-2-mapseq-reveals-structured-correlations-in-the-distribution-of-olfactory-bulb-projections-to-their-targets",normalizedTitle:"figure 2. mapseq reveals structured correlations in the distribution of olfactory bulb projections to their targets",charIndex:18176},{level:3,title:"Figure 3. Olfactory bulb pMC projections tile the anterior-posterior axis of the piriform cortex and are interdependent with co-projections to extra-piriform target regions",slug:"figure-3-olfactory-bulb-pmc-projections-tile-the-anterior-posterior-axis-of-the-piriform-cortex-and-are-interdependent-with-co-projections-to-extra-piriform-target-regions",normalizedTitle:"figure 3. olfactory bulb pmc projections tile the anterior-posterior axis of the piriform cortex and are interdependent with co-projections to extra-piriform target regions",charIndex:18697},{level:3,title:"Figure 4. Narrowly and broadly projecting pMC tile differentially the anterior-posterior axis of the piriform cortex",slug:"figure-4-narrowly-and-broadly-projecting-pmc-tile-differentially-the-anterior-posterior-axis-of-the-piriform-cortex",normalizedTitle:"figure 4. narrowly and broadly projecting pmc tile differentially the anterior-posterior axis of the piriform cortex",charIndex:19085},{level:3,title:"Figure 5. Projection patterns of piriform cortex output neurons are systematically organized along the anterior-posterior axis with respect to their extra-piriform targets",slug:"figure-5-projection-patterns-of-piriform-cortex-output-neurons-are-systematically-organized-along-the-anterior-posterior-axis-with-respect-to-their-extra-piriform-targets",normalizedTitle:"figure 5. projection patterns of piriform cortex output neurons are systematically organized along the anterior-posterior axis with respect to their extra-piriform targets",charIndex:19707},{level:3,title:"Figure 6. Matched input and output circuit motifs in the piriform cortex",slug:"figure-6-matched-input-and-output-circuit-motifs-in-the-piriform-cortex",normalizedTitle:"figure 6. matched input and output circuit motifs in the piriform cortex",charIndex:20225},{level:3,title:"Figure 7. Cartoon schematics of parallel olfactory processing streams structured along the anterior-posterior axis of the piriform cortex according to triadic connectivity motifs",slug:"figure-7-cartoon-schematics-of-parallel-olfactory-processing-streams-structured-along-the-anterior-posterior-axis-of-the-piriform-cortex-according-to-triadic-connectivity-motifs",normalizedTitle:"figure 7. cartoon schematics of parallel olfactory processing streams structured along the anterior-posterior axis of the piriform cortex according to triadic connectivity motifs",charIndex:20572},{level:2,title:"Supplemental Figure",slug:"supplemental-figure",normalizedTitle:"supplemental figure",charIndex:21001},{level:3,title:"Figure S1. Olfactory bulb barcoded Sindbis virus injections and normalization schemes",slug:"figure-s1-olfactory-bulb-barcoded-sindbis-virus-injections-and-normalization-schemes",normalizedTitle:"figure s1. olfactory bulb barcoded sindbis virus injections and normalization schemes",charIndex:21025},{level:3,title:"Figure S2. Classification of the olfactory bulb output neurons based on their projection patterns using BARseq data as template",slug:"figure-s2-classification-of-the-olfactory-bulb-output-neurons-based-on-their-projection-patterns-using-barseq-data-as-template",normalizedTitle:"figure s2. classification of the olfactory bulb output neurons based on their projection patterns using barseq data as template",charIndex:21423},{level:3,title:"Figure S3. Spatial distribution of barcode labeled pMC somata and projection patterns as a function of different locations within the olfactory bulb",slug:"figure-s3-spatial-distribution-of-barcode-labeled-pmc-somata-and-projection-patterns-as-a-function-of-different-locations-within-the-olfactory-bulb",normalizedTitle:"figure s3. spatial distribution of barcode labeled pmc somata and projection patterns as a function of different locations within the olfactory bulb",charIndex:21936},{level:3,title:"Figure S4. Reproducibility across individuals, shuffled controls and downsampling analysis on the projection patterns of the olfactory bulb output neurons",slug:"figure-s4-reproducibility-across-individuals-shuffled-controls-and-downsampling-analysis-on-the-projection-patterns-of-the-olfactory-bulb-output-neurons",normalizedTitle:"figure s4. reproducibility across individuals, shuffled controls and downsampling analysis on the projection patterns of the olfactory bulb output neurons",charIndex:22411},{level:3,title:"Figure S5. Validation of the spatial organization of pMC projections using CAV-Cre bulk retrograde labeling",slug:"figure-s5-validation-of-the-spatial-organization-of-pmc-projections-using-cav-cre-bulk-retrograde-labeling",normalizedTitle:"figure s5. validation of the spatial organization of pmc projections using cav-cre bulk retrograde labeling",charIndex:22616},{level:3,title:"Figure S6. Spatial organization of the projections of broadly projecting and narrowly projecting cells",slug:"figure-s6-spatial-organization-of-the-projections-of-broadly-projecting-and-narrowly-projecting-cells",normalizedTitle:"figure s6. spatial organization of the projections of broadly projecting and narrowly projecting cells",charIndex:22774},{level:3,title:"Figure S7. Differential brain-wide projections of groups of piriform cortex output neurons organized in a graded manner along the A-P axis",slug:"figure-s7-differential-brain-wide-projections-of-groups-of-piriform-cortex-output-neurons-organized-in-a-graded-manner-along-the-a-p-axis",normalizedTitle:"figure s7. differential brain-wide projections of groups of piriform cortex output neurons organized in a graded manner along the a-p axis",charIndex:22915},{level:3,title:"Figure S8. Validation of piriform cortex output projection patterns along A-P axis via bulk retrograde tracing",slug:"figure-s8-validation-of-piriform-cortex-output-projection-patterns-along-a-p-axis-via-bulk-retrograde-tracing",normalizedTitle:"figure s8. validation of piriform cortex output projection patterns along a-p axis via bulk retrograde tracing",charIndex:23106}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2022.09.038" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2022.09.038<OutboundLink/></a><br>\nCite: Chen Y, Chen X, Baserdem B. et al. High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex. Cell. 2022 Oct 27;185(22):4117-4134.e28.</p>\n',headersStr:"Abstract INTRODUCTION RESULTS DISCUSSION Limitations of the study Conclusions Figure Figure 1. MAPseq and BARseq projection mapping of individual olfactory bulb neurons Figure 2. MAPseq reveals structured correlations in the distribution of olfactory bulb projections to their targets Figure 3. Olfactory bulb pMC projections tile the anterior-posterior axis of the piriform cortex and are interdependent with co-projections to extra-piriform target regions Figure 4. Narrowly and broadly projecting pMC tile differentially the anterior-posterior axis of the piriform cortex Figure 5. Projection patterns of piriform cortex output neurons are systematically organized along the anterior-posterior axis with respect to their extra-piriform targets Figure 6. Matched input and output circuit motifs in the piriform cortex Figure 7. Cartoon schematics of parallel olfactory processing streams structured along the anterior-posterior axis of the piriform cortex according to triadic connectivity motifs Supplemental Figure Figure S1. Olfactory bulb barcoded Sindbis virus injections and normalization schemes Figure S2. Classification of the olfactory bulb output neurons based on their projection patterns using BARseq data as template Figure S3. Spatial distribution of barcode labeled pMC somata and projection patterns as a function of different locations within the olfactory bulb Figure S4. Reproducibility across individuals, shuffled controls and downsampling analysis on the projection patterns of the olfactory bulb output neurons Figure S5. Validation of the spatial organization of pMC projections using CAV-Cre bulk retrograde labeling Figure S6. Spatial organization of the projections of broadly projecting and narrowly projecting cells Figure S7. Differential brain-wide projections of groups of piriform cortex output neurons organized in a graded manner along the A-P axis Figure S8. Validation of piriform cortex output projection patterns along A-P axis via bulk retrograde tracing",content:': \nDOI: https://doi.org/10.1016/j.cell.2022.09.038\nCite: Chen Y, Chen X, Baserdem B. et al. High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex. Cell. 2022 Oct 27;185(22):4117-4134.e28.\n\n\n\n\n\n\n\nDINU FLORIN ALBEANU             ALEXEI KOULAKOV                 ANTHONY ZADOR\n                                                                \nCold Spring Harbor Laboratory   Cold Spring Harbor Laboratory   Cold Spring Harbor Laboratory\nalbeanu@cshl.edu                koulakov@cshl.edu               zador@cshl.edu\n\n\n# Abstract\n\nolfactory cortex 5,309 olfactory bulb 30,433 piriform cortexanterior-posterior axis\n\n\n# INTRODUCTION\n\n\n\nolfactionolfactory bulbOBolfactory cortexpiriform intra-piriformolfactory cortex  piriform cortex anterior and posterior parts of olfactory cortex  piriform cortex \n\n olfactory cortex  olfactory cortex MAPseqOB5,309piriform cortex30,433 OB  piriform cortex  OB piriform cortex \n\n olfactory cortex  OB  piriform cortex  piriform cortex  piriform cortex A-P- piriform cortex  OB  piriform cortex  piriform cortex what/where\n\n\n# RESULTS\n\nMapping the brain-wide projections of olfactory bulb outputs using neuronal barcoding\n\n\nMAPseq  BARseq (Figures 1A1C, S1, and STAR Methods)\n\nOB mitraltufted mitral  tufted  OB  BARseqBARseq  MAPseq Figure 1D A-P axis 200 OB  anterior piriform cortexAPC posterior piriform cortexPPCanterior olfactory nucleusAONolfactory tubercleOTolfactory striatum  cortical amygdalaCoAlateral entorhinal cortexlENT 5,309  OB  BARseq  415  MAPseq  4,894 \n\nThe distribution of olfactory bulb projections to their targets displays structured correlations\n\n\n BARseq  mitral cells  tufted cellsFigures 1D and 1E putative mitral cellspMC mitral cell layerMCL putative tufted cells (pTC)  external plexiform layerEPLpTC  AON / OT pMC  anterior-to-posterior piriform cortex Figure 1E putative deep cells (pDC) granule cell layerGCLBARseq \n\n BARseq  MAPseq  MAPseq  MAPseq  BARseq  MAPseq  4,894 mitral, tufted or deep cell BARseq Figure 1F,S2AS2G, and STAR Methods BARseq  MAPseq  mitral cell layer pMC  pTC  pDC Figures 1B, D, E, 2A, STAR Methods MAPseq  4,665  pMC90  pTC  139  pDCFigures 2A2C BARseq Figure 1E mitral cell layer >85% classification accuracy; Figure S2H, STAR Methods pMC 4,388\n\n pTC  pDC  mitral cells Figures 2A2Dinverse participation ratioIPR mitral cells Figures 2D, S2I, and STAR Methods IPR  pMC  mitral cells  pMC  AON  OT  Figures 2E and S2J for pMC; Figure S2K for pTC and pDC PPC  CoA  lENT  pMC  shuffling projections pMCFigures 2FFigure S3A-S3C  STAR Methods A-P (Figures S3DS3F) pMC \n\nOlfactory bulb-to-piriform cortex connectivity along the A-P axis and bulb co-innervation of extra-piriform target regions are interdependent\n A-P \n\nAnterior and posterior piriform cortex  2.5 mm pMC  piriform cortex  extra-piriform  olfactory bulb  piriform cortex  extra-piriform  extra-piriform P(target|PC location) piriform cortex (200um)(Figures 3A-3CS4A-S4CSTAR Methods) extra-piriform  piriform cortex \n\nP(target|PC location) piriform cortex  AONCoA  lENT  OT pMC  AON  piriform cortex (Figures 3CFigures S4B and S4C) CoA  lENT  piriform cortex (Figures 3CFigures S4B and S4C)P(target|PC location)(Figures 3C) anterior and posterior piriform cortex (Figures 3C)(Figures S4D)\n\npMC  OT anterior piriform cortex  posterior piriform cortex piriform cortex  locality model (Figures S4B) CAV-2 Cre (Figures S5A-S5CSTAR Methods) extra-piriform  piriform cortex \n\nProjections of narrowly and broadly projecting pMC differentially tile the A-P axis of the piriform cortex and innervate distinct domains of extra-piriform bulb target regions\n pMC \n\n pMC narrowly projecting(NP) broadly projecting(BP) piriform cortex  extra-piriform (Figures 4A-4DS6  STAR Methods)NP (20%)(80%)BP pMC (Figures 4A)NP  APC (Figures 4B and 4C) BP  APC  anterior and posterior piriform cortex (Figures 4B and 4D)NP  BP  anterior piriform cortex (Figures 4C and 4D) BP  posterior piriform cortex  NP  APC  PPC (Figures 4C and 4D) NP (Figures 4C STAR Methods)\n\nx(n)=wexp[a(Nn)/N]x(n)=w exp[- a(N-n)/N]x(n)=wexp[a(Nn)/N] (Equation 1)\n\n x  APC w  APC n N a = 3.8  anterior piriform cortex (x)=dn/dx=N/ax.\\rho(x)=|dn/dx| = N/ax.(x)=dn/dx=N/ax.  NP  APC  x0 \n\nBP  anterior piriform cortex  (red curve in Figure 4D):\n\nx(n)=w[1exp(an/N)]x(n)=w[1 - exp(-an/N)]x(n)=w[1exp(an/N)] (Equation 2)\n\n a = 2.9Equation 2  BP  piriform cortex  posterior APC  xw (x) = N/a(w-x)Figure 4DBP  posterior piriform cortex  NP  APC  [Equation 1 with a = 6:5] BP  NP  A-P  posterior piriform cortex  anterior piriform cortex (Figure 4E4J,S6ES6J, and STAR Methods) extra-piriform  A-P  NP  BP Figure S6B NP BP OTCoA  lENT  AON \n\n NP  BP  piriform A-P  pMC  NP  BP  piriform cortex  extra-piriform Figure S6C AON  piriform cortex  NP  BP  CoA  lENT  piriform cortex (Figure S6D and Table S1)Figure 3C piriform cortex  extra-piriform \n\n A-P  olfactory bulb-to-piriform cortex mitral versus tufted cells extra-piriform NPBP NP  BP  piriform cortex  extra-piriform \n\nPiriform cortex output neurons distributed along the anterior-posterior axis project to different target regions\n\n\n piriform cortex  A-P  extra-piriform  rostro-caudal intra-piriform  piriform cortex  piriform cortex  A-P  MAPseq  A-P 5 piriform cortex  30,433 Figure 5A piriform cortex APC and PPC, Figure 5A, right 14 Table S2 and Figure 5A, left piriform cortex (blue line in Figure 5A, Right)200 intra-piriform Figures 5B, 5C, and S7A extra-piriform (Figures S7BS7E and STAR Methods)\n\n piriform cortex  intra-piriform Lorentzian function,STAR Methods, Figure 5D2 intra-piriform  anterior  posterior  anterior-to-posterior Figures 5A5Cmitral cell Figures 3A and 5D>10 intra-piriform  intra-piriform  piriform cortex \n\n piriform cortex  A-P  extra-piriform  Louvain  (Figure 5E and STAR Methods)  piriform cortex  extra-piriform  A-P  intra-piriform (Figures 5E, S7FS7K, and STAR Methods)14 OB ORB(orbitofrontal cortex) anterior piriform cortex 3 lENT posterior piriform cortex Figure 5FFigure S7H piriform cortex \n\nMatching projection gradients of piriform cortex inputs and outputs along the anterior-posterior axis\n\n\n piriform cortex  olfactory bulb-to-piriform cortex  olfactory bulb-to-piriform cortex  piriform cortex  AONlENT  CoA  A-P Figure 6Apiriform cortex  AON  piriform cortex  anterior  posterior  piriform cortex  lENT  CoA piriform cortex  OT  anterior-to-posterior piriform cortex  piriform cortex  extra-piriform  piriform cortex PCAAPC/PPCFigure 6BS7LS7MSTAR MethodsRetroBeadsFigure S8A-S8CA-P\n\n olfactory bulb-to-piriform cortex  piriform cortex  piriform A-P Figure 3C5A5F6A A-P piriform cortex  AONCoA  lENT  pMC P(targetj|PC locationFigure 6C piriform cortex Figure 6C AONy piriform cortex  pMC  AONx piriform cortex  AON  piriform cortex  pMC  AON  CoA  lENT  mitral cell  piriform cortex  A-P  piriform cortex  OT \n\n piriform cortex  AONlENT  CoA Figure 7 piriform cortex  piriform cortex  extra-piriform  piriform cortex  A-P AONCoAlENT\n\n\n# DISCUSSION\n\n-Figure 3Figure 6Figure 3 and 4A-PFigure 5-Figure 6 and 7\n\nStructured bulb-to-piriform cortex connectivity\n\n\nA-P(mitral versus tufted cells)NPBPA-POBNPBPFigure 4G and 4J\n\n mitral cellspiriform cortex  extra-piriform mitral cells  mitral cells  mitral cells  mitral cells  mitral  tufted \n\nFunction- and location-based connectivity schemes\n\n\nFigure 3C/OTA-PFigure 3CFigure 6BAONFigure 5EFigure 3C and 6A\n\nLocal piriform cortex connectivity\n\n\nFigure 5Dintra-piriform A-P2intra-piriform  intra-piriform STAR Methods\n\nMatched triadic circuit motifs\n\n\nA-PAONCoA entorhinal cortex AONCoAlENT\n\nResNet----\n\nImplications for olfactory processing\n\n\n"what""where" two-stream what/where  perception[]OB-APC-AONvalence[]OB-PPC-CoA action[]OB-PPC-lENTOBAPCOBAONAPCAON () OBAPCAONAONAPCAONAPCOBPPCOBCoAPPCCoAOBPPCOBlENTPPClENTPPCAPC\n\n\n# Limitations of the study\n\n\n\n\n# Conclusions\n\n\n\n\n# Figure\n\n\n# Figure 1. MAPseq and BARseq projection mapping of individual olfactory bulb neurons\n\n\n\nFigure 1.  MAPseq  BARseq \n\n(A) MAPseq  RNA barcodes \n(B)  barcodes EGFP Sindbis  mitral cells  tufted cells\n(C)  Allen \n(D) mitral, tufted, deep cells  laminar positions barcoded cells  BARseq  BARseq  barcode  6 \n(E)  BARseq  (415 neurons, 2 mice) mitral cell layer (MCL)  pDCpMCpTC MCL pTC  AON / OT mitral cells \n(F) BARseq  MCL  GL  BARseq \n\n\n# Figure 2. MAPseq reveals structured correlations in the distribution of olfactory bulb projections to their targets\n\n\n\nFigure 2. MAPseq \n\n(A-C)  MAPseq (A)(B) BARseq-based classifier  5 (C)\n(D) [IPR]x\n(E) putative mitral cell(pMC)  Bonferroni \n(F)  BARseq-based classifier  pMC  pMC  city block distance area shuffled\n\n\n# Figure 3. Olfactory bulb pMC projections tile the anterior-posterior axis of the piriform cortex and are interdependent with co-projections to extra-piriform target regions\n\n\n\nFigure 3.  pMC \n\n(A) pMC  A-P  A-P \n(B)  pMC \n(C) pMC  P(target|PC location)\n\n\n# Figure 4. Narrowly and broadly projecting pMC tile differentially the anterior-posterior axis of the piriform cortex\n\n\n\nFigure 4.  pMC \n\n(A)  (inverse participation ratio [IPR]) clustersNP  BP putative mitral cells (pMC)Watershed clustering \n(B) NP  BP \n(C-D) NP(C)  BP(D) pMC  A-P NP BP  \n(E-F)  APC  PPC  NP  APC(E)  PPC(F)  A-P \n(G) NP  APC  PPC  A-P \n(H-I)  APC  PPC  BP  APC(H)  PPC(I)  A-P \n(J) BP  APC  PPC  A-P \n\n\n# Figure 5. Projection patterns of piriform cortex output neurons are systematically organized along the anterior-posterior axis with respect to their extra-piriform targets\n\n\n\nFigure 5. \n\n(A)  A-P  A-P  A-P \n(B)  A-P  A-P \n(C) \n(D) pMC \n(E) \n(F)  A-P \n\n\n# Figure 6. Matched input and output circuit motifs in the piriform cortex\n\n\n\nFigure 6. \n\n(A)  barcode  A-P \n(B)  AONCoAlENT  OT  slice\n(C)  A-P y axis pMC  A-P P(target|PC location) (x axis)\n\n\n# Figure 7. Cartoon schematics of parallel olfactory processing streams structured along the anterior-posterior axis of the piriform cortex according to triadic connectivity motifs\n\n\n\nFigure 7. \n\nOBanterior olfactory nucleusAON AON CoAlENT A-P \n\n\n# Supplemental Figure\n\n\n# Figure S1. Olfactory bulb barcoded Sindbis virus injections and normalization schemes\n\n\n\nFigure S1. \n\n(A) barcoded Sindbis virus \n(B)  Allen \n(C) YC61 spike-in RNAspike-in RNA and b-actin (measured by qPCR)spike-in RNA and dissected brain area size \n(D) \n(E) \n\n\n# Figure S2. Classification of the olfactory bulb output neurons based on their projection patterns using BARseq data as template\n\n\n\nFigure S2.  BARseq \n\n(A) BARseq  MCL  GL \n(B)  ROC \n(C) \n(D) MAPseq \n(E) \n(F) \n(G)  pMCs 85%\n(H) 85%\n(I) IPR \n(J)  pMC \n(K) pDC  pTC \n\n\n# Figure S3. Spatial distribution of barcode labeled pMC somata and projection patterns as a function of different locations within the olfactory bulb\n\n\n\nFigure S3.  pMC \n\n(A)  Barcode counts A-P \n(B)  pMC \n(C)  pMC \n(D)  pMC  A-P \n(E)  pMC  A-P \n(F)  A-P  pMC  A-P \n\n\n# Figure S4. Reproducibility across individuals, shuffled controls and downsampling analysis on the projection patterns of the olfactory bulb output neurons\n\n\n\nFigure S4. \n\n\n# Figure S5. Validation of the spatial organization of pMC projections using CAV-Cre bulk retrograde labeling\n\n\n\nFigure S5.  CAV-Cre  pMC \n\n\n# Figure S6. Spatial organization of the projections of broadly projecting and narrowly projecting cells\n\n\n\nFigure S6. \n\n\n# Figure S7. Differential brain-wide projections of groups of piriform cortex output neurons organized in a graded manner along the A-P axis\n\n\n\nFigure S7.  A-P \n\n\n# Figure S8. Validation of piriform cortex output projection patterns along A-P axis via bulk retrograde tracing\n\n\n\nFigure S8.  A-P ',normalizedContent:': \ndoi: https://doi.org/10.1016/j.cell.2022.09.038\ncite: chen y, chen x, baserdem b. et al. high-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex. cell. 2022 oct 27;185(22):4117-4134.e28.\n\n\n\n\n\n\n\ndinu florin albeanu             alexei koulakov                 anthony zador\n                                                                \ncold spring harbor laboratory   cold spring harbor laboratory   cold spring harbor laboratory\nalbeanu@cshl.edu                koulakov@cshl.edu               zador@cshl.edu\n\n\n# abstract\n\nolfactory cortex 5,309 olfactory bulb 30,433 piriform cortexanterior-posterior axis\n\n\n# introduction\n\n\n\nolfactionolfactory bulbobolfactory cortexpiriform intra-piriformolfactory cortex  piriform cortex anterior and posterior parts of olfactory cortex  piriform cortex \n\n olfactory cortex  olfactory cortex mapseqob5,309piriform cortex30,433 ob  piriform cortex  ob piriform cortex \n\n olfactory cortex  ob  piriform cortex  piriform cortex  piriform cortex a-p- piriform cortex  ob  piriform cortex  piriform cortex what/where\n\n\n# results\n\nmapping the brain-wide projections of olfactory bulb outputs using neuronal barcoding\n\n\nmapseq  barseq (figures 1a1c, s1, and star methods)\n\nob mitraltufted mitral  tufted  ob  barseqbarseq  mapseq figure 1d a-p axis 200 ob  anterior piriform cortexapc posterior piriform cortexppcanterior olfactory nucleusaonolfactory tubercleotolfactory striatum  cortical amygdalacoalateral entorhinal cortexlent 5,309  ob  barseq  415  mapseq  4,894 \n\nthe distribution of olfactory bulb projections to their targets displays structured correlations\n\n\n barseq  mitral cells  tufted cellsfigures 1d and 1e putative mitral cellspmc mitral cell layermcl putative tufted cells (ptc)  external plexiform layereplptc  aon / ot pmc  anterior-to-posterior piriform cortex figure 1e putative deep cells (pdc) granule cell layergclbarseq \n\n barseq  mapseq  mapseq  mapseq  barseq  mapseq  4,894 mitral, tufted or deep cell barseq figure 1f,s2as2g, and star methods barseq  mapseq  mitral cell layer pmc  ptc  pdc figures 1b, d, e, 2a, star methods mapseq  4,665  pmc90  ptc  139  pdcfigures 2a2c barseq figure 1e mitral cell layer >85% classification accuracy; figure s2h, star methods pmc 4,388\n\n ptc  pdc  mitral cells figures 2a2dinverse participation ratioipr mitral cells figures 2d, s2i, and star methods ipr  pmc  mitral cells  pmc  aon  ot  figures 2e and s2j for pmc; figure s2k for ptc and pdc ppc  coa  lent  pmc  shuffling projections pmcfigures 2ffigure s3a-s3c  star methods a-p (figures s3ds3f) pmc \n\nolfactory bulb-to-piriform cortex connectivity along the a-p axis and bulb co-innervation of extra-piriform target regions are interdependent\n a-p \n\nanterior and posterior piriform cortex  2.5 mm pmc  piriform cortex  extra-piriform  olfactory bulb  piriform cortex  extra-piriform  extra-piriform p(target|pc location) piriform cortex (200um)(figures 3a-3cs4a-s4cstar methods) extra-piriform  piriform cortex \n\np(target|pc location) piriform cortex  aoncoa  lent  ot pmc  aon  piriform cortex (figures 3cfigures s4b and s4c) coa  lent  piriform cortex (figures 3cfigures s4b and s4c)p(target|pc location)(figures 3c) anterior and posterior piriform cortex (figures 3c)(figures s4d)\n\npmc  ot anterior piriform cortex  posterior piriform cortex piriform cortex  locality model (figures s4b) cav-2 cre (figures s5a-s5cstar methods) extra-piriform  piriform cortex \n\nprojections of narrowly and broadly projecting pmc differentially tile the a-p axis of the piriform cortex and innervate distinct domains of extra-piriform bulb target regions\n pmc \n\n pmc narrowly projecting(np) broadly projecting(bp) piriform cortex  extra-piriform (figures 4a-4ds6  star methods)np (20%)(80%)bp pmc (figures 4a)np  apc (figures 4b and 4c) bp  apc  anterior and posterior piriform cortex (figures 4b and 4d)np  bp  anterior piriform cortex (figures 4c and 4d) bp  posterior piriform cortex  np  apc  ppc (figures 4c and 4d) np (figures 4c star methods)\n\nx(n)=wexp[a(nn)/n]x(n)=w exp[- a(n-n)/n]x(n)=wexp[a(nn)/n] (equation 1)\n\n x  apc w  apc n n a = 3.8  anterior piriform cortex (x)=dn/dx=n/ax.\\rho(x)=|dn/dx| = n/ax.(x)=dn/dx=n/ax.  np  apc  x0 \n\nbp  anterior piriform cortex  (red curve in figure 4d):\n\nx(n)=w[1exp(an/n)]x(n)=w[1 - exp(-an/n)]x(n)=w[1exp(an/n)] (equation 2)\n\n a = 2.9equation 2  bp  piriform cortex  posterior apc  xw (x) = n/a(w-x)figure 4dbp  posterior piriform cortex  np  apc  [equation 1 with a = 6:5] bp  np  a-p  posterior piriform cortex  anterior piriform cortex (figure 4e4j,s6es6j, and star methods) extra-piriform  a-p  np  bp figure s6b np bp otcoa  lent  aon \n\n np  bp  piriform a-p  pmc  np  bp  piriform cortex  extra-piriform figure s6c aon  piriform cortex  np  bp  coa  lent  piriform cortex (figure s6d and table s1)figure 3c piriform cortex  extra-piriform \n\n a-p  olfactory bulb-to-piriform cortex mitral versus tufted cells extra-piriform npbp np  bp  piriform cortex  extra-piriform \n\npiriform cortex output neurons distributed along the anterior-posterior axis project to different target regions\n\n\n piriform cortex  a-p  extra-piriform  rostro-caudal intra-piriform  piriform cortex  piriform cortex  a-p  mapseq  a-p 5 piriform cortex  30,433 figure 5a piriform cortex apc and ppc, figure 5a, right 14 table s2 and figure 5a, left piriform cortex (blue line in figure 5a, right)200 intra-piriform figures 5b, 5c, and s7a extra-piriform (figures s7bs7e and star methods)\n\n piriform cortex  intra-piriform lorentzian function,star methods, figure 5d2 intra-piriform  anterior  posterior  anterior-to-posterior figures 5a5cmitral cell figures 3a and 5d>10 intra-piriform  intra-piriform  piriform cortex \n\n piriform cortex  a-p  extra-piriform  louvain  (figure 5e and star methods)  piriform cortex  extra-piriform  a-p  intra-piriform (figures 5e, s7fs7k, and star methods)14 ob orb(orbitofrontal cortex) anterior piriform cortex 3 lent posterior piriform cortex figure 5ffigure s7h piriform cortex \n\nmatching projection gradients of piriform cortex inputs and outputs along the anterior-posterior axis\n\n\n piriform cortex  olfactory bulb-to-piriform cortex  olfactory bulb-to-piriform cortex  piriform cortex  aonlent  coa  a-p figure 6apiriform cortex  aon  piriform cortex  anterior  posterior  piriform cortex  lent  coa piriform cortex  ot  anterior-to-posterior piriform cortex  piriform cortex  extra-piriform  piriform cortex pcaapc/ppcfigure 6bs7ls7mstar methodsretrobeadsfigure s8a-s8ca-p\n\n olfactory bulb-to-piriform cortex  piriform cortex  piriform a-p figure 3c5a5f6a a-p piriform cortex  aoncoa  lent  pmc p(targetj|pc locationfigure 6c piriform cortex figure 6c aony piriform cortex  pmc  aonx piriform cortex  aon  piriform cortex  pmc  aon  coa  lent  mitral cell  piriform cortex  a-p  piriform cortex  ot \n\n piriform cortex  aonlent  coa figure 7 piriform cortex  piriform cortex  extra-piriform  piriform cortex  a-p aoncoalent\n\n\n# discussion\n\n-figure 3figure 6figure 3 and 4a-pfigure 5-figure 6 and 7\n\nstructured bulb-to-piriform cortex connectivity\n\n\na-p(mitral versus tufted cells)npbpa-pobnpbpfigure 4g and 4j\n\n mitral cellspiriform cortex  extra-piriform mitral cells  mitral cells  mitral cells  mitral cells  mitral  tufted \n\nfunction- and location-based connectivity schemes\n\n\nfigure 3c/ota-pfigure 3cfigure 6baonfigure 5efigure 3c and 6a\n\nlocal piriform cortex connectivity\n\n\nfigure 5dintra-piriform a-p2intra-piriform  intra-piriform star methods\n\nmatched triadic circuit motifs\n\n\na-paoncoa entorhinal cortex aoncoalent\n\nresnet----\n\nimplications for olfactory processing\n\n\n"what""where" two-stream what/where  perception[]ob-apc-aonvalence[]ob-ppc-coa action[]ob-ppc-lentobapcobaonapcaon () obapcaonaonapcaonapcobppcobcoappccoaobppcoblentppclentppcapc\n\n\n# limitations of the study\n\n\n\n\n# conclusions\n\n\n\n\n# figure\n\n\n# figure 1. mapseq and barseq projection mapping of individual olfactory bulb neurons\n\n\n\nfigure 1.  mapseq  barseq \n\n(a) mapseq  rna barcodes \n(b)  barcodes egfp sindbis  mitral cells  tufted cells\n(c)  allen \n(d) mitral, tufted, deep cells  laminar positions barcoded cells  barseq  barseq  barcode  6 \n(e)  barseq  (415 neurons, 2 mice) mitral cell layer (mcl)  pdcpmcptc mcl ptc  aon / ot mitral cells \n(f) barseq  mcl  gl  barseq \n\n\n# figure 2. mapseq reveals structured correlations in the distribution of olfactory bulb projections to their targets\n\n\n\nfigure 2. mapseq \n\n(a-c)  mapseq (a)(b) barseq-based classifier  5 (c)\n(d) [ipr]x\n(e) putative mitral cell(pmc)  bonferroni \n(f)  barseq-based classifier  pmc  pmc  city block distance area shuffled\n\n\n# figure 3. olfactory bulb pmc projections tile the anterior-posterior axis of the piriform cortex and are interdependent with co-projections to extra-piriform target regions\n\n\n\nfigure 3.  pmc \n\n(a) pmc  a-p  a-p \n(b)  pmc \n(c) pmc  p(target|pc location)\n\n\n# figure 4. narrowly and broadly projecting pmc tile differentially the anterior-posterior axis of the piriform cortex\n\n\n\nfigure 4.  pmc \n\n(a)  (inverse participation ratio [ipr]) clustersnp  bp putative mitral cells (pmc)watershed clustering \n(b) np  bp \n(c-d) np(c)  bp(d) pmc  a-p np bp  \n(e-f)  apc  ppc  np  apc(e)  ppc(f)  a-p \n(g) np  apc  ppc  a-p \n(h-i)  apc  ppc  bp  apc(h)  ppc(i)  a-p \n(j) bp  apc  ppc  a-p \n\n\n# figure 5. projection patterns of piriform cortex output neurons are systematically organized along the anterior-posterior axis with respect to their extra-piriform targets\n\n\n\nfigure 5. \n\n(a)  a-p  a-p  a-p \n(b)  a-p  a-p \n(c) \n(d) pmc \n(e) \n(f)  a-p \n\n\n# figure 6. matched input and output circuit motifs in the piriform cortex\n\n\n\nfigure 6. \n\n(a)  barcode  a-p \n(b)  aoncoalent  ot  slice\n(c)  a-p y axis pmc  a-p p(target|pc location) (x axis)\n\n\n# figure 7. cartoon schematics of parallel olfactory processing streams structured along the anterior-posterior axis of the piriform cortex according to triadic connectivity motifs\n\n\n\nfigure 7. \n\nobanterior olfactory nucleusaon aon coalent a-p \n\n\n# supplemental figure\n\n\n# figure s1. olfactory bulb barcoded sindbis virus injections and normalization schemes\n\n\n\nfigure s1. \n\n(a) barcoded sindbis virus \n(b)  allen \n(c) yc61 spike-in rnaspike-in rna and b-actin (measured by qpcr)spike-in rna and dissected brain area size \n(d) \n(e) \n\n\n# figure s2. classification of the olfactory bulb output neurons based on their projection patterns using barseq data as template\n\n\n\nfigure s2.  barseq \n\n(a) barseq  mcl  gl \n(b)  roc \n(c) \n(d) mapseq \n(e) \n(f) \n(g)  pmcs 85%\n(h) 85%\n(i) ipr \n(j)  pmc \n(k) pdc  ptc \n\n\n# figure s3. spatial distribution of barcode labeled pmc somata and projection patterns as a function of different locations within the olfactory bulb\n\n\n\nfigure s3.  pmc \n\n(a)  barcode counts a-p \n(b)  pmc \n(c)  pmc \n(d)  pmc  a-p \n(e)  pmc  a-p \n(f)  a-p  pmc  a-p \n\n\n# figure s4. reproducibility across individuals, shuffled controls and downsampling analysis on the projection patterns of the olfactory bulb output neurons\n\n\n\nfigure s4. \n\n\n# figure s5. validation of the spatial organization of pmc projections using cav-cre bulk retrograde labeling\n\n\n\nfigure s5.  cav-cre  pmc \n\n\n# figure s6. spatial organization of the projections of broadly projecting and narrowly projecting cells\n\n\n\nfigure s6. \n\n\n# figure s7. differential brain-wide projections of groups of piriform cortex output neurons organized in a graded manner along the a-p axis\n\n\n\nfigure s7.  a-p \n\n\n# figure s8. validation of piriform cortex output projection patterns along a-p axis via bulk retrograde tracing\n\n\n\nfigure s8.  a-p ',charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Single-neuron projectome of mouse prefrontal cortex",frontmatter:{title:" --- Single-neuron projectome of mouse prefrontal cortex",categories:[""],tags:[""],date:"2022-11-19T15:34:21.000Z",permalink:"/pages/0be7e3/",readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20220331.(2022.03.31%20%E7%A5%9E%E7%BB%8F%E6%89%80fMOST%20Nat%20Neurosci)%20Single-neuron%20projectome%20of%20mouse%20prefrontal%20cortex.html",relativePath:"02./11./02./20220331.(2022.03.31 fMOST Nat Neurosci) Single-neuron projectome of mouse prefrontal cortex.md",key:"v-e6a0d06a",path:"/pages/0be7e3/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:622},{level:2,title:"Introduction",slug:"introduction",normalizedTitle:"introduction",charIndex:1070},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:2416},{level:3,title:"Reconstruction and classification of PFC projection neurons",slug:"reconstruction-and-classification-of-pfc-projection-neurons",normalizedTitle:"reconstruction and classification of pfc projection neurons",charIndex:2428},{level:3,title:"Analysis of axon morphology of PFC projection neurons",slug:"analysis-of-axon-morphology-of-pfc-projection-neurons",normalizedTitle:"analysis of axon morphology of pfc projection neurons",charIndex:5275},{level:3,title:"Single-neuron projectomes of IT neurons",slug:"single-neuron-projectomes-of-it-neurons",normalizedTitle:"single-neuron projectomes of it neurons",charIndex:5822},{level:3,title:"Single-neuron projectomes of PT neurons",slug:"single-neuron-projectomes-of-pt-neurons",normalizedTitle:"single-neuron projectomes of pt neurons",charIndex:11165},{level:3,title:"Single-neuron projectomes of CT neurons",slug:"single-neuron-projectomes-of-ct-neurons",normalizedTitle:"single-neuron projectomes of ct neurons",charIndex:12888},{level:3,title:"Hierarchical ordering and recurrent connectivity within PFC",slug:"hierarchical-ordering-and-recurrent-connectivity-within-pfc",normalizedTitle:"hierarchical ordering and recurrent connectivity within pfc",charIndex:14221},{level:3,title:"Correspondence between transcriptome and projectome",slug:"correspondence-between-transcriptome-and-projectome",normalizedTitle:"correspondence between transcriptome and projectome",charIndex:15846},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:17928},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:18504},{level:3,title:"Fig1. Projectome reconstruction and classification of PFC projection neurons",slug:"fig1-projectome-reconstruction-and-classification-of-pfc-projection-neurons",normalizedTitle:"fig1. projectome reconstruction and classification of pfc projection neurons",charIndex:18515},{level:3,title:"Fig2. Cortical projections of PFC IT neurons",slug:"fig2-cortical-projections-of-pfc-it-neurons",normalizedTitle:"fig2. cortical projections of pfc it neurons",charIndex:19627},{level:3,title:"Fig3. Subcortical projections of PFC IT neurons",slug:"fig3-subcortical-projections-of-pfc-it-neurons",normalizedTitle:"fig3. subcortical projections of pfc it neurons",charIndex:21604},{level:3,title:"Fig4. Single-neuron projectomes of PFC PT neurons",slug:"fig4-single-neuron-projectomes-of-pfc-pt-neurons",normalizedTitle:"fig4. single-neuron projectomes of pfc pt neurons",charIndex:22949},{level:3,title:"Fig5. Single-neuron projectomes of PFC CT neurons",slug:"fig5-single-neuron-projectomes-of-pfc-ct-neurons",normalizedTitle:"fig5. single-neuron projectomes of pfc ct neurons",charIndex:24082},{level:3,title:"Fig6. Modularity and hierarchy of intra-PFC connectivity",slug:"fig6-modularity-and-hierarchy-of-intra-pfc-connectivity",normalizedTitle:"fig6. modularity and hierarchy of intra-pfc connectivity",charIndex:24749},{level:3,title:"Fig7. Association between single-cell transcriptome and single-cell projectome",slug:"fig7-association-between-single-cell-transcriptome-and-single-cell-projectome",normalizedTitle:"fig7. association between single-cell transcriptome and single-cell projectome",charIndex:25867},{level:2,title:"Extended Figure",slug:"extended-figure",normalizedTitle:"extended figure",charIndex:26757},{level:3,title:"Extended Data Fig1. Design and application of FNT and image registration",slug:"extended-data-fig1-design-and-application-of-fnt-and-image-registration",normalizedTitle:"extended data fig1. design and application of fnt and image registration",charIndex:26777},{level:3,title:"Extended Data Fig2. Somata distribution of neurons of each neuron classes and comparison with bulk labeling experiments",slug:"extended-data-fig2-somata-distribution-of-neurons-of-each-neuron-classes-and-comparison-with-bulk-labeling-experiments",normalizedTitle:"extended data fig2. somata distribution of neurons of each neuron classes and comparison with bulk labeling experiments",charIndex:27249},{level:3,title:"Extended Data Fig3. Relationship between soma distance and projectome dissimilarity, consensus neuron model",slug:"extended-data-fig3-relationship-between-soma-distance-and-projectome-dissimilarity-consensus-neuron-model",normalizedTitle:"extended data fig3. relationship between soma distance and projectome dissimilarity, consensus neuron model",charIndex:27619},{level:3,title:"Extended Data Fig4. Comparison of morphological features among 64 projectome subtypes",slug:"extended-data-fig4-comparison-of-morphological-features-among-64-projectome-subtypes",normalizedTitle:"extended data fig4. comparison of morphological features among 64 projectome subtypes",charIndex:28416},{level:3,title:"Extended Data Fig5. Cortical projection of IT neurons",slug:"extended-data-fig5-cortical-projection-of-it-neurons",normalizedTitle:"extended data fig5. cortical projection of it neurons",charIndex:28992},{level:3,title:"Extended Data Fig6. Diversity of projections to lateral and central cortical subnetworks",slug:"extended-data-fig6-diversity-of-projections-to-lateral-and-central-cortical-subnetworks",normalizedTitle:"extended data fig6. diversity of projections to lateral and central cortical subnetworks",charIndex:29805},{level:3,title:"Extended Data Fig7. Striatal subdomains defined by PFC inputs and comparison with previous studies",slug:"extended-data-fig7-striatal-subdomains-defined-by-pfc-inputs-and-comparison-with-previous-studies",normalizedTitle:"extended data fig7. striatal subdomains defined by pfc inputs and comparison with previous studies",charIndex:30725},{level:3,title:"Extended Data Fig8. Mutually exclusive projection of PT neurons in PL/ORB, AId/v, and MOs",slug:"extended-data-fig8-mutually-exclusive-projection-of-pt-neurons-in-pl-orb-aid-v-and-mos",normalizedTitle:"extended data fig8. mutually exclusive projection of pt neurons in pl/orb, aid/v, and mos",charIndex:31406},{level:3,title:"Extended Data Fig9. Association of transcriptome subtypes and projectome subtypes",slug:"extended-data-fig9-association-of-transcriptome-subtypes-and-projectome-subtypes",normalizedTitle:"extended data fig9. association of transcriptome subtypes and projectome subtypes",charIndex:31786}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-022-01041-5" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-022-01041-5<OutboundLink/></a><br>\nCite: Gao L, Liu S, Gou L, et al. Single-neuron projectome of mouse prefrontal cortex. <em>Nat Neurosci</em> 25, 515529 (2022).</p>\n',headersStr:"Abstract Introduction Results Reconstruction and classification of PFC projection neurons Analysis of axon morphology of PFC projection neurons Single-neuron projectomes of IT neurons Single-neuron projectomes of PT neurons Single-neuron projectomes of CT neurons Hierarchical ordering and recurrent connectivity within PFC Correspondence between transcriptome and projectome Discussion Figure Fig1. Projectome reconstruction and classification of PFC projection neurons Fig2. Cortical projections of PFC IT neurons Fig3. Subcortical projections of PFC IT neurons Fig4. Single-neuron projectomes of PFC PT neurons Fig5. Single-neuron projectomes of PFC CT neurons Fig6. Modularity and hierarchy of intra-PFC connectivity Fig7. Association between single-cell transcriptome and single-cell projectome Extended Figure Extended Data Fig1. Design and application of FNT and image registration Extended Data Fig2. Somata distribution of neurons of each neuron classes and comparison with bulk labeling experiments Extended Data Fig3. Relationship between soma distance and projectome dissimilarity, consensus neuron model Extended Data Fig4. Comparison of morphological features among 64 projectome subtypes Extended Data Fig5. Cortical projection of IT neurons Extended Data Fig6. Diversity of projections to lateral and central cortical subnetworks Extended Data Fig7. Striatal subdomains defined by PFC inputs and comparison with previous studies Extended Data Fig8. Mutually exclusive projection of PT neurons in PL/ORB, AId/v, and MOs Extended Data Fig9. Association of transcriptome subtypes and projectome subtypes",content:": \nDOI: https://doi.org/10.1038/s41593-022-01041-5\nCite: Gao L, Liu S, Gou L, et al. Single-neuron projectome of mouse prefrontal cortex. Nat Neurosci 25, 515529 (2022).\n\n\n\n\n\n\n\n 1.  --- Single-neuron projectome of mouse prefrontal cortex\n 2.  --- Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n\n\nJUN YAN()        NINGLONG XU()   HUI GONG()\n                                      \n              \njunyan@ion.ac.cn   xunl@ion.ac.cn     huigong@mail.hust.edu.cn\n\n\n# Abstract\n\nPFCcognitive centerPFC  6,357  PFC  64 - PFC first-order axon collaterals PFC somatargetterminal arbor PFC  PFC  PFC  PFC  PFC \n\n\n# Introduction\n\nneocortexintratelencephalicITpyramidal tractPT(corticothalamicCT) Cre  scRNA-seq  scRNA-seq  mPFC  PFC  PFC  scRNA-seq  TB(terabyte) \n\nPFC  PFC  PFC  PFC  PFC PFC  PFC \n\nFast Neurite Tracer (FNT) TB-sized  FNT PFC  6,357  64  4 -first-order axon collateralssomata PFC primary axoncollateral branch target  target axon terminal arbors target  subdomains intra-PFC  PFC subregions  3  intra-PFC modules  PFC subregions  PFC  PFC \n\n\n# Results\n\n\n# Reconstruction and classification of PFC projection neurons\n\nPFC \n\n PFC fluorescence micro-optical sectioning tomography, fMOST 161 Fig. 1a, Methods, Supplementary Fig. 1 and Supplementary Table 1 (FNT) 16 FNT-slice2cubeFNT-tracerFNT-viewer TB-size Extended Data Fig. 1, Supplementary Methods 1 and 2 and Supplementary Video 1\n\n FNT 6,357  PFC Extended Data Fig. 1dfaxons 783.27 m Allen Brain Atlas (ABA) (Extended Data Fig. 1h,i) PFC prelimbic (PL)infralimbic (ILA)frontal pole (FRP)orbito-frontal (ORB)dorsal and ventral agranular insular (AId/v)anterior cingulate (ACA)secondary motor (MOs) areas (Fig. 1b) Supplementary Table 2 AId/v medio-dorsal nucleus (MD)  PFC  Fig.1c  (https://mouse.braindatacenter.cn) (Supplementary Video 2)\n\n PFC FNT-distFig. 1d and Supplementary Method 2Fig. 1e and Methods ITPTCT IT  L2/3  L6PT  L5CT  L5  L6 (Fig.1e) PFC  (Extended Data Fig. 2a) PFC IT 3,744IT-ipsiIT-contra PT1,528 CT 1,085 bulk labeling  bulk labeling 0.9 bulk labeling Extended Data Fig. 2b ITPT CT  64  subtypes44 for IT, 12 for PT, 8 for CTFig. 1e 64  Supplementary Figs. 24, Supplementary Table 4 and Supplementary Video 3\n\n subtype  laminar layers  PFC subregions Fig. 1e PFC subregions entropy subtype 38 of 64 subtypes  subregion-specific  subregion-biased subtypes subtypes  2-5  PFC subregions  subtypes subtypes22 PFC subregions  homologous subtypesFig. 1e, Supplementary Table 4 and Methods t-SNE constellation plot 64  projection subtypes Fig. 1f,g ITPTCT Extended Data Fig. 3a ACA  AId/v medio-lateral axisFig. 1f ITPTCT  29%45%51%Extended Data Fig. 3a subtypes  subtype (Extended Data Fig. 3b,c)PFC \n\n FNT-merge  subtype Extended Data Fig. 3d, Supplementary Figs. 24 and Supplementary Method 2primary axonfirst-order axon collaterals subtype  subtype  subtype higher-order collaterals (Extended Data Fig. 3e,f)\n\n\n# Analysis of axon morphology of PFC projection neurons\n\nPFC \n\nExtended Data Fig. 4a and Methods ITPTCT  IT  CT PT  ITPT  CT  subtype Extended Data Fig. 4bIT subtype26  IT subtype8 Extended Data Fig. 4b,c power law () (Extended Data Fig. 4d) 64  subtypes Extended Data Fig. 4e IT  CT  PT \n\n\n# Single-neuron projectomes of IT neurons\n\nIT \n\nIT  subtypes  subnetworks  bulk labeling - medial ()lateral ()central ()prefrontal ()  subnetworks  4  subnetworks  PFC IT ipsilateral projections IT  1  IT subtypes  PFC  234  IT subtypes  medial ()lateral ()central () subnetworksFig.2a subnetworks  Fig.1e  subtype group 4Fig.2b PFC  subnetworks  IT subtype  PFC subregions  PFC  IT subtype  PFC subregionsFig. 2cIT-contra neuron subtype  subnetworksExtended Data Fig. 5a,b IT subtypes (such as 20, 27 and 31)  subnetworks subnetworks interactions \n\nAxon collateral segregation subnetworks   subnetworks FNT-dist  IT first-order axon collaterals clusters clusters  subnetworks Fig. 2d and Methods subnetworks  IT  (VIS)  IT  (AUD) (RSP)  (PTLP)  medial subnetwork Fig. 2e IT  lateral and central subnetworks  Extended Data Fig. 5c,d subnetwork  PFC IT  cortical subnetworks \n\n subnetwork  IT   PFC IT  subnetwork  IT subtypes  subnetwork medial subnetwork-projecting neurons  subtypes  subtypes  medial cortical areas Fig.2f,g subtypes  anterior cingulate (ACA)  ventro-lateral orbito-frontal (ORBvl)  subregions laminar layers  L2/3  subtype6  L5  subtypes lateral and central subnetwork-projecting neurons  IT  Extended Data Fig. 6a,b,d,e cortical subnetwork  IT \n\nIT terminal arbors laminar   IT  laminar  IT subtype  laminar distribution IT subtype  laminar distribution Fig. 2hsubtype 31  RSPPTLpAUD  VIS  L1  laminar distribution subtype 6 located in ORBvl terminated in L2/3 and L5; subtype 7 located in ORBvl terminated in L5 and L6; subtype 10 located in ACA terminated in L6; and subtypes 11 and 31 located in ACA terminated in L1IT  lateral and central subnetworks  laminar  Extended Data Fig. 6c,f. IT  IT subtypes  laminar \n\nIT  Subcortical target   cortico-cortical PFC IT  subcortical targets BLACPACBCLAPAL subcortical targets  one-to-one  one-to-many  PFC subregions  laminar layers  IT subtypes  subcortical targets  (Fig. 3a)subtype 19  PL/ILA/ORBm  L2/3 BLA one-to-one subtype 38  PL/ILA/ORBm  L5 one-to-many  subtype 20  AId/v  L6  one-to-one  PFC subregions BLA-ventral striatum-PAL-  PL/ILA/ORBm/AId/v PFC IT  subcortical  cortical layers  PFC subregions \n\n subcortical projection targets  mPFC neuron subtypes   PFC subregion mPFCPL/ILA/ORBm 12  IT neuron subtypesFig. 3b subtypes -(medio-lateral)-(dorso-ventral)subtype 19 BLA L2/3 subtype 21 BLA  PAL subtype 38 PAL L5 (Fig. 3c,d) BLA-projecting  PAL-projecting IT neurons -(medio-lateral) BLA  PAL Fig. 3dPL/ILA/ORBm  BLA-projecting subtypes (1921)  mPFC  L5 IT -(dorso-ventral)subtypes 26 and 40subtype 21(subtype 38)- (Fig. 3e) topographic cortico-striatal map\n\nstriatum IT axon arbors   IT  14004 100100100m3 PFC IT n=2351 9 ventro-medial (STRvm), ventro-lateral (STRvl), dorso-lateral (STRdl), ventro-anterior (STRva), anterior (STRa), dorso-medial (STRdm), dorso-anterior (STRda), latero-posterior (STRlp) and central (STRc) striatum (Fig. 3f,g, Extended Data Fig. 7a and Methods) AId  IT  STRvl ORBm  ILA  IT  STRa ACAd  ORBvl  IT  STRc (Fig. 3g) bulk tracing studies  PFC Extended Data Fig. 7b non-PFC  PFC  non-PFC  44  PFC IT neuron subtypes  14  non-PFC  bulk labeling experiments Extended Data Fig. 7c and Methodsstriatum-projecting PFC IT neuron  striatal subdomains striatal subdomains -cortico-basal ganglia loopSTRdm  ACAd  PFC IT subtype 11Extended Data Fig. 7d PFC  PFC  IT \n\n\n# Single-neuron projectomes of PT neurons\n\nPT \n\nPT  PFC PT  subcortical regions 12  PT neurons subtypes53-64Fig. 4a and Supplementary Fig. 3 PT subtypes  PFC subregions Fig. 4b and Supplementary Fig. 3 PFC PT cortico-cortical projection ACA  MOs  PT neuron subtype 53  RSP  VIS PFC PT Fig. 4a\n\nPT   PT primary axons  first-order axon collaterals  5  clustersFig. 4c and Methods clusters  axon collaterals  primary axons  subcortical targets PT primary axons  VTA  SNc/SNr  collateral clustercluster 4 PT  PFC Fig. 4c) PFC  VTA  SNc  PT neuron subtypes  collateral cluster Fig. 4d neuron subtypes  collateral cluster  first-order collaterals  subtypes \n\nPT   PT  (MY)  (TH) PT  ACA  PT  subtypes 53/55  54/57  SCm  PCGFig. 4e,f) ACA Fig. 4g ACA  SCm-projecting  PCG-projecting Fig. 4h primary axons SCm-projecting  PCG-projecting  primary axons  SCm  PCG Fig. 4i PFC subregions  PT subregions subregions  PL/ORB subtypes  PAG  PCGAId/v subtypes  MY  ACBMOs subtypes  medulla  thalamus Fig. 4j see also Extended Data Fig. 8 PFC subregions \n\n\n# Single-neuron projectomes of CT neurons\n\nCT \n\nCT  cortico-thalamic   8  CT neuron subtypes subtypes Fig. 5a and Supplementary Fig. 4Subtypes 47/49  ventral group of thalamus subtype 48  anterior and lateral groups of thalamus subtypes 51/52  medial and midline groups of thalamus medio-dorsal thalamus (MD)  PFC CT  ORBvl  ORB1  subtype 46 CT  MD  SMT CT  subtypes 45  52 CLA GPe/GPiSIBSTZILHAFig. 5a 8  CT subtypes  PFC Fig. 5bCT  primary axons - (medio-lateral)  PFC  CT - (medio-lateral)  (anterior-posterior) CT  (anterior-posterior) Fig. 5c non-MD-projecting PFC CT neuron subtypenon-thalamic targets of PFC CT neurons  PFC cortico-thalamic \n\n MD   MD  PFC CT  MD  MD subdomainsMDaMDmMDlMDvFig. 5d and MethodsORBm/vl  CT  MDa AId  CT  MDv CT  PT  MD CT  PT Fig. 5e,f MD  PT  CT Fig. 5g PFC  MD \n\n\n# Hierarchical ordering and recurrent connectivity within PFC\n\nPFC \n\n PFC  PFC   PFC  60 cortical units PFC PFC units  intra-PFC Fig. 6a and Methods Louvain  intra-PFC Fig. 6b ABA  subregions module 1  MOs  AId  unitsmodule 2  ORBvlORB1ACAposterior MOs  unitsmodule 3  PLILORBmAIv  unitsFig. 6c PFC modules module \n\nPFC   intra-PFC connectivity  60  PFC units VISAUD and SS (MOp) (GU)  (ENT) Fig. 6d module 1 module 2 module 3 Fig. 6dintra-cortical connectivity \n\nPFC   (FF)  (FB)  laminar  60  cortical units  intra-PFC axon projections  laminar distributionsFig. 6e bulk-labeled  FF  FB 3  FF  3  FB 60  PFC units  units Fig. 6fh and Methods PFC  PFC units Fig. 6g and Methods PFC  (FRP)  units 24  5 units 23  48  (ORB1)Fig.6h\n\n cortical unitsFig. 6h PFC  units 22  56within module 2Fig. 6ij units 22  units 56 Fig. 6ij PFC \n\n\n# Correspondence between transcriptome and projectome\n\n\n\n PFC  scRNA-seq  9 Lypd1 and Penk for L2/3 IT neurons; Rorb, Deptor and Nnat for L5/6 IT neurons; Npnt and Lypd1 for PT neurons; and Nnat and Tpbg for CT neurons (Fig. 7a,b) (smFISH) PLACAdORBMOs Fig. 7c MOs  PL/ORB -medio-lateralLypd1+ L2/3 IT neuronsNnat+ L5/6 IT neuronsLypd1+ PT neuronsNnat+ CT neurons  medial PFC (PL/ORBm)  ventro-lateral (ORBvl)  lateral orbito-frontal (ORBl) PFC \n\n markers  smFISH PL/ORB  ITPT  CT  L5/6 IT  CP  ACBPT  PAG  PCGCT  MD  SMTprojectome subtypes  first-order axons  L5/6 IT  ACB-projecting IT neuronssubtypes 21/38 Nnat  CP-projecting IT neuronssubtypes 26 Deptor Fig. 7d PT  PAG-projecting PT neuronssubtypes 57/62 Lypd1+  PCG-projecting PT neuronssubtypes 61Fig. 7e CT  MD-projecting CT neuronssubtypes 51/52 Nnat+ SMT-projecting CT neuronssubtypes 47/50 Tpbg+Fig. 7f PL/ORB  PFC subregion first-order axons \n\n PL/ORB  PFC subregion  ACA  smFISH SCm  PCG ACA  PT  Npnt  SCm-projecting PT neuronssubtypes 53/55 PCG-projecting PT neuronssubtypes 54/57Extended Data Fig. 9aNpnt  MOs thalamus-projecting PT neurons medulla-projecting PT neurons Npnt+ PT neurons PFC subregions  scRNA-seq  medial PFC  ALM  PFC  scRNA-seq Extended Data Fig. 9be PFC \n\n\n# Discussion\n\n PFC  FNT-dist  intra-module  PFC  scRNA-seq  PFC  PFC  PFC  PFC \n\n\n# Figure\n\n\n# Fig1. Projectome reconstruction and classification of PFC projection neurons\n\n\n\nFig1. PFC \n\na. PFC  fMOST  161  (FNT)  6,357  PFC \nb. PFC  PFC  PFC \nc.  PFC (b)\nd. FNT-dist (d12)  L1  L2 \ne.  PFC 64  subtypes  subtype  C-index C-index  64  subtypes subtypes  laminar distributions subtypes  PFC subregions PFC subregions subtypes  11  PFC subregions \nf.  FNT-dist  tSNE  ITPTCT IT  L2/3  L6PT  L5CT  L5  L6IT 3,744IT-ipsiIT-contra PT1,528 CT 1,085 tSNE  ITPTCT  ACA  AId/v medio-lateral axis\ng. 64  subtypes  t-SNE  subtype  subtype  subtype  subtype \n\n\n# Fig2. Cortical projections of PFC IT neurons\n\n\n\nFig2. PFC IT \n\na.  44  IT subtypes  4 Groups4  prefrontal (), lateral (), central (), medial ()  subnetworkssubtype groups  target subnetworks \nb. Fig.1e  subtypes  target subnetworks \nc. 4  IT subtype groups  PFC  subnetworks  IT subtype  PFC subregions  PFC  IT subtype  PFC subregionsIT-contra neuron subtype  subnetworksExtended Data Fig. 5a,b\nd.  soma, primary axon, first-order axon collateral, terminal arbor first-order axon collaterals  subnetwork  4  clusters collaterals  target subnetworks \ne.  medial subnetwork  Over-represented  under-represented '+'  over-represented pattern  PTLpRSPVISAUD subnetworks  IT  (VIS)  IT  (AUD) (RSP)  (PTLP)  medial subnetwork \nf. 8   medial cortical subnetwork  IT subtypes  subtypes  ()  laminar () \ng. 8   medial cortical subnetwork  IT subtypes  targets  cortical subnetwork  IT \nh. 8  medial subnetwork  IT subtypes axon terminals laminar distribution medial subnetwork  medial subnetwork-projecting subtypes  subtype  subtypes  laminar distribution neuron subtype pair  laminar distribution subtype 31  (RSP, PTLp and AUD)  laminar distribution VIS-projecting IT subtypes (6, 7, 10, 11 and 31)  VIS  laminar distributions IT subtype  laminar distribution IT subtype  laminar distribution \n\n\n# Fig3. Subcortical projections of PFC IT neurons\n\n\n\nFig3. PFC IT \n\na. 44  IT subtypes  (y axis) subtypes  PFC subregions top labels laminar lower labels\nb. medial PFC (PL-ILA-ORBm)  5  IT subtypes \nc. b  5  IT neuron subtypes \nd.  IT subtypes (192138)  mPFC  BLA  PAL Left laminar distribution Middle IT subtypes Right BLA (red)  PAL (green)  ()  BLA  PAL  medio-lateral laminar  (subtype 21)  BLA  PAL\ne. striatum () -(Dorso-ventral)Left PFC IT  PFC  striatum-projecting IT  dorso-lateral position  striatum-projecting IT neurons  first-order branch point (left)  centroid position of their terminal arbors (middle) branch point location  centroid position (right) \nf.  IT Top left (n=100) Top rightBottom\ng.  IT  subdomains  PFC Top left f 9 Right 8  Bottom left  PFC  PFC  IT \n\n\n# Fig4. Single-neuron projectomes of PFC PT neurons\n\n\n\nFig4. PFC PT \n\na. 12  PT subtypes  PFC subregions  12  PT neurons subtypes53-64 PFC PT cortico-cortical projection ACA  MOs  PT neuron subtype 53  RSP  VIS PFC PT \nb. PT subtypes  PFC \nc. PT  first-order collaterals  PT  clusters  first-order collaterals  primary axon  collateral cluster  PT targets\nd.  PT  5  collateral cluster \ne. ACA  PT subtypes \nf.  targets  PCG  SCm \ng. subtypes 53/55 (SCm-projecting)  subtypes 54/57 (PCG-projecting)  (anterior-posterior) \nh. SCm-projecting  PCG-projecting  PT  ACA ACA SCm  PCG \ni. subtypes 53/55 (SCm-projecting)  subtypes 54/57 (PCG-projecting)  SCm  PCG  primary axon  first-order collaterals \nj.  PT neuron subtypes  PFC subregions \n\n\n# Fig5. Single-neuron projectomes of PFC CT neurons\n\n\n\nFig5. PFC CT \n\na. 8  CT subtypes  PFC subregions \nb. CT subtypes  PFC \nc.  lateral-medial axis  PFC CT Top left CT  CT  lateral-medial \nd.  (MD) Top leftMD Bottom left MD subdomains  CT  PT Right MD subdomains  7 \ne. CT PT PFC  MD \nf. CT PT PFC subdomains  MD \ng. CT  PT  MD CT PTMD  CT  PT  \n\n\n# Fig6. Modularity and hierarchy of intra-PFC connectivity\n\n\n\nFig6. PFC \n\na.  PFC  60  cortical units  intra-PFC connectivity\nb.  cortical units  3  modules\nc.  PFC  high intra-PFC connectivity  PFC modules \nd.  PFC  (VISAUDSSMOpGUENT)  Target-based functional map\ne.  intra-PFC connections  laminar distribution  6  laminar patterns laminar \nf.  e  FF  FB intra-PFC connections  Harris et al. \ng.  intra-PFC connections \nh. intra-PFC \ni.  ( unit 22 unit 56) unit 22  ()  unit 56  L1('FB')unit 56  ()  unit 22  L2/3  L5 ('FF') VIS  laminar distributions\nj.  unit 22  unit 56  unit 22  unit 56  unit 22  unit 56 unit 56  unit 22  L2/3  L5  unit 22  unit 56  L1  L5  units 22  units 56 \n\n\n# Fig7. Association between single-cell transcriptome and single-cell projectome\n\n\n\nFig7. \n\na. PFC Yao et al.  scRNA-seq  t-SNE  9  L2/3  IT  L5/6  ITPT  CT \nb.  PFC subregions PL-ILA-ORB, MOs-FRP, AId/v and ACAd/v\nc.  markers \nd.  PL/ORB L5/6 IT  (Nnat+  Deptor+)  (262138)  L5/6 IT  ACB-projecting IT neuronssubtypes 21/38 Nnat  CP-projecting IT neuronssubtypes 26 Deptor \ne. PL/ORB  PT  (Lypd1+ versus Npnt+)  (576162)  PT  PAG-projecting PT neuronssubtypes 57/62 Lypd1+  PCG-projecting PT neuronssubtypes 61\nf. PL/ORB  CT  (Nnat+ vsrsus Tpbg+)  (47505152)  CT  MD-projecting CT neuronssubtypes 51/52 Nnat+ SMT-projecting CT neuronssubtypes 47/50 Tpbg+\n\n\n# Extended Figure\n\n\n# Extended Data Fig1. Design and application of FNT and image registration\n\n\n\nExtended Data Fig1.  FNT \n\na. FNT workflow\nb. FNT tracing \nc.  multiple tracers  Pipeline \nd. \ne. 35  \nf. 600 \ng. \nhi.  Allen Brain Atlas (ABA) \n\n\n# Extended Data Fig2. Somata distribution of neurons of each neuron classes and comparison with bulk labeling experiments\n\n\n\nExtended Data Fig2.  bulk labeling experiments \n\na. L2/3 ITL5 ITPTCTL6 IT  PFC flatmap  11  CCFv3  PFC subregions \nb.  bulk labeling experiments  PFC subregions \n\n\n# Extended Data Fig3. Relationship between soma distance and projectome dissimilarity, consensus neuron model\n\n\n\nExtended Data Fig3. \n\na. \nb.  subtype pair  subtype ITCTPT subtype \nc. neuron subtype  ITPTCT  Moran's I  (IT: subtypes 25 and 12; CT: subtypes 52 and 49; PT: subtypes 63 and 55)  (IT: subtypes 25 and 3; CT: subtypes 52 and 49; PT: subtypes 61 and 53) \nd.  neuron subtype FNT-dist  FNT-merge \nef.  (zero-order)  f  subtype 60 (n=27)\n\n\n# Extended Data Fig4. Comparison of morphological features among 64 projectome subtypes\n\n\n\nExtended Data Fig4.  64 \n\na. ITPT  CT CTIT  PT \nbc. 64  neuron subtype b ,  64  neuron subtype Subtype26  subtype8  c  IT  CT PT \nd. ITPT  CT axon lengthnumber of branch points power law ()ITCT  PT  0.90.3  0.5R2  0.80.4  0.6\ne. axon lengthnumber of branch points\n\n\n# Extended Data Fig5. Cortical projection of IT neurons\n\n\n\nExtended Data Fig5. IT \n\na. 44  IT subtypes Ipsi-contra-lateral cortical projection patterns\nb.  IT subtypes Ipsi-contra-lateral cortical Symmetry Spearman's rank 4IT subtypes 23 and 34  (AI) IT subtypes 7 and 30 subtype 7 subtype 30  (for example, ECT and ENT) IT \nc. Over- and under- represented patterns lateral subnetwork  AIpECTENTPERI  TEa over-represented \nd. Over- and under- represented patterns central subnetwork  MOp, SSp, and SSs over-represented \n\n\n# Extended Data Fig6. Diversity of projections to lateral and central cortical subnetworks\n\n\n\nExtended Data Fig6.  lateral and central cortical subnetworks \n\na. 7  lateral cortical subnetwork  IT subtypes  regional ()  laminar () \nb. IT subtypes  lateral cortical subnetwork  a \nc. 7  IT subtypes  laminar  lateral subnetwork  lateral subnetwork-projecting subtypes \nd.  central cortical subnetwork  8  IT subtypes  regional ()  laminar () \ne. IT subtypes  central cortical subnetwork  d \nf.  central subnetwork  8  IT subtypes  laminar central subnetwork  central subnetwork-projecting subtypes \n\n\n# Extended Data Fig7. Striatal subdomains defined by PFC inputs and comparison with previous studies\n\n\n\nExtended Data Fig7.  PFC \n\na. 9  striatal subdomains \nb.  Hintiryan et al. Left Hintiryan et al.  striatal subdomainsRightHintiryan et al.  9  striatal subdomains  striatal subdomains \nc. 44  IT neuron subtypes  non-PFC  striatal subdomains Left44  IT neuron subtypes  striatal subdomains Right14  bulk labeling experiments  7  non-PFC  striatal subdomains \nd. LeftPL-ILA-ORBm  IT subtype 21 STRa/va  ENTRightACAd  IT subtype 11 STRdm  VIS\n\n\n# Extended Data Fig8. Mutually exclusive projection of PT neurons in PL/ORB, AId/v, and MOs\n\n\n\nExtended Data Fig8. PT  PL/ORBAId/vMOs \n\na. PT subtypes  PL-ILA-ORB  PAG vs. PCG \nb. PL-ORB  PAG  subtypes 57/62  PCG  subtypes 61 3D  PAG  PCG\nc-d. PT subtypes  AId/v \ne-f. PT subtypes  MOs \n\n\n# Extended Data Fig9. Association of transcriptome subtypes and projectome subtypes\n\n\n\nExtended Data Fig9. \n\na. PT  (Lypd1+ vs. Npnt+)  ACA 53545557 Npnt  SCm-projecting PT neuronssubtypes 53/55 PCG-projecting PT neuronssubtypes 54/57\nb.  Lui et al.  vmPFC  Retro-seq  Fig.7a  9 \nc.  b \nd.  Tasic et al.  ALM  Retro-seq  Fig.7a  9 \ne.  d ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-022-01041-5\ncite: gao l, liu s, gou l, et al. single-neuron projectome of mouse prefrontal cortex. nat neurosci 25, 515529 (2022).\n\n\n\n\n\n\n\n 1.  --- single-neuron projectome of mouse prefrontal cortex\n 2.  --- single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n\n\njun yan()        ninglong xu()   hui gong()\n                                      \n              \njunyan@ion.ac.cn   xunl@ion.ac.cn     huigong@mail.hust.edu.cn\n\n\n# abstract\n\npfccognitive centerpfc  6,357  pfc  64 - pfc first-order axon collaterals pfc somatargetterminal arbor pfc  pfc  pfc  pfc  pfc \n\n\n# introduction\n\nneocortexintratelencephalicitpyramidal tractpt(corticothalamicct) cre  scrna-seq  scrna-seq  mpfc  pfc  pfc  scrna-seq  tb(terabyte) \n\npfc  pfc  pfc  pfc  pfc pfc  pfc \n\nfast neurite tracer (fnt) tb-sized  fnt pfc  6,357  64  4 -first-order axon collateralssomata pfc primary axoncollateral branch target  target axon terminal arbors target  subdomains intra-pfc  pfc subregions  3  intra-pfc modules  pfc subregions  pfc  pfc \n\n\n# results\n\n\n# reconstruction and classification of pfc projection neurons\n\npfc \n\n pfc fluorescence micro-optical sectioning tomography, fmost 161 fig. 1a, methods, supplementary fig. 1 and supplementary table 1 (fnt) 16 fnt-slice2cubefnt-tracerfnt-viewer tb-size extended data fig. 1, supplementary methods 1 and 2 and supplementary video 1\n\n fnt 6,357  pfc extended data fig. 1dfaxons 783.27 m allen brain atlas (aba) (extended data fig. 1h,i) pfc prelimbic (pl)infralimbic (ila)frontal pole (frp)orbito-frontal (orb)dorsal and ventral agranular insular (aid/v)anterior cingulate (aca)secondary motor (mos) areas (fig. 1b) supplementary table 2 aid/v medio-dorsal nucleus (md)  pfc  fig.1c  (https://mouse.braindatacenter.cn) (supplementary video 2)\n\n pfc fnt-distfig. 1d and supplementary method 2fig. 1e and methods itptct it  l2/3  l6pt  l5ct  l5  l6 (fig.1e) pfc  (extended data fig. 2a) pfc it 3,744it-ipsiit-contra pt1,528 ct 1,085 bulk labeling  bulk labeling 0.9 bulk labeling extended data fig. 2b itpt ct  64  subtypes44 for it, 12 for pt, 8 for ctfig. 1e 64  supplementary figs. 24, supplementary table 4 and supplementary video 3\n\n subtype  laminar layers  pfc subregions fig. 1e pfc subregions entropy subtype 38 of 64 subtypes  subregion-specific  subregion-biased subtypes subtypes  2-5  pfc subregions  subtypes subtypes22 pfc subregions  homologous subtypesfig. 1e, supplementary table 4 and methods t-sne constellation plot 64  projection subtypes fig. 1f,g itptct extended data fig. 3a aca  aid/v medio-lateral axisfig. 1f itptct  29%45%51%extended data fig. 3a subtypes  subtype (extended data fig. 3b,c)pfc \n\n fnt-merge  subtype extended data fig. 3d, supplementary figs. 24 and supplementary method 2primary axonfirst-order axon collaterals subtype  subtype  subtype higher-order collaterals (extended data fig. 3e,f)\n\n\n# analysis of axon morphology of pfc projection neurons\n\npfc \n\nextended data fig. 4a and methods itptct  it  ct pt  itpt  ct  subtype extended data fig. 4bit subtype26  it subtype8 extended data fig. 4b,c power law () (extended data fig. 4d) 64  subtypes extended data fig. 4e it  ct  pt \n\n\n# single-neuron projectomes of it neurons\n\nit \n\nit  subtypes  subnetworks  bulk labeling - medial ()lateral ()central ()prefrontal ()  subnetworks  4  subnetworks  pfc it ipsilateral projections it  1  it subtypes  pfc  234  it subtypes  medial ()lateral ()central () subnetworksfig.2a subnetworks  fig.1e  subtype group 4fig.2b pfc  subnetworks  it subtype  pfc subregions  pfc  it subtype  pfc subregionsfig. 2cit-contra neuron subtype  subnetworksextended data fig. 5a,b it subtypes (such as 20, 27 and 31)  subnetworks subnetworks interactions \n\naxon collateral segregation subnetworks   subnetworks fnt-dist  it first-order axon collaterals clusters clusters  subnetworks fig. 2d and methods subnetworks  it  (vis)  it  (aud) (rsp)  (ptlp)  medial subnetwork fig. 2e it  lateral and central subnetworks  extended data fig. 5c,d subnetwork  pfc it  cortical subnetworks \n\n subnetwork  it   pfc it  subnetwork  it subtypes  subnetwork medial subnetwork-projecting neurons  subtypes  subtypes  medial cortical areas fig.2f,g subtypes  anterior cingulate (aca)  ventro-lateral orbito-frontal (orbvl)  subregions laminar layers  l2/3  subtype6  l5  subtypes lateral and central subnetwork-projecting neurons  it  extended data fig. 6a,b,d,e cortical subnetwork  it \n\nit terminal arbors laminar   it  laminar  it subtype  laminar distribution it subtype  laminar distribution fig. 2hsubtype 31  rspptlpaud  vis  l1  laminar distribution subtype 6 located in orbvl terminated in l2/3 and l5; subtype 7 located in orbvl terminated in l5 and l6; subtype 10 located in aca terminated in l6; and subtypes 11 and 31 located in aca terminated in l1it  lateral and central subnetworks  laminar  extended data fig. 6c,f. it  it subtypes  laminar \n\nit  subcortical target   cortico-cortical pfc it  subcortical targets blacpacbclapal subcortical targets  one-to-one  one-to-many  pfc subregions  laminar layers  it subtypes  subcortical targets  (fig. 3a)subtype 19  pl/ila/orbm  l2/3 bla one-to-one subtype 38  pl/ila/orbm  l5 one-to-many  subtype 20  aid/v  l6  one-to-one  pfc subregions bla-ventral striatum-pal-  pl/ila/orbm/aid/v pfc it  subcortical  cortical layers  pfc subregions \n\n subcortical projection targets  mpfc neuron subtypes   pfc subregion mpfcpl/ila/orbm 12  it neuron subtypesfig. 3b subtypes -(medio-lateral)-(dorso-ventral)subtype 19 bla l2/3 subtype 21 bla  pal subtype 38 pal l5 (fig. 3c,d) bla-projecting  pal-projecting it neurons -(medio-lateral) bla  pal fig. 3dpl/ila/orbm  bla-projecting subtypes (1921)  mpfc  l5 it -(dorso-ventral)subtypes 26 and 40subtype 21(subtype 38)- (fig. 3e) topographic cortico-striatal map\n\nstriatum it axon arbors   it  14004 100100100m3 pfc it n=2351 9 ventro-medial (strvm), ventro-lateral (strvl), dorso-lateral (strdl), ventro-anterior (strva), anterior (stra), dorso-medial (strdm), dorso-anterior (strda), latero-posterior (strlp) and central (strc) striatum (fig. 3f,g, extended data fig. 7a and methods) aid  it  strvl orbm  ila  it  stra acad  orbvl  it  strc (fig. 3g) bulk tracing studies  pfc extended data fig. 7b non-pfc  pfc  non-pfc  44  pfc it neuron subtypes  14  non-pfc  bulk labeling experiments extended data fig. 7c and methodsstriatum-projecting pfc it neuron  striatal subdomains striatal subdomains -cortico-basal ganglia loopstrdm  acad  pfc it subtype 11extended data fig. 7d pfc  pfc  it \n\n\n# single-neuron projectomes of pt neurons\n\npt \n\npt  pfc pt  subcortical regions 12  pt neurons subtypes53-64fig. 4a and supplementary fig. 3 pt subtypes  pfc subregions fig. 4b and supplementary fig. 3 pfc pt cortico-cortical projection aca  mos  pt neuron subtype 53  rsp  vis pfc pt fig. 4a\n\npt   pt primary axons  first-order axon collaterals  5  clustersfig. 4c and methods clusters  axon collaterals  primary axons  subcortical targets pt primary axons  vta  snc/snr  collateral clustercluster 4 pt  pfc fig. 4c) pfc  vta  snc  pt neuron subtypes  collateral cluster fig. 4d neuron subtypes  collateral cluster  first-order collaterals  subtypes \n\npt   pt  (my)  (th) pt  aca  pt  subtypes 53/55  54/57  scm  pcgfig. 4e,f) aca fig. 4g aca  scm-projecting  pcg-projecting fig. 4h primary axons scm-projecting  pcg-projecting  primary axons  scm  pcg fig. 4i pfc subregions  pt subregions subregions  pl/orb subtypes  pag  pcgaid/v subtypes  my  acbmos subtypes  medulla  thalamus fig. 4j see also extended data fig. 8 pfc subregions \n\n\n# single-neuron projectomes of ct neurons\n\nct \n\nct  cortico-thalamic   8  ct neuron subtypes subtypes fig. 5a and supplementary fig. 4subtypes 47/49  ventral group of thalamus subtype 48  anterior and lateral groups of thalamus subtypes 51/52  medial and midline groups of thalamus medio-dorsal thalamus (md)  pfc ct  orbvl  orb1  subtype 46 ct  md  smt ct  subtypes 45  52 cla gpe/gpisibstzilhafig. 5a 8  ct subtypes  pfc fig. 5bct  primary axons - (medio-lateral)  pfc  ct - (medio-lateral)  (anterior-posterior) ct  (anterior-posterior) fig. 5c non-md-projecting pfc ct neuron subtypenon-thalamic targets of pfc ct neurons  pfc cortico-thalamic \n\n md   md  pfc ct  md  md subdomainsmdamdmmdlmdvfig. 5d and methodsorbm/vl  ct  mda aid  ct  mdv ct  pt  md ct  pt fig. 5e,f md  pt  ct fig. 5g pfc  md \n\n\n# hierarchical ordering and recurrent connectivity within pfc\n\npfc \n\n pfc  pfc   pfc  60 cortical units pfc pfc units  intra-pfc fig. 6a and methods louvain  intra-pfc fig. 6b aba  subregions module 1  mos  aid  unitsmodule 2  orbvlorb1acaposterior mos  unitsmodule 3  plilorbmaiv  unitsfig. 6c pfc modules module \n\npfc   intra-pfc connectivity  60  pfc units visaud and ss (mop) (gu)  (ent) fig. 6d module 1 module 2 module 3 fig. 6dintra-cortical connectivity \n\npfc   (ff)  (fb)  laminar  60  cortical units  intra-pfc axon projections  laminar distributionsfig. 6e bulk-labeled  ff  fb 3  ff  3  fb 60  pfc units  units fig. 6fh and methods pfc  pfc units fig. 6g and methods pfc  (frp)  units 24  5 units 23  48  (orb1)fig.6h\n\n cortical unitsfig. 6h pfc  units 22  56within module 2fig. 6ij units 22  units 56 fig. 6ij pfc \n\n\n# correspondence between transcriptome and projectome\n\n\n\n pfc  scrna-seq  9 lypd1 and penk for l2/3 it neurons; rorb, deptor and nnat for l5/6 it neurons; npnt and lypd1 for pt neurons; and nnat and tpbg for ct neurons (fig. 7a,b) (smfish) placadorbmos fig. 7c mos  pl/orb -medio-laterallypd1+ l2/3 it neuronsnnat+ l5/6 it neuronslypd1+ pt neuronsnnat+ ct neurons  medial pfc (pl/orbm)  ventro-lateral (orbvl)  lateral orbito-frontal (orbl) pfc \n\n markers  smfish pl/orb  itpt  ct  l5/6 it  cp  acbpt  pag  pcgct  md  smtprojectome subtypes  first-order axons  l5/6 it  acb-projecting it neuronssubtypes 21/38 nnat  cp-projecting it neuronssubtypes 26 deptor fig. 7d pt  pag-projecting pt neuronssubtypes 57/62 lypd1+  pcg-projecting pt neuronssubtypes 61fig. 7e ct  md-projecting ct neuronssubtypes 51/52 nnat+ smt-projecting ct neuronssubtypes 47/50 tpbg+fig. 7f pl/orb  pfc subregion first-order axons \n\n pl/orb  pfc subregion  aca  smfish scm  pcg aca  pt  npnt  scm-projecting pt neuronssubtypes 53/55 pcg-projecting pt neuronssubtypes 54/57extended data fig. 9anpnt  mos thalamus-projecting pt neurons medulla-projecting pt neurons npnt+ pt neurons pfc subregions  scrna-seq  medial pfc  alm  pfc  scrna-seq extended data fig. 9be pfc \n\n\n# discussion\n\n pfc  fnt-dist  intra-module  pfc  scrna-seq  pfc  pfc  pfc  pfc \n\n\n# figure\n\n\n# fig1. projectome reconstruction and classification of pfc projection neurons\n\n\n\nfig1. pfc \n\na. pfc  fmost  161  (fnt)  6,357  pfc \nb. pfc  pfc  pfc \nc.  pfc (b)\nd. fnt-dist (d12)  l1  l2 \ne.  pfc 64  subtypes  subtype  c-index c-index  64  subtypes subtypes  laminar distributions subtypes  pfc subregions pfc subregions subtypes  11  pfc subregions \nf.  fnt-dist  tsne  itptct it  l2/3  l6pt  l5ct  l5  l6it 3,744it-ipsiit-contra pt1,528 ct 1,085 tsne  itptct  aca  aid/v medio-lateral axis\ng. 64  subtypes  t-sne  subtype  subtype  subtype  subtype \n\n\n# fig2. cortical projections of pfc it neurons\n\n\n\nfig2. pfc it \n\na.  44  it subtypes  4 groups4  prefrontal (), lateral (), central (), medial ()  subnetworkssubtype groups  target subnetworks \nb. fig.1e  subtypes  target subnetworks \nc. 4  it subtype groups  pfc  subnetworks  it subtype  pfc subregions  pfc  it subtype  pfc subregionsit-contra neuron subtype  subnetworksextended data fig. 5a,b\nd.  soma, primary axon, first-order axon collateral, terminal arbor first-order axon collaterals  subnetwork  4  clusters collaterals  target subnetworks \ne.  medial subnetwork  over-represented  under-represented '+'  over-represented pattern  ptlprspvisaud subnetworks  it  (vis)  it  (aud) (rsp)  (ptlp)  medial subnetwork \nf. 8   medial cortical subnetwork  it subtypes  subtypes  ()  laminar () \ng. 8   medial cortical subnetwork  it subtypes  targets  cortical subnetwork  it \nh. 8  medial subnetwork  it subtypes axon terminals laminar distribution medial subnetwork  medial subnetwork-projecting subtypes  subtype  subtypes  laminar distribution neuron subtype pair  laminar distribution subtype 31  (rsp, ptlp and aud)  laminar distribution vis-projecting it subtypes (6, 7, 10, 11 and 31)  vis  laminar distributions it subtype  laminar distribution it subtype  laminar distribution \n\n\n# fig3. subcortical projections of pfc it neurons\n\n\n\nfig3. pfc it \n\na. 44  it subtypes  (y axis) subtypes  pfc subregions top labels laminar lower labels\nb. medial pfc (pl-ila-orbm)  5  it subtypes \nc. b  5  it neuron subtypes \nd.  it subtypes (192138)  mpfc  bla  pal left laminar distribution middle it subtypes right bla (red)  pal (green)  ()  bla  pal  medio-lateral laminar  (subtype 21)  bla  pal\ne. striatum () -(dorso-ventral)left pfc it  pfc  striatum-projecting it  dorso-lateral position  striatum-projecting it neurons  first-order branch point (left)  centroid position of their terminal arbors (middle) branch point location  centroid position (right) \nf.  it top left (n=100) top rightbottom\ng.  it  subdomains  pfc top left f 9 right 8  bottom left  pfc  pfc  it \n\n\n# fig4. single-neuron projectomes of pfc pt neurons\n\n\n\nfig4. pfc pt \n\na. 12  pt subtypes  pfc subregions  12  pt neurons subtypes53-64 pfc pt cortico-cortical projection aca  mos  pt neuron subtype 53  rsp  vis pfc pt \nb. pt subtypes  pfc \nc. pt  first-order collaterals  pt  clusters  first-order collaterals  primary axon  collateral cluster  pt targets\nd.  pt  5  collateral cluster \ne. aca  pt subtypes \nf.  targets  pcg  scm \ng. subtypes 53/55 (scm-projecting)  subtypes 54/57 (pcg-projecting)  (anterior-posterior) \nh. scm-projecting  pcg-projecting  pt  aca aca scm  pcg \ni. subtypes 53/55 (scm-projecting)  subtypes 54/57 (pcg-projecting)  scm  pcg  primary axon  first-order collaterals \nj.  pt neuron subtypes  pfc subregions \n\n\n# fig5. single-neuron projectomes of pfc ct neurons\n\n\n\nfig5. pfc ct \n\na. 8  ct subtypes  pfc subregions \nb. ct subtypes  pfc \nc.  lateral-medial axis  pfc ct top left ct  ct  lateral-medial \nd.  (md) top leftmd bottom left md subdomains  ct  pt right md subdomains  7 \ne. ct pt pfc  md \nf. ct pt pfc subdomains  md \ng. ct  pt  md ct ptmd  ct  pt  \n\n\n# fig6. modularity and hierarchy of intra-pfc connectivity\n\n\n\nfig6. pfc \n\na.  pfc  60  cortical units  intra-pfc connectivity\nb.  cortical units  3  modules\nc.  pfc  high intra-pfc connectivity  pfc modules \nd.  pfc  (visaudssmopguent)  target-based functional map\ne.  intra-pfc connections  laminar distribution  6  laminar patterns laminar \nf.  e  ff  fb intra-pfc connections  harris et al. \ng.  intra-pfc connections \nh. intra-pfc \ni.  ( unit 22 unit 56) unit 22  ()  unit 56  l1('fb')unit 56  ()  unit 22  l2/3  l5 ('ff') vis  laminar distributions\nj.  unit 22  unit 56  unit 22  unit 56  unit 22  unit 56 unit 56  unit 22  l2/3  l5  unit 22  unit 56  l1  l5  units 22  units 56 \n\n\n# fig7. association between single-cell transcriptome and single-cell projectome\n\n\n\nfig7. \n\na. pfc yao et al.  scrna-seq  t-sne  9  l2/3  it  l5/6  itpt  ct \nb.  pfc subregions pl-ila-orb, mos-frp, aid/v and acad/v\nc.  markers \nd.  pl/orb l5/6 it  (nnat+  deptor+)  (262138)  l5/6 it  acb-projecting it neuronssubtypes 21/38 nnat  cp-projecting it neuronssubtypes 26 deptor \ne. pl/orb  pt  (lypd1+ versus npnt+)  (576162)  pt  pag-projecting pt neuronssubtypes 57/62 lypd1+  pcg-projecting pt neuronssubtypes 61\nf. pl/orb  ct  (nnat+ vsrsus tpbg+)  (47505152)  ct  md-projecting ct neuronssubtypes 51/52 nnat+ smt-projecting ct neuronssubtypes 47/50 tpbg+\n\n\n# extended figure\n\n\n# extended data fig1. design and application of fnt and image registration\n\n\n\nextended data fig1.  fnt \n\na. fnt workflow\nb. fnt tracing \nc.  multiple tracers  pipeline \nd. \ne. 35  \nf. 600 \ng. \nhi.  allen brain atlas (aba) \n\n\n# extended data fig2. somata distribution of neurons of each neuron classes and comparison with bulk labeling experiments\n\n\n\nextended data fig2.  bulk labeling experiments \n\na. l2/3 itl5 itptctl6 it  pfc flatmap  11  ccfv3  pfc subregions \nb.  bulk labeling experiments  pfc subregions \n\n\n# extended data fig3. relationship between soma distance and projectome dissimilarity, consensus neuron model\n\n\n\nextended data fig3. \n\na. \nb.  subtype pair  subtype itctpt subtype \nc. neuron subtype  itptct  moran's i  (it: subtypes 25 and 12; ct: subtypes 52 and 49; pt: subtypes 63 and 55)  (it: subtypes 25 and 3; ct: subtypes 52 and 49; pt: subtypes 61 and 53) \nd.  neuron subtype fnt-dist  fnt-merge \nef.  (zero-order)  f  subtype 60 (n=27)\n\n\n# extended data fig4. comparison of morphological features among 64 projectome subtypes\n\n\n\nextended data fig4.  64 \n\na. itpt  ct ctit  pt \nbc. 64  neuron subtype b ,  64  neuron subtype subtype26  subtype8  c  it  ct pt \nd. itpt  ct axon lengthnumber of branch points power law ()itct  pt  0.90.3  0.5r2  0.80.4  0.6\ne. axon lengthnumber of branch points\n\n\n# extended data fig5. cortical projection of it neurons\n\n\n\nextended data fig5. it \n\na. 44  it subtypes ipsi-contra-lateral cortical projection patterns\nb.  it subtypes ipsi-contra-lateral cortical symmetry spearman's rank 4it subtypes 23 and 34  (ai) it subtypes 7 and 30 subtype 7 subtype 30  (for example, ect and ent) it \nc. over- and under- represented patterns lateral subnetwork  aipectentperi  tea over-represented \nd. over- and under- represented patterns central subnetwork  mop, ssp, and sss over-represented \n\n\n# extended data fig6. diversity of projections to lateral and central cortical subnetworks\n\n\n\nextended data fig6.  lateral and central cortical subnetworks \n\na. 7  lateral cortical subnetwork  it subtypes  regional ()  laminar () \nb. it subtypes  lateral cortical subnetwork  a \nc. 7  it subtypes  laminar  lateral subnetwork  lateral subnetwork-projecting subtypes \nd.  central cortical subnetwork  8  it subtypes  regional ()  laminar () \ne. it subtypes  central cortical subnetwork  d \nf.  central subnetwork  8  it subtypes  laminar central subnetwork  central subnetwork-projecting subtypes \n\n\n# extended data fig7. striatal subdomains defined by pfc inputs and comparison with previous studies\n\n\n\nextended data fig7.  pfc \n\na. 9  striatal subdomains \nb.  hintiryan et al. left hintiryan et al.  striatal subdomainsrighthintiryan et al.  9  striatal subdomains  striatal subdomains \nc. 44  it neuron subtypes  non-pfc  striatal subdomains left44  it neuron subtypes  striatal subdomains right14  bulk labeling experiments  7  non-pfc  striatal subdomains \nd. leftpl-ila-orbm  it subtype 21 stra/va  entrightacad  it subtype 11 strdm  vis\n\n\n# extended data fig8. mutually exclusive projection of pt neurons in pl/orb, aid/v, and mos\n\n\n\nextended data fig8. pt  pl/orbaid/vmos \n\na. pt subtypes  pl-ila-orb  pag vs. pcg \nb. pl-orb  pag  subtypes 57/62  pcg  subtypes 61 3d  pag  pcg\nc-d. pt subtypes  aid/v \ne-f. pt subtypes  mos \n\n\n# extended data fig9. association of transcriptome subtypes and projectome subtypes\n\n\n\nextended data fig9. \n\na. pt  (lypd1+ vs. npnt+)  aca 53545557 npnt  scm-projecting pt neuronssubtypes 53/55 pcg-projecting pt neuronssubtypes 54/57\nb.  lui et al.  vmpfc  retro-seq  fig.7a  9 \nc.  b \nd.  tasic et al.  alm  retro-seq  fig.7a  9 \ne.  d ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex",frontmatter:{title:" --- Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex",date:"2023-06-02T21:28:58.000Z",permalink:"/pages/e528dd/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20230522.(2023.5.22%20%E4%B8%A5%E5%86%9BfMOST%20Nat%20Neurosci)%20Single-neuron%20analysis%20of%20dendrites%20and%20axons%20reveals%20the%20network%20organization%20in%20mouse%20prefrontal%20cortex.html",relativePath:"02./11./02./20230522.(2023.5.22 fMOST Nat Neurosci) Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex.md",key:"v-d1ce449a",path:"/pages/e528dd/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:541},{level:2,title:"Introduction",slug:"introduction",normalizedTitle:"introduction",charIndex:928},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:2223},{level:3,title:"Complete reconstruction of PFC projection neurons",slug:"complete-reconstruction-of-pfc-projection-neurons",normalizedTitle:"complete reconstruction of pfc projection neurons",charIndex:2235},{level:3,title:"Scaling of PFC neuron morphology with cytoarchitecture",slug:"scaling-of-pfc-neuron-morphology-with-cytoarchitecture",normalizedTitle:"scaling of pfc neuron morphology with cytoarchitecture",charIndex:3627},{level:3,title:"Dendritic morphology of typical pyramidal PFC neurons",slug:"dendritic-morphology-of-typical-pyramidal-pfc-neurons",normalizedTitle:"dendritic morphology of typical pyramidal pfc neurons",charIndex:5059},{level:3,title:"Dendritic morphology of atypical pyramidal PFC neurons",slug:"dendritic-morphology-of-atypical-pyramidal-pfc-neurons",normalizedTitle:"dendritic morphology of atypical pyramidal pfc neurons",charIndex:7137},{level:3,title:"Comparison with dendrites from sensorimotor cortices",slug:"comparison-with-dendrites-from-sensorimotor-cortices",normalizedTitle:"comparison with dendrites from sensorimotor cortices",charIndex:10545},{level:3,title:"Relationship between dendrites and local axons",slug:"relationship-between-dendrites-and-local-axons",normalizedTitle:"relationship between dendrites and local axons",charIndex:11362},{level:3,title:"Correspondence between dendrites and long-range axons",slug:"correspondence-between-dendrites-and-long-range-axons",normalizedTitle:"correspondence between dendrites and long-range axons",charIndex:12583},{level:3,title:"PFC connectivity map estimated from dendriteaxon morphology",slug:"pfc-connectivity-map-estimated-from-dendrite-axon-morphology",normalizedTitle:"pfc connectivity map estimated from dendriteaxon morphology",charIndex:14559},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:16379},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:17653},{level:3,title:"Fig1. Reconstruction and basic statistics for dendrites and axons of PFC neurons",slug:"fig1-reconstruction-and-basic-statistics-for-dendrites-and-axons-of-pfc-neurons",normalizedTitle:"fig1. reconstruction and basic statistics for dendrites and axons of pfc neurons",charIndex:17664},{level:3,title:"Fig2. Classification of PFC typical pyramidal projection neurons based on dendritic morphology",slug:"fig2-classification-of-pfc-typical-pyramidal-projection-neurons-based-on-dendritic-morphology",normalizedTitle:"fig2. classification of pfc typical pyramidal projection neurons based on dendritic morphology",charIndex:18354},{level:3,title:"Fig3. Classification of PFC atypical pyramidal projection neurons",slug:"fig3-classification-of-pfc-atypical-pyramidal-projection-neurons",normalizedTitle:"fig3. classification of pfc atypical pyramidal projection neurons",charIndex:19182},{level:3,title:"Fig4. Comparison of dendrite morphology of typical pyramidal neurons from PFC and sensorimotor cortical neurons",slug:"fig4-comparison-of-dendrite-morphology-of-typical-pyramidal-neurons-from-pfc-and-sensorimotor-cortical-neurons",normalizedTitle:"fig4. comparison of dendrite morphology of typical pyramidal neurons from pfc and sensorimotor cortical neurons",charIndex:20425},{level:3,title:"Fig5. Joint analysis of dendrites and local axons",slug:"fig5-joint-analysis-of-dendrites-and-local-axons",normalizedTitle:"fig5. joint analysis of dendrites and local axons",charIndex:20999},{level:3,title:"Fig6. Relationship between dendrites and long-range axons",slug:"fig6-relationship-between-dendrites-and-long-range-axons",normalizedTitle:"fig6. relationship between dendrites and long-range axons",charIndex:21580},{level:3,title:"Fig7. PFC connectivity map predicted from dendriteaxon pairing",slug:"fig7-pfc-connectivity-map-predicted-from-dendrite-axon-pairing",normalizedTitle:"fig7. pfc connectivity map predicted from dendriteaxon pairing",charIndex:22410},{level:2,title:"Extended Figure",slug:"extended-figure",normalizedTitle:"extended figure",charIndex:23141},{level:3,title:"Extended Data Fig1. Analysis of basic dendrite morphology",slug:"extended-data-fig1-analysis-of-basic-dendrite-morphology",normalizedTitle:"extended data fig1. analysis of basic dendrite morphology",charIndex:23161},{level:3,title:"Extended Data Fig2. Description and justification of dendrite classification",slug:"extended-data-fig2-description-and-justification-of-dendrite-classification",normalizedTitle:"extended data fig2. description and justification of dendrite classification",charIndex:23914},{level:3,title:"Extended Data Fig3. Classification and morphological analysis of atypical pyramidal neurons and spiny stellate neurons",slug:"extended-data-fig3-classification-and-morphological-analysis-of-atypical-pyramidal-neurons-and-spiny-stellate-neurons",normalizedTitle:"extended data fig3. classification and morphological analysis of atypical pyramidal neurons and spiny stellate neurons",charIndex:24379},{level:3,title:"Extended Data Fig4. The laminar distribution of local axons and long-range axons for each dendrite subtype",slug:"extended-data-fig4-the-laminar-distribution-of-local-axons-and-long-range-axons-for-each-dendrite-subtype",normalizedTitle:"extended data fig4. the laminar distribution of local axons and long-range axons for each dendrite subtype",charIndex:24999},{level:3,title:"Extended Data Fig5. Relationship between dendrites and long-range axonal projection for PT and CT neurons",slug:"extended-data-fig5-relationship-between-dendrites-and-long-range-axonal-projection-for-pt-and-ct-neurons",normalizedTitle:"extended data fig5. relationship between dendrites and long-range axonal projection for pt and ct neurons",charIndex:25278},{level:3,title:"Extended Data Fig6. Relationship between dendrite and long-range axonal projection for IT neurons",slug:"extended-data-fig6-relationship-between-dendrite-and-long-range-axonal-projection-for-it-neurons",normalizedTitle:"extended data fig6. relationship between dendrite and long-range axonal projection for it neurons",charIndex:25972},{level:3,title:"Extended Data Fig7. Intra-column and homotopic contralateral connectivity",slug:"extended-data-fig7-intra-column-and-homotopic-contralateral-connectivity",normalizedTitle:"extended data fig7. intra-column and homotopic contralateral connectivity",charIndex:26547}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-023-01339-y" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-023-01339-y<OutboundLink/></a><br>\nCite: Gao, L., Liu, S., Wang, Y. et al. Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex. <em>Nat Neurosci</em> (2023).</p>\n',headersStr:"Abstract Introduction Results Complete reconstruction of PFC projection neurons Scaling of PFC neuron morphology with cytoarchitecture Dendritic morphology of typical pyramidal PFC neurons Dendritic morphology of atypical pyramidal PFC neurons Comparison with dendrites from sensorimotor cortices Relationship between dendrites and local axons Correspondence between dendrites and long-range axons PFC connectivity map estimated from dendriteaxon morphology Discussion Figure Fig1. Reconstruction and basic statistics for dendrites and axons of PFC neurons Fig2. Classification of PFC typical pyramidal projection neurons based on dendritic morphology Fig3. Classification of PFC atypical pyramidal projection neurons Fig4. Comparison of dendrite morphology of typical pyramidal neurons from PFC and sensorimotor cortical neurons Fig5. Joint analysis of dendrites and local axons Fig6. Relationship between dendrites and long-range axons Fig7. PFC connectivity map predicted from dendriteaxon pairing Extended Figure Extended Data Fig1. Analysis of basic dendrite morphology Extended Data Fig2. Description and justification of dendrite classification Extended Data Fig3. Classification and morphological analysis of atypical pyramidal neurons and spiny stellate neurons Extended Data Fig4. The laminar distribution of local axons and long-range axons for each dendrite subtype Extended Data Fig5. Relationship between dendrites and long-range axonal projection for PT and CT neurons Extended Data Fig6. Relationship between dendrite and long-range axonal projection for IT neurons Extended Data Fig7. Intra-column and homotopic contralateral connectivity",content:": \nDOI: https://doi.org/10.1038/s41593-023-01339-y\nCite: Gao, L., Liu, S., Wang, Y. et al. Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex. Nat Neurosci (2023).\n\n\n\n\n\n\n\n 1.  --- Single-neuron projectome of mouse prefrontal cortex\n 2.  --- Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n\n\nJUN YAN()\n\n\njunyan@ion.ac.cn\n\n\n# Abstract\n\n (PFC)  2000  PFC  1515 405  24 - PFC  intra-columninter-hemisphericinter-column  PFC \n\n\n# Introduction\n\nDendritesaxons Patch-sequencin -\n\n pyramidal neurons  laminar layerslayer 5B pyramidal neurons  (V1) -\n\n (PFC)  PFC  PFC  2,000  PFC  PFC  PFC  L2 horizontal neurons, L3 vertical neurons, L4-like spiny stellate neurons and L6 atypical pyramidal neurons PFC  PFC \n\n\n# Results\n\n\n# Complete reconstruction of PFC projection neurons\n\n PFC \n\n PFC  6,357  (3D)  fMOST Fig. 1a and Methods 1,920  PFC subregion Allen Brain Atlas  11  subregionsintratelencephalic (IT), pyramidal tract (PT) and corticothalamic (CT) neuronsFig. 1b,c Supplementary Table 1 \n\n ITPTCT  PT PT: 10.72.7mm, meanSD IT IT6.9  2.0 mmCT CT5.31.2mmFig.1d 4  compartments apical tuftapical trunkoblique dendritebasal dendriteMethods basal dendrite  dendritic compartments  basal dendritetuftapical trunk  ITPT  CT Extended Data Fig. 1a PFC subregions ITPTCT  PT CT Fig. 1d IT  PT Extended Data Fig. 1b PFC subregions  PT  IT  CT  PFC subregions Extended Data Fig. 1c laminar  PFC  ITPTCT  apical tuftbasal dendriteapical trunk Extended Data Fig. 1d ITPTCT  laminar layers / PFC subregions \n\n\n# Scaling of PFC neuron morphology with cytoarchitecture\n\n PFC \n\n-cortical thicknessneuron density PFC  PFC  PT  IT  PFC Fig. 1e and Extended Data Fig. 1e PT  IT  dendritic compartments  cable  CT  apical trunk Fig. 1fapical trunk Fig. 1g PTITCT Fig. 1h PFC  IT  PT Fig. 1i and Extended Data Fig. 1f PFC  cortical column  PFC Extended Data Fig. 1g\n\n ITPT  CT  PFC  apical  basal Extended Data Fig. 1h,i IT  CT PT  apical tuftsPT: 0.6760.017, CT: 0.6870.036, IT: 0.7260.013, meanSD basal dendrites PT, 0.7150.023; IT: 0.6820.016; CT, 0.6250.031; meanSDExtended Data Fig. 1h (0.660.72) Extended Data Fig. 1i apical  basal  (0.5)  (2/3) PFC ITPT  CT  PFC \n\n\n# Dendritic morphology of typical pyramidal PFC neurons\n\n PFC \n\n PFC  (typical pyramidal neuronsn = 1,515) (atypical pyramidal neuronsn = 392)  (spiny stellate neuronsn = 13Fig. 2a and Methods PFC  Supplementary Table 2  43 Methods and Supplementary Table 2 apical tuftbasal dendriteapical trunkobliques  24 Fig. 2b,c and Extended Data Fig. 2a) Extended Data Fig. 2a Extended Data Fig. 2b dendrite clusters  L2  L6  laminar layers  laminar distributionsFig. 2d, left dendrite subtypes  PFC subregions  PFC subregions  dendrite subtype 9  ACA Fig. 2d, middle, and Supplementary Table 3\n\n ITPTCT PT  tufts IT  tufts CT  tufts Fig. 2b and Extended Data Fig. 2cIT  PT  CT  basal dendrites CT  IT  PT  apical trunkExtended Data Fig. 2cCT  PT Extended Data Fig. 2dITPTCTFig. 2e-g IT  tufts L2/3  IT  dendrite subtypes 13  15 tufts IT  basal dendrites  basal dendrites  dendrite subtype 14  basal dendrites  dendrite subtype 17 PT  IT  tufts  dendrite subtype 22dendrite subtypes 23 and 24 CT  CT  L4 PFC  CT  L1 truck dendrite subtypes 1 and 2 versus subtypes 3 and 4Extended Data Fig. 2e  dendrite subtype  laminar layersL2/3L5  L6ITPT  CT apical tufts (few, slender or thick)  basal dendrites (small, medium or large) dendrite subtypes Fig. 2h and Supplementary Table 4\n\n\n# Dendritic morphology of atypical pyramidal PFC neurons\n\n PFC \n\n 405  L2/3 neurons without any typical apical dendrite (n = 212)L4-like spiny stellate neurons (n = 13)L6 atypical pyramidal neurons (n = 154)other rare type neurons (n = 26) (Fig. 3a) dendrite subtypesExtended Data Fig. 3a\n\nL2/3 horizontal and vertical neurons  L2  L3  L2/3  212  L2/3 PFC  PFC  L2/3  laminar  apical dendrite L2/3  7  subtypesFig. 3b,c, Extended Data Fig. 3b and Methodssubtypes 1-4 L2/3 Fig. 3cSubtype 1 neurons (n = 58)  apical-like main dendrites L2 Subtype 2 neurons (n = 24)  subtype 3 neurons (n = 12) subtype 4 neurons (n = 59) Extended Data Fig. 3b and Methodssubtypes 5-7 L2/3  L2/3 Fig. 3csubtypes 5 (n = 23)  subtypes 6 (n = 4)  L2/3  L1  L2/3  L2/3 Extended Data Fig. 3c L2/3  PFC \n\n L2/3  L2/3  BLA Fig. 3d L2/3 Fig. 3d PFC  L2/3 L2  L3 \n\nL4-like spiny stellate neurons Spiny stellate neurons  L4  apical dendrites dendritic spinesPFC  L4  L2/3  L5  13  L4 spiny stellate neurons Fig. 3e spiny stellate neurons  PFC subregionsSupplementary Table 4 atypical pyramidal neurons Extended Data Fig. 3dPFC spiny stellate neurons  L2/3 Fig. 3e barrel cortex  L4 spiny stellate neurons  spiny stellate neurons  PFC spiny stellate neurons  L2/3 L4 spiny stellate neurons  PFC  spiny stellate neurons  IT n=11, ITc, contralateral-projecting IT neurons; n=2, ITi, ipsilateral-projecting IT neuronsFig. 3f PFC  L4 L4 spiny stellate neurons  PFC \n\nL6 atypical pyramidal neurons L6 multipolarinvertedhorizontally orientedtangentially oriented PFC  154  L6 atypical pyramidal neurons dendrite subtypesFig. 3gh apical and inverted dendrites  L6  L6 bipolar neurons (n = 26) typical pyramidal and inverted neurons  Sholl  L6 bipolar neurons  L6  80-120m Extended Data Fig. 3e L6 Extended Data Fig. 3f\n\n L6 atypical pyramidal neurons  L6 atypical pyramidal neurons Fig. 3imultipolar neurons  ipsilateral-only cortical axon projections (ITi)horizontal neurons with CT axon projectionsinverted neurons containing contralateral cortical axon projections (ITc)Fig. 3i L6 atypical pyramidal neurons \n\n\n# Comparison with dendrites from sensorimotor cortices\n\n\n\n PFC (n = 1,515)  (n = 49) (n = 122)  (n = 202) Fig. 4a and Methods PFC  ITPT  CT  ITPT  CT Fig. 4b ITPT  CT  apical tuft, apical trunk and basal dendrite  PFC Fig. 4c PFC Fig. 4d,ePFC  apical trunk  PFC  L4PFC IT  PT  basal dendritesPFC PT  apical tufts  PFC  basal  apical  PFC Fig. 4e IT  PT Fig. 4fg\n\n\n# Relationship between dendrites and local axons\n\n\n\n cortical column Fig. 5a and Methods ITPTCT IT CT Fig. 5b PFC subregions  ITPTCT Fig. 5b,c and Extended Data Fig. 4a\n\n IT  PT  CT Fig. 5d and MethodsFig. 5e and MethodsFig. 5f,gL2/3 IT dendrite subtypes 13141624  7 L2/3  L5 butterfly L2/3  L5  L4  laminar layers\n\n IT Methods, Fig. 5h and Extended Data Fig. 4b (MOs)  dendrite subtype 24  L1  (MOp)  (ECT)  L1  laminar layers  subtype 20  MO  L5 MOp  (SS)  L6Fig. 5h,i\n\n\n# Correspondence between dendrites and long-range axons\n\n\n\n ITPTCT Extended Data Fig. 5a ITPTCT Extended Data Fig. 5b and Methods dendrite subtypes  axon subtypes Fig. 6a dendrite subtype 22  IT  PT  axon subtype 53 Fig. 6a  PT  ACA  IT  RSP Extended Data Fig. 6b scRNAseq  RSP  ACA  IT  PT  dendrite subtype 22  IT/PT \n\n PL  ACA  PT   PL  anterior ACA  PCG-projecting neurons  posterior ACA  SCm-projecting neurons  PCG-projecting  SCm-projecting  dendrite subtypes Fig. 6c SCm-projecting PCG-projecting  basal dendritesPCG-projecting  SCm-projecting Fig. 6d PCG  SCm  PL/ACA  PT Fig. 6dg, Supplementary Table 5 and Methods PCG-projecting  SCm-projecting Fig. 6fPCG-projecting  (AP)  SCm-projecting  AP  AP Fig. 6gFig. 6hPL/ACA PT \n\n AI  ORB1/vl  PT Extended Data Fig. 5ce PL/ACA  L6 CT  L5 IT Extended Data Figs. 5fi and 6adaxon subtypes 21  38  IT  dendrite subtype 19  axon subtypes 21  IT  axon subtypes 38  IT Extended Data Fig. 6e,fExtended Data Fig. 6g\n\n\n# PFC connectivity map estimated from dendriteaxon morphology\n\n- PFC \n\n cortical column  PFC  1,929  PFC L2/3 ITL5 ITL5 PT L6 ITL6 CT neurons 21  PFC columns Fig. 7a,b and Methods cortical column inter-hemispheric homotopic columns  PFC  column  intra-column connectivityFig. 7c,d and Extended Data Fig. 7a L2/3 IT  L5 IT  L5 PTL6 IT  L6 CT  cortical column \n\n PFC  PFC  homotopic cortical columnsmiddle, Fig. 7b PFC Extended Data Fig. 7b,eL5 PT Fig. 7f L5 IT  L5 PT  L5 IT  L5 IT  PT  L2/3  L5 IT  L5 PT  apical dendrite Extended Data Fig. 7c,d) PFC  PT apical dendrite  IT Extended Data Fig. 7e\n\n PFC  cortical columns  cortical columns  PFC  clustersFig. 7g,h 3  4 L6 IT  L6 IT 1 L2/3 IT  L5 PT 2 Extended Data Fig. 7f PFC  PFC  PFC Fig. 7h and Methods PFC Extended Data Fig. 7g L5 PT  L6 IT  Fig. 7h  intra-column connectivity  PFC  cortical columns 2, 5 and 36 L5 PT  PFC columns Fig. 7d PFC  PT  PFC  PFC \n\n\n# Discussion\n\n SCm-projecting  PCG-projecting PT neurons  SCm-projecting PCG-projecting ACA/PL PT  SCm  PCG  Parm1+ PT  Npnt  SCm  PCG /\n\n ITPT  CT  PFC L2  BLA  BLA  L2/3 Lypd1+  scRNAseq  PFC \n\n PFC  PFC  PFC  PFC  80%  PFC  PFC \n\n\n# Figure\n\n\n# Fig1. Reconstruction and basic statistics for dendrites and axons of PFC neurons\n\n\n\nFig1. PFC \n\na.  PFC  4  compartments  tufttrunkobliquebasal\nb. PFC \nc. PFC n=1,920 from 113 samples PFC subregions\nd.  ITPTCT  PT PT: 10.72.7mm, meanSD IT IT6.9  2.0 mmCT CT5.31.2mm\ne. LeftPT  IT Right PFC \nf. PT  IT  dendritic compartments  CT  Trunk \ng. Apical trunk  PTITCT \nh. \ni. IT  PT \n\n\n# Fig2. Classification of PFC typical pyramidal projection neurons based on dendritic morphology\n\n\n\nFig2.  PFC \n\na.  PFC  (typical pyramidal neuronsn = 1,515) (atypical pyramidal neuronsn = 392) (spiny stellate neuronsn = 13\nb.  43  typical pyramidal neurons  24 \nc.  24  dendrite subtypes ITPTCT t-SNE \nd.  24  dendrite subtypes  L2  L6  laminar distributions dendrite subtypes  PFC subregions  PFC subregions  subtype 9  ACA \ne. ITPTCT  dendrite subtypes \nf. 24  dendrite subtypes  Tuft lengthPT  tufts IT  tufts \ng. 24  dendrite subtypes  Basal lengthIT  PT  CT  basal\nh. 24  dendrite subtypes \n\n\n# Fig3. Classification of PFC atypical pyramidal projection neurons\n\n\n\nFig3. PFC \n\na. t-SNE  405 \nbc.  L2/3 atypical pyramidal neurons  7  SubtypesHorizontal neuronsSubtypes 1-4 L2/3 Subtype 1 neurons (n = 58)  apical-like main dendrites L2 Subtype 2 neurons (n = 24) subtype 3 neurons (n = 12) subtype 4 neurons (n = 59) Vertical neuronssubtypes 5-7 L2/3  L2/3 subtypes 5 (n = 23)  subtypes 6 (n = 4)  L2/3  L1 \nd. Horizontal neurons  Vertical neurons  L2/3 typical pyramidal neurons Horizontal neurons  BLA  Vertical neurons  L2/3 typical pyramidal neurons\ne.  spiny stellate neurons \nf. ITi  ITc  L4-like spiny stellate neurons L4 spiny stellate neurons  PFC  spiny stellate neurons  IT \ng. L6 atypical pyramidal neurons invertedHorizontalTangentialBipolar Multipolar\nh.  subtype g laminar \ng.  L6 atypical pyramidal neurons \n\n\n# Fig4. Comparison of dendrite morphology of typical pyramidal neurons from PFC and sensorimotor cortical neurons\n\n\n\nFig4. PFC \n\na.  PFCMOpSSVIS \nb. t-SNE  ITPTCT  PFCMOpSSVIS  ITPT  CT  ITPT  CT \nc.  ITPTCT  apical tuft, apical trunk and basal dendrite  PFC \nd. PFC \ne. PFCMOpSSVIS  ITPTCT \nf. ITPTCT \ng. IT PT\n\n\n# Fig5. Joint analysis of dendrites and local axons\n\n\n\nFig5. \n\na.  local long-range\nb.  ITn=1041PTn = 533 CTn = 331 11  PFC subregions IT CT  PFC subregions \nc. \nd.  IT  PT  CT \ne. \nf. dendrite subtypes  local axon subtypes \ng. local axon subtypes  dendrite subtypes \nh.  laminar \ni. dendrite subtypes 24  20 \n\n\n# Fig6. Relationship between dendrites and long-range axons\n\n\n\nFig6. \n\na.  dendrite subtypes  long-range axon subtypes \nb. 8  PT dendrite subtypes  PFC subregions \nc. PCG-projecting  SCm-projecting  dendrite subtypes \nd. TopPL/ACA  PCG-projecting  SCm-projecting MiddleSCm-projecting  PCG-projecting BottomPCG-projecting  SCm-projecting  SCm-projecting PCG-projecting  basal dendritesPCG-projecting  SCm-projecting \ne. PL/ACA  PCG-projecting  SCm-projecting \nf. PCG-projecting  SCm-projecting \ng. PCG-projecting  (AP)  SCm-projecting  AP  AP \nh. \n\n\n# Fig7. PFC connectivity map predicted from dendriteaxon pairing\n\n\n\nFig7. - PFC \n\na. \nb. 21  cortical columnsbottom righttop right Cortical columns  Gao et al.  intra-PFC modules \nc. column  b  intra-column connectivity \nd. intra-column connectivity self-connectionasymmetric connectionsymmetric connections\ne.  b  contralateral homotopic column connectivity  contralateral homotopic PFC column  Inter-hemispheric connectivity strengths\nf. PFC  homotopic contralateral connectivity \ng.  cortical columns \nh.  g \ni. PFC columns \n\n\n# Extended Figure\n\n\n# Extended Data Fig1. Analysis of basic dendrite morphology\n\n\n\nExtended Data Fig1. \n\na.  4  compartments  tufttrunkobliquebasal  basal dendrite  compartments  basaltufttrunk  ITPTCT \nb.  IT  PT \nc.  PT  IT  CT  PFC subregions \nd. ITPTCT soma volumetuft lengthtrunk lengthbasal length \ne. CT  PFC \nf. CT  PFC \ng.  PFC \nh. ITn = 1,049PTn = 537CTn = 334 Apical tufts  Basal dendrites \ni. ITn = 1,049PTn = 537CTn = 334 Apical tufts  Basal dendrites \n\n\n# Extended Data Fig2. Description and justification of dendrite classification\n\n\n\nExtended Data Fig2. \n\na.  70%  1000 \nb.  dendrite subtypes dendrite subtypes dendrite subtypes \nc. ITn = 1,049PTn = 537CTn = 334 tuft lengthtrunk lengthbasal length \nd. 24  dendrite subtypes CT  PT \ne.  dendrite subtype \n\n\n# Extended Data Fig3. Classification and morphological analysis of atypical pyramidal neurons and spiny stellate neurons\n\n\n\nExtended Data Fig3. \n\na.  dendrite subtypes dendrite subtypes dendrite subtypes \nb.  L2/3 atypical pyramidal neurons \nc. L2/3 atypical neuron subtypes  L2/3 typical pyramidal neurons \nd. atypical pyramidal subtypes  spiny stellate neurons \ne. L6 atypical neuron subtypes  Sholl Bipolar neurons  L6 atypical neuron subtypes\nf. L6 atypical pyramidal neuron subtypes \n\n\n# Extended Data Fig4. The laminar distribution of local axons and long-range axons for each dendrite subtype\n\n\n\nExtended Data Fig4.  laminar \n\na. ITPTCT\nb.  dendrite subtype  laminar  IT \n\n\n# Extended Data Fig5. Relationship between dendrites and long-range axonal projection for PT and CT neurons\n\n\n\nExtended Data Fig5. PT  CT \n\na. ITPTCT \nb.  ITPTCT \nc. AI/ORBl/ORBvl  PT MY vs ACB Pearson \nd.  dendrite/axon subtype  MY-projecting  ACB-projecting \ne.  PT dendrite subtypes /\nf.  CT dendrite subtypes  PFC \ng. ACA/PL/MOs  CT  VM  RT  Pearson \nh. / VM-projecting  RT-projecting \ni.  CT dendrite subtypes /\n\n\n# Extended Data Fig6. Relationship between dendrite and long-range axonal projection for IT neurons\n\n\n\nExtended Data Fig6. IT \n\na.  IT dendrite subtypes  PFC \nb. ACA/PL  L5 IT  STR  STR  Pearson \nc.  PL/ACA  IT \nd.  L5 IT dendrite subtypes /\ne. Axon subtype 21/38  t-SNE top t-SNE bottom\nf. PL/ACA  IT \ng.  L4-like spiny stellate neuron  pyramidal neuron\n\n\n# Extended Data Fig7. Intra-column and homotopic contralateral connectivity\n\n\n\nExtended Data Fig7. \n\na.  5 L2/3 ITL5 ITL5 PTL6 ITL6 CT single cortical columns  \nb.  5 L2/3 ITL5 ITL5 PTL6 ITL6 CT homotopic contralateral columns \nc. contralateral homotopic regions  L2/3L5L6 IT \nd. Left PT RightL2/3 IT  L5 IT  apical tuftapical trunkobliquebasal dendrites \ne. Top PFC subregion  PT  laminar distributionBottomPT apical tufts contralateral homotopical regions \nf.  Columnspattern 1: n=22, pattern 2: n=16, pattern 3: n=30, pattern 4: n=43, pattern 5: n=17 inter-column \ng. ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-023-01339-y\ncite: gao, l., liu, s., wang, y. et al. single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex. nat neurosci (2023).\n\n\n\n\n\n\n\n 1.  --- single-neuron projectome of mouse prefrontal cortex\n 2.  --- single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex\n\n\n\njun yan()\n\n\njunyan@ion.ac.cn\n\n\n# abstract\n\n (pfc)  2000  pfc  1515 405  24 - pfc  intra-columninter-hemisphericinter-column  pfc \n\n\n# introduction\n\ndendritesaxons patch-sequencin -\n\n pyramidal neurons  laminar layerslayer 5b pyramidal neurons  (v1) -\n\n (pfc)  pfc  pfc  2,000  pfc  pfc  pfc  l2 horizontal neurons, l3 vertical neurons, l4-like spiny stellate neurons and l6 atypical pyramidal neurons pfc  pfc \n\n\n# results\n\n\n# complete reconstruction of pfc projection neurons\n\n pfc \n\n pfc  6,357  (3d)  fmost fig. 1a and methods 1,920  pfc subregion allen brain atlas  11  subregionsintratelencephalic (it), pyramidal tract (pt) and corticothalamic (ct) neuronsfig. 1b,c supplementary table 1 \n\n itptct  pt pt: 10.72.7mm, meansd it it6.9  2.0 mmct ct5.31.2mmfig.1d 4  compartments apical tuftapical trunkoblique dendritebasal dendritemethods basal dendrite  dendritic compartments  basal dendritetuftapical trunk  itpt  ct extended data fig. 1a pfc subregions itptct  pt ct fig. 1d it  pt extended data fig. 1b pfc subregions  pt  it  ct  pfc subregions extended data fig. 1c laminar  pfc  itptct  apical tuftbasal dendriteapical trunk extended data fig. 1d itptct  laminar layers / pfc subregions \n\n\n# scaling of pfc neuron morphology with cytoarchitecture\n\n pfc \n\n-cortical thicknessneuron density pfc  pfc  pt  it  pfc fig. 1e and extended data fig. 1e pt  it  dendritic compartments  cable  ct  apical trunk fig. 1fapical trunk fig. 1g ptitct fig. 1h pfc  it  pt fig. 1i and extended data fig. 1f pfc  cortical column  pfc extended data fig. 1g\n\n itpt  ct  pfc  apical  basal extended data fig. 1h,i it  ct pt  apical tuftspt: 0.6760.017, ct: 0.6870.036, it: 0.7260.013, meansd basal dendrites pt, 0.7150.023; it: 0.6820.016; ct, 0.6250.031; meansdextended data fig. 1h (0.660.72) extended data fig. 1i apical  basal  (0.5)  (2/3) pfc itpt  ct  pfc \n\n\n# dendritic morphology of typical pyramidal pfc neurons\n\n pfc \n\n pfc  (typical pyramidal neuronsn = 1,515) (atypical pyramidal neuronsn = 392)  (spiny stellate neuronsn = 13fig. 2a and methods pfc  supplementary table 2  43 methods and supplementary table 2 apical tuftbasal dendriteapical trunkobliques  24 fig. 2b,c and extended data fig. 2a) extended data fig. 2a extended data fig. 2b dendrite clusters  l2  l6  laminar layers  laminar distributionsfig. 2d, left dendrite subtypes  pfc subregions  pfc subregions  dendrite subtype 9  aca fig. 2d, middle, and supplementary table 3\n\n itptct pt  tufts it  tufts ct  tufts fig. 2b and extended data fig. 2cit  pt  ct  basal dendrites ct  it  pt  apical trunkextended data fig. 2cct  pt extended data fig. 2ditptctfig. 2e-g it  tufts l2/3  it  dendrite subtypes 13  15 tufts it  basal dendrites  basal dendrites  dendrite subtype 14  basal dendrites  dendrite subtype 17 pt  it  tufts  dendrite subtype 22dendrite subtypes 23 and 24 ct  ct  l4 pfc  ct  l1 truck dendrite subtypes 1 and 2 versus subtypes 3 and 4extended data fig. 2e  dendrite subtype  laminar layersl2/3l5  l6itpt  ct apical tufts (few, slender or thick)  basal dendrites (small, medium or large) dendrite subtypes fig. 2h and supplementary table 4\n\n\n# dendritic morphology of atypical pyramidal pfc neurons\n\n pfc \n\n 405  l2/3 neurons without any typical apical dendrite (n = 212)l4-like spiny stellate neurons (n = 13)l6 atypical pyramidal neurons (n = 154)other rare type neurons (n = 26) (fig. 3a) dendrite subtypesextended data fig. 3a\n\nl2/3 horizontal and vertical neurons  l2  l3  l2/3  212  l2/3 pfc  pfc  l2/3  laminar  apical dendrite l2/3  7  subtypesfig. 3b,c, extended data fig. 3b and methodssubtypes 1-4 l2/3 fig. 3csubtype 1 neurons (n = 58)  apical-like main dendrites l2 subtype 2 neurons (n = 24)  subtype 3 neurons (n = 12) subtype 4 neurons (n = 59) extended data fig. 3b and methodssubtypes 5-7 l2/3  l2/3 fig. 3csubtypes 5 (n = 23)  subtypes 6 (n = 4)  l2/3  l1  l2/3  l2/3 extended data fig. 3c l2/3  pfc \n\n l2/3  l2/3  bla fig. 3d l2/3 fig. 3d pfc  l2/3 l2  l3 \n\nl4-like spiny stellate neurons spiny stellate neurons  l4  apical dendrites dendritic spinespfc  l4  l2/3  l5  13  l4 spiny stellate neurons fig. 3e spiny stellate neurons  pfc subregionssupplementary table 4 atypical pyramidal neurons extended data fig. 3dpfc spiny stellate neurons  l2/3 fig. 3e barrel cortex  l4 spiny stellate neurons  spiny stellate neurons  pfc spiny stellate neurons  l2/3 l4 spiny stellate neurons  pfc  spiny stellate neurons  it n=11, itc, contralateral-projecting it neurons; n=2, iti, ipsilateral-projecting it neuronsfig. 3f pfc  l4 l4 spiny stellate neurons  pfc \n\nl6 atypical pyramidal neurons l6 multipolarinvertedhorizontally orientedtangentially oriented pfc  154  l6 atypical pyramidal neurons dendrite subtypesfig. 3gh apical and inverted dendrites  l6  l6 bipolar neurons (n = 26) typical pyramidal and inverted neurons  sholl  l6 bipolar neurons  l6  80-120m extended data fig. 3e l6 extended data fig. 3f\n\n l6 atypical pyramidal neurons  l6 atypical pyramidal neurons fig. 3imultipolar neurons  ipsilateral-only cortical axon projections (iti)horizontal neurons with ct axon projectionsinverted neurons containing contralateral cortical axon projections (itc)fig. 3i l6 atypical pyramidal neurons \n\n\n# comparison with dendrites from sensorimotor cortices\n\n\n\n pfc (n = 1,515)  (n = 49) (n = 122)  (n = 202) fig. 4a and methods pfc  itpt  ct  itpt  ct fig. 4b itpt  ct  apical tuft, apical trunk and basal dendrite  pfc fig. 4c pfc fig. 4d,epfc  apical trunk  pfc  l4pfc it  pt  basal dendritespfc pt  apical tufts  pfc  basal  apical  pfc fig. 4e it  pt fig. 4fg\n\n\n# relationship between dendrites and local axons\n\n\n\n cortical column fig. 5a and methods itptct it ct fig. 5b pfc subregions  itptct fig. 5b,c and extended data fig. 4a\n\n it  pt  ct fig. 5d and methodsfig. 5e and methodsfig. 5f,gl2/3 it dendrite subtypes 13141624  7 l2/3  l5 butterfly l2/3  l5  l4  laminar layers\n\n it methods, fig. 5h and extended data fig. 4b (mos)  dendrite subtype 24  l1  (mop)  (ect)  l1  laminar layers  subtype 20  mo  l5 mop  (ss)  l6fig. 5h,i\n\n\n# correspondence between dendrites and long-range axons\n\n\n\n itptct extended data fig. 5a itptct extended data fig. 5b and methods dendrite subtypes  axon subtypes fig. 6a dendrite subtype 22  it  pt  axon subtype 53 fig. 6a  pt  aca  it  rsp extended data fig. 6b scrnaseq  rsp  aca  it  pt  dendrite subtype 22  it/pt \n\n pl  aca  pt   pl  anterior aca  pcg-projecting neurons  posterior aca  scm-projecting neurons  pcg-projecting  scm-projecting  dendrite subtypes fig. 6c scm-projecting pcg-projecting  basal dendritespcg-projecting  scm-projecting fig. 6d pcg  scm  pl/aca  pt fig. 6dg, supplementary table 5 and methods pcg-projecting  scm-projecting fig. 6fpcg-projecting  (ap)  scm-projecting  ap  ap fig. 6gfig. 6hpl/aca pt \n\n ai  orb1/vl  pt extended data fig. 5ce pl/aca  l6 ct  l5 it extended data figs. 5fi and 6adaxon subtypes 21  38  it  dendrite subtype 19  axon subtypes 21  it  axon subtypes 38  it extended data fig. 6e,fextended data fig. 6g\n\n\n# pfc connectivity map estimated from dendriteaxon morphology\n\n- pfc \n\n cortical column  pfc  1,929  pfc l2/3 itl5 itl5 pt l6 itl6 ct neurons 21  pfc columns fig. 7a,b and methods cortical column inter-hemispheric homotopic columns  pfc  column  intra-column connectivityfig. 7c,d and extended data fig. 7a l2/3 it  l5 it  l5 ptl6 it  l6 ct  cortical column \n\n pfc  pfc  homotopic cortical columnsmiddle, fig. 7b pfc extended data fig. 7b,el5 pt fig. 7f l5 it  l5 pt  l5 it  l5 it  pt  l2/3  l5 it  l5 pt  apical dendrite extended data fig. 7c,d) pfc  pt apical dendrite  it extended data fig. 7e\n\n pfc  cortical columns  cortical columns  pfc  clustersfig. 7g,h 3  4 l6 it  l6 it 1 l2/3 it  l5 pt 2 extended data fig. 7f pfc  pfc  pfc fig. 7h and methods pfc extended data fig. 7g l5 pt  l6 it  fig. 7h  intra-column connectivity  pfc  cortical columns 2, 5 and 36 l5 pt  pfc columns fig. 7d pfc  pt  pfc  pfc \n\n\n# discussion\n\n scm-projecting  pcg-projecting pt neurons  scm-projecting pcg-projecting aca/pl pt  scm  pcg  parm1+ pt  npnt  scm  pcg /\n\n itpt  ct  pfc l2  bla  bla  l2/3 lypd1+  scrnaseq  pfc \n\n pfc  pfc  pfc  pfc  80%  pfc  pfc \n\n\n# figure\n\n\n# fig1. reconstruction and basic statistics for dendrites and axons of pfc neurons\n\n\n\nfig1. pfc \n\na.  pfc  4  compartments  tufttrunkobliquebasal\nb. pfc \nc. pfc n=1,920 from 113 samples pfc subregions\nd.  itptct  pt pt: 10.72.7mm, meansd it it6.9  2.0 mmct ct5.31.2mm\ne. leftpt  it right pfc \nf. pt  it  dendritic compartments  ct  trunk \ng. apical trunk  ptitct \nh. \ni. it  pt \n\n\n# fig2. classification of pfc typical pyramidal projection neurons based on dendritic morphology\n\n\n\nfig2.  pfc \n\na.  pfc  (typical pyramidal neuronsn = 1,515) (atypical pyramidal neuronsn = 392) (spiny stellate neuronsn = 13\nb.  43  typical pyramidal neurons  24 \nc.  24  dendrite subtypes itptct t-sne \nd.  24  dendrite subtypes  l2  l6  laminar distributions dendrite subtypes  pfc subregions  pfc subregions  subtype 9  aca \ne. itptct  dendrite subtypes \nf. 24  dendrite subtypes  tuft lengthpt  tufts it  tufts \ng. 24  dendrite subtypes  basal lengthit  pt  ct  basal\nh. 24  dendrite subtypes \n\n\n# fig3. classification of pfc atypical pyramidal projection neurons\n\n\n\nfig3. pfc \n\na. t-sne  405 \nbc.  l2/3 atypical pyramidal neurons  7  subtypeshorizontal neuronssubtypes 1-4 l2/3 subtype 1 neurons (n = 58)  apical-like main dendrites l2 subtype 2 neurons (n = 24) subtype 3 neurons (n = 12) subtype 4 neurons (n = 59) vertical neuronssubtypes 5-7 l2/3  l2/3 subtypes 5 (n = 23)  subtypes 6 (n = 4)  l2/3  l1 \nd. horizontal neurons  vertical neurons  l2/3 typical pyramidal neurons horizontal neurons  bla  vertical neurons  l2/3 typical pyramidal neurons\ne.  spiny stellate neurons \nf. iti  itc  l4-like spiny stellate neurons l4 spiny stellate neurons  pfc  spiny stellate neurons  it \ng. l6 atypical pyramidal neurons invertedhorizontaltangentialbipolar multipolar\nh.  subtype g laminar \ng.  l6 atypical pyramidal neurons \n\n\n# fig4. comparison of dendrite morphology of typical pyramidal neurons from pfc and sensorimotor cortical neurons\n\n\n\nfig4. pfc \n\na.  pfcmopssvis \nb. t-sne  itptct  pfcmopssvis  itpt  ct  itpt  ct \nc.  itptct  apical tuft, apical trunk and basal dendrite  pfc \nd. pfc \ne. pfcmopssvis  itptct \nf. itptct \ng. it pt\n\n\n# fig5. joint analysis of dendrites and local axons\n\n\n\nfig5. \n\na.  local long-range\nb.  itn=1041ptn = 533 ctn = 331 11  pfc subregions it ct  pfc subregions \nc. \nd.  it  pt  ct \ne. \nf. dendrite subtypes  local axon subtypes \ng. local axon subtypes  dendrite subtypes \nh.  laminar \ni. dendrite subtypes 24  20 \n\n\n# fig6. relationship between dendrites and long-range axons\n\n\n\nfig6. \n\na.  dendrite subtypes  long-range axon subtypes \nb. 8  pt dendrite subtypes  pfc subregions \nc. pcg-projecting  scm-projecting  dendrite subtypes \nd. toppl/aca  pcg-projecting  scm-projecting middlescm-projecting  pcg-projecting bottompcg-projecting  scm-projecting  scm-projecting pcg-projecting  basal dendritespcg-projecting  scm-projecting \ne. pl/aca  pcg-projecting  scm-projecting \nf. pcg-projecting  scm-projecting \ng. pcg-projecting  (ap)  scm-projecting  ap  ap \nh. \n\n\n# fig7. pfc connectivity map predicted from dendriteaxon pairing\n\n\n\nfig7. - pfc \n\na. \nb. 21  cortical columnsbottom righttop right cortical columns  gao et al.  intra-pfc modules \nc. column  b  intra-column connectivity \nd. intra-column connectivity self-connectionasymmetric connectionsymmetric connections\ne.  b  contralateral homotopic column connectivity  contralateral homotopic pfc column  inter-hemispheric connectivity strengths\nf. pfc  homotopic contralateral connectivity \ng.  cortical columns \nh.  g \ni. pfc columns \n\n\n# extended figure\n\n\n# extended data fig1. analysis of basic dendrite morphology\n\n\n\nextended data fig1. \n\na.  4  compartments  tufttrunkobliquebasal  basal dendrite  compartments  basaltufttrunk  itptct \nb.  it  pt \nc.  pt  it  ct  pfc subregions \nd. itptct soma volumetuft lengthtrunk lengthbasal length \ne. ct  pfc \nf. ct  pfc \ng.  pfc \nh. itn = 1,049ptn = 537ctn = 334 apical tufts  basal dendrites \ni. itn = 1,049ptn = 537ctn = 334 apical tufts  basal dendrites \n\n\n# extended data fig2. description and justification of dendrite classification\n\n\n\nextended data fig2. \n\na.  70%  1000 \nb.  dendrite subtypes dendrite subtypes dendrite subtypes \nc. itn = 1,049ptn = 537ctn = 334 tuft lengthtrunk lengthbasal length \nd. 24  dendrite subtypes ct  pt \ne.  dendrite subtype \n\n\n# extended data fig3. classification and morphological analysis of atypical pyramidal neurons and spiny stellate neurons\n\n\n\nextended data fig3. \n\na.  dendrite subtypes dendrite subtypes dendrite subtypes \nb.  l2/3 atypical pyramidal neurons \nc. l2/3 atypical neuron subtypes  l2/3 typical pyramidal neurons \nd. atypical pyramidal subtypes  spiny stellate neurons \ne. l6 atypical neuron subtypes  sholl bipolar neurons  l6 atypical neuron subtypes\nf. l6 atypical pyramidal neuron subtypes \n\n\n# extended data fig4. the laminar distribution of local axons and long-range axons for each dendrite subtype\n\n\n\nextended data fig4.  laminar \n\na. itptct\nb.  dendrite subtype  laminar  it \n\n\n# extended data fig5. relationship between dendrites and long-range axonal projection for pt and ct neurons\n\n\n\nextended data fig5. pt  ct \n\na. itptct \nb.  itptct \nc. ai/orbl/orbvl  pt my vs acb pearson \nd.  dendrite/axon subtype  my-projecting  acb-projecting \ne.  pt dendrite subtypes /\nf.  ct dendrite subtypes  pfc \ng. aca/pl/mos  ct  vm  rt  pearson \nh. / vm-projecting  rt-projecting \ni.  ct dendrite subtypes /\n\n\n# extended data fig6. relationship between dendrite and long-range axonal projection for it neurons\n\n\n\nextended data fig6. it \n\na.  it dendrite subtypes  pfc \nb. aca/pl  l5 it  str  str  pearson \nc.  pl/aca  it \nd.  l5 it dendrite subtypes /\ne. axon subtype 21/38  t-sne top t-sne bottom\nf. pl/aca  it \ng.  l4-like spiny stellate neuron  pyramidal neuron\n\n\n# extended data fig7. intra-column and homotopic contralateral connectivity\n\n\n\nextended data fig7. \n\na.  5 l2/3 itl5 itl5 ptl6 itl6 ct single cortical columns  \nb.  5 l2/3 itl5 itl5 ptl6 itl6 ct homotopic contralateral columns \nc. contralateral homotopic regions  l2/3l5l6 it \nd. left pt rightl2/3 it  l5 it  apical tuftapical trunkobliquebasal dendrites \ne. top pfc subregion  pt  laminar distributionbottompt apical tufts contralateral homotopical regions \nf.  columnspattern 1: n=22, pattern 2: n=16, pattern 3: n=30, pattern 4: n=43, pattern 5: n=17 inter-column \ng. ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Brain-wide correspondence of neuronal epigenomics and distant projections",frontmatter:{title:" --- Brain-wide correspondence of neuronal epigenomics and distant projections",date:"2023-12-20T10:51:03.000Z",permalink:"/pages/b405a1/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20231213.(2023.12.13%20epi-retro-seq%20Nature)%20Temporal%20controls%20over%20inter-areal%20cortical%20projection%20neuron%20fate%20diversity.html",relativePath:"02./11./02./20231213.(2023.12.13 epi-retro-seq Nature) Temporal controls over inter-areal cortical projection neuron fate diversity.md",key:"v-128dac97",path:"/pages/b405a1/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:554},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:1140},{level:3,title:"Fig. 1 The epigenomic landscape of brain-wide projection neurons",slug:"fig-1-the-epigenomic-landscape-of-brain-wide-projection-neurons",normalizedTitle:"fig. 1 the epigenomic landscape of brain-wide projection neurons",charIndex:1151},{level:3,title:"Fig. 2 Distinguishability of neurons projecting to different targets across the entire brain",slug:"fig-2-distinguishability-of-neurons-projecting-to-different-targets-across-the-entire-brain",normalizedTitle:"fig. 2 distinguishability of neurons projecting to different targets across the entire brain",charIndex:3163},{level:3,title:"Fig. 3 The diversity of cell type, spatial location and gene regulation of hypothalamic projection neurons",slug:"fig-3-the-diversity-of-cell-type-spatial-location-and-gene-regulation-of-hypothalamic-projection-neurons",normalizedTitle:"fig. 3 the diversity of cell type, spatial location and gene regulation of hypothalamic projection neurons",charIndex:4995},{level:3,title:"Fig. 4 The diversity of cell type, spatial location and gene regulation of thalamic projection neurons",slug:"fig-4-the-diversity-of-cell-type-spatial-location-and-gene-regulation-of-thalamic-projection-neurons",normalizedTitle:"fig. 4 the diversity of cell type, spatial location and gene regulation of thalamic projection neurons",charIndex:7418},{level:3,title:"Fig. 5 Neurotransmitter usage in HB, AMY and VTA projection neurons",slug:"fig-5-neurotransmitter-usage-in-hb-amy-and-vta-projection-neurons",normalizedTitle:"fig. 5 neurotransmitter usage in hb, amy and vta projection neurons",charIndex:8101}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-023-06823-w" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-023-06823-w<OutboundLink/></a><br>\nCite: Bhattacherjee, A., Zhang, C., Watson, B.R. et al. Spatial transcriptomics reveals the distinct\norganization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. <em>Nat Neurosci</em> (2023).</p>\n',headersStr:"Abstract Figure Fig. 1 The epigenomic landscape of brain-wide projection neurons Fig. 2 Distinguishability of neurons projecting to different targets across the entire brain Fig. 3 The diversity of cell type, spatial location and gene regulation of hypothalamic projection neurons Fig. 4 The diversity of cell type, spatial location and gene regulation of thalamic projection neurons Fig. 5 Neurotransmitter usage in HB, AMY and VTA projection neurons",content:": \nDOI: https://doi.org/10.1038/s41586-023-06823-w\nCite: Bhattacherjee, A., Zhang, C., Watson, B.R. et al. Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. Nat Neurosci (2023).\n\n\n\n\n\n\n\nEDWARD CALLAWAY                      JOSEPH ECKER\n                                     \nUniversity of California San Diego   The Salk Institute for Biological Studies\ncallaway@salk.edu                    ecker@salk.edu\n\n\n# Abstract\n\n 'cell-type' clusters clusters  epi-repro-seq  33,034  32  24 225 source-to-target combinations 926  source  target  consensus clusters  clusters  source \n\n\n# Figure\n\n\n# Fig. 1 The epigenomic landscape of brain-wide projection neurons\n\n\n\nFig. 1 \n\na.  epi-retro-seq Cre-recombinase  AAV  (AAV-retro-Cre)  Cre-dependent, nuclear-GFP-expressing  (INTACT-Cre)  24  4  (two male and two female) isocortex (CTX), hippocampal formation, olfactory areas, amygdala (AMY), cerebral nuclei (CNU), interbrain (IB), midbrain (MB), hindbrain (HB) and cerebellum2 weeks  32  source regions target  target  sources  source  GFP+ NeuN+  snmC-seq \nb.  epi-retro-seq (blue dots)  32  source regions (rows)  225  source-target combinations  24  targets (columns)\nc.  epi-retro-seq  225  source-target combinations  33,034  single-nucleus methylomes map  276,187  unbiased snmC-seq  t-SNEunbiased snmC-seq epi-retro-seq  subclass (top)source regions (middle)projection targets (bottom) \nd.  source regions   targets  9  targets  AMY  DNA  targets  AUROC \ne.  epi-retro-seq n=35,743unbias snmC-seq n=266,740 87  260  scRNA-seq n=2,434,472 t-SNE  subclass \nf.  AMY  (prop.) target  z-score \ng.  source  source  MERFISH  MERFISH  subclasses \nh.  RNA DMRsCREsTFs binding motifs  DMRs\ni.  motif  DNA  TFsDMRs GRNsGRNs \n\n\n# Fig. 2 Distinguishability of neurons projecting to different targets across the entire brain\n\n\n\nFig. 2 \n\na.  (source)  ET  IT targets  10  region groups  IT  ET  Joint t-SNE (CTX)IT  ET  cluster RHPTH IT  ET IT  ET  IT  ET  AUROC \nb. 22  (source)  IT  ET  AUROC \nc.  (source)  ET  IT  22  (source)  1  logistic  (source)  ET  IT  (isocortical)  (isocortical) ENT and PIR ET  IT \nd.  (source)  ET  IT DMGs 2,919  c  AUROC  DMGs  mCH  (isocortical)  (archicortical)MOB  AON TH  MB  fold changesMOB  AON AMYTH  MRN \ne.  target groups  ET  IT targets ITCTXMOBCNUETIBMBHB target groups pairs  ET  IT  target groups pairs  ET  IT  CNU  IB  ET  IT \nf.  AUROC (neurotransmitter receptors)(neuropeptides and receptors)(ion channels)(TFs)(neuron projection development) 5  19  666  AUROC  target  628  AUROC  0.88 9906 AUROC 0.91GO ET  IT \n\n\n# Fig. 3 The diversity of cell type, spatial location and gene regulation of hypothalamic projection neurons\n\n\n\nFig. 3 \n\nDNAhypothalamic, HY\na.  epi-retro-seqn=1,572unbiased snmC-seqn=11,554scRNA-seqn=148,840hypothalamic, HY 94  17 \nb-d.  17  10  targets HY-to-STR  cluster 76  HY-to-AMY  cluster 64  HY HY-to-P  HY-to-MY  HY-to-P  cluster 76 HY-to-PFC  HY-to-MOB  cluster 50  cluster 29  HY-to-MOB  cluster 17 HY  targets  targets \ne-f.  HY  MERFISH  17  HY cluster 0376  HY  targets clusters  HY  clusters  HYHY-to-TH  cluster 12323  HY HY-to-HB  cluster  HY  HY  HY  MERFISH \ng-i.  epi-retro-sequnbiased snmC-seqscRNA-seq  17  clusters  1163 DEGs cluster  DEGs clusters  target DEGs  mCH  clusters  DEGs  148,897  DMRsDEG-associated DMRs  mCG  gene-body mCH \nj-k.  DEGs  CREs  binding motifs  TFs HY clusters  GRN 389  TFs46,075  DMRs  8,184  clusters Zic1  motif  cluster 17  39  hypo-CG-methylated DMRshypo-DMRs cluster  cluster  HY  MOB  cluster  TF-encoding gene Zic4  ZIC1  15  DMRs mCG  ZIC1  Zic4  clusters  TFs  HY-to-TH cluster  TFs  HY-to-P  HY-to-MY GRNs  TFs  DMRs  HY \n\n\n# Fig. 4 The diversity of cell type, spatial location and gene regulation of thalamic projection neurons\n\n\n\nFig. 4 \n\nTH  HY TH TH LGd GABAergic TH  GABAergic  HY TH  TH  HY  TH  HY  12  epi-retro-seq  unbiased snmC-seq  scRNA-seq  58  33  epi-retro-seq  TH  HY \n\n\n# Fig. 5 Neurotransmitter usage in HB, AMY and VTA projection neurons\n\n\n\nFig. 5 HBAMY  VTA \n\n 12  9  Slc17a7 (Vglut1), Slc17a6 (Vglut2) and Slc17a8 (Vglut3) for glutamatergic neuronsSlc32a1 (Vgat) for GABAergic neuronsSlc6a2 (Net) for noradrenergic neuronsSlc6a3 (Dat) for dopaminergic neuronsSlc6a4 (Sert) for serotonergic neuronsSlc6a5 (Glyt2) for glycinergic neuronsSlc18a3 (Vacht) for cholinergic neuronsHdc for histaminergic neurons\na.  11  HB  P  MY  5  targets --- THHYSCcerebellar nucleicerebellar cortex (CBX) P  MY 128  HB clusters  20 \nb.  20  6  MY-to-CBX  cluster 76 Slc17a6 Slc17a7 VGLUT1 Slc17a6 VGLUT2 cluster 0  P-to-CBX clusters 1011  27  P-to-HY cluster 10  GABAergiccluster 11  glutamatergiccluster 27  serotonergic Slc6a4 SERT Slc17a8 VGLUT3\nc.  MERFISH  clusters  clusters 0761027 HB \nd.  P  MY  CBX  P  MY  CBX  gene-body DNA  P-to-CBX cluster(0)  P clusters  MY-to-CBX cluster(76)  MY cluster  top 100 genes  5 Slit3Phactr3Pcbp3Atp10aCdk14 5  mossy fibres  climbing fibres  CBX MY  P / cerebellar Purkinje cells  granule cells \ne.  CBX  DEGs  HB  223,839  CBX  hypo-DMRs DMRs  P-to-CBX  MY-to-CBX CBX  P  MY \nf.  AMY  9  targetsPFCENTHIPMOBSTRTHVTAPMY  16  AMY clusters  clusters  IT targets  ET targets \ng.  IT  clusters  Slc17a7 / Slc17a6 ET  clusters  Slc17a6  clusters  Slc32a1  GABAergic  clusters AMY \nh-i.  VTA  targets  marker genes  mCH  tyrosine hydroxylase (Th) for dopaminergic neurons Gad2 for GABAergic neuronsSlc17a6 for glutamatergic neurons VTA  Th  mCH VTA-to-STR  Th CTX  Slc17a6  mCH  Slc17a6  CTX  VTA  Th+ and Slc17a6+ VTA-to-CTX VTA-to-STR Th+Slc17a6  ThSlc17a6+ Th+  Slc17a6+ VTA ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-023-06823-w\ncite: bhattacherjee, a., zhang, c., watson, b.r. et al. spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. nat neurosci (2023).\n\n\n\n\n\n\n\nedward callaway                      joseph ecker\n                                     \nuniversity of california san diego   the salk institute for biological studies\ncallaway@salk.edu                    ecker@salk.edu\n\n\n# abstract\n\n 'cell-type' clusters clusters  epi-repro-seq  33,034  32  24 225 source-to-target combinations 926  source  target  consensus clusters  clusters  source \n\n\n# figure\n\n\n# fig. 1 the epigenomic landscape of brain-wide projection neurons\n\n\n\nfig. 1 \n\na.  epi-retro-seq cre-recombinase  aav  (aav-retro-cre)  cre-dependent, nuclear-gfp-expressing  (intact-cre)  24  4  (two male and two female) isocortex (ctx), hippocampal formation, olfactory areas, amygdala (amy), cerebral nuclei (cnu), interbrain (ib), midbrain (mb), hindbrain (hb) and cerebellum2 weeks  32  source regions target  target  sources  source  gfp+ neun+  snmc-seq \nb.  epi-retro-seq (blue dots)  32  source regions (rows)  225  source-target combinations  24  targets (columns)\nc.  epi-retro-seq  225  source-target combinations  33,034  single-nucleus methylomes map  276,187  unbiased snmc-seq  t-sneunbiased snmc-seq epi-retro-seq  subclass (top)source regions (middle)projection targets (bottom) \nd.  source regions   targets  9  targets  amy  dna  targets  auroc \ne.  epi-retro-seq n=35,743unbias snmc-seq n=266,740 87  260  scrna-seq n=2,434,472 t-sne  subclass \nf.  amy  (prop.) target  z-score \ng.  source  source  merfish  merfish  subclasses \nh.  rna dmrscrestfs binding motifs  dmrs\ni.  motif  dna  tfsdmrs grnsgrns \n\n\n# fig. 2 distinguishability of neurons projecting to different targets across the entire brain\n\n\n\nfig. 2 \n\na.  (source)  et  it targets  10  region groups  it  et  joint t-sne (ctx)it  et  cluster rhpth it  et it  et  it  et  auroc \nb. 22  (source)  it  et  auroc \nc.  (source)  et  it  22  (source)  1  logistic  (source)  et  it  (isocortical)  (isocortical) ent and pir et  it \nd.  (source)  et  it dmgs 2,919  c  auroc  dmgs  mch  (isocortical)  (archicortical)mob  aon th  mb  fold changesmob  aon amyth  mrn \ne.  target groups  et  it targets itctxmobcnuetibmbhb target groups pairs  et  it  target groups pairs  et  it  cnu  ib  et  it \nf.  auroc (neurotransmitter receptors)(neuropeptides and receptors)(ion channels)(tfs)(neuron projection development) 5  19  666  auroc  target  628  auroc  0.88 9906 auroc 0.91go et  it \n\n\n# fig. 3 the diversity of cell type, spatial location and gene regulation of hypothalamic projection neurons\n\n\n\nfig. 3 \n\ndnahypothalamic, hy\na.  epi-retro-seqn=1,572unbiased snmc-seqn=11,554scrna-seqn=148,840hypothalamic, hy 94  17 \nb-d.  17  10  targets hy-to-str  cluster 76  hy-to-amy  cluster 64  hy hy-to-p  hy-to-my  hy-to-p  cluster 76 hy-to-pfc  hy-to-mob  cluster 50  cluster 29  hy-to-mob  cluster 17 hy  targets  targets \ne-f.  hy  merfish  17  hy cluster 0376  hy  targets clusters  hy  clusters  hyhy-to-th  cluster 12323  hy hy-to-hb  cluster  hy  hy  hy  merfish \ng-i.  epi-retro-sequnbiased snmc-seqscrna-seq  17  clusters  1163 degs cluster  degs clusters  target degs  mch  clusters  degs  148,897  dmrsdeg-associated dmrs  mcg  gene-body mch \nj-k.  degs  cres  binding motifs  tfs hy clusters  grn 389  tfs46,075  dmrs  8,184  clusters zic1  motif  cluster 17  39  hypo-cg-methylated dmrshypo-dmrs cluster  cluster  hy  mob  cluster  tf-encoding gene zic4  zic1  15  dmrs mcg  zic1  zic4  clusters  tfs  hy-to-th cluster  tfs  hy-to-p  hy-to-my grns  tfs  dmrs  hy \n\n\n# fig. 4 the diversity of cell type, spatial location and gene regulation of thalamic projection neurons\n\n\n\nfig. 4 \n\nth  hy th th lgd gabaergic th  gabaergic  hy th  th  hy  th  hy  12  epi-retro-seq  unbiased snmc-seq  scrna-seq  58  33  epi-retro-seq  th  hy \n\n\n# fig. 5 neurotransmitter usage in hb, amy and vta projection neurons\n\n\n\nfig. 5 hbamy  vta \n\n 12  9  slc17a7 (vglut1), slc17a6 (vglut2) and slc17a8 (vglut3) for glutamatergic neuronsslc32a1 (vgat) for gabaergic neuronsslc6a2 (net) for noradrenergic neuronsslc6a3 (dat) for dopaminergic neuronsslc6a4 (sert) for serotonergic neuronsslc6a5 (glyt2) for glycinergic neuronsslc18a3 (vacht) for cholinergic neuronshdc for histaminergic neurons\na.  11  hb  p  my  5  targets --- thhysccerebellar nucleicerebellar cortex (cbx) p  my 128  hb clusters  20 \nb.  20  6  my-to-cbx  cluster 76 slc17a6 slc17a7 vglut1 slc17a6 vglut2 cluster 0  p-to-cbx clusters 1011  27  p-to-hy cluster 10  gabaergiccluster 11  glutamatergiccluster 27  serotonergic slc6a4 sert slc17a8 vglut3\nc.  merfish  clusters  clusters 0761027 hb \nd.  p  my  cbx  p  my  cbx  gene-body dna  p-to-cbx cluster(0)  p clusters  my-to-cbx cluster(76)  my cluster  top 100 genes  5 slit3phactr3pcbp3atp10acdk14 5  mossy fibres  climbing fibres  cbx my  p / cerebellar purkinje cells  granule cells \ne.  cbx  degs  hb  223,839  cbx  hypo-dmrs dmrs  p-to-cbx  my-to-cbx cbx  p  my \nf.  amy  9  targetspfcenthipmobstrthvtapmy  16  amy clusters  clusters  it targets  et targets \ng.  it  clusters  slc17a7 / slc17a6 et  clusters  slc17a6  clusters  slc32a1  gabaergic  clusters amy \nh-i.  vta  targets  marker genes  mch  tyrosine hydroxylase (th) for dopaminergic neurons gad2 for gabaergic neuronsslc17a6 for glutamatergic neurons vta  th  mch vta-to-str  th ctx  slc17a6  mch  slc17a6  ctx  vta  th+ and slc17a6+ vta-to-ctx vta-to-str th+slc17a6  thslc17a6+ th+  slc17a6+ vta ",charsets:{cjk:!0},lastUpdated:"2024/02/19, 20:54:06",lastUpdatedTimestamp:1708347246e3},{title:" --- Whole-brain spatial organization of hippocampal single-neuron projectomes",frontmatter:{title:" --- Whole-brain spatial organization of hippocampal single-neuron projectomes",date:"2024-02-19T10:55:46.000Z",permalink:"/pages/f20dad/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20240202.(2024.02.02%20%E5%BE%90%E6%98%A5fmost%E6%B5%B7%E9%A9%AC%E4%BD%93%20Science)%20Whole-brain%20spatial%20organization%20of%20hippocampal%20single-neuron%20projectomes.html",relativePath:"02./11./02./20240202.(2024.02.02 fmost Science) Whole-brain spatial organization of hippocampal single-neuron projectomes.md",key:"v-722b1d75",path:"/pages/f20dad/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:752},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:1062},{level:3,title:"Fig. 1 Projectome subtype classifications of HIP neurons",slug:"fig-1-projectome-subtype-classifications-of-hip-neurons",normalizedTitle:"fig. 1 projectome subtype classifications of hip neurons",charIndex:1073},{level:3,title:"Fig. 2 Projection targetdependent soma locations of HIP neurons",slug:"fig-2-projection-target-dependent-soma-locations-of-hip-neurons",normalizedTitle:"fig. 2 projection targetdependent soma locations of hip neurons",charIndex:2102},{level:3,title:"Fig. 3 Spatial correlation between transcriptome and projectome subtypes",slug:"fig-3-spatial-correlation-between-transcriptome-and-projectome-subtypes",normalizedTitle:"fig. 3 spatial correlation between transcriptome and projectome subtypes",charIndex:3280},{level:3,title:"Fig. 4 Bihemispheric projecting neurons in HIP",slug:"fig-4-bihemispheric-projecting-neurons-in-hip",normalizedTitle:"fig. 4 bihemispheric projecting neurons in hip",charIndex:4513},{level:3,title:"Fig. 5 Projection profiles of CA1, SUB, and SUBr neurons",slug:"fig-5-projection-profiles-of-ca1-sub-and-subr-neurons",normalizedTitle:"fig. 5 projection profiles of ca1, sub, and subr neurons",charIndex:5956},{level:3,title:"Fig. 6 Longitudinal spread of Schaffer collaterals and mossy fiber axons",slug:"fig-6-longitudinal-spread-of-schaffer-collaterals-and-mossy-fiber-axons",normalizedTitle:"fig. 6 longitudinal spread of schaffer collaterals and mossy fiber axons",charIndex:7448},{level:3,title:"Fig. 7 Dependence of axonal arbor distribution on the transverse location of soma",slug:"fig-7-dependence-of-axonal-arbor-distribution-on-the-transverse-location-of-soma",normalizedTitle:"fig. 7 dependence of axonal arbor distribution on the transverse location of soma",charIndex:8522},{level:3,title:"Fig. 8 Dependence of axonal arbor distribution on the longitudinal location of soma",slug:"fig-8-dependence-of-axonal-arbor-distribution-on-the-longitudinal-location-of-soma",normalizedTitle:"fig. 8 dependence of axonal arbor distribution on the longitudinal location of soma",charIndex:9660},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:10399}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1126/science.adj9198" target="_blank" rel="noopener noreferrer">https://doi.org/10.1126/science.adj9198<OutboundLink/></a><br>\nCite: Shou Qiu et al. ,Whole-brain spatial organization of hippocampal single-neuron projectomes.Science383,eadj9198(2024).</p>\n',headersStr:"Abstract Figure Fig. 1 Projectome subtype classifications of HIP neurons Fig. 2 Projection targetdependent soma locations of HIP neurons Fig. 3 Spatial correlation between transcriptome and projectome subtypes Fig. 4 Bihemispheric projecting neurons in HIP Fig. 5 Projection profiles of CA1, SUB, and SUBr neurons Fig. 6 Longitudinal spread of Schaffer collaterals and mossy fiber axons Fig. 7 Dependence of axonal arbor distribution on the transverse location of soma Fig. 8 Dependence of axonal arbor distribution on the longitudinal location of soma Discussion",content:": \nDOI: https://doi.org/10.1126/science.adj9198\nCite: Shou Qiu et al. ,Whole-brain spatial organization of hippocampal single-neuron projectomes.Science383,eadj9198(2024).\n\n\n\n\n\n\n\n                                                                                                         \n                                                                                                                      \n                                    \nchun.xu@ion.ac.cn     yangang.sun@ion.ac.cn   huigong@mail.hust.edu.cn   haishanyao@ion.ac.cn   tonylicy@ion.ac.cn    ydsun@ion.ac.cn\n\n\n# Abstract\n\nHIP 10,100  HIP  43  HIP  HIP  HIP HPF HIP  HPF  HPF  HIP  HIP \n\n\n# Figure\n\n\n# Fig. 1 Projectome subtype classifications of HIP neurons\n\n\n\nFig. 1 HIP \n\n(A)  EGFP  HIP 100 um20 um\n(B)  HIP  HIP  HIP DVAP HIP  HIP \n(C) \n(D)  HIP  7  HIP DGCA3CA2CA1SUBProSSUBr D-V  9  HPFISOOLFCTXspCNUTHHYMB table S3 \n(E)  HIP Ward's method 13 RCD HIP  D-V \n(F-H) group 1  subtype 3Fgroup 2  subtype 26Ggroup 3  subtype 37H\n\n\n# Fig. 2 Projection targetdependent soma locations of HIP neurons\n\n\n\nFig. 2 HIP \n\n(A)  50  extra-HPF  intra-HPFc  intra-HPFc \n(B)  HIP  LEC  MEC \n(C)  LEC-MEC  bins  LEC  MEC sel.LEC  MEC  HIP CA1SUBProSSUBr\n(D)  CA1  SUB  LEC MEC HIP D-V  D-V mean  SEMCA1  SUB  LEC-  MEC- \n(E)  HIP RSP-onlyMBO-onlyRSPMBORSP  MBO \n(F)  HIP NAc-only, mPFC-only, BA-only, NAc/mPFC, NAc/BA, mPFC/BA, NAc/mPFC/BANAcBAmPFC\n(G) subtypes383637  (E) subtypes383637  31%91%  98% D-V mean  SEM\n(H) (F)  7  D-V mean  SEMNAc/mPFC/BA 145Nac 1160mPFC 144BA 108NAc/mPFC 1866NAc/BA 2137mPFC/BA 316 D-V  Nac-only \n\n\n# Fig. 3 Spatial correlation between transcriptome and projectome subtypes\n\n\n\nFig. 3 \n\n(A)  Stereo-seq  HIPbregma -0.9 to -4.4 mm10 m  Stereo-seq  mRNA  RNA B\n(B) UMAP  23  HIP  CA1 top, Snap25  Plp1bottom Stereo-seq  50  50 bin DNA  25 m\n(C) cluster 1 [C1, defined in (B)]  bregma  HIP  HIP  C1  Nnat Nnat  mRNA  C1  bin HIP \n(D) UMAP  C1  C1  14  subclusters 14  subclusters \n(E)  Stereo-seq  bregma  C1 subclusters  bregma  CA1 \n(F and H) subcluster  bregma Spearman subcluster 6  subtype 39 (F)P < 0.001subcluster 8  subtype 10 (H)P < 0.0001\n(G and I) subcluster  CA1  (G)  subtype 39subcluster 6  Snap25(I)  subtype 10subcluster 8  Gap43\n(J to L)  subclusters 25678  GABA  (J) (K)  GABA  (L) \n\n\n# Fig. 4 Bihemispheric projecting neurons in HIP\n\n\n\nFig. 4 HIP \n\n(A)  HIP (n = 803)RHP (n = 533)LSX (n = 806)NAc (n = 55)MSC (n = 140)AON (n = 23)TT (n = 17)ATN (n = 85)MBO (n = 1153)LZ ( n = 157)  PVR (n = 86)\n(B)  LSXNAcMSCAONATNMBO \n(C) CA1  HIP HIP \n(D)  DG (5.6%)CA1 (10.3%)CA2 (2.7%)CA3 (81.1%)SUB (0.2%)  HIP  (n = 803) \n(E)  D-V  HIP  CA1 CA3CA1  CA3 \n(F)  SUBr  RHP ENTRHP  MEC 0.5 mm\n(G)  CA1 (1.9%)CA2 (0.4%)CA3 (4.5%)SUB (9.8%)POST (20.6%)PAR (17.6%) PRE43.5%ProS1.3% RHP  (n = 533)  HIP-amygdalar HATA0.19%\n(H)  LEC  MEC  (G)  ProS  PARPREPOST \n(I)  11 \n(J)  HIP  7  ISOOLFCTXspTHHYMB\n(K) AON  HIP  HIP  HIP \n(L) \n(M)  11 \n\n\n# Fig. 5 Projection profiles of CA1, SUB, and SUBr neurons\n\n\n\nFig. 5 CA1SUB  SUBr \n\n(A)  CA1SUBProSSUBr  15  CA1SUBProSSUBr\n(B)  CA1SUBProSSUBr CA1SUBProSSUBr  P  0.0001\n(C) CA1SUBProSSUBr 13 mean  SEM\n(D to G)  CA1 (D)SUB (E)ProS (F)SUBr (G) OLFCNUCTXsp OLFCCNUCCTXspC CNUc  CTXspc CA1DSUBEProSFSUBrG 13  HPFCHPFRISOCISOROLFCOLFRCTXspCCTXspRCNUCCNURTHRHYRMBR CA1SUBProSSUBr \n(H) HPF  HPF  HPF HPF  HPF CA1SUBProSSUBr D-V \n(I)  HPF mean  SEMCA1SUBProSSUBr\n(J)  LECLEC  MECMEC  HPF \n(K) --HPF HPF CASUBPREPOSTProSLECMEC (I) \n(L) --HPF (K)  LEC  OLFR  CA  HYR \n\n\n# Fig. 6 Longitudinal spread of Schaffer collaterals and mossy fiber axons\n\n\n\nFig. 6 Schaffer  mossy \n\n(A)  CA3  CA3 CA1 HIP  100  binlt TL  T-L \n(B)  CA3  CA3  CA3  CA1 \n(C and D)  100  bin  T-L  CA3 n = 352n = 544D-V  T-L  bin  CA3  T-L  CA3 C CA1 D\n(E)  D-V  DG  DG  CA3 \n(F)  100  bin  T-L  D-V n = 741n = 2318DG  T-L  bin  DG  T-L  DG  CA3 \n(G)  HIP  HIP  DG  CA3  HIP  DG  CA3 \n(H)  CA3  CA1  CA3  T-L \n(I)  DG  CA3  DG  T-L \n(J)  CA3  CA1  D-V \n(K)  DG  CA3  D-V \n\n\n# Fig. 7 Dependence of axonal arbor distribution on the transverse location of soma\n\n\n\nFig. 7 \n\n(A) HIP  CA3  CA1  P-D  P-D  CA1 \n(B and C)  300 mm  HIP  PI  CA1  stratumstr.-l-m(B)  CA3  CA1 (C)  CA3  CA1  P-D \n(D and E) CA3  P-D  P-D D CA1 En = 765rCB\n(F) CA3 P-D  CA1  P-D \n(G) CA1  LEC  MEC  P-D  LEC  MEC  NPC \n(H) CA1  P-D  ENT  NPC n = 1633\n(I) CA1  EP G\n(J) CA1  P-D  EP  NPC n = 160\n(K)  P-D  CA1  CA1\n(L)  CA1  CA1 n = 89 CA1  P-D  CA1  P-D \n(M) CA1  P-D ML\n\n\n# Fig. 8 Dependence of axonal arbor distribution on the longitudinal location of soma\n\n\n\nFig. 8 \n\n(A)  CA1  MEC \n(B) HIP  LEC  MEC  D-V \n(C) HIP  D-V  ENT  NPC  [ (B) Pearson r = 0.80P < 0.0001n = 3177 ]\n(D)  D-V  HIP \n(E) CA1SUBProSSUBr  D-V \n(F) CA1SUBProSSUBrmean  SEM D-V \n(G) D-V  bin HIP  HPFISOOLFCTXspCNUTHHYMB bin  HIP \n\n\n# Discussion\n\n fMOST  HIP  10,100  HIP  CA1 https://mouse.digital-brain.cn/hippi 43  HPF ii HPFiiiivv HIP ",normalizedContent:": \ndoi: https://doi.org/10.1126/science.adj9198\ncite: shou qiu et al. ,whole-brain spatial organization of hippocampal single-neuron projectomes.science383,eadj9198(2024).\n\n\n\n\n\n\n\n                                                                                                         \n                                                                                                                      \n                                    \nchun.xu@ion.ac.cn     yangang.sun@ion.ac.cn   huigong@mail.hust.edu.cn   haishanyao@ion.ac.cn   tonylicy@ion.ac.cn    ydsun@ion.ac.cn\n\n\n# abstract\n\nhip 10,100  hip  43  hip  hip  hip hpf hip  hpf  hpf  hip  hip \n\n\n# figure\n\n\n# fig. 1 projectome subtype classifications of hip neurons\n\n\n\nfig. 1 hip \n\n(a)  egfp  hip 100 um20 um\n(b)  hip  hip  hip dvap hip  hip \n(c) \n(d)  hip  7  hip dgca3ca2ca1subprossubr d-v  9  hpfisoolfctxspcnuthhymb table s3 \n(e)  hip ward's method 13 rcd hip  d-v \n(f-h) group 1  subtype 3fgroup 2  subtype 26ggroup 3  subtype 37h\n\n\n# fig. 2 projection targetdependent soma locations of hip neurons\n\n\n\nfig. 2 hip \n\n(a)  50  extra-hpf  intra-hpfc  intra-hpfc \n(b)  hip  lec  mec \n(c)  lec-mec  bins  lec  mec sel.lec  mec  hip ca1subprossubr\n(d)  ca1  sub  lec mec hip d-v  d-v mean  semca1  sub  lec-  mec- \n(e)  hip rsp-onlymbo-onlyrspmborsp  mbo \n(f)  hip nac-only, mpfc-only, ba-only, nac/mpfc, nac/ba, mpfc/ba, nac/mpfc/banacbampfc\n(g) subtypes383637  (e) subtypes383637  31%91%  98% d-v mean  sem\n(h) (f)  7  d-v mean  semnac/mpfc/ba 145nac 1160mpfc 144ba 108nac/mpfc 1866nac/ba 2137mpfc/ba 316 d-v  nac-only \n\n\n# fig. 3 spatial correlation between transcriptome and projectome subtypes\n\n\n\nfig. 3 \n\n(a)  stereo-seq  hipbregma -0.9 to -4.4 mm10 m  stereo-seq  mrna  rna b\n(b) umap  23  hip  ca1 top, snap25  plp1bottom stereo-seq  50  50 bin dna  25 m\n(c) cluster 1 [c1, defined in (b)]  bregma  hip  hip  c1  nnat nnat  mrna  c1  bin hip \n(d) umap  c1  c1  14  subclusters 14  subclusters \n(e)  stereo-seq  bregma  c1 subclusters  bregma  ca1 \n(f and h) subcluster  bregma spearman subcluster 6  subtype 39 (f)p < 0.001subcluster 8  subtype 10 (h)p < 0.0001\n(g and i) subcluster  ca1  (g)  subtype 39subcluster 6  snap25(i)  subtype 10subcluster 8  gap43\n(j to l)  subclusters 25678  gaba  (j) (k)  gaba  (l) \n\n\n# fig. 4 bihemispheric projecting neurons in hip\n\n\n\nfig. 4 hip \n\n(a)  hip (n = 803)rhp (n = 533)lsx (n = 806)nac (n = 55)msc (n = 140)aon (n = 23)tt (n = 17)atn (n = 85)mbo (n = 1153)lz ( n = 157)  pvr (n = 86)\n(b)  lsxnacmscaonatnmbo \n(c) ca1  hip hip \n(d)  dg (5.6%)ca1 (10.3%)ca2 (2.7%)ca3 (81.1%)sub (0.2%)  hip  (n = 803) \n(e)  d-v  hip  ca1 ca3ca1  ca3 \n(f)  subr  rhp entrhp  mec 0.5 mm\n(g)  ca1 (1.9%)ca2 (0.4%)ca3 (4.5%)sub (9.8%)post (20.6%)par (17.6%) pre43.5%pros1.3% rhp  (n = 533)  hip-amygdalar hata0.19%\n(h)  lec  mec  (g)  pros  parprepost \n(i)  11 \n(j)  hip  7  isoolfctxspthhymb\n(k) aon  hip  hip  hip \n(l) \n(m)  11 \n\n\n# fig. 5 projection profiles of ca1, sub, and subr neurons\n\n\n\nfig. 5 ca1sub  subr \n\n(a)  ca1subprossubr  15  ca1subprossubr\n(b)  ca1subprossubr ca1subprossubr  p  0.0001\n(c) ca1subprossubr 13 mean  sem\n(d to g)  ca1 (d)sub (e)pros (f)subr (g) olfcnuctxsp olfccnucctxspc cnuc  ctxspc ca1dsubeprosfsubrg 13  hpfchpfrisocisorolfcolfrctxspcctxsprcnuccnurthrhyrmbr ca1subprossubr \n(h) hpf  hpf  hpf hpf  hpf ca1subprossubr d-v \n(i)  hpf mean  semca1subprossubr\n(j)  leclec  mecmec  hpf \n(k) --hpf hpf casubprepostproslecmec (i) \n(l) --hpf (k)  lec  olfr  ca  hyr \n\n\n# fig. 6 longitudinal spread of schaffer collaterals and mossy fiber axons\n\n\n\nfig. 6 schaffer  mossy \n\n(a)  ca3  ca3 ca1 hip  100  binlt tl  t-l \n(b)  ca3  ca3  ca3  ca1 \n(c and d)  100  bin  t-l  ca3 n = 352n = 544d-v  t-l  bin  ca3  t-l  ca3 c ca1 d\n(e)  d-v  dg  dg  ca3 \n(f)  100  bin  t-l  d-v n = 741n = 2318dg  t-l  bin  dg  t-l  dg  ca3 \n(g)  hip  hip  dg  ca3  hip  dg  ca3 \n(h)  ca3  ca1  ca3  t-l \n(i)  dg  ca3  dg  t-l \n(j)  ca3  ca1  d-v \n(k)  dg  ca3  d-v \n\n\n# fig. 7 dependence of axonal arbor distribution on the transverse location of soma\n\n\n\nfig. 7 \n\n(a) hip  ca3  ca1  p-d  p-d  ca1 \n(b and c)  300 mm  hip  pi  ca1  stratumstr.-l-m(b)  ca3  ca1 (c)  ca3  ca1  p-d \n(d and e) ca3  p-d  p-d d ca1 en = 765rcb\n(f) ca3 p-d  ca1  p-d \n(g) ca1  lec  mec  p-d  lec  mec  npc \n(h) ca1  p-d  ent  npc n = 1633\n(i) ca1  ep g\n(j) ca1  p-d  ep  npc n = 160\n(k)  p-d  ca1  ca1\n(l)  ca1  ca1 n = 89 ca1  p-d  ca1  p-d \n(m) ca1  p-d ml\n\n\n# fig. 8 dependence of axonal arbor distribution on the longitudinal location of soma\n\n\n\nfig. 8 \n\n(a)  ca1  mec \n(b) hip  lec  mec  d-v \n(c) hip  d-v  ent  npc  [ (b) pearson r = 0.80p < 0.0001n = 3177 ]\n(d)  d-v  hip \n(e) ca1subprossubr  d-v \n(f) ca1subprossubrmean  sem d-v \n(g) d-v  bin hip  hpfisoolfctxspcnuthhymb bin  hip \n\n\n# discussion\n\n fmost  hip  10,100  hip  ca1 https://mouse.digital-brain.cn/hippi 43  hpf ii hpfiiiivv hip ",charsets:{cjk:!0},lastUpdated:"2024/03/07, 10:44:36",lastUpdatedTimestamp:1709779476e3},{title:" --- Vision-dependent specification of cell types and function in the developing cortex",frontmatter:{title:" --- Vision-dependent specification of cell types and function in the developing cortex",date:"2023-07-11T20:38:38.000Z",permalink:"/pages/b79c35/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/03.%E5%8F%91%E8%82%B2/20220120.(2022.01.20%20%E8%A7%86%E8%A7%89%E7%9A%AE%E5%B1%82%E5%8F%91%E8%82%B2%20Cell)%20Vision-dependent%20specification%20of%20cell%20types%20and%20function%20in%20the%20developing%20cortex.html",relativePath:"02./11./03./20220120.(2022.01.20  Cell) Vision-dependent specification of cell types and function in the developing cortex.md",key:"v-9ab7ce14",path:"/pages/b79c35/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:512},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:901},{level:3,title:"Figure 1. snRNA-seq profiling of V1 during postnatal development",slug:"figure-1-snrna-seq-profiling-of-v1-during-postnatal-development",normalizedTitle:"figure 1. snrna-seq profiling of v1 during postnatal development",charIndex:913},{level:3,title:"Figure 2. Transcriptomic diversity of V1 glutamatergic neurons during postnatal development",slug:"figure-2-transcriptomic-diversity-of-v1-glutamatergic-neurons-during-postnatal-development",normalizedTitle:"figure 2. transcriptomic diversity of v1 glutamatergic neurons during postnatal development",charIndex:1350},{level:3,title:"Figure 3. Anatomical and transcriptomic maturation of L2/3 glutamatergic neuron types",slug:"figure-3-anatomical-and-transcriptomic-maturation-of-l2-3-glutamatergic-neuron-types",normalizedTitle:"figure 3. anatomical and transcriptomic maturation of l2/3 glutamatergic neuron types",charIndex:2025},{level:3,title:"Figure 4. Visual experience is required to maintain L2/3 glutamatergic neuron types",slug:"figure-4-visual-experience-is-required-to-maintain-l2-3-glutamatergic-neuron-types",normalizedTitle:"figure 4. visual experience is required to maintain l2/3 glutamatergic neuron types",charIndex:2416},{level:3,title:"Figure 5. Vision is required to establish L2/3 glutamatergic neuron types",slug:"figure-5-vision-is-required-to-establish-l2-3-glutamatergic-neuron-types",normalizedTitle:"figure 5. vision is required to establish l2/3 glutamatergic neuron types",charIndex:3262},{level:3,title:"Figure 6. Continuous variation of L2/3 neuron types and vision-dependent gene gradients implicated in wiring",slug:"figure-6-continuous-variation-of-l2-3-neuron-types-and-vision-dependent-gene-gradients-implicated-in-wiring",normalizedTitle:"figure 6. continuous variation of l2/3 neuron types and vision-dependent gene gradients implicated in wiring",charIndex:3562},{level:3,title:"Figure 7. Igsf9b is required for vision-dependent maturation of binocular neurons in V1B L2/3",slug:"figure-7-igsf9b-is-required-for-vision-dependent-maturation-of-binocular-neurons-in-v1b-l2-3",normalizedTitle:"figure 7. igsf9b is required for vision-dependent maturation of binocular neurons in v1b l2/3",charIndex:4328},{level:2,title:"Supplemental Figure",slug:"supplemental-figure",normalizedTitle:"supplemental figure",charIndex:5439},{level:3,title:"Figure S1. Expression of GABAergic and glutamatergic markers with age, preprocessing snRNA-seq data, and comparison to transcriptomic signatures of adult visual and motor cortices (Tasic et al., 2018), related to Figure 1",slug:"figure-s1-expression-of-gabaergic-and-glutamatergic-markers-with-age-preprocessing-snrna-seq-data-and-comparison-to-transcriptomic-signatures-of-adult-visual-and-motor-cortices-tasic-et-al-2018-related-to-figure-1",normalizedTitle:"figure s1. expression of gabaergic and glutamatergic markers with age, preprocessing snrna-seq data, and comparison to transcriptomic signatures of adult visual and motor cortices (tasic et al., 2018), related to figure 1",charIndex:5463},{level:3,title:"Figure S2. Additional data related to transcriptomic maturation of V1 cell types, related to Figure 2",slug:"figure-s2-additional-data-related-to-transcriptomic-maturation-of-v1-cell-types-related-to-figure-2",normalizedTitle:"figure s2. additional data related to transcriptomic maturation of v1 cell types, related to figure 2",charIndex:5783},{level:3,title:"Figure S3. Cell-type-specific markers for glutamatergic, GABAergic, and non-neuronal subclasses, related to Figure 2",slug:"figure-s3-cell-type-specific-markers-for-glutamatergic-gabaergic-and-non-neuronal-subclasses-related-to-figure-2",normalizedTitle:"figure s3. cell-type-specific markers for glutamatergic, gabaergic, and non-neuronal subclasses, related to figure 2",charIndex:5926},{level:3,title:"Figure S4. In situ analysis of L2/3 glutamatergic neuron types in normally reared mice, related to Figure 3",slug:"figure-s4-in-situ-analysis-of-l2-3-glutamatergic-neuron-types-in-normally-reared-mice-related-to-figure-3",normalizedTitle:"figure s4. in situ analysis of l2/3 glutamatergic neuron types in normally reared mice, related to figure 3",charIndex:6126},{level:3,title:"Figure S5. Transcriptomic changes in V1 cell type diversity in dark-rearing experiments, related to Figure 4",slug:"figure-s5-transcriptomic-changes-in-v1-cell-type-diversity-in-dark-rearing-experiments-related-to-figure-4",normalizedTitle:"figure s5. transcriptomic changes in v1 cell type diversity in dark-rearing experiments, related to figure 4",charIndex:6296},{level:3,title:"Figure S6. Graded gene expression among L2/3 types, and selective maturation of L2/3 types, related to Figure 6",slug:"figure-s6-graded-gene-expression-among-l2-3-types-and-selective-maturation-of-l2-3-types-related-to-figure-6",normalizedTitle:"figure s6. graded gene expression among l2/3 types, and selective maturation of l2/3 types, related to figure 6",charIndex:6447},{level:3,title:"Figure S7. Immunohistological characterization of synaptic markers in Igsf9b knockout (KO), mapping V1b, and measuring receptive field tuning via 2-photon calcium imaging of awake mice, related to Figure 7",slug:"figure-s7-immunohistological-characterization-of-synaptic-markers-in-igsf9b-knockout-ko-mapping-v1b-and-measuring-receptive-field-tuning-via-2-photon-calcium-imaging-of-awake-mice-related-to-figure-7",normalizedTitle:"figure s7. immunohistological characterization of synaptic markers in igsf9b knockout (ko), mapping v1b, and measuring receptive field tuning via 2-photon calcium imaging of awake mice, related to figure 7",charIndex:6613}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2021.12.022" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2021.12.022<OutboundLink/></a><br>\nCite: Cheng S, Butrus S, Tan L, et al. Vision-dependent specification of cell types and function in the developing cortex. <em>Cell</em>. 2022;185(2):311-327.e24.</p>\n',headersStr:"Abstract Results Figure 1. snRNA-seq profiling of V1 during postnatal development Figure 2. Transcriptomic diversity of V1 glutamatergic neurons during postnatal development Figure 3. Anatomical and transcriptomic maturation of L2/3 glutamatergic neuron types Figure 4. Visual experience is required to maintain L2/3 glutamatergic neuron types Figure 5. Vision is required to establish L2/3 glutamatergic neuron types Figure 6. Continuous variation of L2/3 neuron types and vision-dependent gene gradients implicated in wiring Figure 7. Igsf9b is required for vision-dependent maturation of binocular neurons in V1B L2/3 Supplemental Figure Figure S1. Expression of GABAergic and glutamatergic markers with age, preprocessing snRNA-seq data, and comparison to transcriptomic signatures of adult visual and motor cortices (Tasic et al., 2018), related to Figure 1 Figure S2. Additional data related to transcriptomic maturation of V1 cell types, related to Figure 2 Figure S3. Cell-type-specific markers for glutamatergic, GABAergic, and non-neuronal subclasses, related to Figure 2 Figure S4. In situ analysis of L2/3 glutamatergic neuron types in normally reared mice, related to Figure 3 Figure S5. Transcriptomic changes in V1 cell type diversity in dark-rearing experiments, related to Figure 4 Figure S6. Graded gene expression among L2/3 types, and selective maturation of L2/3 types, related to Figure 6 Figure S7. Immunohistological characterization of synaptic markers in Igsf9b knockout (KO), mapping V1b, and measuring receptive field tuning via 2-photon calcium imaging of awake mice, related to Figure 7",content:": \nDOI: https://doi.org/10.1016/j.cell.2021.12.022\nCite: Cheng S, Butrus S, Tan L, et al. Vision-dependent specification of cell types and function in the developing cortex. Cell. 2022;185(2):311-327.e24.\n\n\n\n\n\n\n\nS. LAWRENCE ZIPURSKY                    KARTHIK SHEKHAR\n                                        \nUniversity of California, Los Angeles   University of California, Berkeley\nlzipursky@mednet.ucla.edu               kshekhar@berkeley.edu\n\n\n# Abstract\n\npostnatal snRNAseqV1L2/3/4glutamatergic  glutamatergicGABAergicnon-neuronal L2/3  ~200  Igsf9b  L2/3  glutamatergic \n\n\n# Results\n\n\n# Figure 1. snRNA-seq profiling of V1 during postnatal development\n\n\n\nFigure 1.  V1  snRNA-seq \n\n(A) Primary visual cortex (V1)A, anterior; AL, anterolateral; AM, anteromedial; LI, laterointermediate; LM, lateromedial; P, posterior; PM, posteromedial; POR, postrhinal; RL, rostrolateral; TEA, temporal anterior areas\n(B) V1 snRNA-seq  6 \n(C) V1 \n(D)  V1  UMAP \n\n\n# Figure 2. Transcriptomic diversity of V1 glutamatergic neurons during postnatal development\n\n\n\nFigure 2.  V1 \n\n(A) V1  L1-L6 \n(B) Tracks plot  1000 \n(C) \n(D)  P21  V1  FISHCcbe1  L2/3 glutamatergic neurons Cux2  L2/3 and L4 glutamatergic neuronsinhibitory neuronsnon-neuronal cells \n(E)  V1 L5  L6 L2/3  L4 L2/3 P8  P14 \n(F) ARI\n(G) L2/3  L4  L5  L6\n\n\n# Figure 3. Anatomical and transcriptomic maturation of L2/3 glutamatergic neuron types\n\n\n\nFigure 3. L2/3 \n\n(A) V1  L2/3  UMAP \n(B) Dot plot  L2/3  L2/3 \n(C-D) P8  P38  L2/3  Cdh13Trpc6Chrm2  FISH \n(E) 6  L2/3  Cdh13Trpc6Chrm2 \n(F) \n(G-H) FISH  snRNA-seq \n\n\n# Figure 4. Visual experience is required to maintain L2/3 glutamatergic neuron types\n\n\n\nFigure 4.  L2/3 \n\n(A)  P21-P28 P28DR P21-P38 P38DR P21-P28 +8hP28DL\n(B) P28DRP38DRP28DL  V1 snRNAseq UMAP\n(C) P28DRP38DRP28DL ARI dark reared (DR)  L2/3  L4  L5  L6 \n(D-E)  (NR)  L2/3  dark reared (DR)  L4  L5  L6 \n(F)  (NR) dark reared (DR)  Trpc6  Cdh13 \n(G-H)  P38 dark reared (DR)  P28  dark reared (DR)  P21  P28  dark reared (DR)  8  (DL)  L2/3  L2/3 \n(I-J) FISH  snRNA-seq  L2/3 \n\n\n# Figure 5. Vision is required to establish L2/3 glutamatergic neuron types\n\n\n\nFigure 5.  L2/3 \n\n(A) \n(B-D)  P17DR Trpc6 Cdh13  L5  L2/3 \n\n\n# Figure 6. Continuous variation of L2/3 neuron types and vision-dependent gene gradients implicated in wiring\n\n\n\nFigure 6.  L2/3 \n\n(A)  (NR)  L2/3_AL2/3_BL2/3_C  DRDL  L2/3 \n(B-C) cell surface molecules, CSMs P21P38  DR  DL \n(D) MDGA1  IGSF9B  NLGN2 MDGA1  NLGN2  NRXN IGSF9B  NLGN2 SSCAM\n(E-J)  Igsf9b IGSF9B 2Nlgn2Igsf9b DR  Igsf9b  L2/3  DR  8 DLIgsf9b  Mdga1Nlgn2  Igsf9b \n\n\n# Figure 7. Igsf9b is required for vision-dependent maturation of binocular neurons in V1B L2/3\n\n\n\nFigure 7.  V1B L2/3  Igsf9b\n\n(A)  V1  L2/3  Igsf9bV1  P21  P36 NR WT  Igsf9b KO  NR  DR  gcamp6 \n(B-D)  L2/3  top (type A)middle (type B)bottom (type C)  Igsf9b \n(E)  P21 KO  WT  P21  L2/3  Igsf9b \n(F-G)  P36  Igsf9b KO  DR \n(H-I)  KO tuned cellsUntuned cells\n(J)  P36  75%NR 66%DR Igsf9b KO  47%\n(K) KO  P21  P36 SNRsKO  DR \n(L) KO  NR WT  L2/3A B  C  Igsf9b  pial-ventricular axis \n Igsf9b  L2/3 pial-ventricular axis  L2/3 \n\n\n# Supplemental Figure\n\n\n# Figure S1. Expression of GABAergic and glutamatergic markers with age, preprocessing snRNA-seq data, and comparison to transcriptomic signatures of adult visual and motor cortices (Tasic et al., 2018), related to Figure 1\n\n\n\nFigure S1. GABAergic  glutamatergic markers  snRNA-seq \n\n\n# Figure S2. Additional data related to transcriptomic maturation of V1 cell types, related to Figure 2\n\n\n\nFigure S2.  V1 \n\n\n# Figure S3. Cell-type-specific markers for glutamatergic, GABAergic, and non-neuronal subclasses, related to Figure 2\n\n\n\nFigure S3. glutamatergicGABAergicnon-neuronal subclasses  markers\n\n\n# Figure S4. In situ analysis of L2/3 glutamatergic neuron types in normally reared mice, related to Figure 3\n\n\n\nFigure S4.  L2/3 glutamatergic neuron \n\n\n# Figure S5. Transcriptomic changes in V1 cell type diversity in dark-rearing experiments, related to Figure 4\n\n\n\nFigure S5.  V1 \n\n\n# Figure S6. Graded gene expression among L2/3 types, and selective maturation of L2/3 types, related to Figure 6\n\n\n\nFigure S6. L2/3  L2/3 \n\n\n# Figure S7. Immunohistological characterization of synaptic markers in Igsf9b knockout (KO), mapping V1b, and measuring receptive field tuning via 2-photon calcium imaging of awake mice, related to Figure 7\n\n\n\nFigure S7. Igsf9b KO V1b",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2021.12.022\ncite: cheng s, butrus s, tan l, et al. vision-dependent specification of cell types and function in the developing cortex. cell. 2022;185(2):311-327.e24.\n\n\n\n\n\n\n\ns. lawrence zipursky                    karthik shekhar\n                                        \nuniversity of california, los angeles   university of california, berkeley\nlzipursky@mednet.ucla.edu               kshekhar@berkeley.edu\n\n\n# abstract\n\npostnatal snrnaseqv1l2/3/4glutamatergic  glutamatergicgabaergicnon-neuronal l2/3  ~200  igsf9b  l2/3  glutamatergic \n\n\n# results\n\n\n# figure 1. snrna-seq profiling of v1 during postnatal development\n\n\n\nfigure 1.  v1  snrna-seq \n\n(a) primary visual cortex (v1)a, anterior; al, anterolateral; am, anteromedial; li, laterointermediate; lm, lateromedial; p, posterior; pm, posteromedial; por, postrhinal; rl, rostrolateral; tea, temporal anterior areas\n(b) v1 snrna-seq  6 \n(c) v1 \n(d)  v1  umap \n\n\n# figure 2. transcriptomic diversity of v1 glutamatergic neurons during postnatal development\n\n\n\nfigure 2.  v1 \n\n(a) v1  l1-l6 \n(b) tracks plot  1000 \n(c) \n(d)  p21  v1  fishccbe1  l2/3 glutamatergic neurons cux2  l2/3 and l4 glutamatergic neuronsinhibitory neuronsnon-neuronal cells \n(e)  v1 l5  l6 l2/3  l4 l2/3 p8  p14 \n(f) ari\n(g) l2/3  l4  l5  l6\n\n\n# figure 3. anatomical and transcriptomic maturation of l2/3 glutamatergic neuron types\n\n\n\nfigure 3. l2/3 \n\n(a) v1  l2/3  umap \n(b) dot plot  l2/3  l2/3 \n(c-d) p8  p38  l2/3  cdh13trpc6chrm2  fish \n(e) 6  l2/3  cdh13trpc6chrm2 \n(f) \n(g-h) fish  snrna-seq \n\n\n# figure 4. visual experience is required to maintain l2/3 glutamatergic neuron types\n\n\n\nfigure 4.  l2/3 \n\n(a)  p21-p28 p28dr p21-p38 p38dr p21-p28 +8hp28dl\n(b) p28drp38drp28dl  v1 snrnaseq umap\n(c) p28drp38drp28dl ari dark reared (dr)  l2/3  l4  l5  l6 \n(d-e)  (nr)  l2/3  dark reared (dr)  l4  l5  l6 \n(f)  (nr) dark reared (dr)  trpc6  cdh13 \n(g-h)  p38 dark reared (dr)  p28  dark reared (dr)  p21  p28  dark reared (dr)  8  (dl)  l2/3  l2/3 \n(i-j) fish  snrna-seq  l2/3 \n\n\n# figure 5. vision is required to establish l2/3 glutamatergic neuron types\n\n\n\nfigure 5.  l2/3 \n\n(a) \n(b-d)  p17dr trpc6 cdh13  l5  l2/3 \n\n\n# figure 6. continuous variation of l2/3 neuron types and vision-dependent gene gradients implicated in wiring\n\n\n\nfigure 6.  l2/3 \n\n(a)  (nr)  l2/3_al2/3_bl2/3_c  drdl  l2/3 \n(b-c) cell surface molecules, csms p21p38  dr  dl \n(d) mdga1  igsf9b  nlgn2 mdga1  nlgn2  nrxn igsf9b  nlgn2 sscam\n(e-j)  igsf9b igsf9b 2nlgn2igsf9b dr  igsf9b  l2/3  dr  8 dligsf9b  mdga1nlgn2  igsf9b \n\n\n# figure 7. igsf9b is required for vision-dependent maturation of binocular neurons in v1b l2/3\n\n\n\nfigure 7.  v1b l2/3  igsf9b\n\n(a)  v1  l2/3  igsf9bv1  p21  p36 nr wt  igsf9b ko  nr  dr  gcamp6 \n(b-d)  l2/3  top (type a)middle (type b)bottom (type c)  igsf9b \n(e)  p21 ko  wt  p21  l2/3  igsf9b \n(f-g)  p36  igsf9b ko  dr \n(h-i)  ko tuned cellsuntuned cells\n(j)  p36  75%nr 66%dr igsf9b ko  47%\n(k) ko  p21  p36 snrsko  dr \n(l) ko  nr wt  l2/3a b  c  igsf9b  pial-ventricular axis \n igsf9b  l2/3 pial-ventricular axis  l2/3 \n\n\n# supplemental figure\n\n\n# figure s1. expression of gabaergic and glutamatergic markers with age, preprocessing snrna-seq data, and comparison to transcriptomic signatures of adult visual and motor cortices (tasic et al., 2018), related to figure 1\n\n\n\nfigure s1. gabaergic  glutamatergic markers  snrna-seq \n\n\n# figure s2. additional data related to transcriptomic maturation of v1 cell types, related to figure 2\n\n\n\nfigure s2.  v1 \n\n\n# figure s3. cell-type-specific markers for glutamatergic, gabaergic, and non-neuronal subclasses, related to figure 2\n\n\n\nfigure s3. glutamatergicgabaergicnon-neuronal subclasses  markers\n\n\n# figure s4. in situ analysis of l2/3 glutamatergic neuron types in normally reared mice, related to figure 3\n\n\n\nfigure s4.  l2/3 glutamatergic neuron \n\n\n# figure s5. transcriptomic changes in v1 cell type diversity in dark-rearing experiments, related to figure 4\n\n\n\nfigure s5.  v1 \n\n\n# figure s6. graded gene expression among l2/3 types, and selective maturation of l2/3 types, related to figure 6\n\n\n\nfigure s6. l2/3  l2/3 \n\n\n# figure s7. immunohistological characterization of synaptic markers in igsf9b knockout (ko), mapping v1b, and measuring receptive field tuning via 2-photon calcium imaging of awake mice, related to figure 7\n\n\n\nfigure s7. igsf9b ko v1b",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:" --- High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labelling",frontmatter:{title:" --- High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labelling",date:"2024-04-13T11:11:56.000Z",permalink:"/pages/723a6a/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20240223.(2024.02.23%20MERGE-seq%20eLife)%20High-throughput%20mapping%20of%20single-neuron%20projection%20and%20molecular%20features%20by%20%20retrograde%20barcoded%20labelling.html",relativePath:"02./11./02./20240223.(2024.02.23 MERGE-seq eLife) High-throughput mapping of single-neuron projection and molecular features by  retrograde barcoded labelling.md",key:"v-f79bb6f6",path:"/pages/723a6a/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:329},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:756},{level:3,title:"Figure 1. MERGE-seq characterized single-neuron transcriptomes and projectomes simultaneously",slug:"figure-1-merge-seq-characterized-single-neuron-transcriptomes-and-projectomes-simultaneously",normalizedTitle:"figure 1. merge-seq characterized single-neuron transcriptomes and projectomes simultaneously",charIndex:767},{level:3,title:"Figure 2. MERGE-seq unravels transcriptomic heterogeneity of projection target-defined vmPFC neurons",slug:"figure-2-merge-seq-unravels-transcriptomic-heterogeneity-of-projection-target-defined-vmpfc-neurons",normalizedTitle:"figure 2. merge-seq unravels transcriptomic heterogeneity of projection target-defined vmpfc neurons",charIndex:1451},{level:3,title:"Figure 3. MERGE-seq reveals projection diversity within the vmPFC",slug:"figure-3-merge-seq-reveals-projection-diversity-within-the-vmpfc",normalizedTitle:"figure 3. merge-seq reveals projection diversity within the vmpfc",charIndex:2234},{level:3,title:"Figure 4. Transcriptional profiling of projection target-specific vmPFC neurons",slug:"figure-4-transcriptional-profiling-of-projection-target-specific-vmpfc-neurons",normalizedTitle:"figure 4. transcriptional profiling of projection target-specific vmpfc neurons",charIndex:2865},{level:3,title:"Figure 5. Molecular features of single vmPFC neuron with collateral projections to downstream targets",slug:"figure-5-molecular-features-of-single-vmpfc-neuron-with-collateral-projections-to-downstream-targets",normalizedTitle:"figure 5. molecular features of single vmpfc neuron with collateral projections to downstream targets",charIndex:3320},{level:3,title:"Figure 6. Machine learning-based modeling predicts projection patterns based on gene expression",slug:"figure-6-machine-learning-based-modeling-predicts-projection-patterns-based-on-gene-expression",normalizedTitle:"figure 6. machine learning-based modeling predicts projection patterns based on gene expression",charIndex:4069}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.7554/eLife.85419" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.85419<OutboundLink/></a><br>\nCite: Peibo Xu, Jian Peng, Tingli Yuan, et al.  (2024) High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labeling eLife 13:e85419.</p>\n',headersStr:"Abstract Figure Figure 1. MERGE-seq characterized single-neuron transcriptomes and projectomes simultaneously Figure 2. MERGE-seq unravels transcriptomic heterogeneity of projection target-defined vmPFC neurons Figure 3. MERGE-seq reveals projection diversity within the vmPFC Figure 4. Transcriptional profiling of projection target-specific vmPFC neurons Figure 5. Molecular features of single vmPFC neuron with collateral projections to downstream targets Figure 6. Machine learning-based modeling predicts projection patterns based on gene expression",content:": \nDOI: https://doi.org/10.7554/eLife.85419\nCite: Peibo Xu, Jian Peng, Tingli Yuan, et al. (2024) High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labeling eLife 13:e85419.\n\n\n\n\n\n\n\n\n\n\nyuejunchen@ion.ac.cn\n\n\n# Abstract\n\n rAAV2-retro  (vmPFC)  5 dedicatedcollateral vmPFC  vmPFC  Pou3f1 \n\n\n# Figure\n\n\n# Figure 1. MERGE-seq characterized single-neuron transcriptomes and projectomes simultaneously\n\n\n\nFigure 1. MERGE-seq \n\n(A) \n(B) rAAV2  read 1  barcode  UMI read 2  EGFP 3' tail  barcode 10X Genomics read 1  28 bpread 2  91 bp150 bp  cell barcodeUMI  virus barcode \n(C)  vmPFC  Umap embedding\n(D)  cluster marker gene \n(E)  Bhattacherjee et al. \n(F)  barcoded cells  Umap embedding\n(G)  barcoded non-barcoded\n (C-E)  21,261  (F,G)  20,047  1214  nUMI \n\n\n# Figure 2. MERGE-seq unravels transcriptomic heterogeneity of projection target-defined vmPFC neurons\n\n\n\nFigure 2. MERGE-seq  vmPFC \n\n(A)  Umap embedding\n(B)  barcoded non-barcoded\n(C)  marker \n(D)  MERFISH  scRNA-seq MERFISH  Bhattacherjee et al., 2023\n(E)  barcoded Umap embeddings barcoded \n(F)  barcoded  (A)  (E) \n(G) \n (ACD)  9368  (BEF)  8210  (G) L2/3-Calb1=72 cells, L2/3-Rorb=331 cells, L5-Bcl6=145 cells, L5-S100b=766 cells, L6-Npy=526 cells, L6-Syt6=1264 cells\n\n\n# Figure 3. MERGE-seq reveals projection diversity within the vmPFC\n\n\n\nFigure 3. MERGE-seq  vmPFC \n\n(A)  MERGEseq  vmPFC barcoded \n(B)  A  B \n(C)  MERGE-seq  1155  fMOST Gao et al., 2022\n(D)  MERGE-seq  fMOST \n(E)  10 \n(F)  10 \n(G)  (E) \n (AB)  2050 CD MERGE-seq  2050  fMOST  1155 Gao et al., 2022 (E-G)  1853  10 \n\n\n# Figure 4. Transcriptional profiling of projection target-specific vmPFC neurons\n\n\n\nFigure 4.  vmPFC \n\n(A)  MD-  non-MD- vmPFC  DEGs DEGsLog2 fold change = 0.5, p value cutoff=1010\n(B) EGFP (MD)  tdTomato (LH)  Syt6  RNA FISH(i, ii) (B)  (i)  6 (ii)  5  Syt6+EGFP+ \n(C) (B) (B) 200 m (B)  iii50 m N=3 (A)  8210 \n\n\n# Figure 5. Molecular features of single vmPFC neuron with collateral projections to downstream targets\n\n\n\nFigure 5.  vmPFC \n\n(A)  10  DEGs  log2 fold change  10  DEGs\n(B)  DMS-DMS + LH \n(C)  DMS LH  DMS + LH  DEGs \n(D-F)  RNA FISH  Pou3f1  DMS + LH  Figure 3figure supplement 1 200um\n(E-F) (D)  Pou3f1+EGFP+tdTomato+  Pou3f1+EGFP-tdTomato+  Pou3f1-EGFP+tdTomato-  Pou3f1+EGFP-tdTomato- 50 m\n(G) (D) N=3  meanSD (A)  1,853  10  (C)  805  DMS + LH = 35LH = 176DMS = 594\n\n\n# Figure 6. Machine learning-based modeling predicts projection patterns based on gene expression\n\n\n\nFigure 6. \n\n(A) \n(B) top HVGs  accuracyAUC score F1 score 8210  1000  100  50  HVGs  50  HVGs  p  two-sided Wilcoxon  meanSD\n(C, E) DMS  MD  SHAP  0  1  50  HVGs \n(D, F)  DMS (D)  MD (F)  PC1  DMS  (E)  barcode 1  MD  (F)  barcode 2  PC2 ",normalizedContent:": \ndoi: https://doi.org/10.7554/elife.85419\ncite: peibo xu, jian peng, tingli yuan, et al. (2024) high-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labeling elife 13:e85419.\n\n\n\n\n\n\n\n\n\n\nyuejunchen@ion.ac.cn\n\n\n# abstract\n\n raav2-retro  (vmpfc)  5 dedicatedcollateral vmpfc  vmpfc  pou3f1 \n\n\n# figure\n\n\n# figure 1. merge-seq characterized single-neuron transcriptomes and projectomes simultaneously\n\n\n\nfigure 1. merge-seq \n\n(a) \n(b) raav2  read 1  barcode  umi read 2  egfp 3' tail  barcode 10x genomics read 1  28 bpread 2  91 bp150 bp  cell barcodeumi  virus barcode \n(c)  vmpfc  umap embedding\n(d)  cluster marker gene \n(e)  bhattacherjee et al. \n(f)  barcoded cells  umap embedding\n(g)  barcoded non-barcoded\n (c-e)  21,261  (f,g)  20,047  1214  numi \n\n\n# figure 2. merge-seq unravels transcriptomic heterogeneity of projection target-defined vmpfc neurons\n\n\n\nfigure 2. merge-seq  vmpfc \n\n(a)  umap embedding\n(b)  barcoded non-barcoded\n(c)  marker \n(d)  merfish  scrna-seq merfish  bhattacherjee et al., 2023\n(e)  barcoded umap embeddings barcoded \n(f)  barcoded  (a)  (e) \n(g) \n (acd)  9368  (bef)  8210  (g) l2/3-calb1=72 cells, l2/3-rorb=331 cells, l5-bcl6=145 cells, l5-s100b=766 cells, l6-npy=526 cells, l6-syt6=1264 cells\n\n\n# figure 3. merge-seq reveals projection diversity within the vmpfc\n\n\n\nfigure 3. merge-seq  vmpfc \n\n(a)  mergeseq  vmpfc barcoded \n(b)  a  b \n(c)  merge-seq  1155  fmost gao et al., 2022\n(d)  merge-seq  fmost \n(e)  10 \n(f)  10 \n(g)  (e) \n (ab)  2050 cd merge-seq  2050  fmost  1155 gao et al., 2022 (e-g)  1853  10 \n\n\n# figure 4. transcriptional profiling of projection target-specific vmpfc neurons\n\n\n\nfigure 4.  vmpfc \n\n(a)  md-  non-md- vmpfc  degs degslog2 fold change = 0.5, p value cutoff=1010\n(b) egfp (md)  tdtomato (lh)  syt6  rna fish(i, ii) (b)  (i)  6 (ii)  5  syt6+egfp+ \n(c) (b) (b) 200 m (b)  iii50 m n=3 (a)  8210 \n\n\n# figure 5. molecular features of single vmpfc neuron with collateral projections to downstream targets\n\n\n\nfigure 5.  vmpfc \n\n(a)  10  degs  log2 fold change  10  degs\n(b)  dms-dms + lh \n(c)  dms lh  dms + lh  degs \n(d-f)  rna fish  pou3f1  dms + lh  figure 3figure supplement 1 200um\n(e-f) (d)  pou3f1+egfp+tdtomato+  pou3f1+egfp-tdtomato+  pou3f1-egfp+tdtomato-  pou3f1+egfp-tdtomato- 50 m\n(g) (d) n=3  meansd (a)  1,853  10  (c)  805  dms + lh = 35lh = 176dms = 594\n\n\n# figure 6. machine learning-based modeling predicts projection patterns based on gene expression\n\n\n\nfigure 6. \n\n(a) \n(b) top hvgs  accuracyauc score f1 score 8210  1000  100  50  hvgs  50  hvgs  p  two-sided wilcoxon  meansd\n(c, e) dms  md  shap  0  1  50  hvgs \n(d, f)  dms (d)  md (f)  pc1  dms  (e)  barcode 1  md  (f)  barcode 2  pc2 ",charsets:{cjk:!0},lastUpdated:"2024/05/28, 12:17:48",lastUpdatedTimestamp:1716869868e3},{title:" --- Deciphering the genetic code of neuronal type connectivity through bilinear modeling",frontmatter:{title:" --- Deciphering the genetic code of neuronal type connectivity through bilinear modeling",date:"2024-07-25T19:19:55.000Z",permalink:"/pages/9c9ddc/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/02.%E6%8A%95%E5%B0%84%E7%BB%84/20240610.(2024.06.10%20%E5%8F%8C%E7%BA%BF%E6%80%A7%E8%BF%9E%E6%8E%A5%E7%BB%84%E6%A8%A1%E5%9E%8B%20eLife)%20Deciphering%20the%20genetic%20code%20of%20neuronal%20type%20connectivity%20through%20bilinear%20modeling.html",relativePath:"02./11./02./20240610.(2024.06.10  eLife) Deciphering the genetic code of neuronal type connectivity through bilinear modeling.md",key:"v-444950d4",path:"/pages/9c9ddc/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:286}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.7554/eLife.91532.3" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.91532.3<OutboundLink/></a><br>\nCite: Mu Qiao (2024) Deciphering the genetic code of neuronal type connectivity through bilinear\nmodeling eLife 12:RP91532.</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.7554/eLife.91532.3\nCite: Mu Qiao (2024) Deciphering the genetic code of neuronal type connectivity through bilinear modeling eLife 12:RP91532.\n\n\n\n\n\n\n\nMU QIAO\n\nLinkedIn\nmuqiao0626@gmail.com\n\n\n# Abstract\n\nSCM SCM ",normalizedContent:": \ndoi: https://doi.org/10.7554/elife.91532.3\ncite: mu qiao (2024) deciphering the genetic code of neuronal type connectivity through bilinear modeling elife 12:rp91532.\n\n\n\n\n\n\n\nmu qiao\n\nlinkedin\nmuqiao0626@gmail.com\n\n\n# abstract\n\nscm scm ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution",frontmatter:{title:" --- Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution",date:"2024-08-30T19:35:00.000Z",permalink:"/pages/0b119c/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/03.%E5%8F%91%E8%82%B2/20221110.(2022.11.10%20%E4%BA%BA%E7%B1%BBPFC%E5%8F%91%E8%82%B2%20Cell)%20Human%20prefrontal%20cortex%20gene%20regulatory%20dynamics%20from%20gestation%20to%20adulthood%20at%20single-cell%20resolution.html",relativePath:"02./11./03./20221110.(2022.11.10 PFC Cell) Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution.md",key:"v-09d0538e",path:"/pages/0b119c/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:394}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2022.09.039" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2022.09.039<OutboundLink/></a><br>\nCite: Herring CA, Simmons RK, Freytag S, et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell. 2022;185(23):4428-4447.e28.</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1016/j.cell.2022.09.039\nCite: Herring CA, Simmons RK, Freytag S, et al. Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. Cell. 2022;185(23):4428-4447.e28.\n\n\n\n\n\n\n\nRYAN LISTER\n\nThe University of Western Australia\nryan.lister@uwa.edu.au\n\n\n# Abstract\n\n",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2022.09.039\ncite: herring ca, simmons rk, freytag s, et al. human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution. cell. 2022;185(23):4428-4447.e28.\n\n\n\n\n\n\n\nryan lister\n\nthe university of western australia\nryan.lister@uwa.edu.au\n\n\n# abstract\n\n",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Molecular and spatial signatures of mouse brain aging at single-cell resolution",frontmatter:{title:" --- Molecular and spatial signatures of mouse brain aging at single-cell resolution",date:"2024-08-30T19:35:07.000Z",permalink:"/pages/16bdaa/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/03.%E5%8F%91%E8%82%B2/20221228.(2022.12.28%20%E8%A1%B0%E8%80%81MERFISH%20Cell)%20Molecular%20and%20spatial%20signatures%20of%20mouse%20brain%20aging%20at%20single-cell%20resolution.html",relativePath:"02./11./03./20221228.(2022.12.28 MERFISH Cell) Molecular and spatial signatures of mouse brain aging at single-cell resolution.md",key:"v-82511ca6",path:"/pages/16bdaa/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:526}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2022.12.010" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2022.12.010<OutboundLink/></a><br>\nCite: William E. Allen, Timothy R. Blosser, Zuri A. Sullivan, et al. Molecular and spatial signatures of mouse brain aging at single-cell resolution. Cell, Volume 186, Issue 1, 2023, Pages 194-208.e18, ISSN 0092-8674.</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1016/j.cell.2022.12.010\nCite: William E. Allen, Timothy R. Blosser, Zuri A. Sullivan, et al. Molecular and spatial signatures of mouse brain aging at single-cell resolution. Cell, Volume 186, Issue 1, 2023, Pages 194-208.e18, ISSN 0092-8674.\n\n\n\n\n\n\n\nXIAOWEI ZHUANG ()           CATHERINE DULAC\n                               \nHarvard University             Harvard University\nzhuang@chemistry.harvard.edu   dulac@fas.harvard.edu\n\n\n# Abstract\n\n",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2022.12.010\ncite: william e. allen, timothy r. blosser, zuri a. sullivan, et al. molecular and spatial signatures of mouse brain aging at single-cell resolution. cell, volume 186, issue 1, 2023, pages 194-208.e18, issn 0092-8674.\n\n\n\n\n\n\n\nxiaowei zhuang ()           catherine dulac\n                               \nharvard university             harvard university\nzhuang@chemistry.harvard.edu   dulac@fas.harvard.edu\n\n\n# abstract\n\n",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Detecting Activated Cell Populations Using Single-Cell RNA-Seq",frontmatter:{title:" --- Detecting Activated Cell Populations Using Single-Cell RNA-Seq",date:"2022-12-08T16:39:55.000Z",permalink:"/pages/8bf2bf/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20171011.(2017.10.11%20Act-seq%20Neuron)%20Detecting%20Activated%20Cell%20Populations%20Using%20Single-Cell%20RNA-Seq.html",relativePath:"02./11./04./20171011.(2017.10.11 Act-seq Neuron) Detecting Activated Cell Populations Using Single-Cell RNA-Seq.md",key:"v-4ca30be4",path:"/pages/8bf2bf/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:343},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:692},{level:3,title:"Fig1. Classifification of MeA Major Cell Types Using Drop-Seq",slug:"fig1-classifification-of-mea-major-cell-types-using-drop-seq",normalizedTitle:"fig1. classifification of mea major cell types using drop-seq",charIndex:704},{level:3,title:"Fig2. Neuronal Subtypes in the MeA",slug:"fig2-neuronal-subtypes-in-the-mea",normalizedTitle:"fig2. neuronal subtypes in the mea",charIndex:1503},{level:3,title:"Fig3. Astrocyte Heterogeneity in the MeA",slug:"fig3-astrocyte-heterogeneity-in-the-mea",normalizedTitle:"fig3. astrocyte heterogeneity in the mea",charIndex:2088},{level:3,title:"Fig4. Act-Seq Suppresses Artificially Induced Gene Expression during Single-Cell Dissociation",slug:"fig4-act-seq-suppresses-artificially-induced-gene-expression-during-single-cell-dissociation",normalizedTitle:"fig4. act-seq suppresses artificially induced gene expression during single-cell dissociation",charIndex:3004},{level:3,title:"Fig5. Confounding Effects of Artificially Induced Gene Expression on Baseline Cell Clustering",slug:"fig5-confounding-effects-of-artificially-induced-gene-expression-on-baseline-cell-clustering",normalizedTitle:"fig5. confounding effects of artificially induced gene expression on baseline cell clustering",charIndex:3553},{level:3,title:"Fig6. Act-Seq Detects Seizure-Induced IEG Upregulation",slug:"fig6-act-seq-detects-seizure-induced-ieg-upregulation",normalizedTitle:"fig6. act-seq detects seizure-induced ieg upregulation",charIndex:3948},{level:3,title:"Fig7. Activation of Neuronal and Astrocyte Subtypes by Physiological Stimuli",slug:"fig7-activation-of-neuronal-and-astrocyte-subtypes-by-physiological-stimuli",normalizedTitle:"fig7. activation of neuronal and astrocyte subtypes by physiological stimuli",charIndex:4650}],excerpt:'<p>:  RNA-Seq <br>\nDOI: <a href="https://doi.org/10.1016/j.neuron.2017.09.026" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.neuron.2017.09.026<OutboundLink/></a><br>\nCite: Wu YE, Pan L, Zuo Y, Li X, Hong W. Detecting Activated Cell Populations Using Single-Cell RNA-Seq. <em>Neuron</em>. 2017 Oct 11;96(2):313-329.e6.</p>\n',headersStr:"Abstract Results Fig1. Classifification of MeA Major Cell Types Using Drop-Seq Fig2. Neuronal Subtypes in the MeA Fig3. Astrocyte Heterogeneity in the MeA Fig4. Act-Seq Suppresses Artificially Induced Gene Expression during Single-Cell Dissociation Fig5. Confounding Effects of Artificially Induced Gene Expression on Baseline Cell Clustering Fig6. Act-Seq Detects Seizure-Induced IEG Upregulation Fig7. Activation of Neuronal and Astrocyte Subtypes by Physiological Stimuli",content:':  RNA-Seq \nDOI: https://doi.org/10.1016/j.neuron.2017.09.026\nCite: Wu YE, Pan L, Zuo Y, Li X, Hong W. Detecting Activated Cell Populations Using Single-Cell RNA-Seq. Neuron. 2017 Oct 11;96(2):313-329.e6.\n\n\n\n RNA-Seq \n\n\n\nWEIZHE HONG()\n\nUniversity of California Los Angeles\nwhong@ucla.edu\n\n\n# Abstract\n\nRNAAct-seq/Act-seqAct-seqCckAct-seq\n\n\n# Results\n\n\n# Fig1. Classifification of MeA Major Cell Types Using Drop-Seq\n\n\n\nFig1.  Drop-Seq  MeA \n\nAB. MeA(A) Drop-seq(B)\nCD. (PC) 1  2 (C)  20679  MeA  tSNE (D)Olig., oligodendrocytes; Endoth., endothelial cells\nE.  top cell-type marker genes (fold change > 3, FDR < 0.05)\nF. marker\nG. MeADAPI(NeuN)FISH(Gja1C1qcPdgfraOpalinCldn5Vtn)\nH. MeADAPIMean  SD, n = 3 \nI. WGCNAN, neurons; AS, astrocytes; MG, microglia; OL, oligodendrocytes; EN, endothelial cells\nJ. (R)FDRpFDR<0.05\n\n\n# Fig2. Neuronal Subtypes in the MeA\n\n\n\nFig2. MeA \n\nA. subclustertop genes(fold change>1.5FDR<0.05)subcluster\nB. 16subclustersubtype markers\nC-H. Dual-color FISH(dFISH)subclusters(CckCartptSst)(Gad2Vglut1) markers50um (C) Cartpt and Gad2; (D) Sst and Gad2; (E) Cck and Gad2; (F) Cck and Vglut1; (G) Cartpt and Sst; (H) Cartpt and Cck (I-J) dFISHIoverlap(J) overlapMean  SD, n R 3 mice\n\n\n# Fig3. Astrocyte Heterogeneity in the MeA\n\n\n\nFig3. MeA \n\nA. MeA(AS1AS2AS3)tSNE\nB. top marker genes (fold change>2FDR<0.05)\nC. markers\nD. WGCNA\nE. top GO terms(FDR<0.15)\nF-I. dFISHmarker(Gja1)markerSlc7a10(enriched in AS1 and AS2)Myoc(enriched in AS3))Lcat(enriched in AS2 and AS3)(F) Slc7a10 and Gja1; (G) Myoc and Gja1; (H) Lcat and Gja1; (I) Myoc and Slc7a10Scale bars, 50 mm\nJK. dFISH(J) The percentage of Gja1+ cells in Slc7a10+, Myoc+,and Lcat+ cells; (K) the percentage of Slc7a10+, Myoc+, and Lcat+ cells in Gja1+ cells and the overlap between Myoc+ and Lcat+ cells. Mean  SD, n = 2 mice for Lcat/Gja1 dFISH, n = 3 mice for all other groups\n\n\n# Fig4. Act-Seq Suppresses Artificially Induced Gene Expression during Single-Cell Dissociation\n\n\n\nFig4. Act-Seq \n\nA-E. IEGsFos(A) Egr1Fos1(B)Fosb(C)Nr4a1(D)Dusp1(E)400"+ inhibitor"ActD\nF-I. IEG(STAR)(F) 34(G-I) 34(G)22(H)11(I)\nJ-O. IEGs(JLN)20top markers(KMO)fold changeFisher\'s exact testFDR-adjustedp\n\n\n# Fig5. Confounding Effects of Artificially Induced Gene Expression on Baseline Cell Clustering\n\n\n\nFig5. \n\nA-D. (microglia)tSNEIEGscluster(Ared dots)IEG(D)3411ActDcluster(BC)IEG(D)\nE-G. IEG(E)34(F)11(G)ActDtSNE\n\n\n# Fig6. Act-Seq Detects Seizure-Induced IEG Upregulation\n\n\n\nFig6. Act-Seq  IEG \n\nA1-A6. Act-seqIEGAct-seqiegvb1200\nB1-C6. iegFisherfdrpieg\nD1-D6. top 50 markersfold changeFisherfdrp\nE1-E6. xtop iegAlltopIEGs\nF1-F6. top regulated IEGsAllE1-E6IEGs300(E1-E5)100(E6)/\n\n\n# Fig7. Activation of Neuronal and Astrocyte Subtypes by Physiological Stimuli\n\n\n\nFig7. \n\nA-H. MeA(AB)(A)(B)ALLIEGFisherp IEGIEGs(p<0.05Fisherp<0.05)',normalizedContent:':  rna-seq \ndoi: https://doi.org/10.1016/j.neuron.2017.09.026\ncite: wu ye, pan l, zuo y, li x, hong w. detecting activated cell populations using single-cell rna-seq. neuron. 2017 oct 11;96(2):313-329.e6.\n\n\n\n rna-seq \n\n\n\nweizhe hong()\n\nuniversity of california los angeles\nwhong@ucla.edu\n\n\n# abstract\n\nrnaact-seq/act-seqact-seqcckact-seq\n\n\n# results\n\n\n# fig1. classifification of mea major cell types using drop-seq\n\n\n\nfig1.  drop-seq  mea \n\nab. mea(a) drop-seq(b)\ncd. (pc) 1  2 (c)  20679  mea  tsne (d)olig., oligodendrocytes; endoth., endothelial cells\ne.  top cell-type marker genes (fold change > 3, fdr < 0.05)\nf. marker\ng. meadapi(neun)fish(gja1c1qcpdgfraopalincldn5vtn)\nh. meadapimean  sd, n = 3 \ni. wgcnan, neurons; as, astrocytes; mg, microglia; ol, oligodendrocytes; en, endothelial cells\nj. (r)fdrpfdr<0.05\n\n\n# fig2. neuronal subtypes in the mea\n\n\n\nfig2. mea \n\na. subclustertop genes(fold change>1.5fdr<0.05)subcluster\nb. 16subclustersubtype markers\nc-h. dual-color fish(dfish)subclusters(cckcartptsst)(gad2vglut1) markers50um (c) cartpt and gad2; (d) sst and gad2; (e) cck and gad2; (f) cck and vglut1; (g) cartpt and sst; (h) cartpt and cck (i-j) dfishioverlap(j) overlapmean  sd, n r 3 mice\n\n\n# fig3. astrocyte heterogeneity in the mea\n\n\n\nfig3. mea \n\na. mea(as1as2as3)tsne\nb. top marker genes (fold change>2fdr<0.05)\nc. markers\nd. wgcna\ne. top go terms(fdr<0.15)\nf-i. dfishmarker(gja1)markerslc7a10(enriched in as1 and as2)myoc(enriched in as3))lcat(enriched in as2 and as3)(f) slc7a10 and gja1; (g) myoc and gja1; (h) lcat and gja1; (i) myoc and slc7a10scale bars, 50 mm\njk. dfish(j) the percentage of gja1+ cells in slc7a10+, myoc+,and lcat+ cells; (k) the percentage of slc7a10+, myoc+, and lcat+ cells in gja1+ cells and the overlap between myoc+ and lcat+ cells. mean  sd, n = 2 mice for lcat/gja1 dfish, n = 3 mice for all other groups\n\n\n# fig4. act-seq suppresses artificially induced gene expression during single-cell dissociation\n\n\n\nfig4. act-seq \n\na-e. iegsfos(a) egr1fos1(b)fosb(c)nr4a1(d)dusp1(e)400"+ inhibitor"actd\nf-i. ieg(star)(f) 34(g-i) 34(g)22(h)11(i)\nj-o. iegs(jln)20top markers(kmo)fold changefisher\'s exact testfdr-adjustedp\n\n\n# fig5. confounding effects of artificially induced gene expression on baseline cell clustering\n\n\n\nfig5. \n\na-d. (microglia)tsneiegscluster(ared dots)ieg(d)3411actdcluster(bc)ieg(d)\ne-g. ieg(e)34(f)11(g)actdtsne\n\n\n# fig6. act-seq detects seizure-induced ieg upregulation\n\n\n\nfig6. act-seq  ieg \n\na1-a6. act-seqiegact-seqiegvb1200\nb1-c6. iegfisherfdrpieg\nd1-d6. top 50 markersfold changefisherfdrp\ne1-e6. xtop iegalltopiegs\nf1-f6. top regulated iegsalle1-e6iegs300(e1-e5)100(e6)/\n\n\n# fig7. activation of neuronal and astrocyte subtypes by physiological stimuli\n\n\n\nfig7. \n\na-h. mea(ab)(a)(b)alliegfisherp iegiegs(p<0.05fisherp<0.05)',charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Div-Seq:Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons",frontmatter:{title:" --- Div-Seq:Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons",date:"2023-03-07T16:21:43.000Z",permalink:"/pages/9b041f/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20160628.(2016.06.28%20Div-Seq%20Science)%20Div-Seq%20Single-nucleus%20RNA-Seq%20reveals%20dynamics%20of%20rare%20adult%20newborn%20neurons.html",relativePath:"02./11./04./20160628.(2016.06.28 Div-Seq Science) Div-Seq Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons.md",key:"v-704e8ccd",path:"/pages/9b041f/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:532},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:877},{level:3,title:"Fig. 1. sNuc-Seq identifies cell types in adult mouse brain",slug:"fig-1-snuc-seq-identifies-cell-types-in-adult-mouse-brain",normalizedTitle:"fig. 1. snuc-seq identifies cell types in adult mouse brain",charIndex:890},{level:3,title:"Fig. 2. sNuc-Seq and biSNE distinguish cell subtypes and spatial expression patterns",slug:"fig-2-snuc-seq-and-bisne-distinguish-cell-subtypes-and-spatial-expression-patterns",normalizedTitle:"fig. 2. snuc-seq and bisne distinguish cell subtypes and spatial expression patterns",charIndex:1215},{level:3,title:"Fig. 3. Transcriptional dynamics of adult neurogenesis by Div-Seq",slug:"fig-3-transcriptional-dynamics-of-adult-neurogenesis-by-div-seq",normalizedTitle:"fig. 3. transcriptional dynamics of adult neurogenesis by div-seq",charIndex:1682},{level:3,title:"Fig. 4. Dynamics of adult newborn GABAergic neurons in SC",slug:"fig-4-dynamics-of-adult-newborn-gabaergic-neurons-in-sc",normalizedTitle:"fig. 4. dynamics of adult newborn gabaergic neurons in sc",charIndex:2049}],excerpt:'<p>: Div-SeqRNA-Seq<br>\nDOI: <a href="https://doi.org/10.1126/science.aad7038" target="_blank" rel="noopener noreferrer">https://doi.org/10.1126/science.aad7038<OutboundLink/></a><br>\nCite: Habib N, Li Y, Heidenreich M. et al. Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Science. 2016 Aug 26;353(6302):925-8.</p>\n',headersStr:"Abstract: Results: Fig. 1. sNuc-Seq identifies cell types in adult mouse brain Fig. 2. sNuc-Seq and biSNE distinguish cell subtypes and spatial expression patterns Fig. 3. Transcriptional dynamics of adult neurogenesis by Div-Seq Fig. 4. Dynamics of adult newborn GABAergic neurons in SC",content:": Div-SeqRNA-Seq\nDOI: https://doi.org/10.1126/science.aad7038\nCite: Habib N, Li Y, Heidenreich M. et al. Div-Seq: Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Science. 2016 Aug 26;353(6302):925-8.\n\n\n\nDiv-SeqRNA-Seq\n\n\n\nAVIV REGEV                           FENG ZHANG()\n                                     \nBroad Institute of MIT and Harvard   Broad Institute of MIT and Harvard\naviv.regev.sc@gmail.com              zhang@broadinstitute.org\n\n\n# Abstract:\n\n RNA Div-SeqRNAsNuc-Seq5--2'-EdUsNuc-Seq  Div-Seq  Div-Seq sNuc-Seq  Div-Seq \n\n\n# Results:\n\n\n# Fig. 1. sNuc-Seq identifies cell types in adult mouse brain\n\n\n\nFig. 1. sNuc-Seq \n\n(A) sNuc-Seq  FACS  Smart-Seq2 \n(B)  sNuc-Seq  1188  tSNE \n(C) B\n(D)  DGCA1CA2CA3 \n\n\n# Fig. 2. sNuc-Seq and biSNE distinguish cell subtypes and spatial expression patterns\n\n\n\nFig. 2. sNuc-Seq  biSNE \n\n(A) CA1  CA3 biSNE subclustersCA1 clusterCA3 cluster\n(B) CA1  CA3 subclusters A\n(C) CA1  CA3  Allen ISH \n(D)  subclusters  Penk Oprd17525 Pvalb  Vip GABA\n(E) Penk  Oprd  GABA  [Pvalb and Vip as in (D)]  Co-FISH\n\n\n# Fig. 3. Transcriptional dynamics of adult neurogenesis by Div-Seq\n\n\n\nFig. 3. Div-Seq \n\n(A) DivSeq EdU  EdU  click-IT chemistry  FACS  sNuc-Seq\n(B)  DG EdU EdU \n(C) DG  biSNE 2D embeddingn = 269\n(D) \n\n\n# Fig. 4. Dynamics of adult newborn GABAergic neurons in SC\n\n\n\nFig. 4.  GABA \n\n(A) Div-Seq OPCsnon-EdU-labeled 6~7  EdU-labeled \n(B) Div-Seq Sox11 GABAergicGad1\n(C)  SC \n(D)  SC  DG \n(E) \n(F) EdU6-7",normalizedContent:": div-seqrna-seq\ndoi: https://doi.org/10.1126/science.aad7038\ncite: habib n, li y, heidenreich m. et al. div-seq: single-nucleus rna-seq reveals dynamics of rare adult newborn neurons. science. 2016 aug 26;353(6302):925-8.\n\n\n\ndiv-seqrna-seq\n\n\n\naviv regev                           feng zhang()\n                                     \nbroad institute of mit and harvard   broad institute of mit and harvard\naviv.regev.sc@gmail.com              zhang@broadinstitute.org\n\n\n# abstract:\n\n rna div-seqrnasnuc-seq5--2'-edusnuc-seq  div-seq  div-seq snuc-seq  div-seq \n\n\n# results:\n\n\n# fig. 1. snuc-seq identifies cell types in adult mouse brain\n\n\n\nfig. 1. snuc-seq \n\n(a) snuc-seq  facs  smart-seq2 \n(b)  snuc-seq  1188  tsne \n(c) b\n(d)  dgca1ca2ca3 \n\n\n# fig. 2. snuc-seq and bisne distinguish cell subtypes and spatial expression patterns\n\n\n\nfig. 2. snuc-seq  bisne \n\n(a) ca1  ca3 bisne subclustersca1 clusterca3 cluster\n(b) ca1  ca3 subclusters a\n(c) ca1  ca3  allen ish \n(d)  subclusters  penk oprd17525 pvalb  vip gaba\n(e) penk  oprd  gaba  [pvalb and vip as in (d)]  co-fish\n\n\n# fig. 3. transcriptional dynamics of adult neurogenesis by div-seq\n\n\n\nfig. 3. div-seq \n\n(a) divseq edu  edu  click-it chemistry  facs  snuc-seq\n(b)  dg edu edu \n(c) dg  bisne 2d embeddingn = 269\n(d) \n\n\n# fig. 4. dynamics of adult newborn gabaergic neurons in sc\n\n\n\nfig. 4.  gaba \n\n(a) div-seq opcsnon-edu-labeled 6~7  edu-labeled \n(b) div-seq sox11 gabaergicgad1\n(c)  sc \n(d)  sc  dg \n(e) \n(f) edu6-7",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons",frontmatter:{title:" --- Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons",date:"2024-01-04T10:48:03.000Z",permalink:"/pages/a61efc/",categories:[""],tags:["-"],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20160816.(2016.08.16%20%E7%BB%86%E8%83%9E%E9%BB%8F%E9%99%84%E5%88%86%E5%AD%90%20PNAS)%20Single-cell%20RNAseq%20reveals%20cell%20adhesion%20molecule%20profiles%20in%20electrophysiologically%20defined%20neurons.html",relativePath:"02./11./04./20160816.(2016.08.16  PNAS) Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons.md",key:"v-2b6b4421",path:"/pages/a61efc/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:512},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:949},{level:3,title:"Fig. 1 RNAseq of the total hippocampus during development: analysis of the changing landscape of CAMs",slug:"fig-1-rnaseq-of-the-total-hippocampus-during-development-analysis-of-the-changing-landscape-of-cams",normalizedTitle:"fig. 1 rnaseq of the total hippocampus during development: analysis of the changing landscape of cams",charIndex:960},{level:3,title:"Fig. 2 Single-cell RNAseq in situ of electrophysiologically characterized neurons in identified microcircuits",slug:"fig-2-single-cell-rnaseq-in-situ-of-electrophysiologically-characterized-neurons-in-identified-microcircuits",normalizedTitle:"fig. 2 single-cell rnaseq in situ of electrophysiologically characterized neurons in identified microcircuits",charIndex:1694},{level:3,title:"Fig. 3 Combined electrophysiological and transcriptional analysis",slug:"fig-3-combined-electrophysiological-and-transcriptional-analysis",normalizedTitle:"fig. 3 combined electrophysiological and transcriptional analysis",charIndex:3045},{level:3,title:"Fig. 4 Hierarchy of CAMs in single neurons",slug:"fig-4-hierarchy-of-cams-in-single-neurons",normalizedTitle:"fig. 4 hierarchy of cams in single neurons",charIndex:4117},{level:3,title:"Fig. 5 Cell-specific expression of exocytosis and RhoGAP/RhoGEF signal transducer molecules",slug:"fig-5-cell-specific-expression-of-exocytosis-and-rhogap-rhogef-signal-transducer-molecules",normalizedTitle:"fig. 5 cell-specific expression of exocytosis and rhogap/rhogef signal transducer molecules",charIndex:4772},{level:3,title:"Fig. 6 Coregulation and coexpression of synaptic molecules",slug:"fig-6-coregulation-and-coexpression-of-synaptic-molecules",normalizedTitle:"fig. 6 coregulation and coexpression of synaptic molecules",charIndex:5596},{level:3,title:"Fig. 7 Molecular identity of BS-PYR and RS-PYR neurons in the subiculum",slug:"fig-7-molecular-identity-of-bs-pyr-and-rs-pyr-neurons-in-the-subiculum",normalizedTitle:"fig. 7 molecular identity of bs-pyr and rs-pyr neurons in the subiculum",charIndex:7546},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:9087}],excerpt:'<p>:  RNAseq <br>\nDOI: <a href="https://doi.org/10.1073/pnas.1610155113" target="_blank" rel="noopener noreferrer">https://doi.org/10.1073/pnas.1610155113<OutboundLink/></a><br>\nCite: Fldy C, Darmanis S, Aoto J, Malenka RC, Quake SR, Sdhof TC. Single-cell RNAseq reveals cell\nadhesion molecule profiles in electrophysiologically defined neurons. Proc Natl Acad Sci U S A.\n2016;113(35):E5222-E5231.</p>\n',headersStr:"Abstract Figure Fig. 1 RNAseq of the total hippocampus during development: analysis of the changing landscape of CAMs Fig. 2 Single-cell RNAseq in situ of electrophysiologically characterized neurons in identified microcircuits Fig. 3 Combined electrophysiological and transcriptional analysis Fig. 4 Hierarchy of CAMs in single neurons Fig. 5 Cell-specific expression of exocytosis and RhoGAP/RhoGEF signal transducer molecules Fig. 6 Coregulation and coexpression of synaptic molecules Fig. 7 Molecular identity of BS-PYR and RS-PYR neurons in the subiculum Discussion",content:':  RNAseq \nDOI: https://doi.org/10.1073/pnas.1610155113\nCite: Fldy C, Darmanis S, Aoto J, Malenka RC, Quake SR, Sdhof TC. Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons. Proc Natl Acad Sci U S A. 2016;113(35):E5222-E5231.\n\n\n\n RNAseq \n\n\n\nCSABA FLDY            THOMAS SUDHOF\n                       \nUniversity of Zrich   Stanford University\nfoldy@hifo.uzh.ch      tcs1@stanford.edu\n\n\n# Abstract\n\n scRNAseq  rho/rac \n\n\n# Figure\n\n\n# Fig. 1 RNAseq of the total hippocampus during development: analysis of the changing landscape of CAMs\n\n\n\nFig. 1  RNAseqCAMs \n\n A  mRNA  RNA Fig.1A P0P7P14P21P28 Fig.1B,C CAMs 406 Fig.1DFig.1EAppCntnNrxnPtpr Ncam1  Sdc3 P0  P28  60%Fig.1F,G\n\n\n# Fig. 2 Single-cell RNAseq in situ of electrophysiologically characterized neurons in identified microcircuits\n\n\n\nFig. 2  RNAseq \n\n RNA  CA1-PYR  FS  RS  target CA1-PYR  FS  PYR APs FS  RS Fig.2AFS  PV+ - RS  CCK+  RNA Fig.2A RNA  18  RS-INT9  FS-INT  14  CA1-PYR \n\n RS-INTFS-INT  PYR Fig.2BGad1  Gad2  PYR  CCK  RS CCK  CCK  cannabinoid type-1 receptor (Cnr1)  RS CCK  FS-PV  PYR Htr3a  PVparvalbumin RS-INT  FS-INT  FS  mRNA  RS-INT  PYR  PYR  Camk2a  Neurod6 Neurod6  PYR Grik3kainate receptor GluR7 FS-INT \n\nINT CA1-PYR Fig.2C RS-INT  FS-INT  CA1-PYR Fig.2D CA1 \n\n\n# Fig. 3 Combined electrophysiological and transcriptional analysis\n\n\n\nFig. 3 \n\n 140 voltage-gated ion channelsFig.3A RS-INTFS-INT  CA1-PYR Fig.3A RS-INT  FS-INT  CA1-PYR Fig.3B,C FS-INT  Na+  K+ Fig.3D,E\n\nn = 11Fig.3A,E 0.35  -0.35 /Fig.3E AP  FS-INT  Na+  K+ Fig.3E Kcnc1  Kcnc2 AP  FS-INT  Kcnc1  Kcnc2  FS PV  RNA \n\n FS-INT  Hcn1  Hcn  Ih  Ih  Hcn1  Kcnc3Scn1a  Trpc6  FS Fig.3E PV  Kcng4  FS \n\n RNA \n\n\n# Fig. 4 Hierarchy of CAMs in single neurons\n\n\n\nFig. 4  CAMs \n\n CAMs  383.2  1.1 RS-INTFS-INT  CA1-PYR  CAMs  CAMs  CAMs Fig.4A RS-INTFS-INT  CA1-PYR  NRS-INT = 121.6  11.2NFS-INT = 128.7  12.7NCA1-PYR = 82.7  4.8  CAM Fig.4B CAMs  NRS-INT vs. FS-INT = 48NRS-INT vs. CA1-PYR = 58NFS-INT vs. CA1-PYR = 62  CAMs PYRs  30  CAMs RS-INTFS-INT  CA1-PYR  AppChl1Cntn1NptnNrxn1PtprsFig.4C\n\n\n# Fig. 5 Cell-specific expression of exocytosis and RhoGAP/RhoGEF signal transducer molecules\n\n\n\nFig. 5  RhoGAP/RhoGEF \n\nCAMs  CAMs RhoGAP/RhoGEF Fig.5A,BSnap25Synaptobrevin-2Vamp2Snap47  Syt11 Cplx1Stx1aSyt13  Synaptobrevin-1Vamp1 Sncg  Syt6  RS-INT \n\n RhoGAP  RhoGEF Rho/rac/CDC42 RhoGAP  RhoGEF  RS-INTFS-INT  CA1-PYR  RhoGAP  RhoGEF Fig.5C,D chimaerin-1Chn1 CA1-PYR \n\n\n# Fig. 6 Coregulation and coexpression of synaptic molecules\n\n\n\nFig. 6 \n\n CAMsexocytosisRhoGAPs/RhoGEFs  632  428 \n\nFig.6A,BFig. 6EFig. 6C 269  692  "hubs"\n\nFig.6D RhoGAP/RhoGEF  20 percentile Fig.6EFig.6E 1  2  >2 Fig.6F 2 P21  1Fig.6F\n\n Fig.6B  RS-INTFS-INT  CA1-PYR  RS-INT  FS-INT  GABA  1  2  80%  Fig.6D Fig.6G RhoGEF  RhoGAP \n\n\n# Fig. 7 Molecular identity of BS-PYR and RS-PYR neurons in the subiculum\n\n\n\nFig. 7 subiculum BS-PYR  RS-PYR \n\n Rho iii12 RNA  subiculum  BS-PYR  RS-PYR Fig.7A CA1-PYR \n\n BS-PYR  RS-PYR  9,671  278  9,499  405 Fig.7B CA1-PYR Fig.7CFig.7DcortistatinCort c-Fos FigfBS-PYR  RS-PYR Fig.7EFig.7F RhoGAP-/RhoGEF \n\n CA1  PYR CAMssubiculum PYR  CA1-PYR Fig.4C,7G 42  CAM  subiculum PYR  CAM  CA1-PYR  CAM Subiculum PYR  CA1-PYR  CAMClstn3CA1Ptpn5  Ptprn2 FS-PV  Lrrc4 CA1-PYR  subiculum PYR  BS-PYR  RS-PYR PCAFig.7H\n\nsubiculum  Fig.6  CA1  subiculum PYR  RNA  CA1 RSFS  PYR Fig.7I CA1  subgraph 1  subgraph 2 \n\n\n# Discussion\n\n RNA  RNA  RS  FS  CA1-PYR  CA1-PYR subiculum BS-PYR  RS-PYR  mRNA \n\n CAMs CAMs ',normalizedContent:':  rnaseq \ndoi: https://doi.org/10.1073/pnas.1610155113\ncite: foldy c, darmanis s, aoto j, malenka rc, quake sr, sudhof tc. single-cell rnaseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons. proc natl acad sci u s a. 2016;113(35):e5222-e5231.\n\n\n\n rnaseq \n\n\n\ncsaba foldy            thomas sudhof\n                       \nuniversity of zurich   stanford university\nfoldy@hifo.uzh.ch      tcs1@stanford.edu\n\n\n# abstract\n\n scrnaseq  rho/rac \n\n\n# figure\n\n\n# fig. 1 rnaseq of the total hippocampus during development: analysis of the changing landscape of cams\n\n\n\nfig. 1  rnaseqcams \n\n a  mrna  rna fig.1a p0p7p14p21p28 fig.1b,c cams 406 fig.1dfig.1eappcntnnrxnptpr ncam1  sdc3 p0  p28  60%fig.1f,g\n\n\n# fig. 2 single-cell rnaseq in situ of electrophysiologically characterized neurons in identified microcircuits\n\n\n\nfig. 2  rnaseq \n\n rna  ca1-pyr  fs  rs  target ca1-pyr  fs  pyr aps fs  rs fig.2afs  pv+ - rs  cck+  rna fig.2a rna  18  rs-int9  fs-int  14  ca1-pyr \n\n rs-intfs-int  pyr fig.2bgad1  gad2  pyr  cck  rs cck  cck  cannabinoid type-1 receptor (cnr1)  rs cck  fs-pv  pyr htr3a  pvparvalbumin rs-int  fs-int  fs  mrna  rs-int  pyr  pyr  camk2a  neurod6 neurod6  pyr grik3kainate receptor glur7 fs-int \n\nint ca1-pyr fig.2c rs-int  fs-int  ca1-pyr fig.2d ca1 \n\n\n# fig. 3 combined electrophysiological and transcriptional analysis\n\n\n\nfig. 3 \n\n 140 voltage-gated ion channelsfig.3a rs-intfs-int  ca1-pyr fig.3a rs-int  fs-int  ca1-pyr fig.3b,c fs-int  na+  k+ fig.3d,e\n\nn = 11fig.3a,e 0.35  -0.35 /fig.3e ap  fs-int  na+  k+ fig.3e kcnc1  kcnc2 ap  fs-int  kcnc1  kcnc2  fs pv  rna \n\n fs-int  hcn1  hcn  ih  ih  hcn1  kcnc3scn1a  trpc6  fs fig.3e pv  kcng4  fs \n\n rna \n\n\n# fig. 4 hierarchy of cams in single neurons\n\n\n\nfig. 4  cams \n\n cams  383.2  1.1 rs-intfs-int  ca1-pyr  cams  cams  cams fig.4a rs-intfs-int  ca1-pyr  nrs-int = 121.6  11.2nfs-int = 128.7  12.7nca1-pyr = 82.7  4.8  cam fig.4b cams  nrs-int vs. fs-int = 48nrs-int vs. ca1-pyr = 58nfs-int vs. ca1-pyr = 62  cams pyrs  30  cams rs-intfs-int  ca1-pyr  appchl1cntn1nptnnrxn1ptprsfig.4c\n\n\n# fig. 5 cell-specific expression of exocytosis and rhogap/rhogef signal transducer molecules\n\n\n\nfig. 5  rhogap/rhogef \n\ncams  cams rhogap/rhogef fig.5a,bsnap25synaptobrevin-2vamp2snap47  syt11 cplx1stx1asyt13  synaptobrevin-1vamp1 sncg  syt6  rs-int \n\n rhogap  rhogef rho/rac/cdc42 rhogap  rhogef  rs-intfs-int  ca1-pyr  rhogap  rhogef fig.5c,d chimaerin-1chn1 ca1-pyr \n\n\n# fig. 6 coregulation and coexpression of synaptic molecules\n\n\n\nfig. 6 \n\n camsexocytosisrhogaps/rhogefs  632  428 \n\nfig.6a,bfig. 6efig. 6c 269  692  "hubs"\n\nfig.6d rhogap/rhogef  20 percentile fig.6efig.6e 1  2  >2 fig.6f 2 p21  1fig.6f\n\n fig.6b  rs-intfs-int  ca1-pyr  rs-int  fs-int  gaba  1  2  80%  fig.6d fig.6g rhogef  rhogap \n\n\n# fig. 7 molecular identity of bs-pyr and rs-pyr neurons in the subiculum\n\n\n\nfig. 7 subiculum bs-pyr  rs-pyr \n\n rho iii12 rna  subiculum  bs-pyr  rs-pyr fig.7a ca1-pyr \n\n bs-pyr  rs-pyr  9,671  278  9,499  405 fig.7b ca1-pyr fig.7cfig.7dcortistatincort c-fos figfbs-pyr  rs-pyr fig.7efig.7f rhogap-/rhogef \n\n ca1  pyr camssubiculum pyr  ca1-pyr fig.4c,7g 42  cam  subiculum pyr  cam  ca1-pyr  cam subiculum pyr  ca1-pyr  camclstn3ca1ptpn5  ptprn2 fs-pv  lrrc4 ca1-pyr  subiculum pyr  bs-pyr  rs-pyr pcafig.7h\n\nsubiculum  fig.6  ca1  subiculum pyr  rna  ca1 rsfs  pyr fig.7i ca1  subgraph 1  subgraph 2 \n\n\n# discussion\n\n rna  rna  rs  fs  ca1-pyr  ca1-pyr subiculum bs-pyr  rs-pyr  mrna \n\n cams cams ',charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Molecular atlas of the adult mouse brain",frontmatter:{title:" --- Molecular atlas of the adult mouse brain",date:"2023-10-28T15:51:11.000Z",permalink:"/pages/37a825/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20200626.(2020.06.26%20%E9%BC%A0%E8%84%91%E5%88%86%E5%AD%90%E5%9B%BE%E8%B0%B1%20Sci%20Adv)%20Molecular%20atlas%20of%20the%20adult%20mouse%20brain.html",relativePath:"02./11./04./20200626.(2020.06.26  Sci Adv) Molecular atlas of the adult mouse brain.md",key:"v-9245513c",path:"/pages/37a825/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:414},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:728},{level:3,title:"Fig. 1. ST to generate a whole-brain molecular atlas",slug:"fig-1-st-to-generate-a-whole-brain-molecular-atlas",normalizedTitle:"fig. 1. st to generate a whole-brain molecular atlas",charIndex:740},{level:3,title:"Fig. 2. Computational approaches to extract new spatial domains from the molecular atlas",slug:"fig-2-computational-approaches-to-extract-new-spatial-domains-from-the-molecular-atlas",normalizedTitle:"fig. 2. computational approaches to extract new spatial domains from the molecular atlas",charIndex:1292},{level:3,title:"Fig. 3. New molecular definitions of subregions and layers in isocortex and striatum",slug:"fig-3-new-molecular-definitions-of-subregions-and-layers-in-isocortex-and-striatum",normalizedTitle:"fig. 3. new molecular definitions of subregions and layers in isocortex and striatum",charIndex:2078},{level:3,title:"Fig. 4. Mapping the spatial origin of single cells using the molecular atlas",slug:"fig-4-mapping-the-spatial-origin-of-single-cells-using-the-molecular-atlas",normalizedTitle:"fig. 4. mapping the spatial origin of single cells using the molecular atlas",charIndex:3424},{level:3,title:"Fig. 5. A reduced brain palette captures the whole-brain spatial organization",slug:"fig-5-a-reduced-brain-palette-captures-the-whole-brain-spatial-organization",normalizedTitle:"fig. 5. a reduced brain palette captures the whole-brain spatial organization",charIndex:3987}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1126/sciadv.abb3446" target="_blank" rel="noopener noreferrer">https://doi.org/10.1126/sciadv.abb3446<OutboundLink/></a><br>\nCite: Cantin Ortiz et al. ,Molecular atlas of the adult mouse brain. <em>Sci. Adv.</em> 6,eabb3446(2020).</p>\n',headersStr:"Abstract Results Fig. 1. ST to generate a whole-brain molecular atlas Fig. 2. Computational approaches to extract new spatial domains from the molecular atlas Fig. 3. New molecular definitions of subregions and layers in isocortex and striatum Fig. 4. Mapping the spatial origin of single cells using the molecular atlas Fig. 5. A reduced brain palette captures the whole-brain spatial organization",content:": \nDOI: https://doi.org/10.1126/sciadv.abb3446\nCite: Cantin Ortiz et al. ,Molecular atlas of the adult mouse brain. Sci. Adv. 6,eabb3446(2020).\n\n\n\n\n\n\n\nKONSTANTINOS MELETIS             JOAKIM LUNDEBERG\n                                 \nKarolinska Institutet            KTH Royal Institute of Technology\njoakim.lundeberg@scilifelab.se   joakim.lundeberg@scilifelab.se\n\n\n# Abstract\n\n- isocortex  hippocampus striatum  RNA \n\n\n# Results\n\n\n# Fig. 1. ST to generate a whole-brain molecular atlas\n\n\n\nFig. 1. \n\n(A) \n(B)  3D H&E  ST array  ABA \n(C) (B)  ST spot  ABA \n(D)  spots  3D \n(E)  spots  ABA \n(F)  isocortexRasgrf2striatumGpr88 thalamusRora ABA  ISH  3D  95  spots ABA  striatum thalamus 3D \n\n\n# Fig. 2. Computational approaches to extract new spatial domains from the molecular atlas\n\n\n\nFig. 2. \n\n(A) independent component (ICs) IC  IC  2D  3D  IC biological ICtechnical IC\n(B) 2D t-SNE  181  molecular cluster27 unique colors spots\n(C) 3D t-SNE  clusters \n(D)  3D  spots\n(E) 2D t-SNE  spots\n(F) 2D t-SNE  ABA  spots\n(G) 181 Molecular clusters  ABA  ABA  ABA \n(H)  clusters ABA  clustersMolecular clusters \n(I) ABA  molecular clusters \n(J) \n\n\n# Fig. 3. New molecular definitions of subregions and layers in isocortex and striatum\n\n\n\nFig. 3. isocortex  striatum \n\n(A) t-SNE  isocortical clusters\n(B) t-SNE  ABA  layer identity  spots \n(C) Circular phylogram  isocortical clusters  layer identity\n(D) isocortical molecular clusters  layer markers averaged z scored\n(E)  isocortical molecular clusters  isocortical subregions \n(F)  isocortical clusters AP axis spots clusters\n(G-I)  isocortical domains (G) anterior isocortex (PL, ILA, ACA, and MO)  molecular clusters(H) isocortical  molecular clustersSSMOp and PTLp(I)  medial  posterior  isocortical  molecular clustersRSPPOST and PRE\n(J) t-SNE  striatal clusters\n(K) t-SNE  ABA reference  spots STRv, ventral striatum; STRd, dorsal striatum; sAMY, striatum-like amygdala nuclei; LSX, lateral septal complex\n(L) Circular phylogram  striatal clusters  ABA \n(M) striatal molecular clusters  reference markers averaged z scored\n(N)  striatal clusters \n(O) striatal clusters ABA striatal DEM spots AEJLOBD layer identityKMstriatal identity\n\n\n# Fig. 4. Mapping the spatial origin of single cells using the molecular atlas\n\n\n\nFig. 4. \n\n(A) NN\n(B)  ALM VISp Molecular atlas clusters ABA reference3D  molecular clusters\n(C) glutamatergic\n(D)  VISp  scRNA-seq clusters ALMABA  scRNA-seq \n(E)  NN  scRNA-seq 13323,822 scRNA-seq  ABA \n\n\n# Fig. 5. A reduced brain palette captures the whole-brain spatial organization\n\n\n\nFig. 5. \n\n(A)  normalized mutual information index (NMI)  brain palette NMI  top IC loads top SVM weights\n(B) t-SNE  brain palette266 genes 181 \n(C) t-SNE  spots ABA reference colors\n(D) Circular phylogram  brain palette atlas  181  ABA \n(E) y axis brain palette x axis\n(F)  brain palette \n(G) brain palette  isocorticalretrohippocampal  olfactory clusters \n(H) brain palette  isocortical clusters  3D BEGH",normalizedContent:": \ndoi: https://doi.org/10.1126/sciadv.abb3446\ncite: cantin ortiz et al. ,molecular atlas of the adult mouse brain. sci. adv. 6,eabb3446(2020).\n\n\n\n\n\n\n\nkonstantinos meletis             joakim lundeberg\n                                 \nkarolinska institutet            kth royal institute of technology\njoakim.lundeberg@scilifelab.se   joakim.lundeberg@scilifelab.se\n\n\n# abstract\n\n- isocortex  hippocampus striatum  rna \n\n\n# results\n\n\n# fig. 1. st to generate a whole-brain molecular atlas\n\n\n\nfig. 1. \n\n(a) \n(b)  3d h&e  st array  aba \n(c) (b)  st spot  aba \n(d)  spots  3d \n(e)  spots  aba \n(f)  isocortexrasgrf2striatumgpr88 thalamusrora aba  ish  3d  95  spots aba  striatum thalamus 3d \n\n\n# fig. 2. computational approaches to extract new spatial domains from the molecular atlas\n\n\n\nfig. 2. \n\n(a) independent component (ics) ic  ic  2d  3d  ic biological ictechnical ic\n(b) 2d t-sne  181  molecular cluster27 unique colors spots\n(c) 3d t-sne  clusters \n(d)  3d  spots\n(e) 2d t-sne  spots\n(f) 2d t-sne  aba  spots\n(g) 181 molecular clusters  aba  aba  aba \n(h)  clusters aba  clustersmolecular clusters \n(i) aba  molecular clusters \n(j) \n\n\n# fig. 3. new molecular definitions of subregions and layers in isocortex and striatum\n\n\n\nfig. 3. isocortex  striatum \n\n(a) t-sne  isocortical clusters\n(b) t-sne  aba  layer identity  spots \n(c) circular phylogram  isocortical clusters  layer identity\n(d) isocortical molecular clusters  layer markers averaged z scored\n(e)  isocortical molecular clusters  isocortical subregions \n(f)  isocortical clusters ap axis spots clusters\n(g-i)  isocortical domains (g) anterior isocortex (pl, ila, aca, and mo)  molecular clusters(h) isocortical  molecular clustersssmop and ptlp(i)  medial  posterior  isocortical  molecular clustersrsppost and pre\n(j) t-sne  striatal clusters\n(k) t-sne  aba reference  spots strv, ventral striatum; strd, dorsal striatum; samy, striatum-like amygdala nuclei; lsx, lateral septal complex\n(l) circular phylogram  striatal clusters  aba \n(m) striatal molecular clusters  reference markers averaged z scored\n(n)  striatal clusters \n(o) striatal clusters aba striatal dem spots aejlobd layer identitykmstriatal identity\n\n\n# fig. 4. mapping the spatial origin of single cells using the molecular atlas\n\n\n\nfig. 4. \n\n(a) nn\n(b)  alm visp molecular atlas clusters aba reference3d  molecular clusters\n(c) glutamatergic\n(d)  visp  scrna-seq clusters almaba  scrna-seq \n(e)  nn  scrna-seq 13323,822 scrna-seq  aba \n\n\n# fig. 5. a reduced brain palette captures the whole-brain spatial organization\n\n\n\nfig. 5. \n\n(a)  normalized mutual information index (nmi)  brain palette nmi  top ic loads top svm weights\n(b) t-sne  brain palette266 genes 181 \n(c) t-sne  spots aba reference colors\n(d) circular phylogram  brain palette atlas  181  aba \n(e) y axis brain palette x axis\n(f)  brain palette \n(g) brain palette  isocorticalretrohippocampal  olfactory clusters \n(h) brain palette  isocortical clusters  3d begh",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Decoding the development of the human hippocampus",frontmatter:{title:" --- Decoding the development of the human hippocampus",categories:[""],tags:[""],date:"2022-11-19T10:07:52.000Z",permalink:"/pages/0b79fa/",readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20200115.(2020.01.15%20%E4%BA%BA%E7%B1%BB%E6%B5%B7%E9%A9%AC%20nature)%20Decoding%20the%20development%20of%20the%20human%20%20hippocampus.html",relativePath:"02./11./04./20200115.(2020.01.15  nature) Decoding the development of the human  hippocampus.md",key:"v-1f4da734",path:"/pages/0b79fa/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:357},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:799},{level:3,title:"Fig1. ",slug:"fig1-",normalizedTitle:"fig1. ",charIndex:811},{level:3,title:"Fig2. ",slug:"fig2-",normalizedTitle:"fig2. ",charIndex:1878},{level:3,title:"Fig3. ",slug:"fig3-",normalizedTitle:"fig3. ",charIndex:2863},{level:3,title:"Fig4. ",slug:"fig4-",normalizedTitle:"fig4. ",charIndex:3594}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-019-1917-5" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-019-1917-5<OutboundLink/></a><br>\nCite: Zhong S, Ding W, Sun L, et al. Decoding the development of the human hippocampus. <em>Nature</em> 577, 531536 (2020).</p>\n',headersStr:"Abstract Results Fig1.  Fig2.  Fig3.  Fig4. ",content:": \nDOI: https://doi.org/10.1038/s41586-019-1917-5\nCite: Zhong S, Ding W, Sun L, et al. Decoding the development of the human hippocampus. Nature 577, 531536 (2020).\n\n\n\n\n\n\n\nXIAOQUN WANG()        QIAN WU()\n                         \n             \nxiaoqunwang@bnu.edu.cn   qianwu@bnu.edu.cn\n\n\n# Abstract\n\n scRNAseq  ATAC-seq  16-27  30,416  47  PAX6+  HOPX+  CA1CA3  DG  DG  PROX1  16-20  0-5  NBPF1  PROX1+ \n\n\n# Results\n\n\n# Fig1. \n\n\n\nFig1. \n\na. 11 major clusters  t-SNE 3D progenitors()Non-DG ExN(DG)DG ExN(DG)Astrocyte()Microglia()CGE-derived InN(CGE-)MGE-derived InN(MGE-)CajalRetzius(CR)OPC()Oligodendrocyte()Endothelial cell()\nb. 11 major clusters  t-SNE 2D \nc. markers gene PROX1  DG ExN  markerLHX6  MGE-derived InN  markerNR2F2  CGE-derived InN  marker\nd. MEIS2  PROX1\nef. PROX1 DG ATAC-seq  PROX1  LEF1  TCF4  motif\ng.  47AC(astrocyte); P(progenitor); CR(CajalRetzius cell); M(microglia); EC(endothelial cell)\nh. (HP)(PFC)\ni. PFC  HP SOX4  SOX11  SOXC \njk. HP non-DG ExN  PFC ExN  HP DG ExN  PFC ExN PFC  HP  InN  HP MGE-derived InN  HP CGE-derived InN \nlm. OLIG2  MBP GW16 HP  MBP+ cells GW16 PFC  MBP+ cells\n\n\n# Fig2. \n\n\n\nFig2. \n\na. \nb. \ncd. PAX6+  HOPX+  PAX6  HOPX GW11(cortical hemCH)(dentate neuroepitheliumDNE)(ammonic neuroepitheliumANE)DNE  ANE  SOX2 ANE  PAX6+ SOX2+ DNE  HOPX+ SOX2+ \ne.  GW14  PAX6+ (ventricular zone) DG (PROX1+) primary matrix (I)  secondary matrix (II)HOPX+  pial side\nf.  GW16 PAX6+  HOPX+  hilus DG  tertiary matrix (III)PAX6+  HOPX+   GW22 PAX6+  DG  HOPX+  hilus HOPX+  cornu ammonis (CA)\ngh.  GW25  CA  DG  PAX6+ NEUROD1+  CA  PAX6+ GFAP+  HOXP+ \ni. PAX6+  HOPX+  GW11 to GW22 \n\n\n# Fig3. \n\n\n\nFig3. \n\na.  t-SNE \nb. \nc.  marker genes DGSEMA5ACA1PID1CA3SULF2NRIP3\nd. DGSEMA5AExN 03CA1PID1ExN 01CA3SULF2NRIP3ExN 02 CA1  CA3  DG \ne-g.  GW  DEGs  GO GW16-18 GW20-22 GW25-27 \nh. WGCNA \nij.  ATAC-seq  PROX1 motif  KCNJ6NFIADUSP1NPTX2 i KCNJ6( GIRK2)G-- K+ \nkl.  MGE  CGE  MGE-derived InN (LHX6+)  CGE-derived InN (NR2F1/2+)  CCKCALB2VIP  InN  CGE  SATB1SST  InN  MGE \n\n\n# Fig4. \n\n\n\nFig4. \n\na.  GW16-20  P0-5 \nb.  DEGs\nc.  DEGs  t-SNE ATAC-seq \nd-g. (NBPF) DUF1220 NBPFNBPF1 8  DUF1220  NBPF1  NBPF1 13.5(E13.5) NBPF1 PROX1+  E15.5  E18.5 ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-019-1917-5\ncite: zhong s, ding w, sun l, et al. decoding the development of the human hippocampus. nature 577, 531536 (2020).\n\n\n\n\n\n\n\nxiaoqun wang()        qian wu()\n                         \n             \nxiaoqunwang@bnu.edu.cn   qianwu@bnu.edu.cn\n\n\n# abstract\n\n scrnaseq  atac-seq  16-27  30,416  47  pax6+  hopx+  ca1ca3  dg  dg  prox1  16-20  0-5  nbpf1  prox1+ \n\n\n# results\n\n\n# fig1. \n\n\n\nfig1. \n\na. 11 major clusters  t-sne 3d progenitors()non-dg exn(dg)dg exn(dg)astrocyte()microglia()cge-derived inn(cge-)mge-derived inn(mge-)cajalretzius(cr)opc()oligodendrocyte()endothelial cell()\nb. 11 major clusters  t-sne 2d \nc. markers gene prox1  dg exn  markerlhx6  mge-derived inn  markernr2f2  cge-derived inn  marker\nd. meis2  prox1\nef. prox1 dg atac-seq  prox1  lef1  tcf4  motif\ng.  47ac(astrocyte); p(progenitor); cr(cajalretzius cell); m(microglia); ec(endothelial cell)\nh. (hp)(pfc)\ni. pfc  hp sox4  sox11  soxc \njk. hp non-dg exn  pfc exn  hp dg exn  pfc exn pfc  hp  inn  hp mge-derived inn  hp cge-derived inn \nlm. olig2  mbp gw16 hp  mbp+ cells gw16 pfc  mbp+ cells\n\n\n# fig2. \n\n\n\nfig2. \n\na. \nb. \ncd. pax6+  hopx+  pax6  hopx gw11(cortical hemch)(dentate neuroepitheliumdne)(ammonic neuroepitheliumane)dne  ane  sox2 ane  pax6+ sox2+ dne  hopx+ sox2+ \ne.  gw14  pax6+ (ventricular zone) dg (prox1+) primary matrix (i)  secondary matrix (ii)hopx+  pial side\nf.  gw16 pax6+  hopx+  hilus dg  tertiary matrix (iii)pax6+  hopx+   gw22 pax6+  dg  hopx+  hilus hopx+  cornu ammonis (ca)\ngh.  gw25  ca  dg  pax6+ neurod1+  ca  pax6+ gfap+  hoxp+ \ni. pax6+  hopx+  gw11 to gw22 \n\n\n# fig3. \n\n\n\nfig3. \n\na.  t-sne \nb. \nc.  marker genes dgsema5aca1pid1ca3sulf2nrip3\nd. dgsema5aexn 03ca1pid1exn 01ca3sulf2nrip3exn 02 ca1  ca3  dg \ne-g.  gw  degs  go gw16-18 gw20-22 gw25-27 \nh. wgcna \nij.  atac-seq  prox1 motif  kcnj6nfiadusp1nptx2 i kcnj6( girk2)g-- k+ \nkl.  mge  cge  mge-derived inn (lhx6+)  cge-derived inn (nr2f1/2+)  cckcalb2vip  inn  cge  satb1sst  inn  mge \n\n\n# fig4. \n\n\n\nfig4. \n\na.  gw16-20  p0-5 \nb.  degs\nc.  degs  t-sne atac-seq \nd-g. (nbpf) duf1220 nbpfnbpf1 8  duf1220  nbpf1  nbpf1 13.5(e13.5) nbpf1 prox1+  e15.5  e18.5 ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction",frontmatter:{title:" --- Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction",date:"2022-12-11T16:23:48.000Z",permalink:"/pages/c60442/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20190913.(2019.09.13%20%E5%BC%A0%E6%AF%85mPFC%20Nat%20Commun)%20Cell%20type-specific%20transcriptional%20programs%20in%20mouse%20prefrontal%20cortex%20during%20adolescence%20and%20addiction.html",relativePath:"02./11./04./20190913.(2019.09.13 mPFC Nat Commun) Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction.md",key:"v-3eebebda",path:"/pages/c60442/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:438},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:788},{level:3,title:"Fig1. Transcriptome-based cell classifification in mouse PFC",slug:"fig1-transcriptome-based-cell-classifification-in-mouse-pfc",normalizedTitle:"fig1. transcriptome-based cell classifification in mouse pfc",charIndex:800},{level:3,title:"Fig2. PFC contains distinct excitatory and inhibitory neuron subtypes",slug:"fig2-pfc-contains-distinct-excitatory-and-inhibitory-neuron-subtypes",normalizedTitle:"fig2. pfc contains distinct excitatory and inhibitory neuron subtypes",charIndex:1022},{level:3,title:"Fig3. Distinct transcriptional features of PFC excitatory neurons relative to those of VISp and ALM",slug:"fig3-distinct-transcriptional-features-of-pfc-excitatory-neurons-relative-to-those-of-visp-and-alm",normalizedTitle:"fig3. distinct transcriptional features of pfc excitatory neurons relative to those of visp and alm",charIndex:1572},{level:3,title:"Fig4. Widespread transcriptional changes in PFC cell types between P21 and P60 mice",slug:"fig4-widespread-transcriptional-changes-in-pfc-cell-types-between-p21-and-p60-mice",normalizedTitle:"fig4. widespread transcriptional changes in pfc cell types between p21 and p60 mice",charIndex:1831},{level:3,title:"Fig5. Expression of many GWAS candidates is enriched in specific clusters and dynamically changed during adolescence",slug:"fig5-expression-of-many-gwas-candidates-is-enriched-in-specific-clusters-and-dynamically-changed-during-adolescence",normalizedTitle:"fig5. expression of many gwas candidates is enriched in specific clusters and dynamically changed during adolescence",charIndex:2452},{level:3,title:"Fig6. Chronic cocaine IVSA induces transcriptional changes in multiple PFC cell types",slug:"fig6-chronic-cocaine-ivsa-induces-transcriptional-changes-in-multiple-pfc-cell-types",normalizedTitle:"fig6. chronic cocaine ivsa induces transcriptional changes in multiple pfc cell types",charIndex:3035}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41467-019-12054-3" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-019-12054-3<OutboundLink/></a><br>\nCite: Bhattacherjee, A., Djekidel, M.N., Chen, R. et al. Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction. <em>Nat Commun</em> 10, 4169 (2019).</p>\n',headersStr:"Abstract Results Fig1. Transcriptome-based cell classifification in mouse PFC Fig2. PFC contains distinct excitatory and inhibitory neuron subtypes Fig3. Distinct transcriptional features of PFC excitatory neurons relative to those of VISp and ALM Fig4. Widespread transcriptional changes in PFC cell types between P21 and P60 mice Fig5. Expression of many GWAS candidates is enriched in specific clusters and dynamically changed during adolescence Fig6. Chronic cocaine IVSA induces transcriptional changes in multiple PFC cell types",content:": \nDOI: https://doi.org/10.1038/s41467-019-12054-3\nCite: Bhattacherjee, A., Djekidel, M.N., Chen, R. et al. Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction. Nat Commun 10, 4169 (2019).\n\n\n\n\n\n\n\nYI ZHANG()\n\nBOSTON CHILDREN'S HOSPITAL\nBROAD INSTITUTE\nHARVARD MEDICAL SCHOOL\nyzhang@genetics.med.harvard.edu\n\n\n# Abstract\n\n(PFC)RNA(scRNA-seq)PFC(P21P60)PFC\n\n\n# Results\n\n\n# Fig1. Transcriptome-based cell classifification in mouse PFC\n\n\n\nFig1.  PFC \n\na. \nb. t-SNE  PFC \nc.  t-SNE \nd.  PFC \n\n\n# Fig2. PFC contains distinct excitatory and inhibitory neuron subtypes\n\n\n\nFig2. PFC \n\na. t-SNE PFC  13 \nb.  13 \nc.  FISH (L5  Pou3f1  Tshz2) Foxp2  L6)\nd.  t-SNE \ne.  L5  L5  Etv1 (Poh3f1+  Tshz2+)\nf. t-SNE  PFC  12 \ng.  12 \nh.  FISH (Cartpt+CartptSst)(Pbx3+inset)\n\n\n# Fig3. Distinct transcriptional features of PFC excitatory neurons relative to those of VISp and ALM\n\n\n\nFig3. PFC  VISp  ALM \n\na. t-SNE PFC \nb.  a  Cluster  marker\nc. PFC  VISp  ALM \n\n\n# Fig4. Widespread transcriptional changes in PFC cell types between P21 and P60 mice\n\n\n\nFig4. P21  P60  PFC \n\na. t-SNE  P60 P21PFC  correlation analysis(CCA) \nb. t-SNE  P60 P21PFC \nc. \nd.  clusters (cutoffFC>1.5 and q-value<0.05)\ne.  P21  P60  -log10(p-value)(right-tailed Fisher exact test)\nf.  P21  P60  PFC (cutoff: FC > 1.5 and q-value < 0.05; numbers 113 on left axis indicate the clusters)\n\n\n# Fig5. Expression of many GWAS candidates is enriched in specific clusters and dynamically changed during adolescence\n\n\n\nFig5.  GWAS  Clusters \n\na. t-SNE  GWAS Top-panel  global t-SNE  4  bottom panels  clusters  t-SNEAllenBrainISH \nb. \nc.  GWAS \nd.  12  pfc  GWAS \ne. t-sne  P21  P60  GWAS \n\n\n# Fig6. Chronic cocaine IVSA induces transcriptional changes in multiple PFC cell types\n\n\n\nFig6.  IVSA  PFC \n\na.  IVSA  BioRender.com \nb.  IVSA  15 \nc. t-sne \nd. \ne.  clusters  clusters  15  30%\nf.  15  clusters  -log10(p)",normalizedContent:": \ndoi: https://doi.org/10.1038/s41467-019-12054-3\ncite: bhattacherjee, a., djekidel, m.n., chen, r. et al. cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction. nat commun 10, 4169 (2019).\n\n\n\n\n\n\n\nyi zhang()\n\nboston children's hospital\nbroad institute\nharvard medical school\nyzhang@genetics.med.harvard.edu\n\n\n# abstract\n\n(pfc)rna(scrna-seq)pfc(p21p60)pfc\n\n\n# results\n\n\n# fig1. transcriptome-based cell classifification in mouse pfc\n\n\n\nfig1.  pfc \n\na. \nb. t-sne  pfc \nc.  t-sne \nd.  pfc \n\n\n# fig2. pfc contains distinct excitatory and inhibitory neuron subtypes\n\n\n\nfig2. pfc \n\na. t-sne pfc  13 \nb.  13 \nc.  fish (l5  pou3f1  tshz2) foxp2  l6)\nd.  t-sne \ne.  l5  l5  etv1 (poh3f1+  tshz2+)\nf. t-sne  pfc  12 \ng.  12 \nh.  fish (cartpt+cartptsst)(pbx3+inset)\n\n\n# fig3. distinct transcriptional features of pfc excitatory neurons relative to those of visp and alm\n\n\n\nfig3. pfc  visp  alm \n\na. t-sne pfc \nb.  a  cluster  marker\nc. pfc  visp  alm \n\n\n# fig4. widespread transcriptional changes in pfc cell types between p21 and p60 mice\n\n\n\nfig4. p21  p60  pfc \n\na. t-sne  p60 p21pfc  correlation analysis(cca) \nb. t-sne  p60 p21pfc \nc. \nd.  clusters (cutofffc>1.5 and q-value<0.05)\ne.  p21  p60  -log10(p-value)(right-tailed fisher exact test)\nf.  p21  p60  pfc (cutoff: fc > 1.5 and q-value < 0.05; numbers 113 on left axis indicate the clusters)\n\n\n# fig5. expression of many gwas candidates is enriched in specific clusters and dynamically changed during adolescence\n\n\n\nfig5.  gwas  clusters \n\na. t-sne  gwas top-panel  global t-sne  4  bottom panels  clusters  t-sneallenbrainish \nb. \nc.  gwas \nd.  12  pfc  gwas \ne. t-sne  p21  p60  gwas \n\n\n# fig6. chronic cocaine ivsa induces transcriptional changes in multiple pfc cell types\n\n\n\nfig6.  ivsa  pfc \n\na.  ivsa  biorender.com \nb.  ivsa  15 \nc. t-sne \nd. \ne.  clusters  clusters  15  30%\nf.  15  clusters  -log10(p)",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single-cell transcriptomic evidence for dense intracortical neuropeptide networks",frontmatter:{title:" --- Single-cell transcriptomic evidence for dense intracortical neuropeptide networks",date:"2023-11-03T10:59:09.000Z",permalink:"/pages/bb2e69/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20191111.(2019.11.11%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A5%9E%E7%BB%8F%E8%82%BD%E7%BD%91%E7%BB%9C%20Elife)%20Single-cell%20transcriptomic%20evidence%20for%20dense%20intracortical%20neuropeptide%20networks.html",relativePath:"02./11./04./20191111.(2019.11.11  Elife) Single-cell transcriptomic evidence for dense intracortical neuropeptide networks.md",key:"v-13e0acb8",path:"/pages/bb2e69/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:348},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:746},{level:3,title:"Figure 1. Single-cell NPP gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 18 NPP genes are highly differential within both neocortical areas but highly conserved between areas",slug:"figure-1-single-cell-npp-gene-expression-and-co-expression-statistics-for-distant-neocortical-areas-visp-and-alm-show-that-expression-patterns-for-18-npp-genes-are-highly-differential-within-both-neocortical-areas-but-highly-conserved-between-areas",normalizedTitle:"figure 1. single-cell npp gene expression and co-expression statistics for distant neocortical areas visp and alm show that expression patterns for 18 npp genes are highly differential within both neocortical areas but highly conserved between areas",charIndex:758},{level:3,title:"Figure 2. Single-cell NP-GPCR gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 29 NP-GPCR genes are highly differential within neocortical areas but conserved between areas",slug:"figure-2-single-cell-np-gpcr-gene-expression-and-co-expression-statistics-for-distant-neocortical-areas-visp-and-alm-show-that-expression-patterns-for-29-np-gpcr-genes-are-highly-differential-within-neocortical-areas-but-conserved-between-areas",normalizedTitle:"figure 2. single-cell np-gpcr gene expression and co-expression statistics for distant neocortical areas visp and alm show that expression patterns for 29 np-gpcr genes are highly differential within neocortical areas but conserved between areas",charIndex:1562},{level:3,title:"Figure 3",slug:"figure-3",normalizedTitle:"figure 3",charIndex:2384},{level:3,title:"Figure 4. Neurons can be clustered effectively based on just 47 NP genes (18 NPP and 29 NP-GPCR)",slug:"figure-4-neurons-can-be-clustered-effectively-based-on-just-47-np-genes-18-npp-and-29-np-gpcr",normalizedTitle:"figure 4. neurons can be clustered effectively based on just 47 np genes (18 npp and 29 np-gpcr)",charIndex:3024},{level:3,title:"Figure 5. Neurotaxonomic heatmaps show strong conservation of NPP and NP-GPCR expression patterns between two distant neocortical areas",slug:"figure-5-neurotaxonomic-heatmaps-show-strong-conservation-of-npp-and-np-gpcr-expression-patterns-between-two-distant-neocortical-areas",normalizedTitle:"figure 5. neurotaxonomic heatmaps show strong conservation of npp and np-gpcr expression patterns between two distant neocortical areas",charIndex:3952},{level:3,title:"Figure 6. Neurotaxonomic expression profiles of 37 cognate NPP/NP-GPCR pairs predict 37 peptidergic networks",slug:"figure-6-neurotaxonomic-expression-profiles-of-37-cognate-npp-np-gpcr-pairs-predict-37-peptidergic-networks",normalizedTitle:"figure 6. neurotaxonomic expression profiles of 37 cognate npp/np-gpcr pairs predict 37 peptidergic networks",charIndex:4458},{level:3,title:"Figure 7. Pooling NP-GPCRs by primary Ga family enables neurotaxonomic prediction of primary NP-GPCR signaling impacts across 37 cognate NP pairs (see text) for each of areas VISp and ALM",slug:"figure-7-pooling-np-gpcrs-by-primary-ga-family-enables-neurotaxonomic-prediction-of-primary-np-gpcr-signaling-impacts-across-37-cognate-np-pairs-see-text-for-each-of-areas-visp-and-alm",normalizedTitle:"figure 7. pooling np-gpcrs by primary ga family enables neurotaxonomic prediction of primary np-gpcr signaling impacts across 37 cognate np pairs (see text) for each of areas visp and alm",charIndex:5256},{level:3,title:"Figure 8. Neurotaxonomy offers a framework for integrating statistical descriptions of multiple modulatory and synaptic networks, as schematized here for purely fictitious neuron types, transcriptomic and connectomic data",slug:"figure-8-neurotaxonomy-offers-a-framework-for-integrating-statistical-descriptions-of-multiple-modulatory-and-synaptic-networks-as-schematized-here-for-purely-fictitious-neuron-types-transcriptomic-and-connectomic-data",normalizedTitle:"figure 8. neurotaxonomy offers a framework for integrating statistical descriptions of multiple modulatory and synaptic networks, as schematized here for purely fictitious neuron types, transcriptomic and connectomic data",charIndex:5841},{level:2,title:"Conclusions",slug:"conclusions",normalizedTitle:"conclusions",charIndex:6534}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.7554/eLife.47889" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.47889<OutboundLink/></a><br>\nCite: Smith SJ, Smbl U, Graybuck LT, et al. Single-cell transcriptomic evidence for dense intracortical neuropeptide networks. Elife. 2019;8:e47889.</p>\n',headersStr:"Abstract Results Figure 1. Single-cell NPP gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 18 NPP genes are highly differential within both neocortical areas but highly conserved between areas Figure 2. Single-cell NP-GPCR gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 29 NP-GPCR genes are highly differential within neocortical areas but conserved between areas Figure 3 Figure 4. Neurons can be clustered effectively based on just 47 NP genes (18 NPP and 29 NP-GPCR) Figure 5. Neurotaxonomic heatmaps show strong conservation of NPP and NP-GPCR expression patterns between two distant neocortical areas Figure 6. Neurotaxonomic expression profiles of 37 cognate NPP/NP-GPCR pairs predict 37 peptidergic networks Figure 7. Pooling NP-GPCRs by primary Ga family enables neurotaxonomic prediction of primary NP-GPCR signaling impacts across 37 cognate NP pairs (see text) for each of areas VISp and ALM Figure 8. Neurotaxonomy offers a framework for integrating statistical descriptions of multiple modulatory and synaptic networks, as schematized here for purely fictitious neuron types, transcriptomic and connectomic data Conclusions",content:": \nDOI: https://doi.org/10.7554/eLife.47889\nCite: Smith SJ, Smbl U, Graybuck LT, et al. Single-cell transcriptomic evidence for dense intracortical neuropeptide networks. Elife. 2019;8:e47889.\n\n\n\n\n\n\n\nMICHAEL HAWRYLYCZ\n\nAllen Institute for Brain Science\nMikeH@alleninstitute.org\n\n\n# Abstract\n\n 22,439  scRNA-seq NPP G NP-GPCR 18  NPPs  29  NP-GPCRs  NP  47  37  NPP/NP-GPCR  NP  37 \n\n\n# Results\n\n\n# Figure 1. Single-cell NPP gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 18 NPP genes are highly differential within both neocortical areas but highly conserved between areas\n\n\n\nFigure 1.  NPP  VISp  ALM 18  NPP \n\n(A)  NPP  22,439  VISp+ALM  NPP  CPM  18 IiiiiiCPM  1  NPP  A.i Max NPP Gene NPP  CPM \n(B)  13,491  VISp  8,948  ALM  NPP  NPP  A \n(C)  NPP  VISp  ALM \n\n\n# Figure 2. Single-cell NP-GPCR gene expression and co-expression statistics for distant neocortical areas VISp and ALM show that expression patterns for 29 NP-GPCR genes are highly differential within neocortical areas but conserved between areas\n\n\n\nFigure 2.  NP-GPCR  VISp  ALM  29  NP-GPCR \n\n(A)  NP-GPCR  22,439  VISp+ALM  NP-GPCR  NP-GPCR  CPM  29  5 i-v CPM  1 NP-GPCR  A. I maxNP-GPCR gene CPM NP-GPCR  CPM \n(B)  NP-GPCR  13,491  VISp  8,948  ALM  A  NP-GPCR \n(C)  NP-GPCR  VISp  ALM \n\n\n# Figure 3\n\n\n\nFigure 3\n\n(A-B)  VISp  ALM  22,439  (A) 18  NPP  (B) 29  NP-GPCR 115  VISp+ALM glutamatergic GABAergic  Trimmed-mean (5% trim) CPM  log10 scales  5 NPP 3 NP-GPCRITPTNPCTL6b  glutamatergic Lamp5SncgSerpinf1VipChodlSstPvalb  GABAergic 12\n(C)  11  18  NPP  CPM variation (CV) CV  CV/global CV = 0.239\n(D) 29  NP-GPCR  CV=0.427\n\n\n# Figure 4. Neurons can be clustered effectively based on just 47 NP genes (18 NPP and 29 NP-GPCR)\n\n\n\nFigure 4.  47  NP 18  NPP  29  NP-GPCR\n\n(A)  6083 highly expressed, HE 22,439  Tasic et al. (2018)  islands \n(B)  Z2  47  NP \n(C)  Z1  Z2\n(D)  RI resolution index, RI NP  RI  Tasic 2018 leaf nodes, RI = 1.0RI <1.0 RI  RI  HE 4020 DE 47  DE  47  NP  RI p<0.01bootstrap 47 Rand47, n = 100 subsets NP Rand47 ExpMatch, n = 100 subsets\n\n\n# Figure 5. Neurotaxonomic heatmaps show strong conservation of NPP and NP-GPCR expression patterns between two distant neocortical areas\n\n\n\nFigure 5.  NPP  NP-GPCR \n\n(A)  13,491  VISp 18  NPP  29  NP-GPCR \n(B)  ALM  8948  Figure 3  VISp  ALM  VISp  ALM  GABAergic Figure 5B  ALM  GABA Glutamatergic leaf\n\n\n# Figure 6. Neurotaxonomic expression profiles of 37 cognate NPP/NP-GPCR pairs predict 37 peptidergic networks\n\n\n\nFigure 6. 37  NPP/NP-GPCR  37 \n\n(A-E) A-EF local peptidergic coupling NPP  NP-GPCR CPM CPMCPM CPMCPMCPM\n(A)  Npy  Npy1r NP-GPCR  93  VISp \n(B) A 84  84  Npy-Npy1r A 84  ALM  glutamatergic ('E')  GABAergic ('I')  NPP/NP-GPCR  (E E), (E I), (I I) and (I E) Tasic 2018  Figure 3A \n(C-E)  G CGi/oDGsEGq/11  37  VISp  NPP/NP-GPCR \n(F)  A-E  CPM \n\n\n# Figure 7. Pooling NP-GPCRs by primary Ga family enables neurotaxonomic prediction of primary NP-GPCR signaling impacts across 37 cognate NP pairs (see text) for each of areas VISp and ALM\n\n\n\nFigure 7.  Ga  NP-GPCRs VISp  ALM  37  NP  NP-GPCR \n\n(A) ISQ  Tasic 2018  Gi/oGs Gq/11 Ga \n(B) A Ga \n(C) AB Tasic 2018  VISp  ALM  glutamatergic  GABAergic E EEIII  IE Figure 6 \n\n\n# Figure 8. Neurotaxonomy offers a framework for integrating statistical descriptions of multiple modulatory and synaptic networks, as schematized here for purely fictitious neuron types, transcriptomic and connectomic data\n\n\n\nFigure 8. \n\n(A-C) A-CDEF\n(A)  NPP Npp1,2,3 NP-GPCR Npr1,2,3a-c d-g \n(B) A NPP  NP-GPCR \n(C)  NP NP1,2,3-\n(D) \n(E) D\n(F) \n\n\n# Conclusions\n\n scRNAseq  scRNAseq  NPP  NP-GPCR  scRNAseq ",normalizedContent:": \ndoi: https://doi.org/10.7554/elife.47889\ncite: smith sj, sumbul u, graybuck lt, et al. single-cell transcriptomic evidence for dense intracortical neuropeptide networks. elife. 2019;8:e47889.\n\n\n\n\n\n\n\nmichael hawrylycz\n\nallen institute for brain science\nmikeh@alleninstitute.org\n\n\n# abstract\n\n 22,439  scrna-seq npp g np-gpcr 18  npps  29  np-gpcrs  np  47  37  npp/np-gpcr  np  37 \n\n\n# results\n\n\n# figure 1. single-cell npp gene expression and co-expression statistics for distant neocortical areas visp and alm show that expression patterns for 18 npp genes are highly differential within both neocortical areas but highly conserved between areas\n\n\n\nfigure 1.  npp  visp  alm 18  npp \n\n(a)  npp  22,439  visp+alm  npp  cpm  18 iiiiiicpm  1  npp  a.i max npp gene npp  cpm \n(b)  13,491  visp  8,948  alm  npp  npp  a \n(c)  npp  visp  alm \n\n\n# figure 2. single-cell np-gpcr gene expression and co-expression statistics for distant neocortical areas visp and alm show that expression patterns for 29 np-gpcr genes are highly differential within neocortical areas but conserved between areas\n\n\n\nfigure 2.  np-gpcr  visp  alm  29  np-gpcr \n\n(a)  np-gpcr  22,439  visp+alm  np-gpcr  np-gpcr  cpm  29  5 i-v cpm  1 np-gpcr  a. i maxnp-gpcr gene cpm np-gpcr  cpm \n(b)  np-gpcr  13,491  visp  8,948  alm  a  np-gpcr \n(c)  np-gpcr  visp  alm \n\n\n# figure 3\n\n\n\nfigure 3\n\n(a-b)  visp  alm  22,439  (a) 18  npp  (b) 29  np-gpcr 115  visp+alm glutamatergic gabaergic  trimmed-mean (5% trim) cpm  log10 scales  5 npp 3 np-gpcritptnpctl6b  glutamatergic lamp5sncgserpinf1vipchodlsstpvalb  gabaergic 12\n(c)  11  18  npp  cpm variation (cv) cv  cv/global cv = 0.239\n(d) 29  np-gpcr  cv=0.427\n\n\n# figure 4. neurons can be clustered effectively based on just 47 np genes (18 npp and 29 np-gpcr)\n\n\n\nfigure 4.  47  np 18  npp  29  np-gpcr\n\n(a)  6083 highly expressed, he 22,439  tasic et al. (2018)  islands \n(b)  z2  47  np \n(c)  z1  z2\n(d)  ri resolution index, ri np  ri  tasic 2018 leaf nodes, ri = 1.0ri <1.0 ri  ri  he 4020 de 47  de  47  np  ri p<0.01bootstrap 47 rand47, n = 100 subsets np rand47 expmatch, n = 100 subsets\n\n\n# figure 5. neurotaxonomic heatmaps show strong conservation of npp and np-gpcr expression patterns between two distant neocortical areas\n\n\n\nfigure 5.  npp  np-gpcr \n\n(a)  13,491  visp 18  npp  29  np-gpcr \n(b)  alm  8948  figure 3  visp  alm  visp  alm  gabaergic figure 5b  alm  gaba glutamatergic leaf\n\n\n# figure 6. neurotaxonomic expression profiles of 37 cognate npp/np-gpcr pairs predict 37 peptidergic networks\n\n\n\nfigure 6. 37  npp/np-gpcr  37 \n\n(a-e) a-ef local peptidergic coupling npp  np-gpcr cpm cpmcpm cpmcpmcpm\n(a)  npy  npy1r np-gpcr  93  visp \n(b) a 84  84  npy-npy1r a 84  alm  glutamatergic ('e')  gabaergic ('i')  npp/np-gpcr  (e e), (e i), (i i) and (i e) tasic 2018  figure 3a \n(c-e)  g cgi/odgsegq/11  37  visp  npp/np-gpcr \n(f)  a-e  cpm \n\n\n# figure 7. pooling np-gpcrs by primary ga family enables neurotaxonomic prediction of primary np-gpcr signaling impacts across 37 cognate np pairs (see text) for each of areas visp and alm\n\n\n\nfigure 7.  ga  np-gpcrs visp  alm  37  np  np-gpcr \n\n(a) isq  tasic 2018  gi/ogs gq/11 ga \n(b) a ga \n(c) ab tasic 2018  visp  alm  glutamatergic  gabaergic e eeiii  ie figure 6 \n\n\n# figure 8. neurotaxonomy offers a framework for integrating statistical descriptions of multiple modulatory and synaptic networks, as schematized here for purely fictitious neuron types, transcriptomic and connectomic data\n\n\n\nfigure 8. \n\n(a-c) a-cdef\n(a)  npp npp1,2,3 np-gpcr npr1,2,3a-c d-g \n(b) a npp  np-gpcr \n(c)  np np1,2,3-\n(d) \n(e) d\n(f) \n\n\n# conclusions\n\n scrnaseq  scrnaseq  npp  np-gpcr  scrnaseq ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing",frontmatter:{title:" --- Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing",date:"2024-02-19T19:22:45.000Z",permalink:"/pages/b5aaa4/",categories:[""],tags:["-"],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20201207.(2020.12.07%20%E8%82%A0%E7%A5%9E%E7%BB%8F%E5%85%83%20Nat%20Neurosci)%20Diversification%20of%20molecularly%20defined%20myenteric%20neuron%20classes%20revealed%20by%20single-cell%20RNA%20sequencing.html",relativePath:"02./11./04./20201207.(2020.12.07  Nat Neurosci) Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing.md",key:"v-cb41435c",path:"/pages/b5aaa4/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:392},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:743},{level:3,title:"Fig. 1 Molecular definition of 12 enteric neuron classes in the mouse small intestine myenteric plexus",slug:"fig-1-molecular-definition-of-12-enteric-neuron-classes-in-the-mouse-small-intestine-myenteric-plexus",normalizedTitle:"fig. 1 molecular definition of 12 enteric neuron classes in the mouse small intestine myenteric plexus",charIndex:754},{level:3,title:"Fig. 2 Gene categories conferring functional characteristics in enteric neuron classes",slug:"fig-2-gene-categories-conferring-functional-characteristics-in-enteric-neuron-classes",normalizedTitle:"fig. 2 gene categories conferring functional characteristics in enteric neuron classes",charIndex:2904},{level:3,title:"Fig. 3 Immunohistochemical validation of marker genes for enteric neuron classes",slug:"fig-3-immunohistochemical-validation-of-marker-genes-for-enteric-neuron-classes",normalizedTitle:"fig. 3 immunohistochemical validation of marker genes for enteric neuron classes",charIndex:4101},{level:3,title:"Fig. 4 Assessment of IPAN characteristics in ENCs 6, 7 and 12",slug:"fig-4-assessment-of-ipan-characteristics-in-encs-6-7-and-12",normalizedTitle:"fig. 4 assessment of ipan characteristics in encs 6, 7 and 12",charIndex:4773},{level:3,title:"Fig. 5 Single-cell transcriptome analysis of the developing ENS reveals transcriptional programs of generic cell states",slug:"fig-5-single-cell-transcriptome-analysis-of-the-developing-ens-reveals-transcriptional-programs-of-generic-cell-states",normalizedTitle:"fig. 5 single-cell transcriptome analysis of the developing ens reveals transcriptional programs of generic cell states",charIndex:6444},{level:3,title:"Fig. 6 ENC identities emerge through binary branching followed by postmitotic neuronal diversification",slug:"fig-6-enc-identities-emerge-through-binary-branching-followed-by-postmitotic-neuronal-diversification",normalizedTitle:"fig. 6 enc identities emerge through binary branching followed by postmitotic neuronal diversification",charIndex:7620},{level:3,title:"Fig. 7 Loss of PBX3 expression impairs the ENC8/9 to ENC12 transition",slug:"fig-7-loss-of-pbx3-expression-impairs-the-enc8-9-to-enc12-transition",normalizedTitle:"fig. 7 loss of pbx3 expression impairs the enc8/9 to enc12 transition",charIndex:9064},{level:3,title:"Fig. 8 Schematic of the major principle of neuronal diversification in the developing CNS versus ENS",slug:"fig-8-schematic-of-the-major-principle-of-neuronal-diversification-in-the-developing-cns-versus-ens",normalizedTitle:"fig. 8 schematic of the major principle of neuronal diversification in the developing cns versus ens",charIndex:10428}],excerpt:'<p>:  RNA <br>\nDOI: <a href="https://doi.org/10.1038/s41593-020-00736-x" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-020-00736-x<OutboundLink/></a><br>\nCite: Morarach, K., Mikhailova, A., Knoflach, V. et al. Diversification of molecularly defined myenteric\nneuron classes revealed by single-cell RNA sequencing. <em>Nat Neurosci</em> 24, 3446 (2021).</p>\n',headersStr:"Abstract Figure Fig. 1 Molecular definition of 12 enteric neuron classes in the mouse small intestine myenteric plexus Fig. 2 Gene categories conferring functional characteristics in enteric neuron classes Fig. 3 Immunohistochemical validation of marker genes for enteric neuron classes Fig. 4 Assessment of IPAN characteristics in ENCs 6, 7 and 12 Fig. 5 Single-cell transcriptome analysis of the developing ENS reveals transcriptional programs of generic cell states Fig. 6 ENC identities emerge through binary branching followed by postmitotic neuronal diversification Fig. 7 Loss of PBX3 expression impairs the ENC8/9 to ENC12 transition Fig. 8 Schematic of the major principle of neuronal diversification in the developing CNS versus ENS",content:":  RNA \nDOI: https://doi.org/10.1038/s41593-020-00736-x\nCite: Morarach, K., Mikhailova, A., Knoflach, V. et al. Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing. Nat Neurosci 24, 3446 (2021).\n\n\n\n RNA \n\n\n\nULRIKA MARKLUND\n\nKarolinska Institutet\nulrika.marklund@ki.se\n\n\n# Abstract\n\nENS scRNAseq  12  ENS  Pbx3  ENS \n\n\n# Figure\n\n\n# Fig. 1 Molecular definition of 12 enteric neuron classes in the mouse small intestine myenteric plexus\n\n\n\nFig. 1  12 \n\n pan-neuronal Baf53b-Cre line  R26R-Tomato reporter mice  TOM  elav-like protein 3/4 (HUC/D)  SOX2/10  TOM+  HUC/D SOX2/10Fig. 1aFig. 1b   (duodenum, 2,138 cells) (jejunum, 2,791 cells) (ileum, 3,180 cells)  TOM  HUC/D+  HUC/D+  TOM~98% Baf53b-Cre  R26R-Tomato  Baf53b-Cre  R26R-Tomato  P21 TOM+  10x Genomics Chromium RNA sequencing Fig. 1c\n\n 9,141  600 UMIFig. 1d Wnt1-Cre;R26R-Tomato  10%  90% Fig. 1e 4,892  12  ENCsFig.1f clusters clusters clusters\n\nFig. 1g ENC5 (somatostatin; Sst)ENC6 (neuromedin U; Nmu)ENC7 (cholecystokinin (Cck) and urocortin 3 (Ucn3))ENC10 (neuronal differentiation 6; Neurod6)ENC12 (neurexophilin-2; Nxph2)Fig. 1gENC14  Tac1Calb2Ndufa4l2GdaPenkFut9 ENC8  ENC9  Nos1  C1ql1 ENC9  Npy  Cox8b ENC11  Npy Nos1\n\n ENS  ENC Fig. 1hENC Fig.1h,iENC1-4 Tac1/Calb2 ENC8  ENC9 Nos1/Gal/Vip/NpyENC67  12  IPANsCalca/Calcb/Nfem/Calb1/Calb2 IPANs  NmuUcn-3/Cck  Nxph2 ENC6  ENC12  IPAN  ENC7 Fig. 4ENC10  IN1Nos1Slc18a3ENC5  Sst/Calcb/Calb2  IN2 ENC12 5- IN3 DdcSlc18a2  Slc6a4Fig. 1g\n\nvGlut2  Gad2 ENC7  ENC12 GABA ENC10Fig. 1g,jENC11  Dbh  ThFig. 1g,j Ddc \n\n\n# Fig. 2 Gene categories conferring functional characteristics in enteric neuron classes\n\n\n\nFig. 2 \n\n inputoutput /Fig2a-d  ENCs Fig. 2a-d/ Grm5  ENC7  Sstr5  ENC12Fig.2a ENS  Kit  BdnfENC6 /Nmu  NogFig.1gFig.2bENC6  Kcnn3  Ano2 ENC12  Piezo2ENCs semaphorins  ephrinsFig.2cNxph1-4 ENC1-12 \n\n ENC  ENC10Neurod6 ENC12Pbx3  Onecut2Fig.2dENC6  Phox2a Etv1  Bnc2  ENC Fig.2d,e\n\n ENC  Paul et al.  HUGO Gene Nomenclature Committee (HGNC)  ENC Fig.2f,g 15%  copines  annexinsFig.2h//\n\n\n# Fig. 3 Immunohistochemical validation of marker genes for enteric neuron classes\n\n\n\nFig. 3 \n\n ENC  ENC Fig. 3 23  ENC ENC  ENC  ENS  NDUFA4L2ENC13FUT9ENC4UCN3ENC7NEUROD6ENC10NXPH2ENC12NEUROD6  NOS1  MNF-MENC10Fig. 3CALR 9  12  C57BL/6  ENC \n\n5-HT5- ENC12 Fig. 3 GABA GAD2VGLUT2Fig. 1j and 3\n\n\n# Fig. 4 Assessment of IPAN characteristics in ENCs 6, 7 and 12\n\n\n\nFig. 4  ENCs 6712  IPAN \n\n IPANENCs 67  12Fig. 1ij ENC6  ENC7 Fig. 3 Nmu-Cre  Cck-IRES-Cre  R26R-Tomato  RNAscope 95.8%2.9%  Nmu+  TOM93.2%1.6%  TOM+  Nmu Nmu-Cre;R26R-Tomato Cck-IRES-Cre;R26R-Tomato  TOM Cck  Cck+  Cck-IRES-Cre DIO-EYFP or DIO-Ruby3AAV-PHP-S Cck+ 95.8%4.1%  EYFP  ENC12 G1NTNG1CALB\n\n IPAN CGRP-MNF-MCALB RNA  IPAN Fig. 1gCGRP  ENC6  ENC7  ENC12 Fig 4a-dNF-M  ENC12 100% ENC6  ENC7 CALB ENC12  ENC6  ENC7  IPAN ENC6CALB CGRP+ NF-MENC7CALB CGRP NF-MENC12CALB+ CGRP NF-M+\n\n IPANs  ENC Fig.4j-lFig.4m-o Cajal ANO1+NOS1+ENK+  CALR+Fig.4pENCs 67  12 \n\nIPANs  NMU+ Fig.4q NMU+  CCK+  NTNG1+/CALB+ Fig.4r,s\n\n ENC6  II  IPANs Nmu-Cre ENC12 PIEZO2Fig2b and 3 IPANs  ENC7  NF-M  CALB IPANs IPAN  IPANs \n\n\n# Fig. 5 Single-cell transcriptome analysis of the developing ENS reveals transcriptional programs of generic cell states\n\n\n\nFig. 5  ENS \n\n ENC1-12  Wnt1-Cre;R26R-Tomato  E15.5  E18.5  scRNA-seq\n\n 3,000-3,500  ENS E15.5 3,260 E18.5  2,733  UMAPFig.5ab/ Sox10Fig.5cFig.5d Elavl4High Fig.5c Sox10+  Elavl4High  Ascl1High  E18.5 Plp1highS100bhighFig.5b ENS  SCPSCPDhhMal  Mpzhttp://mousebrain.orgApoe  Nkain4http://mousebrain.org RNA Fig.5e\n\nMef2cSox5/8Notch1/2Fig.5fDll3Hes6Fig.5gMyt1lPtprnFig.5hSCPGfra3Wnt6Fig.5iHes5FrzbFig.5iNotch  ENS Rcor2Bcl11bFig.5g\n\n\n# Fig. 6 ENC identities emerge through binary branching followed by postmitotic neuronal diversification\n\n\n\nFig. 6 ENC \n\n E15.5  E18.5  UMAP  A  B ENC1-12  PCA  ENC  0.5  ENC8-12  A  ENC1-7  B Fig.6a-dENC5  ENC11 Fig.6d ENC8  ENC1 Fig.6cGal/Vip/Nos1ENC 8ENC9Ndufa4l2ENC 1ENC2Fig.6e A ENC10Neurod6 ENC12Ntng1Calb1Nxph2Fig.6eENC10-12 Nos1  Slc18a2 Fig.6e/Fig.1g B  B1 ENC3  ENC4Gda  Fut9 B2 ENC6Ano2 ENC7Mgat4c  Ucn3\n\n ENCs Etv1/Ebf1/Tbx3  Bnc2/Dlx5  A  B Etv1Ebf1Fig.6f ENCs  Etv1  Bnc2 Fig.2eNeurod6  Onecut2Zfp804a  Trps1Fig.6fEtv4  Dlx5\n\nPAGA E15.5  E18.5 Fig.6gPAGA  AB1  B2  ENCs Fig.6h 12  ENC  ENC1  ENC8Fig.6cENC2-7  ENC9-12\n\n\n# Fig. 7 Loss of PBX3 expression impairs the ENC8/9 to ENC12 transition\n\n\n\nFig. 7 PBX3  ENC8/9  ENC12 \n\nNOS1+ A  ENC8/9  ENC12 Fig.6a Gal/Vip/Nos1  Ntng1/Nfem/Calb1 Fig.6e and 7a RNA Fig.5e ENC8/9  ENC12 Pbx3 Fig.7a CALB  NOS1  PBX3+  E18.5  1.5%0.4%n=3 NOS1+  PBX3 NOS1ENC8/9  PBX3 \n\n Pbx3  ENS  Pbx3-/-  Pbx3-/-  E15.5  E18.5  ENS  Pbx3-/-  E15.5  90 5--2'-EdU S  SOX10+ Fig.7b HUC/D+  SOX10+ Fig.7cPbx3 \n\nPbx3  ENC12  ENC8/9  E18.5  HUC/D+  ENC8/9 GALNOS1VIP ENC12  CALB  Pbx3-/-  VIP20.5%NOS128.7%GAL37.1%Fig.7de CALB+  43.8%Fig.7de PBX3  ENC8/9  ENC12  PBX3  CALB+ ENC12  GAL+/VIP+/NOS+ Fig.7fENC7  PBX3Fig.6f Pbx3-/-  CCK 72.1%1.43%n=4\n\n\n# Fig. 8 Schematic of the major principle of neuronal diversification in the developing CNS versus ENS\n\n\n\nFig. 8  ENS \n\n\n\nENS ENC1  ENC8  ENC2-7  ENC9-12 ENC ",normalizedContent:":  rna \ndoi: https://doi.org/10.1038/s41593-020-00736-x\ncite: morarach, k., mikhailova, a., knoflach, v. et al. diversification of molecularly defined myenteric neuron classes revealed by single-cell rna sequencing. nat neurosci 24, 3446 (2021).\n\n\n\n rna \n\n\n\nulrika marklund\n\nkarolinska institutet\nulrika.marklund@ki.se\n\n\n# abstract\n\nens scrnaseq  12  ens  pbx3  ens \n\n\n# figure\n\n\n# fig. 1 molecular definition of 12 enteric neuron classes in the mouse small intestine myenteric plexus\n\n\n\nfig. 1  12 \n\n pan-neuronal baf53b-cre line  r26r-tomato reporter mice  tom  elav-like protein 3/4 (huc/d)  sox2/10  tom+  huc/d sox2/10fig. 1afig. 1b   (duodenum, 2,138 cells) (jejunum, 2,791 cells) (ileum, 3,180 cells)  tom  huc/d+  huc/d+  tom~98% baf53b-cre  r26r-tomato  baf53b-cre  r26r-tomato  p21 tom+  10x genomics chromium rna sequencing fig. 1c\n\n 9,141  600 umifig. 1d wnt1-cre;r26r-tomato  10%  90% fig. 1e 4,892  12  encsfig.1f clusters clusters clusters\n\nfig. 1g enc5 (somatostatin; sst)enc6 (neuromedin u; nmu)enc7 (cholecystokinin (cck) and urocortin 3 (ucn3))enc10 (neuronal differentiation 6; neurod6)enc12 (neurexophilin-2; nxph2)fig. 1genc14  tac1calb2ndufa4l2gdapenkfut9 enc8  enc9  nos1  c1ql1 enc9  npy  cox8b enc11  npy nos1\n\n ens  enc fig. 1henc fig.1h,ienc1-4 tac1/calb2 enc8  enc9 nos1/gal/vip/npyenc67  12  ipanscalca/calcb/nfem/calb1/calb2 ipans  nmuucn-3/cck  nxph2 enc6  enc12  ipan  enc7 fig. 4enc10  in1nos1slc18a3enc5  sst/calcb/calb2  in2 enc12 5- in3 ddcslc18a2  slc6a4fig. 1g\n\nvglut2  gad2 enc7  enc12 gaba enc10fig. 1g,jenc11  dbh  thfig. 1g,j ddc \n\n\n# fig. 2 gene categories conferring functional characteristics in enteric neuron classes\n\n\n\nfig. 2 \n\n inputoutput /fig2a-d  encs fig. 2a-d/ grm5  enc7  sstr5  enc12fig.2a ens  kit  bdnfenc6 /nmu  nogfig.1gfig.2benc6  kcnn3  ano2 enc12  piezo2encs semaphorins  ephrinsfig.2cnxph1-4 enc1-12 \n\n enc  enc10neurod6 enc12pbx3  onecut2fig.2denc6  phox2a etv1  bnc2  enc fig.2d,e\n\n enc  paul et al.  hugo gene nomenclature committee (hgnc)  enc fig.2f,g 15%  copines  annexinsfig.2h//\n\n\n# fig. 3 immunohistochemical validation of marker genes for enteric neuron classes\n\n\n\nfig. 3 \n\n enc  enc fig. 3 23  enc enc  enc  ens  ndufa4l2enc13fut9enc4ucn3enc7neurod6enc10nxph2enc12neurod6  nos1  mnf-menc10fig. 3calr 9  12  c57bl/6  enc \n\n5-ht5- enc12 fig. 3 gaba gad2vglut2fig. 1j and 3\n\n\n# fig. 4 assessment of ipan characteristics in encs 6, 7 and 12\n\n\n\nfig. 4  encs 6712  ipan \n\n ipanencs 67  12fig. 1ij enc6  enc7 fig. 3 nmu-cre  cck-ires-cre  r26r-tomato  rnascope 95.8%2.9%  nmu+  tom93.2%1.6%  tom+  nmu nmu-cre;r26r-tomato cck-ires-cre;r26r-tomato  tom cck  cck+  cck-ires-cre dio-eyfp or dio-ruby3aav-php-s cck+ 95.8%4.1%  eyfp  enc12 g1ntng1calb\n\n ipan cgrp-mnf-mcalb rna  ipan fig. 1gcgrp  enc6  enc7  enc12 fig 4a-dnf-m  enc12 100% enc6  enc7 calb enc12  enc6  enc7  ipan enc6calb cgrp+ nf-menc7calb cgrp nf-menc12calb+ cgrp nf-m+\n\n ipans  enc fig.4j-lfig.4m-o cajal ano1+nos1+enk+  calr+fig.4pencs 67  12 \n\nipans  nmu+ fig.4q nmu+  cck+  ntng1+/calb+ fig.4r,s\n\n enc6  ii  ipans nmu-cre enc12 piezo2fig2b and 3 ipans  enc7  nf-m  calb ipans ipan  ipans \n\n\n# fig. 5 single-cell transcriptome analysis of the developing ens reveals transcriptional programs of generic cell states\n\n\n\nfig. 5  ens \n\n enc1-12  wnt1-cre;r26r-tomato  e15.5  e18.5  scrna-seq\n\n 3,000-3,500  ens e15.5 3,260 e18.5  2,733  umapfig.5ab/ sox10fig.5cfig.5d elavl4high fig.5c sox10+  elavl4high  ascl1high  e18.5 plp1highs100bhighfig.5b ens  scpscpdhhmal  mpzhttp://mousebrain.orgapoe  nkain4http://mousebrain.org rna fig.5e\n\nmef2csox5/8notch1/2fig.5fdll3hes6fig.5gmyt1lptprnfig.5hscpgfra3wnt6fig.5ihes5frzbfig.5inotch  ens rcor2bcl11bfig.5g\n\n\n# fig. 6 enc identities emerge through binary branching followed by postmitotic neuronal diversification\n\n\n\nfig. 6 enc \n\n e15.5  e18.5  umap  a  b enc1-12  pca  enc  0.5  enc8-12  a  enc1-7  b fig.6a-denc5  enc11 fig.6d enc8  enc1 fig.6cgal/vip/nos1enc 8enc9ndufa4l2enc 1enc2fig.6e a enc10neurod6 enc12ntng1calb1nxph2fig.6eenc10-12 nos1  slc18a2 fig.6e/fig.1g b  b1 enc3  enc4gda  fut9 b2 enc6ano2 enc7mgat4c  ucn3\n\n encs etv1/ebf1/tbx3  bnc2/dlx5  a  b etv1ebf1fig.6f encs  etv1  bnc2 fig.2eneurod6  onecut2zfp804a  trps1fig.6fetv4  dlx5\n\npaga e15.5  e18.5 fig.6gpaga  ab1  b2  encs fig.6h 12  enc  enc1  enc8fig.6cenc2-7  enc9-12\n\n\n# fig. 7 loss of pbx3 expression impairs the enc8/9 to enc12 transition\n\n\n\nfig. 7 pbx3  enc8/9  enc12 \n\nnos1+ a  enc8/9  enc12 fig.6a gal/vip/nos1  ntng1/nfem/calb1 fig.6e and 7a rna fig.5e enc8/9  enc12 pbx3 fig.7a calb  nos1  pbx3+  e18.5  1.5%0.4%n=3 nos1+  pbx3 nos1enc8/9  pbx3 \n\n pbx3  ens  pbx3-/-  pbx3-/-  e15.5  e18.5  ens  pbx3-/-  e15.5  90 5--2'-edu s  sox10+ fig.7b huc/d+  sox10+ fig.7cpbx3 \n\npbx3  enc12  enc8/9  e18.5  huc/d+  enc8/9 galnos1vip enc12  calb  pbx3-/-  vip20.5%nos128.7%gal37.1%fig.7de calb+  43.8%fig.7de pbx3  enc8/9  enc12  pbx3  calb+ enc12  gal+/vip+/nos+ fig.7fenc7  pbx3fig.6f pbx3-/-  cck 72.1%1.43%n=4\n\n\n# fig. 8 schematic of the major principle of neuronal diversification in the developing cns versus ens\n\n\n\nfig. 8  ens \n\n\n\nens enc1  enc8  enc2-7  enc9-12 enc ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease",frontmatter:{title:" --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease",date:"2023-03-17T10:42:35.000Z",permalink:"/pages/25c02b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20210708.(2021.07.08%20hdWGCNA%20Nat%20Genet)%20Single-nucleus%20chromatin%20accessibility%20and%20transcriptomic%20characterization%20of%20Alzheimer's%20disease.html",relativePath:"02./11./04./20210708.(2021.07.08 hdWGCNA Nat Genet) Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease.md",key:"v-53c700f9",path:"/pages/25c02b/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:923},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:1306},{level:3,title:"Figure 1. Single-nucleus ATAC-seq and single-nucleus RNA-seq for the study of cellular diversity in the diseased brain",slug:"figure-1-single-nucleus-atac-seq-and-single-nucleus-rna-seq-for-the-study-of-cellular-diversity-in-the-diseased-brain",normalizedTitle:"figure 1. single-nucleus atac-seq and single-nucleus rna-seq for the study of cellular diversity in the diseased brain",charIndex:1319},{level:3,title:"Figure 2. Epigenetically and transcriptionally distinct cell subpopulations in human AD prefrontal cortex",slug:"figure-2-epigenetically-and-transcriptionally-distinct-cell-subpopulations-in-human-ad-prefrontal-cortex",normalizedTitle:"figure 2. epigenetically and transcriptionally distinct cell subpopulations in human ad prefrontal cortex",charIndex:1768},{level:3,title:"Figure 3. Linking cis-regulatory elements to downstream target genes in specific cell types",slug:"figure-3-linking-cis-regulatory-elements-to-downstream-target-genes-in-specific-cell-types",normalizedTitle:"figure 3. linking cis-regulatory elements to downstream target genes in specific cell types",charIndex:2335},{level:3,title:"Figure 4. Cell-subpopulation-specific transcription factor regulation in late-stage AD",slug:"figure-4-cell-subpopulation-specific-transcription-factor-regulation-in-late-stage-ad",normalizedTitle:"figure 4. cell-subpopulation-specific transcription factor regulation in late-stage ad",charIndex:3013},{level:3,title:"Figure 5. Multi-omic oligodendrocyte trajectory analysis",slug:"figure-5-multi-omic-oligodendrocyte-trajectory-analysis",normalizedTitle:"figure 5. multi-omic oligodendrocyte trajectory analysis",charIndex:3681},{level:3,title:"Figure 6. Multi-omic microglia and astrocyte trajectory analyses",slug:"figure-6-multi-omic-microglia-and-astrocyte-trajectory-analyses",normalizedTitle:"figure 6. multi-omic microglia and astrocyte trajectory analyses",charIndex:4371},{level:3,title:"Figure 7. Cell-type-specific regulatory landscapes of GWAS loci in the AD brain",slug:"figure-7-cell-type-specific-regulatory-landscapes-of-gwas-loci-in-the-ad-brain",normalizedTitle:"figure 7. cell-type-specific regulatory landscapes of gwas loci in the ad brain",charIndex:4535},{level:3,title:"Figure 8. Robust gene expression modules revealed using integrated bulk and single-cell coexpression network analysis",slug:"figure-8-robust-gene-expression-modules-revealed-using-integrated-bulk-and-single-cell-coexpression-network-analysis",normalizedTitle:"figure 8. robust gene expression modules revealed using integrated bulk and single-cell coexpression network analysis",charIndex:4843},{level:2,title:"Extended Data:",slug:"extended-data",normalizedTitle:"extended data:",charIndex:5267},{level:3,title:"Extended Data Figure 1. Batch correction of snATAC-seq, snRNA-seq, and merged datasets",slug:"extended-data-figure-1-batch-correction-of-snatac-seq-snrna-seq-and-merged-datasets",normalizedTitle:"extended data figure 1. batch correction of snatac-seq, snrna-seq, and merged datasets",charIndex:5286},{level:3,title:"Extended Data Figure 2. Cell-type immunostaining and in situ hybridization",slug:"extended-data-figure-2-cell-type-immunostaining-and-in-situ-hybridization",normalizedTitle:"extended data figure 2. cell-type immunostaining and in situ hybridization",charIndex:5654},{level:3,title:"Extended Data Figure 3. Comparison of gene expression and gene activity",slug:"extended-data-figure-3-comparison-of-gene-expression-and-gene-activity",normalizedTitle:"extended data figure 3. comparison of gene expression and gene activity",charIndex:5979},{level:3,title:"Extended Data Figure 4. NEAT1 validation and neuronal TFs",slug:"extended-data-figure-4-neat1-validation-and-neuronal-tfs",normalizedTitle:"extended data figure 4. neat1 validation and neuronal tfs",charIndex:6199},{level:3,title:"Extended Data Figure 5. Schematics of analyses",slug:"extended-data-figure-5-schematics-of-analyses",normalizedTitle:"extended data figure 5. schematics of analyses",charIndex:6867},{level:3,title:"Extended Data Figure 6. Pseudotime trajectory analysis to identify dysregulated TFs and gene expression in glia",slug:"extended-data-figure-6-pseudotime-trajectory-analysis-to-identify-dysregulated-tfs-and-gene-expression-in-glia",normalizedTitle:"extended data figure 6. pseudotime trajectory analysis to identify dysregulated tfs and gene expression in glia",charIndex:7250},{level:3,title:"Extended Data Figure 7. Metacell aggregation and SREBP",slug:"extended-data-figure-7-metacell-aggregation-and-srebp",normalizedTitle:"extended data figure 7. metacell aggregation and srebp",charIndex:7419},{level:3,title:"Extended Data Figure 8. iNMF integration of snRNA-seq with Mathys et al. snRNA-seq",slug:"extended-data-figure-8-inmf-integration-of-snrna-seq-with-mathys-et-al-snrna-seq",normalizedTitle:"extended data figure 8. inmf integration of snrna-seq with mathys et al. snrna-seq",charIndex:7524},{level:3,title:"Extended Data Figure 9. scWGCNA in microglia and astrocytes",slug:"extended-data-figure-9-scwgcna-in-microglia-and-astrocytes",normalizedTitle:"extended data figure 9. scwgcna in microglia and astrocytes",charIndex:7684},{level:3,title:"Extended Data Figure 10. scWGCNA in neurons",slug:"extended-data-figure-10-scwgcna-in-neurons",normalizedTitle:"extended data figure 10. scwgcna in neurons",charIndex:7800}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41588-021-00894-z" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41588-021-00894-z<OutboundLink/></a><br>\nCite: Morabito, S., Miyoshi, E., Michael, N. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer\'s disease. Nat Genet 53, 11431155 (2021).</p>\n',headersStr:"Abstract: Results: Figure 1. Single-nucleus ATAC-seq and single-nucleus RNA-seq for the study of cellular diversity in the diseased brain Figure 2. Epigenetically and transcriptionally distinct cell subpopulations in human AD prefrontal cortex Figure 3. Linking cis-regulatory elements to downstream target genes in specific cell types Figure 4. Cell-subpopulation-specific transcription factor regulation in late-stage AD Figure 5. Multi-omic oligodendrocyte trajectory analysis Figure 6. Multi-omic microglia and astrocyte trajectory analyses Figure 7. Cell-type-specific regulatory landscapes of GWAS loci in the AD brain Figure 8. Robust gene expression modules revealed using integrated bulk and single-cell coexpression network analysis Extended Data: Extended Data Figure 1. Batch correction of snATAC-seq, snRNA-seq, and merged datasets Extended Data Figure 2. Cell-type immunostaining and in situ hybridization Extended Data Figure 3. Comparison of gene expression and gene activity Extended Data Figure 4. NEAT1 validation and neuronal TFs Extended Data Figure 5. Schematics of analyses Extended Data Figure 6. Pseudotime trajectory analysis to identify dysregulated TFs and gene expression in glia Extended Data Figure 7. Metacell aggregation and SREBP Extended Data Figure 8. iNMF integration of snRNA-seq with Mathys et al. snRNA-seq Extended Data Figure 9. scWGCNA in microglia and astrocytes Extended Data Figure 10. scWGCNA in neurons",content:": \nDOI: https://doi.org/10.1038/s41588-021-00894-z\nCite: Morabito, S., Miyoshi, E., Michael, N. et al. Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease. Nat Genet 53, 11431155 (2021).\n\n\n\n\n\n\n\n 1.  --- WGCNAan R package for weighted correlation network analysis\n 2. WGCNAan R package for weighted correlation network analysis\n 3.  --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n 4.  --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdWGCNAhigh dimensional WGCNA\n\n\n\nSAMUEL MORABITO                        VIVEK SWARUP\n                                       \nUniversity of California Irvine(UCI)   University of California Irvine(UCI)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# Abstract:\n\n 191,890 AD APOE  CLU  AD  SREBF1 AD \n\n\n# Results:\n\n\n# Figure 1. Single-nucleus ATAC-seq and single-nucleus RNA-seq for the study of cellular diversity in the diseased brain\n\n\n\nFigure 1. snATAC-seq  snRNA-seq \n\na.  BioRender.com \nb. 130,418  snATAC-seq UMAP\nc. 61,472  snRNA-seq UMAP\nd.  marker genes \ne.  marker genes \nf. snATAC-seq  snRNA-seq  186,167  UMAP \ng.  UMAP  snATAC-seq  snRNA-seq \n\n\n# Figure 2. Epigenetically and transcriptionally distinct cell subpopulations in human AD prefrontal cortex\n\n\n\nFigure 2.  AD \n\na. snRNA-seq  OPC  oligodendrocyte  top25 DEGs\nb. snATAC-seq  OPC  oligodendrocyte  top25 DEGs\nc. OPC snATAC-seq  Pseudo-bulk \nd. snATAC-seq  UMAPs AD  control \ne. snRNA-seq  UMAPs AD  control \nf. snATAC-seq  AD  control \ng. snRNA-seq  AD  control \n AD \n\n\n# Figure 3. Linking cis-regulatory elements to downstream target genes in specific cell types\n\n\n\nFigure 3. \n\na.  1  25  linked cCREs \nb. Upset  cCRE-linked genes  cCRE-linked genes \nc. Venn  cCRE-linked genescell-type DEGsdiagnosis DEGs  overlaps\nd.  AD  snATAC-seq cluster  pseudo-bulk cCREs NMF  AD \ne.  intronicexonicdistal regions  gl-cCREs \nf.  snATAC-seq cluster  NMF  control  late-stage AD \ng.  NMF  GO \n\n\n# Figure 4. Cell-subpopulation-specific transcription factor regulation in late-stage AD\n\n\n\nFigure 4. AD \n\na. Left: snATAC-seq  snRNA-seq  UMAP SPI1 motif variability  microglia Right:  AD  snATAC-seq  SPI1 motif variability \nb. Left: snATAC-seq  snRNA-seq  UMAP SPI1 target gene score  microglia Right:  AD  snRNA-seq  SPI1 target gene score \nc. snATAC-seq microglia cluster (top)  AD diagnosis (bottom)  Tn5 bias-subtracted TF footprinting for SPI1 TF binding motif\nd-f.  a-c ODC  NRF1 TF\ng-h. TF  Microglia  Oligodendrocyte \n\n\n# Figure 5. Multi-omic oligodendrocyte trajectory analysis\n\n\n\nFigure 5. \n\na. snATAC-seq  snRNA-seq  UMAP pseudotime trajectory \nb.  bin50 AD \nc.  NF-ODC, MF-ODC, mature ODC  module score \nd. Left:  RVAE  9,231  gl-cCREs Right:  RVAE  1,563  t-DEGs  DEGs AD \ne.  RVAE  t-DEGs gl-cCREsLeft:  75% Right:  RVAE  AD \nf.  t-DEG  NRF1 SREBF1motif  TF  AD AD DEGs\n\n\n# Figure 6. Multi-omic microglia and astrocyte trajectory analyses\n\n\n\nFigure 6. \n\na-f.  Fig. 5 Microglia \ng-I.  Fig. 5 Astrocytes \n\n\n# Figure 7. Cell-type-specific regulatory landscapes of GWAS loci in the AD brain\n\n\n\nFigure 7. AD  GWAS \n\na.  snATAC-seq  GWAS traits  disorders  LDSC \nb-c.  pseudotime trajectory  distal peaks (b)  gene-proximal peaks (c)  gchromVAR \nd-i. GWAS \n\n\n# Figure 8. Robust gene expression modules revealed using integrated bulk and single-cell coexpression network analysis\n\n\n\nFigure 8.  bulk \n\na.  OM1OM2OM4  OM5 \nb.  AD diagnosis \nc.  SREBF1 \nd.  SREBF1  RNAtopbottom AD \ne.  MOG+  SREBF1 \nf.  MOG+  ACSL4 \ng.  RNAscope \n\n\n# Extended Data:\n\n\n# Extended Data Figure 1. Batch correction of snATAC-seq, snRNA-seq, and merged datasets\n\n\n\nExtended Data Figure 1.  snATAC-seqsnRNA-seq \n\na. snRNA-seq  iNMF  UMAP\nb. snATAC-seq  MNN  UMAP\nc.  snRNA-seq cluster  iNMF metagene  Dot plot\nd.  metagene  snRNA-seq UMAPs \ne.  metagene  top30 genes iNMF loading\n\n\n# Extended Data Figure 2. Cell-type immunostaining and in situ hybridization\n\n\n\nExtended Data Figure 2. \n\na-d.  AD  Iba-1(a), GFAP(b) MAP2(c), 6E10(d) \ne. 6E10-positive amyloid plaques \nf.  PDGFRA  OLIG2 \ngh. CNP(g)  PLP1(h)  RNAscope \n\n\n# Extended Data Figure 3. Comparison of gene expression and gene activity\n\n\n\nExtended Data Figure 3. \n\na.  snATAC-seq  snRNA-seq \nb. \n\n\n# Extended Data Figure 4. NEAT1 validation and neuronal TFs\n\n\n\nExtended Data Figure 4. NEAT1  TFs\n\na-b.  RNAscope NEAT1  AQP4  (a)  NEAT1  MOG (b)DAPI \nc.  AQP4+  NEAT1 \nd.  MOG+  NEAT1 \ne. snATAC-seq neuron cluster (top)  AD diagnosis (bottom)  Tn5 bias subtracted TF footprinting TF binding motif\nf. Left: JUN motif variability (top)  JUN target gene score (bottom)  UMAPRight:  JUN motif variability (top)  JUN target gene score (bottom) \ng-h.  e-f EGR1\ni. snATAC-seq  SREBF1 motif variability \nj. snRNA-seq  SREBF1 expression \n\n\n# Extended Data Figure 5. Schematics of analyses\n\n\n\nExtended Data Figure 5. \n\na. cis-regulatory elements, cCRE target genes  AD  NMF  gl-cCREs \nb.  TF \nc. scWGCNA  Mathys et al. 2019  iNMF metacell \n\n\n# Extended Data Figure 6. Pseudotime trajectory analysis to identify dysregulated TFs and gene expression in glia\n\n\n\nExtended Data Figure 6. \n\n\n# Extended Data Figure 7. Metacell aggregation and SREBP\n\n\n\nExtended Data Figure 7. Metacell  SREBP\n\n\n# Extended Data Figure 8. iNMF integration of snRNA-seq with Mathys et al. snRNA-seq\n\n\n\nExtended Data Figure 8. snRNA-seq  Mathys et al. snRNA-seq  iNMF \n\n\n# Extended Data Figure 9. scWGCNA in microglia and astrocytes\n\n\n\nExtended Data Figure 9.  scWGCNA \n\n\n# Extended Data Figure 10. scWGCNA in neurons\n\n\n\nExtended Data Figure 10.  scWGCNA ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41588-021-00894-z\ncite: morabito, s., miyoshi, e., michael, n. et al. single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease. nat genet 53, 11431155 (2021).\n\n\n\n\n\n\n\n 1.  --- wgcnaan r package for weighted correlation network analysis\n 2. wgcnaan r package for weighted correlation network analysis\n 3.  --- single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n 4.  --- hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdwgcnahigh dimensional wgcna\n\n\n\nsamuel morabito                        vivek swarup\n                                       \nuniversity of california irvine(uci)   university of california irvine(uci)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# abstract:\n\n 191,890 ad apoe  clu  ad  srebf1 ad \n\n\n# results:\n\n\n# figure 1. single-nucleus atac-seq and single-nucleus rna-seq for the study of cellular diversity in the diseased brain\n\n\n\nfigure 1. snatac-seq  snrna-seq \n\na.  biorender.com \nb. 130,418  snatac-seq umap\nc. 61,472  snrna-seq umap\nd.  marker genes \ne.  marker genes \nf. snatac-seq  snrna-seq  186,167  umap \ng.  umap  snatac-seq  snrna-seq \n\n\n# figure 2. epigenetically and transcriptionally distinct cell subpopulations in human ad prefrontal cortex\n\n\n\nfigure 2.  ad \n\na. snrna-seq  opc  oligodendrocyte  top25 degs\nb. snatac-seq  opc  oligodendrocyte  top25 degs\nc. opc snatac-seq  pseudo-bulk \nd. snatac-seq  umaps ad  control \ne. snrna-seq  umaps ad  control \nf. snatac-seq  ad  control \ng. snrna-seq  ad  control \n ad \n\n\n# figure 3. linking cis-regulatory elements to downstream target genes in specific cell types\n\n\n\nfigure 3. \n\na.  1  25  linked ccres \nb. upset  ccre-linked genes  ccre-linked genes \nc. venn  ccre-linked genescell-type degsdiagnosis degs  overlaps\nd.  ad  snatac-seq cluster  pseudo-bulk ccres nmf  ad \ne.  intronicexonicdistal regions  gl-ccres \nf.  snatac-seq cluster  nmf  control  late-stage ad \ng.  nmf  go \n\n\n# figure 4. cell-subpopulation-specific transcription factor regulation in late-stage ad\n\n\n\nfigure 4. ad \n\na. left: snatac-seq  snrna-seq  umap spi1 motif variability  microglia right:  ad  snatac-seq  spi1 motif variability \nb. left: snatac-seq  snrna-seq  umap spi1 target gene score  microglia right:  ad  snrna-seq  spi1 target gene score \nc. snatac-seq microglia cluster (top)  ad diagnosis (bottom)  tn5 bias-subtracted tf footprinting for spi1 tf binding motif\nd-f.  a-c odc  nrf1 tf\ng-h. tf  microglia  oligodendrocyte \n\n\n# figure 5. multi-omic oligodendrocyte trajectory analysis\n\n\n\nfigure 5. \n\na. snatac-seq  snrna-seq  umap pseudotime trajectory \nb.  bin50 ad \nc.  nf-odc, mf-odc, mature odc  module score \nd. left:  rvae  9,231  gl-ccres right:  rvae  1,563  t-degs  degs ad \ne.  rvae  t-degs gl-ccresleft:  75% right:  rvae  ad \nf.  t-deg  nrf1 srebf1motif  tf  ad ad degs\n\n\n# figure 6. multi-omic microglia and astrocyte trajectory analyses\n\n\n\nfigure 6. \n\na-f.  fig. 5 microglia \ng-i.  fig. 5 astrocytes \n\n\n# figure 7. cell-type-specific regulatory landscapes of gwas loci in the ad brain\n\n\n\nfigure 7. ad  gwas \n\na.  snatac-seq  gwas traits  disorders  ldsc \nb-c.  pseudotime trajectory  distal peaks (b)  gene-proximal peaks (c)  gchromvar \nd-i. gwas \n\n\n# figure 8. robust gene expression modules revealed using integrated bulk and single-cell coexpression network analysis\n\n\n\nfigure 8.  bulk \n\na.  om1om2om4  om5 \nb.  ad diagnosis \nc.  srebf1 \nd.  srebf1  rnatopbottom ad \ne.  mog+  srebf1 \nf.  mog+  acsl4 \ng.  rnascope \n\n\n# extended data:\n\n\n# extended data figure 1. batch correction of snatac-seq, snrna-seq, and merged datasets\n\n\n\nextended data figure 1.  snatac-seqsnrna-seq \n\na. snrna-seq  inmf  umap\nb. snatac-seq  mnn  umap\nc.  snrna-seq cluster  inmf metagene  dot plot\nd.  metagene  snrna-seq umaps \ne.  metagene  top30 genes inmf loading\n\n\n# extended data figure 2. cell-type immunostaining and in situ hybridization\n\n\n\nextended data figure 2. \n\na-d.  ad  iba-1(a), gfap(b) map2(c), 6e10(d) \ne. 6e10-positive amyloid plaques \nf.  pdgfra  olig2 \ngh. cnp(g)  plp1(h)  rnascope \n\n\n# extended data figure 3. comparison of gene expression and gene activity\n\n\n\nextended data figure 3. \n\na.  snatac-seq  snrna-seq \nb. \n\n\n# extended data figure 4. neat1 validation and neuronal tfs\n\n\n\nextended data figure 4. neat1  tfs\n\na-b.  rnascope neat1  aqp4  (a)  neat1  mog (b)dapi \nc.  aqp4+  neat1 \nd.  mog+  neat1 \ne. snatac-seq neuron cluster (top)  ad diagnosis (bottom)  tn5 bias subtracted tf footprinting tf binding motif\nf. left: jun motif variability (top)  jun target gene score (bottom)  umapright:  jun motif variability (top)  jun target gene score (bottom) \ng-h.  e-f egr1\ni. snatac-seq  srebf1 motif variability \nj. snrna-seq  srebf1 expression \n\n\n# extended data figure 5. schematics of analyses\n\n\n\nextended data figure 5. \n\na. cis-regulatory elements, ccre target genes  ad  nmf  gl-ccres \nb.  tf \nc. scwgcna  mathys et al. 2019  inmf metacell \n\n\n# extended data figure 6. pseudotime trajectory analysis to identify dysregulated tfs and gene expression in glia\n\n\n\nextended data figure 6. \n\n\n# extended data figure 7. metacell aggregation and srebp\n\n\n\nextended data figure 7. metacell  srebp\n\n\n# extended data figure 8. inmf integration of snrna-seq with mathys et al. snrna-seq\n\n\n\nextended data figure 8. snrna-seq  mathys et al. snrna-seq  inmf \n\n\n# extended data figure 9. scwgcna in microglia and astrocytes\n\n\n\nextended data figure 9.  scwgcna \n\n\n# extended data figure 10. scwgcna in neurons\n\n\n\nextended data figure 10.  scwgcna ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations",frontmatter:{title:" --- Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations",categories:[""],tags:[""],date:"2022-11-19T16:13:27.000Z",permalink:"/pages/f481a1/",readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20220530.(2022.05.30%20%E7%8C%95%E7%8C%B4%E6%B5%B7%E9%A9%AC%20Nat%20Neurosci)%20Single-cell%20transcriptomics%20of%20adult%20macaque%20%20hippocampus%20reveals%20neural%20precursor%20cell%20%20populations.html",relativePath:"02./11./04./20220530.(2022.05.30  Nat Neurosci) Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations.md",key:"v-f88540b8",path:"/pages/f481a1/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:565},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1053},{level:3,title:"Fig1. ",slug:"fig1-",normalizedTitle:"fig1. ",charIndex:1065},{level:3,title:"Fig2. ",slug:"fig2-",normalizedTitle:"fig2. ",charIndex:2643},{level:3,title:"Fig3. RGLs  astrocytes ",slug:"fig3-rgls--astrocytes-",normalizedTitle:"fig3. rgls  astrocytes ",charIndex:3038},{level:3,title:"Fig4.  DG ",slug:"fig4--dg-",normalizedTitle:"fig4.  dg ",charIndex:4350},{level:3,title:"Fig5. HMGB2  RGL  IPCs ",slug:"fig5-hmgb2--rgl--ipcs-",normalizedTitle:"fig5. hmgb2  rgl  ipcs ",charIndex:5098},{level:3,title:"Fig6.  DG  GCs ",slug:"fig6--dg--gcs-",normalizedTitle:"fig6.  dg  gcs ",charIndex:6162},{level:3,title:"Fig7.  DG  GCs ",slug:"fig7--dg--gcs-",normalizedTitle:"fig7.  dg  gcs ",charIndex:6976},{level:2,title:"Extended Results",slug:"extended-results",normalizedTitle:"extended results",charIndex:7674},{level:3,title:"Extended Data Fig1. ",slug:"extended-data-fig1-",normalizedTitle:"extended data fig1. ",charIndex:7696},{level:3,title:"Extended Data Fig2. (gene regulatory networksGRNstermed regulons)",slug:"extended-data-fig2--gene-regulatory-networks-grns-termed-regulons",normalizedTitle:"extended data fig2. (gene regulatory networksgrnstermed regulons)",charIndex:7898},{level:3,title:"Extended Data Fig3. ",slug:"extended-data-fig3-",normalizedTitle:"extended data fig3. ",charIndex:8166},{level:3,title:"Extended Data Fig4. ",slug:"extended-data-fig4-",normalizedTitle:"extended data fig4. ",charIndex:8750},{level:3,title:"Extended Data Fig5. SGZ",slug:"extended-data-fig5-sgz",normalizedTitle:"extended data fig5. sgz",charIndex:9265},{level:3,title:"Extended Data Fig6. ",slug:"extended-data-fig6-",normalizedTitle:"extended data fig6. ",charIndex:9689},{level:3,title:"Extended Data Fig7. ",slug:"extended-data-fig7-",normalizedTitle:"extended data fig7. ",charIndex:9786},{level:3,title:"Extended Data Fig8. ",slug:"extended-data-fig8-",normalizedTitle:"extended data fig8. ",charIndex:9883},{level:3,title:"Extended Data Fig9. ",slug:"extended-data-fig9-",normalizedTitle:"extended data fig9. ",charIndex:10181},{level:3,title:"Extended Data Fig10. ",slug:"extended-data-fig10-",normalizedTitle:"extended data fig10. ",charIndex:10491}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-022-01073-x" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-022-01073-x<OutboundLink/></a><br>\nCite: Hao, ZZ., Wei, JR., Xiao, D. et al. Single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations. <em>Nat Neurosci</em> 25, 805817 (2022).</p>\n',headersStr:"Abstract Results Fig1.  Fig2.  Fig3. RGLs  astrocytes  Fig4.  DG  Fig5. HMGB2  RGL  IPCs  Fig6.  DG  GCs  Fig7.  DG  GCs  Extended Results Extended Data Fig1.  Extended Data Fig2. (gene regulatory networksGRNstermed regulons) Extended Data Fig3.  Extended Data Fig4.  Extended Data Fig5. SGZ Extended Data Fig6.  Extended Data Fig7.  Extended Data Fig8.  Extended Data Fig9.  Extended Data Fig10. ",content:": \nDOI: https://doi.org/10.1038/s41593-022-01073-x\nCite: Hao, ZZ., Wei, JR., Xiao, D. et al. Single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations. Nat Neurosci 25, 805817 (2022).\n\n\n\n\n\n\n\nSHENG LIU()              YIZHI LIU()       MENGQING XIANG()\n                                                \n            \nliush87@mail.sysu.edu.cn   yzliu62@yahoo.com    xiangmq3@mail.sysu.edu.cn\n\n\n# Abstract\n\n(neurogenesis) single-cell RNA sequencing pipeline 207,785  34 (radial glia-like cellsRGLs)(intermediate progenitor cellsIPCs)(neuroblastsNB) HMGB2  IPC  neurogenesis (neural precursor cellsNPCs)\n\n\n# Results\n\n\n# Fig1. \n\n\n\nFig1. \n\na.  8  ( 6  2 )  DG  10x Genomics  207,785 \nb. 207,785  UMAP  34 \ncd.  marker gene\n(1) neuroglia \nAstro, Astrocytes ;AQP4+\nAstro_im, immature astrocytes ;AQP4+\nOPC, oligodendrocyte progenitor cells ; C1QL1+\nNFOL, newly formed oligodendrocytes ; BCAS1+\nMOL, mature oligodendrocytes ; MOG+\n(2) neurons \nGABA, GABAergic neurons GABA; GAD2+\nPyr, pyramidal neurons ; RGS4+\nGC_im, immature granule cells ; PROX1+\nGC, granule cells ; PROX1+\n(3) vascular cells \nEndothelialFLT1+\nVLMC, vascular leptomeningeal cells ; DCN+\n(4) immune cells \nMicroglia C1QA+\nPVM, perivascular macrophages ; LYVE1+\n(5) CR, CajalRetzius cells; RELN+\n(6) EpendymalCFAP126+\n(7) NPCsneural precursor cells \nRGL_1 and RGL_2, radial glia-like cells ; SLC1A3+\nIPCintermediate progenitor cells ; IPC_1HMGB2+ASCL1+\nIPC_2HMGB2+ASCL1 \nNB, neuroblasts ; SOX4+\n(8) Unk, Unknown\n marker genes marker genes\nIPCsRFC4\nneuroblastsNNAT\nGCsNPY1R\ne. RGLs  local environment cell-cell signaling pathways  RGLs  Wnt signaling pathway RGLs \nf. (radial glia-like cellsRGLs)(neural stem cellsNSCs)RGLs (astrocytes)(GFAP)(neural precursor) SOX2  SLC1A3 DG  SGZ  RGLs\n\n\n# Fig2. \n\n\n\nFig2. \n\na.  scVI (n=15,878 cells)(n=9,810 cellsHochgerer_1 dataset) UMAP  Harmony \nb. \ncd.  a  UMAP \ne.  neural genesis cells and neurons (n=800 cells)  NPCs(n=132 cells) \nf. (n=15,878 cells)(n=6,323 cells)\n\n\n# Fig3. RGLs  astrocytes \n\n\n\nFig3. RGLs  astrocytes \n\n astrocytes  RGLs  RGLs  astrocytes RGLs  immature astrocytes scVelo  RGLs  astrocytes  RNA velocity(trajectories)scVelo  neurogenesis  newly transcribed, unspliced pre-mRNAs with mature, spliced mRNAs  RGLs  astrocytes  astrocyte genesis trajectory\n\na.  RGL  astrocyte maturation-related  UMAP  (RGL_1, n=1,107 cells; RGL_2, n=1,246; Astro_im1, n=1,661; Astro_im2, n=698; Astro_1, n=12,322) RNA velocities (RNA) \nb. scVelo  UMAP astrocyte genesis trajectory  RGL_1  RGL_2  immature astrocytes mature astrocytes RGLs \nc.  colorbar  a \nd.  immature astrocytes  astrocytes  UMAP \ne. DG  RGL markers (SOX2  GFAP) (apical processes) SGZ  ML()\nf. GFAP  hilus  astrocytes RGLs \ng. RGL_1  RGL_2  5575  DEGs GO  KEGG RGL_1 - RGL_2  Wnt pathway  WIF1(Wnt inhibitory factor 1)RGCCregulator of cell cycle NSCs (neural stem cell) TPT1 1\n\n\n# Fig4.  DG \n\n\n\nFig4.  DG \n\na.  neurogenic-related  immature GCs  UMAP  mKNN graph plot\nIPCs (n=163 cells, )  neuroblasts (n=1,904 cells) and GC_im cells (n=7,547 cells),  (Pyr_1 and Pyr_2, n=1,384 and 123 cells)mKNN  IPC_1  neuroblasts  immature GCs  neuroblasts  pyramidal neurons \nb.  a  mKNN graphs  proliferating cells(MKI67) NPCs(SOX2)IPCs(ASCL1) neuroblasts(EOMES and NEUROD2)  immature GCs (PROX1) \nc. xy\nd.  SOX2(precursor cell marker)  MKI67(proliferating cell marker)  NSCs (neural stem cell) \n\n\n# Fig5. HMGB2  RGL  IPCs \n\n\n\nFig5. HMGB2  RGL  IPCs \n\na. IPCs  UMAP  MKI67  EZH2 IPCs IPC_2  EZH2\nb. RGLs  IPCs  GFAP  HMGB2 IPCs  HMGB2  NSCs \nc. IPC  HMGB2  RGL  GFAP  RGLs  IPCs Box1 and white arrowheads: RGLGFAPHMGB2Box2 and yellow arrowheads: GFAPHMGB2Box3 and magenta arrowheads: HMGB2+HMGB2GFAP\nd.  RGLs (Box 1 and the green cell, GFAP+HMGB2)RGLs  IPCs  (Box 2 and the light green cell, GFAP+ HMBG2+)IPCs (Box 3 and the red cell, GFAP HMGB2+)\ne. HMGB2  GFAP  GFAP HMGB2+ IPC  GFAP+ HMGB2+  RGL  IPC  HMGB2  GFAP \nf. HMGB2  GFAP GFAPHighHMGB2LowRGLGFAPLowHMGB2HighIPCGFAP  HMGB2 \n\n\n# Fig6.  DG  GCs \n\n\n\nFig6.  DG  GCs \n\na. GC genesis-related  (neuroblast, n=1,898 cells; GC_im, n=7,547 cells; and GC_1, n=23,491 cellsUMAP)  UMAP  RNA velocities(RNA) RNA velocity (neuroblastsNB) immature GCs mature GCs  GC specification trajectory\nb.  RNA velocities  GC \nc.  GC  immature GC(GC_imn=7,547)  mature GC(GC_1n=23,491) DEG GCs(chemical synaptic transmission), (neuron projection morphogenesis)  SNARE binding pathwaysGCs\nd.  SGZ  immature GCsDCX, immature neuron marker; PROX1, GC marker; DAPI, nuclei\ne.  UMAP  neuroblasts (ASAP2 and TMEM159), immature GCs (COTL1)mature GCs (NOV)\n\n\n# Fig7.  DG  GCs \n\n\n\nFig7.  DG  GCs \n\n NSC  scRNA-seq  6988 \naj. \nk. (n=6988 cells) marker genes UMAP NES(RGL marker); HMGB2(IPC marker); MKI67(proliferating cell marker); PAX6(neural progenitor marker)\ni.  NSC marker NESTIN \nm.  NESTIN  EdU \nn. proliferating neural progenitor marker HMGB2\no. proliferating cell marker MKI67\np. neural progenitor marker PAX6\nq. neuronal marker MAP2\nr. astrocyte marker GFAP\ns. oligodendrocyte marker CNPase\nt. UMAPSCCAF1dSCCAFUMAP\n\n\n# Extended Results\n\n\n# Extended Data Fig1. \n\n\n\nExtended Data Fig1. \n\nab. counts  gene \nc.  10x version  animal ID  UMAP\nd.  sample  UMI counts\ne.  sample  genes\nf. \ng. \n\n\n# Extended Data Fig2. (gene regulatory networksGRNstermed regulons)\n\n\n\nExtended Data Fig2. (gene regulatory networksGRNstermed regulons)\n\na. neurogenic-related  regulon score tSNE plot\nb. \nc-f.  SCENIC \n\n\n# Extended Data Fig3. \n\n\n\nExtended Data Fig3. \n\na. \nb. Y46(Young)  Y1315(Middle-aged) neuroblasts(NB) DEGs\nc. Young  Middle-aged neurogenic-related  Log2 FC  stripchart\nd. Young adult  Middle-aged adult  neurogenic-related  Pair-wise comparison Log2FC>0.5 \ne.  Young adult  Middle-aged adult   Young adult  Middle-aged adult  neurogenesis \n\n\n# Extended Data Fig4. \n\n\n\nExtended Data Fig4. \n\na. \nb. Y46(Young)  Y1315(Middle-aged) neuroblasts(NB) DEGs\nc. females and males neurogenic-related  Log2 FC  stripchart\nd. females and males  neurogenic-related  Pair-wise comparison Log2FC>0.5 \ne.  females  males  (n=6)(n=2)(neurogenic process)\n\n\n# Extended Data Fig5. SGZ\n\n\n\nExtended Data Fig5. SGZ\n\na.  NESTIN (neural precursor cellsNPCs) SGZ  ML \nb. (proliferating marker) MKI67  SGZ  proliferating NPCs \nc.  HOPX  SOX2  GFAP  SGZ  RGL  RGLs\nd. ASCL1+ SOX2+ GFAP+  RGLs (proliferating neural progenitors)\n\n\n# Extended Data Fig6. \n\n\n\nExtended Data Fig6. \n\n\n# Extended Data Fig7. \n\n\n\nExtended Data Fig7. \n\n\n# Extended Data Fig8. \n\n\n\nExtended Data Fig8. \n\na. neurogenic-related population  PCA \nb.  neurogenic marker genes  PCA \nc.  scVelo  RNA velocities  PCA \nd. \ne.  Slingshot \nf.  Slingshot  RGLIPCNBGC \n\n\n# Extended Data Fig9. \n\n\n\nExtended Data Fig9. \n\na.  Neurogenic markers\nb.  marker genes\nc.  RGLsIPCsneuroblasts(NB)GCsAstrocytesOligodendrocytes \nd. (n=39555)(n=3711) \n\n\n# Extended Data Fig10. \n\n\n\nExtended Data Fig10. \n\na-d. \ne. (n=6,988 cells) UMAP VIM(RGL and astrocyte marker); NNAT(novel neuroblast marker); SOX4(neuroblast marker); TUJ1(neuronal marker)\nf-i.  marker gene \nj.  Hochgerner_1  UMAP IPC, intermediate progenitor cell; NB, neuroblast; GC, granule cell; Astro, astrocyte",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-022-01073-x\ncite: hao, zz., wei, jr., xiao, d. et al. single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations. nat neurosci 25, 805817 (2022).\n\n\n\n\n\n\n\nsheng liu()              yizhi liu()       mengqing xiang()\n                                                \n            \nliush87@mail.sysu.edu.cn   yzliu62@yahoo.com    xiangmq3@mail.sysu.edu.cn\n\n\n# abstract\n\n(neurogenesis) single-cell rna sequencing pipeline 207,785  34 (radial glia-like cellsrgls)(intermediate progenitor cellsipcs)(neuroblastsnb) hmgb2  ipc  neurogenesis (neural precursor cellsnpcs)\n\n\n# results\n\n\n# fig1. \n\n\n\nfig1. \n\na.  8  ( 6  2 )  dg  10x genomics  207,785 \nb. 207,785  umap  34 \ncd.  marker gene\n(1) neuroglia \nastro, astrocytes ;aqp4+\nastro_im, immature astrocytes ;aqp4+\nopc, oligodendrocyte progenitor cells ; c1ql1+\nnfol, newly formed oligodendrocytes ; bcas1+\nmol, mature oligodendrocytes ; mog+\n(2) neurons \ngaba, gabaergic neurons gaba; gad2+\npyr, pyramidal neurons ; rgs4+\ngc_im, immature granule cells ; prox1+\ngc, granule cells ; prox1+\n(3) vascular cells \nendothelialflt1+\nvlmc, vascular leptomeningeal cells ; dcn+\n(4) immune cells \nmicroglia c1qa+\npvm, perivascular macrophages ; lyve1+\n(5) cr, cajalretzius cells; reln+\n(6) ependymalcfap126+\n(7) npcsneural precursor cells \nrgl_1 and rgl_2, radial glia-like cells ; slc1a3+\nipcintermediate progenitor cells ; ipc_1hmgb2+ascl1+\nipc_2hmgb2+ascl1 \nnb, neuroblasts ; sox4+\n(8) unk, unknown\n marker genes marker genes\nipcsrfc4\nneuroblastsnnat\ngcsnpy1r\ne. rgls  local environment cell-cell signaling pathways  rgls  wnt signaling pathway rgls \nf. (radial glia-like cellsrgls)(neural stem cellsnscs)rgls (astrocytes)(gfap)(neural precursor) sox2  slc1a3 dg  sgz  rgls\n\n\n# fig2. \n\n\n\nfig2. \n\na.  scvi (n=15,878 cells)(n=9,810 cellshochgerer_1 dataset) umap  harmony \nb. \ncd.  a  umap \ne.  neural genesis cells and neurons (n=800 cells)  npcs(n=132 cells) \nf. (n=15,878 cells)(n=6,323 cells)\n\n\n# fig3. rgls  astrocytes \n\n\n\nfig3. rgls  astrocytes \n\n astrocytes  rgls  rgls  astrocytes rgls  immature astrocytes scvelo  rgls  astrocytes  rna velocity(trajectories)scvelo  neurogenesis  newly transcribed, unspliced pre-mrnas with mature, spliced mrnas  rgls  astrocytes  astrocyte genesis trajectory\n\na.  rgl  astrocyte maturation-related  umap  (rgl_1, n=1,107 cells; rgl_2, n=1,246; astro_im1, n=1,661; astro_im2, n=698; astro_1, n=12,322) rna velocities (rna) \nb. scvelo  umap astrocyte genesis trajectory  rgl_1  rgl_2  immature astrocytes mature astrocytes rgls \nc.  colorbar  a \nd.  immature astrocytes  astrocytes  umap \ne. dg  rgl markers (sox2  gfap) (apical processes) sgz  ml()\nf. gfap  hilus  astrocytes rgls \ng. rgl_1  rgl_2  5575  degs go  kegg rgl_1 - rgl_2  wnt pathway  wif1(wnt inhibitory factor 1)rgccregulator of cell cycle nscs (neural stem cell) tpt1 1\n\n\n# fig4.  dg \n\n\n\nfig4.  dg \n\na.  neurogenic-related  immature gcs  umap  mknn graph plot\nipcs (n=163 cells, )  neuroblasts (n=1,904 cells) and gc_im cells (n=7,547 cells),  (pyr_1 and pyr_2, n=1,384 and 123 cells)mknn  ipc_1  neuroblasts  immature gcs  neuroblasts  pyramidal neurons \nb.  a  mknn graphs  proliferating cells(mki67) npcs(sox2)ipcs(ascl1) neuroblasts(eomes and neurod2)  immature gcs (prox1) \nc. xy\nd.  sox2(precursor cell marker)  mki67(proliferating cell marker)  nscs (neural stem cell) \n\n\n# fig5. hmgb2  rgl  ipcs \n\n\n\nfig5. hmgb2  rgl  ipcs \n\na. ipcs  umap  mki67  ezh2 ipcs ipc_2  ezh2\nb. rgls  ipcs  gfap  hmgb2 ipcs  hmgb2  nscs \nc. ipc  hmgb2  rgl  gfap  rgls  ipcs box1 and white arrowheads: rglgfaphmgb2box2 and yellow arrowheads: gfaphmgb2box3 and magenta arrowheads: hmgb2+hmgb2gfap\nd.  rgls (box 1 and the green cell, gfap+hmgb2)rgls  ipcs  (box 2 and the light green cell, gfap+ hmbg2+)ipcs (box 3 and the red cell, gfap hmgb2+)\ne. hmgb2  gfap  gfap hmgb2+ ipc  gfap+ hmgb2+  rgl  ipc  hmgb2  gfap \nf. hmgb2  gfap gfaphighhmgb2lowrglgfaplowhmgb2highipcgfap  hmgb2 \n\n\n# fig6.  dg  gcs \n\n\n\nfig6.  dg  gcs \n\na. gc genesis-related  (neuroblast, n=1,898 cells; gc_im, n=7,547 cells; and gc_1, n=23,491 cellsumap)  umap  rna velocities(rna) rna velocity (neuroblastsnb) immature gcs mature gcs  gc specification trajectory\nb.  rna velocities  gc \nc.  gc  immature gc(gc_imn=7,547)  mature gc(gc_1n=23,491) deg gcs(chemical synaptic transmission), (neuron projection morphogenesis)  snare binding pathwaysgcs\nd.  sgz  immature gcsdcx, immature neuron marker; prox1, gc marker; dapi, nuclei\ne.  umap  neuroblasts (asap2 and tmem159), immature gcs (cotl1)mature gcs (nov)\n\n\n# fig7.  dg  gcs \n\n\n\nfig7.  dg  gcs \n\n nsc  scrna-seq  6988 \naj. \nk. (n=6988 cells) marker genes umap nes(rgl marker); hmgb2(ipc marker); mki67(proliferating cell marker); pax6(neural progenitor marker)\ni.  nsc marker nestin \nm.  nestin  edu \nn. proliferating neural progenitor marker hmgb2\no. proliferating cell marker mki67\np. neural progenitor marker pax6\nq. neuronal marker map2\nr. astrocyte marker gfap\ns. oligodendrocyte marker cnpase\nt. umapsccaf1dsccafumap\n\n\n# extended results\n\n\n# extended data fig1. \n\n\n\nextended data fig1. \n\nab. counts  gene \nc.  10x version  animal id  umap\nd.  sample  umi counts\ne.  sample  genes\nf. \ng. \n\n\n# extended data fig2. (gene regulatory networksgrnstermed regulons)\n\n\n\nextended data fig2. (gene regulatory networksgrnstermed regulons)\n\na. neurogenic-related  regulon score tsne plot\nb. \nc-f.  scenic \n\n\n# extended data fig3. \n\n\n\nextended data fig3. \n\na. \nb. y46(young)  y1315(middle-aged) neuroblasts(nb) degs\nc. young  middle-aged neurogenic-related  log2 fc  stripchart\nd. young adult  middle-aged adult  neurogenic-related  pair-wise comparison log2fc>0.5 \ne.  young adult  middle-aged adult   young adult  middle-aged adult  neurogenesis \n\n\n# extended data fig4. \n\n\n\nextended data fig4. \n\na. \nb. y46(young)  y1315(middle-aged) neuroblasts(nb) degs\nc. females and males neurogenic-related  log2 fc  stripchart\nd. females and males  neurogenic-related  pair-wise comparison log2fc>0.5 \ne.  females  males  (n=6)(n=2)(neurogenic process)\n\n\n# extended data fig5. sgz\n\n\n\nextended data fig5. sgz\n\na.  nestin (neural precursor cellsnpcs) sgz  ml \nb. (proliferating marker) mki67  sgz  proliferating npcs \nc.  hopx  sox2  gfap  sgz  rgl  rgls\nd. ascl1+ sox2+ gfap+  rgls (proliferating neural progenitors)\n\n\n# extended data fig6. \n\n\n\nextended data fig6. \n\n\n# extended data fig7. \n\n\n\nextended data fig7. \n\n\n# extended data fig8. \n\n\n\nextended data fig8. \n\na. neurogenic-related population  pca \nb.  neurogenic marker genes  pca \nc.  scvelo  rna velocities  pca \nd. \ne.  slingshot \nf.  slingshot  rglipcnbgc \n\n\n# extended data fig9. \n\n\n\nextended data fig9. \n\na.  neurogenic markers\nb.  marker genes\nc.  rglsipcsneuroblasts(nb)gcsastrocytesoligodendrocytes \nd. (n=39555)(n=3711) \n\n\n# extended data fig10. \n\n\n\nextended data fig10. \n\na-d. \ne. (n=6,988 cells) umap vim(rgl and astrocyte marker); nnat(novel neuroblast marker); sox4(neuroblast marker); tuj1(neuronal marker)\nf-i.  marker gene \nj.  hochgerner_1  umap ipc, intermediate progenitor cell; nb, neuroblast; gc, granule cell; astro, astrocyte",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Molecular logic of cellular diversification in the mouse cerebral cortex",frontmatter:{title:" --- Molecular logic of cellular diversification in the mouse cerebral cortex",date:"2022-12-09T19:01:17.000Z",permalink:"/pages/8a50f3/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20210623.(2021.06.23%20%E5%B0%8F%E9%BC%A0%E7%9A%AE%E5%B1%82%E5%8F%91%E8%82%B2%20nature)%20Molecular%20logic%20of%20cellular%20diversification%20in%20the%20mouse%20cerebral%20cortex.html",relativePath:"02./11./04./20210623.(2021.06.23  nature) Molecular logic of cellular diversification in the mouse cerebral cortex.md",key:"v-7b5d7342",path:"/pages/8a50f3/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:450},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:854},{level:3,title:"Fig1. Comprehensive atlas of mouse cortical development",slug:"fig1-comprehensive-atlas-of-mouse-cortical-development",normalizedTitle:"fig1. comprehensive atlas of mouse cortical development",charIndex:866},{level:3,title:"Fig2. Spatial distribution of cell types in the developing cortex",slug:"fig2-spatial-distribution-of-cell-types-in-the-developing-cortex",normalizedTitle:"fig2. spatial distribution of cell types in the developing cortex",charIndex:2108},{level:3,title:"Fig3. Molecular developmental trajectories of neocortical cell types",slug:"fig3-molecular-developmental-trajectories-of-neocortical-cell-types",normalizedTitle:"fig3. molecular developmental trajectories of neocortical cell types",charIndex:2379},{level:3,title:"Fig4. scATAC-seq landscape of the developing neocortex",slug:"fig4-scatac-seq-landscape-of-the-developing-neocortex",normalizedTitle:"fig4. scatac-seq landscape of the developing neocortex",charIndex:2684},{level:3,title:"Fig5. FEZF2 prevents the acquisition of callosal identity in CFuPN",slug:"fig5-fezf2-prevents-the-acquisition-of-callosal-identity-in-cfupn",normalizedTitle:"fig5. fezf2 prevents the acquisition of callosal identity in cfupn",charIndex:2997},{level:2,title:"Extended",slug:"extended",normalizedTitle:"extended",charIndex:3388},{level:3,title:"Extended Data Fig1. Classification of cell types in scRNA-seq data from individual time points",slug:"extended-data-fig1-classification-of-cell-types-in-scrna-seq-data-from-individual-time-points",normalizedTitle:"extended data fig1. classification of cell types in scrna-seq data from individual time points",charIndex:3401},{level:3,title:"Extended Data Fig2. Molecular signatures and interneuron heterogeneity in the developing cerebral cortex",slug:"extended-data-fig2-molecular-signatures-and-interneuron-heterogeneity-in-the-developing-cerebral-cortex",normalizedTitle:"extended data fig2. molecular signatures and interneuron heterogeneity in the developing cerebral cortex",charIndex:4260},{level:3,title:"Extended Data Fig3. Spatial mappings of cell types in the developing cerebral cortex",slug:"extended-data-fig3-spatial-mappings-of-cell-types-in-the-developing-cerebral-cortex",normalizedTitle:"extended data fig3. spatial mappings of cell types in the developing cerebral cortex",charIndex:5056},{level:3,title:"Extended Data Fig4. Consistent ordering of cells in developmental trajectories and characterization of branching tree of cortical development",slug:"extended-data-fig4-consistent-ordering-of-cells-in-developmental-trajectories-and-characterization-of-branching-tree-of-cortical-development",normalizedTitle:"extended data fig4. consistent ordering of cells in developmental trajectories and characterization of branching tree of cortical development",charIndex:5462},{level:3,title:"Extended Data Fig5. Neuronal cell types diverge post-mitotically",slug:"extended-data-fig5-neuronal-cell-types-diverge-post-mitotically",normalizedTitle:"extended data fig5. neuronal cell types diverge post-mitotically",charIndex:5945},{level:3,title:"Extended Data Fig6. Novel expression pattern of selected genes and NMF gene modules",slug:"extended-data-fig6-novel-expression-pattern-of-selected-genes-and-nmf-gene-modules",normalizedTitle:"extended data fig6. novel expression pattern of selected genes and nmf gene modules",charIndex:6446},{level:3,title:"Extended Data Fig7. Genetic cascades accompanying the development of cortical cell types",slug:"extended-data-fig7-genetic-cascades-accompanying-the-development-of-cortical-cell-types",normalizedTitle:"extended data fig7. genetic cascades accompanying the development of cortical cell types",charIndex:6576},{level:3,title:"Extended Data Fig8. Extended analysis of genes distinguishing between branches in URD tree",slug:"extended-data-fig8-extended-analysis-of-genes-distinguishing-between-branches-in-urd-tree",normalizedTitle:"extended data fig8. extended analysis of genes distinguishing between branches in urd tree",charIndex:6710},{level:3,title:"Extended Data Fig9. Characterization of scATACseq atlas and developmental trajectories of accessible elements through cortical development",slug:"extended-data-fig9-characterization-of-scatac-seq-atlas-and-developmental-trajectories-of-accessible-elements-through-cortical-development",normalizedTitle:"extended data fig9. characterization of scatacseq atlas and developmental trajectories of accessible elements through cortical development",charIndex:6845},{level:3,title:"Extended Data Fig10. Transcription factors with binding sites enriched along cortical development",slug:"extended-data-fig10-transcription-factors-with-binding-sites-enriched-along-cortical-development",normalizedTitle:"extended data fig10. transcription factors with binding sites enriched along cortical development",charIndex:7045},{level:3,title:"Extended Data Fig11. Use of developmental atlas to investigate Fezf2-knockout cortex",slug:"extended-data-fig11-use-of-developmental-atlas-to-investigate-fezf2-knockout-cortex",normalizedTitle:"extended data fig11. use of developmental atlas to investigate fezf2-knockout cortex",charIndex:7183},{level:3,title:"Extended Data Fig12. CFuPN acquire CThPN-like and layers 5&6 CPN-like identities in the absence of Fezf2",slug:"extended-data-fig12-cfupn-acquire-cthpn-like-and-layers-5-6-cpn-like-identities-in-the-absence-of-fezf2",normalizedTitle:"extended data fig12. cfupn acquire cthpn-like and layers 5&amp;6 cpn-like identities in the absence of fezf2",charIndex:null}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-021-03670-5" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-021-03670-5<OutboundLink/></a><br>\nCite: Di Bella, D.J., Habibi, E., Stickels, R.R. et al. Molecular logic of cellular diversification in the mouse cerebral cortex. <em>Nature</em> 595, 554559 (2021).</p>\n',headersStr:"Abstract Results Fig1. Comprehensive atlas of mouse cortical development Fig2. Spatial distribution of cell types in the developing cortex Fig3. Molecular developmental trajectories of neocortical cell types Fig4. scATAC-seq landscape of the developing neocortex Fig5. FEZF2 prevents the acquisition of callosal identity in CFuPN Extended Extended Data Fig1. Classification of cell types in scRNA-seq data from individual time points Extended Data Fig2. Molecular signatures and interneuron heterogeneity in the developing cerebral cortex Extended Data Fig3. Spatial mappings of cell types in the developing cerebral cortex Extended Data Fig4. Consistent ordering of cells in developmental trajectories and characterization of branching tree of cortical development Extended Data Fig5. Neuronal cell types diverge post-mitotically Extended Data Fig6. Novel expression pattern of selected genes and NMF gene modules Extended Data Fig7. Genetic cascades accompanying the development of cortical cell types Extended Data Fig8. Extended analysis of genes distinguishing between branches in URD tree Extended Data Fig9. Characterization of scATACseq atlas and developmental trajectories of accessible elements through cortical development Extended Data Fig10. Transcription factors with binding sites enriched along cortical development Extended Data Fig11. Use of developmental atlas to investigate Fezf2-knockout cortex Extended Data Fig12. CFuPN acquire CThPN-like and layers 5&6 CPN-like identities in the absence of Fezf2",content:": \nDOI: https://doi.org/10.1038/s41586-021-03670-5\nCite: Di Bella, D.J., Habibi, E., Stickels, R.R. et al. Molecular logic of cellular diversification in the mouse cerebral cortex. Nature 595, 554559 (2021).\n\n\n\n\n\n\n\nPAOLA ARLOTTA               AVIV REGEV\n                            \nHarvard University          MIT (Department of Biology)\npaola_arlotta@harvard.edu   aviv.regev.sc@gmail.com\n\n\n# Abstract\n\n scRNAseq  scATACseq \n\n\n# Results\n\n\n# Fig1. Comprehensive atlas of mouse cortical development\n\n\n\nFig1. \n\na. \n\n * CR cells(CajalRetzius cells); CP(cortical plate); SVZ(subventricular zone); VZ(ventricular zone)\n * \n   * E10.5 and E11.5\n   * E12.5 and E13.5L6  L5 \n   * E14.5 to E17.5L4  L2/3 \n   * E18.5, P1, P4\n\nb.  scRNA-seq  UMAP \n\nc. normalized\n\n * E10.5, E11.5 apical progenitors(AP; expressing Sox2, Pax6 and Hes5)  intermediate progenitors(IP; Eomes, Neurog2, Btg2) \n *  E12.5  progenitors  projection neurons(PN; Neurod2, Tubb3, Neurod6)  corticofugal projection neurons(CFuPN)  callosal projection neurons(CPN)\n *  E13.5  inhibitory interneurons(Dlx2, Gad1, Gad2)medial ganglionic eminence (MGE)-derived interneurons (Sst, Npy, Lhx6, Nxph1, Nxph2)  E13.5 \n *  E15.5  caudal ganglionic eminence (CGE)-derived interneurons (Pax6, Sp8, Cxcl14, Htr3a)\n *  E17.5  Oligodendrocyte precursor cells(OPCOlig1Olig2Pdgfra) astrocytes (Apoe, Aldh1l1, Slc1a3)\n *  E18.5  Htr3a-positive interneurons(Meis2Etv1Sp8)-\n\nd.  UMAP \n\n\n# Fig2. Spatial distribution of cell types in the developing cortex\n\n\n\nFig2. \n\na.  Tangram  scRNA-seq  spatial transcriptomes \nb.  E15.5 Migrating neurons  radial axis  migrating neuron substates \n\n\n# Fig3. Molecular developmental trajectories of neocortical cell types\n\n\n\nFig3. \n\na. URD branching treeRoot  E10.5 earliest progenitorstips  P4 terminal cell types identity \nb. layers 2&3 CPN and SCPN \nc.  NMF \nd.  10  TFs\n\n\n# Fig4. scATAC-seq landscape of the developing neocortex\n\n\n\nFig4. scATAC-seq\n\na.  scATAC-seq  UMAP  scRNA-seq \nbc. URD Root  E13.5 tips  E18.5 >70%(b)(c)\nd. TF motifs  ATAC tree \ne. \n\n\n# Fig5. FEZF2 prevents the acquisition of callosal identity in CFuPN\n\n\n\nFig5. FEZF2  CFuPN \n\na.  Fezf2 \nb. Fezf2 KOE15.5 SCPN  CThPN  30  APIP(N)\nc.  E15.5  scRNA-seq  UMAP \nd. \ne. UMAPCThPN  layer 5&6 CPNDL neurons, deep-layer neurons\n\n\n# Extended\n\n\n# Extended Data Fig1. Classification of cell types in scRNA-seq data from individual time points\n\n\n\nExtended Data Fig1.  scRNA-seq \n\na. \nb. mRNAcounts\nc. (apical, AP)(intermediate progenitors, IP)AP+IP  E10. 5 77% E11.5  80% E12.5 69% E13.5  66%\nd. E12.5  E18.5  male(M, Xist expression < 1)  female(F, Xist expression > 1) (Pearson)x Xist  Tsixy Ddx3y  Eif2s3ye\ne.  E18.5 male  female \nf.  UMAP  dorsal derivativesEmx1apical progenitorsSox2intermediate progenitorsEomesexcitatory neuronsNeurod2Fezf2Satb2inhibitory interneuronsDlx2 glial cellsApoenormalized\n\n\n# Extended Data Fig2. Molecular signatures and interneuron heterogeneity in the developing cerebral cortex\n\n\n\nExtended Data Fig2. \n\na.  scRNA-seq normalized\nb.  top20 \nc.  marker gene  UMAP \nd.  UMAP  UMAP  inhibitory markers Dlx2  Gad2  dorsally-derived cell types marker (Emx1) \ne.  (MGE)  NpySstLhx6  Nxph1Medial ganglionic eminence (MGE)-derived interneurons Htr3aProx1Cxcl14  Sp8 caudal ganglionic eminence (CGE)-derived interneurons  Htr3a, Prox1, Cxcl14  Sp8 Meis2+ P-SP boundary derived interneurons  Meis2, Etv1  Sp8\n\n\n# Extended Data Fig3. Spatial mappings of cell types in the developing cerebral cortex\n\n\n\nExtended Data Fig3. \n\na.  Tangram  scRNA-seq developmental atlas  tissue spatial transcriptomes (Slide-seq v2) \nb. \nc.  marker gene \nd. P1 cortex  deep layers \ne. \nf. E15.5  migrating neurons sub-states \n\n\n# Extended Data Fig4. Consistent ordering of cells in developmental trajectories and characterization of branching tree of cortical development\n\n\n\nExtended Data Fig4. \n\na.  scRNA-seq  pseudotime UMAP  5 \nb. URD  pseudotime values  4 \nc.  Monocle3 \nd.  Monocle3  URD  pseudotime \ne.  URD \nf.  binned pseudotime \ng.  binned pseudotime \n\n\n# Extended Data Fig5. Neuronal cell types diverge post-mitotically\n\n\n\nExtended Data Fig5. \n\na. Branching trees  marker gene apical progenitors (Sox2, Hes5)intermediate progenitors (Eomes)excitatory neurons (Neurod2)callosal neurons (Satb2, Cux2)layer 4 stellate neurons (Rorb)corticofugal neurons (Fezf2, Tle4, Pcp4, Tcerg1l)putative near-projecting neurons (Tshz2)astrocytes (Slc1a3, Aqp4, Aldh1l1)ependymocytes (Foxj1)\nb. \nc. \nd. Tangram \n\n\n# Extended Data Fig6. Novel expression pattern of selected genes and NMF gene modules\n\n\n\nExtended Data Fig6. NMF\n\n\n# Extended Data Fig7. Genetic cascades accompanying the development of cortical cell types\n\n\n\nExtended Data Fig7. \n\n\n# Extended Data Fig8. Extended analysis of genes distinguishing between branches in URD tree\n\n\n\nExtended Data Fig8. URD\n\n\n# Extended Data Fig9. Characterization of scATACseq atlas and developmental trajectories of accessible elements through cortical development\n\n\n\nExtended Data Fig9. scATAC-seq\n\n\n# Extended Data Fig10. Transcription factors with binding sites enriched along cortical development\n\n\n\nExtended Data Fig10. \n\n\n# Extended Data Fig11. Use of developmental atlas to investigate Fezf2-knockout cortex\n\n\n\nExtended Data Fig11. fezf2\n\n\n# Extended Data Fig12. CFuPN acquire CThPN-like and layers 5&6 CPN-like identities in the absence of Fezf2\n\n\n\nExtended Data Fig12. Fezf2CFuPNCthPN56cpn",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-021-03670-5\ncite: di bella, d.j., habibi, e., stickels, r.r. et al. molecular logic of cellular diversification in the mouse cerebral cortex. nature 595, 554559 (2021).\n\n\n\n\n\n\n\npaola arlotta               aviv regev\n                            \nharvard university          mit (department of biology)\npaola_arlotta@harvard.edu   aviv.regev.sc@gmail.com\n\n\n# abstract\n\n scrnaseq  scatacseq \n\n\n# results\n\n\n# fig1. comprehensive atlas of mouse cortical development\n\n\n\nfig1. \n\na. \n\n * cr cells(cajalretzius cells); cp(cortical plate); svz(subventricular zone); vz(ventricular zone)\n * \n   * e10.5 and e11.5\n   * e12.5 and e13.5l6  l5 \n   * e14.5 to e17.5l4  l2/3 \n   * e18.5, p1, p4\n\nb.  scrna-seq  umap \n\nc. normalized\n\n * e10.5, e11.5 apical progenitors(ap; expressing sox2, pax6 and hes5)  intermediate progenitors(ip; eomes, neurog2, btg2) \n *  e12.5  progenitors  projection neurons(pn; neurod2, tubb3, neurod6)  corticofugal projection neurons(cfupn)  callosal projection neurons(cpn)\n *  e13.5  inhibitory interneurons(dlx2, gad1, gad2)medial ganglionic eminence (mge)-derived interneurons (sst, npy, lhx6, nxph1, nxph2)  e13.5 \n *  e15.5  caudal ganglionic eminence (cge)-derived interneurons (pax6, sp8, cxcl14, htr3a)\n *  e17.5  oligodendrocyte precursor cells(opcolig1olig2pdgfra) astrocytes (apoe, aldh1l1, slc1a3)\n *  e18.5  htr3a-positive interneurons(meis2etv1sp8)-\n\nd.  umap \n\n\n# fig2. spatial distribution of cell types in the developing cortex\n\n\n\nfig2. \n\na.  tangram  scrna-seq  spatial transcriptomes \nb.  e15.5 migrating neurons  radial axis  migrating neuron substates \n\n\n# fig3. molecular developmental trajectories of neocortical cell types\n\n\n\nfig3. \n\na. urd branching treeroot  e10.5 earliest progenitorstips  p4 terminal cell types identity \nb. layers 2&3 cpn and scpn \nc.  nmf \nd.  10  tfs\n\n\n# fig4. scatac-seq landscape of the developing neocortex\n\n\n\nfig4. scatac-seq\n\na.  scatac-seq  umap  scrna-seq \nbc. urd root  e13.5 tips  e18.5 >70%(b)(c)\nd. tf motifs  atac tree \ne. \n\n\n# fig5. fezf2 prevents the acquisition of callosal identity in cfupn\n\n\n\nfig5. fezf2  cfupn \n\na.  fezf2 \nb. fezf2 koe15.5 scpn  cthpn  30  apip(n)\nc.  e15.5  scrna-seq  umap \nd. \ne. umapcthpn  layer 5&6 cpndl neurons, deep-layer neurons\n\n\n# extended\n\n\n# extended data fig1. classification of cell types in scrna-seq data from individual time points\n\n\n\nextended data fig1.  scrna-seq \n\na. \nb. mrnacounts\nc. (apical, ap)(intermediate progenitors, ip)ap+ip  e10. 5 77% e11.5  80% e12.5 69% e13.5  66%\nd. e12.5  e18.5  male(m, xist expression < 1)  female(f, xist expression > 1) (pearson)x xist  tsixy ddx3y  eif2s3ye\ne.  e18.5 male  female \nf.  umap  dorsal derivativesemx1apical progenitorssox2intermediate progenitorseomesexcitatory neuronsneurod2fezf2satb2inhibitory interneuronsdlx2 glial cellsapoenormalized\n\n\n# extended data fig2. molecular signatures and interneuron heterogeneity in the developing cerebral cortex\n\n\n\nextended data fig2. \n\na.  scrna-seq normalized\nb.  top20 \nc.  marker gene  umap \nd.  umap  umap  inhibitory markers dlx2  gad2  dorsally-derived cell types marker (emx1) \ne.  (mge)  npysstlhx6  nxph1medial ganglionic eminence (mge)-derived interneurons htr3aprox1cxcl14  sp8 caudal ganglionic eminence (cge)-derived interneurons  htr3a, prox1, cxcl14  sp8 meis2+ p-sp boundary derived interneurons  meis2, etv1  sp8\n\n\n# extended data fig3. spatial mappings of cell types in the developing cerebral cortex\n\n\n\nextended data fig3. \n\na.  tangram  scrna-seq developmental atlas  tissue spatial transcriptomes (slide-seq v2) \nb. \nc.  marker gene \nd. p1 cortex  deep layers \ne. \nf. e15.5  migrating neurons sub-states \n\n\n# extended data fig4. consistent ordering of cells in developmental trajectories and characterization of branching tree of cortical development\n\n\n\nextended data fig4. \n\na.  scrna-seq  pseudotime umap  5 \nb. urd  pseudotime values  4 \nc.  monocle3 \nd.  monocle3  urd  pseudotime \ne.  urd \nf.  binned pseudotime \ng.  binned pseudotime \n\n\n# extended data fig5. neuronal cell types diverge post-mitotically\n\n\n\nextended data fig5. \n\na. branching trees  marker gene apical progenitors (sox2, hes5)intermediate progenitors (eomes)excitatory neurons (neurod2)callosal neurons (satb2, cux2)layer 4 stellate neurons (rorb)corticofugal neurons (fezf2, tle4, pcp4, tcerg1l)putative near-projecting neurons (tshz2)astrocytes (slc1a3, aqp4, aldh1l1)ependymocytes (foxj1)\nb. \nc. \nd. tangram \n\n\n# extended data fig6. novel expression pattern of selected genes and nmf gene modules\n\n\n\nextended data fig6. nmf\n\n\n# extended data fig7. genetic cascades accompanying the development of cortical cell types\n\n\n\nextended data fig7. \n\n\n# extended data fig8. extended analysis of genes distinguishing between branches in urd tree\n\n\n\nextended data fig8. urd\n\n\n# extended data fig9. characterization of scatacseq atlas and developmental trajectories of accessible elements through cortical development\n\n\n\nextended data fig9. scatac-seq\n\n\n# extended data fig10. transcription factors with binding sites enriched along cortical development\n\n\n\nextended data fig10. \n\n\n# extended data fig11. use of developmental atlas to investigate fezf2-knockout cortex\n\n\n\nextended data fig11. fezf2\n\n\n# extended data fig12. cfupn acquire cthpn-like and layers 5&6 cpn-like identities in the absence of fezf2\n\n\n\nextended data fig12. fezf2cfupncthpn56cpn",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Single-cell spatial transcriptome reveals cell-typ organization in the macaque cortex",frontmatter:{title:" --- Single-cell spatial transcriptome reveals cell-typ organization in the macaque cortex",date:"2023-09-22T11:05:34.000Z",permalink:"/pages/8c91b0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20230712.(2023.07.12%20stereo-seq%E7%8C%95%E7%8C%B4%E7%9A%AE%E5%B1%82%20Cell)%20Single-cell%20spatial%20transcriptome%20reveals%20cell-type%20organization%20in%20the%20macaque%20cortex.html",relativePath:"02./11./04./20230712.(2023.07.12 stereo-seq Cell) Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex.md",key:"v-52142b16",path:"/pages/8c91b0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:305},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:657},{level:3,title:"Figure 1. Transcriptomic cell-type taxonomy of the macaque cortex",slug:"figure-1-transcriptomic-cell-type-taxonomy-of-the-macaque-cortex",normalizedTitle:"figure 1. transcriptomic cell-type taxonomy of the macaque cortex",charIndex:669},{level:3,title:"Figure 2. Single-cell spatial transcriptome map of the macaque cortex",slug:"figure-2-single-cell-spatial-transcriptome-map-of-the-macaque-cortex",normalizedTitle:"figure 2. single-cell spatial transcriptome map of the macaque cortex",charIndex:954},{level:3,title:"Figure 3. Association between cell-type distribution and cortical hierarchy",slug:"figure-3-association-between-cell-type-distribution-and-cortical-hierarchy",normalizedTitle:"figure 3. association between cell-type distribution and cortical hierarchy",charIndex:1700},{level:3,title:"Figure 4. Layer and regional preferences of glutamatergic cell types",slug:"figure-4-layer-and-regional-preferences-of-glutamatergic-cell-types",normalizedTitle:"figure 4. layer and regional preferences of glutamatergic cell types",charIndex:2042},{level:3,title:"Figure 5. Organization of GABAergic cell types",slug:"figure-5-organization-of-gabaergic-cell-types",normalizedTitle:"figure 5. organization of gabaergic cell types",charIndex:2554},{level:3,title:"Figure 6. Distribution of non-neuronal cell types",slug:"figure-6-distribution-of-non-neuronal-cell-types",normalizedTitle:"figure 6. distribution of non-neuronal cell types",charIndex:3079},{level:3,title:"Figure 7. Primate-specific cell types",slug:"figure-7-primate-specific-cell-types",normalizedTitle:"figure 7. primate-specific cell types",charIndex:3436},{level:2,title:"Supplemental Figure",slug:"supplemental-figure",normalizedTitle:"supplemental figure",charIndex:3819},{level:3,title:"Figure S1. Regional parcellation and cell-type identification, related to Figure 1",slug:"figure-s1-regional-parcellation-and-cell-type-identification-related-to-figure-1",normalizedTitle:"figure s1. regional parcellation and cell-type identification, related to figure 1",charIndex:3843},{level:3,title:"Figure S2. Stereo-seq single-cell registration and cell-type annotation, related to Figure 2",slug:"figure-s2-stereo-seq-single-cell-registration-and-cell-type-annotation-related-to-figure-2",normalizedTitle:"figure s2. stereo-seq single-cell registration and cell-type annotation, related to figure 2",charIndex:4378},{level:3,title:"Figure S3. Cell-type distribution across hierarchies, related to Figure 3",slug:"figure-s3-cell-type-distribution-across-hierarchies-related-to-figure-3",normalizedTitle:"figure s3. cell-type distribution across hierarchies, related to figure 3",charIndex:5099},{level:3,title:"Figure S4. Distribution of glutamatergic cell types, related to Figure 4",slug:"figure-s4-distribution-of-glutamatergic-cell-types-related-to-figure-4",normalizedTitle:"figure s4. distribution of glutamatergic cell types, related to figure 4",charIndex:5382},{level:3,title:"Figure S5. GABAergic cell types, related to Figure 5",slug:"figure-s5-gabaergic-cell-types-related-to-figure-5",normalizedTitle:"figure s5. gabaergic cell types, related to figure 5",charIndex:5671},{level:3,title:"Figure S6. Non-neuronal cell types, related to Figure 6",slug:"figure-s6-non-neuronal-cell-types-related-to-figure-6",normalizedTitle:"figure s6. non-neuronal cell types, related to figure 6",charIndex:6255},{level:3,title:"Figure S7. Cross-species comparison of glutamatergic neurons, related to Figure 7",slug:"figure-s7-cross-species-comparison-of-glutamatergic-neurons-related-to-figure-7",normalizedTitle:"figure s7. cross-species comparison of glutamatergic neurons, related to figure 7",charIndex:6685}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2023.06.009" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2023.06.009<OutboundLink/></a><br>\nCite: Chen A, Sun Y, Lei Y, et al. Single-cell spatial transcriptome reveals cell-type organization in the\nmacaque cortex. <em>Cell</em>. 2023;186(17):3726-3743.e24.</p>\n',headersStr:"Abstract Results Figure 1. Transcriptomic cell-type taxonomy of the macaque cortex Figure 2. Single-cell spatial transcriptome map of the macaque cortex Figure 3. Association between cell-type distribution and cortical hierarchy Figure 4. Layer and regional preferences of glutamatergic cell types Figure 5. Organization of GABAergic cell types Figure 6. Distribution of non-neuronal cell types Figure 7. Primate-specific cell types Supplemental Figure Figure S1. Regional parcellation and cell-type identification, related to Figure 1 Figure S2. Stereo-seq single-cell registration and cell-type annotation, related to Figure 2 Figure S3. Cell-type distribution across hierarchies, related to Figure 3 Figure S4. Distribution of glutamatergic cell types, related to Figure 4 Figure S5. GABAergic cell types, related to Figure 5 Figure S6. Non-neuronal cell types, related to Figure 6 Figure S7. Cross-species comparison of glutamatergic neurons, related to Figure 7",content:": \nDOI: https://doi.org/10.1016/j.cell.2023.06.009\nCite: Chen A, Sun Y, Lei Y, et al. Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex. Cell. 2023;186(17):3726-3743.e24.\n\n\n\n\n\n\n\n\n\n\n# Abstract\n\n 143  snRNAseq  264  glutamatergicglutamatergic  4 \n\n\n# Results\n\n\n# Figure 1. Transcriptomic cell-type taxonomy of the macaque cortex\n\n\n\nFigure 1. \n\nA.  snRNA-seq  143  1493,240  stereo-seq  50 m  646,393  846,847 \nB. snRNA-seq  UMAP  264  clusters\n\n\n# Figure 2. Single-cell spatial transcriptome map of the macaque cortex\n\n\n\nFigure 2. \n\nA.  stereo-seq (Aa)  mRNA(Ab)  mRNA mRNA(Ac) nucleic acid (Ad) ConA (Ae)  nucleic acid  ConA  AI \nB. Ab\nC.  genes/cell \nD.  5 23  cell subclasses \nE. SII D  cell subclasses  spatial mapbottom density profilestop\nF.  cell types  marker genes \nG. regional cell-type  UMAP \nH. V146dF1  cell subclasses \n(I)9(J)\nI.  regions  cortical layers  Neighborhood complexity\nJ.  cortical layers  9  regions  Neighborhood complexity\n\n\n# Figure 3. Association between cell-type distribution and cortical hierarchy\n\n\n\nFigure 3. \n\nA-B. (A)(B) Felleman  Van Essen \nC-D.  UMAP region clusters\nE-F. \nG. (E)(F)\n\n\n# Figure 4. Layer and regional preferences of glutamatergic cell types\n\n\n\nFigure 4. \n\nA. top9 middlebelow\nB.  A-P EBZ \nC-D. 3  V1F1temporal pole  (C)  (D)\nE.  (D)  top marker genes \nF. (D)  marker gene \nG. V1  NTNG1  RNAscope \nH. \nI.  (C)  UMAP\nJ. AMPANMDA \n\n\n# Figure 5. Organization of GABAergic cell types\n\n\n\nFigure 5. GABAergic \n\nA. GABAergic \nB. 3  GABAergic subclasses  A-P \nC.  SST  side  medial \nD.  G1  G2 SST groups \nE. SST  UMAP \nF.  CHC \nG.  (F)  CHC  marker genes \nH. UMAP  CHC CHC  UMAP \nI. PV vs. SST \nJ. PV  SST  PC1 \nK. PV  SST \n\n\n# Figure 6. Distribution of non-neuronal cell types\n\n\n\nFigure 6. \n\nA. \nB. ASC  5 \nC.  L1-  L6-  ASC  DEG \nD.  3  ASC \nE.  A-P \nF.  ASC  OLG \nG-H.  F1-enriched OLG.7 (G)  marker gene (H)\nI.  OLG \n\n\n# Figure 7. Primate-specific cell types\n\n\n\nFigure 7. \n\nA-C.  PFC (A)F1 (B)V1 (C)  snRNA-seq/scRNA-seq  UMAP\nD. 6  5 \nE.  DEGs \nF.  GO \nG.  3 \nH.  FOXP2EPHA3DCC  4 \nI. RNAscope  FOXP2  EPHA3 \n\n\n# Supplemental Figure\n\n\n# Figure S1. Regional parcellation and cell-type identification, related to Figure 1\n\n\n\nFigure S1. \n\nA-B.  Total mRNA  nucleic acid \nC.  A  NeuN \nD. \nE-F.  cortical layersV1V2 (E)  PEc  PGm (F)  C \nG. UMAP \nH.  marker genes  Jaccard \nI. \nJ.  subclasses \nK. Dot plot  subclasses  marker genes \nL.  glutamatergic subclasses  glutamatergic subclasses \n\n\n# Figure S2. Stereo-seq single-cell registration and cell-type annotation, related to Figure 2\n\n\n\nFigure S2. Stereo-seq \n\nA. stereo-seq (Aa)  mRNA(Ab)  mRNA mRNA(Ac) nucleic acid (Ad) ConA (Ae)  nucleic acid  ConA  AI \nB. Ab\nC. \nD.  snRNA-seq  stereo-seq  subclasses \nE. #2  #3  264 \nF.  EBZ +10  #1 \nG. \nH. \nI.  #2  #3 \nJ.  23  cell subclasses \nK.  snRNA-seq  regional cell-type  UMAP \nL. 9  brain regions \n\n\n# Figure S3. Cell-type distribution across hierarchies, related to Figure 3\n\n\n\nFigure S3. \n\nA. 143 \nB-C. (B)(C)\nD.  3C  3D  UMAP \nE. \nF. (E) marker gene \nG. (F) GO \n\n\n# Figure S4. Distribution of glutamatergic cell types, related to Figure 4\n\n\n\nFigure S4. \n\nA.  V1F1temporal pole \nB. (A)  marker gene \nC. \nD. AMPANMDAKainateDelta\n\n\n# Figure S5. GABAergic cell types, related to Figure 5\n\n\n\nFigure S5. GABAergic \n\nA. GABAergic subclasses \nB.  9  GABAergic \nC. GABAergic  3D (upper) and 2D (bottom) \nD. SST RNAscope  DAPI SST  Sankey \nE.  SST  marker genes \nF. GO terms for the genes in (E)\nG. SST  marker gene \nH.  SST \nI. GO terms among DEGs in G1- and G2-SST cells\nJ.  CHC \nK.  CHC \nL. PV  SST \nM.  PV  SST  PC1 \n\n\n# Figure S6. Non-neuronal cell types, related to Figure 6\n\n\n\nFigure S6. \n\nA.  marker gene \nB. #2  #3  5  ASCs \nC.  ASC.2  ASC.11  DEGs\nD.  ASC.11  ASC.1  ASC.2  DEGs \nE. GO terms  L1-  L6-enriched  ASCs  DEGs \nF. UMAP \nG.  marker gene \nH.  (G)  4  GO \nI.  F1-enriched OLG  marker genes\n\n\n# Figure S7. Cross-species comparison of glutamatergic neurons, related to Figure 7\n\n\n\nFigure S7. \n\nA-B. (A)(B) 3  PFCF1V1  8 /\nC. \nD.  3225  13  PL-ILA-ORB  snRNA-seq/scRNA-seq  UMAP\nE-F.  (E) (F)\nG.  PFC \nH.  PFCF1V1  GABAergic  snRNA-seq/scRNA-seq  UMAP",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2023.06.009\ncite: chen a, sun y, lei y, et al. single-cell spatial transcriptome reveals cell-type organization in the macaque cortex. cell. 2023;186(17):3726-3743.e24.\n\n\n\n\n\n\n\n\n\n\n# abstract\n\n 143  snrnaseq  264  glutamatergicglutamatergic  4 \n\n\n# results\n\n\n# figure 1. transcriptomic cell-type taxonomy of the macaque cortex\n\n\n\nfigure 1. \n\na.  snrna-seq  143  1493,240  stereo-seq  50 m  646,393  846,847 \nb. snrna-seq  umap  264  clusters\n\n\n# figure 2. single-cell spatial transcriptome map of the macaque cortex\n\n\n\nfigure 2. \n\na.  stereo-seq (aa)  mrna(ab)  mrna mrna(ac) nucleic acid (ad) cona (ae)  nucleic acid  cona  ai \nb. ab\nc.  genes/cell \nd.  5 23  cell subclasses \ne. sii d  cell subclasses  spatial mapbottom density profilestop\nf.  cell types  marker genes \ng. regional cell-type  umap \nh. v146df1  cell subclasses \n(i)9(j)\ni.  regions  cortical layers  neighborhood complexity\nj.  cortical layers  9  regions  neighborhood complexity\n\n\n# figure 3. association between cell-type distribution and cortical hierarchy\n\n\n\nfigure 3. \n\na-b. (a)(b) felleman  van essen \nc-d.  umap region clusters\ne-f. \ng. (e)(f)\n\n\n# figure 4. layer and regional preferences of glutamatergic cell types\n\n\n\nfigure 4. \n\na. top9 middlebelow\nb.  a-p ebz \nc-d. 3  v1f1temporal pole  (c)  (d)\ne.  (d)  top marker genes \nf. (d)  marker gene \ng. v1  ntng1  rnascope \nh. \ni.  (c)  umap\nj. ampanmda \n\n\n# figure 5. organization of gabaergic cell types\n\n\n\nfigure 5. gabaergic \n\na. gabaergic \nb. 3  gabaergic subclasses  a-p \nc.  sst  side  medial \nd.  g1  g2 sst groups \ne. sst  umap \nf.  chc \ng.  (f)  chc  marker genes \nh. umap  chc chc  umap \ni. pv vs. sst \nj. pv  sst  pc1 \nk. pv  sst \n\n\n# figure 6. distribution of non-neuronal cell types\n\n\n\nfigure 6. \n\na. \nb. asc  5 \nc.  l1-  l6-  asc  deg \nd.  3  asc \ne.  a-p \nf.  asc  olg \ng-h.  f1-enriched olg.7 (g)  marker gene (h)\ni.  olg \n\n\n# figure 7. primate-specific cell types\n\n\n\nfigure 7. \n\na-c.  pfc (a)f1 (b)v1 (c)  snrna-seq/scrna-seq  umap\nd. 6  5 \ne.  degs \nf.  go \ng.  3 \nh.  foxp2epha3dcc  4 \ni. rnascope  foxp2  epha3 \n\n\n# supplemental figure\n\n\n# figure s1. regional parcellation and cell-type identification, related to figure 1\n\n\n\nfigure s1. \n\na-b.  total mrna  nucleic acid \nc.  a  neun \nd. \ne-f.  cortical layersv1v2 (e)  pec  pgm (f)  c \ng. umap \nh.  marker genes  jaccard \ni. \nj.  subclasses \nk. dot plot  subclasses  marker genes \nl.  glutamatergic subclasses  glutamatergic subclasses \n\n\n# figure s2. stereo-seq single-cell registration and cell-type annotation, related to figure 2\n\n\n\nfigure s2. stereo-seq \n\na. stereo-seq (aa)  mrna(ab)  mrna mrna(ac) nucleic acid (ad) cona (ae)  nucleic acid  cona  ai \nb. ab\nc. \nd.  snrna-seq  stereo-seq  subclasses \ne. #2  #3  264 \nf.  ebz +10  #1 \ng. \nh. \ni.  #2  #3 \nj.  23  cell subclasses \nk.  snrna-seq  regional cell-type  umap \nl. 9  brain regions \n\n\n# figure s3. cell-type distribution across hierarchies, related to figure 3\n\n\n\nfigure s3. \n\na. 143 \nb-c. (b)(c)\nd.  3c  3d  umap \ne. \nf. (e) marker gene \ng. (f) go \n\n\n# figure s4. distribution of glutamatergic cell types, related to figure 4\n\n\n\nfigure s4. \n\na.  v1f1temporal pole \nb. (a)  marker gene \nc. \nd. ampanmdakainatedelta\n\n\n# figure s5. gabaergic cell types, related to figure 5\n\n\n\nfigure s5. gabaergic \n\na. gabaergic subclasses \nb.  9  gabaergic \nc. gabaergic  3d (upper) and 2d (bottom) \nd. sst rnascope  dapi sst  sankey \ne.  sst  marker genes \nf. go terms for the genes in (e)\ng. sst  marker gene \nh.  sst \ni. go terms among degs in g1- and g2-sst cells\nj.  chc \nk.  chc \nl. pv  sst \nm.  pv  sst  pc1 \n\n\n# figure s6. non-neuronal cell types, related to figure 6\n\n\n\nfigure s6. \n\na.  marker gene \nb. #2  #3  5  ascs \nc.  asc.2  asc.11  degs\nd.  asc.11  asc.1  asc.2  degs \ne. go terms  l1-  l6-enriched  ascs  degs \nf. umap \ng.  marker gene \nh.  (g)  4  go \ni.  f1-enriched olg  marker genes\n\n\n# figure s7. cross-species comparison of glutamatergic neurons, related to figure 7\n\n\n\nfigure s7. \n\na-b. (a)(b) 3  pfcf1v1  8 /\nc. \nd.  3225  13  pl-ila-orb  snrna-seq/scrna-seq  umap\ne-f.  (e) (f)\ng.  pfc \nh.  pfcf1v1  gabaergic  snrna-seq/scrna-seq  umap",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- The neuropeptide landscape of human prefrontal cortex",frontmatter:{title:" --- The neuropeptide landscape of human prefrontal cortex",date:"2023-05-19T16:53:32.000Z",permalink:"/pages/444fce/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20220621.(2022.06.21%20hPFC%E7%A5%9E%E7%BB%8F%E8%82%BD%E5%88%86%E6%9E%90%20PNAS)%20The%20neuropeptide%20landscape%20of%20human%20prefrontal%20cortex.html",relativePath:"02./11./04./20220621.(2022.06.21 hPFC PNAS) The neuropeptide landscape of human prefrontal cortex.md",key:"v-95c1c078",path:"/pages/444fce/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:320},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:930},{level:3,title:"Fig. 1. Transcriptomic analysis of the 17 microdissected PFC regions and 3 cortical reference regions",slug:"fig-1-transcriptomic-analysis-of-the-17-microdissected-pfc-regions-and-3-cortical-reference-regions",normalizedTitle:"fig. 1. transcriptomic analysis of the 17 microdissected pfc regions and 3 cortical reference regions",charIndex:942},{level:3,title:"Fig. 2. Expression levels of NPs and NPRs in hPFC",slug:"fig-2-expression-levels-of-nps-and-nprs-in-hpfc",normalizedTitle:"fig. 2. expression levels of nps and nprs in hpfc",charIndex:1400},{level:3,title:"Fig. 3. Regional variation of NPs and NPRs in hPFC",slug:"fig-3-regional-variation-of-nps-and-nprs-in-hpfc",normalizedTitle:"fig. 3. regional variation of nps and nprs in hpfc",charIndex:1755},{level:3,title:"Fig. 4. Categorization of peptidergic cell types in ACC based on expression profiles of NPs and their NPRs",slug:"fig-4-categorization-of-peptidergic-cell-types-in-acc-based-on-expression-profiles-of-nps-and-their-nprs",normalizedTitle:"fig. 4. categorization of peptidergic cell types in acc based on expression profiles of nps and their nprs",charIndex:1980},{level:3,title:"Fig. 5. Distribution of CCK, VGLUT1, and VGAT mRNA-positive cells in ACC",slug:"fig-5-distribution-of-cck-vglut1-and-vgat-mrna-positive-cells-in-acc",normalizedTitle:"fig. 5. distribution of cck, vglut1, and vgat mrna-positive cells in acc",charIndex:2905},{level:3,title:"Fig. 6. Expression of the GAL system in ACC",slug:"fig-6-expression-of-the-gal-system-in-acc",normalizedTitle:"fig. 6. expression of the gal system in acc",charIndex:3268},{level:3,title:"Fig. 7. Peptides from diverse afferent sources",slug:"fig-7-peptides-from-diverse-afferent-sources",normalizedTitle:"fig. 7. peptides from diverse afferent sources",charIndex:3658}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1073/pnas.2123146119" target="_blank" rel="noopener noreferrer">https://doi.org/10.1073/pnas.2123146119<OutboundLink/></a><br>\nCite: Zhong W, Barde S, Mitsios N, et al. The neuropeptide landscape of human prefrontal cortex. <em>Proc Natl Acad Sci U S A</em>. 2022 Aug 16;119(33):e2123146119.</p>\n',headersStr:"Abstract Results Fig. 1. Transcriptomic analysis of the 17 microdissected PFC regions and 3 cortical reference regions Fig. 2. Expression levels of NPs and NPRs in hPFC Fig. 3. Regional variation of NPs and NPRs in hPFC Fig. 4. Categorization of peptidergic cell types in ACC based on expression profiles of NPs and their NPRs Fig. 5. Distribution of CCK, VGLUT1, and VGAT mRNA-positive cells in ACC Fig. 6. Expression of the GAL system in ACC Fig. 7. Peptides from diverse afferent sources",content:': \nDOI: https://doi.org/10.1073/pnas.2123146119\nCite: Zhong W, Barde S, Mitsios N, et al. The neuropeptide landscape of human prefrontal cortex. Proc Natl Acad Sci U S A. 2022 Aug 16;119(33):e2123146119.\n\n\n\n\n\n\n\nTOMAS HKFELT\n\nKarolinska Institutet\ntomas.hokfelt@ki.se\n\n\n# Abstract\n\nhPFC hPFC  17 RNA  RNA-scope  hPFC  60  60 PFC / PFC  anterior cingulate cortex/orbitofrontal gyrus PFC >5-fold PFC //YY hPFC / PFC \n\n\n# Results\n\n\n# Fig. 1. Transcriptomic analysis of the 17 microdissected PFC regions and 3 cortical reference regions\n\n\n\nFig. 1.  17  PFC  3 \n\n(A) hPFC  17  3 (frontal, parietal, and temporal cortex)\n(B) 165  bulk RNAseq \n(C) 4  marker gene  20 GAD1 SLC17A7 SLC17A6  (17-fold) RefCorts PVALB  (14-fold)\n\n\n# Fig. 2. Expression levels of NPs and NPRs in hPFC\n\n\n\nFig. 2. hPFC  NPs  NPRs \n\n(A) 17  hPFC NPs78  NPs  60 nTPM>0.1\n(B) 17  hPFC NPRs83  NPRs  60 nTPM>0.1\n(C) RNAscope ADMPOMC1CRHR1NP/HCRTOXTHCRT/ORX2HCRTR2\n\n\n# Fig. 3. Regional variation of NPs and NPRs in hPFC\n\n\n\nFig. 3. hPFC  NPs  NPRs \n\n(A)  Z-score  NPs  NPs  "NP-rich" \n(B) 60  NPs  20 \n(C) 60  NPRs  20 \n\n\n# Fig. 4. Categorization of peptidergic cell types in ACC based on expression profiles of NPs and their NPRs\n\n\n\nFig. 4.  NPs  NPRs  ACC \n\n 78  NPs  83  NPRs  ACC  Khrameeva et al. ACC snRNAseq GABAergicglutamatergicnon-neuronal cells\n 8 \n(1) inhibitory interneuronsNP and its NPR[s] expressed in inhibitory neurons, n = 25\n(2) excitatory interneuronsNP and its NPR[s] expressed in excitatory neurons, n = 17\n(3) interneuron candidatesboth NP and its NPR[s] expressed, n = 20\n(4) excitatory projection neuronsNP expressed in excitatory neurons, n = 20\n(5) projection neuron candidatesNP expressed, but not its NPR, n = 8\n(6) non-neuronal cellsNP expressed in non-neuronal cells, n = 14\n(7) afferent input, either neuronal from other brain regions or blood-borne peptidesNPR expressed, but not NP, n = 17\n(8) precursors/NPsno receptor known, n = 10\n\n\n# Fig. 5. Distribution of CCK, VGLUT1, and VGAT mRNA-positive cells in ACC\n\n\n\nFig. 5. CCKVGLUT1VGAT mRNA- ACC \n\n(AE) CCK mRNA  glutamatergic  GABAergic \n(F) CCKVGLUT1VGAT  lamina IV  CCK \n(G-H) CCKVGLUT1VGAT CCK  VGLUT1  VGAT VGLUT1 G1 VGAT H1\n(I-J2) CCKVGLUT1VGAT \n\n\n# Fig. 6. Expression of the GAL system in ACC\n\n\n\nFig. 6. GAL  ACC \n\n(A)  GAL  GALR1  GALR3  20  glutamatergic (VGLUT1) and GABAergic (VGAT)  ISH \n(B) HCRT/ORX  HCRTR2/ORXR2  20 \n(C) GAL GAL  NA (locus coeruleus) GALR1 GALR3 glutamatergic  glutamatergic  GAL \n\n\n# Fig. 7. Peptides from diverse afferent sources\n\n\n\nFig. 7. \n\n hPFC  16  NPs  hPFC  hPFC  volume transmission ',normalizedContent:': \ndoi: https://doi.org/10.1073/pnas.2123146119\ncite: zhong w, barde s, mitsios n, et al. the neuropeptide landscape of human prefrontal cortex. proc natl acad sci u s a. 2022 aug 16;119(33):e2123146119.\n\n\n\n\n\n\n\ntomas hokfelt\n\nkarolinska institutet\ntomas.hokfelt@ki.se\n\n\n# abstract\n\nhpfc hpfc  17 rna  rna-scope  hpfc  60  60 pfc / pfc  anterior cingulate cortex/orbitofrontal gyrus pfc >5-fold pfc //yy hpfc / pfc \n\n\n# results\n\n\n# fig. 1. transcriptomic analysis of the 17 microdissected pfc regions and 3 cortical reference regions\n\n\n\nfig. 1.  17  pfc  3 \n\n(a) hpfc  17  3 (frontal, parietal, and temporal cortex)\n(b) 165  bulk rnaseq \n(c) 4  marker gene  20 gad1 slc17a7 slc17a6  (17-fold) refcorts pvalb  (14-fold)\n\n\n# fig. 2. expression levels of nps and nprs in hpfc\n\n\n\nfig. 2. hpfc  nps  nprs \n\n(a) 17  hpfc nps78  nps  60 ntpm>0.1\n(b) 17  hpfc nprs83  nprs  60 ntpm>0.1\n(c) rnascope admpomc1crhr1np/hcrtoxthcrt/orx2hcrtr2\n\n\n# fig. 3. regional variation of nps and nprs in hpfc\n\n\n\nfig. 3. hpfc  nps  nprs \n\n(a)  z-score  nps  nps  "np-rich" \n(b) 60  nps  20 \n(c) 60  nprs  20 \n\n\n# fig. 4. categorization of peptidergic cell types in acc based on expression profiles of nps and their nprs\n\n\n\nfig. 4.  nps  nprs  acc \n\n 78  nps  83  nprs  acc  khrameeva et al. acc snrnaseq gabaergicglutamatergicnon-neuronal cells\n 8 \n(1) inhibitory interneuronsnp and its npr[s] expressed in inhibitory neurons, n = 25\n(2) excitatory interneuronsnp and its npr[s] expressed in excitatory neurons, n = 17\n(3) interneuron candidatesboth np and its npr[s] expressed, n = 20\n(4) excitatory projection neuronsnp expressed in excitatory neurons, n = 20\n(5) projection neuron candidatesnp expressed, but not its npr, n = 8\n(6) non-neuronal cellsnp expressed in non-neuronal cells, n = 14\n(7) afferent input, either neuronal from other brain regions or blood-borne peptidesnpr expressed, but not np, n = 17\n(8) precursors/npsno receptor known, n = 10\n\n\n# fig. 5. distribution of cck, vglut1, and vgat mrna-positive cells in acc\n\n\n\nfig. 5. cckvglut1vgat mrna- acc \n\n(ae) cck mrna  glutamatergic  gabaergic \n(f) cckvglut1vgat  lamina iv  cck \n(g-h) cckvglut1vgat cck  vglut1  vgat vglut1 g1 vgat h1\n(i-j2) cckvglut1vgat \n\n\n# fig. 6. expression of the gal system in acc\n\n\n\nfig. 6. gal  acc \n\n(a)  gal  galr1  galr3  20  glutamatergic (vglut1) and gabaergic (vgat)  ish \n(b) hcrt/orx  hcrtr2/orxr2  20 \n(c) gal gal  na (locus coeruleus) galr1 galr3 glutamatergic  glutamatergic  gal \n\n\n# fig. 7. peptides from diverse afferent sources\n\n\n\nfig. 7. \n\n hpfc  16  nps  hpfc  hpfc  volume transmission ',charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain",frontmatter:{title:" --- Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain",date:"2023-10-19T20:18:22.000Z",permalink:"/pages/42c71a/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20231016.(2023.10.16%20mPFC%20MERFISH%20Nat%20Neurosci)%20Spatial%20transcriptomics%20reveals%20the%20distinct%20organization%20of%20mouse%20prefrontal%20cortex%20and%20neuronal%20subtypes%20regulating%20chronic%20pain.html",relativePath:"02./11./04./20231016.(2023.10.16 mPFC MERFISH Nat Neurosci) Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain.md",key:"v-6de6612b",path:"/pages/42c71a/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:576},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:930},{level:3,title:"Figure 1. MERFISH reveals the molecularly diverse cell types and subtypes comprising the PFC and adjoining cortices",slug:"figure-1-merfish-reveals-the-molecularly-diverse-cell-types-and-subtypes-comprising-the-pfc-and-adjoining-cortices",normalizedTitle:"figure 1. merfish reveals the molecularly diverse cell types and subtypes comprising the pfc and adjoining cortices",charIndex:942},{level:3,title:"Figure 2. Spatial organization of different neuron subtypes in PFC",slug:"figure-2-spatial-organization-of-different-neuron-subtypes-in-pfc",normalizedTitle:"figure 2. spatial organization of different neuron subtypes in pfc",charIndex:1459},{level:3,title:"Figure 3. Distinct neuron subtypes are selectively enriched or depleted in PFC relative to the adjacent cortical regions",slug:"figure-3-distinct-neuron-subtypes-are-selectively-enriched-or-depleted-in-pfc-relative-to-the-adjacent-cortical-regions",normalizedTitle:"figure 3. distinct neuron subtypes are selectively enriched or depleted in pfc relative to the adjacent cortical regions",charIndex:1924},{level:3,title:"Figure 4. Genes with expression enriched or depleted in PFC",slug:"figure-4-genes-with-expression-enriched-or-depleted-in-pfc",normalizedTitle:"figure 4. genes with expression enriched or depleted in pfc",charIndex:2426},{level:3,title:"Figure 5. Cellcell proximity across all neuronal cell types",slug:"figure-5-cell-cell-proximity-across-all-neuronal-cell-types",normalizedTitle:"figure 5. cellcell proximity across all neuronal cell types",charIndex:3104},{level:3,title:"Figure 6. Spatial and molecular organization of PFC projection to the major subcortical targets",slug:"figure-6-spatial-and-molecular-organization-of-pfc-projection-to-the-major-subcortical-targets",normalizedTitle:"figure 6. spatial and molecular organization of pfc projection to the major subcortical targets",charIndex:3516},{level:3,title:"Figure 7. Chronic pain caused cellular and molecular changes in PFC excitatory neurons",slug:"figure-7-chronic-pain-caused-cellular-and-molecular-changes-in-pfc-excitatory-neurons",normalizedTitle:"figure 7. chronic pain caused cellular and molecular changes in pfc excitatory neurons",charIndex:4179},{level:2,title:"Extended Data",slug:"extended-data",normalizedTitle:"extended data",charIndex:4890},{level:3,title:"Extended Data Fig. 1 The workflow and quality control for MERFISH profiling",slug:"extended-data-fig-1-the-workflow-and-quality-control-for-merfish-profiling",normalizedTitle:"extended data fig. 1 the workflow and quality control for merfish profiling",charIndex:4908},{level:3,title:"Extended Data Fig. 2 MERFISH and scRNA-seq based clusters are consistent",slug:"extended-data-fig-2-merfish-and-scrna-seq-based-clusters-are-consistent",normalizedTitle:"extended data fig. 2 merfish and scrna-seq based clusters are consistent",charIndex:5284},{level:3,title:"Extended Data Fig. 3 The marker gene expression in the different neuronal subtypes",slug:"extended-data-fig-3-the-marker-gene-expression-in-the-different-neuronal-subtypes",normalizedTitle:"extended data fig. 3 the marker gene expression in the different neuronal subtypes",charIndex:5670},{level:3,title:"Extended Data Fig. 4 Single molecule FISH validates expression of inhibitory neuron markers and their overlap with several subtype-specific markers in PFC",slug:"extended-data-fig-4-single-molecule-fish-validates-expression-of-inhibitory-neuron-markers-and-their-overlap-with-several-subtype-specific-markers-in-pfc",normalizedTitle:"extended data fig. 4 single molecule fish validates expression of inhibitory neuron markers and their overlap with several subtype-specific markers in pfc",charIndex:5859},{level:3,title:"Extended Data Fig. 5 Comparative analysis between our dataset and those of published MERFISH dataset",slug:"extended-data-fig-5-comparative-analysis-between-our-dataset-and-those-of-published-merfish-dataset",normalizedTitle:"extended data fig. 5 comparative analysis between our dataset and those of published merfish dataset",charIndex:6186},{level:3,title:"Extended Data Fig. 6 Spatial distribution of molecularly defined excitatory neuron subtypes along the anterior to posterior axis",slug:"extended-data-fig-6-spatial-distribution-of-molecularly-defined-excitatory-neuron-subtypes-along-the-anterior-to-posterior-axis",normalizedTitle:"extended data fig. 6 spatial distribution of molecularly defined excitatory neuron subtypes along the anterior to posterior axis",charIndex:6411},{level:3,title:"Extended Data Fig. 7 Spatial location of all molecularly defined PFC cell types and subtypes",slug:"extended-data-fig-7-spatial-location-of-all-molecularly-defined-pfc-cell-types-and-subtypes",normalizedTitle:"extended data fig. 7 spatial location of all molecularly defined pfc cell types and subtypes",charIndex:6687},{level:3,title:"Extended Data Fig. 8 Specific gene expression signatures of PFC and PFC subregions",slug:"extended-data-fig-8-specific-gene-expression-signatures-of-pfc-and-pfc-subregions",normalizedTitle:"extended data fig. 8 specific gene expression signatures of pfc and pfc subregions",charIndex:6935},{level:3,title:"Extended Data Fig. 9 Integrated MERFISH and scRNA-seq data to predict neuronal projections",slug:"extended-data-fig-9-integrated-merfish-and-scrna-seq-data-to-predict-neuronal-projections",normalizedTitle:"extended data fig. 9 integrated merfish and scrna-seq data to predict neuronal projections",charIndex:7404},{level:3,title:"Extended Data Fig. 10 Pain model: Behavior, MERFISH and smFISH measurements",slug:"extended-data-fig-10-pain-model-behavior-merfish-and-smfish-measurements",normalizedTitle:"extended data fig. 10 pain model: behavior, merfish and smfish measurements",charIndex:7910}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-023-01455-9" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-023-01455-9<OutboundLink/></a><br>\nCite: Bhattacherjee, A., Zhang, C., Watson, B.R. et al. Spatial transcriptomics reveals the distinct\norganization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. <em>Nat Neurosci</em> (2023).</p>\n',headersStr:"Abstract Results Figure 1. MERFISH reveals the molecularly diverse cell types and subtypes comprising the PFC and adjoining cortices Figure 2. Spatial organization of different neuron subtypes in PFC Figure 3. Distinct neuron subtypes are selectively enriched or depleted in PFC relative to the adjacent cortical regions Figure 4. Genes with expression enriched or depleted in PFC Figure 5. Cellcell proximity across all neuronal cell types Figure 6. Spatial and molecular organization of PFC projection to the major subcortical targets Figure 7. Chronic pain caused cellular and molecular changes in PFC excitatory neurons Extended Data Extended Data Fig. 1 The workflow and quality control for MERFISH profiling Extended Data Fig. 2 MERFISH and scRNA-seq based clusters are consistent Extended Data Fig. 3 The marker gene expression in the different neuronal subtypes Extended Data Fig. 4 Single molecule FISH validates expression of inhibitory neuron markers and their overlap with several subtype-specific markers in PFC Extended Data Fig. 5 Comparative analysis between our dataset and those of published MERFISH dataset Extended Data Fig. 6 Spatial distribution of molecularly defined excitatory neuron subtypes along the anterior to posterior axis Extended Data Fig. 7 Spatial location of all molecularly defined PFC cell types and subtypes Extended Data Fig. 8 Specific gene expression signatures of PFC and PFC subregions Extended Data Fig. 9 Integrated MERFISH and scRNA-seq data to predict neuronal projections Extended Data Fig. 10 Pain model: Behavior, MERFISH and smFISH measurements",content:": \nDOI: https://doi.org/10.1038/s41593-023-01455-9\nCite: Bhattacherjee, A., Zhang, C., Watson, B.R. et al. Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. Nat Neurosci (2023).\n\n\n\n\n\n\n\nYI ZHANG()                      JEFFREY R. MOFFITT\n                                  \nHarvard Stem Cell Institute       Harvard Medical School\nyzhang@genetics.med.harvard.edu   jeffrey.moffitt@childrens.harvard.edu\n\n\n# Abstract\n\n (PFC)  PFC PFC  PFC  PFC PFC  PFC  PFC \n\n\n# Results\n\n\n# Figure 1. MERFISH reveals the molecularly diverse cell types and subtypes comprising the PFC and adjoining cortices\n\n\n\nFigure 1. MERFISH  PFC \n\na. MERFISH  UMAP  = 487,224\nb. marker genes \nc. \nd.  PFC \ne.  MERFISH  scRNA-seq scRNA-seq  Allen Brain Atlas  PFC \n\n\n# Figure 2. Spatial organization of different neuron subtypes in PFC\n\n\n\nFigure 2. PFC \n\na.  MERFISH  PFC  PFC Fig.1c \nb.  PFC A to P\nc. L4/5 IT1  L5 ET 1  A  P \nde.  PFC (d)(e)= 121,6171\nf.  5  L2/3 IT 2L5 ET 1L5/6 NP Lamp5 3  Pvalb 3\n\n\n# Figure 3. Distinct neuron subtypes are selectively enriched or depleted in PFC relative to the adjacent cortical regions\n\n\n\nFigure 3.  PFC \n\na.  MERFISH  UMAP  PFC in =121617 out = 285,445 \nb. Barplot  PFC  log2\nc.  PFC  enricheddepleted unbiased PFC\nd. PFC \ne. \nf.  Allen Brain Atlas CCF v3 \n\n\n# Figure 4. Genes with expression enriched or depleted in PFC\n\n\n\nFigure 4.  PFC \n\na.  PFC  PFC  enriched  depleted  DEGsin=121,617out=285,445PFC  enricheddepleted \nb. Nnat Scn4b PFC \nc.  Allen Brain Atlas  ISH  Nnat  Scn4b \nd.  MERFISH  UMAP PFC  10  enriched  10  depleted \ne.  PFC  PFC \nf.  iSpatial PFC  enriched  depleted  20,733 MERFISH iSpatial \ng. PFC  enriched  depleted  gene ontology \nh. PFC \n\n\n# Figure 5. Cellcell proximity across all neuronal cell types\n\n\n\nFigure 5. \n\na. -= 32,811 PFC  PFC  log2  cluster a'  PFC inoutside\nb.  Pvalb 1  L5 IT 3  Pvalb 6  L5 ET 2 \nc.  L2/3 IT 3  L4/5 IT 2  PFC \n\n\n# Figure 6. Spatial and molecular organization of PFC projection to the major subcortical targets\n\n\n\nFigure 6. PFC \n\na. MERFISH clusters  MERFISH  scRNA-seq SVM  MERFISH \nb.  MERFISH9544scRNA-seq4294 UMAP \nc. ROC  6 w/o \nd.  6 \ne. \nf. PFC  PAG  mCherry  AAV  PAGPAG  PFC  smFISHmCherry L5 ET 1 Pou3f1mCherry  Pou3f1 RNA-FISH  Pou3f1  mCherry  mCherry+  Pou3f1+\n\n\n# Figure 7. Chronic pain caused cellular and molecular changes in PFC excitatory neurons\n\n\n\nFigure 7.  PFC \n\na.  MERFISH  control  pain n=3  sham  SNI  7 \nb. Augur AUC  ROC \nc.  control  pain  ARG  PFC \nd.  control  pain  PFC  ARG \ne. control  pain  PFC  ARG  control-pain  ID\nf.  sham(control)  SNIchronic pain cFos  Pou3f1L5 ET marker smFISH \ng.  sham  chronic pain  cFos  Pou3f1 mRNA \nh. Barplot  cFos+  Pou3f1+ \n\n\n# Extended Data\n\n\n# Extended Data Fig. 1 The workflow and quality control for MERFISH profiling\n\n\n\nExtended Data Fig. 1 MERFISH \n\na. MERFISH  PFC  MERFISH \nb.  MERFISH  RNA counts  spearman \nc.  MERFISH  bulk RNAseq  RNA counts counts \nd. MERFISH  Allen ISH\n\n\n# Extended Data Fig. 2 MERFISH and scRNA-seq based clusters are consistent\n\n\n\nExtended Data Fig. 2 MERFISH  scRNA-seq \n\na. UMAP  MERFISH  scRNA-seq GSE124952\nbc. UMAP  scRNA-seq MERFISH\nd-f.  MERFISH  scRNA-seq (d)(e)(f)\ng. scRNA-seq  MERFISH MERFISH = 121617scRNAseq = 24822\n\n\n# Extended Data Fig. 3 The marker gene expression in the different neuronal subtypes\n\n\n\nExtended Data Fig. 3 \n\n(a)= 247098(b)= 51794\n\n\n# Extended Data Fig. 4 Single molecule FISH validates expression of inhibitory neuron markers and their overlap with several subtype-specific markers in PFC\n\n\n\nExtended Data Fig. 4  FISH  PFC \n\nab. smFISH  Sst \ncd. smFISH  Pvalb \ne.  Vip clusters \n\n\n# Extended Data Fig. 5 Comparative analysis between our dataset and those of published MERFISH dataset\n\n\n\nExtended Data Fig. 5  MERFISH \n\na. \nb. \n\n\n# Extended Data Fig. 6 Spatial distribution of molecularly defined excitatory neuron subtypes along the anterior to posterior axis\n\n\n\nExtended Data Fig. 6 \n\na.  Allen  CCF-v3 \nb. \nc. \n\n\n# Extended Data Fig. 7 Spatial location of all molecularly defined PFC cell types and subtypes\n\n\n\nExtended Data Fig. 7  PFC \n\na. \nb. \nc. \nd.  PFC \n\n\n# Extended Data Fig. 8 Specific gene expression signatures of PFC and PFC subregions\n\n\n\nExtended Data Fig. 8 PFC  PFC \n\nab. PFC  enrichedCacna1hCxcl12Cdh13 depletedAbcd2 iSpatial  b Allen  ISH  PFC \nc. IPA PFC  enriched  depleted/ in/out PFC \nd.  PFC  4  PFC  ISH  PFC \n\n\n# Extended Data Fig. 9 Integrated MERFISH and scRNA-seq data to predict neuronal projections\n\n\n\nExtended Data Fig. 9  MERFISH  scRNA-seq \n\na. UMAP  scRNA-seq MERFISH scRNA-seq  MERFISH \nb.  6 \nc.  AAV  mCherry DAPI  mCherry mCherry  Pou3f1L5 ET marker Foxp2L6 CT marker smFISH 3\nd.  mCherry  Foxp2  Pou3f1  smFISH \n\n\n# Extended Data Fig. 10 Pain model: Behavior, MERFISH and smFISH measurements\n\n\n\nExtended Data Fig. 10 MERFISH  smFISH \n\na. Von Frey  6 SNI  6 sham  SNI 6 \nb.  UMAP \nc. \nd. \ne.  ARG \nf.  Fos smFISH Sham\ng. Npas4  Arc (h)  Pou3f1+ ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-023-01455-9\ncite: bhattacherjee, a., zhang, c., watson, b.r. et al. spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain. nat neurosci (2023).\n\n\n\n\n\n\n\nyi zhang()                      jeffrey r. moffitt\n                                  \nharvard stem cell institute       harvard medical school\nyzhang@genetics.med.harvard.edu   jeffrey.moffitt@childrens.harvard.edu\n\n\n# abstract\n\n (pfc)  pfc pfc  pfc  pfc pfc  pfc  pfc \n\n\n# results\n\n\n# figure 1. merfish reveals the molecularly diverse cell types and subtypes comprising the pfc and adjoining cortices\n\n\n\nfigure 1. merfish  pfc \n\na. merfish  umap  = 487,224\nb. marker genes \nc. \nd.  pfc \ne.  merfish  scrna-seq scrna-seq  allen brain atlas  pfc \n\n\n# figure 2. spatial organization of different neuron subtypes in pfc\n\n\n\nfigure 2. pfc \n\na.  merfish  pfc  pfc fig.1c \nb.  pfc a to p\nc. l4/5 it1  l5 et 1  a  p \nde.  pfc (d)(e)= 121,6171\nf.  5  l2/3 it 2l5 et 1l5/6 np lamp5 3  pvalb 3\n\n\n# figure 3. distinct neuron subtypes are selectively enriched or depleted in pfc relative to the adjacent cortical regions\n\n\n\nfigure 3.  pfc \n\na.  merfish  umap  pfc in =121617 out = 285,445 \nb. barplot  pfc  log2\nc.  pfc  enricheddepleted unbiased pfc\nd. pfc \ne. \nf.  allen brain atlas ccf v3 \n\n\n# figure 4. genes with expression enriched or depleted in pfc\n\n\n\nfigure 4.  pfc \n\na.  pfc  pfc  enriched  depleted  degsin=121,617out=285,445pfc  enricheddepleted \nb. nnat scn4b pfc \nc.  allen brain atlas  ish  nnat  scn4b \nd.  merfish  umap pfc  10  enriched  10  depleted \ne.  pfc  pfc \nf.  ispatial pfc  enriched  depleted  20,733 merfish ispatial \ng. pfc  enriched  depleted  gene ontology \nh. pfc \n\n\n# figure 5. cellcell proximity across all neuronal cell types\n\n\n\nfigure 5. \n\na. -= 32,811 pfc  pfc  log2  cluster a'  pfc inoutside\nb.  pvalb 1  l5 it 3  pvalb 6  l5 et 2 \nc.  l2/3 it 3  l4/5 it 2  pfc \n\n\n# figure 6. spatial and molecular organization of pfc projection to the major subcortical targets\n\n\n\nfigure 6. pfc \n\na. merfish clusters  merfish  scrna-seq svm  merfish \nb.  merfish9544scrna-seq4294 umap \nc. roc  6 w/o \nd.  6 \ne. \nf. pfc  pag  mcherry  aav  pagpag  pfc  smfishmcherry l5 et 1 pou3f1mcherry  pou3f1 rna-fish  pou3f1  mcherry  mcherry+  pou3f1+\n\n\n# figure 7. chronic pain caused cellular and molecular changes in pfc excitatory neurons\n\n\n\nfigure 7.  pfc \n\na.  merfish  control  pain n=3  sham  sni  7 \nb. augur auc  roc \nc.  control  pain  arg  pfc \nd.  control  pain  pfc  arg \ne. control  pain  pfc  arg  control-pain  id\nf.  sham(control)  snichronic pain cfos  pou3f1l5 et marker smfish \ng.  sham  chronic pain  cfos  pou3f1 mrna \nh. barplot  cfos+  pou3f1+ \n\n\n# extended data\n\n\n# extended data fig. 1 the workflow and quality control for merfish profiling\n\n\n\nextended data fig. 1 merfish \n\na. merfish  pfc  merfish \nb.  merfish  rna counts  spearman \nc.  merfish  bulk rnaseq  rna counts counts \nd. merfish  allen ish\n\n\n# extended data fig. 2 merfish and scrna-seq based clusters are consistent\n\n\n\nextended data fig. 2 merfish  scrna-seq \n\na. umap  merfish  scrna-seq gse124952\nbc. umap  scrna-seq merfish\nd-f.  merfish  scrna-seq (d)(e)(f)\ng. scrna-seq  merfish merfish = 121617scrnaseq = 24822\n\n\n# extended data fig. 3 the marker gene expression in the different neuronal subtypes\n\n\n\nextended data fig. 3 \n\n(a)= 247098(b)= 51794\n\n\n# extended data fig. 4 single molecule fish validates expression of inhibitory neuron markers and their overlap with several subtype-specific markers in pfc\n\n\n\nextended data fig. 4  fish  pfc \n\nab. smfish  sst \ncd. smfish  pvalb \ne.  vip clusters \n\n\n# extended data fig. 5 comparative analysis between our dataset and those of published merfish dataset\n\n\n\nextended data fig. 5  merfish \n\na. \nb. \n\n\n# extended data fig. 6 spatial distribution of molecularly defined excitatory neuron subtypes along the anterior to posterior axis\n\n\n\nextended data fig. 6 \n\na.  allen  ccf-v3 \nb. \nc. \n\n\n# extended data fig. 7 spatial location of all molecularly defined pfc cell types and subtypes\n\n\n\nextended data fig. 7  pfc \n\na. \nb. \nc. \nd.  pfc \n\n\n# extended data fig. 8 specific gene expression signatures of pfc and pfc subregions\n\n\n\nextended data fig. 8 pfc  pfc \n\nab. pfc  enrichedcacna1hcxcl12cdh13 depletedabcd2 ispatial  b allen  ish  pfc \nc. ipa pfc  enriched  depleted/ in/out pfc \nd.  pfc  4  pfc  ish  pfc \n\n\n# extended data fig. 9 integrated merfish and scrna-seq data to predict neuronal projections\n\n\n\nextended data fig. 9  merfish  scrna-seq \n\na. umap  scrna-seq merfish scrna-seq  merfish \nb.  6 \nc.  aav  mcherry dapi  mcherry mcherry  pou3f1l5 et marker foxp2l6 ct marker smfish 3\nd.  mcherry  foxp2  pou3f1  smfish \n\n\n# extended data fig. 10 pain model: behavior, merfish and smfish measurements\n\n\n\nextended data fig. 10 merfish  smfish \n\na. von frey  6 sni  6 sham  sni 6 \nb.  umap \nc. \nd. \ne.  arg \nf.  fos smfish sham\ng. npas4  arc (h)  pou3f1+ ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- The neuropeptidergic connectome of C. elegans",frontmatter:{title:" --- The neuropeptidergic connectome of C. elegans",date:"2024-02-21T09:23:13.000Z",permalink:"/pages/a1ce18/",categories:[""],tags:["-"],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20231115.(2023.11.15%20%E7%BA%BF%E8%99%AB%E7%A5%9E%E7%BB%8F%E8%82%BD%20Neuron)%20The%20neuropeptidergic%20connectome%20of%20C.%20elegans.html",relativePath:"02./11./04./20231115.(2023.11.15  Neuron) The neuropeptidergic connectome of C. elegans.md",key:"v-51b78e88",path:"/pages/a1ce18/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:326},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:648},{level:3,title:"Fig. 1 Construction of the neuropeptidergic connectome",slug:"fig-1-construction-of-the-neuropeptidergic-connectome",normalizedTitle:"fig. 1 construction of the neuropeptidergic connectome",charIndex:659},{level:3,title:"Fig. 2 Assessment of gene expression thresholds using single-copy knockin reporters",slug:"fig-2-assessment-of-gene-expression-thresholds-using-single-copy-knockin-reporters",normalizedTitle:"fig. 2 assessment of gene expression thresholds using single-copy knockin reporters",charIndex:825},{level:3,title:"Fig. 3 Assessment of the spatial scale of neuropeptide signaling",slug:"fig-3-assessment-of-the-spatial-scale-of-neuropeptide-signaling",normalizedTitle:"fig. 3 assessment of the spatial scale of neuropeptide signaling",charIndex:1185},{level:3,title:"Fig. 4 Individual NPP-GPCR networks exhibit different topologies",slug:"fig-4-individual-npp-gpcr-networks-exhibit-different-topologies",normalizedTitle:"fig. 4 individual npp-gpcr networks exhibit different topologies",charIndex:1732},{level:3,title:"Fig. 5 The aggregate neuropeptide connectome connects all neurons in a dense network",slug:"fig-5-the-aggregate-neuropeptide-connectome-connects-all-neurons-in-a-dense-network",normalizedTitle:"fig. 5 the aggregate neuropeptide connectome connects all neurons in a dense network",charIndex:2289},{level:3,title:"Fig. 6 Analysis of peptidergic network degree highlights hubs and a large rich club",slug:"fig-6-analysis-of-peptidergic-network-degree-highlights-hubs-and-a-large-rich-club",normalizedTitle:"fig. 6 analysis of peptidergic network degree highlights hubs and a large rich club",charIndex:2540},{level:3,title:"Fig. 7 Mesoscale structure of the neuropeptide connectome",slug:"fig-7-mesoscale-structure-of-the-neuropeptide-connectome",normalizedTitle:"fig. 7 mesoscale structure of the neuropeptide connectome",charIndex:3175},{level:3,title:"Fig. 8 Co-expression between GPCRs and their ligands potentiates autocrine and paracrine signaling",slug:"fig-8-co-expression-between-gpcrs-and-their-ligands-potentiates-autocrine-and-paracrine-signaling",normalizedTitle:"fig. 8 co-expression between gpcrs and their ligands potentiates autocrine and paracrine signaling",charIndex:3672},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:4212}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.neuron.2023.09.043" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.neuron.2023.09.043<OutboundLink/></a><br>\nCite: Ripoll-Snchez L, Watteyne J, Sun H, et al. The neuropeptidergic connectome of C. elegans. <em>Neuron</em>. 2023;111(22):3570-3589.e5.</p>\n',headersStr:"Abstract Figure Fig. 1 Construction of the neuropeptidergic connectome Fig. 2 Assessment of gene expression thresholds using single-copy knockin reporters Fig. 3 Assessment of the spatial scale of neuropeptide signaling Fig. 4 Individual NPP-GPCR networks exhibit different topologies Fig. 5 The aggregate neuropeptide connectome connects all neurons in a dense network Fig. 6 Analysis of peptidergic network degree highlights hubs and a large rich club Fig. 7 Mesoscale structure of the neuropeptide connectome Fig. 8 Co-expression between GPCRs and their ligands potentiates autocrine and paracrine signaling Discussion",content:": \nDOI: https://doi.org/10.1016/j.neuron.2023.09.043\nCite: Ripoll-Snchez L, Watteyne J, Sun H, et al. The neuropeptidergic connectome of C. elegans. Neuron. 2023;111(22):3570-3589.e5.\n\n\n\n\n\n\n\nWILLIAM SCHAFER\n\nMRC Laboratory of Molecular Biology\nwschafer@mrc-lmb.cam.ac.uk\n\n\n# Abstract\n\n-\n\n\n# Figure\n\n\n# Fig. 1 Construction of the neuropeptidergic connectome\n\n\n\nFig. 1 \n\n(A) -GPCR\n(B) \n\n\n# Fig. 2 Assessment of gene expression thresholds using single-copy knockin reporters\n\n\n\nFig. 2 \n\nGFP- NeuroPAL  10um\n(A)  NeuroPAL \n(B)  NPP  nlp-45 flp-20\n(C)  GPCR  tkr-1 dmsr-6 17  NPP  9  GPCR  Figure S1 \n\n\n# Fig. 3 Assessment of the spatial scale of neuropeptide signaling\n\n\n\nFig. 3 \n\n(A) \n(B) \n(C) 23  GPCRs  GPCR\n(D) C 23  x  y GPCR  NPP  Figure S2 \n\n\n# Fig. 4 Individual NPP-GPCR networks exhibit different topologies\n\n\n\nFig. 4  NPP-GPCR \n\n(A)  NPP  92  GPCR  NPP Local NPP  GPCR 50Pervasive NPP  GPCR 50broadcaster GPCR 50 NPP 50integrative NPP 50 GPCR 50CAPA-1/NMUR-1 FLP-18/NPR-5 FLP-1/FRPR-7 NLP-47/GNRR-1\n(B)  NPP-GPCR  Figure S4\n\n\n# Fig. 5 The aggregate neuropeptide connectome connects all neurons in a dense network\n\n\n\nFig. 5 \n\n NPP  GPCR 1-18 NPP-GPCR  5 \n\n\n# Fig. 6 Analysis of peptidergic network degree highlights hubs and a large rich club\n\n\n\nFig. 6  hubs  rich club\n\n(A)  nodesedgesdegreehub rich club hub\n(B-E)  degree degree in-degreeout-degree 10  degree  hubs \n(F) Rich club  rich club  F(k)  degree  100  rich club  F_random(k)  rich club  156  166  Figure S7\n(G)  degrees r = 0.54p = 3.1e-14 rich club  hubs \n\n\n# Fig. 7 Mesoscale structure of the neuropeptide connectome\n\n\n\nFig. 7 \n\n(A and B)  t-SNEA PCABEuclidean distance, perplexity 30Hubs  166  rich club  112 \n(C) AB\n(D)  indegree  indegree 139hubs26719854 Kruskal-Wallis  indegree  Tukey-Kramer \n(E) \n\n\n# Fig. 8 Co-expression between GPCRs and their ligands potentiates autocrine and paracrine signaling\n\n\n\nFig. 8 GPCRs \n\n(A) 92  NPP-GPCR  NPP  GPCR  NPP  GPCR \n(B)  NPP-GPCR \n(C)  92  NPP-GPCR \n(D)  degreein-degree out-degree\n(E)  NPP-GPCR  NPP-GPCR \n\n\n# Discussion\n\nRipoll-Sanchez ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.neuron.2023.09.043\ncite: ripoll-sanchez l, watteyne j, sun h, et al. the neuropeptidergic connectome of c. elegans. neuron. 2023;111(22):3570-3589.e5.\n\n\n\n\n\n\n\nwilliam schafer\n\nmrc laboratory of molecular biology\nwschafer@mrc-lmb.cam.ac.uk\n\n\n# abstract\n\n-\n\n\n# figure\n\n\n# fig. 1 construction of the neuropeptidergic connectome\n\n\n\nfig. 1 \n\n(a) -gpcr\n(b) \n\n\n# fig. 2 assessment of gene expression thresholds using single-copy knockin reporters\n\n\n\nfig. 2 \n\ngfp- neuropal  10um\n(a)  neuropal \n(b)  npp  nlp-45 flp-20\n(c)  gpcr  tkr-1 dmsr-6 17  npp  9  gpcr  figure s1 \n\n\n# fig. 3 assessment of the spatial scale of neuropeptide signaling\n\n\n\nfig. 3 \n\n(a) \n(b) \n(c) 23  gpcrs  gpcr\n(d) c 23  x  y gpcr  npp  figure s2 \n\n\n# fig. 4 individual npp-gpcr networks exhibit different topologies\n\n\n\nfig. 4  npp-gpcr \n\n(a)  npp  92  gpcr  npp local npp  gpcr 50pervasive npp  gpcr 50broadcaster gpcr 50 npp 50integrative npp 50 gpcr 50capa-1/nmur-1 flp-18/npr-5 flp-1/frpr-7 nlp-47/gnrr-1\n(b)  npp-gpcr  figure s4\n\n\n# fig. 5 the aggregate neuropeptide connectome connects all neurons in a dense network\n\n\n\nfig. 5 \n\n npp  gpcr 1-18 npp-gpcr  5 \n\n\n# fig. 6 analysis of peptidergic network degree highlights hubs and a large rich club\n\n\n\nfig. 6  hubs  rich club\n\n(a)  nodesedgesdegreehub rich club hub\n(b-e)  degree degree in-degreeout-degree 10  degree  hubs \n(f) rich club  rich club  f(k)  degree  100  rich club  f_random(k)  rich club  156  166  figure s7\n(g)  degrees r = 0.54p = 3.1e-14 rich club  hubs \n\n\n# fig. 7 mesoscale structure of the neuropeptide connectome\n\n\n\nfig. 7 \n\n(a and b)  t-snea pcabeuclidean distance, perplexity 30hubs  166  rich club  112 \n(c) ab\n(d)  indegree  indegree 139hubs26719854 kruskal-wallis  indegree  tukey-kramer \n(e) \n\n\n# fig. 8 co-expression between gpcrs and their ligands potentiates autocrine and paracrine signaling\n\n\n\nfig. 8 gpcrs \n\n(a) 92  npp-gpcr  npp  gpcr  npp  gpcr \n(b)  npp-gpcr \n(c)  92  npp-gpcr \n(d)  degreein-degree out-degree\n(e)  npp-gpcr  npp-gpcr \n\n\n# discussion\n\nripoll-sanchez ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Whole-cortex in situ sequencing reveals input-dependent area identity",frontmatter:{title:" --- Whole-cortex in situ sequencing reveals input-dependent area identity",date:"2024-07-25T19:35:36.000Z",permalink:"/pages/666cd3/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20240424.(2024.04.24%20BARseq%E5%85%A8%E7%9A%AE%E5%B1%82%20Nature)%20Whole-cortex%20in%20situ%20sequencing%20reveals%20input-dependent%20area%20identity.html",relativePath:"02./11./04./20240424.(2024.04.24 BARseq Nature) Whole-cortex in situ sequencing reveals input-dependent area identity.md",key:"v-6f2dd4c9",path:"/pages/666cd3/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:589}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-024-07221-6" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-024-07221-6<OutboundLink/></a><br>\nCite: Chen, X., Fischer, S., Rue, M.C.P. et al. Whole-cortex in situ sequencing reveals input-dependent\narea identity. Nature (2024).</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1038/s41586-024-07221-6\nCite: Chen, X., Fischer, S., Rue, M.C.P. et al. Whole-cortex in situ sequencing reveals input-dependent area identity. Nature (2024).\n\n\n\n\n\n\n\nXIAOYIN CHEN                        ANTHONY M. ZADOR                JESSE GILLIS\n                                                                    \nAllen Institute for Brain Science   Cold Spring Harbor Laboratory   University of Toronto\nxiaoyin.chen@alleninstitute.org     zador@cshl.edu                  jesse.gillis@utoronto.ca\n\n\n# Abstract\n\n BARseq 10.3  104  9  4,194,658  RNA  BARseq  ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-024-07221-6\ncite: chen, x., fischer, s., rue, m.c.p. et al. whole-cortex in situ sequencing reveals input-dependent area identity. nature (2024).\n\n\n\n\n\n\n\nxiaoyin chen                        anthony m. zador                jesse gillis\n                                                                    \nallen institute for brain science   cold spring harbor laboratory   university of toronto\nxiaoyin.chen@alleninstitute.org     zador@cshl.edu                  jesse.gillis@utoronto.ca\n\n\n# abstract\n\n barseq 10.3  104  9  4,194,658  rna  barseq  ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types",frontmatter:{title:" --- A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types",date:"2024-02-01T11:30:42.000Z",permalink:"/pages/12b484/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20231101.(2023.11.01%20%E9%87%91%E9%B1%BC%E7%AB%AF%E8%84%91%E7%A9%BA%E9%97%B4%E5%9B%BE%E8%B0%B1%20Sci.%20Adv.)%20A%20telencephalon%20cell%20type%20atlas%20for%20goldfish%20reveals%20diversity%20in%20the%20evolution%20of%20spatial%20structure%20and%20cell%20types.html",relativePath:"02./11./04./20231101.(2023.11.01  Sci. Adv.) A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types.md",key:"v-2cc66357",path:"/pages/12b484/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:544},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:911},{level:3,title:"Fig. 1. Composition of goldfish telencephalon cells",slug:"fig-1-composition-of-goldfish-telencephalon-cells",normalizedTitle:"fig. 1. composition of goldfish telencephalon cells",charIndex:922},{level:3,title:"Fig. 2. Axial and regional parcellation of goldfish telencephalon",slug:"fig-2-axial-and-regional-parcellation-of-goldfish-telencephalon",normalizedTitle:"fig. 2. axial and regional parcellation of goldfish telencephalon",charIndex:1356},{level:3,title:"Fig. 3. GABAergic neurons in the goldfish telencephalon",slug:"fig-3-gabaergic-neurons-in-the-goldfish-telencephalon",normalizedTitle:"fig. 3. gabaergic neurons in the goldfish telencephalon",charIndex:1881},{level:3,title:"Fig. 4. Glutamatergic neurons in the goldfish telencephalon",slug:"fig-4-glutamatergic-neurons-in-the-goldfish-telencephalon",normalizedTitle:"fig. 4. glutamatergic neurons in the goldfish telencephalon",charIndex:2359},{level:3,title:"Fig. 5. Region-specific marker genes and cell types",slug:"fig-5-region-specific-marker-genes-and-cell-types",normalizedTitle:"fig. 5. region-specific marker genes and cell types",charIndex:2831},{level:3,title:"Fig. 6. Goldfish and mouse cross species comparison of the telencephalon cell types",slug:"fig-6-goldfish-and-mouse-cross-species-comparison-of-the-telencephalon-cell-types",normalizedTitle:"fig. 6. goldfish and mouse cross species comparison of the telencephalon cell types",charIndex:3316},{level:3,title:"Fig. 7. Cross-species comparison of teleostean telencephalon cell types",slug:"fig-7-cross-species-comparison-of-teleostean-telencephalon-cell-types",normalizedTitle:"fig. 7. cross-species comparison of teleostean telencephalon cell types",charIndex:4093},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:4500}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1126/sciadv.adh7693" target="_blank" rel="noopener noreferrer">https://doi.org/10.1126/sciadv.adh7693<OutboundLink/></a><br>\nCite: Muhammad Tibi et al. ,A telencephalon cell type atlas for goldfish reveals diversity in the\nevolution of spatial structure and cell types. <em>Sci. Adv.</em> 9,eadh7693(2023).</p>\n',headersStr:"Abstract Figure Fig. 1. Composition of goldfish telencephalon cells Fig. 2. Axial and regional parcellation of goldfish telencephalon Fig. 3. GABAergic neurons in the goldfish telencephalon Fig. 4. Glutamatergic neurons in the goldfish telencephalon Fig. 5. Region-specific marker genes and cell types Fig. 6. Goldfish and mouse cross species comparison of the telencephalon cell types Fig. 7. Cross-species comparison of teleostean telencephalon cell types Discussion",content:": \nDOI: https://doi.org/10.1126/sciadv.adh7693\nCite: Muhammad Tibi et al. ,A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types. Sci. Adv. 9,eadh7693(2023).\n\n\n\n\n\n\n\nAMIT ZEISEL                               RONEN SEGEV\n                                          \nTechnionIsrael Institute of Technology   Ben-Gurion University of the Negev\namit.zeisel@technion.ac.il                ronensgv@bgu.ac.il\n\n\n# Abstract\n\nRNA4.5\n\n\n# Figure\n\n\n# Fig. 1. Composition of goldfish telencephalon cells\n\n\n\nFig. 1 \n\n(A) \n(B) OTSC\n(C)  t-SNE \n(D)  t-SNE \n(E) \n(F) Top marker gene  cluster \n\n\n# Fig. 2. Axial and regional parcellation of goldfish telencephalon\n\n\n\nFig. 2 \n\n(A) GABAergic  glutamatergic  Visium ST 1 Northcutt DVDcDm VsstSstPpaapdvml\n(B)  lateral-medial dorsal-ventralSD12 top \n(C) 14 \n\n\n# Fig. 3. GABAergic neurons in the goldfish telencephalon\n\n\n\nFig. 3  GABAergic \n\n(A)  GABAergic  t-SNE  marker gene \n(B)  GABA GABA1 to GABA40 top marker gene UMI\n(C)  marker gene [(A)] ST  anterior-posterior \n(D) GABAergic  Visium spots  anterior-posterior  scRNA-seq \n\n\n# Fig. 4. Glutamatergic neurons in the goldfish telencephalon\n\n\n\nFig. 4  Glutamatergic \n\n(A)  Glutamatergic  t-SNE \n(B)  Glutamatergic GLUT1  GLUT48 top marker gene UMI cluster \n(C)  NR2F2  CNR1  t-SNE [(A)] ST \n(D) Glutamatergic  Visium spots - scRNA-seq \n\n\n# Fig. 5. Region-specific marker genes and cell types\n\n\n\nFig. 5 \n\n(A)  ST  top  marker gene\n(B)  (A) -\n(C) \n(D)  ST  HCR-FISH  NEUROD6 HCR 0.5 mmST  HCR \n(E) GLUT20GLUT21GLUT30  GLUT31GLUT5GLUT12GLUT16  GLUT18 Dm-Dl 3D/\n\n\n# Fig. 6. Goldfish and mouse cross species comparison of the telencephalon cell types\n\n\n\nFig. 6 \n\n(A) HarmonyDBSCANKNN\n(B) KNN\n(C)  t-SNE Schwann \n(D) GABA KNN\n(E) SST  ST  ISH  isocortex  Sst Allen Mouse Brain ISH Atlas\n(F) () ST ISH\n(G)  markers  ST ISH \n(H) STISH()\n(I) GLUT KNN \n(J) STISH\n(K)  ST  ISH /\n\n\n# Fig. 7. Cross-species comparison of teleostean telencephalon cell types\n\n\n\nFig. 7 \n\n(A-B) t-SNE (A)  (B) Harmony DBSCAN \n(C)  KNN \n(D) CBLN1PENK  SST\n(E)  HCR-FISH  ST  Visium HCR\n\n\n# Discussion\n\n ST  scRNA-seq",normalizedContent:": \ndoi: https://doi.org/10.1126/sciadv.adh7693\ncite: muhammad tibi et al. ,a telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types. sci. adv. 9,eadh7693(2023).\n\n\n\n\n\n\n\namit zeisel                               ronen segev\n                                          \ntechnionisrael institute of technology   ben-gurion university of the negev\namit.zeisel@technion.ac.il                ronensgv@bgu.ac.il\n\n\n# abstract\n\nrna4.5\n\n\n# figure\n\n\n# fig. 1. composition of goldfish telencephalon cells\n\n\n\nfig. 1 \n\n(a) \n(b) otsc\n(c)  t-sne \n(d)  t-sne \n(e) \n(f) top marker gene  cluster \n\n\n# fig. 2. axial and regional parcellation of goldfish telencephalon\n\n\n\nfig. 2 \n\n(a) gabaergic  glutamatergic  visium st 1 northcutt dvdcdm vsstsstppaapdvml\n(b)  lateral-medial dorsal-ventralsd12 top \n(c) 14 \n\n\n# fig. 3. gabaergic neurons in the goldfish telencephalon\n\n\n\nfig. 3  gabaergic \n\n(a)  gabaergic  t-sne  marker gene \n(b)  gaba gaba1 to gaba40 top marker gene umi\n(c)  marker gene [(a)] st  anterior-posterior \n(d) gabaergic  visium spots  anterior-posterior  scrna-seq \n\n\n# fig. 4. glutamatergic neurons in the goldfish telencephalon\n\n\n\nfig. 4  glutamatergic \n\n(a)  glutamatergic  t-sne \n(b)  glutamatergic glut1  glut48 top marker gene umi cluster \n(c)  nr2f2  cnr1  t-sne [(a)] st \n(d) glutamatergic  visium spots - scrna-seq \n\n\n# fig. 5. region-specific marker genes and cell types\n\n\n\nfig. 5 \n\n(a)  st  top  marker gene\n(b)  (a) -\n(c) \n(d)  st  hcr-fish  neurod6 hcr 0.5 mmst  hcr \n(e) glut20glut21glut30  glut31glut5glut12glut16  glut18 dm-dl 3d/\n\n\n# fig. 6. goldfish and mouse cross species comparison of the telencephalon cell types\n\n\n\nfig. 6 \n\n(a) harmonydbscanknn\n(b) knn\n(c)  t-sne schwann \n(d) gaba knn\n(e) sst  st  ish  isocortex  sst allen mouse brain ish atlas\n(f) () st ish\n(g)  markers  st ish \n(h) stish()\n(i) glut knn \n(j) stish\n(k)  st  ish /\n\n\n# fig. 7. cross-species comparison of teleostean telencephalon cell types\n\n\n\nfig. 7 \n\n(a-b) t-sne (a)  (b) harmony dbscan \n(c)  knn \n(d) cbln1penk  sst\n(e)  hcr-fish  st  visium hcr\n\n\n# discussion\n\n st  scrna-seq",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma",frontmatter:{title:" --- Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma",date:"2023-03-27T20:07:26.000Z",permalink:"/pages/3eedee/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/12.%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/20220220.(2022.02.20%20CRD%E8%82%BF%E7%98%A4%20J%20Pineal%20Res)Single%E2%80%90cell%20transcriptomic%20analysis%20reveals%20circadianrhythm%20disruption%20associated%20with%20poor%20prognosisand%20drug%E2%80%90resistance%20in%20lung%20adenocarcinoma.html",relativePath:"02./12./20220220.(2022.02.20 CRD J Pineal Res)Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma.md",key:"v-c424057a",path:"/pages/3eedee/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:504},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1028},{level:3,title:"Figure 1. CRD heterogeneity is found across different sample origins in malignant cells",slug:"figure-1-crd-heterogeneity-is-found-across-different-sample-origins-in-malignant-cells",normalizedTitle:"figure 1. crd heterogeneity is found across different sample origins in malignant cells",charIndex:1040},{level:3,title:"Figure 2. CRD is associated with the response of TKI therapy",slug:"figure-2-crd-is-associated-with-the-response-of-tki-therapy",normalizedTitle:"figure 2. crd is associated with the response of tki therapy",charIndex:1799},{level:3,title:"Figure 3. Cellcell ligand-receptor (LRs) and cytokine-related pathway network analysis",slug:"figure-3-cell-cell-ligand-receptor-lrs-and-cytokine-related-pathway-network-analysis",normalizedTitle:"figure 3. cellcell ligand-receptor (lrs) and cytokine-related pathway network analysis",charIndex:2940},{level:3,title:"Figure 4. Validation of 358 gene expression signatures for CRD status in bulk transcriptomic datasets",slug:"figure-4-validation-of-358-gene-expression-signatures-for-crd-status-in-bulk-transcriptomic-datasets",normalizedTitle:"figure 4. validation of 358 gene expression signatures for crd status in bulk transcriptomic datasets",charIndex:3668},{level:3,title:"Figure 5. Clinical characteristics of CRD groups (high and low) in multiple independent cohorts",slug:"figure-5-clinical-characteristics-of-crd-groups-high-and-low-in-multiple-independent-cohorts",normalizedTitle:"figure 5. clinical characteristics of crd groups (high and low) in multiple independent cohorts",charIndex:4351},{level:3,title:"Figure 6. CRGm contributed to predicting the response to immunotherapy",slug:"figure-6-crgm-contributed-to-predicting-the-response-to-immunotherapy",normalizedTitle:"figure 6. crgm contributed to predicting the response to immunotherapy",charIndex:4960},{level:3,title:"Figure 7. Validated experiments related to CRD",slug:"figure-7-validated-experiments-related-to-crd",normalizedTitle:"figure 7. validated experiments related to crd",charIndex:5617}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1111/jpi.12803" target="_blank" rel="noopener noreferrer">https://doi.org/10.1111/jpi.12803<OutboundLink/></a><br>\nCite: He L, Fan Y, Zhang Y, Tu T, Zhang Q, Yuan F, Cheng C. Single-cell transcriptomic analysis reveals circadian rhythm disruption associated with poor prognosis and drug-resistance in lung adenocarcinoma. J Pineal Res. 2022 Aug;73(1):e12803.</p>\n',headersStr:"Abstract Results Figure 1. CRD heterogeneity is found across different sample origins in malignant cells Figure 2. CRD is associated with the response of TKI therapy Figure 3. Cellcell ligand-receptor (LRs) and cytokine-related pathway network analysis Figure 4. Validation of 358 gene expression signatures for CRD status in bulk transcriptomic datasets Figure 5. Clinical characteristics of CRD groups (high and low) in multiple independent cohorts Figure 6. CRGm contributed to predicting the response to immunotherapy Figure 7. Validated experiments related to CRD",content:": \nDOI: https://doi.org/10.1111/jpi.12803\nCite: He L, Fan Y, Zhang Y, Tu T, Zhang Q, Yuan F, Cheng C. Single-cell transcriptomic analysis reveals circadian rhythm disruption associated with poor prognosis and drug-resistance in lung adenocarcinoma. J Pineal Res. 2022 Aug;73(1):e12803.\n\n\n\n\n\n\n\nCHAO CHENG                 FAHU YUAN\n                     \ncnchao_cheng@hust.edu.cn   yuanfh@jhun.edu.cn\n\n\n# Abstract\n\nCRDCRDRCRDscoreRNACRDscore-CRDT22CoxCRDscore2314RNAH1299CRDscoreSNAILTWISTCRDCRDT\n\n\n# Results\n\n\n# Figure 1. CRD heterogeneity is found across different sample origins in malignant cells\n\n\n\nFigure 1.  CRD \n\n(A)  GSE131907  GSE123904  30 lung adenocarcinoma, LUAD Figure 1B \n(B) n=17n=10n=3total cells=96204 UMAP\n(C) \n(D)  2091  WGCNA  2091 \n(E)  358 \n(F)  CRDscore 11  clusters  CRDhigh  cluster \n(G) CRDhigh CRDlow\n(H) CRD  z-score \n\n\n# Figure 2. CRD is associated with the response of TKI therapy\n\n\n\nFigure 2. CRD  TKI \n\n(A)  22  TKI  (total cells=21604)  UMAP\n(B)  Epithelial scores TKI  CRDscore \n(C)  tSNE  TKI TKI TNRDPD\n(D) 4258  TKI  CRD \n(E)  CRDscore \n(F) CRDhigh  CRDlow \n(G)  CRDscore  CRDhigh  CRDlow  CRDhigh  CRDlow \n(H)  CRDhigh NES, normalized enrichment score\n(I)  PD RD CRDhigh  CRDlow  GSEA  log2 fold changes  positive log2 fold changes  negative log2 fold changes \n(J-K)  CRDscore  30 /J  K CRDscore J  Kx  marginal boxplots  CRDscore  y CRD, circadian rhythm disruption; GSEA, Gene Set Enrichment Analysis; SD, standard deviation; TKI, tyrosine kinase inhibitor\n\n\n# Figure 3. Cellcell ligand-receptor (LRs) and cytokine-related pathway network analysis\n\n\n\nFigure 3. -- (LRs) \n\n(A-C)  T  LR (chemokines) (A)(costimulatory factors) (B) (coinhibitory factors) (C)\n(DF)  T \n(G-H)  CRDhigh (G)  CRDlow  (H) / T -\n(I) T CRDhigh malignant cells(red bars in upper panel)  LUAD samples (cyan bars, bottom panel)  36  T cell markers (purple bars, right panel)  macrophage markers (yellow bars, left panel)  (r > .2 & p < .01) \n(J)  malignant cells with CRD (high vs. low)T cellsmacrophages \n\n\n# Figure 4. Validation of 358 gene expression signatures for CRD status in bulk transcriptomic datasets\n\n\n\nFigure 4.  bulk  CRD  358 \n\n(A)  358  CRG  CRDscore  z-score  GEO  CRD  control \n(B) CRDscore  CRDscore  CRDscore  GSVA score \n(C)  TCGA-LUAD  CRDhigh  CRDlow \n(D) Circos  TCGA-LUAD  CRDscore  TIL \n(E) CRD \n(F) TCGA-LUAD  CRDhigh  CRDlow \n\n\n# Figure 5. Clinical characteristics of CRD groups (high and low) in multiple independent cohorts\n\n\n\nFigure 5.  CRD \n\n(A-C)  smoothHR  TCGA-LUAD (A)GSE31210 (B)  GSE72094 (C)  CRDscore \n(DF)  CRDhigh  CRDlow  KaplanMeier \n(G)  CRDhigh  CRDlow 95%  Cox \n(H)  22  TCGA  7577  CRDscore CRDscore \n(I)  CRDscore  CRDhigh  CRDlow \n(J)  TCGA  (n = 17)  CRDscore \n(K)  (n = 22)  Cox CRD \n\n\n# Figure 6. CRGm contributed to predicting the response to immunotherapy\n\n\n\nFigure 6. CRGm \n\n(A-B)  GSE42127 (A)  GSE14814 (B)  CRDscore  OS Kaplan-Meier \n(C) TKI  CRDhigh  CRDlow  Kaplan-Meier \n(D) PD/NR  CRDscore  BRCA  RD \n(E)  TCGA-LUAD  CRDscore  KaplanMeier \n(F) CRG  LASSO  SVM-RFE  LASSO  lambda  SVM-REF \n(G)  PD/NR  RD  CRGmGEPscore  PDL1 \n(H) CRGm  ROC \n(I)  CRGm  14 \n(J)  CRGm \n\n\n# Figure 7. Validated experiments related to CRD\n\n\n\nFigure 7. CRD \n\n(A) clockper1cry1  mRNA  BEAS-2BA549H1299  PCR \n(B)  siRNA  CLOCK  24 \n(C) PCA siCLOCK  PERMANOVA PCoA pc1  pc2 \n(D) Venn siCLOCK  H1299  DEGs\n(E)  GO  DEGs 100\n(F)  SNAILlog2(TPM+1) CRDscore \n(G)  CRDscoret",normalizedContent:": \ndoi: https://doi.org/10.1111/jpi.12803\ncite: he l, fan y, zhang y, tu t, zhang q, yuan f, cheng c. single-cell transcriptomic analysis reveals circadian rhythm disruption associated with poor prognosis and drug-resistance in lung adenocarcinoma. j pineal res. 2022 aug;73(1):e12803.\n\n\n\n\n\n\n\nchao cheng                 fahu yuan\n                     \ncnchao_cheng@hust.edu.cn   yuanfh@jhun.edu.cn\n\n\n# abstract\n\ncrdcrdrcrdscorernacrdscore-crdt22coxcrdscore2314rnah1299crdscoresnailtwistcrdcrdt\n\n\n# results\n\n\n# figure 1. crd heterogeneity is found across different sample origins in malignant cells\n\n\n\nfigure 1.  crd \n\n(a)  gse131907  gse123904  30 lung adenocarcinoma, luad figure 1b \n(b) n=17n=10n=3total cells=96204 umap\n(c) \n(d)  2091  wgcna  2091 \n(e)  358 \n(f)  crdscore 11  clusters  crdhigh  cluster \n(g) crdhigh crdlow\n(h) crd  z-score \n\n\n# figure 2. crd is associated with the response of tki therapy\n\n\n\nfigure 2. crd  tki \n\n(a)  22  tki  (total cells=21604)  umap\n(b)  epithelial scores tki  crdscore \n(c)  tsne  tki tki tnrdpd\n(d) 4258  tki  crd \n(e)  crdscore \n(f) crdhigh  crdlow \n(g)  crdscore  crdhigh  crdlow  crdhigh  crdlow \n(h)  crdhigh nes, normalized enrichment score\n(i)  pd rd crdhigh  crdlow  gsea  log2 fold changes  positive log2 fold changes  negative log2 fold changes \n(j-k)  crdscore  30 /j  k crdscore j  kx  marginal boxplots  crdscore  y crd, circadian rhythm disruption; gsea, gene set enrichment analysis; sd, standard deviation; tki, tyrosine kinase inhibitor\n\n\n# figure 3. cellcell ligand-receptor (lrs) and cytokine-related pathway network analysis\n\n\n\nfigure 3. -- (lrs) \n\n(a-c)  t  lr (chemokines) (a)(costimulatory factors) (b) (coinhibitory factors) (c)\n(df)  t \n(g-h)  crdhigh (g)  crdlow  (h) / t -\n(i) t crdhigh malignant cells(red bars in upper panel)  luad samples (cyan bars, bottom panel)  36  t cell markers (purple bars, right panel)  macrophage markers (yellow bars, left panel)  (r > .2 & p < .01) \n(j)  malignant cells with crd (high vs. low)t cellsmacrophages \n\n\n# figure 4. validation of 358 gene expression signatures for crd status in bulk transcriptomic datasets\n\n\n\nfigure 4.  bulk  crd  358 \n\n(a)  358  crg  crdscore  z-score  geo  crd  control \n(b) crdscore  crdscore  crdscore  gsva score \n(c)  tcga-luad  crdhigh  crdlow \n(d) circos  tcga-luad  crdscore  til \n(e) crd \n(f) tcga-luad  crdhigh  crdlow \n\n\n# figure 5. clinical characteristics of crd groups (high and low) in multiple independent cohorts\n\n\n\nfigure 5.  crd \n\n(a-c)  smoothhr  tcga-luad (a)gse31210 (b)  gse72094 (c)  crdscore \n(df)  crdhigh  crdlow  kaplanmeier \n(g)  crdhigh  crdlow 95%  cox \n(h)  22  tcga  7577  crdscore crdscore \n(i)  crdscore  crdhigh  crdlow \n(j)  tcga  (n = 17)  crdscore \n(k)  (n = 22)  cox crd \n\n\n# figure 6. crgm contributed to predicting the response to immunotherapy\n\n\n\nfigure 6. crgm \n\n(a-b)  gse42127 (a)  gse14814 (b)  crdscore  os kaplan-meier \n(c) tki  crdhigh  crdlow  kaplan-meier \n(d) pd/nr  crdscore  brca  rd \n(e)  tcga-luad  crdscore  kaplanmeier \n(f) crg  lasso  svm-rfe  lasso  lambda  svm-ref \n(g)  pd/nr  rd  crgmgepscore  pdl1 \n(h) crgm  roc \n(i)  crgm  14 \n(j)  crgm \n\n\n# figure 7. validated experiments related to crd\n\n\n\nfigure 7. crd \n\n(a) clockper1cry1  mrna  beas-2ba549h1299  pcr \n(b)  sirna  clock  24 \n(c) pca siclock  permanova pcoa pc1  pc2 \n(d) venn siclock  h1299  degs\n(e)  go  degs 100\n(f)  snaillog2(tpm+1) crdscore \n(g)  crdscoret",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:" --- A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain",frontmatter:{title:" --- A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain",date:"2024-02-19T19:38:41.000Z",permalink:"/pages/895303/",categories:[""],tags:["-"],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6/04.%E5%85%B6%E4%BB%96/20231213.(2023.12.13%20Allen%E5%85%A8%E8%84%91%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84%20bioRxiv)%20A%20high-resolution%20transcriptomic%20and%20spatial%20atlas%20of%20cell%20types%20in%20the%20whole%20mouse%20brain.html",relativePath:"02./11./04./20231213.(2023.12.13 Allen bioRxiv) A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain.md",key:"v-6086e0c3",path:"/pages/895303/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:391},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:1049},{level:3,title:"Fig. 1 Transcriptomic cell type taxonomy of the whole mouse brain",slug:"fig-1-transcriptomic-cell-type-taxonomy-of-the-whole-mouse-brain",normalizedTitle:"fig. 1 transcriptomic cell type taxonomy of the whole mouse brain",charIndex:1060},{level:3,title:"Fig. 2 Neuronal cell type classification and distribution across the brain",slug:"fig-2-neuronal-cell-type-classification-and-distribution-across-the-brain",normalizedTitle:"fig. 2 neuronal cell type classification and distribution across the brain",charIndex:1749},{level:3,title:"Fig. 3 Neurotransmitter types and their distribution throughout the mouse brain",slug:"fig-3-neurotransmitter-types-and-their-distribution-throughout-the-mouse-brain",normalizedTitle:"fig. 3 neurotransmitter types and their distribution throughout the mouse brain",charIndex:1980},{level:3,title:"Fig. 4 Non-neuronal cell types and immature neuronal types",slug:"fig-4-non-neuronal-cell-types-and-immature-neuronal-types",normalizedTitle:"fig. 4 non-neuronal cell types and immature neuronal types",charIndex:2580},{level:3,title:"Fig. 5 Transcription factor modules across the whole mouse brain",slug:"fig-5-transcription-factor-modules-across-the-whole-mouse-brain",normalizedTitle:"fig. 5 transcription factor modules across the whole mouse brain",charIndex:3337},{level:3,title:"Fig. 6 Region specific features and transitional cell types",slug:"fig-6-region-specific-features-and-transitional-cell-types",normalizedTitle:"fig. 6 region specific features and transitional cell types",charIndex:4037}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41586-023-06812-z" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41586-023-06812-z<OutboundLink/></a><br>\nCite: Yao, Z., van Velthoven, C.T.J., Kunst, M. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. <em>Nature</em> 624, 317332 (2023).</p>\n',headersStr:"Abstract Figure Fig. 1 Transcriptomic cell type taxonomy of the whole mouse brain Fig. 2 Neuronal cell type classification and distribution across the brain Fig. 3 Neurotransmitter types and their distribution throughout the mouse brain Fig. 4 Non-neuronal cell types and immature neuronal types Fig. 5 Transcription factor modules across the whole mouse brain Fig. 6 Region specific features and transitional cell types",content:": \nDOI: https://doi.org/10.1038/s41586-023-06812-z\nCite: Yao, Z., van Velthoven, C.T.J., Kunst, M. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. Nature 624, 317332 (2023).\n\n\n\n\n\n\n\nHONGKUI ZENG()\n\nAllen Institute for Brain Science\nhongkuiz@alleninstitute.org\n\n\n# Abstract\n\n700RNAscRNA-seq430 MERFISH 7323061,0455,200\n\n\n# Figure\n\n\n# Fig. 1 Transcriptomic cell type taxonomy of the whole mouse brain\n\n\n\nFig. 1 \n\n(a)  306  subclasses 10xv2: n = 1,708,450 10v3 n = 2,349,599  divisions classNT typeBroad region clusters RNA-seqMERFISH subclasses  subclasses subclasses  subclasses 15  nearest neighbors subclass  subclass  subclass  nearest neighbors  subclass log P value  subclass  odds < 0.2  logPval <-10 \n(b-e)  UMAP  divisionbclasscsubclassdbrain regione\n\n\n# Fig. 2 Neuronal cell type classification and distribution across the brain\n\n\n\nFig. 2 \n\nPallium glut (a,f)Subpallium GABA (b,g)PAL-sAMY-HY (c,h)TH-EPI (d,i) MB-HB-CB (e,j)  UMAP  MERFISH  subclass \n\n\n# Fig. 3 Neurotransmitter types and their distribution throughout the mouse brain\n\n\n\nFig. 3 \n\n(a-c) UMAP  subclasses glutamate-GABA  clustersUMAPs  subclass (a), neurotransmitter type (b), and cluster (c) \n(d) UMAPs  glutamate, GABA, glycine \n(e-g) UMAP  subclasses clusters glutamate and/or GABA UMAPs  subclass (e), neurotransmitter type (f), and cluster (g) \n(h)  glutamate, GABA, modulatory neurotransmitters  UMAPs\n(i-j)  MERFISH  glutamate, GABA, modulatory neurotransmitters \n\n\n# Fig. 4 Non-neuronal cell types and immature neuronal types\n\n\n\nFig. 4 \n\n(a)  marker gene  subclasses  subclasses \n(b) UMAP  subclass astrocytes (c), ependymal cells (d), VLMC (e)\n(c-e) astrocytes (c), ependymal cells (d), VLMC (e)  UMAP  MERFISH  cluster  subclasses\n(f) VLMC cluster 5181Tanycyte cluster 5133  MERFISH \n(g) VLMC cluster 5180CHOR cluster 5142Ependymal clusters 5137 and 5138 \n(h) VLMCsInterlaminar astrocytes(ILA) \n(i) UMAP supertype  dentate gyrus (DG) (j)inner main olfactory bulb (k)outer main olfactory bulb (l) \n(j-l)  MERFISH  supertypes \n\n\n# Fig. 5 Transcription factor modules across the whole mouse brain\n\n\n\nFig. 5 \n\n(a)  TFs divisions (pink), classes (apple green), subclasses (sea green), within subclasses (dark blue)\n(b)  8,108  marker genes (pink) 499  marker genes (sea green) 499  marker genes (dark blue)  cluster (top panel)  subclass (bottom panel) \n(c)  499  TF markers  subclasses  subclasses  Jaccard \n(d)  cluster  TFs (logCPM)  Y  TFclusters  X \n(e)  subclass mod divisions classes\n\n\n# Fig. 6 Region specific features and transitional cell types\n\n\n\nFig. 6 \n\n(a)  clusters  cluster \n(b)  logCPM > 3  cluster  cluster  Homeobox TFs  cluster  TFs  cluster \n(c)  cluster  DEGs  0.10.2...0.9  0.1  0.9  DEGs \n(d)  cluster  MERFISH XY Z clusters \n(e-g)  subclasses  UMAP  MERFISH (e,g)  subclasses (f)  (g)  CCF ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41586-023-06812-z\ncite: yao, z., van velthoven, c.t.j., kunst, m. et al. a high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. nature 624, 317332 (2023).\n\n\n\n\n\n\n\nhongkui zeng()\n\nallen institute for brain science\nhongkuiz@alleninstitute.org\n\n\n# abstract\n\n700rnascrna-seq430 merfish 7323061,0455,200\n\n\n# figure\n\n\n# fig. 1 transcriptomic cell type taxonomy of the whole mouse brain\n\n\n\nfig. 1 \n\n(a)  306  subclasses 10xv2: n = 1,708,450 10v3 n = 2,349,599  divisions classnt typebroad region clusters rna-seqmerfish subclasses  subclasses subclasses  subclasses 15  nearest neighbors subclass  subclass  subclass  nearest neighbors  subclass log p value  subclass  odds < 0.2  logpval <-10 \n(b-e)  umap  divisionbclasscsubclassdbrain regione\n\n\n# fig. 2 neuronal cell type classification and distribution across the brain\n\n\n\nfig. 2 \n\npallium glut (a,f)subpallium gaba (b,g)pal-samy-hy (c,h)th-epi (d,i) mb-hb-cb (e,j)  umap  merfish  subclass \n\n\n# fig. 3 neurotransmitter types and their distribution throughout the mouse brain\n\n\n\nfig. 3 \n\n(a-c) umap  subclasses glutamate-gaba  clustersumaps  subclass (a), neurotransmitter type (b), and cluster (c) \n(d) umaps  glutamate, gaba, glycine \n(e-g) umap  subclasses clusters glutamate and/or gaba umaps  subclass (e), neurotransmitter type (f), and cluster (g) \n(h)  glutamate, gaba, modulatory neurotransmitters  umaps\n(i-j)  merfish  glutamate, gaba, modulatory neurotransmitters \n\n\n# fig. 4 non-neuronal cell types and immature neuronal types\n\n\n\nfig. 4 \n\n(a)  marker gene  subclasses  subclasses \n(b) umap  subclass astrocytes (c), ependymal cells (d), vlmc (e)\n(c-e) astrocytes (c), ependymal cells (d), vlmc (e)  umap  merfish  cluster  subclasses\n(f) vlmc cluster 5181tanycyte cluster 5133  merfish \n(g) vlmc cluster 5180chor cluster 5142ependymal clusters 5137 and 5138 \n(h) vlmcsinterlaminar astrocytes(ila) \n(i) umap supertype  dentate gyrus (dg) (j)inner main olfactory bulb (k)outer main olfactory bulb (l) \n(j-l)  merfish  supertypes \n\n\n# fig. 5 transcription factor modules across the whole mouse brain\n\n\n\nfig. 5 \n\n(a)  tfs divisions (pink), classes (apple green), subclasses (sea green), within subclasses (dark blue)\n(b)  8,108  marker genes (pink) 499  marker genes (sea green) 499  marker genes (dark blue)  cluster (top panel)  subclass (bottom panel) \n(c)  499  tf markers  subclasses  subclasses  jaccard \n(d)  cluster  tfs (logcpm)  y  tfclusters  x \n(e)  subclass mod divisions classes\n\n\n# fig. 6 region specific features and transitional cell types\n\n\n\nfig. 6 \n\n(a)  clusters  cluster \n(b)  logcpm > 3  cluster  cluster  homeobox tfs  cluster  tfs  cluster \n(c)  cluster  degs  0.10.2...0.9  0.1  0.9  degs \n(d)  cluster  merfish xy z clusters \n(e-g)  subclasses  umap  merfish (e,g)  subclasses (f)  (g)  ccf ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules",frontmatter:{title:" --- Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules",date:"2024-07-25T19:52:02.000Z",permalink:"/pages/c40bc0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/12.%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/20230127.(2023.01.27%20%CF%80-FISH%20rainbow%20Nat%20Commun)%20Highly%20efficient%20and%20robust%20%CF%80-FISH%20rainbow%20for%20multiplexed%20in%20situ%20detection%20of%20diverse%20biomolecules.html",relativePath:"02./12./20230127.(2023.01.27 -FISH rainbow Nat Commun) Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules.md",key:"v-3c7ee81a",path:"/pages/c40bc0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:458}],excerpt:'<p>:  -FISH rainbow <br>\nDOI: <a href="https://doi.org/10.1038/s41467-023-36137-4" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-023-36137-4<OutboundLink/></a><br>\nCite: Tao, Y., Zhou, X., Sun, L. et al. Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules. Nat Commun 14, 443 (2023).</p>\n',headersStr:"Abstract",content:":  -FISH rainbow \nDOI: https://doi.org/10.1038/s41467-023-36137-4\nCite: Tao, Y., Zhou, X., Sun, L. et al. Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules. Nat Commun 14, 443 (2023).\n\n\n\n -FISH rainbow \n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# Abstract\n\nFISH RNAs  -FISH rainbow DNARNA 21  -FISH rainbow  -FISH+  microRNA  ARV7 ",normalizedContent:":  -fish rainbow \ndoi: https://doi.org/10.1038/s41467-023-36137-4\ncite: tao, y., zhou, x., sun, l. et al. highly efficient and robust -fish rainbow for multiplexed in situ detection of diverse biomolecules. nat commun 14, 443 (2023).\n\n\n\n -fish rainbow \n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# abstract\n\nfish rnas  -fish rainbow dnarna 21  -fish rainbow  -fish+  microrna  arv7 ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Systematic comparison of sequencing-based spatial transcriptomic methods",frontmatter:{title:" --- Systematic comparison of sequencing-based spatial transcriptomic methods",date:"2024-07-25T19:52:15.000Z",permalink:"/pages/b55e03/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/12.%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/20240704.(2024.07.04%20%E7%94%B0%E9%B2%81%E4%BA%A6%20Nat%20Methods)%20Systematic%20comparison%20of%20sequencing-based%20spatial%20transcriptomic%20methods.html",relativePath:"02./12./20240704.(2024.07.04  Nat Methods) Systematic comparison of sequencing-based spatial transcriptomic methods.md",key:"v-72b734c5",path:"/pages/b55e03/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:383}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41592-024-02325-3" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-024-02325-3<OutboundLink/></a><br>\nCite: You, Y., Fu, Y., Li, L. et al. Systematic comparison of sequencing-based spatial transcriptomic methods. Nat Methods (2024).</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1038/s41592-024-02325-3\nCite: You, Y., Fu, Y., Li, L. et al. Systematic comparison of sequencing-based spatial transcriptomic methods. Nat Methods (2024).\n\n\n\n\n\n\n\n                     \n                        \n                   \ntian_luyi@gzlab.ac.cn   liuxiaodong@westlake.edu.cn\n\n\n# Abstract\n\nsST 11  sST  markers  sST ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41592-024-02325-3\ncite: you, y., fu, y., li, l. et al. systematic comparison of sequencing-based spatial transcriptomic methods. nat methods (2024).\n\n\n\n\n\n\n\n                     \n                        \n                   \ntian_luyi@gzlab.ac.cn   liuxiaodong@westlake.edu.cn\n\n\n# abstract\n\nsst 11  sst  markers  sst ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing",frontmatter:{title:" --- Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing",date:"2024-07-25T19:52:09.000Z",permalink:"/pages/e6566f/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/12.%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/20240514.(2024.05.14%20MiP-seq%20Nat.%20Biomed.%20Eng)%20Spatial%20multi-omics%20at%20subcellular%20resolution%20via%20high-throughput%20in%20situ%20pairwise%20sequencing.html",relativePath:"02./12./20240514.(2024.05.14 MiP-seq Nat. Biomed. Eng) Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing.md",key:"v-6ed06999",path:"/pages/e6566f/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:423}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41551-024-01205-7" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41551-024-01205-7<OutboundLink/></a><br>\nCite: Wu, X., Xu, W., Deng, L. et al. Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing. Nat. Biomed. Eng (2024).</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1038/s41551-024-01205-7\nCite: Wu, X., Xu, W., Deng, L. et al. Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing. Nat. Biomed. Eng (2024).\n\n\n\n\n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# Abstract\n\n (MiP-seq)  DNAsRNAsMiP-seq  RNA MiP-seq ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41551-024-01205-7\ncite: wu, x., xu, w., deng, l. et al. spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing. nat. biomed. eng (2024).\n\n\n\n\n\n\n\n                      \n                        \n                  \ngcao@mail.hzau.edu.cn   jxdai@mail.hzau.edu.cn\n\n\n# abstract\n\n (mip-seq)  dnasrnasmip-seq  rna mip-seq ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture",frontmatter:{title:" --- Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture",date:"2022-12-08T20:37:04.000Z",permalink:"/pages/76fbba/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/13.%E4%B8%89%E7%BB%B4%E5%9F%BA%E5%9B%A0%E7%BB%84/20180426.(2018.04.26%20DLO%20Hi-C%20Nat%20Genet)%20Digestion-ligation-only%20Hi-C%20is%20an%20efficient%20and%20cost-effective%20method%20for%20chromosome%20conformation%20capture.html",relativePath:"02./13./20180426.(2018.04.26 DLO Hi-C Nat Genet) Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture.md",key:"v-2607359c",path:"/pages/76fbba/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:441},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:821},{level:3,title:"Fig1. Flowchart of the digestion-ligation-only Hi-C (DLO Hi-C) method",slug:"fig1-flowchart-of-the-digestion-ligation-only-hi-c-dlo-hi-c-method",normalizedTitle:"fig1. flowchart of the digestion-ligation-only hi-c (dlo hi-c) method",charIndex:833},{level:3,title:"Fig2. The early quality-control step in the DLO Hi-C method",slug:"fig2-the-early-quality-control-step-in-the-dlo-hi-c-method",normalizedTitle:"fig2. the early quality-control step in the dlo hi-c method",charIndex:1197},{level:3,title:"Fig3. DLO Hi-C data analysis workflow and comparison of DLO Hi-C, in situ Hi-C, in situ DLO Hi-C, Hi-C, and DNase Hi-C for 3D genome structure analysis of K562 cells",slug:"fig3-dlo-hi-c-data-analysis-workflow-and-comparison-of-dlo-hi-c-in-situ-hi-c-in-situ-dlo-hi-c-hi-c-and-dnase-hi-c-for-3d-genome-structure-analysis-of-k562-cells",normalizedTitle:"fig3. dlo hi-c data analysis workflow and comparison of dlo hi-c, in situ hi-c, in situ dlo hi-c, hi-c, and dnase hi-c for 3d genome structure analysis of k562 cells",charIndex:1649},{level:3,title:"Fig4. Consistency of A/B compartments, TADs, and chromatin loops between DLO Hi-C, in situ DLO Hi-C, and in situ Hi-C",slug:"fig4-consistency-of-a-b-compartments-tads-and-chromatin-loops-between-dlo-hi-c-in-situ-dlo-hi-c-and-in-situ-hi-c",normalizedTitle:"fig4. consistency of a/b compartments, tads, and chromatin loops between dlo hi-c, in situ dlo hi-c, and in situ hi-c",charIndex:2218},{level:3,title:"Fig5. Detection of chromosomal translocations in THP-1 cells",slug:"fig5-detection-of-chromosomal-translocations-in-thp-1-cells",normalizedTitle:"fig5. detection of chromosomal translocations in thp-1 cells",charIndex:2895}],excerpt:'<p>: Digestion-ligation-only Hi-C <br>\nDOI: <a href="https://doi.org/10.1038/s41588-018-0111-2" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41588-018-0111-2<OutboundLink/></a><br>\nCite: Lin, D., Hong, P., Zhang, S. et al. Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture. <em>Nat Genet</em> 50, 754763 (2018).</p>\n',headersStr:"Abstract Results Fig1. Flowchart of the digestion-ligation-only Hi-C (DLO Hi-C) method Fig2. The early quality-control step in the DLO Hi-C method Fig3. DLO Hi-C data analysis workflow and comparison of DLO Hi-C, in situ Hi-C, in situ DLO Hi-C, Hi-C, and DNase Hi-C for 3D genome structure analysis of K562 cells Fig4. Consistency of A/B compartments, TADs, and chromatin loops between DLO Hi-C, in situ DLO Hi-C, and in situ Hi-C Fig5. Detection of chromosomal translocations in THP-1 cells",content:": Digestion-ligation-only Hi-C \nDOI: https://doi.org/10.1038/s41588-018-0111-2\nCite: Lin, D., Hong, P., Zhang, S. et al. Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture. Nat Genet 50, 754763 (2018).\n\n\n\nGANG CAO()            GUOLIANG LI()\n                        \n                  \ngcao@mail.hzau.edu.cn   guoliang.li@mail.hzau.edu.cn\n\n\n# Abstract\n\n (3C)  3D  3D  Hi-C (digestion-ligation-only Hi-CDLO Hi-C)  3D  DNA  DNA DLO Hi-C  DLO Hi-C  THP-1  K562 \n\n\n# Results\n\n\n# Fig1. Flowchart of the digestion-ligation-only Hi-C (DLO Hi-C) method\n\n\n\nFig1. DLO Hi-C \n\na. EGSstep 1step 220bpstep 3step 4DNAMmeIstep 580-bp DLO Hi-C DNADLO Hi-C DNAIllumina step 6\nb. \n\n\n# Fig2. The early quality-control step in the DLO Hi-C method\n\n\n\nFig2. DLO Hi-C \n\na. linker A  linker BHindIIIClaI\nb. ClaIHindIIIDLO Hi-CDLO Hi-C\nc. ClaIHindIII200-bp DLO Hi-CDNAn=6 tPImageJv1.8.0ROI, region of interest\n\n\n# Fig3. DLO Hi-C data analysis workflow and comparison of DLO Hi-C, in situ Hi-C, in situ DLO Hi-C, Hi-C, and DNase Hi-C for 3D genome structure analysis of K562 cells\n\n\n\nFig3. DLO Hi-C  DLO Hi-Cin situ Hi-Cin situ DLO Hi-CHi-C  DNase Hi-C  K562  3D \n\na. DLO Hi-Clinker\nb. Hi-C\nc. DLO Hi-Cdownsampled in situ DLO Hi-Cdownsampled in situ Hi-CHi-CDNase Hi-CK562(1Mb for the entire genome, 100kb for the individual chromosome, and 10kb for the selected regions)\n\n\n# Fig4. Consistency of A/B compartments, TADs, and chromatin loops between DLO Hi-C, in situ DLO Hi-C, and in situ Hi-C\n\n\n\nFig4. DLO Hi-Cin situ DLO Hi-C  in situ Hi-C  A/B compartmentsTAD  loops \n\na-c. DLO Hi-C253 million raw readsin situ DLO Hi-C632 million raw reads)in situ Hi-C(1.37 billion raw readsA/B compartments (a)TAD boundaries (b)loops (c)overlap\nd. in situ DLO Hi-Cin situ Hi-C5-kblooploops\ne-g. DLO Hi-C253 million raw reads(e)in situ DLO Hi-C (632 million raw reads) (f)in situ Hi-C (1.37 billion raw reads) (g) 5-kbAPA\nh. in situ Hi-C peak listin situ DLO Hi-C 5-kbK562APA\n\n\n# Fig5. Detection of chromosomal translocations in THP-1 cells\n\n\n\nFig5.  THP-1 ",normalizedContent:": digestion-ligation-only hi-c \ndoi: https://doi.org/10.1038/s41588-018-0111-2\ncite: lin, d., hong, p., zhang, s. et al. digestion-ligation-only hi-c is an efficient and cost-effective method for chromosome conformation capture. nat genet 50, 754763 (2018).\n\n\n\ngang cao()            guoliang li()\n                        \n                  \ngcao@mail.hzau.edu.cn   guoliang.li@mail.hzau.edu.cn\n\n\n# abstract\n\n (3c)  3d  3d  hi-c (digestion-ligation-only hi-cdlo hi-c)  3d  dna  dna dlo hi-c  dlo hi-c  thp-1  k562 \n\n\n# results\n\n\n# fig1. flowchart of the digestion-ligation-only hi-c (dlo hi-c) method\n\n\n\nfig1. dlo hi-c \n\na. egsstep 1step 220bpstep 3step 4dnammeistep 580-bp dlo hi-c dnadlo hi-c dnaillumina step 6\nb. \n\n\n# fig2. the early quality-control step in the dlo hi-c method\n\n\n\nfig2. dlo hi-c \n\na. linker a  linker bhindiiiclai\nb. claihindiiidlo hi-cdlo hi-c\nc. claihindiii200-bp dlo hi-cdnan=6 tpimagejv1.8.0roi, region of interest\n\n\n# fig3. dlo hi-c data analysis workflow and comparison of dlo hi-c, in situ hi-c, in situ dlo hi-c, hi-c, and dnase hi-c for 3d genome structure analysis of k562 cells\n\n\n\nfig3. dlo hi-c  dlo hi-cin situ hi-cin situ dlo hi-chi-c  dnase hi-c  k562  3d \n\na. dlo hi-clinker\nb. hi-c\nc. dlo hi-cdownsampled in situ dlo hi-cdownsampled in situ hi-chi-cdnase hi-ck562(1mb for the entire genome, 100kb for the individual chromosome, and 10kb for the selected regions)\n\n\n# fig4. consistency of a/b compartments, tads, and chromatin loops between dlo hi-c, in situ dlo hi-c, and in situ hi-c\n\n\n\nfig4. dlo hi-cin situ dlo hi-c  in situ hi-c  a/b compartmentstad  loops \n\na-c. dlo hi-c253 million raw readsin situ dlo hi-c632 million raw reads)in situ hi-c(1.37 billion raw readsa/b compartments (a)tad boundaries (b)loops (c)overlap\nd. in situ dlo hi-cin situ hi-c5-kblooploops\ne-g. dlo hi-c253 million raw reads(e)in situ dlo hi-c (632 million raw reads) (f)in situ hi-c (1.37 billion raw reads) (g) 5-kbapa\nh. in situ hi-c peak listin situ dlo hi-c 5-kbk562apa\n\n\n# fig5. detection of chromosomal translocations in thp-1 cells\n\n\n\nfig5.  thp-1 ",charsets:{cjk:!0},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" --- WGCNAan R package for weighted correlation network analysis",frontmatter:{title:" --- WGCNAan R package for weighted correlation network analysis",date:"2022-12-11T21:08:43.000Z",permalink:"/pages/ffdcc0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20081228.(2008.12.28%20WGCNA%20BMC%20Bioinformatics)%20WGCNA%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis.html",relativePath:"02./14./20081228.(2008.12.28 WGCNA BMC Bioinformatics) WGCNA an R package for weighted correlation network analysis.md",key:"v-49747540",path:"/pages/ffdcc0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:728},{level:2,title:"WGCNA ",slug:"wgcna-",normalizedTitle:"wgcna ",charIndex:1381},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1396},{level:3,title:"Scale-free Network",slug:"-scale-free-network",normalizedTitle:"scale-free network",charIndex:1829},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:2318},{level:4,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:2333},{level:4,title:"2. ",slug:"_2-",normalizedTitle:"2. ",charIndex:2486},{level:4,title:"3. TOM",slug:"_3-tom",normalizedTitle:"3. tom",charIndex:3170},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:3748},{level:2,title:"Figure",slug:"figure",normalizedTitle:"figure",charIndex:3910},{level:3,title:"Fig1. Overview of WGCNA methodology",slug:"fig1-overview-of-wgcna-methodology",normalizedTitle:"fig1. overview of wgcna methodology",charIndex:3921},{level:3,title:"Fig2. Network visualization plots",slug:"fig2-network-visualization-plots",normalizedTitle:"fig2. network visualization plots",charIndex:4065},{level:3,title:"Fig3. Module and eigengene network plots",slug:"fig3-module-and-eigengene-network-plots",normalizedTitle:"fig3. module and eigengene network plots",charIndex:4246},{level:3,title:"Fig4. WGCNA analysis of liver expression data in female mice",slug:"fig4-wgcna-analysis-of-liver-expression-data-in-female-mice",normalizedTitle:"fig4. wgcna analysis of liver expression data in female mice",charIndex:4515}],excerpt:'<p>: WGCNA R <br>\nDOI: <a href="https://doi.org/10.1186/1471-2105-9-559" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/1471-2105-9-559<OutboundLink/></a><br>\nCite: Langfelder P, Horvath S. WGCNA: an R package for weighted correlation network analysis. <em>BMC Bioinformatics</em>. 2008 Dec 29;9:559.</p>\n',headersStr:"Abstract WGCNA   Scale-free Network  1.  2.  3. TOM  Figure Fig1. Overview of WGCNA methodology Fig2. Network visualization plots Fig3. Module and eigengene network plots Fig4. WGCNA analysis of liver expression data in female mice",content:": WGCNA R \nDOI: https://doi.org/10.1186/1471-2105-9-559\nCite: Langfelder P, Horvath S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics. 2008 Dec 29;9:559.\n\n\n\nWGCNA R \n\n\n\n 1.  --- WGCNAan R package for weighted correlation network analysis\n 2. WGCNAan R package for weighted correlation network analysis\n 3.  --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n 4.  --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdWGCNAhigh dimensional WGCNA\n\n\n\nSTEVE HORVATH\n\nUniversity of California Los Angeles(UCLA)\nshorvath@mednet.ucla.edu\n\n\n# Abstract\n\nBackground: WGCNA\n\nResults: WGCNA RRRR\n\nConclusion: WGCNARR https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/index.html\n\n\n# WGCNA \n\n\n# \n\n\n\n\n\naij={1,ifsij0,otherwisea_{ij} = \\begin{cases} 1, &if s_{ij} \\geqslant \\tau \\\\ 0, &otherwise \\end{cases}a ij ={ 1, 0, ifs ij  otherwise   (Hard Threshold)\n\n\n\n\n\n 1  0 \n\n gene  0.79 0.8 gene \n\n\n\n\n\n\n\n\n\n\n# Scale-free Network\n\n\n\n\n\n\n\n gene  gene  hub gene:\n\n * : hubhub\n * : hub\n\n\n\n(power law): P(k)P(k)P(k)~kk^{-\\gamma}k \nkkk:P(k)P(k)P(k): k\\gamma: \n\n\n# \n\n# 1. \n\n Sij=cor(xi,xj)S_{ij}=|cor(x_i,x_j)|S ij =cor(x i ,x j )\nxix_ix i xjx_jx j \n:\n\n\n\n\n\n# 2. \n\n aij=Sija_{ij}=S_{ij}^\\betaa ij =S ij \n\\beta\nlog\nP(k)P(k)P(k)~kk^{-\\gamma}k   log(P(k))=log(k)log(P(k))=-\\gamma log(k)log(P(k))=log(k)\nP(k)P(k)P(k): kki=i=1,ijnaijk_i=\\sum_{i=1, i \\neq j}^n a_{ij}k i = i=1,ij n a ij\n \\beta log(k)log(k)log(k)  log(P(k))log(P(k))log(P(k))  \\beta  R2R^2R 2  R2R^2R 2  \\beta R2R^2R 2 0.8\n WGCNA  R2R^2R 2  0.9 R2R^2R 2  0.9 6\n\n\n\n\n\n\n SSS  =6\\beta=6=6  aij=Sija_{ij}=S_{ij}^\\betaa ij =S ij  \n\n\n\n\n\n# 3. TOM\n\nTOM TOMij=ui,j(aiuauj)aijmin{uiaiu,ujauj}+1aijTOM_{ij} = \\frac{\\sum_{u \\neq i,j}(a_{iu}*a_{uj})-a_{ij}}{min\\lbrace \\sum_{u \\neq i}a_{iu},\\sum_{u \\neq j}a_{uj} \\rbrace +1 -a_{ij}}TOM ij = min{ ui a iu , uj a uj }+1a ij  ui,j (a iu a uj )a ij\n\ngeneABABABABAB\n\nu i  j  u  i  j  i  j  u \n\n\n\nTOMgene\n\n\n# \n\nModule eigengene(ME)\nGS(Gene Signicance)\nMM(Module Membership)\n\n\n# Figure\n\n\n# Fig1. Overview of WGCNA methodology\n\n\n\nFig1. WGCNA \n\n\n\n 1. \n 2. \n 3. \n 4. \n 5. \n\n\n# Fig2. Network visualization plots\n\n\n\nFig2. \n\nA. log(k)  log(P(k)) \nB.  multi-dimensional scaling plot \nC. \n\n\n# Fig3. Module and eigengene network plots\n\n\n\nFig3. \n\nA.  gene significance(GS)\nB. module membership(MM)  gene significance(GS)  geneMM  GS \nC. module eigengenes(ME) \nD.  module eigengenes(ME) \n\n\n# Fig4. WGCNA analysis of liver expression data in female mice\n\n\n\nFig4.  WGCNA \n\nA.  average linkage hierarchical clustering  Dynamic Tree Cut \nB. TOM  TOM \nC. module eigengenes(ME) \nD.  module eigengenes(ME) \nE. module membership(MM)  gene significance(GS) GS  MM  hub genes \nF.  30  TOM  0.08 \n\n\n\nhttps://www.jianshu.com/p/dcae5fb86559",normalizedContent:": wgcna r \ndoi: https://doi.org/10.1186/1471-2105-9-559\ncite: langfelder p, horvath s. wgcna: an r package for weighted correlation network analysis. bmc bioinformatics. 2008 dec 29;9:559.\n\n\n\nwgcna r \n\n\n\n 1.  --- wgcnaan r package for weighted correlation network analysis\n 2. wgcnaan r package for weighted correlation network analysis\n 3.  --- single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n 4.  --- hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdwgcnahigh dimensional wgcna\n\n\n\nsteve horvath\n\nuniversity of california los angeles(ucla)\nshorvath@mednet.ucla.edu\n\n\n# abstract\n\nbackground: wgcna\n\nresults: wgcna rrrr\n\nconclusion: wgcnarr https://horvath.genetics.ucla.edu/html/coexpressionnetwork/rpackages/wgcna/index.html\n\n\n# wgcna \n\n\n# \n\n\n\n\n\naij={1,ifsij0,otherwisea_{ij} = \\begin{cases} 1, &if s_{ij} \\geqslant \\tau \\\\ 0, &otherwise \\end{cases}a ij ={ 1, 0, ifs ij  otherwise   (hard threshold)\n\n\n\n\n\n 1  0 \n\n gene  0.79 0.8 gene \n\n\n\n\n\n\n\n\n\n\n# scale-free network\n\n\n\n\n\n\n\n gene  gene  hub gene:\n\n * : hubhub\n * : hub\n\n\n\n(power law): p(k)p(k)p(k)~kk^{-\\gamma}k \nkkk:p(k)p(k)p(k): k\\gamma: \n\n\n# \n\n# 1. \n\n sij=cor(xi,xj)s_{ij}=|cor(x_i,x_j)|s ij =cor(x i ,x j )\nxix_ix i xjx_jx j \n:\n\n\n\n\n\n# 2. \n\n aij=sija_{ij}=s_{ij}^\\betaa ij =s ij \n\\beta\nlog\np(k)p(k)p(k)~kk^{-\\gamma}k   log(p(k))=log(k)log(p(k))=-\\gamma log(k)log(p(k))=log(k)\np(k)p(k)p(k): kki=i=1,i=jnaijk_i=\\sum_{i=1, i \\neq j}^n a_{ij}k i = i=1,i=j n a ij\n \\beta log(k)log(k)log(k)  log(p(k))log(p(k))log(p(k))  \\beta  r2r^2r 2  r2r^2r 2  \\beta r2r^2r 2 0.8\n wgcna  r2r^2r 2  0.9 r2r^2r 2  0.9 6\n\n\n\n\n\n\n sss  =6\\beta=6=6  aij=sija_{ij}=s_{ij}^\\betaa ij =s ij  \n\n\n\n\n\n# 3. tom\n\ntom tomij=u=i,j(aiuauj)aijmin{u=iaiu,u=jauj}+1aijtom_{ij} = \\frac{\\sum_{u \\neq i,j}(a_{iu}*a_{uj})-a_{ij}}{min\\lbrace \\sum_{u \\neq i}a_{iu},\\sum_{u \\neq j}a_{uj} \\rbrace +1 -a_{ij}}tom ij = min{ u=i a iu , u=j a uj }+1a ij  u=i,j (a iu a uj )a ij\n\ngeneababababab\n\nu i  j  u  i  j  i  j  u \n\n\n\ntomgene\n\n\n# \n\nmodule eigengene(me)\ngs(gene signicance)\nmm(module membership)\n\n\n# figure\n\n\n# fig1. overview of wgcna methodology\n\n\n\nfig1. wgcna \n\n\n\n 1. \n 2. \n 3. \n 4. \n 5. \n\n\n# fig2. network visualization plots\n\n\n\nfig2. \n\na. log(k)  log(p(k)) \nb.  multi-dimensional scaling plot \nc. \n\n\n# fig3. module and eigengene network plots\n\n\n\nfig3. \n\na.  gene significance(gs)\nb. module membership(mm)  gene significance(gs)  genemm  gs \nc. module eigengenes(me) \nd.  module eigengenes(me) \n\n\n# fig4. wgcna analysis of liver expression data in female mice\n\n\n\nfig4.  wgcna \n\na.  average linkage hierarchical clustering  dynamic tree cut \nb. tom  tom \nc. module eigengenes(me) \nd.  module eigengenes(me) \ne. module membership(mm)  gene significance(gs) gs  mm  hub genes \nf.  30  tom  0.08 \n\n\n\nhttps://www.jianshu.com/p/dcae5fb86559",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:" --- Spatial reconstruction of single-cell gene expression data",frontmatter:{title:" --- Spatial reconstruction of single-cell gene expression data",date:"2023-08-11T15:49:54.000Z",permalink:"/pages/0d1fce/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20150413.(2015.04.13%20Seurat%20V1%20Nat%20Biotechnol)%20Spatial%20reconstruction%20of%20single-cell%20gene%20expression%20data.html",relativePath:"02./14./20150413.(2015.04.13 Seurat V1 Nat Biotechnol) Spatial reconstruction of single-cell gene expression data.md",key:"v-35f11cad",path:"/pages/0d1fce/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:1047},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1426},{level:3,title:"Figure 1. Overview",slug:"figure-1-overview",normalizedTitle:"figure 1. overview",charIndex:1438},{level:3,title:"Figure 2. Single-cell RNA-seq from zebrafish embryos",slug:"figure-2-single-cell-rna-seq-from-zebrafish-embryos",normalizedTitle:"figure 2. single-cell rna-seq from zebrafish embryos",charIndex:2219},{level:3,title:"Figure 3. Seurat correctly infers the spatial position of cells",slug:"figure-3-seurat-correctly-infers-the-spatial-position-of-cells",normalizedTitle:"figure 3. seurat correctly infers the spatial position of cells",charIndex:2647},{level:3,title:"Figure 4. Nine archetypal patterns discovered through spatial clustering",slug:"figure-4-nine-archetypal-patterns-discovered-through-spatial-clustering",normalizedTitle:"figure 4. nine archetypal patterns discovered through spatial clustering",charIndex:3308},{level:3,title:"Figure 5. Seurat spatially characterizes rare cell populations",slug:"figure-5-seurat-spatially-characterizes-rare-cell-populations",normalizedTitle:"figure 5. seurat spatially characterizes rare cell populations",charIndex:3732}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/nbt.3192" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/nbt.3192<OutboundLink/></a><br>\nCite: Satija, R., Farrell, J., Gennert, D. et al. Spatial reconstruction of single-cell gene expression data. Nat Biotechnol 33, 495502 (2015).</p>\n',headersStr:"Abstract Results Figure 1. Overview Figure 2. Single-cell RNA-seq from zebrafish embryos Figure 3. Seurat correctly infers the spatial position of cells Figure 4. Nine archetypal patterns discovered through spatial clustering Figure 5. Seurat spatially characterizes rare cell populations",content:": \nDOI: https://doi.org/10.1038/nbt.3192\nCite: Satija, R., Farrell, J., Gennert, D. et al. Spatial reconstruction of single-cell gene expression data. Nat Biotechnol 33, 495502 (2015).\n\n\n\n(Seurat V1) \n\n\n\n 1.  --- (Seurat V1) Spatial reconstruction of single-cell gene expression data\n 2.  --- (Seurat V2) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (Seurat V3) Comprehensive Integration of Single-Cell Data\n 4.  --- (Seurat V4) Integrated analysis of multimodal single-cell data\n 5.  --- (Seurat V5) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nRAHUL SATIJA           AVIV REGEV                           ALEXANDER F SCHIER\n                                                            \nOxford University      Broad Institute of MIT and Harvard   Broad Institute of MIT and Harvard\nrsatija@nygenome.org   aviv.regev.sc@gmail.com              schier@fas.harvard.edu\n\n\n# Abstract\n\nRNA  RNA-seq  Seurat single-cell RNA-seq  in situ RNA  Seurat  (Danio rerio)  851  Seurat Seurat Seurat \n\n\n# Results\n\n\n# Figure 1. Overview\n\n\n\nFigure 1. \n\n Seurat (i)(ii) landmark genes 'bins' bin landmark genes  'on'  'off'Seurat  landmark genes  bins\n\nSeurat (i)  single-cell RNA-seq impute landmark genes  single-cell RNA-seq (ii)  landmark genes  RNA-seq (iii)  bin landmark genes  (iv)  cell-bin pair  single-cell RNA-seq  Seurat\n\n\n# Figure 2. Single-cell RNA-seq from zebrafish embryos\n\n\n\nFigure 2.  single-cell RNA-seq\n\n(a)  Seurat50% \n(b)  47 i.e., landmark genes 64  binslateralanimal bin  landmark  'on'  'off'\n(c) single-cell RNA-seq protocol  851 \n\n\n# Figure 3. Seurat correctly infers the spatial position of cells\n\n\n\nFigure 3. Seurat \n\n(a) Seurat \n(b)  animal cap  7 \n(c)  'reference' \n(d) Seurat  'reference'  bin  100% \n(e)  'reference' \n(f)  landmark gene Seurat top ROC bottom\n(g-h)  landmark  ROC median ROC = 0.96 (g)  landmark  ROC  (h)\n\n\n# Figure 4. Nine archetypal patterns discovered through spatial clustering\n\n\n\nFigure 4.  9 \n\n(a)  Seurat  290  9  'archetypes'RM, restricted to margin; VM, ventral margin; DEM, dorsally enriched margin; DRM, dorsally restricted margin; EM, extended margin; V, ventral; DA, dorsal animal; VA, ventral animal; A, animal\n(b)  Seurat \n\n\n# Figure 5. Seurat spatially characterizes rare cell populations\n\n\n\nFigure 5. Seurat \n\n(a) \n(b)  PCA sox32cxcr4agscripply1marginendoPCP\n(c) Seurat \n(d) Seurat  ripply1 \n(e) gsc ripply1 ripply1 \n(f)  PCA  PC4 \n(g) Seurat  'apoptotic-like' \n(h)  isg15  mat2al  'apoptotic-like' \n(i) 'apoptotic-like' \n(j) aplnrb isg15Seurat  aplnrb  isg15",normalizedContent:": \ndoi: https://doi.org/10.1038/nbt.3192\ncite: satija, r., farrell, j., gennert, d. et al. spatial reconstruction of single-cell gene expression data. nat biotechnol 33, 495502 (2015).\n\n\n\n(seurat v1) \n\n\n\n 1.  --- (seurat v1) spatial reconstruction of single-cell gene expression data\n 2.  --- (seurat v2) integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (seurat v3) comprehensive integration of single-cell data\n 4.  --- (seurat v4) integrated analysis of multimodal single-cell data\n 5.  --- (seurat v5) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nrahul satija           aviv regev                           alexander f schier\n                                                            \noxford university      broad institute of mit and harvard   broad institute of mit and harvard\nrsatija@nygenome.org   aviv.regev.sc@gmail.com              schier@fas.harvard.edu\n\n\n# abstract\n\nrna  rna-seq  seurat single-cell rna-seq  in situ rna  seurat  (danio rerio)  851  seurat seurat seurat \n\n\n# results\n\n\n# figure 1. overview\n\n\n\nfigure 1. \n\n seurat (i)(ii) landmark genes 'bins' bin landmark genes  'on'  'off'seurat  landmark genes  bins\n\nseurat (i)  single-cell rna-seq impute landmark genes  single-cell rna-seq (ii)  landmark genes  rna-seq (iii)  bin landmark genes  (iv)  cell-bin pair  single-cell rna-seq  seurat\n\n\n# figure 2. single-cell rna-seq from zebrafish embryos\n\n\n\nfigure 2.  single-cell rna-seq\n\n(a)  seurat50% \n(b)  47 i.e., landmark genes 64  binslateralanimal bin  landmark  'on'  'off'\n(c) single-cell rna-seq protocol  851 \n\n\n# figure 3. seurat correctly infers the spatial position of cells\n\n\n\nfigure 3. seurat \n\n(a) seurat \n(b)  animal cap  7 \n(c)  'reference' \n(d) seurat  'reference'  bin  100% \n(e)  'reference' \n(f)  landmark gene seurat top roc bottom\n(g-h)  landmark  roc median roc = 0.96 (g)  landmark  roc  (h)\n\n\n# figure 4. nine archetypal patterns discovered through spatial clustering\n\n\n\nfigure 4.  9 \n\n(a)  seurat  290  9  'archetypes'rm, restricted to margin; vm, ventral margin; dem, dorsally enriched margin; drm, dorsally restricted margin; em, extended margin; v, ventral; da, dorsal animal; va, ventral animal; a, animal\n(b)  seurat \n\n\n# figure 5. seurat spatially characterizes rare cell populations\n\n\n\nfigure 5. seurat \n\n(a) \n(b)  pca sox32cxcr4agscripply1marginendopcp\n(c) seurat \n(d) seurat  ripply1 \n(e) gsc ripply1 ripply1 \n(f)  pca  pc4 \n(g) seurat  'apoptotic-like' \n(h)  isg15  mat2al  'apoptotic-like' \n(i) 'apoptotic-like' \n(j) aplnrb isg15seurat  aplnrb  isg15",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- SCANPYlarge-scale single-cell gene expression data analysis",frontmatter:{title:" --- SCANPYlarge-scale single-cell gene expression data analysis",date:"2023-09-28T20:19:03.000Z",permalink:"/pages/ebb632/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20180206.(2018.02.06%20scanpy%20Genome%20Biol)%20SCANPY%20large-scale%20single-cell%20gene%20expression%20data%20analysis.html",relativePath:"02./14./20180206.(2018.02.06 scanpy Genome Biol) SCANPY large-scale single-cell gene expression data analysis.md",key:"v-26ae0bd4",path:"/pages/ebb632/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:338},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:582},{level:3,title:"Fig. 1",slug:"fig-1",normalizedTitle:"fig. 1",charIndex:594}],excerpt:'<p>: SCANPY<br>\nDOI: <a href="https://doi.org/10.1186/s13059-017-1382-0" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13059-017-1382-0<OutboundLink/></a><br>\nCite: Wolf, F., Angerer, P. &amp; Theis, F. SCANPY: large-scale single-cell gene expression data analysis. Genome Biol 19, 15 (2018).</p>\n',headersStr:"Abstract Results Fig. 1",content:": SCANPY\nDOI: https://doi.org/10.1186/s13059-017-1382-0\nCite: Wolf, F., Angerer, P. & Theis, F. SCANPY: large-scale single-cell gene expression data analysis. Genome Biol 19, 15 (2018).\n\n\n\nSCANPY\n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# Abstract\n\nSCANPY  Python  100  (https://github.com/theislab/Scanpy) SCANPY  ANNDATA (https://github.com/theislab/anndata)\n\n\n# Results\n\n\n# Fig. 1\n\n\n\nFig. 1\n\na. SCANPY  68,579  PBMCs tSNE  graph-drawing (FruchtermanReingold)  bulk  Louvain  cluster 7  B  MS4A1 marker gene bulk  CD34+ cluster  root  tSNE  diffusion maps \nb. Scanpy  Cell Ranger R \nc.  130  E18  10x Genomics ",normalizedContent:": scanpy\ndoi: https://doi.org/10.1186/s13059-017-1382-0\ncite: wolf, f., angerer, p. & theis, f. scanpy: large-scale single-cell gene expression data analysis. genome biol 19, 15 (2018).\n\n\n\nscanpy\n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# abstract\n\nscanpy  python  100  (https://github.com/theislab/scanpy) scanpy  anndata (https://github.com/theislab/anndata)\n\n\n# results\n\n\n# fig. 1\n\n\n\nfig. 1\n\na. scanpy  68,579  pbmcs tsne  graph-drawing (fruchtermanreingold)  bulk  louvain  cluster 7  b  ms4a1 marker gene bulk  cd34+ cluster  root  tsne  diffusion maps \nb. scanpy  cell ranger r \nc.  130  e18  10x genomics ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Recovering Gene Interactions from Single-Cell Data Using Data Diffusion",frontmatter:{title:" --- Recovering Gene Interactions from Single-Cell Data Using Data Diffusion",date:"2022-12-12T19:04:32.000Z",permalink:"/pages/06c5a0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20180628.(2018.06.28%20MAGIC%20Cell)%20Recovering%20Gene%20Interactions%20from%20Single-Cell%20Data%20Using%20Data%20Diffusion.html",relativePath:"02./14./20180628.(2018.06.28 MAGIC Cell) Recovering Gene Interactions from Single-Cell Data Using Data Diffusion.md",key:"v-9dcdff76",path:"/pages/06c5a0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:309},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:656},{level:3,title:"Fig1. Steps of the MAGIC Algorithm",slug:"fig1-steps-of-the-magic-algorithm",normalizedTitle:"fig1. steps of the magic algorithm",charIndex:668},{level:3,title:"Fig2. MAGIC Applied to Mouse Myeloid Progenitor Data",slug:"fig2-magic-applied-to-mouse-myeloid-progenitor-data",normalizedTitle:"fig2. magic applied to mouse myeloid progenitor data",charIndex:1002},{level:3,title:"Fig3. MAGIC Preserves Cluster Structure",slug:"fig3-magic-preserves-cluster-structure",normalizedTitle:"fig3. magic preserves cluster structure",charIndex:1087},{level:3,title:"Fig4. MAGIC Recovers a State Space in EMT Data",slug:"fig4-magic-recovers-a-state-space-in-emt-data",normalizedTitle:"fig4. magic recovers a state space in emt data",charIndex:1153},{level:3,title:"Fig5. Gene-Gene Relationships and kNN-DREMI",slug:"fig5-gene-gene-relationships-and-knn-dremi",normalizedTitle:"fig5. gene-gene relationships and knn-dremi",charIndex:1235},{level:3,title:"Fig6. Gene Expression Dynamics Underlying EMT and TF Target Predictions",slug:"fig6-gene-expression-dynamics-underlying-emt-and-tf-target-predictions",normalizedTitle:"fig6. gene expression dynamics underlying emt and tf target predictions",charIndex:1311},{level:3,title:"Fig7. Comparison of MAGIC to Other Imputation and Smoothing Methods",slug:"fig7-comparison-of-magic-to-other-imputation-and-smoothing-methods",normalizedTitle:"fig7. comparison of magic to other imputation and smoothing methods",charIndex:1418}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2018.05.061" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2018.05.061<OutboundLink/></a><br>\nCite: van Dijk D, Sharma R, Nainys J, et al. Recovering Gene Interactions from Single-Cell Data Using Data Diffusion. <em>Cell</em>. 2018 Jul 26;174(3):716-729.e27.</p>\n',headersStr:"Abstract Results Fig1. Steps of the MAGIC Algorithm Fig2. MAGIC Applied to Mouse Myeloid Progenitor Data Fig3. MAGIC Preserves Cluster Structure Fig4. MAGIC Recovers a State Space in EMT Data Fig5. Gene-Gene Relationships and kNN-DREMI Fig6. Gene Expression Dynamics Underlying EMT and TF Target Predictions Fig7. Comparison of MAGIC to Other Imputation and Smoothing Methods",content:': \nDOI: https://doi.org/10.1016/j.cell.2018.05.061\nCite: van Dijk D, Sharma R, Nainys J, et al. Recovering Gene Interactions from Single-Cell Data Using Data Diffusion. Cell. 2018 Jul 26;174(3):716-729.e27.\n\n\n\nDANA PE\'ER\n\nMemorial Sloan Kettering Cancer Center\npeerd@mskcc.org\n\n\n# Abstract\n\nscRNAseq  mRNA  "dropout"- MAGIC(Markov affinity-based graph imputation of cells) MAGIC-MAGIC \n\n\n# Results\n\n\n# Fig1. Steps of the MAGIC Algorithm\n\n\n\nFig1. MAGIC \n\ni. MAGIC -\nii. (distance matrix)\niii.  Gaussian kernel (affinity matrix)\niv. (Markov matrix)\nv.  diffusion t \nvi.  denoised and imputed \n\n\n# Fig2. MAGIC Applied to Mouse Myeloid Progenitor Data\n\n\n\nFig2. MAGIC \n\n\n# Fig3. MAGIC Preserves Cluster Structure\n\n\n\nFig3. MAGIC \n\n\n# Fig4. MAGIC Recovers a State Space in EMT Data\n\n\n\nFig4. MAGIC  EMT \n\n\n# Fig5. Gene-Gene Relationships and kNN-DREMI\n\n\n\nFig5. - kNN-DREMI\n\n\n# Fig6. Gene Expression Dynamics Underlying EMT and TF Target Predictions\n\n\n\nFig6. EMT  TF \n\n\n# Fig7. Comparison of MAGIC to Other Imputation and Smoothing Methods\n\n\n\nFig7. MAGIC  imputation  smoothing ',normalizedContent:': \ndoi: https://doi.org/10.1016/j.cell.2018.05.061\ncite: van dijk d, sharma r, nainys j, et al. recovering gene interactions from single-cell data using data diffusion. cell. 2018 jul 26;174(3):716-729.e27.\n\n\n\ndana pe\'er\n\nmemorial sloan kettering cancer center\npeerd@mskcc.org\n\n\n# abstract\n\nscrnaseq  mrna  "dropout"- magic(markov affinity-based graph imputation of cells) magic-magic \n\n\n# results\n\n\n# fig1. steps of the magic algorithm\n\n\n\nfig1. magic \n\ni. magic -\nii. (distance matrix)\niii.  gaussian kernel (affinity matrix)\niv. (markov matrix)\nv.  diffusion t \nvi.  denoised and imputed \n\n\n# fig2. magic applied to mouse myeloid progenitor data\n\n\n\nfig2. magic \n\n\n# fig3. magic preserves cluster structure\n\n\n\nfig3. magic \n\n\n# fig4. magic recovers a state space in emt data\n\n\n\nfig4. magic  emt \n\n\n# fig5. gene-gene relationships and knn-dremi\n\n\n\nfig5. - knn-dremi\n\n\n# fig6. gene expression dynamics underlying emt and tf target predictions\n\n\n\nfig6. emt  tf \n\n\n# fig7. comparison of magic to other imputation and smoothing methods\n\n\n\nfig7. magic  imputation  smoothing ',charsets:{cjk:!0},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" --- An interactive framework for whole-brain maps at cellular resolution",frontmatter:{title:" --- An interactive framework for whole-brain maps at cellular resolution",date:"2024-08-30T20:23:45.000Z",permalink:"/pages/e020de/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20171204.(2017.12.04%20WholeBrain%20Nat%20Neurosci)%20An%20interactive%20framework%20for%20whole-brain%20maps%20at%20cellular%20resolution.html",relativePath:"02./14./20171204.(2017.12.04 WholeBrain Nat Neurosci) An interactive framework for whole-brain maps at cellular resolution.md",key:"v-195442fc",path:"/pages/e020de/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:431}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41593-017-0027-7" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41593-017-0027-7<OutboundLink/></a><br>\nCite: Frth, D., Vaissire, T., Tzortzi, O. et al. An interactive framework for whole-brain maps at cellular resolution. Nat Neurosci 21, 139149 (2018).</p>\n',headersStr:"Abstract",content:": \nDOI: https://doi.org/10.1038/s41593-017-0027-7\nCite: Frth, D., Vaissire, T., Tzortzi, O. et al. An interactive framework for whole-brain maps at cellular resolution. Nat Neurosci 21, 139149 (2018).\n\n\n\n\n\n\n\nDANIEL FRTH             KONSTANTINOS MELETIS\n                         \nSciLifeLab               Karolinska Institutet\nfurth@scilifelab.uu.se   dinos.meletis@ki.se\n\n\n# Abstract\n\n",normalizedContent:": \ndoi: https://doi.org/10.1038/s41593-017-0027-7\ncite: furth, d., vaissiere, t., tzortzi, o. et al. an interactive framework for whole-brain maps at cellular resolution. nat neurosci 21, 139149 (2018).\n\n\n\n\n\n\n\ndaniel furth             konstantinos meletis\n                         \nscilifelab               karolinska institutet\nfurth@scilifelab.uu.se   dinos.meletis@ki.se\n\n\n# abstract\n\n",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Comprehensive Integration of Single-Cell Data",frontmatter:{title:" --- Comprehensive Integration of Single-Cell Data",date:"2023-08-15T21:11:29.000Z",permalink:"/pages/d26f9e/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20190606.(2019.06.06%20Seurat%20V3%20Cell)%20Comprehensive%20Integration%20of%20Single-Cell%20Data.html",relativePath:"02./14./20190606.(2019.06.06 Seurat V3 Cell) Comprehensive Integration of Single-Cell Data.md",key:"v-5d420fe2",path:"/pages/d26f9e/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:766},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1158},{level:3,title:'Figure 1. Schematic Overview of Reference "Assembly" Integration in Seurat v3',slug:"figure-1-schematic-overview-of-reference-assembly-integration-in-seurat-v3",normalizedTitle:"figure 1. schematic overview of reference &quot;assembly&quot; integration in seurat v3",charIndex:null},{level:3,title:"Figure 2. Comparison of Multi-dataset Integration Methods for scRNA-Seq",slug:"figure-2-comparison-of-multi-dataset-integration-methods-for-scrna-seq",normalizedTitle:"figure 2. comparison of multi-dataset integration methods for scrna-seq",charIndex:1789},{level:3,title:"Figure 3. Transferring Cell State Classifications across Datasets",slug:"figure-3-transferring-cell-state-classifications-across-datasets",normalizedTitle:"figure 3. transferring cell state classifications across datasets",charIndex:2224},{level:3,title:"Figure 4. Imputing Immunophenotypes in a Transcriptomic Atlas of the Human Bone Marrow",slug:"figure-4-imputing-immunophenotypes-in-a-transcriptomic-atlas-of-the-human-bone-marrow",normalizedTitle:"figure 4. imputing immunophenotypes in a transcriptomic atlas of the human bone marrow",charIndex:2991},{level:3,title:"Figure 5. Spatial Patterns of Gene Expression in the Mouse Brain",slug:"figure-5-spatial-patterns-of-gene-expression-in-the-mouse-brain",normalizedTitle:"figure 5. spatial patterns of gene expression in the mouse brain",charIndex:3646}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2019.05.031" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2019.05.031<OutboundLink/></a><br>\nCite: Stuart T, Butler A, Hoffman P, et al. Comprehensive Integration of Single-Cell Data. <em>Cell</em>. 2019;177(7):1888-1902.e21.</p>\n',headersStr:'Abstract Results Figure 1. Schematic Overview of Reference "Assembly" Integration in Seurat v3 Figure 2. Comparison of Multi-dataset Integration Methods for scRNA-Seq Figure 3. Transferring Cell State Classifications across Datasets Figure 4. Imputing Immunophenotypes in a Transcriptomic Atlas of the Human Bone Marrow Figure 5. Spatial Patterns of Gene Expression in the Mouse Brain',content:': \nDOI: https://doi.org/10.1016/j.cell.2019.05.031\nCite: Stuart T, Butler A, Hoffman P, et al. Comprehensive Integration of Single-Cell Data. Cell. 2019;177(7):1888-1902.e21.\n\n\n\n(Seurat V3) \n\n\n\n 1.  --- (Seurat V1) Spatial reconstruction of single-cell gene expression data\n 2.  --- (Seurat V2) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (Seurat V3) Comprehensive Integration of Single-Cell Data\n 4.  --- (Seurat V4) Integrated analysis of multimodal single-cell data\n 5.  --- (Seurat V5) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nRAHUL SATIJA\n\nNew York Genome Center\nrsatija@nygenome.org\n\n\n# Abstract\n\n ("anchor")  scRNA-seq  scRNA-seq  scRNA-seq  scATAC-seq  scRNA-seq  (imputation)\n\n\n# Results\n\n\n# Figure 1. Schematic Overview of Reference "Assembly" Integration in Seurat v3\n\n\n\nFigure 1. Seurat v3  Reference "Assembly" Integration \n\n(A) referencequery\n(B) canonical correlation analysis, CCA L2 normalization\n(C)  MNNmutual nearest neighborsanchors\n(D) \n(E) \n\n\n# Figure 2. Comparison of Multi-dataset Integration Methods for scRNA-Seq\n\n\n\nFigure 2.  scRNA-Seq \n\n(AH)  Seurat v3A and ESeurat v2B and FmnnCorrectC and GScanoramaD and HADEH UMAP \n(I-J) anchor scoresI countsJ\n(K-L) KL\n\n\n# Figure 3. Transferring Cell State Classifications across Datasets\n\n\n\nFigure 3. \n\n(A) \n(B)  alpha  alpha  "Unassigned"\n(C)  166  test/training \n(D)  B \n(E)  scRNA-seq  scATAC-seq  scRNA-seq reference scATAC-seqquery anchors\n(F)  pseudo-bulk ATAC-seq RPKM y \n(G)  PV  DNA motifs Mef2c  Rora motifs  motifs\n(H) Mef2c  Rora  scRNA-seq  PV interneurons \n\n\n# Figure 4. Imputing Immunophenotypes in a Transcriptomic Atlas of the Human Bone Marrow\n\n\n\nFigure 4. \n\n(A)  35,543  25  CITE-seq \n(B)  anchors \n(C)  HCA  274,932  CITE-seq  HCA \n(D)  CD8+ CD69+  RNA \n(E)  scRNA-seq  CD69+  FACS  CD8+ CD69+/-  RNA \n(F)  HCA  CITE-seq  CD69  CD8+  CD69 Moran\'s ICD69+  HCA  Moran\'s I \n\n\n# Figure 5. Spatial Patterns of Gene Expression in the Mouse Brain\n\n\n\nFigure 5. \n\n(A) scRNA-seq  STARmap  transfer  anchors  transfer  STARmap \n(B)  STARmap \n(C) RorbSyt6Lamp5  Sox10  osmFISH\n(D)  STARmap \n(E)  STARmap  Moran\'s I  VLMCs\n(F)  SMART-seq2  transferred  STARmap \n(G)  STARmap ',normalizedContent:': \ndoi: https://doi.org/10.1016/j.cell.2019.05.031\ncite: stuart t, butler a, hoffman p, et al. comprehensive integration of single-cell data. cell. 2019;177(7):1888-1902.e21.\n\n\n\n(seurat v3) \n\n\n\n 1.  --- (seurat v1) spatial reconstruction of single-cell gene expression data\n 2.  --- (seurat v2) integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (seurat v3) comprehensive integration of single-cell data\n 4.  --- (seurat v4) integrated analysis of multimodal single-cell data\n 5.  --- (seurat v5) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nrahul satija\n\nnew york genome center\nrsatija@nygenome.org\n\n\n# abstract\n\n ("anchor")  scrna-seq  scrna-seq  scrna-seq  scatac-seq  scrna-seq  (imputation)\n\n\n# results\n\n\n# figure 1. schematic overview of reference "assembly" integration in seurat v3\n\n\n\nfigure 1. seurat v3  reference "assembly" integration \n\n(a) referencequery\n(b) canonical correlation analysis, cca l2 normalization\n(c)  mnnmutual nearest neighborsanchors\n(d) \n(e) \n\n\n# figure 2. comparison of multi-dataset integration methods for scrna-seq\n\n\n\nfigure 2.  scrna-seq \n\n(ah)  seurat v3a and eseurat v2b and fmnncorrectc and gscanoramad and hadeh umap \n(i-j) anchor scoresi countsj\n(k-l) kl\n\n\n# figure 3. transferring cell state classifications across datasets\n\n\n\nfigure 3. \n\n(a) \n(b)  alpha  alpha  "unassigned"\n(c)  166  test/training \n(d)  b \n(e)  scrna-seq  scatac-seq  scrna-seq reference scatac-seqquery anchors\n(f)  pseudo-bulk atac-seq rpkm y \n(g)  pv  dna motifs mef2c  rora motifs  motifs\n(h) mef2c  rora  scrna-seq  pv interneurons \n\n\n# figure 4. imputing immunophenotypes in a transcriptomic atlas of the human bone marrow\n\n\n\nfigure 4. \n\n(a)  35,543  25  cite-seq \n(b)  anchors \n(c)  hca  274,932  cite-seq  hca \n(d)  cd8+ cd69+  rna \n(e)  scrna-seq  cd69+  facs  cd8+ cd69+/-  rna \n(f)  hca  cite-seq  cd69  cd8+  cd69 moran\'s icd69+  hca  moran\'s i \n\n\n# figure 5. spatial patterns of gene expression in the mouse brain\n\n\n\nfigure 5. \n\n(a) scrna-seq  starmap  transfer  anchors  transfer  starmap \n(b)  starmap \n(c) rorbsyt6lamp5  sox10  osmfish\n(d)  starmap \n(e)  starmap  moran\'s i  vlmcs\n(f)  smart-seq2  transferred  starmap \n(g)  starmap ',charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- Integrating single-cell transcriptomic data across different conditions, technologies, and species",frontmatter:{title:" --- Integrating single-cell transcriptomic data across different conditions, technologies, and species",date:"2023-08-11T21:03:46.000Z",permalink:"/pages/f37280/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20180402.(2018.04.02%20Seurat%20V2%20Nat%20Biotechnol)%20Integrating%20single-cell%20transcriptomic%20data%20across%20different%20conditions,%20technologies,%20and%20species.html",relativePath:"02./14./20180402.(2018.04.02 Seurat V2 Nat Biotechnol) Integrating single-cell transcriptomic data across different conditions, technologies, and species.md",key:"v-778d5c48",path:"/pages/f37280/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:845},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1269},{level:3,title:"Figure 1. Overview of Seurat alignment of single-cell RNA-seq data sets",slug:"figure-1-overview-of-seurat-alignment-of-single-cell-rna-seq-data-sets",normalizedTitle:"figure 1. overview of seurat alignment of single-cell rna-seq data sets",charIndex:1281},{level:3,title:"Figure 2. Integrated analysis of resting and stimulated PBMCs",slug:"figure-2-integrated-analysis-of-resting-and-stimulated-pbmcs",normalizedTitle:"figure 2. integrated analysis of resting and stimulated pbmcs",charIndex:1794},{level:3,title:"Figure 3. Comparative analysis of mouse hematopoietic progenitors across scRNA-seq technologies",slug:"figure-3-comparative-analysis-of-mouse-hematopoietic-progenitors-across-scrna-seq-technologies",normalizedTitle:"figure 3. comparative analysis of mouse hematopoietic progenitors across scrna-seq technologies",charIndex:2444},{level:3,title:"Figure 4. Joint identification of cell types across human and mouse islet scRNA-seq atlases",slug:"figure-4-joint-identification-of-cell-types-across-human-and-mouse-islet-scrna-seq-atlases",normalizedTitle:"figure 4. joint identification of cell types across human and mouse islet scrna-seq atlases",charIndex:3103},{level:3,title:"Figure 5. Benchmarking alignment and batch correction methods",slug:"figure-5-benchmarking-alignment-and-batch-correction-methods",normalizedTitle:"figure 5. benchmarking alignment and batch correction methods",charIndex:3642}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/nbt.4096" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/nbt.4096<OutboundLink/></a><br>\nCite: Butler, A., Hoffman, P., Smibert, P. et al. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol 36, 411420 (2018).</p>\n',headersStr:"Abstract Results Figure 1. Overview of Seurat alignment of single-cell RNA-seq data sets Figure 2. Integrated analysis of resting and stimulated PBMCs Figure 3. Comparative analysis of mouse hematopoietic progenitors across scRNA-seq technologies Figure 4. Joint identification of cell types across human and mouse islet scRNA-seq atlases Figure 5. Benchmarking alignment and batch correction methods",content:": \nDOI: https://doi.org/10.1038/nbt.4096\nCite: Butler, A., Hoffman, P., Smibert, P. et al. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nat Biotechnol 36, 411420 (2018).\n\n\n\n(Seurat V2) \n\n\n\n 1.  --- (Seurat V1) Spatial reconstruction of single-cell gene expression data\n 2.  --- (Seurat V2) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (Seurat V3) Comprehensive Integration of Single-Cell Data\n 4.  --- (Seurat V4) Integrated analysis of multimodal single-cell data\n 5.  --- (Seurat V5) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nRAHUL SATIJA\n\nNew York Genome Center\nrsatija@nygenome.org\n\n\n# Abstract\n\n single-cell RNA-seq (scRNA-seq)  scRNA-seq  R  Seurat (http://satijalab.org/seurat/)  scRNA-seq  ('atlases') scRNA-seq \n\n\n# Results\n\n\n# Figure 1. Overview of Seurat alignment of single-cell RNA-seq data sets\n\n\n\nFigure 1.  RNA-seq  Seurat \n\n(a) - scRNAseq\n(b) Seurat canonical correlation analysis, CCAcanonical correlation vectors, CCdynamic time warping, DTW t-SNE  2D \n(c) \n\n\n# Figure 2. Integrated analysis of resting and stimulated PBMCs\n\n\n\nFigure 2.  PBMCs \n\n(a-c) 14,039  PBMCs  control  IFN- (a) (b) (c)  13 \n(d) Cell-type-specific markers IFN- Uniform IFN- markers IFN-  Cell-type-specific IFN- markers cluster  cluster \n(e)  clustern = 13 clusters8 \n(f) DC  pDC  IFN- \n(g) IFN- n = 430 genespDC IFN- \n\n\n# Figure 3. Comparative analysis of mouse hematopoietic progenitors across scRNA-seq technologies\n\n\n\nFigure 3.  scRNA-seq \n\n(a-c)  MARS-Seq (2,686)  SMART-Seq2SS2; 765 3,451  t-SNE (a) (b,c)\n(c-d)  SMART-Seq2  MARS-Seq cluster from Paul et al.t-SNE  (c) SMART-Seq2 from Nestorawa et al. MARS-Seq clusters(d) \n(e-f)  MARS-Seq  SMART-Seq2  MARS-Seq cluster (e)  MARS-Seq cluster(f) \n(g-h) (g) '' (h)\n(i) \n\n\n# Figure 4. Joint identification of cell types across human and mouse islet scRNA-seq atlases\n\n\n\nFigure 4.  scRNA-seq \n\n(a-c)  (n=8,424 cells)  (n=1,767 cells)  10,191 (a) (b)  t-SNE  (c)  10 \n(d-e)  (d)  (e)  DE \n(f) n=2,431 cellsn=762 cellsn=126 cellsn=10 cells\n(g)  'ER-stress'  n = 100  GOplot R  GO \n\n\n# Figure 5. Benchmarking alignment and batch correction methods\n\n\n\nFigure 5. \n\n(a,d,g,j,m) ComBat  t-SNE \n(b,e,h,k,n) limma  t-SNE \n(c,f,i,l,o)  SeuratComBatlimma Seurat ",normalizedContent:": \ndoi: https://doi.org/10.1038/nbt.4096\ncite: butler, a., hoffman, p., smibert, p. et al. integrating single-cell transcriptomic data across different conditions, technologies, and species. nat biotechnol 36, 411420 (2018).\n\n\n\n(seurat v2) \n\n\n\n 1.  --- (seurat v1) spatial reconstruction of single-cell gene expression data\n 2.  --- (seurat v2) integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (seurat v3) comprehensive integration of single-cell data\n 4.  --- (seurat v4) integrated analysis of multimodal single-cell data\n 5.  --- (seurat v5) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nrahul satija\n\nnew york genome center\nrsatija@nygenome.org\n\n\n# abstract\n\n single-cell rna-seq (scrna-seq)  scrna-seq  r  seurat (http://satijalab.org/seurat/)  scrna-seq  ('atlases') scrna-seq \n\n\n# results\n\n\n# figure 1. overview of seurat alignment of single-cell rna-seq data sets\n\n\n\nfigure 1.  rna-seq  seurat \n\n(a) - scrnaseq\n(b) seurat canonical correlation analysis, ccacanonical correlation vectors, ccdynamic time warping, dtw t-sne  2d \n(c) \n\n\n# figure 2. integrated analysis of resting and stimulated pbmcs\n\n\n\nfigure 2.  pbmcs \n\n(a-c) 14,039  pbmcs  control  ifn- (a) (b) (c)  13 \n(d) cell-type-specific markers ifn- uniform ifn- markers ifn-  cell-type-specific ifn- markers cluster  cluster \n(e)  clustern = 13 clusters8 \n(f) dc  pdc  ifn- \n(g) ifn- n = 430 genespdc ifn- \n\n\n# figure 3. comparative analysis of mouse hematopoietic progenitors across scrna-seq technologies\n\n\n\nfigure 3.  scrna-seq \n\n(a-c)  mars-seq (2,686)  smart-seq2ss2; 765 3,451  t-sne (a) (b,c)\n(c-d)  smart-seq2  mars-seq cluster from paul et al.t-sne  (c) smart-seq2 from nestorawa et al. mars-seq clusters(d) \n(e-f)  mars-seq  smart-seq2  mars-seq cluster (e)  mars-seq cluster(f) \n(g-h) (g) '' (h)\n(i) \n\n\n# figure 4. joint identification of cell types across human and mouse islet scrna-seq atlases\n\n\n\nfigure 4.  scrna-seq \n\n(a-c)  (n=8,424 cells)  (n=1,767 cells)  10,191 (a) (b)  t-sne  (c)  10 \n(d-e)  (d)  (e)  de \n(f) n=2,431 cellsn=762 cellsn=126 cellsn=10 cells\n(g)  'er-stress'  n = 100  goplot r  go \n\n\n# figure 5. benchmarking alignment and batch correction methods\n\n\n\nfigure 5. \n\n(a,d,g,j,m) combat  t-sne \n(b,e,h,k,n) limma  t-sne \n(c,f,i,l,o)  seuratcombatlimma seurat ",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- DoubletFinder:Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors",frontmatter:{title:" --- DoubletFinder:Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors",date:"2022-12-12T19:19:36.000Z",permalink:"/pages/71b694/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20190403.(2019.04.03%20DoubletFinder%20Cell%20Systems)%20DoubletFinder%20Doublet%20Detection%20in%20Single-Cell%20RNA%20Sequencing%20Data%20Using%20Artificial%20Nearest%20Neighbors.html",relativePath:"02./14./20190403.(2019.04.03 DoubletFinder Cell Systems) DoubletFinder Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors.md",key:"v-084ec0e5",path:"/pages/71b694/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:636},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1127},{level:3,title:"Box1. DoubletFinder Real-World Workflow Interfaces with Seurat",slug:"box1-doubletfinder-real-world-workflow-interfaces-with-seurat",normalizedTitle:"box1. doubletfinder real-world workflow interfaces with seurat",charIndex:1139},{level:3,title:"Fig1. DoubletFinder Overview, nUMI Comparison, and Ground-Truth Benchmarking",slug:"fig1-doubletfinder-overview-numi-comparison-and-ground-truth-benchmarking",normalizedTitle:"fig1. doubletfinder overview, numi comparison, and ground-truth benchmarking",charIndex:2085},{level:3,title:'Fig2. BC Maximization Estimates Ideal DoubletFinder Parameters for Real-World scRNA-Seq Data and Facilitates DoubletFinder Application to Mouse Kidney Data with "Hybrid" Cell States',slug:"fig2-bcmvn-maximization-estimates-ideal-doubletfinder-parameters-for-real-world-scrna-seq-data-and-facilitates-doubletfinder-application-to-mouse-kidney-data-with-hybrid-cell-states",normalizedTitle:"fig2. bc maximization estimates ideal doubletfinder parameters for real-world scrna-seq data and facilitates doubletfinder application to mouse kidney data with &quot;hybrid&quot; cell states",charIndex:null}],excerpt:'<p>: DoubletFinderRNA<br>\nDOI: <a href="https://doi.org/10.1016/j.cels.2019.03.003" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cels.2019.03.003<OutboundLink/></a><br>\nCite: McGinnis CS, Murrow LM, Gartner ZJ. DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors. <em>Cell Syst</em>. 2019 Apr 24;8(4):329-337.e4.</p>\n',headersStr:'Abstract Results Box1. DoubletFinder Real-World Workflow Interfaces with Seurat Fig1. DoubletFinder Overview, nUMI Comparison, and Ground-Truth Benchmarking Fig2. BC Maximization Estimates Ideal DoubletFinder Parameters for Real-World scRNA-Seq Data and Facilitates DoubletFinder Application to Mouse Kidney Data with "Hybrid" Cell States',content:': DoubletFinderRNA\nDOI: https://doi.org/10.1016/j.cels.2019.03.003\nCite: McGinnis CS, Murrow LM, Gartner ZJ. DoubletFinder: Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors. Cell Syst. 2019 Apr 24;8(4):329-337.e4.\n\n\n\nDoubletFinderRNA\n\n\n\n 1.  --- DoubletFinder:Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors\n 2. DoubletFinderR package for detecting doublets in single-cell RNA sequencing data\n\n\n\nZEV GARTNER\n\nUniversity of California San Francisco(UCSF)\nzev.gartner@ucsf.edu\n\n\n# Abstract\n\nscRNA-seq  "doublets"  doublet  DoubletFinder doubletsDoubletFinder  doublets  doublets doublets identity  scRNA-seq  DoubletFinder  doublets doublets  DoubletFinder  scRNA-seq  DoubletFinder  "best practices" DoubletFinder  "" \n\n\n# Results\n\n\n# Box1. DoubletFinder Real-World Workflow Interfaces with Seurat\n\n\n\nBox1. DoubletFinder  Seurat \n\nSeurat  Gene/Cell-filter  raw RNA UMI counts  log2-normalization  regression  normalized  centered  scaled variably expressed genes PCAunsupervised clustering  literature annotation  downstream analyses\n\nDoubletFinder parameter selection  doublet classification raw RNA UMI count  artificial doubletspN artificial doublets  scRNA-seq  Seurat  PCA  PC pK pANN  pN  pK  pANN  BCmvn  pK pK doublet classification  homotypic doublet  doublet  doublet  heterotypic doublets Homotypic doublets  pANN  doublets classifications doublets\n\n\n# Fig1. DoubletFinder Overview, nUMI Comparison, and Ground-Truth Benchmarking\n\n\n\nFig1. DoubletFinder nUMI  Ground-Truth \n\nA. DoubletFinder Doublet  scRNA-seq  Seurat  scRNA-seq DoubletFinder  (pN)  doubletsDoubletFinder  (pK) (pANN)  pANN  doubletsDoublet  scRNA-seq    doublets \nB.  pN-pK  singletspN  artificial doubletspK  pANN neighbors \nC.  Demuxlet  Cell Hashing pN-pK  AUC \nD.  DoubletFinder nUMIs nUMIs  DoubletFinder ROC \nE.  PBMC  Demuxlet  DoubletFinder doublets  t-SNE False-negative DoubletFinder classifications singlets putative false-positive DoubletFinder classifications heterotypic doubletsMono, monocytes; NK, natural killer cells; MK, megakaryocytes; and DC, dendritic cells\nF.  ground-truth singlets and doublets  ground-truth doublet neighbors Singlets doublet neighbor  doublets doublet neighbor Homotypic (Hom.)  heterotypic (Het.) doublets  single  doublet t-SNE homotypic doublets singlets heterotypic doublets\nG. true-positive doublets ()putative false-negative doublets ()singlets ()  RNA UMI \nH. true-positive doubletsfalse-negative doubletsB cellsCD14 monocytes \nI.  doublet \nJ.  doublet \n\n\n# Fig2. BCMVN Maximization Estimates Ideal DoubletFinder Parameters for Real-World scRNA-Seq Data and Facilitates DoubletFinder Application to Mouse Kidney Data with "Hybrid" Cell States\n\n\n\nFig2. BCMVN  scRNA-Seq  DoubletFinder  DoubletFinder \n\nA.  scRNA-seq topbottom doubletsred pDE, probability of differential expression\nB.  pN-pK  AUC  pK pDE 10.0%top=8bottomDoubletFinder \nC.  BCMVN AUC  Demuxlet  Cell Hashing  AUC pK  BCMVN  pK  BCMVN  pK \nD.  DoubletFinder doublet  t-SNE \nE. doublet singlets RNA UMI \nF. doubletPT cells DCT cells marker gene \nG.  doublet ',normalizedContent:': doubletfinderrna\ndoi: https://doi.org/10.1016/j.cels.2019.03.003\ncite: mcginnis cs, murrow lm, gartner zj. doubletfinder: doublet detection in single-cell rna sequencing data using artificial nearest neighbors. cell syst. 2019 apr 24;8(4):329-337.e4.\n\n\n\ndoubletfinderrna\n\n\n\n 1.  --- doubletfinder:doublet detection in single-cell rna sequencing data using artificial nearest neighbors\n 2. doubletfinderr package for detecting doublets in single-cell rna sequencing data\n\n\n\nzev gartner\n\nuniversity of california san francisco(ucsf)\nzev.gartner@ucsf.edu\n\n\n# abstract\n\nscrna-seq  "doublets"  doublet  doubletfinder doubletsdoubletfinder  doublets  doublets doublets identity  scrna-seq  doubletfinder  doublets doublets  doubletfinder  scrna-seq  doubletfinder  "best practices" doubletfinder  "" \n\n\n# results\n\n\n# box1. doubletfinder real-world workflow interfaces with seurat\n\n\n\nbox1. doubletfinder  seurat \n\nseurat  gene/cell-filter  raw rna umi counts  log2-normalization  regression  normalized  centered  scaled variably expressed genes pcaunsupervised clustering  literature annotation  downstream analyses\n\ndoubletfinder parameter selection  doublet classification raw rna umi count  artificial doubletspn artificial doublets  scrna-seq  seurat  pca  pc pk pann  pn  pk  pann  bcmvn  pk pk doublet classification  homotypic doublet  doublet  doublet  heterotypic doublets homotypic doublets  pann  doublets classifications doublets\n\n\n# fig1. doubletfinder overview, numi comparison, and ground-truth benchmarking\n\n\n\nfig1. doubletfinder numi  ground-truth \n\na. doubletfinder doublet  scrna-seq  seurat  scrna-seq doubletfinder  (pn)  doubletsdoubletfinder  (pk) (pann)  pann  doubletsdoublet  scrna-seq    doublets \nb.  pn-pk  singletspn  artificial doubletspk  pann neighbors \nc.  demuxlet  cell hashing pn-pk  auc \nd.  doubletfinder numis numis  doubletfinder roc \ne.  pbmc  demuxlet  doubletfinder doublets  t-sne false-negative doubletfinder classifications singlets putative false-positive doubletfinder classifications heterotypic doubletsmono, monocytes; nk, natural killer cells; mk, megakaryocytes; and dc, dendritic cells\nf.  ground-truth singlets and doublets  ground-truth doublet neighbors singlets doublet neighbor  doublets doublet neighbor homotypic (hom.)  heterotypic (het.) doublets  single  doublet t-sne homotypic doublets singlets heterotypic doublets\ng. true-positive doublets ()putative false-negative doublets ()singlets ()  rna umi \nh. true-positive doubletsfalse-negative doubletsb cellscd14 monocytes \ni.  doublet \nj.  doublet \n\n\n# fig2. bcmvn maximization estimates ideal doubletfinder parameters for real-world scrna-seq data and facilitates doubletfinder application to mouse kidney data with "hybrid" cell states\n\n\n\nfig2. bcmvn  scrna-seq  doubletfinder  doubletfinder \n\na.  scrna-seq topbottom doubletsred pde, probability of differential expression\nb.  pn-pk  auc  pk pde 10.0%top=8bottomdoubletfinder \nc.  bcmvn auc  demuxlet  cell hashing  auc pk  bcmvn  pk  bcmvn  pk \nd.  doubletfinder doublet  t-sne \ne. doublet singlets rna umi \nf. doubletpt cells dct cells marker gene \ng.  doublet ',charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- GCNGgraph convolutional networks for inferring gene interaction from spatial transcriptomics data",frontmatter:{title:" --- GCNGgraph convolutional networks for inferring gene interaction from spatial transcriptomics data",date:"2024-07-26T11:27:40.000Z",permalink:"/pages/c34d2c/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20201210.(2020.12.10,%20GCNG,%20Genome%20Biol)%20GCNG%20graph%20convolutional%20networks%20for%20inferring%20gene%20interaction%20from%20spatial%20transcriptomics%20data.html",relativePath:"02./14./20201210.(2020.12.10, GCNG, Genome Biol) GCNG graph convolutional networks for inferring gene interaction from spatial transcriptomics data.md",key:"v-620e68fd",path:"/pages/c34d2c/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:362}],excerpt:'<p>: GCNG<br>\nDOI: <a href="https://doi.org/10.1186/s13059-020-02214-w" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13059-020-02214-w<OutboundLink/></a><br>\nCite: Yuan, Y., Bar-Joseph, Z. GCNG: graph convolutional networks for inferring gene interaction from\nspatial transcriptomics data. Genome Biol 21, 300 (2020).</p>\n',headersStr:"Abstract",content:": GCNG\nDOI: https://doi.org/10.1186/s13059-020-02214-w\nCite: Yuan, Y., Bar-Joseph, Z. GCNG: graph convolutional networks for inferring gene interaction from spatial transcriptomics data. Genome Biol 21, 300 (2020).\n\n\n\nGCNG\n\n\n\nZIV BAR-JOSEPH\n\nCarnegie Mellon University\nzivbj@cs.cmu.edu\n\n\n# Abstract\n\nGraph Convolutional Neural networks for Genes, GCNGGCNG GCNG GCNG   https://github.com/xiaoyeye/GCNG",normalizedContent:": gcng\ndoi: https://doi.org/10.1186/s13059-020-02214-w\ncite: yuan, y., bar-joseph, z. gcng: graph convolutional networks for inferring gene interaction from spatial transcriptomics data. genome biol 21, 300 (2020).\n\n\n\ngcng\n\n\n\nziv bar-joseph\n\ncarnegie mellon university\nzivbj@cs.cmu.edu\n\n\n# abstract\n\ngraph convolutional neural networks for genes, gcnggcng gcng gcng   https://github.com/xiaoyeye/gcng",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation",frontmatter:{title:" --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation",date:"2023-04-07T16:28:02.000Z",permalink:"/pages/d38252/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20201207.(2020.12.07%20nnU-Net%20Nat%20Methods)%20nnU-Net%20a%20self-configuring%20method%20for%20deep%20learning-based%20biomedical%20image%20segmentation.html",relativePath:"02./14./20201207.(2020.12.07 nnU-Net Nat Methods) nnU-Net a self-configuring method for deep learning-based biomedical image segmentation.md",key:"v-48e1665e",path:"/pages/d38252/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:699},{level:2,title:"Introduction",slug:"introduction",normalizedTitle:"introduction",charIndex:1036},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:2466},{level:3,title:"Fig.1 nnU-Net handles a broad variety of datasets and target image properties",slug:"fig-1-nnu-net-handles-a-broad-variety-of-datasets-and-target-image-properties",normalizedTitle:"fig.1 nnu-net handles a broad variety of datasets and target image properties",charIndex:2580},{level:3,title:"Fig.2 Proposed automated method configuration for deep learning-based biomedical image segmentation",slug:"fig-2-proposed-automated-method-configuration-for-deep-learning-based-biomedical-image-segmentation",normalizedTitle:"fig.2 proposed automated method configuration for deep learning-based biomedical image segmentation",charIndex:4860},{level:3,title:"Fig.3 nnU-Net outperforms most specialized deep learning pipelines",slug:"fig-3-nnu-net-outperforms-most-specialized-deep-learning-pipelines",normalizedTitle:"fig.3 nnu-net outperforms most specialized deep learning pipelines",charIndex:5954},{level:3,title:"Fig.4 Pipeline fingerprints from KiTS 2019 leaderboard entries",slug:"fig-4-pipeline-fingerprints-from-kits-2019-leaderboard-entries",normalizedTitle:"fig.4 pipeline fingerprints from kits 2019 leaderboard entries",charIndex:7424},{level:3,title:"Fig.5 Data fingerprints across different challenge datasets",slug:"fig-5-data-fingerprints-across-different-challenge-datasets",normalizedTitle:"fig.5 data fingerprints across different challenge datasets",charIndex:8072},{level:3,title:"Fig.6 Evaluation of design decisions across multiple tasks",slug:"fig-6-evaluation-of-design-decisions-across-multiple-tasks",normalizedTitle:"fig.6 evaluation of design decisions across multiple tasks",charIndex:8709},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:9217}],excerpt:'<p>: nnU-Net: <br>\nDOI: <a href="https://doi.org/10.1038/s41592-020-01008-z" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-020-01008-z<OutboundLink/></a><br>\nCite: Isensee, F., Jaeger, P.F., Kohl, S.A.A. et al. nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation. <em>Nat Methods</em> 18, 203211 (2021).</p>\n',headersStr:"Abstract Introduction Results Fig.1 nnU-Net handles a broad variety of datasets and target image properties Fig.2 Proposed automated method configuration for deep learning-based biomedical image segmentation Fig.3 nnU-Net outperforms most specialized deep learning pipelines Fig.4 Pipeline fingerprints from KiTS 2019 leaderboard entries Fig.5 Data fingerprints across different challenge datasets Fig.6 Evaluation of design decisions across multiple tasks Discussion",content:': nnU-Net: \nDOI: https://doi.org/10.1038/s41592-020-01008-z\nCite: Isensee, F., Jaeger, P.F., Kohl, S.A.A. et al. nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation. Nat Methods 18, 203211 (2021).\n\n\n\nnnU-Net: \n\n\n\n 1.  --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation\n 3. nnUNeta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nKLAUS H. MAIER-HEIN\n\nGerman Cancer Research Center\nk.maier-hein@dkfz.de\n\n\n# Abstract\n\n nnU-NetnnU-Net 23 nnU-Net \n\n\n# Introduction\n\n 70%\n\nAutoML AutoML \n\n AutoML \n\n 1. ""\n 2. """"""\n 3. ""\n\n Medical Segmentation Decathlon  nnU-Net nnU-Net nnU-Net nnU-Net \n\nnnU-Net  13  53 nnU-Net \n\n\n# Results\n\nnnU-Net  nnU-Net  Fig.1 \n\n\n# Fig.1 nnU-Net handles a broad variety of datasets and target image properties\n\n\n\nFig1. nnU-Net \n\n nnU-Net 2D3D MITK Workbench  a. CT dataset (D)18)\nb. FM  A549 (D22)\nc. CT (D6)\nd. T1 in-phase MRI (D16)\ne. (D19) (https://cremi.org/)\nf. MRI (T1, T1 with contrast agent, T2, FLAIR) (D1)\ng. CT (D17)\nh. CT  13 D11\ni. CTD8\nj. MRI (D2)\nk. cine MRI (D13)\nl. FM  HL60  (D21)\n\nnnU-Net  Fig. 2  nnU-Net \n\nnnU-Net nnU-Net  "U-Net-like" \'dataset fingerprint\'  \'pipeline fingerprint\'  \'dataset fingerprint\'  \'pipeline fingerprint\' \n\n *  batch size  1  batch size \n *  patch size \n *  patch size \n\n" GPU " Supplementary Note 2  nnU-Net\n\nnnU-Net  nnU-Net nnU-Net nnU-Net nnU-Net  U-Net 2DU-Net3DU-Net 3D U-Net cascade U-Net  U-Net nnU-Net  nnU-Net  "" nnU-Net  13  nnU-Net \n\nnnU-Net  Methods  Supplementary Note 2 nnU-Net  Supplementary Note 6 \n\n\n# Fig.2 Proposed automated method configuration for deep learning-based biomedical image segmentation\n\n\n\nFig2. \n\n \'dataset fingerprint\' \'rule-based parameters\' \'fixed parameters\' five-fold nnU-Net (\'empirical parameters\', )Res., resolution\n\nnnU-Net   11  23  53  (https://cremi.org/  nnU-Net  2D  3D MRICTEMFM \'Challenges\' nnU-Net  nnU-Net  (Supplementary Note 3, sections 1 and 2)nnU-Net  Fig. 1 \n\nnnU-Net  Fig. 3  nnU-Net  53  nnU-Net nnU-Net  53  33 \n\n\n# Fig.3 nnU-Net outperforms most specialized deep learning pipelines\n\n\n\nFig3. nnU-Net \n\n nnU-Net nnU-Net nnU-Net  (D16) section 9 of Supplementary Note 6  (D20-D23)  2020  7  30  2019  12  12 SIM, simulated\n\n MICCAI (KiTS) 2019MICCAI  50% KiTS MICCAI 2019  100  (rank 18 of 100)  ""http://results.kits-challenge.org/miccai2019/manuscripts/peekaboo_2.pdf KiTSFig. 4a  KiTS http://results.kits-challenge.org/miccai2019 15  2016 3DU-Net  15  KiTS \n\nFig. 4b  3D U-Net\n\nnnU-Net  KiTS  3D U-Net  leaderboard nnU-Net  leaderboard  leaderboard  22  (Fig. 3)\n\n\n# Fig.4 Pipeline fingerprints from KiTS 2019 leaderboard entries\n\n\n\nFig4.  KiTS 2019 \n\na. 153D1"3DU-Net-like"\nb.  3D U-net  z-score CE, cross-entropy loss function; Dice, soft Dice loss function; WBCE, weighted binary CE KiTS 2019 challenge  http://results.kits-challenge.org/\n\n  23  Fig. 5 KiTSnnU-Net Fig. 2\n\n\n# Fig.5 Data fingerprints across different challenge datasets\n\n\n\nFig5. \n\n( s.d.  z-score  nnU-Net  23  (see Supplementary Note 1 for detailed dataset descriptions)\n\n nnU-Net  nnU-Net  nnU-Net nnU-Net  nnU-Net Fig. 6\n\n nnU-Net \n\n\n# Fig.6 Evaluation of design decisions across multiple tasks\n\n\n\nFig6. \n\na-j.  10 (D1D10, see Fig. 5 for dataset references)cross-entropy and TopK10AdamSupplementary Note 8  challengeR \nk. \n\n\n# Discussion\n\n nnU-NetnnU-Net nnU-Net  nnU-Net nnU-Net nnU-Net \n\n KiTS Fig. 4 Litjens et al.  """"Fig. 5 Litjens et al. "" nnU-Net  U-Net  CT Fig. 6nnU-Net nnU-Net nnU-Net \n\n nnU-Net  AutoML  nnU-Net "black box algorithms" AutoML nnU-Net \n\n 53  nnU-Net nnU-Net nnU-Net  Dice  CREMI (https://cremi.org) nnU-Net (396) EM  nnU-Net  nnU-Net \n\nnnU-Net  Litjens et al.  """"nnU-Net  nnU-Net ',normalizedContent:': nnu-net: \ndoi: https://doi.org/10.1038/s41592-020-01008-z\ncite: isensee, f., jaeger, p.f., kohl, s.a.a. et al. nnu-net: a self-configuring method for deep learning-based biomedical image segmentation. nat methods 18, 203211 (2021).\n\n\n\nnnu-net: \n\n\n\n 1.  --- nnu-neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- toward data-efficient learninga benchmark for covid-19 ct lung and infection segmentation\n 3. nnuneta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nklaus h. maier-hein\n\ngerman cancer research center\nk.maier-hein@dkfz.de\n\n\n# abstract\n\n nnu-netnnu-net 23 nnu-net \n\n\n# introduction\n\n 70%\n\nautoml automl \n\n automl \n\n 1. ""\n 2. """"""\n 3. ""\n\n medical segmentation decathlon  nnu-net nnu-net nnu-net nnu-net \n\nnnu-net  13  53 nnu-net \n\n\n# results\n\nnnu-net  nnu-net  fig.1 \n\n\n# fig.1 nnu-net handles a broad variety of datasets and target image properties\n\n\n\nfig1. nnu-net \n\n nnu-net 2d3d mitk workbench  a. ct dataset (d)18)\nb. fm  a549 (d22)\nc. ct (d6)\nd. t1 in-phase mri (d16)\ne. (d19) (https://cremi.org/)\nf. mri (t1, t1 with contrast agent, t2, flair) (d1)\ng. ct (d17)\nh. ct  13 d11\ni. ctd8\nj. mri (d2)\nk. cine mri (d13)\nl. fm  hl60  (d21)\n\nnnu-net  fig. 2  nnu-net \n\nnnu-net nnu-net  "u-net-like" \'dataset fingerprint\'  \'pipeline fingerprint\'  \'dataset fingerprint\'  \'pipeline fingerprint\' \n\n *  batch size  1  batch size \n *  patch size \n *  patch size \n\n" gpu " supplementary note 2  nnu-net\n\nnnu-net  nnu-net nnu-net nnu-net nnu-net  u-net 2du-net3du-net 3d u-net cascade u-net  u-net nnu-net  nnu-net  "" nnu-net  13  nnu-net \n\nnnu-net  methods  supplementary note 2 nnu-net  supplementary note 6 \n\n\n# fig.2 proposed automated method configuration for deep learning-based biomedical image segmentation\n\n\n\nfig2. \n\n \'dataset fingerprint\' \'rule-based parameters\' \'fixed parameters\' five-fold nnu-net (\'empirical parameters\', )res., resolution\n\nnnu-net   11  23  53  (https://cremi.org/  nnu-net  2d  3d mrictemfm \'challenges\' nnu-net  nnu-net  (supplementary note 3, sections 1 and 2)nnu-net  fig. 1 \n\nnnu-net  fig. 3  nnu-net  53  nnu-net nnu-net  53  33 \n\n\n# fig.3 nnu-net outperforms most specialized deep learning pipelines\n\n\n\nfig3. nnu-net \n\n nnu-net nnu-net nnu-net  (d16) section 9 of supplementary note 6  (d20-d23)  2020  7  30  2019  12  12 sim, simulated\n\n miccai (kits) 2019miccai  50% kits miccai 2019  100  (rank 18 of 100)  ""http://results.kits-challenge.org/miccai2019/manuscripts/peekaboo_2.pdf kitsfig. 4a  kits http://results.kits-challenge.org/miccai2019 15  2016 3du-net  15  kits \n\nfig. 4b  3d u-net\n\nnnu-net  kits  3d u-net  leaderboard nnu-net  leaderboard  leaderboard  22  (fig. 3)\n\n\n# fig.4 pipeline fingerprints from kits 2019 leaderboard entries\n\n\n\nfig4.  kits 2019 \n\na. 153d1"3du-net-like"\nb.  3d u-net  z-score ce, cross-entropy loss function; dice, soft dice loss function; wbce, weighted binary ce kits 2019 challenge  http://results.kits-challenge.org/\n\n  23  fig. 5 kitsnnu-net fig. 2\n\n\n# fig.5 data fingerprints across different challenge datasets\n\n\n\nfig5. \n\n( s.d.  z-score  nnu-net  23  (see supplementary note 1 for detailed dataset descriptions)\n\n nnu-net  nnu-net  nnu-net nnu-net  nnu-net fig. 6\n\n nnu-net \n\n\n# fig.6 evaluation of design decisions across multiple tasks\n\n\n\nfig6. \n\na-j.  10 (d1d10, see fig. 5 for dataset references)cross-entropy and topk10adamsupplementary note 8  challenger \nk. \n\n\n# discussion\n\n nnu-netnnu-net nnu-net  nnu-net nnu-net nnu-net \n\n kits fig. 4 litjens et al.  """"fig. 5 litjens et al. "" nnu-net  u-net  ct fig. 6nnu-net nnu-net nnu-net \n\n nnu-net  automl  nnu-net "black box algorithms" automl nnu-net \n\n 53  nnu-net nnu-net nnu-net  dice  cremi (https://cremi.org) nnu-net (396) em  nnu-net  nnu-net \n\nnnu-net  litjens et al.  """"nnu-net  nnu-net ',charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- A systematic evaluation of single-cell RNA-sequencing imputation methods",frontmatter:{title:" --- A systematic evaluation of single-cell RNA-sequencing imputation methods",categories:[""],tags:[""],date:"2022-11-20T10:39:56.000Z",permalink:"/pages/c7adb0/",readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20200914.(2020.09.14%20scRNAseq%20imputation%20Genome%20Biol)%20A%20systematic%20evaluation%20of%20single-cell%20RNA-sequencing%20imputation%20methods.html",relativePath:"02./14./20200914.(2020.09.14 scRNAseq imputation Genome Biol) A systematic evaluation of single-cell RNA-sequencing imputation methods.md",key:"v-65c7b89c",path:"/pages/c7adb0/",headers:[{level:2,title:"Abstract:",slug:"abstract",normalizedTitle:"abstract:",charIndex:549},{level:2,title:"Results:",slug:"results",normalizedTitle:"results:",charIndex:683},{level:3,title:"Fig. 1 Motivation and overview of benchmark evaluation of scRNA-seq imputation methods",slug:"fig-1-motivation-and-overview-of-benchmark-evaluation-of-scrna-seq-imputation-methods",normalizedTitle:"fig. 1 motivation and overview of benchmark evaluation of scrna-seq imputation methods",charIndex:1569},{level:3,title:"Fig. 2 Similarity between bulk and imputed single-cell expression data in cell lines",slug:"fig-2-similarity-between-bulk-and-imputed-single-cell-expression-data-in-cell-lines",normalizedTitle:"fig. 2 similarity between bulk and imputed single-cell expression data in cell lines",charIndex:2017},{level:3,title:"Fig. 3 Impact of imputation methods on differential expression analysis",slug:"fig-3-impact-of-imputation-methods-on-differential-expression-analysis",normalizedTitle:"fig. 3 impact of imputation methods on differential expression analysis",charIndex:2996},{level:3,title:"Fig. 4 Impact of imputation methods on unsupervised clustering analysis",slug:"fig-4-impact-of-imputation-methods-on-unsupervised-clustering-analysis",normalizedTitle:"fig. 4 impact of imputation methods on unsupervised clustering analysis",charIndex:3531},{level:3,title:"Fig. 5 Impact of imputation methods on inferred trajectories for pseudotime analysis",slug:"fig-5-impact-of-imputation-methods-on-inferred-trajectories-for-pseudotime-analysis",normalizedTitle:"fig. 5 impact of imputation methods on inferred trajectories for pseudotime analysis",charIndex:4001},{level:3,title:"Fig. 6 Overall summary of results evaluating imputation methods for scRNA-seq data",slug:"fig-6-overall-summary-of-results-evaluating-imputation-methods-for-scrna-seq-data",normalizedTitle:"fig. 6 overall summary of results evaluating imputation methods for scrna-seq data",charIndex:4594}],excerpt:'<p>: scRNAseq imputation <br>\nDOI: <a href="https://doi.org/10.1186/s13059-020-02132-x" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13059-020-02132-x<OutboundLink/></a><br>\nCite: Hou, W., Ji, Z., Ji, H. et al. A systematic evaluation of single-cell RNA-sequencing imputation methods. <em>Genome Biol</em> 21, 218 (2020).</p>\n',headersStr:"Abstract: Results: Fig. 1 Motivation and overview of benchmark evaluation of scRNA-seq imputation methods Fig. 2 Similarity between bulk and imputed single-cell expression data in cell lines Fig. 3 Impact of imputation methods on differential expression analysis Fig. 4 Impact of imputation methods on unsupervised clustering analysis Fig. 5 Impact of imputation methods on inferred trajectories for pseudotime analysis Fig. 6 Overall summary of results evaluating imputation methods for scRNA-seq data",content:": scRNAseq imputation \nDOI: https://doi.org/10.1186/s13059-020-02132-x\nCite: Hou, W., Ji, Z., Ji, H. et al. A systematic evaluation of single-cell RNA-sequencing imputation methods. Genome Biol 21, 218 (2020).\n\n\n\nscRNAseq imputation \n\n\n\nSTEPHANIE HICKS                                   HONGKAI JI\n                                                  \nJohns Hopkins Bloomberg School of Public Health   Johns Hopkins Bloomberg School of Public Health\nshicks19@jhu.edu                                  hji@jhu.edu\n\n\n# Abstract:\n\nBackground: scRNA-seq  imputation  imputation \nResults:  18  scRNA-seq imputation imputed cell profiles  bulk samples ()\nConclusions:  scRNA-seq imputation  imputation  bulk RNA-seq  imputation MAGICkNN-smoothingSAVER \n\n\n# Results:\n\n imputation \n\n 1. bayNorm, SAVER, SAVER-X, scImpute, scRecover, VIPER\n 2.   graph  neighborszero and non-zeroDrImpute, MAGIC, kNN-smoothing\n 3. AutoImpute, DCA, DeepImpute, SAUCIE, scScope, scVIALRA, mcImpute, PBLR\n\n\n# Fig. 1 Motivation and overview of benchmark evaluation of scRNA-seq imputation methods\n\n\n\nFig. 1 scRNA-seq imputation \n\na.  no imputation methodno_imp  18  imputation methods PCA library size features  counts  null simulations data imputation MAGIC kNN-smoothingSAVER  no_imp \nb.  18  scRNA-seq imputation methods \n\n\n# Fig. 2 Similarity between bulk and imputed single-cell expression data in cell lines\n\n\n\nFig. 2  bulk  imputed single-cell \n\na-d:  imputed scRNA-seq profiles  bulk RNA-seq profile 5  CellBench cell lines (sc_10x_5cl)  10x Genomics UMI-based scRNA-seq data 5  ENCODE cell lines (ENCODE_fluidigm_5cl)  Fluidigm C1 plate-based scRNA-seq read count data\na.  H1975 scran normalized log2-transformed scRNA-seq profiles (N = 440 gene)pseudobulk bulk RNA-seq profile  Spearman's correlation coefficient(SCC)\nb.  imputed  (e.g., using scVI)pseudobulk  bulk RNA-seq  SCC\nc.  imputed  single-cell profiles  bulk RNA-seq profiles  SCC\nd.  5  pseudobulk  bulk RNA-seq  SCC SAVER  SAVER-X  10x  Fluidigm scVI, DCA, MAGIC \ne-h:  LFC(log fold change) a-dSAVER  SAVER-X  10x  Fluidigm scVI, DCA, MAGIC \n\n\n# Fig. 3 Impact of imputation methods on differential expression analysis\n\n\n\nFig. 3 imputation \n\na.  bulk RNA-seq  scRNA-seq DEGsH2228  A594 \nb-d:  MAST(x-axis)  Wilcoxon rank-sum test(y-axis)  bulk RNA-seq  scRNA-seq  DEGs  overlap\ne-h:  MAST(x-axis)  Wilcoxon rank-sum test(y-axis)  DEGs  log-fold change (LFC)\ni. CD19 PBMC  ROC ImputationFail \njk:  bulk  single-cell  DEGstop 10%  bottom 10%  overlap\n\n\n# Fig. 4 Impact of imputation methods on unsupervised clustering analysis\n\n\n\nFig. 4 imputation \n\na. entropy of cluster accuracy (Hacc), entropy of cluster purity (Hpur), adjusted Rand index (ARI), and median Silhouette index\nb.  PCA  no imputation MAGIC imputation\nc. Louvain clustering (x-axis)  k-means clustering (y-axis) \ndf:  a-c  PBMC_10x_tissue dataset\n\n\n# Fig. 5 Impact of imputation methods on inferred trajectories for pseudotime analysis\n\n\n\nFig. 5 imputation \n\na.  TSCAN  RNA mixture and cell mixture datasets  Pearson correlation coefficients (PCC)TrajjectoryFail \nb.  TSCAN  overlap\nc.  Monocle 2 (x-axis)  TSCAN (y-axis)  PCC\nd.  Monocle 2 (x-axis)  TSCAN (y-axis)  overlap\ne.  HCA_10x_tissue  Monocle 2  kNN-smoothing  imputation bulk RNA-seq \nf.  Monocle 2 (x-axis)  TSCAN (y-axis) \n\n\n# Fig. 6 Overall summary of results evaluating imputation methods for scRNA-seq data\n\n\n\nFig. 6  scRNA-seq  imputation \n\n\n\n * imputed single-cell  bulk profiles  (bulk correlation\n * differential expression (differential)\n * unsupervised clustering (clustering)\n * trajectory inference (trajectory)\n * time, memory usage, and scalability\\\n\na-c: a. UMI-based and plate-based (Fluidigm) protocolsb. using UMI-based data onlyc. using Fluidigm data only\nd.  accuracy/efficiencyUMI-based/plate-based",normalizedContent:": scrnaseq imputation \ndoi: https://doi.org/10.1186/s13059-020-02132-x\ncite: hou, w., ji, z., ji, h. et al. a systematic evaluation of single-cell rna-sequencing imputation methods. genome biol 21, 218 (2020).\n\n\n\nscrnaseq imputation \n\n\n\nstephanie hicks                                   hongkai ji\n                                                  \njohns hopkins bloomberg school of public health   johns hopkins bloomberg school of public health\nshicks19@jhu.edu                                  hji@jhu.edu\n\n\n# abstract:\n\nbackground: scrna-seq  imputation  imputation \nresults:  18  scrna-seq imputation imputed cell profiles  bulk samples ()\nconclusions:  scrna-seq imputation  imputation  bulk rna-seq  imputation magicknn-smoothingsaver \n\n\n# results:\n\n imputation \n\n 1. baynorm, saver, saver-x, scimpute, screcover, viper\n 2.   graph  neighborszero and non-zerodrimpute, magic, knn-smoothing\n 3. autoimpute, dca, deepimpute, saucie, scscope, scvialra, mcimpute, pblr\n\n\n# fig. 1 motivation and overview of benchmark evaluation of scrna-seq imputation methods\n\n\n\nfig. 1 scrna-seq imputation \n\na.  no imputation methodno_imp  18  imputation methods pca library size features  counts  null simulations data imputation magic knn-smoothingsaver  no_imp \nb.  18  scrna-seq imputation methods \n\n\n# fig. 2 similarity between bulk and imputed single-cell expression data in cell lines\n\n\n\nfig. 2  bulk  imputed single-cell \n\na-d:  imputed scrna-seq profiles  bulk rna-seq profile 5  cellbench cell lines (sc_10x_5cl)  10x genomics umi-based scrna-seq data 5  encode cell lines (encode_fluidigm_5cl)  fluidigm c1 plate-based scrna-seq read count data\na.  h1975 scran normalized log2-transformed scrna-seq profiles (n = 440 gene)pseudobulk bulk rna-seq profile  spearman's correlation coefficient(scc)\nb.  imputed  (e.g., using scvi)pseudobulk  bulk rna-seq  scc\nc.  imputed  single-cell profiles  bulk rna-seq profiles  scc\nd.  5  pseudobulk  bulk rna-seq  scc saver  saver-x  10x  fluidigm scvi, dca, magic \ne-h:  lfc(log fold change) a-dsaver  saver-x  10x  fluidigm scvi, dca, magic \n\n\n# fig. 3 impact of imputation methods on differential expression analysis\n\n\n\nfig. 3 imputation \n\na.  bulk rna-seq  scrna-seq degsh2228  a594 \nb-d:  mast(x-axis)  wilcoxon rank-sum test(y-axis)  bulk rna-seq  scrna-seq  degs  overlap\ne-h:  mast(x-axis)  wilcoxon rank-sum test(y-axis)  degs  log-fold change (lfc)\ni. cd19 pbmc  roc imputationfail \njk:  bulk  single-cell  degstop 10%  bottom 10%  overlap\n\n\n# fig. 4 impact of imputation methods on unsupervised clustering analysis\n\n\n\nfig. 4 imputation \n\na. entropy of cluster accuracy (hacc), entropy of cluster purity (hpur), adjusted rand index (ari), and median silhouette index\nb.  pca  no imputation magic imputation\nc. louvain clustering (x-axis)  k-means clustering (y-axis) \ndf:  a-c  pbmc_10x_tissue dataset\n\n\n# fig. 5 impact of imputation methods on inferred trajectories for pseudotime analysis\n\n\n\nfig. 5 imputation \n\na.  tscan  rna mixture and cell mixture datasets  pearson correlation coefficients (pcc)trajjectoryfail \nb.  tscan  overlap\nc.  monocle 2 (x-axis)  tscan (y-axis)  pcc\nd.  monocle 2 (x-axis)  tscan (y-axis)  overlap\ne.  hca_10x_tissue  monocle 2  knn-smoothing  imputation bulk rna-seq \nf.  monocle 2 (x-axis)  tscan (y-axis) \n\n\n# fig. 6 overall summary of results evaluating imputation methods for scrna-seq data\n\n\n\nfig. 6  scrna-seq  imputation \n\n\n\n * imputed single-cell  bulk profiles  (bulk correlation\n * differential expression (differential)\n * unsupervised clustering (clustering)\n * trajectory inference (trajectory)\n * time, memory usage, and scalability\\\n\na-c: a. umi-based and plate-based (fluidigm) protocolsb. using umi-based data onlyc. using fluidigm data only\nd.  accuracy/efficiencyumi-based/plate-based",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- Cellposea generalist algorithm for cellular segmentation",frontmatter:{title:" --- Cellposea generalist algorithm for cellular segmentation",date:"2023-04-07T17:34:07.000Z",permalink:"/pages/1f3ea0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20201214.(2020.12.14%20Cellpose%20Nat%20Methods)%20Cellpose%20a%20generalist%20algorithm%20for%20cellular%20segmentation.html",relativePath:"02./14./20201214.(2020.12.14 Cellpose Nat Methods) Cellpose a generalist algorithm for cellular segmentation.md",key:"v-10ed743c",path:"/pages/1f3ea0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:350},{level:2,title:"Introduction",slug:"introduction",normalizedTitle:"introduction",charIndex:675},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1582},{level:3,title:"Fig.1 Model architecture",slug:"fig-1-model-architecture",normalizedTitle:"fig.1 model architecture",charIndex:2558},{level:3,title:"Fig.2 Visualization of the diverse training dataset",slug:"fig-2-visualization-of-the-diverse-training-dataset",normalizedTitle:"fig.2 visualization of the diverse training dataset",charIndex:3933},{level:3,title:"Fig.3 Example Cellpose segmentations for 36 test images",slug:"fig-3-example-cellpose-segmentations-for-36-test-images",normalizedTitle:"fig.3 example cellpose segmentations for 36 test images",charIndex:4054},{level:3,title:"Fig.4 Segmentation performance of specialist and generalist algorithms",slug:"fig-4-segmentation-performance-of-specialist-and-generalist-algorithms",normalizedTitle:"fig.4 segmentation performance of specialist and generalist algorithms",charIndex:5341},{level:3,title:"Fig.5 Model performance across image types and ROI statistics",slug:"fig-5-model-performance-across-image-types-and-roi-statistics",normalizedTitle:"fig.5 model performance across image types and roi statistics",charIndex:6517},{level:3,title:"Fig.6 Segmentation in 3D without 3D-labeled data",slug:"fig-6-segmentation-in-3d-without-3d-labeled-data",normalizedTitle:"fig.6 segmentation in 3d without 3d-labeled data",charIndex:8067},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:8528},{level:2,title:"Extended Data Figure",slug:"extended-data-figure",normalizedTitle:"extended data figure",charIndex:9456},{level:3,title:"Extended Data Fig.1 Graphical User Interface (GUI)",slug:"extended-data-fig-1-graphical-user-interface-gui",normalizedTitle:"extended data fig.1 graphical user interface (gui)",charIndex:9481},{level:3,title:"Extended Data Fig.2 Effect of network architecture and training set exclusions",slug:"extended-data-fig-2-effect-of-network-architecture-and-training-set-exclusions",normalizedTitle:"extended data fig.2 effect of network architecture and training set exclusions",charIndex:9951},{level:3,title:"Extended Data Fig.3 Prediction of median object diameter",slug:"extended-data-fig-3-prediction-of-median-object-diameter",normalizedTitle:"extended data fig.3 prediction of median object diameter",charIndex:10298},{level:3,title:"Extended Data Fig.4 Example Stardist segmentations",slug:"extended-data-fig-4-example-stardist-segmentations",normalizedTitle:"extended data fig.4 example stardist segmentations",charIndex:10392},{level:3,title:"Extended Data Fig.5 Example Mask R-CNN segmentations",slug:"extended-data-fig-5-example-mask-r-cnn-segmentations",normalizedTitle:"extended data fig.5 example mask r-cnn segmentations",charIndex:10485},{level:3,title:"Extended Data Fig.6 Other performance metrics",slug:"extended-data-fig-6-other-performance-metrics",normalizedTitle:"extended data fig.6 other performance metrics",charIndex:10582},{level:3,title:"Extended Data Fig.7 Benchmarks for dataset of nuclei",slug:"extended-data-fig-7-benchmarks-for-dataset-of-nuclei",normalizedTitle:"extended data fig.7 benchmarks for dataset of nuclei",charIndex:10662},{level:3,title:"Extended Data Fig.8 Example cellpose segmentations for nuclei",slug:"extended-data-fig-8-example-cellpose-segmentations-for-nuclei",normalizedTitle:"extended data fig.8 example cellpose segmentations for nuclei",charIndex:10754}],excerpt:'<p>: Cellpose: <br>\nDOI: <a href="https://doi.org/10.1038/s41592-020-01018-x" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-020-01018-x<OutboundLink/></a><br>\nCite: Stringer, C., Wang, T., Michaelos, M. et al. Cellpose: a generalist algorithm for cellular segmentation. <em>Nat Methods</em> 18, 100106 (2021).</p>\n',headersStr:"Abstract Introduction Results Fig.1 Model architecture Fig.2 Visualization of the diverse training dataset Fig.3 Example Cellpose segmentations for 36 test images Fig.4 Segmentation performance of specialist and generalist algorithms Fig.5 Model performance across image types and ROI statistics Fig.6 Segmentation in 3D without 3D-labeled data Discussion Extended Data Figure Extended Data Fig.1 Graphical User Interface (GUI) Extended Data Fig.2 Effect of network architecture and training set exclusions Extended Data Fig.3 Prediction of median object diameter Extended Data Fig.4 Example Stardist segmentations Extended Data Fig.5 Example Mask R-CNN segmentations Extended Data Fig.6 Other performance metrics Extended Data Fig.7 Benchmarks for dataset of nuclei Extended Data Fig.8 Example cellpose segmentations for nuclei",content:': Cellpose: \nDOI: https://doi.org/10.1038/s41592-020-01018-x\nCite: Stringer, C., Wang, T., Michaelos, M. et al. Cellpose: a generalist algorithm for cellular segmentation. Nat Methods 18, 100106 (2021).\n\n\n\nCellpose: \n\n\n\nMARIUS PACHITARIU\n\nHHMI Janelia Research Campus\npachitarium@janelia.hhmi.org\n\n\n# Abstract\n\n Cellpose Cellpose  70,000  Cellpose  Cellpose \n\n\n# Introduction\n\nRNA \n\n\n\n(Data Science Bowl) Mask R-CNN  Stardist  nucleAIzer \n\nCellpose Cellpose  3D Cellpose Extended Data Fig. 1 Python 3 Cellpose  <www.cellpose.org> \n\n\n# Results\n\nModel design. watershed algorithmbasinsFig. 1a,b\n\n ground-truth masksFig. 1aROIFig. 1c,dgradient trackingFig. 1e,f and Methods\n\n U-Net Fig. 1dU-Net  U-Net  "" Extended Data Fig. 2ad\n\nExtended Data Fig. 3ROI Methods Supplementary Table 1 \n\n\n# Fig.1 Model architecture\n\n\n\nFig1. \n\na.  masks  mask x  y  0  360 \nb. \nc. \nd.  U-Net \ne.  masks\nf.  masks\n\nTraining dataset.  """""" for example on OMERO 4,6-diamidino-2-phenylindole-DAPIn=316 imagesn=50n=58n=86n=98\n\n t-SNE Fig. 2 embedding  Python  PyQt  pyqtgraph Extended Data Fig. 1 mask 300-600  Cellpose \n\n 608 100  Cell Image Library \n\n\n# Fig.2 Visualization of the diverse training dataset\n\n\n\nFig2. \n\n t-SNE \n\n\n# Fig.3 Example Cellpose segmentations for 36 test images\n\n\n\nFig3. 36  Cellpose \n\n masks  Extended Data Figs. 4 and 5  Stardist  Mask-R-CNN \n\nBenchmarks. \'generalist\' Cellpose (Fig. 3) Cellpose \'specialized data\'Fig. 4a,bFig. 4abStardistMask R-CNN  U-NetStardist  Mask R-CNN  Extended Data Figs. 4 and 5IoU masks APTPFPFNAP=TP/(TP+FP+FN)Extended Data Fig. 6a,b Jaccard Extended Data Fig. 6c,d\n\nCellpose  IoU  0.5Cellpose  521  ROI  485  18  0.91 Stardist  Mask R-CNN  0.76  0.80 IoU Cellpose Cellpose Fig. 4d\n\nFig. 4e Cellpose Fig. 4f Cellpose  0.5  0.77 Mask R-CNN  Stardist  0.61  0.61Fig. 4gExtended Data Fig. 2e Cellpose \n\n\n# Fig.4 Segmentation performance of specialist and generalist algorithms\n\n\n\nFig4. \n\na.  \'specialist dataset\' 100 \nb. CellposeMask R-CNN  Stardist ground truth \nc. \nd. CellposeMask R-CNNStardistU-Net3  U-Net2 n=11 test imagesIoU  mask  ground-truth mask 1 0.5  masks  bc  Fig.3 \ne.  (n = 57 test images)\nfg.  de\n\nFig. 5aCellpose  Cell Image Library Fig. 3fFig. 5bFig. 5bFig. 5d\n\n Data Science Bowl kaggle see the t-SNE embedding of the segmentation styles in Extended Data Fig. 7aCellpose Extended Data Figs. 7b,c and 8\n\n\n# Fig.5 Model performance across image types and ROI statistics\n\n\n\nFig5.  ROI \n\na. n=17, 5, 9, 13, 13 test images\nb.  ROI ground truth masks  Cellpose \nc. ROI masks \nd.  IoU  0.5  ROI  (n=57 test images) IoU0.5 ROI  IoU (n=57 test images)\n\n3D segmentation.  Cellpose  3D segmentation 3D  2D  2D  30  3D  2D  2D  3D U-Nets 3D  3D  2D \n\n CellposeStardist  Mask R-CNN 2D  3D  Cellpose  3D xyxz  yz  CellposeFig. 6a,b xy z  3D  ROI 3D Fig. 6e\n\n Cellpose3D  2D U-Net  3D Fig. 6e 3D  xy 2D  2D  Stardist Mask R-CNN  ilastik  3D ilastik  Cellpose3D ilastik  3D \n\n 3D  Cellpose3D  DNA  RNA Fig. 6f183 Cellpose3D  172  IoU  0.5  17 Fig. 6g, Videos 1 and 2 IoU Cellpose3D  ilastik Cellpose3D  Cellpose  2D Fig. 6gCellpose3D U-Net  3D \n\n\n# Fig.6 Segmentation in 3D without 3D-labeled data\n\n\n\nFig6.  3D  3D \n\na. 3D  xyxz  yz \nb. \nc.  Cellpose Fig.4f,g\nd. 6 two each for x, y and z xyz xy  z  xy \ne.  3D sinks  sink 3D masks\nf.  z stack  ground-truthmasks  xy \ng. Cellpose3D  IoU \n\n\n# Discussion\n\n CellposeCellpose ROI\n\n Cellpose Cellpose Cellpose  Cellpose for example, Fig. 4e\n\n 2D  3D  Cellpose Cellpose  3D  3D Cellpose Fig. 5dCellpose \n\n\n# Extended Data Figure\n\n\n# Extended Data Fig.1 Graphical User Interface (GUI)\n\n\n\nExtended Data Fig.1 GUI\n\nGUI  Cellpose  GUI 1 Cellpose ""2 Cellpose  image channelcellpose vector flows  cellpose predicted pixel probabilities  outlinestransparent masksboth  mask \n\n\n# Extended Data Fig.2 Effect of network architecture and training set exclusions\n\n\n\nExtended Data Fig.2 \n\na-d.  4-network averagefull Cellpose architecture\na. \nb. \nc. \nd.  Unet  (in other words a, b and c combined)\ne.  Cellpose \n\n\n# Extended Data Fig.3 Prediction of median object diameter\n\n\n\nExtended Data Fig.3 \n\n\n# Extended Data Fig.4 Example Stardist segmentations\n\n\n\nExtended Data Fig.4 Stardist \n\n\n# Extended Data Fig.5 Example Mask R-CNN segmentations\n\n\n\nExtended Data Fig.5 Mask R-CNN \n\n\n# Extended Data Fig.6 Other performance metrics\n\n\n\nExtended Data Fig.6 \n\n\n# Extended Data Fig.7 Benchmarks for dataset of nuclei\n\n\n\nExtended Data Fig.7 \n\n\n# Extended Data Fig.8 Example cellpose segmentations for nuclei\n\n\n\nExtended Data Fig.8  Cellpose ',normalizedContent:': cellpose: \ndoi: https://doi.org/10.1038/s41592-020-01018-x\ncite: stringer, c., wang, t., michaelos, m. et al. cellpose: a generalist algorithm for cellular segmentation. nat methods 18, 100106 (2021).\n\n\n\ncellpose: \n\n\n\nmarius pachitariu\n\nhhmi janelia research campus\npachitarium@janelia.hhmi.org\n\n\n# abstract\n\n cellpose cellpose  70,000  cellpose  cellpose \n\n\n# introduction\n\nrna \n\n\n\n(data science bowl) mask r-cnn  stardist  nucleaizer \n\ncellpose cellpose  3d cellpose extended data fig. 1 python 3 cellpose  <www.cellpose.org> \n\n\n# results\n\nmodel design. watershed algorithmbasinsfig. 1a,b\n\n ground-truth masksfig. 1aroifig. 1c,dgradient trackingfig. 1e,f and methods\n\n u-net fig. 1du-net  u-net  "" extended data fig. 2ad\n\nextended data fig. 3roi methods supplementary table 1 \n\n\n# fig.1 model architecture\n\n\n\nfig1. \n\na.  masks  mask x  y  0  360 \nb. \nc. \nd.  u-net \ne.  masks\nf.  masks\n\ntraining dataset.  """""" for example on omero 4,6-diamidino-2-phenylindole-dapin=316 imagesn=50n=58n=86n=98\n\n t-sne fig. 2 embedding  python  pyqt  pyqtgraph extended data fig. 1 mask 300-600  cellpose \n\n 608 100  cell image library \n\n\n# fig.2 visualization of the diverse training dataset\n\n\n\nfig2. \n\n t-sne \n\n\n# fig.3 example cellpose segmentations for 36 test images\n\n\n\nfig3. 36  cellpose \n\n masks  extended data figs. 4 and 5  stardist  mask-r-cnn \n\nbenchmarks. \'generalist\' cellpose (fig. 3) cellpose \'specialized data\'fig. 4a,bfig. 4abstardistmask r-cnn  u-netstardist  mask r-cnn  extended data figs. 4 and 5iou masks aptpfpfnap=tp/(tp+fp+fn)extended data fig. 6a,b jaccard extended data fig. 6c,d\n\ncellpose  iou  0.5cellpose  521  roi  485  18  0.91 stardist  mask r-cnn  0.76  0.80 iou cellpose cellpose fig. 4d\n\nfig. 4e cellpose fig. 4f cellpose  0.5  0.77 mask r-cnn  stardist  0.61  0.61fig. 4gextended data fig. 2e cellpose \n\n\n# fig.4 segmentation performance of specialist and generalist algorithms\n\n\n\nfig4. \n\na.  \'specialist dataset\' 100 \nb. cellposemask r-cnn  stardist ground truth \nc. \nd. cellposemask r-cnnstardistu-net3  u-net2 n=11 test imagesiou  mask  ground-truth mask 1 0.5  masks  bc  fig.3 \ne.  (n = 57 test images)\nfg.  de\n\nfig. 5acellpose  cell image library fig. 3ffig. 5bfig. 5bfig. 5d\n\n data science bowl kaggle see the t-sne embedding of the segmentation styles in extended data fig. 7acellpose extended data figs. 7b,c and 8\n\n\n# fig.5 model performance across image types and roi statistics\n\n\n\nfig5.  roi \n\na. n=17, 5, 9, 13, 13 test images\nb.  roi ground truth masks  cellpose \nc. roi masks \nd.  iou  0.5  roi  (n=57 test images) iou0.5 roi  iou (n=57 test images)\n\n3d segmentation.  cellpose  3d segmentation 3d  2d  2d  30  3d  2d  2d  3d u-nets 3d  3d  2d \n\n cellposestardist  mask r-cnn 2d  3d  cellpose  3d xyxz  yz  cellposefig. 6a,b xy z  3d  roi 3d fig. 6e\n\n cellpose3d  2d u-net  3d fig. 6e 3d  xy 2d  2d  stardist mask r-cnn  ilastik  3d ilastik  cellpose3d ilastik  3d \n\n 3d  cellpose3d  dna  rna fig. 6f183 cellpose3d  172  iou  0.5  17 fig. 6g, videos 1 and 2 iou cellpose3d  ilastik cellpose3d  cellpose  2d fig. 6gcellpose3d u-net  3d \n\n\n# fig.6 segmentation in 3d without 3d-labeled data\n\n\n\nfig6.  3d  3d \n\na. 3d  xyxz  yz \nb. \nc.  cellpose fig.4f,g\nd. 6 two each for x, y and z xyz xy  z  xy \ne.  3d sinks  sink 3d masks\nf.  z stack  ground-truthmasks  xy \ng. cellpose3d  iou \n\n\n# discussion\n\n cellposecellpose roi\n\n cellpose cellpose cellpose  cellpose for example, fig. 4e\n\n 2d  3d  cellpose cellpose  3d  3d cellpose fig. 5dcellpose \n\n\n# extended data figure\n\n\n# extended data fig.1 graphical user interface (gui)\n\n\n\nextended data fig.1 gui\n\ngui  cellpose  gui 1 cellpose ""2 cellpose  image channelcellpose vector flows  cellpose predicted pixel probabilities  outlinestransparent masksboth  mask \n\n\n# extended data fig.2 effect of network architecture and training set exclusions\n\n\n\nextended data fig.2 \n\na-d.  4-network averagefull cellpose architecture\na. \nb. \nc. \nd.  unet  (in other words a, b and c combined)\ne.  cellpose \n\n\n# extended data fig.3 prediction of median object diameter\n\n\n\nextended data fig.3 \n\n\n# extended data fig.4 example stardist segmentations\n\n\n\nextended data fig.4 stardist \n\n\n# extended data fig.5 example mask r-cnn segmentations\n\n\n\nextended data fig.5 mask r-cnn \n\n\n# extended data fig.6 other performance metrics\n\n\n\nextended data fig.6 \n\n\n# extended data fig.7 benchmarks for dataset of nuclei\n\n\n\nextended data fig.7 \n\n\n# extended data fig.8 example cellpose segmentations for nuclei\n\n\n\nextended data fig.8  cellpose ',charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- Inference and analysis of cell-cell communication using CellChat",frontmatter:{title:" --- Inference and analysis of cell-cell communication using CellChat",date:"2023-05-17T15:42:57.000Z",permalink:"/pages/2737d2/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20210217.(2021.02.17%20CellChat%20Nat%20Commun)%20Inference%20and%20analysis%20of%20cell-cell%20communication%20using%20CellChat.html",relativePath:"02./14./20210217.(2021.02.17 CellChat Nat Commun) Inference and analysis of cell-cell communication using CellChat.md",key:"v-7bb51432",path:"/pages/2737d2/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:679},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1084},{level:3,title:"Fig.1 Overview of CellChat",slug:"fig-1-overview-of-cellchat",normalizedTitle:"fig.1 overview of cellchat",charIndex:1096},{level:3,title:"Fig.2 CellChat analysis of the communications between skin cells during wound repair",slug:"fig-2-cellchat-analysis-of-the-communications-between-skin-cells-during-wound-repair",normalizedTitle:"fig.2 cellchat analysis of the communications between skin cells during wound repair",charIndex:1790},{level:3,title:"Fig.3 Application of CellChat to continuous cell states along pseudotemporal trajectories during embryonic skin development",slug:"fig-3-application-of-cellchat-to-continuous-cell-states-along-pseudotemporal-trajectories-during-embryonic-skin-development",normalizedTitle:"fig.3 application of cellchat to continuous cell states along pseudotemporal trajectories during embryonic skin development",charIndex:3351},{level:3,title:"Fig.4 Application of CellChat to communications between spatially colocalized cell populations",slug:"fig-4-application-of-cellchat-to-communications-between-spatially-colocalized-cell-populations",normalizedTitle:"fig.4 application of cellchat to communications between spatially colocalized cell populations",charIndex:4156},{level:3,title:"Fig.5 Comparison analysis of epidermal-dermal communications between different skin developmental stages",slug:"fig-5-comparison-analysis-of-epidermal-dermal-communications-between-different-skin-developmental-stages",normalizedTitle:"fig.5 comparison analysis of epidermal-dermal communications between different skin developmental stages",charIndex:5184},{level:3,title:"Fig.6 Joint identification of conserved and context-specific communication patterns between two skin states",slug:"fig-6-joint-identification-of-conserved-and-context-specific-communication-patterns-between-two-skin-states",normalizedTitle:"fig.6 joint identification of conserved and context-specific communication patterns between two skin states",charIndex:6125},{level:3,title:"Fig.7 Identification of major signaling changes in diseased human skin",slug:"fig-7-identification-of-major-signaling-changes-in-diseased-human-skin",normalizedTitle:"fig.7 identification of major signaling changes in diseased human skin",charIndex:6795}],excerpt:'<p>:  CellChat -<br>\nDOI: <a href="https://doi.org/10.1038/s41467-021-21246-9" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-021-21246-9<OutboundLink/></a><br>\nCite: Jin, S., Guerrero-Juarez, C.F., Zhang, L. et al. Inference and analysis of cell-cell communication using CellChat. Nat Commun 12, 1088 (2021).</p>\n',headersStr:"Abstract Results Fig.1 Overview of CellChat Fig.2 CellChat analysis of the communications between skin cells during wound repair Fig.3 Application of CellChat to continuous cell states along pseudotemporal trajectories during embryonic skin development Fig.4 Application of CellChat to communications between spatially colocalized cell populations Fig.5 Comparison analysis of epidermal-dermal communications between different skin developmental stages Fig.6 Joint identification of conserved and context-specific communication patterns between two skin states Fig.7 Identification of major signaling changes in diseased human skin",content:":  CellChat -\nDOI: https://doi.org/10.1038/s41467-021-21246-9\nCite: Jin, S., Guerrero-Juarez, C.F., Zhang, L. et al. Inference and analysis of cell-cell communication using CellChat. Nat Commun 12, 1088 (2021).\n\n\n\n CellChat -\n\n\n\n\n\n 1.  --- Inference and analysis of cell-cell communication using CellChat\n 2. CellChatInference and analysis of cell-cell communication\n 3.  --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat\n 4. NeuronChatInferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nQING NIE\n\nUniversity of California, Irvine\nqnie@uci.edu\n\n\n# Abstract\n\n CellChat scRNA-seq CellChat CellChat  CellChat  CellChat http://www.cellchat.org/-\n\n\n# Results\n\n\n# Fig.1 Overview of CellChat\n\n\n\nFig.1 CellChat \n\na. -CellChatDB -CellChatDB  2021  L-R  60% secreting interactions 48% \nb. CellChat  meta \nc. CellChat  -law of mass action \nd. hierarchy plot  (Circle plot)bubble plot\ne. CellChat CellChat \n\n\n# Fig.2 CellChat analysis of the communications between skin cells during wound repair\n\n\n\nFig.2  CellChat \n\n scRNA-seq  CellChat  12  scRNA-seq  CellChat  21,898  25  9 FIB5 MYL 6 ENDO T TCB BCDCLYME CellChat  25  60 - 25 TGF WNTncWNTTNFSPP1PTNPDGFCXCLCCL  MIF myeloid cell  fibroblasts  TGF \n\na. Hierarchical plot  TGF  fibroblast  non-fibroblast  source  target source \nb.  TGF \nc.  TGF  Tgfb1  Tgfbr1/Tgfbr2 \nd-f.  ncWNT  TGF ncWNT Wnt5a fibroblastsFIB-D fibroblast-to-fibroblastfibroblast-to-endothelialfibroblast-to-myeloid signaling\ng.  fibroblast signaling #4 ncWNTSPP1MK  PROS myeloid cell signaling #2 TGFTNFCSFIL1  RANKL\nh.  fibroblast signaling #1#3 TGFncWNTPDGFTNFMKPTNmyeloid cell signaling #4CXCL\ni. \nj.  sources/targets \nk. \n\n\n# Fig.3 Application of CellChat to continuous cell states along pseudotemporal trajectories during embryonic skin development\n\n\n\nFig.3 CellChat \n\na.  dermal skin cells  fibroblasts DC (dermal condensate) cells \nb.  epidermal trajectory \nc. Hierarchical plot  WNT dermal  epidermal  dermal trajectory  epidermal trajectory\ne.  WNT \nf.  ncWNT  dermal  epidermal \ng. alluvial plot \nh. \n\n\n# Fig.4 Application of CellChat to communications between spatially colocalized cell populations\n\n\n\nFig.4 CellChat \n\na.  E14.5 hair follicle (HF)  placode, pre-DC, DC1 and DC2 cells DC: dermal condensate\nb. --\nc.  FGF  WNT \nd. - placode  DC  p p  one-sided permutation test \ne. \nf. \ng.  Pros1-Axl  Pros1 Axl  Thy1  scRNA-seq \nh. RNAscope n =4  E14.5  Edn3Axl Thy1Epithelial placode  dermal condensate (DC)  CellChat  Pros1-Axl DAPI\ni.  Edn3-Ednrb  DctEdn3  Ednrb  scRNA-seq DC: dermal condensate; MELA: melanocytes\nj. RNAscope n = 4  E14.5  DctEdn3 Ednrb Edn3-Ednrb \n\n\n# Fig.5 Comparison analysis of epidermal-dermal communications between different skin developmental stages\n\n\n\nFig.5 -\n\na. embryonic skin  E13.5  E14.5 FIB-A: fibroblast type A; FIB-B: fibroblast type B; FIB-P: proliferative fibroblasts. MYL: myeloid cell; ENDO: endothelial cell; MELA: melanocytes\nb.  E13.5  E14.5  E13.5  E14.5 \nc. \nd. \ne.  E13.5  WNT \nf. - E13.5  WNT \ng. E13.5  WNT \nh.  E14.5  WNT \ni. - E14.5 \nj. E14.5  WNT \nk.  E13.5 E13.5  E14.5 E14.5\nl.  E13.5  E14.5 \n\n\n# Fig.6 Joint identification of conserved and context-specific communication patterns between two skin states\n\n\n\nFig.6 \n\na.  E13.5  12 \nb.  E13.5  E13.5 \nc. \nd.  E13.5  MK \ne-f. Hierarchical plot  E13.5   MK  \ng.  E13.5  E13.5 \n\n\n# Fig.7 Identification of major signaling changes in diseased human skin\n\n\n\nFig.7 \n\na. scRNA-seq NLLS\nb.  NL  LS  NL  LS \nc.  NL  LS  NL  LS \nd.  NL  LS - from Inflam.FIB (i.e., inflammatory fibroblasts) to dendritic cells (DC) and T cells (TC) including cDC1, cDC2, LS, Inflam.DC, TC, Inflam.TC, CD40LG + TC, and NKT subpopulations CCL19-CCR7  p p  one-sided permutation test \ne. Ligand CCL19  CCR7  NL LS\nf.  CCL19-CCR7  LS CellChat  NL  CCL19-CCR7 ",normalizedContent:":  cellchat -\ndoi: https://doi.org/10.1038/s41467-021-21246-9\ncite: jin, s., guerrero-juarez, c.f., zhang, l. et al. inference and analysis of cell-cell communication using cellchat. nat commun 12, 1088 (2021).\n\n\n\n cellchat -\n\n\n\n\n\n 1.  --- inference and analysis of cell-cell communication using cellchat\n 2. cellchatinference and analysis of cell-cell communication\n 3.  --- inferring neuron-neuron communications from single-cell transcriptomics through neuronchat\n 4. neuronchatinferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nqing nie\n\nuniversity of california, irvine\nqnie@uci.edu\n\n\n# abstract\n\n cellchat scrna-seq cellchat cellchat  cellchat  cellchat http://www.cellchat.org/-\n\n\n# results\n\n\n# fig.1 overview of cellchat\n\n\n\nfig.1 cellchat \n\na. -cellchatdb -cellchatdb  2021  l-r  60% secreting interactions 48% \nb. cellchat  meta \nc. cellchat  -law of mass action \nd. hierarchy plot  (circle plot)bubble plot\ne. cellchat cellchat \n\n\n# fig.2 cellchat analysis of the communications between skin cells during wound repair\n\n\n\nfig.2  cellchat \n\n scrna-seq  cellchat  12  scrna-seq  cellchat  21,898  25  9 fib5 myl 6 endo t tcb bcdclyme cellchat  25  60 - 25 tgf wntncwnttnfspp1ptnpdgfcxclccl  mif myeloid cell  fibroblasts  tgf \n\na. hierarchical plot  tgf  fibroblast  non-fibroblast  source  target source \nb.  tgf \nc.  tgf  tgfb1  tgfbr1/tgfbr2 \nd-f.  ncwnt  tgf ncwnt wnt5a fibroblastsfib-d fibroblast-to-fibroblastfibroblast-to-endothelialfibroblast-to-myeloid signaling\ng.  fibroblast signaling #4 ncwntspp1mk  pros myeloid cell signaling #2 tgftnfcsfil1  rankl\nh.  fibroblast signaling #1#3 tgfncwntpdgftnfmkptnmyeloid cell signaling #4cxcl\ni. \nj.  sources/targets \nk. \n\n\n# fig.3 application of cellchat to continuous cell states along pseudotemporal trajectories during embryonic skin development\n\n\n\nfig.3 cellchat \n\na.  dermal skin cells  fibroblasts dc (dermal condensate) cells \nb.  epidermal trajectory \nc. hierarchical plot  wnt dermal  epidermal  dermal trajectory  epidermal trajectory\ne.  wnt \nf.  ncwnt  dermal  epidermal \ng. alluvial plot \nh. \n\n\n# fig.4 application of cellchat to communications between spatially colocalized cell populations\n\n\n\nfig.4 cellchat \n\na.  e14.5 hair follicle (hf)  placode, pre-dc, dc1 and dc2 cells dc: dermal condensate\nb. --\nc.  fgf  wnt \nd. - placode  dc  p p  one-sided permutation test \ne. \nf. \ng.  pros1-axl  pros1 axl  thy1  scrna-seq \nh. rnascope n =4  e14.5  edn3axl thy1epithelial placode  dermal condensate (dc)  cellchat  pros1-axl dapi\ni.  edn3-ednrb  dctedn3  ednrb  scrna-seq dc: dermal condensate; mela: melanocytes\nj. rnascope n = 4  e14.5  dctedn3 ednrb edn3-ednrb \n\n\n# fig.5 comparison analysis of epidermal-dermal communications between different skin developmental stages\n\n\n\nfig.5 -\n\na. embryonic skin  e13.5  e14.5 fib-a: fibroblast type a; fib-b: fibroblast type b; fib-p: proliferative fibroblasts. myl: myeloid cell; endo: endothelial cell; mela: melanocytes\nb.  e13.5  e14.5  e13.5  e14.5 \nc. \nd. \ne.  e13.5  wnt \nf. - e13.5  wnt \ng. e13.5  wnt \nh.  e14.5  wnt \ni. - e14.5 \nj. e14.5  wnt \nk.  e13.5 e13.5  e14.5 e14.5\nl.  e13.5  e14.5 \n\n\n# fig.6 joint identification of conserved and context-specific communication patterns between two skin states\n\n\n\nfig.6 \n\na.  e13.5  12 \nb.  e13.5  e13.5 \nc. \nd.  e13.5  mk \ne-f. hierarchical plot  e13.5   mk  \ng.  e13.5  e13.5 \n\n\n# fig.7 identification of major signaling changes in diseased human skin\n\n\n\nfig.7 \n\na. scrna-seq nlls\nb.  nl  ls  nl  ls \nc.  nl  ls  nl  ls \nd.  nl  ls - from inflam.fib (i.e., inflammatory fibroblasts) to dendritic cells (dc) and t cells (tc) including cdc1, cdc2, ls, inflam.dc, tc, inflam.tc, cd40lg + tc, and nkt subpopulations ccl19-ccr7  p p  one-sided permutation test \ne. ligand ccl19  ccr7  nl ls\nf.  ccl19-ccr7  ls cellchat  nl  ccl19-ccr7 ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- FAN-C:a feature-rich framework for the analysis and visualisation of chromosome conformation capture data",frontmatter:{title:" --- FAN-C:a feature-rich framework for the analysis and visualisation of chromosome conformation capture data",date:"2022-12-12T17:16:45.000Z",permalink:"/pages/1f53c5/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20201217.(2020.12.17%20FAN-C%20Genome%20Biol)%20FAN-C%20a%20feature-rich%20framework%20for%20the%20analysis%20and%20visualisation%20of%20chromosome%20conformation%20capture%20data.html",relativePath:"02./14./20201217.(2020.12.17 FAN-C Genome Biol) FAN-C a feature-rich framework for the analysis and visualisation of chromosome conformation capture data.md",key:"v-ca793f2a",path:"/pages/1f53c5/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:358},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:652},{level:3,title:"Fig1. Overview of FAN-C functionality",slug:"fig1-overview-of-fan-c-functionality",normalizedTitle:"fig1. overview of fan-c functionality",charIndex:664},{level:3,title:"Fig2. FAN-C matrix generation",slug:"fig2-fan-c-matrix-generation",normalizedTitle:"fig2. fan-c matrix generation",charIndex:885},{level:3,title:"Fig3. FAN-C analysis features",slug:"fig3-fan-c-analysis-features",normalizedTitle:"fig3. fan-c analysis features",charIndex:1290},{level:3,title:"Fig4. FAN-C comparison workflow for neuronal differentiation",slug:"fig4-fan-c-comparison-workflow-for-neuronal-differentiation",normalizedTitle:"fig4. fan-c comparison workflow for neuronal differentiation",charIndex:1357}],excerpt:'<p>: FAN-C<br>\nDOI: <a href="https://doi.org/10.1186/s13059-020-02215-9" target="_blank" rel="noopener noreferrer">https://doi.org/10.1186/s13059-020-02215-9<OutboundLink/></a><br>\nCite: Kruse K, Hug CB, Vaquerizas JM. FAN-C: a feature-rich framework for the analysis and visualisation of chromosome conformation capture data. <em>Genome Biol</em>. 2020 Dec 17;21(1):303.</p>\n',headersStr:"Abstract Results Fig1. Overview of FAN-C functionality Fig2. FAN-C matrix generation Fig3. FAN-C analysis features Fig4. FAN-C comparison workflow for neuronal differentiation",content:": FAN-C\nDOI: https://doi.org/10.1186/s13059-020-02215-9\nCite: Kruse K, Hug CB, Vaquerizas JM. FAN-C: a feature-rich framework for the analysis and visualisation of chromosome conformation capture data. Genome Biol. 2020 Dec 17;21(1):303.\n\n\n\nJUANMA VAQUERIZAS\n\nUniversidad Autnoma de Madrid\nj.vaquerizas@lms.mrc.ac.uk\n\n\n# Abstract\n\n Hi-C Hi-C  FAN-C Python API C-like  (https://github.com/vaquerizaslab/fanc) Hi-C FAN-C  Hi-C \n\n\n# Results\n\n\n# Fig1. Overview of FAN-C functionality\n\n\n\nFig1. FAN-C \n\nA. Hi-C RE: restriction enzyme\nB. Matrix generation features\nC. Hi-C matrix analysis features\nD. Hi-C visualisation features\nE. Helper tools\n\n\n# Fig2. FAN-C matrix generation\n\n\n\nFig2. FAN-C matrix generation\n\na-c. matrix generation pipeline (a) Hi-C read pairs (b) read pairs  Hi-C\ndf. FAN-C  HUVEC Hi-C d. e.  read insert sizef.  read pairs\ng.  1 kb binned  Hi-C  25%  coverage cutoff\n\n\n# Fig3. FAN-C analysis features\n\n\n\nFig3. FAN-C analysis features\n\n\n# Fig4. FAN-C comparison workflow for neuronal differentiation\n\n\n\nFig4.  FAN-C ",normalizedContent:": fan-c\ndoi: https://doi.org/10.1186/s13059-020-02215-9\ncite: kruse k, hug cb, vaquerizas jm. fan-c: a feature-rich framework for the analysis and visualisation of chromosome conformation capture data. genome biol. 2020 dec 17;21(1):303.\n\n\n\njuanma vaquerizas\n\nuniversidad autonoma de madrid\nj.vaquerizas@lms.mrc.ac.uk\n\n\n# abstract\n\n hi-c hi-c  fan-c python api c-like  (https://github.com/vaquerizaslab/fanc) hi-c fan-c  hi-c \n\n\n# results\n\n\n# fig1. overview of fan-c functionality\n\n\n\nfig1. fan-c \n\na. hi-c re: restriction enzyme\nb. matrix generation features\nc. hi-c matrix analysis features\nd. hi-c visualisation features\ne. helper tools\n\n\n# fig2. fan-c matrix generation\n\n\n\nfig2. fan-c matrix generation\n\na-c. matrix generation pipeline (a) hi-c read pairs (b) read pairs  hi-c\ndf. fan-c  huvec hi-c d. e.  read insert sizef.  read pairs\ng.  1 kb binned  hi-c  25%  coverage cutoff\n\n\n# fig3. fan-c analysis features\n\n\n\nfig3. fan-c analysis features\n\n\n# fig4. fan-c comparison workflow for neuronal differentiation\n\n\n\nfig4.  fan-c ",charsets:{cjk:!0},lastUpdated:"2023/06/16, 10:28:34",lastUpdatedTimestamp:1686882514e3},{title:" --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation",frontmatter:{title:" --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation",date:"2023-03-27T20:47:33.000Z",permalink:"/pages/1c1b2b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20210206.(2021.02.06%20%20COVID-19%20CT%20Medical%20Physics)%20Toward%20data-efficient%20learning%20A%20benchmark%20for%20COVID-19%20CT%20lung%20and%20infection%20segmentation.html",relativePath:"02./14./20210206.(2021.02.06  COVID-19 CT Medical Physics) Toward data-efficient learning A benchmark for COVID-19 CT lung and infection segmentation.md",key:"v-9366df52",path:"/pages/1c1b2b/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:657},{level:2,title:"1. INTRODUCTION",slug:"_1-introduction",normalizedTitle:"1. introduction",charIndex:1312},{level:2,title:"2. MATERIALS",slug:"_2-materials",normalizedTitle:"2. materials",charIndex:3968},{level:3,title:"2.A.  CT ",slug:"_2-a--ct-",normalizedTitle:"2.a.  ct ",charIndex:4374},{level:4,title:"2.A.1. StructSeg ",slug:"_2-a-1-structseg-",normalizedTitle:"2.a.1. structseg ",charIndex:4396},{level:4,title:"2.A.2. NSCLC ",slug:"_2-a-2-nsclc-",normalizedTitle:"2.a.2. nsclc ",charIndex:4544},{level:3,title:"2.B.  CT ",slug:"_2-b--ct-",normalizedTitle:"2.b.  ct ",charIndex:4605},{level:4,title:"2.B.1. MSD ",slug:"_2-b-1-msd-",normalizedTitle:"2.b.1. msd ",charIndex:4629},{level:4,title:"2.B.2. StructSeg ",slug:"_2-b-2-structseg-",normalizedTitle:"2.b.2. structseg ",charIndex:4791},{level:4,title:"2.B.3. NSCLC PE",slug:"_2-b-3-nsclc--pe-",normalizedTitle:"2.b.3. nsclc pe",charIndex:4890},{level:4,title:"2.B.4. MosMed dataset",slug:"_2-b-4-mosmed-dataset",normalizedTitle:"2.b.4. mosmed dataset",charIndex:4972},{level:3,title:"2.C.  COVID-19-CT-Seg ",slug:"_2-c--covid-19-ct-seg-",normalizedTitle:"2.c.  covid-19-ct-seg ",charIndex:5074},{level:2,title:"3. METHODS",slug:"_3-methods",normalizedTitle:"3. methods",charIndex:5538},{level:3,title:"3.A. Task 1: ",slug:"_3-a-task-1-",normalizedTitle:"3.a. task 1: ",charIndex:5844},{level:3,title:"3.B. Task 2:  COVID-19 CT  COVID-19 CT ",slug:"_3-b-task-2--covid-19-ct--covid-19-ct-",normalizedTitle:"3.b. task 2:  covid-19 ct  covid-19 ct ",charIndex:6168},{level:3,title:"3.C. Task 3:  COVID-19  non-COVID-19 CT ",slug:"_3-c-task-3--covid-19--non-covid-19-ct-",normalizedTitle:"3.c. task 3:  covid-19  non-covid-19 ct ",charIndex:6590},{level:3,title:"3.D. ",slug:"_3-d-",normalizedTitle:"3.d. ",charIndex:6907},{level:4,title:"3.D.1. ",slug:"_3-d-1-",normalizedTitle:"3.d.1. ",charIndex:7143},{level:3,title:"3.D.2. ",slug:"_3-d-2-",normalizedTitle:"3.d.2. ",charIndex:7164},{level:3,title:"3.E. U-Net baselines: Oldies but goldies",slug:"_3-e-u-net-baselines-oldies-but-goldies",normalizedTitle:"3.e. u-net baselines: oldies but goldies",charIndex:7185},{level:2,title:"4. RESULTS AND DISCUSSION",slug:"_4-results-and-discussion",normalizedTitle:"4. results and discussion",charIndex:8194},{level:3,title:"4.A. Results of task 1: ",slug:"_4-a-results-of-task-1-",normalizedTitle:"4.a. results of task 1: ",charIndex:8330},{level:3,title:"4.B. Results of task 2:  COVID- 19 CT  non-COVID-19 CT ",slug:"_4-b-results-of-task-2--covid-19-ct--non-covid-19-ct-",normalizedTitle:"4.b. results of task 2:  covid- 19 ct  non-covid-19 ct ",charIndex:9135},{level:3,title:"4.C. Results of task 3:  COVID-19  non-COVID-19CT ",slug:"_4-c-results-of-task-3--covid-19--non-covid-19ct-",normalizedTitle:"4.c. results of task 3:  covid-19  non-covid-19ct ",charIndex:9844},{level:3,title:"4.D. ",slug:"_4-d-",normalizedTitle:"4.d. ",charIndex:11e3},{level:3,title:"4.E. Limitation",slug:"_4-e-limitation",normalizedTitle:"4.e. limitation",charIndex:13677},{level:2,title:"5. CONCLUSION",slug:"_5-conclusion",normalizedTitle:"5. conclusion",charIndex:14057}],excerpt:'<p>: COVID-19 CT <br>\nDOI: <a href="https://doi.org/10.1002/mp.14676" target="_blank" rel="noopener noreferrer">https://doi.org/10.1002/mp.14676<OutboundLink/></a><br>\nCite: Ma, J., Wang, Y., An, X., et al. (2021). Toward dataefficient learning: A benchmark for COVID19 CT lung and infection segmentation. Medical Physics, 48(3), 11971210.</p>\n',headersStr:"Abstract 1. INTRODUCTION 2. MATERIALS 2.A.  CT  2.A.1. StructSeg  2.A.2. NSCLC  2.B.  CT  2.B.1. MSD  2.B.2. StructSeg  2.B.3. NSCLC PE 2.B.4. MosMed dataset 2.C.  COVID-19-CT-Seg  3. METHODS 3.A. Task 1:  3.B. Task 2:  COVID-19 CT  COVID-19 CT  3.C. Task 3:  COVID-19  non-COVID-19 CT  3.D.  3.D.1.  3.D.2.  3.E. U-Net baselines: Oldies but goldies 4. RESULTS AND DISCUSSION 4.A. Results of task 1:  4.B. Results of task 2:  COVID- 19 CT  non-COVID-19 CT  4.C. Results of task 3:  COVID-19  non-COVID-19CT  4.D.  4.E. Limitation 5. CONCLUSION",content:": COVID-19 CT \nDOI: https://doi.org/10.1002/mp.14676\nCite: Ma, J., Wang, Y., An, X., et al. (2021). Toward dataefficient learning: A benchmark for COVID19 CT lung and infection segmentation. Medical Physics, 48(3), 11971210.\n\n\n\nCOVID-19 CT \n\n\n\n\n\n 1.  --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation\n 3. nnUNeta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nXIAOPING YANG()\n\nxpyang@nju.edu.cn\n\n\n# Abstract\n\n  COVID-19 CTlunginfection COVID-19 CT \n  70  COVID-19 \n  40  COVID-19  dice DSC 97.397.7  67.3 diceNSD 90.691.4  70.0\n  COVID-19 CT \n\n\n# 1. INTRODUCTION\n\nCoronavirus disease 2019COVID-19 2020  11  30  6100 CT COVID-19 CT -RT-PCR COVID-19 Wang et al.  325  COVID-19 CT  740  RT-PCR  46  COVID-19 CT  COVID-19 \n\nAI COVID-19AI  COVID-19  COVID-19 \n\n CT  COVID-19 \n\n U-Netor V-Net U-Net  83.1  91.6  Dice  Shan et al.  V-Net  human-in-the-loop  36114  249  CT  85.191.0 91.6  diceHuang et al.  774  U-Net  Shan et al.  3D CT \n\n COVID-19 CT \n\n\n\n *  COVID-19 CT 3D  COVID-19  CT \n *  MedSeghttp://medicalsegmentation.com/covid19/ 100  COVID-19 \n *  COVID-19 U-Net \n\n COVID-19 CT  COVID-19 3D CT  U-Net  baselines\n\n\n\n * Few-shot learning\n * Domain generalization\n * Knowledge transfer////\n\n\n\n *  COVID-19 CT \n *  COVID-19 CT \n *  40  COVID-19 CT \n\n\n# 2. MATERIALS\n\nCOVID-19 CT  CT  COVID-19 CT Figure 1 \n\n\n\nFIG. 1. CT\n\n CT  ground truth ground truth MSD Lung Tumor  MosMed [Color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 2.A.  CT \n\n# 2.A.1. StructSeg \n\n 50  CT  2019  MICCAI \n\n# 2.A.2. NSCLC \n\n 402  CT \n\n\n# 2.B.  CT \n\n# 2.B.1. MSD \n\n (Palo Alto, CA, USA) NSCLC TCIA  2018  MICCAI  63  CT \n\n# 2.B.2. StructSeg \n\n StructSeg  50  CT \n\n# 2.B.3. NSCLC PE\n\n CT  NSCLC  78 \n\n# 2.B.4. MosMed dataset\n\n 50  COVID-19 CT \n\n\n# 2.C.  COVID-19-CT-Seg \n\n Coronacases Initiative  Radiopaedia  20  COVID-19 CT  CC BY-NC-SA  COVID-19  0.01  59  1-5  5-10  10  ITK-SNAP  250  CT  400 x 45  300  1800  https://zenodo.org/record/3757476  CC BY-NC-SA \n\n\n# 3. METHODS\n\n 1  COVID-CT  COVID-19 CT \n\n *  COVID-19 CT \n *  COVID-19  CT \n *  COVID-19  COVID-19 CT \n\n\n\n\n# 3.A. Task 1: \n\nTask 1 (Table I)  COVID-19-CT-Seg  fivefold cross-validation  fold  4  16 MosMed \n\n\n\nTABLE I. COVID-19 CT  Task 1  COVID-19-CT-Seg \n\n\n# 3.B. Task 2:  COVID-19 CT  COVID-19 CT \n\nTask 2 (Table II)  COVID-19  StructSeg Lung  NSCLC Lung  MSD Lung TumorStructSeg Gross Target  NSCLC Pleural Effusion  80%  20%  COVID-19 CT \n\n\n\nTABLE II. Task 2  non-COVID-19 CT  COVID-19 CT  COVID-19 CT \n\n\n# 3.C. Task 3:  COVID-19  non-COVID-19 CT \n\nTask 3 (Table III)  80%  non-COVID-19  20%  COVID-19  20%  80% MosMed \n\n\n\nTABLE III. Task 3 COVID-19  non-COVID-19 CT  COVID-19 CT \n\n\n# 3.D. \n\nDice DSC DiceNSD   ground truth 100%  GS  ground truth \n\n# 3.D.1. \n\n\n\n\n# 3.D.2. \n\n\n\n\n# 3.E. U-Net baselines: Oldies but goldies\n\nU-Net  5  U-NetnnU-Netno-new-U-Net 3D nnU-Net  19  49  https://github.com/MIC-DKFZ/nnUNet \n\n COVID-19 CT U-Net  U-Net  nnU-Net  U-Net  Task 2 Task 3  Task 1 Figure 2  U-Net \n\n Z-ScorennU-Net  Dice 0.01 Nesterov 0.99PolyLR 19219264  1000  epoch epoch  250 batch size 2\n\n\n\nFIG. 2.  3D U-Net \n\n325616092122[Color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 4. RESULTS AND DISCUSSION\n\nMosMed\n\n\n# 4.A. Results of task 1: \n\nTable IV  Task 1  DSC  NSD \n\n *  folds  DSC  NSD  fold  fivefold cross-validation  onefold \n *  COVID-19 CT \n * \n\n\n\nTABLE IV.  Task 1  COVID-19-CT-Seg  fivefold cross validation  fold DSC  NSD  80 =5 folds  16 testing cases per fold\n\nFigure 3  Task 1 \n\n\n\nFIG. 3. Task 1 \n\nTask1-Separate Task1-Union [Color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 4.B. Results of task 2:  COVID- 19 CT  non-COVID-19 CT \n\nCOVID-19 CTTable V  DSC  NSD \n\n * 3D U-Net  DSC  NSD  DSC \n *  NSCLC Lung  StuctSeg lung  StructSeg  COVID-19-CT \n\n\n\nTABLE V. Task 2 \n\nTable VI  DSC  NSD \n\n * Table V MICCAI \n *  COVID-19  CT  COVID-19 \n\n\n\nTABLE VI. Task 2 \n\n\n# 4.C. Results of task 3:  COVID-19  non-COVID-19CT \n\n Task 3  COVID-19  non-COVID-19  COVID-19 CT \n\nTable VII  fivefold cross-validation  DSC  NSD \n\n *  DSC  NSD  Task 2  96.4% DSC 97.2%  DSC\n *  DSC  NSD  CT non-COVID-19  COVID-19 CT \n\n\n\nTABLE VII. Task 3  fivefold cross validation \n\nTable VIII  fivefold cross validation \n\n *  non-COVID-19 fivefold cross validation  fivefold cross validation \n *  Task 2 Table VI COVID-19  StructSeg DSC  7.5%NSCLC  DSC  1.1% MSD  3.3%\n *  4%-14%  DSC  11%-16%  NSD COVID-19  non-COVID-19  COVID-19 CT  non-COVID-19  COVID-19 \n\n\n\nTABLE VIII. Task 3  fivefold cross validation \n\n\n# 4.D. \n\nTasks 1-3  COVID-19 CT Tasks 1-3 Table IX  DSC  NSD Task 1-Separate  -Union Figure 4  DSC  NSD  COVID-19-CT-Seg \n\n\n\n * Task 3  97.3 DSC  90.6 NSD  97.7 DSC  91.4 NSD\n *  Task 1  Task 2 StructSeg  StructSeg  COVID-19 CT  300  NSCLC  COVID-19 \n *  Task 1  Task 3 StructSeg  NSCLC non-COVID CT  COVID-19 CT  COVID-19  COVID-19 CT \n *  Task 2  Task 3  COVID-19 StructSeg  NSCLC NSCLC DSC  34 COVID-19 \n\n\n\n * Task 1  DSC  67.3  NSD  70.0\n *  Task 1  Task 2  COVID-19 CT  COVID-19 \n *  Task 1  Task 3 40-62non-COVID-19  COVID-19 \n *  Task 2  Task 3  COVID-19  COVID-19 \n\n\n\nTABLE IX. COVID-19 CT  DSC  NSD \n\n\n\nFIG. 4.  COVID-19-CT-Seg DSC  NSD[Color figure can be viewed at wileyonlinelibrary.com]\n\nTable X Figure 5 \n\n\n\nTABLE X.  COVID-19 CT \n\n\n\nFIG. 5.  [Color figure can be viewed at wileyonlinelibrary.com]\n\n non-COVID-19  CT  3D COVID-19 CT \n\nTask 1Task 2 non-COVID-19 Task 3 non-COVID-19  COVID-19  SOTA  non-COVID-19 \n\n\n# 4.E. Limitation\n\nCOVID-19-CT-Seg1620MosMed50 MICCAI 2020 StructSeg 2020 10  ASOCA 20 \n\n\n# 5. CONCLUSION\n\n COVID-19  COVID-19  COVID-19 CT  40  https://github.com/JunMa11/COVID-19-CT-Seg-Benchmark  COVID-19 ",normalizedContent:": covid-19 ct \ndoi: https://doi.org/10.1002/mp.14676\ncite: ma, j., wang, y., an, x., et al. (2021). toward dataefficient learning: a benchmark for covid19 ct lung and infection segmentation. medical physics, 48(3), 11971210.\n\n\n\ncovid-19 ct \n\n\n\n\n\n 1.  --- nnu-neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- toward data-efficient learninga benchmark for covid-19 ct lung and infection segmentation\n 3. nnuneta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nxiaoping yang()\n\nxpyang@nju.edu.cn\n\n\n# abstract\n\n  covid-19 ctlunginfection covid-19 ct \n  70  covid-19 \n  40  covid-19  dice dsc 97.397.7  67.3 dicensd 90.691.4  70.0\n  covid-19 ct \n\n\n# 1. introduction\n\ncoronavirus disease 2019covid-19 2020  11  30  6100 ct covid-19 ct -rt-pcr covid-19 wang et al.  325  covid-19 ct  740  rt-pcr  46  covid-19 ct  covid-19 \n\nai covid-19ai  covid-19  covid-19 \n\n ct  covid-19 \n\n u-netor v-net u-net  83.1  91.6  dice  shan et al.  v-net  human-in-the-loop  36114  249  ct  85.191.0 91.6  dicehuang et al.  774  u-net  shan et al.  3d ct \n\n covid-19 ct \n\n\n\n *  covid-19 ct 3d  covid-19  ct \n *  medseghttp://medicalsegmentation.com/covid19/ 100  covid-19 \n *  covid-19 u-net \n\n covid-19 ct  covid-19 3d ct  u-net  baselines\n\n\n\n * few-shot learning\n * domain generalization\n * knowledge transfer////\n\n\n\n *  covid-19 ct \n *  covid-19 ct \n *  40  covid-19 ct \n\n\n# 2. materials\n\ncovid-19 ct  ct  covid-19 ct figure 1 \n\n\n\nfig. 1. ct\n\n ct  ground truth ground truth msd lung tumor  mosmed [color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 2.a.  ct \n\n# 2.a.1. structseg \n\n 50  ct  2019  miccai \n\n# 2.a.2. nsclc \n\n 402  ct \n\n\n# 2.b.  ct \n\n# 2.b.1. msd \n\n (palo alto, ca, usa) nsclc tcia  2018  miccai  63  ct \n\n# 2.b.2. structseg \n\n structseg  50  ct \n\n# 2.b.3. nsclc pe\n\n ct  nsclc  78 \n\n# 2.b.4. mosmed dataset\n\n 50  covid-19 ct \n\n\n# 2.c.  covid-19-ct-seg \n\n coronacases initiative  radiopaedia  20  covid-19 ct  cc by-nc-sa  covid-19  0.01  59  1-5  5-10  10  itk-snap  250  ct  400 x 45  300  1800  https://zenodo.org/record/3757476  cc by-nc-sa \n\n\n# 3. methods\n\n 1  covid-ct  covid-19 ct \n\n *  covid-19 ct \n *  covid-19  ct \n *  covid-19  covid-19 ct \n\n\n\n\n# 3.a. task 1: \n\ntask 1 (table i)  covid-19-ct-seg  fivefold cross-validation  fold  4  16 mosmed \n\n\n\ntable i. covid-19 ct  task 1  covid-19-ct-seg \n\n\n# 3.b. task 2:  covid-19 ct  covid-19 ct \n\ntask 2 (table ii)  covid-19  structseg lung  nsclc lung  msd lung tumorstructseg gross target  nsclc pleural effusion  80%  20%  covid-19 ct \n\n\n\ntable ii. task 2  non-covid-19 ct  covid-19 ct  covid-19 ct \n\n\n# 3.c. task 3:  covid-19  non-covid-19 ct \n\ntask 3 (table iii)  80%  non-covid-19  20%  covid-19  20%  80% mosmed \n\n\n\ntable iii. task 3 covid-19  non-covid-19 ct  covid-19 ct \n\n\n# 3.d. \n\ndice dsc dicensd   ground truth 100%  gs  ground truth \n\n# 3.d.1. \n\n\n\n\n# 3.d.2. \n\n\n\n\n# 3.e. u-net baselines: oldies but goldies\n\nu-net  5  u-netnnu-netno-new-u-net 3d nnu-net  19  49  https://github.com/mic-dkfz/nnunet \n\n covid-19 ct u-net  u-net  nnu-net  u-net  task 2 task 3  task 1 figure 2  u-net \n\n z-scorennu-net  dice 0.01 nesterov 0.99polylr 19219264  1000  epoch epoch  250 batch size 2\n\n\n\nfig. 2.  3d u-net \n\n325616092122[color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 4. results and discussion\n\nmosmed\n\n\n# 4.a. results of task 1: \n\ntable iv  task 1  dsc  nsd \n\n *  folds  dsc  nsd  fold  fivefold cross-validation  onefold \n *  covid-19 ct \n * \n\n\n\ntable iv.  task 1  covid-19-ct-seg  fivefold cross validation  fold dsc  nsd  80 =5 folds  16 testing cases per fold\n\nfigure 3  task 1 \n\n\n\nfig. 3. task 1 \n\ntask1-separate task1-union [color figure can be viewed at wileyonlinelibrary.com]\n\n\n# 4.b. results of task 2:  covid- 19 ct  non-covid-19 ct \n\ncovid-19 cttable v  dsc  nsd \n\n * 3d u-net  dsc  nsd  dsc \n *  nsclc lung  stuctseg lung  structseg  covid-19-ct \n\n\n\ntable v. task 2 \n\ntable vi  dsc  nsd \n\n * table v miccai \n *  covid-19  ct  covid-19 \n\n\n\ntable vi. task 2 \n\n\n# 4.c. results of task 3:  covid-19  non-covid-19ct \n\n task 3  covid-19  non-covid-19  covid-19 ct \n\ntable vii  fivefold cross-validation  dsc  nsd \n\n *  dsc  nsd  task 2  96.4% dsc 97.2%  dsc\n *  dsc  nsd  ct non-covid-19  covid-19 ct \n\n\n\ntable vii. task 3  fivefold cross validation \n\ntable viii  fivefold cross validation \n\n *  non-covid-19 fivefold cross validation  fivefold cross validation \n *  task 2 table vi covid-19  structseg dsc  7.5%nsclc  dsc  1.1% msd  3.3%\n *  4%-14%  dsc  11%-16%  nsd covid-19  non-covid-19  covid-19 ct  non-covid-19  covid-19 \n\n\n\ntable viii. task 3  fivefold cross validation \n\n\n# 4.d. \n\ntasks 1-3  covid-19 ct tasks 1-3 table ix  dsc  nsd task 1-separate  -union figure 4  dsc  nsd  covid-19-ct-seg \n\n\n\n * task 3  97.3 dsc  90.6 nsd  97.7 dsc  91.4 nsd\n *  task 1  task 2 structseg  structseg  covid-19 ct  300  nsclc  covid-19 \n *  task 1  task 3 structseg  nsclc non-covid ct  covid-19 ct  covid-19  covid-19 ct \n *  task 2  task 3  covid-19 structseg  nsclc nsclc dsc  34 covid-19 \n\n\n\n * task 1  dsc  67.3  nsd  70.0\n *  task 1  task 2  covid-19 ct  covid-19 \n *  task 1  task 3 40-62non-covid-19  covid-19 \n *  task 2  task 3  covid-19  covid-19 \n\n\n\ntable ix. covid-19 ct  dsc  nsd \n\n\n\nfig. 4.  covid-19-ct-seg dsc  nsd[color figure can be viewed at wileyonlinelibrary.com]\n\ntable x figure 5 \n\n\n\ntable x.  covid-19 ct \n\n\n\nfig. 5.  [color figure can be viewed at wileyonlinelibrary.com]\n\n non-covid-19  ct  3d covid-19 ct \n\ntask 1task 2 non-covid-19 task 3 non-covid-19  covid-19  sota  non-covid-19 \n\n\n# 4.e. limitation\n\ncovid-19-ct-seg1620mosmed50 miccai 2020 structseg 2020 10  asoca 20 \n\n\n# 5. conclusion\n\n covid-19  covid-19  covid-19 ct  40  https://github.com/junma11/covid-19-ct-seg-benchmark  covid-19 ",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:" --- Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease",frontmatter:{title:" --- Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease",date:"2023-03-23T21:03:37.000Z",permalink:"/pages/6a4bcc/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20210701.(2021.07.01%20%20Liu%20et%20al.%20Front.%20Genet)%20Discovery%20and%20Validation%20of%20Key%20Biomarkers%20Based%20on%20Immune%20Infiltrates%20in%20Alzheimer's%20Disease.html",relativePath:"02./14./20210701.(2021.07.01  Liu et al. Front. Genet) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease.md",key:"v-78e6cd76",path:"/pages/6a4bcc/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:282},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:642},{level:3,title:"FIGURE 1. Selection of diagnostic biomarkers and identification of hub genes",slug:"figure-1-selection-of-diagnostic-biomarkers-and-identification-of-hub-genes",normalizedTitle:"figure 1. selection of diagnostic biomarkers and identification of hub genes",charIndex:1133},{level:3,title:"FIGURE 2. The landscape of immune infiltration between AD and normal controls",slug:"figure-2-the-landscape-of-immune-infiltration-between-ad-and-normal-controls",normalizedTitle:"figure 2. the landscape of immune infiltration between ad and normal controls",charIndex:1522},{level:3,title:"FIGURE 3. The association between the hub genes and immune cell infiltration",slug:"figure-3-the-association-between-the-hub-genes-and-immune-cell-infiltration",normalizedTitle:"figure 3. the association between the hub genes and immune cell infiltration",charIndex:1781},{level:3,title:"FIGURE 4. Enrichment analysis of pathway and gene ontology (GO) involved hub genes",slug:"figure-4-enrichment-analysis-of-pathway-and-gene-ontology-go-involved-hub-genes",normalizedTitle:"figure 4. enrichment analysis of pathway and gene ontology (go) involved hub genes",charIndex:1984},{level:3,title:"FIGURE 5. Molecular regulatory mechanism of core gene-related pathway and GO functional enrichment analyses",slug:"figure-5-molecular-regulatory-mechanism-of-core-gene-related-pathway-and-go-functional-enrichment-analyses",normalizedTitle:"figure 5. molecular regulatory mechanism of core gene-related pathway and go functional enrichment analyses",charIndex:2167},{level:3,title:"FIGURE 6. Closeness score of semantic similarities between GO terms and coexpression analysis of hub genes",slug:"figure-6-closeness-score-of-semantic-similarities-between-go-terms-and-coexpression-analysis-of-hub-genes",normalizedTitle:"figure 6. closeness score of semantic similarities between go terms and coexpression analysis of hub genes",charIndex:2400},{level:3,title:"FIGURE 7. ROC curves for evaluating the accuracy of logistic regression analysis of training set [(A), GSE63061] and validation [(B), GSE85426]",slug:"figure-7-roc-curves-for-evaluating-the-accuracy-of-logistic-regression-analysis-of-training-set-a-gse63061-and-validation-b-gse85426",normalizedTitle:"figure 7. roc curves for evaluating the accuracy of logistic regression analysis of training set [(a), gse63061] and validation [(b), gse85426]",charIndex:2651},{level:3,title:"FIGURE 8. RT-PCR validation of the hub gene between AD and normal controls",slug:"figure-8-rt-pcr-validation-of-the-hub-gene-between-ad-and-normal-controls",normalizedTitle:"figure 8. rt-pcr validation of the hub gene between ad and normal controls",charIndex:2871},{level:3,title:"FIGURE 9. The workflow of analysis",slug:"figure-9-the-workflow-of-analysis",normalizedTitle:"figure 9. the workflow of analysis",charIndex:3070}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.3389/fgene.2021.658323" target="_blank" rel="noopener noreferrer">https://doi.org/10.3389/fgene.2021.658323<OutboundLink/></a><br>\nCite: Liu Z, Li H and Pan S (2021) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer\'s Disease. Front. Genet. 12:658323.</p>\n',headersStr:"Abstract Results FIGURE 1. Selection of diagnostic biomarkers and identification of hub genes FIGURE 2. The landscape of immune infiltration between AD and normal controls FIGURE 3. The association between the hub genes and immune cell infiltration FIGURE 4. Enrichment analysis of pathway and gene ontology (GO) involved hub genes FIGURE 5. Molecular regulatory mechanism of core gene-related pathway and GO functional enrichment analyses FIGURE 6. Closeness score of semantic similarities between GO terms and coexpression analysis of hub genes FIGURE 7. ROC curves for evaluating the accuracy of logistic regression analysis of training set [(A), GSE63061] and validation [(B), GSE85426] FIGURE 8. RT-PCR validation of the hub gene between AD and normal controls FIGURE 9. The workflow of analysis",content:": \nDOI: https://doi.org/10.3389/fgene.2021.658323\nCite: Liu Z, Li H and Pan S (2021) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease. Front. Genet. 12:658323.\n\n\n\n\n\n\n# Abstract\n\nBackground: ADAD  AD \n\nMethods:  NCBI GEO  139  AD  134  CIBERSORT  AD  22  Least Absolute Shrinkage Selection Operator (LASSO)  SVM-RFE  GO  hub genes\n\nResults: AD  monocytesM0 macrophagesdendritic cells  NK cell restingT-cell CD4 naiveT-cell CD4 memory activationeosinophilsABCA2CREBRFCD72CETN2KCNG1NDUFA2RPL36AL hub gene hub gene GSEA GO ROC  AUC  0.845  0.839ABCA2NDUFA2CREBRF  CD72  mRNA  hub gene  RT-PCR \n\nConclusion:  AD  AD \n\n\n# Results\n\n\n# FIGURE 1. Selection of diagnostic biomarkers and identification of hub genes\n\n\n\nFIGURE 1. Selection of diagnostic biomarkers and identification of hub genes\n\n(A)  11  LASSO \n(B)  misclassification \n(C)  LASSO  SVM-RFE  Venn LASSO, least absolute shrinkage and selection operator; SVM, support vector machine; RFE, recursive feature elimination\n\n\n# FIGURE 2. The landscape of immune infiltration between AD and normal controls\n\n\n\nFIGURE 2. AD \n\n(A-B)  AD  AD \n(C)  CIBERSORT \n(D) AD (red)  normal controls (blue) \n\n\n# FIGURE 3. The association between the hub genes and immune cell infiltration\n\n\n\nFIGURE 3. \n\n(A-G) \n(H-K) \n(L) -\n\n\n# FIGURE 4. Enrichment analysis of pathway and gene ontology (GO) involved hub genes\n\n\n\nFIGURE 4.  GO \n\n(A-G) ABCA2CREBRFCD72CETN2KCNG1NDUFA2RPL36AL \n\n\n# FIGURE 5. Molecular regulatory mechanism of core gene-related pathway and GO functional enrichment analyses\n\n\n\nFIGURE 5.  GO \n\n(A-G) ABCA2CREBRFCD72CETN2KCNG1NDUFA2RPL36AL  GSEA-related ccgraph plot\n\n\n# FIGURE 6. Closeness score of semantic similarities between GO terms and coexpression analysis of hub genes\n\n\n\nFIGURE 6. GO terms \n\n(A) GO \n(B) GO terms  Raincloud plot\n(C) circos diagram  hub genes \n\n\n# FIGURE 7. ROC curves for evaluating the accuracy of logistic regression analysis of training set [(A), GSE63061] and validation [(B), GSE85426]\n\n\n\nFIGURE 7. [(A), GSE63061] [(B), GSE85426]  ROC \n\n\n# FIGURE 8. RT-PCR validation of the hub gene between AD and normal controls\n\n\n\nFIGURE 8. AD  RT-PCR \n\n(A-G) 3 mean  SEM *p < 0.05**p < 0.01ns, no significance\n\n\n# FIGURE 9. The workflow of analysis\n\n\n\nFIGURE 9. ",normalizedContent:": \ndoi: https://doi.org/10.3389/fgene.2021.658323\ncite: liu z, li h and pan s (2021) discovery and validation of key biomarkers based on immune infiltrates in alzheimer's disease. front. genet. 12:658323.\n\n\n\n\n\n\n# abstract\n\nbackground: adad  ad \n\nmethods:  ncbi geo  139  ad  134  cibersort  ad  22  least absolute shrinkage selection operator (lasso)  svm-rfe  go  hub genes\n\nresults: ad  monocytesm0 macrophagesdendritic cells  nk cell restingt-cell cd4 naivet-cell cd4 memory activationeosinophilsabca2crebrfcd72cetn2kcng1ndufa2rpl36al hub gene hub gene gsea go roc  auc  0.845  0.839abca2ndufa2crebrf  cd72  mrna  hub gene  rt-pcr \n\nconclusion:  ad  ad \n\n\n# results\n\n\n# figure 1. selection of diagnostic biomarkers and identification of hub genes\n\n\n\nfigure 1. selection of diagnostic biomarkers and identification of hub genes\n\n(a)  11  lasso \n(b)  misclassification \n(c)  lasso  svm-rfe  venn lasso, least absolute shrinkage and selection operator; svm, support vector machine; rfe, recursive feature elimination\n\n\n# figure 2. the landscape of immune infiltration between ad and normal controls\n\n\n\nfigure 2. ad \n\n(a-b)  ad  ad \n(c)  cibersort \n(d) ad (red)  normal controls (blue) \n\n\n# figure 3. the association between the hub genes and immune cell infiltration\n\n\n\nfigure 3. \n\n(a-g) \n(h-k) \n(l) -\n\n\n# figure 4. enrichment analysis of pathway and gene ontology (go) involved hub genes\n\n\n\nfigure 4.  go \n\n(a-g) abca2crebrfcd72cetn2kcng1ndufa2rpl36al \n\n\n# figure 5. molecular regulatory mechanism of core gene-related pathway and go functional enrichment analyses\n\n\n\nfigure 5.  go \n\n(a-g) abca2crebrfcd72cetn2kcng1ndufa2rpl36al  gsea-related ccgraph plot\n\n\n# figure 6. closeness score of semantic similarities between go terms and coexpression analysis of hub genes\n\n\n\nfigure 6. go terms \n\n(a) go \n(b) go terms  raincloud plot\n(c) circos diagram  hub genes \n\n\n# figure 7. roc curves for evaluating the accuracy of logistic regression analysis of training set [(a), gse63061] and validation [(b), gse85426]\n\n\n\nfigure 7. [(a), gse63061] [(b), gse85426]  roc \n\n\n# figure 8. rt-pcr validation of the hub gene between ad and normal controls\n\n\n\nfigure 8. ad  rt-pcr \n\n(a-g) 3 mean  sem *p < 0.05**p < 0.01ns, no significance\n\n\n# figure 9. the workflow of analysis\n\n\n\nfigure 9. ",charsets:{cjk:!0},lastUpdated:"2023/09/04, 10:33:44",lastUpdatedTimestamp:1693794824e3},{title:" --- Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram",frontmatter:{title:" --- Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram",date:"2023-09-18T20:45:57.000Z",permalink:"/pages/c3bbd9/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20211028.(2021.10.28%20Tangram%20Nat%20Methods)%20Deep%20learning%20and%20alignment%20of%20spatially%20resolved%20single-cell%20transcriptomes%20with%20Tangram.html",relativePath:"02./14./20211028.(2021.10.28 Tangram Nat Methods) Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram.md",key:"v-c46c1bf8",path:"/pages/c3bbd9/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:825},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1238},{level:3,title:"Fig. 1 Tangram learns spatial transcriptome-wide patterns at single-cell resolution from sc/snRNA-seq data and corresponding spatial data",slug:"fig-1-tangram-learns-spatial-transcriptome-wide-patterns-at-single-cell-resolution-from-sc-snrna-seq-data-and-corresponding-spatial-data",normalizedTitle:"fig. 1 tangram learns spatial transcriptome-wide patterns at single-cell resolution from sc/snrna-seq data and corresponding spatial data",charIndex:1250},{level:3,title:"Fig. 2 Tangram maps cells with high-resolution MERFISH measurements and expands them to genome scale",slug:"fig-2-tangram-maps-cells-with-high-resolution-merfish-measurements-and-expands-them-to-genome-scale",normalizedTitle:"fig. 2 tangram maps cells with high-resolution merfish measurements and expands them to genome scale",charIndex:1951},{level:3,title:"Fig. 3 Correction of low-quality genes by mapping snRNA-seq on STARmap data",slug:"fig-3-correction-of-low-quality-genes-by-mapping-snrna-seq-on-starmap-data",normalizedTitle:"fig. 3 correction of low-quality genes by mapping snrna-seq on starmap data",charIndex:2576},{level:3,title:"Fig. 4 Mapping snRNA-seq data to Spatial Transcriptomics data (Visium) demonstrates deconvolution and imputation of dropouts",slug:"fig-4-mapping-snrna-seq-data-to-spatial-transcriptomics-data-visium-demonstrates-deconvolution-and-imputation-of-dropouts",normalizedTitle:"fig. 4 mapping snrna-seq data to spatial transcriptomics data (visium) demonstrates deconvolution and imputation of dropouts",charIndex:3079},{level:3,title:"Fig. 5 Tangram mapping of multi-omic SHARE-seq profiles yields spatial patterns of chromatin accessibility and transcription factor activity",slug:"fig-5-tangram-mapping-of-multi-omic-share-seq-profiles-yields-spatial-patterns-of-chromatin-accessibility-and-transcription-factor-activity",normalizedTitle:"fig. 5 tangram mapping of multi-omic share-seq profiles yields spatial patterns of chromatin accessibility and transcription factor activity",charIndex:3899},{level:3,title:"Fig. 6 Tangram mapping of snRNA-seq profiles to histological and anatomical mouse brain atlases",slug:"fig-6-tangram-mapping-of-snrna-seq-profiles-to-histological-and-anatomical-mouse-brain-atlases",normalizedTitle:"fig. 6 tangram mapping of snrna-seq profiles to histological and anatomical mouse brain atlases",charIndex:4399}],excerpt:'<p>:  Tangram <br>\nDOI: <a href="https://doi.org/10.1038/s41592-021-01264-7" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-021-01264-7<OutboundLink/></a><br>\nCite: Biancalani, T., Scalia, G., Buffoni, L. et al. Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram. <em>Nat Methods</em> 18, 13521362 (2021).</p>\n',headersStr:"Abstract Results Fig. 1 Tangram learns spatial transcriptome-wide patterns at single-cell resolution from sc/snRNA-seq data and corresponding spatial data Fig. 2 Tangram maps cells with high-resolution MERFISH measurements and expands them to genome scale Fig. 3 Correction of low-quality genes by mapping snRNA-seq on STARmap data Fig. 4 Mapping snRNA-seq data to Spatial Transcriptomics data (Visium) demonstrates deconvolution and imputation of dropouts Fig. 5 Tangram mapping of multi-omic SHARE-seq profiles yields spatial patterns of chromatin accessibility and transcription factor activity Fig. 6 Tangram mapping of snRNA-seq profiles to histological and anatomical mouse brain atlases",content:":  Tangram \nDOI: https://doi.org/10.1038/s41592-021-01264-7\nCite: Biancalani, T., Scalia, G., Buffoni, L. et al. Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram. Nat Methods 18, 13521362 (2021).\n\n\n\n\n\n 1.  Tangram \n 2. Tangram Tutorial 1 MOp  snRNAseq  Slide-seq2 \n 3. Tangram Tutorial 2 squidy  Tangram\n 4. Tangram \n\n\n\n\n 1.  --- Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram\n 2. TangramSpatial alignment of single cell transcriptomic data\n\n\n\nTOMMASO BIANCALANI             AVIV REGEV\n                               \nGenentech                      Howard Hughes Medical Institute\ntommaso.biancalani@gmail.com   aviv.regev.sc@gmail.com\n\n\n# Abstract\n\n RNA-seq (sc/snRNA-seq)  Tangram sc/snRNA-seq  MERFISHSTARmapsmFISH (Visium) Tangram  sc/snRNA-seq  SHARE-seq  Tangram\n\n\n# Results\n\n\n# Fig. 1 Tangram learns spatial transcriptome-wide patterns at single-cell resolution from sc/snRNA-seq data and corresponding spatial data\n\n\n\nFig. 1 Tangram  sc/snRNA-seq \n\na. sc/snRNA-seq data  spatial data\nb.  smFISH  Allen ISH atlas \nc.  markers  Visium markers  Visium \nd.  snRNA-seqlegend smFISH \ne.  Visium  snRNA-seq  Visium \nf.  SHARE-seq  RNA  MERFISH \n\n\n# Fig. 2 Tangram maps cells with high-resolution MERFISH measurements and expands them to genome scale\n\n\n\nFig. 2 Tangram  MERFISH \n\na. MERFISH  snRNA-seq \nb. MERFISH  snRNA-seq profile  Tangram \nc.  Tangram \nd. xCDF75%  >40%\ne. Tangram  Allen Brain Atlas  ISH  11  MERFISH \nf. MERIFSH Tangram  Allen Brain Atlas  ISH  MERFISH  Allen ISH \n\n\n# Fig. 3 Correction of low-quality genes by mapping snRNA-seq on STARmap data\n\n\n\nFig. 3  snRNA-seq  STARmap \n\na. snRNA-seq  STARmap \nb.  STARmap  Tangram  snRNA-seq profile \nc.  Tangram \nd. Tangram STARmap  Allen atlas  STARmap  Allen atlas \ne. Tangram  Allen Brain Atlas  ISH  STARmap \n\n\n# Fig. 4 Mapping snRNA-seq data to Spatial Transcriptomics data (Visium) demonstrates deconvolution and imputation of dropouts\n\n\n\nFig. 4  snRNA-seq Visium\n\na.  Visium  Visium  snRNA-seq \nb.  Visium  snRNA-seq \nc.  Tangram  Visium \nd.  1,237  Tangram \ne.  Tangram  Visium  Visium \nf.  Visium \ng.  Tangram  Visium \nh.  Tangram Visium MERFISH Visium  MERFISH  Visium \n\n\n# Fig. 5 Tangram mapping of multi-omic SHARE-seq profiles yields spatial patterns of chromatin accessibility and transcription factor activity\n\n\n\nFig. 5 Tangram  SHARE-seq \n\na. SHARE-seq profiles  MERFISH  SHARE-seq profiles  RNA \nb.  MERFISH  Tangram  SHARE-seqRNAprofiles \nc. MERFISH  Tangram \nd. Tangram  SHARE-seq  z-score \n\n\n# Fig. 6 Tangram mapping of snRNA-seq profiles to histological and anatomical mouse brain atlases\n\n\n\nFig. 6 Tangram  snRNA-seq \n\na. ROIsNissl  snRNA-seq \nbc. b Allen c Blue Brain  ROI\nd.  ROI  snRNA-seq  ROI",normalizedContent:":  tangram \ndoi: https://doi.org/10.1038/s41592-021-01264-7\ncite: biancalani, t., scalia, g., buffoni, l. et al. deep learning and alignment of spatially resolved single-cell transcriptomes with tangram. nat methods 18, 13521362 (2021).\n\n\n\n\n\n 1.  tangram \n 2. tangram tutorial 1 mop  snrnaseq  slide-seq2 \n 3. tangram tutorial 2 squidy  tangram\n 4. tangram \n\n\n\n\n 1.  --- deep learning and alignment of spatially resolved single-cell transcriptomes with tangram\n 2. tangramspatial alignment of single cell transcriptomic data\n\n\n\ntommaso biancalani             aviv regev\n                               \ngenentech                      howard hughes medical institute\ntommaso.biancalani@gmail.com   aviv.regev.sc@gmail.com\n\n\n# abstract\n\n rna-seq (sc/snrna-seq)  tangram sc/snrna-seq  merfishstarmapsmfish (visium) tangram  sc/snrna-seq  share-seq  tangram\n\n\n# results\n\n\n# fig. 1 tangram learns spatial transcriptome-wide patterns at single-cell resolution from sc/snrna-seq data and corresponding spatial data\n\n\n\nfig. 1 tangram  sc/snrna-seq \n\na. sc/snrna-seq data  spatial data\nb.  smfish  allen ish atlas \nc.  markers  visium markers  visium \nd.  snrna-seqlegend smfish \ne.  visium  snrna-seq  visium \nf.  share-seq  rna  merfish \n\n\n# fig. 2 tangram maps cells with high-resolution merfish measurements and expands them to genome scale\n\n\n\nfig. 2 tangram  merfish \n\na. merfish  snrna-seq \nb. merfish  snrna-seq profile  tangram \nc.  tangram \nd. xcdf75%  >40%\ne. tangram  allen brain atlas  ish  11  merfish \nf. merifsh tangram  allen brain atlas  ish  merfish  allen ish \n\n\n# fig. 3 correction of low-quality genes by mapping snrna-seq on starmap data\n\n\n\nfig. 3  snrna-seq  starmap \n\na. snrna-seq  starmap \nb.  starmap  tangram  snrna-seq profile \nc.  tangram \nd. tangram starmap  allen atlas  starmap  allen atlas \ne. tangram  allen brain atlas  ish  starmap \n\n\n# fig. 4 mapping snrna-seq data to spatial transcriptomics data (visium) demonstrates deconvolution and imputation of dropouts\n\n\n\nfig. 4  snrna-seq visium\n\na.  visium  visium  snrna-seq \nb.  visium  snrna-seq \nc.  tangram  visium \nd.  1,237  tangram \ne.  tangram  visium  visium \nf.  visium \ng.  tangram  visium \nh.  tangram visium merfish visium  merfish  visium \n\n\n# fig. 5 tangram mapping of multi-omic share-seq profiles yields spatial patterns of chromatin accessibility and transcription factor activity\n\n\n\nfig. 5 tangram  share-seq \n\na. share-seq profiles  merfish  share-seq profiles  rna \nb.  merfish  tangram  share-seqrnaprofiles \nc. merfish  tangram \nd. tangram  share-seq  z-score \n\n\n# fig. 6 tangram mapping of snrna-seq profiles to histological and anatomical mouse brain atlases\n\n\n\nfig. 6 tangram  snrna-seq \n\na. roisnissl  snrna-seq \nbc. b allen c blue brain  roi\nd.  roi  snrna-seq  roi",charsets:{cjk:!0},lastUpdated:"2023/09/30, 10:34:48",lastUpdatedTimestamp:1696041288e3},{title:" --- Squidpya scalable framework for spatial omics analysis",frontmatter:{title:" --- Squidpya scalable framework for spatial omics analysis",date:"2023-10-07T09:16:18.000Z",permalink:"/pages/76476f/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20220131.(2022.01.31%20Squidpy%20Nat%20Methods)%20Squidpy%20a%20scalable%20framework%20for%20spatial%20omics%20analysis.html",relativePath:"02./14./20220131.(2022.01.31 Squidpy Nat Methods) Squidpy a scalable framework for spatial omics analysis.md",key:"v-15038c83",path:"/pages/76476f/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:356},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:599},{level:3,title:"Fig. 1 Squidpy is a software framework for the analysis of spatial omics data",slug:"fig-1-squidpy-is-a-software-framework-for-the-analysis-of-spatial-omics-data",normalizedTitle:"fig. 1 squidpy is a software framework for the analysis of spatial omics data",charIndex:611},{level:3,title:"Fig. 2 Analysis of spatial omics datasets across diverse experimental techniques using Squidpy",slug:"fig-2-analysis-of-spatial-omics-datasets-across-diverse-experimental-techniques-using-squidpy",normalizedTitle:"fig. 2 analysis of spatial omics datasets across diverse experimental techniques using squidpy",charIndex:1010},{level:3,title:"Fig. 3 Image analysis and relating images to molecular profiles with Squidpy",slug:"fig-3-image-analysis-and-relating-images-to-molecular-profiles-with-squidpy",normalizedTitle:"fig. 3 image analysis and relating images to molecular profiles with squidpy",charIndex:1570},{level:3,title:"Fig. 4 Analysis of mouse brain Visium dataset using Squidpy",slug:"fig-4-analysis-of-mouse-brain-visium-dataset-using-squidpy",normalizedTitle:"fig. 4 analysis of mouse brain visium dataset using squidpy",charIndex:2477},{level:2,title:"Discussion",slug:"discussion",normalizedTitle:"discussion",charIndex:2981}],excerpt:'<p>: Squidpy: <br>\nDOI: <a href="https://doi.org/10.1038/s41592-021-01358-2" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-021-01358-2<OutboundLink/></a><br>\nCite: Palla, G., Spitzer, H., Klein, M. et al. Squidpy: a scalable framework for spatial omics analysis. Nat Methods 19, 171178 (2022).</p>\n',headersStr:"Abstract Results Fig. 1 Squidpy is a software framework for the analysis of spatial omics data Fig. 2 Analysis of spatial omics datasets across diverse experimental techniques using Squidpy Fig. 3 Image analysis and relating images to molecular profiles with Squidpy Fig. 4 Analysis of mouse brain Visium dataset using Squidpy Discussion",content:": Squidpy: \nDOI: https://doi.org/10.1038/s41592-021-01358-2\nCite: Palla, G., Spitzer, H., Klein, M. et al. Squidpy: a scalable framework for spatial omics analysis. Nat Methods 19, 171178 (2022).\n\n\n\n\n\n 1. Squidpy: \n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# Abstract\n\n Squidpy Python Squidpy Squidpy \n\n\n# Results\n\n\n# Fig. 1 Squidpy is a software framework for the analysis of spatial omics data\n\n\n\nFig. 1 Squidpy \n\na. Squidpy  spot\nb.  Scanpy  Anndata Squidpy \nc. -\n\n\n# Fig. 2 Analysis of spatial omics datasets across diverse experimental techniques using Squidpy\n\n\n\nFig. 2  Squidpy \n\na. Squidpy \nb.  clusters \nc.  seqFISH  clusters \nd.  MERFISH  3D  clusters\ne. \nf.  HeLa \ng.  cluster \nh.  SlideseqV2 \ni.  'Ependymal' cluster  clusters  cluster \nj.  Ripley's L \nk.  SlideseqV2  Moran's I \n\n\n# Fig. 3 Image analysis and relating images to molecular profiles with Squidpy\n\n\n\nFig. 3  Squidpy \n\na. ImageContainer  Anndata ImageContainer  xyz left Anndata top rightAnndata  ImageContainer  napari bottom right\nb.  Visium  DAPIanti-NeuNanti-GFAP c \nc.  b  DAPIanti-NeuNanti-GFAP  DAPI \nd.  b  Visium spot  Visium spot  Visium spot  anti-NeuN  Visium spot  anti-GFAP \ne.  anti-GFAP  anti-NeuN  Visium spot  Gfap  Rbfox3 \nf.  MIBI-TOF  CD45CKvimentinleftcenter left Squidpy  CD45 center right Squidpy  CK right\n\n\n# Fig. 4 Analysis of mouse brain Visium dataset using Squidpy\n\n\n\nFig. 4  Squidpy  Visium \n\nab.  Moran's I Mobp  Nrgn\nc.  Sepal Krt18\nd. \ne.  cluster 'Hippocampus'  cluster 'Pyramidal layer'  'Pyramidal layer dentate gyrus' -\nf. 'Hippocampus'  cluster \ng. H&E \nh.  H&E  spot \ni.  StarDist  Visium spot \nj.  cluster  Visium spot \n\n\n# Discussion\n\nSquidpy Squidpy  Scanpy  Python Squidpy Squidpy  GPU  Dask Squidpy ",normalizedContent:": squidpy: \ndoi: https://doi.org/10.1038/s41592-021-01358-2\ncite: palla, g., spitzer, h., klein, m. et al. squidpy: a scalable framework for spatial omics analysis. nat methods 19, 171178 (2022).\n\n\n\n\n\n 1. squidpy: \n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# abstract\n\n squidpy python squidpy squidpy \n\n\n# results\n\n\n# fig. 1 squidpy is a software framework for the analysis of spatial omics data\n\n\n\nfig. 1 squidpy \n\na. squidpy  spot\nb.  scanpy  anndata squidpy \nc. -\n\n\n# fig. 2 analysis of spatial omics datasets across diverse experimental techniques using squidpy\n\n\n\nfig. 2  squidpy \n\na. squidpy \nb.  clusters \nc.  seqfish  clusters \nd.  merfish  3d  clusters\ne. \nf.  hela \ng.  cluster \nh.  slideseqv2 \ni.  'ependymal' cluster  clusters  cluster \nj.  ripley's l \nk.  slideseqv2  moran's i \n\n\n# fig. 3 image analysis and relating images to molecular profiles with squidpy\n\n\n\nfig. 3  squidpy \n\na. imagecontainer  anndata imagecontainer  xyz left anndata top rightanndata  imagecontainer  napari bottom right\nb.  visium  dapianti-neunanti-gfap c \nc.  b  dapianti-neunanti-gfap  dapi \nd.  b  visium spot  visium spot  visium spot  anti-neun  visium spot  anti-gfap \ne.  anti-gfap  anti-neun  visium spot  gfap  rbfox3 \nf.  mibi-tof  cd45ckvimentinleftcenter left squidpy  cd45 center right squidpy  ck right\n\n\n# fig. 4 analysis of mouse brain visium dataset using squidpy\n\n\n\nfig. 4  squidpy  visium \n\nab.  moran's i mobp  nrgn\nc.  sepal krt18\nd. \ne.  cluster 'hippocampus'  cluster 'pyramidal layer'  'pyramidal layer dentate gyrus' -\nf. 'hippocampus'  cluster \ng. h&e \nh.  h&e  spot \ni.  stardist  visium spot \nj.  cluster  visium spot \n\n\n# discussion\n\nsquidpy squidpy  scanpy  python squidpy squidpy  gpu  dask squidpy ",charsets:{cjk:!0},lastUpdated:"2023/10/19, 21:28:34",lastUpdatedTimestamp:1697722114e3},{title:" --- Cell2location maps fine-grained cell types in spatial transcriptomics",frontmatter:{title:" --- Cell2location maps fine-grained cell types in spatial transcriptomics",date:"2024-08-30T20:35:49.000Z",permalink:"/pages/9912cc/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20220113.(2022.01.13%20Cell2location%20Nat%20Biotechnol)%20Cell2location%20maps%20fine-grained%20cell%20types%20in%20spatial%20transcriptomics.html",relativePath:"02./14./20220113.(2022.01.13 Cell2location Nat Biotechnol) Cell2location maps fine-grained cell types in spatial transcriptomics.md",key:"v-c3ba952a",path:"/pages/9912cc/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:482}],excerpt:'<p>: Cell2location <br>\nDOI: <a href="https://doi.org/10.1038/s41587-021-01139-4" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41587-021-01139-4<OutboundLink/></a><br>\nCite: Kleshchevnikov, V., Shmatko, A., Dann, E. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Nat Biotechnol 40, 661671 (2022).</p>\n',headersStr:"Abstract",content:": Cell2location \nDOI: https://doi.org/10.1038/s41587-021-01139-4\nCite: Kleshchevnikov, V., Shmatko, A., Dann, E. et al. Cell2location maps fine-grained cell types in spatial transcriptomics. Nat Biotechnol 40, 661671 (2022).\n\n\n\nCell2location \n\n\n\nOMER ALI BAYRAKTAR          OLIVER STEGLE\n                            \nWellcome Sanger Institute   Wellcome Sanger Institute\nob5@sanger.ac.uk            oliver.stegle@embl.de\n\n\n# Abstract\n\n Cell2locationCell2location  Cell2location pre-germinal center B cell  Cell2location ",normalizedContent:": cell2location \ndoi: https://doi.org/10.1038/s41587-021-01139-4\ncite: kleshchevnikov, v., shmatko, a., dann, e. et al. cell2location maps fine-grained cell types in spatial transcriptomics. nat biotechnol 40, 661671 (2022).\n\n\n\ncell2location \n\n\n\nomer ali bayraktar          oliver stegle\n                            \nwellcome sanger institute   wellcome sanger institute\nob5@sanger.ac.uk            oliver.stegle@embl.de\n\n\n# abstract\n\n cell2locationcell2location  cell2location pre-germinal center b cell  cell2location ",charsets:{cjk:!0},lastUpdated:"2024/08/31, 09:20:00",lastUpdatedTimestamp:17250672e5},{title:" --- Integrated analysis of multimodal single-cell data",frontmatter:{title:" --- Integrated analysis of multimodal single-cell data",date:"2023-08-15T22:19:22.000Z",permalink:"/pages/b4dfe0/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20210624.(2021.06.24%20Seurat%20V4%20Cell)%20Integrated%20analysis%20of%20multimodal%20single-cell%20data.html",relativePath:"02./14./20210624.(2021.06.24 Seurat V4 Cell) Integrated analysis of multimodal single-cell data.md",key:"v-1056f6de",path:"/pages/b4dfe0/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:780},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1185},{level:3,title:"Figure 1. Schematic overview of multimodal integration using weighted nearest neighbor analysis",slug:"figure-1-schematic-overview-of-multimodal-integration-using-weighted-nearest-neighbor-analysis",normalizedTitle:"figure 1. schematic overview of multimodal integration using weighted nearest neighbor analysis",charIndex:1197},{level:3,title:"Figure 2. Benchmarking and robustness analysis for WNN integration",slug:"figure-2-benchmarking-and-robustness-analysis-for-wnn-integration",normalizedTitle:"figure 2. benchmarking and robustness analysis for wnn integration",charIndex:1870},{level:3,title:"Figure 3. A multimodal atlas of human PBMC",slug:"figure-3-a-multimodal-atlas-of-human-pbmc",normalizedTitle:"figure 3. a multimodal atlas of human pbmc",charIndex:2401},{level:3,title:"Figure 4. Multimodal biomarkers of immune cell states",slug:"figure-4-multimodal-biomarkers-of-immune-cell-states",normalizedTitle:"figure 4. multimodal biomarkers of immune cell states",charIndex:2812},{level:3,title:"Figure 5. Characterizing heterogeneity within lymphoid populations",slug:"figure-5-characterizing-heterogeneity-within-lymphoid-populations",normalizedTitle:"figure 5. characterizing heterogeneity within lymphoid populations",charIndex:3352},{level:3,title:"Figure 6. Identifying cell-type-specific responses to vaccination",slug:"figure-6-identifying-cell-type-specific-responses-to-vaccination",normalizedTitle:"figure 6. identifying cell-type-specific responses to vaccination",charIndex:4068},{level:3,title:"Figure 7. Supervised mapping of immune perturbations",slug:"figure-7-supervised-mapping-of-immune-perturbations",normalizedTitle:"figure 7. supervised mapping of immune perturbations",charIndex:4605}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1016/j.cell.2021.04.048" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.cell.2021.04.048<OutboundLink/></a><br>\nCite: Hao Y, Hao S, Andersen-Nissen E, et al. Integrated analysis of multimodal single-cell data. <em>Cell</em>. 2021;184(13):3573-3587.e29.</p>\n',headersStr:"Abstract Results Figure 1. Schematic overview of multimodal integration using weighted nearest neighbor analysis Figure 2. Benchmarking and robustness analysis for WNN integration Figure 3. A multimodal atlas of human PBMC Figure 4. Multimodal biomarkers of immune cell states Figure 5. Characterizing heterogeneity within lymphoid populations Figure 6. Identifying cell-type-specific responses to vaccination Figure 7. Supervised mapping of immune perturbations",content:": \nDOI: https://doi.org/10.1016/j.cell.2021.04.048\nCite: Hao Y, Hao S, Andersen-Nissen E, et al. Integrated analysis of multimodal single-cell data. Cell. 2021;184(13):3573-3587.e29.\n\n\n\n(Seurat V4) \n\n\n\n 1.  --- (Seurat V1) Spatial reconstruction of single-cell gene expression data\n 2.  --- (Seurat V2) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (Seurat V3) Comprehensive Integration of Single-Cell Data\n 4.  --- (Seurat V4) Integrated analysis of multimodal single-cell data\n 5.  --- (Seurat V5) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nRAHUL SATIJA\n\nNew York Genome Center\nrsatija@nygenome.org\n\n\n# Abstract\n\n \"weighted-nearest neighbor\"  211,000  (PBMC)  CITE-seq  228  2019  (COVID-19) \n\n\n# Results\n\n\n# Figure 1. Schematic overview of multimodal integration using weighted nearest neighbor analysis\n\n\n\nFigure 1. \n\n(A-B)  CITE-seq  (A)  (B)  (A)  (B)  (A)  (B)  k = 20 \n(C)  RNA  Seurat  RNA  T CD4  CD8 \n(D)  (C)  CD8 T \n(E)  RNA (WNN) UMAP \n(F)  RNA \n\n\n# Figure 2. Benchmarking and robustness analysis for WNN integration\n\n\n\nFigure 2. WNN \n\n(A)  CITE-seq  ( 25 ) UMAP  RNA WNN  WNN  T Figure S2 \n(B)  T  panel \n(C)  WNN  0\n(D-E)  WNN  totalVI  MOFA+ (D) WNN  25 (E) WNN Figure S2 \n\n\n# Figure 3. A multimodal atlas of human PBMC\n\n\n\nFigure 3.  PBMC \n\n(A) CITE-seq PBMC  8  (day 0)  (day 3 and day 7) 10x3' (228 )  10x5' (54  + BCR + TCR)  CITE-seq  210,911 \n(B-D)  RNA  (B) (C)  WNN  (D)  161,764  10x3'  UMAP  WNN  3  8  57 49,147  10X 5'  UMAP  3' \n\n\n# Figure 4. Multimodal biomarkers of immune cell states\n\n\n\nFigure 4. \n\n(A) CD4+ T  (DE)  RNA Heatmap  PBMC \n(B)  (A)  CD8+ T Figure S4 \n(C)  57  CITE-seq  panel panel in silico panel  Table S2\n(D) CD8_TEM_5  (C)  panel  bulk RNA-seq CITE-seq Heatmap  DE  bulk RNA-seq \n(E)  (D)  CD4 CTL \n\n\n# Figure 5. Characterizing heterogeneity within lymphoid populations\n\n\n\nFigure 5. \n\n(A) CITE-seq  CD8+ T  integrin proteins CD103  CD49a \n(B) CITE-seq  CD103+ CD49a-  CD103+ CD49+  integrin-7 \n(C-D)  (A-B) \n(E-F) CD103+ CD49a-  CD103- CD49+ GO \n(G)  15  T  VDJ  y  CDR3  cmv \n(H)  T  10X 5'  TCR \n(I-K) NK  CD16  CD38 \n(I) NK  CD16  CD16 (J)  (I)  CD38\n(K) CD38  CD16  NK \n\n\n# Figure 6. Identifying cell-type-specific responses to vaccination\n\n\n\nFigure 6. \n\n(A)  2  (day 0)  day 3 () () \n(B)  Nebulosa  IFI27 \n(C-D) Violin  day 3  Siglecl-1 (CD169)  (D) (A-D)  CD14+ CD16+  cDC2 DC \n(E) Bar plot  (1) \n(F)  NK \n(G) \n\n\n# Figure 7. Supervised mapping of immune perturbations\n\n\n\nFigure 7. \n\n(A)  CITE-seq  WNN  T cell Level 2 \n(B)  PBMC  CITE-seq  (A) \n(C) UMAP  scRNA-seq  44,721  COVID-19  PBMC UMAP \n(D)  (C)  2 \n(E)  scRNA-seq  (y axis)  CyTOF (x axis)  MAIT  Pearson  0.911\n(F) CyTOF  COVID-19  PBMC  MAIT ",normalizedContent:": \ndoi: https://doi.org/10.1016/j.cell.2021.04.048\ncite: hao y, hao s, andersen-nissen e, et al. integrated analysis of multimodal single-cell data. cell. 2021;184(13):3573-3587.e29.\n\n\n\n(seurat v4) \n\n\n\n 1.  --- (seurat v1) spatial reconstruction of single-cell gene expression data\n 2.  --- (seurat v2) integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (seurat v3) comprehensive integration of single-cell data\n 4.  --- (seurat v4) integrated analysis of multimodal single-cell data\n 5.  --- (seurat v5) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nrahul satija\n\nnew york genome center\nrsatija@nygenome.org\n\n\n# abstract\n\n \"weighted-nearest neighbor\"  211,000  (pbmc)  cite-seq  228  2019  (covid-19) \n\n\n# results\n\n\n# figure 1. schematic overview of multimodal integration using weighted nearest neighbor analysis\n\n\n\nfigure 1. \n\n(a-b)  cite-seq  (a)  (b)  (a)  (b)  (a)  (b)  k = 20 \n(c)  rna  seurat  rna  t cd4  cd8 \n(d)  (c)  cd8 t \n(e)  rna (wnn) umap \n(f)  rna \n\n\n# figure 2. benchmarking and robustness analysis for wnn integration\n\n\n\nfigure 2. wnn \n\n(a)  cite-seq  ( 25 ) umap  rna wnn  wnn  t figure s2 \n(b)  t  panel \n(c)  wnn  0\n(d-e)  wnn  totalvi  mofa+ (d) wnn  25 (e) wnn figure s2 \n\n\n# figure 3. a multimodal atlas of human pbmc\n\n\n\nfigure 3.  pbmc \n\n(a) cite-seq pbmc  8  (day 0)  (day 3 and day 7) 10x3' (228 )  10x5' (54  + bcr + tcr)  cite-seq  210,911 \n(b-d)  rna  (b) (c)  wnn  (d)  161,764  10x3'  umap  wnn  3  8  57 49,147  10x 5'  umap  3' \n\n\n# figure 4. multimodal biomarkers of immune cell states\n\n\n\nfigure 4. \n\n(a) cd4+ t  (de)  rna heatmap  pbmc \n(b)  (a)  cd8+ t figure s4 \n(c)  57  cite-seq  panel panel in silico panel  table s2\n(d) cd8_tem_5  (c)  panel  bulk rna-seq cite-seq heatmap  de  bulk rna-seq \n(e)  (d)  cd4 ctl \n\n\n# figure 5. characterizing heterogeneity within lymphoid populations\n\n\n\nfigure 5. \n\n(a) cite-seq  cd8+ t  integrin proteins cd103  cd49a \n(b) cite-seq  cd103+ cd49a-  cd103+ cd49+  integrin-7 \n(c-d)  (a-b) \n(e-f) cd103+ cd49a-  cd103- cd49+ go \n(g)  15  t  vdj  y  cdr3  cmv \n(h)  t  10x 5'  tcr \n(i-k) nk  cd16  cd38 \n(i) nk  cd16  cd16 (j)  (i)  cd38\n(k) cd38  cd16  nk \n\n\n# figure 6. identifying cell-type-specific responses to vaccination\n\n\n\nfigure 6. \n\n(a)  2  (day 0)  day 3 () () \n(b)  nebulosa  ifi27 \n(c-d) violin  day 3  siglecl-1 (cd169)  (d) (a-d)  cd14+ cd16+  cdc2 dc \n(e) bar plot  (1) \n(f)  nk \n(g) \n\n\n# figure 7. supervised mapping of immune perturbations\n\n\n\nfigure 7. \n\n(a)  cite-seq  wnn  t cell level 2 \n(b)  pbmc  cite-seq  (a) \n(c) umap  scrna-seq  44,721  covid-19  pbmc umap \n(d)  (c)  2 \n(e)  scrna-seq  (y axis)  cytof (x axis)  mait  pearson  0.911\n(f) cytof  covid-19  pbmc  mait ",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- Vivmultiscale visualization of high-resolution multiplexed bioimaging data on the web",frontmatter:{title:" --- Vivmultiscale visualization of high-resolution multiplexed bioimaging data on the web",date:"2024-07-26T10:20:25.000Z",permalink:"/pages/112d0b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20220511.(2022.05.11%20Viv%20Nat%20Methods)%20Viv%20multiscale%20visualization%20of%20high-resolution%20multiplexed%20bioimaging%20data%20on%20the%20web.html",relativePath:"02./14./20220511.(2022.05.11 Viv Nat Methods) Viv multiscale visualization of high-resolution multiplexed bioimaging data on the web.md",key:"v-cebb617c",path:"/pages/112d0b/",headers:[{level:2,title:"SUMMARY",slug:"summary",normalizedTitle:"summary",charIndex:373}],excerpt:'<p>: Viv web <br>\nDOI: <a href="https://doi.org/10.1038/s41592-022-01482-7" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-022-01482-7<OutboundLink/></a><br>\nCite: Manz, T., Gold, I., Patterson, N.H. et al. Viv: multiscale visualization of high-resolution multiplexed bioimaging data on the web. Nat Methods 19, 515516 (2022).</p>\n',headersStr:"SUMMARY",content:": Viv web \nDOI: https://doi.org/10.1038/s41592-022-01482-7\nCite: Manz, T., Gold, I., Patterson, N.H. et al. Viv: multiscale visualization of high-resolution multiplexed bioimaging data on the web. Nat Methods 19, 515516 (2022).\n\n\n\nViv web \n\n\n\nNILS GEHLENBORG\n\nHarvard Medical School\nnils@hms.harvard.edu\n\n\n# SUMMARY\n\n (NGFF)  Vivhttps://github.com/hms-dbmi/viv web  OME-TIFF  OME-NGFFViv  web  TB ",normalizedContent:": viv web \ndoi: https://doi.org/10.1038/s41592-022-01482-7\ncite: manz, t., gold, i., patterson, n.h. et al. viv: multiscale visualization of high-resolution multiplexed bioimaging data on the web. nat methods 19, 515516 (2022).\n\n\n\nviv web \n\n\n\nnils gehlenborg\n\nharvard medical school\nnils@hms.harvard.edu\n\n\n# summary\n\n (ngff)  vivhttps://github.com/hms-dbmi/viv web  ome-tiff  ome-ngffviv  web  tb ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat",frontmatter:{title:" --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat",date:"2023-04-24T19:30:31.000Z",permalink:"/pages/794f06/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20230215.(2023.02.15%20NeuronChat%20Nat%20Commun)%20Inferring%20neuron-neuron%20communications%20from%20single-cell%20transcriptomics%20through%20NeuronChat.html",relativePath:"02./14./20230215.(2023.02.15 NeuronChat Nat Commun) Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat.md",key:"v-01f55ac7",path:"/pages/794f06/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:713},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1101},{level:3,title:"Fig. 1 Overview of NeuronChat",slug:"fig-1-overview-of-neuronchat",normalizedTitle:"fig. 1 overview of neuronchat",charIndex:1113},{level:3,title:"Fig. 2 Benchmarking NeuronChat on projection networks of two mouse cortex regions: VISp and ALM",slug:"fig-2-benchmarking-neuronchat-on-projection-networks-of-two-mouse-cortex-regions-visp-and-alm",normalizedTitle:"fig. 2 benchmarking neuronchat on projection networks of two mouse cortex regions: visp and alm",charIndex:1961},{level:3,title:"Fig. 3 Comparison of NeuronChat, CellPhoneDB, and CellChat in predicting VISp and ALM projection networks",slug:"fig-3-comparison-of-neuronchat-cellphonedb-and-cellchat-in-predicting-visp-and-alm-projection-networks",normalizedTitle:"fig. 3 comparison of neuronchat, cellphonedb, and cellchat in predicting visp and alm projection networks",charIndex:2591},{level:3,title:"Fig. 4 Visualization and analysis of intercellular communication networks for mouse primary visual cortex",slug:"fig-4-visualization-and-analysis-of-intercellular-communication-networks-for-mouse-primary-visual-cortex",normalizedTitle:"fig. 4 visualization and analysis of intercellular communication networks for mouse primary visual cortex",charIndex:2969},{level:3,title:"Fig. 5 Comparison analysis of interlaminar excitatory communication networks for ALM and VISp",slug:"fig-5-comparison-analysis-of-interlaminar-excitatory-communication-networks-for-alm-and-visp",normalizedTitle:"fig. 5 comparison analysis of interlaminar excitatory communication networks for alm and visp",charIndex:3661},{level:3,title:"Fig. 6 Comparison analysis of cell-cell communications in cortex between ASD patients and controls",slug:"fig-6-comparison-analysis-of-cell-cell-communications-in-cortex-between-asd-patients-and-controls",normalizedTitle:"fig. 6 comparison analysis of cell-cell communications in cortex between asd patients and controls",charIndex:4082},{level:3,title:"Fig. 7 Multi-layered visualization for spatial data and inference of spatially constrained communication network",slug:"fig-7-multi-layered-visualization-for-spatial-data-and-inference-of-spatially-constrained-communication-network",normalizedTitle:"fig. 7 multi-layered visualization for spatial data and inference of spatially constrained communication network",charIndex:4609}],excerpt:'<p>:  NeuronChat -<br>\nDOI: <a href="https://doi.org/10.1038/s41467-023-36800-w" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-023-36800-w<OutboundLink/></a><br>\nCite: Zhao, W., Johnston, K.G., Ren, H. et al. Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat. Nat Commun 14, 1128 (2023).</p>\n',headersStr:"Abstract Results Fig. 1 Overview of NeuronChat Fig. 2 Benchmarking NeuronChat on projection networks of two mouse cortex regions: VISp and ALM Fig. 3 Comparison of NeuronChat, CellPhoneDB, and CellChat in predicting VISp and ALM projection networks Fig. 4 Visualization and analysis of intercellular communication networks for mouse primary visual cortex Fig. 5 Comparison analysis of interlaminar excitatory communication networks for ALM and VISp Fig. 6 Comparison analysis of cell-cell communications in cortex between ASD patients and controls Fig. 7 Multi-layered visualization for spatial data and inference of spatially constrained communication network",content:':  NeuronChat -\nDOI: https://doi.org/10.1038/s41467-023-36800-w\nCite: Zhao, W., Johnston, K.G., Ren, H. et al. Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat. Nat Commun 14, 1128 (2023).\n\n\n\n NeuronChat -\n\n\n\n\n\n 1.  --- Inference and analysis of cell-cell communication using CellChat\n 2. CellChatInference and analysis of cell-cell communication\n 3.  --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat\n 4. NeuronChatInferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nQING NIE\n\nUniversity of California, Irvine\nqnie@uci.edu\n\n\n# Abstract\n\nNeuronChatNeuronChat NeuronChat  NeuronChat -\n\n\n# Results\n\n\n# Fig. 1 Overview of NeuronChat\n\n\n\nFig. 1 NeuronChat \n\na. NeuronChat database NeuronChat database  chemical synapseelectrical synapsesynaptic adhesion ligand-target pairs-NeuronChat database  373  ligand-target 5 \nb.  NeuronChat  normalized cells-genes count matrixcell type  cell type  ligand-target synthesis  vesicular transport ligands  targets cell type  cell type  ligands  cell type  target  permutation test 0\nc. NeuronChat 1,2.3.4.\n\n\n# Fig. 2 Benchmarking NeuronChat on projection networks of two mouse cortex regions: VISp and ALM\n\n\n\nFig. 2  VISp  ALM  NeuronChat \n\na.  7  VISp  ground truth\nb. NeuronChat  VISp  NeuronChat  NeuronChat 0.028\nc. NeuronChat  VISp  ROC  AUROC  0.8316327 AUROC  0.5 \nd. NeuronChat  VISp  PR  AUPRC  0.9152577 AUPRC  0.5 \ne.  interaction pairs  AUROC  AUPRC \nf-j.  ALM \n\n\n# Fig. 3 Comparison of NeuronChat, CellPhoneDB, and CellChat in predicting VISp and ALM projection networks\n\n\n\nFig. 3 NeuronChat, CellPhoneDB, CellChat  VISp  ALM \n\na.  ROC  PR \nb.  VISp  100  AUROC AUPRC 100 \nc. \nd-e.  VISp  AUROC (d)  AUPRC (e) \nf-j.  ALM \n\n\n# Fig. 4 Visualization and analysis of intercellular communication networks for mouse primary visual cortex\n\n\n\nFig. 4 \n\na.  VISp  "weight"  circle plot  glutamatergic, GABAergic and non-neuronal chord diagram  heatmap \nb. 60\nc. \nd. \n\n\n# Fig. 5 Comparison analysis of interlaminar excitatory communication networks for ALM and VISp\n\n\n\nFig. 5  ALM  VISp \n\na. ALM VISp "weight" \nb. Glu-Grin3a  ALM VISp\nc.  ALM  VISp \nd.  ALM  VISp VISpALM\ne. (d)\n\n\n# Fig. 6 Comparison analysis of cell-cell communications in cortex between ASD patients and controls\n\n\n\nFig. 6 ASD -\n\na.  ASD \nb. ASD ASD \nc.  ASD  ASD \nd.  ASD \ne. ASD\n\n\n# Fig. 7 Multi-layered visualization for spatial data and inference of spatially constrained communication network\n\n\n\nFig. 7 \n\na-c. seqFISH+aMERFISHbVisiumc//10- b Allen \nd. 64MERFISHscore>0score<0\ne. GABA-scRNA-seq262044---',normalizedContent:':  neuronchat -\ndoi: https://doi.org/10.1038/s41467-023-36800-w\ncite: zhao, w., johnston, k.g., ren, h. et al. inferring neuron-neuron communications from single-cell transcriptomics through neuronchat. nat commun 14, 1128 (2023).\n\n\n\n neuronchat -\n\n\n\n\n\n 1.  --- inference and analysis of cell-cell communication using cellchat\n 2. cellchatinference and analysis of cell-cell communication\n 3.  --- inferring neuron-neuron communications from single-cell transcriptomics through neuronchat\n 4. neuronchatinferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nqing nie\n\nuniversity of california, irvine\nqnie@uci.edu\n\n\n# abstract\n\nneuronchatneuronchat neuronchat  neuronchat -\n\n\n# results\n\n\n# fig. 1 overview of neuronchat\n\n\n\nfig. 1 neuronchat \n\na. neuronchat database neuronchat database  chemical synapseelectrical synapsesynaptic adhesion ligand-target pairs-neuronchat database  373  ligand-target 5 \nb.  neuronchat  normalized cells-genes count matrixcell type  cell type  ligand-target synthesis  vesicular transport ligands  targets cell type  cell type  ligands  cell type  target  permutation test 0\nc. neuronchat 1,2.3.4.\n\n\n# fig. 2 benchmarking neuronchat on projection networks of two mouse cortex regions: visp and alm\n\n\n\nfig. 2  visp  alm  neuronchat \n\na.  7  visp  ground truth\nb. neuronchat  visp  neuronchat  neuronchat 0.028\nc. neuronchat  visp  roc  auroc  0.8316327 auroc  0.5 \nd. neuronchat  visp  pr  auprc  0.9152577 auprc  0.5 \ne.  interaction pairs  auroc  auprc \nf-j.  alm \n\n\n# fig. 3 comparison of neuronchat, cellphonedb, and cellchat in predicting visp and alm projection networks\n\n\n\nfig. 3 neuronchat, cellphonedb, cellchat  visp  alm \n\na.  roc  pr \nb.  visp  100  auroc auprc 100 \nc. \nd-e.  visp  auroc (d)  auprc (e) \nf-j.  alm \n\n\n# fig. 4 visualization and analysis of intercellular communication networks for mouse primary visual cortex\n\n\n\nfig. 4 \n\na.  visp  "weight"  circle plot  glutamatergic, gabaergic and non-neuronal chord diagram  heatmap \nb. 60\nc. \nd. \n\n\n# fig. 5 comparison analysis of interlaminar excitatory communication networks for alm and visp\n\n\n\nfig. 5  alm  visp \n\na. alm visp "weight" \nb. glu-grin3a  alm visp\nc.  alm  visp \nd.  alm  visp vispalm\ne. (d)\n\n\n# fig. 6 comparison analysis of cell-cell communications in cortex between asd patients and controls\n\n\n\nfig. 6 asd -\n\na.  asd \nb. asd asd \nc.  asd  asd \nd.  asd \ne. asd\n\n\n# fig. 7 multi-layered visualization for spatial data and inference of spatially constrained communication network\n\n\n\nfig. 7 \n\na-c. seqfish+amerfishbvisiumc//10- b allen \nd. 64merfishscore>0score<0\ne. gaba-scrna-seq262044---',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- Dictionary learning for integrative, multimodal and scalable single-cell analysis",frontmatter:{title:" --- Dictionary learning for integrative, multimodal and scalable single-cell analysis",date:"2023-08-16T19:44:13.000Z",permalink:"/pages/8c5839/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20230325.(2023.03.25%20Seurat%20V5%20Nat%20Biotechnol)%20Dictionary%20learning%20for%20integrative,%20multimodal,%20and%20scalable%20single-cell%20analysis.html",relativePath:"02./14./20230325.(2023.03.25 Seurat V5 Nat Biotechnol) Dictionary learning for integrative, multimodal, and scalable single-cell analysis.md",key:"v-01625f39",path:"/pages/8c5839/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:823},{level:2,title:"Results",slug:"results",normalizedTitle:"results",charIndex:1274},{level:3,title:"Figure 1. Integrating across modalities with molecular bridges",slug:"figure-1-integrating-across-modalities-with-molecular-bridges",normalizedTitle:"figure 1. integrating across modalities with molecular bridges",charIndex:1286},{level:3,title:"Figure 2. Mapping scATAC-seq data onto scRNA-seq references",slug:"figure-2-mapping-scatac-seq-data-onto-scrna-seq-references",normalizedTitle:"figure 2. mapping scatac-seq data onto scrna-seq references",charIndex:1726},{level:3,title:"Figure 3. Robustness and benchmarking analysis for bridge integration",slug:"figure-3-robustness-and-benchmarking-analysis-for-bridge-integration",normalizedTitle:"figure 3. robustness and benchmarking analysis for bridge integration",charIndex:2495},{level:3,title:"Figure 4. Using dictionary learning for massively scalable integration",slug:"figure-4-using-dictionary-learning-for-massively-scalable-integration",normalizedTitle:"figure 4. using dictionary learning for massively scalable integration",charIndex:3500},{level:3,title:"Figure 5. 'Community-scale' integration of sequencing and cytometry immune datasets",slug:"figure-5-community-scale-integration-of-sequencing-and-cytometry-immune-datasets",normalizedTitle:"figure 5. 'community-scale' integration of sequencing and cytometry immune datasets",charIndex:4059}],excerpt:'<p>: <br>\nDOI: <a href="https://doi.org/10.1038/s41587-023-01767-y" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41587-023-01767-y<OutboundLink/></a><br>\nCite: Hao, Y., Stuart, T., Kowalski, M.H. et al. Dictionary learning for integrative, multimodal and scalable single-cell analysis. <em>Nat Biotechnol</em> (2023).</p>\n',headersStr:"Abstract Results Figure 1. Integrating across modalities with molecular bridges Figure 2. Mapping scATAC-seq data onto scRNA-seq references Figure 3. Robustness and benchmarking analysis for bridge integration Figure 4. Using dictionary learning for massively scalable integration Figure 5. 'Community-scale' integration of sequencing and cytometry immune datasets",content:": \nDOI: https://doi.org/10.1038/s41587-023-01767-y\nCite: Hao, Y., Stuart, T., Kowalski, M.H. et al. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Nat Biotechnol (2023).\n\n\n\n(Seurat V5) \n\n\n\n 1.  --- (Seurat V1) Spatial reconstruction of single-cell gene expression data\n 2.  --- (Seurat V2) Integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (Seurat V3) Comprehensive Integration of Single-Cell Data\n 4.  --- (Seurat V4) Integrated analysis of multimodal single-cell data\n 5.  --- (Seurat V5) Dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nRAHUL SATIJA\n\nNew York Genome Center\nrsatija@nygenome.org\n\n\n# Abstract\n\n RNA  'bridge integration' 'dictionary' DNA  dictionary learning  sketching  860  Seurat  5  (http://www.satijalab.org/seurat) \n\n\n# Results\n\n\n# Figure 1. Integrating across modalities with molecular bridges\n\n\n\nFigure 1. \n\na. bridge integration workflow scRNA-seq  scATAC-seq10x multiome bridge integration10x multiomePaired-TagsnmC2TCITE-seq bridge \nb. bridge integration \n\n\n# Figure 2. Mapping scATAC-seq data onto scRNA-seq references\n\n\n\nFigure 2.  scATAC-seq  scRNA-seq \n\na.  scRNA-seq  UMAP  297,627  scRNA-seq profiles\nb. Granja et al. scATAC-seq  UMAP  26,159  profiles CD34 \nc. bridge integration query cells  scRNA-seq \nd-f. Coverage plots  query cells \ng. DC scRNA-seq  scATAC-seq \nh.  MPO i\ni. MPO  LMPP \nj. MPO  RNA  GMP \nk.  236  KEGG \nl. \n\n\n# Figure 3. Robustness and benchmarking analysis for bridge integration\n\n\n\nFigure 3. bridge integration \n\na. bridge integration  bridge integration 21 \nb.  bridge integrationmultiVICobolt  SIGLEC6  Coverage plots bridge integration  ASDCs  Supplementary Fig.2e,f \nc.  10x multiome  RNA  ATAC profiles  scATAC-seq  scRNA-seq  Jaccard n=30,253 cell pairs Supplementary Fig. 3  Paired-Tag H3K27acn=10,906 cellsH3K27me3n=6,280 cells H3K4me1n=12,638 cellsbridge integration  Jaccard \nd.  scRNA-seq \ne-f.  single-cell DNA methylation profiles  snmC2T-seq bridge e bridge integration  scRNA-seq f Supplementary Fig.3 \n\n\n# Figure 4. Using dictionary learning for massively scalable integration\n\n\n\nFigure 4. \n\na. atomic sketch integration  (harmonized profiles)\nb-c. UMAP  1,525,710  atomic sketch integration  55  (b)  (c)\nd.  FOXI1 \ne.  (PNECs) pseudobulk \nf.  (GO) \ng.  ()  pseudobulk  () \n\n\n# Figure 5. 'Community-scale' integration of sequencing and cytometry immune datasets\n\n\n\nFigure 5. '' \n\na.  atomic sketch integration UMAP  14  639  3,461,171  PBMC scRNA-seq profiles\nb. CD14+  COVID-19  506  CD14+  pseudobulk \nc.  CITE-seq  119  5,170,249  CyTOF profiles  CyTOF profile  scRNA-seq \nd.  scRNA-seq  FOXP3  T  20 \ne.  CyTOF  34  pseudobulk ",normalizedContent:": \ndoi: https://doi.org/10.1038/s41587-023-01767-y\ncite: hao, y., stuart, t., kowalski, m.h. et al. dictionary learning for integrative, multimodal and scalable single-cell analysis. nat biotechnol (2023).\n\n\n\n(seurat v5) \n\n\n\n 1.  --- (seurat v1) spatial reconstruction of single-cell gene expression data\n 2.  --- (seurat v2) integrating single-cell transcriptomic data across different conditions, technologies, and species\n 3.  --- (seurat v3) comprehensive integration of single-cell data\n 4.  --- (seurat v4) integrated analysis of multimodal single-cell data\n 5.  --- (seurat v5) dictionary learning for integrative, multimodal and scalable single-cell analysis\n\n\n\nrahul satija\n\nnew york genome center\nrsatija@nygenome.org\n\n\n# abstract\n\n rna  'bridge integration' 'dictionary' dna  dictionary learning  sketching  860  seurat  5  (http://www.satijalab.org/seurat) \n\n\n# results\n\n\n# figure 1. integrating across modalities with molecular bridges\n\n\n\nfigure 1. \n\na. bridge integration workflow scrna-seq  scatac-seq10x multiome bridge integration10x multiomepaired-tagsnmc2tcite-seq bridge \nb. bridge integration \n\n\n# figure 2. mapping scatac-seq data onto scrna-seq references\n\n\n\nfigure 2.  scatac-seq  scrna-seq \n\na.  scrna-seq  umap  297,627  scrna-seq profiles\nb. granja et al. scatac-seq  umap  26,159  profiles cd34 \nc. bridge integration query cells  scrna-seq \nd-f. coverage plots  query cells \ng. dc scrna-seq  scatac-seq \nh.  mpo i\ni. mpo  lmpp \nj. mpo  rna  gmp \nk.  236  kegg \nl. \n\n\n# figure 3. robustness and benchmarking analysis for bridge integration\n\n\n\nfigure 3. bridge integration \n\na. bridge integration  bridge integration 21 \nb.  bridge integrationmultivicobolt  siglec6  coverage plots bridge integration  asdcs  supplementary fig.2e,f \nc.  10x multiome  rna  atac profiles  scatac-seq  scrna-seq  jaccard n=30,253 cell pairs supplementary fig. 3  paired-tag h3k27acn=10,906 cellsh3k27me3n=6,280 cells h3k4me1n=12,638 cellsbridge integration  jaccard \nd.  scrna-seq \ne-f.  single-cell dna methylation profiles  snmc2t-seq bridge e bridge integration  scrna-seq f supplementary fig.3 \n\n\n# figure 4. using dictionary learning for massively scalable integration\n\n\n\nfigure 4. \n\na. atomic sketch integration  (harmonized profiles)\nb-c. umap  1,525,710  atomic sketch integration  55  (b)  (c)\nd.  foxi1 \ne.  (pnecs) pseudobulk \nf.  (go) \ng.  ()  pseudobulk  () \n\n\n# figure 5. 'community-scale' integration of sequencing and cytometry immune datasets\n\n\n\nfigure 5. '' \n\na.  atomic sketch integration umap  14  639  3,461,171  pbmc scrna-seq profiles\nb. cd14+  covid-19  506  cd14+  pseudobulk \nc.  cite-seq  119  5,170,249  cytof profiles  cytof profile  scrna-seq \nd.  scrna-seq  foxp3  t  20 \ne.  cytof  34  pseudobulk ",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:" --- SpaceXgene co-expression network estimation for spatial transcriptomics",frontmatter:{title:" --- SpaceXgene co-expression network estimation for spatial transcriptomics",date:"2024-07-26T15:14:43.000Z",permalink:"/pages/4cba92/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20220930.(2022.09.30,%20SpaceX,%20Bioinformatics)%20SpaceX%20gene%20co-expression%20network%20estimation%20for%20spatial%20transcriptomics.html",relativePath:"02./14./20220930.(2022.09.30, SpaceX, Bioinformatics) SpaceX gene co-expression network estimation for spatial transcriptomics.md",key:"v-2766f654",path:"/pages/4cba92/",headers:[{level:2,title:"SUMMARY",slug:"summary",normalizedTitle:"summary",charIndex:539}],excerpt:'<p>: SpaceX<br>\nDOI: <a href="https://doi.org/10.1093/bioinformatics/btac645" target="_blank" rel="noopener noreferrer">https://doi.org/10.1093/bioinformatics/btac645<OutboundLink/></a><br>\nCite: Satwik Acharyya, Xiang Zhou, Veerabhadran Baladandayuthapani, SpaceX: gene co-expression\nnetwork estimation for spatial transcriptomics, Bioinformatics, Volume 38, Issue 22, 15 November 2022, Pages 50335041.</p>\n',headersStr:"SUMMARY",content:": SpaceX\nDOI: https://doi.org/10.1093/bioinformatics/btac645\nCite: Satwik Acharyya, Xiang Zhou, Veerabhadran Baladandayuthapani, SpaceX: gene co-expression network estimation for spatial transcriptomics, Bioinformatics, Volume 38, Issue 22, 15 November 2022, Pages 50335041.\n\n\n\nSpaceX\n\n\n\nVEERABHADRAN BALADANDAYUTHAPANI   XIANG ZHOU\n                                  \nUniversity of Michigan            University of Michigan\nveerab@umich.edu                  xzhousph@umich.edu\n\n\n# SUMMARY\n\nMotivation\n - \nResults\n SpaceXSpaceX  SpaceX \nAvailability and implementation\nSpaceX R  https://github.com/bayesrx/SpaceX \nSupplementary information\n Bioinformatics ",normalizedContent:": spacex\ndoi: https://doi.org/10.1093/bioinformatics/btac645\ncite: satwik acharyya, xiang zhou, veerabhadran baladandayuthapani, spacex: gene co-expression network estimation for spatial transcriptomics, bioinformatics, volume 38, issue 22, 15 november 2022, pages 50335041.\n\n\n\nspacex\n\n\n\nveerabhadran baladandayuthapani   xiang zhou\n                                  \nuniversity of michigan            university of michigan\nveerab@umich.edu                  xzhousph@umich.edu\n\n\n# summary\n\nmotivation\n - \nresults\n spacexspacex  spacex \navailability and implementation\nspacex r  https://github.com/bayesrx/spacex \nsupplementary information\n bioinformatics ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- The scverse project provides a computational ecosystem for single-cell omics data analysis",frontmatter:{title:" --- The scverse project provides a computational ecosystem for single-cell omics data analysis",date:"2023-09-30T11:18:11.000Z",permalink:"/pages/124edd/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20230410.(2023.04.10%20scverse%20Nat%20Biotechnol)%20The%20scverse%20project%20provides%20a%20computational%20ecosystem%20for%20single-cell%20omics%20data%20analysis.html",relativePath:"02./14./20230410.(2023.04.10 scverse Nat Biotechnol) The scverse project provides a computational ecosystem for single-cell omics data analysis.md",key:"v-3121a636",path:"/pages/124edd/",headers:[{level:2,title:"Fig. 1 Scverse ",slug:"fig-1-scverse-",normalizedTitle:"fig. 1 scverse ",charIndex:404},{level:2,title:"Code availability",slug:"code-availability",normalizedTitle:"code availability",charIndex:3774}],excerpt:'<p>: scverse <br>\nDOI: <a href="https://doi.org/10.1038/s41587-023-01733-8" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41587-023-01733-8<OutboundLink/></a><br>\nCite: Virshup, I., Bredikhin, D., Heumos, L. et al. The scverse project provides a computational ecosystem for single-cell omics data analysis. Nat Biotechnol 41, 604606 (2023).</p>\n',headersStr:"Fig. 1 Scverse  Code availability",content:": scverse \nDOI: https://doi.org/10.1038/s41587-023-01733-8\nCite: Virshup, I., Bredikhin, D., Heumos, L. et al. The scverse project provides a computational ecosystem for single-cell omics data analysis. Nat Biotechnol 41, 604606 (2023).\n\n\n\nscverse \n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# Fig. 1 Scverse \n\n\n\nFig. 1 Scverse \n\na. scverse scverse  AnnData  MuData scverse  scverse \nb.  scverse Scverse  scverse  scverse \n\nAPIs scversehttps://scverse.orgScverse  Python scverse Scverse  Python  scanpyscvi-tools  muon\n\n\n\n scverse  AnnData  MuData  Bioconductor  SingleCellExperiment  MultiAssayExperiment  k  HDF5  Zarr  AnnData  MuData vitessceJavaScriptzellkonverterRMuDataSeuratRscVI.jlJuliaanndata-rsRust scverse  OME-NGFF \n\nscverse Scanpy  Python  muon scirpy  T  B squidpy scvi-tools Fig. 1a scverse  scverse \n\nscverse  Python https://scientific-python.orgPyTorch  Jax scverse \n\nScverse Fig. 1b scverse cellxgene  vitessce sfaira  PathML  scverse  scverse \n\n scverse.org  GitHub https://github.com/scverse/ecosystem-packages Bioconductor  scverse  https://scverse.org \n\n scverse  GitHub  Zulip  Discourse https://discourse.scverse.org Bioconductor https://github.com/scverse/cookiecutter-scverseFig. 1b Bioconductor  GitHub ActionsPyPI  Read the Docs \n\nscverse \n\n\n# Code availability\n\nscverse  https://github.com/scverse  https://gist.github.com/ivirshup/4bff45c0a8d38b97c5289c8b2407dfcf ",normalizedContent:": scverse \ndoi: https://doi.org/10.1038/s41587-023-01733-8\ncite: virshup, i., bredikhin, d., heumos, l. et al. the scverse project provides a computational ecosystem for single-cell omics data analysis. nat biotechnol 41, 604606 (2023).\n\n\n\nscverse \n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# fig. 1 scverse \n\n\n\nfig. 1 scverse \n\na. scverse scverse  anndata  mudata scverse  scverse \nb.  scverse scverse  scverse  scverse \n\napis scversehttps://scverse.orgscverse  python scverse scverse  python  scanpyscvi-tools  muon\n\n\n\n scverse  anndata  mudata  bioconductor  singlecellexperiment  multiassayexperiment  k  hdf5  zarr  anndata  mudata vitesscejavascriptzellkonverterrmudataseuratrscvi.jljuliaanndata-rsrust scverse  ome-ngff \n\nscverse scanpy  python  muon scirpy  t  b squidpy scvi-tools fig. 1a scverse  scverse \n\nscverse  python https://scientific-python.orgpytorch  jax scverse \n\nscverse fig. 1b scverse cellxgene  vitessce sfaira  pathml  scverse  scverse \n\n scverse.org  github https://github.com/scverse/ecosystem-packages bioconductor  scverse  https://scverse.org \n\n scverse  github  zulip  discourse https://discourse.scverse.org bioconductor https://github.com/scverse/cookiecutter-scversefig. 1b bioconductor  github actionspypi  read the docs \n\nscverse \n\n\n# code availability\n\nscverse  https://github.com/scverse  https://gist.github.com/ivirshup/4bff45c0a8d38b97c5289c8b2407dfcf ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data",frontmatter:{title:" --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data",date:"2023-06-30T11:09:22.000Z",permalink:"/pages/a24249/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20230612.(2023.06.12%20hdWGCNA%20Cell%20Reports%20Methods)%20hdWGCNA%20identifies%20co-expression%20networks%20in%20high-dimensional%20transcriptomics%20data.html",relativePath:"02./14./20230612.(2023.06.12 hdWGCNA Cell Reports Methods) hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data.md",key:"v-29874988",path:"/pages/a24249/",headers:[{level:2,title:"SUMMARY",slug:"summary",normalizedTitle:"summary",charIndex:909},{level:2,title:"RESULTS",slug:"results",normalizedTitle:"results",charIndex:1363},{level:3,title:"Figure 1. Overview of the hdWGCNA workflow and application in the human prefrontal cortex",slug:"figure-1-overview-of-the-hdwgcna-workflow-and-application-in-the-human-prefrontal-cortex",normalizedTitle:"figure 1. overview of the hdwgcna workflow and application in the human prefrontal cortex",charIndex:1375},{level:3,title:"Figure 2. Runtime, memory usage, and performance of hdWGCNA",slug:"figure-2-runtime-memory-usage-and-performance-of-hdwgcna",normalizedTitle:"figure 2. runtime, memory usage, and performance of hdwgcna",charIndex:2460},{level:3,title:"Figure 3. Spatial co-expression networks represent regional expression patterns in the mouse brain",slug:"figure-3-spatial-co-expression-networks-represent-regional-expression-patterns-in-the-mouse-brain",normalizedTitle:"figure 3. spatial co-expression networks represent regional expression patterns in the mouse brain",charIndex:3014},{level:3,title:"Figure 4. Isoform co-expression network analysis reveals fate-specific expression programs in the hippocampal radial glia lineage",slug:"figure-4-isoform-co-expression-network-analysis-reveals-fate-specific-expression-programs-in-the-hippocampal-radial-glia-lineage",normalizedTitle:"figure 4. isoform co-expression network analysis reveals fate-specific expression programs in the hippocampal radial glia lineage",charIndex:3432},{level:3,title:"Figure 5. Co-expression network analysis of inhibitory neurons in Autism spectrum disorder",slug:"figure-5-co-expression-network-analysis-of-inhibitory-neurons-in-autism-spectrum-disorder",normalizedTitle:"figure 5. co-expression network analysis of inhibitory neurons in autism spectrum disorder",charIndex:4396},{level:3,title:"Figure 6. Consensus network analysis of microglia in AD",slug:"figure-6-consensus-network-analysis-of-microglia-in-ad",normalizedTitle:"figure 6. consensus network analysis of microglia in ad",charIndex:4946},{level:3,title:"Figure 7. Projecting bulk RNA-seq co-expression modules into a single-cell dataset",slug:"figure-7-projecting-bulk-rna-seq-co-expression-modules-into-a-single-cell-dataset",normalizedTitle:"figure 7. projecting bulk rna-seq co-expression modules into a single-cell dataset",charIndex:5808}],excerpt:'<p>: hdWGCNA <br>\nDOI: <a href="https://doi.org/10.1016/j.crmeth.2023.100498" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.crmeth.2023.100498<OutboundLink/></a><br>\nCite: Samuel, M., Fairlie, R., Negin, R. et al. hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data. Cell Reports Methods (2023).</p>\n',headersStr:"SUMMARY RESULTS Figure 1. Overview of the hdWGCNA workflow and application in the human prefrontal cortex Figure 2. Runtime, memory usage, and performance of hdWGCNA Figure 3. Spatial co-expression networks represent regional expression patterns in the mouse brain Figure 4. Isoform co-expression network analysis reveals fate-specific expression programs in the hippocampal radial glia lineage Figure 5. Co-expression network analysis of inhibitory neurons in Autism spectrum disorder Figure 6. Consensus network analysis of microglia in AD Figure 7. Projecting bulk RNA-seq co-expression modules into a single-cell dataset",content:": hdWGCNA \nDOI: https://doi.org/10.1016/j.crmeth.2023.100498\nCite: Samuel, M., Fairlie, R., Negin, R. et al. hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data. Cell Reports Methods (2023).\n\n\n\nhdWGCNA \n\n\n\n\n\n 1.  --- WGCNAan R package for weighted correlation network analysis\n 2. WGCNAan R package for weighted correlation network analysis\n 3.  --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n 4.  --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdWGCNAhigh dimensional WGCNA\n\n\n\nSAMUEL MORABITO                        VIVEK SWARUP\n                                       \nUniversity of California Irvine(UCI)   University of California Irvine(UCI)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# SUMMARY\n\n hdWGCNA scRNAseq  spatial RNA-seqhdWGCNA  scRNAseq hdWGCNA  long-read single-cell  hdWGCNAhdWGCNA  Seurat Seurat  R  100  hdWGCNA \n\n\n# RESULTS\n\n\n# Figure 1. Overview of the hdWGCNA workflow and application in the human prefrontal cortex\n\n\n\nFigure 1. hdWGCNA \n\n(A)  scRNA-seq  hdWGCNA UMAP  Zhou et al.  PFC  36,671  ASC: astrocytes; EX: excitatory neurons; INH: inhibitory neurons; MG: microglia; ODC: oligodendrocytes; OPC: oligodendrocyte progenitor cells\n(B)  K  metacell  sc  single cell k  metacell single cell  0 metacell \n(C) single-cell (sc), pseudo-bulk (pb),  K  metacell (1 - sparsity\n(D) kME  INH-M6EX-M2ODC-M3OPC-M2ASC-M18MG-M14  top5 hub genes  hub gene  hub gene \n(E) d  module eigengene (ME)  snRNA-seq UMAP module eigengenes  PCA \n(F) ODC  UMAP  kME  hub genes  downsampled\n(G) 10  ODC  snRNA-seq UMAP \n(H) z preservation \n\n\n# Figure 2. Runtime, memory usage, and performance of hdWGCNA\n\n\n\nFigure 2. hdWGCNA \n\n(A and B)  54  65,415  hdWGCNA R package  1,000  50,000  (A)  (B)\n(C)  PFC AUC EGAD neighbor-voting \n(D)  PFC  AUC \n(E) XGBoost  Zhou et al.  PFC  96  10  hub genes RMSE split RMSE \n\n\n# Figure 3. Spatial co-expression networks represent regional expression patterns in the mouse brain\n\n\n\nFigure 3. \n\n(A)  Visium ST Louvain clusters \n(B)  UMAP  hub gene  kME  top5 hub genes  downsampled\n(C)  12 MEs ME \n\n\n# Figure 4. Isoform co-expression network analysis reveals fate-specific expression programs in the hippocampal radial glia lineage\n\n\n\nFigure 4. \n\n(A)  ScISOrSeq  UMAP  6,832  96,093  isoforms()  31,053 \n(B) UMAP Monocle3 EPDASCNPC\n(C)  isoforms  UMAP  isoforms isoforms  hub isoforms  kMEiso  downsampled\n(D)  isoforms isoforms  isoforms \n(E) Module eigenisoforms (MEiso)  loess \n(F)  GO \n(G) Gfap  isoforms \n(H) H3f3b   isoforms \n(I) Cd9   isoforms Swan  Cd9 / TSS \n\n\n# Figure 5. Co-expression network analysis of inhibitory neurons in Autism spectrum disorder\n\n\n\nFigure 5. \n\n(A)  snRNA-seq  ASD  121,451  UMAP \n(B)  TOM  UMAP  SFARI  ASD  hub genes hub gene  downsampled\n(C)  SFARI  INH  ASD \n(D) snRNA-seq UMAP  A MEs\n(E)  INH cluster  MEs Two-sided Wilcoxon test  ASD \n(F)  GO \n\n\n# Figure 6. Consensus network analysis of microglia in AD\n\n\n\nFigure 6. \n\n(A) Left:  AD snRNA-seq Right:  snRNA-seq  UMAP \n(B)  UMAP  Monocle3 \n(C)  UMAP  hub genes  kME  top10 hub genes \n(D) GO\n(E) MEs 50  bins  MEs loess  95%\n(F)  ME  UMAP \n(G) DME 10  bins bins ADpositive fold change DME \n(H) Top:  UMAP AD scDRSZ-score Bottom: scDRS Z-score  50  bins  scDRS Z-scores 95% \n(I) MEs  scDRS Z-scores  AD \n(J)  Z-summary \n\n\n# Figure 7. Projecting bulk RNA-seq co-expression modules into a single-cell dataset\n\n\n\nFigure 7.  bulk RNA-seq \n\n(A)  AD  PFC  snRNA-seq  57,950  UMAP \n(B)  Morabito et al. 2020 bulk RNA-seq  snRNA-seq  a \n(C)  AMP-AD bulk RNA-seq  snRNA-seq  a ",normalizedContent:": hdwgcna \ndoi: https://doi.org/10.1016/j.crmeth.2023.100498\ncite: samuel, m., fairlie, r., negin, r. et al. hdwgcna identifies co-expression networks in high-dimensional transcriptomics data. cell reports methods (2023).\n\n\n\nhdwgcna \n\n\n\n\n\n 1.  --- wgcnaan r package for weighted correlation network analysis\n 2. wgcnaan r package for weighted correlation network analysis\n 3.  --- single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n 4.  --- hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdwgcnahigh dimensional wgcna\n\n\n\nsamuel morabito                        vivek swarup\n                                       \nuniversity of california irvine(uci)   university of california irvine(uci)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# summary\n\n hdwgcna scrnaseq  spatial rna-seqhdwgcna  scrnaseq hdwgcna  long-read single-cell  hdwgcnahdwgcna  seurat seurat  r  100  hdwgcna \n\n\n# results\n\n\n# figure 1. overview of the hdwgcna workflow and application in the human prefrontal cortex\n\n\n\nfigure 1. hdwgcna \n\n(a)  scrna-seq  hdwgcna umap  zhou et al.  pfc  36,671  asc: astrocytes; ex: excitatory neurons; inh: inhibitory neurons; mg: microglia; odc: oligodendrocytes; opc: oligodendrocyte progenitor cells\n(b)  k  metacell  sc  single cell k  metacell single cell  0 metacell \n(c) single-cell (sc), pseudo-bulk (pb),  k  metacell (1 - sparsity\n(d) kme  inh-m6ex-m2odc-m3opc-m2asc-m18mg-m14  top5 hub genes  hub gene  hub gene \n(e) d  module eigengene (me)  snrna-seq umap module eigengenes  pca \n(f) odc  umap  kme  hub genes  downsampled\n(g) 10  odc  snrna-seq umap \n(h) z preservation \n\n\n# figure 2. runtime, memory usage, and performance of hdwgcna\n\n\n\nfigure 2. hdwgcna \n\n(a and b)  54  65,415  hdwgcna r package  1,000  50,000  (a)  (b)\n(c)  pfc auc egad neighbor-voting \n(d)  pfc  auc \n(e) xgboost  zhou et al.  pfc  96  10  hub genes rmse split rmse \n\n\n# figure 3. spatial co-expression networks represent regional expression patterns in the mouse brain\n\n\n\nfigure 3. \n\n(a)  visium st louvain clusters \n(b)  umap  hub gene  kme  top5 hub genes  downsampled\n(c)  12 mes me \n\n\n# figure 4. isoform co-expression network analysis reveals fate-specific expression programs in the hippocampal radial glia lineage\n\n\n\nfigure 4. \n\n(a)  scisorseq  umap  6,832  96,093  isoforms()  31,053 \n(b) umap monocle3 epdascnpc\n(c)  isoforms  umap  isoforms isoforms  hub isoforms  kmeiso  downsampled\n(d)  isoforms isoforms  isoforms \n(e) module eigenisoforms (meiso)  loess \n(f)  go \n(g) gfap  isoforms \n(h) h3f3b   isoforms \n(i) cd9   isoforms swan  cd9 / tss \n\n\n# figure 5. co-expression network analysis of inhibitory neurons in autism spectrum disorder\n\n\n\nfigure 5. \n\n(a)  snrna-seq  asd  121,451  umap \n(b)  tom  umap  sfari  asd  hub genes hub gene  downsampled\n(c)  sfari  inh  asd \n(d) snrna-seq umap  a mes\n(e)  inh cluster  mes two-sided wilcoxon test  asd \n(f)  go \n\n\n# figure 6. consensus network analysis of microglia in ad\n\n\n\nfigure 6. \n\n(a) left:  ad snrna-seq right:  snrna-seq  umap \n(b)  umap  monocle3 \n(c)  umap  hub genes  kme  top10 hub genes \n(d) go\n(e) mes 50  bins  mes loess  95%\n(f)  me  umap \n(g) dme 10  bins bins adpositive fold change dme \n(h) top:  umap ad scdrsz-score bottom: scdrs z-score  50  bins  scdrs z-scores 95% \n(i) mes  scdrs z-scores  ad \n(j)  z-summary \n\n\n# figure 7. projecting bulk rna-seq co-expression modules into a single-cell dataset\n\n\n\nfigure 7.  bulk rna-seq \n\n(a)  ad  pfc  snrna-seq  57,950  umap \n(b)  morabito et al. 2020 bulk rna-seq  snrna-seq  a \n(c)  amp-ad bulk rna-seq  snrna-seq  a ",charsets:{cjk:!0},lastUpdated:"2023/09/04, 10:33:44",lastUpdatedTimestamp:1693794824e3},{title:" --- SPACELdeep learning-based characterization of spatial transcriptome architectures",frontmatter:{title:" --- SPACELdeep learning-based characterization of spatial transcriptome architectures",date:"2024-07-26T10:47:49.000Z",permalink:"/pages/da0fb5/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20231122.(2023.11.22,%20SPACEL,%20Nat%20Commun)%20SPACEL%20characterizing%20spatial%20transcriptome%20architectures%20by%20deep-learning.html",relativePath:"02./14./20231122.(2023.11.22, SPACEL, Nat Commun) SPACEL characterizing spatial transcriptome architectures by deep-learning.md",key:"v-24a06b6a",path:"/pages/da0fb5/",headers:[{level:2,title:"Abstract",slug:"abstract",normalizedTitle:"abstract",charIndex:314}],excerpt:'<p>: SPACEL<br>\nDOI: <a href="https://doi.org/10.1038/s41467-023-43220-3" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-023-43220-3<OutboundLink/></a><br>\nCite: Xu, H., Wang, S., Fang, M. et al. SPACEL: deep learning-based characterization of spatial transcriptome architectures. Nat Commun 14, 7603 (2023).</p>\n',headersStr:"Abstract",content:": SPACEL\nDOI: https://doi.org/10.1038/s41467-023-43220-3\nCite: Xu, H., Wang, S., Fang, M. et al. SPACEL: deep learning-based characterization of spatial transcriptome architectures. Nat Commun 14, 7603 (2023).\n\n\n\nSPACEL\n\n\n\n\n\n\nqukun@ustc.edu.cn\n\n\n# Abstract\n\nST cells/spots  mRNA 2D ST 3D ST SPACELSPACEL Spoint  ST  spot Splane  ST Scube  ST  ST  19  SPACEL  3D  SPACEL  ST ",normalizedContent:": spacel\ndoi: https://doi.org/10.1038/s41467-023-43220-3\ncite: xu, h., wang, s., fang, m. et al. spacel: deep learning-based characterization of spatial transcriptome architectures. nat commun 14, 7603 (2023).\n\n\n\nspacel\n\n\n\n\n\n\nqukun@ustc.edu.cn\n\n\n# abstract\n\nst cells/spots  mrna 2d st 3d st spacelspacel spoint  st  spot splane  st scube  st  st  19  spacel  3d  spacel  st ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- ezSingleCellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists",frontmatter:{title:" --- ezSingleCellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists",date:"2024-07-26T10:48:02.000Z",permalink:"/pages/02ef79/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20240703.(2024.07.03,%20ezSingleCell,%20Nat%20Commun)%20ezSingleCell%20an%20integrated%20one-stop%20single-cell%20and%20spatial%20omics%20analysis%20platform%20for%20bench%20scientists.html",relativePath:"02./14./20240703.(2024.07.03, ezSingleCell, Nat Commun) ezSingleCell an integrated one-stop single-cell and spatial omics analysis platform for bench scientists.md",key:"v-0ef330be",path:"/pages/02ef79/",headers:[{level:2,title:"SUMMARY",slug:"summary",normalizedTitle:"summary",charIndex:388}],excerpt:'<p>: ezSingleCell<br>\nDOI: <a href="https://doi.org/10.1038/s41467-024-48188-2" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41467-024-48188-2<OutboundLink/></a><br>\nCite: Sethi, R., Ang, K.S., Li, M. et al. ezSingleCell: an integrated one-stop single-cell and spatial omics analysis platform for bench scientists. Nat Commun 15, 5600 (2024).</p>\n',headersStr:"SUMMARY",content:": ezSingleCell\nDOI: https://doi.org/10.1038/s41467-024-48188-2\nCite: Sethi, R., Ang, K.S., Li, M. et al. ezSingleCell: an integrated one-stop single-cell and spatial omics analysis platform for bench scientists. Nat Commun 15, 5600 (2024).\n\n\n\nezSingleCell \n\n\n\nSTEFANO MANGIOLA\n\n\njinmiao@gmail.com\n\n\n# SUMMARY\n\nezSingleCell ezSingleCell ezSingleCell ezSingleCell  3000  scRNA-seq ezSingleCell  Web https://immunesinglecell.org/ezsc/ shinyApp https://github.com/JinmiaoChenLab/ezSingleCell2",normalizedContent:": ezsinglecell\ndoi: https://doi.org/10.1038/s41467-024-48188-2\ncite: sethi, r., ang, k.s., li, m. et al. ezsinglecell: an integrated one-stop single-cell and spatial omics analysis platform for bench scientists. nat commun 15, 5600 (2024).\n\n\n\nezsinglecell \n\n\n\nstefano mangiola\n\n\njinmiao@gmail.com\n\n\n# summary\n\nezsinglecell ezsinglecell ezsinglecell ezsinglecell  3000  scrna-seq ezsinglecell  web https://immunesinglecell.org/ezsc/ shinyapp https://github.com/jinmiaochenlab/ezsinglecell2",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" --- The tidyomics ecosystemenhancing omic data analyses",frontmatter:{title:" --- The tidyomics ecosystemenhancing omic data analyses",date:"2024-07-26T10:47:55.000Z",permalink:"/pages/32a729/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/02.%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/14.%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6/20240614.(2024.06.14,%20tidyomics,%20Nat%20Methods)%20The%20tidyomics%20ecosystem%20enhancing%20omic%20data%20analyses.html",relativePath:"02./14./20240614.(2024.06.14, tidyomics, Nat Methods) The tidyomics ecosystem enhancing omic data analyses.md",key:"v-22df1dd0",path:"/pages/32a729/",headers:[{level:2,title:"SUMMARY",slug:"summary",normalizedTitle:"summary",charIndex:668}],excerpt:'<p>: tidyomics <br>\nDOI: <a href="https://doi.org/10.1038/s41592-024-02299-2" target="_blank" rel="noopener noreferrer">https://doi.org/10.1038/s41592-024-02299-2<OutboundLink/></a><br>\nCite: Hutchison, W.J., Keyes, T.J., The tidyomics Consortium. et al. The tidyomics ecosystem:\nenhancing omic data analyses. Nat Methods 21, 11661170 (2024).</p>\n',headersStr:"SUMMARY",content:": tidyomics \nDOI: https://doi.org/10.1038/s41592-024-02299-2\nCite: Hutchison, W.J., Keyes, T.J., The tidyomics Consortium. et al. The tidyomics ecosystem: enhancing omic data analyses. Nat Methods 21, 11661170 (2024).\n\n\n\ntidyomics \n\n\n\nSTEFANO MANGIOLA                   MICHAEL I. LOVE                            ANTHONY T. PAPENFUSS\n                                                                              \nThe University of Adelaide         University of North Carolina-Chapel Hill   University of Melbourne\nstefano.mangiola@adelaide.edu.au   michaelisaiahlove@gmail.com                papenfuss@wehi.edu.au\n\n\n# SUMMARY\n\nBioconductor tidy R  tidyomics  Bioconductor  tidy R  750  tidyomics ",normalizedContent:": tidyomics \ndoi: https://doi.org/10.1038/s41592-024-02299-2\ncite: hutchison, w.j., keyes, t.j., the tidyomics consortium. et al. the tidyomics ecosystem: enhancing omic data analyses. nat methods 21, 11661170 (2024).\n\n\n\ntidyomics \n\n\n\nstefano mangiola                   michael i. love                            anthony t. papenfuss\n                                                                              \nthe university of adelaide         university of north carolina-chapel hill   university of melbourne\nstefano.mangiola@adelaide.edu.au   michaelisaiahlove@gmail.com                papenfuss@wehi.edu.au\n\n\n# summary\n\nbioconductor tidy r  tidyomics  bioconductor  tidy r  750  tidyomics ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"SeuratR toolkit for single cell genomics",frontmatter:{title:"SeuratR toolkit for single cell genomics",date:"2023-01-31T16:57:11.000Z",permalink:"/pages/d207e7/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/001.Seurat.html",relativePath:"03./01. - /001.Seurat.md",key:"v-41ca1859",path:"/pages/d207e7/",headers:[{level:2,title:"Seurat ",slug:"seurat-",normalizedTitle:"seurat ",charIndex:744},{level:2,title:"Seurat Tutorial",slug:"seurat-tutorial",normalizedTitle:"seurat tutorial",charIndex:322},{level:3,title:"1 Introductory Vignettes",slug:"_1-introductory-vignettes",normalizedTitle:"1 introductory vignettes",charIndex:1205},{level:4,title:"1.1 Guided tutorial  2,700 PBMCs",slug:"_1-1-guided-tutorial-2-700-pbmcs",normalizedTitle:"1.1 guided tutorial  2,700 pbmcs",charIndex:1475},{level:4,title:"1.2 Multimodal analysis",slug:"_1-2-multimodal-analysis",normalizedTitle:"1.2 multimodal analysis",charIndex:1817},{level:3,title:"2 Data Integration",slug:"_2-data-integration",normalizedTitle:"2 data integration",charIndex:2119},{level:4,title:"2.1 Introduction to scRNA-seq integration",slug:"_2-1-introduction-to-scrna-seq-integration",normalizedTitle:"2.1 introduction to scrna-seq integration",charIndex:2300},{level:4,title:"2.2 Mapping and annotating query datasets",slug:"_2-2-mapping-and-annotating-query-datasets",normalizedTitle:"2.2 mapping and annotating query datasets",charIndex:2663},{level:4,title:"2.3 Fast integration using reciprocal PCA (RPCA)",slug:"_2-3-fast-integration-using-reciprocal-pca-rpca",normalizedTitle:"2.3 fast integration using reciprocal pca (rpca)",charIndex:3056},{level:4,title:"2.4 Tips for integrating large datasets",slug:"_2-4-tips-for-integrating-large-datasets",normalizedTitle:"2.4 tips for integrating large datasets",charIndex:3458},{level:4,title:"2.5 Integrating scRNA-seq and scATAC-seq data",slug:"_2-5-integrating-scrna-seq-and-scatac-seq-data",normalizedTitle:"2.5 integrating scrna-seq and scatac-seq data",charIndex:3820},{level:4,title:"2.6 Multimodal Reference Mapping",slug:"_2-6-multimodal-reference-mapping",normalizedTitle:"2.6 multimodal reference mapping",charIndex:4197},{level:3,title:"3 Additional New Methods",slug:"_3-additional-new-methods",normalizedTitle:"3 additional new methods",charIndex:4486},{level:4,title:"3.1 Weighted Nearest Neighbor Analysis",slug:"_3-1-weighted-nearest-neighbor-analysis",normalizedTitle:"3.1 weighted nearest neighbor analysis",charIndex:4712},{level:4,title:"3.2 Mixscape",slug:"_3-2-mixscape",normalizedTitle:"3.2 mixscape",charIndex:4754},{level:4,title:"3.3 SCTransform",slug:"_3-3-sctransform",normalizedTitle:"3.3 sctransform",charIndex:4770},{level:4,title:"3.4 SCTransform, v2 regularization",slug:"_3-4-sctransform-v2-regularization",normalizedTitle:"3.4 sctransform, v2 regularization",charIndex:4789},{level:3,title:"4 Other",slug:"_4-other",normalizedTitle:"4 other",charIndex:4828},{level:4,title:"4.1 Visualization",slug:"_4-1-visualization",normalizedTitle:"4.1 visualization",charIndex:4915},{level:4,title:"4.2 Cell Cycle Regression",slug:"_4-2-cell-cycle-regression",normalizedTitle:"4.2 cell cycle regression",charIndex:4936},{level:4,title:"4.3 Differential Expression Testing",slug:"_4-3-differential-expression-testing",normalizedTitle:"4.3 differential expression testing",charIndex:4965},{level:4,title:"4.4 Demultiplex Cell Hashing data",slug:"_4-4-demultiplex-cell-hashing-data",normalizedTitle:"4.4 demultiplex cell hashing data",charIndex:5004},{level:4,title:"4.5 Interoperability with Other Analysis Tools",slug:"_4-5-interoperability-with-other-analysis-tools",normalizedTitle:"4.5 interoperability with other analysis tools",charIndex:5041},{level:4,title:"4.6 Parallelization",slug:"_4-6-parallelization",normalizedTitle:"4.6 parallelization",charIndex:5091},{level:3,title:"5 SeuratWrappers",slug:"_5-seuratwrappers",normalizedTitle:"5 seuratwrappers",charIndex:5115}],excerpt:'<p>GitHub: <a href="https://github.com/satijalab/seurat" target="_blank" rel="noopener noreferrer">https://github.com/satijalab/seurat<OutboundLink/></a><br>\nDoc<a href="https://satijalab.org/seurat/index.html" target="_blank" rel="noopener noreferrer">https://satijalab.org/seurat/index.html<OutboundLink/></a><br>\n<img src="https://satijalab.org/seurat/articles/assets/seurat_banner.jpg" alt="SEURAT"></p>\n',headersStr:"Seurat  Seurat Tutorial 1 Introductory Vignettes 1.1 Guided tutorial  2,700 PBMCs 1.2 Multimodal analysis 2 Data Integration 2.1 Introduction to scRNA-seq integration 2.2 Mapping and annotating query datasets 2.3 Fast integration using reciprocal PCA (RPCA) 2.4 Tips for integrating large datasets 2.5 Integrating scRNA-seq and scATAC-seq data 2.6 Multimodal Reference Mapping 3 Additional New Methods 3.1 Weighted Nearest Neighbor Analysis 3.2 Mixscape 3.3 SCTransform 3.4 SCTransform, v2 regularization 4 Other 4.1 Visualization 4.2 Cell Cycle Regression 4.3 Differential Expression Testing 4.4 Demultiplex Cell Hashing data 4.5 Interoperability with Other Analysis Tools 4.6 Parallelization 5 SeuratWrappers",content:"GitHub: https://github.com/satijalab/seurat\nDochttps://satijalab.org/seurat/index.html\n\n\n\n\n\n\n 1.  (Seurat V1) \n 2.  (Seurat V2) \n 3.  (Seurat V3) \n 4.  (Seurat V4) \n 5.  (Seurat V5) (opens new window)\n 6.  Seurat Tutorial 1 PBMC 3K \n 7.  Seurat Tutorial 2 Seurat \n 8.  Seurat Tutorial 3scRNA-seq \n 9.  Seurat Tutorial 4\n 10. Seurat Tutorial 5 reciprocal PCA (RPCA) \n 11. Seurat Tutorial 6\n 12. Seurat Tutorial 7 scRNA-seq  scATAC-seq \n 13. Seurat Tutorial 8\n 14. Seurat Tutorial 9\n\n\n\nRAHUL SATIJA\n\nOxford University\nrsatija@nygenome.org\n\n\n# Seurat \n\nSeurat  R  single-cell RNA-seq Seurat \n\nSeurat \n\nSeurat :\n\n * Hao, et al., bioRxiv 2022 [Seurat V5]\n * Hao*, Hao*, et al., Cell 2021 [Seurat V4]\n * Stuart*, Butler*, et al., Cell 2019 [Seurat V3]\n * Butler* et al., Nat Biotechnol 2018 [Seurat V2]\n * Satija*, Farrell*, et al., Nat Biotechnol 2015 [Seurat V1]\n\n\n# Seurat Tutorial\n\n\n# 1 Introductory Vignettes\n\n Seurat  10X Genomics  2,700  (PBMC)  QC \n\nSeurat  CITE-seq  10x multiome  10x Visium  Vizgen MERFISH\n\n# 1.1 Guided tutorial  2,700 PBMCs\n\nSeurat \n https://satijalab.org/seurat/articles/pbmc3k_tutorial.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/Seurat - Guided Clustering Tutorial.nb.html\n Seurat Tutorial 1 PBMC 3K \n\n# 1.2 Multimodal analysis\n\n Seurat \n https://satijalab.org/seurat/articles/multimodal_vignette\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/multimodal_vignette.html\n Seurat Tutorial 2 Seurat \n\n\n# 2 Data Integration\n\nSeurat  PBMC  scRNA-seq reference  query cells\n\n# 2.1 Introduction to scRNA-seq integration\n\n scRNA-seq \n https://satijalab.org/seurat/articles/integration_introduction.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/Introduction to scRNA-seq integration.nb.html\n Seurat Tutorial 3scRNA-seq \n\n# 2.2 Mapping and annotating query datasets\n\n query scRNA-seq dataset  reference dataset  query cells\n https://satijalab.org/seurat/articles/integration_mapping.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/Mapping and annotating query datasets.nb.html\n Seurat Tutorial 4\n\n# 2.3 Fast integration using reciprocal PCA (RPCA)\n\n reciprocal PCA (RPCA)  anchors\n https://satijalab.org/seurat/articles/integration_rpca.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/Fast integration using reciprocal PCA (RPCA).nb.html\n Seurat Tutorial 5 reciprocal PCA (RPCA) \n\n# 2.4 Tips for integrating large datasets\n\n scRNA-seq  >200,000 \n https://satijalab.org/seurat/articles/integration_large_datasets.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/Tips for integrating large datasets.nb.html\n Seurat Tutorial 6\n\n# 2.5 Integrating scRNA-seq and scATAC-seq data\n\n scRNA-seq  scATAC-seq   https://satijalab.org/seurat/articles/atacseq_integration_vignette\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/atacseq_integration_vignette.html\n Seurat Tutorial 7 scRNA-seq  scATAC-seq \n\n# 2.6 Multimodal Reference Mapping\n\n  https://satijalab.org/seurat/articles/multimodal_reference_mapping\n http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.Seurat/multimodal_reference_mapping.html\n\n\n\n# 3 Additional New Methods\n\nSeurat \n\n * WNN paper\n * Mixscape:  CRISPR  paper\n * SCTransform:  RNA-seq  paper\n * SCTransform, v2 regularization paper\n\n# 3.1 Weighted Nearest Neighbor Analysis\n\n# 3.2 Mixscape\n\n# 3.3 SCTransform\n\n# 3.4 SCTransform, v2 regularization\n\n\n# 4 Other\n\n Seurat \n\n# 4.1 Visualization\n\n# 4.2 Cell Cycle Regression\n\n# 4.3 Differential Expression Testing\n\n# 4.4 Demultiplex Cell Hashing data\n\n# 4.5 Interoperability with Other Analysis Tools\n\n# 4.6 Parallelization\n\n\n# 5 SeuratWrappers\n\n Seurat  Seurat Wrappers  Seurat  Seurat ",normalizedContent:"github: https://github.com/satijalab/seurat\ndochttps://satijalab.org/seurat/index.html\n\n\n\n\n\n\n 1.  (seurat v1) \n 2.  (seurat v2) \n 3.  (seurat v3) \n 4.  (seurat v4) \n 5.  (seurat v5) (opens new window)\n 6.  seurat tutorial 1 pbmc 3k \n 7.  seurat tutorial 2 seurat \n 8.  seurat tutorial 3scrna-seq \n 9.  seurat tutorial 4\n 10. seurat tutorial 5 reciprocal pca (rpca) \n 11. seurat tutorial 6\n 12. seurat tutorial 7 scrna-seq  scatac-seq \n 13. seurat tutorial 8\n 14. seurat tutorial 9\n\n\n\nrahul satija\n\noxford university\nrsatija@nygenome.org\n\n\n# seurat \n\nseurat  r  single-cell rna-seq seurat \n\nseurat \n\nseurat :\n\n * hao, et al., biorxiv 2022 [seurat v5]\n * hao*, hao*, et al., cell 2021 [seurat v4]\n * stuart*, butler*, et al., cell 2019 [seurat v3]\n * butler* et al., nat biotechnol 2018 [seurat v2]\n * satija*, farrell*, et al., nat biotechnol 2015 [seurat v1]\n\n\n# seurat tutorial\n\n\n# 1 introductory vignettes\n\n seurat  10x genomics  2,700  (pbmc)  qc \n\nseurat  cite-seq  10x multiome  10x visium  vizgen merfish\n\n# 1.1 guided tutorial  2,700 pbmcs\n\nseurat \n https://satijalab.org/seurat/articles/pbmc3k_tutorial.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/seurat - guided clustering tutorial.nb.html\n seurat tutorial 1 pbmc 3k \n\n# 1.2 multimodal analysis\n\n seurat \n https://satijalab.org/seurat/articles/multimodal_vignette\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/multimodal_vignette.html\n seurat tutorial 2 seurat \n\n\n# 2 data integration\n\nseurat  pbmc  scrna-seq reference  query cells\n\n# 2.1 introduction to scrna-seq integration\n\n scrna-seq \n https://satijalab.org/seurat/articles/integration_introduction.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/introduction to scrna-seq integration.nb.html\n seurat tutorial 3scrna-seq \n\n# 2.2 mapping and annotating query datasets\n\n query scrna-seq dataset  reference dataset  query cells\n https://satijalab.org/seurat/articles/integration_mapping.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/mapping and annotating query datasets.nb.html\n seurat tutorial 4\n\n# 2.3 fast integration using reciprocal pca (rpca)\n\n reciprocal pca (rpca)  anchors\n https://satijalab.org/seurat/articles/integration_rpca.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/fast integration using reciprocal pca (rpca).nb.html\n seurat tutorial 5 reciprocal pca (rpca) \n\n# 2.4 tips for integrating large datasets\n\n scrna-seq  >200,000 \n https://satijalab.org/seurat/articles/integration_large_datasets.html\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/tips for integrating large datasets.nb.html\n seurat tutorial 6\n\n# 2.5 integrating scrna-seq and scatac-seq data\n\n scrna-seq  scatac-seq   https://satijalab.org/seurat/articles/atacseq_integration_vignette\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/atacseq_integration_vignette.html\n seurat tutorial 7 scrna-seq  scatac-seq \n\n# 2.6 multimodal reference mapping\n\n  https://satijalab.org/seurat/articles/multimodal_reference_mapping\n http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.seurat/multimodal_reference_mapping.html\n\n\n\n# 3 additional new methods\n\nseurat \n\n * wnn paper\n * mixscape:  crispr  paper\n * sctransform:  rna-seq  paper\n * sctransform, v2 regularization paper\n\n# 3.1 weighted nearest neighbor analysis\n\n# 3.2 mixscape\n\n# 3.3 sctransform\n\n# 3.4 sctransform, v2 regularization\n\n\n# 4 other\n\n seurat \n\n# 4.1 visualization\n\n# 4.2 cell cycle regression\n\n# 4.3 differential expression testing\n\n# 4.4 demultiplex cell hashing data\n\n# 4.5 interoperability with other analysis tools\n\n# 4.6 parallelization\n\n\n# 5 seuratwrappers\n\n seurat  seurat wrappers  seurat  seurat ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"DoubletFinderR package for detecting doublets in single-cell RNA sequencing data",frontmatter:{title:"DoubletFinderR package for detecting doublets in single-cell RNA sequencing data",categories:[""," - "],tags:[""],permalink:"/pages/a35823/",date:"2022-11-19T09:25:14.000Z",readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/002.DoubletFinder.html",relativePath:"03./01. - /002.DoubletFinder.md",key:"v-616419cb",path:"/pages/a35823/",headers:[{level:2,title:"DoubletFinder ",slug:"doubletfinder-",normalizedTitle:"doubletfinder ",charIndex:405},{level:2,title:"DoubletFinder ",slug:"doubletfinder-",normalizedTitle:"doubletfinder ",charIndex:939}],excerpt:'<p>GitHub: <a href="https://github.com/chris-mcginnis-ucsf/DoubletFinder" target="_blank" rel="noopener noreferrer">https://github.com/chris-mcginnis-ucsf/DoubletFinder<OutboundLink/></a></p>\n',headersStr:"DoubletFinder  DoubletFinder ",content:'GitHub: https://github.com/chris-mcginnis-ucsf/DoubletFinder\n\n\n\nDoubletFinderscRNAseqR\n\n\n\n\n 1.  --- DoubletFinder:Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors\n 2. DoubletFinderR package for detecting doublets in single-cell RNA sequencing data\n\n\n\nZEV GARTNER\n\nUniversity of California, San Francisco\nzev.gartner@ucsf.edu\n\n\n# DoubletFinder \n\nDoubletFinder :\n\n * Original Data:  Seurat  ()\n * Simulate Doublets: DoubletFinder  scRNA-seq  (pN)  artificial doublets ()\n * PCA: DoubletFinder  PCA \n * Compute pANN: DoubletFinder  PC  (pK)   (pANN)\n * Threshold pANN: DoubletFinder  pANN  n n  (nExp) \n * Doublets Removed:  Seurat  DoubletFinder \n\n\n# DoubletFinder \n\nDoubletFinder :\n\n * seu:  Seurat  ( NormalizeDataFindVariableGenesScaleDataRunPCARunTSNE )\n * PCs:  PCs = 1:10 \n * pN:  pN = 0.25DoubletFinder  pN \n * pK:  pANN (proportion of artificial nearest neighbors)  PC BC (bimodality coefficient) DoubletFinder  find.pK()  BC BC  pK \n * nExp:  10X  1000  0.8% \n\n\n\nlibrary(Seurat)\nlibrary(dplyr)\nlibrary(DoubletFinder)\n\n# Seuratpbmc3k\nseuratobject <- readRDS("pbmc3k_final.rds")\nncol(seuratobject) #  2638\nDimPlot(seuratobject, reduction=\'umap\')\n\n# pK Identification (no ground-truth) ------------------------------------------\nsweep.res.list <- paramSweep_v3(seuratobject, PCs = 1:10, sct = FALSE)\nsweep.stats <- summarizeSweep(sweep.res.list, GT = FALSE)\nbcmvn <- find.pK(sweep.stats)\npK <- bcmvn$pK[which.max(bcmvn$BCmetric)] %>% as.character() %>% as.numeric()\n\n# Homotypic Doublet Proportion Estimate ----------------------------------------\nDoubletRate <- ncol(seuratobject)/1000*0.008  # DoubletRate10X10000.8%\nhomotypic.prop <- modelHomotypic(seuratobject$seurat_clusters)\nnExp_poi <- round(DoubletRate*nrow(seuratobject@meta.data))\nnExp_poi.adj <- round(nExp_poi*(1-homotypic.prop))\n\n## Run DoubletFinder with varying classification stringencies ------------------\nseuratobject <- doubletFinder_v3(seuratobject, PCs = 1:10, pN = 0.25, pK = pK, nExp = nExp_poi, reuse.pANN = FALSE, sct = FALSE)\nseuratobject <- doubletFinder_v3(seuratobject, PCs = 1:10, pN = 0.25, pK = pK, nExp = nExp_poi.adj, reuse.pANN = FALSE, sct = FALSE)\n\n# DoubletFinder\nseuratobject@meta.data[,"DF_hi.lo"] <- seuratobject$DF.classifications_0.25_0.03_56\nseuratobject@meta.data$DF_hi.lo[which(seuratobject@meta.data$DF_hi.lo == "Doublet" & \n                                      seuratobject@meta.data$DF.classifications_0.25_0.03_46 == "Singlet")] <- "Doublet-Low Confidience"\nseuratobject@meta.data$DF_hi.lo[which(seuratobject@meta.data$DF_hi.lo == "Doublet" & \n                                      seuratobject@meta.data$DF.classifications_0.25_0.03_46 == "Doublet")] <- "Doublet-High Confidience"\ntable(seuratobject@meta.data$DF_hi.lo) # high:46, low:10, singlet:2582\nDimPlot(seuratobject, group.by ="DF_hi.lo", cols =c("black","red","gold"),reduction = \'umap\')\nseuratobject <- subset(seuratobject, cells = rownames(seuratobject@meta.data)[which(seuratobject@meta.data$DF_hi.lo == \'Singlet\')])\nncol(seuratobject) #  258256\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n\n\nDoubletFinder:',normalizedContent:'github: https://github.com/chris-mcginnis-ucsf/doubletfinder\n\n\n\ndoubletfinderscrnaseqr\n\n\n\n\n 1.  --- doubletfinder:doublet detection in single-cell rna sequencing data using artificial nearest neighbors\n 2. doubletfinderr package for detecting doublets in single-cell rna sequencing data\n\n\n\nzev gartner\n\nuniversity of california, san francisco\nzev.gartner@ucsf.edu\n\n\n# doubletfinder \n\ndoubletfinder :\n\n * original data:  seurat  ()\n * simulate doublets: doubletfinder  scrna-seq  (pn)  artificial doublets ()\n * pca: doubletfinder  pca \n * compute pann: doubletfinder  pc  (pk)   (pann)\n * threshold pann: doubletfinder  pann  n n  (nexp) \n * doublets removed:  seurat  doubletfinder \n\n\n# doubletfinder \n\ndoubletfinder :\n\n * seu:  seurat  ( normalizedatafindvariablegenesscaledatarunpcaruntsne )\n * pcs:  pcs = 1:10 \n * pn:  pn = 0.25doubletfinder  pn \n * pk:  pann (proportion of artificial nearest neighbors)  pc bc (bimodality coefficient) doubletfinder  find.pk()  bc bc  pk \n * nexp:  10x  1000  0.8% \n\n\n\nlibrary(seurat)\nlibrary(dplyr)\nlibrary(doubletfinder)\n\n# seuratpbmc3k\nseuratobject <- readrds("pbmc3k_final.rds")\nncol(seuratobject) #  2638\ndimplot(seuratobject, reduction=\'umap\')\n\n# pk identification (no ground-truth) ------------------------------------------\nsweep.res.list <- paramsweep_v3(seuratobject, pcs = 1:10, sct = false)\nsweep.stats <- summarizesweep(sweep.res.list, gt = false)\nbcmvn <- find.pk(sweep.stats)\npk <- bcmvn$pk[which.max(bcmvn$bcmetric)] %>% as.character() %>% as.numeric()\n\n# homotypic doublet proportion estimate ----------------------------------------\ndoubletrate <- ncol(seuratobject)/1000*0.008  # doubletrate10x10000.8%\nhomotypic.prop <- modelhomotypic(seuratobject$seurat_clusters)\nnexp_poi <- round(doubletrate*nrow(seuratobject@meta.data))\nnexp_poi.adj <- round(nexp_poi*(1-homotypic.prop))\n\n## run doubletfinder with varying classification stringencies ------------------\nseuratobject <- doubletfinder_v3(seuratobject, pcs = 1:10, pn = 0.25, pk = pk, nexp = nexp_poi, reuse.pann = false, sct = false)\nseuratobject <- doubletfinder_v3(seuratobject, pcs = 1:10, pn = 0.25, pk = pk, nexp = nexp_poi.adj, reuse.pann = false, sct = false)\n\n# doubletfinder\nseuratobject@meta.data[,"df_hi.lo"] <- seuratobject$df.classifications_0.25_0.03_56\nseuratobject@meta.data$df_hi.lo[which(seuratobject@meta.data$df_hi.lo == "doublet" & \n                                      seuratobject@meta.data$df.classifications_0.25_0.03_46 == "singlet")] <- "doublet-low confidience"\nseuratobject@meta.data$df_hi.lo[which(seuratobject@meta.data$df_hi.lo == "doublet" & \n                                      seuratobject@meta.data$df.classifications_0.25_0.03_46 == "doublet")] <- "doublet-high confidience"\ntable(seuratobject@meta.data$df_hi.lo) # high:46, low:10, singlet:2582\ndimplot(seuratobject, group.by ="df_hi.lo", cols =c("black","red","gold"),reduction = \'umap\')\nseuratobject <- subset(seuratobject, cells = rownames(seuratobject@meta.data)[which(seuratobject@meta.data$df_hi.lo == \'singlet\')])\nncol(seuratobject) #  258256\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n\n\ndoubletfinder:',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"scrattch.hicatHierarchical, Iterative Clustering for Analysis of Transcriptomics",frontmatter:{title:"scrattch.hicatHierarchical, Iterative Clustering for Analysis of Transcriptomics",date:"2023-05-15T10:10:25.000Z",permalink:"/pages/8dbd6f/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/005.scrattch.html",relativePath:"03./01. - /005.scrattch.md",key:"v-1d158259",path:"/pages/8dbd6f/",headers:[{level:2,title:"scrattch.hicat ",slug:"scrattch-hicat-",normalizedTitle:"scrattch.hicat ",charIndex:115},{level:3,title:"What is hicat?",slug:"what-is-hicat",normalizedTitle:"what is hicat?",charIndex:139},{level:3,title:"Breaking it all down",slug:"breaking-it-all-down",normalizedTitle:"breaking it all down",charIndex:500},{level:3,title:"Online Resources",slug:"online-resources",normalizedTitle:"online resources",charIndex:1045},{level:2,title:"Tutorials",slug:"tutorials",normalizedTitle:"tutorials",charIndex:1439}],excerpt:'<p>GitHub: <a href="https://github.com/AllenInstitute/scrattch.hicat" target="_blank" rel="noopener noreferrer">https://github.com/AllenInstitute/scrattch.hicat<OutboundLink/></a><br>\nDoc: <a href="https://taxonomy.shinyapps.io/scrattch_tutorial/" target="_blank" rel="noopener noreferrer">https://taxonomy.shinyapps.io/scrattch_tutorial/<OutboundLink/></a></p>\n',headersStr:"scrattch.hicat  What is hicat? Breaking it all down Online Resources Tutorials",content:"GitHub: https://github.com/AllenInstitute/scrattch.hicat\nDoc: https://taxonomy.shinyapps.io/scrattch_tutorial/\n\n\n# scrattch.hicat \n\n\n\n\n# What is hicat?\n\nhicat  Hierarchical, Iterative Clustering for Analysis of Transcriptomics clusters  markers \n\n Pvalb+ \n\n\n\n Tasic et al. 2018  hicat \n\n\n\n\n# Breaking it all down\n\n(iterative clustering)\n\nThe pipeline consists of the following key steps:\n\n * Dataset formatting and setup\n * Parameter specification\n * Dimension filtering based on QC correlation\n * High variance gene selection and dimensionality reduction\n * Jaccard-Louvain or hierarchichal (Ward) clustering\n * Cluster merging based on presence of differentially expressed genes.\n\n cluster  cluster  cluster \n\n\n\n\n# Online Resources\n\n Github  scrattch.hicat: https://github.com/AllenInstitute/scrattch.hicat\n\nTasic, et al., Nature Neuroscience (2016).\n\n Github  R : https://github.com/AllenInstitute/tasic2016data\n\n\n\nscrattch.hicat  Allen Institute  RNA-seq  scrattch \n\n\n# Tutorials\n\nhttps://taxonomy.shinyapps.io/scrattch_tutorial/\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//005.scrattch/scrattch_tutorial.nb.html",normalizedContent:"github: https://github.com/alleninstitute/scrattch.hicat\ndoc: https://taxonomy.shinyapps.io/scrattch_tutorial/\n\n\n# scrattch.hicat \n\n\n\n\n# what is hicat?\n\nhicat  hierarchical, iterative clustering for analysis of transcriptomics clusters  markers \n\n pvalb+ \n\n\n\n tasic et al. 2018  hicat \n\n\n\n\n# breaking it all down\n\n(iterative clustering)\n\nthe pipeline consists of the following key steps:\n\n * dataset formatting and setup\n * parameter specification\n * dimension filtering based on qc correlation\n * high variance gene selection and dimensionality reduction\n * jaccard-louvain or hierarchichal (ward) clustering\n * cluster merging based on presence of differentially expressed genes.\n\n cluster  cluster  cluster \n\n\n\n\n# online resources\n\n github  scrattch.hicat: https://github.com/alleninstitute/scrattch.hicat\n\ntasic, et al., nature neuroscience (2016).\n\n github  r : https://github.com/alleninstitute/tasic2016data\n\n\n\nscrattch.hicat  allen institute  rna-seq  scrattch \n\n\n# tutorials\n\nhttps://taxonomy.shinyapps.io/scrattch_tutorial/\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//005.scrattch/scrattch_tutorial.nb.html",charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"anndataa Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray",frontmatter:{title:"anndataa Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray",date:"2023-05-15T09:31:32.000Z",permalink:"/pages/7d0fcf/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/003.anndata.html",relativePath:"03./01. - /003.anndata.md",key:"v-e06f21f6",path:"/pages/7d0fcf/",headers:[{level:2,title:"anndata ",slug:"anndata-",normalizedTitle:"anndata ",charIndex:211},{level:2,title:"Tutorials",slug:"tutorials",normalizedTitle:"tutorials",charIndex:751},{level:3,title:"Getting started with anndata",slug:"getting-started-with-anndata",normalizedTitle:"getting started with anndata",charIndex:765}],excerpt:'<p>GitHub: <a href="https://github.com/scverse/anndata" target="_blank" rel="noopener noreferrer">https://github.com/scverse/anndata<OutboundLink/></a><br>\nDoc: <a href="https://anndata-tutorials.readthedocs.io/en/latest/index.html" target="_blank" rel="noopener noreferrer">https://anndata-tutorials.readthedocs.io/en/latest/index.html<OutboundLink/></a></p>\n',headersStr:"anndata  Tutorials Getting started with anndata",content:"GitHub: https://github.com/scverse/anndata\nDoc: https://anndata-tutorials.readthedocs.io/en/latest/index.html\n\n\n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# anndata \n\n\n\nanndata  Python  pandas  xarray anndata  PyTorch \n\n * Discuss development on GitHub.\n * Read the documentation.\n * Ask questions on the scverse Discourse.\n * Install via pip install anndata or conda install anndata -c conda-forge.\n * See Scanpy's documentation for usage related to single cell data. anndata was initially built for Scanpy.\n\nanndata  scverse  (website), governance) \n\nanndata \n\n * bioRxiv 2021 Dec 19.\n * Nat Biotechnol. 2023 Apr 10.\n\n\n# Tutorials\n\n\n# Getting started with anndata\n\nhttps://anndata-tutorials.readthedocs.io/en/latest/getting-started.html\nhttps://nbviewer.org/github/zhenghu159/Bioinformatics-Tools/blob/main//003.anndata/getting-started.ipynb\nAnnData  1 anndata",normalizedContent:"github: https://github.com/scverse/anndata\ndoc: https://anndata-tutorials.readthedocs.io/en/latest/index.html\n\n\n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# anndata \n\n\n\nanndata  python  pandas  xarray anndata  pytorch \n\n * discuss development on github.\n * read the documentation.\n * ask questions on the scverse discourse.\n * install via pip install anndata or conda install anndata -c conda-forge.\n * see scanpy's documentation for usage related to single cell data. anndata was initially built for scanpy.\n\nanndata  scverse  (website), governance) \n\nanndata \n\n * biorxiv 2021 dec 19.\n * nat biotechnol. 2023 apr 10.\n\n\n# tutorials\n\n\n# getting started with anndata\n\nhttps://anndata-tutorials.readthedocs.io/en/latest/getting-started.html\nhttps://nbviewer.org/github/zhenghu159/bioinformatics-tools/blob/main//003.anndata/getting-started.ipynb\nanndata  1 anndata",charsets:{cjk:!0},lastUpdated:"2023/10/19, 21:28:34",lastUpdatedTimestamp:1697722114e3},{title:"MetaNeighbora method to rapidly assess cell type identity using both functional and random gene sets",frontmatter:{title:"MetaNeighbora method to rapidly assess cell type identity using both functional and random gene sets",date:"2023-04-28T09:56:18.000Z",permalink:"/pages/9d48f3/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/004.MetaNeighbor.html",relativePath:"03./01. - /004.MetaNeighbor.md",key:"v-0da46b4e",path:"/pages/9d48f3/",headers:[{level:2,title:"1 Introduction",slug:"_1-introduction",normalizedTitle:"1 introduction",charIndex:202},{level:2,title:"2 Data type requirements",slug:"_2-data-type-requirements",normalizedTitle:"2 data type requirements",charIndex:1090},{level:2,title:"3 System requirements/estimated run times",slug:"_3-system-requirements-estimated-run-times",normalizedTitle:"3 system requirements/estimated run times",charIndex:2222},{level:2,title:"4 Installation",slug:"_4-installation",normalizedTitle:"4 installation",charIndex:2464},{level:2,title:"5 Methods",slug:"_5-methods",normalizedTitle:"5 methods",charIndex:2665},{level:3,title:"5.1 Part 1: Supervised MetaNeighbor",slug:"_5-1-part-1-supervised-metaneighbor",normalizedTitle:"5.1 part 1: supervised metaneighbor",charIndex:2980}],excerpt:'<p>GitHub: <a href="https://github.com/maggiecrow/MetaNeighbor" target="_blank" rel="noopener noreferrer">https://github.com/maggiecrow/MetaNeighbor<OutboundLink/></a><br>\nDoc: <a href="https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html" target="_blank" rel="noopener noreferrer">https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html<OutboundLink/></a></p>\n',headersStr:"1 Introduction 2 Data type requirements 3 System requirements/estimated run times 4 Installation 5 Methods 5.1 Part 1: Supervised MetaNeighbor",content:'GitHub: https://github.com/maggiecrow/MetaNeighbor\nDoc: https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html\n\n\n\nJESSE GILLIS\n\nCold Spring Harbor Laboratory\njgillis@cshl.edu\n\n\n# 1 Introduction\n\n RNA  (scRNA-seq)MetaNeighbor  T  T  T \n\n\n\nFigure 1\n\nA. \nB. \n\n MetaNeighbor -MethodsMetaNeighbor 1. a gene-by-sample expression matrix ("data")2. a set of labels indicating each samples dataset of origin ("experiment labels")3. a set of labels indicating each sample\'s cell type ("cell type labels")4. a set of genes ("gene sets")"AUROC scores"AUROC (e.g. neuron vs. non-neuronal cell)AUROC  0.5  Part 2 \n\n\n# 2 Data type requirements\n\n MetaNeighbor SummarizedExperiment objec (SEO)\n\n 1. The gene-by-sample matrix should be an assay of SEO.\n 2. Gene sets of interest should be provided as a list of vectors.\n 3. Additional data should have following vectors:\n\n * sample_id: A character vector (length equal to the number of samples) containing a unique identifier for each sample.\n * study_id: A character vector (length equal to the number of samples) that indicates the source of each sample (ex.GSE60361 = Zeisel et al, GSE71585 = Tasic et al, as in mn_data).\n * cell_type: A character vector (length equal to the number of samples) that indicates the celltype of each sample.\n\n 4. cell_labels should be provided as a sample-by-cell label matrix. This should be a binary (0,1) matrix where 1 indicates cell type membership and 0 indicates non-membership.\n\n:\n\n 1. It is critical that genes within gene sets match the gene names of the expression matrix.\n 2. Gene sets should contain more than one gene.\n 3. The row names of the cell_labels object should match the column names of the expression matrix.\n\n\n# 3 System requirements/estimated run times\n\nLaptop (OSX 10.10.4, 1.6 GHz, 8GB RAM, R 3.3.2, Rstudio 0.99.446) for the default version.\n\nDesktop (Ubuntu 18.04.1 in an Oracle VM, 4 x 3.6 GHz, 10.9GB RAM, R 3.4.4) for the low-memory version.\n\n\n# 4 Installation\n\n MetaNeighbor \n\nif (!require(\'devtools\')) {\n  install.packages(\'devtools\', quiet=TRUE)\n}\ndevtools::install_git(\'https://github.com/gillislab/MetaNeighbor\')\n\n\n1\n2\n3\n4\n\n\n\n# 5 Methods\n\nMetaNeighbor  0-1  (AUROC) AUROC\n\n\n# 5.1 Part 1: Supervised MetaNeighbor\n\n * https://www.bioconductor.org/packages/release/bioc/vignettes/MetaNeighbor/inst/doc/MetaNeighbor.pdf',normalizedContent:'github: https://github.com/maggiecrow/metaneighbor\ndoc: https://www.bioconductor.org/packages/release/bioc/html/metaneighbor.html\n\n\n\njesse gillis\n\ncold spring harbor laboratory\njgillis@cshl.edu\n\n\n# 1 introduction\n\n rna  (scrna-seq)metaneighbor  t  t  t \n\n\n\nfigure 1\n\na. \nb. \n\n metaneighbor -methodsmetaneighbor 1. a gene-by-sample expression matrix ("data")2. a set of labels indicating each samples dataset of origin ("experiment labels")3. a set of labels indicating each sample\'s cell type ("cell type labels")4. a set of genes ("gene sets")"auroc scores"auroc (e.g. neuron vs. non-neuronal cell)auroc  0.5  part 2 \n\n\n# 2 data type requirements\n\n metaneighbor summarizedexperiment objec (seo)\n\n 1. the gene-by-sample matrix should be an assay of seo.\n 2. gene sets of interest should be provided as a list of vectors.\n 3. additional data should have following vectors:\n\n * sample_id: a character vector (length equal to the number of samples) containing a unique identifier for each sample.\n * study_id: a character vector (length equal to the number of samples) that indicates the source of each sample (ex.gse60361 = zeisel et al, gse71585 = tasic et al, as in mn_data).\n * cell_type: a character vector (length equal to the number of samples) that indicates the celltype of each sample.\n\n 4. cell_labels should be provided as a sample-by-cell label matrix. this should be a binary (0,1) matrix where 1 indicates cell type membership and 0 indicates non-membership.\n\n:\n\n 1. it is critical that genes within gene sets match the gene names of the expression matrix.\n 2. gene sets should contain more than one gene.\n 3. the row names of the cell_labels object should match the column names of the expression matrix.\n\n\n# 3 system requirements/estimated run times\n\nlaptop (osx 10.10.4, 1.6 ghz, 8gb ram, r 3.3.2, rstudio 0.99.446) for the default version.\n\ndesktop (ubuntu 18.04.1 in an oracle vm, 4 x 3.6 ghz, 10.9gb ram, r 3.4.4) for the low-memory version.\n\n\n# 4 installation\n\n metaneighbor \n\nif (!require(\'devtools\')) {\n  install.packages(\'devtools\', quiet=true)\n}\ndevtools::install_git(\'https://github.com/gillislab/metaneighbor\')\n\n\n1\n2\n3\n4\n\n\n\n# 5 methods\n\nmetaneighbor  0-1  (auroc) auroc\n\n\n# 5.1 part 1: supervised metaneighbor\n\n * https://www.bioconductor.org/packages/release/bioc/vignettes/metaneighbor/inst/doc/metaneighbor.pdf',charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"WGCNAan R package for weighted correlation network analysis",frontmatter:{title:"WGCNAan R package for weighted correlation network analysis",date:"2023-05-15T10:42:38.000Z",permalink:"/pages/9ffdf7/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/006.WGCNA.html",relativePath:"03./01. - /006.WGCNA.md",key:"v-8449846e",path:"/pages/9ffdf7/",headers:[{level:2,title:"WGCNA ",slug:"wgcna-",normalizedTitle:"wgcna ",charIndex:654},{level:2,title:"Tutorials for the WGCNA package",slug:"tutorials-for-the-wgcna-package",normalizedTitle:"tutorials for the wgcna package",charIndex:827},{level:3,title:"I. Network analysis of liver expression data from female mice: finding modules related to body weight",slug:"i-network-analysis-of-liver-expression-data-from-female-mice-finding-modules-related-to-body-weight",normalizedTitle:"i. network analysis of liver expression data from female mice: finding modules related to body weight",charIndex:1617},{level:3,title:". Consensus analysis of female and male liver expression data",slug:"ii-consensus-analysis-of-female-and-male-liver-expression-data",normalizedTitle:". consensus analysis of female and male liver expression data",charIndex:2581}],excerpt:'<p>GitHub: <a href="https://github.com/cran/WGCNA" target="_blank" rel="noopener noreferrer">https://github.com/cran/WGCNA<OutboundLink/></a><br>\nDoc: <a href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/" target="_blank" rel="noopener noreferrer">https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/<OutboundLink/></a></p>\n',headersStr:"WGCNA  Tutorials for the WGCNA package I. Network analysis of liver expression data from female mice: finding modules related to body weight . Consensus analysis of female and male liver expression data",content:"GitHub: https://github.com/cran/WGCNA\nDoc: https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/\n\n\n\n\n\n 1. WGCNA Tutorial \n    \n\n\n\n 1.  --- WGCNAan R package for weighted correlation network analysis\n 2. WGCNAan R package for weighted correlation network analysis\n 3.  --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease\n 4.  --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdWGCNAhigh dimensional WGCNA\n\n\n\nSTEVE HORVATH\n\nUniversity of California Los Angeles(UCLA)\nshorvath@mednet.ucla.edu\n\n\n# WGCNA \n\nWGCNA R R  R  R \n\n\n# Tutorials for the WGCNA package\n\n WGCNA    WGCNA  main WGCNA page  installation instructions\n\nI-III\n\n * Tutorial I  WGCNA \n * Tutorial II consensus module analysis Tutorial I \n * Tutorial III \n * Jeremy Miller  collapseRows  userListEnrichment  VisANT  here\n\n WGCNA  2016.2.13 \n\n\n\n * WGCNA background and glossary of important terms\n\n\n# I. Network analysis of liver expression data from female mice: finding modules related to body weight\n\nI. \n\nData description and download\n\n F2  Ghazalpour et. al. (2006) Integrating Genetics and Network Analysis to Characterize Genes Related to Mouse Weight (link to paper; link to additional information) 3600  20,000 \n\n * zipped data sets,\n\n R  R \n\nR Tutorial\n\n\n\n\n\nhttps://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/Tutorials/index.html\nhttps://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&mid=2247483968&idx=1&sn=87ef3c6f85952ae0fda59f1008af54dd&chksm=c0a7b29df7d03b8b861a6a37a7a5f4de13a4a4f172512f679762b94ff1c80321d9a02a7b302c&token=549238884&lang=zh_CN#rd\n\n\n# . Consensus analysis of female and male liver expression data\n\n. ",normalizedContent:"github: https://github.com/cran/wgcna\ndoc: https://horvath.genetics.ucla.edu/html/coexpressionnetwork/rpackages/wgcna/\n\n\n\n\n\n 1. wgcna tutorial \n    \n\n\n\n 1.  --- wgcnaan r package for weighted correlation network analysis\n 2. wgcnaan r package for weighted correlation network analysis\n 3.  --- single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer's disease\n 4.  --- hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdwgcnahigh dimensional wgcna\n\n\n\nsteve horvath\n\nuniversity of california los angeles(ucla)\nshorvath@mednet.ucla.edu\n\n\n# wgcna \n\nwgcna r r  r  r \n\n\n# tutorials for the wgcna package\n\n wgcna    wgcna  main wgcna page  installation instructions\n\ni-iii\n\n * tutorial i  wgcna \n * tutorial ii consensus module analysis tutorial i \n * tutorial iii \n * jeremy miller  collapserows  userlistenrichment  visant  here\n\n wgcna  2016.2.13 \n\n\n\n * wgcna background and glossary of important terms\n\n\n# i. network analysis of liver expression data from female mice: finding modules related to body weight\n\ni. \n\ndata description and download\n\n f2  ghazalpour et. al. (2006) integrating genetics and network analysis to characterize genes related to mouse weight (link to paper; link to additional information) 3600  20,000 \n\n * zipped data sets,\n\n r  r \n\nr tutorial\n\n\n\n\n\nhttps://horvath.genetics.ucla.edu/html/coexpressionnetwork/rpackages/wgcna/tutorials/index.html\nhttps://mp.weixin.qq.com/s?__biz=mzkwmjm0mza5ma==&mid=2247483968&idx=1&sn=87ef3c6f85952ae0fda59f1008af54dd&chksm=c0a7b29df7d03b8b861a6a37a7a5f4de13a4a4f172512f679762b94ff1c80321d9a02a7b302c&token=549238884&lang=zh_cn#rd\n\n\n# . consensus analysis of female and male liver expression data\n\n. ",charsets:{cjk:!0},lastUpdated:"2023/08/17, 09:47:20",lastUpdatedTimestamp:169223684e4},{title:"MonocleAn analysis toolkit for single-cell RNA-seq",frontmatter:{title:"MonocleAn analysis toolkit for single-cell RNA-seq",date:"2022-12-04T09:51:09.000Z",permalink:"/pages/b62f01/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/008.Monocle.html",relativePath:"03./01. - /008.Monocle.md",key:"v-0387c6df",path:"/pages/b62f01/",headers:[{level:2,title:"Monocle 3 ",slug:"monocle-3-",normalizedTitle:"monocle 3 ",charIndex:171},{level:2,title:"Monocle 3 ",slug:"monocle-3-",normalizedTitle:"monocle 3 ",charIndex:555},{level:2,title:" Monocle 3",slug:"-monocle-3",normalizedTitle:" monocle 3",charIndex:1045},{level:2,title:"Monocle3 Tutorial --- Clustering and classifying your cells",slug:"monocle3-tutorial-clustering-and-classifying-your-cells",normalizedTitle:"monocle3 tutorial --- clustering and classifying your cells",charIndex:1664},{level:2,title:"Monocle3 Tutorial --- Constructing single-cell trajectories",slug:"monocle3-tutorial-constructing-single-cell-trajectories",normalizedTitle:"monocle3 tutorial --- constructing single-cell trajectories",charIndex:1961}],excerpt:'<p>GitHub: <a href="https://github.com/cole-trapnell-lab/monocle3" target="_blank" rel="noopener noreferrer">https://github.com/cole-trapnell-lab/monocle3<OutboundLink/></a><br>\nDoc: <a href="http://cole-trapnell-lab.github.io/monocle3/" target="_blank" rel="noopener noreferrer">http://cole-trapnell-lab.github.io/monocle3/<OutboundLink/></a></p>\n',headersStr:"Monocle 3  Monocle 3   Monocle 3 Monocle3 Tutorial --- Clustering and classifying your cells Monocle3 Tutorial --- Constructing single-cell trajectories",content:"GitHub: https://github.com/cole-trapnell-lab/monocle3\nDoc: http://cole-trapnell-lab.github.io/monocle3/\n\n\n\nCOLE TRAPNELL\n\nUniversity of Washington\ncoletrap@uw.edu\n\n\n# Monocle 3 \n\nMonocle 3  R \n\n * Clustering, classifying, and counting cells. \n * Constructing single-cell trajectories. \n * Differential expression analysis. \n\nMonocle :\n\n * Cole Trapnell et al., Nat Biotechnol 2014 [Monocle 1]\n * Qiu et al., Nat Methods 2017 [Monocle 2]\n * Cao et al., Nature 2019 [Monocle 3]\n\n\n# Monocle 3 \n\n * millions of cells\n * \n   * \n   * UMAP\n   *  roots \n   * \n   *  \"approximate graph abstraction\" \n   *  differentialGeneTest()  BEAM() \n   *  query data  reference \n   *  reference  query data \n   *  Monocle objects  transformation models\n   *  fit_models \n   *  3D \n\n\n#  Monocle 3\n\n Bioconductor\n\nif (!requireNamespace(\"BiocManager\", quietly = TRUE))\ninstall.packages(\"BiocManager\")\nBiocManager::install()\n\n\n1\n2\n3\n\n\n Bioconductor dependencies\n\nBiocManager::install(c('BiocGenerics', 'DelayedArray', 'DelayedMatrixStats',\n                       'limma', 'lme4', 'S4Vectors', 'SingleCellExperiment',\n                       'SummarizedExperiment', 'batchelor', 'HDF5Array',\n                       'terra', 'ggrastr'))\n\n\n1\n2\n3\n4\n\n\n monocle3\n\ninstall.packages(\"devtools\")\ndevtools::install_github('cole-trapnell-lab/monocle3')\n\n\n1\n2\n\n\n monocle3 \n\nlibrary(monocle3)\n\n\n1\n\n\n\n# Monocle3 Tutorial --- Clustering and classifying your cells\n\nhttps://cole-trapnell-lab.github.io/monocle3/docs/clustering/\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//008.Monocle3/Clustering and classifying your cells.nb.html\n\n\n# Monocle3 Tutorial --- Constructing single-cell trajectories\n\nhttps://cole-trapnell-lab.github.io/monocle3/docs/trajectories/\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//008.Monocle3/Constructing single-cell trajectories.nb.html",normalizedContent:"github: https://github.com/cole-trapnell-lab/monocle3\ndoc: http://cole-trapnell-lab.github.io/monocle3/\n\n\n\ncole trapnell\n\nuniversity of washington\ncoletrap@uw.edu\n\n\n# monocle 3 \n\nmonocle 3  r \n\n * clustering, classifying, and counting cells. \n * constructing single-cell trajectories. \n * differential expression analysis. \n\nmonocle :\n\n * cole trapnell et al., nat biotechnol 2014 [monocle 1]\n * qiu et al., nat methods 2017 [monocle 2]\n * cao et al., nature 2019 [monocle 3]\n\n\n# monocle 3 \n\n * millions of cells\n * \n   * \n   * umap\n   *  roots \n   * \n   *  \"approximate graph abstraction\" \n   *  differentialgenetest()  beam() \n   *  query data  reference \n   *  reference  query data \n   *  monocle objects  transformation models\n   *  fit_models \n   *  3d \n\n\n#  monocle 3\n\n bioconductor\n\nif (!requirenamespace(\"biocmanager\", quietly = true))\ninstall.packages(\"biocmanager\")\nbiocmanager::install()\n\n\n1\n2\n3\n\n\n bioconductor dependencies\n\nbiocmanager::install(c('biocgenerics', 'delayedarray', 'delayedmatrixstats',\n                       'limma', 'lme4', 's4vectors', 'singlecellexperiment',\n                       'summarizedexperiment', 'batchelor', 'hdf5array',\n                       'terra', 'ggrastr'))\n\n\n1\n2\n3\n4\n\n\n monocle3\n\ninstall.packages(\"devtools\")\ndevtools::install_github('cole-trapnell-lab/monocle3')\n\n\n1\n2\n\n\n monocle3 \n\nlibrary(monocle3)\n\n\n1\n\n\n\n# monocle3 tutorial --- clustering and classifying your cells\n\nhttps://cole-trapnell-lab.github.io/monocle3/docs/clustering/\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//008.monocle3/clustering and classifying your cells.nb.html\n\n\n# monocle3 tutorial --- constructing single-cell trajectories\n\nhttps://cole-trapnell-lab.github.io/monocle3/docs/trajectories/\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//008.monocle3/constructing single-cell trajectories.nb.html",charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"hdWGCNAhigh dimensional WGCNA",frontmatter:{title:"hdWGCNAhigh dimensional WGCNA",date:"2023-05-04T08:25:04.000Z",permalink:"/pages/649303/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/007.hdWGCNA.html",relativePath:"03./01. - /007.hdWGCNA.md",key:"v-97fad502",path:"/pages/649303/",headers:[{level:2,title:"hdWGCNA ",slug:"hdwgcna-",normalizedTitle:"hdwgcna ",charIndex:707},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:1020},{level:2,title:"hdWGCNA Vignettes",slug:"hdwgcna-vignettes",normalizedTitle:"hdwgcna vignettes",charIndex:1571},{level:3,title:"1 Co-expression network analysis",slug:"_1-co-expression-network-analysis",normalizedTitle:"1 co-expression network analysis",charIndex:1593},{level:4,title:"1.1 hdWGCNA in single-cell data",slug:"_1-1-hdwgcna-in-single-cell-data",normalizedTitle:"1.1 hdwgcna in single-cell data",charIndex:1672},{level:4,title:"1.2 hdWGCNA in spatial transcriptomics data",slug:"_1-2-hdwgcna-in-spatial-transcriptomics-data",normalizedTitle:"1.2 hdwgcna in spatial transcriptomics data",charIndex:1974},{level:4,title:"1.3 Isoform co-expression network analysis with PacBio MAS-Seq",slug:"_1-3-isoform-co-expression-network-analysis-with-pacbio-mas-seq",normalizedTitle:"1.3 isoform co-expression network analysis with pacbio mas-seq",charIndex:2129},{level:4,title:"1.4 Network visualization",slug:"_1-4-network-visualization",normalizedTitle:"1.4 network visualization",charIndex:2321},{level:3,title:"2 Biological context for co-expression modules",slug:"_2-biological-context-for-co-expression-modules",normalizedTitle:"2 biological context for co-expression modules",charIndex:2610},{level:4,title:"2.1 Differential module eigengene (DME) analysis",slug:"_2-1-differential-module-eigengene-dme-analysis",normalizedTitle:"2.1 differential module eigengene (dme) analysis",charIndex:2712},{level:4,title:"2.2 Module trait correlation",slug:"_2-2-module-trait-correlation",normalizedTitle:"2.2 module trait correlation",charIndex:3044},{level:4,title:"2.3 Enrichment analysis",slug:"_2-3-enrichment-analysis",normalizedTitle:"2.3 enrichment analysis",charIndex:3400},{level:3,title:"3 Exploring modules in external datasets",slug:"_3-exploring-modules-in-external-datasets",normalizedTitle:"3 exploring modules in external datasets",charIndex:3712},{level:4,title:"3.1 Projecting modules to new datasets",slug:"_3-1-projecting-modules-to-new-datasets",normalizedTitle:"3.1 projecting modules to new datasets",charIndex:3756},{level:4,title:"3.2 Module preservation and reproducibility",slug:"_3-2-module-preservation-and-reproducibility",normalizedTitle:"3.2 module preservation and reproducibility",charIndex:3914},{level:4,title:"3.3 Cross-species and cross-modality analysis",slug:"_3-3-cross-species-and-cross-modality-analysis",normalizedTitle:"3.3 cross-species and cross-modality analysis",charIndex:4073},{level:3,title:"4 Advanced topics",slug:"_4-advanced-topics",normalizedTitle:"4 advanced topics",charIndex:4285},{level:4,title:"4.1 Consensus network analysis",slug:"_4-1-consensus-network-analysis",normalizedTitle:"4.1 consensus network analysis",charIndex:4306},{level:4,title:"4.2 Co-expression module dynamics with pseudotime",slug:"_4-2-co-expression-module-dynamics-with-pseudotime",normalizedTitle:"4.2 co-expression module dynamics with pseudotime",charIndex:4444},{level:4,title:"4.3 Motif Analysis",slug:"_4-3-motif-analysis",normalizedTitle:"4.3 motif analysis",charIndex:4592},{level:4,title:"4.4 Alternative metacell algorithms",slug:"_4-4-alternative-metacell-algorithms",normalizedTitle:"4.4 alternative metacell algorithms",charIndex:4684},{level:3,title:"5 Other",slug:"_5-other",normalizedTitle:"5 other",charIndex:4777},{level:4,title:"5.1 Module customization",slug:"_5-1-module-customization",normalizedTitle:"5.1 module customization",charIndex:4788},{level:4,title:"5.2 Using SCTransform normalized data",slug:"_5-2-using-sctransform-normalized-data",normalizedTitle:"5.2 using sctransform normalized data",charIndex:4917}],excerpt:'<p>GitHub: <a href="https://github.com/smorabit/hdWGCNA/" target="_blank" rel="noopener noreferrer">https://github.com/smorabit/hdWGCNA/<OutboundLink/></a><br>\nDoc<a href="https://smorabit.github.io/hdWGCNA/" target="_blank" rel="noopener noreferrer">https://smorabit.github.io/hdWGCNA/<OutboundLink/></a></p>\n',headersStr:"hdWGCNA  Installation hdWGCNA Vignettes 1 Co-expression network analysis 1.1 hdWGCNA in single-cell data 1.2 hdWGCNA in spatial transcriptomics data 1.3 Isoform co-expression network analysis with PacBio MAS-Seq 1.4 Network visualization 2 Biological context for co-expression modules 2.1 Differential module eigengene (DME) analysis 2.2 Module trait correlation 2.3 Enrichment analysis 3 Exploring modules in external datasets 3.1 Projecting modules to new datasets 3.2 Module preservation and reproducibility 3.3 Cross-species and cross-modality analysis 4 Advanced topics 4.1 Consensus network analysis 4.2 Co-expression module dynamics with pseudotime 4.3 Motif Analysis 4.4 Alternative metacell algorithms 5 Other 5.1 Module customization 5.2 Using SCTransform normalized data",content:'GitHub: https://github.com/smorabit/hdWGCNA/\nDochttps://smorabit.github.io/hdWGCNA/\n\n\n\n 1.  --- WGCNAan R package for weighted correlation network analysis\n 2. WGCNAan R package for weighted correlation network analysis\n 3.  --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer\'s disease\n 4.  --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdWGCNAhigh dimensional WGCNA\n\n\n\nSAMUEL MORABITO                        VIVEK SWARUP\n                                       \nUniversity of California Irvine(UCI)   University of California Irvine(UCI)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# hdWGCNA \n\nhdWGCNA  R  scRNA-seq (WGCNA)hdWGCNA hdWGNCA hdWGCNA  Seurat \n\nWGCNA :\n\n * Morabito et al. bioRxiv 2022\n * Morabito & Miyoshi et al. Nature Genetics 2021\n\n\n# Installation\n\n hdWGCNA  R conda \n\n# create new conda environment for R\nconda create -n hdWGCNA -c conda-forge r-base r-essentials\n\n# activate conda environment\nconda activate hdWGCNA\n\n\n1\n2\n3\n4\n5\n\n\n R \n\n# install BiocManager\ninstall.packages("BiocManager")\n\n# install Bioconductor core packages\nBiocManager::install()\n\n# install additional packages:\ninstall.packages(c("Seurat", "WGCNA", "igraph", "devtools"))\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n devtools  hdWGCNA \n\ndevtools::install_github(\'smorabit/hdWGCNA\', ref=\'dev\')\n\n\n1\n\n\n\n# hdWGCNA Vignettes\n\n\n# 1 Co-expression network analysis\n\n\n\n# 1.1 hdWGCNA in single-cell data\n\n hdWGCNA \nhttps://smorabit.github.io/hdWGCNA/articles/basic_tutorial.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//007.hdWGCNA/hdWGCNA in single-cell data.nb.html\n\n\n\n# 1.2 hdWGCNA in spatial transcriptomics data\n\n hdWGCNA \nhttps://smorabit.github.io/hdWGCNA/articles/ST_basics.html\n\n\n\n# 1.3 Isoform co-expression network analysis with PacBio MAS-Seq\n\n hdWGCNA  PacBio-MAS-Seq \nhttps://smorabit.github.io/hdWGCNA/articles/isoform_pbmcs.html\n\n# 1.4 Network visualization\n\n hdWGCNA \nhttps://smorabit.github.io/hdWGCNA/articles/network_visualizations.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//007.hdWGCNA/Network Visualization.nb.html\n\n\n\n\n# 2 Biological context for co-expression modules\n\n\n\n# 2.1 Differential module eigengene (DME) analysis\n\n module eigengenes\nhttps://smorabit.github.io/hdWGCNA/articles/differential_MEs.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//007.hdWGCNA/Differential module eigengene (DME) analysis.nb.html\n\n\n\n# 2.2 Module trait correlation\n\n module eigengenes  module expression scores \nhttps://smorabit.github.io/hdWGCNA/articles/module_trait_correlation.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//007.hdWGCNA/Module Trait Correlation.nb.html\n\n\n\n# 2.3 Enrichment analysis\n\n Enrichr \nhttps://smorabit.github.io/hdWGCNA/articles/enrichment_analysis.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//007.hdWGCNA/Enrichment analysis.nb.html\n\n\n\n\n# 3 Exploring modules in external datasets\n\n# 3.1 Projecting modules to new datasets\n\n reference  query \nhttps://smorabit.github.io/hdWGCNA/articles/projecting_modules.html\n\n\n\n# 3.2 Module preservation and reproducibility\n\n\nhttps://smorabit.github.io/hdWGCNA/articles/module_preservation.html\n\n\n\n# 3.3 Cross-species and cross-modality analysis\n\n reference  query  reference  query \nhttps://smorabit.github.io/hdWGCNA/articles/projecting_modules_cross.html\n\n\n\n\n# 4 Advanced topics\n\n# 4.1 Consensus network analysis\n\n hdWGCNA \nhttps://smorabit.github.io/hdWGCNA/articles/consensus_wgcna.html\n\n# 4.2 Co-expression module dynamics with pseudotime\n\n\nhttps://smorabit.github.io/hdWGCNA/articles/pseudotime.html\n\n# 4.3 Motif Analysis\n\nhttps://smorabit.github.io/hdWGCNA/articles/motif_analysis.html\n\n# 4.4 Alternative metacell algorithms\n\n metacell \n\n\n# 5 Other\n\n# 5.1 Module customization\n\n hdWGCNA \nhttps://smorabit.github.io/hdWGCNA/articles/customization.html\n\n\n\n# 5.2 Using SCTransform normalized data\n\n hdWGCNA  SCTransform normalized \nhttps://smorabit.github.io/hdWGCNA/articles/sctransform.html\n\n',normalizedContent:'github: https://github.com/smorabit/hdwgcna/\ndochttps://smorabit.github.io/hdwgcna/\n\n\n\n 1.  --- wgcnaan r package for weighted correlation network analysis\n 2. wgcnaan r package for weighted correlation network analysis\n 3.  --- single-nucleus chromatin accessibility and transcriptomic characterization of alzheimer\'s disease\n 4.  --- hdwgcna identifies co-expression networks in high-dimensional transcriptomics data\n 5. hdwgcnahigh dimensional wgcna\n\n\n\nsamuel morabito                        vivek swarup\n                                       \nuniversity of california irvine(uci)   university of california irvine(uci)\nsmorabit@uci.edu                       vswarup@uci.edu\n\n\n# hdwgcna \n\nhdwgcna  r  scrna-seq (wgcna)hdwgcna hdwgnca hdwgcna  seurat \n\nwgcna :\n\n * morabito et al. biorxiv 2022\n * morabito & miyoshi et al. nature genetics 2021\n\n\n# installation\n\n hdwgcna  r conda \n\n# create new conda environment for r\nconda create -n hdwgcna -c conda-forge r-base r-essentials\n\n# activate conda environment\nconda activate hdwgcna\n\n\n1\n2\n3\n4\n5\n\n\n r \n\n# install biocmanager\ninstall.packages("biocmanager")\n\n# install bioconductor core packages\nbiocmanager::install()\n\n# install additional packages:\ninstall.packages(c("seurat", "wgcna", "igraph", "devtools"))\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n devtools  hdwgcna \n\ndevtools::install_github(\'smorabit/hdwgcna\', ref=\'dev\')\n\n\n1\n\n\n\n# hdwgcna vignettes\n\n\n# 1 co-expression network analysis\n\n\n\n# 1.1 hdwgcna in single-cell data\n\n hdwgcna \nhttps://smorabit.github.io/hdwgcna/articles/basic_tutorial.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//007.hdwgcna/hdwgcna in single-cell data.nb.html\n\n\n\n# 1.2 hdwgcna in spatial transcriptomics data\n\n hdwgcna \nhttps://smorabit.github.io/hdwgcna/articles/st_basics.html\n\n\n\n# 1.3 isoform co-expression network analysis with pacbio mas-seq\n\n hdwgcna  pacbio-mas-seq \nhttps://smorabit.github.io/hdwgcna/articles/isoform_pbmcs.html\n\n# 1.4 network visualization\n\n hdwgcna \nhttps://smorabit.github.io/hdwgcna/articles/network_visualizations.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//007.hdwgcna/network visualization.nb.html\n\n\n\n\n# 2 biological context for co-expression modules\n\n\n\n# 2.1 differential module eigengene (dme) analysis\n\n module eigengenes\nhttps://smorabit.github.io/hdwgcna/articles/differential_mes.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//007.hdwgcna/differential module eigengene (dme) analysis.nb.html\n\n\n\n# 2.2 module trait correlation\n\n module eigengenes  module expression scores \nhttps://smorabit.github.io/hdwgcna/articles/module_trait_correlation.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//007.hdwgcna/module trait correlation.nb.html\n\n\n\n# 2.3 enrichment analysis\n\n enrichr \nhttps://smorabit.github.io/hdwgcna/articles/enrichment_analysis.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//007.hdwgcna/enrichment analysis.nb.html\n\n\n\n\n# 3 exploring modules in external datasets\n\n# 3.1 projecting modules to new datasets\n\n reference  query \nhttps://smorabit.github.io/hdwgcna/articles/projecting_modules.html\n\n\n\n# 3.2 module preservation and reproducibility\n\n\nhttps://smorabit.github.io/hdwgcna/articles/module_preservation.html\n\n\n\n# 3.3 cross-species and cross-modality analysis\n\n reference  query  reference  query \nhttps://smorabit.github.io/hdwgcna/articles/projecting_modules_cross.html\n\n\n\n\n# 4 advanced topics\n\n# 4.1 consensus network analysis\n\n hdwgcna \nhttps://smorabit.github.io/hdwgcna/articles/consensus_wgcna.html\n\n# 4.2 co-expression module dynamics with pseudotime\n\n\nhttps://smorabit.github.io/hdwgcna/articles/pseudotime.html\n\n# 4.3 motif analysis\n\nhttps://smorabit.github.io/hdwgcna/articles/motif_analysis.html\n\n# 4.4 alternative metacell algorithms\n\n metacell \n\n\n# 5 other\n\n# 5.1 module customization\n\n hdwgcna \nhttps://smorabit.github.io/hdwgcna/articles/customization.html\n\n\n\n# 5.2 using sctransform normalized data\n\n hdwgcna  sctransform normalized \nhttps://smorabit.github.io/hdwgcna/articles/sctransform.html\n\n',charsets:{cjk:!0},lastUpdated:"2023/07/05, 20:21:07",lastUpdatedTimestamp:1688559667e3},{title:"CellChatInference and analysis of cell-cell communication",frontmatter:{title:"CellChatInference and analysis of cell-cell communication",date:"2023-05-15T15:26:27.000Z",permalink:"/pages/f26000/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/009.CellChat.html",relativePath:"03./01. - /009.CellChat.md",key:"v-bf406dce",path:"/pages/f26000/",headers:[{level:2,title:"CellChat ",slug:"cellchat-",normalizedTitle:"cellchat ",charIndex:203},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:543},{level:2,title:"Tutorials",slug:"tutorials",normalizedTitle:"tutorials",charIndex:609},{level:3,title:"Full tutorial for CellChat analysis of a single dataset with detailed explanation of each function",slug:"full-tutorial-for-cellchat-analysis-of-a-single-dataset-with-detailed-explanation-of-each-function",normalizedTitle:"full tutorial for cellchat analysis of a single dataset with detailed explanation of each function",charIndex:623},{level:3,title:"Brief tutorial for CellChat analysis of a single spatial imaging dataset",slug:"brief-tutorial-for-cellchat-analysis-of-a-single-spatial-imaging-dataset",normalizedTitle:"brief tutorial for cellchat analysis of a single spatial imaging dataset",charIndex:888},{level:3,title:"Full tutorial for comparison analysis of multiple datasets",slug:"full-tutorial-for-comparison-analysis-of-multiple-datasets",normalizedTitle:"full tutorial for comparison analysis of multiple datasets",charIndex:1104},{level:3,title:"Comparison analysis of multiple datasets with different cellular compositions",slug:"comparison-analysis-of-multiple-datasets-with-different-cellular-compositions",normalizedTitle:"comparison analysis of multiple datasets with different cellular compositions",charIndex:1305},{level:3,title:"Interface with other single-cell analysis toolkits (e.g., Seurat, SingleCellExperiment, Scanpy)",slug:"interface-with-other-single-cell-analysis-toolkits-e-g-seurat-singlecellexperiment-scanpy",normalizedTitle:"interface with other single-cell analysis toolkits (e.g., seurat, singlecellexperiment, scanpy)",charIndex:1562},{level:3,title:"Tutorial for updating ligand-receptor database CellChatDB",slug:"tutorial-for-updating-ligand-receptor-database-cellchatdb",normalizedTitle:"tutorial for updating ligand-receptor database cellchatdb",charIndex:1810}],excerpt:'<p>GitHub: <a href="https://github.com/sqjin/CellChat" target="_blank" rel="noopener noreferrer">https://github.com/sqjin/CellChat<OutboundLink/></a></p>\n',headersStr:"CellChat  Installation Tutorials Full tutorial for CellChat analysis of a single dataset with detailed explanation of each function Brief tutorial for CellChat analysis of a single spatial imaging dataset Full tutorial for comparison analysis of multiple datasets Comparison analysis of multiple datasets with different cellular compositions Interface with other single-cell analysis toolkits (e.g., Seurat, SingleCellExperiment, Scanpy) Tutorial for updating ligand-receptor database CellChatDB",content:'GitHub: https://github.com/sqjin/CellChat\n\n\n\n\n\n 1.  CellChat -\n 2. CellChat v2  1 CellChat \n\n\n\nQING NIE\n\nUniversity of California, Irvine\nqnie@uci.edu\n\n\n# CellChat \n\n\n\nCellChat  scRNA-seq \n\n * \n * \n * \n * \n\nNeuronChat \n\n 1. Jin et al., Nature Communications, 2021\n\n\n# Installation\n\ndevtools::install_github("sqjin/CellChat")\n\n\n1\n\n\n\n# Tutorials\n\n\n# Full tutorial for CellChat analysis of a single dataset with detailed explanation of each function\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/CellChat-vignette.html\n CellChat v2  1 CellChat \n\n\n# Brief tutorial for CellChat analysis of a single spatial imaging dataset\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/CellChat_analysis_of_spatial_imaging_data.html\n\n\n# Full tutorial for comparison analysis of multiple datasets\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/Comparison_analysis_of_multiple_datasets.html\n\n\n# Comparison analysis of multiple datasets with different cellular compositions\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/Comparison_analysis_of_multiple_datasets_with_different_cellular_compositions.html\n\n\n# Interface with other single-cell analysis toolkits (e.g., Seurat, SingleCellExperiment, Scanpy)\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/Interface_with_other_single-cell_analysis_toolkits.html\n\n\n# Tutorial for updating ligand-receptor database CellChatDB\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/CellChat/blob/master/tutorial/Update-CellChatDB.html',normalizedContent:'github: https://github.com/sqjin/cellchat\n\n\n\n\n\n 1.  cellchat -\n 2. cellchat v2  1 cellchat \n\n\n\nqing nie\n\nuniversity of california, irvine\nqnie@uci.edu\n\n\n# cellchat \n\n\n\ncellchat  scrna-seq \n\n * \n * \n * \n * \n\nneuronchat \n\n 1. jin et al., nature communications, 2021\n\n\n# installation\n\ndevtools::install_github("sqjin/cellchat")\n\n\n1\n\n\n\n# tutorials\n\n\n# full tutorial for cellchat analysis of a single dataset with detailed explanation of each function\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/cellchat-vignette.html\n cellchat v2  1 cellchat \n\n\n# brief tutorial for cellchat analysis of a single spatial imaging dataset\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/cellchat_analysis_of_spatial_imaging_data.html\n\n\n# full tutorial for comparison analysis of multiple datasets\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/comparison_analysis_of_multiple_datasets.html\n\n\n# comparison analysis of multiple datasets with different cellular compositions\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/comparison_analysis_of_multiple_datasets_with_different_cellular_compositions.html\n\n\n# interface with other single-cell analysis toolkits (e.g., seurat, singlecellexperiment, scanpy)\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/interface_with_other_single-cell_analysis_toolkits.html\n\n\n# tutorial for updating ligand-receptor database cellchatdb\n\nhttps://htmlpreview.github.io/?https://github.com/sqjin/cellchat/blob/master/tutorial/update-cellchatdb.html',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"NeuronChatInferring neuron-neuron communications from single-cell transcriptomics",frontmatter:{title:"NeuronChatInferring neuron-neuron communications from single-cell transcriptomics",date:"2023-04-26T20:22:25.000Z",permalink:"/pages/7a726f/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/010.NeuronChat.html",relativePath:"03./01. - /010.NeuronChat.md",key:"v-434e1f39",path:"/pages/7a726f/",headers:[{level:2,title:"NeuronChat ",slug:"neuronchat-",normalizedTitle:"neuronchat ",charIndex:457},{level:2,title:"NeuronChat ",slug:"neuronchat-",normalizedTitle:"neuronchat ",charIndex:596},{level:2,title:"Usage & Tutorial",slug:"usage-tutorial",normalizedTitle:"usage &amp; tutorial",charIndex:null},{level:3,title:"Basic usage of NeuronChat",slug:"basic-usage-of-neuronchat",normalizedTitle:"basic usage of neuronchat",charIndex:722},{level:3,title:"Full tutorials",slug:"full-tutorials",normalizedTitle:"full tutorials",charIndex:1173},{level:4,title:"1.",slug:"_1-",normalizedTitle:"1.",charIndex:1212},{level:4,title:"2.",slug:"_2-",normalizedTitle:"2.",charIndex:1567},{level:4,title:"3. NeuronChat ",slug:"_3--neuronchat-",normalizedTitle:"3. neuronchat ",charIndex:1712},{level:4,title:"4.NeuronChat predicting neural connectivity",slug:"_4-neuronchat-predicting-neural-connectivity",normalizedTitle:"4.neuronchat predicting neural connectivity",charIndex:1868}],excerpt:'<p>GitHub: <a href="https://github.com/Wei-BioMath/NeuronChat" target="_blank" rel="noopener noreferrer">https://github.com/Wei-BioMath/NeuronChat<OutboundLink/></a></p>\n',headersStr:"NeuronChat  NeuronChat  Usage & Tutorial Basic usage of NeuronChat Full tutorials 1. 2. 3. NeuronChat  4.NeuronChat predicting neural connectivity",content:"GitHub: https://github.com/Wei-BioMath/NeuronChat\n\n\n\n 1.  --- Inference and analysis of cell-cell communication using CellChat\n 2. CellChatInference and analysis of cell-cell communication\n 3.  --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat\n 4. NeuronChatInferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nQING NIE\n\nUniversity of California, Irvine\nqnie@uci.edu\n\n\n# NeuronChat \n\nNeuronChat \n\nNeuronChat :\n\n * Zhao et al., Nat Commun 2023 [NeuronChat]\n\n\n# NeuronChat \n\n NeuronChat \n\ndevtools::install_github(\"Wei-BioMath/NeuronChat\")\n\n\n1\n\n\n\n# Usage & Tutorial\n\n\n# Basic usage of NeuronChat\n\nlibrary(NeuronChat)\n# creat NeuronChat object \nx <- createNeuronChat(normalized_count_mtx,DB='mouse',group.by = cell_group_vector) # use DB='human' for human data\n# calculation of communication networks  \nx <- run_NeuronChat(x,M=100)\n# aggregating networks over interaction pairs\nnet_aggregated_x <- net_aggregation(x@net,method = 'weight')\n# visualization\nnetVisual_circle_neuron(net_aggregated_x)\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# Full tutorials\n\nNeuronChat \n\n# 1.\n\nhttps://htmlpreview.github.io/?https://github.com/Wei-BioMath/NeuronChat/blob/main/vignettes/NeuronChat-Tutorial.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//010.NeuronChat/Inference visualization and analysis of neural-specific communication network.nb.html\n\n# 2.\n\nhttps://htmlpreview.github.io/?https://github.com/Wei-BioMath/NeuronChat/blob/main/vignettes/Spatial_analysis.html\n\n# 3. NeuronChat \n\nhttps://htmlpreview.github.io/?https://github.com/Wei-BioMath/NeuronChat/blob/main/vignettes/Update_NeuronChat_database.html\n\n# 4.NeuronChat predicting neural connectivity\n\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//010.NeuronChat/NeuronChat predicting neural connectivity.nb.html",normalizedContent:"github: https://github.com/wei-biomath/neuronchat\n\n\n\n 1.  --- inference and analysis of cell-cell communication using cellchat\n 2. cellchatinference and analysis of cell-cell communication\n 3.  --- inferring neuron-neuron communications from single-cell transcriptomics through neuronchat\n 4. neuronchatinferring neuron-neuron communications from single-cell transcriptomics\n\n\n\nqing nie\n\nuniversity of california, irvine\nqnie@uci.edu\n\n\n# neuronchat \n\nneuronchat \n\nneuronchat :\n\n * zhao et al., nat commun 2023 [neuronchat]\n\n\n# neuronchat \n\n neuronchat \n\ndevtools::install_github(\"wei-biomath/neuronchat\")\n\n\n1\n\n\n\n# usage & tutorial\n\n\n# basic usage of neuronchat\n\nlibrary(neuronchat)\n# creat neuronchat object \nx <- createneuronchat(normalized_count_mtx,db='mouse',group.by = cell_group_vector) # use db='human' for human data\n# calculation of communication networks  \nx <- run_neuronchat(x,m=100)\n# aggregating networks over interaction pairs\nnet_aggregated_x <- net_aggregation(x@net,method = 'weight')\n# visualization\nnetvisual_circle_neuron(net_aggregated_x)\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# full tutorials\n\nneuronchat \n\n# 1.\n\nhttps://htmlpreview.github.io/?https://github.com/wei-biomath/neuronchat/blob/main/vignettes/neuronchat-tutorial.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//010.neuronchat/inference visualization and analysis of neural-specific communication network.nb.html\n\n# 2.\n\nhttps://htmlpreview.github.io/?https://github.com/wei-biomath/neuronchat/blob/main/vignettes/spatial_analysis.html\n\n# 3. neuronchat \n\nhttps://htmlpreview.github.io/?https://github.com/wei-biomath/neuronchat/blob/main/vignettes/update_neuronchat_database.html\n\n# 4.neuronchat predicting neural connectivity\n\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//010.neuronchat/neuronchat predicting neural connectivity.nb.html",charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"scCustomizeA collection of functions created and/or curated to aid in the visualization and analysis of single-cell data using R",frontmatter:{title:"scCustomizeA collection of functions created and/or curated to aid in the visualization and analysis of single-cell data using R",date:"2023-05-14T09:45:23.000Z",permalink:"/pages/9e5aae/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/011.scCustomize.html",relativePath:"03./01. - /011.scCustomize.md",key:"v-d477d30a",path:"/pages/9e5aae/",headers:[{level:2,title:"scCustomize ",slug:"sccustomize-",normalizedTitle:"sccustomize ",charIndex:115},{level:2,title:"Installing scCustomize",slug:"installing-sccustomize",normalizedTitle:"installing sccustomize",charIndex:3e3},{level:2,title:"Plotting Vignettes",slug:"plotting-vignettes",normalizedTitle:"plotting vignettes",charIndex:3065},{level:3,title:"Plotting #1: Analysis Plots",slug:"plotting-1-analysis-plots",normalizedTitle:"plotting #1: analysis plots",charIndex:3088},{level:3,title:"Plotting #2: QC Plots",slug:"plotting-2-qc-plots",normalizedTitle:"plotting #2: qc plots",charIndex:3367},{level:3,title:"Plotting #3: Sequencing QC Plots/Analysis",slug:"plotting-3-sequencing-qc-plots-analysis",normalizedTitle:"plotting #3: sequencing qc plots/analysis",charIndex:3465},{level:3,title:"Plotting #4: Iterative Plotting Functions",slug:"plotting-4-iterative-plotting-functions",normalizedTitle:"plotting #4: iterative plotting functions",charIndex:3594},{level:3,title:"Customized Color Palettes & Themes",slug:"customized-color-palettes-themes",normalizedTitle:"customized color palettes &amp; themes",charIndex:null}],excerpt:'<p>GitHub: <a href="https://github.com/samuel-marsh/scCustomize" target="_blank" rel="noopener noreferrer">https://github.com/samuel-marsh/scCustomize<OutboundLink/></a><br>\nDoc: <a href="https://samuel-marsh.github.io/scCustomize/index.html" target="_blank" rel="noopener noreferrer">https://samuel-marsh.github.io/scCustomize/index.html<OutboundLink/></a></p>\n',headersStr:"scCustomize  Installing scCustomize Plotting Vignettes Plotting #1: Analysis Plots Plotting #2: QC Plots Plotting #3: Sequencing QC Plots/Analysis Plotting #4: Iterative Plotting Functions Customized Color Palettes & Themes",content:'GitHub: https://github.com/samuel-marsh/scCustomize\nDoc: https://samuel-marsh.github.io/scCustomize/index.html\n\n\n# scCustomize \n\n\n\nscCustomize  R \n\nscCustomize \n\n 1. \n 2.  scRNA-seq \n\nscCustomize \n\n * \n   \n   * FeaturePlot(..., order = TRUE)\n   *  ggplot2 /plot + scale_color_continuous(...) + ggtitle(...) + theme(plot.title = element_text(...), legend.position = ...) + guides(...)\n   *  1.  Seurat (e.g., QC_Plot_Genes()) 2. (e.g., Seq_QC_Plot_Reads_per_Cell() or Plot_Median_Genes()) 3. both(e.g., QC_Plot_UMIvsGene(..., combination = TRUE))\n   * (e.g., high and low cutoff parameters in QC_Plot_UMIvsGene())\n *   loopsapplypurrr \n   * scCustomize \n   *  PDF (e.g., png, tiff, jpeg, pdf, etc)\n *  //\n   * /\n   * \n   *  Seurat  R  CellBender\n *  /\n   * /scCustomize  Add_Mito_Ribo_Seurat() LIGER  Add_Mito_Ribo_Seurat(seurat_object = obj_name, species = "Human")\n     *  HumanMouseRatZebrafishDrosophilaMarmosetRhesus Macaque  PR\n   *  Seurat::FindAllMarkers()  identity  top10 152025 scCustomize  `Extract_Top_Markers()``   scCustomize \n   \n   markers_df <- FindAllMarkers(object = obj_name)\n   \n   # Get vector/string\n   top10_list <- Extract_Top_Markers(marker_dataframe = markers_df)\n   \n   # or for data.frame\n   top10_df <- Extract_Top_Markers(marker_dataframe = markers_df, dataframe = TRUE)\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   \n   \n   \n   markers_df <- FindAllMarkers(object = obj_name)\n   \n   # Get vector/string\n   top10_list <- markers_df %>%\n     rownames_to_column("rownames") %>%\n     group_by(cluster) %>%\n     slice_max(n = 10, order_by = avg_log2FC) %>%\n     pull("gene")\n   \n   # or for data.frame\n   top10_df <- markers_df %>%\n     rownames_to_column("rownames") %>%\n     group_by(cluster) %>%\n     slice_max(n = 10, order_by = avg_log2FC) %>%\n     column_to_rownames("rownames")\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   13\n   14\n   15\n   \n *   Seurat Base R  R \n   * scCustomize / cli/rlang /\n     * Add_Mito_Ribo_Seurat()  metadata \n     * Rename_Clusters() \n\n\n# Installing scCustomize\n\ninstall.packages("scCustomize")\n\n\n1\n\n\n\n# Plotting Vignettes\n\n\n# Plotting #1: Analysis Plots\n\nhttps://samuel-marsh.github.io/scCustomize/articles/Gene_Expression_Plotting.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//011.scCustomize/Plotting %231 Analysis Plots.nb.html\n\n\n# Plotting #2: QC Plots\n\nhttps://samuel-marsh.github.io/scCustomize/articles/QC_Plots.html\n\n\n# Plotting #3: Sequencing QC Plots/Analysis\n\nhttps://samuel-marsh.github.io/scCustomize/articles/Sequencing_QC_Plots.html\n\n\n# Plotting #4: Iterative Plotting Functions\n\nhttps://samuel-marsh.github.io/scCustomize/articles/Iterative_Plotting.html\n\n\n# Customized Color Palettes & Themes\n\nhttps://samuel-marsh.github.io/scCustomize/articles/Color_Palettes.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//011.scCustomize/Customized Color Palettes & Themes.nb.html',normalizedContent:'github: https://github.com/samuel-marsh/sccustomize\ndoc: https://samuel-marsh.github.io/sccustomize/index.html\n\n\n# sccustomize \n\n\n\nsccustomize  r \n\nsccustomize \n\n 1. \n 2.  scrna-seq \n\nsccustomize \n\n * \n   \n   * featureplot(..., order = true)\n   *  ggplot2 /plot + scale_color_continuous(...) + ggtitle(...) + theme(plot.title = element_text(...), legend.position = ...) + guides(...)\n   *  1.  seurat (e.g., qc_plot_genes()) 2. (e.g., seq_qc_plot_reads_per_cell() or plot_median_genes()) 3. both(e.g., qc_plot_umivsgene(..., combination = true))\n   * (e.g., high and low cutoff parameters in qc_plot_umivsgene())\n *   loopsapplypurrr \n   * sccustomize \n   *  pdf (e.g., png, tiff, jpeg, pdf, etc)\n *  //\n   * /\n   * \n   *  seurat  r  cellbender\n *  /\n   * /sccustomize  add_mito_ribo_seurat() liger  add_mito_ribo_seurat(seurat_object = obj_name, species = "human")\n     *  humanmouseratzebrafishdrosophilamarmosetrhesus macaque  pr\n   *  seurat::findallmarkers()  identity  top10 152025 sccustomize  `extract_top_markers()``   sccustomize \n   \n   markers_df <- findallmarkers(object = obj_name)\n   \n   # get vector/string\n   top10_list <- extract_top_markers(marker_dataframe = markers_df)\n   \n   # or for data.frame\n   top10_df <- extract_top_markers(marker_dataframe = markers_df, dataframe = true)\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   \n   \n   \n   markers_df <- findallmarkers(object = obj_name)\n   \n   # get vector/string\n   top10_list <- markers_df %>%\n     rownames_to_column("rownames") %>%\n     group_by(cluster) %>%\n     slice_max(n = 10, order_by = avg_log2fc) %>%\n     pull("gene")\n   \n   # or for data.frame\n   top10_df <- markers_df %>%\n     rownames_to_column("rownames") %>%\n     group_by(cluster) %>%\n     slice_max(n = 10, order_by = avg_log2fc) %>%\n     column_to_rownames("rownames")\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   13\n   14\n   15\n   \n *   seurat base r  r \n   * sccustomize / cli/rlang /\n     * add_mito_ribo_seurat()  metadata \n     * rename_clusters() \n\n\n# installing sccustomize\n\ninstall.packages("sccustomize")\n\n\n1\n\n\n\n# plotting vignettes\n\n\n# plotting #1: analysis plots\n\nhttps://samuel-marsh.github.io/sccustomize/articles/gene_expression_plotting.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//011.sccustomize/plotting %231 analysis plots.nb.html\n\n\n# plotting #2: qc plots\n\nhttps://samuel-marsh.github.io/sccustomize/articles/qc_plots.html\n\n\n# plotting #3: sequencing qc plots/analysis\n\nhttps://samuel-marsh.github.io/sccustomize/articles/sequencing_qc_plots.html\n\n\n# plotting #4: iterative plotting functions\n\nhttps://samuel-marsh.github.io/sccustomize/articles/iterative_plotting.html\n\n\n# customized color palettes & themes\n\nhttps://samuel-marsh.github.io/sccustomize/articles/color_palettes.html\nhttps://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//011.sccustomize/customized color palettes & themes.nb.html',charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"ScanpySingle-Cell Analysis in Python",frontmatter:{title:"ScanpySingle-Cell Analysis in Python",date:"2023-09-28T19:53:16.000Z",permalink:"/pages/61a60f/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/013.scanpy.html",relativePath:"03./01. - /013.scanpy.md",key:"v-2833738e",path:"/pages/61a60f/",headers:[{level:2,title:"Scanpy ",slug:"scanpy-",normalizedTitle:"scanpy ",charIndex:260}],excerpt:'<p>GitHub: <a href="https://github.com/scverse/scanpy" target="_blank" rel="noopener noreferrer">https://github.com/scverse/scanpy<OutboundLink/></a><br>\nDoc<a href="https://scanpy.readthedocs.io/en/stable/" target="_blank" rel="noopener noreferrer">https://scanpy.readthedocs.io/en/stable/<OutboundLink/></a></p>\n',headersStr:"Scanpy ",content:"GitHub: https://github.com/scverse/scanpy\nDochttps://scanpy.readthedocs.io/en/stable/\n\n\n\n\n\n 1. SCANPY\n 2. scanpy  1 3k PBMCs\n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# Scanpy \n\nScanpy  anndata  Python \n\n *  Discourse  GitHub \n *  turorialsusage principles  API \n *  release notes \n *  external API  ecosystem page  anndata  Scanpy \n *  contributing guide\n *  Genome Biology (2018)  references",normalizedContent:"github: https://github.com/scverse/scanpy\ndochttps://scanpy.readthedocs.io/en/stable/\n\n\n\n\n\n 1. scanpy\n 2. scanpy  1 3k pbmcs\n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# scanpy \n\nscanpy  anndata  python \n\n *  discourse  github \n *  turorialsusage principles  api \n *  release notes \n *  external api  ecosystem page  anndata  scanpy \n *  contributing guide\n *  genome biology (2018)  references",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"squidpySpatial Single Cell Analysis in Python",frontmatter:{title:"squidpySpatial Single Cell Analysis in Python",date:"2024-07-26T20:42:25.000Z",permalink:"/pages/37f62d/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/014.squidpy.html",relativePath:"03./01. - /014.squidpy.md",key:"v-8d171136",path:"/pages/37f62d/",headers:[{level:2,title:"Squidpy ",slug:"squidpy-",normalizedTitle:"squidpy ",charIndex:348},{level:2,title:"Tutorials",slug:"tutorials",normalizedTitle:"tutorials",charIndex:460}],excerpt:'<p>GitHub: <a href="https://github.com/scverse/squidpy" target="_blank" rel="noopener noreferrer">https://github.com/scverse/squidpy<OutboundLink/></a><br>\nDoc<a href="https://squidpy.readthedocs.io/en/stable/" target="_blank" rel="noopener noreferrer">https://squidpy.readthedocs.io/en/stable/<OutboundLink/></a></p>\n',headersStr:"Squidpy  Tutorials",content:"GitHub: https://github.com/scverse/squidpy\nDochttps://squidpy.readthedocs.io/en/stable/\n\n\n\n\n\n 1. Squidpy: \n 2. squidpy  1 AnnData  Squidpy \n 3. squidpy  2ImageContainer \n 4. squidpy  3 Napari \n\n\n\nFABIAN J. THEIS\n\nTechnische Universitt Mnchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# Squidpy \n\nSquidpy  scanpy  anndata \n\n\n# Tutorials",normalizedContent:"github: https://github.com/scverse/squidpy\ndochttps://squidpy.readthedocs.io/en/stable/\n\n\n\n\n\n 1. squidpy: \n 2. squidpy  1 anndata  squidpy \n 3. squidpy  2imagecontainer \n 4. squidpy  3 napari \n\n\n\nfabian j. theis\n\ntechnische universitat munchen\nfabian.theis@helmholtz-muenchen.de\n\n\n# squidpy \n\nsquidpy  scanpy  anndata \n\n\n# tutorials",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"XGBoostScalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library",frontmatter:{title:"XGBoostScalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library",date:"2023-05-10T20:43:26.000Z",permalink:"/pages/006dc9/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/02.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/001.XGBoost.html",relativePath:"03./02. - /001.XGBoost.md",key:"v-35793acf",path:"/pages/006dc9/",headers:[{level:2,title:"XGBoost ",slug:"xgboost-",normalizedTitle:"xgboost ",charIndex:227},{level:2,title:"R Package",slug:"r-package",normalizedTitle:"r package",charIndex:405},{level:3,title:"XGBoost R Tutorial",slug:"xgboost-r-tutorial",normalizedTitle:"xgboost r tutorial",charIndex:419},{level:3,title:"Understand your dataset with XGBoost",slug:"understand-your-dataset-with-xgboost",normalizedTitle:"understand your dataset with xgboost",charIndex:781},{level:3,title:"SHAP for XGBoost in R: SHAPforxgboost",slug:"shap-for-xgboost-in-r-shapforxgboost",normalizedTitle:"shap for xgboost in r: shapforxgboost",charIndex:1176},{level:3,title:"Binary Classification",slug:"binary-classification",normalizedTitle:"binary classification",charIndex:1572}],excerpt:'<p>GitHub: <a href="https://github.com/dmlc/xgboost" target="_blank" rel="noopener noreferrer">https://github.com/dmlc/xgboost<OutboundLink/></a><br>\nDoc: <a href="https://xgboost.readthedocs.io/en/stable/index.html" target="_blank" rel="noopener noreferrer">https://xgboost.readthedocs.io/en/stable/index.html<OutboundLink/></a></p>\n',headersStr:"XGBoost  R Package XGBoost R Tutorial Understand your dataset with XGBoost SHAP for XGBoost in R: SHAPforxgboost Binary Classification",content:"GitHub: https://github.com/dmlc/xgboost\nDoc: https://xgboost.readthedocs.io/en/stable/index.html\n\n\n\n\n\n 1. XGBoost R  1 XGBoost  R \n 2. XGBoost R  2 XGBoost \n 3. XGBoost R  3SHAPforxgboost\n\n\n# XGBoost \n\n\n\nXGBoost XGBoost  GBDTGBMHadoopSGEMPI\n\n\n# R Package\n\n\n# XGBoost R Tutorial\n\nhttps://xgboost.readthedocs.io/en/stable/R-package/xgboostPresentation.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.XGBoost/XGBoost R Tutorial.nb.html\nhttps://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&mid=2247485417&idx=1&sn=dc6af33884c985821db5a18ea8f1670e\n\n\n# Understand your dataset with XGBoost\n\nhttps://xgboost.readthedocs.io/en/stable/R-package/discoverYourData.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.XGBoost/Understand your dataset with XGBoost.nb.html\nhttps://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&mid=2247485441&idx=1&sn=4921764f6833a34d568dfac4afdee784\n\n\n# SHAP for XGBoost in R: SHAPforxgboost\n\nhttps://liuyanguu.github.io/post/2019/07/18/visualization-of-shap-for-xgboost/\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.XGBoost/visualization-of-shap-for-xgboost.html\nhttps://mp.weixin.qq.com/s?__biz=MzkwMjM0MzA5MA==&mid=2247485486&idx=1&sn=9adf56c32de5e0537a73056d8f568276\n\n\n# Binary Classification\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.XGBoost/Binary Classification.nb.html",normalizedContent:"github: https://github.com/dmlc/xgboost\ndoc: https://xgboost.readthedocs.io/en/stable/index.html\n\n\n\n\n\n 1. xgboost r  1 xgboost  r \n 2. xgboost r  2 xgboost \n 3. xgboost r  3shapforxgboost\n\n\n# xgboost \n\n\n\nxgboost xgboost  gbdtgbmhadoopsgempi\n\n\n# r package\n\n\n# xgboost r tutorial\n\nhttps://xgboost.readthedocs.io/en/stable/r-package/xgboostpresentation.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.xgboost/xgboost r tutorial.nb.html\nhttps://mp.weixin.qq.com/s?__biz=mzkwmjm0mza5ma==&mid=2247485417&idx=1&sn=dc6af33884c985821db5a18ea8f1670e\n\n\n# understand your dataset with xgboost\n\nhttps://xgboost.readthedocs.io/en/stable/r-package/discoveryourdata.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.xgboost/understand your dataset with xgboost.nb.html\nhttps://mp.weixin.qq.com/s?__biz=mzkwmjm0mza5ma==&mid=2247485441&idx=1&sn=4921764f6833a34d568dfac4afdee784\n\n\n# shap for xgboost in r: shapforxgboost\n\nhttps://liuyanguu.github.io/post/2019/07/18/visualization-of-shap-for-xgboost/\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.xgboost/visualization-of-shap-for-xgboost.html\nhttps://mp.weixin.qq.com/s?__biz=mzkwmjm0mza5ma==&mid=2247485486&idx=1&sn=9adf56c32de5e0537a73056d8f568276\n\n\n# binary classification\n\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//001.xgboost/binary classification.nb.html",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"TangramSpatial alignment of single cell transcriptomic data",frontmatter:{title:"TangramSpatial alignment of single cell transcriptomic data",date:"2023-09-19T09:58:09.000Z",permalink:"/pages/c7e56a/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/01.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%8D%95%E7%BB%86%E8%83%9E%E7%A9%BA%E9%97%B4%E7%BB%84/012.Tangram.html",relativePath:"03./01. - /012.Tangram.md",key:"v-66ffee6f",path:"/pages/c7e56a/",headers:[{level:2,title:"Tangram ",slug:"tangram-",normalizedTitle:"tangram ",charIndex:671},{level:2,title:"Tangram ",slug:"tangram-",normalizedTitle:"tangram ",charIndex:955},{level:2,title:"Tangram ",slug:"tangram-",normalizedTitle:"tangram ",charIndex:1394},{level:3,title:"Cell Level",slug:"cell-level",normalizedTitle:"cell level",charIndex:1411},{level:3,title:"Cluster Level",slug:"cluster-level",normalizedTitle:"cluster level",charIndex:2127},{level:2,title:"Tangram Tutorials",slug:"tangram-tutorials",normalizedTitle:"tangram tutorials",charIndex:2667},{level:3,title:"Tutorial 1: Tutorial for mapping data with Tangram",slug:"tutorial-1-tutorial-for-mapping-data-with-tangram",normalizedTitle:"tutorial 1: tutorial for mapping data with tangram",charIndex:2689},{level:3,title:"Tutorial 2: Tutorial for spatial mapping using Tangram",slug:"tutorial-2-tutorial-for-spatial-mapping-using-tangram",normalizedTitle:"tutorial 2: tutorial for spatial mapping using tangram",charIndex:3016}],excerpt:'<p>GitHub: <a href="https://github.com/broadinstitute/Tangram" target="_blank" rel="noopener noreferrer">https://github.com/broadinstitute/Tangram<OutboundLink/></a><br>\nDoc<a href="https://tangram-sc.readthedocs.io/en/latest/index.html" target="_blank" rel="noopener noreferrer">https://tangram-sc.readthedocs.io/en/latest/index.html<OutboundLink/></a></p>\n',headersStr:"Tangram  Tangram  Tangram  Cell Level Cluster Level Tangram Tutorials Tutorial 1: Tutorial for mapping data with Tangram Tutorial 2: Tutorial for spatial mapping using Tangram",content:"GitHub: https://github.com/broadinstitute/Tangram\nDochttps://tangram-sc.readthedocs.io/en/latest/index.html\n\n\n\n\n\n 1.  Tangram \n 2. Tangram Tutorial 1 MOp  snRNAseq  Slide-seq2 \n 3. Tangram Tutorial 2 squidy  Tangram\n 4. Tangram \n\n\n\n\n 1.  --- Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram\n 2. TangramSpatial alignment of single cell transcriptomic data\n\n\n\nTOMMASO BIANCALANI             AVIV REGEV\n                               \nGenentech                      Howard Hughes Medical Institute\ntommaso.biancalani@gmail.com   aviv.regev.sc@gmail.com\n\n\n# Tangram \n\nTangram  Python  PyTorch  scanpy single-cellor single-nucleus/Tangram  Tangram  tutorial  documentation squidpy tutorial\n\n\n# Tangram \n\n Tangram PyTorch  scanpy environment.yml \n\n * set up conda environment for Tangram\n   \n   conda env create -f environment.yml\n   \n   \n   1\n   \n * install tangram-sc from shell:\n   \n   conda activate tangram-env\n   pip install tangram-sc\n   \n   \n   1\n   2\n   \n * To start using Tangram, import tangram in your jupyter notebooks or/and scripts\n   \n   import tangram as tg\n   \n   \n   1\n   \n\n\n# Tangram \n\n\n# Cell Level\n\n AnnData  tg.pp_adatas \n\nad_sp = sc.read_h5ad(path)\nad_sc = sc.read_h5ad(path)\ntg.pp_adatas(ad_sc, ad_sp, genes=None)\n\n\n1\n2\n3\n\n\n pp_adatas  adata_scadata_sp  adatas.un  gene  genes=None Tangram \n\nad_map = tg.map_cells_to_space(ad_sc, ad_sp)\n\n\n1\n\n\n AnnDataad_map  cell-by-voxel  ad_map.X[i, j]  cell i  voxel j  tg.project_genes \n\nad_ge = tg.project_genes(ad_map, ad_sc)\n\n\n1\n\n\n ad_ge  voxel-by-gene AnnData ad_sp single cells  transfer \n\n\n# Cluster Level\n\nTangram  cell cluster level  Sten Linnarsson \n\n cell level Tangram \n\nad_map = tg.map_cells_to_space(\n  ad_sc,\n  ad_sp,\n  mode='clusters',\n  cluster_label='subclass_label')\n\n\n1\n2\n3\n4\n5\n\n\n cluster_label  ad_sc.obs subclass_label subclass_label  ad_sc.obs \n\n tg.project_genes  cluster_label  cluster label\n\nad_ge = tg.project_genes(\n  ad_map,\n  ad_sc,\n  cluster_label='subclass_label')\n\n\n1\n2\n3\n4\n\n\n\n# Tangram Tutorials\n\n\n# Tutorial 1: Tutorial for mapping data with Tangram\n\n https://tangram-sc.readthedocs.io/en/latest/tutorial_link.html\n https://nbviewer.org/github/zhenghu159/Bioinformatics-Tools/blob/main//012.Tangram/tutorial_tangram_without_squidpy.ipynb\n Tangram Tutorial 1 MOp  snRNAseq  Slide-seq2 \n\n\n# Tutorial 2: Tutorial for spatial mapping using Tangram\n\n https://tangram-sc.readthedocs.io/en/latest/tutorial_sq_link.html\n https://nbviewer.org/github/zhenghu159/Bioinformatics-Tools/blob/main//012.Tangram/tutorial_tangram_with_squidpy.ipynb\n Tangram Tutorial 2 squidy  Tangram",normalizedContent:"github: https://github.com/broadinstitute/tangram\ndochttps://tangram-sc.readthedocs.io/en/latest/index.html\n\n\n\n\n\n 1.  tangram \n 2. tangram tutorial 1 mop  snrnaseq  slide-seq2 \n 3. tangram tutorial 2 squidy  tangram\n 4. tangram \n\n\n\n\n 1.  --- deep learning and alignment of spatially resolved single-cell transcriptomes with tangram\n 2. tangramspatial alignment of single cell transcriptomic data\n\n\n\ntommaso biancalani             aviv regev\n                               \ngenentech                      howard hughes medical institute\ntommaso.biancalani@gmail.com   aviv.regev.sc@gmail.com\n\n\n# tangram \n\ntangram  python  pytorch  scanpy single-cellor single-nucleus/tangram  tangram  tutorial  documentation squidpy tutorial\n\n\n# tangram \n\n tangram pytorch  scanpy environment.yml \n\n * set up conda environment for tangram\n   \n   conda env create -f environment.yml\n   \n   \n   1\n   \n * install tangram-sc from shell:\n   \n   conda activate tangram-env\n   pip install tangram-sc\n   \n   \n   1\n   2\n   \n * to start using tangram, import tangram in your jupyter notebooks or/and scripts\n   \n   import tangram as tg\n   \n   \n   1\n   \n\n\n# tangram \n\n\n# cell level\n\n anndata  tg.pp_adatas \n\nad_sp = sc.read_h5ad(path)\nad_sc = sc.read_h5ad(path)\ntg.pp_adatas(ad_sc, ad_sp, genes=none)\n\n\n1\n2\n3\n\n\n pp_adatas  adata_scadata_sp  adatas.un  gene  genes=none tangram \n\nad_map = tg.map_cells_to_space(ad_sc, ad_sp)\n\n\n1\n\n\n anndataad_map  cell-by-voxel  ad_map.x[i, j]  cell i  voxel j  tg.project_genes \n\nad_ge = tg.project_genes(ad_map, ad_sc)\n\n\n1\n\n\n ad_ge  voxel-by-gene anndata ad_sp single cells  transfer \n\n\n# cluster level\n\ntangram  cell cluster level  sten linnarsson \n\n cell level tangram \n\nad_map = tg.map_cells_to_space(\n  ad_sc,\n  ad_sp,\n  mode='clusters',\n  cluster_label='subclass_label')\n\n\n1\n2\n3\n4\n5\n\n\n cluster_label  ad_sc.obs subclass_label subclass_label  ad_sc.obs \n\n tg.project_genes  cluster_label  cluster label\n\nad_ge = tg.project_genes(\n  ad_map,\n  ad_sc,\n  cluster_label='subclass_label')\n\n\n1\n2\n3\n4\n\n\n\n# tangram tutorials\n\n\n# tutorial 1: tutorial for mapping data with tangram\n\n https://tangram-sc.readthedocs.io/en/latest/tutorial_link.html\n https://nbviewer.org/github/zhenghu159/bioinformatics-tools/blob/main//012.tangram/tutorial_tangram_without_squidpy.ipynb\n tangram tutorial 1 mop  snrnaseq  slide-seq2 \n\n\n# tutorial 2: tutorial for spatial mapping using tangram\n\n https://tangram-sc.readthedocs.io/en/latest/tutorial_sq_link.html\n https://nbviewer.org/github/zhenghu159/bioinformatics-tools/blob/main//012.tangram/tutorial_tangram_with_squidpy.ipynb\n tangram tutorial 2 squidy  tangram",charsets:{cjk:!0},lastUpdated:"2023/09/30, 10:34:48",lastUpdatedTimestamp:1696041288e3},{title:"nnUNeta self-configuring method for deep learning-based biomedical image segmentation",frontmatter:{title:"nnUNeta self-configuring method for deep learning-based biomedical image segmentation",date:"2023-05-15T17:07:45.000Z",permalink:"/pages/8633a5/",categories:[""," - "],tags:[" - "],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/04.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E5%9B%BE%E5%83%8F%E5%A4%84%E7%90%86/001.nnUNet.html",relativePath:"03./04. - /001.nnUNet.md",key:"v-5de954f1",path:"/pages/8633a5/",headers:[{level:2,title:"nnUNet ",slug:"nnunet-",normalizedTitle:"nnunet ",charIndex:429},{level:3,title:"What is nnU-Net?",slug:"what-is-nnu-net",normalizedTitle:"what is nnu-net?",charIndex:443},{level:3,title:"What can nnU-Net do for you?",slug:"what-can-nnu-net-do-for-you",normalizedTitle:"what can nnu-net do for you?",charIndex:1287},{level:3,title:"What is the scope of nnU-Net?",slug:"what-is-the-scope-of-nnu-net",normalizedTitle:"what is the scope of nnu-net?",charIndex:1659},{level:3,title:"How does nnU-Net work?",slug:"how-does-nnu-net-work",normalizedTitle:"how does nnu-net work?",charIndex:2046},{level:2,title:"nnUNet ",slug:"nnunet-",normalizedTitle:"nnunet ",charIndex:2822},{level:3,title:"Dataset conversion",slug:"dataset-conversion",normalizedTitle:"dataset conversion",charIndex:2836},{level:3,title:"Setting up Paths",slug:"setting-up-paths",normalizedTitle:"setting up paths",charIndex:3048},{level:3,title:"Usage instructions",slug:"usage-instructions",normalizedTitle:"usage instructions",charIndex:3258},{level:3,title:"nnUNet ",slug:"nnunet-",normalizedTitle:"nnunet ",charIndex:3473}],excerpt:'<p>GitHub: <a href="https://github.com/MIC-DKFZ/nnUNet" target="_blank" rel="noopener noreferrer">https://github.com/MIC-DKFZ/nnUNet<OutboundLink/></a></p>\n',headersStr:"nnUNet  What is nnU-Net? What can nnU-Net do for you? What is the scope of nnU-Net? How does nnU-Net work? nnUNet  Dataset conversion Setting up Paths Usage instructions nnUNet ",content:"GitHub: https://github.com/MIC-DKFZ/nnUNet\n\n\n\n 1.  --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation\n 3. nnUNeta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nKLAUS H. MAIER-HEIN\n\nGerman Cancer Research Center\nk.maier-hein@dkfz.de\n\n\n# nnUNet \n\n\n# What is nnU-Net?\n\n2D3D/RGBCTMRI\n\n\n\nnnU-Net  U-Net \n\nnnU-Net  23  nnU-Net nnU-Net  baseline MICCAI 2020  10  9 MICCAI 2021 7  5  nnU-Net  nnU-Net  AMOS2022 \n\nPlease cite the following paper when using nnU-Net:\n\nIsensee, F., Jaeger, P. F., Kohl, S. A., Petersen, J., & Maier-Hein, K. H. (2021). nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation. Nature methods, 18(2), 203-211.\n\n\n# What can nnU-Net do for you?\n\nnnU-Net  nnU-Net \n\nnnU-Net\n\n *  baseline \n *  pipelines\n * \n * \n\n\n# What is the scope of nnU-Net?\n\nnnU-Net / 2D  3D \n\nnnU-Net nnU-Net \n\nnnU-Net  3D  40x40x40  1500x1500x1500  2D  40x40  ~30000x30000  RAM \n\n\n# How does nnU-Net work?\n\nnnU-Net  'dataset fingerprint'nnU-Net  U-Net \n\n * 2d 2D U-Net 2D  3D \n * 3d_fullres 3D U-Net 3D \n * 3d_lowres  3d_cascade_fullres 3D U-Net  3D U-Net  3D U-Net  3D \n\n U-Net U-Net  3d_lowres  U-Net \n\nnnU-Net pipelines\n\n *  nnU-Net  nnU-Net \n *  dataset fingerprint pipelines  GPU \n *  U-Net 2D3D 3D 3D \n\n\n# nnUNet \n\n\n# Dataset conversion\n\nhttps://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/dataset_format.md\nhttps://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.nnUNet/Dataset conversion.md\n\n\n# Setting up Paths\n\nhttps://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/setting_up_paths.md\nhttps://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.nnUNet/Setting up Paths.md\n\n\n# Usage instructions\n\nhttps://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/how_to_use_nnunet.md\nhttps://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.nnUNet/Usage instructions.md\n\n\n# nnUNet \n\nhttps://github.com/zhenghu159/Bioinformatics-Tools/blob/main//001.nnUNet/.pdf",normalizedContent:"github: https://github.com/mic-dkfz/nnunet\n\n\n\n 1.  --- nnu-neta self-configuring method for deep learning-based biomedical image segmentation\n 2.  --- toward data-efficient learninga benchmark for covid-19 ct lung and infection segmentation\n 3. nnuneta self-configuring method for deep learning-based biomedical image segmentation\n\n\n\nklaus h. maier-hein\n\ngerman cancer research center\nk.maier-hein@dkfz.de\n\n\n# nnunet \n\n\n# what is nnu-net?\n\n2d3d/rgbctmri\n\n\n\nnnu-net  u-net \n\nnnu-net  23  nnu-net nnu-net  baseline miccai 2020  10  9 miccai 2021 7  5  nnu-net  nnu-net  amos2022 \n\nplease cite the following paper when using nnu-net:\n\nisensee, f., jaeger, p. f., kohl, s. a., petersen, j., & maier-hein, k. h. (2021). nnu-net: a self-configuring method for deep learning-based biomedical image segmentation. nature methods, 18(2), 203-211.\n\n\n# what can nnu-net do for you?\n\nnnu-net  nnu-net \n\nnnu-net\n\n *  baseline \n *  pipelines\n * \n * \n\n\n# what is the scope of nnu-net?\n\nnnu-net / 2d  3d \n\nnnu-net nnu-net \n\nnnu-net  3d  40x40x40  1500x1500x1500  2d  40x40  ~30000x30000  ram \n\n\n# how does nnu-net work?\n\nnnu-net  'dataset fingerprint'nnu-net  u-net \n\n * 2d 2d u-net 2d  3d \n * 3d_fullres 3d u-net 3d \n * 3d_lowres  3d_cascade_fullres 3d u-net  3d u-net  3d u-net  3d \n\n u-net u-net  3d_lowres  u-net \n\nnnu-net pipelines\n\n *  nnu-net  nnu-net \n *  dataset fingerprint pipelines  gpu \n *  u-net 2d3d 3d 3d \n\n\n# nnunet \n\n\n# dataset conversion\n\nhttps://github.com/mic-dkfz/nnunet/blob/master/documentation/dataset_format.md\nhttps://github.com/zhenghu159/bioinformatics-tools/blob/main//001.nnunet/dataset conversion.md\n\n\n# setting up paths\n\nhttps://github.com/mic-dkfz/nnunet/blob/master/documentation/setting_up_paths.md\nhttps://github.com/zhenghu159/bioinformatics-tools/blob/main//001.nnunet/setting up paths.md\n\n\n# usage instructions\n\nhttps://github.com/mic-dkfz/nnunet/blob/master/documentation/how_to_use_nnunet.md\nhttps://github.com/zhenghu159/bioinformatics-tools/blob/main//001.nnunet/usage instructions.md\n\n\n# nnunet \n\nhttps://github.com/zhenghu159/bioinformatics-tools/blob/main//001.nnunet/.pdf",charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"glmnetLasso and Elastic-Net Regularized Generalized Linear Models",frontmatter:{title:"glmnetLasso and Elastic-Net Regularized Generalized Linear Models",date:"2023-05-10T21:13:35.000Z",permalink:"/pages/ee4b70/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/03.%E7%94%9F%E4%BF%A1%E5%AD%A6%E4%B9%A0/02.%E7%94%9F%E4%BF%A1%E5%B7%A5%E5%85%B7%20-%20%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0/002.glmnet.html",relativePath:"03./02. - /002.glmnet.md",key:"v-b2b8642e",path:"/pages/ee4b70/",headers:[{level:2,title:"glmnet ",slug:"glmnet-",normalizedTitle:"glmnet ",charIndex:77},{level:2,title:"An Introduction to glmnet",slug:"an-introduction-to-glmnet",normalizedTitle:"an introduction to glmnet",charIndex:983}],excerpt:'<p>GitHub: <a href="https://github.com/cran/glmnet" target="_blank" rel="noopener noreferrer">https://github.com/cran/glmnet<OutboundLink/></a><br>\nDoc: <a href="https://glmnet.stanford.edu/" target="_blank" rel="noopener noreferrer">https://glmnet.stanford.edu/<OutboundLink/></a></p>\n',headersStr:"glmnet  An Introduction to glmnet",content:"GitHub: https://github.com/cran/glmnet\nDoc: https://glmnet.stanford.edu/\n\n\n# glmnet \n\n\n\nglmnet  Cox  lasso  elastic-net Friedman, Hastie, and Tibshirani (2010), Simon et al. (2011), Tibshirani et al. (2012), Simon, Friedman, and Hastie (2013)\n\nVersion 3.0 \n\n * Relaxed fitting CV \n * \n *  cv.glmnet  ROC \n *  CV \n *  glmnet  x  one-hot-encoding\n *  glmnet  GLM \n\nVersion 4.0  GLM \n\nVersion 4.1  X survival:survfit\n\nglmnet \n\n 1. https://doi.org/10.18637/jss.v033.i01\n 2. https://doi.org/10.18637/jss.v039.i05\n 3. https://doi.org/10.1111/j.1467-9868.2011.01004.x\n 4. https://doi.org/10.18637/jss.v106.i01\n\n\n# An Introduction to glmnet\n\nhttps://glmnet.stanford.edu/articles/glmnet.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main//002.glmnet/Get started.nb.html",normalizedContent:"github: https://github.com/cran/glmnet\ndoc: https://glmnet.stanford.edu/\n\n\n# glmnet \n\n\n\nglmnet  cox  lasso  elastic-net friedman, hastie, and tibshirani (2010), simon et al. (2011), tibshirani et al. (2012), simon, friedman, and hastie (2013)\n\nversion 3.0 \n\n * relaxed fitting cv \n * \n *  cv.glmnet  roc \n *  cv \n *  glmnet  x  one-hot-encoding\n *  glmnet  glm \n\nversion 4.0  glm \n\nversion 4.1  x survival:survfit\n\nglmnet \n\n 1. https://doi.org/10.18637/jss.v033.i01\n 2. https://doi.org/10.18637/jss.v039.i05\n 3. https://doi.org/10.1111/j.1467-9868.2011.01004.x\n 4. https://doi.org/10.18637/jss.v106.i01\n\n\n# an introduction to glmnet\n\nhttps://glmnet.stanford.edu/articles/glmnet.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main//002.glmnet/get started.nb.html",charsets:{cjk:!0},lastUpdated:"2023/05/17, 17:05:53",lastUpdatedTimestamp:1684314353e3},{title:"HTML",frontmatter:{title:"HTML",date:"2023-04-21T20:59:41.000Z",permalink:"/pages/791228/",categories:[" - HTML"],tags:["HTML"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/11.%E5%89%8D%E7%AB%AF%20-%20HTML/01.HTML%E5%9F%BA%E7%A1%80.html",relativePath:"04.Web/11. - HTML/01.HTML.md",key:"v-7c78e838",path:"/pages/791228/",headers:[{level:2,title:"1 ",slug:"_1-",normalizedTitle:"1 ",charIndex:58},{level:3,title:"1.1 ",slug:"_1-1-",normalizedTitle:"1.1 ",charIndex:69},{level:4,title:"1.1.1 ",slug:"_1-1-1-",normalizedTitle:"1.1.1 ",charIndex:81},{level:4,title:"1.1.2  HTML",slug:"_1-1-2--html",normalizedTitle:"1.1.2  html",charIndex:267},{level:4,title:"1.1.3 ",slug:"_1-1-3-",normalizedTitle:"1.1.3 ",charIndex:535},{level:3,title:"1.2 ",slug:"_1-2-",normalizedTitle:"1.2 ",charIndex:602},{level:3,title:"1.3 Web ",slug:"_1-3-web-",normalizedTitle:"1.3 web ",charIndex:875},{level:2,title:"2 HTML ",slug:"_2-html-",normalizedTitle:"2 html ",charIndex:1085},{level:3,title:"2.1 HTML",slug:"_2-1-html",normalizedTitle:"2.1 html",charIndex:1101},{level:3,title:"2.2 HTML",slug:"_2-2-html",normalizedTitle:"2.2 html",charIndex:1477},{level:3,title:"2.3 HTML",slug:"_2-3-html",normalizedTitle:"2.3 html",charIndex:1541}],excerpt:'<div class="custom-block note"><p class="custom-block-title"></p>\n<p><a href="https://www.bilibili.com/video/BV1Kg411T7t9?p=1" target="_blank" rel="noopener noreferrer">https://www.bilibili.com/video/BV1Kg411T7t9?p=1<OutboundLink/></a></p>\n</div>\n',headersStr:"1  1.1  1.1.1  1.1.2  HTML 1.1.3  1.2  1.3 Web  2 HTML  2.1 HTML 2.2 HTML 2.3 HTML",content:"\n\nhttps://www.bilibili.com/video/BV1Kg411T7t9?p=1\n\n\n# 1 \n\n\n# 1.1 \n\n# 1.1.1 \n\n HTML \n\n HTML \n\n  .html  HTML \n\n# 1.1.2  HTML\n\nHTML  (Hyper Text Markup Language) \n\nHTML  (markup language)\n\n (markup tag)    \n\n 2 \n\n 1.  \n 2.  \n\n# 1.1.3 \n\n html \n\n\n# 1.2 \n\n\n\n\n\n\n\n                    \nIE             Trident   IE360\nFireFox        Gecko     \nSafari         Webkit    \nChrome/Opera   Blink     BlinkWebkit\n\n\n# 1.3 Web \n\nWeb  W3C W3C\n\nWeb :\n\n              \n   HTML         \n   CSS          \n   JavaScript   \n\n\n# 2 HTML \n\n\n# 2.1 HTML\n\n\n\n<html>\n  <head>\n    <title></title>\n  </head>\n  <body>\n    \n  </body>\n</html>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n                      \n<html></html>     HTML   \n<head></head>         headtitle\n<title></title>       \n<body></body>         body\n\n\n# 2.2 HTML\n\nVSCode Alt + Shift + a\n\n\x3c!--  --\x3e \n\n\n1\n\n\n\n# 2.3 HTML\n\n         \n&nbsp;      \n&lt;        <\n&gt;        >\n&amp;       &\n&yen;       \n&copy;      \n&reg;       \n&deg;       \n&plusmn;    \n&times;     \n&divide;    \n&sup2;      \n&sup3;      ",normalizedContent:"\n\nhttps://www.bilibili.com/video/bv1kg411t7t9?p=1\n\n\n# 1 \n\n\n# 1.1 \n\n# 1.1.1 \n\n html \n\n html \n\n  .html  html \n\n# 1.1.2  html\n\nhtml  (hyper text markup language) \n\nhtml  (markup language)\n\n (markup tag)    \n\n 2 \n\n 1.  \n 2.  \n\n# 1.1.3 \n\n html \n\n\n# 1.2 \n\n\n\n\n\n\n\n                    \nie             trident   ie360\nfirefox        gecko     \nsafari         webkit    \nchrome/opera   blink     blinkwebkit\n\n\n# 1.3 web \n\nweb  w3c w3c\n\nweb :\n\n              \n   html         \n   css          \n   javascript   \n\n\n# 2 html \n\n\n# 2.1 html\n\n\n\n<html>\n  <head>\n    <title></title>\n  </head>\n  <body>\n    \n  </body>\n</html>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n                      \n<html></html>     html   \n<head></head>         headtitle\n<title></title>       \n<body></body>         body\n\n\n# 2.2 html\n\nvscode alt + shift + a\n\n\x3c!--  --\x3e \n\n\n1\n\n\n\n# 2.3 html\n\n         \n&nbsp;      \n&lt;        <\n&gt;        >\n&amp;       &\n&yen;       \n&copy;      \n&reg;       \n&deg;       \n&plusmn;    \n&times;     \n&divide;    \n&sup2;      \n&sup3;      ",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"CSS",frontmatter:{title:"CSS",date:"2023-04-22T15:15:38.000Z",permalink:"/pages/bb1275/",categories:[" - CSS"],tags:["CSS"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/12.%E5%89%8D%E7%AB%AF%20-%20CSS/01.CSS%E5%9F%BA%E7%A1%80.html",relativePath:"04.Web/12. - CSS/01.CSS.md",key:"v-c279f104",path:"/pages/bb1275/",headers:[{level:2,title:"1 CSS",slug:"_1-css",normalizedTitle:"1 css",charIndex:59},{level:2,title:"2 CSS",slug:"_2-css",normalizedTitle:"2 css",charIndex:534},{level:3,title:"2.1 ",slug:"_2-1-",normalizedTitle:"2.1 ",charIndex:547},{level:3,title:"2.2 ",slug:"_2-2-",normalizedTitle:"2.2 ",charIndex:1259},{level:2,title:"3 CSS",slug:"_3-css",normalizedTitle:"3 css",charIndex:3004},{level:2,title:"4 CSS",slug:"_4-css",normalizedTitle:"4 css",charIndex:3635},{level:2,title:"5 CSS",slug:"_5-css",normalizedTitle:"5 css",charIndex:4256},{level:2,title:"6 CSS",slug:"_6-css",normalizedTitle:"6 css",charIndex:5607},{level:3,title:"6.1 ",slug:"_6-1-",normalizedTitle:"6.1 ",charIndex:5621},{level:3,title:"6.2 ",slug:"_6-2-",normalizedTitle:"6.2 ",charIndex:5682},{level:3,title:"6.3 ",slug:"_6-3-",normalizedTitle:"6.3 ",charIndex:6084},{level:3,title:"6.4 ",slug:"_6-4-",normalizedTitle:"6.4 ",charIndex:6641},{level:3,title:"6.5 Flex",slug:"_6-5-flex",normalizedTitle:"6.5 flex",charIndex:7436},{level:2,title:"7 CSS/",slug:"_7-css-",normalizedTitle:"7 css/",charIndex:8383},{level:3,title:"7.1 ",slug:"_7-1-",normalizedTitle:"7.1 ",charIndex:8400},{level:3,title:"7.2 ",slug:"_7-2-",normalizedTitle:"7.2 ",charIndex:9258},{level:2,title:"7 CSS",slug:"_7-css",normalizedTitle:"7 css",charIndex:10978}],excerpt:'<div class="custom-block note"><p class="custom-block-title"></p>\n<p><a href="https://www.bilibili.com/video/BV1Kg411T7t9?p=45" target="_blank" rel="noopener noreferrer">https://www.bilibili.com/video/BV1Kg411T7t9?p=45<OutboundLink/></a></p>\n</div>\n',headersStr:"1 CSS 2 CSS 2.1  2.2  3 CSS 4 CSS 5 CSS 6 CSS 6.1  6.2  6.3  6.4  6.5 Flex 7 CSS/ 7.1  7.2  7 CSS",content:'\n\nhttps://www.bilibili.com/video/BV1Kg411T7t9?p=45\n\n\n# 1 CSS\n\nCSSCascading style sheets\n\nCSSHTML\n\nCSS:\n\n\n\nCSS:\n\nVSCode Alt + Shift + a\n\n/*css*/\n\n\n1\n\n\nCSS:\n\n\n\nCSS <style> <style>  HTML  <head> \n\n<style>\n  div {\n    color: red;\n    font-size: 12px;\n  }\n</style>\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n\nCSS.csslink\n\n<link rel="stylesheet" href="style.css">\n\n\n1\n\n\n\n\nCSSstyle\n\n<div style="color: red; font-size: 12px;"></div>\n\n\n1\n\n\n\n# 2 CSS\n\n\n# 2.1 \n\n\n\n *  { css }\n * \n * \n   1. \n   2. \n * \n   \n\n\n\n * . { css }\n * \n * \n   1. classclass\n   2. \n   3. \n   4. \n * \n   \n\nid\n\n * #id { css }\n * idid\n * \n   1. id\n   2. id\n   3. id\n   4. id\n * \n   \n\n\n\n * * { css }\n * \n * \n   1. \n   2. marginpadding\n * \n   \n\n\n# 2.2 \n\n\n\n * 1 2 { css }\n *  HTML   \n * \n   1. \n   2.   \n * \n   \n\n\n\n * 1 > 2 { css }\n *  HTML   \n * \n   1. \n   2.  > \n * \n   \n\n\n\n * 1  2 { css }\n * \n * \n   1.  , \n   2. \n   3. \n * \n   \n\n\n\n * 12 { css }\n *   \n * \n   1. \n   2. \n * \n   \n\n\n\n * \n   * a:link { css }a\n   * a:visiter { css }a\n   * a:hover { css }\n   * a:active { css }\n * \n * \n   1.  :hover \n * \n   \n\n\n\n * :focus { css }\n * \n * \n   1.  <input> \n * \n   \n\n\n\n * \n   * E:first-child {}E\n   * E:last-child {}E\n   * E:nth-child(n) {}nE\n   * E:nth-last-child(n) {}nE\n * \n   * n\n     \n              \n              2neven\n              2n+12n-1odd\n     5      -n+5\n     5   n+5\n\n\n\n * \n * \n   1. HTML \n   2.  CSS \n * \n   1. ::before\n   2. ::after\n * \n   1. content\n   2. \n * \n\n.father::before {\n  /*  */\n  /* content,  */\n  content: \'\';\n  color: green;\n  width: 100px;\n  height: 100px;\n  background-color: blue;\n  /* ,  */\n  display: block;\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# 3 CSS\n\n\n\n * \n   1. \n   2. \n   3. \n * \n   * divphullidldtddformheadernavfooter\n\n\n\n * \n   1. \n   2. \n   3. \n * \n   * aspan buisstronginsemdel\n\n\n\n * \n   1. \n   2. \n * \n   * inputtextareabuttonselect\n   * imgChromeinline\n\n\n\n:  <a> \n\n\n\n * display:block;\n * display:inline;\n * display:inline-block;\n\n\n\n\n\n\n\ndisplay: block; \n\n\n\n\n# 4 CSS\n\n\n\n * \n * ()\n   1. color\n   2. font-stylefont-weightfont-sizefont-family\n   3. text-indenttext-align\n   4. line-height\n   5. \n * \n   * \n     \n\n\n\n * \n * \n   * \n     \n\n\n\n * \n * \n   *  <  <  <  < id <  < !important\n * \n   1. !important\n   2. !important\n   3.  !important \n\n\n\ndiv {\n  color: pink !important;\n}\n\n\n1\n2\n3\n\n\n\n# 5 CSS\n\n\n\n * \n   1.  \n   2.  \n * \n   * CSS contentpaddingborder margin \n     \n\n\n\n *  width  height  content \n * width / height\n * +px\n\n border \n\n * \n * \n   \n                    \n      border-width   +px\n      border-style    solid dashed dotted\n      border-color   \n\n * border : 10px solid red;\n\n padding \n\n *     \n * padding\n * \n   \n                                     \n      padding: 10px;                  10px\n      padding: 10px 20px;             10px20px\n      padding: 10px 20px 30px;        10px20px30px\n      padding: 10px 20px 30px 40px;   10px20px30px30px\n\n * \n\nmargin\n\n * \n * margin\n * \n   \n                                    \n      margin: 10px;                  10px\n      margin: 10px 20px;             10px20px\n      margin: 10px 20px 30px;        10px20px30px\n      margin: 10px 20px 30px 40px;   10px20px30px30px\n\n * \n\n\n# 6 CSS\n\n\n# 6.1 \n\n 1.   \n 2. /  \n\n\n# 6.2 \n\n * \n   1. \n   2. \n * \n   1. float:left;\n   2. float:right;\n * \n   1. \n   2. \n   3. \n   4.    \n * \n   * \n     \n   * 1\n     \n   * 2 overflow:hidden\n     \n\n\n# 6.3 \n\n * \n   1. \n   2. \n * \n   * \n   1. position:static;\n   2. position:relative;\n   3. position:absolute;\n   4. position:fixed;\n * \n   * \n   * \n   * left/right/bottom/top:px\n\n\n\n margin:0 auto; \n\n: transform:translate(-50%,-50%);\n\n\n\n 1. -\n 2. \n 3. \n\n\n\n z-index: z-index0\n\n\n# 6.4 \n\n\n\n * vertical-align\n * \n   * baseline: \n   * top: \n   * middle: \n   * bottom: \n\n\n\n * \n * cursor\n * \n   * default\n   * pointer\n   * text\n   * move\n\n\n\n * \n * border-radius\n * +px \n\n\n\n * \n * overflow\n * \n   * visible\n   * hidden\n   * scroll\n   * auto\n\n\n\n * :hover\n * \n   1. visibility:hidden\n   2. display:none\n * \n   1. visibility:hidden  \n   2. display:none  \n * \n   *  display\n   * display:none; display:block;\n\n\n# 6.5 Flex\n\n * \n   *  Flex \n   * Flex\n * \n   *  display: flex\n * \n   * \n   * \n   * \n   *  / \n     \n\n\n\n * :justify-content\n * \n   * flex-start, \n   * flex-end\n   * center\n   * space-around, \n   * space-between, \n   * space-evenly, \n\n\n\n * :\n   * align-items\n   * align-self\n * \n   * flex-start, \n   * flex-end\n   * center\n   * stretch, \n\n\n\n * \n   * flex:;\n * \n   * ()  : \n\n\n\n * , \n * : flex-direction\n * \n   * row, ()\n   * column* , \n   * row-reverse, \n   * column-reverse, \n\n\n\n *  : flex-wrap\n *  align-content\n   *  justify-content \n\n\n# 7 CSS/\n\n\n# 7.1 \n\n\n\n * \n   * transform: translate(, );\n * \n   * \n   * \n   * XY\n * \n   * translate(), x\n   * translateX() & translateY()\n * translate\n\n\n\n * \n   * transform: rotate();\n   * deg\n * \n   * , \n   * , \n * \n\nimg:hover {\n  transform: rotate(360deg);\n}\n\n\n1\n2\n3\n\n\n\n\n * \n   * \n   * transform-origin:  ;\n * \n * lefttoprightbottomcenter\n * \n * \n\n\n\n * transform: translate() rotate();\n\n\n\n * \n   * transform: scale(x, y);\n * \n   * , scale, xy\n   * transform: scale();\n   * scale1, scale1\n * \n\n\n\nbackground-image: linear-gradient(\n  transparent,\n  rgba(0,0,0,.6)\n)\n\n\n1\n2\n3\n4\n\n\n\n# 7.2 \n\n\n\n * \n   * transform: translate3d(x, y, z);\n   * transform: translateX();\n   * transform: translateY();\n   * transform: translateZ();\n * \n   * \n   * \n\n\n\n * \n   * perspective: ;\n   *  800  1200\n * \n   * \n\n\n\n * \n   * transform: rotateZ();  Z \n   * transform: rotateX();  X \n   * transform: rotateY();  Y \n * \n   * : , , \n * \n   * rotate3d(x, y, z, ) \n   * xyz 0-1\n\n\n\n * \n   *  transform-style: preserve-3d;\n   * 3d\n * \n   1. `transform-style: preserve-3d``\n   2. \n * \n   * \n * 3D\n\n\n\n * \n   * transform: scaleX();\n   * transform: scaleY();\n   * transform: scaleZ();\n   * transform: scale3d(x, y, z);\n\n\n\n * \n   1. \n   \n   @keyframes  {\n     from {}\n     to {}\n   }\n   \n   @keyframes change {\n     0% {\n       width: 200px;\n     }\n     50% {\n       width: 500px;\n     }\n     100% {\n       width: 800px;\n     }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   13\n   14\n   15\n   16\n   \n   2. \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   animation:  ;\n   \n   .box {\n     width: 200px;\n     background-color: pink;\n   \n     /*  */\n     animation: change 1s;\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   \n * \n   * animation:       ;\n * \n   * \n   * \n   * 2\n * \n   \n   animation:\n     1,\n     2,\n     N\n   ;\n   \n   \n   1\n   2\n   3\n   4\n   5\n   \n * \n\n\n# 7 CSS\n\n\n\n * \n   \n   @media () {\n     {\n       \n     }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   \n * \n   * max-width\n   * min-width\n * :\n   \n   /* 768px  */\n   @media (max-width: 768px) {\n       body {\n           background-color: pink;\n       }\n   }\n   /* 1200px skyblue */\n   @media (min-width: 1200px) {\n       body {\n           background-color: skyblue;\n       }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   \n\nrem\n\nrem10 HTML 1/10\n\nflexible.js\n\n * flexible.jsjs\n * htmlfont-size\n\nvw/vh\n\n * \n * \n * vwviewport width\n   * 1vw = 1/100\n * vhviewport height\n   * 1vh = 1/100\n * vw\n   1. vw 1/100\n   *    ()  vw 1/100 \n   2. vw = px / ( 1/100  )\n * vh\n   1. vh 1/100\n   *    ()  vh 1/100 \n   2. vh = px / ( 1/100  )\n * vwvh',normalizedContent:'\n\nhttps://www.bilibili.com/video/bv1kg411t7t9?p=45\n\n\n# 1 css\n\ncsscascading style sheets\n\ncsshtml\n\ncss:\n\n\n\ncss:\n\nvscode alt + shift + a\n\n/*css*/\n\n\n1\n\n\ncss:\n\n\n\ncss <style> <style>  html  <head> \n\n<style>\n  div {\n    color: red;\n    font-size: 12px;\n  }\n</style>\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n\ncss.csslink\n\n<link rel="stylesheet" href="style.css">\n\n\n1\n\n\n\n\ncssstyle\n\n<div style="color: red; font-size: 12px;"></div>\n\n\n1\n\n\n\n# 2 css\n\n\n# 2.1 \n\n\n\n *  { css }\n * \n * \n   1. \n   2. \n * \n   \n\n\n\n * . { css }\n * \n * \n   1. classclass\n   2. \n   3. \n   4. \n * \n   \n\nid\n\n * #id { css }\n * idid\n * \n   1. id\n   2. id\n   3. id\n   4. id\n * \n   \n\n\n\n * * { css }\n * \n * \n   1. \n   2. marginpadding\n * \n   \n\n\n# 2.2 \n\n\n\n * 1 2 { css }\n *  html   \n * \n   1. \n   2.   \n * \n   \n\n\n\n * 1 > 2 { css }\n *  html   \n * \n   1. \n   2.  > \n * \n   \n\n\n\n * 1  2 { css }\n * \n * \n   1.  , \n   2. \n   3. \n * \n   \n\n\n\n * 12 { css }\n *   \n * \n   1. \n   2. \n * \n   \n\n\n\n * \n   * a:link { css }a\n   * a:visiter { css }a\n   * a:hover { css }\n   * a:active { css }\n * \n * \n   1.  :hover \n * \n   \n\n\n\n * :focus { css }\n * \n * \n   1.  <input> \n * \n   \n\n\n\n * \n   * e:first-child {}e\n   * e:last-child {}e\n   * e:nth-child(n) {}ne\n   * e:nth-last-child(n) {}ne\n * \n   * n\n     \n              \n              2neven\n              2n+12n-1odd\n     5      -n+5\n     5   n+5\n\n\n\n * \n * \n   1. html \n   2.  css \n * \n   1. ::before\n   2. ::after\n * \n   1. content\n   2. \n * \n\n.father::before {\n  /*  */\n  /* content,  */\n  content: \'\';\n  color: green;\n  width: 100px;\n  height: 100px;\n  background-color: blue;\n  /* ,  */\n  display: block;\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# 3 css\n\n\n\n * \n   1. \n   2. \n   3. \n * \n   * divphullidldtddformheadernavfooter\n\n\n\n * \n   1. \n   2. \n   3. \n * \n   * aspan buisstronginsemdel\n\n\n\n * \n   1. \n   2. \n * \n   * inputtextareabuttonselect\n   * imgchromeinline\n\n\n\n:  <a> \n\n\n\n * display:block;\n * display:inline;\n * display:inline-block;\n\n\n\n\n\n\n\ndisplay: block; \n\n\n\n\n# 4 css\n\n\n\n * \n * ()\n   1. color\n   2. font-stylefont-weightfont-sizefont-family\n   3. text-indenttext-align\n   4. line-height\n   5. \n * \n   * \n     \n\n\n\n * \n * \n   * \n     \n\n\n\n * \n * \n   *  <  <  <  < id <  < !important\n * \n   1. !important\n   2. !important\n   3.  !important \n\n\n\ndiv {\n  color: pink !important;\n}\n\n\n1\n2\n3\n\n\n\n# 5 css\n\n\n\n * \n   1.  \n   2.  \n * \n   * css contentpaddingborder margin \n     \n\n\n\n *  width  height  content \n * width / height\n * +px\n\n border \n\n * \n * \n   \n                    \n      border-width   +px\n      border-style    solid dashed dotted\n      border-color   \n\n * border : 10px solid red;\n\n padding \n\n *     \n * padding\n * \n   \n                                     \n      padding: 10px;                  10px\n      padding: 10px 20px;             10px20px\n      padding: 10px 20px 30px;        10px20px30px\n      padding: 10px 20px 30px 40px;   10px20px30px30px\n\n * \n\nmargin\n\n * \n * margin\n * \n   \n                                    \n      margin: 10px;                  10px\n      margin: 10px 20px;             10px20px\n      margin: 10px 20px 30px;        10px20px30px\n      margin: 10px 20px 30px 40px;   10px20px30px30px\n\n * \n\n\n# 6 css\n\n\n# 6.1 \n\n 1.   \n 2. /  \n\n\n# 6.2 \n\n * \n   1. \n   2. \n * \n   1. float:left;\n   2. float:right;\n * \n   1. \n   2. \n   3. \n   4.    \n * \n   * \n     \n   * 1\n     \n   * 2 overflow:hidden\n     \n\n\n# 6.3 \n\n * \n   1. \n   2. \n * \n   * \n   1. position:static;\n   2. position:relative;\n   3. position:absolute;\n   4. position:fixed;\n * \n   * \n   * \n   * left/right/bottom/top:px\n\n\n\n margin:0 auto; \n\n: transform:translate(-50%,-50%);\n\n\n\n 1. -\n 2. \n 3. \n\n\n\n z-index: z-index0\n\n\n# 6.4 \n\n\n\n * vertical-align\n * \n   * baseline: \n   * top: \n   * middle: \n   * bottom: \n\n\n\n * \n * cursor\n * \n   * default\n   * pointer\n   * text\n   * move\n\n\n\n * \n * border-radius\n * +px \n\n\n\n * \n * overflow\n * \n   * visible\n   * hidden\n   * scroll\n   * auto\n\n\n\n * :hover\n * \n   1. visibility:hidden\n   2. display:none\n * \n   1. visibility:hidden  \n   2. display:none  \n * \n   *  display\n   * display:none; display:block;\n\n\n# 6.5 flex\n\n * \n   *  flex \n   * flex\n * \n   *  display: flex\n * \n   * \n   * \n   * \n   *  / \n     \n\n\n\n * :justify-content\n * \n   * flex-start, \n   * flex-end\n   * center\n   * space-around, \n   * space-between, \n   * space-evenly, \n\n\n\n * :\n   * align-items\n   * align-self\n * \n   * flex-start, \n   * flex-end\n   * center\n   * stretch, \n\n\n\n * \n   * flex:;\n * \n   * ()  : \n\n\n\n * , \n * : flex-direction\n * \n   * row, ()\n   * column* , \n   * row-reverse, \n   * column-reverse, \n\n\n\n *  : flex-wrap\n *  align-content\n   *  justify-content \n\n\n# 7 css/\n\n\n# 7.1 \n\n\n\n * \n   * transform: translate(, );\n * \n   * \n   * \n   * xy\n * \n   * translate(), x\n   * translatex() & translatey()\n * translate\n\n\n\n * \n   * transform: rotate();\n   * deg\n * \n   * , \n   * , \n * \n\nimg:hover {\n  transform: rotate(360deg);\n}\n\n\n1\n2\n3\n\n\n\n\n * \n   * \n   * transform-origin:  ;\n * \n * lefttoprightbottomcenter\n * \n * \n\n\n\n * transform: translate() rotate();\n\n\n\n * \n   * transform: scale(x, y);\n * \n   * , scale, xy\n   * transform: scale();\n   * scale1, scale1\n * \n\n\n\nbackground-image: linear-gradient(\n  transparent,\n  rgba(0,0,0,.6)\n)\n\n\n1\n2\n3\n4\n\n\n\n# 7.2 \n\n\n\n * \n   * transform: translate3d(x, y, z);\n   * transform: translatex();\n   * transform: translatey();\n   * transform: translatez();\n * \n   * \n   * \n\n\n\n * \n   * perspective: ;\n   *  800  1200\n * \n   * \n\n\n\n * \n   * transform: rotatez();  z \n   * transform: rotatex();  x \n   * transform: rotatey();  y \n * \n   * : , , \n * \n   * rotate3d(x, y, z, ) \n   * xyz 0-1\n\n\n\n * \n   *  transform-style: preserve-3d;\n   * 3d\n * \n   1. `transform-style: preserve-3d``\n   2. \n * \n   * \n * 3d\n\n\n\n * \n   * transform: scalex();\n   * transform: scaley();\n   * transform: scalez();\n   * transform: scale3d(x, y, z);\n\n\n\n * \n   1. \n   \n   @keyframes  {\n     from {}\n     to {}\n   }\n   \n   @keyframes change {\n     0% {\n       width: 200px;\n     }\n     50% {\n       width: 500px;\n     }\n     100% {\n       width: 800px;\n     }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   13\n   14\n   15\n   16\n   \n   2. \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   \n   animation:  ;\n   \n   .box {\n     width: 200px;\n     background-color: pink;\n   \n     /*  */\n     animation: change 1s;\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   \n * \n   * animation:       ;\n * \n   * \n   * \n   * 2\n * \n   \n   animation:\n     1,\n     2,\n     n\n   ;\n   \n   \n   1\n   2\n   3\n   4\n   5\n   \n * \n\n\n# 7 css\n\n\n\n * \n   \n   @media () {\n     {\n       \n     }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   \n * \n   * max-width\n   * min-width\n * :\n   \n   /* 768px  */\n   @media (max-width: 768px) {\n       body {\n           background-color: pink;\n       }\n   }\n   /* 1200px skyblue */\n   @media (min-width: 1200px) {\n       body {\n           background-color: skyblue;\n       }\n   }\n   \n   \n   1\n   2\n   3\n   4\n   5\n   6\n   7\n   8\n   9\n   10\n   11\n   12\n   \n\nrem\n\nrem10 html 1/10\n\nflexible.js\n\n * flexible.jsjs\n * htmlfont-size\n\nvw/vh\n\n * \n * \n * vwviewport width\n   * 1vw = 1/100\n * vhviewport height\n   * 1vh = 1/100\n * vw\n   1. vw 1/100\n   *    ()  vw 1/100 \n   2. vw = px / ( 1/100  )\n * vh\n   1. vh 1/100\n   *    ()  vh 1/100 \n   2. vh = px / ( 1/100  )\n * vwvh',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"HTML",frontmatter:{title:"HTML",date:"2023-04-22T14:55:52.000Z",permalink:"/pages/18d66f/",categories:[" - HTML"],tags:["HTML"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/11.%E5%89%8D%E7%AB%AF%20-%20HTML/02.HTML%E6%A0%87%E7%AD%BE.html",relativePath:"04.Web/11. - HTML/02.HTML.md",key:"v-4e38c05c",path:"/pages/18d66f/",headers:[{level:2,title:"HTML ",slug:"html--",normalizedTitle:"html ",charIndex:2},{level:2,title:"HTML ",slug:"html--",normalizedTitle:"html ",charIndex:68}],headersStr:"HTML  HTML ",content:"# HTML \n\nhttps://www.w3schools.cn/tags/default.html\n\n\n# HTML \n\nhttps://www.w3schools.cn/tags/ref_byfunc.html",normalizedContent:"# html \n\nhttps://www.w3schools.cn/tags/default.html\n\n\n# html \n\nhttps://www.w3schools.cn/tags/ref_byfunc.html",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"JavaScript ",frontmatter:{title:"JavaScript ",date:"2024-04-02T17:11:13.000Z",permalink:"/pages/592a6d/",categories:[""],tags:[" - JavaScript"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/13.%E5%89%8D%E7%AB%AF%20-%20JS&TS/01.JavaScript%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/13. - JS&TS/01.JavaScript.md",key:"v-2a4dc049",path:"/pages/592a6d/",excerpt:"<p> b  pink  JavaScript </p>\n",headersStr:null,content:" b  pink  JavaScript \n\n\n\n\n\n 1.  JavaScript Day1\n 2.  JavaScript Day2\n 3.  JavaScript Day3\n 4.  JavaScript Day4\n 5.  JavaScript Day5\n 6.  JavaScript WEB APIs Day1\n 7.  JavaScript WEB APIs Day2\n 8.  JavaScript WEB APIs Day3\n 9.  JavaScript WEB APIs Day4\n 10. JavaScript WEB APIs Day5\n 11. JavaScript WEB APIs Day6\n 12. JavaScript JS Day1\n 13. JavaScript JS Day2\n 14. JavaScript JS Day3\n 15. JavaScript JS Day4",normalizedContent:" b  pink  javascript \n\n\n\n\n\n 1.  javascript day1\n 2.  javascript day2\n 3.  javascript day3\n 4.  javascript day4\n 5.  javascript day5\n 6.  javascript web apis day1\n 7.  javascript web apis day2\n 8.  javascript web apis day3\n 9.  javascript web apis day4\n 10. javascript web apis day5\n 11. javascript web apis day6\n 12. javascript js day1\n 13. javascript js day2\n 14. javascript js day3\n 15. javascript js day4",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"Markdown ",frontmatter:{title:"Markdown ",categories:[""],tags:["Markdown"],date:"2022-11-24T20:37:33.000Z",permalink:"/pages/da13bb/",readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/14.%E5%89%8D%E7%AB%AF%20-%20Markdown/01.Markdown.html",relativePath:"04.Web/14. - Markdown/01.Markdown.md",key:"v-2754ef54",path:"/pages/da13bb/",headers:[{level:2,title:"MathJax ",slug:"mathjax-",normalizedTitle:"mathjax ",charIndex:109},{level:3,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:193},{level:3,title:"2.",slug:"_2-",normalizedTitle:"2.",charIndex:418},{level:3,title:"3. ",slug:"_3-",normalizedTitle:"3. ",charIndex:1229},{level:3,title:"4.",slug:"_4-",normalizedTitle:"4.",charIndex:1492},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:2026}],excerpt:'<p>Markdown  <a href="https://markdown.com.cn/" target="_blank" rel="noopener noreferrer">https://markdown.com.cn/<OutboundLink/></a><br>\nMarkdown Markdown.md</p>\n',headersStr:"MathJax  1.  2. 3.  4. ",content:"Markdown  https://markdown.com.cn/\nMarkdown Markdown.md\n\n\n# MathJax \n\nhttps://blog.csdn.net/oYinHeZhiGuang/article/details/119943358\n\n\n# 1. \n\n              \n\\alpha        \\alpha\n\\beta         \\beta\n\\gamma        \\gamma\n\\varphi       \\varphi\n\\rho          \\rho\n\\Delta        \\Delta\n\\epsilon      \\epsilon\n\\varepsilon   \\varepsilon\n\n\n# 2.\n\n                          \n{\\lbrace{                   \\lbrace\n}\\rbrace}                   \\rbrace\n{x}\\lbrace x \\rbrace{x}     \\lbrace x \\rbrace\nlim\\limlim                  \\lim\nlimx\\lim\\limits_{x} x lim   \\lim\\limits_{x}\n\\rightarrow               \\rightarrow\n\\to                       \\to\n\\infty                    \\infty\n\\neq                      \\neq\n\\leq                      \\leq\n\\geq                      \\geq\n\\sim                      \\sim\n\\bigtriangleup            \\bigtriangleup\n\\bigtriangledown          \\bigtriangledown\n\\triangle                 \\triangle\n\\pm                       \\pm\n\\partial                  \\partial\n\\in                       \\in\n\\notin                    \\notin\n\\bigcap                   \\bigcap\n\\bigcup                   \\bigcup\n\n\n# 3. \n\n                    \nx0x_0x 0              x_0\n12\\frac{1}{2} 2 1     \\frac{1}{2}\nx\\sqrt{x} x          \\sqrt{x}\nxn\\sqrt[n]{x} n  x   \\sqrt[n]{x}\nx\\vec{x} x          \\vec{x}\nx\\overline{x} x      \\overline{x}\n\\mathring U           \\mathring{U}\n\n\n# 4.\n\n                          \nlog\\loglog                  \\log\nln\\lnln                     \\ln\nsin\\sinsin                  \\sin\ncos\\coscos                  \\cos\ntan\\tantan                  \\tan\ncot\\cotcot                  \\cot\nsec\\secsec                  \\sec\ncsc\\csccsc                  \\csc\narcsin\\arcsinarcsin         \\arcsin\narccos\\arccosarccos         \\arccos\narctan\\arctanarctan         \\arctan\narccot\\text{arccot}arccot   \\text{arccot}\nsh\\text{sh}sh               \\text{sh}\nch\\text{ch}ch               \\text{ch}\n\n\n# \n\n                                                             \nf(x)={x,x0x,x<0f(x) = \\begin{cases} \\sqrt{x}, &x\\geqslant0   f(x) = \\begin{cases} \\sqrt{x}, &x \\geqslant 0 \\\\ -x, &x<0\n\\\\ -x, &x<0 \\end{cases}f(x)={  x , x, x0 x<0                \\end{cases}",normalizedContent:"markdown  https://markdown.com.cn/\nmarkdown markdown.md\n\n\n# mathjax \n\nhttps://blog.csdn.net/oyinhezhiguang/article/details/119943358\n\n\n# 1. \n\n              \n\\alpha        \\alpha\n\\beta         \\beta\n\\gamma        \\gamma\n\\varphi       \\varphi\n\\rho          \\rho\n\\delta        \\delta\n\\epsilon      \\epsilon\n\\varepsilon   \\varepsilon\n\n\n# 2.\n\n                          \n{\\lbrace{                   \\lbrace\n}\\rbrace}                   \\rbrace\n{x}\\lbrace x \\rbrace{x}     \\lbrace x \\rbrace\nlim\\limlim                  \\lim\nlimx\\lim\\limits_{x} x lim   \\lim\\limits_{x}\n\\rightarrow               \\rightarrow\n\\to                       \\to\n\\infty                    \\infty\n=\\neq=                      \\neq\n\\leq                      \\leq\n\\geq                      \\geq\n\\sim                      \\sim\n\\bigtriangleup            \\bigtriangleup\n\\bigtriangledown          \\bigtriangledown\n\\triangle                 \\triangle\n\\pm                       \\pm\n\\partial                  \\partial\n\\in                       \\in\n\\notin                    \\notin\n\\bigcap                   \\bigcap\n\\bigcup                   \\bigcup\n\n\n# 3. \n\n                    \nx0x_0x 0              x_0\n12\\frac{1}{2} 2 1     \\frac{1}{2}\nx\\sqrt{x} x          \\sqrt{x}\nxn\\sqrt[n]{x} n  x   \\sqrt[n]{x}\nx\\vec{x} x          \\vec{x}\nx\\overline{x} x      \\overline{x}\n\\mathring u           \\mathring{u}\n\n\n# 4.\n\n                          \nlog\\loglog                  \\log\nln\\lnln                     \\ln\nsin\\sinsin                  \\sin\ncos\\coscos                  \\cos\ntan\\tantan                  \\tan\ncot\\cotcot                  \\cot\nsec\\secsec                  \\sec\ncsc\\csccsc                  \\csc\narcsin\\arcsinarcsin         \\arcsin\narccos\\arccosarccos         \\arccos\narctan\\arctanarctan         \\arctan\narccot\\text{arccot}arccot   \\text{arccot}\nsh\\text{sh}sh               \\text{sh}\nch\\text{ch}ch               \\text{ch}\n\n\n# \n\n                                                             \nf(x)={x,x0x,x<0f(x) = \\begin{cases} \\sqrt{x}, &x\\geqslant0   f(x) = \\begin{cases} \\sqrt{x}, &x \\geqslant 0 \\\\ -x, &x<0\n\\\\ -x, &x<0 \\end{cases}f(x)={  x , x, x0 x<0                \\end{cases}",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"TypeScript",frontmatter:{title:"TypeScript",date:"2024-07-26T21:28:51.000Z",permalink:"/pages/6ce63c/",categories:[""],tags:[" - TypeScript"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/13.%E5%89%8D%E7%AB%AF%20-%20JS&TS/02.TypeScript%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/13. - JS&TS/02.TypeScript.md",key:"v-c8068dca",path:"/pages/6ce63c/",excerpt:"<p> b  TypeScript </p>\n",headersStr:null,content:" b  TypeScript \n\n\n\n\n\n 1. TypeScript Day1\n 2. TypeScript Day2\n 3. TypeScript Day3\n 4. TypeScript Day4",normalizedContent:" b  typescript \n\n\n\n\n\n 1. typescript day1\n 2. typescript day2\n 3. typescript day3\n 4. typescript day4",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"Vue3",frontmatter:{title:"Vue3",date:"2024-07-26T21:35:25.000Z",permalink:"/pages/f03da3/",categories:["Web"],tags:[" - Vue"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/15.%E5%89%8D%E7%AB%AF%20-%20Vue/01.Vue3%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/15. - Vue/01.Vue3.md",key:"v-1d78ed0e",path:"/pages/f03da3/",excerpt:"<p> Vue3 </p>\n",headersStr:null,content:" Vue3 \n\n\n\n\n\n 1. Vue3 Day1\n 2. Vue3 Day2\n 3. Vue3 Day3\n 4. Vue3 Day4\n 5. Vue3 Day5\n 6. Vue3 Day6",normalizedContent:" vue3 \n\n\n\n\n\n 1. vue3 day1\n 2. vue3 day2\n 3. vue3 day3\n 4. vue3 day4\n 5. vue3 day5\n 6. vue3 day6",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"MySQL",frontmatter:{title:"MySQL",date:"2024-07-26T21:55:15.000Z",permalink:"/pages/49a104/",categories:["Web"],tags:[" - MySQL"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/21.%E5%90%8E%E7%AB%AF%20-%20MySQL/01.MySQL%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/21. - MySQL/01.MySQL.md",key:"v-4e102365",path:"/pages/49a104/",excerpt:"<p> MySQL </p>\n",headersStr:null,content:" MySQL \n\n\n\n\n\n 1. MySQL  Day1\n 2. MySQL  Day2\n 3. MySQL  Day3",normalizedContent:" mysql \n\n\n\n\n\n 1. mysql  day1\n 2. mysql  day2\n 3. mysql  day3",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"AJAX",frontmatter:{title:"AJAX",date:"2024-07-26T21:54:53.000Z",permalink:"/pages/ce8da3/",categories:["Web"],tags:[" - Ajax"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/16.%E5%89%8D%E7%AB%AF%20-%20Ajax/01.AJAX%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/16. - Ajax/01.AJAX.md",key:"v-f12adf96",path:"/pages/ce8da3/",excerpt:"<p> AJAX </p>\n",headersStr:null,content:" AJAX \n\n\n\n\n\n 1. AJAX Day1\n 2. AJAX Day3\n 3. AJAX Day4",normalizedContent:" ajax \n\n\n\n\n\n 1. ajax day1\n 2. ajax day3\n 3. ajax day4",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"",frontmatter:{title:"",date:"2022-12-28T19:32:00.000Z",permalink:"/pages/d7e3cb/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/01.%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80/1.%E9%AB%98%E7%AD%89%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80.html",relativePath:"05./01./1..md",key:"v-43390cd4",path:"/pages/d7e3cb/",headers:[{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:2},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:14}],excerpt:"",headersStr:"   ",content:"#  \n\n\n#  \n\n(N\\mathbb{N}N) {0,1,2,3,...}\\lbrace 0,1,2,3,... \\rbrace{0,1,2,3,...} \n\n(Z\\mathbb{Z}Z) {...,3,2,1,0,1,2,3,...}\\lbrace ...,-3,-2,-1,0,1,2,3,... \\rbrace{...,3,2,1,0,1,2,3,...} \n\n(Q\\mathbb{Q}Q)0 {aba,bZandb0}\\lbrace \\frac{a}{b}|a,b\\in\\mathbb{Z} \\,and\\, b \\neq 0 \\rbrace{ b a a,bZandb0} \n\n(A\\mathbb{A}A) \n\n(R\\mathbb{R}R) {xx[,+]}\\lbrace x|x\\in[-\\infty,+\\infty] \\rbrace{xx[,+]} \n\n(C\\mathbb{C}C) z=a+biz=a+biz=a+biaaabbb aaa bbb iii i2=1i^2 = -1i 2 =1 {x+yix,yR}\\lbrace x+yi|x,y\\in\\mathbb{R} \\rbrace{x+yix,yR} ",normalizedContent:"#  \n\n\n#  \n\n(n\\mathbb{n}n) {0,1,2,3,...}\\lbrace 0,1,2,3,... \\rbrace{0,1,2,3,...} \n\n(z\\mathbb{z}z) {...,3,2,1,0,1,2,3,...}\\lbrace ...,-3,-2,-1,0,1,2,3,... \\rbrace{...,3,2,1,0,1,2,3,...} \n\n(q\\mathbb{q}q)0 {aba,bzandb=0}\\lbrace \\frac{a}{b}|a,b\\in\\mathbb{z} \\,and\\, b \\neq 0 \\rbrace{ b a a,bzandb=0} \n\n(a\\mathbb{a}a) \n\n(r\\mathbb{r}r) {xx[,+]}\\lbrace x|x\\in[-\\infty,+\\infty] \\rbrace{xx[,+]} \n\n(c\\mathbb{c}c) z=a+biz=a+biz=a+biaaabbb aaa bbb iii i2=1i^2 = -1i 2 =1 {x+yix,yr}\\lbrace x+yi|x,y\\in\\mathbb{r} \\rbrace{x+yix,yr} ",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"Django",frontmatter:{title:"Django",date:"2024-07-26T21:55:35.000Z",permalink:"/pages/b75e1a/",categories:["Web"],tags:[" - Django"],readingShow:"top"},regularPath:"/04.Web%E5%BC%80%E5%8F%91/22.%E5%90%8E%E7%AB%AF%20-%20Django/01.Django%E5%AD%A6%E4%B9%A0%E7%AC%94%E8%AE%B0.html",relativePath:"04.Web/22. - Django/01.Django.md",key:"v-8a435fea",path:"/pages/b75e1a/",excerpt:"<p> Django </p>\n",headersStr:null,content:" Django \n\n\n\n\n\n 1. Django Day1",normalizedContent:" django \n\n\n\n\n\n 1. django day1",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"1 ",frontmatter:{title:"1 ",date:"2022-12-04T20:48:11.000Z",permalink:"/pages/4a4e4d/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/11.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E8%A5%BF%E7%93%9C%E4%B9%A6/1.%E7%BB%AA%E8%AE%BA.html",relativePath:"05./11. - /1..md",key:"v-6770800e",path:"/pages/4a4e4d/",headers:[{level:2,title:"1.1 ",slug:"_1-1-",normalizedTitle:"1.1 ",charIndex:67},{level:2,title:"1.2 ",slug:"_1-2-",normalizedTitle:"1.2 ",charIndex:495},{level:2,title:"1.3 ",slug:"_1-3-",normalizedTitle:"1.3 ",charIndex:2551},{level:2,title:"1.4 ",slug:"_1-4-",normalizedTitle:"1.4 ",charIndex:2719}],excerpt:'<p><a href="https://github.com/Vay-keen/Machine-learning-learning-notes" target="_blank" rel="noopener noreferrer">https://github.com/Vay-keen/Machine-learning-learning-notes<OutboundLink/></a></p>\n',headersStr:"1.1  1.2  1.3  1.4 ",content:"https://github.com/Vay-keen/Machine-learning-learning-notes\n\n\n# 1.1 \n\n(model)(learning algorithm)\n\nMitchell  1997 \n\n * T\n * P T \n * E  E  T  P  E \n\n\n# 1.2 \n\n===)===)===......=\n\n * (data set): \n * (instance)/(sample): \n * (attribute)/(feature):   \n * (attribute value):   \n * (attribute space)/(sample space):     \n * (feature vector): \n * /(dimensionality):  D={x1,x2,...,xm}D = \\lbrace x_1, x_2, ..., x_m \\rbraceD={x 1 ,x 2 ,...,x m }  mmm  ddd 3 xi=(xi1,xi2,...,xid)x_i = (x_{i1}, x_{i2}, ..., x_{id})x i =(x i1 ,x i2 ,...,x id )  ddd  \\chi xix_i \\in \\chix i  xijx_{ij}x ij  xix_ix i  jjj 32ddd  xix_ix i \n * (learning)/(training): \n * (training data): \n * (training sample): \n * (training set): \n * (test set): \n * (hypothesis): \n * (ground-truth): \n * (label): ((===))\n * (example): \n * (label space): \n * (classification):  \n * (regression):  0.950.37\n * (binary classification): positive classnegative class\n * (multi-class classification): \n * (clustering): cluster  \n * (supervised learning): \n * (unsupervised learning): \n * (generalization): \n * (independent and identically distributed, i.d.d.i.d.d.i.d.d.): (distribution) DDD DDD \n\n\n# 1.3 \n\n\n\nhypothesis space\n\n\n# 1.4 \n\n(inductive bias)",normalizedContent:"https://github.com/vay-keen/machine-learning-learning-notes\n\n\n# 1.1 \n\n(model)(learning algorithm)\n\nmitchell  1997 \n\n * t\n * p t \n * e  e  t  p  e \n\n\n# 1.2 \n\n===)===)===......=\n\n * (data set): \n * (instance)/(sample): \n * (attribute)/(feature):   \n * (attribute value):   \n * (attribute space)/(sample space):     \n * (feature vector): \n * /(dimensionality):  d={x1,x2,...,xm}d = \\lbrace x_1, x_2, ..., x_m \\rbraced={x 1 ,x 2 ,...,x m }  mmm  ddd 3 xi=(xi1,xi2,...,xid)x_i = (x_{i1}, x_{i2}, ..., x_{id})x i =(x i1 ,x i2 ,...,x id )  ddd  \\chi xix_i \\in \\chix i  xijx_{ij}x ij  xix_ix i  jjj 32ddd  xix_ix i \n * (learning)/(training): \n * (training data): \n * (training sample): \n * (training set): \n * (test set): \n * (hypothesis): \n * (ground-truth): \n * (label): ((===))\n * (example): \n * (label space): \n * (classification):  \n * (regression):  0.950.37\n * (binary classification): positive classnegative class\n * (multi-class classification): \n * (clustering): cluster  \n * (supervised learning): \n * (unsupervised learning): \n * (generalization): \n * (independent and identically distributed, i.d.d.i.d.d.i.d.d.): (distribution) ddd ddd \n\n\n# 1.3 \n\n\n\nhypothesis space\n\n\n# 1.4 \n\n(inductive bias)",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{title:"",date:"2023-01-02T20:09:47.000Z",permalink:"/pages/4c1aba/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/01.%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80/3.%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B0.html",relativePath:"05./01./3..md",key:"v-bb151b66",path:"/pages/4c1aba/",headers:[{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:218},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:269},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:1196},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1206},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1225},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:2497},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:4911},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5290},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5663},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:5904},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5914},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6047},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6797},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:7165},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:7244},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:7259},{level:2,title:" SVD",slug:"svd",normalizedTitle:" svd",charIndex:7660}],excerpt:'<div class="custom-block note"><p class="custom-block-title"></p>\n<ol>\n<li><a href="https://www.bilibili.com/video/BV1pW4y127eJ?p=20&amp;vd_source=282029fd85715bf2f6e3073da94c3f5b" target="_blank" rel="noopener noreferrer">https://www.bilibili.com/video/BV1pW4y127eJ?p=20&amp;vd_source=282029fd85715bf2f6e3073da94c3f5b<OutboundLink/></a></li>\n<li><a href="https://www.bilibili.com/video/BV1G64y1S7qj/?spm_id_from=333.788&amp;vd_source=282029fd85715bf2f6e3073da94c3f5b" target="_blank" rel="noopener noreferrer">https://www.bilibili.com/video/BV1G64y1S7qj/?spm_id_from=333.788&amp;vd_source=282029fd85715bf2f6e3073da94c3f5b<OutboundLink/></a></li>\n</ol>\n</div>\n',headersStr:"                                 SVD",content:"\n\n 1. https://www.bilibili.com/video/BV1pW4y127eJ?p=20&vd_source=282029fd85715bf2f6e3073da94c3f5b\n 2. https://www.bilibili.com/video/BV1G64y1S7qj/?spm_id_from=333.788&vd_source=282029fd85715bf2f6e3073da94c3f5b\n\n\n#  \n\nAAA XXX bbb \n\n\n\n\n\n\n#  \n\n{a11x1+a12x2=b1a21x1+a22x2=b2\\begin{cases} a_{11}x_1 + a_{12}x_2 = b_1 \\\\ a_{21}x_1 + a_{22}x_2 = b_2 \\end{cases}{ a 11 x 1 +a 12 x 2 =b 1 a 21 x 1 +a 22 x 2 =b 2\n a11a22a12a210a_{11}a_{22} - a_{12}a_{21} \\neq 0a 11 a 22 a 12 a 21 0  x1=b1a22a12b2a11a22a12a21x_1 = \\frac{b_1a_{22}-a_{12}b_2}{a_{11}a_{22}-a_{12}a_{21}}x 1 = a 11 a 22 a 12 a 21 b 1 a 22 a 12 b 2 x2=b2a11a21b1a11a22a12a21x_2 = \\frac{b_2a_{11}-a_{21}b_1}{a_{11}a_{22}-a_{12}a_{21}}x 2 = a 11 a 22 a 12 a 21 b 2 a 11 a 21 b 1\n\n\n\n a11a22a12a21a_{11}a_{22} - a_{12}a_{21}a 11 a 22 a 12 a 21 \n\n\n\n D=a11a12a21a22=a11a22a12a21D = \\left | \\begin{matrix} a_{11} &a_{12} \\\\ a_{21} &a_{22} \\end{matrix} \\right | = a_{11}a_{22} - a_{12}a_{21}D=     a 11 a 21 a 12 a 22     =a 11 a 22 a 12 a 21  aij(i=1,2;j=1,2)a_{ij}(i=1,2;j=1,2)a ij (i=1,2;j=1,2) iii jjj \n\n\n\n\n\n\n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n (a1a2an)\\begin{pmatrix} a_1 &a_2 &\\cdots &a_n \\end{pmatrix}( a 1 a 2  a n ) (a1a2an)\\begin{pmatrix} a_1 \\\\ a_2 \\\\ \\vdots \\\\ a_n \\end{pmatrix}     a 1 a 2  a n    \n\n \n\n(diagonal matrix) 0 (a1000a2000an) \\begin{pmatrix} a_1 &0 &\\cdots &0 \\\\ 0 &a_2 &\\cdots &0 \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ 0 &0 &\\cdots &a_n \\end{pmatrix}     a 1 0  0 0 a 2  0     0 0  a n    \n\n 10 (100010001) \\begin{pmatrix} 1 &0 &\\cdots &0 \\\\ 0 &1 &\\cdots &0 \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ 0 &0 &\\cdots &1 \\end{pmatrix}     1 0  0 0 1  0     0 0  1    \n\n (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7    (1438439)\\begin{pmatrix} 14 &3 \\\\ 8 &4 \\\\ 3 &9 \\end{pmatrix}   14 8 3 3 4 9  \n\n (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7    (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7  \n\n  AT=AA^T=AA T =A  AAA (10111)\\begin{pmatrix} 10 &1 \\\\ 1 &-1 \\end{pmatrix}( 10 1 1 1 ) (324207475)\\begin{pmatrix} -3 &2 &-4 \\\\ 2 &0 &7 \\\\ -4 &7 &5 \\end{pmatrix}   3 2 4 2 0 7 4 7 5  \n\n\n#  \n\n mnm*nmn  A=(a11a12a1na21a22a2nam1am2amn)A = \\begin{pmatrix} a_{11} &a_{12} &\\cdots &a_{1n} \\\\ a_{21} &a_{22} &\\cdots &a_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ a_{m1} &a_{m2} &\\cdots &a_{mn} \\end{pmatrix} A=     a 11 a 21  a m1 a 12 a 22  a m2     a 1n a 2n  a mn      B=(b11b12b1nb21b22b2nbm1bm2bmn)B = \\begin{pmatrix} b_{11} &b_{12} &\\cdots &b_{1n} \\\\ b_{21} &b_{22} &\\cdots &b_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ b_{m1} &b_{m2} &\\cdots &b_{mn} \\end{pmatrix} B=     b 11 b 21  b m1 b 12 b 22  b m2     b 1n b 2n  b mn    \n\n \nA+B=(a11+b11a12+b12a1n+b1na21+b21a22+b22a2n+b2nam1+bm1am2+bm2amn+bmn)A + B = \\begin{pmatrix} a_{11}+b_{11} &a_{12}+b_{12} &\\cdots &a_{1n}+b_{1n} \\\\ a_{21}+b_{21} &a_{22}+b_{22} &\\cdots &a_{2n}+b_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ a_{m1}+b_{m1} &a_{m2}+b_{m2} &\\cdots &a_{mn}+b_{mn} \\end{pmatrix} A+B=     a 11 +b 11 a 21 +b 21  a m1 +b m1 a 12 +b 12 a 22 +b 22  a m2 +b m2     a 1n +b 1n a 2n +b 2n  a mn +b mn    \n\n A=A(a11a12a1na21a22a2nam1am2amn)\\lambda A = A \\lambda \\begin{pmatrix} \\lambda a_{11} &\\lambda a_{12} &\\cdots &\\lambda a_{1n} \\\\ \\lambda a_{21} &\\lambda a_{22} &\\cdots &\\lambda a_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ \\lambda a_{m1} &\\lambda a_{m2} &\\cdots &\\lambda a_{mn} \\end{pmatrix} A=A     a 11 a 21  a m1 a 12 a 22  a m2     a 1n a 2n  a mn    \n\n AB ABBAAB \\neq BAABBA\nC=AB=(a11a12a13a21a22a23)(b11b12b21b22b31b32)=(a11b11+a12b21+a13b31a11b12+a12b22+a13b32a21b11+a22b21+a23b31a21b12+a22b22+a23b32)C=AB= \\begin{pmatrix} a_{11} &a_{12} &a_{13} \\\\ a_{21} &a_{22} &a_{23} \\end{pmatrix} \\begin{pmatrix} b_{11} &b_{12} \\\\ b_{21} &b_{22} \\\\ b_{31} &b_{32} \\end{pmatrix}= \\begin{pmatrix} a_{11}b_{11}+a_{12}b_{21}+a_{13}b_{31} &a_{11}b_{12}+a_{12}b_{22}+a_{13}b_{32} \\\\ a_{21}b_{11}+a_{22}b_{21}+a_{23}b_{31} &a_{21}b_{12}+a_{22}b_{22}+a_{23}b_{32} \\end{pmatrix} C=AB=( a 11 a 21 a 12 a 22 a 13 a 23 )   b 11 b 21 b 31 b 12 b 22 b 32   =( a 11 b 11 +a 12 b 21 +a 13 b 31 a 21 b 11 +a 22 b 21 +a 23 b 31 a 11 b 12 +a 12 b 22 +a 13 b 32 a 21 b 12 +a 22 b 22 +a 23 b 32 )\n\n\n(AB)C=A(BC)(AB)C = A(BC)(AB)C=A(BC)\n(AB)=(A)B)=A(B)\\lambda(AB) = (\\lambda A)B) = A(\\lambda B)(AB)=(A)B)=A(B)\nA(B+C)=AB+ACA(B+C)=AB+ACA(B+C)=AB+AC\n(B+C)A=BA+CA(B+C)A=BA+CA(B+C)A=BA+CA\n\n\n#  \n\n A=(120311)A=\\begin{pmatrix} 1 &2 &0 \\\\ 3 &-1 &1 \\end{pmatrix}A=( 1 3 2 1 0 1 ) AT=(132101)A^T=\\begin{pmatrix} 1 &3 \\\\ 2 &-1 \\\\ 0 &1 \\end{pmatrix}A T =   1 2 0 3 1 1  \n\n\n(AT)T=A(A^T)^T = A(A T ) T =A\n(A+B)T=AT+BT(A+B)^T = A^T+B^T(A+B) T =A T +B T\n(A)T=AT(\\lambda A)^T = \\lambda A^T(A) T =A T\n(AB)T=BTAT(AB)^T = B^TA^T(AB) T =B T A T\n\n\n#  \n\n  AAA nn BBBAB=BA=EAB=BA=EAB=BA=E ()  AAA  BBB  AAA B=A1B=A^{-1}B=A 1 \n\n\n(AT)1=(A1)T(A^T)^{-1} = (A^{-1})^T(A T ) 1 =(A 1 ) T\n(A1)1=A(A^{-1})^{-1} = A(A 1 ) 1 =A\n(A)1=1A1(\\lambda A)^{-1} = \\frac{1}{\\lambda}A^{-1}(A) 1 =  1 A 1\n(AB)1=B1A1(A^B)^{-1} = B^{-1}A^{-1}(A B ) 1 =B 1 A 1\n\n\n#  \n\n  nmn*mnm Ammm()nnn()A=\n\n ()/\n__bilibili\n21\n\n\n\n\n\n\n\n#  \n\n\n#  \n\n mnm \\times nmn  mmm  nnn  nnn \n\n\n#  \n\n  nnn x=(x1x2xn)x=\\begin{pmatrix} x_1 \\\\ x_2 \\\\ \\vdots \\\\ x_n \\end{pmatrix}x=     x 1 x 2  x n     y=(y1y2yn)y=\\begin{pmatrix} y_1 \\\\ y_2 \\\\ \\vdots \\\\ y_n \\end{pmatrix}y=     y 1 y 2  y n     xy=xTy=(x1x2xn)(y1y2yn)=x1y1+x2y2++xnynx \\bullet y = x^Ty = \\begin{pmatrix} x_1 &x_2 &\\cdots &x_n \\end{pmatrix} \\begin{pmatrix} y_1 \\\\ y_2 \\\\ \\vdots \\\\ y_n \\end{pmatrix} = x_1y_1 + x_2y_2 + \\cdots + x_ny_nxy=x T y=( x 1 x 2  x n )     y 1 y 2  y n     =x 1 y 1 +x 2 y 2 ++x n y n\n\n\nxy=yxx \\bullet y = y \\bullet xxy=yx\n(x)y=(xy)(\\lambda x) \\bullet y = \\lambda (x \\bullet y)(x)y=(xy)(x+y)z=xz+yz(x+y) \\bullet z = x \\bullet z + y \\bullet z(x+y)z=xz+yz\n\n\n#  \n\nnnn  xxx x=xx=x12+x22++xn20||x|| = \\sqrt{x \\bullet x} = \\sqrt{x_1^2 + x_2^2 + \\cdots + x_n^2} \\geq 0x= xx = x 1 2 +x 2 2 ++x n 2 0\n\n x=1||x|| = 1x=1 \n\nx=x\\lambda\\Vert x \\Vert = |\\lambda|\\cdot \\Vert x \\Vertx=x\n\nx+yx+y\\Vert x+y \\Vert \\leq \\Vert x \\Vert + \\Vert y \\Vertx+yx+y\n\n\n#  \n\nxy=0x \\bullet y = 0xy=0 xxx  yyy xyx \\bot yxy\n\n\n#  \n\n\n\n\n#  \n\n  AAA  nnn  \\lambda  xxx  Ax=x(x0)Ax = \\lambda x (x \\neq 0)Ax=x(x0)\\lambda  AAA (eigenvalue)xxx  AAA  \\lambda (eigenvector)\n\n\n\n\n\n\n\n \n\n  AAA A=PP1A = P \\Lambda P^{-1}A=PP 1 PPP  AAA \\Lambda \n\n\n#  SVD",normalizedContent:"\n\n 1. https://www.bilibili.com/video/bv1pw4y127ej?p=20&vd_source=282029fd85715bf2f6e3073da94c3f5b\n 2. https://www.bilibili.com/video/bv1g64y1s7qj/?spm_id_from=333.788&vd_source=282029fd85715bf2f6e3073da94c3f5b\n\n\n#  \n\naaa xxx bbb \n\n\n\n\n\n\n#  \n\n{a11x1+a12x2=b1a21x1+a22x2=b2\\begin{cases} a_{11}x_1 + a_{12}x_2 = b_1 \\\\ a_{21}x_1 + a_{22}x_2 = b_2 \\end{cases}{ a 11 x 1 +a 12 x 2 =b 1 a 21 x 1 +a 22 x 2 =b 2\n a11a22a12a21=0a_{11}a_{22} - a_{12}a_{21} \\neq 0a 11 a 22 a 12 a 21 =0  x1=b1a22a12b2a11a22a12a21x_1 = \\frac{b_1a_{22}-a_{12}b_2}{a_{11}a_{22}-a_{12}a_{21}}x 1 = a 11 a 22 a 12 a 21 b 1 a 22 a 12 b 2 x2=b2a11a21b1a11a22a12a21x_2 = \\frac{b_2a_{11}-a_{21}b_1}{a_{11}a_{22}-a_{12}a_{21}}x 2 = a 11 a 22 a 12 a 21 b 2 a 11 a 21 b 1\n\n\n\n a11a22a12a21a_{11}a_{22} - a_{12}a_{21}a 11 a 22 a 12 a 21 \n\n\n\n d=a11a12a21a22=a11a22a12a21d = \\left | \\begin{matrix} a_{11} &a_{12} \\\\ a_{21} &a_{22} \\end{matrix} \\right | = a_{11}a_{22} - a_{12}a_{21}d=     a 11 a 21 a 12 a 22     =a 11 a 22 a 12 a 21  aij(i=1,2;j=1,2)a_{ij}(i=1,2;j=1,2)a ij (i=1,2;j=1,2) iii jjj \n\n\n\n\n\n\n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n (a1a2an)\\begin{pmatrix} a_1 &a_2 &\\cdots &a_n \\end{pmatrix}( a 1 a 2  a n ) (a1a2an)\\begin{pmatrix} a_1 \\\\ a_2 \\\\ \\vdots \\\\ a_n \\end{pmatrix}     a 1 a 2  a n    \n\n \n\n(diagonal matrix) 0 (a1000a2000an) \\begin{pmatrix} a_1 &0 &\\cdots &0 \\\\ 0 &a_2 &\\cdots &0 \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ 0 &0 &\\cdots &a_n \\end{pmatrix}     a 1 0  0 0 a 2  0     0 0  a n    \n\n 10 (100010001) \\begin{pmatrix} 1 &0 &\\cdots &0 \\\\ 0 &1 &\\cdots &0 \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ 0 &0 &\\cdots &1 \\end{pmatrix}     1 0  0 0 1  0     0 0  1    \n\n (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7    (1438439)\\begin{pmatrix} 14 &3 \\\\ 8 &4 \\\\ 3 &9 \\end{pmatrix}   14 8 3 3 4 9  \n\n (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7    (125637)\\begin{pmatrix} 1 &2 \\\\ 5 &6 \\\\ 3 &7 \\end{pmatrix}   1 5 3 2 6 7  \n\n  at=aa^t=aa t =a  aaa (10111)\\begin{pmatrix} 10 &1 \\\\ 1 &-1 \\end{pmatrix}( 10 1 1 1 ) (324207475)\\begin{pmatrix} -3 &2 &-4 \\\\ 2 &0 &7 \\\\ -4 &7 &5 \\end{pmatrix}   3 2 4 2 0 7 4 7 5  \n\n\n#  \n\n mnm*nmn  a=(a11a12a1na21a22a2nam1am2amn)a = \\begin{pmatrix} a_{11} &a_{12} &\\cdots &a_{1n} \\\\ a_{21} &a_{22} &\\cdots &a_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ a_{m1} &a_{m2} &\\cdots &a_{mn} \\end{pmatrix} a=     a 11 a 21  a m1 a 12 a 22  a m2     a 1n a 2n  a mn      b=(b11b12b1nb21b22b2nbm1bm2bmn)b = \\begin{pmatrix} b_{11} &b_{12} &\\cdots &b_{1n} \\\\ b_{21} &b_{22} &\\cdots &b_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ b_{m1} &b_{m2} &\\cdots &b_{mn} \\end{pmatrix} b=     b 11 b 21  b m1 b 12 b 22  b m2     b 1n b 2n  b mn    \n\n \na+b=(a11+b11a12+b12a1n+b1na21+b21a22+b22a2n+b2nam1+bm1am2+bm2amn+bmn)a + b = \\begin{pmatrix} a_{11}+b_{11} &a_{12}+b_{12} &\\cdots &a_{1n}+b_{1n} \\\\ a_{21}+b_{21} &a_{22}+b_{22} &\\cdots &a_{2n}+b_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ a_{m1}+b_{m1} &a_{m2}+b_{m2} &\\cdots &a_{mn}+b_{mn} \\end{pmatrix} a+b=     a 11 +b 11 a 21 +b 21  a m1 +b m1 a 12 +b 12 a 22 +b 22  a m2 +b m2     a 1n +b 1n a 2n +b 2n  a mn +b mn    \n\n a=a(a11a12a1na21a22a2nam1am2amn)\\lambda a = a \\lambda \\begin{pmatrix} \\lambda a_{11} &\\lambda a_{12} &\\cdots &\\lambda a_{1n} \\\\ \\lambda a_{21} &\\lambda a_{22} &\\cdots &\\lambda a_{2n} \\\\ \\vdots &\\vdots &\\vdots &\\vdots \\\\ \\lambda a_{m1} &\\lambda a_{m2} &\\cdots &\\lambda a_{mn} \\end{pmatrix} a=a     a 11 a 21  a m1 a 12 a 22  a m2     a 1n a 2n  a mn    \n\n ab ab=baab \\neq baab=ba\nc=ab=(a11a12a13a21a22a23)(b11b12b21b22b31b32)=(a11b11+a12b21+a13b31a11b12+a12b22+a13b32a21b11+a22b21+a23b31a21b12+a22b22+a23b32)c=ab= \\begin{pmatrix} a_{11} &a_{12} &a_{13} \\\\ a_{21} &a_{22} &a_{23} \\end{pmatrix} \\begin{pmatrix} b_{11} &b_{12} \\\\ b_{21} &b_{22} \\\\ b_{31} &b_{32} \\end{pmatrix}= \\begin{pmatrix} a_{11}b_{11}+a_{12}b_{21}+a_{13}b_{31} &a_{11}b_{12}+a_{12}b_{22}+a_{13}b_{32} \\\\ a_{21}b_{11}+a_{22}b_{21}+a_{23}b_{31} &a_{21}b_{12}+a_{22}b_{22}+a_{23}b_{32} \\end{pmatrix} c=ab=( a 11 a 21 a 12 a 22 a 13 a 23 )   b 11 b 21 b 31 b 12 b 22 b 32   =( a 11 b 11 +a 12 b 21 +a 13 b 31 a 21 b 11 +a 22 b 21 +a 23 b 31 a 11 b 12 +a 12 b 22 +a 13 b 32 a 21 b 12 +a 22 b 22 +a 23 b 32 )\n\n\n(ab)c=a(bc)(ab)c = a(bc)(ab)c=a(bc)\n(ab)=(a)b)=a(b)\\lambda(ab) = (\\lambda a)b) = a(\\lambda b)(ab)=(a)b)=a(b)\na(b+c)=ab+aca(b+c)=ab+aca(b+c)=ab+ac\n(b+c)a=ba+ca(b+c)a=ba+ca(b+c)a=ba+ca\n\n\n#  \n\n a=(120311)a=\\begin{pmatrix} 1 &2 &0 \\\\ 3 &-1 &1 \\end{pmatrix}a=( 1 3 2 1 0 1 ) at=(132101)a^t=\\begin{pmatrix} 1 &3 \\\\ 2 &-1 \\\\ 0 &1 \\end{pmatrix}a t =   1 2 0 3 1 1  \n\n\n(at)t=a(a^t)^t = a(a t ) t =a\n(a+b)t=at+bt(a+b)^t = a^t+b^t(a+b) t =a t +b t\n(a)t=at(\\lambda a)^t = \\lambda a^t(a) t =a t\n(ab)t=btat(ab)^t = b^ta^t(ab) t =b t a t\n\n\n#  \n\n  aaa nn bbbab=ba=eab=ba=eab=ba=e ()  aaa  bbb  aaa b=a1b=a^{-1}b=a 1 \n\n\n(at)1=(a1)t(a^t)^{-1} = (a^{-1})^t(a t ) 1 =(a 1 ) t\n(a1)1=a(a^{-1})^{-1} = a(a 1 ) 1 =a\n(a)1=1a1(\\lambda a)^{-1} = \\frac{1}{\\lambda}a^{-1}(a) 1 =  1 a 1\n(ab)1=b1a1(a^b)^{-1} = b^{-1}a^{-1}(a b ) 1 =b 1 a 1\n\n\n#  \n\n  nmn*mnm ammm()nnn()a=\n\n ()/\n__bilibili\n21\n\n\n\n\n\n\n\n#  \n\n\n#  \n\n mnm \\times nmn  mmm  nnn  nnn \n\n\n#  \n\n  nnn x=(x1x2xn)x=\\begin{pmatrix} x_1 \\\\ x_2 \\\\ \\vdots \\\\ x_n \\end{pmatrix}x=     x 1 x 2  x n     y=(y1y2yn)y=\\begin{pmatrix} y_1 \\\\ y_2 \\\\ \\vdots \\\\ y_n \\end{pmatrix}y=     y 1 y 2  y n     xy=xty=(x1x2xn)(y1y2yn)=x1y1+x2y2++xnynx \\bullet y = x^ty = \\begin{pmatrix} x_1 &x_2 &\\cdots &x_n \\end{pmatrix} \\begin{pmatrix} y_1 \\\\ y_2 \\\\ \\vdots \\\\ y_n \\end{pmatrix} = x_1y_1 + x_2y_2 + \\cdots + x_ny_nxy=x t y=( x 1 x 2  x n )     y 1 y 2  y n     =x 1 y 1 +x 2 y 2 ++x n y n\n\n\nxy=yxx \\bullet y = y \\bullet xxy=yx\n(x)y=(xy)(\\lambda x) \\bullet y = \\lambda (x \\bullet y)(x)y=(xy)(x+y)z=xz+yz(x+y) \\bullet z = x \\bullet z + y \\bullet z(x+y)z=xz+yz\n\n\n#  \n\nnnn  xxx x=xx=x12+x22++xn20||x|| = \\sqrt{x \\bullet x} = \\sqrt{x_1^2 + x_2^2 + \\cdots + x_n^2} \\geq 0x= xx = x 1 2 +x 2 2 ++x n 2 0\n\n x=1||x|| = 1x=1 \n\nx=x\\lambda\\vert x \\vert = |\\lambda|\\cdot \\vert x \\vertx=x\n\nx+yx+y\\vert x+y \\vert \\leq \\vert x \\vert + \\vert y \\vertx+yx+y\n\n\n#  \n\nxy=0x \\bullet y = 0xy=0 xxx  yyy xyx \\bot yxy\n\n\n#  \n\n\n\n\n#  \n\n  aaa  nnn  \\lambda  xxx  ax=x(x=0)ax = \\lambda x (x \\neq 0)ax=x(x=0)\\lambda  aaa (eigenvalue)xxx  aaa  \\lambda (eigenvector)\n\n\n\n\n\n\n\n \n\n  aaa a=pp1a = p \\lambda p^{-1}a=pp 1 ppp  aaa \\lambda \n\n\n#  svd",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{title:"",date:"2023-01-02T19:49:17.000Z",permalink:"/pages/02fc82/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/01.%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80/2.%E5%BE%AE%E7%A7%AF%E5%88%86.html",relativePath:"05./01./2..md",key:"v-636d4942",path:"/pages/02fc82/",headers:[{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:2},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:12},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:159},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:518},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:697},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:707},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1058},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1612},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:2749},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:2919},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:3321},{level:3,title:" ",slug:"-2",normalizedTitle:" ",charIndex:3331},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5470},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6261},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:8102},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:9709},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:9720},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:9735}],excerpt:"",headersStr:"                                   ",content:"#  \n\n\n#  \n\n * y=f(x)y=f(x)y=f(x) xy\n * y0=yx=x0y_0=y|_{x=x_0}y 0 =y x=x 0 y0=f(x0)y_0=f(x_0)y 0 =f(x 0 )  x0x_0x 0 \n\n\n#  \n\n *  f(x)={x,x0x,x<0f(x) = \\begin{cases} \\sqrt{x}, &x \\geqslant 0 \\\\ -x, &x<0 \\end{cases}f(x)={  x , x, x0 x<0\n *  h=12gt2t=2hgh=\\frac{1}{2}gt^2 \\to t=\\sqrt{\\frac{2h}{g}}h= 2 1 gt 2 t= g 2h  t=t(h)t=t(h)t=t(h)  h=h(t)h=h(t)h=h(t) \n *  y=x2+1y=x^2+1y=x 2 +1\n *  F(x,y)=0F(x,y)=0F(x,y)=0  3x+y4=03x+y-4=03x+y4=0\n\n\n#  \n\n * \n   *  f(x)=f(x)f(-x)=f(x)f(x)=f(x)y\n   *  f(x)=f(x)f(-x)=-f(x)f(x)=f(x)\n *  f(x+T)=f(x)f(x+T)=f(x)f(x+T)=f(x)\n *  \n\n\n#  \n\n\n#  \n\n  u1,u2,...,un,...u_1,u_2,...,u_n,...u 1 ,u 2 ,...,u n ,... unu_nu n \n\n  {un}\\lbrace u_n \\rbrace{u n }  nnn  AAA AAA  AAA limn13n=0\\lim\\limits_{n \\to \\infty} \\frac{1}{3^n} = 0 n lim 3 n 1 =0 limn2n\\lim\\limits_{n \\to \\infty} 2^n n lim 2 n \n\n\n#  \n\n  x0x_0x 0  x0(x0,x0+)x_0\\in(x_0-\\delta,x_0+\\delta)x 0 (x 0 ,x 0 +)\n\n  x0x_0x 0  x0x_0x 0 \n\n\n\n\n\n  x0x_0x 0  AAA \\varepsilonf(x)A<|f(x)-A|<\\varepsilonf(x)A< AAA  xx0x \\to x_0xx 0  limxx0f(x)=A\\lim\\limits_{x \\to x_0}f(x)=A xx 0 lim f(x)=A\n limxx0f(x)=limxx0+f(x)=A\\lim\\limits_{x \\to x_0^-}f(x)=\\lim\\limits_{x \\to x_0^+}f(x)=A xx 0  lim f(x)= xx 0 + lim f(x)=A\n\n\n\n\n\n\n\n#  \n\n limx1x=0\\lim\\limits_{x \\to \\infty} \\frac{1}{x} = 0 x lim x 1 =0 1x\\frac{1}{x} x 1  xx \\to \\inftyx limx2(3x6)=0\\lim\\limits_{x \\to 2} (3x-6) = 0 x2 lim (3x6)=0 3x63x-63x6  x2x \\to 2x2 \n\n  ++\\infty+-\\infty\n\n\n\n * \n * \n * \n * \n   \n * limx0x2x=12\\lim\\limits_{x \\to 0} \\frac{x}{2x} = \\frac{1}{2} x0 lim 2x x = 2 1\n\n limxx0f(x)=A\\lim\\limits_{x \\to x_0}f(x)=A xx 0 lim f(x)=A  f(x)=A+a(x)f(x)=A+a(x)f(x)=A+a(x)  a(x)a(x)a(x)  xx0x \\to x_0xx 0 \n\n a=a(x),b=b(x)a=a(x),b=b(x)a=a(x),b=b(x) limxx0a(x)=0\\lim\\limits_{x \\to x_0}a(x)=0 xx 0 lim a(x)=0limxx0b(x)=0\\lim\\limits_{x \\to x_0}b(x)=0 xx 0 lim b(x)=0\n\n *  limxx0ba=0\\lim\\limits_{x \\to x_0}\\frac{b}{a}=0 xx 0 lim a b =0 bbb  aaa \n *  limxx0ba=\\lim\\limits_{x \\to x_0}\\frac{b}{a}=\\infty xx 0 lim a b = bbb  aaa \n *  limxx0ba=C0\\lim\\limits_{x \\to x_0}\\frac{b}{a}=C\\neq0 xx 0 lim a b =C0 bbb  aaa \n\n\n#  \n\n  y=f(x)y=f(x)y=f(x)  x0x_0x 0  x\\triangle xx  y\\triangle yy  y=f(x)y=f(x)y=f(x)  x0x_0x 0 \n\n\n#  \n\n  f(x)f(x)f(x)  x=x0x=x_0x=x 0 \n\n\n\n *  xx0x \\to x_0xx 0 f(x)f(x)f(x)  x0x_0x 0  f(x)f(x)f(x) \n * limx0f(x)\\lim\\limits_{x \\to 0^-}f(x) x0  lim f(x)  limx0+f(x)\\lim\\limits_{x \\to 0^+}f(x) x0 + lim f(x) \n * limxx0f(x)\\lim\\limits_{x \\to x_0}f(x) xx 0 lim f(x)  f(x0)f(x_0)f(x 0 )\n\n\n#  \n\n\n#  \n\n  limx0yx=limx0f(x0+x)f(x0)x\\lim\\limits_{\\triangle x \\to 0} \\frac{\\triangle y}{\\triangle x} = \\lim\\limits_{\\triangle x \\to 0} \\frac{f(x_0 + \\triangle x)-f(x_0)}{\\triangle x} x0 lim x y = x0 lim x f(x 0 +x)f(x 0 )  y=f(x)y=f(x)y=f(x)  x0x_0x 0 f(x0)f'(x_0)f  (x 0 )yx=x0y'|_{x=x_0}y   x=x 0 dydxx=x0\\frac{dy}{dx}|_{x=x_0} dx dy  x=x 0  df(x)dxx=x0\\frac{df(x)}{dx}|_{x=x_0} dx df(x)  x=x 0\n\n\n\n * (uv)=uv(u \\pm v)' = u' \\pm v'(uv)  =u  v \n * (uv)=uv+uv(uv)' = u'v + uv'(uv)  =u  v+uv \n * (uv)=uvuvv2(v0)(\\frac{u}{v})' = \\frac{u'v-uv'}{v^2}(v \\neq 0)( v u )  = v 2 u  vuv  (v0)\n * (Cu)=Cu(Cu)'=Cu'(Cu)  =Cu \n * (Cv)=Cvv2(\\frac{C}{v})'=-\\frac{Cv'}{v^2}( v C )  = v 2 Cv  (C)\n\n\n\n                                   \ny=Cy=Cy=C (CCC )                   y=0y'=0y  =0\ny=axy=a^xy=a x                        y=axlnay'=a^x\\ln ay  =a x lna\ny=exy=e^xy=e x                        y=exy'=e^xy  =e x\ny=xny=x^ny=x n                        y=nxn1y'=nx^{n-1}y  =nx n1\ny=logaxy=\\log_a xy=log a x            y=1xlnay'=\\frac{1}{x \\ln a}y  = xlna 1\ny=lnxy=\\ln xy=lnx                     y=1xy'=\\frac{1}{x}y  = x 1\ny=exy=e^xy=e x                        y=exy'=e^xy  =e x\ny=sinxy=\\sin xy=sinx                  y=cosxy'=\\cos xy  =cosx\ny=cosxy=\\cos xy=cosx                  y=sinxy'=-\\sin xy  =sinx\ny=tanxy=\\tan xy=tanx                  y=sec2xy'=\\sec^2 xy  =sec 2 x\ny=cotxy=\\cot xy=cotx                  y=csc2xy'=-\\csc^2 xy  =csc 2 x\ny=secxy=\\sec xy=secx                  y=secxtanxy'=\\sec x \\tan xy  =secxtanx\ny=cscxy=\\csc xy=cscx                  y=cscxcotxy'=-\\csc x \\cot xy  =cscxcotx\ny=arcsinxy=\\arcsin xy=arcsinx         y=11x2y'=\\frac{1}{\\sqrt{1-x^2}}y  =  1x 2 1\ny=arccosxy=\\arccos xy=arccosx         y=11x2y'=-\\frac{1}{\\sqrt{1-x^2}}y  =  1x 2 1\ny=arctanxy=\\arctan xy=arctanx         y=11+x2y'=\\frac{1}{1+x^2}y  = 1+x 2 1\ny=arccotxy=\\text{arccot} xy=arccotx   y=11+x2y'=-\\frac{1}{1+x^2}y  = 1+x 2 1\ny=shxy=\\text{sh} xy=shx               y=chxy'=\\text{ch} xy  =chx\n\n\n#  \n\n z=f(x,y)z=f(x,y)z=f(x,y)  (x0,y0)(x_0,y_0)(x 0 ,y 0 )  y=y0y=y_0y=y 0  f(x,y0)f(x,y_0)f(x,y 0 )  x=x0x=x_0x=x 0  limx0f(x0+x,y0)f(x0,y0)x=A\\lim\\limits_{\\bigtriangleup x \\to 0} \\frac{f(x_0 + \\bigtriangleup x, y_0)-f(x_0,y_0)}{\\bigtriangleup x}=A x0 lim x f(x 0 +x,y 0 )f(x 0 ,y 0 ) =A AAA  z=f(x,y)z=f(x,y)z=f(x,y)  (x0,y0)(x_0,y_0)(x 0 ,y 0 )  xxx fx(x0,y0)f_x(x_0,y_0)f x (x 0 ,y 0 )  zxx=x0y=y0\\frac{\\partial z}{\\partial x} | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}} x z  x y =x 0 =y 0  fxx=x0y=y0\\frac{\\partial f}{\\partial x} | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}} x f  x y =x 0 =y 0  zxx=x0y=y0z_x | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}}z x  x y =x 0 =y 0 \n\n\n#  \n\n l=(a,b)\\vec{l}=(a,b) l  =(a,b) lll  l0=1a2+b2(a,b)=(aa2+b2,ba2+b2)=(cos,cos)\\vec{l^0} = \\frac{1}{\\sqrt{a^2+b^2}}(a,b) = (\\frac{a}{\\sqrt{a^2+b^2}},\\frac{b}{\\sqrt{a^2+b^2}}) = (\\cos \\alpha, \\cos \\beta) l 0  =  a 2 +b 2 1 (a,b)=(  a 2 +b 2 a ,  a 2 +b 2 b )=(cos,cos) \n\n\n\n\n\n z=f(x,y)z=f(x,y)z=f(x,y)  P(x,y)P(x,y)P(x,y)  U(P)U(P)U(P)  PPP  lll xxx  lll  \\varphiP(x+x,y+y)U(P)P'(x + \\triangle x, y + \\triangle y) \\in U(P)P  (x+x,y+y)U(P)  lll  lim0f(x+x,y+y)f(x,y)(=(x)2+(y)2)\\lim\\limits_{\\rho \\to 0}\\frac{f(x + \\triangle x, y + \\triangle y) - f(x,y)}{\\rho}(\\rho = \\sqrt{(\\triangle x)^2+(\\triangle y)^2}) 0 lim  f(x+x,y+y)f(x,y) (= (x) 2 +(y) 2 )  f(x,y)f(x,y)f(x,y)  PPP  lll  fl\\frac{\\partial f}{\\partial l} l f  fl=fxcos+fysin\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos \\varphi + \\frac{\\partial f}{\\partial y}\\sin \\varphi l f = x f cos+ y f sin \n\n u=f(x,y,z)u=f(x,y,z)u=f(x,y,z)  P(x,y,z)P(x,y,z)P(x,y,z)  lll ( ,,\\alpha,\\beta,\\gamma,,) fl=lim0f(x+x,y+y,z+z)f(x,y,z)\\frac{\\partial f}{\\partial l} = \\lim\\limits_{\\rho \\to 0} \\frac{f(x + \\triangle x, y + \\triangle y, z + \\triangle z) - f(x,y,z)}{\\rho} l f = 0 lim  f(x+x,y+y,z+z)f(x,y,z)  =(x)2+(y)2+(z)2\\rho = \\sqrt{(\\triangle x)^2 + (\\triangle y)^2 + (\\triangle z)^2}= (x) 2 +(y) 2 +(z) 2  P(x+x,y+y,z+z)P'(x + \\triangle x, y + \\triangle y, z + \\triangle z)P  (x+x,y+y,z+z)  lll  fl=fxcos+fycos+fzcos\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos\\alpha + \\frac{\\partial f}{\\partial y}\\cos\\beta + \\frac{\\partial f}{\\partial z}\\cos\\gamma l f = x f cos+ y f cos+ z f cos \n\n\n#  \n\n z=f(x,y)z=f(x,y)z=f(x,y)  DDD  P(x,y)P(x,y)P(x,y)  {fx,fy}=fx(x,y)i+fy(x,y)j\\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace = f_x(x,y)\\vec{i} + f_y(x,y)\\vec{j}{ x f , y f }=f x (x,y) i  +f y (x,y) j   z=f(x,y)z=f(x,y)z=f(x,y)  P(x,y)P(x,y)P(x,y)  gradf(x,y)gradf(x,y)gradf(x,y)  f(x,y)\\bigtriangledown f(x,y)f(x,y)gradf(x,y)=f(x,y)={fx,fy}=fx(x,y)i+fy(x,y)jgradf(x,y) = \\bigtriangledown f(x,y) = \\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace = f_x(x,y)\\vec{i} + f_y(x,y)\\vec{j}gradf(x,y)=f(x,y)={ x f , y f }=f x (x,y) i  +f y (x,y) j \n e={cos,cos}e=\\lbrace \\cos\\alpha, \\cos\\beta \\rbracee={cos,cos}  lll \nfl=fxcos+fycos={fx,fy}{cos,cos}=gradf(x,y)e=gradf(x,y)ecos[gradf(x,y),e]\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos\\alpha + \\frac{\\partial f}{\\partial y}\\cos\\beta = \\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace \\lbrace\\cos\\alpha,\\cos\\beta\\rbrace = gradf(x,y)e = |gradf(x,y)|*|e|*\\cos[gradf(x,y),e] l f = x f cos+ y f cos={ x f , y f }{cos,cos}=gradf(x,y)e=gradf(x,y)ecos[gradf(x,y),e]\n lll \ncos[gradf(x,y),e]=1\\cos[gradf(x,y),e]=1cos[gradf(x,y),e]=1\n lll  fl\\frac{\\partial f}{\\partial l} l f \ngradf(x,y)=(fx)2+(fy)2|gradf(x,y)|=\\sqrt{(\\frac{\\partial f}{\\partial x})^2 + (\\frac{\\partial f}{\\partial y})^2}gradf(x,y)= ( x f ) 2 +( y f ) 2\n\n\n\n#  \n\n\n#  \n\n\n#  ",normalizedContent:"#  \n\n\n#  \n\n * y=f(x)y=f(x)y=f(x) xy\n * y0=yx=x0y_0=y|_{x=x_0}y 0 =y x=x 0 y0=f(x0)y_0=f(x_0)y 0 =f(x 0 )  x0x_0x 0 \n\n\n#  \n\n *  f(x)={x,x0x,x<0f(x) = \\begin{cases} \\sqrt{x}, &x \\geqslant 0 \\\\ -x, &x<0 \\end{cases}f(x)={  x , x, x0 x<0\n *  h=12gt2t=2hgh=\\frac{1}{2}gt^2 \\to t=\\sqrt{\\frac{2h}{g}}h= 2 1 gt 2 t= g 2h  t=t(h)t=t(h)t=t(h)  h=h(t)h=h(t)h=h(t) \n *  y=x2+1y=x^2+1y=x 2 +1\n *  f(x,y)=0f(x,y)=0f(x,y)=0  3x+y4=03x+y-4=03x+y4=0\n\n\n#  \n\n * \n   *  f(x)=f(x)f(-x)=f(x)f(x)=f(x)y\n   *  f(x)=f(x)f(-x)=-f(x)f(x)=f(x)\n *  f(x+t)=f(x)f(x+t)=f(x)f(x+t)=f(x)\n *  \n\n\n#  \n\n\n#  \n\n  u1,u2,...,un,...u_1,u_2,...,u_n,...u 1 ,u 2 ,...,u n ,... unu_nu n \n\n  {un}\\lbrace u_n \\rbrace{u n }  nnn  aaa aaa  aaa limn13n=0\\lim\\limits_{n \\to \\infty} \\frac{1}{3^n} = 0 n lim 3 n 1 =0 limn2n\\lim\\limits_{n \\to \\infty} 2^n n lim 2 n \n\n\n#  \n\n  x0x_0x 0  x0(x0,x0+)x_0\\in(x_0-\\delta,x_0+\\delta)x 0 (x 0 ,x 0 +)\n\n  x0x_0x 0  x0x_0x 0 \n\n\n\n\n\n  x0x_0x 0  aaa \\varepsilonf(x)a<|f(x)-a|<\\varepsilonf(x)a< aaa  xx0x \\to x_0xx 0  limxx0f(x)=a\\lim\\limits_{x \\to x_0}f(x)=a xx 0 lim f(x)=a\n limxx0f(x)=limxx0+f(x)=a\\lim\\limits_{x \\to x_0^-}f(x)=\\lim\\limits_{x \\to x_0^+}f(x)=a xx 0  lim f(x)= xx 0 + lim f(x)=a\n\n\n\n\n\n\n\n#  \n\n limx1x=0\\lim\\limits_{x \\to \\infty} \\frac{1}{x} = 0 x lim x 1 =0 1x\\frac{1}{x} x 1  xx \\to \\inftyx limx2(3x6)=0\\lim\\limits_{x \\to 2} (3x-6) = 0 x2 lim (3x6)=0 3x63x-63x6  x2x \\to 2x2 \n\n  ++\\infty+-\\infty\n\n\n\n * \n * \n * \n * \n   \n * limx0x2x=12\\lim\\limits_{x \\to 0} \\frac{x}{2x} = \\frac{1}{2} x0 lim 2x x = 2 1\n\n limxx0f(x)=a\\lim\\limits_{x \\to x_0}f(x)=a xx 0 lim f(x)=a  f(x)=a+a(x)f(x)=a+a(x)f(x)=a+a(x)  a(x)a(x)a(x)  xx0x \\to x_0xx 0 \n\n a=a(x),b=b(x)a=a(x),b=b(x)a=a(x),b=b(x) limxx0a(x)=0\\lim\\limits_{x \\to x_0}a(x)=0 xx 0 lim a(x)=0limxx0b(x)=0\\lim\\limits_{x \\to x_0}b(x)=0 xx 0 lim b(x)=0\n\n *  limxx0ba=0\\lim\\limits_{x \\to x_0}\\frac{b}{a}=0 xx 0 lim a b =0 bbb  aaa \n *  limxx0ba=\\lim\\limits_{x \\to x_0}\\frac{b}{a}=\\infty xx 0 lim a b = bbb  aaa \n *  limxx0ba=c=0\\lim\\limits_{x \\to x_0}\\frac{b}{a}=c\\neq0 xx 0 lim a b =c=0 bbb  aaa \n\n\n#  \n\n  y=f(x)y=f(x)y=f(x)  x0x_0x 0  x\\triangle xx  y\\triangle yy  y=f(x)y=f(x)y=f(x)  x0x_0x 0 \n\n\n#  \n\n  f(x)f(x)f(x)  x=x0x=x_0x=x 0 \n\n\n\n *  xx0x \\to x_0xx 0 f(x)f(x)f(x)  x0x_0x 0  f(x)f(x)f(x) \n * limx0f(x)\\lim\\limits_{x \\to 0^-}f(x) x0  lim f(x)  limx0+f(x)\\lim\\limits_{x \\to 0^+}f(x) x0 + lim f(x) \n * limxx0f(x)\\lim\\limits_{x \\to x_0}f(x) xx 0 lim f(x)  f(x0)f(x_0)f(x 0 )\n\n\n#  \n\n\n#  \n\n  limx0yx=limx0f(x0+x)f(x0)x\\lim\\limits_{\\triangle x \\to 0} \\frac{\\triangle y}{\\triangle x} = \\lim\\limits_{\\triangle x \\to 0} \\frac{f(x_0 + \\triangle x)-f(x_0)}{\\triangle x} x0 lim x y = x0 lim x f(x 0 +x)f(x 0 )  y=f(x)y=f(x)y=f(x)  x0x_0x 0 f(x0)f'(x_0)f  (x 0 )yx=x0y'|_{x=x_0}y   x=x 0 dydxx=x0\\frac{dy}{dx}|_{x=x_0} dx dy  x=x 0  df(x)dxx=x0\\frac{df(x)}{dx}|_{x=x_0} dx df(x)  x=x 0\n\n\n\n * (uv)=uv(u \\pm v)' = u' \\pm v'(uv)  =u  v \n * (uv)=uv+uv(uv)' = u'v + uv'(uv)  =u  v+uv \n * (uv)=uvuvv2(v=0)(\\frac{u}{v})' = \\frac{u'v-uv'}{v^2}(v \\neq 0)( v u )  = v 2 u  vuv  (v=0)\n * (cu)=cu(cu)'=cu'(cu)  =cu \n * (cv)=cvv2(\\frac{c}{v})'=-\\frac{cv'}{v^2}( v c )  = v 2 cv  (c)\n\n\n\n                                   \ny=cy=cy=c (ccc )                   y=0y'=0y  =0\ny=axy=a^xy=a x                        y=axlnay'=a^x\\ln ay  =a x lna\ny=exy=e^xy=e x                        y=exy'=e^xy  =e x\ny=xny=x^ny=x n                        y=nxn1y'=nx^{n-1}y  =nx n1\ny=logaxy=\\log_a xy=log a x            y=1xlnay'=\\frac{1}{x \\ln a}y  = xlna 1\ny=lnxy=\\ln xy=lnx                     y=1xy'=\\frac{1}{x}y  = x 1\ny=exy=e^xy=e x                        y=exy'=e^xy  =e x\ny=sinxy=\\sin xy=sinx                  y=cosxy'=\\cos xy  =cosx\ny=cosxy=\\cos xy=cosx                  y=sinxy'=-\\sin xy  =sinx\ny=tanxy=\\tan xy=tanx                  y=sec2xy'=\\sec^2 xy  =sec 2 x\ny=cotxy=\\cot xy=cotx                  y=csc2xy'=-\\csc^2 xy  =csc 2 x\ny=secxy=\\sec xy=secx                  y=secxtanxy'=\\sec x \\tan xy  =secxtanx\ny=cscxy=\\csc xy=cscx                  y=cscxcotxy'=-\\csc x \\cot xy  =cscxcotx\ny=arcsinxy=\\arcsin xy=arcsinx         y=11x2y'=\\frac{1}{\\sqrt{1-x^2}}y  =  1x 2 1\ny=arccosxy=\\arccos xy=arccosx         y=11x2y'=-\\frac{1}{\\sqrt{1-x^2}}y  =  1x 2 1\ny=arctanxy=\\arctan xy=arctanx         y=11+x2y'=\\frac{1}{1+x^2}y  = 1+x 2 1\ny=arccotxy=\\text{arccot} xy=arccotx   y=11+x2y'=-\\frac{1}{1+x^2}y  = 1+x 2 1\ny=shxy=\\text{sh} xy=shx               y=chxy'=\\text{ch} xy  =chx\n\n\n#  \n\n z=f(x,y)z=f(x,y)z=f(x,y)  (x0,y0)(x_0,y_0)(x 0 ,y 0 )  y=y0y=y_0y=y 0  f(x,y0)f(x,y_0)f(x,y 0 )  x=x0x=x_0x=x 0  limx0f(x0+x,y0)f(x0,y0)x=a\\lim\\limits_{\\bigtriangleup x \\to 0} \\frac{f(x_0 + \\bigtriangleup x, y_0)-f(x_0,y_0)}{\\bigtriangleup x}=a x0 lim x f(x 0 +x,y 0 )f(x 0 ,y 0 ) =a aaa  z=f(x,y)z=f(x,y)z=f(x,y)  (x0,y0)(x_0,y_0)(x 0 ,y 0 )  xxx fx(x0,y0)f_x(x_0,y_0)f x (x 0 ,y 0 )  zxx=x0y=y0\\frac{\\partial z}{\\partial x} | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}} x z  x y =x 0 =y 0  fxx=x0y=y0\\frac{\\partial f}{\\partial x} | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}} x f  x y =x 0 =y 0  zxx=x0y=y0z_x | _{\\begin{aligned} x&=x_0\\\\ y&=y_0 \\end{aligned}}z x  x y =x 0 =y 0 \n\n\n#  \n\n l=(a,b)\\vec{l}=(a,b) l  =(a,b) lll  l0=1a2+b2(a,b)=(aa2+b2,ba2+b2)=(cos,cos)\\vec{l^0} = \\frac{1}{\\sqrt{a^2+b^2}}(a,b) = (\\frac{a}{\\sqrt{a^2+b^2}},\\frac{b}{\\sqrt{a^2+b^2}}) = (\\cos \\alpha, \\cos \\beta) l 0  =  a 2 +b 2 1 (a,b)=(  a 2 +b 2 a ,  a 2 +b 2 b )=(cos,cos) \n\n\n\n\n\n z=f(x,y)z=f(x,y)z=f(x,y)  p(x,y)p(x,y)p(x,y)  u(p)u(p)u(p)  ppp  lll xxx  lll  \\varphip(x+x,y+y)u(p)p'(x + \\triangle x, y + \\triangle y) \\in u(p)p  (x+x,y+y)u(p)  lll  lim0f(x+x,y+y)f(x,y)(=(x)2+(y)2)\\lim\\limits_{\\rho \\to 0}\\frac{f(x + \\triangle x, y + \\triangle y) - f(x,y)}{\\rho}(\\rho = \\sqrt{(\\triangle x)^2+(\\triangle y)^2}) 0 lim  f(x+x,y+y)f(x,y) (= (x) 2 +(y) 2 )  f(x,y)f(x,y)f(x,y)  ppp  lll  fl\\frac{\\partial f}{\\partial l} l f  fl=fxcos+fysin\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos \\varphi + \\frac{\\partial f}{\\partial y}\\sin \\varphi l f = x f cos+ y f sin \n\n u=f(x,y,z)u=f(x,y,z)u=f(x,y,z)  p(x,y,z)p(x,y,z)p(x,y,z)  lll ( ,,\\alpha,\\beta,\\gamma,,) fl=lim0f(x+x,y+y,z+z)f(x,y,z)\\frac{\\partial f}{\\partial l} = \\lim\\limits_{\\rho \\to 0} \\frac{f(x + \\triangle x, y + \\triangle y, z + \\triangle z) - f(x,y,z)}{\\rho} l f = 0 lim  f(x+x,y+y,z+z)f(x,y,z)  =(x)2+(y)2+(z)2\\rho = \\sqrt{(\\triangle x)^2 + (\\triangle y)^2 + (\\triangle z)^2}= (x) 2 +(y) 2 +(z) 2  p(x+x,y+y,z+z)p'(x + \\triangle x, y + \\triangle y, z + \\triangle z)p  (x+x,y+y,z+z)  lll  fl=fxcos+fycos+fzcos\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos\\alpha + \\frac{\\partial f}{\\partial y}\\cos\\beta + \\frac{\\partial f}{\\partial z}\\cos\\gamma l f = x f cos+ y f cos+ z f cos \n\n\n#  \n\n z=f(x,y)z=f(x,y)z=f(x,y)  ddd  p(x,y)p(x,y)p(x,y)  {fx,fy}=fx(x,y)i+fy(x,y)j\\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace = f_x(x,y)\\vec{i} + f_y(x,y)\\vec{j}{ x f , y f }=f x (x,y) i  +f y (x,y) j   z=f(x,y)z=f(x,y)z=f(x,y)  p(x,y)p(x,y)p(x,y)  gradf(x,y)gradf(x,y)gradf(x,y)  f(x,y)\\bigtriangledown f(x,y)f(x,y)gradf(x,y)=f(x,y)={fx,fy}=fx(x,y)i+fy(x,y)jgradf(x,y) = \\bigtriangledown f(x,y) = \\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace = f_x(x,y)\\vec{i} + f_y(x,y)\\vec{j}gradf(x,y)=f(x,y)={ x f , y f }=f x (x,y) i  +f y (x,y) j \n e={cos,cos}e=\\lbrace \\cos\\alpha, \\cos\\beta \\rbracee={cos,cos}  lll \nfl=fxcos+fycos={fx,fy}{cos,cos}=gradf(x,y)e=gradf(x,y)ecos[gradf(x,y),e]\\frac{\\partial f}{\\partial l} = \\frac{\\partial f}{\\partial x}\\cos\\alpha + \\frac{\\partial f}{\\partial y}\\cos\\beta = \\lbrace \\frac{\\partial f}{\\partial x},\\frac{\\partial f}{\\partial y} \\rbrace \\lbrace\\cos\\alpha,\\cos\\beta\\rbrace = gradf(x,y)e = |gradf(x,y)|*|e|*\\cos[gradf(x,y),e] l f = x f cos+ y f cos={ x f , y f }{cos,cos}=gradf(x,y)e=gradf(x,y)ecos[gradf(x,y),e]\n lll \ncos[gradf(x,y),e]=1\\cos[gradf(x,y),e]=1cos[gradf(x,y),e]=1\n lll  fl\\frac{\\partial f}{\\partial l} l f \ngradf(x,y)=(fx)2+(fy)2|gradf(x,y)|=\\sqrt{(\\frac{\\partial f}{\\partial x})^2 + (\\frac{\\partial f}{\\partial y})^2}gradf(x,y)= ( x f ) 2 +( y f ) 2\n\n\n\n#  \n\n\n#  \n\n\n#  ",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{title:"",date:"2023-01-02T20:10:43.000Z",permalink:"/pages/cbc016/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/01.%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80/4.%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1.html",relativePath:"05./01./4..md",key:"v-7384231e",path:"/pages/cbc016/",headers:[{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:2},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:848},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:864},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1107},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1399},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:1794},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1812},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1828},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:1980},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:1997},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:2557},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:2733},{level:3,title:" (SD, standard Deviation)",slug:"-sd-standard-deviation",normalizedTitle:" (sd, standard deviation)",charIndex:3128},{level:2,title:" (Covariance)",slug:"-covariance",normalizedTitle:" (covariance)",charIndex:3385},{level:2,title:" (absolute error)",slug:"-absolute-error",normalizedTitle:" (absolute error)",charIndex:3536},{level:2,title:" (Relative Error)",slug:"-relative-error",normalizedTitle:" (relative error)",charIndex:3616},{level:2,title:" (residual)",slug:"-residual",normalizedTitle:" (residual)",charIndex:3685},{level:2,title:" (RSS, residual sum of squares)",slug:"-rss-residual-sum-of-squares",normalizedTitle:" (rss, residual sum of squares)",charIndex:3788},{level:2,title:" (MSE, Mean Squared Error)",slug:"-mse-mean-squared-error",normalizedTitle:" (mse, mean squared error)",charIndex:3927},{level:2,title:" (RMSE, Root Mean Squared Error)",slug:"-rmse-root-mean-squared-error",normalizedTitle:" (rmse, root mean squared error)",charIndex:4035},{level:2,title:" (MAE, Mean Absolute Error)",slug:"-mae-mean-absolute-error",normalizedTitle:" (mae, mean absolute error)",charIndex:4155},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:4216},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:4228},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:4934},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:4946},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:4992},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5009},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5026},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5104},{level:2,title:" ",slug:"",normalizedTitle:" ",charIndex:5576},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5590},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:5877},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6205},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6268},{level:3,title:" ",slug:"",normalizedTitle:" ",charIndex:6379},{level:3,title:" Beta",slug:"beta",normalizedTitle:" beta",charIndex:6585}],excerpt:"",headersStr:"                         (SD, standard Deviation)  (Covariance)  (absolute error)  (Relative Error)  (residual)  (RSS, residual sum of squares)  (MSE, Mean Squared Error)  (RMSE, Root Mean Squared Error)  (MAE, Mean Absolute Error)                              Beta",content:"#  \n\n  EEE\n\n  EEE S={0,1,2,...}S=\\lbrace 0,1,2,... \\rbraceS={0,1,2,...}\n\n AAA  nnn fn(A)=nAnf_n(A) = \\frac{n_A}{n}f n (A)= n n A  nAn_An A  AAA ()nnn \n\n fn(A)f_n(A)f n (A)  ppp  AAA  P(A)=pP(A)=pP(A)=p\n\n  EEE \n\n  AAA  BBB P(AB)=P(AB)P(B)P(A|B)=\\frac{P(AB)}{P(B)}P(AB)= P(B) P(AB) \n\n  AAABBB  P(AB)=P(A)P(B)P(AB)=P(A)*P(B)P(AB)=P(A)P(B) AAABBB \n\nn  AAA  ,  nnn  nnn  nnn  AAA  p(0<p<1)p(0<p<1)p(0<p<1) nnn  AAA  kkk Pn(k)=CnkpkqnkP_n(k)=C_n^kp^kq^{n-k}P n (k)=C n k p k q nk  k=0,1,...,nk=0,1,...,nk=0,1,...,nq=1pq=1-pq=1p\n\n\n#  \n\n\n#  \n\n  XXX  XXX \n\n  XXX  XXX  XXX  xi(i=1,2,...,n)x_i(i=1,2,...,n)x i (i=1,2,...,n) P(X=xi)=piP(X=x_i)=p_iP(X=x i )=p i  XXX \n\n\n#  \n\n  XXX  XXX \n\n  XXX  f(x)f(x)f(x)  F(x)=xf(t)dt(<x<)F(x)=\\int_{-\\infty}^{x}f(t)dt (-\\infty<x<\\infty)F(x)=  x f(t)dt(<x<) F(x)F(x)F(x)  XXX f(x)\n\n\n\n\n\n\n#  \n\n\n(1)  X1,X2,...,XnX_1,X_2,...,X_nX 1 ,X 2 ,...,X n  (2)  X1,X2,...,XnX_1,X_2,...,X_nX 1 ,X 2 ,...,X n X\n\n F(X1,X2,...,Xn)=i=1nF(Xi)F(X_1,X_2,...,X_n)=\\prod \\limits_{i=1}^{n}F(X_i)F(X 1 ,X 2 ,...,X n )= i=1  n F(X i )\n\n f(X1,X2,...,Xn)=i=1nf(xi)f(X_1,X_2,...,X_n)=\\prod \\limits_{i=1}^{n}f(x_i)f(X 1 ,X 2 ,...,X n )= i=1  n f(x i )\n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n  (X,Y)(X,Y)(X,Y)  F(x,y)F(x,y)F(x,y) XXX  YYY FX(x)F_X(x)F X (x)FY(y)F_Y(y)F Y (y) \n\n\n#  \n\n\n#  \n\n\n\n  XXX P(X=xk)=pkP(X=x_k)=p_kP(X=x k )=p k (k=1,2,...k=1,2,...k=1,2,...) k=1xkpk\\sum \\limits_{k=1}^{\\infty}x_kp_k k=1   x k p k  XXX E(X)=k=1xkpkE(X)=\\sum \\limits_{k=1}^{\\infty}x_kp_kE(X)= k=1   x k p k \n\n  XXX  f(x)f(x)f(x) +xf(x)dx\\int_{-\\infty}^{+\\infty}xf(x)dx  + xf(x)dx  +xf(x)dx\\int_{-\\infty}^{+\\infty}xf(x)dx  + xf(x)dx  XXX E(X)=+xf(x)dxE(X)=\\int_{-\\infty}^{+\\infty}xf(x)dxE(X)=  + xf(x)dx\n\n\n#  \n\n =i=1NxiN\\mu = \\frac{\\sum_{i=1}^N x_i}{N}= N  i=1 N x i\nNNN \n\n x=i=1nxin\\overline{x} = \\frac{\\sum_{i=1}^n x_i}{n} x = n  i=1 n x i\nnnn \n\n\n#  \n\n\n\n 2=i=1N(xi)2N\\sigma^2=\\frac{\\sum_{i=1}^N(x_i-\\mu)^2}{N} 2 = N  i=1 N (x i ) 2\n\n S2=i=1n(xix)2n1S^2=\\frac{\\sum_{i=1}^n(x_i-\\overline{x})^2}{n-1}S 2 = n1  i=1 n (x i  x ) 2\n\n XXX  E[XE(X)]2E[X-E(X)]^2E[XE(X)] 2  XXX  D(X)=E[XE(X)]2=E(X2)[E(X)]2D(X)=E[X-E(X)]^2=E(X^2)-[E(X)]^2D(X)=E[XE(X)] 2 =E(X 2 )[E(X)] 2\n\n\n#  (SD, standard Deviation)\n\n\n\n =i=1N(xi)2N\\sigma = \\sqrt{\\frac{\\sum_{i=1}^N(x_i-\\mu)^2}{N}}= N  i=1 N (x i ) 2\n\n S=i=1n(xix)2n1S = \\sqrt{\\frac{\\sum_{i=1}^n(x_i-\\overline{x})^2}{n-1}}S= n1  i=1 n (x i  x ) 2\n\n\n#  (Covariance)\n\n\n\nCov(X,Y)=E[(Xx)(Yy)]Cov(X,Y)=E[(X-\\mu_x)(Y-\\mu_y)]Cov(X,Y)=E[(X x )(Y y )]\n\n\n#  (absolute error)\n\n\n\n\\varepsilon = -\n\n\n#  (Relative Error)\n\n\n\n\\delta = /100%\n\n\n#  (residual)\n\n\n\n^=yy^\\hat{\\varepsilon} = y - \\hat{y}  ^ =y y ^\n\n\n#  (RSS, residual sum of squares)\n\n\n\nRSS=^2=(yy^)2RSS=\\hat{\\varepsilon}^2=(y - \\hat{y})^2RSS=  ^ 2 =(y y ^ ) 2\n\n\n#  (MSE, Mean Squared Error)\n\nRSS\n\nMSE=E(yy^)2MSE = E(y - \\hat{y})^2MSE=E(y y ^ ) 2\n\n\n#  (RMSE, Root Mean Squared Error)\n\n\n\nRMSE=E(yy^)2RMSE = \\sqrt{E(y - \\hat{y})^2}RMSE= E(y y ^ ) 2\n\n\n#  (MAE, Mean Absolute Error)\n\n\n\nMAE = /N\n\n\n#  \n\n\n#  \n\n  xxx  \\theta  L(x)L(\\theta|x)L(x)  \\theta  XXX L(x)=P(X=x)L(\\theta|x)=P(X=x|\\theta)L(x)=P(X=x)\n\\theta11-\\theta1L()=3(1)2L(\\theta)=\\theta^3(1-\\theta)^2L()= 3 (1) 2\n\n L()=i=1np(xi)L(\\theta)=\\prod \\limits_{i=1}^{n}p(x_i|\\theta)L()= i=1  n p(x i )\n\n L()=i=1nf(xi)L(\\theta)=\\prod \\limits_{i=1}^{n}f(x_i|\\theta)L()= i=1  n f(x i )\n\n:  L(x)L(\\theta|x)L(x)  \\theta \n\n\n1.L()L(\\theta)L()\n2.lnL()\\ln L(\\theta)lnL()\n3.dlnLd=0\\frac{d \\ln L}{d \\theta}=0 d dlnL =0\n4. \\theta \n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n P(BA)P(B|A)P(BA)  P(AB)P(A|B)P(AB) P(AB)=P(BA)P(A)P(B)P(A|B)=\\frac{P(B|A)P(A)}{P(B)}P(AB)= P(B) P(BA)P(A)  P(BA)P(B|A)P(BA)  P(AB)P(A|B)P(AB)\n\n * P(AB)P(A|B)P(AB) (posterior)\n * P(BA)P(B|A)P(BA) (likelihood) AAA  BBB(evidence)\n * P(A)P(A)P(A) (prior) AAA \n * P(B)P(B)P(B) (evidence) BBB(evidence)\n\n\n#  \n\n\n#  \n\nNormal distributionGaussian distribution\n\n XXX  \\mu 2\\sigma^2 2  XN(,2)X \\sim N(\\mu,\\sigma^2)XN(, 2 )    \\sigma =0,=1\\mu=0,\\sigma=1=0,=1 \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n XXX  nnn  ppp  XB(N,P)X \\sim B(N,P)XB(N,P)\n\n\n\n\n k=0,1,2,...,nk=0,1,2,...,nk=0,1,2,...,n(nk)=Cnk=C(n,k)=n!k!(nk)!\\begin{pmatrix} n \\\\ k \\end{pmatrix}=C_n^k=C(n,k)=\\frac{n!}{k!(n-k)!}( n k )=C n k =C(n,k)= k!(nk)! n! \n\n\n\n\n\n\n\n\n\n\n#  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n16n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n nnn  1,2,...,n\\xi_1,\\xi_2,...,\\xi_n 1 , 2 ,..., n  nnn chi-square distribution\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  Beta",normalizedContent:"#  \n\n  eee\n\n  eee s={0,1,2,...}s=\\lbrace 0,1,2,... \\rbraces={0,1,2,...}\n\n aaa  nnn fn(a)=nanf_n(a) = \\frac{n_a}{n}f n (a)= n n a  nan_an a  aaa ()nnn \n\n fn(a)f_n(a)f n (a)  ppp  aaa  p(a)=pp(a)=pp(a)=p\n\n  eee \n\n  aaa  bbb p(ab)=p(ab)p(b)p(a|b)=\\frac{p(ab)}{p(b)}p(ab)= p(b) p(ab) \n\n  aaabbb  p(ab)=p(a)p(b)p(ab)=p(a)*p(b)p(ab)=p(a)p(b) aaabbb \n\nn  aaa  a,  nnn  nnn  nnn  aaa  p(0<p<1)p(0<p<1)p(0<p<1) nnn  aaa  kkk pn(k)=cnkpkqnkp_n(k)=c_n^kp^kq^{n-k}p n (k)=c n k p k q nk  k=0,1,...,nk=0,1,...,nk=0,1,...,nq=1pq=1-pq=1p\n\n\n#  \n\n\n#  \n\n  xxx  xxx \n\n  xxx  xxx  xxx  xi(i=1,2,...,n)x_i(i=1,2,...,n)x i (i=1,2,...,n) p(x=xi)=pip(x=x_i)=p_ip(x=x i )=p i  xxx \n\n\n#  \n\n  xxx  xxx \n\n  xxx  f(x)f(x)f(x)  f(x)=xf(t)dt(<x<)f(x)=\\int_{-\\infty}^{x}f(t)dt (-\\infty<x<\\infty)f(x)=  x f(t)dt(<x<) f(x)f(x)f(x)  xxx f(x)\n\n\n\n\n\n\n#  \n\n\n(1)  x1,x2,...,xnx_1,x_2,...,x_nx 1 ,x 2 ,...,x n  (2)  x1,x2,...,xnx_1,x_2,...,x_nx 1 ,x 2 ,...,x n x\n\n f(x1,x2,...,xn)=i=1nf(xi)f(x_1,x_2,...,x_n)=\\prod \\limits_{i=1}^{n}f(x_i)f(x 1 ,x 2 ,...,x n )= i=1  n f(x i )\n\n f(x1,x2,...,xn)=i=1nf(xi)f(x_1,x_2,...,x_n)=\\prod \\limits_{i=1}^{n}f(x_i)f(x 1 ,x 2 ,...,x n )= i=1  n f(x i )\n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n  (x,y)(x,y)(x,y)  f(x,y)f(x,y)f(x,y) xxx  yyy fx(x)f_x(x)f x (x)fy(y)f_y(y)f y (y) \n\n\n#  \n\n\n#  \n\n\n\n  xxx p(x=xk)=pkp(x=x_k)=p_kp(x=x k )=p k (k=1,2,...k=1,2,...k=1,2,...) k=1xkpk\\sum \\limits_{k=1}^{\\infty}x_kp_k k=1   x k p k  xxx e(x)=k=1xkpke(x)=\\sum \\limits_{k=1}^{\\infty}x_kp_ke(x)= k=1   x k p k \n\n  xxx  f(x)f(x)f(x) +xf(x)dx\\int_{-\\infty}^{+\\infty}xf(x)dx  + xf(x)dx  +xf(x)dx\\int_{-\\infty}^{+\\infty}xf(x)dx  + xf(x)dx  xxx e(x)=+xf(x)dxe(x)=\\int_{-\\infty}^{+\\infty}xf(x)dxe(x)=  + xf(x)dx\n\n\n#  \n\n =i=1nxin\\mu = \\frac{\\sum_{i=1}^n x_i}{n}= n  i=1 n x i\nnnn \n\n x=i=1nxin\\overline{x} = \\frac{\\sum_{i=1}^n x_i}{n} x = n  i=1 n x i\nnnn \n\n\n#  \n\n\n\n 2=i=1n(xi)2n\\sigma^2=\\frac{\\sum_{i=1}^n(x_i-\\mu)^2}{n} 2 = n  i=1 n (x i ) 2\n\n s2=i=1n(xix)2n1s^2=\\frac{\\sum_{i=1}^n(x_i-\\overline{x})^2}{n-1}s 2 = n1  i=1 n (x i  x ) 2\n\n xxx  e[xe(x)]2e[x-e(x)]^2e[xe(x)] 2  xxx  d(x)=e[xe(x)]2=e(x2)[e(x)]2d(x)=e[x-e(x)]^2=e(x^2)-[e(x)]^2d(x)=e[xe(x)] 2 =e(x 2 )[e(x)] 2\n\n\n#  (sd, standard deviation)\n\n\n\n =i=1n(xi)2n\\sigma = \\sqrt{\\frac{\\sum_{i=1}^n(x_i-\\mu)^2}{n}}= n  i=1 n (x i ) 2\n\n s=i=1n(xix)2n1s = \\sqrt{\\frac{\\sum_{i=1}^n(x_i-\\overline{x})^2}{n-1}}s= n1  i=1 n (x i  x ) 2\n\n\n#  (covariance)\n\n\n\ncov(x,y)=e[(xx)(yy)]cov(x,y)=e[(x-\\mu_x)(y-\\mu_y)]cov(x,y)=e[(x x )(y y )]\n\n\n#  (absolute error)\n\n\n\n\\varepsilon = -\n\n\n#  (relative error)\n\n\n\n\\delta = /100%\n\n\n#  (residual)\n\n\n\n^=yy^\\hat{\\varepsilon} = y - \\hat{y}  ^ =y y ^\n\n\n#  (rss, residual sum of squares)\n\n\n\nrss=^2=(yy^)2rss=\\hat{\\varepsilon}^2=(y - \\hat{y})^2rss=  ^ 2 =(y y ^ ) 2\n\n\n#  (mse, mean squared error)\n\nrss\n\nmse=e(yy^)2mse = e(y - \\hat{y})^2mse=e(y y ^ ) 2\n\n\n#  (rmse, root mean squared error)\n\n\n\nrmse=e(yy^)2rmse = \\sqrt{e(y - \\hat{y})^2}rmse= e(y y ^ ) 2\n\n\n#  (mae, mean absolute error)\n\n\n\nmae = /n\n\n\n#  \n\n\n#  \n\n  xxx  \\theta  l(x)l(\\theta|x)l(x)  \\theta  xxx l(x)=p(x=x)l(\\theta|x)=p(x=x|\\theta)l(x)=p(x=x)\n\\theta11-\\theta1l()=3(1)2l(\\theta)=\\theta^3(1-\\theta)^2l()= 3 (1) 2\n\n l()=i=1np(xi)l(\\theta)=\\prod \\limits_{i=1}^{n}p(x_i|\\theta)l()= i=1  n p(x i )\n\n l()=i=1nf(xi)l(\\theta)=\\prod \\limits_{i=1}^{n}f(x_i|\\theta)l()= i=1  n f(x i )\n\n:  l(x)l(\\theta|x)l(x)  \\theta \n\n\n1.l()l(\\theta)l()\n2.lnl()\\ln l(\\theta)lnl()\n3.dlnld=0\\frac{d \\ln l}{d \\theta}=0 d dlnl =0\n4. \\theta \n\n\n#  \n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n\n\n\n#  \n\n p(ba)p(b|a)p(ba)  p(ab)p(a|b)p(ab) p(ab)=p(ba)p(a)p(b)p(a|b)=\\frac{p(b|a)p(a)}{p(b)}p(ab)= p(b) p(ba)p(a)  p(ba)p(b|a)p(ba)  p(ab)p(a|b)p(ab)\n\n * p(ab)p(a|b)p(ab) (posterior)\n * p(ba)p(b|a)p(ba) (likelihood) aaa  bbb(evidence)\n * p(a)p(a)p(a) (prior) aaa \n * p(b)p(b)p(b) (evidence) bbb(evidence)\n\n\n#  \n\n\n#  \n\nnormal distributiongaussian distribution\n\n xxx  \\mu 2\\sigma^2 2  xn(,2)x \\sim n(\\mu,\\sigma^2)xn(, 2 )    \\sigma =0,=1\\mu=0,\\sigma=1=0,=1 \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n xxx  nnn  ppp  xb(n,p)x \\sim b(n,p)xb(n,p)\n\n\n\n\n k=0,1,2,...,nk=0,1,2,...,nk=0,1,2,...,n(nk)=cnk=c(n,k)=n!k!(nk)!\\begin{pmatrix} n \\\\ k \\end{pmatrix}=c_n^k=c(n,k)=\\frac{n!}{k!(n-k)!}( n k )=c n k =c(n,k)= k!(nk)! n! \n\n\n\n\n\n\n\n\n\n\n#  \n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n16n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  \n\n nnn  1,2,...,n\\xi_1,\\xi_2,...,\\xi_n 1 , 2 ,..., n  nnn chi-square distribution\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n#  beta",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"3 ",frontmatter:{title:"3 ",date:"2022-12-20T15:37:40.000Z",permalink:"/pages/3e695c/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/11.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E8%A5%BF%E7%93%9C%E4%B9%A6/3.%E7%BA%BF%E6%80%A7%E6%A8%A1%E5%9E%8B.html",relativePath:"05./11. - /3..md",key:"v-0f0ab9bb",path:"/pages/3e695c/",headers:[{level:2,title:"3.1 ",slug:"_3-1-",normalizedTitle:"3.1 ",charIndex:2}],excerpt:"",headersStr:"3.1 ",content:"# 3.1 \n\n ddd  x=(x1;x2;...;xd)x=(x_1;x_2;...;x_d)x=(x 1 ;x 2 ;...;x d ) xix_ix i  xxx  iii (linear model)\n\nf(x)=w1x1+w2x2+...+wdxd+bf(x)=w_1x_1 + w_2x_2 + ... + w_dx_d + b f(x)=w 1 x 1 +w 2 x 2 +...+w d x d +b\n\n\n\nf(x)=wTx+bf(x)=w^Tx + b f(x)=w T x+b\n\n w=(w1;w2;...;wd)w=(w_1;w_2;...;w_d)w=(w 1 ;w 2 ;...;w d )www  bbb \n\n(nonlinear model) www (comprehensibility) fff(x)=0.2x(x)=0.2*x(x)=0.2x + 0.5x0.5*x0.5x + 0.3x0.3*x0.3x1",normalizedContent:"# 3.1 \n\n ddd  x=(x1;x2;...;xd)x=(x_1;x_2;...;x_d)x=(x 1 ;x 2 ;...;x d ) xix_ix i  xxx  iii (linear model)\n\nf(x)=w1x1+w2x2+...+wdxd+bf(x)=w_1x_1 + w_2x_2 + ... + w_dx_d + b f(x)=w 1 x 1 +w 2 x 2 +...+w d x d +b\n\n\n\nf(x)=wtx+bf(x)=w^tx + b f(x)=w t x+b\n\n w=(w1;w2;...;wd)w=(w_1;w_2;...;w_d)w=(w 1 ;w 2 ;...;w d )www  bbb \n\n(nonlinear model) www (comprehensibility) fff(x)=0.2x(x)=0.2*x(x)=0.2x + 0.5x0.5*x0.5x + 0.3x0.3*x0.3x1",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"2 ",frontmatter:{title:"2 ",date:"2022-12-05T21:15:19.000Z",permalink:"/pages/f7fe10/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/11.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E8%A5%BF%E7%93%9C%E4%B9%A6/2.%E6%A8%A1%E5%9E%8B%E8%AF%84%E4%BC%B0%E4%B8%8E%E9%80%89%E6%8B%A9.html",relativePath:"05./11. - /2..md",key:"v-6d8cebcc",path:"/pages/f7fe10/",headers:[{level:2,title:"2.1 ",slug:"_2-1-",normalizedTitle:"2.1 ",charIndex:2},{level:2,title:"2.2 ",slug:"_2-2-",normalizedTitle:"2.2 ",charIndex:580},{level:3,title:"2.2.1 ",slug:"_2-2-1-",normalizedTitle:"2.2.1 ",charIndex:1265},{level:3,title:"2.2.2 ",slug:"_2-2-2-",normalizedTitle:"2.2.2 ",charIndex:1535},{level:3,title:"2.2.3 ",slug:"_2-2-3-",normalizedTitle:"2.2.3 ",charIndex:2103},{level:3,title:"2.2.4 ",slug:"_2-2-4-",normalizedTitle:"2.2.4 ",charIndex:2621},{level:2,title:"2.3 ",slug:"_2-3-",normalizedTitle:"2.3 ",charIndex:3125},{level:3,title:"2.3.1 ",slug:"_2-3-1-",normalizedTitle:"2.3.1 ",charIndex:3493},{level:3,title:"2.3.2 //F1",slug:"_2-3-2---f1",normalizedTitle:"2.3.2 //f1",charIndex:3598},{level:3,title:"2.3.3 ROC  AUC",slug:"_2-3-3-roc--auc",normalizedTitle:"2.3.3 roc  auc",charIndex:4799},{level:3,title:"2.3.4 ",slug:"_2-3-4-",normalizedTitle:"2.3.4 ",charIndex:5575},{level:2,title:"2.4 ",slug:"_2-4-",normalizedTitle:"2.4 ",charIndex:6052},{level:3,title:"2.4.1 ",slug:"_2-4-1-",normalizedTitle:"2.4.1 ",charIndex:6248},{level:3,title:"2.4.2 t",slug:"_2-4-2-t",normalizedTitle:"2.4.2 t",charIndex:6263},{level:3,title:"2.4.3 McNemar",slug:"_2-4-3-mcnemar",normalizedTitle:"2.4.3 mcnemar",charIndex:6281},{level:3,title:"2.4.4 FriedmanNemenyi",slug:"_2-4-4-friedmannemenyi",normalizedTitle:"2.4.4 friedmannemenyi",charIndex:6301},{level:2,title:"2.5 ",slug:"_2-5-",normalizedTitle:"2.5 ",charIndex:6334}],excerpt:"",headersStr:"2.1  2.2  2.2.1  2.2.2  2.2.3  2.2.4  2.3  2.3.1  2.3.2 //F1 2.3.3 ROC  AUC 2.3.4  2.4  2.4.1  2.4.2 t 2.4.3 McNemar 2.4.4 FriedmanNemenyi 2.5 ",content:'# 2.1 \n\n(error): \n\n * training errorempirical error\n * test error\n * generalization error\n\n\n\n(overfitting): \n(underfitting): \n\n\n\n\n\n\n# 2.2 \n\n\n\nwhy\n\n10 1010 """"\n\n mmm  D={(x1,y1),(x2,y2),...,(xm,ym)}D = \\lbrace (x_1,y_1),(x_2,y_2),...,(x_m,y_m) \\rbraceD={(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m )}  DDD  SSS  TTT\n\n\n# 2.2.1 \n\n DDD  SSS TTT D=STD=S\\bigcup TD=ST  ST=S \\bigcap T=\\emptysetST= 2/3 ~ 4/5 /\n\n\n# 2.2.2 \n\n DDD  kkk  D=D1D2...DkD=D_1 \\bigcup D_2 \\bigcup ... \\bigcup D_kD=D 1 D 2 ...D k DiDj=(ij)D_i \\bigcap D_j =\\emptyset (i \\neq j)D i D j =(ij) k1k-1k1  KKK / kkk  kkk kk-fold cross validationkkk 1010\n\n\n\n DDD  kkk Kppk1010100/ kkk \n\n\n# 2.2.3 \n\n DDD  DDD \n\n mmm  DDD DDD  DD\'D   DDD  mmm  mmm  DD\'D   mmm \n\n\n\n DDD  36.8%  DD\'D   DD\'D  D\\D\' /\n\n\n# 2.2.4 \n\n(parameter) """" (parameter tuning)\n\n35/ 53=1255^3= 1255 3 =125 \n\n(validation set)\n\n\n# 2.3 \n\n(performance measure)\n\n D={(x1,y1),(x2,y2),...,(xm,ym))}D=\\lbrace (x_1,y_1),(x_2,y_2),...,(x_m,y_m))\\rbraceD={(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m ))} yiy_iy i  xix_ix i  fff  f(x)f(x)f(x)  yyy \n\n(mean squared error, MSE)\n\n\n\n\n# 2.3.1 \n\n+=1\n\n\n\n\n\n# 2.3.2 //F1\n\n/precision/recall\n\n(true positive)(false positive)(true negative)(false negative)TPFPTNFN TP+FP+TN+FN=(confusion matrix)\n\n\n\n\n\n-P-RP-R\n\n\n\nP-RACP-RABP-R\n\nF-MeasureF-ScoreF-MeasurePR\n\n\n\n\n=1F1PRF1\n\n\n\n\nPRPmacro-PRmacro-RFF1TPFPTNFNPRFF1\n\n\n\n\n# 2.3.3 ROC  AUC\n\nROC(Receiver Operating Characteristic)P-RROCP-R, ROC(True Positive RateTPR)(False PositiveRateFPR)2.1\n\n\n\n\n\n ROC ROC(0,1)\n\n\n\n\n\nP-R ROC  ROC  ROC  AUC(Area Under ROC Curve) \n\nAUC  ROC  ROC  {(x1,y1),(x2,y2),...,(xm,ym)}\\lbrace(x_1, y_1),(x_2, y_2),...,(x_m,y_m)\\rbrace{(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m )}  (x1=0,xm=1)(x_1=0,x_m=1)(x 1 =0,x m =1) AUC :\n\n\n\n\n\n\n# 2.3.4 \n\n--\x3e--\x3ecost matrix\n\n\n\n\n\n\n\n\n\n\n\n ROC [0,1]p[0,1]\n\n\n\n\n\n\n ROC (TPRFPR)  FNR(0FPR) (1FNR)  ROC \n\n\n\n\n\n\n# 2.4 \n\n(hypothesis test)ABAB \\epsilon \n\n\n# 2.4.1 \n\n\n# 2.4.2 t\n\n\n# 2.4.3 McNemar\n\n\n# 2.4.4 FriedmanNemenyi\n\n\n# 2.5 \n\n-\n\n * =+\n * \n * \n\n-bias-variance dilamma\n\n\n\n',normalizedContent:'# 2.1 \n\n(error): \n\n * training errorempirical error\n * test error\n * generalization error\n\n\n\n(overfitting): \n(underfitting): \n\n\n\n\n\n\n# 2.2 \n\n\n\nwhy\n\n10 1010 """"\n\n mmm  d={(x1,y1),(x2,y2),...,(xm,ym)}d = \\lbrace (x_1,y_1),(x_2,y_2),...,(x_m,y_m) \\rbraced={(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m )}  ddd  sss  ttt\n\n\n# 2.2.1 \n\n ddd  sss ttt d=std=s\\bigcup td=st  st=s \\bigcap t=\\emptysetst= 2/3 ~ 4/5 /\n\n\n# 2.2.2 \n\n ddd  kkk  d=d1d2...dkd=d_1 \\bigcup d_2 \\bigcup ... \\bigcup d_kd=d 1 d 2 ...d k didj=(i=j)d_i \\bigcap d_j =\\emptyset (i \\neq j)d i d j =(i=j) k1k-1k1  kkk / kkk  kkk kk-fold cross validationkkk 1010\n\n\n\n ddd  kkk kppk1010100/ kkk \n\n\n# 2.2.3 \n\n ddd  ddd \n\n mmm  ddd ddd  dd\'d   ddd  mmm  mmm  dd\'d   mmm \n\n\n\n ddd  36.8%  dd\'d   dd\'d  d\\d\' /\n\n\n# 2.2.4 \n\n(parameter) """" (parameter tuning)\n\n35/ 53=1255^3= 1255 3 =125 \n\n(validation set)\n\n\n# 2.3 \n\n(performance measure)\n\n d={(x1,y1),(x2,y2),...,(xm,ym))}d=\\lbrace (x_1,y_1),(x_2,y_2),...,(x_m,y_m))\\rbraced={(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m ))} yiy_iy i  xix_ix i  fff  f(x)f(x)f(x)  yyy \n\n(mean squared error, mse)\n\n\n\n\n# 2.3.1 \n\n+=1\n\n\n\n\n\n# 2.3.2 //f1\n\n/precision/recall\n\n(true positive)(false positive)(true negative)(false negative)tpfptnfn tp+fp+tn+fn=(confusion matrix)\n\n\n\n\n\n-p-rp-r\n\n\n\np-racp-rabp-r\n\nf-measuref-scoref-measurepr\n\n\n\n\n=1f1prf1\n\n\n\n\nprpmacro-prmacro-rff1tpfptnfnprff1\n\n\n\n\n# 2.3.3 roc  auc\n\nroc(receiver operating characteristic)p-rrocp-r, roc(true positive ratetpr)(false positiveratefpr)2.1\n\n\n\n\n\n roc roc(0,1)\n\n\n\n\n\np-r roc  roc  roc  auc(area under roc curve) \n\nauc  roc  roc  {(x1,y1),(x2,y2),...,(xm,ym)}\\lbrace(x_1, y_1),(x_2, y_2),...,(x_m,y_m)\\rbrace{(x 1 ,y 1 ),(x 2 ,y 2 ),...,(x m ,y m )}  (x1=0,xm=1)(x_1=0,x_m=1)(x 1 =0,x m =1) auc :\n\n\n\n\n\n\n# 2.3.4 \n\n--\x3e--\x3ecost matrix\n\n\n\n\n\n\n\n\n\n\n\n roc [0,1]p[0,1]\n\n\n\n\n\n\n roc (tprfpr)  fnr(0fpr) (1fnr)  roc \n\n\n\n\n\n\n# 2.4 \n\n(hypothesis test)abab \\epsilon \n\n\n# 2.4.1 \n\n\n# 2.4.2 t\n\n\n# 2.4.3 mcnemar\n\n\n# 2.4.4 friedmannemenyi\n\n\n# 2.5 \n\n-\n\n * =+\n * \n * \n\n-bias-variance dilamma\n\n\n\n',charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"(Introduction)",frontmatter:{title:"(Introduction)",date:"2023-04-21T19:15:41.000Z",permalink:"/pages/e2487d/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE/01.%E5%BC%95%E8%A8%80.html",relativePath:"05./12. - /01..md",key:"v-1e343bfb",path:"/pages/e2487d/",headers:[{level:2,title:"1.1 ",slug:"_1-1-",normalizedTitle:"1.1 ",charIndex:2},{level:2,title:"1.2 ",slug:"_1-2-",normalizedTitle:"1.2 ",charIndex:80},{level:2,title:"1.3 Supervised Learning",slug:"_1-3--supervised-learning",normalizedTitle:"1.3 supervised learning",charIndex:315},{level:2,title:"1.4 Unsupervised Learning",slug:"_1-4--unsupervised-learning",normalizedTitle:"1.4 unsupervised learning",charIndex:445}],headersStr:"1.1  1.2  1.3 Supervised Learning 1.4 Unsupervised Learning",content:"# 1.1 \n\n\n\n1-1.__bilibili\n\n\n\n\n# 1.2 \n\n\n\n1-2.__bilibili\n\n Arthur SamuelSamuel \n\n Tom Mitchell Tom  E  T P E  P  T \n\n\n# 1.3 Supervised Learning\n\n\n\n1-3.__bilibili\n\n\n\n\n\n\n\n\n# 1.4 Unsupervised Learning\n\n\n\n1-4.__bilibili\n\n\n\n\n\n",normalizedContent:"# 1.1 \n\n\n\n1-1.__bilibili\n\n\n\n\n# 1.2 \n\n\n\n1-2.__bilibili\n\n arthur samuelsamuel \n\n tom mitchell tom  e  t p e  p  t \n\n\n# 1.3 supervised learning\n\n\n\n1-3.__bilibili\n\n\n\n\n\n\n\n\n# 1.4 unsupervised learning\n\n\n\n1-4.__bilibili\n\n\n\n\n\n",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"(Linear Algebra Review)",frontmatter:{title:"(Linear Algebra Review)",date:"2023-05-05T21:18:57.000Z",permalink:"/pages/0952ad/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE/03.%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B0%E5%9B%9E%E9%A1%BE.html",relativePath:"05./12. - /03..md",key:"v-5bf683a0",path:"/pages/0952ad/",headers:[{level:2,title:"3.1 Matrices and vectors",slug:"_3-1--matrices-and-vectors",normalizedTitle:"3.1 matrices and vectors",charIndex:2},{level:2,title:"3.2 Additon and scalar multiplication",slug:"_3-2--additon-and-scalar-multiplication",normalizedTitle:"3.2 additon and scalar multiplication",charIndex:808},{level:2,title:"3.3 Matrix-vector multiplication",slug:"_3-3--matrix-vector-multiplication",normalizedTitle:"3.3 matrix-vector multiplication",charIndex:1547},{level:2,title:"3.4 Matrix-matrix multiplication",slug:"_3-4--matrix-matrix-multiplication",normalizedTitle:"3.4 matrix-matrix multiplication",charIndex:1713},{level:2,title:"3.5 Matrix multiplication properties",slug:"_3-5--matrix-multiplication-properties",normalizedTitle:"3.5 matrix multiplication properties",charIndex:1891},{level:2,title:"3.6 ",slug:"_3-6-",normalizedTitle:"3.6 ",charIndex:2247}],headersStr:"3.1 Matrices and vectors 3.2 Additon and scalar multiplication 3.3 Matrix-vector multiplication 3.4 Matrix-matrix multiplication 3.5 Matrix multiplication properties 3.6 ",content:"# 3.1 Matrices and vectors\n\n\n\n3-1.__bilibili\n\n 42  4  2  mmm nnn  mn  42\n\n\n\n \n\nA=[1402191137182194914371471448]A=\\left[\\begin{matrix} 1402 & 191 \\\\ 1371 & 821 \\\\ 949 & 1437 \\\\ 147 & 1448 \\end{matrix} \\right]A=     1402 1371 949 147 191 821 1437 1448    \n\nAijA_{ij}A ij  iii  jjj \n\ny=[460232315178]y=\\left[\\begin{matrix} {460} \\\\ {232} \\\\ {315} \\\\ {178} \\end{matrix} \\right]y=     460 232 315 178    \n\n41\n\n 1  0  1 \n\ny=[y1y2y3y4]y=\\left[\\begin{matrix} y_1 \\\\ y_2 \\\\ y_3 \\\\ y_4 \\end{matrix} \\right]y=     y 1 y 2 y 3 y 4     y=[y0y1y2y3]y=\\left[\\begin{matrix} y_0 \\\\ y_1 \\\\ y_2 \\\\ y_3 \\end{matrix} \\right]y=     y 0 y 1 y 2 y 3    \n\n\n# 3.2 Additon and scalar multiplication\n\n\n\n3-2.__bilibili\n\n\n\n[102531]\\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right]   1 2 3 0 5 1   + [40.52501]\\left[ \\begin{matrix} 4 & 0.5 \\\\ 2 & 5 \\\\ 0 & 1 \\end{matrix} \\right]   4 2 0 0.5 5 1   = [50.541032]\\left[ \\begin{matrix} 5 & 0.5 \\\\ 4 & 10 \\\\ 3 & 2 \\end{matrix} \\right]   5 4 3 0.5 10 2  \n\n\n\n3[102531]3 \\times \\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right]3   1 2 3 0 5 1   = [3061593]\\left[ \\begin{matrix} 3 & 0 \\\\ 6 & 15 \\\\ 9 & 3 \\end{matrix} \\right]   3 6 9 0 15 3   = [102531]3\\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right] \\times 3   1 2 3 0 5 1   3\n\n\n# 3.3 Matrix-vector multiplication\n\n\n\n3-3.__bilibili\n\nmnm \\times nmn  n1n \\times 1n1  m1m \\times 1m1 \n\n\n\n\n\n\n# 3.4 Matrix-matrix multiplication\n\n\n\n3-4.__bilibili\n\nmnm \\times nmn  non \\times ono  mom \\times omo \n\n AAA  BBB\n\n\n\n\n# 3.5 Matrix multiplication properties\n\n\n\n3-5.__bilibili\n\n\n\nABBA\n\nA(BC)=(AB)C\n\n 111 III  EEE  III  111 0\n\nAA1=A1A=IA{A^{-1}}={A^{-1}}A=IAA 1 =A 1 A=I\n\nAI=IA=AAI=IA=AAI=IA=A\n\n\n# 3.6 \n\n\n\n3-6.__bilibili\n\n AAA  mm AA1=A1A=IA{A}^{-1}={A}^{-1}A=IAA 1 =A 1 A=I\n\nOCTAVEMATLAB\n\n AAA  mn  mmm  nnn  iii  jjj  a(i,j)a(i,j)a(i,j)A=a(i,j)A=a(i,j)A=a(i,j)\n\n AAA  nm  BBB B=a(j,i)B=a(j,i)B=a(j,i) b(i,j)=a(j,i)b (i,j)=a(j,i)b(i,j)=a(j,i)BBB  iii  jjj  AAA  jjj  iii  AT=B{A}^{T}=BA T =B( A=BA'=BA  =B\n\n AAA  1  1  45  AAA \n\n\n\nabcdefT=acebdf{\\left| \\begin{matrix} a& b \\\\ c& d \\\\ e& f \\end{matrix} \\right|}^{T}=\\left|\\begin{matrix} a& c & e \\\\ b& d & f \\end{matrix} \\right|       a c e b d f       T =     a b c d e f    ",normalizedContent:"# 3.1 matrices and vectors\n\n\n\n3-1.__bilibili\n\n 42  4  2  mmm nnn  mn  42\n\n\n\n \n\na=[1402191137182194914371471448]a=\\left[\\begin{matrix} 1402 & 191 \\\\ 1371 & 821 \\\\ 949 & 1437 \\\\ 147 & 1448 \\end{matrix} \\right]a=     1402 1371 949 147 191 821 1437 1448    \n\naija_{ij}a ij  iii  jjj \n\ny=[460232315178]y=\\left[\\begin{matrix} {460} \\\\ {232} \\\\ {315} \\\\ {178} \\end{matrix} \\right]y=     460 232 315 178    \n\n41\n\n 1  0  1 \n\ny=[y1y2y3y4]y=\\left[\\begin{matrix} y_1 \\\\ y_2 \\\\ y_3 \\\\ y_4 \\end{matrix} \\right]y=     y 1 y 2 y 3 y 4     y=[y0y1y2y3]y=\\left[\\begin{matrix} y_0 \\\\ y_1 \\\\ y_2 \\\\ y_3 \\end{matrix} \\right]y=     y 0 y 1 y 2 y 3    \n\n\n# 3.2 additon and scalar multiplication\n\n\n\n3-2.__bilibili\n\n\n\n[102531]\\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right]   1 2 3 0 5 1   + [40.52501]\\left[ \\begin{matrix} 4 & 0.5 \\\\ 2 & 5 \\\\ 0 & 1 \\end{matrix} \\right]   4 2 0 0.5 5 1   = [50.541032]\\left[ \\begin{matrix} 5 & 0.5 \\\\ 4 & 10 \\\\ 3 & 2 \\end{matrix} \\right]   5 4 3 0.5 10 2  \n\n\n\n3[102531]3 \\times \\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right]3   1 2 3 0 5 1   = [3061593]\\left[ \\begin{matrix} 3 & 0 \\\\ 6 & 15 \\\\ 9 & 3 \\end{matrix} \\right]   3 6 9 0 15 3   = [102531]3\\left[ \\begin{matrix} 1 & 0 \\\\ 2 & 5 \\\\ 3 & 1 \\end{matrix} \\right] \\times 3   1 2 3 0 5 1   3\n\n\n# 3.3 matrix-vector multiplication\n\n\n\n3-3.__bilibili\n\nmnm \\times nmn  n1n \\times 1n1  m1m \\times 1m1 \n\n\n\n\n\n\n# 3.4 matrix-matrix multiplication\n\n\n\n3-4.__bilibili\n\nmnm \\times nmn  non \\times ono  mom \\times omo \n\n aaa  bbb\n\n\n\n\n# 3.5 matrix multiplication properties\n\n\n\n3-5.__bilibili\n\n\n\nab=ba\n\na(bc)=(ab)c\n\n 111 iii  eee  iii  111 0\n\naa1=a1a=ia{a^{-1}}={a^{-1}}a=iaa 1 =a 1 a=i\n\nai=ia=aai=ia=aai=ia=a\n\n\n# 3.6 \n\n\n\n3-6.__bilibili\n\n aaa  mm aa1=a1a=ia{a}^{-1}={a}^{-1}a=iaa 1 =a 1 a=i\n\noctavematlab\n\n aaa  mn  mmm  nnn  iii  jjj  a(i,j)a(i,j)a(i,j)a=a(i,j)a=a(i,j)a=a(i,j)\n\n aaa  nm  bbb b=a(j,i)b=a(j,i)b=a(j,i) b(i,j)=a(j,i)b (i,j)=a(j,i)b(i,j)=a(j,i)bbb  iii  jjj  aaa  jjj  iii  at=b{a}^{t}=ba t =b( a=ba'=ba  =b\n\n aaa  1  1  45  aaa \n\n\n\nabcdeft=acebdf{\\left| \\begin{matrix} a& b \\\\ c& d \\\\ e& f \\end{matrix} \\right|}^{t}=\\left|\\begin{matrix} a& c & e \\\\ b& d & f \\end{matrix} \\right|       a c e b d f       t =     a b c d e f    ",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"(Linear Regression with One Variable)",frontmatter:{title:"(Linear Regression with One Variable)",date:"2023-04-21T19:16:35.000Z",permalink:"/pages/64abf4/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE/02.%E5%8D%95%E5%8F%98%E9%87%8F%E7%BA%BF%E6%80%A7%E5%9B%9E%E5%BD%92.html",relativePath:"05./12. - /02..md",key:"v-220be6fa",path:"/pages/64abf4/",headers:[{level:2,title:"2.1 Model Representation",slug:"_2-1--model-representation",normalizedTitle:"2.1 model representation",charIndex:2},{level:2,title:"2.2 Cost Function",slug:"_2-2--cost-function",normalizedTitle:"2.2 cost function",charIndex:541},{level:2,title:"2.3 Cost function intuition ",slug:"_2-3-i-cost-function-intuition-i",normalizedTitle:"2.3 cost function intuition ",charIndex:1037},{level:2,title:"2.4 Cost function intuition ",slug:"_2-4-ii-cost-function-intuition-ii",normalizedTitle:"2.4 cost function intuition ",charIndex:1243},{level:2,title:"2.5 Gradient descent",slug:"_2-5--gradient-descent",normalizedTitle:"2.5 gradient descent",charIndex:1473},{level:2,title:"2.6 Gradient descent intuition",slug:"_2-6--gradient-descent-intuition",normalizedTitle:"2.6 gradient descent intuition",charIndex:2111},{level:2,title:"2.7 ",slug:"_2-7-",normalizedTitle:"2.7 ",charIndex:3210}],headersStr:"2.1 Model Representation 2.2 Cost Function 2.3 Cost function intuition  2.4 Cost function intuition  2.5 Gradient descent 2.6 Gradient descent intuition 2.7 ",content:'# 2.1 Model Representation\n\n\n\n2-1.__bilibili\n\n\n\n\n\nTraining Set\n\n\n\n:\n\n * mmm \n * xxx /\n * yyy /\n * (x,y)(x,y)(x,y) \n * (x(i),y(i))(x^{(i)},y^{(i)})(x (i) ,y (i) )  iii \n * hhh hypothesis\n\n\n\n h hypothesis()h  x  y \n\n h h(x)=0+1xh_{\\theta}(x) = \\theta_0 + \\theta_1xh  (x)= 0 + 1 x\n\n/\n\n\n# 2.2 Cost Function\n\n\n\n2-2.__bilibili\n\nh(x)=0+1xh_{\\theta}(x) = \\theta_0 + \\theta_1xh  (x)= 0 + 1 x\n\nparameters0\\theta_0 0  1\\theta_1 1  y \n\nmodeling error\n\n\n\n J(0,1)=12mi=1m(h(x(i))y(i))2J(\\theta_0,\\theta_1) = \\frac{1}{2m}\\sum_{i=1}^m(h_{\\theta}(x^{(i)})-y^{(i)})^2J( 0 , 1 )= 2m 1  i=1 m (h  (x (i) )y (i) ) 2 \n\n\n\n\n# 2.3 Cost function intuition \n\n\n\n2-3._bilibili\n\n 0=0\\theta_0=0 0 =0  1\\theta_1 1  J(1)J(\\theta_1)J( 1 ) \n\n\n\n J(1)J(\\theta_1)J( 1 ) \n\n\n# 2.4 Cost function intuition \n\n\n\n2-4._bilibili\n\n 0\\theta_0 0 ,1\\theta_1 1  J(0,1)J(\\theta_0,\\theta_1)J( 0 , 1 ) \n\n\n\n J(0,1)J(\\theta_0,\\theta_1)J( 0 , 1 ) \n\n\n# 2.5 Gradient descent\n\n\n\n2-5.__bilibili\n\n J(0,1)J(\\theta_{0}, \\theta_{1})J( 0 , 1 ) \n\n (0,1,......,n)\\left( {\\theta_{0}},{\\theta_{1}},......,{\\theta_{n}} \\right)( 0 , 1 ,......, n ) local minimumglobal minimum\n\n\n\nbatch gradient descent\n\n\n\n aaa learning rate\n\n 0{\\theta_{0}} 0  1{\\theta_{1}} 1 \n\n\n# 2.6 Gradient descent intuition\n\n\n\n2-6.__bilibili\n\nj:=jjJ(){\\theta_{j}}:={\\theta_{j}}-\\alpha \\frac{\\partial }{\\partial {\\theta_{j}}}J\\left(\\theta \\right) j := j   j  J()\n\n J()J\\left(\\theta \\right)J()  \\theta \n\n\n\n \\theta  J()J\\left(\\theta \\right)J()  aaa learning rate\n\n 1{\\theta_{1}} 1  1{\\theta_{1}} 1  J()J\\left(\\theta \\right)J() \n\n aaa  aaa \n\n\n\n aaa  aaa \n\n 1{\\theta_{1}} 1  0 1{\\theta_{1}} 1 \n\n\n\n\n\n J()J\\left(\\theta \\right)J()  1{\\theta_{1}} 1 \n\n\n# 2.7 \n\n\n\n2-7.__bilibili\n\n\n\n\n\n\n\njJ(0,1)=j12mi=1m(h(x(i))y(i))2\\frac{\\partial}{\\partial \\theta_j}J(\\theta_0,\\theta_1) = \\frac{\\partial}{\\partial \\theta_j} \\frac{1}{2m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)}) - y^{(i)})^2  j  J( 0 , 1 )=  j  2m 1 i=1  m (h  (x (i) )y (i) ) 2\n\nj=0j=0j=0  0J(0,1)=1mi=1m(h(x(i))y(i))\\frac{\\partial}{\\partial \\theta_0}J(\\theta_0,\\theta_1) = \\frac{1}{m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)}) - y^{(i)})  0  J( 0 , 1 )= m 1 i=1  m (h  (x (i) )y (i) )\n\nj=1j=1j=1  1J(0,1)=1mi=1m((h(x(i))y(i))x(i))\\frac{\\partial}{\\partial \\theta_1}J(\\theta_0,\\theta_1) = \\frac{1}{m} \\sum\\limits_{i=1}^{m} ((h_{\\theta}(x^{(i)}) - y^{(i)}) \\cdot x^{(i)})  1  J( 0 , 1 )= m 1 i=1  m ((h  (x (i) )y (i) )x (i) )\n\n\n\nRepeat {\n\n0:=0a1mi=1m(h(x(i))y(i))\\theta_0 := \\theta_0 - a \\frac{1}{m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)})-y^{(i)}) 0 := 0 a m 1 i=1  m (h  (x (i) )y (i) )\n\n1:=1a1mi=1m((h(x(i))y(i))x(i))\\theta_1 := \\theta_1 - a \\frac{1}{m} \\sum\\limits_{i=1}^{m} ((h_{\\theta}(x^{(i)})-y^{(i)}) \\cdot x^{(i)}) 1 := 1 a m 1 i=1  m ((h  (x (i) )y (i) )x (i) )\n\n}\n\n""""',normalizedContent:'# 2.1 model representation\n\n\n\n2-1.__bilibili\n\n\n\n\n\ntraining set\n\n\n\n:\n\n * mmm \n * xxx /\n * yyy /\n * (x,y)(x,y)(x,y) \n * (x(i),y(i))(x^{(i)},y^{(i)})(x (i) ,y (i) )  iii \n * hhh hypothesis\n\n\n\n h hypothesis()h  x  y \n\n h h(x)=0+1xh_{\\theta}(x) = \\theta_0 + \\theta_1xh  (x)= 0 + 1 x\n\n/\n\n\n# 2.2 cost function\n\n\n\n2-2.__bilibili\n\nh(x)=0+1xh_{\\theta}(x) = \\theta_0 + \\theta_1xh  (x)= 0 + 1 x\n\nparameters0\\theta_0 0  1\\theta_1 1  y \n\nmodeling error\n\n\n\n j(0,1)=12mi=1m(h(x(i))y(i))2j(\\theta_0,\\theta_1) = \\frac{1}{2m}\\sum_{i=1}^m(h_{\\theta}(x^{(i)})-y^{(i)})^2j( 0 , 1 )= 2m 1  i=1 m (h  (x (i) )y (i) ) 2 \n\n\n\n\n# 2.3 cost function intuition \n\n\n\n2-3._bilibili\n\n 0=0\\theta_0=0 0 =0  1\\theta_1 1  j(1)j(\\theta_1)j( 1 ) \n\n\n\n j(1)j(\\theta_1)j( 1 ) \n\n\n# 2.4 cost function intuition \n\n\n\n2-4._bilibili\n\n 0\\theta_0 0 ,1\\theta_1 1  j(0,1)j(\\theta_0,\\theta_1)j( 0 , 1 ) \n\n\n\n j(0,1)j(\\theta_0,\\theta_1)j( 0 , 1 ) \n\n\n# 2.5 gradient descent\n\n\n\n2-5.__bilibili\n\n j(0,1)j(\\theta_{0}, \\theta_{1})j( 0 , 1 ) \n\n (0,1,......,n)\\left( {\\theta_{0}},{\\theta_{1}},......,{\\theta_{n}} \\right)( 0 , 1 ,......, n ) local minimumglobal minimum\n\n\n\nbatch gradient descent\n\n\n\n aaa learning rate\n\n 0{\\theta_{0}} 0  1{\\theta_{1}} 1 \n\n\n# 2.6 gradient descent intuition\n\n\n\n2-6.__bilibili\n\nj:=jjj(){\\theta_{j}}:={\\theta_{j}}-\\alpha \\frac{\\partial }{\\partial {\\theta_{j}}}j\\left(\\theta \\right) j := j   j  j()\n\n j()j\\left(\\theta \\right)j()  \\theta \n\n\n\n \\theta  j()j\\left(\\theta \\right)j()  aaa learning rate\n\n 1{\\theta_{1}} 1  1{\\theta_{1}} 1  j()j\\left(\\theta \\right)j() \n\n aaa  aaa \n\n\n\n aaa  aaa \n\n 1{\\theta_{1}} 1  0 1{\\theta_{1}} 1 \n\n\n\n\n\n j()j\\left(\\theta \\right)j()  1{\\theta_{1}} 1 \n\n\n# 2.7 \n\n\n\n2-7.__bilibili\n\n\n\n\n\n\n\njj(0,1)=j12mi=1m(h(x(i))y(i))2\\frac{\\partial}{\\partial \\theta_j}j(\\theta_0,\\theta_1) = \\frac{\\partial}{\\partial \\theta_j} \\frac{1}{2m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)}) - y^{(i)})^2  j  j( 0 , 1 )=  j  2m 1 i=1  m (h  (x (i) )y (i) ) 2\n\nj=0j=0j=0  0j(0,1)=1mi=1m(h(x(i))y(i))\\frac{\\partial}{\\partial \\theta_0}j(\\theta_0,\\theta_1) = \\frac{1}{m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)}) - y^{(i)})  0  j( 0 , 1 )= m 1 i=1  m (h  (x (i) )y (i) )\n\nj=1j=1j=1  1j(0,1)=1mi=1m((h(x(i))y(i))x(i))\\frac{\\partial}{\\partial \\theta_1}j(\\theta_0,\\theta_1) = \\frac{1}{m} \\sum\\limits_{i=1}^{m} ((h_{\\theta}(x^{(i)}) - y^{(i)}) \\cdot x^{(i)})  1  j( 0 , 1 )= m 1 i=1  m ((h  (x (i) )y (i) )x (i) )\n\n\n\nrepeat {\n\n0:=0a1mi=1m(h(x(i))y(i))\\theta_0 := \\theta_0 - a \\frac{1}{m} \\sum\\limits_{i=1}^{m} (h_{\\theta}(x^{(i)})-y^{(i)}) 0 := 0 a m 1 i=1  m (h  (x (i) )y (i) )\n\n1:=1a1mi=1m((h(x(i))y(i))x(i))\\theta_1 := \\theta_1 - a \\frac{1}{m} \\sum\\limits_{i=1}^{m} ((h_{\\theta}(x^{(i)})-y^{(i)}) \\cdot x^{(i)}) 1 := 1 a m 1 i=1  m ((h  (x (i) )y (i) )x (i) )\n\n}\n\n""""',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"(Linear Regression with Multiple Variables)",frontmatter:{title:"(Linear Regression with Multiple Variables)",date:"2023-05-07T20:52:02.000Z",permalink:"/pages/95b5f9/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE/04.%E5%A4%9A%E5%8F%98%E9%87%8F%E7%BA%BF%E6%80%A7%E5%9B%9E%E5%BD%92.html",relativePath:"05./12. - /04..md",key:"v-0c0e50cc",path:"/pages/95b5f9/",headers:[{level:2,title:"4.1 Multiple features",slug:"_4-1--multiple-features",normalizedTitle:"4.1 multiple features",charIndex:2},{level:2,title:"4.2 Gradient descent for multiple variables",slug:"_4-2--gradient-descent-for-multiple-variables",normalizedTitle:"4.2 gradient descent for multiple variables",charIndex:1278},{level:2,title:"4.3 1-Gradient descent in practice I: Feature Scaling",slug:"_4-3-1--gradient-descent-in-practice-i-feature-scaling",normalizedTitle:"4.3 1-gradient descent in practice i: feature scaling",charIndex:2137},{level:2,title:"4.4 2-Gradient descent in practice II: Learning rate",slug:"_4-4-2--gradient-descent-in-practice-ii-learning-rate",normalizedTitle:"4.4 2-gradient descent in practice ii: learning rate",charIndex:2574},{level:2,title:"4.5 Features and polynomial regression",slug:"_4-5--features-and-polynomial-regression",normalizedTitle:"4.5 features and polynomial regression",charIndex:3012},{level:2,title:"4.6 Normal equation",slug:"_4-6--normal-equation",normalizedTitle:"4.6 normal equation",charIndex:4340},{level:2,title:"4.7 Normal equation and non-invertibility",slug:"_4-7--normal-equation-and-non-invertibility",normalizedTitle:"4.7 normal equation and non-invertibility",charIndex:5664}],headersStr:"4.1 Multiple features 4.2 Gradient descent for multiple variables 4.3 1-Gradient descent in practice I: Feature Scaling 4.4 2-Gradient descent in practice II: Learning rate 4.5 Features and polynomial regression 4.6 Normal equation 4.7 Normal equation and non-invertibility",content:"# 4.1 Multiple features\n\n\n\n4-1.__bilibili\n\n/ (x1,x2,...,xn)\\left( {x_{1}},{x_{2}},...,{x_{n}} \\right)(x 1 ,x 2 ,...,x n )\n\n\n\n\n\nnnn \n\nx(i){x^{\\left( i \\right)}}x (i)  iii  iii vector x(2)=[14163240]{x}^{(2)}\\text{=}\\begin{bmatrix} 1416\\\\\\ 3\\\\\\ 2\\\\\\ 40 \\end{bmatrix}x (2) =     1416 3 2 40    \n\nxj(i){x}_{j}^{\\left( i \\right)}x j (i)  iii  jjj  iii  jjj  x2(2)=3,x3(2)=2x_{2}^{\\left( 2 \\right)}=3, x_{3}^{\\left( 2 \\right)}=2x 2 (2) =3,x 3 (2) =2\n\n hhh h(x)=0+1x1+2x2+...+nxnh_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= 0 + 1 x 1 + 2 x 2 +...+ n x n \n\n n+1n+1n+1  nnn  x0=1x_{0}=1x 0 =1h(x)=0x0+1x1+2x2+...+nxnh_{\\theta} \\left( x \\right)={\\theta_{0}}{x_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= 0 x 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n n+1n+1n+1  n+1n+1n+1  XXX  m(n+1)m*(n+1)m(n+1) h(x)=TXh_{\\theta} \\left( x \\right)={\\theta^{T}}Xh  (x)= T X TTT \n\n\n\n\n# 4.2 Gradient descent for multiple variables\n\n\n\n4-2.__bilibili\n\n\n\n hhh h(x)=TX=0+1x1+2x2+...+nxnh_{\\theta}\\left( x \\right)={\\theta^{T}}X={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= T X= 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n=0,1,...,n{\\theta}={\\theta_{0},\\theta_{1},...,\\theta_{n}}= 0 , 1 ,..., n\n\nJ(0,1...n)=12mi=1m(h(x(i))y(i))2J\\left( {\\theta_{0}},{\\theta_{1}}...{\\theta_{n}} \\right)=\\frac{1}{2m}\\sum\\limits_{i=1}^{m}{\\left( h_{\\theta} \\left({x}^{\\left( i \\right)} \\right)-{y}^{\\left( i \\right)} \\right)}^{2}J( 0 , 1 ... n )= 2m 1 i=1  m (h  (x (i) )y (i) ) 2\n\n\n\nRepeat {\n\nj:=jjJ(0,...,n)\\theta_j := \\theta_j - \\frac{\\partial}{\\partial \\theta_j}J(\\theta_0,...,\\theta_n) j := j   j  J( 0 ,..., n )\n\n}\n\n/\n\n\n\n\n# 4.3 1-Gradient descent in practice I: Feature Scaling\n\n\n\n4-3..I..__bilibili\n\n\n\n 0-2000  0-5\n\n\n\n -1  1 \n\n\n\n Z-Scorexn=xnnsnx_n=\\frac{x_n-\\mu_n}{s_n}x n = s n x n  n  n{\\mu_{n}} n sn{s_{n}}s n \n\n\n# 4.4 2-Gradient descent in practice II: Learning rate\n\n\n\n4-4.II..__bilibili\n\n\n\n\n\n0.001\n\n aaa  aaa \n\n\n\n=0.01,0.03,0.1,0.3,1,3,10\\alpha=0.01,0.03,0.1,0.3,1,3,10=0.01,0.03,0.1,0.3,1,3,10\n\n\n# 4.5 Features and polynomial regression\n\n\n\n4-5.__bilibili\n\n\n\n\n\nh(x)=0+1frontage+2depthh_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}\\times{frontage}+{\\theta_{2}}\\times{depth}h  (x)= 0 + 1 frontage+ 2 depth\n\nx1=frontage{x_{1}}=frontagex 1 =frontagex2=depth{x_{2}}=depthx 2 =depthx=frontagedepth=areax=frontage*depth=areax=frontagedepth=areah(x)=0+1x{h_{\\theta}}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}xh  (x)= 0 + 1 x\n\nh(x)=0+1x1+2x22h_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}^2}h  (x)= 0 + 1 x 1 + 2 x 2 2  h(x)=0+1x1+2x22+3x33h_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}^2}+{\\theta_{3}}{x_{3}^3}h  (x)= 0 + 1 x 1 + 2 x 2 2 + 3 x 3 3\n\n\n\n\n\nx2=x22,x3=x33{x}_{2}=x_{2}^{2},{x}_{3}=x_{3}^{3}x 2 =x 2 2 ,x 3 =x 3 3 \n\n\n\nh(x)=0+1(size)+2(size)2{h}_{\\theta}(x)={\\theta}_{0}\\text{+}{\\theta}_{1}(size)+{\\theta}_{2}{(size)}^{2}h  (x)= 0 + 1 (size)+ 2 (size) 2\n\n:\n\nh(x)=0+1(size)+2size{h}_{\\theta}(x)={\\theta}_{0}\\text{+}{\\theta}_{1}(size)+{\\theta}_{2}\\sqrt{size}h  (x)= 0 + 1 (size)+ 2  size\n\n\n\n\n# 4.6 Normal equation\n\n\n\n4-6._bilibili\n\n\n\n\n\njJ(j)=0\\frac{\\partial}{\\partial{\\theta_{j}}}J\\left( {\\theta_{j}} \\right)=0  j  J( j )=0 \n\n XXX x0=1{x}_{0}=1x 0 =1 yyy =(XTX)1XTy\\theta ={\\left({X^T}X \\right)}^{-1}{X^{T}}y=(X T X) 1 X T y  T  -1  A=XTXA=X^{T}XA=X T X(XTX)1=A1{\\left({X^T}X \\right)}^{-1}=A^{-1}(X T X) 1 =A 1\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n                 \n\\alpha      \n               \n nnn     (XTX)1\\left({X}^{T}X \\right)^{-1}(X T X) 1  n\n                      O(n3)O\\left({n}^{3} \\right)O(n 3\n                     ) nnn  10000 \n           \n\n \\theta \n\n\n\n\n# 4.7 Normal equation and non-invertibility\n\n\n\n4-7.__bilibili",normalizedContent:"# 4.1 multiple features\n\n\n\n4-1.__bilibili\n\n/ (x1,x2,...,xn)\\left( {x_{1}},{x_{2}},...,{x_{n}} \\right)(x 1 ,x 2 ,...,x n )\n\n\n\n\n\nnnn \n\nx(i){x^{\\left( i \\right)}}x (i)  iii  iii vector x(2)=[14163240]{x}^{(2)}\\text{=}\\begin{bmatrix} 1416\\\\\\ 3\\\\\\ 2\\\\\\ 40 \\end{bmatrix}x (2) =     1416 3 2 40    \n\nxj(i){x}_{j}^{\\left( i \\right)}x j (i)  iii  jjj  iii  jjj  x2(2)=3,x3(2)=2x_{2}^{\\left( 2 \\right)}=3, x_{3}^{\\left( 2 \\right)}=2x 2 (2) =3,x 3 (2) =2\n\n hhh h(x)=0+1x1+2x2+...+nxnh_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= 0 + 1 x 1 + 2 x 2 +...+ n x n \n\n n+1n+1n+1  nnn  x0=1x_{0}=1x 0 =1h(x)=0x0+1x1+2x2+...+nxnh_{\\theta} \\left( x \\right)={\\theta_{0}}{x_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= 0 x 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n n+1n+1n+1  n+1n+1n+1  xxx  m(n+1)m*(n+1)m(n+1) h(x)=txh_{\\theta} \\left( x \\right)={\\theta^{t}}xh  (x)= t x ttt \n\n\n\n\n# 4.2 gradient descent for multiple variables\n\n\n\n4-2.__bilibili\n\n\n\n hhh h(x)=tx=0+1x1+2x2+...+nxnh_{\\theta}\\left( x \\right)={\\theta^{t}}x={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= t x= 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n=0,1,...,n{\\theta}={\\theta_{0},\\theta_{1},...,\\theta_{n}}= 0 , 1 ,..., n\n\nj(0,1...n)=12mi=1m(h(x(i))y(i))2j\\left( {\\theta_{0}},{\\theta_{1}}...{\\theta_{n}} \\right)=\\frac{1}{2m}\\sum\\limits_{i=1}^{m}{\\left( h_{\\theta} \\left({x}^{\\left( i \\right)} \\right)-{y}^{\\left( i \\right)} \\right)}^{2}j( 0 , 1 ... n )= 2m 1 i=1  m (h  (x (i) )y (i) ) 2\n\n\n\nrepeat {\n\nj:=jjj(0,...,n)\\theta_j := \\theta_j - \\frac{\\partial}{\\partial \\theta_j}j(\\theta_0,...,\\theta_n) j := j   j  j( 0 ,..., n )\n\n}\n\n/\n\n\n\n\n# 4.3 1-gradient descent in practice i: feature scaling\n\n\n\n4-3..i..__bilibili\n\n\n\n 0-2000  0-5\n\n\n\n -1  1 \n\n\n\n z-scorexn=xnnsnx_n=\\frac{x_n-\\mu_n}{s_n}x n = s n x n  n  n{\\mu_{n}} n sn{s_{n}}s n \n\n\n# 4.4 2-gradient descent in practice ii: learning rate\n\n\n\n4-4.ii..__bilibili\n\n\n\n\n\n0.001\n\n aaa  aaa \n\n\n\n=0.01,0.03,0.1,0.3,1,3,10\\alpha=0.01,0.03,0.1,0.3,1,3,10=0.01,0.03,0.1,0.3,1,3,10\n\n\n# 4.5 features and polynomial regression\n\n\n\n4-5.__bilibili\n\n\n\n\n\nh(x)=0+1frontage+2depthh_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}\\times{frontage}+{\\theta_{2}}\\times{depth}h  (x)= 0 + 1 frontage+ 2 depth\n\nx1=frontage{x_{1}}=frontagex 1 =frontagex2=depth{x_{2}}=depthx 2 =depthx=frontagedepth=areax=frontage*depth=areax=frontagedepth=areah(x)=0+1x{h_{\\theta}}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}xh  (x)= 0 + 1 x\n\nh(x)=0+1x1+2x22h_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}^2}h  (x)= 0 + 1 x 1 + 2 x 2 2  h(x)=0+1x1+2x22+3x33h_{\\theta}\\left( x \\right)={\\theta_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}^2}+{\\theta_{3}}{x_{3}^3}h  (x)= 0 + 1 x 1 + 2 x 2 2 + 3 x 3 3\n\n\n\n\n\nx2=x22,x3=x33{x}_{2}=x_{2}^{2},{x}_{3}=x_{3}^{3}x 2 =x 2 2 ,x 3 =x 3 3 \n\n\n\nh(x)=0+1(size)+2(size)2{h}_{\\theta}(x)={\\theta}_{0}\\text{+}{\\theta}_{1}(size)+{\\theta}_{2}{(size)}^{2}h  (x)= 0 + 1 (size)+ 2 (size) 2\n\n:\n\nh(x)=0+1(size)+2size{h}_{\\theta}(x)={\\theta}_{0}\\text{+}{\\theta}_{1}(size)+{\\theta}_{2}\\sqrt{size}h  (x)= 0 + 1 (size)+ 2  size\n\n\n\n\n# 4.6 normal equation\n\n\n\n4-6._bilibili\n\n\n\n\n\njj(j)=0\\frac{\\partial}{\\partial{\\theta_{j}}}j\\left( {\\theta_{j}} \\right)=0  j  j( j )=0 \n\n xxx x0=1{x}_{0}=1x 0 =1 yyy =(xtx)1xty\\theta ={\\left({x^t}x \\right)}^{-1}{x^{t}}y=(x t x) 1 x t y  t  -1  a=xtxa=x^{t}xa=x t x(xtx)1=a1{\\left({x^t}x \\right)}^{-1}=a^{-1}(x t x) 1 =a 1\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n                 \n\\alpha      \n               \n nnn     (xtx)1\\left({x}^{t}x \\right)^{-1}(x t x) 1  n\n                      o(n3)o\\left({n}^{3} \\right)o(n 3\n                     ) nnn  10000 \n           \n\n \\theta \n\n\n\n\n# 4.7 normal equation and non-invertibility\n\n\n\n4-7.__bilibili",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"ggplot2",frontmatter:{title:"ggplot2",date:"2023-07-03T21:14:18.000Z",permalink:"/pages/98868d/",categories:["R","tidyverse"],tags:["tidyverse"],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/12.R%20-%20tidyverse/02.ggplot2.html",relativePath:"06./12.R - tidyverse/02.ggplot2.md",key:"v-3ee28eee",path:"/pages/98868d/",headers:[{level:2,title:"Overview",slug:"overview",normalizedTitle:"overview",charIndex:86},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:212},{level:2,title:"Cheatsheet",slug:"cheatsheet",normalizedTitle:"cheatsheet",charIndex:516},{level:2,title:"Usage",slug:"usage",normalizedTitle:"usage",charIndex:533},{level:2,title:"Lifecycle",slug:"lifecycle",normalizedTitle:"lifecycle",charIndex:876},{level:2,title:"Learning ggplot2",slug:"learning-ggplot2",normalizedTitle:"learning ggplot2",charIndex:1118},{level:2,title:"Getting help",slug:"getting-help",normalizedTitle:"getting help",charIndex:1730},{level:2,title:"Articles",slug:"articles",normalizedTitle:"articles",charIndex:1904},{level:3,title:"Aesthetic specifications",slug:"aesthetic-specifications",normalizedTitle:"aesthetic specifications",charIndex:1917}],excerpt:'<p><a href="https://ggplot2.tidyverse.org"><img src="https://github.com/tidyverse/ggplot2/raw/main/man/figures/logo.png" align="right" height="138" /></a><br>\nGitHub: <a href="https://github.com/tidyverse/ggplot2" target="_blank" rel="noopener noreferrer">https://github.com/tidyverse/ggplot2<OutboundLink/></a><br>\nDoc: <a href="https://ggplot2.tidyverse.org/" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org/<OutboundLink/></a></p>\n',headersStr:"Overview Installation Cheatsheet Usage Lifecycle Learning ggplot2 Getting help Articles Aesthetic specifications",content:'\nGitHub: https://github.com/tidyverse/ggplot2\nDoc: https://ggplot2.tidyverse.org/\n\n\n# Overview\n\nggplot2  The Grammar of Graphics ggplot2 aesthetics\n\n\n# Installation\n\n# The easiest way to get ggplot2 is to install the whole tidyverse:\ninstall.packages("tidyverse")\n\n# Alternatively, install just ggplot2:\ninstall.packages("ggplot2")\n\n# Or the development version from GitHub:\n# install.packages("pak")\npak::pak("tidyverse/ggplot2")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# Cheatsheet\n\n\n\n\n# Usage\n\n ggplot2  ggplot()  aes() layers geom_point()  geom_histogram()scales scale_colour_brewer()faceting specifications facet_wrap()coordinate systems coord_flip()\n\nlibrary(ggplot2)\n\nggplot(mpg, aes(displ, hwy, colour = class)) + \n  geom_point()\n\n\n1\n2\n3\n4\n\n\n\n# Lifecycle\n\nggplot2  10 ggplot2 \n\n ggplot2 https://exts.ggplot2.tidyverse.org/gallery/\n\n\n# Learning ggplot2\n\n ggplot2 \n\n 1. R for Data Science  Data Visualisation  Graphics for communication R for Data Science  tidyverse  ggplot2 \n\n 2.  Kara Woo  Data Visualization in R With ggplot2\n\n 3.  Thomas Lin Pedersen  Plotting Anything with ggplot2\n\n 4.  Winston Chang  The R Graphics Cookbook\n\n ggplot2:Elegant Graphics for Data Analysis ggplot2  ggplot2 \n\n\n# Getting help\n\n ggplot2 \n\n 1. RStudio community  ggplot2 \n\n 2. Stack Overflow  ggplot2 \n\n\n# Articles\n\n\n# Aesthetic specifications\n\nhttps://ggplot2.tidyverse.org/articles/ggplot2-specs.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R/tidyverse/02.ggplot2/ggplot2-specs.html',normalizedContent:'\ngithub: https://github.com/tidyverse/ggplot2\ndoc: https://ggplot2.tidyverse.org/\n\n\n# overview\n\nggplot2  the grammar of graphics ggplot2 aesthetics\n\n\n# installation\n\n# the easiest way to get ggplot2 is to install the whole tidyverse:\ninstall.packages("tidyverse")\n\n# alternatively, install just ggplot2:\ninstall.packages("ggplot2")\n\n# or the development version from github:\n# install.packages("pak")\npak::pak("tidyverse/ggplot2")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# cheatsheet\n\n\n\n\n# usage\n\n ggplot2  ggplot()  aes() layers geom_point()  geom_histogram()scales scale_colour_brewer()faceting specifications facet_wrap()coordinate systems coord_flip()\n\nlibrary(ggplot2)\n\nggplot(mpg, aes(displ, hwy, colour = class)) + \n  geom_point()\n\n\n1\n2\n3\n4\n\n\n\n# lifecycle\n\nggplot2  10 ggplot2 \n\n ggplot2 https://exts.ggplot2.tidyverse.org/gallery/\n\n\n# learning ggplot2\n\n ggplot2 \n\n 1. r for data science  data visualisation  graphics for communication r for data science  tidyverse  ggplot2 \n\n 2.  kara woo  data visualization in r with ggplot2\n\n 3.  thomas lin pedersen  plotting anything with ggplot2\n\n 4.  winston chang  the r graphics cookbook\n\n ggplot2:elegant graphics for data analysis ggplot2  ggplot2 \n\n\n# getting help\n\n ggplot2 \n\n 1. rstudio community  ggplot2 \n\n 2. stack overflow  ggplot2 \n\n\n# articles\n\n\n# aesthetic specifications\n\nhttps://ggplot2.tidyverse.org/articles/ggplot2-specs.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r/tidyverse/02.ggplot2/ggplot2-specs.html',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"(Logistic Regression)",frontmatter:{title:"(Logistic Regression)",date:"2023-05-09T21:01:11.000Z",permalink:"/pages/81f7ef/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/05.%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90/12.%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%20-%20%E5%90%B4%E6%81%A9%E8%BE%BE/06.%E9%80%BB%E8%BE%91%E5%9B%9E%E5%BD%92.html",relativePath:"05./12. - /06..md",key:"v-5e86d73a",path:"/pages/81f7ef/",headers:[{level:2,title:"6.1 Classification",slug:"_6-1--classification",normalizedTitle:"6.1 classification",charIndex:2},{level:2,title:"6.2 Hypothesis Representation",slug:"_6-2--hypothesis-representation",normalizedTitle:"6.2 hypothesis representation",charIndex:756},{level:2,title:"6.3 Decision boundary",slug:"_6-3--decision-boundary",normalizedTitle:"6.3 decision boundary",charIndex:2037},{level:2,title:"6.4 Cost function",slug:"_6-4--cost-function",normalizedTitle:"6.4 cost function",charIndex:3214},{level:2,title:"6.5 Simplified cost function and gradient descent",slug:"_6-5--simplified-cost-function-and-gradient-descent",normalizedTitle:"6.5 simplified cost function and gradient descent",charIndex:4706},{level:2,title:"6.6 Advanced optimization",slug:"_6-6--advanced-optimization",normalizedTitle:"6.6 advanced optimization",charIndex:6728},{level:2,title:"6.7 Multi-class classification: One-vs-all",slug:"_6-7---multi-class-classification-one-vs-all",normalizedTitle:"6.7 multi-class classification: one-vs-all",charIndex:7419}],headersStr:"6.1 Classification 6.2 Hypothesis Representation 6.3 Decision boundary 6.4 Cost function 6.5 Simplified cost function and gradient descent 6.6 Advanced optimization 6.7 Multi-class classification: One-vs-all",content:'# 6.1 Classification\n\n\n\n6-1.__bilibili\n\n\n\n yyy   (Logistic Regression) \n\n\n\n\n\n (dependent variable)  negative class  positive class y0,1y \\in {0,1}y0,1  0 1 \n\n\n\nyyy  0  1 1 0 yyy  0  1 0  1 1  0  0  1 \n\n yyy 1 0 0 1\n\n\n\n\n# 6.2 Hypothesis Representation\n\n\n\n6-2.__bilibili\n\n 0  1  0  1 \n\n\n\n\n\n 0  1\n\n h(x)>=0.5{h_\\theta}\\left( x \\right)>=0.5h  (x)>=0.5  y=1y=1y=1\n\n h(x)<0.5{h_\\theta}\\left( x \\right)<0.5h  (x)<0.5  y=0y=0y=0 \n\n 0.5  [0,1] \n\n 0  1 \n\nh(x)=g(TX)h_\\theta \\left( x \\right)=g\\left(\\theta^{T}X \\right)h  (x)=g( T X)\n\n\n\nXXX \n\nggg logistic function)  S Sigmoid function g(z)=11+ezg\\left( z \\right)=\\frac{1}{1+{e}^{-z}}g(z)= 1+e z 1 \n\n\n\n\n\n g(z)=11+ezg\\left( z \\right)=\\frac{1}{1+{e}^{-z}}g(z)= 1+e z 1 \n\nh(x)h_\\theta \\left( x \\right)h  (x) =1estimated probablityh(x)=P(y=1x;)h_\\theta \\left( x \\right)=P\\left( y=1|x;\\theta \\right)h  (x)=P(y=1x;)  xxx h(x)=0.7h_\\theta \\left( x \\right)=0.7h  (x)=0.7 70%  yyy  yyy  1-0.7=0.3\n\n\n# 6.3 Decision boundary\n\n\n\n6-3.__bilibili\n\n(decision boundary)\n\n\n\n\n\n h(x)>=0.5{h_\\theta}\\left( x \\right)>=0.5h  (x)>=0.5  y=1y=1y=1\n\n h(x)<0.5{h_\\theta}\\left( x \\right)<0.5h  (x)<0.5  y=0y=0y=0 \n\n S \n\nz=0z=0z=0  g(z)=0.5g(z)=0.5g(z)=0.5\n\nz>0z>0z>0  g(z)>0.5g(z)>0.5g(z)>0.5\n\nz<0z<0z<0  g(z)<0.5g(z)<0.5g(z)<0.5\n\n z=Txz={\\theta^{T}}xz= T x \n\nTx>=0{\\theta^{T}}x>=0 T x>=0  y=1y=1y=1\n\nTx<0{\\theta^{T}}x<0 T x<0  y=0y=0y=0\n\n\n\n\n\n \\theta  [-3 1 1]  3+x1+x20-3+{x_1}+{x_2} \\geq 03+x 1 +x 2 0 x1+x23{x_1}+{x_2} \\geq 3x 1 +x 2 3  y=1y=1y=1 x1+x2=3{x_1}+{x_2} = 3x 1 +x 2 =3 1  0 \n\n\n\n\n\n\n\n y=0y=0y=0  y=1y=1y=1 h(x)=g(0+1x1+2x2+3x12+4x22){h_\\theta}\\left( x \\right)=g\\left( {\\theta_0}+{\\theta_1}{x_1}+{\\theta_{2}}{x_{2}}+{\\theta_{3}}x_{1}^{2}+{\\theta_{4}}x_{2}^{2} \\right)h  (x)=g( 0 + 1 x 1 + 2 x 2 + 3 x 1 2 + 4 x 2 2 ) \\theta  [-1 0 0 1 1] 1 \n\n\n\n\n# 6.4 Cost function\n\n\n\n6-4.__bilibili\n\n \\theta \n\n\n\n h(x)=11+eTx{h_\\theta}\\left( x \\right)=\\frac{1}{1+{e^{-\\theta^{T}x}}}h  (x)= 1+e  T x 1 non-convexfunction\n\n\n\n\n\nJ()=1mi=1m12(h(x(i))y(i))2J\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}{\\frac{1}{2}{\\left( {h_\\theta}\\left({x}^{\\left( i \\right)} \\right)-{y}^{\\left( i \\right)} \\right)}^{2}}J()= m 1 i=1  m 2 1 (h  (x (i) )y (i) ) 2 \n\nJ()=1mi=1mCost(h(x(i)),y(i))J\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}{{Cost}\\left( {h_\\theta}\\left( {x}^{\\left( i \\right)} \\right),{y}^{\\left( i \\right)} \\right)}J()= m 1 i=1  m Cost(h  (x (i) ),y (i) ):\n\n\n\nh(x){h_\\theta}\\left( x \\right)h  (x)  Cost(h(x),y)Cost\\left( {h_\\theta}\\left( x \\right),y \\right)Cost(h  (x),y) \n\n\n\n Cost(h(x),y)Cost\\left( {h_\\theta}\\left( x \\right),y \\right)Cost(h  (x),y) \n\n *  y=1y=1y=1  h(x){h_\\theta}\\left( x \\right)h  (x)  1  0 y=1y=1y=1  h(x){h_\\theta}\\left( x \\right)h  (x)  1  h(x){h_\\theta}\\left( x \\right)h  (x) \n *  y=0y=0y=0  h(x){h_\\theta}\\left( x \\right)h  (x)  0  0 y=0y=0y=0  h(x){h_\\theta}\\left( x \\right)h  (x)  0  h(x){h_\\theta}\\left( x \\right)h  (x) \n\n\n# 6.5 Simplified cost function and gradient descent\n\n\n\n6-5.__bilibili\n\n\n\n\n\n\n\n\n\nCost(h(x),y)=ylog(h(x))(1y)log(1h(x))Cost\\left( {h_\\theta}\\left( x \\right),y \\right)=-y\\times log\\left( {h_\\theta}\\left( x \\right) \\right)-(1-y)\\times log\\left( 1-{h_\\theta}\\left( x \\right) \\right)Cost(h  (x),y)=ylog(h  (x))(1y)log(1h  (x)) =1mi=1m[y(i)log(h(x(i)))+(1y(i))log(1h(x(i)))]=-\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{y}^{(i)}\\log \\left( {h_\\theta}\\left( {x}^{(i)} \\right) \\right)+\\left( 1-{y}^{(i)} \\right)\\log \\left( 1-{h_\\theta}\\left( {x}^{(i)} \\right) \\right)]= m 1 i=1  m [y (i) log(h  (x (i) ))+(1y (i) )log(1h  (x (i) ))]\n\n J()J\\left( \\theta \\right)J()  \\theta\n\n(gradient descent)\n\n\n\nJ()=1mi=1m[y(i)log(h(x(i)))+(1y(i))log(1h(x(i)))]J\\left( \\theta \\right)=-\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{y}^{(i)}\\log \\left( {h_\\theta}\\left( {x}^{(i)} \\right) \\right)+\\left( 1-{y}^{(i)} \\right)\\log \\left( 1-{h_\\theta}\\left( {x}^{(i)} \\right) \\right)]J()= m 1 i=1  m [y (i) log(h  (x (i) ))+(1y (i) )log(1h  (x (i) ))]\n\n\n\njJ()=1mi=1m[h(x(i))y(i)]xj(i)\\frac{\\partial }{\\partial {\\theta_{j}}}J\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{h_\\theta}\\left( {x}^{(i)} \\right)-{y}^{(i)}]x_j^{(i)}  j  J()= m 1 i=1  m [h  (x (i) )y (i) ]x j (i)\n\n \\theta \n\n\n\n\n\n\n\nh(x)=TX=0x0+1x1+2x2+...+nxn{h_\\theta}\\left( x \\right)={\\theta^T}X={\\theta_{0}}{x_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= T X= 0 x 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n\n\nh(x)=11+eTX{h_\\theta}\\left( x \\right)=\\frac{1}{1+{e}^{-{\\theta^T}X}}h  (x)= 1+e  T X 1\n\n\n\n\n\n\n# 6.6 Advanced optimization\n\n\n\n6-6.__bilibili\n\n J()J\\left( \\theta \\right)J()\n\nBFGS ()L-BFGS ()  J()J\\left( \\theta \\right)J()\n\n\n\n \\alpha(line search) \\alpha aaa\n\n\n\n\n# 6.7 Multi-class classification: One-vs-all\n\n\n\n6-7.__bilibili\n\n (logistic regression)"" (one-vs-all)\n\n\n\n y=1y=1y=1y=2y=2y=2y=3y=3y=3y=4y=4y=4 \n\n y=1y=1y=1 y=2y=2y=2 y=3y=3y=3 \n\n y=1y=1y=1y=2y=2y=2y=3y=3y=3y=4y=4y=4 \n\n\n\n\n\n""\n\n 3  y=1y=1y=1 y=2y=2y=2 y=3y=3y=3 3 \n\n\n\n 1 "" 2  3  1 \n\n 1 0\n\ny=1y=1y=1h(1)(x)h_\\theta^{\\left( 1 \\right)}\\left( x \\right)h  (1) (x)y=2y=2y=2 h(2)(x)h_\\theta^{\\left( 2 \\right)}\\left( x \\right)h  (2) (x),  h(i)(x)=p(y=ix;)h_\\theta^{\\left( i \\right)}\\left( x \\right)=p\\left( y=i|x;\\theta \\right)h  (i) (x)=p(y=ix;)i=(1,2,3....k)i=\\left( 1,2,3....k \\right)i=(1,2,3....k)\n\n',normalizedContent:'# 6.1 classification\n\n\n\n6-1.__bilibili\n\n\n\n yyy   (logistic regression) \n\n\n\n\n\n (dependent variable)  negative class  positive class y0,1y \\in {0,1}y0,1  0 1 \n\n\n\nyyy  0  1 1 0 yyy  0  1 0  1 1  0  0  1 \n\n yyy 1 0 0 1\n\n\n\n\n# 6.2 hypothesis representation\n\n\n\n6-2.__bilibili\n\n 0  1  0  1 \n\n\n\n\n\n 0  1\n\n h(x)>=0.5{h_\\theta}\\left( x \\right)>=0.5h  (x)>=0.5  y=1y=1y=1\n\n h(x)<0.5{h_\\theta}\\left( x \\right)<0.5h  (x)<0.5  y=0y=0y=0 \n\n 0.5  [0,1] \n\n 0  1 \n\nh(x)=g(tx)h_\\theta \\left( x \\right)=g\\left(\\theta^{t}x \\right)h  (x)=g( t x)\n\n\n\nxxx \n\nggg logistic function)  s sigmoid function g(z)=11+ezg\\left( z \\right)=\\frac{1}{1+{e}^{-z}}g(z)= 1+e z 1 \n\n\n\n\n\n g(z)=11+ezg\\left( z \\right)=\\frac{1}{1+{e}^{-z}}g(z)= 1+e z 1 \n\nh(x)h_\\theta \\left( x \\right)h  (x) =1estimated probablityh(x)=p(y=1x;)h_\\theta \\left( x \\right)=p\\left( y=1|x;\\theta \\right)h  (x)=p(y=1x;)  xxx h(x)=0.7h_\\theta \\left( x \\right)=0.7h  (x)=0.7 70%  yyy  yyy  1-0.7=0.3\n\n\n# 6.3 decision boundary\n\n\n\n6-3.__bilibili\n\n(decision boundary)\n\n\n\n\n\n h(x)>=0.5{h_\\theta}\\left( x \\right)>=0.5h  (x)>=0.5  y=1y=1y=1\n\n h(x)<0.5{h_\\theta}\\left( x \\right)<0.5h  (x)<0.5  y=0y=0y=0 \n\n s \n\nz=0z=0z=0  g(z)=0.5g(z)=0.5g(z)=0.5\n\nz>0z>0z>0  g(z)>0.5g(z)>0.5g(z)>0.5\n\nz<0z<0z<0  g(z)<0.5g(z)<0.5g(z)<0.5\n\n z=txz={\\theta^{t}}xz= t x \n\ntx>=0{\\theta^{t}}x>=0 t x>=0  y=1y=1y=1\n\ntx<0{\\theta^{t}}x<0 t x<0  y=0y=0y=0\n\n\n\n\n\n \\theta  [-3 1 1]  3+x1+x20-3+{x_1}+{x_2} \\geq 03+x 1 +x 2 0 x1+x23{x_1}+{x_2} \\geq 3x 1 +x 2 3  y=1y=1y=1 x1+x2=3{x_1}+{x_2} = 3x 1 +x 2 =3 1  0 \n\n\n\n\n\n\n\n y=0y=0y=0  y=1y=1y=1 h(x)=g(0+1x1+2x2+3x12+4x22){h_\\theta}\\left( x \\right)=g\\left( {\\theta_0}+{\\theta_1}{x_1}+{\\theta_{2}}{x_{2}}+{\\theta_{3}}x_{1}^{2}+{\\theta_{4}}x_{2}^{2} \\right)h  (x)=g( 0 + 1 x 1 + 2 x 2 + 3 x 1 2 + 4 x 2 2 ) \\theta  [-1 0 0 1 1] 1 \n\n\n\n\n# 6.4 cost function\n\n\n\n6-4.__bilibili\n\n \\theta \n\n\n\n h(x)=11+etx{h_\\theta}\\left( x \\right)=\\frac{1}{1+{e^{-\\theta^{t}x}}}h  (x)= 1+e  t x 1 non-convexfunction\n\n\n\n\n\nj()=1mi=1m12(h(x(i))y(i))2j\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}{\\frac{1}{2}{\\left( {h_\\theta}\\left({x}^{\\left( i \\right)} \\right)-{y}^{\\left( i \\right)} \\right)}^{2}}j()= m 1 i=1  m 2 1 (h  (x (i) )y (i) ) 2 \n\nj()=1mi=1mcost(h(x(i)),y(i))j\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}{{cost}\\left( {h_\\theta}\\left( {x}^{\\left( i \\right)} \\right),{y}^{\\left( i \\right)} \\right)}j()= m 1 i=1  m cost(h  (x (i) ),y (i) ):\n\n\n\nh(x){h_\\theta}\\left( x \\right)h  (x)  cost(h(x),y)cost\\left( {h_\\theta}\\left( x \\right),y \\right)cost(h  (x),y) \n\n\n\n cost(h(x),y)cost\\left( {h_\\theta}\\left( x \\right),y \\right)cost(h  (x),y) \n\n *  y=1y=1y=1  h(x){h_\\theta}\\left( x \\right)h  (x)  1  0 y=1y=1y=1  h(x){h_\\theta}\\left( x \\right)h  (x)  1  h(x){h_\\theta}\\left( x \\right)h  (x) \n *  y=0y=0y=0  h(x){h_\\theta}\\left( x \\right)h  (x)  0  0 y=0y=0y=0  h(x){h_\\theta}\\left( x \\right)h  (x)  0  h(x){h_\\theta}\\left( x \\right)h  (x) \n\n\n# 6.5 simplified cost function and gradient descent\n\n\n\n6-5.__bilibili\n\n\n\n\n\n\n\n\n\ncost(h(x),y)=ylog(h(x))(1y)log(1h(x))cost\\left( {h_\\theta}\\left( x \\right),y \\right)=-y\\times log\\left( {h_\\theta}\\left( x \\right) \\right)-(1-y)\\times log\\left( 1-{h_\\theta}\\left( x \\right) \\right)cost(h  (x),y)=ylog(h  (x))(1y)log(1h  (x)) =1mi=1m[y(i)log(h(x(i)))+(1y(i))log(1h(x(i)))]=-\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{y}^{(i)}\\log \\left( {h_\\theta}\\left( {x}^{(i)} \\right) \\right)+\\left( 1-{y}^{(i)} \\right)\\log \\left( 1-{h_\\theta}\\left( {x}^{(i)} \\right) \\right)]= m 1 i=1  m [y (i) log(h  (x (i) ))+(1y (i) )log(1h  (x (i) ))]\n\n j()j\\left( \\theta \\right)j()  \\theta\n\n(gradient descent)\n\n\n\nj()=1mi=1m[y(i)log(h(x(i)))+(1y(i))log(1h(x(i)))]j\\left( \\theta \\right)=-\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{y}^{(i)}\\log \\left( {h_\\theta}\\left( {x}^{(i)} \\right) \\right)+\\left( 1-{y}^{(i)} \\right)\\log \\left( 1-{h_\\theta}\\left( {x}^{(i)} \\right) \\right)]j()= m 1 i=1  m [y (i) log(h  (x (i) ))+(1y (i) )log(1h  (x (i) ))]\n\n\n\njj()=1mi=1m[h(x(i))y(i)]xj(i)\\frac{\\partial }{\\partial {\\theta_{j}}}j\\left( \\theta \\right)=\\frac{1}{m}\\sum\\limits_{i=1}^{m}[{h_\\theta}\\left( {x}^{(i)} \\right)-{y}^{(i)}]x_j^{(i)}  j  j()= m 1 i=1  m [h  (x (i) )y (i) ]x j (i)\n\n \\theta \n\n\n\n\n\n\n\nh(x)=tx=0x0+1x1+2x2+...+nxn{h_\\theta}\\left( x \\right)={\\theta^t}x={\\theta_{0}}{x_{0}}+{\\theta_{1}}{x_{1}}+{\\theta_{2}}{x_{2}}+...+{\\theta_{n}}{x_{n}}h  (x)= t x= 0 x 0 + 1 x 1 + 2 x 2 +...+ n x n\n\n\n\nh(x)=11+etx{h_\\theta}\\left( x \\right)=\\frac{1}{1+{e}^{-{\\theta^t}x}}h  (x)= 1+e  t x 1\n\n\n\n\n\n\n# 6.6 advanced optimization\n\n\n\n6-6.__bilibili\n\n j()j\\left( \\theta \\right)j()\n\nbfgs ()l-bfgs ()  j()j\\left( \\theta \\right)j()\n\n\n\n \\alpha(line search) \\alpha aaa\n\n\n\n\n# 6.7 multi-class classification: one-vs-all\n\n\n\n6-7.__bilibili\n\n (logistic regression)"" (one-vs-all)\n\n\n\n y=1y=1y=1y=2y=2y=2y=3y=3y=3y=4y=4y=4 \n\n y=1y=1y=1 y=2y=2y=2 y=3y=3y=3 \n\n y=1y=1y=1y=2y=2y=2y=3y=3y=3y=4y=4y=4 \n\n\n\n\n\n""\n\n 3  y=1y=1y=1 y=2y=2y=2 y=3y=3y=3 3 \n\n\n\n 1 "" 2  3  1 \n\n 1 0\n\ny=1y=1y=1h(1)(x)h_\\theta^{\\left( 1 \\right)}\\left( x \\right)h  (1) (x)y=2y=2y=2 h(2)(x)h_\\theta^{\\left( 2 \\right)}\\left( x \\right)h  (2) (x),  h(i)(x)=p(y=ix;)h_\\theta^{\\left( i \\right)}\\left( x \\right)=p\\left( y=i|x;\\theta \\right)h  (i) (x)=p(y=ix;)i=(1,2,3....k)i=\\left( 1,2,3....k \\right)i=(1,2,3....k)\n\n',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"tidyverse",frontmatter:{title:"tidyverse",date:"2023-07-03T21:14:09.000Z",permalink:"/pages/c63497/",categories:["R","tidyverse"],tags:["tidyverse"],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/12.R%20-%20tidyverse/01.tidyverse.html",relativePath:"06./12.R - tidyverse/01.tidyverse.md",key:"v-1b276464",path:"/pages/c63497/",headers:[{level:2,title:"Overview",slug:"overview",normalizedTitle:"overview",charIndex:90},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:303},{level:2,title:"Usage",slug:"usage",normalizedTitle:"usage",charIndex:457},{level:2,title:"Packages",slug:"packages",normalizedTitle:"packages",charIndex:2277}],excerpt:'<p><a href=\'https://tidyverse.tidyverse.org\'><img src=\'https://tidyverse.tidyverse.org/logo.png\' align="right" height="138.5" /></a><br>\nGitHub: <a href="https://github.com/tidyverse/tidyverse" target="_blank" rel="noopener noreferrer">https://github.com/tidyverse/tidyverse<OutboundLink/></a><br>\nDoc: <a href="https://tidyverse.tidyverse.org/" target="_blank" rel="noopener noreferrer">https://tidyverse.tidyverse.org/<OutboundLink/></a></p>\n',headersStr:"Overview Installation Usage Packages",content:'\nGitHub: https://github.com/tidyverse/tidyverse\nDoc: https://tidyverse.tidyverse.org/\n\n\n# Overview\n\ntidyverse  API tidyverse  tidyverse \n\n tidyverseR for data science\n\n * Original Book: r4ds\n\n * Translation Book: r4ds-zh-cn\n\n\n# Installation\n\n# Install from CRAN\ninstall.packages("tidyverse")\n\n\n1\n2\n\n\n pak::pkg_system_requirements("tidyverse")\n\n\n# Usage\n\nlibrary(tidyverse)  tidyverse \n\n * ggplot2, for data visualisation.\n * dplyr, for data manipulation.\n * tidyr, for data tidying.\n * readr, for data import.\n * purrr, for functional programming.\n * tibble, for tibbles, a modern re-imagining of data frames.\n * stringr, for strings.\n * forcats, for factors.\n * lubridate, for date/times.\n\n\n\nlibrary(tidyverse)\n#>  Attaching core tidyverse packages  tidyverse 2.0.0 \n#>  dplyr     1.1.0      readr     2.1.4\n#>  forcats   1.0.0      stringr   1.5.0\n#>  ggplot2   3.4.1      tibble    3.1.8\n#>  lubridate 1.9.2      tidyr     1.3.0\n#>  purrr     1.0.1     \n#>  Conflicts  tidyverse_conflicts() \n#>  dplyr::filter() masks stats::filter()\n#>  dplyr::lag()    masks stats::lag()\n#>  Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n tidyverse_conflicts() \n\nlibrary(MASS)\n#> \n#> Attaching package: \'MASS\'\n#> The following object is masked from \'package:dplyr\':\n#> \n#>     select\ntidyverse_conflicts()\n#>  Conflicts  tidyverse_conflicts() \n#>  dplyr::filter() masks stats::filter()\n#>  dplyr::lag()    masks stats::lag()\n#>  MASS::select()  masks dplyr::select()\n#>  Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n\n\n tidyverse_update()  tidyverse \n\ntidyverse_update()\n#> The following packages are out of date:\n#>  * broom (0.4.0 -> 0.4.1)\n#>  * DBI   (0.4.1 -> 0.5)\n#>  * Rcpp  (0.12.6 -> 0.12.7)\n#>  \n#> Start a clean R session then run:\n#> install.packages(c("broom", "DBI", "Rcpp"))\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n\n# Packages\n\n tidyverse  \n\n * Working with specific types of vectors:\n   \n   * hms, for times.\n\n * Importing other types of data:\n   \n   * feather, for sharing with Python and other languages.\n   * haven, for SPSS, SAS and Stata files.\n   * httr, for web apis.\n   * jsonlite for JSON.\n   * readxl, for .xls and .xlsx files.\n   * rvest, for web scraping.\n   * xml2, for XML.\n\n * Modelling\n   \n   * modelr, for modelling within a pipeline\n   * broom, for turning models into tidy data',normalizedContent:'\ngithub: https://github.com/tidyverse/tidyverse\ndoc: https://tidyverse.tidyverse.org/\n\n\n# overview\n\ntidyverse  api tidyverse  tidyverse \n\n tidyverser for data science\n\n * original book: r4ds\n\n * translation book: r4ds-zh-cn\n\n\n# installation\n\n# install from cran\ninstall.packages("tidyverse")\n\n\n1\n2\n\n\n pak::pkg_system_requirements("tidyverse")\n\n\n# usage\n\nlibrary(tidyverse)  tidyverse \n\n * ggplot2, for data visualisation.\n * dplyr, for data manipulation.\n * tidyr, for data tidying.\n * readr, for data import.\n * purrr, for functional programming.\n * tibble, for tibbles, a modern re-imagining of data frames.\n * stringr, for strings.\n * forcats, for factors.\n * lubridate, for date/times.\n\n\n\nlibrary(tidyverse)\n#>  attaching core tidyverse packages  tidyverse 2.0.0 \n#>  dplyr     1.1.0      readr     2.1.4\n#>  forcats   1.0.0      stringr   1.5.0\n#>  ggplot2   3.4.1      tibble    3.1.8\n#>  lubridate 1.9.2      tidyr     1.3.0\n#>  purrr     1.0.1     \n#>  conflicts  tidyverse_conflicts() \n#>  dplyr::filter() masks stats::filter()\n#>  dplyr::lag()    masks stats::lag()\n#>  use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n tidyverse_conflicts() \n\nlibrary(mass)\n#> \n#> attaching package: \'mass\'\n#> the following object is masked from \'package:dplyr\':\n#> \n#>     select\ntidyverse_conflicts()\n#>  conflicts  tidyverse_conflicts() \n#>  dplyr::filter() masks stats::filter()\n#>  dplyr::lag()    masks stats::lag()\n#>  mass::select()  masks dplyr::select()\n#>  use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n\n\n tidyverse_update()  tidyverse \n\ntidyverse_update()\n#> the following packages are out of date:\n#>  * broom (0.4.0 -> 0.4.1)\n#>  * dbi   (0.4.1 -> 0.5)\n#>  * rcpp  (0.12.6 -> 0.12.7)\n#>  \n#> start a clean r session then run:\n#> install.packages(c("broom", "dbi", "rcpp"))\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n\n\n\n# packages\n\n tidyverse  \n\n * working with specific types of vectors:\n   \n   * hms, for times.\n\n * importing other types of data:\n   \n   * feather, for sharing with python and other languages.\n   * haven, for spss, sas and stata files.\n   * httr, for web apis.\n   * jsonlite for json.\n   * readxl, for .xls and .xlsx files.\n   * rvest, for web scraping.\n   * xml2, for xml.\n\n * modelling\n   \n   * modelr, for modelling within a pipeline\n   * broom, for turning models into tidy data',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"linkET",frontmatter:{title:"linkET",date:"2023-07-15T16:11:13.000Z",permalink:"/pages/142715/",categories:["R","R "],tags:["R "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/15.R%20-%20%E7%BB%98%E5%9B%BE/006.linkET.html",relativePath:"06./15.R - /006.linkET.md",key:"v-ba6ecaea",path:"/pages/142715/",headers:[{level:2,title:"linkET ",slug:"linket-",normalizedTitle:"linket ",charIndex:104},{level:2,title:" Mantel test ",slug:"-mantel-test-",normalizedTitle:" mantel test ",charIndex:172},{level:2,title:"Mantel test ",slug:"mantel-test-",normalizedTitle:"mantel test ",charIndex:642},{level:2,title:"linkET ",slug:"linket-",normalizedTitle:"linket ",charIndex:808}],excerpt:'<p>GitHub: <a href="https://github.com/Hy4m/linkET" target="_blank" rel="noopener noreferrer">https://github.com/Hy4m/linkET<OutboundLink/></a></p>\n',headersStr:"linkET   Mantel test  Mantel test  linkET ",content:"GitHub: https://github.com/Hy4m/linkET\n\n\n\n 1. R Mantel_test  -  (bilibili.com)\n\n\n# linkET \n\nlinkET  Mantel test \n\n\n\n\n#  Mantel test \n\nMantel test  Mantel tests  A  B \n\nMantel test  A  B  A  nx120 x  B  mx120 x  Mantel testMantel test  P  A  B  A  B \n\nhttps://doi.org/10.1111/2041-210x.12018\n\n\n# Mantel test \n\n 1.  A  B \n 2.  pearson \n 3.  N \n 4.  R  R  R  R \n\n\n# linkET \n\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//006.linkET/linkET-tutorial.html",normalizedContent:"github: https://github.com/hy4m/linket\n\n\n\n 1. r mantel_test  -  (bilibili.com)\n\n\n# linket \n\nlinket  mantel test \n\n\n\n\n#  mantel test \n\nmantel test  mantel tests  a  b \n\nmantel test  a  b  a  nx120 x  b  mx120 x  mantel testmantel test  p  a  b  a  b \n\nhttps://doi.org/10.1111/2041-210x.12018\n\n\n# mantel test \n\n 1.  a  b \n 2.  pearson \n 3.  n \n 4.  r  r  r  r \n\n\n# linket \n\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//006.linket/linket-tutorial.html",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"ggalluvial",frontmatter:{title:"ggalluvial",date:"2023-05-24T16:07:05.000Z",permalink:"/pages/29214b/",categories:["R","R "],tags:["R "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/15.R%20-%20%E7%BB%98%E5%9B%BE/004.ggalluvial.html",relativePath:"06./15.R - /004.ggalluvial.md",key:"v-aaf5ea96",path:"/pages/29214b/",headers:[{level:2,title:"ggalluvial ",slug:"ggalluvial-",normalizedTitle:"ggalluvial ",charIndex:100},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:550},{level:2,title:"Usage",slug:"usage",normalizedTitle:"usage",charIndex:1065},{level:2,title:"Documentation",slug:"documentation",normalizedTitle:"documentation",charIndex:2930}],excerpt:'<p>GitHub: <a href="https://github.com/corybrunson/ggalluvial" target="_blank" rel="noopener noreferrer">https://github.com/corybrunson/ggalluvial<OutboundLink/></a><br>\nDoc: <a href="http://corybrunson.github.io/ggalluvial/" target="_blank" rel="noopener noreferrer">http://corybrunson.github.io/ggalluvial/<OutboundLink/></a></p>\n',headersStr:"ggalluvial  Installation Usage Documentation",content:'GitHub: https://github.com/corybrunson/ggalluvial\nDoc: http://corybrunson.github.io/ggalluvial/\n\n\n# ggalluvial \n\nggalluvial  alluvial  ggplot2 \n\n * alluvial vectors, lists, data frames ggalluvial  data frame\n * alluvial  ggalluvial \n * alluvial  alluvia strata ggalluvial stats and geoms strataalluviallodesflows\n\nggalluvial  axis  strata \n\n\n# Installation\n\n CRAN \n\ninstall.packages("ggalluvial")\n\n\n1\n\n\ncran  CRAN  master \n\n GitHub  main \n\nremotes::install_github("corybrunson/ggalluvial@main", build_vignettes = TRUE)\n\n\n1\n\n\n vignettes alluvialggfittext  ggrepel build_vignettes  FALSE\n\noptimization #6\n\nremotes::install_github("corybrunson/ggalluvial", ref = "optimization")\n\n\n1\n\n\n CRAN \n\n\n# Usage\n\n alluvial plot \n\ntitanic_wide <- data.frame(Titanic)\nhead(titanic_wide)\n#>   Class    Sex   Age Survived Freq\n#> 1   1st   Male Child       No    0\n#> 2   2nd   Male Child       No    0\n#> 3   3rd   Male Child       No   35\n#> 4  Crew   Male Child       No    0\n#> 5   1st Female Child       No    0\n#> 6   2nd Female Child       No    0\nggplot(data = titanic_wide,\n       aes(axis1 = Class, axis2 = Sex, axis3 = Age,\n           y = Freq)) +\n  scale_x_discrete(limits = c("Class", "Sex", "Age"), expand = c(.2, .05)) +\n  xlab("Demographic") +\n  geom_alluvium(aes(fill = Survived)) +\n  geom_stratum() +\n  geom_text(stat = "stratum", aes(label = after_stat(stratum))) +\n  theme_minimal() +\n  ggtitle("passengers on the maiden voyage of the Titanic",\n          "stratified by demographics and survival")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n\n\n\n\n wide  ggalluvial  long \n\ntitanic_long <- to_lodes_form(data.frame(Titanic),\n                              key = "Demographic",\n                              axes = 1:3)\nhead(titanic_long)\n#>   Survived Freq alluvium Demographic stratum\n#> 1       No    0        1       Class     1st\n#> 2       No    0        2       Class     2nd\n#> 3       No   35        3       Class     3rd\n#> 4       No    0        4       Class    Crew\n#> 5       No    0        5       Class     1st\n#> 6       No    0        6       Class     2nd\nggplot(data = titanic_long,\n       aes(x = Demographic, stratum = stratum, alluvium = alluvium,\n           y = Freq, label = stratum)) +\n  geom_alluvium(aes(fill = Survived)) +\n  geom_stratum() + geom_text(stat = "stratum") +\n  theme_minimal() +\n  ggtitle("passengers on the maiden voyage of the Titanic",\n          "stratified by demographics and survival")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n\n\n\n\n\n# Documentation\n\n: http://corybrunson.github.io/ggalluvial/articles/ggalluvial.html\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//004.ggalluvial/Get started.nb.html',normalizedContent:'github: https://github.com/corybrunson/ggalluvial\ndoc: http://corybrunson.github.io/ggalluvial/\n\n\n# ggalluvial \n\nggalluvial  alluvial  ggplot2 \n\n * alluvial vectors, lists, data frames ggalluvial  data frame\n * alluvial  ggalluvial \n * alluvial  alluvia strata ggalluvial stats and geoms strataalluviallodesflows\n\nggalluvial  axis  strata \n\n\n# installation\n\n cran \n\ninstall.packages("ggalluvial")\n\n\n1\n\n\ncran  cran  master \n\n github  main \n\nremotes::install_github("corybrunson/ggalluvial@main", build_vignettes = true)\n\n\n1\n\n\n vignettes alluvialggfittext  ggrepel build_vignettes  false\n\noptimization #6\n\nremotes::install_github("corybrunson/ggalluvial", ref = "optimization")\n\n\n1\n\n\n cran \n\n\n# usage\n\n alluvial plot \n\ntitanic_wide <- data.frame(titanic)\nhead(titanic_wide)\n#>   class    sex   age survived freq\n#> 1   1st   male child       no    0\n#> 2   2nd   male child       no    0\n#> 3   3rd   male child       no   35\n#> 4  crew   male child       no    0\n#> 5   1st female child       no    0\n#> 6   2nd female child       no    0\nggplot(data = titanic_wide,\n       aes(axis1 = class, axis2 = sex, axis3 = age,\n           y = freq)) +\n  scale_x_discrete(limits = c("class", "sex", "age"), expand = c(.2, .05)) +\n  xlab("demographic") +\n  geom_alluvium(aes(fill = survived)) +\n  geom_stratum() +\n  geom_text(stat = "stratum", aes(label = after_stat(stratum))) +\n  theme_minimal() +\n  ggtitle("passengers on the maiden voyage of the titanic",\n          "stratified by demographics and survival")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n\n\n\n\n wide  ggalluvial  long \n\ntitanic_long <- to_lodes_form(data.frame(titanic),\n                              key = "demographic",\n                              axes = 1:3)\nhead(titanic_long)\n#>   survived freq alluvium demographic stratum\n#> 1       no    0        1       class     1st\n#> 2       no    0        2       class     2nd\n#> 3       no   35        3       class     3rd\n#> 4       no    0        4       class    crew\n#> 5       no    0        5       class     1st\n#> 6       no    0        6       class     2nd\nggplot(data = titanic_long,\n       aes(x = demographic, stratum = stratum, alluvium = alluvium,\n           y = freq, label = stratum)) +\n  geom_alluvium(aes(fill = survived)) +\n  geom_stratum() + geom_text(stat = "stratum") +\n  theme_minimal() +\n  ggtitle("passengers on the maiden voyage of the titanic",\n          "stratified by demographics and survival")\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n\n\n\n\n\n# documentation\n\n: http://corybrunson.github.io/ggalluvial/articles/ggalluvial.html\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//004.ggalluvial/get started.nb.html',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"Biorplot",frontmatter:{title:"Biorplot",date:"2024-07-27T09:40:16.000Z",permalink:"/pages/22fafa/",categories:[""],tags:["R - "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/15.R%20-%20%E7%BB%98%E5%9B%BE/008.Biorplot.html",relativePath:"06./15.R - /008.Biorplot.md",key:"v-371f414b",path:"/pages/22fafa/",excerpt:'<p>GitHub: <a href="https://github.com/zhenghu159/Biorplot" target="_blank" rel="noopener noreferrer">https://github.com/zhenghu159/Biorplot<OutboundLink/></a><br>\nDoc: <a href="https://zhenghu159.github.io/Biorplot/" target="_blank" rel="noopener noreferrer">https://zhenghu159.github.io/Biorplot/<OutboundLink/></a></p>\n',headersStr:null,content:"GitHub: https://github.com/zhenghu159/Biorplot\nDoc: https://zhenghu159.github.io/Biorplot/\n\n\n\n\n\n 1. R --- Biorplot  --- 1\n 2. R --- Biorplot  --- 2\n 3. R --- Biorplot  --- 3\n 4. R --- Biorplot  --- 4\n 5. R --- Biorplot  --- 5",normalizedContent:"github: https://github.com/zhenghu159/biorplot\ndoc: https://zhenghu159.github.io/biorplot/\n\n\n\n\n\n 1. r --- biorplot  --- 1\n 2. r --- biorplot  --- 2\n 3. r --- biorplot  --- 3\n 4. r --- biorplot  --- 4\n 5. r --- biorplot  --- 5",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"quarto",frontmatter:{title:"quarto",date:"2024-07-27T09:30:13.000Z",permalink:"/pages/02875c/",categories:[""],tags:["R - "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/16.R%20-%20%E5%85%B6%E4%BB%96/01.quarto.html",relativePath:"06./16.R - /01.quarto.md",key:"v-5c93088f",path:"/pages/02875c/",excerpt:'<p>GitHub: <a href="https://github.com/quarto-dev/quarto-cli" target="_blank" rel="noopener noreferrer">https://github.com/quarto-dev/quarto-cli<OutboundLink/></a><br>\nDoc<a href="https://quarto.org/" target="_blank" rel="noopener noreferrer">https://quarto.org/<OutboundLink/></a></p>\n',headersStr:null,content:"GitHub: https://github.com/quarto-dev/quarto-cli\nDochttps://quarto.org/\n\n\n\n\n\n 1. quarto  1Hello, Quarto\n 2. quarto  2Computations\n 3. quarto  3Authoring\n 4. Quarto Dashboards  1Overview\n 5. Quarto Dashboards  2Dashboard Layout\n 6. Quarto Dashboards  3Dashboard Data Display",normalizedContent:"github: https://github.com/quarto-dev/quarto-cli\ndochttps://quarto.org/\n\n\n\n\n\n 1. quarto  1hello, quarto\n 2. quarto  2computations\n 3. quarto  3authoring\n 4. quarto dashboards  1overview\n 5. quarto dashboards  2dashboard layout\n 6. quarto dashboards  3dashboard data display",charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"DiagrammeR",frontmatter:{title:"DiagrammeR",date:"2023-06-25T09:22:14.000Z",permalink:"/pages/6580bf/",categories:["R","R "],tags:["R "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/15.R%20-%20%E7%BB%98%E5%9B%BE/005.DiagrammeR.html",relativePath:"06./15.R - /005.DiagrammeR.md",key:"v-0ec22c8d",path:"/pages/6580bf/",headers:[{level:2,title:"DiagrammeR ",slug:"diagrammer-",normalizedTitle:"diagrammer ",charIndex:112},{level:2,title:"Installation",slug:"installation",normalizedTitle:"installation",charIndex:1399},{level:2,title:"Documentation",slug:"documentation",normalizedTitle:"documentation",charIndex:1692},{level:3,title:"Get start",slug:"get-start",normalizedTitle:"get start",charIndex:1710},{level:3,title:"Graphviz and mermaid in DiagrammeR",slug:"graphviz-and-mermaid-in-diagrammer",normalizedTitle:"graphviz and mermaid in diagrammer",charIndex:1923}],excerpt:'<p>GitHub: <a href="https://github.com/rich-iannone/DiagrammeR" target="_blank" rel="noopener noreferrer">https://github.com/rich-iannone/DiagrammeR<OutboundLink/></a><br>\nDoc: <a href="http://rich-iannone.github.io/DiagrammeR/index.html" target="_blank" rel="noopener noreferrer">http://rich-iannone.github.io/DiagrammeR/index.html<OutboundLink/></a></p>\n',headersStr:"DiagrammeR  Installation Documentation Get start Graphviz and mermaid in DiagrammeR",content:'GitHub: https://github.com/rich-iannone/DiagrammeR\nDoc: http://rich-iannone.github.io/DiagrammeR/index.html\n\n\n# DiagrammeR \n\n\n\n DiagrammeR network graph diagrams R Markdown  Shiny web apps \n\n\n\n DiagrammeR \n\nexample_graph <-\n  create_graph() %>%\n  add_pa_graph(\n    n = 50, m = 1,\n    set_seed = 23\n  ) %>%\n  add_gnp_graph(\n    n = 50, p = 1/100,\n    set_seed = 23\n  ) %>%\n  join_node_attrs(df = get_betweenness(.)) %>%\n  join_node_attrs(df = get_degree_total(.)) %>%\n  colorize_node_attrs(\n    node_attr_from = total_degree,\n    node_attr_to = fillcolor,\n    palette = "Greens",\n    alpha = 90\n  ) %>%\n  rescale_node_attrs(\n    node_attr_from = betweenness,\n    to_lower_bound = 0.5,\n    to_upper_bound = 1.0,\n    node_attr_to = height\n  ) %>%\n  select_nodes_by_id(nodes = get_articulation_points(.)) %>%\n  set_node_attrs_ws(node_attr = peripheries, value = 2) %>%\n  set_node_attrs_ws(node_attr = penwidth, value = 3) %>%\n  clear_selection() %>%\n  set_node_attr_to_display(attr = NULL)\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n\n\nrender_graph(example_graph, layout = "nicely")\n\n\n1\n\n\nDiametermeR node data framesedge data frames\n\n\n# Installation\n\nDiametermeR  R   R Comprehensive R Archive Network (CRAN) \n\n devtools  GitHub  DiagrammeR \n\ndevtools::install_github("rich-iannone/DiagrammeR")\n\n\n1\n\n\n CRAN \n\ninstall.packages("DiagrammeR")\n\n\n1\n\n\n\n\n\n# Documentation\n\n\n# Get start\n\n: http://rich-iannone.github.io/DiagrammeR/#graph-basics\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//005.DiagrammeR/get_start.nb.html\n\n\n# Graphviz and mermaid in DiagrammeR\n\n: http://rich-iannone.github.io/DiagrammeR/articles/graphviz-mermaid.html\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//005.DiagrammeR/graphviz-mermaid.nb.html',normalizedContent:'github: https://github.com/rich-iannone/diagrammer\ndoc: http://rich-iannone.github.io/diagrammer/index.html\n\n\n# diagrammer \n\n\n\n diagrammer network graph diagrams r markdown  shiny web apps \n\n\n\n diagrammer \n\nexample_graph <-\n  create_graph() %>%\n  add_pa_graph(\n    n = 50, m = 1,\n    set_seed = 23\n  ) %>%\n  add_gnp_graph(\n    n = 50, p = 1/100,\n    set_seed = 23\n  ) %>%\n  join_node_attrs(df = get_betweenness(.)) %>%\n  join_node_attrs(df = get_degree_total(.)) %>%\n  colorize_node_attrs(\n    node_attr_from = total_degree,\n    node_attr_to = fillcolor,\n    palette = "greens",\n    alpha = 90\n  ) %>%\n  rescale_node_attrs(\n    node_attr_from = betweenness,\n    to_lower_bound = 0.5,\n    to_upper_bound = 1.0,\n    node_attr_to = height\n  ) %>%\n  select_nodes_by_id(nodes = get_articulation_points(.)) %>%\n  set_node_attrs_ws(node_attr = peripheries, value = 2) %>%\n  set_node_attrs_ws(node_attr = penwidth, value = 3) %>%\n  clear_selection() %>%\n  set_node_attr_to_display(attr = null)\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n\n\nrender_graph(example_graph, layout = "nicely")\n\n\n1\n\n\ndiametermer node data framesedge data frames\n\n\n# installation\n\ndiametermer  r   r comprehensive r archive network (cran) \n\n devtools  github  diagrammer \n\ndevtools::install_github("rich-iannone/diagrammer")\n\n\n1\n\n\n cran \n\ninstall.packages("diagrammer")\n\n\n1\n\n\n\n\n\n# documentation\n\n\n# get start\n\n: http://rich-iannone.github.io/diagrammer/#graph-basics\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//005.diagrammer/get_start.nb.html\n\n\n# graphviz and mermaid in diagrammer\n\n: http://rich-iannone.github.io/diagrammer/articles/graphviz-mermaid.html\n: http://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//005.diagrammer/graphviz-mermaid.nb.html',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:"circlize",frontmatter:{title:"circlize",date:"2023-08-21T20:26:22.000Z",permalink:"/pages/a41a6b/",categories:["","R"],tags:["R "],readingShow:"top"},regularPath:"/06.%E7%BC%96%E7%A8%8B%E5%AD%A6%E4%B9%A0/15.R%20-%20%E7%BB%98%E5%9B%BE/007.circlize.html",relativePath:"06./15.R - /007.circlize.md",key:"v-e53f033e",path:"/pages/a41a6b/",headers:[{level:2,title:"circlize ",slug:"circlize-",normalizedTitle:"circlize ",charIndex:152},{level:2,title:"Examples",slug:"examples",normalizedTitle:"examples",charIndex:344},{level:2,title:"Install",slug:"install",normalizedTitle:"install",charIndex:409},{level:2,title:"book",slug:"book",normalizedTitle:"book",charIndex:86},{level:3,title:"About",slug:"about",normalizedTitle:"about",charIndex:584},{level:3,title:"1 Introduction",slug:"_1-introduction",normalizedTitle:"1 introduction",charIndex:787},{level:3,title:"6 The circos.heatmap() function",slug:"_6-the-circos-heatmap-function",normalizedTitle:"6 the circos.heatmap() function",charIndex:1013}],excerpt:'<p>GitHub: <a href="https://github.com/jokergoo/circlize" target="_blank" rel="noopener noreferrer">https://github.com/jokergoo/circlize<OutboundLink/></a><br>\nDoc: <a href="https://jokergoo.github.io/circlize_book/book/" target="_blank" rel="noopener noreferrer">https://jokergoo.github.io/circlize_book/book/<OutboundLink/></a><br>\nDOI: <a href="https://doi.org/10.1093/bioinformatics/btu393" target="_blank" rel="noopener noreferrer">https://doi.org/10.1093/bioinformatics/btu393<OutboundLink/></a></p>\n',headersStr:"circlize  Examples Install book About 1 Introduction 6 The circos.heatmap() function",content:'GitHub: https://github.com/jokergoo/circlize\nDoc: https://jokergoo.github.io/circlize_book/book/\nDOI: https://doi.org/10.1093/bioinformatics/btu393\n\n\n# circlize \n\n\n\ncirclize  R  R circlize \n\n\n# Examples\n\nSee https://jokergoo.github.io/circlize_examples/.\n\n\n# Install\n\nThe package can be installed from CRAN:\n\ninstall.packages("circlize")\n\n\n1\n\n\nor directly from GitHub:\n\ndevtools::install_github("jokergoo/circlize")\n\n\n1\n\n\n\n# book\n\n\n# About\n\nhttps://jokergoo.github.io/circlize_book/book/index.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//007.circlize/00-about.html\n\n\n# 1 Introduction\n\nhttps://jokergoo.github.io/circlize_book/book/introduction.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//007.circlize/01-introduction.html\n\n\n# 6 The circos.heatmap() function\n\nhttps://jokergoo.github.io/circlize_book/book/circos-heatmap.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/Bioinformatics-Tools/blob/main/R//007.circlize/06-circos-heatmap.html',normalizedContent:'github: https://github.com/jokergoo/circlize\ndoc: https://jokergoo.github.io/circlize_book/book/\ndoi: https://doi.org/10.1093/bioinformatics/btu393\n\n\n# circlize \n\n\n\ncirclize  r  r circlize \n\n\n# examples\n\nsee https://jokergoo.github.io/circlize_examples/.\n\n\n# install\n\nthe package can be installed from cran:\n\ninstall.packages("circlize")\n\n\n1\n\n\nor directly from github:\n\ndevtools::install_github("jokergoo/circlize")\n\n\n1\n\n\n\n# book\n\n\n# about\n\nhttps://jokergoo.github.io/circlize_book/book/index.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//007.circlize/00-about.html\n\n\n# 1 introduction\n\nhttps://jokergoo.github.io/circlize_book/book/introduction.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//007.circlize/01-introduction.html\n\n\n# 6 the circos.heatmap() function\n\nhttps://jokergoo.github.io/circlize_book/book/circos-heatmap.html\nhttp://htmlpreview.github.io/?https://github.com/zhenghu159/bioinformatics-tools/blob/main/r//007.circlize/06-circos-heatmap.html',charsets:{cjk:!0},lastUpdated:"2024/07/27, 10:10:31",lastUpdatedTimestamp:1722046231e3},{title:" - ",frontmatter:{title:" - ",date:"2022-07-30T22:29:33.000Z",permalink:"/about/me/introduce/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/01.%E5%85%B3%E4%BA%8E%20-%20%E8%87%AA%E6%88%91/01.%E8%87%AA%E6%88%91%20-%20%E4%BB%8B%E7%BB%8D.html",relativePath:"100./01. - /01. - .md",key:"v-114caebc",path:"/about/me/introduce/",headers:[{level:2,title:"About Me",slug:"about-me",normalizedTitle:"about me",charIndex:2},{level:2,title:"My Skill",slug:"my-skill",normalizedTitle:"my skill",charIndex:135},{level:2,title:"Contact Me",slug:"contact-me",normalizedTitle:"contact me",charIndex:156},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:225}],headersStr:"About Me My Skill Contact Me ",content:"# About Me\n\n HZAU \n\n \n\n  NERvGear \n\n  Blog \n\n\n# My Skill\n\n\n\n\n\n\n\n\n\n\n# Contact Me\n\n: 1308155474@qq.com\n: https://github.com/ZhengTiger/\n\n\n# \n\n TigerZ  \n\n",normalizedContent:"# about me\n\n hzau \n\n \n\n  nervgear \n\n  blog \n\n\n# my skill\n\n\n\n\n\n\n\n\n\n\n# contact me\n\n: 1308155474@qq.com\n: https://github.com/zhengtiger/\n\n\n# \n\n tigerz  \n\n",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"1 ",frontmatter:{title:"1 ",date:"2023-05-03T20:16:27.000Z",permalink:"/pages/14b7e3/",categories:[""," - "],tags:[""],readingShow:"top"},regularPath:"/07.%E7%94%9F%E7%89%A9%E7%9F%A5%E8%AF%86/11.%E7%A5%9E%E7%BB%8F%E7%94%9F%E7%89%A9%E5%AD%A6%20-%20%E4%B8%81%E6%96%90/01.%E7%AC%AC1%E7%AB%A0%20%E7%BB%AA%E8%AE%BA.html",relativePath:"07./11. - /01.1 .md",key:"v-5ffa3dac",path:"/pages/14b7e3/",headers:[{level:2,title:"1.1 ",slug:"_1-1-",normalizedTitle:"1.1 ",charIndex:2},{level:3,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:19},{level:3,title:"2. ",slug:"_2-",normalizedTitle:"2. ",charIndex:415},{level:3,title:"3. ",slug:"_3-",normalizedTitle:"3. ",charIndex:751},{level:3,title:"4. ",slug:"_4-",normalizedTitle:"4. ",charIndex:1470},{level:3,title:"5. ",slug:"_5-",normalizedTitle:"5. ",charIndex:2542},{level:2,title:"1.2 ",slug:"_1-2-",normalizedTitle:"1.2 ",charIndex:3195},{level:2,title:"1.3 ",slug:"_1-3-",normalizedTitle:"1.3 ",charIndex:3547},{level:3,title:"1.3.1 ",slug:"_1-3-1-",normalizedTitle:"1.3.1 ",charIndex:3568},{level:4,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:3583},{level:4,title:"2. ",slug:"_2---",normalizedTitle:"2. ",charIndex:4315},{level:4,title:"3. ",slug:"_3---",normalizedTitle:"3. ",charIndex:4731},{level:4,title:"4. ",slug:"_4-",normalizedTitle:"4. ",charIndex:6449},{level:4,title:"5. ",slug:"_5-",normalizedTitle:"5. ",charIndex:7131},{level:4,title:"6. ",slug:"_6-",normalizedTitle:"6. ",charIndex:7988},{level:4,title:"7. ",slug:"_7-",normalizedTitle:"7. ",charIndex:8462},{level:3,title:"1.3.2 ",slug:"_1-3-2-",normalizedTitle:"1.3.2 ",charIndex:9137},{level:4,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:9280},{level:4,title:"2. ",slug:"_2-",normalizedTitle:"2. ",charIndex:10276},{level:4,title:"3. ",slug:"_3-",normalizedTitle:"3. ",charIndex:11195},{level:3,title:"1.3.3 ",slug:"_1-3-3-",normalizedTitle:"1.3.3 ",charIndex:11724},{level:4,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:11844},{level:4,title:"2. ",slug:"_2-",normalizedTitle:"2. ",charIndex:12375},{level:4,title:"3. ",slug:"_3-",normalizedTitle:"3. ",charIndex:12541},{level:4,title:"4. ",slug:"_4-",normalizedTitle:"4. ",charIndex:13123}],excerpt:"",headersStr:"1.1  1.  2.  3.  4.  5.  1.2  1.3  1.3.1  1.  2.  3.  4.  5.  6.  7.  1.3.2  1.  2.  3.  1.3.3  1.  2.  3.  4. ",content:"# 1.1 \n\n\n# 1. \n\nCajal1980\n\n1897Sherrington\n\n1910Sherrington\n\n202050\n\n2090\n\n\n# 2. \n\n1791Galvani\n\n19\n\n2030Young\n\n40HodgkinHuxleyKatzNa+K+K+Na+\n\n\n# 3. \n\n1905Elyot\n\n1921LoewiDale1914(acetyIcholine,Ach)LoewiAch1926DaleLoewiAchAch1932DaleAchAch\n\n(norepinephrine/nonadrenaline,NE)Cannon19341935(sympathin)1946Eule\n\n30(dopamine,DA),NE\n\n2070\n\n\n# 4. \n\n19\n\n1980\n\n2030FoersterPenfield12\n\nSperry204060\n\n1970Caton40\n\n1925Berger()\n\n2050Dawson5060(Fourier)\n\n80(positronemission tomography,PET)(functional magnetic resonance imaging,fMRI)\n\n\n# 5. \n\n(artificial intelligence,AI)AIAIMarvin MinskySeymour PapertJohn McCarthyAI\n\n2080AIAIAIAlphaGoAIAIAI\n\n\n# 1.2 \n\nDNA\n\n\n# 1.3 \n\n\n# 1.3.1 \n\n# 1. \n\n-(haematoxylin and eosin,HE)(nissl body)(silver staining)(nissl staining)\n\n(GolgiGolgi-Cox)(HolmesCajal)(Bielschowsky)(Eager)\n\n1892Nissl( cresyl violet)(thionine)( toluidine blue)()\n\n# 2. \n\n()(histochemistry,cytochemistry)()\n\n()()()()Ach\n\n# 3. \n\n()(immunohistochemistry,immunocytochemistry)()()(chemical neuroanatomy)\n\n()4%(1-1)()()(biotin)(avidin)(14)\n\nA. B. C. D. \n\n(fluorescein isothiocyanate,FYTC)490 nm530 nm(tetramethyl rhodamine isothiocyana te,TRITC)580 nm610 nm( taxes red,TR)550 nm615 nm()\n\n(horseradish peroxidase,HRP)( alkaline phosphatase,AP)HRP(HO)-(3,3'-diaminobenzidine,DAB)AP-()( crossstaining)(DAB)\n\n\n\n# 4. \n\n(tract tracing)2070\n\n# (1) \n\n()(axoplasmic transport)HRP\n\n# (2) \n\n(degeneration tracing)()() DiI(1,1' - dioctadecyl -3,3,3',3'-tetramethylindo-carbocyanine perchlorate)\n\n# 5. \n\n 0.2 um(electronmicroscope) 0.1~0.2 nm(electronmicroscopy)(transmission electron microscope,TEM)(scanning electron microscope,SEM)(l mm3)(50~100 nm)TEMSEM\n\n(cryoelectron microscopy)(Cryo-SEM)(/)(-185)\n\n# 6. \n\n(laser scanning confocal microscope,LSCM)2080LSCMLSCMCTLSCM\n\n# 7. \n\n(morphometry)(planimetry)(stereology)\n\n()(image analyzer)\n\n(clarity)()\n\n\n# 1.3.2 \n\n\n\n# 1. \n\n\n\n# (1) \n\nl) (knee jerk reflex)(righting reflex)(blink reflex)()\n\n2) \n\n3) (operant conditioning reflex)\n\n# (2) \n\nl) (rota-rod test)(beam balance test)( beam walking test)\n\n2) (maze test)(step-down test)(passive avoidance test)\n\n3) ( pain threshold measured)( visual field)(olfactory disorder)\n\n4) (inclined plane test)BBB(Basso-Beattle-Bresnahan locomotorrating score)(measurement of forelimb placing)(bilateral forepawsgrasp)(walking track analysis)\n\n# 2. \n\n\n\n# (1) \n\n\n\n# (2) \n\n 0.5~3.0 mm\n\n# (3) \n\n 150~400 um 5% CO295% O2  Krebs-37  5~15 min  37  2~5 \n\n# (4) \n\n(synaptosome)( G-15 )\n\n# 3. \n\n\n\n# (1) \n\n(microelectrophoresis)() \n\n# (2) \n\n\n\n\n# 1.3.3 \n\n\n\n# 1. \n\n# (1) \n\n(intracellular recording) \n\n# (2) \n\n(extracellular recording)\n\n# 2. \n\n\n\n# 3. \n\n# (1) \n\n(orthodromic impulse)\n\n# (2) \n\n(antidromic impulse)(collision)(forbidden period)\n\n# 4. \n\n# (1) \n\n(voltage clamp)\n\n# (2) \n\n(patch clamp) 1 pA l um  10 us ,-(cell-attached mode)( whole cell recording)(outside-out)( inside-out)",normalizedContent:"# 1.1 \n\n\n# 1. \n\ncajal1980\n\n1897sherrington\n\n1910sherrington\n\n202050\n\n2090\n\n\n# 2. \n\n1791galvani\n\n19\n\n2030young\n\n40hodgkinhuxleykatzna+k+k+na+\n\n\n# 3. \n\n1905elyot\n\n1921loewidale1914(acetyicholine,ach)loewiach1926daleloewiachach1932daleachach\n\n(norepinephrine/nonadrenaline,ne)cannon19341935(sympathin)1946eule\n\n30(dopamine,da),ne\n\n2070\n\n\n# 4. \n\n19\n\n1980\n\n2030foersterpenfield12\n\nsperry204060\n\n1970caton40\n\n1925berger()\n\n2050dawson5060(fourier)\n\n80(positronemission tomography,pet)(functional magnetic resonance imaging,fmri)\n\n\n# 5. \n\n(artificial intelligence,ai)aiaimarvin minskyseymour papertjohn mccarthyai\n\n2080aiaiaialphagoaiaiai\n\n\n# 1.2 \n\ndna\n\n\n# 1.3 \n\n\n# 1.3.1 \n\n# 1. \n\n-(haematoxylin and eosin,he)(nissl body)(silver staining)(nissl staining)\n\n(golgigolgi-cox)(holmescajal)(bielschowsky)(eager)\n\n1892nissl( cresyl violet)(thionine)( toluidine blue)()\n\n# 2. \n\n()(histochemistry,cytochemistry)()\n\n()()()()ach\n\n# 3. \n\n()(immunohistochemistry,immunocytochemistry)()()(chemical neuroanatomy)\n\n()4%(1-1)()()(biotin)(avidin)(14)\n\na. b. c. d. \n\n(fluorescein isothiocyanate,fytc)490 nm530 nm(tetramethyl rhodamine isothiocyana te,tritc)580 nm610 nm( taxes red,tr)550 nm615 nm()\n\n(horseradish peroxidase,hrp)( alkaline phosphatase,ap)hrp(ho)-(3,3'-diaminobenzidine,dab)ap-()( crossstaining)(dab)\n\n\n\n# 4. \n\n(tract tracing)2070\n\n# (1) \n\n()(axoplasmic transport)hrp\n\n# (2) \n\n(degeneration tracing)()() dii(1,1' - dioctadecyl -3,3,3',3'-tetramethylindo-carbocyanine perchlorate)\n\n# 5. \n\n 0.2 um(electronmicroscope) 0.1~0.2 nm(electronmicroscopy)(transmission electron microscope,tem)(scanning electron microscope,sem)(l mm3)(50~100 nm)temsem\n\n(cryoelectron microscopy)(cryo-sem)(/)(-185)\n\n# 6. \n\n(laser scanning confocal microscope,lscm)2080lscmlscmctlscm\n\n# 7. \n\n(morphometry)(planimetry)(stereology)\n\n()(image analyzer)\n\n(clarity)()\n\n\n# 1.3.2 \n\n\n\n# 1. \n\n\n\n# (1) \n\nl) (knee jerk reflex)(righting reflex)(blink reflex)()\n\n2) \n\n3) (operant conditioning reflex)\n\n# (2) \n\nl) (rota-rod test)(beam balance test)( beam walking test)\n\n2) (maze test)(step-down test)(passive avoidance test)\n\n3) ( pain threshold measured)( visual field)(olfactory disorder)\n\n4) (inclined plane test)bbb(basso-beattle-bresnahan locomotorrating score)(measurement of forelimb placing)(bilateral forepawsgrasp)(walking track analysis)\n\n# 2. \n\n\n\n# (1) \n\n\n\n# (2) \n\n 0.5~3.0 mm\n\n# (3) \n\n 150~400 um 5% co295% o2  krebs-37  5~15 min  37  2~5 \n\n# (4) \n\n(synaptosome)( g-15 )\n\n# 3. \n\n\n\n# (1) \n\n(microelectrophoresis)() \n\n# (2) \n\n\n\n\n# 1.3.3 \n\n\n\n# 1. \n\n# (1) \n\n(intracellular recording) \n\n# (2) \n\n(extracellular recording)\n\n# 2. \n\n\n\n# 3. \n\n# (1) \n\n(orthodromic impulse)\n\n# (2) \n\n(antidromic impulse)(collision)(forbidden period)\n\n# 4. \n\n# (1) \n\n(voltage clamp)\n\n# (2) \n\n(patch clamp) 1 pa l um  10 us ,-(cell-attached mode)( whole cell recording)(outside-out)( inside-out)",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"-",frontmatter:{title:"-",date:"2023-04-19T19:53:03.000Z",permalink:"/pages/093f81/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/04.%E6%9C%AC%E7%AB%99-%E6%95%B0%E5%AD%A6%E5%85%AC%E5%BC%8F.html",relativePath:"100./02. - /04.-.md",key:"v-41e3f643",path:"/pages/093f81/",headers:[{level:2,title:" markdown-it-katex",slug:"-markdown-it-katex",normalizedTitle:" markdown-it-katex",charIndex:50},{level:2,title:" markdown-it-katex",slug:"-markdown-it-katex",normalizedTitle:" markdown-it-katex",charIndex:114},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:341},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2}],headersStr:" markdown-it-katex  markdown-it-katex  ",content:" markdown-it-katex  Markdown \n\n\n#  markdown-it-katex\n\nnpm install markdown-it-katex -D\n\n\n1\n\n\n\n#  markdown-it-katex\n\n .vuepress/config.ts \n\nmarkdown: {\n    // markdown-it-katex\n    extendMarkdown: md => {\n      md.set({\n        html: true\n      })\n      md.use(MarkdownItKatex)\n    },\n},\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# \n\n .vuepress/config/head.ts \n\nexport default <HeadTags>[\n  // markdown-it-katex\n  ['link', {\n    rel: 'stylesheet',\n    href: 'https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css'\n  }],\n  ['link', {\n    rel: \"stylesheet\",\n    href: \"https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/2.10.0/github-markdown.min.css\"\n  }]\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\n Markdown  LaTeX \n\n\n\n$$\\begin{array}{c}\n\n\\nabla \\times \\vec{\\mathbf{B}} -\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{E}}}{\\partial t} &\n= \\frac{4\\pi}{c}\\vec{\\mathbf{j}}    \\nabla \\cdot \\vec{\\mathbf{E}} & = 4 \\pi \\rho \\\\\n\n\\nabla \\times \\vec{\\mathbf{E}}\\, +\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{B}}}{\\partial t} & = \\vec{\\mathbf{0}} \\\\\n\n\\nabla \\cdot \\vec{\\mathbf{B}} & = 0\n\n\\end{array}$$\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n\nB1cEt=4cjE=4E+1cBt=0B=0\\begin{array}{c} \\nabla \\times \\vec{\\mathbf{B}} -\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{E}}}{\\partial t} & = \\frac{4\\pi}{c}\\vec{\\mathbf{j}} \\nabla \\cdot \\vec{\\mathbf{E}} & = 4 \\pi \\rho \\\\ \\nabla \\times \\vec{\\mathbf{E}}\\, +\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{B}}}{\\partial t} & = \\vec{\\mathbf{0}} \\\\ \\nabla \\cdot \\vec{\\mathbf{B}} & = 0 \\end{array}  B   c 1 t  E   E  + c 1 t  B   B  = c 4 j   E  = 0  =0 =4",normalizedContent:" markdown-it-katex  markdown \n\n\n#  markdown-it-katex\n\nnpm install markdown-it-katex -d\n\n\n1\n\n\n\n#  markdown-it-katex\n\n .vuepress/config.ts \n\nmarkdown: {\n    // markdown-it-katex\n    extendmarkdown: md => {\n      md.set({\n        html: true\n      })\n      md.use(markdownitkatex)\n    },\n},\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# \n\n .vuepress/config/head.ts \n\nexport default <headtags>[\n  // markdown-it-katex\n  ['link', {\n    rel: 'stylesheet',\n    href: 'https://cdnjs.cloudflare.com/ajax/libs/katex/0.7.1/katex.min.css'\n  }],\n  ['link', {\n    rel: \"stylesheet\",\n    href: \"https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/2.10.0/github-markdown.min.css\"\n  }]\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\n markdown  latex \n\n\n\n$$\\begin{array}{c}\n\n\\nabla \\times \\vec{\\mathbf{b}} -\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{e}}}{\\partial t} &\n= \\frac{4\\pi}{c}\\vec{\\mathbf{j}}    \\nabla \\cdot \\vec{\\mathbf{e}} & = 4 \\pi \\rho \\\\\n\n\\nabla \\times \\vec{\\mathbf{e}}\\, +\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{b}}}{\\partial t} & = \\vec{\\mathbf{0}} \\\\\n\n\\nabla \\cdot \\vec{\\mathbf{b}} & = 0\n\n\\end{array}$$\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n\nb1cet=4cje=4e+1cbt=0b=0\\begin{array}{c} \\nabla \\times \\vec{\\mathbf{b}} -\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{e}}}{\\partial t} & = \\frac{4\\pi}{c}\\vec{\\mathbf{j}} \\nabla \\cdot \\vec{\\mathbf{e}} & = 4 \\pi \\rho \\\\ \\nabla \\times \\vec{\\mathbf{e}}\\, +\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{b}}}{\\partial t} & = \\vec{\\mathbf{0}} \\\\ \\nabla \\cdot \\vec{\\mathbf{b}} & = 0 \\end{array}  b   c 1 t  e   e  + c 1 t  b   b  = c 4 j   e  = 0  =0 =4",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"-",frontmatter:{title:"-",date:"2023-04-18T08:28:49.000Z",permalink:"/pages/94f5fc/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/03.%E6%9C%AC%E7%AB%99-%E8%AF%84%E8%AE%BA%E6%A0%8F.html",relativePath:"100./02. - /03.-.md",key:"v-16897589",path:"/pages/94f5fc/",headers:[{level:2,title:"Vercel ",slug:"vercel-",normalizedTitle:"vercel ",charIndex:155},{level:2,title:"Vdoing ",slug:"vdoing-",normalizedTitle:"vdoing ",charIndex:685},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:965},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1220},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1257}],headersStr:"Vercel  Vdoing    ",content:"Twikoo:\nhttps://twikoo.js.org/\nhttps://notes.youngkbt.cn/about/website/comment/#twikoo-\nhttps://www.bilibili.com/video/BV1Fh411e7ZH\n\n\n# Vercel \n\n 1.  MongoDB \n 2.  MongoDB  AWS / N. Virginia (us-east-1)\n 3.  Clusters  CONNECT IP  <password> \n 4.  Vercel \n 5.  Twikoo  Vercel\n 6.  Settings - Environment Variables MONGODB_URI 3 \n 7.  Deployments ,  , Redeploy , Redeploy\n 8.  Overview Domains  Twikoo  \n 9. Vercel Domains https://  https://xxx.vercel.app id\n\n\n# Vdoing \n\n .vuepress/components/Twikoo.vue  67  evnId  Vercel  Twikoo  Vercel Domains https://xxx.vercel.app\n\n https://xxx.vercel.app  Vercel  \n\n frontmatter  comment  false\n\n\n# \n\n Twikoo    MongoDB \n\n\n\n * \n * \n * \n * \n\n\n# \n\n \n\n\n# \n\nAkismet (Automattic Kismet)  WordPress  Matt MullenwegAkismet  WordPress \n\n 1.  akismet.com\n 2.  Akismet Personal  Akismet API Key Twikoo \n\n Akismet \n\n viagra-test-123  akismet-guaranteed-spam@example.com \n\n Akismet  6 Twikoo   ",normalizedContent:"twikoo:\nhttps://twikoo.js.org/\nhttps://notes.youngkbt.cn/about/website/comment/#twikoo-\nhttps://www.bilibili.com/video/bv1fh411e7zh\n\n\n# vercel \n\n 1.  mongodb \n 2.  mongodb  aws / n. virginia (us-east-1)\n 3.  clusters  connect ip  <password> \n 4.  vercel \n 5.  twikoo  vercel\n 6.  settings - environment variables mongodb_uri 3 \n 7.  deployments ,  , redeploy , redeploy\n 8.  overview domains  twikoo  \n 9. vercel domains https://  https://xxx.vercel.app id\n\n\n# vdoing \n\n .vuepress/components/twikoo.vue  67  evnid  vercel  twikoo  vercel domains https://xxx.vercel.app\n\n https://xxx.vercel.app  vercel  \n\n frontmatter  comment  false\n\n\n# \n\n twikoo    mongodb \n\n\n\n * \n * \n * \n * \n\n\n# \n\n \n\n\n# \n\nakismet (automattic kismet)  wordpress  matt mullenwegakismet  wordpress \n\n 1.  akismet.com\n 2.  akismet personal  akismet api key twikoo \n\n akismet \n\n viagra-test-123  akismet-guaranteed-spam@example.com \n\n akismet  6 twikoo   ",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"-",frontmatter:{title:"-",date:"2023-04-16T15:07:33.000Z",permalink:"/pages/d3830b/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/01.%E6%9C%AC%E7%AB%99-%E5%9F%BA%E6%9C%AC%E9%85%8D%E7%BD%AE.html",relativePath:"100./02. - /01.-.md",key:"v-623ea45e",path:"/pages/d3830b/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1344},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1547},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2105}],headersStr:"   ",content:"# \n\n .vuepress/common/info.ts \n\n//  penName  title docs/index  4 \n//  | \nconst penName: string = 'TigerZ ';\n//  penName\nconst title: string = 'Blog';\n// \nconst link: string = \"https://zhenghu159.github.io/\";\n//  | \nconst footerTitle: string = 'Blog';\n// \nconst avatar: string = 'img/avatar.jpg'\n// \nconst name: string = 'TigerZ'\n// \nconst slogan: string = '';\n\nexport {\n  penName,\n  title,\n  link,\n  footerTitle,\n  avatar,\n  name,\n  slogan\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n\n\n .vuepress/config/themeConfig.ts \n\nimport { penName, link, avatar, name, slogan } from '../common/info';\n\nexport default <VdoingThemeConfig>{\n  // md String | {name: String, link: String}\n  author: {\n    name: penName, // \n    link, // \n  },\n  // \n  blogger: {\n    avatar,\n    name,\n    slogan,\n  },\n  // \n  social: {\n    icons: [\n      {\n        iconClass: 'icon-youjian',\n        title: '',\n        link: 'mailto:1308155474@qq.com',\n      },\n      {\n        iconClass: \"icon-github\",\n        title: \"GitHub\",\n        link: \"https://github.com/zhenghu159\",\n      },\n    ],\n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n\n\n\n# \n\n .vuepress/config/themeConfig.ts \n\nexport default <VdoingThemeConfig>{\n  // \n  contentBgStyle: 1, // , 1 =>  | 2 =>  | 3 =>  | 4 =>  | 5 =>  | 6 => \n}\n\n\n1\n2\n3\n4\n\n\n\n# \n\n .vuepress/common/footer.ts \n\nimport { penName, footerTitle } from '../common/info'\n\ninterface Footer {\n  createYear: number, // \n  copyrightInfo: string //  a \n}\n\nexport default <Footer>{\n  // \n  createYear: 2022,\n  copyrightInfo:\n    penName + ' | ' + footerTitle + '<br> <a href=\"http://beian.miit.gov.cn/\" target=\"_blank\"></a>',\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n .vuepress/config/themeConfig.ts \n\nimport footer from '../common/footer';\n\nexport default <VdoingThemeConfig>{\n  // \n  footer,\n}\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n# \n\n .vuepress/config/themeConfig.ts \n\nimport { readFileList, readTotalFileWords, readEachFileWords } from '../webSiteInfo/readFile';\n\nexport default <VdoingThemeConfig>{\n  //  & \n  blogInfo: {\n    blogCreate: \"2022\", // \n    indexView: true, //  true\n    pageView: true, //  true\n    readingTime: true, //  eachFileWords true eachFileWords  readEachFileWords  1  300 160 \n    eachFileWords: readEachFileWords([\"\"], 300, 160), // readEachFileWords(['xx'])  xx  1 readEachFileWords()  article  false \n    mdFileCountType: \"archives\", // 1. archives 2.  readFileList(['xx'])  xx readFileList()  docs  md  `.vuepress`  `@pages` \n    totalWords: \"archives\", // 1. archives  archives  eachFileWords2. readTotalFileWords(['xx'])  xx \n    moutedEvent: \".tags-wrapper\", //  '.tags-wrapper' '.categories-wrapper' '.blogger-wrapper' \n    // \n    indexIteration: 2500, //  5  3  + 2+ 3 ......\n    pageIteration: 2500, //  5  3  + 2+ 3 ......\n    //  + 1 100  + 3\n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n",normalizedContent:"# \n\n .vuepress/common/info.ts \n\n//  penname  title docs/index  4 \n//  | \nconst penname: string = 'tigerz ';\n//  penname\nconst title: string = 'blog';\n// \nconst link: string = \"https://zhenghu159.github.io/\";\n//  | \nconst footertitle: string = 'blog';\n// \nconst avatar: string = 'img/avatar.jpg'\n// \nconst name: string = 'tigerz'\n// \nconst slogan: string = '';\n\nexport {\n  penname,\n  title,\n  link,\n  footertitle,\n  avatar,\n  name,\n  slogan\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n\n\n .vuepress/config/themeconfig.ts \n\nimport { penname, link, avatar, name, slogan } from '../common/info';\n\nexport default <vdoingthemeconfig>{\n  // md string | {name: string, link: string}\n  author: {\n    name: penname, // \n    link, // \n  },\n  // \n  blogger: {\n    avatar,\n    name,\n    slogan,\n  },\n  // \n  social: {\n    icons: [\n      {\n        iconclass: 'icon-youjian',\n        title: '',\n        link: 'mailto:1308155474@qq.com',\n      },\n      {\n        iconclass: \"icon-github\",\n        title: \"github\",\n        link: \"https://github.com/zhenghu159\",\n      },\n    ],\n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n\n\n\n# \n\n .vuepress/config/themeconfig.ts \n\nexport default <vdoingthemeconfig>{\n  // \n  contentbgstyle: 1, // , 1 =>  | 2 =>  | 3 =>  | 4 =>  | 5 =>  | 6 => \n}\n\n\n1\n2\n3\n4\n\n\n\n# \n\n .vuepress/common/footer.ts \n\nimport { penname, footertitle } from '../common/info'\n\ninterface footer {\n  createyear: number, // \n  copyrightinfo: string //  a \n}\n\nexport default <footer>{\n  // \n  createyear: 2022,\n  copyrightinfo:\n    penname + ' | ' + footertitle + '<br> <a href=\"http://beian.miit.gov.cn/\" target=\"_blank\"></a>',\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n .vuepress/config/themeconfig.ts \n\nimport footer from '../common/footer';\n\nexport default <vdoingthemeconfig>{\n  // \n  footer,\n}\n\n\n1\n2\n3\n4\n5\n6\n\n\n\n# \n\n .vuepress/config/themeconfig.ts \n\nimport { readfilelist, readtotalfilewords, readeachfilewords } from '../websiteinfo/readfile';\n\nexport default <vdoingthemeconfig>{\n  //  & \n  bloginfo: {\n    blogcreate: \"2022\", // \n    indexview: true, //  true\n    pageview: true, //  true\n    readingtime: true, //  eachfilewords true eachfilewords  readeachfilewords  1  300 160 \n    eachfilewords: readeachfilewords([\"\"], 300, 160), // readeachfilewords(['xx'])  xx  1 readeachfilewords()  article  false \n    mdfilecounttype: \"archives\", // 1. archives 2.  readfilelist(['xx'])  xx readfilelist()  docs  md  `.vuepress`  `@pages` \n    totalwords: \"archives\", // 1. archives  archives  eachfilewords2. readtotalfilewords(['xx'])  xx \n    moutedevent: \".tags-wrapper\", //  '.tags-wrapper' '.categories-wrapper' '.blogger-wrapper' \n    // \n    indexiteration: 2500, //  5  3  + 2+ 3 ......\n    pageiteration: 2500, //  5  3  + 2+ 3 ......\n    //  + 1 100  + 3\n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"-",frontmatter:{title:"-",date:"2023-04-16T15:39:00.000Z",permalink:"/pages/e6cfba/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/02.%E6%9C%AC%E7%AB%99-%E9%A6%96%E9%A1%B5%E5%A4%A7%E5%9B%BE.html",relativePath:"100./02. - /02.-.md",key:"v-6406ed3f",path:"/pages/e6cfba/",headers:[{level:2,title:"index.md ",slug:"index-md-",normalizedTitle:"index.md ",charIndex:67},{level:2,title:"themeConfig.ts ",slug:"themeconfig-ts-",normalizedTitle:"themeconfig.ts ",charIndex:322},{level:2,title:"IndexBigImag.vue ",slug:"indexbigimag-vue-",normalizedTitle:"indexbigimag.vue ",charIndex:1897}],headersStr:"index.md  themeConfig.ts  IndexBigImag.vue ",content:': https://notes.youngkbt.cn/about/website/index-big-img/\n\n\n# index.md \n\n .vuepress/index.md \n\n---\nhome: true\nheroText: TigerZ  \ntagline:  --- TigerZ\npageClass: vdoing-index-class\nbannerBg: \'img/bannerBg.gif\'\n---\n\n<ClientOnly>\n  <IndexBigImg />\n</ClientOnly>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# themeConfig.ts \n\n .vuepress/config/themeConfig.ts \n\nexport default <VdoingThemeConfig>{\n  // body\n  bodyBgImg: ["/img/bodyBgImg/DNF1.jpg"], // bodyStringArray\n  bodyBgImgOpacity: 1, // body\n  bodyBgImgInterval: 15, // body\n\n  // \n  indexImg: {\n    navColor: 2, // 1 2  1\n    switchNavColor: true, // navColor  false\n    //  navColor  2(true) +  = \n    bgTimeColor: true, //  false\n    bgTimeColorArray: [\n      "transparent",\n      "transparent",\n      "transparent",\n      "transparent",\n    ], // bgTimeColor  true \n    descFade: true, //  false\n    desc: [\n      " --- TigerZ"\n    ], //  index.md  tagline index.md  taglinedescFadeIn  true \n    descFontSize: "1.4rem", // desc  1.4rem 1.1rem\n    descFadeInTime: 200, // descFade  true  200 \n    descFadeOutTime: 100, // descFade  true  100 \n    descNextTime: 800, //  desc  desc  desc 800 \n    bubble: false, //  false\n    bubblePosition: 0, // 0-100050bubble  true  0\n    bubbleNum: 200, // bubble  true  200 \n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n\n\n\n# IndexBigImag.vue \n\n\n\n .vuepress/components/IndexBigImag.vue \n\n/*  */\n.banner-conent {\n  margin-top: 14vh !important;\n}\n\n\n1\n2\n3\n4\n',normalizedContent:': https://notes.youngkbt.cn/about/website/index-big-img/\n\n\n# index.md \n\n .vuepress/index.md \n\n---\nhome: true\nherotext: tigerz  \ntagline:  --- tigerz\npageclass: vdoing-index-class\nbannerbg: \'img/bannerbg.gif\'\n---\n\n<clientonly>\n  <indexbigimg />\n</clientonly>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# themeconfig.ts \n\n .vuepress/config/themeconfig.ts \n\nexport default <vdoingthemeconfig>{\n  // body\n  bodybgimg: ["/img/bodybgimg/dnf1.jpg"], // bodystringarray\n  bodybgimgopacity: 1, // body\n  bodybgimginterval: 15, // body\n\n  // \n  indeximg: {\n    navcolor: 2, // 1 2  1\n    switchnavcolor: true, // navcolor  false\n    //  navcolor  2(true) +  = \n    bgtimecolor: true, //  false\n    bgtimecolorarray: [\n      "transparent",\n      "transparent",\n      "transparent",\n      "transparent",\n    ], // bgtimecolor  true \n    descfade: true, //  false\n    desc: [\n      " --- tigerz"\n    ], //  index.md  tagline index.md  taglinedescfadein  true \n    descfontsize: "1.4rem", // desc  1.4rem 1.1rem\n    descfadeintime: 200, // descfade  true  200 \n    descfadeouttime: 100, // descfade  true  100 \n    descnexttime: 800, //  desc  desc  desc 800 \n    bubble: false, //  false\n    bubbleposition: 0, // 0-100050bubble  true  0\n    bubblenum: 200, // bubble  true  200 \n  },\n}\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n\n\n\n# indexbigimag.vue \n\n\n\n .vuepress/components/indexbigimag.vue \n\n/*  */\n.banner-conent {\n  margin-top: 14vh !important;\n}\n\n\n1\n2\n3\n4\n',charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"-",frontmatter:{title:"-",date:"2023-04-20T21:41:32.000Z",permalink:"/pages/25bbfb/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/07.%E6%9C%AC%E7%AB%99-%E5%9B%BE%E5%BA%8A.html",relativePath:"100./02. - /07.-.md",key:"v-4a93e361",path:"/pages/25bbfb/",headers:[{level:2,title:"?",slug:"",normalizedTitle:"?",charIndex:36},{level:2,title:"PicX ",slug:"picx-",normalizedTitle:"picx ",charIndex:338},{level:3,title:" GitHub Token",slug:"-github-token",normalizedTitle:" github token",charIndex:350},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:587},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:145}],headersStr:"? PicX   GitHub Token  ",content:" PicX  PicX\n\n\n# ?\n\nURLmarkdown+url~\n\nPicX  GitHub API  jsDelivr  CDN \n\n\n# PicX \n\n\n#  GitHub Token\n\nPicX  GitHub Token repo  GitHub Token\n\n https://github.com/settings/tokens/new  GitHub Token\n\n\n\n Generate token  GitHub Token\n\n\n\n\n\n\n\n Token \n\n\n# \n\n\n\n GitHub Token   GitHub \n\n * picx-images-hosting\n * master\n * yyyyMMdd (20230403)\n\n PicX\n\n\n\n\n# \n\nPicX \n\n\n\n",normalizedContent:" picx  picx\n\n\n# ?\n\nurlmarkdown+url~\n\npicx  github api  jsdelivr  cdn \n\n\n# picx \n\n\n#  github token\n\npicx  github token repo  github token\n\n https://github.com/settings/tokens/new  github token\n\n\n\n generate token  github token\n\n\n\n\n\n\n\n token \n\n\n# \n\n\n\n github token   github \n\n * picx-images-hosting\n * master\n * yyyymmdd (20230403)\n\n picx\n\n\n\n\n# \n\npicx \n\n\n\n",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"-Markdown",frontmatter:{title:"-Markdown",date:"2023-04-19T20:35:21.000Z",permalink:"/pages/50efc7/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/05.%E6%9C%AC%E7%AB%99-Markdown%E6%8B%93%E5%B1%95.html",relativePath:"100./02. - /05.-Markdown.md",key:"v-7c712705",path:"/pages/50efc7/",headers:[{level:2,title:"GitHub ",slug:"github-",normalizedTitle:"github ",charIndex:104},{level:2,title:"Emoji",slug:"emoji",normalizedTitle:"emoji",charIndex:500},{level:2,title:"Markdown ",slug:"markdown-",normalizedTitle:"markdown ",charIndex:761},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:777},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1086},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1113},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:1697},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:4379},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:6454},{level:4,title:"1. ",slug:"_1-",normalizedTitle:"1. ",charIndex:6516},{level:4,title:"2. ",slug:"_2-",normalizedTitle:"2. ",charIndex:6597},{level:4,title:"3. ",slug:"_3-",normalizedTitle:"3. ",charIndex:6862},{level:2,title:"Markdown ",slug:"markdown-",normalizedTitle:"markdown ",charIndex:7692},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:15},{level:4,title:"",slug:"",normalizedTitle:"",charIndex:7929},{level:3,title:"",slug:"",normalizedTitle:"",charIndex:8127},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:8130}],headersStr:"GitHub  Emoji Markdown        1.  2.  3.  Markdown     ",content:"Markdown Markdown .mdVuepress  VdoingMarkdown\n\n\n# GitHub \n\n\n\n| Tables        | Are           | Cool  |\n| ------------- |:-------------:| -----:|\n| col 3 is      | right-aligned | $1600 |\n| col 2 is      | centered      |   $12 |\n| zebra stripes | are neat      |    $1 |\n\n\n1\n2\n3\n4\n5\n\n\n\n\nTABLES          ARE             COOL\ncol 3 is        right-aligned   $1600\ncol 2 is        centered        $12\nzebra stripes   are neat        $1\n\n\n# Emoji\n\n\n\n:tada: :100:\n\n\n1\n\n\n\n\n \n\n Emoji \n\n * https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json\n * https://www.webfx.com/tools/emoji-cheat-sheet\n * https://emojipedia.org/\n * https://github.com/carloscuesta/gitmoji\n\n\n# Markdown \n\n\n# \n\n\n\n::: tip\n\n:::\n\n::: warning\n\n:::\n\n::: danger\n\n:::\n\n::: note\n <Badge text=\"v1.5.0 +\" /> ~\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n ~\n\n\n\n::: tip \n\n:::\n\n\n1\n2\n3\n\n\n\n\n\n\n\n# \n\n\n\n::: center\n  ### \n  \n:::\n\n::: right\n  [](https://zh.wikipedia.org/wiki/%E7%89%9B%E9%A1%BF%E8%BF%90%E5%8A%A8%E5%AE%9A%E5%BE%8B)\n:::\n\n::: details\n IE / Edge \n```js\nconsole.log('')\n```\n:::\n\n::: theorem \n\n::: right\n [](https://zh.wikipedia.org/wiki/%E7%89%9B%E9%A1%BF%E8%BF%90%E5%8A%A8%E5%AE%9A%E5%BE%8B)\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n\n\n# \n\n\n\n\n\n\n\n IE / Edge \n\nconsole.log('')\n\n\n1\n\n\n\n\n\n\n \n\n\n# \n\n\n\n\n\n::: cardList\n```yaml\n- name: \n  desc: \n  avatar: https://fastly.jsdelivr.net/gh/xugaoyi/image_store/blog/20200122153807.jpg # \n  link: https://www.cnblogs.com/miluluyo/ # \n  bgColor: '#CBEAFA' # var(--bodyBg)#\n  textColor: '#6854A1' # var(--textColor)\n- name: XAOXUU\n  desc: '#IOS #Volantis'\n  avatar: https://fastly.jsdelivr.net/gh/xaoxuu/assets@master/avatar/avatar.png\n  link: https://xaoxuu.com\n  bgColor: '#718971'\n  textColor: '#fff'\n- name: \n  desc: \n  avatar: https://reinness.com/avatar.png\n  link: https://reinness.com\n  bgColor: '#FCDBA0'\n  textColor: '#A05F2C'\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n\n\n\n\n\n\n\n\nXAOXUU\n\n#IOS #Volantis\n\n\n\n\n\n- name: \n  desc: \n  avatar: https://fastly.jsdelivr.net/gh/xugaoyi/image_store/blog/20200122153807.jpg # \n  link: https://www.cnblogs.com/miluluyo/ # \n  bgColor: '#CBEAFA' # var(--bodyBg)#\n  textColor: '#6854A1' # var(--textColor)\n- name: XAOXUU\n  desc: '#IOS #Volantis'\n  avatar: https://fastly.jsdelivr.net/gh/xaoxuu/assets@master/avatar/avatar.png\n  link: https://xaoxuu.com\n  bgColor: '#718971'\n  textColor: '#fff'\n- name: \n  desc: \n  avatar: https://reinness.com/avatar.png\n  link: https://reinness.com\n  bgColor: '#FCDBA0'\n  textColor: '#A05F2C'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n\n\n\n\n::: cardList <>\n``` yaml\n- name: \n  desc: \n  avatar: https://xxx.jpg # \n  link: https://xxx/ # \n  bgColor: '#CBEAFA' # var(--bodyBg)#\n  textColor: '#6854A1' # var(--textColor)\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n * <> 1~43\n * yaml\n * yaml\n * \n   * name\n   * desc\n   * avatar\n   * link\n   * bgColorvar(--bodyBg)#\n   * textColorvar(--textColor)\n\n\n\n\n\n::: cardList 2\n```yaml\n- name: \n  desc: \n  bgColor: '#F0DFB1'\n  textColor: '#242A38'\n- name: Vdoing\n  desc: VuePress &(blog) \n  link: https://github.com/xugaoyi/vuepress-theme-vdoing\n  bgColor: '#DFEEE7'\n  textColor: '#2A3344'\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n\n\n\n\n\nVdoing\n\nVuePress &(blog) \n\n- name: \n  desc: \n  bgColor: '#F0DFB1'\n  textColor: '#242A38'\n- name: Vdoing\n  desc: VuePress &(blog) \n  link: https://github.com/xugaoyi/vuepress-theme-vdoing\n  bgColor: '#DFEEE7'\n  textColor: '#2A3344'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# \n\n\n\n\n\n::: cardImgList\n```yaml\n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n\n\n\n\n\n\n\n\nTigerZ\n\n\n\n\n\nTigerZ\n\n\n\n\n\nTigerZ\n\n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: TigerZ # \n  avatar: https://tigerz.online/img/avatar.jpg # \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n\n\n\n\n::: cardImgList <>\n``` yaml\n- img: https://xxx.jpg # \n  link: https://xxx.com # \n  name: \n  desc:  # \n  author:  # \n  avatar: https://xxx.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n * <> 1~43\n * yaml\n * yaml\n * \n   * img\n   * link\n   * name\n   * desc\n   * author\n   * avatar\n\n\n# \n\nVdoing  v1.9.0+ \n\n# 1. \n\ntarget\n\n * _blank\n   * _self \n   * _blank \n\n# 2. \n\nimgHeight\n\n *  auto\n   * \n\nobjectFit\n\n * (object-fit) cover\n   * fill  ()\n   * contain  ()\n   * cover  ()\n   * none  ()\n   * scale-down  ()\n\nlineClamp\n\n * 1\n\n# 3. \n\n\n\n::: cardImgList\n```yaml\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 3\n\ndata:\n  - img: https://tigerz.online/img/avatar.jpg\n    link: https://zhenghu159.github.io/\n    name: \n    desc:  # \n    author: TigerZ # \n    avatar: https://tigerz.online/img/avatar.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n\n\n\n\n\n\n\n\nTigerZ\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 3\n\ndata:\n  - img: https://tigerz.online/img/avatar.jpg\n    link: https://zhenghu159.github.io/\n    name: \n    desc:  # \n    author: TigerZ # \n    avatar: https://tigerz.online/img/avatar.jpg # \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n# Markdown \n\nMarkdownhtml\n\n\n# \n\n * Props:\n   \n   * text - string\n   * type - string,  tip | warning | error tip\n   * vertical - string,  top | middle top\n\n * Usage:\n\n\n\n####  <Badge text=\"\"/>\n<Badge text=\"\" type=\"warning\"/>\n\n> <Badge text=\"\" type=\"error\" vertical=\"middle\"/>: \n\n\n1\n2\n3\n4\n\n * \n\n# \n\n\n\n> : \n\n\n# \n\n<code-group><code-block><code-block>titletabactivetab\n\n<code-group>\n  <code-block title=\"YARN\" active>\n  ```bash\n  yarn add vuepress-theme-vdoing -D\n  ```\n  </code-block>\n\n  <code-block title=\"NPM\">\n  ```bash\n  npm install vuepress-theme-vdoing -D\n  ```\n  </code-block>\n</code-group>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n\nyarn add vuepress-theme-vdoing -D\n\n\n1\n\n\nnpm install vuepress-theme-vdoing -D\n\n\n1\n\n\n\n# \n\nvuepress-plugin-tabs \n\n\n\nnpm install vuepress-plugin-tabs vue-tabs-component\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <UserPlugins>[\n  // vuepress-plugin-tabshttps://github.com/pskordilakis/vuepress-plugin-tabs\n  ['tabs'],\n]\n\n\n1\n2\n3\n4\n\n\n .vuepress/style/index.styl \n\n@require '~vuepress-plugin-tabs/dist/themes/default.styl'\n\n\n1\n\n\n\n\n:::: tabs\n\n::: tab title\n__markdown content__\n:::\n\n\n::: tab javascript\n``` javascript\n() => {\n  console.log('Javascript code example')\n}\n```\n:::\n\n::::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n\n\n\n\nmarkdown content\n\n() => {\n  console.log('Javascript code example')\n}\n\n\n1\n2\n3\n",normalizedContent:"markdown markdown .mdvuepress  vdoingmarkdown\n\n\n# github \n\n\n\n| tables        | are           | cool  |\n| ------------- |:-------------:| -----:|\n| col 3 is      | right-aligned | $1600 |\n| col 2 is      | centered      |   $12 |\n| zebra stripes | are neat      |    $1 |\n\n\n1\n2\n3\n4\n5\n\n\n\n\ntables          are             cool\ncol 3 is        right-aligned   $1600\ncol 2 is        centered        $12\nzebra stripes   are neat        $1\n\n\n# emoji\n\n\n\n:tada: :100:\n\n\n1\n\n\n\n\n \n\n emoji \n\n * https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json\n * https://www.webfx.com/tools/emoji-cheat-sheet\n * https://emojipedia.org/\n * https://github.com/carloscuesta/gitmoji\n\n\n# markdown \n\n\n# \n\n\n\n::: tip\n\n:::\n\n::: warning\n\n:::\n\n::: danger\n\n:::\n\n::: note\n <badge text=\"v1.5.0 +\" /> ~\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n ~\n\n\n\n::: tip \n\n:::\n\n\n1\n2\n3\n\n\n\n\n\n\n\n# \n\n\n\n::: center\n  ### \n  \n:::\n\n::: right\n  [](https://zh.wikipedia.org/wiki/%e7%89%9b%e9%a1%bf%e8%bf%90%e5%8a%a8%e5%ae%9a%e5%be%8b)\n:::\n\n::: details\n ie / edge \n```js\nconsole.log('')\n```\n:::\n\n::: theorem \n\n::: right\n [](https://zh.wikipedia.org/wiki/%e7%89%9b%e9%a1%bf%e8%bf%90%e5%8a%a8%e5%ae%9a%e5%be%8b)\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n\n\n\n\n\n# \n\n\n\n\n\n\n\n ie / edge \n\nconsole.log('')\n\n\n1\n\n\n\n\n\n\n \n\n\n# \n\n\n\n\n\n::: cardlist\n```yaml\n- name: \n  desc: \n  avatar: https://fastly.jsdelivr.net/gh/xugaoyi/image_store/blog/20200122153807.jpg # \n  link: https://www.cnblogs.com/miluluyo/ # \n  bgcolor: '#cbeafa' # var(--bodybg)#\n  textcolor: '#6854a1' # var(--textcolor)\n- name: xaoxuu\n  desc: '#ios #volantis'\n  avatar: https://fastly.jsdelivr.net/gh/xaoxuu/assets@master/avatar/avatar.png\n  link: https://xaoxuu.com\n  bgcolor: '#718971'\n  textcolor: '#fff'\n- name: \n  desc: \n  avatar: https://reinness.com/avatar.png\n  link: https://reinness.com\n  bgcolor: '#fcdba0'\n  textcolor: '#a05f2c'\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n\n\n\n\n\n\n\n\nxaoxuu\n\n#ios #volantis\n\n\n\n\n\n- name: \n  desc: \n  avatar: https://fastly.jsdelivr.net/gh/xugaoyi/image_store/blog/20200122153807.jpg # \n  link: https://www.cnblogs.com/miluluyo/ # \n  bgcolor: '#cbeafa' # var(--bodybg)#\n  textcolor: '#6854a1' # var(--textcolor)\n- name: xaoxuu\n  desc: '#ios #volantis'\n  avatar: https://fastly.jsdelivr.net/gh/xaoxuu/assets@master/avatar/avatar.png\n  link: https://xaoxuu.com\n  bgcolor: '#718971'\n  textcolor: '#fff'\n- name: \n  desc: \n  avatar: https://reinness.com/avatar.png\n  link: https://reinness.com\n  bgcolor: '#fcdba0'\n  textcolor: '#a05f2c'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n\n\n\n\n::: cardlist <>\n``` yaml\n- name: \n  desc: \n  avatar: https://xxx.jpg # \n  link: https://xxx/ # \n  bgcolor: '#cbeafa' # var(--bodybg)#\n  textcolor: '#6854a1' # var(--textcolor)\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n * <> 1~43\n * yaml\n * yaml\n * \n   * name\n   * desc\n   * avatar\n   * link\n   * bgcolorvar(--bodybg)#\n   * textcolorvar(--textcolor)\n\n\n\n\n\n::: cardlist 2\n```yaml\n- name: \n  desc: \n  bgcolor: '#f0dfb1'\n  textcolor: '#242a38'\n- name: vdoing\n  desc: vuepress &(blog) \n  link: https://github.com/xugaoyi/vuepress-theme-vdoing\n  bgcolor: '#dfeee7'\n  textcolor: '#2a3344'\n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n\n\n\n\n\nvdoing\n\nvuepress &(blog) \n\n- name: \n  desc: \n  bgcolor: '#f0dfb1'\n  textcolor: '#242a38'\n- name: vdoing\n  desc: vuepress &(blog) \n  link: https://github.com/xugaoyi/vuepress-theme-vdoing\n  bgcolor: '#dfeee7'\n  textcolor: '#2a3344'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n\n\n\n# \n\n\n\n\n\n::: cardimglist\n```yaml\n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n\n\n\n\n\n\n\n\ntigerz\n\n\n\n\n\ntigerz\n\n\n\n\n\ntigerz\n\n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n- img: https://tigerz.online/img/avatar.jpg\n  link: https://zhenghu159.github.io/\n  name: \n  desc:  # \n  author: tigerz # \n  avatar: https://tigerz.online/img/avatar.jpg # \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n\n\n\n\n::: cardimglist <>\n``` yaml\n- img: https://xxx.jpg # \n  link: https://xxx.com # \n  name: \n  desc:  # \n  author:  # \n  avatar: https://xxx.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n\n * <> 1~43\n * yaml\n * yaml\n * \n   * img\n   * link\n   * name\n   * desc\n   * author\n   * avatar\n\n\n# \n\nvdoing  v1.9.0+ \n\n# 1. \n\ntarget\n\n * _blank\n   * _self \n   * _blank \n\n# 2. \n\nimgheight\n\n *  auto\n   * \n\nobjectfit\n\n * (object-fit) cover\n   * fill  ()\n   * contain  ()\n   * cover  ()\n   * none  ()\n   * scale-down  ()\n\nlineclamp\n\n * 1\n\n# 3. \n\n\n\n::: cardimglist\n```yaml\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 3\n\ndata:\n  - img: https://tigerz.online/img/avatar.jpg\n    link: https://zhenghu159.github.io/\n    name: \n    desc:  # \n    author: tigerz # \n    avatar: https://tigerz.online/img/avatar.jpg # \n```\n:::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n\n\n\n\n\n\n\n\ntigerz\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 3\n\ndata:\n  - img: https://tigerz.online/img/avatar.jpg\n    link: https://zhenghu159.github.io/\n    name: \n    desc:  # \n    author: tigerz # \n    avatar: https://tigerz.online/img/avatar.jpg # \n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n# markdown \n\nmarkdownhtml\n\n\n# \n\n * props:\n   \n   * text - string\n   * type - string,  tip | warning | error tip\n   * vertical - string,  top | middle top\n\n * usage:\n\n\n\n####  <badge text=\"\"/>\n<badge text=\"\" type=\"warning\"/>\n\n> <badge text=\"\" type=\"error\" vertical=\"middle\"/>: \n\n\n1\n2\n3\n4\n\n * \n\n# \n\n\n\n> : \n\n\n# \n\n<code-group><code-block><code-block>titletabactivetab\n\n<code-group>\n  <code-block title=\"yarn\" active>\n  ```bash\n  yarn add vuepress-theme-vdoing -d\n  ```\n  </code-block>\n\n  <code-block title=\"npm\">\n  ```bash\n  npm install vuepress-theme-vdoing -d\n  ```\n  </code-block>\n</code-group>\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n\nyarn add vuepress-theme-vdoing -d\n\n\n1\n\n\nnpm install vuepress-theme-vdoing -d\n\n\n1\n\n\n\n# \n\nvuepress-plugin-tabs \n\n\n\nnpm install vuepress-plugin-tabs vue-tabs-component\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <userplugins>[\n  // vuepress-plugin-tabshttps://github.com/pskordilakis/vuepress-plugin-tabs\n  ['tabs'],\n]\n\n\n1\n2\n3\n4\n\n\n .vuepress/style/index.styl \n\n@require '~vuepress-plugin-tabs/dist/themes/default.styl'\n\n\n1\n\n\n\n\n:::: tabs\n\n::: tab title\n__markdown content__\n:::\n\n\n::: tab javascript\n``` javascript\n() => {\n  console.log('javascript code example')\n}\n```\n:::\n\n::::\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n\n\n\n\nmarkdown content\n\n() => {\n  console.log('javascript code example')\n}\n\n\n1\n2\n3\n",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{title:"",date:"2023-04-23T21:22:07.000Z",permalink:"/pages/30653b/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/99.%E7%94%9F%E6%B4%BB/02.%E6%91%98%E6%8A%84.html",relativePath:"99./02..md",key:"v-0701bf55",path:"/pages/30653b/",headersStr:null,content:'\n\n-----\n\n\n\n\n\n-----\n\n\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 10\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/02-/Fig0001_.6hfmgmafy080.png\n    link: https://ZhengTiger.github.io/picx-images-hosting/99-/02-/Fig0001_.6hfmgmafy080.png\n    name:  <p align="right"><strong>-----</strong></p>\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/02-/Fig0002_.5snxmcmdyio0.webp\n    link: https://ZhengTiger.github.io/picx-images-hosting/99-/02-/Fig0002_.5snxmcmdyio0.webp\n    name:  <p align="right"><strong>-----</strong></p>\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n',normalizedContent:'\n\n-----\n\n\n\n\n\n-----\n\n\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 10\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/02-/fig0001_.6hfmgmafy080.png\n    link: https://zhengtiger.github.io/picx-images-hosting/99-/02-/fig0001_.6hfmgmafy080.png\n    name:  <p align="right"><strong>-----</strong></p>\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/02-/fig0002_.5snxmcmdyio0.webp\n    link: https://zhengtiger.github.io/picx-images-hosting/99-/02-/fig0002_.5snxmcmdyio0.webp\n    name:  <p align="right"><strong>-----</strong></p>\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n',charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"-",frontmatter:{title:"-",date:"2023-04-20T08:10:37.000Z",permalink:"/pages/c4fda3/",categories:[" - "],tags:[""],readingShow:"top"},regularPath:"/100.%E5%85%B3%E4%BA%8E/02.%E5%85%B3%E4%BA%8E%20-%20%E6%9C%AC%E7%AB%99/06.%E6%9C%AC%E7%AB%99-%E8%8A%B1%E9%87%8C%E8%83%A1%E5%93%A8%E7%9A%84%E6%8F%92%E4%BB%B6.html",relativePath:"100./02. - /06.-.md",key:"v-7931d701",path:"/pages/c4fda3/",headers:[{level:2,title:"",slug:"",normalizedTitle:"",charIndex:2},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:517},{level:2,title:"",slug:"",normalizedTitle:"",charIndex:1e3}],headersStr:"  ",content:"# \n\nvuepress-plugin-cursor-effects \n\n\n\nnpm i vuepress-plugin-cursor-effects -D\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <UserPlugins>[\n  // , https://github.com/moefyit/vuepress-plugin-dynamic-title\n  [\n    'cursor-effects',\n    {\n      size: 2, // size of the particle, default: 2\n      shape: 'circle', // shape of the particle, default: 'star'\n      zIndex: 999999999, // z-index property of the canvas, default: 999999999\n    },\n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\nvuepress-plugin-dynamic-title \n\n\n\nnpm i vuepress-plugin-dynamic-title -D\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <UserPlugins>[\n    // https://github.com/moefyit/vuepress-plugin-dynamic-title\n  [\n    'dynamic-title',\n    {\n      showIcon: '/favicon.ico',\n      showText: '(*)o',\n      hideIcon: '/failure.ico',\n      hideText: '_',\n      recoverTime: 2000,   //  \n    },\n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n# \n\nvuepress-plugin-meting \n\n\n\nnpm i vuepress-plugin-meting -D\n\n\n1\n\n\n\n\n meting  Vue \n\nnpm i vue@2.6.14\nnpm i vue-server-renderer@2.6.14\nnpm i vue-template-compiler@2.6.14\n\n\n1\n2\n3\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <UserPlugins>[\n  [\n    'meting',\n    {\n      meting: {\n        server: 'netease', // \n        type: 'playlist', // \n        mid: '8334143269', //  id\n      }, \n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n\n .md \n\n<Meting server=\"netease\"\n        type=\"playlist\"\n        mid=\"8334143269\"\n        :lrc-type=\"3\"/>\n\n\n1\n2\n3\n4\n\n\n",normalizedContent:"# \n\nvuepress-plugin-cursor-effects \n\n\n\nnpm i vuepress-plugin-cursor-effects -d\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <userplugins>[\n  // , https://github.com/moefyit/vuepress-plugin-dynamic-title\n  [\n    'cursor-effects',\n    {\n      size: 2, // size of the particle, default: 2\n      shape: 'circle', // shape of the particle, default: 'star'\n      zindex: 999999999, // z-index property of the canvas, default: 999999999\n    },\n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n# \n\nvuepress-plugin-dynamic-title \n\n\n\nnpm i vuepress-plugin-dynamic-title -d\n\n\n1\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <userplugins>[\n    // https://github.com/moefyit/vuepress-plugin-dynamic-title\n  [\n    'dynamic-title',\n    {\n      showicon: '/favicon.ico',\n      showtext: '(*)o',\n      hideicon: '/failure.ico',\n      hidetext: '_',\n      recovertime: 2000,   //  \n    },\n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n\n\n\n# \n\nvuepress-plugin-meting \n\n\n\nnpm i vuepress-plugin-meting -d\n\n\n1\n\n\n\n\n meting  vue \n\nnpm i vue@2.6.14\nnpm i vue-server-renderer@2.6.14\nnpm i vue-template-compiler@2.6.14\n\n\n1\n2\n3\n\n\n\n\n .vuepress/config/plugins.ts \n\nexport default <userplugins>[\n  [\n    'meting',\n    {\n      meting: {\n        server: 'netease', // \n        type: 'playlist', // \n        mid: '8334143269', //  id\n      }, \n  ],\n]\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n\n\n\n\n .md \n\n<meting server=\"netease\"\n        type=\"playlist\"\n        mid=\"8334143269\"\n        :lrc-type=\"3\"/>\n\n\n1\n2\n3\n4\n\n\n",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:" - ",frontmatter:{title:" - ",date:"2023-04-17T16:49:26.000Z",permalink:"/message-area/",categories:[null],tags:[null],sidebar:!1,article:!1,editLink:!1,readingShow:"top"},regularPath:"/101.%E6%9C%AC%E7%AB%99%20-%20%E7%95%99%E8%A8%80%E5%8C%BA.html",relativePath:"101. - .md",key:"v-0b9c81ee",path:"/message-area/",headersStr:null,content:"\n\n 10s ",normalizedContent:"\n\n 10s ",charsets:{cjk:!0},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3},{title:"",frontmatter:{title:"",date:"2023-08-26T21:49:11.000Z",permalink:"/pages/aef061/",categories:["",""],tags:[""],readingShow:"top"},regularPath:"/99.%E7%94%9F%E6%B4%BB/03.%E9%9F%B3%E4%B9%90/002.%E4%BB%8E%E5%88%AB%E5%90%8E.html",relativePath:"99./03./002..md",key:"v-8d2263ae",path:"/pages/aef061/",headersStr:null,content:" () - AZA\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n",normalizedContent:" () - aza\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n",charsets:{cjk:!0},lastUpdated:"2023/09/04, 10:33:44",lastUpdatedTimestamp:1693794824e3},{title:"",frontmatter:{title:"",date:"2023-08-26T15:18:30.000Z",permalink:"/pages/46a15a/",categories:["",""],tags:[""],readingShow:"top"},regularPath:"/99.%E7%94%9F%E6%B4%BB/03.%E9%9F%B3%E4%B9%90/001.%E7%92%80%E7%92%A8%E5%86%92%E9%99%A9%E4%BA%BA.html",relativePath:"99./03./001..md",key:"v-14a60cb5",path:"/pages/46a15a/",headersStr:null,content:" ()  \n\n \n \n\n \n \n \n\n \n \n \n \n\n \n \n  \n \n \n \n \n \n   \n \n \n \nRap\n  \n  \n \n \n  \n  \n \n \n\n \n \n \n\n \n \n  \n \n \n \n \n \n   \n \n \n \n  \n  \n  \n \n\n\n ",normalizedContent:" ()  \n\n \n \n\n \n \n \n\n \n \n \n \n\n \n \n  \n \n \n \n \n \n   \n \n \n \nrap\n  \n  \n \n \n  \n  \n \n \n\n \n \n \n\n \n \n  \n \n \n \n \n \n   \n \n \n \n  \n  \n  \n \n\n\n ",charsets:{cjk:!0},lastUpdated:"2023/09/04, 10:33:44",lastUpdatedTimestamp:1693794824e3},{title:"",frontmatter:{title:"",date:"2023-04-24T08:40:27.000Z",permalink:"/pages/eccd67/",categories:[""],tags:[""],readingShow:"top"},regularPath:"/99.%E7%94%9F%E6%B4%BB/01.%E8%A7%82%E5%BD%B1.html",relativePath:"99./01..md",key:"v-e022c6c4",path:"/pages/eccd67/",headersStr:null,content:"\n\n\n\n1993.10.16 ...\n\nEVA \n\n\n\n1995.10.04 2000913\n\n\n\n\n\n1996.01.08 ...\n\n\n\n\n\n1999.03.07 ...\n\n\n\n\n\n1999.10.20 ...\n\n\n\n\n\n2000.10.16 ...\n\n\n\n\n\n2002.10.03 ...\n\n\n\n\n\n2005.10.22 ...\n\n\n\n\n\n2007.02.18 ...\n\n\n\n\n\n2007.10.04 ...\n\n\n\n\n\n2008.01.01 ...\n\n\n\n\n\n2011.04.01 BOSS...\n\n\n\n\n\n2011.10.13 GHQ\n\n\n\n\n\n2012.07.07 2022Sword Art Online\n\n\n\n\n\n2013.04.07 107743\n\n\n\n\n\n2014.02.14 ...\n\n\n\n\n\n2015.03.05 ...\n\n\n\n\n\n2016.06.24 ...\n\n\n\n\n\n2016.07.07 ...\n\n\n\n\n\n2016.07.08  19...\n\n\n\n\n\n2016.09.02 ...\n\n\n\n\n\n2017.01.07 41011...\n\n\n\n\n\n2017.04.07 ...\n\n\n\n\n\n2017.04.28 ...\n\n\n\n\n\n2017.06.01 ...\n\n\n\n\n\n2018.01.20 ...\n\n\n\n\n\n2018.04.25 ...\n\n\n\n\n\n2018.07.08 372...\n\n\n\n\n\n2018.07.09 ...\n\n\n\n\n\n2018.10.05 ...\n\n\n\n\n\n2018.12.20 ...\n\n\n\n\n\n2019.01.16 ...\n\n\n\n\n\n2019.02.06 ...\n\n\n\n\n\n2019.04.07 \n\nDr.STONE\n\n\n\n2019.07.05 ...\n\nRe:zero\n\n\n\n2020.01.01 \n\n\n\n\n\n2020.03.08 ...\n\n\n\n\n\n2020.05.18 \n\n\n\n\n\n2020.10.03 ...\n\n\n\n\n\n2021.10.15 ...\n\n\n\n\n\n2022.12.10 400...\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 4\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/19931016_.1tf9x3vp0ra8.webp\n    link: https://www.iqiyi.com/a_19rrhb244t.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1993.10.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/19951004_.54xk80t0jac0.webp\n    link: https://www.bilibili.com/bangumi/media/md1635\n    name: <div align=\"center\"><strong>EVA </strong></div>\n    desc: '1995.10.04 2000913'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/19960108_.mnzk5bq0bao.webp\n    link: https://www.bilibili.com/bangumi/media/md28228775\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1996.01.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/19990307_.6oiuc1sl1q00.webp\n    link: https://www.iqiyi.com/a_19rrhb1dmp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.03.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/19991020_.7kjnqb9t11k0.webp\n    link: https://www.iqiyi.com/a_19rrhb3xvl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.10.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20001016_.4giytspav300.webp\n    link: https://www.bilibili.com/bangumi/media/md28222083\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2000.10.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20021003_.137hxhreolyo.webp\n    link: https://list.youku.com/show/id_zcc001f06962411de83b1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.10.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20051022_.49caw1b25w00.webp\n    link: https://www.bilibili.com/bangumi/media/md1715\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.10.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20070218_.3vjqy34u6ey0.webp\n    link: https://v.qq.com/x/cover/mzc002001kt6n30.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.02.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20071004_.j1fa5f529tc.webp\n    link: https://www.bilibili.com/bangumi/media/md28234156\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.10.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20080101_.6brhw733lmc.webp\n    link: https://tv.cctv.com/2013/03/13/VIDA1363159660527562.shtml\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.01.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20110401_.u8jats1l5bk.webp\n    link: https://so.youku.com/search_video/q_%E8%B6%85%E5%85%BD%E6%AD%A6%E8%A3%85?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.01 BOSS...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20111013_.2n2iyugy5ik0.webp\n    link: https://www.bilibili.com/bangumi/media/md28229899\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.10.13 GHQ'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20120707_.3eafv3bbwco0.webp\n    link: https://www.bilibili.com/bangumi/media/md24755609\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.07.07 2022Sword Art Online'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20130407_.6c2jaojmt800.webp\n    link: https://www.5yju.com/detail-4710.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.04.07 107743'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20140214_.2f90npse2tes.webp\n    link: https://www.bilibili.com/bangumi/media/md348\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.02.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20150305_.3ikjbq28vfc0.webp\n    link: https://www.bilibili.com/bangumi/media/md1689\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.03.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20160624_.3fl7lj6uj6u0.webp\n    link: https://v.qq.com/x/search/?q=%E6%AD%A6%E5%BA%9A%E7%BA%AA+%E7%AC%AC1%E5%AD%A3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.06.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20160707_.54jz1xk32vk0.webp\n    link: https://www.bilibili.com/bangumi/media/md5045\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20160708_.2jv84aqhe620.webp\n    link: https://www.bilibili.com/bangumi/media/md5050\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.08  19...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20160902_.3efd3k56a4i0.webp\n    link: https://v.qq.com/x/search/?q=%E5%85%A8%E8%81%8C%E6%B3%95%E5%B8%88+%E7%AC%AC%E4%B8%80%E5%AD%A3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20170107_.19vngp2vjmg0.webp\n    link: https://v.qq.com/x/cover/xvlhj7cglyfwx8x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.01.07 41011...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20170407_.2kbo32579ro0.webp\n    link: https://v.qq.com/x/search/?q=%E5%85%A8%E8%81%8C%E9%AB%98%E6%89%8B+%E7%AC%AC1%E5%AD%A3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.04.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20170428_.1rnkhuwvgy1s.webp\n    link: https://www.bilibili.com/bangumi/media/md5855\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.04.28 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20170601_.2jj44cjb1w00.webp\n    link: https://v.qq.com/x/search/?q=%E9%9B%84%E5%85%B5%E8%BF%9E1%E5%A4%A9%E6%B2%B3%E6%88%98%E5%BD%B9&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.06.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20180120_.1da1j48ygehs.webp\n    link: https://v.qq.com/x/cover/m441e3rjq9kwpsc/m00253deqqo.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20180425_.6ts6stu55t40.webp\n    link: https://www.bilibili.com/bangumi/media/md6360\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.04.25 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20180708_.13myg5syhgf4.webp\n    link: https://www.bilibili.com/bangumi/media/md102392\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.08 372...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20180709_.3ekdvmwqgz00.webp\n    link: https://v.qq.com/x/cover/k4mutekomtrdbux.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20181005_.1kfq0sa12ue8.webp\n    link: https://v.qq.com/x/search/?q=%E6%98%9F%E8%BE%B0%E5%8F%98+%E7%AC%AC1%E5%AD%A3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.10.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20181220_.3mvps241qbu0.webp\n    link: https://v.qq.com/x/search/?q=%E9%9B%AA%E9%B9%B0%E9%A2%86%E4%B8%BB&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20190116_.2blgxmin20sg.webp\n    link: https://v.qq.com/x/cover/2w2legt0g8z26al/u00296p6t50.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.01.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20190206_.4zsqt9ufkog0.webp\n    link: https://v.qq.com/x/search/?q=%E4%B8%87%E7%95%8C%E7%A5%9E%E4%B8%BB&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20190407_.5cy5r0x6rf40.webp\n    link: https://www.bilibili.com/bangumi/media/md22718131\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.04.07 '\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20190705_.34ese4dynua0.webp\n    link: https://www.bilibili.com/bangumi/media/md28221387\n    name: <div align=\"center\"><strong>Dr.STONE</strong></div>\n    desc: '2019.07.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20200101_.3qz0nn8hz6y0.webp\n    link: https://www.bilibili.com/bangumi/media/md28224394\n    name: <div align=\"center\"><strong>Re:zero</strong></div>\n    desc: '2020.01.01 '\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20200308_.5f9r7d2tgh00.webp\n    link: https://v.qq.com/x/search/?q=%E6%AD%A6%E7%A5%9E%E4%B8%BB%E5%AE%B0&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.03.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20200518_.3tsj8vulnz80.webp\n    link: https://v.qq.com/x/search/?q=%E6%97%A0%E4%B8%8A%E7%A5%9E%E5%B8%9D&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.05.18 '\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20201003_.29yakemsnocg.webp\n    link: https://www.bilibili.com/bangumi/media/md28229899\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.10.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20211015_.59ztgxeyhxs0.webp\n    link: https://www.bilibili.com/bangumi/media/md28235154\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.10.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/01-/20221210_.ii0z3wwyeqo.webp\n    link: https://www.bilibili.com/bangumi/media/md4315402\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.12.10 400...'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n\n\n\n\n\n\n1986.02.09 ...\n\n2002\n\n\n\n2002.12.01 ...\n\n\n\n\n\n2005.01.24 ...\n\n\n\n\n\n2005.12.16 ...\n\n2006\n\n\n\n2006.03.14 ...\n\n\n\n\n\n2008.04.08 2060...\n\n2010\n\n\n\n2010.05.02 ...\n\n\n\n\n\n2011.04.17 StarkEddard Star...\n\n\n\n\n\n2012.09.29 ...\n\n\n\n\n\n2012.10.01 2090...\n\n2013\n\n\n\n2013.02.06 ...\n\n\n\n\n\n2013.10.23 21...\n\n\n\n\n\n2014.08.25 CIB...\n\n\n\n\n\n2015.02.26 70...\n\n\n\n\n\n2015.06.09 ...\n\n1988\n\n\n\n2015.11.06 386881971...\n\n\n\n\n\n2015.11.29 ...\n\n\n\n\n\n2016.04.08 ...\n\n\n\n\n\n2016.05.23 ...\n\n2017\n\n\n\n2017.01.09 ...\n\n\n\n\n\n2017.03.28 ...\n\n\n\n\n\n2017.10.12 ...\n\n\n\n\n\n2017.11.09 19...\n\n\n\n\n\n2017.12.13 ...\n\n\n\n\n\n2018.01.12 ...\n\n\n\n\n\n2018.07.06 ...\n\n\n\n\n\n2018.08.02 ...\n\n\n\n\n\n2018.08.07 ...\n\n\n\n\n\n2018.09.03 ...\n\n\n\n\n\n2018.10.31 ...\n\n\n\n\n\n2019.03.01 ...\n\n\n\n\n\n2019.06.09 ...\n\n\n\n\n\n2019.06.27 ...\n\n\n\n\n\n2019.07.24 ...\n\n\n\n\n\n2019.11.12 ...\n\n\n\n\n\n2020.01.30 ...\n\n\n\n\n\n2020.05.18 ...\n\n\n\n\n\n2020.07.14 ...\n\n\n\n\n\n2020.07.17 ...\n\n\n\n\n\n2020.12.16 ...\n\n\n\n\n\n2021.02.01 ...\n\n\n\n\n\n2021.02.05 ...\n\n\n\n\n\n2021.08.09 ...\n\n\n\n\n\n2021.12.15 ...\n\n\n\n\n\n2023.01.15 2007...\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 4\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/19860209_.79ogmd93d1w0.webp\n    link: https://www.bilibili.com/bangumi/media/md28229051\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.02.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20021201_2002.cd67c6eixwo.webp\n    link: https://v.qq.com/x/search/?q=%E5%80%9A%E5%A4%A9%E5%B1%A0%E9%BE%99%E8%AE%B0\n    name: <div align=\"center\"><strong>2002</strong></div>\n    desc: '2002.12.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20050124_.388fvzqltx40.jpg\n    link: https://so.youku.com/search_video/q_%E4%BB%99%E5%89%91%E5%A5%87%E4%BE%A0%E4%BC%A01?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.01.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20051216_.2k6naze37oe0.webp\n    link: https://www.iqiyi.com/v_19rr8rigrg.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.12.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20060314_2006.1s88vtlv7jmo.webp\n    link: https://www.iqiyi.com/a_19rrhbznr5.html\n    name: <div align=\"center\"><strong>2006</strong></div>\n    desc: '2006.03.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20080408_.jfb32ho0ik8.jpg\n    link: https://www.bilibili.com/bangumi/media/md21317\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.04.08 2060...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20100502_2010.5mjdhel0ios0.webp\n    link: https://v.qq.com/x/search/?q=%E6%96%B0%E4%B8%89%E5%9B%BD&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong>2010</strong></div>\n    desc: '2010.05.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20110417_.5p9g0bf0xu00.webp\n    link: https://v.qq.com/x/search/?q=%E6%9D%83%E5%8A%9B%E7%9A%84%E6%B8%B8%E6%88%8F+%E7%AC%AC%E4%B8%80%E5%AD%A3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.17 StarkEddard Star...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20120929_.1ediygtve9ds.jpg\n    link: https://www.iqiyi.com/a_19rrk60bih.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.09.29 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20121001_.1jlhspre7abk.jpg\n    link: https://www.iqiyi.com/a_df79g6zw2d.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.10.01 2090...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20130206_2013.5flyqx85ql00.webp\n    link: https://v.qq.com/x/search/?q=%E7%AC%91%E5%82%B2%E6%B1%9F%E6%B9%962012&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong>2013</strong></div>\n    desc: '2013.02.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20131023_.2196eobkgim8.webp\n    link: https://v.qq.com/x/search/?q=%E6%88%91%E6%98%AF%E7%89%B9%E7%A7%8D%E5%85%B5%E4%B9%8B%E7%81%AB%E5%87%A4%E5%87%B0&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.10.23 21...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20140825_.2fqg2kkq5xes.webp\n    link: https://so.youku.com/search_video/q_%E4%BD%BF%E5%BE%92%E8%A1%8C%E8%80%85?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.25 CIB...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20150226_.2jlrtsu3xj40.jpg\n    link: https://v.qq.com/x/search/?q=%E5%B9%B3%E5%87%A1%E7%9A%84%E4%B8%96%E7%95%8C&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.02.26 70...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20150609_.4zh82za05gc0.jpg\n    link: https://v.qq.com/x/search/?q=%E8%8A%B1%E5%8D%83%E9%AA%A82015&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.06.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20151106_1988.4y7jc8pdglo0.webp\n    link: https://www.iqiyi.com/a_19rrhbhrod.html\n    name: <div align=\"center\"><strong>1988</strong></div>\n    desc: '2015.11.06 386881971...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20151129_.7gp4byilqhw0.jpg\n    link: https://so.youku.com/search_video/q_%E7%A7%A6%E6%97%B6%E6%98%8E%E6%9C%88%20TV%E7%89%88?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.11.29 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20160408_.4wbsl0c5qjg0.jpg\n    link: https://www.iqiyi.com/a_19rrhbd6ad.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.04.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20160523_.4l6y3ciqseg0.jpg\n    link: https://www.iqiyi.com/a_19rrhalg9p.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.05.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20170109_2017.2ue452phb380.jpg\n    link: https://www.iqiyi.com/a_19rrhatc1p.html\n    name: <div align=\"center\"><strong>2017</strong></div>\n    desc: '2017.01.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20170328_.3hvq2621f0o0.jpg\n    link: https://so.youku.com/search_video/q_%E4%BA%BA%E6%B0%91%E7%9A%84%E5%90%8D%E4%B9%89%20DVD%E7%89%88?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.03.28 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20171012_.3z5x1nb00wu0.webp\n    link: https://www.bilibili.com/bangumi/media/md28223924\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.10.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20171109_.e9b50ahf5g0.webp\n    link: https://v.qq.com/x/search/?q=%E8%87%B4%E6%88%91%E4%BB%AC%E5%8D%95%E7%BA%AF%E7%9A%84%E5%B0%8F%E7%BE%8E%E5%A5%BD\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.11.09 19...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20171213_.3tgisjll9j60.webp\n    link: https://v.qq.com/x/search/?q=%E6%9F%92%E4%B8%AA%E6%88%91&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.12.13 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20180112_.1rb9eoupn9sw.webp\n    link: https://v.qq.com/x/search/?q=%E6%81%8B%E7%88%B1%E5%85%88%E7%94%9F&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20180706_.k85olwpnme8.jpg\n    link: https://v.qq.com/x/search/?q=%E7%8C%8E%E6%AF%92%E4%BA%BA&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20180802_.4ino5hb73go0.jpg\n    link: https://v.qq.com/x/search/?q=%E9%A6%99%E8%9C%9C%E6%B2%89%E6%B2%89%E7%83%AC%E5%A6%82%E9%9C%9C&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.08.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20180807_.6ziki0x64p80.webp\n    link: https://so.youku.com/search_video/q_%E6%AD%A6%E5%8A%A8%E4%B9%BE%E5%9D%A4%E4%B9%8B%E8%8B%B1%E9%9B%84%E5%87%BA%E5%B0%91%E5%B9%B4?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.08.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20180903_.2w48ukotxp40.jpg\n    link: https://v.qq.com/x/search/?q=%E6%96%97%E7%A0%B4%E8%8B%8D%E7%A9%B9%E7%94%B5%E8%A7%86%E5%89%A7%E7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20181031_.3u5j5bhocjm0.jpg\n    link: https://v.qq.com/x/search/?q=%E5%B0%86%E5%A4%9C&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.10.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20190301_.4po188rk3tw0.jpg\n    link: https://v.qq.com/x/search/?q=%E9%83%BD%E6%8C%BA%E5%A5%BD&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.03.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20190609_.72uauzsw8fw0.webp\n    link: https://v.qq.com/x/search/?q=%E5%B0%91%E5%B9%B4%E6%B4%BE&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.06.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20190627_.5xgcebh2uso0.webp\n    link: https://v.qq.com/x/search/?q=%E9%99%88%E6%83%85%E4%BB%A4&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.06.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20190724_.1siafz4zj39c.jpg\n    link: https://v.qq.com/x/search/?q=%E5%85%A8%E8%81%8C%E9%AB%98%E6%89%8B&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20191112_.5ch827ubz780.webp\n    link: https://v.qq.com/x/search/?q=%E4%BB%8E%E5%89%8D%E6%9C%89%E5%BA%A7%E7%81%B5%E5%89%91%E5%B1%B1&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.11.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20200130_.54b413a0gow0.jpg\n    link: https://v.qq.com/x/search/?q=%E5%A4%A7%E4%B8%BB%E5%AE%B0%E7%94%B5%E8%A7%86%E5%89%A7%E7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.01.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20200518_.1ovq5z8el1s0.jpg\n    link: https://v.qq.com/x/search/?q=%E4%BC%A0%E9%97%BB%E4%B8%AD%E7%9A%84%E9%99%88%E8%8A%8A%E8%8A%8A&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.05.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20200714_.lv4ylx4ftc0.jpg\n    link: https://www.iqiyi.com/a_19rrhlatf5.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.07.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20200717_.5f5boxy06oo0.jpg\n    link: https://v.qq.com/x/search/?q=%E4%B8%89%E5%8D%81%E8%80%8C%E5%B7%B2&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.07.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20201216_.3bi3t5ya18c0.webp\n    link: https://v.qq.com/x/search/?q=%E9%BB%91%E7%99%BD%E7%A6%81%E5%8C%BA&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.12.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20210201_.eyf1e4ab8ls.png\n    link: https://www.iqiyi.com/a_19rrhymcsx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20210205_.28qkpusy6xog.webp\n    link: https://v.qq.com/x/search/?q=%E6%96%97%E7%BD%97%E5%A4%A7%E9%99%86%E7%94%B5%E8%A7%86%E5%89%A7%E7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20210809_.3tsj8vv1sqg0.webp\n    link: https://v.qq.com/x/search/?q=%E6%89%AB%E9%BB%91%E9%A3%8E%E6%9A%B4&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.08.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20211215_.c9tomc12dk0.webp\n    link: https://v.qq.com/x/search/?q=%E9%9B%AA%E4%B8%AD%E6%82%8D%E5%88%80%E8%A1%8C&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.12.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/02-/20230115_.4ex9mpvnnh80.jpg\n    link: https://v.qq.com/x/search/?q=%E4%B8%89%E4%BD%93&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.01.15 2007...'\n\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n\n\n\n\n\n\n1984.03.11 7...\n\n\n\n\n\n1986.08.02 ...\n\n\n\n\n\n1988.04.16 14...\n\n\n\n\n\n1988.04.16 ...\n\n\n\n\n\n1994.06.15 ...\n\n\n\n\n\n1997.07.12 ...\n\n\n\n\n\n2001.07.20 ...\n\n1\n\n\n\n2002.03.15 ...\n\n1\n\n\n\n2003.05.30 ...\n\n2\n\n\n\n2006.06.09 ...\n\n\n\n\n\n2007.03.03 ...\n\n1\n\n\n\n2008.06.06 ...\n\n\n\n\n\n2008.06.27 2805...\n\n\n\n\n\n2008.07.19 ...\n\n3\n\n\n\n2009.07.09 MannyDiego...\n\n\n\n\n\n2009.08.04 ...\n\n1\n\n\n\n2010.03.18 ...\n\n1\n\n\n\n2010.07.09 ...\n\n1\n\n\n\n2011.04.08 ...\n\n4\n\n\n\n2012.07.27 ...\n\n1\n\n\n\n2013.04.20 Grug...\n\n\n\n\n\n2013.05.31 15...\n\n2\n\n\n\n2013.07.03 ...\n\n\n\n\n\n2013.07.20 ...\n\n1\n\n\n\n2013.11.27 ...\n\n2\n\n\n\n2014.04.11 ...\n\n2\n\n\n\n2014.08.14 ...\n\n\n\n\n\n2014.11.07 +...\n\n\n\n\n\n2015.07.10 ...\n\n\n\n\n\n2015.10.16 ...\n\n\n\n\n\n2016.03.04 ...\n\n\n\n\n\n2016.07.08 ...\n\n5\n\n\n\n2016.07.22 ...\n\n\n\n\n\n2016.08.26 ...\n\n\n\n\n\n2016.09.17 ...\n\n3\n\n\n\n2017.07.07 ...\n\n\n\n\n\n2017.11.24 ...\n\n\n\n\n\n2018.01.16 ...\n\n3\n\n\n\n2019.01.03 ...\n\n\n\n\n\n2019.07.19 ...\n\n\n\n\n\n2019.07.26 ...\n\n2\n\n\n\n2019.11.22 ...\n\n1\n\n\n\n2020.08.06 ...\n\n\n\n\n\n2020.10.01 ...\n\n2\n\n\n\n2020.11.27 ...\n\n2\n\n\n\n2021.02.12 ...\n\n2\n\n\n\n2021.07.23 ...\n\n\n\n\n\n2022.08.19 ...\n\n\n\n\n\n2022.11.11 17...\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 4\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19840311_.5s5bo6s5it80.webp\n    link: https://yhdm6.top/index.php/vod/play/id/2941/sid/3/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1984.03.11 7...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19860802_.3dtwcq95uti0.webp\n    link: https://yhdm6.top/index.php/vod/play/id/13098/sid/2/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.08.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19880416_.5fotjlnro5o0.webp\n    link: https://yhdm6.top/index.php/vod/play/id/15833/sid/1/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1988.04.16 14...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19880416_.3crzn4o5wrs0.webp\n    link: https://www.iqiyi.com/v_19rr7r4y5g.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1988.04.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19940615_.4xwlvm0mblw0.webp\n    link: https://v.qq.com/x/cover/lh4cj69w87hfodz/g0022161p32.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.06.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/19970712_.5v2hez1s3e00.webp\n    link: https://yhdm6.top/index.php/vod/play/id/2978/sid/3/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.07.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20010720_.12vl0rpxlmcg.webp\n    link: https://yhdm6.top/index.php/vod/play/id/4905/sid/4/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2001.07.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20020315_1.5jjr4e6a9m00.webp\n    link: https://v.qq.com/x/cover/thrjyzyxu8wkgel/c0020pocfxn.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2002.03.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20030530_1.6npsnasdflw0.webp\n    link: https://v.qq.com/x/cover/5frrueiimmg47zt/a0012tsyz90.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2003.05.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20060609_2.72kcughb67c0.webp\n    link: https://v.qq.com/x/cover/c5gf072ldgv9yv4/d0020h4v7fy.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2006.06.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20070303_5.3g9j51t6nyq0.webp\n    link: https://www.bilibili.com/bangumi/media/md2688\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.03.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20080606_1.36fr0nofrvy0.webp\n    link: https://www.iqiyi.com/v_19rrnppf4o.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.06.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20080627_.2mwbngr1wwi0.webp\n    link: https://v.qq.com/x/cover/vqeduf4egjyrc8w/v00121f4a8u.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.06.27 2805...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20080719_.2oe6ls26p660.webp\n    link: https://www.iqiyi.com/v_1htw2y3hjpw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.07.19 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20090709_3.vofcyn3v1ps.webp\n    link: https://v.qq.com/x/cover/d5mkraazz53p6gt/h00202lepiv.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2009.07.09 MannyDiego...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20090804_.65cwlcgoh7s0.webp\n    link: https://v.qq.com/x/cover/w9gk2xdkyvez6yp/a0012lgk0o8.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.08.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20100318_1.5la2x412id80.webp\n    link: https://v.qq.com/x/cover/3imeqzq7nzw130g/h00202vxfcc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2010.03.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20100709_1.4jlxpjihq940.webp\n    link: https://v.qq.com/x/cover/oylsefopm3zevpj/j0013tua8os.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2010.07.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20110408_1.5l2avz1fpeo0.webp\n    link: https://v.qq.com/x/cover/zb24obs27w4mfdw/k0020jbkc1h.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.04.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20120727_4.xsmyljyzfy8.webp\n    link: https://v.qq.com/x/cover/qszw1ugyfz10izl/f0020g3u5lt.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2012.07.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20130420_1.2st3r9p88xw0.webp\n    link: https://v.qq.com/x/cover/m4ry5h1naszjlzh/t0012d28gss.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.04.20 Grug...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20130531_.4bai9idgntk0.webp\n    link: https://www.bilibili.com/bangumi/media/md2546\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.05.31 15...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20130703_2.1zsawgnf9dvk.webp\n    link: https://v.qq.com/x/cover/ksbt3iklh13mawf.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.07.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20130720_.1v0z4701y3y8.webp\n    link: https://yhdm6.top/index.php/vod/play/id/9933/sid/4/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.07.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20131127_1.7eao2yh06sg0.webp\n    link: https://v.qq.com/x/cover/bv8m91i5jj1w1ww/k0014rpm1vd.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.11.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20140411_2.3a37750gzlw0.webp\n    link: https://v.qq.com/x/cover/vatxsyi3vgs4ul0/o0024gk1tb6.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.04.11 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20140814_2.6zdxwe6aoj80.webp\n    link: https://v.qq.com/x/cover/z46ef2t37pfuojr/p00157g180n.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.08.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20141107_.6g8sik1uxho0.webp\n    link: https://v.qq.com/x/cover/hjdb95f8lxcyyew/o0016yn71wh.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.11.07 +...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20150710_.l7b55jgk39s.webp\n    link: https://v.qq.com/x/cover/3o56brnhunm5bwx/x0018dcg7rm.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.07.10 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20151016_.53ghml9vurg0.webp\n    link: https://www.iqiyi.com/v_19rrkdlco0.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.10.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20160304_.3raxkduicgo0.webp\n    link: https://v.qq.com/x/cover/fhe2h7sop52qzza/l0020h13orj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.03.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20160708_.2jaud65cuow0.webp\n    link: https://v.qq.com/x/cover/we35g3aduiwkudp/u0021lcrkp2.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20160722_5.38zblcp4w1w0.webp\n    link: https://v.qq.com/x/cover/ofbgmce1glm7kwe/v0033fnq85s.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2016.07.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20160826_.3utr585ulmy0.webp\n    link: https://v.qq.com/x/cover/gz9q7kd8wviwkxv/m0024nvac3j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.08.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20160917_.i7howzh6h74.webp\n    link: https://v.qq.com/x/cover/feqfsbsngr2re6e/k0024mc33fl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20170707_3.4otba4tpotq0.webp\n    link: https://v.qq.com/x/cover/y0vxr2r3ptzxbic/o0024bwaj7l.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.07.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20171124_.9ul2qsozwyg.webp\n    link: https://v.qq.com/x/cover/vjrvzv3s517g6m8/w0025hgvzin.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.11.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20180116_.6cn7onig27c0.webp\n    link: https://v.qq.com/x/cover/hzssbvhzc7xyr95/y0030o2gbjj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20190103_3.26mranwrt868.webp\n    link: https://v.qq.com/x/cover/8rpyk6div4sku7g/q0030fnxigq.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2019.01.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20190719_.48y4gaq3i5y0.webp\n    link: https://www.bilibili.com/bangumi/media/md28228734\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.19 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20190726_.3y34fkkdbki0.webp\n    link: https://v.qq.com/x/cover/zr5a67l333ehzu9/d0032x85stp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20191122_2.115crnsv5xuo.webp\n    link: https://v.qq.com/x/cover/06f338d6r1wa3my/a003369kmz4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.11.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20200806_1.2akqrcgkw45c.webp\n    link: https://v.qq.com/x/cover/mzc002002smgao4.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2020.08.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20201001_.3zwlnse3eqq0.webp\n    link: https://v.qq.com/x/cover/mzc00200892n01l/s0034x8bf06.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.10.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20201127_2.3a46n8dn2d40.webp\n    link: https://v.qq.com/x/cover/ccd36tevohjt9ze/u0035tjlfmu.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2020.11.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20210212_2.6wonqcu3ap80.webp\n    link: https://v.qq.com/x/cover/mzc00200jq7kvn8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2021.02.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20210723_.3vcutfuxa9y0.webp\n    link: https://v.qq.com/x/cover/mzc002003vpbryd.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2021.07.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20220819_.6mi6ccvsn6o0.webp\n    link: https://v.qq.com/x/cover/mzc00200mjy32e7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.08.19 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/03-/20221111_.eghwhxmw1io.webp\n    link: \n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.11.11 17...'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n234\n235\n236\n237\n238\n239\n240\n241\n242\n243\n244\n245\n246\n247\n248\n249\n250\n251\n\n\n\n\n\n\n1925.06.26 ...\n\n\n\n\n\n1928.01.06 ...\n\n\n\n\n\n1936.02.25 2020...\n\n\n\n\n\n1949.12.01 ...\n\n\n\n\n\n1965.03.02 ...\n\n1\n\n\n\n1975.06.20 ...\n\n1\n\n\n\n1979.05.25 ...\n\n1\n\n\n\n1979.12.06 ...\n\n\n\n\n\n1980.05.23 ...\n\n\n\n\n\n1985.11.07 ...\n\n2\n\n\n\n1986.07.18 ...\n\n\n\n\n\n1986.08.15 ...\n\n\n\n\n\n1987.12.17 ...\n\n\n\n\n\n1989.07.26 ...\n\n\n\n\n\n1989.12.14 ...\n\n1\n\n\n\n1991.07.18 ...\n\n2\n\n\n\n1992.04.09 ...\n\n3\n\n\n\n1992.05.22 ...\n\n\n\n\n\n1992.10.01 ...\n\n\n\n\n\n1992.12.17 ...\n\n3\n\n\n\n1993.01.01 ...\n\n1\n\n\n\n1993.06.11 ...\n\n\n\n\n\n1993.07.01 ...\n\n\n\n\n\n1994.03.31 ...\n\n\n\n\n\n1994.06.30 ...\n\n\n\n\n\n1994.07.06 75...\n\n\n\n\n\n1994.10.14 ...\n\n\n\n\n\n1995.01.21 ...\n\n\n\n\n\n1995.02.04 ...\n\n\n\n\n\n1995.07.06 ...\n\n\n\n\n\n1995.08.19 ...\n\n\n\n\n\n1997.08.01 ...\n\n2\n\n\n\n1997.08.07 ....\n\n\n\n\n\n1997.12.05 ...\n\n\n\n\n\n1997.12.19  ...\n\n\n\n\n\n1997.12.20 ...\n\n\n\n\n\n1998.01.31 ...\n\n\n\n\n\n1998.06.05 ...\n\n\n\n\n\n1999.02.13 ...\n\n\n\n\n\n1999.03.31 ...\n\n\n\n\n\n1999.04.16 ...\n\n\n\n\n\n1999.11.04 ...\n\n1\n\n\n\n2000.03.17 -...\n\n1\n\n\n\n2001.06.22 ...\n\n\n\n\n\n2001.07.12 ...\n\n3\n\n\n\n2001.07.18 ....\n\n1\n\n\n\n2002.03.15 ...\n\n\n\n\n\n2002.09.25 ...\n\n\n\n\n\n2002.10.18 ...\n\n\n\n\n\n2002.12.12 1991...\n\n2\n\n\n\n2003.01.31 ...\n\n2\n\n\n\n2003.06.03 FBI...\n\n1\n\n\n\n2003.11.21 (Caribbean Sea)...\n\n\n\n\n\n2004.03.17 1949...\n\n\n\n\n\n2004.05.28 ...\n\n\n\n\n\n2004.09.09 ThunJaneJane...\n\n1\n\n\n\n2004.10.29 ...\n\n\n\n\n\n2004.12.23 1940...\n\nD\n\n\n\n2005.06.23 ...\n\n\n\n\n\n2005.09.23 ...\n\n2\n\n\n\n2005.10.28  ...\n\n\n\n\n\n2005.12.14 ...\n\n\n\n\n\n2006.02.02 12...\n\n\n\n\n\n2006.04.21 RoseRose...\n\n3\n\n\n\n2006.06.16 ...\n\n2\n\n\n\n2006.07.07 ...\n\n3\n\n\n\n2006.10.27 ...\n\n\n\n\n\n2006.12.15 ...\n\n\n\n\n\n2007.03.16 ....\n\n3\n\n\n\n2007.06.12 ...\n\n1\n\n\n\n2007.07.11 ...\n\n\n\n\n\n2007.07.27 ...\n\n\n\n\n\n2007.07.31 ...\n\n4\n\n\n\n2007.10.25 SWAT...\n\n\n\n\n\n2007.11.16 19Florentino ArizaFermina Daza...\n\n\n\n\n\n2008.01.30 ...\n\n1\n\n\n\n2008.04.30 ...\n\n\n\n\n\n2008.06.13 Bruce Banner...\n\n5\n\n\n\n2008.10.23 ...\n\n1\n\n\n\n2008.12.12 2030...\n\n4\n\n\n\n2009.04.15 1...\n\n1\n\n\n\n2009.05.03 1845...\n\n2\n\n\n\n2009.06.24 ...\n\n6\n\n\n\n2009.10.23 ...\n\n2012\n\n\n\n2009.11.13 ...\n\n\n\n\n\n2009.11.27 1990...\n\n1\n\n\n\n2009.12.18 ...\n\n\n\n\n\n2009.12.25 ...\n\n2\n\n\n\n2010.04.27 ...\n\n2\n\n\n\n2010.05.07 ...\n\n\n\n\n\n2010.06.04 ...\n\n\n\n\n\n2010.07.22 1976...\n\n\n\n\n\n2010.07.26 ...\n\n7\n\n\n\n2010.10.29 ...\n\n\n\n\n\n2011.04.21 ...\n\n1\n\n\n\n2011.05.08 ...\n\n5\n\n\n\n2011.05.12 ...\n\n4\n\n\n\n2011.05.20 ...\n\n3\n\n\n\n2011.07.21 50...\n\n1\n\n\n\n2011.07.22 40...\n\n\n\n\n\n2011.11.02 ...\n\nHello\n\n\n\n2011.11.04 ...\n\n1\n\n\n\n2012.05.05 ...\n\n\n\n\n\n2012.12.12 ...\n\n3\n\n\n\n2013.05.01 ...\n\n\n\n\n\n2013.05.09 ...\n\n1\n\n\n\n2013.07.12 ...\n\n\n\n\n\n2013.07.19 20...\n\n6\n\n\n\n2013.07.26 ...\n\n2\n\n\n\n2013.10.17 ...\n\n\n\n\n\n2013.10.24 ...\n\n\n\n\n\n2013.11.05 1942...\n\n2\n\n\n\n2013.11.08 ...\n\n\n\n\n\n2013.11.19 ...\n\n2\n\n\n\n2014.04.04 ...\n\n1\n\n\n\n2014.06.13 15...\n\n4\n\n\n\n2014.06.27 ...\n\n1\n\n\n\n2014.08.01 ...\n\n\n\n\n\n2014.08.30 ...\n\n\n\n\n\n2014.08.30 ...\n\n\n\n\n\n2014.11.12 ...\n\n1\n\n\n\n2015.04.02 ...\n\n7\n\n\n\n2015.04.12 ...\n\n2\n\n\n\n2015.05.12 ...\n\n4\n\n\n\n2015.06.10 21...\n\n1\n\n\n\n2015.07.17 ...\n\n\n\n\n\n2015.09.30 ...\n\n\n\n\n\n2015.11.25 ...\n\n3\n\n\n\n2015.12.24 ...\n\n1\n\n\n\n2015.12.31 ...\n\n\n\n\n\n2016.02.08 ...\n\n3\n\n\n\n2016.05.06 ...\n\n\n\n\n\n2016.09.29 DJ...\n\n\n\n\n\n2016.10.06 201110513...\n\n1\n\n\n\n2016.11.06 ...\n\n\n\n\n\n2016.12.23 ...\n\n8\n\n\n\n2017.04.14 ...\n\n2\n\n\n\n2017.05.05 ...\n\n3\n\n\n\n2017.05.23 2024X...\n\n5\n\n\n\n2017.05.26 320 ...\n\n5\n\n\n\n2017.06.23 ...\n\n2\n\n\n\n2017.07.27 ...\n\n\n\n\n\n2017.08.11 ...\n\n\n\n\n\n2017.09.08 1989...\n\n1\n\n\n\n2017.09.08 ...\n\n\n\n\n\n2017.09.30 ...\n\n8\n\n\n\n2017.10.27 ...\n\n3\n\n\n\n2017.11.03 ...\n\n\n\n\n\n2017.12.28 1972...\n\n2\n\n\n\n2018.02.16 ...\n\n\n\n\n\n2018.02.16 ...\n\n1\n\n\n\n2018.02.16 ...\n\n2\n\n\n\n2018.03.23 ...\n\n\n\n\n\n2018.03.28 2060...\n\n\n\n\n\n2018.03.30 2045...\n\n3\n\n\n\n2018.05.11 ...\n\n1\n\n\n\n2018.05.18 ...\n\n\n\n\n\n2018.07.05 ...\n\n2\n\n\n\n2018.07.06 FBI...\n\n\n\n\n\n2018.07.27 ...\n\n\n\n\n\n2018.09.20 ...\n\n\n\n\n\n2018.09.30 ...\n\n\n\n\n\n2018.11.16 ...\n\n\n\n\n\n2018.12.07 ...\n\n\n\n\n\n2018.12.21 B-1271987...\n\n\n\n\n\n2019.02.05 ...\n\n1\n\n\n\n2019.02.05 ...\n\n\n\n\n\n2019.02.05 ...\n\n1\n\n\n\n2019.03.08 ...\n\n4\n\n\n\n2019.04.24 ...\n\n\n\n\n\n2019.05.10 ...\n\n2\n\n\n\n2019.05.31 ...\n\n2\n\n\n\n2019.06.28 ...\n\n\n\n\n\n2019.07.12 ...\n\n\n\n\n\n2019.07.18 ...\n\n\n\n\n\n2019.08.09 ...\n\n1\n\n\n\n2019.09.13 ...\n\n\n\n\n\n2019.09.30 20185143U8633...\n\n\n\n\n\n2019.09.30 196015...\n\n4\n\n\n\n2019.12.20 ...\n\n\n\n\n\n2020.08.07 ...\n\n\n\n\n\n2020.08.21 193788262524420...\n\n\n\n\n\n2021.01.15 ...\n\n3\n\n\n\n2021.02.12 CRIMASTER...\n\n\n\n\n\n2021.02.12 ...\n\n\n\n\n\n2021.03.26 ...\n\n3\n\n\n\n2021.12.15 4...\n\n\n\n\n\n2022.02.01 ...\n\n\n\n\n\n2022.03.05 ...\n\n\n\n\n\n2022.06.24 ...\n\n\n\n\n\n2022.07.29 ...\n\n\n\n\n\n2022.09.30  ...\n\n2\n\n\n\n2022.12.16 ...\n\n2\n\n\n\n2023.01.22 ...\n\n\n\n\n\n2023.01.22 ...\n\n\n\n\n\n2023.04.28 ...\n\nconfig:\n    target: _blank\n    imgHeight: auto\n    objectFit: cover\n    lineClamp: 4\n\ndata:\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19250626_.77ecc6ish680.webp\n    link: https://v.qq.com/x/cover/tqgmpsxckcen61k.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1925.06.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19280106_.21s2zois81ls.webp\n    link: https://v.qq.com/x/cover/coy40bv9c0qtpq4/i00221744g0.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1928.01.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19360225_.4lz3vmrcj660.webp\n    link: https://v.qq.com/x/cover/9w35g1rr7w3smmr/d0015rhc18d.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1936.02.25 2020...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19491201_.7cr8ewa5nz00.webp\n    link: https://v.qq.com/x/cover/rz7x3oii2cafdnb/w0189iuvk50.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1949.12.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19650302_.2dzx6n830268.webp\n    link: https://v.qq.com/x/cover/nlzswcxr4umy1fr.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1965.03.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19750620_.k0h738dexi8.webp\n    link: https://v.qq.com/x/cover/44jy24n86wmax84/x0011vrknod.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1975.06.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19790525_1.1q8uz1ypcw0w.webp\n    link: https://v.qq.com/x/cover/87zd07p6ok4h7r6/p0020mgmuki.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1979.05.25 ...'\n  \n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19791206_1.66xbpi87g3s0.webp\n    link: https://v.qq.com/x/cover/l25jwdqjjwvyhqb/d0017c9p14q.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1979.12.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19800523_.5ehebvqdojk0.webp\n    link: https://v.qq.com/x/cover/di25h17qqkgj08p/u0022mdu41x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1980.05.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19851107_.39vipm2b6ds0.webp\n    link: https://v.youku.com/v_show/id_XNjEzNDIyOTE2.html?tpa=dW5pb25faWQ9MTAzMzUyXzEwMDAwMl8wMV8wMQ\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1985.11.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19860718_2.57nxdru77gg0.webp\n    link: https://v.qq.com/x/cover/enubkr6cyzh68pa/k0024z2sfx4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1986.07.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19860815_.br0n8toi0bs.webp\n    link: https://v.youku.com/v_show/id_XNjEyNTkwMDYw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.08.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19871217_.4t6ut3gh4740.webp\n    link: https://v.qq.com/x/cover/lnot50y3bsf1b1c/g00420mtyzt.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1987.12.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19890726_.56wgerj6veo0.webp\n    link: https://v.qq.com/x/cover/sg1e3flajdea6rm/w004284klfx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1989.07.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19891214_1.3spxq13lw7c0.webp\n    link: https://v.qq.com/x/cover/msbt3iret5tgwu8/m0025c8sgbz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1989.12.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19910718_1.3qf48pi9vlc.webp\n    link: https://v.qq.com/x/cover/mxwqna0wtgezxzk/s0025swakuf.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1991.07.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19920409_2.5hc6m6ld35c.webp\n    link: https://v.qq.com/x/cover/777ghazyu5gty9r/9emtBOmNpIF.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1992.04.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19920522_3.2a3t2jgyppa8.webp\n    link: https://v.qq.com/x/cover/qjpt50lj1rdoh2o/n00204bfdnp.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '1992.05.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19921001_.6xueoq1qcn40.webp\n    link: https://v.youku.com/v_show/id_XODUzMzMwNjMy.html?tpa=dW5pb25faWQ9MTAzMzUyXzEwMDAwMl8wMV8wMQ\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1992.10.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19921217_.3hb57tosdg00.webp\n    link: https://v.qq.com/x/cover/0psszmdu9yyngb4/p00159wnhh7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1992.12.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19930101_3.1892d57ctj5s.webp\n    link: https://v.qq.com/x/cover/wk8xgcdq0gdayni/p0033w2dzid.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '1993.01.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19930611_1.8raqm7exgzc.webp\n    link: https://v.qq.com/x/cover/idubxef33n76slz/g0011afrrvv.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1993.06.11 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19930701_.2aa7h5ck7lus.webp\n    link: https://v.qq.com/x/cover/e3pdjwjjbbcmsow/s0025wfup09.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1993.07.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19940331_.4y2f0kdo5ps0.webp\n    link: https://v.qq.com/x/cover/gwz8nqvc78oexgm/9skha95wrzG.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.03.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19940630_.3j0l4244ciu0.webp\n    link: https://www.bilibili.com/video/BV1HE41177c1/\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.06.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19940706_.3bjvm56r9mq0.webp\n    link: https://v.qq.com/x/cover/r6hc2kqgvnmiejn/b0016ws00gf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.07.06 75...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19941014_.1sa7s0ps23i8.webp\n    link: https://v.qq.com/x/cover/1o29ui77e85grdr/h0022ah1yrf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.10.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19950121_.2pwfqj7xweg0.webp\n    link: https://quanying.cc/vodplay/2603-1-1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.01.21 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19950204_.78h4y9j8owg0.webp\n    link: https://v.qq.com/x/cover/fse52rd4klx7qn2/v0023cbono5.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.02.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19950706_.41cvwctiggo0.webp\n    link: https://www.iqiyi.com/v_19rrj52998.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.07.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19950819_.2oau32bkf9k0.webp\n    link: https://www.iqiyi.com/v_19rrk62ru4.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.08.19 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19970801_.2qa7zucayyq0.webp\n    link: https://v.qq.com/x/cover/9n30k2ayqn1f063/f0021e9sjk4.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.08.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19970807_2.5x31v2evink0.webp\n    link: https://v.qq.com/x/cover/wwo3k53jc1k4apu/r001291y965.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1997.08.07 ....'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19971205_.1qxrs7x2oz7k.webp\n    link: https://v.qq.com/x/cover/mhl665xf8ina3t2/k0024erhg9j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19971219_.34pwxzlzr600.webp\n    link: https://v.qq.com/x/cover/t5jqhgw8pix81mw/v0024wohpa1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.19  ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19971220_.6cnibhfco380.webp\n    link: https://v.qq.com/x/cover/1weku8f23n0j12s/o0024s1dn9b.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19980131_.1qxkp01n2tsw.webp\n    link: https://555dyy9.com/vodplay/261439-3-1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1998.01.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19980605_.7h5gr4d6k7s0.webp\n    link: https://v.qq.com/x/cover/31082i4u5ovkrl0/s00211yt19u.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1998.06.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19990213_.24x4b7prenog.webp\n    link: https://v.youku.com/v_show/id_XMzYyMDUzMTU3Ng==.html?spm=a2h1n.8261147.0.0&s=cc04f3b4962411de83b1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.02.13 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19990331_.j6u8oblb80o.webp\n    link: https://www.dianyinggou.com/mov/1291843.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.03.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19990416_.8aglf1ttkk8.webp\n    link: https://v.qq.com/x/cover/znrpdjaupklt9q0/d0012mrzi68.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.04.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/19991104_.3tsmsi6caus0.webp\n    link: https://www.dianyinggou.com/mov/1305114.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.11.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20000317_1.6emakqwcl7k.webp\n    link: https://v.qq.com/x/cover/e8072tbfo3q2ccy/b0022bfu6up.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2000.03.17 -...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20010622_1.sqyzco31wm8.webp\n    link: https://v.qq.com/x/cover/o66no96ex54bksd/o0011g0p889.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2001.06.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20010712_.3rzudjwmv7o0.webp\n    link: https://www.dianyinggou.com/mov/1297747.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2001.07.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20010718_3.6filmh0xa300.webp\n    link: https://v.qq.com/x/cover/pbknv4h3h8qvo01/m0011ireuji.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2001.07.18 ....'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20020315_1.rdxpw92uc0w.webp\n    link: https://v.qq.com/x/cover/sixf5demwk331r9/g00144sraal.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2002.03.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20020925_.637t38849sw0.webp\n    link: https://v.qq.com/x/cover/g6n8noz3gmbiz4o/k0030lgbvhd.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.09.25 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20021018_.33ljm5m2m8i0.webp\n    link: https://www.dianyinggou.com/mov/1306808.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.10.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20021212_.74nvmtxwtqc0.webp\n    link: https://v.qq.com/x/cover/sg5p4qp1s6jbyj8/o0025wzpzfw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.12.12 1991...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20030131_2.6pfmpz2btnc0.webp\n    link: https://www.dianyinggou.com/mov/1307956.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2003.01.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20030603_2.1lbff08vyxr4.webp\n    link: https://v.qq.com/x/cover/xia8ahis63c8l3n/j0022gmjck0.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2003.06.03 FBI...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20031121_1.3phuilveo960.webp\n    link: https://v.qq.com/x/cover/6bxb2yc7mskrdoz/b0012l31auu.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2003.11.21 (Caribbean Sea)...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20040317_.asnjkyhmogc.webp\n    link: https://www.dianyinggou.com/mov/1291549.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.03.17 1949...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20040528_.1ba5waxnp4lc.webp\n    link: https://v.qq.com/x/cover/y60o6957ucc46qo/d0021ykrfle.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.05.28 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20040909_.1j6x6jivfbeo.webp\n    link: https://v.qq.com/x/cover/v8vd4y9td09bv4n/w0010KhTfom.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.09.09 ThunJaneJane...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20041029_1.3uhcig9a0sk0.webp\n    link: https://www.dianyinggou.com/mov/1417598.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2004.10.29 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20041223_.7fyxfvrhnmg0.webp\n    link: https://v.qq.com/x/cover/uiq0rxuywu508qr/o001241efrp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.12.23 1940...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20050623_D.nrkj47xifzk.webp\n    link: https://v.qq.com/x/cover/n16pc4sk2x29mjo.html\n    name: <div align=\"center\"><strong>D</strong></div>\n    desc: '2005.06.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20050923_.46b7cvys43g0.webp\n    link: https://www.dianyinggou.com/mov/1307023.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.09.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20051028_2.2xypiw9autg0.webp\n    link: https://www.dianyinggou.com/mov/1309045.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2005.10.28  ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20051214_.3pg2pn0i4wa0.webp\n    link: https://v.qq.com/x/cover/ztnpctziozsz5y5/l0012dovkvk.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.12.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20060202_.6gksywf2ncs0.webp\n    link: https://v.qq.com/x/cover/yp5rjb7mlt7encw/b0010MCBkOC.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.02.02 12...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20060421_.60n92fu92vs0.webp\n    link: https://www.dianyinggou.com/mov/1466007.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.04.21 RoseRose...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20060616_3.5bv9lcd1kf40.webp\n    link: https://v.qq.com/x/cover/p0wd3xfsrh2ntwo/t0011fugmo3.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2006.06.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20060707_2.19b2mcq2knls.webp\n    link: https://v.qq.com/x/cover/7qgeyfoqp4ks3dl/a0012cer60c.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2006.07.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20061027_3.5em4h4osg600.webp\n    link: https://www.dianyinggou.com/mov/1474189.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2006.10.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20061215_.4527fey60m80.webp\n    link: https://v.qq.com/x/cover/vbr7hkoebjpsrxb/a001483446x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.12.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20070316_.6ezlhv9keis0.webp\n    link: https://v.qq.com/x/cover/f8637674mcfvlhi/n0011ongycd.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.03.16 ....'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20070612_3.57bwxfkqo980.webp\n    link: https://v.qq.com/x/cover/dluq5thl5pcvtl5/f0012jbwanr.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2007.06.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20070711_1.1m9lfg8n848w.webp\n    link: https://v.qq.com/x/cover/dwt56ndspm9xm89/b0016sl196d.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2007.07.11 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20070727_.5co7qhpt0lo0.webp\n    link: https://v.qq.com/x/cover/uqxmnkbvq4qzxlv/p00216kbjh9.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.07.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20070731_.ojj867ivvcw.webp\n    link: https://www.dianyinggou.com/mov/2124724.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.07.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20071025_4.1788n8x8rbeo.webp\n    link: https://www.dianyinggou.com/mov/1922273.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2007.10.25 SWAT...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20071116_.37ixt6jiekq0.webp\n    link: https://www.dianyinggou.com/mov/2002743.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.11.16 19Florentino ArizaFermina Daza...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20080130_.2uadfxhtdf20.webp\n    link: https://v.qq.com/x/cover/f0qjcexig8rjzpo/t0014c8lrw7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.01.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20080430_1.4vxm59ui6ve0.webp\n    link: https://v.qq.com/x/cover/ciwsbplwbrje0gt/y00207r9opc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.04.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20080613_.543j36je12s0.webp\n    link: https://v.qq.com/x/cover/vwq7gmy475harwj/e002120vmdu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.06.13 Bruce Banner...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20081023_5.6yxhxmvbzlg0.webp\n    link: https://www.dianyinggou.com/mov/2350904.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2008.10.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20081212_1.2gfcvqwo724g.webp\n    link: https://v.qq.com/x/cover/rali88m8udw2x0u/z0040nen92x.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.12.12 2030...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20090415_4.7clf9y0v0iw0.webp\n    link: https://v.qq.com/x/cover/w48cjplx8eitbb4/b0021aeu6fu.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2009.04.15 1...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20090503_1.rxtqzfvzkpc.webp\n    link: https://v.qq.com/x/cover/ti9lk63vdjl2zph/q0020ad0xxi.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2009.05.03 1845...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20090624_2.4vqbu6jgfag0.webp\n    link: https://v.qq.com/x/cover/vehyv3t0ith3r55/m0016pq3xum.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2009.06.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20091023_6.6awospvzdyg0.webp\n    link: https://www.dianyinggou.com/mov/3075983.html\n    name: <div align=\"center\"><strong>6</strong></div>\n    desc: '2009.10.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20091113_2012.72smln9a7780.webp\n    link: https://v.qq.com/x/cover/92lbu34b5yvzel6/s002085eoew.html\n    name: <div align=\"center\"><strong>2012</strong></div>\n    desc: '2009.11.13 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20091127_.vvbhmb1kzo0.webp\n    link: https://v.qq.com/x/cover/nvw3lm5ouqs54cc/i00141yi8pe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.11.27 1990...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20091218_1.5g7j1dpqb5g0.webp\n    link: https://v.qq.com/x/cover/ldl1811bamppdrd/y0040wzq199.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2009.12.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20091225_.4ahgkr6ql860.webp\n    link: https://v.qq.com/x/cover/x5ul2annjsbcwh8/e00335q391x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.12.25 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20100427_2.4iouosxdjka0.webp\n    link: https://www.iqiyi.com/v_19rrj5kq94.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2010.04.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20100507_2.2th14ce6oxo0.webp\n    link: https://v.qq.com/x/cover/90qkh14unn4q8xu/v0020sk2lw4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2010.05.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20100604_.5pqa5iy6qs00.webp\n    link: https://v.qq.com/x/cover/wlb7ba6jg8x1rjm/i0014q7qgmu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.06.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20100722_.6p3bmszpv7w0.webp\n    link: https://v.qq.com/x/cover/upd61lkzbo5ko1p/8TCBch3V618.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.07.22 1976...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20100726_.40al0bhna460.webp\n    link: https://v.qq.com/x/cover/997m2fipgc6erv7/h0022fp0vk9.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.07.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20101029_7.3i4l30q4v060.webp\n    link: https://www.dianyinggou.com/mov/3878157.html\n    name: <div align=\"center\"><strong>7</strong></div>\n    desc: '2010.10.29 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110421_.s4pvn02iuu8.webp\n    link: https://v.qq.com/x/cover/h8aovufr2nix8pv/k0014qbypb2.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.21 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110508_1.161i8sg48ko0.webp\n    link: https://v.qq.com/x/cover/sjfw7p2cz0ff6m9/i0018msd6dc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.05.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110512_5.ultu3lug4v4.webp\n    link: https://v.qq.com/x/cover/unhg39knn3fjkb7/n0015hqcguy.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2011.05.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110520_4.3nnv6hfnrmw.webp\n    link: https://v.qq.com/x/cover/m63g578728dv56w/f00120mtond.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2011.05.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110721_3.3l2c3gm2a180.webp\n    link: https://v.qq.com/x/cover/lpc2ry21etiyhmn/j00169ib5er.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2011.07.21 50...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20110722_1.1kw5zk2rs8tc.webp\n    link: https://v.qq.com/x/cover/891a0latn4c5chp/r0018t60y1z.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.07.22 40...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20111102_.5i2oiygn4ls0.webp\n    link: https://v.qq.com/x/cover/km0frs6ajyrstog/t0040nwsohx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.11.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20111104_Hello.2ljhh8b8i560.webp\n    link: https://v.qq.com/x/cover/g6055xnwc815rrr/p00189kcnwz.html\n    name: <div align=\"center\"><strong>Hello</strong></div>\n    desc: '2011.11.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20120505_1.7cyipzl7fk40.webp\n    link: https://v.qq.com/x/cover/cz2szzeahb8aj1k/t0020bvo93n.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2012.05.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20121212_.48hvkrakfdm0.webp\n    link: https://v.qq.com/x/cover/tjtiyg1qe0zkhdk/k0033s8cekl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.12.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20130501_3.7an0t9dajkw0.webp\n    link: https://www.dianyinggou.com/mov/3231742.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2013.05.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20130509_.35inzx3bl780.webp\n    link: https://v.qq.com/x/cover/7jwfxt0rnldwcip/h002171rs0n.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.05.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20130712_1.2m7bap31oco0.webp\n    link: https://v.qq.com/x/cover/3xkxy6w6cdj5bh1/m0024wxr19j.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.07.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20130719_.1e3dupqlyoqo.webp\n    link: https://v.qq.com/x/cover/igy79xk26amdv6z/u00235gga3e.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.07.19 20...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20130726_6.1t6bco1hcp1c.webp\n    link: https://v.qq.com/x/cover/wiqfi65xzyj7rzz/u00137ae5kq.html\n    name: <div align=\"center\"><strong>6</strong></div>\n    desc: '2013.07.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20131017_2.1secnm1vz9og.webp\n    link: https://v.qq.com/x/cover/e7d88bfyq7trhy5/z002025ta8o.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.10.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20131024_.fv7fbdl0k3k.webp\n    link: https://v.qq.com/x/cover/skl46pqqu1e7bmx/b0021iwtlcp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.10.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20131105_.dpbkbjj5vpc.webp\n    link: https://v.qq.com/x/cover/3zpk72rw56ggqh9/b0032ruhjan.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.11.05 1942...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20131108_2.5e18zy4t86c0.webp\n    link: https://v.qq.com/x/cover/gib6uvlt5rfudib/b00202o3rv6.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.11.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20131119_.1go8ibxmktz4.webp\n    link: https://v.qq.com/x/cover/1jx00d87vzuv04a/y0024xp2k03.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.11.19 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140404_2.2ih7ercbxn80.webp\n    link: https://v.qq.com/x/cover/ix5jw6tz5ghz4dc/w00157zn04f.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.04.04 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140613_1.5qu27pbt19g0.webp\n    link: https://v.qq.com/x/cover/803p673mlosoeog/r0024n8ir8m.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2014.06.13 15...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140627_4.3w4pyw0k1ae0.webp\n    link: https://v.qq.com/x/cover/am6jxwt5rzq6kba/n0022sql7nn.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2014.06.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140801_1.5ws0util27g0.webp\n    link: https://v.qq.com/x/cover/ht4z382bhs7h94u/v0015hiufvt.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2014.08.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140830_.53gs9f6sgnk0.webp\n    link: https://v.qq.com/x/cover/1o7ou00t80apgko/w0018fcfqfv.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20140830_.5j71usgk9jk0.webp\n    link: https://v.qq.com/x/cover/yz98ta4nfjtaa6q/h0017hm4g46.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20141112_.29df38gu9hhc.webp\n    link: https://v.qq.com/x/cover/c2seabnsfozypl8/d00161yroup.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.11.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150402_1.6nxrrnr71ds0.webp\n    link: https://v.qq.com/x/cover/1bnyfttijufe0cq/m0016nwmqws.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.04.02 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150412_7.2ndjz3r9fro0.webp\n    link: https://v.qq.com/x/cover/dv71ekj7urmxkvp/a00177o0w31.html\n    name: <div align=\"center\"><strong>7</strong></div>\n    desc: '2015.04.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150512_2.3lmirdm0ui00.webp\n    link: https://v.qq.com/x/cover/w7ju7urc1x4w3jc/t0017mlfdg4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2015.05.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150610_4.9p9nubqki4w.webp\n    link: https://v.qq.com/x/cover/p06la73u7kd0hzx/z0018okldyy.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2015.06.10 21...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150717_1.2q5ardigl700.webp\n    link: https://v.qq.com/x/cover/zl6n65vy9hgrarl/u0019j97d7o.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.07.17 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20150930_.5poszdwfmng0.webp\n    link: https://v.qq.com/x/cover/ge1vg9evjfeb6l0/u0019esirxq.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.09.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20151125_.2si2r0woz5k0.webp\n    link: https://v.qq.com/x/cover/gk11svuomdpsgl4/f0027zr3wyu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.11.25 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20151224_3.thcyno7i4ls.webp\n    link: https://v.qq.com/x/cover/ra1dhxmhz872ry8/d0020tn1crm.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2015.12.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20151231_1.373342r3dfi0.webp\n    link: https://v.qq.com/x/cover/f3qroleq1lyuavy/r0019z1dctq.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.12.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20160208_.1e80e04yc7.webp\n    link: https://v.qq.com/x/cover/xg95sxi4q7zc4uo/s00429cehc3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.02.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20160506_3.gj4sdys9w48.webp\n    link: https://v.qq.com/x/cover/kzhcitf07bg2pxy/t0021qb4zzg.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2016.05.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20160929_.20yu7pcvcn34.webp\n    link: https://v.qq.com/x/cover/hrr3qneqbo22q06/o0022hmagay.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.29 DJ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20161006_.2blynol8x78k.webp\n    link: https://v.qq.com/x/cover/h188mdjtrh3up22/y0022eckcdz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.10.06 201110513...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20161106_1.2x8m6f2bttq0.webp\n    link: https://v.qq.com/x/cover/uvl5rs2euhfgoip/h0022otg21y.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2016.11.06 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20161223_.512tqgjydsg0.webp\n    link: https://v.qq.com/x/cover/2xxul4n2j8y0rxi/a0024c3w6lf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.12.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170414_8.2wrz4fs6c2s0.webp\n    link: https://v.qq.com/x/cover/l9vxrwfdjcw1q4g/h00248zwzvg.html\n    name: <div align=\"center\"><strong>8</strong></div>\n    desc: '2017.04.14 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170505_2.18nfw3y0qjts.webp\n    link: https://v.qq.com/x/cover/4gl2i78zd9idpi0/j0024zknymk.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2017.05.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170523_3.4icqouq77ac0.webp\n    link: https://www.dianyinggou.com/mov/25765735.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.05.23 2024X...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170526_5.5e3bfqsv6rc0.webp\n    link: https://v.qq.com/x/cover/00jxecd5him5kmn/v0024zl735r.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2017.05.26 320 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170623_5.139nwwr7a2n4.webp\n    link: https://v.qq.com/x/cover/qee75hz1x7m6n3s/x00243heka3.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2017.06.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170727_2.57o80lncmos0.webp\n    link: https://v.qq.com/x/cover/wi8e2p5kirdaf3j/s00242sxrne.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2017.07.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170811_.7k9bjcbk1xc0.webp\n    link: https://v.qq.com/x/cover/q5iz3d3xh80tkuf/f0026f5cjwe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.08.11 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170908_.76fl22wqdjo0.webp\n    link: https://www.dianyinggou.com/mov/3604148.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.09.08 1989...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170908_1.315wtu21t3c0.webp\n    link: https://v.qq.com/x/cover/iqwvbujgzj5obv1/d00256xcmw9.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2017.09.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20170930_.4pt20bwp1vk0.webp\n    link: https://v.qq.com/x/cover/jpjl78i91mx05bu/m0025mcvah8.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.09.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20171027_8.ws0bx91q1gw.webp\n    link: https://www.dianyinggou.com/mov/25788426.html\n    name: <div align=\"center\"><strong>8</strong></div>\n    desc: '2017.10.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20171103_3.ee4g0mqwvo.webp\n    link: https://v.qq.com/x/cover/9ikkuz4o8ejavdc/k0026phfnm2.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.11.03 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20171228_.pis8bi60vsg.webp\n    link: https://v.qq.com/x/cover/hirg4ixfe73pras/i0031e1i2s6.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.12.28 1972...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180216_2.7ktb41g308o0.webp\n    link: https://v.qq.com/x/cover/t2t7544bcr1pw8r/a0026k0ccqn.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180216_.4c3288342d40.webp\n    link: https://v.qq.com/x/cover/7casb7nes159mrl/o0026p7tyf1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180216_1.4poiyatpwem0.webp\n    link: https://v.qq.com/x/cover/7casb7nes159mrl/o0026p7tyf1.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180323_2.849a11nxtks.webp\n    link: https://v.qq.com/x/cover/f7pqur8uhmzltps/o0026dn1s6q.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.03.23 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180328_.76ijed08lt80.webp\n    link: https://www.dianyinggou.com/mov/26945085.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.03.28 2060...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180330_.1jw4mjae6mtc.webp\n    link: https://v.qq.com/x/cover/zatx0zuk94d9aru/p0026621ctf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.03.30 2045...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180511_3.218e1t7eoykg.webp\n    link: https://v.qq.com/x/cover/coqnq6i120wojq6/x0027u7i1ip.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2018.05.11 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180518_.3i3w9r647h80.webp\n    link: https://v.qq.com/x/cover/uuq07xfnxk00zqd/n0027ozlk06.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2018.05.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180705_.5atulsgmnr40.webp\n    link: https://v.qq.com/x/cover/og0eputlxwet1cn/r0028uz12ua.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180706_2.620vljvk4ws0.webp\n    link: https://v.qq.com/x/cover/sxbxm9xaakyynzp/i0028uor0n8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.07.06 FBI...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180727_.3tey2szp1de0.webp\n    link: https://v.qq.com/x/cover/z6j3ixjjcokafyc/l0028g8hyd3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.27 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180920_.1ttqzoy3kni8.webp\n    link: https://v.qq.com/x/cover/eaa7qkjmlecfnp1/q0031poot4r.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20180930_.38ckrlgaiae0.webp\n    link: https://v.qq.com/x/cover/v3sfrz8ws9ew7fw/m0029o2tv2o.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.30 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20181116_.3yen5v900260.webp\n    link: https://v.qq.com/x/cover/r9umzdo0yjelfg1/n0030x2ucda.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.11.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20181207_.29kib3wgex34.webp\n    link: https://v.qq.com/x/cover/l8r3gm1zzu5u3nk/j0030f3wne1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20181221_.1sxcs7t8utxc.webp\n    link: https://v.qq.com/x/cover/i43tfclvbhiy0kn/a0030h7pjsj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.21 B-1271987...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190205_.23sgcjwoui3k.webp\n    link: https://v.qq.com/x/cover/lt0w27nj9tpfllw/c003070g88f.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190205_1.4tsm6rfwlvk0.webp\n    link: https://v.qq.com/x/cover/3fvg46217gw800n/h0030qj4fov.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190205_.69z41xiiz8c0.webp\n    link: https://v.qq.com/x/cover/8kw2uo89elwb7as/v0030gvdd5b.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190308_1.4j98fxsrq6m0.webp\n    link: https://v.qq.com/x/cover/xyne4253g35nak3/m0031od9ekb.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.03.08 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190424_4.43xn0d2t9jc0.webp\n    link: https://v.qq.com/x/cover/v2098lbuihuqs11/u00315w81xn.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2019.04.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190510_.2i8u3yqea8e0.webp\n    link: https://v.qq.com/x/cover/2acva9s87eg7m8g/b00326bxu3h.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.05.10 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190531_2.4d5d49ez8po0.webp\n    link: https://v.qq.com/x/cover/e0bk8kf7wllv7r8/z0034ixucs8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.05.31 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190628_2.49mu68wserk0.webp\n    link: https://v.qq.com/x/cover/96sxjj429lictza/s0032ab6wjm.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.06.28 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190712_.6etl9p4u4x80.webp\n    link: https://v.qq.com/x/cover/ehqo76prcwku2oq/x0032rq56lh.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190718_.54yjo4k7fug0.webp\n    link: https://v.qq.com/x/cover/p6xvg0ft471pcj8/l00321cue7j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.18 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190809_.453dyq6rpiw0.webp\n    link: https://v.qq.com/x/cover/hswhwhxw05n7jso/r0032c9m0dx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.08.09 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190913_1.5velexcpmc00.webp\n    link: https://v.qq.com/x/cover/5vfbg80goc4mkay/f003249hywz.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.09.13 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190930_.3zg98n0uivs0.webp\n    link: https://v.qq.com/x/cover/rj8uc45tm8a17wm/z0032udy82e.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.09.30 20185143U8633...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20190930_.7ij3a8ehm8s0.webp\n    link: https://v.qq.com/x/cover/u9z5s6rt4j30fjh/a0032xozepq.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.09.30 196015...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20191220_4.72sq59700a40.webp\n    link: https://v.qq.com/x/cover/m5zzglrbt5zdv6d/v0033476tnr.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2019.12.20 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20200807_.20vos7dxi880.webp\n    link: https://v.qq.com/x/cover/mzc0020067i3zrk/m0034gnwg46.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.08.07 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20200821_.6yoxjmdyq1c0.webp\n    link: https://v.qq.com/x/cover/9f8tpq2gly9p5ts/r0034rmwzpe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.08.21 193788262524420...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20210115_.524tzo2o4ws0.webp\n    link: https://v.qq.com/x/cover/mzc00200zflvank/w0035jtpsb1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.01.15 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20210212_3.qufv8t4xeeo.webp\n    link: https://v.qq.com/x/cover/tk3l27paq7sqr0z/p0036mrfj8h.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2021.02.12 CRIMASTER...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20210212_.2srtocmnxmi0.webp\n    link: https://v.qq.com/x/cover/mzc00200vx37bvf/c0035hwvmv4.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.12 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20210326_.5mz13avkmqc0.webp\n    link: https://v.qq.com/x/cover/babks6vmrn25n9c/l0037j1ebig.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.03.26 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20211215_3.3v0n9vtsbl20.webp\n    link: https://www.dianyinggou.com/mov/26933210.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2021.12.15 4...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20220201_.13cf5yz9w6r4.webp\n    link: https://v.qq.com/x/cover/mzc002001uvqi5d/i0042h9vxm3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.02.01 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20220305_.5ck2uwdp3fo0.webp\n    link: https://v.qq.com/x/cover/mzc00200owkcck4/f0043bpxr53.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.03.05 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20220624_.1sh7ga7oegjk.webp\n    link: https://v.qq.com/x/cover/mzc00200zlh907s/j0044wpymeo.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.06.24 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20220729_.4a7wqnc78wu0.webp\n    link: https://v.qq.com/x/cover/mzc002000y0q8uy/r0044rrsowz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.07.29 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20220930_.60ctbuk9gw00.webp\n    link: https://v.qq.com/x/cover/mzc00200qv892j3/k0045ztu6tu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.09.30  ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20221216_2.2z8zjag8ga40.webp\n    link: https://v.qq.com/x/cover/mzc00200qv892j3/k0045ztu6tu.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2022.12.16 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20230122_2.5zpvevc8ab40.webp\n    link: https://v.qq.com/x/cover/mzc002008v8rka3/d0046v4pddg.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2023.01.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20230122_.1x5dt1w3vbk0.webp\n    link: https://v.qq.com/x/cover/mzc00200bd3gkta/b0046wa81dm.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.01.22 ...'\n\n  - img: https://ZhengTiger.github.io/picx-images-hosting/99-/01-/04-/20230428_.stmosyfuy4g.webp\n    link: \n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.04.28 ...'\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n234\n235\n236\n237\n238\n239\n240\n241\n242\n243\n244\n245\n246\n247\n248\n249\n250\n251\n252\n253\n254\n255\n256\n257\n258\n259\n260\n261\n262\n263\n264\n265\n266\n267\n268\n269\n270\n271\n272\n273\n274\n275\n276\n277\n278\n279\n280\n281\n282\n283\n284\n285\n286\n287\n288\n289\n290\n291\n292\n293\n294\n295\n296\n297\n298\n299\n300\n301\n302\n303\n304\n305\n306\n307\n308\n309\n310\n311\n312\n313\n314\n315\n316\n317\n318\n319\n320\n321\n322\n323\n324\n325\n326\n327\n328\n329\n330\n331\n332\n333\n334\n335\n336\n337\n338\n339\n340\n341\n342\n343\n344\n345\n346\n347\n348\n349\n350\n351\n352\n353\n354\n355\n356\n357\n358\n359\n360\n361\n362\n363\n364\n365\n366\n367\n368\n369\n370\n371\n372\n373\n374\n375\n376\n377\n378\n379\n380\n381\n382\n383\n384\n385\n386\n387\n388\n389\n390\n391\n392\n393\n394\n395\n396\n397\n398\n399\n400\n401\n402\n403\n404\n405\n406\n407\n408\n409\n410\n411\n412\n413\n414\n415\n416\n417\n418\n419\n420\n421\n422\n423\n424\n425\n426\n427\n428\n429\n430\n431\n432\n433\n434\n435\n436\n437\n438\n439\n440\n441\n442\n443\n444\n445\n446\n447\n448\n449\n450\n451\n452\n453\n454\n455\n456\n457\n458\n459\n460\n461\n462\n463\n464\n465\n466\n467\n468\n469\n470\n471\n472\n473\n474\n475\n476\n477\n478\n479\n480\n481\n482\n483\n484\n485\n486\n487\n488\n489\n490\n491\n492\n493\n494\n495\n496\n497\n498\n499\n500\n501\n502\n503\n504\n505\n506\n507\n508\n509\n510\n511\n512\n513\n514\n515\n516\n517\n518\n519\n520\n521\n522\n523\n524\n525\n526\n527\n528\n529\n530\n531\n532\n533\n534\n535\n536\n537\n538\n539\n540\n541\n542\n543\n544\n545\n546\n547\n548\n549\n550\n551\n552\n553\n554\n555\n556\n557\n558\n559\n560\n561\n562\n563\n564\n565\n566\n567\n568\n569\n570\n571\n572\n573\n574\n575\n576\n577\n578\n579\n580\n581\n582\n583\n584\n585\n586\n587\n588\n589\n590\n591\n592\n593\n594\n595\n596\n597\n598\n599\n600\n601\n602\n603\n604\n605\n606\n607\n608\n609\n610\n611\n612\n613\n614\n615\n616\n617\n618\n619\n620\n621\n622\n623\n624\n625\n626\n627\n628\n629\n630\n631\n632\n633\n634\n635\n636\n637\n638\n639\n640\n641\n642\n643\n644\n645\n646\n647\n648\n649\n650\n651\n652\n653\n654\n655\n656\n657\n658\n659\n660\n661\n662\n663\n664\n665\n666\n667\n668\n669\n670\n671\n672\n673\n674\n675\n676\n677\n678\n679\n680\n681\n682\n683\n684\n685\n686\n687\n688\n689\n690\n691\n692\n693\n694\n695\n696\n697\n698\n699\n700\n701\n702\n703\n704\n705\n706\n707\n708\n709\n710\n711\n712\n713\n714\n715\n716\n717\n718\n719\n720\n721\n722\n723\n724\n725\n726\n727\n728\n729\n730\n731\n732\n733\n734\n735\n736\n737\n738\n739\n740\n741\n742\n743\n744\n745\n746\n747\n748\n749\n750\n751\n752\n753\n754\n755\n756\n757\n758\n759\n760\n761\n762\n763\n764\n765\n766\n767\n768\n769\n770\n771\n772\n773\n774\n775\n776\n777\n778\n779\n780\n781\n782\n783\n784\n785\n786\n787\n788\n789\n790\n791\n792\n793\n794\n795\n796\n797\n798\n799\n800\n801\n802\n803\n804\n805\n806\n807\n808\n809\n810\n811\n812\n813\n814\n815\n816\n817\n818\n819\n820\n821\n822\n823\n824\n825\n826\n827\n828\n829\n830\n831\n832\n833\n834\n835\n836\n837\n838\n839\n840\n841\n842\n843\n844\n845\n846\n847\n848\n849\n850\n851\n852\n853\n854\n855\n856\n857\n858\n859\n860\n861\n862\n863\n864\n865\n866\n867\n868\n869\n870\n871\n872\n873\n874\n875\n876\n877\n878\n879\n880\n881\n882\n883\n884\n885\n886\n887\n888\n889\n890\n891\n892\n893\n894\n895\n896\n897\n898\n899\n900\n901\n902\n903\n904\n905\n906\n907\n908\n909\n910\n911\n912\n913\n914\n915\n916\n917\n918\n919\n920\n921\n922\n923\n924\n925\n926\n927\n928\n929\n930\n931\n932\n933\n934\n935\n936\n937\n938\n939\n940\n941\n942\n943\n944\n945\n946\n947\n948\n949\n950\n951\n952\n953\n954\n955\n956\n957\n958\n959\n960\n961\n962\n963\n964\n965\n966\n967\n968\n969\n970\n971\n972\n973\n974\n975\n976\n977\n978\n979\n980\n981\n982\n983\n984\n985\n986\n987\n",normalizedContent:"\n\n\n\n1993.10.16 ...\n\neva \n\n\n\n1995.10.04 2000913\n\n\n\n\n\n1996.01.08 ...\n\n\n\n\n\n1999.03.07 ...\n\n\n\n\n\n1999.10.20 ...\n\n\n\n\n\n2000.10.16 ...\n\n\n\n\n\n2002.10.03 ...\n\n\n\n\n\n2005.10.22 ...\n\n\n\n\n\n2007.02.18 ...\n\n\n\n\n\n2007.10.04 ...\n\n\n\n\n\n2008.01.01 ...\n\n\n\n\n\n2011.04.01 boss...\n\n\n\n\n\n2011.10.13 ghq\n\n\n\n\n\n2012.07.07 2022sword art online\n\n\n\n\n\n2013.04.07 107743\n\n\n\n\n\n2014.02.14 ...\n\n\n\n\n\n2015.03.05 ...\n\n\n\n\n\n2016.06.24 ...\n\n\n\n\n\n2016.07.07 ...\n\n\n\n\n\n2016.07.08  19...\n\n\n\n\n\n2016.09.02 ...\n\n\n\n\n\n2017.01.07 41011...\n\n\n\n\n\n2017.04.07 ...\n\n\n\n\n\n2017.04.28 ...\n\n\n\n\n\n2017.06.01 ...\n\n\n\n\n\n2018.01.20 ...\n\n\n\n\n\n2018.04.25 ...\n\n\n\n\n\n2018.07.08 372...\n\n\n\n\n\n2018.07.09 ...\n\n\n\n\n\n2018.10.05 ...\n\n\n\n\n\n2018.12.20 ...\n\n\n\n\n\n2019.01.16 ...\n\n\n\n\n\n2019.02.06 ...\n\n\n\n\n\n2019.04.07 \n\ndr.stone\n\n\n\n2019.07.05 ...\n\nre:zero\n\n\n\n2020.01.01 \n\n\n\n\n\n2020.03.08 ...\n\n\n\n\n\n2020.05.18 \n\n\n\n\n\n2020.10.03 ...\n\n\n\n\n\n2021.10.15 ...\n\n\n\n\n\n2022.12.10 400...\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 4\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/19931016_.1tf9x3vp0ra8.webp\n    link: https://www.iqiyi.com/a_19rrhb244t.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1993.10.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/19951004_.54xk80t0jac0.webp\n    link: https://www.bilibili.com/bangumi/media/md1635\n    name: <div align=\"center\"><strong>eva </strong></div>\n    desc: '1995.10.04 2000913'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/19960108_.mnzk5bq0bao.webp\n    link: https://www.bilibili.com/bangumi/media/md28228775\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1996.01.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/19990307_.6oiuc1sl1q00.webp\n    link: https://www.iqiyi.com/a_19rrhb1dmp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.03.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/19991020_.7kjnqb9t11k0.webp\n    link: https://www.iqiyi.com/a_19rrhb3xvl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.10.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20001016_.4giytspav300.webp\n    link: https://www.bilibili.com/bangumi/media/md28222083\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2000.10.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20021003_.137hxhreolyo.webp\n    link: https://list.youku.com/show/id_zcc001f06962411de83b1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.10.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20051022_.49caw1b25w00.webp\n    link: https://www.bilibili.com/bangumi/media/md1715\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.10.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20070218_.3vjqy34u6ey0.webp\n    link: https://v.qq.com/x/cover/mzc002001kt6n30.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.02.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20071004_.j1fa5f529tc.webp\n    link: https://www.bilibili.com/bangumi/media/md28234156\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.10.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20080101_.6brhw733lmc.webp\n    link: https://tv.cctv.com/2013/03/13/vida1363159660527562.shtml\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.01.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20110401_.u8jats1l5bk.webp\n    link: https://so.youku.com/search_video/q_%e8%b6%85%e5%85%bd%e6%ad%a6%e8%a3%85?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.01 boss...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20111013_.2n2iyugy5ik0.webp\n    link: https://www.bilibili.com/bangumi/media/md28229899\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.10.13 ghq'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20120707_.3eafv3bbwco0.webp\n    link: https://www.bilibili.com/bangumi/media/md24755609\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.07.07 2022sword art online'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20130407_.6c2jaojmt800.webp\n    link: https://www.5yju.com/detail-4710.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.04.07 107743'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20140214_.2f90npse2tes.webp\n    link: https://www.bilibili.com/bangumi/media/md348\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.02.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20150305_.3ikjbq28vfc0.webp\n    link: https://www.bilibili.com/bangumi/media/md1689\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.03.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20160624_.3fl7lj6uj6u0.webp\n    link: https://v.qq.com/x/search/?q=%e6%ad%a6%e5%ba%9a%e7%ba%aa+%e7%ac%ac1%e5%ad%a3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.06.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20160707_.54jz1xk32vk0.webp\n    link: https://www.bilibili.com/bangumi/media/md5045\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20160708_.2jv84aqhe620.webp\n    link: https://www.bilibili.com/bangumi/media/md5050\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.08  19...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20160902_.3efd3k56a4i0.webp\n    link: https://v.qq.com/x/search/?q=%e5%85%a8%e8%81%8c%e6%b3%95%e5%b8%88+%e7%ac%ac%e4%b8%80%e5%ad%a3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20170107_.19vngp2vjmg0.webp\n    link: https://v.qq.com/x/cover/xvlhj7cglyfwx8x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.01.07 41011...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20170407_.2kbo32579ro0.webp\n    link: https://v.qq.com/x/search/?q=%e5%85%a8%e8%81%8c%e9%ab%98%e6%89%8b+%e7%ac%ac1%e5%ad%a3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.04.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20170428_.1rnkhuwvgy1s.webp\n    link: https://www.bilibili.com/bangumi/media/md5855\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.04.28 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20170601_.2jj44cjb1w00.webp\n    link: https://v.qq.com/x/search/?q=%e9%9b%84%e5%85%b5%e8%bf%9e1%e5%a4%a9%e6%b2%b3%e6%88%98%e5%bd%b9&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.06.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20180120_.1da1j48ygehs.webp\n    link: https://v.qq.com/x/cover/m441e3rjq9kwpsc/m00253deqqo.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20180425_.6ts6stu55t40.webp\n    link: https://www.bilibili.com/bangumi/media/md6360\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.04.25 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20180708_.13myg5syhgf4.webp\n    link: https://www.bilibili.com/bangumi/media/md102392\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.08 372...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20180709_.3ekdvmwqgz00.webp\n    link: https://v.qq.com/x/cover/k4mutekomtrdbux.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20181005_.1kfq0sa12ue8.webp\n    link: https://v.qq.com/x/search/?q=%e6%98%9f%e8%be%b0%e5%8f%98+%e7%ac%ac1%e5%ad%a3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.10.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20181220_.3mvps241qbu0.webp\n    link: https://v.qq.com/x/search/?q=%e9%9b%aa%e9%b9%b0%e9%a2%86%e4%b8%bb&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20190116_.2blgxmin20sg.webp\n    link: https://v.qq.com/x/cover/2w2legt0g8z26al/u00296p6t50.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.01.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20190206_.4zsqt9ufkog0.webp\n    link: https://v.qq.com/x/search/?q=%e4%b8%87%e7%95%8c%e7%a5%9e%e4%b8%bb&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20190407_.5cy5r0x6rf40.webp\n    link: https://www.bilibili.com/bangumi/media/md22718131\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.04.07 '\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20190705_.34ese4dynua0.webp\n    link: https://www.bilibili.com/bangumi/media/md28221387\n    name: <div align=\"center\"><strong>dr.stone</strong></div>\n    desc: '2019.07.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20200101_.3qz0nn8hz6y0.webp\n    link: https://www.bilibili.com/bangumi/media/md28224394\n    name: <div align=\"center\"><strong>re:zero</strong></div>\n    desc: '2020.01.01 '\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20200308_.5f9r7d2tgh00.webp\n    link: https://v.qq.com/x/search/?q=%e6%ad%a6%e7%a5%9e%e4%b8%bb%e5%ae%b0&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.03.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20200518_.3tsj8vulnz80.webp\n    link: https://v.qq.com/x/search/?q=%e6%97%a0%e4%b8%8a%e7%a5%9e%e5%b8%9d&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.05.18 '\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20201003_.29yakemsnocg.webp\n    link: https://www.bilibili.com/bangumi/media/md28229899\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.10.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20211015_.59ztgxeyhxs0.webp\n    link: https://www.bilibili.com/bangumi/media/md28235154\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.10.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/01-/20221210_.ii0z3wwyeqo.webp\n    link: https://www.bilibili.com/bangumi/media/md4315402\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.12.10 400...'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n\n\n\n\n\n\n1986.02.09 ...\n\n2002\n\n\n\n2002.12.01 ...\n\n\n\n\n\n2005.01.24 ...\n\n\n\n\n\n2005.12.16 ...\n\n2006\n\n\n\n2006.03.14 ...\n\n\n\n\n\n2008.04.08 2060...\n\n2010\n\n\n\n2010.05.02 ...\n\n\n\n\n\n2011.04.17 starkeddard star...\n\n\n\n\n\n2012.09.29 ...\n\n\n\n\n\n2012.10.01 2090...\n\n2013\n\n\n\n2013.02.06 ...\n\n\n\n\n\n2013.10.23 21...\n\n\n\n\n\n2014.08.25 cib...\n\n\n\n\n\n2015.02.26 70...\n\n\n\n\n\n2015.06.09 ...\n\n1988\n\n\n\n2015.11.06 386881971...\n\n\n\n\n\n2015.11.29 ...\n\n\n\n\n\n2016.04.08 ...\n\n\n\n\n\n2016.05.23 ...\n\n2017\n\n\n\n2017.01.09 ...\n\n\n\n\n\n2017.03.28 ...\n\n\n\n\n\n2017.10.12 ...\n\n\n\n\n\n2017.11.09 19...\n\n\n\n\n\n2017.12.13 ...\n\n\n\n\n\n2018.01.12 ...\n\n\n\n\n\n2018.07.06 ...\n\n\n\n\n\n2018.08.02 ...\n\n\n\n\n\n2018.08.07 ...\n\n\n\n\n\n2018.09.03 ...\n\n\n\n\n\n2018.10.31 ...\n\n\n\n\n\n2019.03.01 ...\n\n\n\n\n\n2019.06.09 ...\n\n\n\n\n\n2019.06.27 ...\n\n\n\n\n\n2019.07.24 ...\n\n\n\n\n\n2019.11.12 ...\n\n\n\n\n\n2020.01.30 ...\n\n\n\n\n\n2020.05.18 ...\n\n\n\n\n\n2020.07.14 ...\n\n\n\n\n\n2020.07.17 ...\n\n\n\n\n\n2020.12.16 ...\n\n\n\n\n\n2021.02.01 ...\n\n\n\n\n\n2021.02.05 ...\n\n\n\n\n\n2021.08.09 ...\n\n\n\n\n\n2021.12.15 ...\n\n\n\n\n\n2023.01.15 2007...\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 4\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/19860209_.79ogmd93d1w0.webp\n    link: https://www.bilibili.com/bangumi/media/md28229051\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.02.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20021201_2002.cd67c6eixwo.webp\n    link: https://v.qq.com/x/search/?q=%e5%80%9a%e5%a4%a9%e5%b1%a0%e9%be%99%e8%ae%b0\n    name: <div align=\"center\"><strong>2002</strong></div>\n    desc: '2002.12.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20050124_.388fvzqltx40.jpg\n    link: https://so.youku.com/search_video/q_%e4%bb%99%e5%89%91%e5%a5%87%e4%be%a0%e4%bc%a01?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.01.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20051216_.2k6naze37oe0.webp\n    link: https://www.iqiyi.com/v_19rr8rigrg.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.12.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20060314_2006.1s88vtlv7jmo.webp\n    link: https://www.iqiyi.com/a_19rrhbznr5.html\n    name: <div align=\"center\"><strong>2006</strong></div>\n    desc: '2006.03.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20080408_.jfb32ho0ik8.jpg\n    link: https://www.bilibili.com/bangumi/media/md21317\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.04.08 2060...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20100502_2010.5mjdhel0ios0.webp\n    link: https://v.qq.com/x/search/?q=%e6%96%b0%e4%b8%89%e5%9b%bd&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong>2010</strong></div>\n    desc: '2010.05.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20110417_.5p9g0bf0xu00.webp\n    link: https://v.qq.com/x/search/?q=%e6%9d%83%e5%8a%9b%e7%9a%84%e6%b8%b8%e6%88%8f+%e7%ac%ac%e4%b8%80%e5%ad%a3&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.17 starkeddard star...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20120929_.1ediygtve9ds.jpg\n    link: https://www.iqiyi.com/a_19rrk60bih.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.09.29 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20121001_.1jlhspre7abk.jpg\n    link: https://www.iqiyi.com/a_df79g6zw2d.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.10.01 2090...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20130206_2013.5flyqx85ql00.webp\n    link: https://v.qq.com/x/search/?q=%e7%ac%91%e5%82%b2%e6%b1%9f%e6%b9%962012&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong>2013</strong></div>\n    desc: '2013.02.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20131023_.2196eobkgim8.webp\n    link: https://v.qq.com/x/search/?q=%e6%88%91%e6%98%af%e7%89%b9%e7%a7%8d%e5%85%b5%e4%b9%8b%e7%81%ab%e5%87%a4%e5%87%b0&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.10.23 21...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20140825_.2fqg2kkq5xes.webp\n    link: https://so.youku.com/search_video/q_%e4%bd%bf%e5%be%92%e8%a1%8c%e8%80%85?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.25 cib...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20150226_.2jlrtsu3xj40.jpg\n    link: https://v.qq.com/x/search/?q=%e5%b9%b3%e5%87%a1%e7%9a%84%e4%b8%96%e7%95%8c&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.02.26 70...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20150609_.4zh82za05gc0.jpg\n    link: https://v.qq.com/x/search/?q=%e8%8a%b1%e5%8d%83%e9%aa%a82015&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.06.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20151106_1988.4y7jc8pdglo0.webp\n    link: https://www.iqiyi.com/a_19rrhbhrod.html\n    name: <div align=\"center\"><strong>1988</strong></div>\n    desc: '2015.11.06 386881971...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20151129_.7gp4byilqhw0.jpg\n    link: https://so.youku.com/search_video/q_%e7%a7%a6%e6%97%b6%e6%98%8e%e6%9c%88%20tv%e7%89%88?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.11.29 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20160408_.4wbsl0c5qjg0.jpg\n    link: https://www.iqiyi.com/a_19rrhbd6ad.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.04.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20160523_.4l6y3ciqseg0.jpg\n    link: https://www.iqiyi.com/a_19rrhalg9p.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.05.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20170109_2017.2ue452phb380.jpg\n    link: https://www.iqiyi.com/a_19rrhatc1p.html\n    name: <div align=\"center\"><strong>2017</strong></div>\n    desc: '2017.01.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20170328_.3hvq2621f0o0.jpg\n    link: https://so.youku.com/search_video/q_%e4%ba%ba%e6%b0%91%e7%9a%84%e5%90%8d%e4%b9%89%20dvd%e7%89%88?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.03.28 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20171012_.3z5x1nb00wu0.webp\n    link: https://www.bilibili.com/bangumi/media/md28223924\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.10.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20171109_.e9b50ahf5g0.webp\n    link: https://v.qq.com/x/search/?q=%e8%87%b4%e6%88%91%e4%bb%ac%e5%8d%95%e7%ba%af%e7%9a%84%e5%b0%8f%e7%be%8e%e5%a5%bd\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.11.09 19...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20171213_.3tgisjll9j60.webp\n    link: https://v.qq.com/x/search/?q=%e6%9f%92%e4%b8%aa%e6%88%91&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.12.13 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20180112_.1rb9eoupn9sw.webp\n    link: https://v.qq.com/x/search/?q=%e6%81%8b%e7%88%b1%e5%85%88%e7%94%9f&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20180706_.k85olwpnme8.jpg\n    link: https://v.qq.com/x/search/?q=%e7%8c%8e%e6%af%92%e4%ba%ba&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20180802_.4ino5hb73go0.jpg\n    link: https://v.qq.com/x/search/?q=%e9%a6%99%e8%9c%9c%e6%b2%89%e6%b2%89%e7%83%ac%e5%a6%82%e9%9c%9c&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.08.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20180807_.6ziki0x64p80.webp\n    link: https://so.youku.com/search_video/q_%e6%ad%a6%e5%8a%a8%e4%b9%be%e5%9d%a4%e4%b9%8b%e8%8b%b1%e9%9b%84%e5%87%ba%e5%b0%91%e5%b9%b4?searchfrom=1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.08.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20180903_.2w48ukotxp40.jpg\n    link: https://v.qq.com/x/search/?q=%e6%96%97%e7%a0%b4%e8%8b%8d%e7%a9%b9%e7%94%b5%e8%a7%86%e5%89%a7%e7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20181031_.3u5j5bhocjm0.jpg\n    link: https://v.qq.com/x/search/?q=%e5%b0%86%e5%a4%9c&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.10.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20190301_.4po188rk3tw0.jpg\n    link: https://v.qq.com/x/search/?q=%e9%83%bd%e6%8c%ba%e5%a5%bd&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.03.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20190609_.72uauzsw8fw0.webp\n    link: https://v.qq.com/x/search/?q=%e5%b0%91%e5%b9%b4%e6%b4%be&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.06.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20190627_.5xgcebh2uso0.webp\n    link: https://v.qq.com/x/search/?q=%e9%99%88%e6%83%85%e4%bb%a4&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.06.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20190724_.1siafz4zj39c.jpg\n    link: https://v.qq.com/x/search/?q=%e5%85%a8%e8%81%8c%e9%ab%98%e6%89%8b&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20191112_.5ch827ubz780.webp\n    link: https://v.qq.com/x/search/?q=%e4%bb%8e%e5%89%8d%e6%9c%89%e5%ba%a7%e7%81%b5%e5%89%91%e5%b1%b1&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.11.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20200130_.54b413a0gow0.jpg\n    link: https://v.qq.com/x/search/?q=%e5%a4%a7%e4%b8%bb%e5%ae%b0%e7%94%b5%e8%a7%86%e5%89%a7%e7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.01.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20200518_.1ovq5z8el1s0.jpg\n    link: https://v.qq.com/x/search/?q=%e4%bc%a0%e9%97%bb%e4%b8%ad%e7%9a%84%e9%99%88%e8%8a%8a%e8%8a%8a&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.05.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20200714_.lv4ylx4ftc0.jpg\n    link: https://www.iqiyi.com/a_19rrhlatf5.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.07.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20200717_.5f5boxy06oo0.jpg\n    link: https://v.qq.com/x/search/?q=%e4%b8%89%e5%8d%81%e8%80%8c%e5%b7%b2&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.07.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20201216_.3bi3t5ya18c0.webp\n    link: https://v.qq.com/x/search/?q=%e9%bb%91%e7%99%bd%e7%a6%81%e5%8c%ba&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.12.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20210201_.eyf1e4ab8ls.png\n    link: https://www.iqiyi.com/a_19rrhymcsx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20210205_.28qkpusy6xog.webp\n    link: https://v.qq.com/x/search/?q=%e6%96%97%e7%bd%97%e5%a4%a7%e9%99%86%e7%94%b5%e8%a7%86%e5%89%a7%e7%89%88&stag=2&smartbox_ab=4828328\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20210809_.3tsj8vv1sqg0.webp\n    link: https://v.qq.com/x/search/?q=%e6%89%ab%e9%bb%91%e9%a3%8e%e6%9a%b4&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.08.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20211215_.c9tomc12dk0.webp\n    link: https://v.qq.com/x/search/?q=%e9%9b%aa%e4%b8%ad%e6%82%8d%e5%88%80%e8%a1%8c&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.12.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/02-/20230115_.4ex9mpvnnh80.jpg\n    link: https://v.qq.com/x/search/?q=%e4%b8%89%e4%bd%93&stag=0&smartbox_ab=\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.01.15 2007...'\n\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n\n\n\n\n\n\n1984.03.11 7...\n\n\n\n\n\n1986.08.02 ...\n\n\n\n\n\n1988.04.16 14...\n\n\n\n\n\n1988.04.16 ...\n\n\n\n\n\n1994.06.15 ...\n\n\n\n\n\n1997.07.12 ...\n\n\n\n\n\n2001.07.20 ...\n\n1\n\n\n\n2002.03.15 ...\n\n1\n\n\n\n2003.05.30 ...\n\n2\n\n\n\n2006.06.09 ...\n\n\n\n\n\n2007.03.03 ...\n\n1\n\n\n\n2008.06.06 ...\n\n\n\n\n\n2008.06.27 2805...\n\n\n\n\n\n2008.07.19 ...\n\n3\n\n\n\n2009.07.09 mannydiego...\n\n\n\n\n\n2009.08.04 ...\n\n1\n\n\n\n2010.03.18 ...\n\n1\n\n\n\n2010.07.09 ...\n\n1\n\n\n\n2011.04.08 ...\n\n4\n\n\n\n2012.07.27 ...\n\n1\n\n\n\n2013.04.20 grug...\n\n\n\n\n\n2013.05.31 15...\n\n2\n\n\n\n2013.07.03 ...\n\n\n\n\n\n2013.07.20 ...\n\n1\n\n\n\n2013.11.27 ...\n\n2\n\n\n\n2014.04.11 ...\n\n2\n\n\n\n2014.08.14 ...\n\n\n\n\n\n2014.11.07 +...\n\n\n\n\n\n2015.07.10 ...\n\n\n\n\n\n2015.10.16 ...\n\n\n\n\n\n2016.03.04 ...\n\n\n\n\n\n2016.07.08 ...\n\n5\n\n\n\n2016.07.22 ...\n\n\n\n\n\n2016.08.26 ...\n\n\n\n\n\n2016.09.17 ...\n\n3\n\n\n\n2017.07.07 ...\n\n\n\n\n\n2017.11.24 ...\n\n\n\n\n\n2018.01.16 ...\n\n3\n\n\n\n2019.01.03 ...\n\n\n\n\n\n2019.07.19 ...\n\n\n\n\n\n2019.07.26 ...\n\n2\n\n\n\n2019.11.22 ...\n\n1\n\n\n\n2020.08.06 ...\n\n\n\n\n\n2020.10.01 ...\n\n2\n\n\n\n2020.11.27 ...\n\n2\n\n\n\n2021.02.12 ...\n\n2\n\n\n\n2021.07.23 ...\n\n\n\n\n\n2022.08.19 ...\n\n\n\n\n\n2022.11.11 17...\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 4\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19840311_.5s5bo6s5it80.webp\n    link: https://yhdm6.top/index.php/vod/play/id/2941/sid/3/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1984.03.11 7...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19860802_.3dtwcq95uti0.webp\n    link: https://yhdm6.top/index.php/vod/play/id/13098/sid/2/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.08.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19880416_.5fotjlnro5o0.webp\n    link: https://yhdm6.top/index.php/vod/play/id/15833/sid/1/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1988.04.16 14...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19880416_.3crzn4o5wrs0.webp\n    link: https://www.iqiyi.com/v_19rr7r4y5g.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1988.04.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19940615_.4xwlvm0mblw0.webp\n    link: https://v.qq.com/x/cover/lh4cj69w87hfodz/g0022161p32.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.06.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/19970712_.5v2hez1s3e00.webp\n    link: https://yhdm6.top/index.php/vod/play/id/2978/sid/3/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.07.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20010720_.12vl0rpxlmcg.webp\n    link: https://yhdm6.top/index.php/vod/play/id/4905/sid/4/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2001.07.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20020315_1.5jjr4e6a9m00.webp\n    link: https://v.qq.com/x/cover/thrjyzyxu8wkgel/c0020pocfxn.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2002.03.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20030530_1.6npsnasdflw0.webp\n    link: https://v.qq.com/x/cover/5frrueiimmg47zt/a0012tsyz90.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2003.05.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20060609_2.72kcughb67c0.webp\n    link: https://v.qq.com/x/cover/c5gf072ldgv9yv4/d0020h4v7fy.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2006.06.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20070303_5.3g9j51t6nyq0.webp\n    link: https://www.bilibili.com/bangumi/media/md2688\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.03.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20080606_1.36fr0nofrvy0.webp\n    link: https://www.iqiyi.com/v_19rrnppf4o.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.06.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20080627_.2mwbngr1wwi0.webp\n    link: https://v.qq.com/x/cover/vqeduf4egjyrc8w/v00121f4a8u.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.06.27 2805...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20080719_.2oe6ls26p660.webp\n    link: https://www.iqiyi.com/v_1htw2y3hjpw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.07.19 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20090709_3.vofcyn3v1ps.webp\n    link: https://v.qq.com/x/cover/d5mkraazz53p6gt/h00202lepiv.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2009.07.09 mannydiego...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20090804_.65cwlcgoh7s0.webp\n    link: https://v.qq.com/x/cover/w9gk2xdkyvez6yp/a0012lgk0o8.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.08.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20100318_1.5la2x412id80.webp\n    link: https://v.qq.com/x/cover/3imeqzq7nzw130g/h00202vxfcc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2010.03.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20100709_1.4jlxpjihq940.webp\n    link: https://v.qq.com/x/cover/oylsefopm3zevpj/j0013tua8os.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2010.07.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20110408_1.5l2avz1fpeo0.webp\n    link: https://v.qq.com/x/cover/zb24obs27w4mfdw/k0020jbkc1h.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.04.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20120727_4.xsmyljyzfy8.webp\n    link: https://v.qq.com/x/cover/qszw1ugyfz10izl/f0020g3u5lt.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2012.07.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20130420_1.2st3r9p88xw0.webp\n    link: https://v.qq.com/x/cover/m4ry5h1naszjlzh/t0012d28gss.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.04.20 grug...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20130531_.4bai9idgntk0.webp\n    link: https://www.bilibili.com/bangumi/media/md2546\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.05.31 15...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20130703_2.1zsawgnf9dvk.webp\n    link: https://v.qq.com/x/cover/ksbt3iklh13mawf.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.07.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20130720_.1v0z4701y3y8.webp\n    link: https://yhdm6.top/index.php/vod/play/id/9933/sid/4/nid/1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.07.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20131127_1.7eao2yh06sg0.webp\n    link: https://v.qq.com/x/cover/bv8m91i5jj1w1ww/k0014rpm1vd.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.11.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20140411_2.3a37750gzlw0.webp\n    link: https://v.qq.com/x/cover/vatxsyi3vgs4ul0/o0024gk1tb6.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.04.11 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20140814_2.6zdxwe6aoj80.webp\n    link: https://v.qq.com/x/cover/z46ef2t37pfuojr/p00157g180n.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.08.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20141107_.6g8sik1uxho0.webp\n    link: https://v.qq.com/x/cover/hjdb95f8lxcyyew/o0016yn71wh.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.11.07 +...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20150710_.l7b55jgk39s.webp\n    link: https://v.qq.com/x/cover/3o56brnhunm5bwx/x0018dcg7rm.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.07.10 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20151016_.53ghml9vurg0.webp\n    link: https://www.iqiyi.com/v_19rrkdlco0.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.10.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20160304_.3raxkduicgo0.webp\n    link: https://v.qq.com/x/cover/fhe2h7sop52qzza/l0020h13orj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.03.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20160708_.2jaud65cuow0.webp\n    link: https://v.qq.com/x/cover/we35g3aduiwkudp/u0021lcrkp2.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.07.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20160722_5.38zblcp4w1w0.webp\n    link: https://v.qq.com/x/cover/ofbgmce1glm7kwe/v0033fnq85s.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2016.07.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20160826_.3utr585ulmy0.webp\n    link: https://v.qq.com/x/cover/gz9q7kd8wviwkxv/m0024nvac3j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.08.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20160917_.i7howzh6h74.webp\n    link: https://v.qq.com/x/cover/feqfsbsngr2re6e/k0024mc33fl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20170707_3.4otba4tpotq0.webp\n    link: https://v.qq.com/x/cover/y0vxr2r3ptzxbic/o0024bwaj7l.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.07.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20171124_.9ul2qsozwyg.webp\n    link: https://v.qq.com/x/cover/vjrvzv3s517g6m8/w0025hgvzin.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.11.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20180116_.6cn7onig27c0.webp\n    link: https://v.qq.com/x/cover/hzssbvhzc7xyr95/y0030o2gbjj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.01.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20190103_3.26mranwrt868.webp\n    link: https://v.qq.com/x/cover/8rpyk6div4sku7g/q0030fnxigq.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2019.01.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20190719_.48y4gaq3i5y0.webp\n    link: https://www.bilibili.com/bangumi/media/md28228734\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.19 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20190726_.3y34fkkdbki0.webp\n    link: https://v.qq.com/x/cover/zr5a67l333ehzu9/d0032x85stp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20191122_2.115crnsv5xuo.webp\n    link: https://v.qq.com/x/cover/06f338d6r1wa3my/a003369kmz4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.11.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20200806_1.2akqrcgkw45c.webp\n    link: https://v.qq.com/x/cover/mzc002002smgao4.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2020.08.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20201001_.3zwlnse3eqq0.webp\n    link: https://v.qq.com/x/cover/mzc00200892n01l/s0034x8bf06.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.10.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20201127_2.3a46n8dn2d40.webp\n    link: https://v.qq.com/x/cover/ccd36tevohjt9ze/u0035tjlfmu.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2020.11.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20210212_2.6wonqcu3ap80.webp\n    link: https://v.qq.com/x/cover/mzc00200jq7kvn8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2021.02.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20210723_.3vcutfuxa9y0.webp\n    link: https://v.qq.com/x/cover/mzc002003vpbryd.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2021.07.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20220819_.6mi6ccvsn6o0.webp\n    link: https://v.qq.com/x/cover/mzc00200mjy32e7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.08.19 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/03-/20221111_.eghwhxmw1io.webp\n    link: \n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.11.11 17...'\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n234\n235\n236\n237\n238\n239\n240\n241\n242\n243\n244\n245\n246\n247\n248\n249\n250\n251\n\n\n\n\n\n\n1925.06.26 ...\n\n\n\n\n\n1928.01.06 ...\n\n\n\n\n\n1936.02.25 2020...\n\n\n\n\n\n1949.12.01 ...\n\n\n\n\n\n1965.03.02 ...\n\n1\n\n\n\n1975.06.20 ...\n\n1\n\n\n\n1979.05.25 ...\n\n1\n\n\n\n1979.12.06 ...\n\n\n\n\n\n1980.05.23 ...\n\n\n\n\n\n1985.11.07 ...\n\n2\n\n\n\n1986.07.18 ...\n\n\n\n\n\n1986.08.15 ...\n\n\n\n\n\n1987.12.17 ...\n\n\n\n\n\n1989.07.26 ...\n\n\n\n\n\n1989.12.14 ...\n\n1\n\n\n\n1991.07.18 ...\n\n2\n\n\n\n1992.04.09 ...\n\n3\n\n\n\n1992.05.22 ...\n\n\n\n\n\n1992.10.01 ...\n\n\n\n\n\n1992.12.17 ...\n\n3\n\n\n\n1993.01.01 ...\n\n1\n\n\n\n1993.06.11 ...\n\n\n\n\n\n1993.07.01 ...\n\n\n\n\n\n1994.03.31 ...\n\n\n\n\n\n1994.06.30 ...\n\n\n\n\n\n1994.07.06 75...\n\n\n\n\n\n1994.10.14 ...\n\n\n\n\n\n1995.01.21 ...\n\n\n\n\n\n1995.02.04 ...\n\n\n\n\n\n1995.07.06 ...\n\n\n\n\n\n1995.08.19 ...\n\n\n\n\n\n1997.08.01 ...\n\n2\n\n\n\n1997.08.07 ....\n\n\n\n\n\n1997.12.05 ...\n\n\n\n\n\n1997.12.19  ...\n\n\n\n\n\n1997.12.20 ...\n\n\n\n\n\n1998.01.31 ...\n\n\n\n\n\n1998.06.05 ...\n\n\n\n\n\n1999.02.13 ...\n\n\n\n\n\n1999.03.31 ...\n\n\n\n\n\n1999.04.16 ...\n\n\n\n\n\n1999.11.04 ...\n\n1\n\n\n\n2000.03.17 -...\n\n1\n\n\n\n2001.06.22 ...\n\n\n\n\n\n2001.07.12 ...\n\n3\n\n\n\n2001.07.18 ....\n\n1\n\n\n\n2002.03.15 ...\n\n\n\n\n\n2002.09.25 ...\n\n\n\n\n\n2002.10.18 ...\n\n\n\n\n\n2002.12.12 1991...\n\n2\n\n\n\n2003.01.31 ...\n\n2\n\n\n\n2003.06.03 fbi...\n\n1\n\n\n\n2003.11.21 (caribbean sea)...\n\n\n\n\n\n2004.03.17 1949...\n\n\n\n\n\n2004.05.28 ...\n\n\n\n\n\n2004.09.09 thunjanejane...\n\n1\n\n\n\n2004.10.29 ...\n\n\n\n\n\n2004.12.23 1940...\n\nd\n\n\n\n2005.06.23 ...\n\n\n\n\n\n2005.09.23 ...\n\n2\n\n\n\n2005.10.28  ...\n\n\n\n\n\n2005.12.14 ...\n\n\n\n\n\n2006.02.02 12...\n\n\n\n\n\n2006.04.21 roserose...\n\n3\n\n\n\n2006.06.16 ...\n\n2\n\n\n\n2006.07.07 ...\n\n3\n\n\n\n2006.10.27 ...\n\n\n\n\n\n2006.12.15 ...\n\n\n\n\n\n2007.03.16 ....\n\n3\n\n\n\n2007.06.12 ...\n\n1\n\n\n\n2007.07.11 ...\n\n\n\n\n\n2007.07.27 ...\n\n\n\n\n\n2007.07.31 ...\n\n4\n\n\n\n2007.10.25 swat...\n\n\n\n\n\n2007.11.16 19florentino arizafermina daza...\n\n\n\n\n\n2008.01.30 ...\n\n1\n\n\n\n2008.04.30 ...\n\n\n\n\n\n2008.06.13 bruce banner...\n\n5\n\n\n\n2008.10.23 ...\n\n1\n\n\n\n2008.12.12 2030...\n\n4\n\n\n\n2009.04.15 1...\n\n1\n\n\n\n2009.05.03 1845...\n\n2\n\n\n\n2009.06.24 ...\n\n6\n\n\n\n2009.10.23 ...\n\n2012\n\n\n\n2009.11.13 ...\n\n\n\n\n\n2009.11.27 1990...\n\n1\n\n\n\n2009.12.18 ...\n\n\n\n\n\n2009.12.25 ...\n\n2\n\n\n\n2010.04.27 ...\n\n2\n\n\n\n2010.05.07 ...\n\n\n\n\n\n2010.06.04 ...\n\n\n\n\n\n2010.07.22 1976...\n\n\n\n\n\n2010.07.26 ...\n\n7\n\n\n\n2010.10.29 ...\n\n\n\n\n\n2011.04.21 ...\n\n1\n\n\n\n2011.05.08 ...\n\n5\n\n\n\n2011.05.12 ...\n\n4\n\n\n\n2011.05.20 ...\n\n3\n\n\n\n2011.07.21 50...\n\n1\n\n\n\n2011.07.22 40...\n\n\n\n\n\n2011.11.02 ...\n\nhello\n\n\n\n2011.11.04 ...\n\n1\n\n\n\n2012.05.05 ...\n\n\n\n\n\n2012.12.12 ...\n\n3\n\n\n\n2013.05.01 ...\n\n\n\n\n\n2013.05.09 ...\n\n1\n\n\n\n2013.07.12 ...\n\n\n\n\n\n2013.07.19 20...\n\n6\n\n\n\n2013.07.26 ...\n\n2\n\n\n\n2013.10.17 ...\n\n\n\n\n\n2013.10.24 ...\n\n\n\n\n\n2013.11.05 1942...\n\n2\n\n\n\n2013.11.08 ...\n\n\n\n\n\n2013.11.19 ...\n\n2\n\n\n\n2014.04.04 ...\n\n1\n\n\n\n2014.06.13 15...\n\n4\n\n\n\n2014.06.27 ...\n\n1\n\n\n\n2014.08.01 ...\n\n\n\n\n\n2014.08.30 ...\n\n\n\n\n\n2014.08.30 ...\n\n\n\n\n\n2014.11.12 ...\n\n1\n\n\n\n2015.04.02 ...\n\n7\n\n\n\n2015.04.12 ...\n\n2\n\n\n\n2015.05.12 ...\n\n4\n\n\n\n2015.06.10 21...\n\n1\n\n\n\n2015.07.17 ...\n\n\n\n\n\n2015.09.30 ...\n\n\n\n\n\n2015.11.25 ...\n\n3\n\n\n\n2015.12.24 ...\n\n1\n\n\n\n2015.12.31 ...\n\n\n\n\n\n2016.02.08 ...\n\n3\n\n\n\n2016.05.06 ...\n\n\n\n\n\n2016.09.29 dj...\n\n\n\n\n\n2016.10.06 201110513...\n\n1\n\n\n\n2016.11.06 ...\n\n\n\n\n\n2016.12.23 ...\n\n8\n\n\n\n2017.04.14 ...\n\n2\n\n\n\n2017.05.05 ...\n\n3\n\n\n\n2017.05.23 2024x...\n\n5\n\n\n\n2017.05.26 320 ...\n\n5\n\n\n\n2017.06.23 ...\n\n2\n\n\n\n2017.07.27 ...\n\n\n\n\n\n2017.08.11 ...\n\n\n\n\n\n2017.09.08 1989...\n\n1\n\n\n\n2017.09.08 ...\n\n\n\n\n\n2017.09.30 ...\n\n8\n\n\n\n2017.10.27 ...\n\n3\n\n\n\n2017.11.03 ...\n\n\n\n\n\n2017.12.28 1972...\n\n2\n\n\n\n2018.02.16 ...\n\n\n\n\n\n2018.02.16 ...\n\n1\n\n\n\n2018.02.16 ...\n\n2\n\n\n\n2018.03.23 ...\n\n\n\n\n\n2018.03.28 2060...\n\n\n\n\n\n2018.03.30 2045...\n\n3\n\n\n\n2018.05.11 ...\n\n1\n\n\n\n2018.05.18 ...\n\n\n\n\n\n2018.07.05 ...\n\n2\n\n\n\n2018.07.06 fbi...\n\n\n\n\n\n2018.07.27 ...\n\n\n\n\n\n2018.09.20 ...\n\n\n\n\n\n2018.09.30 ...\n\n\n\n\n\n2018.11.16 ...\n\n\n\n\n\n2018.12.07 ...\n\n\n\n\n\n2018.12.21 b-1271987...\n\n\n\n\n\n2019.02.05 ...\n\n1\n\n\n\n2019.02.05 ...\n\n\n\n\n\n2019.02.05 ...\n\n1\n\n\n\n2019.03.08 ...\n\n4\n\n\n\n2019.04.24 ...\n\n\n\n\n\n2019.05.10 ...\n\n2\n\n\n\n2019.05.31 ...\n\n2\n\n\n\n2019.06.28 ...\n\n\n\n\n\n2019.07.12 ...\n\n\n\n\n\n2019.07.18 ...\n\n\n\n\n\n2019.08.09 ...\n\n1\n\n\n\n2019.09.13 ...\n\n\n\n\n\n2019.09.30 20185143u8633...\n\n\n\n\n\n2019.09.30 196015...\n\n4\n\n\n\n2019.12.20 ...\n\n\n\n\n\n2020.08.07 ...\n\n\n\n\n\n2020.08.21 193788262524420...\n\n\n\n\n\n2021.01.15 ...\n\n3\n\n\n\n2021.02.12 crimaster...\n\n\n\n\n\n2021.02.12 ...\n\n\n\n\n\n2021.03.26 ...\n\n3\n\n\n\n2021.12.15 4...\n\n\n\n\n\n2022.02.01 ...\n\n\n\n\n\n2022.03.05 ...\n\n\n\n\n\n2022.06.24 ...\n\n\n\n\n\n2022.07.29 ...\n\n\n\n\n\n2022.09.30  ...\n\n2\n\n\n\n2022.12.16 ...\n\n2\n\n\n\n2023.01.22 ...\n\n\n\n\n\n2023.01.22 ...\n\n\n\n\n\n2023.04.28 ...\n\nconfig:\n    target: _blank\n    imgheight: auto\n    objectfit: cover\n    lineclamp: 4\n\ndata:\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19250626_.77ecc6ish680.webp\n    link: https://v.qq.com/x/cover/tqgmpsxckcen61k.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1925.06.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19280106_.21s2zois81ls.webp\n    link: https://v.qq.com/x/cover/coy40bv9c0qtpq4/i00221744g0.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1928.01.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19360225_.4lz3vmrcj660.webp\n    link: https://v.qq.com/x/cover/9w35g1rr7w3smmr/d0015rhc18d.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1936.02.25 2020...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19491201_.7cr8ewa5nz00.webp\n    link: https://v.qq.com/x/cover/rz7x3oii2cafdnb/w0189iuvk50.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1949.12.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19650302_.2dzx6n830268.webp\n    link: https://v.qq.com/x/cover/nlzswcxr4umy1fr.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1965.03.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19750620_.k0h738dexi8.webp\n    link: https://v.qq.com/x/cover/44jy24n86wmax84/x0011vrknod.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1975.06.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19790525_1.1q8uz1ypcw0w.webp\n    link: https://v.qq.com/x/cover/87zd07p6ok4h7r6/p0020mgmuki.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1979.05.25 ...'\n  \n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19791206_1.66xbpi87g3s0.webp\n    link: https://v.qq.com/x/cover/l25jwdqjjwvyhqb/d0017c9p14q.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1979.12.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19800523_.5ehebvqdojk0.webp\n    link: https://v.qq.com/x/cover/di25h17qqkgj08p/u0022mdu41x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1980.05.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19851107_.39vipm2b6ds0.webp\n    link: https://v.youku.com/v_show/id_xnjezndiyote2.html?tpa=dw5pb25fawq9mtazmzuyxzewmdawml8wmv8wmq\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1985.11.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19860718_2.57nxdru77gg0.webp\n    link: https://v.qq.com/x/cover/enubkr6cyzh68pa/k0024z2sfx4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1986.07.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19860815_.br0n8toi0bs.webp\n    link: https://v.youku.com/v_show/id_xnjeyntkwmdyw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1986.08.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19871217_.4t6ut3gh4740.webp\n    link: https://v.qq.com/x/cover/lnot50y3bsf1b1c/g00420mtyzt.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1987.12.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19890726_.56wgerj6veo0.webp\n    link: https://v.qq.com/x/cover/sg1e3flajdea6rm/w004284klfx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1989.07.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19891214_1.3spxq13lw7c0.webp\n    link: https://v.qq.com/x/cover/msbt3iret5tgwu8/m0025c8sgbz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1989.12.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19910718_1.3qf48pi9vlc.webp\n    link: https://v.qq.com/x/cover/mxwqna0wtgezxzk/s0025swakuf.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1991.07.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19920409_2.5hc6m6ld35c.webp\n    link: https://v.qq.com/x/cover/777ghazyu5gty9r/9emtbomnpif.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1992.04.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19920522_3.2a3t2jgyppa8.webp\n    link: https://v.qq.com/x/cover/qjpt50lj1rdoh2o/n00204bfdnp.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '1992.05.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19921001_.6xueoq1qcn40.webp\n    link: https://v.youku.com/v_show/id_xoduzmzmwnjmy.html?tpa=dw5pb25fawq9mtazmzuyxzewmdawml8wmv8wmq\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1992.10.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19921217_.3hb57tosdg00.webp\n    link: https://v.qq.com/x/cover/0psszmdu9yyngb4/p00159wnhh7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1992.12.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19930101_3.1892d57ctj5s.webp\n    link: https://v.qq.com/x/cover/wk8xgcdq0gdayni/p0033w2dzid.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '1993.01.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19930611_1.8raqm7exgzc.webp\n    link: https://v.qq.com/x/cover/idubxef33n76slz/g0011afrrvv.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '1993.06.11 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19930701_.2aa7h5ck7lus.webp\n    link: https://v.qq.com/x/cover/e3pdjwjjbbcmsow/s0025wfup09.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1993.07.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19940331_.4y2f0kdo5ps0.webp\n    link: https://v.qq.com/x/cover/gwz8nqvc78oexgm/9skha95wrzg.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.03.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19940630_.3j0l4244ciu0.webp\n    link: https://www.bilibili.com/video/bv1he41177c1/\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.06.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19940706_.3bjvm56r9mq0.webp\n    link: https://v.qq.com/x/cover/r6hc2kqgvnmiejn/b0016ws00gf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.07.06 75...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19941014_.1sa7s0ps23i8.webp\n    link: https://v.qq.com/x/cover/1o29ui77e85grdr/h0022ah1yrf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1994.10.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19950121_.2pwfqj7xweg0.webp\n    link: https://quanying.cc/vodplay/2603-1-1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.01.21 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19950204_.78h4y9j8owg0.webp\n    link: https://v.qq.com/x/cover/fse52rd4klx7qn2/v0023cbono5.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.02.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19950706_.41cvwctiggo0.webp\n    link: https://www.iqiyi.com/v_19rrj52998.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.07.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19950819_.2oau32bkf9k0.webp\n    link: https://www.iqiyi.com/v_19rrk62ru4.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1995.08.19 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19970801_.2qa7zucayyq0.webp\n    link: https://v.qq.com/x/cover/9n30k2ayqn1f063/f0021e9sjk4.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.08.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19970807_2.5x31v2evink0.webp\n    link: https://v.qq.com/x/cover/wwo3k53jc1k4apu/r001291y965.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '1997.08.07 ....'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19971205_.1qxrs7x2oz7k.webp\n    link: https://v.qq.com/x/cover/mhl665xf8ina3t2/k0024erhg9j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19971219_.34pwxzlzr600.webp\n    link: https://v.qq.com/x/cover/t5jqhgw8pix81mw/v0024wohpa1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.19  ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19971220_.6cnibhfco380.webp\n    link: https://v.qq.com/x/cover/1weku8f23n0j12s/o0024s1dn9b.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1997.12.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19980131_.1qxkp01n2tsw.webp\n    link: https://555dyy9.com/vodplay/261439-3-1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1998.01.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19980605_.7h5gr4d6k7s0.webp\n    link: https://v.qq.com/x/cover/31082i4u5ovkrl0/s00211yt19u.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1998.06.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19990213_.24x4b7prenog.webp\n    link: https://v.youku.com/v_show/id_xmzyymduzmtu3ng==.html?spm=a2h1n.8261147.0.0&s=cc04f3b4962411de83b1\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.02.13 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19990331_.j6u8oblb80o.webp\n    link: https://www.dianyinggou.com/mov/1291843.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.03.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19990416_.8aglf1ttkk8.webp\n    link: https://v.qq.com/x/cover/znrpdjaupklt9q0/d0012mrzi68.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.04.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/19991104_.3tsmsi6caus0.webp\n    link: https://www.dianyinggou.com/mov/1305114.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '1999.11.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20000317_1.6emakqwcl7k.webp\n    link: https://v.qq.com/x/cover/e8072tbfo3q2ccy/b0022bfu6up.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2000.03.17 -...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20010622_1.sqyzco31wm8.webp\n    link: https://v.qq.com/x/cover/o66no96ex54bksd/o0011g0p889.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2001.06.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20010712_.3rzudjwmv7o0.webp\n    link: https://www.dianyinggou.com/mov/1297747.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2001.07.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20010718_3.6filmh0xa300.webp\n    link: https://v.qq.com/x/cover/pbknv4h3h8qvo01/m0011ireuji.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2001.07.18 ....'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20020315_1.rdxpw92uc0w.webp\n    link: https://v.qq.com/x/cover/sixf5demwk331r9/g00144sraal.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2002.03.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20020925_.637t38849sw0.webp\n    link: https://v.qq.com/x/cover/g6n8noz3gmbiz4o/k0030lgbvhd.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.09.25 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20021018_.33ljm5m2m8i0.webp\n    link: https://www.dianyinggou.com/mov/1306808.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.10.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20021212_.74nvmtxwtqc0.webp\n    link: https://v.qq.com/x/cover/sg5p4qp1s6jbyj8/o0025wzpzfw.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2002.12.12 1991...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20030131_2.6pfmpz2btnc0.webp\n    link: https://www.dianyinggou.com/mov/1307956.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2003.01.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20030603_2.1lbff08vyxr4.webp\n    link: https://v.qq.com/x/cover/xia8ahis63c8l3n/j0022gmjck0.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2003.06.03 fbi...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20031121_1.3phuilveo960.webp\n    link: https://v.qq.com/x/cover/6bxb2yc7mskrdoz/b0012l31auu.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2003.11.21 (caribbean sea)...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20040317_.asnjkyhmogc.webp\n    link: https://www.dianyinggou.com/mov/1291549.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.03.17 1949...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20040528_.1ba5waxnp4lc.webp\n    link: https://v.qq.com/x/cover/y60o6957ucc46qo/d0021ykrfle.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.05.28 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20040909_.1j6x6jivfbeo.webp\n    link: https://v.qq.com/x/cover/v8vd4y9td09bv4n/w0010khtfom.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.09.09 thunjanejane...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20041029_1.3uhcig9a0sk0.webp\n    link: https://www.dianyinggou.com/mov/1417598.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2004.10.29 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20041223_.7fyxfvrhnmg0.webp\n    link: https://v.qq.com/x/cover/uiq0rxuywu508qr/o001241efrp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2004.12.23 1940...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20050623_d.nrkj47xifzk.webp\n    link: https://v.qq.com/x/cover/n16pc4sk2x29mjo.html\n    name: <div align=\"center\"><strong>d</strong></div>\n    desc: '2005.06.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20050923_.46b7cvys43g0.webp\n    link: https://www.dianyinggou.com/mov/1307023.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.09.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20051028_2.2xypiw9autg0.webp\n    link: https://www.dianyinggou.com/mov/1309045.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2005.10.28  ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20051214_.3pg2pn0i4wa0.webp\n    link: https://v.qq.com/x/cover/ztnpctziozsz5y5/l0012dovkvk.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2005.12.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20060202_.6gksywf2ncs0.webp\n    link: https://v.qq.com/x/cover/yp5rjb7mlt7encw/b0010mcbkoc.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.02.02 12...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20060421_.60n92fu92vs0.webp\n    link: https://www.dianyinggou.com/mov/1466007.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.04.21 roserose...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20060616_3.5bv9lcd1kf40.webp\n    link: https://v.qq.com/x/cover/p0wd3xfsrh2ntwo/t0011fugmo3.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2006.06.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20060707_2.19b2mcq2knls.webp\n    link: https://v.qq.com/x/cover/7qgeyfoqp4ks3dl/a0012cer60c.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2006.07.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20061027_3.5em4h4osg600.webp\n    link: https://www.dianyinggou.com/mov/1474189.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2006.10.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20061215_.4527fey60m80.webp\n    link: https://v.qq.com/x/cover/vbr7hkoebjpsrxb/a001483446x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2006.12.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20070316_.6ezlhv9keis0.webp\n    link: https://v.qq.com/x/cover/f8637674mcfvlhi/n0011ongycd.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.03.16 ....'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20070612_3.57bwxfkqo980.webp\n    link: https://v.qq.com/x/cover/dluq5thl5pcvtl5/f0012jbwanr.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2007.06.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20070711_1.1m9lfg8n848w.webp\n    link: https://v.qq.com/x/cover/dwt56ndspm9xm89/b0016sl196d.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2007.07.11 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20070727_.5co7qhpt0lo0.webp\n    link: https://v.qq.com/x/cover/uqxmnkbvq4qzxlv/p00216kbjh9.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.07.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20070731_.ojj867ivvcw.webp\n    link: https://www.dianyinggou.com/mov/2124724.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.07.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20071025_4.1788n8x8rbeo.webp\n    link: https://www.dianyinggou.com/mov/1922273.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2007.10.25 swat...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20071116_.37ixt6jiekq0.webp\n    link: https://www.dianyinggou.com/mov/2002743.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2007.11.16 19florentino arizafermina daza...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20080130_.2uadfxhtdf20.webp\n    link: https://v.qq.com/x/cover/f0qjcexig8rjzpo/t0014c8lrw7.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.01.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20080430_1.4vxm59ui6ve0.webp\n    link: https://v.qq.com/x/cover/ciwsbplwbrje0gt/y00207r9opc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.04.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20080613_.543j36je12s0.webp\n    link: https://v.qq.com/x/cover/vwq7gmy475harwj/e002120vmdu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2008.06.13 bruce banner...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20081023_5.6yxhxmvbzlg0.webp\n    link: https://www.dianyinggou.com/mov/2350904.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2008.10.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20081212_1.2gfcvqwo724g.webp\n    link: https://v.qq.com/x/cover/rali88m8udw2x0u/z0040nen92x.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2008.12.12 2030...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20090415_4.7clf9y0v0iw0.webp\n    link: https://v.qq.com/x/cover/w48cjplx8eitbb4/b0021aeu6fu.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2009.04.15 1...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20090503_1.rxtqzfvzkpc.webp\n    link: https://v.qq.com/x/cover/ti9lk63vdjl2zph/q0020ad0xxi.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2009.05.03 1845...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20090624_2.4vqbu6jgfag0.webp\n    link: https://v.qq.com/x/cover/vehyv3t0ith3r55/m0016pq3xum.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2009.06.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20091023_6.6awospvzdyg0.webp\n    link: https://www.dianyinggou.com/mov/3075983.html\n    name: <div align=\"center\"><strong>6</strong></div>\n    desc: '2009.10.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20091113_2012.72smln9a7780.webp\n    link: https://v.qq.com/x/cover/92lbu34b5yvzel6/s002085eoew.html\n    name: <div align=\"center\"><strong>2012</strong></div>\n    desc: '2009.11.13 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20091127_.vvbhmb1kzo0.webp\n    link: https://v.qq.com/x/cover/nvw3lm5ouqs54cc/i00141yi8pe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.11.27 1990...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20091218_1.5g7j1dpqb5g0.webp\n    link: https://v.qq.com/x/cover/ldl1811bamppdrd/y0040wzq199.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2009.12.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20091225_.4ahgkr6ql860.webp\n    link: https://v.qq.com/x/cover/x5ul2annjsbcwh8/e00335q391x.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2009.12.25 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20100427_2.4iouosxdjka0.webp\n    link: https://www.iqiyi.com/v_19rrj5kq94.html?vfm=m_103_txsp\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2010.04.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20100507_2.2th14ce6oxo0.webp\n    link: https://v.qq.com/x/cover/90qkh14unn4q8xu/v0020sk2lw4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2010.05.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20100604_.5pqa5iy6qs00.webp\n    link: https://v.qq.com/x/cover/wlb7ba6jg8x1rjm/i0014q7qgmu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.06.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20100722_.6p3bmszpv7w0.webp\n    link: https://v.qq.com/x/cover/upd61lkzbo5ko1p/8tcbch3v618.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.07.22 1976...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20100726_.40al0bhna460.webp\n    link: https://v.qq.com/x/cover/997m2fipgc6erv7/h0022fp0vk9.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2010.07.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20101029_7.3i4l30q4v060.webp\n    link: https://www.dianyinggou.com/mov/3878157.html\n    name: <div align=\"center\"><strong>7</strong></div>\n    desc: '2010.10.29 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110421_.s4pvn02iuu8.webp\n    link: https://v.qq.com/x/cover/h8aovufr2nix8pv/k0014qbypb2.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.04.21 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110508_1.161i8sg48ko0.webp\n    link: https://v.qq.com/x/cover/sjfw7p2cz0ff6m9/i0018msd6dc.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.05.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110512_5.ultu3lug4v4.webp\n    link: https://v.qq.com/x/cover/unhg39knn3fjkb7/n0015hqcguy.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2011.05.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110520_4.3nnv6hfnrmw.webp\n    link: https://v.qq.com/x/cover/m63g578728dv56w/f00120mtond.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2011.05.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110721_3.3l2c3gm2a180.webp\n    link: https://v.qq.com/x/cover/lpc2ry21etiyhmn/j00169ib5er.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2011.07.21 50...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20110722_1.1kw5zk2rs8tc.webp\n    link: https://v.qq.com/x/cover/891a0latn4c5chp/r0018t60y1z.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2011.07.22 40...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20111102_.5i2oiygn4ls0.webp\n    link: https://v.qq.com/x/cover/km0frs6ajyrstog/t0040nwsohx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2011.11.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20111104_hello.2ljhh8b8i560.webp\n    link: https://v.qq.com/x/cover/g6055xnwc815rrr/p00189kcnwz.html\n    name: <div align=\"center\"><strong>hello</strong></div>\n    desc: '2011.11.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20120505_1.7cyipzl7fk40.webp\n    link: https://v.qq.com/x/cover/cz2szzeahb8aj1k/t0020bvo93n.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2012.05.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20121212_.48hvkrakfdm0.webp\n    link: https://v.qq.com/x/cover/tjtiyg1qe0zkhdk/k0033s8cekl.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2012.12.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20130501_3.7an0t9dajkw0.webp\n    link: https://www.dianyinggou.com/mov/3231742.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2013.05.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20130509_.35inzx3bl780.webp\n    link: https://v.qq.com/x/cover/7jwfxt0rnldwcip/h002171rs0n.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.05.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20130712_1.2m7bap31oco0.webp\n    link: https://v.qq.com/x/cover/3xkxy6w6cdj5bh1/m0024wxr19j.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2013.07.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20130719_.1e3dupqlyoqo.webp\n    link: https://v.qq.com/x/cover/igy79xk26amdv6z/u00235gga3e.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.07.19 20...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20130726_6.1t6bco1hcp1c.webp\n    link: https://v.qq.com/x/cover/wiqfi65xzyj7rzz/u00137ae5kq.html\n    name: <div align=\"center\"><strong>6</strong></div>\n    desc: '2013.07.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20131017_2.1secnm1vz9og.webp\n    link: https://v.qq.com/x/cover/e7d88bfyq7trhy5/z002025ta8o.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.10.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20131024_.fv7fbdl0k3k.webp\n    link: https://v.qq.com/x/cover/skl46pqqu1e7bmx/b0021iwtlcp.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.10.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20131105_.dpbkbjj5vpc.webp\n    link: https://v.qq.com/x/cover/3zpk72rw56ggqh9/b0032ruhjan.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.11.05 1942...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20131108_2.5e18zy4t86c0.webp\n    link: https://v.qq.com/x/cover/gib6uvlt5rfudib/b00202o3rv6.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2013.11.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20131119_.1go8ibxmktz4.webp\n    link: https://v.qq.com/x/cover/1jx00d87vzuv04a/y0024xp2k03.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2013.11.19 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140404_2.2ih7ercbxn80.webp\n    link: https://v.qq.com/x/cover/ix5jw6tz5ghz4dc/w00157zn04f.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2014.04.04 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140613_1.5qu27pbt19g0.webp\n    link: https://v.qq.com/x/cover/803p673mlosoeog/r0024n8ir8m.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2014.06.13 15...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140627_4.3w4pyw0k1ae0.webp\n    link: https://v.qq.com/x/cover/am6jxwt5rzq6kba/n0022sql7nn.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2014.06.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140801_1.5ws0util27g0.webp\n    link: https://v.qq.com/x/cover/ht4z382bhs7h94u/v0015hiufvt.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2014.08.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140830_.53gs9f6sgnk0.webp\n    link: https://v.qq.com/x/cover/1o7ou00t80apgko/w0018fcfqfv.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20140830_.5j71usgk9jk0.webp\n    link: https://v.qq.com/x/cover/yz98ta4nfjtaa6q/h0017hm4g46.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.08.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20141112_.29df38gu9hhc.webp\n    link: https://v.qq.com/x/cover/c2seabnsfozypl8/d00161yroup.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2014.11.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150402_1.6nxrrnr71ds0.webp\n    link: https://v.qq.com/x/cover/1bnyfttijufe0cq/m0016nwmqws.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.04.02 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150412_7.2ndjz3r9fro0.webp\n    link: https://v.qq.com/x/cover/dv71ekj7urmxkvp/a00177o0w31.html\n    name: <div align=\"center\"><strong>7</strong></div>\n    desc: '2015.04.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150512_2.3lmirdm0ui00.webp\n    link: https://v.qq.com/x/cover/w7ju7urc1x4w3jc/t0017mlfdg4.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2015.05.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150610_4.9p9nubqki4w.webp\n    link: https://v.qq.com/x/cover/p06la73u7kd0hzx/z0018okldyy.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2015.06.10 21...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150717_1.2q5ardigl700.webp\n    link: https://v.qq.com/x/cover/zl6n65vy9hgrarl/u0019j97d7o.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.07.17 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20150930_.5poszdwfmng0.webp\n    link: https://v.qq.com/x/cover/ge1vg9evjfeb6l0/u0019esirxq.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.09.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20151125_.2si2r0woz5k0.webp\n    link: https://v.qq.com/x/cover/gk11svuomdpsgl4/f0027zr3wyu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2015.11.25 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20151224_3.thcyno7i4ls.webp\n    link: https://v.qq.com/x/cover/ra1dhxmhz872ry8/d0020tn1crm.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2015.12.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20151231_1.373342r3dfi0.webp\n    link: https://v.qq.com/x/cover/f3qroleq1lyuavy/r0019z1dctq.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2015.12.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20160208_.1e80e04yc7.webp\n    link: https://v.qq.com/x/cover/xg95sxi4q7zc4uo/s00429cehc3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.02.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20160506_3.gj4sdys9w48.webp\n    link: https://v.qq.com/x/cover/kzhcitf07bg2pxy/t0021qb4zzg.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2016.05.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20160929_.20yu7pcvcn34.webp\n    link: https://v.qq.com/x/cover/hrr3qneqbo22q06/o0022hmagay.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.09.29 dj...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20161006_.2blynol8x78k.webp\n    link: https://v.qq.com/x/cover/h188mdjtrh3up22/y0022eckcdz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.10.06 201110513...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20161106_1.2x8m6f2bttq0.webp\n    link: https://v.qq.com/x/cover/uvl5rs2euhfgoip/h0022otg21y.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2016.11.06 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20161223_.512tqgjydsg0.webp\n    link: https://v.qq.com/x/cover/2xxul4n2j8y0rxi/a0024c3w6lf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2016.12.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170414_8.2wrz4fs6c2s0.webp\n    link: https://v.qq.com/x/cover/l9vxrwfdjcw1q4g/h00248zwzvg.html\n    name: <div align=\"center\"><strong>8</strong></div>\n    desc: '2017.04.14 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170505_2.18nfw3y0qjts.webp\n    link: https://v.qq.com/x/cover/4gl2i78zd9idpi0/j0024zknymk.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2017.05.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170523_3.4icqouq77ac0.webp\n    link: https://www.dianyinggou.com/mov/25765735.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.05.23 2024x...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170526_5.5e3bfqsv6rc0.webp\n    link: https://v.qq.com/x/cover/00jxecd5him5kmn/v0024zl735r.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2017.05.26 320 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170623_5.139nwwr7a2n4.webp\n    link: https://v.qq.com/x/cover/qee75hz1x7m6n3s/x00243heka3.html\n    name: <div align=\"center\"><strong>5</strong></div>\n    desc: '2017.06.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170727_2.57o80lncmos0.webp\n    link: https://v.qq.com/x/cover/wi8e2p5kirdaf3j/s00242sxrne.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2017.07.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170811_.7k9bjcbk1xc0.webp\n    link: https://v.qq.com/x/cover/q5iz3d3xh80tkuf/f0026f5cjwe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.08.11 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170908_.76fl22wqdjo0.webp\n    link: https://www.dianyinggou.com/mov/3604148.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.09.08 1989...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170908_1.315wtu21t3c0.webp\n    link: https://v.qq.com/x/cover/iqwvbujgzj5obv1/d00256xcmw9.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2017.09.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20170930_.4pt20bwp1vk0.webp\n    link: https://v.qq.com/x/cover/jpjl78i91mx05bu/m0025mcvah8.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.09.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20171027_8.ws0bx91q1gw.webp\n    link: https://www.dianyinggou.com/mov/25788426.html\n    name: <div align=\"center\"><strong>8</strong></div>\n    desc: '2017.10.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20171103_3.ee4g0mqwvo.webp\n    link: https://v.qq.com/x/cover/9ikkuz4o8ejavdc/k0026phfnm2.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2017.11.03 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20171228_.pis8bi60vsg.webp\n    link: https://v.qq.com/x/cover/hirg4ixfe73pras/i0031e1i2s6.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2017.12.28 1972...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180216_2.7ktb41g308o0.webp\n    link: https://v.qq.com/x/cover/t2t7544bcr1pw8r/a0026k0ccqn.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180216_.4c3288342d40.webp\n    link: https://v.qq.com/x/cover/7casb7nes159mrl/o0026p7tyf1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180216_1.4poiyatpwem0.webp\n    link: https://v.qq.com/x/cover/7casb7nes159mrl/o0026p7tyf1.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2018.02.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180323_2.849a11nxtks.webp\n    link: https://v.qq.com/x/cover/f7pqur8uhmzltps/o0026dn1s6q.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.03.23 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180328_.76ijed08lt80.webp\n    link: https://www.dianyinggou.com/mov/26945085.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.03.28 2060...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180330_.1jw4mjae6mtc.webp\n    link: https://v.qq.com/x/cover/zatx0zuk94d9aru/p0026621ctf.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.03.30 2045...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180511_3.218e1t7eoykg.webp\n    link: https://v.qq.com/x/cover/coqnq6i120wojq6/x0027u7i1ip.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2018.05.11 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180518_.3i3w9r647h80.webp\n    link: https://v.qq.com/x/cover/uuq07xfnxk00zqd/n0027ozlk06.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2018.05.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180705_.5atulsgmnr40.webp\n    link: https://v.qq.com/x/cover/og0eputlxwet1cn/r0028uz12ua.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180706_2.620vljvk4ws0.webp\n    link: https://v.qq.com/x/cover/sxbxm9xaakyynzp/i0028uor0n8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2018.07.06 fbi...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180727_.3tey2szp1de0.webp\n    link: https://v.qq.com/x/cover/z6j3ixjjcokafyc/l0028g8hyd3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.07.27 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180920_.1ttqzoy3kni8.webp\n    link: https://v.qq.com/x/cover/eaa7qkjmlecfnp1/q0031poot4r.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20180930_.38ckrlgaiae0.webp\n    link: https://v.qq.com/x/cover/v3sfrz8ws9ew7fw/m0029o2tv2o.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.09.30 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20181116_.3yen5v900260.webp\n    link: https://v.qq.com/x/cover/r9umzdo0yjelfg1/n0030x2ucda.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.11.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20181207_.29kib3wgex34.webp\n    link: https://v.qq.com/x/cover/l8r3gm1zzu5u3nk/j0030f3wne1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20181221_.1sxcs7t8utxc.webp\n    link: https://v.qq.com/x/cover/i43tfclvbhiy0kn/a0030h7pjsj.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2018.12.21 b-1271987...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190205_.23sgcjwoui3k.webp\n    link: https://v.qq.com/x/cover/lt0w27nj9tpfllw/c003070g88f.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190205_1.4tsm6rfwlvk0.webp\n    link: https://v.qq.com/x/cover/3fvg46217gw800n/h0030qj4fov.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190205_.69z41xiiz8c0.webp\n    link: https://v.qq.com/x/cover/8kw2uo89elwb7as/v0030gvdd5b.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.02.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190308_1.4j98fxsrq6m0.webp\n    link: https://v.qq.com/x/cover/xyne4253g35nak3/m0031od9ekb.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.03.08 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190424_4.43xn0d2t9jc0.webp\n    link: https://v.qq.com/x/cover/v2098lbuihuqs11/u00315w81xn.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2019.04.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190510_.2i8u3yqea8e0.webp\n    link: https://v.qq.com/x/cover/2acva9s87eg7m8g/b00326bxu3h.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.05.10 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190531_2.4d5d49ez8po0.webp\n    link: https://v.qq.com/x/cover/e0bk8kf7wllv7r8/z0034ixucs8.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.05.31 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190628_2.49mu68wserk0.webp\n    link: https://v.qq.com/x/cover/96sxjj429lictza/s0032ab6wjm.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2019.06.28 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190712_.6etl9p4u4x80.webp\n    link: https://v.qq.com/x/cover/ehqo76prcwku2oq/x0032rq56lh.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190718_.54yjo4k7fug0.webp\n    link: https://v.qq.com/x/cover/p6xvg0ft471pcj8/l00321cue7j.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.07.18 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190809_.453dyq6rpiw0.webp\n    link: https://v.qq.com/x/cover/hswhwhxw05n7jso/r0032c9m0dx.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.08.09 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190913_1.5velexcpmc00.webp\n    link: https://v.qq.com/x/cover/5vfbg80goc4mkay/f003249hywz.html\n    name: <div align=\"center\"><strong>1</strong></div>\n    desc: '2019.09.13 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190930_.3zg98n0uivs0.webp\n    link: https://v.qq.com/x/cover/rj8uc45tm8a17wm/z0032udy82e.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.09.30 20185143u8633...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20190930_.7ij3a8ehm8s0.webp\n    link: https://v.qq.com/x/cover/u9z5s6rt4j30fjh/a0032xozepq.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2019.09.30 196015...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20191220_4.72sq59700a40.webp\n    link: https://v.qq.com/x/cover/m5zzglrbt5zdv6d/v0033476tnr.html\n    name: <div align=\"center\"><strong>4</strong></div>\n    desc: '2019.12.20 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20200807_.20vos7dxi880.webp\n    link: https://v.qq.com/x/cover/mzc0020067i3zrk/m0034gnwg46.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.08.07 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20200821_.6yoxjmdyq1c0.webp\n    link: https://v.qq.com/x/cover/9f8tpq2gly9p5ts/r0034rmwzpe.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2020.08.21 193788262524420...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20210115_.524tzo2o4ws0.webp\n    link: https://v.qq.com/x/cover/mzc00200zflvank/w0035jtpsb1.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.01.15 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20210212_3.qufv8t4xeeo.webp\n    link: https://v.qq.com/x/cover/tk3l27paq7sqr0z/p0036mrfj8h.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2021.02.12 crimaster...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20210212_.2srtocmnxmi0.webp\n    link: https://v.qq.com/x/cover/mzc00200vx37bvf/c0035hwvmv4.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.02.12 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20210326_.5mz13avkmqc0.webp\n    link: https://v.qq.com/x/cover/babks6vmrn25n9c/l0037j1ebig.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2021.03.26 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20211215_3.3v0n9vtsbl20.webp\n    link: https://www.dianyinggou.com/mov/26933210.html\n    name: <div align=\"center\"><strong>3</strong></div>\n    desc: '2021.12.15 4...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20220201_.13cf5yz9w6r4.webp\n    link: https://v.qq.com/x/cover/mzc002001uvqi5d/i0042h9vxm3.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.02.01 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20220305_.5ck2uwdp3fo0.webp\n    link: https://v.qq.com/x/cover/mzc00200owkcck4/f0043bpxr53.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.03.05 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20220624_.1sh7ga7oegjk.webp\n    link: https://v.qq.com/x/cover/mzc00200zlh907s/j0044wpymeo.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.06.24 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20220729_.4a7wqnc78wu0.webp\n    link: https://v.qq.com/x/cover/mzc002000y0q8uy/r0044rrsowz.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.07.29 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20220930_.60ctbuk9gw00.webp\n    link: https://v.qq.com/x/cover/mzc00200qv892j3/k0045ztu6tu.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2022.09.30  ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20221216_2.2z8zjag8ga40.webp\n    link: https://v.qq.com/x/cover/mzc00200qv892j3/k0045ztu6tu.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2022.12.16 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20230122_2.5zpvevc8ab40.webp\n    link: https://v.qq.com/x/cover/mzc002008v8rka3/d0046v4pddg.html\n    name: <div align=\"center\"><strong>2</strong></div>\n    desc: '2023.01.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20230122_.1x5dt1w3vbk0.webp\n    link: https://v.qq.com/x/cover/mzc00200bd3gkta/b0046wa81dm.html\n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.01.22 ...'\n\n  - img: https://zhengtiger.github.io/picx-images-hosting/99-/01-/04-/20230428_.stmosyfuy4g.webp\n    link: \n    name: <div align=\"center\"><strong></strong></div>\n    desc: '2023.04.28 ...'\n\n\n\n1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19\n20\n21\n22\n23\n24\n25\n26\n27\n28\n29\n30\n31\n32\n33\n34\n35\n36\n37\n38\n39\n40\n41\n42\n43\n44\n45\n46\n47\n48\n49\n50\n51\n52\n53\n54\n55\n56\n57\n58\n59\n60\n61\n62\n63\n64\n65\n66\n67\n68\n69\n70\n71\n72\n73\n74\n75\n76\n77\n78\n79\n80\n81\n82\n83\n84\n85\n86\n87\n88\n89\n90\n91\n92\n93\n94\n95\n96\n97\n98\n99\n100\n101\n102\n103\n104\n105\n106\n107\n108\n109\n110\n111\n112\n113\n114\n115\n116\n117\n118\n119\n120\n121\n122\n123\n124\n125\n126\n127\n128\n129\n130\n131\n132\n133\n134\n135\n136\n137\n138\n139\n140\n141\n142\n143\n144\n145\n146\n147\n148\n149\n150\n151\n152\n153\n154\n155\n156\n157\n158\n159\n160\n161\n162\n163\n164\n165\n166\n167\n168\n169\n170\n171\n172\n173\n174\n175\n176\n177\n178\n179\n180\n181\n182\n183\n184\n185\n186\n187\n188\n189\n190\n191\n192\n193\n194\n195\n196\n197\n198\n199\n200\n201\n202\n203\n204\n205\n206\n207\n208\n209\n210\n211\n212\n213\n214\n215\n216\n217\n218\n219\n220\n221\n222\n223\n224\n225\n226\n227\n228\n229\n230\n231\n232\n233\n234\n235\n236\n237\n238\n239\n240\n241\n242\n243\n244\n245\n246\n247\n248\n249\n250\n251\n252\n253\n254\n255\n256\n257\n258\n259\n260\n261\n262\n263\n264\n265\n266\n267\n268\n269\n270\n271\n272\n273\n274\n275\n276\n277\n278\n279\n280\n281\n282\n283\n284\n285\n286\n287\n288\n289\n290\n291\n292\n293\n294\n295\n296\n297\n298\n299\n300\n301\n302\n303\n304\n305\n306\n307\n308\n309\n310\n311\n312\n313\n314\n315\n316\n317\n318\n319\n320\n321\n322\n323\n324\n325\n326\n327\n328\n329\n330\n331\n332\n333\n334\n335\n336\n337\n338\n339\n340\n341\n342\n343\n344\n345\n346\n347\n348\n349\n350\n351\n352\n353\n354\n355\n356\n357\n358\n359\n360\n361\n362\n363\n364\n365\n366\n367\n368\n369\n370\n371\n372\n373\n374\n375\n376\n377\n378\n379\n380\n381\n382\n383\n384\n385\n386\n387\n388\n389\n390\n391\n392\n393\n394\n395\n396\n397\n398\n399\n400\n401\n402\n403\n404\n405\n406\n407\n408\n409\n410\n411\n412\n413\n414\n415\n416\n417\n418\n419\n420\n421\n422\n423\n424\n425\n426\n427\n428\n429\n430\n431\n432\n433\n434\n435\n436\n437\n438\n439\n440\n441\n442\n443\n444\n445\n446\n447\n448\n449\n450\n451\n452\n453\n454\n455\n456\n457\n458\n459\n460\n461\n462\n463\n464\n465\n466\n467\n468\n469\n470\n471\n472\n473\n474\n475\n476\n477\n478\n479\n480\n481\n482\n483\n484\n485\n486\n487\n488\n489\n490\n491\n492\n493\n494\n495\n496\n497\n498\n499\n500\n501\n502\n503\n504\n505\n506\n507\n508\n509\n510\n511\n512\n513\n514\n515\n516\n517\n518\n519\n520\n521\n522\n523\n524\n525\n526\n527\n528\n529\n530\n531\n532\n533\n534\n535\n536\n537\n538\n539\n540\n541\n542\n543\n544\n545\n546\n547\n548\n549\n550\n551\n552\n553\n554\n555\n556\n557\n558\n559\n560\n561\n562\n563\n564\n565\n566\n567\n568\n569\n570\n571\n572\n573\n574\n575\n576\n577\n578\n579\n580\n581\n582\n583\n584\n585\n586\n587\n588\n589\n590\n591\n592\n593\n594\n595\n596\n597\n598\n599\n600\n601\n602\n603\n604\n605\n606\n607\n608\n609\n610\n611\n612\n613\n614\n615\n616\n617\n618\n619\n620\n621\n622\n623\n624\n625\n626\n627\n628\n629\n630\n631\n632\n633\n634\n635\n636\n637\n638\n639\n640\n641\n642\n643\n644\n645\n646\n647\n648\n649\n650\n651\n652\n653\n654\n655\n656\n657\n658\n659\n660\n661\n662\n663\n664\n665\n666\n667\n668\n669\n670\n671\n672\n673\n674\n675\n676\n677\n678\n679\n680\n681\n682\n683\n684\n685\n686\n687\n688\n689\n690\n691\n692\n693\n694\n695\n696\n697\n698\n699\n700\n701\n702\n703\n704\n705\n706\n707\n708\n709\n710\n711\n712\n713\n714\n715\n716\n717\n718\n719\n720\n721\n722\n723\n724\n725\n726\n727\n728\n729\n730\n731\n732\n733\n734\n735\n736\n737\n738\n739\n740\n741\n742\n743\n744\n745\n746\n747\n748\n749\n750\n751\n752\n753\n754\n755\n756\n757\n758\n759\n760\n761\n762\n763\n764\n765\n766\n767\n768\n769\n770\n771\n772\n773\n774\n775\n776\n777\n778\n779\n780\n781\n782\n783\n784\n785\n786\n787\n788\n789\n790\n791\n792\n793\n794\n795\n796\n797\n798\n799\n800\n801\n802\n803\n804\n805\n806\n807\n808\n809\n810\n811\n812\n813\n814\n815\n816\n817\n818\n819\n820\n821\n822\n823\n824\n825\n826\n827\n828\n829\n830\n831\n832\n833\n834\n835\n836\n837\n838\n839\n840\n841\n842\n843\n844\n845\n846\n847\n848\n849\n850\n851\n852\n853\n854\n855\n856\n857\n858\n859\n860\n861\n862\n863\n864\n865\n866\n867\n868\n869\n870\n871\n872\n873\n874\n875\n876\n877\n878\n879\n880\n881\n882\n883\n884\n885\n886\n887\n888\n889\n890\n891\n892\n893\n894\n895\n896\n897\n898\n899\n900\n901\n902\n903\n904\n905\n906\n907\n908\n909\n910\n911\n912\n913\n914\n915\n916\n917\n918\n919\n920\n921\n922\n923\n924\n925\n926\n927\n928\n929\n930\n931\n932\n933\n934\n935\n936\n937\n938\n939\n940\n941\n942\n943\n944\n945\n946\n947\n948\n949\n950\n951\n952\n953\n954\n955\n956\n957\n958\n959\n960\n961\n962\n963\n964\n965\n966\n967\n968\n969\n970\n971\n972\n973\n974\n975\n976\n977\n978\n979\n980\n981\n982\n983\n984\n985\n986\n987\n",charsets:{cjk:!0},lastUpdated:"2024/09/02, 20:17:14",lastUpdatedTimestamp:1725279434e3},{title:"",frontmatter:{tagsPage:!0,title:"",permalink:"/tags/",article:!1,readingShow:"top"},regularPath:"/@pages/tagsPage.html",relativePath:"@pages/tagsPage.md",key:"v-6ea3592e",path:"/tags/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2022/11/18, 22:00:53",lastUpdatedTimestamp:1668780053e3},{title:"",frontmatter:{archivesPage:!0,title:"",permalink:"/archives/",article:!1,readingShow:"top"},regularPath:"/@pages/archivesPage.html",relativePath:"@pages/archivesPage.md",key:"v-0d4bbaae",path:"/archives/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2022/11/18, 22:00:53",lastUpdatedTimestamp:1668780053e3},{title:"",frontmatter:{categoriesPage:!0,title:"",permalink:"/categories/",article:!1,readingShow:"top"},regularPath:"/@pages/categoriesPage.html",relativePath:"@pages/categoriesPage.md",key:"v-22f1f98e",path:"/categories/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2022/11/18, 22:00:53",lastUpdatedTimestamp:1668780053e3},{title:"Home",frontmatter:{home:!0,heroText:"TigerZ  ",tagline:" --- TigerZ",pageClass:"vdoing-index-class",bannerBg:"img/bannerBg.gif",readingShow:"top"},regularPath:"/",relativePath:"index.md",key:"v-42ca3e9a",path:"/",headersStr:null,content:"",normalizedContent:"",charsets:{},lastUpdated:"2023/04/26, 16:12:14",lastUpdatedTimestamp:1682496734e3}],themeConfig:{bodyBgImg:["/img/bodyBgImg/DNF1.jpg"],bodyBgImgOpacity:1,bodyBgImgInterval:15,indexImg:{navColor:2,switchNavColor:!0,bgTimeColor:!0,bgTimeColorArray:["transparent","transparent","transparent","transparent"],descFade:!0,desc:[" --- TigerZ"],descFontSize:"1.4rem",descFadeInTime:200,descFadeOutTime:100,descNextTime:800,bubble:!1,bubblePosition:0,bubbleNum:200},contentBgStyle:1,author:{name:"TigerZ ",link:"https://zhenghu159.github.io/"},blogger:{avatar:"img/avatar.jpg",name:"TigerZ",slogan:""},social:{icons:[{iconClass:"icon-youjian",title:"",link:"mailto:1308155474@qq.com"},{iconClass:"icon-github",title:"GitHub",link:"https://github.com/zhenghu159"},{iconClass:"icon-weixin",title:"",link:"https://open.weixin.qq.com/qr/code?username=gh_55ff0104c230"},{iconClass:"icon-zhihu",title:"",link:"https://www.zhihu.com/people/shu-ru-yong-hu-ming-65-69"},{iconClass:"icon-jianshu",title:"",link:"https://www.jianshu.com/u/6dacd977dea9"},{iconClass:"icon-bokeyuan",title:"",link:"https://www.cnblogs.com/tigerzheng/"},{iconClass:"icon-csdn",title:"CSDN",link:"https://blog.csdn.net/weixin_45851732"}]},footer:{createYear:2022,copyrightInfo:'TigerZ  | Blog<br> <a href="http://beian.miit.gov.cn/" target="_blank"></a>'},nav:[{text:"",link:"/"},{text:"",link:"/navigation/"},{text:"",link:"/pages/585994/"},{text:"",link:"/pages/577e62/",items:[{text:"",items:[{text:"",link:"/pages/09e806/"},{text:"",link:"/pages/4c56e7/"}]},{text:"",items:[{text:"",link:"/pages/17c432/"},{text:"",link:"/pages/e2e6a4/"},{text:"",link:"/pages/e90196/"},{text:"",link:"/pages/69dc52/"}]}]},{text:"",link:"/pages/98350d/",items:[{text:"",items:[{text:"",link:"/pages/11a817/"}]},{text:"",items:[{text:"",link:"/pages/967a30/"},{text:"",link:"/pages/3c3418/"},{text:"",link:"/pages/726c78/"},{text:"",link:"/pages/8b01b8/"},{text:"NGS",link:"/pages/c3f74c/"}]}]},{text:"Web",link:"/pages/4efbc6/",items:[{text:"",items:[{text:"HTML",link:"/pages/3ca8f4/"},{text:"CSS",link:"/pages/f16f8d/"},{text:"JS/TS",link:"/pages/9ec206/"},{text:"Markdown",link:"/pages/1abc5f/"},{text:"Vue",link:"/pages/d2038c/"},{text:"Ajax",link:"/pages/d4684b/"}]},{text:"",items:[{text:"MySQL",link:"/pages/98c9ed/"},{text:"Django",link:"/pages/e1436c/"}]}]},{text:"",link:"/pages/3c0f51/",items:[{text:"",link:"/pages/01057a/"},{text:"",items:[{text:" - ",link:"/pages/99a1ec/"},{text:" - ",link:"/pages/7daf99/"}]}]},{text:"",link:"/pages/baed44/",items:[{text:"",items:[{text:"R ",link:"/pages/9ab953/"}]}]},{text:"",link:"/pages/634e82/",items:[{text:"",items:[{text:"- ",link:"/pages/be5d67/"}]}]},{text:"",link:"/pages/d28eb7/",items:[{text:"",link:"/pages/eccd67/"},{text:"",link:"/pages/30653b/"},{text:"",link:"/pages/1a58a6/"}]},{text:"",link:"/pages/8cb41f/",items:[{text:" - ",link:"/pages/13ae7d/"},{text:" - ",link:"/pages/2b095a/"},{text:" - ",link:"/archives/"}]},{text:"",link:"/message-area/"}],sidebarDepth:2,searchMaxSuggestions:10,lastUpdated:"",docsDir:"docs",sidebar:{"/00./":[{title:" - ",collapsable:!0,children:[["01. - /00..md","","/pages/585994/"]]},{title:" - ",collapsable:!0,children:[["02. - /00..md","","/pages/577e62/"],["02. - /01..md","","/pages/09e806/"],["02. - /02..md","","/pages/4c56e7/"],["02. - /11..md"," --- ","/pages/17c432/"],["02. - /12..md"," --- ","/pages/e2e6a4/"],["02. - /13..md"," --- ","/pages/e90196/"],["02. - /14..md"," --- ","/pages/69dc52/"]]},{title:" - ",collapsable:!0,children:[["03. - /00..md","","/pages/98350d/"],["03. - /01..md","","/pages/11a817/"],["03. - /11. - .md"," - ","/pages/967a30/"],["03. - /12. - .md"," - ","/pages/3c3418/"],["03. - /13. - .md"," - ","/pages/726c78/"],["03. - /14. - .md"," - ","/pages/8b01b8/"],["03. - /15. - NGS.md"," - NGS","/pages/c3f74c/"]]},{title:"Web - ",collapsable:!0,children:[["04.Web - /00.Web.md","Web","/pages/4efbc6/"],["04.Web - /11. - HTML.md"," - HTML","/pages/3ca8f4/"],["04.Web - /12. - CSS.md"," - CSS","/pages/f16f8d/"],["04.Web - /13. - JS&TS.md"," - JavaScript/TypeScript","/pages/9ec206/"],["04.Web - /14. - Markdown.md"," - Markdown","/pages/1abc5f/"],["04.Web - /15. - Vue.md"," - Vue","/pages/d2038c/"],["04.Web - /16. - Ajax.md"," - Ajax","/pages/d4684b/"],["04.Web - /21. - MySQL.md"," - MySQL","/pages/98c9ed/"],["04.Web - /22. - Django.md"," - Django","/pages/e1436c/"]]},{title:" - ",collapsable:!0,children:[["05. - /00..md","","/pages/3c0f51/"],["05. - /01..md","","/pages/01057a/"],["05. - /11. - .md"," - ","/pages/99a1ec/"],["05. - /12. - .md"," - ","/pages/7daf99/"]]},{title:" - ",collapsable:!0,children:[["06. - /00..md","","/pages/baed44/"],["06. - /10.R .md","R ","/pages/9ab953/"]]},{title:" - ",collapsable:!0,children:[["07. - /00..md","","/pages/634e82/"],["07. - /11. - .md"," - ","/pages/be5d67/"]]},{title:" - ",collapsable:!0,children:[["99. - /00..md","","/pages/d28eb7/"],["99. - /03..md","","/pages/1a58a6/"]]},{title:" - ",collapsable:!0,children:[["100. - /00..md","","/pages/8cb41f/"],["100. - /01. - .md"," - ","/pages/13ae7d/"],["100. - /02. - .md"," - ","/pages/2b095a/"]]}],catalogue:{"":"/pages/585994/","":"/pages/577e62/","":"/pages/17c432/","":"/pages/e2e6a4/","":"/pages/e90196/","":"/pages/69dc52/","":"/pages/98350d/"," - ":"/pages/967a30/"," - ":"/pages/3c3418/"," - ":"/pages/726c78/"," - ":"/pages/8b01b8/"," - NGS":"/pages/c3f74c/","Web":"/pages/4efbc6/"," - HTML":"/pages/3ca8f4/"," - CSS":"/pages/f16f8d/"," - JS&TS":"/pages/9ec206/"," - Markdown":"/pages/1abc5f/"," - Vue":"/pages/d2038c/"," - Ajax":"/pages/d4684b/"," - MySQL":"/pages/98c9ed/"," - Django":"/pages/e1436c/","":"/pages/3c0f51/","":"/pages/01057a/"," - ":"/pages/99a1ec/"," - ":"/pages/7daf99/","":"/pages/baed44/","":"/pages/634e82/"," - ":"/pages/be5d67/","":"/pages/8cb41f/"," - ":"/pages/13ae7d/"," - ":"/pages/2b095a/","":"/pages/d28eb7/"},"/01./":[["001.OneNote.md","OneNote","/pages/1260b7/"],["002.Snipaste.md","Snipaste","/pages/413897/"],["003.NCBI.md","NCBI","/pages/420e29/"]],"/02./":[{title:"",collapsable:!0,children:[{title:"",collapsable:!0,children:[["11./01./20121220.(2012.12.20 mPFC  Neuron) The Role of Medial Prefrontal Cortex in Memory and Decision Making.md"," --- The Role of Medial Prefrontal Cortex in Memory and Decision Making","/pages/1aae84/"],["11./01./20180609.(2018.06.09  Curr Opin Neurobiol) Transcriptional control of long-range cortical projections.md"," --- Transcriptional control of long-range cortical projections","/pages/7100c2/"],["11./01./20181116.(2018.11.16  Trends Cogn Sci) Bridging the Gap between Connectome and Transcriptome.md"," --- Bridging the Gap between Connectome and Transcriptome","/pages/9e1c62/"]]},{title:"",collapsable:!0,children:[["11./02./20140402.(2014.04.02 Allen Nature) A mesoscale connectome of the mouse brain.md"," --- A mesoscale connectome of the mouse brain","/pages/7ffbf6/"],["11./02./20160718.(2016.07.18 MAPseq Neuron) High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA.md"," --- High-Throughput Mapping of Single-Neuron Projections by Sequencing of Barcoded RNA","/pages/dc1352/"],["11./02./20180328.(2018.03.28  Nature 2018) The logic of single-cell projections from  visual cortex.md"," --- The logic of single-cell projections from visual cortex","/pages/8cb60c/"],["11./02./20181030.(2018.10.31 PT-thalamus-medulla Nature) Distinct descending motor cortex pathways and their roles in movement.md"," --- Distinct descending motor cortex pathways and their roles in movement","/pages/986ab0/"],["11./02./20181031.(2018.10.31  Nature) Shared and distinct transcriptomic cell types across neocortical areas.md"," --- Shared and distinct transcriptomic cell types across neocortical areas","/pages/af94c8/"],["11./02./20190208.(2019.02.08  Mol Neurodegeneration) Differences in neurotropism and neurotoxicity among retrograde viral tracers.md"," --- Differences in neurotropism and neurotoxicity among retrograde viral tracers","/pages/037be6/"],["11./02./20191017.(2019.10.17 BAR-seq Cell) High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing.md"," --- High-Throughput Mapping of Long-Range Neuronal Projection Using In Situ Sequencing","/pages/da828f/"],["11./02./20200709.(2020.07.09 BRICseq Cell) BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals.md"," --- BRICseq Bridges Brain-wide Interregional Connectivity to Neural Activity and Gene Expression in Single Animals","/pages/afda58/"],["11./02./20200722.(2020.07.22  Nature) Distinct subnetworks of the thalamic reticular nucleus.md"," --- Distinct subnetworks of the thalamic reticular nucleus","/pages/4918af/"],["11./02./20200914.(2020.09.14 MAPseq Nat Neurosci) Circuit and molecular architecture of a ventral hippocampal network.md"," --- Circuit and molecular architecture of a ventral hippocampal network","/pages/fa4bef/"],["11./02./20201214.(2020.12.14 SCM PNAS) Uncovering the genetic blueprint of the C. elegans nervous system.md"," --- Uncovering the genetic blueprint of the C. elegans nervous system","/pages/521e00/"],["11./02./20201217.(2020.12.17  Cell) Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks.md"," --- Differential encoding in prefrontal cortex projection neuron classes across cognitive tasks","/pages/5c79f3/"],["11./02./20210510.(2021.05.10 BARseq2 Neuroscience) Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections.md"," --- Integrating barcoded neuroanatomy with spatial transcriptional profiling enables identification of gene correlates of projections","/pages/084a0b/"],["11./02./20211005.(2021.10.06 epi-retro-seq Nature) Epigenomic diversity of cortical projection neurons in the mouse brain.md"," --- Epigenomic diversity of cortical projection neurons in the mouse brain","/pages/ccc053/"],["11./02./20211006.(2021.10.06 MOp-ul Nature) Cellular anatomy of the mouse primary motor cortex.md"," --- Cellular anatomy of the mouse primary motor cortex","/pages/d5ec9f/"],["11./02./20211221.(2021.12.21 VECTORseq Cell Reports) Virally encoded connectivity transgenic overlay RNA sequencing (VECTORseq) defines projection neuron involved in sensorimotor integration.md"," --- Virally Encoded Connectivity Transgenic Overlay RNA sequencing (VECTORseq) defines projection neurons involved in sensorimotor integration","/pages/40cb7b/"],["11./02./20220331.(2022.03.31 fMOST Nat Neurosci) Single-neuron projectome of mouse prefrontal cortex.md"," --- Single-neuron projectome of mouse prefrontal cortex","/pages/0be7e3/"],["11./02./20221027.(2022.10.27 MAPseq Cell) High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex.md"," --- High-throughput sequencing of single neuron projections reveals spatial organization in the olfactory cortex","/pages/94c1e9/"],["11./02./20230522.(2023.5.22 fMOST Nat Neurosci) Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex.md"," --- Single-neuron analysis of dendrites and axons reveals the network organization in mouse prefrontal cortex","/pages/e528dd/"],["11./02./20231018.(2023.10.18 MAPseq macaque Neuron) Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex.md"," --- Single basolateral amygdala neurons in macaques exhibit distinct connectional motifs with frontal cortex","/pages/aaea7b/"],["11./02./20231213.(2023.12.13 epi-retro-seq Nature) Temporal controls over inter-areal cortical projection neuron fate diversity.md"," --- Brain-wide correspondence of neuronal epigenomics and distant projections","/pages/b405a1/"],["11./02./20240202.(2024.02.02 fmost Science) Whole-brain spatial organization of hippocampal single-neuron projectomes.md"," --- Whole-brain spatial organization of hippocampal single-neuron projectomes","/pages/f20dad/"],["11./02./20240223.(2024.02.23 MERGE-seq eLife) High-throughput mapping of single-neuron projection and molecular features by  retrograde barcoded labelling.md"," --- High-throughput mapping of single-neuron projection and molecular features by retrograde barcoded labelling","/pages/723a6a/"],["11./02./20240610.(2024.06.10  eLife) Deciphering the genetic code of neuronal type connectivity through bilinear modeling.md"," --- Deciphering the genetic code of neuronal type connectivity through bilinear modeling","/pages/9c9ddc/"]]},{title:"",collapsable:!0,children:[["11./03./20220120.(2022.01.20  Cell) Vision-dependent specification of cell types and function in the developing cortex.md"," --- Vision-dependent specification of cell types and function in the developing cortex","/pages/b79c35/"],["11./03./20221110.(2022.11.10 PFC Cell) Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution.md"," --- Human prefrontal cortex gene regulatory dynamics from gestation to adulthood at single-cell resolution","/pages/0b119c/"],["11./03./20221228.(2022.12.28 MERFISH Cell) Molecular and spatial signatures of mouse brain aging at single-cell resolution.md"," --- Molecular and spatial signatures of mouse brain aging at single-cell resolution","/pages/16bdaa/"]]},{title:"",collapsable:!0,children:[["11./04./20160628.(2016.06.28 Div-Seq Science) Div-Seq Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons.md"," --- Div-Seq:Single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons","/pages/9b041f/"],["11./04./20160816.(2016.08.16  PNAS) Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons.md"," --- Single-cell RNAseq reveals cell adhesion molecule profiles in electrophysiologically defined neurons","/pages/a61efc/"],["11./04./20171011.(2017.10.11 Act-seq Neuron) Detecting Activated Cell Populations Using Single-Cell RNA-Seq.md"," --- Detecting Activated Cell Populations Using Single-Cell RNA-Seq","/pages/8bf2bf/"],["11./04./20190913.(2019.09.13 mPFC Nat Commun) Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction.md"," --- Cell type-specific transcriptional programs in mouse prefrontal cortex during adolescence and addiction","/pages/c60442/"],["11./04./20191111.(2019.11.11  Elife) Single-cell transcriptomic evidence for dense intracortical neuropeptide networks.md"," --- Single-cell transcriptomic evidence for dense intracortical neuropeptide networks","/pages/bb2e69/"],["11./04./20200115.(2020.01.15  nature) Decoding the development of the human  hippocampus.md"," --- Decoding the development of the human hippocampus","/pages/0b79fa/"],["11./04./20200626.(2020.06.26  Sci Adv) Molecular atlas of the adult mouse brain.md"," --- Molecular atlas of the adult mouse brain","/pages/37a825/"],["11./04./20201207.(2020.12.07  Nat Neurosci) Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing.md"," --- Diversification of molecularly defined myenteric neuron classes revealed by single-cell RNA sequencing","/pages/b5aaa4/"],["11./04./20210623.(2021.06.23  nature) Molecular logic of cellular diversification in the mouse cerebral cortex.md"," --- Molecular logic of cellular diversification in the mouse cerebral cortex","/pages/8a50f3/"],["11./04./20210708.(2021.07.08 hdWGCNA Nat Genet) Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease.md"," --- Single-nucleus chromatin accessibility and transcriptomic characterization of Alzheimer's disease","/pages/25c02b/"],["11./04./20220530.(2022.05.30  Nat Neurosci) Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations.md"," --- Single-cell transcriptomics of adult macaque  hippocampus reveals neural precursor cell  populations","/pages/f481a1/"],["11./04./20220621.(2022.06.21 hPFC PNAS) The neuropeptide landscape of human prefrontal cortex.md"," --- The neuropeptide landscape of human prefrontal cortex","/pages/444fce/"],["11./04./20230712.(2023.07.12 stereo-seq Cell) Single-cell spatial transcriptome reveals cell-type organization in the macaque cortex.md"," --- Single-cell spatial transcriptome reveals cell-typ organization in the macaque cortex","/pages/8c91b0/"],["11./04./20231016.(2023.10.16 mPFC MERFISH Nat Neurosci) Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain.md"," --- Spatial transcriptomics reveals the distinct organization of mouse prefrontal cortex and neuronal subtypes regulating chronic pain","/pages/42c71a/"],["11./04./20231101.(2023.11.01  Sci. Adv.) A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types.md"," --- A telencephalon cell type atlas for goldfish reveals diversity in the evolution of spatial structure and cell types","/pages/12b484/"],["11./04./20231115.(2023.11.15  Neuron) The neuropeptidergic connectome of C. elegans.md"," --- The neuropeptidergic connectome of C. elegans","/pages/a1ce18/"],["11./04./20231213.(2023.12.13 Allen bioRxiv) A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain.md"," --- A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain","/pages/895303/"],["11./04./20240424.(2024.04.24 BARseq Nature) Whole-cortex in situ sequencing reveals input-dependent area identity.md"," --- Whole-cortex in situ sequencing reveals input-dependent area identity","/pages/666cd3/"]]}]},{title:"",collapsable:!0,children:[["12./20220220.(2022.02.20 CRD J Pineal Res)Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma.md"," --- Singlecell transcriptomic analysis reveals circadianrhythm disruption associated with poor prognosisand drugresistance in lung adenocarcinoma","/pages/3eedee/"],["12./20230127.(2023.01.27 -FISH rainbow Nat Commun) Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules.md"," --- Highly efficient and robust -FISH rainbow for multiplexed in situ detection of diverse biomolecules","/pages/c40bc0/"],["12./20240514.(2024.05.14 MiP-seq Nat. Biomed. Eng) Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing.md"," --- Spatial multi-omics at subcellular resolution via high-throughput in situ pairwise sequencing","/pages/e6566f/"],["12./20240704.(2024.07.04  Nat Methods) Systematic comparison of sequencing-based spatial transcriptomic methods.md"," --- Systematic comparison of sequencing-based spatial transcriptomic methods","/pages/b55e03/"]]},{title:"",collapsable:!0,children:[["13./20180426.(2018.04.26 DLO Hi-C Nat Genet) Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture.md"," --- Digestion-ligation-only Hi-C is an efficient and cost-effective method for chromosome conformation capture","/pages/76fbba/"]]},{title:"",collapsable:!0,children:[["14./20081228.(2008.12.28 WGCNA BMC Bioinformatics) WGCNA an R package for weighted correlation network analysis.md"," --- WGCNAan R package for weighted correlation network analysis","/pages/ffdcc0/"],["14./20150413.(2015.04.13 Seurat V1 Nat Biotechnol) Spatial reconstruction of single-cell gene expression data.md"," --- Spatial reconstruction of single-cell gene expression data","/pages/0d1fce/"],["14./20171204.(2017.12.04 WholeBrain Nat Neurosci) An interactive framework for whole-brain maps at cellular resolution.md"," --- An interactive framework for whole-brain maps at cellular resolution","/pages/e020de/"],["14./20180206.(2018.02.06 scanpy Genome Biol) SCANPY large-scale single-cell gene expression data analysis.md"," --- SCANPYlarge-scale single-cell gene expression data analysis","/pages/ebb632/"],["14./20180402.(2018.04.02 Seurat V2 Nat Biotechnol) Integrating single-cell transcriptomic data across different conditions, technologies, and species.md"," --- Integrating single-cell transcriptomic data across different conditions, technologies, and species","/pages/f37280/"],["14./20180628.(2018.06.28 MAGIC Cell) Recovering Gene Interactions from Single-Cell Data Using Data Diffusion.md"," --- Recovering Gene Interactions from Single-Cell Data Using Data Diffusion","/pages/06c5a0/"],["14./20190403.(2019.04.03 DoubletFinder Cell Systems) DoubletFinder Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors.md"," --- DoubletFinder:Doublet Detection in Single-Cell RNA Sequencing Data Using Artificial Nearest Neighbors","/pages/71b694/"],["14./20190606.(2019.06.06 Seurat V3 Cell) Comprehensive Integration of Single-Cell Data.md"," --- Comprehensive Integration of Single-Cell Data","/pages/d26f9e/"],["14./20200914.(2020.09.14 scRNAseq imputation Genome Biol) A systematic evaluation of single-cell RNA-sequencing imputation methods.md"," --- A systematic evaluation of single-cell RNA-sequencing imputation methods","/pages/c7adb0/"],["14./20201207.(2020.12.07 nnU-Net Nat Methods) nnU-Net a self-configuring method for deep learning-based biomedical image segmentation.md"," --- nnU-Neta self-configuring method for deep learning-based biomedical image segmentation","/pages/d38252/"],["14./20201210.(2020.12.10, GCNG, Genome Biol) GCNG graph convolutional networks for inferring gene interaction from spatial transcriptomics data.md"," --- GCNGgraph convolutional networks for inferring gene interaction from spatial transcriptomics data","/pages/c34d2c/"],["14./20201214.(2020.12.14 Cellpose Nat Methods) Cellpose a generalist algorithm for cellular segmentation.md"," --- Cellposea generalist algorithm for cellular segmentation","/pages/1f3ea0/"],["14./20201217.(2020.12.17 FAN-C Genome Biol) FAN-C a feature-rich framework for the analysis and visualisation of chromosome conformation capture data.md"," --- FAN-C:a feature-rich framework for the analysis and visualisation of chromosome conformation capture data","/pages/1f53c5/"],["14./20210206.(2021.02.06  COVID-19 CT Medical Physics) Toward data-efficient learning A benchmark for COVID-19 CT lung and infection segmentation.md"," --- Toward data-efficient learningA benchmark for COVID-19 CT lung and infection segmentation","/pages/1c1b2b/"],["14./20210217.(2021.02.17 CellChat Nat Commun) Inference and analysis of cell-cell communication using CellChat.md"," --- Inference and analysis of cell-cell communication using CellChat","/pages/2737d2/"],["14./20210624.(2021.06.24 Seurat V4 Cell) Integrated analysis of multimodal single-cell data.md"," --- Integrated analysis of multimodal single-cell data","/pages/b4dfe0/"],["14./20210701.(2021.07.01  Liu et al. Front. Genet) Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease.md"," --- Discovery and Validation of Key Biomarkers Based on Immune Infiltrates in Alzheimer's Disease","/pages/6a4bcc/"],["14./20211028.(2021.10.28 Tangram Nat Methods) Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram.md"," --- Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram","/pages/c3bbd9/"],["14./20220113.(2022.01.13 Cell2location Nat Biotechnol) Cell2location maps fine-grained cell types in spatial transcriptomics.md"," --- Cell2location maps fine-grained cell types in spatial transcriptomics","/pages/9912cc/"],["14./20220131.(2022.01.31 Squidpy Nat Methods) Squidpy a scalable framework for spatial omics analysis.md"," --- Squidpya scalable framework for spatial omics analysis","/pages/76476f/"],["14./20220511.(2022.05.11 Viv Nat Methods) Viv multiscale visualization of high-resolution multiplexed bioimaging data on the web.md"," --- Vivmultiscale visualization of high-resolution multiplexed bioimaging data on the web","/pages/112d0b/"],["14./20220930.(2022.09.30, SpaceX, Bioinformatics) SpaceX gene co-expression network estimation for spatial transcriptomics.md"," --- SpaceXgene co-expression network estimation for spatial transcriptomics","/pages/4cba92/"],["14./20230215.(2023.02.15 NeuronChat Nat Commun) Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat.md"," --- Inferring neuron-neuron communications from single-cell transcriptomics through NeuronChat","/pages/794f06/"],["14./20230325.(2023.03.25 Seurat V5 Nat Biotechnol) Dictionary learning for integrative, multimodal, and scalable single-cell analysis.md"," --- Dictionary learning for integrative, multimodal and scalable single-cell analysis","/pages/8c5839/"],["14./20230410.(2023.04.10 scverse Nat Biotechnol) The scverse project provides a computational ecosystem for single-cell omics data analysis.md"," --- The scverse project provides a computational ecosystem for single-cell omics data analysis","/pages/124edd/"],["14./20230612.(2023.06.12 hdWGCNA Cell Reports Methods) hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data.md"," --- hdWGCNA identifies co-expression networks in high-dimensional transcriptomics data","/pages/a24249/"],["14./20231122.(2023.11.22, SPACEL, Nat Commun) SPACEL characterizing spatial transcriptome architectures by deep-learning.md"," --- SPACELdeep learning-based characterization of spatial transcriptome architectures","/pages/da0fb5/"],["14./20240614.(2024.06.14, tidyomics, Nat Methods) The tidyomics ecosystem enhancing omic data analyses.md"," --- The tidyomics ecosystemenhancing omic data analyses","/pages/32a729/"],["14./20240703.(2024.07.03, ezSingleCell, Nat Commun) ezSingleCell an integrated one-stop single-cell and spatial omics analysis platform for bench scientists.md"," --- ezSingleCellan integrated one-stop single-cell and spatial omics analysis platform for bench scientists","/pages/02ef79/"]]}],"/03./":[{title:" - ",collapsable:!0,children:[["01. - /001.Seurat.md","SeuratR toolkit for single cell genomics","/pages/d207e7/"],["01. - /002.DoubletFinder.md","DoubletFinderR package for detecting doublets in single-cell RNA sequencing data","/pages/a35823/"],["01. - /003.anndata.md","anndataa Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray","/pages/7d0fcf/"],["01. - /004.MetaNeighbor.md","MetaNeighbora method to rapidly assess cell type identity using both functional and random gene sets","/pages/9d48f3/"],["01. - /005.scrattch.md","scrattch.hicatHierarchical, Iterative Clustering for Analysis of Transcriptomics","/pages/8dbd6f/"],["01. - /006.WGCNA.md","WGCNAan R package for weighted correlation network analysis","/pages/9ffdf7/"],["01. - /007.hdWGCNA.md","hdWGCNAhigh dimensional WGCNA","/pages/649303/"],["01. - /008.Monocle.md","MonocleAn analysis toolkit for single-cell RNA-seq","/pages/b62f01/"],["01. - /009.CellChat.md","CellChatInference and analysis of cell-cell communication","/pages/f26000/"],["01. - /010.NeuronChat.md","NeuronChatInferring neuron-neuron communications from single-cell transcriptomics","/pages/7a726f/"],["01. - /011.scCustomize.md","scCustomizeA collection of functions created and/or curated to aid in the visualization and analysis of single-cell data using R","/pages/9e5aae/"],["01. - /012.Tangram.md","TangramSpatial alignment of single cell transcriptomic data","/pages/c7e56a/"],["01. - /013.scanpy.md","ScanpySingle-Cell Analysis in Python","/pages/61a60f/"],["01. - /014.squidpy.md","squidpySpatial Single Cell Analysis in Python","/pages/37f62d/"]]},{title:" - ",collapsable:!0,children:[["02. - /001.XGBoost.md","XGBoostScalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library","/pages/006dc9/"],["02. - /002.glmnet.md","glmnetLasso and Elastic-Net Regularized Generalized Linear Models","/pages/ee4b70/"]]},{title:" - ",collapsable:!0,children:[]},{title:" - ",collapsable:!0,children:[["04. - /001.nnUNet.md","nnUNeta self-configuring method for deep learning-based biomedical image segmentation","/pages/8633a5/"]]},{title:" - NGS",collapsable:!0,children:[]}],"/04.Web/":[{title:" - HTML",collapsable:!0,children:[["11. - HTML/01.HTML.md","HTML","/pages/791228/"],["11. - HTML/02.HTML.md","HTML","/pages/18d66f/"]]},{title:" - CSS",collapsable:!0,children:[["12. - CSS/01.CSS.md","CSS","/pages/bb1275/"]]},{title:" - JS&TS",collapsable:!0,children:[["13. - JS&TS/01.JavaScript.md","JavaScript ","/pages/592a6d/"],["13. - JS&TS/02.TypeScript.md","TypeScript","/pages/6ce63c/"]]},{title:" - Markdown",collapsable:!0,children:[["14. - Markdown/01.Markdown.md","Markdown ","/pages/da13bb/"]]},{title:" - Vue",collapsable:!0,children:[["15. - Vue/01.Vue3.md","Vue3","/pages/f03da3/"]]},{title:" - Ajax",collapsable:!0,children:[["16. - Ajax/01.AJAX.md","AJAX","/pages/ce8da3/"]]},{title:" - MySQL",collapsable:!0,children:[["21. - MySQL/01.MySQL.md","MySQL","/pages/49a104/"]]},{title:" - Django",collapsable:!0,children:[["22. - Django/01.Django.md","Django","/pages/b75e1a/"]]}],"/05./":[{title:"",collapsable:!0,children:[["01./1..md","","/pages/d7e3cb/"],["01./2..md","","/pages/02fc82/"],["01./3..md","","/pages/4c1aba/"],["01./4..md","","/pages/cbc016/"]]},{title:" - ",collapsable:!0,children:[["11. - /1..md","1 ","/pages/4a4e4d/"],["11. - /2..md","2 ","/pages/f7fe10/"],["11. - /3..md","3 ","/pages/3e695c/"]]},{title:" - ",collapsable:!0,children:[["12. - /01..md","(Introduction)","/pages/e2487d/"],["12. - /02..md","(Linear Regression with One Variable)","/pages/64abf4/"],["12. - /03..md","(Linear Algebra Review)","/pages/0952ad/"],["12. - /04..md","(Linear Regression with Multiple Variables)","/pages/95b5f9/"],["12. - /06..md","(Logistic Regression)","/pages/81f7ef/"]]}],"/06./":[{title:"R - books",collapsable:!0,children:[]},{title:"R - tidyverse",collapsable:!0,children:[["12.R - tidyverse/01.tidyverse.md","tidyverse","/pages/c63497/"],["12.R - tidyverse/02.ggplot2.md","ggplot2","/pages/98868d/"]]},{title:"R - ",collapsable:!0,children:[["15.R - /004.ggalluvial.md","ggalluvial","/pages/29214b/"],["15.R - /005.DiagrammeR.md","DiagrammeR","/pages/6580bf/"],["15.R - /006.linkET.md","linkET","/pages/142715/"],["15.R - /007.circlize.md","circlize","/pages/a41a6b/"],["15.R - /008.Biorplot.md","Biorplot","/pages/22fafa/"]]},{title:"R - ",collapsable:!0,children:[["16.R - /01.quarto.md","quarto","/pages/02875c/"]]}],"/07./":[{title:" - ",collapsable:!0,children:[["11. - /01.1 .md","1 ","/pages/14b7e3/"]]}],"/100./":[{title:" - ",collapsable:!0,children:[["01. - /01. - .md"," - ","/about/me/introduce/"]]},{title:" - ",collapsable:!0,children:[["02. - /01.-.md","-","/pages/d3830b/"],["02. - /02.-.md","-","/pages/e6cfba/"],["02. - /03.-.md","-","/pages/94f5fc/"],["02. - /04.-.md","-","/pages/093f81/"],["02. - /05.-Markdown.md","-Markdown","/pages/50efc7/"],["02. - /06.-.md","-","/pages/c4fda3/"],["02. - /07.-.md","-","/pages/25bbfb/"]]}],"/99./":[["01..md","","/pages/eccd67/"],["02..md","","/pages/30653b/"],{title:"",collapsable:!0,children:[["03./001..md","","/pages/46a15a/"],["03./002..md","","/pages/aef061/"]]}]},blogInfo:{blogCreate:"2023",indexView:!0,pageView:!0,readingTime:!0,eachFileWords:[{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\01. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"01."}},title:"",date:"2023-09-03T16:57:24.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/585994/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"02."}},title:"",date:"2022-11-19T10:51:23.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/577e62/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\01..md",wordsCount:"7.3k",readingTime:"33.7m",title:"",date:"2023-02-02T11:18:04.000Z",permalink:"/pages/09e806/",categories:[""],tags:[null]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\02..md",wordsCount:"3.8k",readingTime:"23m",title:"",date:"2023-06-15T19:19:37.000Z",permalink:"/pages/4c56e7/",categories:[""],tags:[null]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\11..md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"02./11."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/17c432/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\12..md",wordsCount:54,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"02./12."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},permalink:"/pages/e2e6a4/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\13..md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"02./13."}},title:" --- ",date:"2022-12-08T20:35:03.000Z",permalink:"/pages/e90196/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\02. - \\14..md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"02./14."}},title:" --- ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/69dc52/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03."}},title:"",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/98350d/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\01..md",wordsCount:556,readingTime:"3.3m",title:"",date:"2023-06-26T19:49:01.000Z",permalink:"/pages/11a817/",categories:[""],tags:[null]},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\11. - .md",wordsCount:58,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03./01. - "}},title:" - ",date:"2022-11-19T10:51:23.000Z",permalink:"/pages/967a30/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\12. - .md",wordsCount:54,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03./02. - "}},title:" - ",date:"2023-05-10T20:47:21.000Z",permalink:"/pages/3c3418/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\13. - .md",wordsCount:56,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03./03. - "}},title:" - ",date:"2023-05-15T16:45:44.000Z",permalink:"/pages/726c78/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\14. - .md",wordsCount:54,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03./04. - "}},title:" - ",date:"2023-05-15T16:46:19.000Z",permalink:"/pages/8b01b8/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - NGS",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\03. - \\15. - NGS.md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"03./05. - NGS"}},title:" - NGS",date:"2023-05-15T16:46:33.000Z",permalink:"/pages/c3f74c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"Web",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\00.Web.md",wordsCount:39,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web"}},title:"Web",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-24T20:29:46.000Z",permalink:"/pages/4efbc6/"},{name:" - HTML",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\11. - HTML.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/11. - HTML"}},title:" - HTML",date:"2023-04-30T11:05:51.000Z",permalink:"/pages/3ca8f4/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - CSS",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\12. - CSS.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/12. - CSS"}},title:" - CSS",date:"2023-04-30T11:05:59.000Z",permalink:"/pages/f16f8d/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - JS&TS",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\13. - JS&TS.md",wordsCount:45,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/13. - JS&TS"}},title:" - JavaScript/TypeScript",date:"2023-04-30T11:06:17.000Z",permalink:"/pages/9ec206/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - Markdown",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\14. - Markdown.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/14. - Markdown"}},title:" - Markdown",date:"2023-04-16T21:17:16.000Z",permalink:"/pages/1abc5f/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - Vue",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\15. - Vue.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/15. - Vue"}},title:" - Vue",date:"2023-04-30T11:16:00.000Z",permalink:"/pages/d2038c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - Ajax",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\16. - Ajax.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/16. - Ajax"}},title:" - Ajax",date:"2024-07-26T21:48:51.000Z",permalink:"/pages/d4684b/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - MySQL",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\21. - MySQL.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/21. - MySQL"}},title:" - MySQL",date:"2024-07-26T21:50:19.000Z",permalink:"/pages/98c9ed/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:" - Django",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\04.Web - \\22. - Django.md",wordsCount:43,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"04.Web/22. - Django"}},title:" - Django",date:"2024-07-26T21:50:22.000Z",permalink:"/pages/e1436c/",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"}},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\05. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"05."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:37:25.000Z",permalink:"/pages/3c0f51/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\05. - \\01..md",wordsCount:46,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"05./01."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:38:52.000Z",permalink:"/pages/01057a/"},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\05. - \\11. - .md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"05./11. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-12-04T20:46:08.000Z",permalink:"/pages/99a1ec/"},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\05. - \\12. - .md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"05./12. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T09:00:26.000Z",permalink:"/pages/7daf99/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\06. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"06."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-28T09:38:06.000Z",permalink:"/pages/baed44/"},{name:"R ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\06. - \\10.R .md",wordsCount:347,readingTime:"2.1m",title:"R ",date:"2023-06-26T19:11:03.000Z",permalink:"/pages/9ab953/",categories:["R"],tags:[null]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\07. - \\00..md",wordsCount:41,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"07."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2022-11-23T22:37:25.000Z",permalink:"/pages/634e82/"},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\07. - \\11. - .md",wordsCount:52,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"07./11. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-28T09:28:26.000Z",permalink:"/pages/be5d67/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\100. - \\00..md",wordsCount:37,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"100."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:35:22.000Z",permalink:"/pages/8cb41f/"},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\100. - \\01. - .md",wordsCount:44,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"100./01. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:36:17.000Z",permalink:"/pages/13ae7d/"},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\100. - \\02. - .md",wordsCount:44,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"100./02. - "}},title:" - ",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-17T16:36:26.000Z",permalink:"/pages/2b095a/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\99. - \\00..md",wordsCount:37,readingTime:"1",pageComponent:{name:"Catalogue",data:{path:"99."}},title:"",sidebar:!1,article:!1,comment:!1,editLink:!1,author:{name:"TigerZ",link:"https://github.com/zhenghu159"},date:"2023-04-23T21:15:10.000Z",permalink:"/pages/d28eb7/"},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\00.\\99. - \\03..md",wordsCount:116,readingTime:"1",title:"",date:"2023-08-26T15:12:36.000Z",permalink:"/pages/1a58a6/",categories:[""],tags:[""]},{name:"OneNote",filePath:"F:\\6.Github\\MyBlog\\docs\\01.\\001.OneNote.md",wordsCount:"1.4k",readingTime:"5.4m",title:"OneNote",date:"2023-09-03T17:03:37.000Z",permalink:"/pages/1260b7/",categories:[""],tags:[null]},{name:"Snipaste",filePath:"F:\\6.Github\\MyBlog\\docs\\01.\\002.Snipaste.md",wordsCount:296,readingTime:"1m",title:"Snipaste",date:"2023-09-04T21:01:39.000Z",permalink:"/pages/413897/",categories:[""],tags:[null]},{name:"NCBI",filePath:"F:\\6.Github\\MyBlog\\docs\\01.\\003.NCBI.md",wordsCount:143,readingTime:"1",title:"NCBI",date:"2024-07-26T21:08:20.000Z",permalink:"/pages/420e29/",categories:[""],tags:[null]},{name:"Seurat",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\001.Seurat.md",wordsCount:"2.4k",readingTime:"11.9m",title:"SeuratR toolkit for single cell genomics",date:"2023-01-31T16:57:11.000Z",permalink:"/pages/d207e7/",categories:[""," - "],tags:[""]},{name:"DoubletFinder",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\002.DoubletFinder.md",wordsCount:958,readingTime:"4.5m",title:"DoubletFinderR package for detecting doublets in single-cell RNA sequencing data",categories:[""," - "],tags:[""],permalink:"/pages/a35823/",date:"2022-11-19T09:25:14.000Z"},{name:"anndata",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\003.anndata.md",wordsCount:376,readingTime:"2m",title:"anndataa Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray",date:"2023-05-15T09:31:32.000Z",permalink:"/pages/7d0fcf/",categories:[""," - "],tags:[""]},{name:"MetaNeighbor",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\004.MetaNeighbor.md",wordsCount:"1.2k",readingTime:"5.3m",title:"MetaNeighbora method to rapidly assess cell type identity using both functional and random gene sets",date:"2023-04-28T09:56:18.000Z",permalink:"/pages/9d48f3/",categories:[""," - "],tags:[""]},{name:"scrattch",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\005.scrattch.md",wordsCount:690,readingTime:"2.9m",title:"scrattch.hicatHierarchical, Iterative Clustering for Analysis of Transcriptomics",date:"2023-05-15T10:10:25.000Z",permalink:"/pages/8dbd6f/",categories:[""," - "],tags:[""]},{name:"WGCNA",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\006.WGCNA.md",wordsCount:"1.3k",readingTime:"5.4m",title:"WGCNAan R package for weighted correlation network analysis",date:"2023-05-15T10:42:38.000Z",permalink:"/pages/9ffdf7/",categories:[""," - "],tags:[""]},{name:"hdWGCNA",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\007.hdWGCNA.md",wordsCount:"1.7k",readingTime:"8.1m",title:"hdWGCNAhigh dimensional WGCNA",date:"2023-05-04T08:25:04.000Z",permalink:"/pages/649303/",categories:[""," - "],tags:[""]},{name:"Monocle",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\008.Monocle.md",wordsCount:639,readingTime:"3m",title:"MonocleAn analysis toolkit for single-cell RNA-seq",date:"2022-12-04T09:51:09.000Z",permalink:"/pages/b62f01/",categories:[""," - "],tags:[""]},{name:"CellChat",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\009.CellChat.md",wordsCount:638,readingTime:"3m",title:"CellChatInference and analysis of cell-cell communication",date:"2023-05-15T15:26:27.000Z",permalink:"/pages/f26000/",categories:[""," - "],tags:[""]},{name:"NeuronChat",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\010.NeuronChat.md",wordsCount:509,readingTime:"2.6m",title:"NeuronChatInferring neuron-neuron communications from single-cell transcriptomics",date:"2023-04-26T20:22:25.000Z",permalink:"/pages/7a726f/",categories:[""," - "],tags:[""]},{name:"scCustomize",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\011.scCustomize.md",wordsCount:"1.3k",readingTime:"5.5m",title:"scCustomizeA collection of functions created and/or curated to aid in the visualization and analysis of single-cell data using R",date:"2023-05-14T09:45:23.000Z",permalink:"/pages/9e5aae/",categories:[""," - "],tags:[""]},{name:"Tangram",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\012.Tangram.md",wordsCount:"1.3k",readingTime:"6m",title:"TangramSpatial alignment of single cell transcriptomic data",date:"2023-09-19T09:58:09.000Z",permalink:"/pages/c7e56a/",categories:[""," - "],tags:[""]},{name:"scanpy",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\013.scanpy.md",wordsCount:421,readingTime:"2m",title:"ScanpySingle-Cell Analysis in Python",date:"2023-09-28T19:53:16.000Z",permalink:"/pages/61a60f/",categories:[""," - "],tags:[""]},{name:"squidpy",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\01. - \\014.squidpy.md",wordsCount:309,readingTime:"1.5m",title:"squidpySpatial Single Cell Analysis in Python",date:"2024-07-26T20:42:25.000Z",permalink:"/pages/37f62d/",categories:[""," - "],tags:[""]},{name:"XGBoost",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\02. - \\001.XGBoost.md",wordsCount:611,readingTime:"3.2m",title:"XGBoostScalable, Portable and Distributed Gradient Boosting (GBDT, GBRT or GBM) Library",date:"2023-05-10T20:43:26.000Z",permalink:"/pages/006dc9/",categories:[""," - "],tags:[""]},{name:"glmnet",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\02. - \\002.glmnet.md",wordsCount:575,readingTime:"2.5m",title:"glmnetLasso and Elastic-Net Regularized Generalized Linear Models",date:"2023-05-10T21:13:35.000Z",permalink:"/pages/ee4b70/",categories:[""," - "],tags:[""]},{name:"nnUNet",filePath:"F:\\6.Github\\MyBlog\\docs\\03.\\04. - \\001.nnUNet.md",wordsCount:"1.8k",readingTime:"7.4m",title:"nnUNeta self-configuring method for deep learning-based biomedical image segmentation",date:"2023-05-15T17:07:45.000Z",permalink:"/pages/8633a5/",categories:[""," - "],tags:[" - "]},{name:"HTML",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\11. - HTML\\01.HTML.md",wordsCount:856,readingTime:"3.2m",title:"HTML",date:"2023-04-21T20:59:41.000Z",permalink:"/pages/791228/",categories:[" - HTML"],tags:["HTML"]},{name:"HTML",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\11. - HTML\\02.HTML.md",wordsCount:58,readingTime:"1",title:"HTML",date:"2023-04-22T14:55:52.000Z",permalink:"/pages/18d66f/",categories:[" - HTML"],tags:["HTML"]},{name:"CSS",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\12. - CSS\\01.CSS.md",wordsCount:"5.9k",readingTime:"23.5m",title:"CSS",date:"2023-04-22T15:15:38.000Z",permalink:"/pages/bb1275/",categories:[" - CSS"],tags:["CSS"]},{name:"JavaScript",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\13. - JS&TS\\01.JavaScript.md",wordsCount:504,readingTime:"2.8m",title:"JavaScript ",date:"2024-04-02T17:11:13.000Z",permalink:"/pages/592a6d/",categories:[""],tags:[" - JavaScript"]},{name:"TypeScript",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\13. - JS&TS\\02.TypeScript.md",wordsCount:168,readingTime:"1",title:"TypeScript",date:"2024-07-26T21:28:51.000Z",permalink:"/pages/6ce63c/",categories:[""],tags:[" - TypeScript"]},{name:"Markdown",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\14. - Markdown\\01.Markdown.md",wordsCount:294,readingTime:"1.5m",title:"Markdown ",categories:[""],tags:["Markdown"],date:"2022-11-24T20:37:33.000Z",permalink:"/pages/da13bb/"},{name:"Vue3",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\15. - Vue\\01.Vue3.md",wordsCount:219,readingTime:"1.2m",title:"Vue3",date:"2024-07-26T21:35:25.000Z",permalink:"/pages/f03da3/",categories:["Web"],tags:[" - Vue"]},{name:"AJAX",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\16. - Ajax\\01.AJAX.md",wordsCount:135,readingTime:"1",title:"AJAX",date:"2024-07-26T21:54:53.000Z",permalink:"/pages/ce8da3/",categories:["Web"],tags:[" - Ajax"]},{name:"MySQL",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\21. - MySQL\\01.MySQL.md",wordsCount:144,readingTime:"1",title:"MySQL",date:"2024-07-26T21:55:15.000Z",permalink:"/pages/49a104/",categories:["Web"],tags:[" - MySQL"]},{name:"Django",filePath:"F:\\6.Github\\MyBlog\\docs\\04.Web\\22. - Django\\01.Django.md",wordsCount:78,readingTime:"1",title:"Django",date:"2024-07-26T21:55:35.000Z",permalink:"/pages/b75e1a/",categories:["Web"],tags:[" - Django"]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\01.\\1..md",wordsCount:256,readingTime:"1.1m",title:"",date:"2022-12-28T19:32:00.000Z",permalink:"/pages/d7e3cb/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\01.\\2..md",wordsCount:"2.2k",readingTime:"10.5m",title:"",date:"2023-01-02T19:49:17.000Z",permalink:"/pages/02fc82/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\01.\\3..md",wordsCount:"1.9k",readingTime:"9.1m",title:"",date:"2023-01-02T20:09:47.000Z",permalink:"/pages/4c1aba/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\01.\\4..md",wordsCount:"2.9k",readingTime:"12.2m",title:"",date:"2023-01-02T20:10:43.000Z",permalink:"/pages/cbc016/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\11. - \\1..md",wordsCount:"1.5k",readingTime:"5.4m",title:"1 ",date:"2022-12-04T20:48:11.000Z",permalink:"/pages/4a4e4d/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\11. - \\2..md",wordsCount:"4.9k",readingTime:"18.2m",title:"2 ",date:"2022-12-05T21:15:19.000Z",permalink:"/pages/f7fe10/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\11. - \\3..md",wordsCount:324,readingTime:"1.2m",title:"3 ",date:"2022-12-20T15:37:40.000Z",permalink:"/pages/3e695c/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\12. - \\01..md",wordsCount:488,readingTime:"1.9m",title:"(Introduction)",date:"2023-04-21T19:15:41.000Z",permalink:"/pages/e2487d/",categories:[" - "],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\12. - \\02..md",wordsCount:"2.4k",readingTime:"9.8m",title:"(Linear Regression with One Variable)",date:"2023-04-21T19:16:35.000Z",permalink:"/pages/64abf4/",categories:[" - "],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\12. - \\03..md",wordsCount:"1.1k",readingTime:"4.8m",title:"(Linear Algebra Review)",date:"2023-05-05T21:18:57.000Z",permalink:"/pages/0952ad/",categories:[" - "],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\12. - \\04..md",wordsCount:"2.6k",readingTime:"10.9m",title:"(Linear Regression with Multiple Variables)",date:"2023-05-07T20:52:02.000Z",permalink:"/pages/95b5f9/",categories:[" - "],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\05.\\12. - \\06..md",wordsCount:"4.4k",readingTime:"17.8m",title:"(Logistic Regression)",date:"2023-05-09T21:01:11.000Z",permalink:"/pages/81f7ef/",categories:[" - "],tags:[""]},{name:"tidyverse",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\12.R - tidyverse\\01.tidyverse.md",wordsCount:669,readingTime:"3.5m",title:"tidyverse",date:"2023-07-03T21:14:09.000Z",permalink:"/pages/c63497/",categories:["R","tidyverse"],tags:["tidyverse"]},{name:"ggplot2",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\12.R - tidyverse\\02.ggplot2.md",wordsCount:942,readingTime:"4.2m",title:"ggplot2",date:"2023-07-03T21:14:18.000Z",permalink:"/pages/98868d/",categories:["R","tidyverse"],tags:["tidyverse"]},{name:"ggalluvial",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\15.R - \\004.ggalluvial.md",wordsCount:786,readingTime:"3.7m",title:"ggalluvial",date:"2023-05-24T16:07:05.000Z",permalink:"/pages/29214b/",categories:["R","R "],tags:["R "]},{name:"DiagrammeR",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\15.R - \\005.DiagrammeR.md",wordsCount:564,readingTime:"2.6m",title:"DiagrammeR",date:"2023-06-25T09:22:14.000Z",permalink:"/pages/6580bf/",categories:["R","R "],tags:["R "]},{name:"linkET",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\15.R - \\006.linkET.md",wordsCount:548,readingTime:"2.2m",title:"linkET",date:"2023-07-15T16:11:13.000Z",permalink:"/pages/142715/",categories:["R","R "],tags:["R "]},{name:"circlize",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\15.R - \\007.circlize.md",wordsCount:388,readingTime:"1.9m",title:"circlize",date:"2023-08-21T20:26:22.000Z",permalink:"/pages/a41a6b/",categories:["","R"],tags:["R "]},{name:"Biorplot",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\15.R - \\008.Biorplot.md",wordsCount:224,readingTime:"1.1m",title:"Biorplot",date:"2024-07-27T09:40:16.000Z",permalink:"/pages/22fafa/",categories:[""],tags:["R - "]},{name:"quarto",filePath:"F:\\6.Github\\MyBlog\\docs\\06.\\16.R - \\01.quarto.md",wordsCount:213,readingTime:"1.2m",title:"quarto",date:"2024-07-27T09:30:13.000Z",permalink:"/pages/02875c/",categories:[""],tags:["R - "]},{name:"1 ",filePath:"F:\\6.Github\\MyBlog\\docs\\07.\\11. - \\01.1 .md",wordsCount:"10.6k",readingTime:"36.5m",title:"1 ",date:"2023-05-03T20:16:27.000Z",permalink:"/pages/14b7e3/",categories:[""," - "],tags:[""]},{name:" - ",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\01. - \\01. - .md",wordsCount:508,readingTime:"2.8m",title:" - ",date:"2022-07-30T22:29:33.000Z",permalink:"/about/me/introduce/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\01.-.md",wordsCount:"1.1k",readingTime:"4.3m",title:"-",date:"2023-04-16T15:07:33.000Z",permalink:"/pages/d3830b/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\02.-.md",wordsCount:696,readingTime:"2.7m",title:"-",date:"2023-04-16T15:39:00.000Z",permalink:"/pages/e6cfba/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\03.-.md",wordsCount:906,readingTime:"3.5m",title:"-",date:"2023-04-18T08:28:49.000Z",permalink:"/pages/94f5fc/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\04.-.md",wordsCount:292,readingTime:"1.6m",title:"-",date:"2023-04-19T19:53:03.000Z",permalink:"/pages/093f81/",categories:[" - "],tags:[""]},{name:"-Markdown",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\05.-Markdown.md",wordsCount:"2.6k",readingTime:"11.3m",title:"-Markdown",date:"2023-04-19T20:35:21.000Z",permalink:"/pages/50efc7/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\06.-.md",wordsCount:383,readingTime:"1.9m",title:"-",date:"2023-04-20T08:10:37.000Z",permalink:"/pages/c4fda3/",categories:[" - "],tags:[""]},{name:"-",filePath:"F:\\6.Github\\MyBlog\\docs\\100.\\02. - \\07.-.md",wordsCount:651,readingTime:"2.5m",title:"-",date:"2023-04-20T21:41:32.000Z",permalink:"/pages/25bbfb/",categories:[" - "],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\99.\\01..md",wordsCount:"32.2k",readingTime:"2h23m",title:"",date:"2023-04-24T08:40:27.000Z",permalink:"/pages/eccd67/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\99.\\02..md",wordsCount:199,readingTime:"1",title:"",date:"2023-04-23T21:22:07.000Z",permalink:"/pages/30653b/",categories:[""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\99.\\03.\\001..md",wordsCount:583,readingTime:"2m",title:"",date:"2023-08-26T15:18:30.000Z",permalink:"/pages/46a15a/",categories:["",""],tags:[""]},{name:"",filePath:"F:\\6.Github\\MyBlog\\docs\\99.\\03.\\002..md",wordsCount:358,readingTime:"1.2m",title:"",date:"2023-08-26T21:49:11.000Z",permalink:"/pages/aef061/",categories:["",""],tags:[""]}],mdFileCountType:"archives",totalWords:"archives",moutedEvent:".tags-wrapper",indexIteration:2500,pageIteration:2500},htmlModules:{homeSidebarB:'<div style="padding: 0.95rem">\n    <p style="\n      color: var(--textColor);\n      opacity: 0.9;\n      font-size: 20px;\n      font-weight: bold;\n      margin: 0 0 8px 0;\n    "></p>\n    <img src="https://open.weixin.qq.com/qr/code?username=gh_55ff0104c230"  style="width:100%;" />\n    \n    </p>\n    </div>'}},locales:{"/":{lang:"zh-CN",title:"TigerZ Blog",description:"TigerZ, VuePress,  Vdoing ",path:"/"}}};var pt=t(98),ut=t(99),mt=t(13);var ht={computed:{$filterPosts(){return this.$site.pages.filter(n=>{const{frontmatter:{pageComponent:e,article:t,home:i}}=n;return!(e||!1===t||!0===i)})},$sortPosts(){return(n=this.$filterPosts).sort((n,e)=>{const t=n.frontmatter.sticky,i=e.frontmatter.sticky;return t&&i?t==i?Object(mt.a)(n,e):t-i:t&&!i?-1:!t&&i?1:Object(mt.a)(n,e)}),n;var n},$sortPostsByDate(){return(n=this.$filterPosts).sort((n,e)=>Object(mt.a)(n,e)),n;var n},$groupPosts(){return function(n){const e={},t={};for(let i=0,a=n.length;i<a;i++){const{frontmatter:{categories:a,tags:r}}=n[i];"array"===Object(mt.n)(a)&&a.forEach(t=>{t&&(e[t]||(e[t]=[]),e[t].push(n[i]))}),"array"===Object(mt.n)(r)&&r.forEach(e=>{e&&(t[e]||(t[e]=[]),t[e].push(n[i]))})}return{categories:e,tags:t}}(this.$sortPosts)},$categoriesAndTags(){return function(n){const e=[],t=[];for(let t in n.categories)e.push({key:t,length:n.categories[t].length});for(let e in n.tags)t.push({key:e,length:n.tags[e].length});return{categories:e,tags:t}}(this.$groupPosts)}}};i.default.component(pt.default),i.default.component(ut.default);function ft(n){return n.toString().padStart(2,"0")}t(246);i.default.component("Aplayer",()=>t.e(15).then(t.bind(null,362))),i.default.component("BlockToggle",()=>Promise.all([t.e(0),t.e(4)]).then(t.bind(null,363))),i.default.component("Bubble",()=>Promise.all([t.e(0),t.e(5)]).then(t.bind(null,364))),i.default.component("Fantasy",()=>Promise.all([t.e(0),t.e(7)]).then(t.bind(null,365))),i.default.component("Card",()=>Promise.all([t.e(0),t.e(6)]).then(t.bind(null,366))),i.default.component("GlobalTip",()=>Promise.all([t.e(0),t.e(8)]).then(t.bind(null,367))),i.default.component("iframeComp",()=>t.e(17).then(t.bind(null,368))),i.default.component("IndexBigImg",()=>Promise.all([t.e(0),t.e(9)]).then(t.bind(null,369))),i.default.component("Login",()=>Promise.all([t.e(0),t.e(10)]).then(t.bind(null,370))),i.default.component("PageInfo",()=>t.e(14).then(t.bind(null,371))),i.default.component("SidebarReverse",()=>t.e(16).then(t.bind(null,372))),i.default.component("WebInfo",()=>Promise.all([t.e(0),t.e(3)]).then(t.bind(null,357))),i.default.component("Twikoo",()=>Promise.all([t.e(0),t.e(11)]).then(t.bind(null,373))),i.default.component("Badge",()=>Promise.all([t.e(0),t.e(12)]).then(t.bind(null,559))),i.default.component("CodeGroup",()=>Promise.resolve().then(t.bind(null,99))),i.default.component("CodeBlock",()=>Promise.resolve().then(t.bind(null,98)));t(247);var bt={name:"ReadingProgress",data:()=>({readingTop:0,readingHeight:1,progressStyle:null,transform:void 0,running:!1}),watch:{$readingShow(){this.progressStyle=this.getProgressStyle(),this.$readingShow&&window.addEventListener("scroll",this.base)}},mounted(){this.transform=this.getTransform(),this.progressStyle=this.getProgressStyle(),this.$readingShow&&window.addEventListener("scroll",this.base)},beforeDestroy(){this.$readingShow&&window.removeEventListener("scroll",this.base)},methods:{base(){this.running||(this.running=!0,requestAnimationFrame(this.getReadingBase))},getReadingBase(){this.readingHeight=this.getReadingHeight()-this.getScreenHeight(),this.readingTop=this.getReadingTop(),this.progressStyle=this.getProgressStyle(),this.running=!1},getReadingHeight:()=>Math.max(document.body.scrollHeight,document.body.offsetHeight,0),getScreenHeight:()=>Math.max(window.innerHeight,document.documentElement.clientHeight,0),getReadingTop:()=>Math.max(window.pageYOffset,document.documentElement.scrollTop,0),getTransform(){const n=document.createElement("div");return["transform","-webkit-transform","-moz-transform","-o-transform","-ms-transform"].find(e=>e in n.style)||void 0},getProgressStyle(){const n=this.readingTop/this.readingHeight;switch(this.$readingShow){case"top":case"bottom":return this.transform?`${this.transform}: scaleX(${n})`:`width: ${100*n}%`;case"left":case"right":return this.transform?`${this.transform}: scaleY(${n})`:`height: ${100*n}%`;default:return null}}}},vt=(t(248),Object(lt.a)(bt,(function(){var n=this.$createElement,e=this._self._c||n;return e("ClientOnly",[this.$readingShow?e("div",{staticClass:"reading-progress",class:this.$readingShow},[e("div",{staticClass:"progress",style:this.progressStyle})]):this._e()])}),[],!1,null,"3640397f",null).exports),yt=t(97),xt=t.n(yt),Ct=(n,e,t)=>{if(!e.has(n))throw TypeError("Cannot "+t)},At=(n,e,t)=>(Ct(n,e,"read from private field"),t?t.call(n):e.get(n)),wt=(n,e,t)=>{if(e.has(n))throw TypeError("Cannot add the same private member more than once");e instanceof WeakSet?e.add(n):e.set(n,t)},kt=(n,e,t,i)=>(Ct(n,e,"write to private field"),i?i.call(n,t):e.set(n,t),t);var Tt,Et;const Ft=class{constructor(n,e,t,i=!0){wt(this,Tt,void 0),wt(this,Et,void 0),kt(this,Tt,{width:0,height:0});const{el:a,ctx:r}=Ft.initCanvas(n);this.el=a,this.ctx=r,kt(this,Et,i),this.size={width:e||window.innerWidth,height:t||window.innerHeight}}get size(){return{...At(this,Tt)}}set size({width:n,height:e}){var t;if(At(this,Tt).width===n&&At(this,Tt).height===e)return;At(this,Tt).width=n,At(this,Tt).height=e;const i=null!=(t=At(this,Et)?window.devicePixelRatio:1)?t:1;this.el.width=Math.round(At(this,Tt).width*i),this.el.height=Math.round(At(this,Tt).height*i),this.el.style.width=At(this,Tt).width+"px",this.el.style.height=At(this,Tt).height+"px",At(this,Et)&&this.ctx.scale(i,i)}clear(){Ft.clearCanvas(this.ctx,{...At(this,Tt)})}to(n){n.ctx.drawImage(this.el,0,0,At(this,Tt).width,At(this,Tt).height)}handleResize(n){this.size={width:window.innerWidth,height:window.innerHeight}}static setCanvasStyle(n,e,t){const i=n.style,{zIndex:a=0,opacity:r=1}=e;i.position="fixed",i.top="0",i.left="0",i.zIndex=a.toString(),i.width=(t?t.width:n.width).toString()+"px",i.height=(t?t.height:n.height).toString()+"px",1!==r&&(i.opacity=r.toString()),i.pointerEvents="none"}static initCanvas(n){n||(n=document.createElement("canvas"));const e=n.getContext("2d");return{el:n,ctx:e}}static createOffscreenCanvas(){return new Ft}static clearCanvas(n,e){const{width:t,height:i}=e;n.clearRect(0,0,t,i)}};let St=Ft;var _t,Pt;Tt=new WeakMap,Et=new WeakMap;class qt{constructor(n,e,t,i=!0,a=!0,r={zIndex:0,opacity:1}){wt(this,_t,void 0),wt(this,Pt,void 0),kt(this,_t,new St(n,e,t,i)),St.setCanvasStyle(At(this,_t).el,r,{width:e,height:t}),kt(this,Pt,a?new St(void 0,e,t,i):null)}get size(){return At(this,_t).size}draw(n){var e;const t=null!=(e=At(this,Pt))?e:At(this,_t);t.clear(),n(t.ctx,{...t.size})}render(){!At(this,Pt)||(At(this,_t).clear(),At(this,Pt).to(At(this,_t)))}handleResize(n){At(this,_t).handleResize(n),At(this,Pt)&&At(this,Pt).handleResize(n)}clear(){At(this,_t).clear(),At(this,Pt)&&At(this,Pt).clear()}}_t=new WeakMap,Pt=new WeakMap;var It=(n,e,t)=>{if(!e.has(n))throw TypeError("Cannot "+t)},zt=(n,e,t)=>(It(n,e,"read from private field"),t?t.call(n):e.get(n));var jt;class Nt{constructor(){((n,e,t)=>{if(e.has(n))throw TypeError("Cannot add the same private member more than once");e instanceof WeakSet?e.add(n):e.set(n,t)})(this,jt,void 0),((n,e,t,i)=>{It(n,e,"write to private field"),i?i.call(n,t):e.set(n,t)})(this,jt,new Map)}add(n,e,t=window){zt(this,jt).has(t)||zt(this,jt).set(t,new Map);const i=zt(this,jt).get(t);i.has(n)||i.set(n,new Set),i.get(n).add(e)}startAll(){for(const[n,e]of zt(this,jt))for(const[t,i]of e)for(const e of i)n.addEventListener(t,e)}stopAll(){for(const[n,e]of zt(this,jt))for(const[t,i]of e)for(const e of i)n.removeEventListener(t,e)}clear(){zt(this,jt).clear()}}function Bt(n){return!!n.touches}jt=new WeakMap;class Dt{static randomFloat(n,e){return Math.random()*(e-n)+n}static randomInt(n,e){return Math.floor(Dt.randomFloat(n,e))}static choice(n){const e=n.length;return n[Math.floor(e*Math.random())]}static color(n="0123456789ABCDEF"){return"#"+Dt.choice(n)+Dt.choice(n)+Dt.choice(n)+Dt.choice(n)+Dt.choice(n)+Dt.choice(n)}}var Rt,Mt,Lt,Ot,Gt,Ht=(n,e,t)=>{if(!e.has(n))throw TypeError("Cannot "+t)},Ut=(n,e,t)=>(Ht(n,e,"read from private field"),t?t.call(n):e.get(n)),Vt=(n,e,t)=>{if(e.has(n))throw TypeError("Cannot add the same private member more than once");e instanceof WeakSet?e.add(n):e.set(n,t)},Zt=(n,e,t,i)=>(Ht(n,e,"write to private field"),i?i.call(n,t):e.set(n,t),t),Wt=(n,e,t)=>(Ht(n,e,"access private method"),t);class Xt{constructor(n,e,t,i,a){Vt(this,Rt,void 0),Vt(this,Mt,void 0),Vt(this,Lt,void 0),this.size=t,this.color=i,Zt(this,Lt,0),Zt(this,Rt,a),Zt(this,Mt,e),this.position={...n}}move(){this.position.x=Math.sin(Ut(this,Rt))*Ut(this,Mt)+this.position.x,this.position.y=Math.cos(Ut(this,Rt))*Ut(this,Mt)+this.position.y+.3*Ut(this,Lt),((n,e,t,i)=>({set _(i){Zt(n,e,i,t)},get _(){return Ut(n,e,i)}}))(this,Lt)._++}shouleRemove(n){return this.position.x<0||this.position.x>n.width||this.position.y>n.height}}Rt=new WeakMap,Mt=new WeakMap,Lt=new WeakMap;Ot=new WeakMap;class $t{static create(n,e,t,i,a,r){return new(this.shapeMap.get(n))(e,t,i,a,r)}}$t.shapeMap=new Map([["star",class extends Xt{constructor(n,e,t,i,a){super(n,e,t,i,a),Vt(this,Ot,0)}draw(n,e){n.fillStyle=this.color,n.beginPath();const t=2*this.size,i=this.size;for(let e=0;e<5;e++)n.lineTo(Math.cos((18+72*e-Ut(this,Ot))/180*Math.PI)*t+this.position.x,-Math.sin((18+72*e-Ut(this,Ot))/180*Math.PI)*t+this.position.y),n.lineTo(Math.cos((54+72*e-Ut(this,Ot))/180*Math.PI)*i+this.position.x,-Math.sin((54+72*e-Ut(this,Ot))/180*Math.PI)*i+this.position.y);n.fill(),Zt(this,Ot,Ut(this,Ot)+5)}}],["circle",class extends Xt{draw(n,e){n.fillStyle=this.color,n.beginPath(),n.arc(this.position.x,this.position.y,this.size,0,2*Math.PI),n.fill()}}]]);class Jt{constructor(n,e,t,i){Vt(this,Gt,void 0),this.stopped=!1,Zt(this,Gt,new Set);for(let a=0;a<i;a++){const i=$t.create(n,e,Dt.randomFloat(1,6),t,Dt.color("89ABCDEF"),Dt.randomFloat(Math.PI-1,Math.PI+1));Ut(this,Gt).add(i)}}move(n){for(const e of Ut(this,Gt))e.shouleRemove(n)?Ut(this,Gt).delete(e):e.move();0===Ut(this,Gt).size&&(this.stopped=!0)}draw(n,e){for(const t of Ut(this,Gt))t.draw(n,e)}}Gt=new WeakMap;var Kt,Yt,Qt,ni,ei,ti,ii,ai,ri,oi,si,li,ci,gi,di,pi,ui,mi,hi,fi;class bi{constructor({shape:n="star",size:e=2,numParticles:t=10}={},i={}){Vt(this,ri),Vt(this,si),Vt(this,ci),Vt(this,di),Vt(this,ui),Vt(this,hi),Vt(this,Kt,void 0),Vt(this,Yt,void 0),Vt(this,Qt,void 0),Vt(this,ni,null),Vt(this,ei,new Set),Vt(this,ti,!1),Vt(this,ii,void 0),Vt(this,ai,new Nt),Zt(this,Kt,n),Zt(this,Yt,e),Zt(this,Qt,t),Zt(this,ii,i),this.animate=this.animate.bind(this)}mount(n){Zt(this,ni,new qt(n,window.innerWidth,window.innerHeight,!0,!0,Ut(this,ii))),Wt(this,ri,oi).call(this),function(n,{leftColor:e="#fff",rightColor:t="#444",leftBgColor:i="#35495e",rightBgColor:a="#00ffc0"}={}){console.log(`%c ${n} %c v0.5.2 112fa81 %c`,`background: ${i}; padding: 2px; color: ${e}; font-weight: bold; text-transform: uppercase;`,`background: ${a}; padding: 2px; color: ${t}; font-weight: bold; text-transform: uppercase;`,"background: transparent")}("Theme Popper ",{leftBgColor:"#ffb366"})}unmount(){Wt(this,si,li).call(this),Zt(this,ti,!1)}animate(){if(Zt(this,ti,!0),0===Ut(this,ei).size)return Zt(this,ti,!1),void Ut(this,ni).clear();requestAnimationFrame(this.animate);for(const n of Ut(this,ei)){if(n.stopped)return void Ut(this,ei).delete(n);n.move(Ut(this,ni).size)}Ut(this,ni).draw((n,e)=>{for(const t of Ut(this,ei))t.draw(n,e)}),Ut(this,ni).render()}}Kt=new WeakMap,Yt=new WeakMap,Qt=new WeakMap,ni=new WeakMap,ei=new WeakMap,ti=new WeakMap,ii=new WeakMap,ai=new WeakMap,ri=new WeakSet,oi=function(){/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)?Ut(this,ai).add("touchstart",Wt(this,ci,gi).bind(this)):Ut(this,ai).add("mousedown",Wt(this,ci,gi).bind(this)),Ut(this,ai).add("visibilitychange",Wt(this,ui,mi).bind(this)),Ut(this,ai).add("resize",function(n,e,t){var i,a,r;void 0===e&&(e=50),void 0===t&&(t={});var o=null!=(i=t.isImmediate)&&i,s=null!=(a=t.callback)&&a,l=t.maxWait,c=Date.now(),g=[];function d(){if(void 0!==l){var n=Date.now()-c;if(n+e>=l)return l-n}return e}var p=function(){var e=[].slice.call(arguments),t=this;return new Promise((function(i,a){var l=o&&void 0===r;if(void 0!==r&&clearTimeout(r),r=setTimeout((function(){if(r=void 0,c=Date.now(),!o){var i=n.apply(t,e);s&&s(i),g.forEach((function(n){return(0,n.resolve)(i)})),g=[]}}),d()),l){var p=n.apply(t,e);return s&&s(p),i(p)}g.push({resolve:i,reject:a})}))};return p.cancel=function(n){void 0!==r&&clearTimeout(r),g.forEach((function(e){return(0,e.reject)(n)})),g=[]},p}(Wt(this,di,pi).bind(this),500)),Ut(this,ai).startAll()},si=new WeakSet,li=function(){Ut(this,ai).stopAll(),Ut(this,ai).clear()},ci=new WeakSet,gi=function(n){const e={x:Bt(n)?n.touches[0].clientX:n.clientX,y:Bt(n)?n.touches[0].clientY:n.clientY},t=new Jt(Ut(this,Kt),{...e},Ut(this,Yt),Ut(this,Qt));Ut(this,ei).add(t),Ut(this,ti)||Wt(this,hi,fi).call(this)},di=new WeakSet,pi=function(n){Ut(this,ni).handleResize(n)},ui=new WeakSet,mi=function(n){Ut(this,ei).clear(),Zt(this,ti,!1)},hi=new WeakSet,fi=function(){requestAnimationFrame(this.animate)};var vi={name:"CursorEffects",data:()=>({popper:new bi({shape:"circle",size:2},{opacity:1,zIndex:999999999})}),mounted(){this.popper.mount(this.$el)},beforeDestroy(){this.popper.unmount()}},yi=Object(lt.a)(vi,(function(){var n=this.$createElement;return(this._self._c||n)("canvas",{attrs:{id:"vuepress-canvas-cursor"}})}),[],!1,null,null,null).exports,xi={name:"DynamicTitle",data:()=>({originTitle:"",recoverTimeout:null,config:{showIcon:"/favicon.ico",showText:"(*)o",hideIcon:"/failure.ico",hideText:"_",recoverTime:2e3}}),mounted(){this.originTitle=document.title,""!==this.config.showIcon&&this.getIconElm().setAttribute("href",this.config.showIcon),document.addEventListener("visibilitychange",()=>{document.hidden?this.hidden():this.visible()})},methods:{hidden(){""!==this.config.hideIcon&&this.getIconElm().setAttribute("href",this.config.hideIcon),document.title=this.config.hideText,clearTimeout(this.recoverTimeout)},visible(){""!==this.config.showIcon&&this.getIconElm().setAttribute("href",this.config.showIcon),document.title=this.config.showText+this.originTitle,this.recoverTimeout=setTimeout(()=>{document.title=this.originTitle},this.config.recoverTime)},getIconElm(){let n=document.querySelector("link[rel=icon]");return null===n&&(n=document.createElement("link"),n.setAttribute("rel","icon"),document.head.appendChild(n)),n}},watch:{$route(n,e){n.path!==e.path&&(this.originTitle=document.title,clearTimeout(this.recoverTimeout))}}},Ci=Object(lt.a)(xi,(function(){var n=this.$createElement;return(this._self._c||n)("div")}),[],!1,null,null,null).exports,Ai={name:"Meting",props:{auto:{required:!1,type:String,default:""},server:{required:!1,type:String,default:""},type:{required:!1,type:String,default:""},mid:{required:!1,type:String,default:""},additionalAudios:{required:!1,type:Array,default:()=>[]},fixed:{required:!1,type:Boolean,default:!1},mini:{required:!1,type:Boolean,default:null},autoplay:{required:!1,type:Boolean,default:!1},theme:{required:!1,type:String,default:"#b7daff"},loop:{required:!1,type:String,default:"all"},order:{required:!1,type:String,default:"list"},preload:{required:!1,type:String,default:"auto"},volume:{required:!1,type:Number,default:.7},customAudioType:{required:!1,type:Object,default:void 0},mutex:{required:!1,type:Boolean,default:!0},lrcType:{required:!1,type:Number,default:0},listFolded:{required:!1,type:Boolean,default:!1},listMaxHeight:{required:!1,type:Number,default:250},storageName:{required:!1,type:String,default:"vuepress-plugin-meting"}},data:()=>({metingApi:"https://api.i-meto.com/meting/api",audio:[]}),mounted(){let n;if(this.auto){const e=this.parse_link(this.auto);n={...e,r:Math.random()}}else n={server:this.server,type:this.type,id:this.mid,r:Math.random()};let e=this.metingApi,t=[];Object.keys(n).forEach(e=>t.push(e+"="+n[e])),e+="?"+t.join("&"),fetch(e,{headers:{referer:null}}).then(n=>n.json()).then(n=>{const e=n.map(n=>({name:n.name||n.title||"Audio name",artist:n.artist||n.author||"Audio artist",url:n.url,cover:n.cover||n.pic,lrc:n.lrc||n.lyric||"",type:n.type||"auto"}));this.audio=e.concat(this.additionalAudios)})},methods:{parse_link(n){let e=[["music.163.com.*song.*id=(\\d+)","netease","song"],["music.163.com.*album.*id=(\\d+)","netease","album"],["music.163.com.*artist.*id=(\\d+)","netease","artist"],["music.163.com.*playlist.*id=(\\d+)","netease","playlist"],["music.163.com.*discover/toplist.*id=(\\d+)","netease","playlist"],["y.qq.com.*song/(\\w+).html","tencent","song"],["y.qq.com.*album/(\\w+).html","tencent","album"],["y.qq.com.*singer/(\\w+).html","tencent","artist"],["y.qq.com.*playsquare/(\\w+).html","tencent","playlist"],["y.qq.com.*playlist/(\\w+).html","tencent","playlist"],["xiami.com.*song/(\\w+)","xiami","song"],["xiami.com.*album/(\\w+)","xiami","album"],["xiami.com.*artist/(\\w+)","xiami","artist"],["xiami.com.*collect/(\\w+)","xiami","playlist"]];for(let t of e){let e=new RegExp(t[0]).exec(n);if(null!==e)return{server:t[1],type:t[2],id:e[1]}}return console.error(`: ${n}`),{server:"",type:"",id:""}}}},wi=Object(lt.a)(Ai,(function(){var n=this,e=n.$createElement;return(n._self._c||e)("APlayer",{attrs:{audio:n.audio,fixed:n.fixed,mini:n.mini,autoplay:n.autoplay,theme:n.theme,loop:n.loop,order:n.order,preload:n.preload,volume:n.volume,"custom-audio-type":n.customAudioType,mutex:n.mutex,"lrc-type":n.lrcType,"list-folded":n.listFolded,"list-max-height":n.listMaxHeight,"storage-name":n.storageName}})}),[],!1,null,null,null).exports,ki={name:"MetingGlobal",data:()=>({auto:"",server:"",type:"",mid:"",additionalAudios:[],mini:null,autoplay:!1,theme:"#b7daff",loop:"all",order:"list",preload:"auto",volume:.7,mutex:!0,lrcType:0,listFolded:!1,listMaxHeight:250,storageName:"vuepress-plugin-meting"}),mounted(){/Android|webOS|iPhone|iPod|BlackBerry/i.test(navigator.userAgent)&&this.aplayer_fixed_mobile_switch()},methods:{aplayer_fixed_mobile_switch(){const n=document.getElementById("aplayer-fixed");if(null===n)return;n.querySelector(".aplayer-body");const e=n.querySelector(".aplayer-miniswitcher");n.querySelector(".aplayer-lrc");let t=!1;function i(){t=!1}i(),e.onclick=()=>{t?i():t=!0}}}},Ti=Object(lt.a)(ki,(function(){var n=this,e=n.$createElement;return(n._self._c||e)("Meting",{attrs:{id:"aplayer-fixed",auto:n.auto,server:n.server,type:n.type,mid:n.mid,"additional-audios":n.additionalAudios,fixed:!0,mini:n.mini,autoplay:n.autoplay,theme:n.theme,loop:n.loop,order:n.order,preload:n.preload,volume:n.volume,mutex:n.mutex,"lrc-type":n.lrcType,"list-folded":n.listFolded,"list-max-height":n.listMaxHeight,"storage-name":n.storageName}})}),[],!1,null,null,null).exports,Ei=[({Vue:n,options:e,router:t,siteData:i,isServer:a})=>{function r(n,e){try{n.forEach(n=>{if(e.username==n.username&&e.password==n.password)throw new Error})}catch(n){return!1}return!0}function o(n,e,i){throw t.push({path:n,query:{toPath:e,verifyMode:i}}),new Error("")}a||i.themeConfig.privatePage&&i.themeConfig.privatePage.openPrivate&&t.beforeEach((n,e,t)=>{try{let{username:e,password:t,loginPath:a,loginKey:s,loginSession:l,loginInfo:c,firstLogin:g,firstLoginKey:d}=i.themeConfig.privatePage;if(!s&&(s="vdoing_manager"),!d&&(d="vdoing_first_login"),l&&window.addEventListener("unload",(function(){localStorage.removeItem(s),localStorage.removeItem(d)})),a==n.path||!a)throw new Error("");let p=JSON.parse(localStorage.getItem(s));if(p&&p.username==e&&p.password==t){if(!(new Date-p.time>p.expire))throw new Error("");localStorage.removeItem(s)}let u=!0;if(1==parseInt(g)||2==parseInt(g)){2==parseInt(g)&&(u=!1);let e=JSON.parse(localStorage.getItem(d));!e&&o(a,n.path,"first"),e&&(c&&c.hasOwnProperty(d)?r(c[d],e)&&o(a,n.path,"first"):o(a,n.path,"first"))}if("/"==n.path)throw new Error("");u&&i.pages.forEach(e=>{if(e.path==n.path&&e.frontmatter.private&&1==e.frontmatter.private){l&&window.addEventListener("unload",(function(){localStorage.removeItem(e.frontmatter.permalink)}));let t=JSON.parse(localStorage.getItem(e.frontmatter.permalink));if(!t&&o(a,n.path,e.frontmatter.loginInfo||e.frontmatter.username||e.frontmatter.password||e.frontmatter.expire?"single":"all"),e.frontmatter.username&&e.frontmatter.password||e.frontmatter.loginInfo){!t&&o(a,n.path,"single"),new Date-t.time>t.expire&&(localStorage.removeItem(e.frontmatter.permalink),o(a,n.path,"single"));let i=!0;Array.isArray(e.frontmatter.loginInfo)&&(i=r(e.frontmatter.loginInfo,t)),i&&t.username!==e.frontmatter.username&&t.password!==e.frontmatter.password&&o(a,n.path,"single")}else{let e=!0;c&&c.hasOwnProperty(n.path)&&(e=r(c[n.path],t)),e&&o(a,n.path,"all")}}})}catch(n){}t()})},({Vue:n,options:e,router:t,siteData:i})=>{i.pages.map(n=>{const{frontmatter:{date:e,author:t}}=n;"string"==typeof e&&"Z"===e.charAt(e.length-1)&&(n.frontmatter.date=function(n){n instanceof Date||(n=new Date(n));return`${n.getUTCFullYear()}-${ft(n.getUTCMonth()+1)}-${ft(n.getUTCDate())} ${ft(n.getUTCHours())}:${ft(n.getUTCMinutes())}:${ft(n.getUTCSeconds())}`}(e)),t?n.author=t:i.themeConfig.author&&(n.author=i.themeConfig.author)}),n.mixin(ht)},{},({Vue:n})=>{n.mixin({computed:{$dataBlock(){return this.$options.__data__block__}}})},{},{},({router:n})=>{"undefined"!=typeof window&&function(){var n=document.createElement("script"),e=window.location.protocol.split(":")[0];n.src="https"===e?"https://zz.bdstatic.com/linksubmit/push.js":"http://push.zhanzhang.baidu.com/push.js";var t=document.getElementsByTagName("script")[0];t.parentNode.insertBefore(n,t)}()},({router:n})=>{"undefined"!=typeof window&&(window._hmt=window._hmt||[],function(){var n=document.createElement("script");n.src="https://hm.baidu.com/hm.js?267c5680c2ffb468ca29c45ffe6801da";var e=document.getElementsByTagName("script")[0];e.parentNode.insertBefore(n,e)}(),n.afterEach((function(n){_hmt.push(["_trackPageview",n.fullPath])})))},({Vue:n})=>{n.component(vt.name,vt),n.mixin({computed:{$readingShow(){return this.$page.frontmatter.readingShow}}})},({Vue:n})=>n.use(xt.a),({Vue:n})=>{n.component("CursorEffects",yi)},({Vue:n})=>{n.component("DynamicTitle",Ci)},({Vue:n})=>{"undefined"!=typeof window&&(localStorage.setItem("aplayer-setting","[]"),n.use(t(249).default,{defaultCover:"https://github.com/SigureMo.png"})),n.component("Meting",wi),n.component("MetingGlobal",Ti)}],Fi=["PageInfo","BlockToggle","GlobalTip","Twikoo","ReadingProgress","CursorEffects","DynamicTitle"];class Si extends class{constructor(){this.store=new i.default({data:{state:{}}})}$get(n){return this.store.state[n]}$set(n,e){i.default.set(this.store.state,n,e)}$emit(...n){this.store.$emit(...n)}$on(...n){this.store.$on(...n)}}{}Object.assign(Si.prototype,{getPageAsyncComponent:ce,getLayoutAsyncComponent:ge,getAsyncComponent:de,getVueComponent:pe});var _i={install(n){const e=new Si;n.$vuepress=e,n.prototype.$vuepress=e}};function Pi(n,e){const t=e.toLowerCase();return n.options.routes.some(n=>n.path.toLowerCase()===t)}var qi={props:{pageKey:String,slotKey:{type:String,default:"default"}},render(n){const e=this.pageKey||this.$parent.$page.key;return me("pageKey",e),i.default.component(e)||i.default.component(e,ce(e)),i.default.component(e)?n(e):n("")}},Ii={functional:!0,props:{slotKey:String,required:!0},render:(n,{props:e,slots:t})=>n("div",{class:["content__"+e.slotKey]},t()[e.slotKey])},zi={computed:{openInNewWindowTitle(){return this.$themeLocaleConfig.openNewWindowText||"(opens new window)"}}},ji=(t(250),t(251),Object(lt.a)(zi,(function(){var n=this.$createElement,e=this._self._c||n;return e("span",[e("svg",{staticClass:"icon outbound",attrs:{xmlns:"http://www.w3.org/2000/svg","aria-hidden":"true",focusable:"false",x:"0px",y:"0px",viewBox:"0 0 100 100",width:"15",height:"15"}},[e("path",{attrs:{fill:"currentColor",d:"M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"}}),this._v(" "),e("polygon",{attrs:{fill:"currentColor",points:"45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"}})]),this._v(" "),e("span",{staticClass:"sr-only"},[this._v(this._s(this.openInNewWindowTitle))])])}),[],!1,null,null,null).exports),Ni={functional:!0,render(n,{parent:e,children:t}){if(e._isMounted)return t;e.$once("hook:mounted",()=>{e.$forceUpdate()})}};i.default.config.productionTip=!1,i.default.use(Wn),i.default.use(_i),i.default.mixin(function(n,e,t=i.default){!function(n){n.locales&&Object.keys(n.locales).forEach(e=>{n.locales[e].path=e});Object.freeze(n)}(e),t.$vuepress.$set("siteData",e);const a=new(n(t.$vuepress.$get("siteData"))),r=Object.getOwnPropertyDescriptors(Object.getPrototypeOf(a)),o={};return Object.keys(r).reduce((n,e)=>(e.startsWith("$")&&(n[e]=r[e].get),n),o),{computed:o}}(n=>class{setPage(n){this.__page=n}get $site(){return n}get $themeConfig(){return this.$site.themeConfig}get $frontmatter(){return this.$page.frontmatter}get $localeConfig(){const{locales:n={}}=this.$site;let e,t;for(const i in n)"/"===i?t=n[i]:0===this.$page.path.indexOf(i)&&(e=n[i]);return e||t||{}}get $siteTitle(){return this.$localeConfig.title||this.$site.title||""}get $canonicalUrl(){const{canonicalUrl:n}=this.$page.frontmatter;return"string"==typeof n&&n}get $title(){const n=this.$page,{metaTitle:e}=this.$page.frontmatter;if("string"==typeof e)return e;const t=this.$siteTitle,i=n.frontmatter.home?null:n.frontmatter.title||n.title;return t?i?i+" | "+t:t:i||"VuePress"}get $description(){const n=function(n){if(n){const e=n.filter(n=>"description"===n.name)[0];if(e)return e.content}}(this.$page.frontmatter.meta);return n||(this.$page.frontmatter.description||this.$localeConfig.description||this.$site.description||"")}get $lang(){return this.$page.frontmatter.lang||this.$localeConfig.lang||"en-US"}get $localePath(){return this.$localeConfig.path||"/"}get $themeLocaleConfig(){return(this.$site.themeConfig.locales||{})[this.$localePath]||{}}get $page(){return this.__page?this.__page:function(n,e){for(let t=0;t<n.length;t++){const i=n[t];if(i.path.toLowerCase()===e.toLowerCase())return i}return{path:"",frontmatter:{}}}(this.$site.pages,this.$route.path)}},dt)),i.default.component("Content",qi),i.default.component("ContentSlotsDistributor",Ii),i.default.component("OutboundLink",ji),i.default.component("ClientOnly",Ni),i.default.component("Layout",ge("Layout")),i.default.component("NotFound",ge("NotFound")),i.default.prototype.$withBase=function(n){const e=this.$site.base;return"/"===n.charAt(0)?e+n.slice(1):n},window.__VUEPRESS__={version:"1.9.5",hash:"8dd3af9"},async function(n){const e="undefined"!=typeof window&&window.__VUEPRESS_ROUTER_BASE__?window.__VUEPRESS_ROUTER_BASE__:dt.routerBase||dt.base,t=new Wn({base:e,mode:"history",fallback:!1,routes:gt,scrollBehavior:(n,e,t)=>t||(n.hash?!i.default.$vuepress.$get("disableScrollBehavior")&&{selector:decodeURIComponent(n.hash)}:{x:0,y:0})});!function(n){n.beforeEach((e,t,i)=>{if(Pi(n,e.path))i();else if(/(\/|\.html)$/.test(e.path))if(/\/$/.test(e.path)){const t=e.path.replace(/\/$/,"")+".html";Pi(n,t)?i(t):i()}else i();else{const t=e.path+"/",a=e.path+".html";Pi(n,a)?i(a):Pi(n,t)?i(t):i()}})}(t);const a={};try{await Promise.all(Ei.filter(n=>"function"==typeof n).map(e=>e({Vue:i.default,options:a,router:t,siteData:dt,isServer:n})))}catch(n){console.error(n)}return{app:new i.default(Object.assign(a,{router:t,render:n=>n("div",{attrs:{id:"app"}},[n("RouterView",{ref:"layout"}),n("div",{class:"global-ui"},Fi.map(e=>n(e)))])})),router:t}}(!1).then(({app:n,router:e})=>{e.onReady(()=>{n.$mount("#app")})})}]);